Index: tools/boot/wheezy/boot-kfreebsd
===================================================================
--- tools/boot/wheezy/boot-kfreebsd	(revision 2289)
+++ tools/boot/wheezy/boot-kfreebsd	(working copy)
@@ -42,6 +42,10 @@
 # Download boot images.
 BOOT_IMAGES="cdrom/debian-cd_info.tar.gz cdrom/kfreebsd.gz cdrom/kfreebsd-9.gz cdrom/initrd.gz"
 
+if [ "$ARCH" = kfreebsd-amd64 ]; then
+	BOOT_IMAGES="$BOOT_IMAGES cdrom/gtk/initrd.gz"
+fi
+
 for image in $BOOT_IMAGES; do
        if [ ! -e "$image" ]; then
                dir=$(dirname $image)
@@ -63,6 +67,11 @@
 cp "cdrom/kfreebsd-9.gz" "$CDDIR/boot/kernel/kfreebsd-9.gz"
 cp "cdrom/initrd.gz" "$CDDIR/boot/mfsroot.gz"
 
+if [ "$ARCH" = kfreebsd-amd64 ]; then
+	mkdir -p $CDDIR/boot/gtk/
+	cp "cdrom/gtk/initrd.gz" "$CDDIR/boot/gtk/mfsroot.gz"
+fi
+
 # Install bootloader
 mkdir -p boot$N
 tar -C boot$N -zxf cdrom/debian-cd_info.tar.gz
