Index: empathy-0.23.3/libempathy/empathy-tp-contact-list.c
===================================================================
--- empathy-0.23.3/libempathy/empathy-tp-contact-list.c
+++ empathy-0.23.3/libempathy/empathy-tp-contact-list.c
@@ -843,8 +843,11 @@
 
 	g_return_if_fail (EMPATHY_IS_TP_CONTACT_LIST (list));
 
+	if (priv->subscribe) {
 	empathy_tp_group_add_member (priv->subscribe, contact, message);
-	if (g_list_find (priv->pendings, contact)) {
+	}
+
+	if (priv->publish && g_list_find (priv->pendings, contact)) {
 		empathy_tp_group_add_member (priv->publish, contact, message);		
 	}
 }
@@ -858,8 +861,12 @@
 
 	g_return_if_fail (EMPATHY_IS_TP_CONTACT_LIST (list));
 
+	if (priv->subscribe) {
 	empathy_tp_group_remove_member (priv->subscribe, contact, message);
+	}
+	if (priv->publish) {
 	empathy_tp_group_remove_member (priv->publish, contact, message);		
+	}
 }
 
 static GList *
@@ -1000,7 +1007,9 @@
 	tp_group = tp_contact_list_get_group (EMPATHY_TP_CONTACT_LIST (list),
 					      group);
 
+	if (tp_group) {
 	empathy_tp_group_add_member (tp_group, contact, "");
+	}
 }
 
 static void
