diff -u bomberclone-0.11.8/debian/bomberclone.dirs bomberclone-0.11.8/debian/bomberclone.dirs
--- bomberclone-0.11.8/debian/bomberclone.dirs
+++ bomberclone-0.11.8/debian/bomberclone.dirs
@@ -2,0 +3 @@
+usr/share/applications
diff -u bomberclone-0.11.8/debian/bomberclone.install bomberclone-0.11.8/debian/bomberclone.install
--- bomberclone-0.11.8/debian/bomberclone.install
+++ bomberclone-0.11.8/debian/bomberclone.install
@@ -1,2 +1,3 @@
 debian/bomberclone.xpm usr/share/pixmaps/
+debian/bomberclone.desktop usr/share/applications/
 debian/tmp/usr/games usr/
only in patch2:
unchanged:
--- bomberclone-0.11.8.orig/debian/bomberclone.desktop
+++ bomberclone-0.11.8/debian/bomberclone.desktop
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Type=Application
+Name=bomberclone
+Comment=Play a Bomberman like game
+Icon=bomberclone
+Exec=bomberclone
+Terminal=false
+Categories=Game;
