From 81465392a4d0ad328be7d4641838ef34a4e14fa1 Mon Sep 17 00:00:00 2001
From: Hilko Bengen <bengen@debian.org>
Date: Thu, 6 Aug 2015 22:24:27 +0200
Subject: [PATCH] Add golang-golang-x-tools (stringer) to build dependencies

---
 debian/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index 832be68..f42d227 100644
--- a/debian/control
+++ b/debian/control
@@ -3,7 +3,7 @@ Section: devel
 Priority: extra
 Maintainer: Debian Go packaging team <pkg-go-maintainers@lists.alioth.debian.org>
 Uploaders: Martín Ferrari <tincho@debian.org>
-Build-Depends: debhelper (>= 9), dh-golang, golang-go
+Build-Depends: debhelper (>= 9), dh-golang, golang-go, golang-golang-x-tools
 Standards-Version: 3.9.6
 Homepage: https://godoc.org/golang.org/x/text
 Vcs-Git: git://anonscm.debian.org/pkg-go/packages/golang-x-text.git
-- 
2.5.0

