diff -Naur apt-0.7.20.orig/apt-pkg/indexcopy.cc apt-0.7.20/apt-pkg/indexcopy.cc
--- apt-0.7.20.orig/apt-pkg/indexcopy.cc	2009-01-12 09:30:45.000000000 +0200
+++ apt-0.7.20/apt-pkg/indexcopy.cc	2009-01-12 09:31:15.000000000 +0200
@@ -582,9 +582,10 @@
       string prefix = *I; 
 
       // a Release.gpg without a Release should never happen
-      if(!FileExists(*I+"Release"))
+      if(!FileExists(*I+"Release")) {
+         delete MetaIndex;
 	 continue;
-
+      }
 
       // verify the gpg signature of "Release"
       // gpg --verify "*I+Release.gpg", "*I+Release"
