--- a/debian/control
+++ b/debian/control
@@ -19,7 +19,7 @@ Build-Depends:
  libdbus-1-dev, libfontconfig1-dev, libfreetype6-dev, libgnutls-dev, 
  libijs-dev, libjpeg-dev, libldap2-dev, 
  libkrb5-dev,
- libpam0g-dev, libpaper-dev, libpng12-dev, libpoppler-dev (>= 0.12.2-2), 
+ libpam0g-dev, libpaper-dev, libpng12-dev,
  libslp-dev, libtiff4-dev, libtool, libusb-1.0-0-dev, 
  patch, pkg-config, po4a (>= 0.31~), po-debconf, poppler-utils, 
  sharutils, zlib1g-dev
