--- devscripts-2.9.27.orig/checkbashisms.pl	2006-11-29 18:39:43.000000000 +0000
+++ devscripts-2.9.27/checkbashisms.pl	2007-02-20 23:50:28.000000000 +0000
@@ -114,6 +114,8 @@
 		'\s\|\&' =>                    q<pipelining is not POSIX>,
 		'\$\[\w+\]' =>                 q<arithmetic not allowed>,
 		'\$\{\w+\:\d+(?::\d+)?\}' =>   q<${foo:3[:1]}>,
+		'\$\{!\w+[@*]\}' => q<${!prefix[*|@]>,
+		'\$\{!\w+\}' => q<${!name}>,
 		'\$\{\w+(/.+?){1,2}\}' =>      q<${parm/?/pat[/str]}>,
 		'[^\\\]\{([^\s]+?,)+[^\\\}\s]+\}' =>
 		                               q<brace expansion>,
