--- a/bin/BackupPC_archive
+++ b/bin/BackupPC_archive
@@ -299,7 +299,7 @@
     if ( $NeedPostCmd ) {
         UserCommandRun("ArchivePostUserCmd");
         if ( $? && $Conf{UserCmdCheckStatus} ) {
-            $stat{hostError} = "RestorePreUserCmd returned error status $?";
+            $stat{hostError} = "ArchivePostUserCmd returned error status $?";
             $stat{xferOK} = 0;
         }
     }
