diff -Nru /tmp/pd5cVOrkNp/developers-reference-3.3.8/debian/changelog /tmp/TJ57TwGEv5/developers-reference-3.3.9/debian/changelog
--- /tmp/pd5cVOrkNp/developers-reference-3.3.8/debian/changelog	2006-12-16 16:53:10.000000000 +0100
+++ /tmp/TJ57TwGEv5/developers-reference-3.3.9/debian/changelog	2007-02-08 07:03:07.000000000 +0100
@@ -1,3 +1,11 @@
+developers-reference (3.3.9) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Added section on team organization and Maintainer and Uploaders
+    fields. Closes: #xxxxxx.
+
+ -- Lucas Nussbaum <lucas@lucas-nussbaum.net>  Thu,  8 Feb 2007 07:01:58 +0100
+
 developers-reference (3.3.8) unstable; urgency=low
 
   * mia-history is replaced by mia-query. Thanks, Christoph Berg.
diff -Nru /tmp/pd5cVOrkNp/developers-reference-3.3.8/developers-reference.sgml /tmp/TJ57TwGEv5/developers-reference-3.3.9/developers-reference.sgml
--- /tmp/pd5cVOrkNp/developers-reference-3.3.8/developers-reference.sgml	2006-11-12 12:05:22.000000000 +0100
+++ /tmp/TJ57TwGEv5/developers-reference-3.3.9/developers-reference.sgml	2007-02-08 09:14:21.000000000 +0100
@@ -3320,7 +3320,8 @@
 Setup the co-maintainer with access to the sources you build the
 package from.  Generally this implies you are using a network-capable
 version control system, such as <prgn>CVS</prgn> or
-<prgn>Subversion</prgn>.</p>
+<prgn>Subversion</prgn>. Alioth (see <ref id="alioth">) provides such
+tools, amongst others.</p>
             </item>
             <item>
               <p>
@@ -3339,8 +3340,29 @@
             </item>
           </list></p>
 	<p>
-Collaborative maintenance can often be further eased by the use of
-tools on Alioth (see <ref id="alioth">).
+Another form of collaborative maintenance is team maintenance, which is
+recommended if you maintain several packages with the same group of
+developers. In that case, the Maintainer and Uploaders field of each
+package must be managed with care. It is recommended to choose between
+one of the two following schemes:
+<enumlist>
+  <item>
+    <p>
+Put the team member mainly responsible for the package in the Maintainer
+field. In the Uploaders, put the mailing list address, and the team members
+who care for the package.</p>
+  </item>
+  <item>
+    <p>
+Put the mailing list address in the Maintainer field. In the Uploaders
+field, put the team members who care for the package.</p>
+  </item>
+</enumlist>
+<p>
+In any case, it is a bad idea to automatically put all team members in
+the Uploaders field. It clutters the Developer's Package Overview listing
+(see <ref id="ddpo">) with packages one doesn't really care for, and
+creates a false sense of good maintenance.
       </sect>
 
     <sect id="testing">
