Index: b/docbook-xsl/manpages/other.xsl
===================================================================
--- a/docbook-xsl/manpages/other.xsl	2016-07-20 01:39:45.756863640 +0200
+++ b/docbook-xsl/manpages/other.xsl	2016-07-20 01:39:45.752864399 +0200
@@ -560,7 +560,14 @@
           <xsl:with-param name="message-prolog">Note: </xsl:with-param>
           <xsl:with-param name="message-epilog"> (soelim stub)</xsl:with-param>
           <xsl:with-param name="content">
-            <xsl:value-of select="'.so '"/>
+            <xsl:choose>
+              <xsl:when test="$man.output.in.separate.dir = 0">
+              <xsl:value-of select="concat('.so man', $section, '/')"/>
+              </xsl:when>
+              <xsl:otherwise>
+              <xsl:value-of select="'.so '"/> <!-- added case -->
+              </xsl:otherwise>
+              </xsl:choose>
             <xsl:variable name="full.filename">
               <xsl:call-template name="make.adjusted.man.filename">
                 <xsl:with-param name="name" select="$first.refname"/>
