From 639854690aa65031a072b8a362b609338955933b Mon Sep 17 00:00:00 2001
From: Josselin Mouette <joss@debian.org>
Date: Thu, 19 May 2011 20:04:36 +0200
Subject: [PATCH] Fix path to the GDM configuration file, which is different
 in Debian.

---
 src/daemon.c |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/src/daemon.c b/src/daemon.c
index a463c46..3b02b63 100644
--- a/src/daemon.c
+++ b/src/daemon.c
@@ -607,7 +607,7 @@ reload_autologin_timeout (Daemon *daemon)
         daemon->priv->autologin_id = 0;
 
         if (!load_autologin (daemon, &name, &enabled, &error)) {
-                g_debug ("failed to load gdms custom.conf: %s", error->message);
+                g_debug ("failed to load gdms daemon.conf: %s", error->message);
                 g_error_free (error);
                 g_free (name);
 
@@ -1352,7 +1352,7 @@ load_autologin (Daemon      *daemon,
         GError *local_error;
         gchar *string;
 
-        filename = "/etc/gdm/custom.conf";
+        filename = "/etc/gdm3/daemon.conf";
 
         keyfile = g_key_file_new ();
         if (!g_key_file_load_from_file (keyfile,
@@ -1402,7 +1402,7 @@ save_autologin (Daemon      *daemon,
         gchar *data;
         gboolean result;
 
-        filename = "/etc/gdm/custom.conf";
+        filename = "/etc/gdm3/daemon.conf";
 
         keyfile = g_key_file_new ();
         if (!g_key_file_load_from_file (keyfile,
-- 
1.7.5.1

