--- device2grub.orig	2009-02-19 16:57:30.000000000 +0100
+++ device2grub	2009-02-19 17:06:03.000000000 +0100
@@ -17,7 +17,7 @@
   $map{$olddevice} = $grubdevice;
 }
 
-$device=~ m#^(/dev/(?:[sh]d\D|ida/c\d*d\d*|cciss/c\d*d\d*))p*(\d*)$# || die "Can't match device: $device\n";
+$device=~ m#^(/dev/(?:[sh]d\D|i2o/hd\D|ida/c\d*d\d*|cciss/c\d*d\d*))p*(\d*)$# || die "Can't match device: $device\n";
 my ($disk,$partition) = ($1,$2);
 
 if ($map{$disk}) {
