diff --git a/debian/README.devel b/debian/README.devel
index e58279a..093ed88 100644
--- a/debian/README.devel
+++ b/debian/README.devel
@@ -39,6 +39,10 @@ we install these as separate packages:
         Claudio Cattazzo <claudio@linux.it>,
         debian-l10n-italian@lists.debian.org,
 
+** ja    (last update (as of 2017-11) 2016-12,
+         victory <victory.deb@gmail.com>,
+         Takuma Yamada <tyamada@takumayamada.com>).
+
 ** fr    (last update (as of 2008-09) 2008-08)
 
     Simon Paillard <simon.paillard@enst-bretagne.fr>
@@ -88,11 +92,6 @@ not in SVN, hopelessly out of date:
        http://bugs.debian.org/327764
        "RM: doc-debian-ko -- RoM; Orphaned, outdated" (Sep 2005)
 
-# ja   doc-debian-ja - Debian FAQ and other documents (Japanese)
-       http://bugs.debian.org/327663
-       "RM: doc-debian-ja -- RoQA; old, unused, very outdated documentation"
-       (Sep 2005)
-
 
 
 peeksheet
diff --git a/debian/changelog b/debian/changelog
index ea80a30..7d559b8 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -13,6 +13,8 @@ debian-faq (9.1) UNRELEASED; urgency=medium
   * Fix file server URL in control file, as mentioned in
     https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=892244#38.
     Closes: #892246, #892247.
+  * Activate package build for Japanese translation; translation is po-based.
+    (Closes: #xxxyyy)
 
   [ Jean-Philippe MENGUAL ]
   * Update French translation (reported as #920492)
diff --git a/debian/control b/debian/control
index d22cba6..c032b00 100644
--- a/debian/control
+++ b/debian/control
@@ -107,6 +107,24 @@ Description: Debian Frequently Asked Questions, in Italian
  This is the translation in Italian of the original English FAQ (available in
  the package debian-faq.)
 
+Package: debian-faq-ja
+Priority: optional
+Architecture: all
+Suggests: www-browser, postscript-viewer
+Depends: ${misc:Depends}
+Description: Debian Frequently Asked Questions, in Japanese
+ In this package you will find the Debian GNU/Linux FAQ, which gives
+ frequently asked questions (with their answers!) about the Debian distribution
+ (Debian GNU/Linux and others) and about the Debian project.
+ Some answers assume some knowledge of Unix-like operating systems.
+ However, as little prior knowledge as possible is assumed: answers to general
+ beginners questions will be kept simple.
+ .
+ The document is supplied in HTML, PDF, PostScript and plain text.
+ .
+ This is the translation in Japanese of the original English FAQ (available in
+ the package debian-faq.)
+
 Package: debian-faq-nl
 Priority: optional
 Architecture: all
