diff -rauN ../orig/devscripts-2.10.54/po4a/devscripts-po4a.conf ./devscripts-2.10.54/po4a/devscripts-po4a.conf
--- ../orig/devscripts-2.10.54/po4a/devscripts-po4a.conf	2009-07-22 13:54:25.000000000 +0200
+++ ./devscripts-2.10.54/po4a/devscripts-po4a.conf	2009-09-06 12:12:21.634025943 +0200
@@ -34,7 +34,7 @@
 [type:man] ../scripts/dcmd.1 \
 	fr:fr/dcmd.fr.1 add_fr:add_fr/translator_man.add
 [type:pod] ../scripts/dcontrol.pl \
-	fr:fr/dcontrol.fr.pl add_fr:add_fr/translator_man.add
+	fr:fr/dcontrol.fr.pl add_fr:add_fr/translator_pod.add
 [type:man] ../scripts/dd-list.1 \
 	fr:fr/dd-list.fr.1 add_fr:add_fr/translator_man.add
 [type:man] ../scripts/debc.1 \
diff -rauN ../orig/devscripts-2.10.54/po4a/Makefile ./devscripts-2.10.54/po4a/Makefile
--- ../orig/devscripts-2.10.54/po4a/Makefile	2007-02-22 18:09:19.000000000 +0100
+++ ./devscripts-2.10.54/po4a/Makefile	2009-09-06 12:12:44.847465702 +0200
@@ -1,4 +1,4 @@
 all:
-	po4a --no-backups devscripts-po4a.conf
+	po4a --previous --no-backups devscripts-po4a.conf
 clean:
-	po4a --rm-translations --no-backups devscripts-po4a.conf
+	po4a --previous --rm-translations --no-backups devscripts-po4a.conf
diff -rauN ../orig/devscripts-2.10.54/debian/control ./devscripts-2.10.54/debian/control
--- ../orig/devscripts-2.10.54/debian/control	2009-08-30 09:40:50.000000000 +0200
+++ ./devscripts-2.10.54/debian/control	2009-09-06 15:38:50.836405012 +0200
@@ -6,7 +6,7 @@
 Standards-Version: 3.8.2
 Build-Depends: perl, debhelper (>= 6.0.0), docbook-xsl, libtimedate-perl,
  libfile-desktopentry-perl, libterm-size-perl,  liburi-perl, libwww-perl,
- po4a (>= 0.26), xsltproc
+ po4a (>= 0.31), xsltproc
 Vcs-Browser: http://svn.debian.org/wsvn/devscripts/trunk/
 Vcs-Svn: svn://svn.debian.org/devscripts/trunk
 
diff -rauN ../orig/devscripts-2.10.54/scripts/licensecheck.pl ./devscripts-2.10.54/scripts/licensecheck.pl
--- ../orig/devscripts-2.10.54/scripts/licensecheck.pl	2009-07-21 14:09:15.000000000 +0200
+++ ./devscripts-2.10.54/scripts/licensecheck.pl	2009-09-06 12:13:12.290363224 +0200
@@ -79,6 +79,7 @@
 Also display copyright text found within the file
 
 =item B<--no-conf> B<--noconf>
+
 Do not read any configuration files. This can only be used as the first
 option given on the command-line.
 
