Index: debian-installer-utils/debian/changelog
===================================================================
--- debian-installer-utils/debian/changelog	(revision 64475)
+++ debian-installer-utils/debian/changelog	(working copy)
@@ -4,6 +4,10 @@
   * Only detect /dev/cd0 as a CD-ROM drive on GNU/kFreeBSD, to prevent
     multiple /etc/fstab entries.
 
+  [ Jeremie Koenig ]
+  * in-target: Check for /target/sys only if /sys/devices exists, to avoid
+    enforcing it on systems without a sysfs (ie. Hurd). Closes: #592684.
+
  -- Aurelien Jarno <aurel32@debian.org>  Sun, 22 Aug 2010 22:42:25 +0200
 
 debian-installer-utils (1.78) unstable; urgency=low
Index: debian-installer-utils/chroot-setup.sh
===================================================================
--- debian-installer-utils/chroot-setup.sh	(revision 64475)
+++ debian-installer-utils/chroot-setup.sh	(working copy)
@@ -10,7 +10,7 @@
 	   [ ! -d /target/proc ]; then
 		return 1
 	fi
-	if [ ! -d /target/sys ]; then
+	if [ -d /sys/devices ] && [ ! -d /target/sys ]; then
 		return 1
 	fi
 
