Index: debian-installer-utils/debian/changelog
===================================================================
--- debian-installer-utils/debian/changelog	(revision 64302)
+++ debian-installer-utils/debian/changelog	(working copy)
@@ -1,10 +1,14 @@
 debian-installer-utils (1.78) UNRELEASED; urgency=low
 
+  [ Otavio Salvador ]
   * When running in live-installer di-utils-reboot ought to quit
     installer if not rebooting/halting the machine. Closes: #589453.
 
- -- Otavio Salvador <otavio@ossystems.com.br>  Wed, 28 Jul 2010 04:07:41 -0300
+  [ Jeremie Koenig ]
+  * in-target: Don't require /sys on hurd. Closes: #<please fill-in>.
 
+ -- Jeremie Koenig <jk@jk.fr.eu.org>  Wed, 11 Aug 2010 20:35:54 +0000
+
 debian-installer-utils (1.77) unstable; urgency=low
 
   [ Updated translations ]
Index: debian-installer-utils/chroot-setup.sh
===================================================================
--- debian-installer-utils/chroot-setup.sh	(revision 64302)
+++ debian-installer-utils/chroot-setup.sh	(working copy)
@@ -10,7 +10,7 @@
 	   [ ! -d /target/proc ]; then
 		return 1
 	fi
-	if [ ! -d /target/sys ]; then
+	if [ ! -d /target/sys ] && [ "$(udpkg --print-os)" != hurd ]; then
 		return 1
 	fi
 
