diff --git a/dpkg/postinst b/dpkg/postinst
index 6a1b7fe..7357325 100644
--- a/dpkg/postinst
+++ b/dpkg/postinst
@@ -169,7 +169,7 @@ dbc_go(){
 				dbc_logline "creating database backup in $dumpfile"
 				# backup before we upgrade
 				_dbc_asuser=""
-				$dbc_dump_cmd $dumpfile
+				$dbc_dump_cmd $dumpfile || dbc_upgrade_error "backing up the old database"
 			fi
 
 			# now perform the updates
