diff -ur cachefilesd-0.9,org/cachefilesd.c cachefilesd-0.9,new/cachefilesd.c
--- cachefilesd-0.9,org/cachefilesd.c	2007-08-09 17:31:39.000000000 +0200
+++ cachefilesd-0.9,new/cachefilesd.c	2011-04-01 08:11:12.058804251 +0200
@@ -603,7 +603,7 @@
 		if (jumpstart_scan) {
 			jumpstart_scan = 0;
 			if (!stop && !scan) {
-				notice("Refilling cull table");
+				debug(1, "Refilling cull table");
 				root.usage++;
 				scan = &root;
 			}
@@ -1307,7 +1307,7 @@
 
 	scan = curr->parent;
 	if (!scan) {
-		info("Scan complete");
+		debug(1, "Scan complete");
 		decant_cull_table();
 	}
 
@@ -1485,7 +1485,7 @@
 	if (!scan && oldest_build <= culltable_size / 2 + 2) {
 		decant_cull_table();
 
-		notice("Refilling cull table");
+		debug(1, "Refilling cull table");
 		root.usage++;
 		scan = &root;
 	}
