Index: common.ent
===================================================================
--- common.ent	(revisione 8928)
+++ common.ent	(copia locale)
@@ -18,7 +18,7 @@
 <!-- standard information -->
 <!ENTITY fsf-addr "Free Software Foundation, Inc.,
   51 Franklin Street, Fifth Floor, Boston, MA 02110-1301, USA">
-<!ENTITY file-GPL "<filename>/usr/share/common-licenses/GPL</filename>">
+<!ENTITY file-GPL "<filename>/usr/share/common-licenses/GPL-2</filename>">
 <!ENTITY debian-formal "Debian GNU/Linux">
 
 <!--
