diff --git a/lib/debian/watch.ex b/lib/debian/watch.ex
index fbe779d..71e4136 100644
--- a/lib/debian/watch.ex
+++ b/lib/debian/watch.ex
@@ -20,7 +20,7 @@ version=4
 # http://sf.net/#PACKAGE#/ #PACKAGE#-(.*)\.tar\.gz debian uupdate
 
 # GitHub hosted projects
-#opts="filenamemangle="s%(?:.*?)?v?(\d[\d.]*)\.tar\.gz%<project>-$1.tar.gz%" \
+#opts="filenamemangle=s%(?:.*?)?v?(\d[\d.]*)\.tar\.gz%<project>-$1.tar.gz%" \
 #   https://github.com/<user>/#PACKAGE#/tags \
 #   (?:.*?/)?v?(\d[\d.]*)\.tar\.gz debian uupdate
 
