diff -ur dhelp-0.5.15.orig/debian/postinst dhelp-0.5.15/debian/postinst
--- dhelp-0.5.15.orig/debian/postinst	2003-01-18 20:09:20.000000000 +0100
+++ dhelp-0.5.15/debian/postinst	2004-03-01 16:14:36.000000000 +0100
@@ -25,6 +25,11 @@
     mkdir /etc/dhelp
 fi
 
+# create /var/lib/dhelp if necessary
+if [ ! -d /var/lib/dhelp ]; then
+    mkdir /var/lib/dhelp
+fi
+
 # store debconf configuration values in files there
 db_get dhelp/www-browser-console 
 echo $RET > /etc/dhelp/www-browser-console
