diff -urN dpkg-1.15.8.10.orig/scripts/Dpkg/Source/Patch.pm dpkg-1.15.8.10/scripts/Dpkg/Source/Patch.pm
--- dpkg-1.15.8.10.orig/scripts/Dpkg/Source/Patch.pm	2011-01-30 20:37:44.000000000 +0100
+++ dpkg-1.15.8.10/scripts/Dpkg/Source/Patch.pm	2011-02-08 17:41:32.000000000 +0100
@@ -113,7 +113,7 @@
             last;
         } elsif (m/^[-+\@ ]/) {
             $difflinefound++;
-        } elsif (m/^\\ No newline at end of file$/) {
+        } elsif (m/^\\/) {
             warning(_g("file %s has no final newline (either " .
                        "original or modified version)"), $new);
         } else {
@@ -417,7 +417,7 @@
 	my $hunk = 0;
 	while (defined($_ = getline($self))) {
 	    # read hunk header (@@)
-	    next if /^\\ No newline/;
+	    next if /^\\/;
 	    last unless (/^@@ -\d+(,(\d+))? \+\d+(,(\d+))? @\@( .*)?$/);
 	    my ($olines, $nlines) = ($1 ? $2 : 1, $3 ? $4 : 1);
 	    # read hunk
@@ -430,7 +430,7 @@
                         error(_g("unexpected end of diff `%s'"), $diff);
                     }
 		}
-		next if /^\\ No newline/;
+		next if /^\\/;
 		# Check stats
 		if    (/^ / || /^$/)  { --$olines; --$nlines; }
 		elsif (/^-/)  { --$olines; }
