=== modified file '1/rules/simple-patchsys.mk.in'
--- 1/rules/simple-patchsys.mk.in	2008-03-28 16:04:32 +0000
+++ 1/rules/simple-patchsys.mk.in	2008-08-08 01:19:14 +0000
@@ -59,7 +59,7 @@
 
 post-patches:: apply-patches
 
-cleanbuilddir:: reverse-patches
+clean:: reverse-patches
 	for dir in $(DEB_PATCHDIRS) ; do \
 	    rm -f $$dir/*.log ; \
 	done

=== modified file 'debian/changelog'
--- debian/changelog	2008-08-01 18:44:48 +0000
+++ debian/changelog	2008-08-08 01:19:14 +0000
@@ -1,5 +1,10 @@
 cdbs (0.4.53) UNRELEASED; urgency=low
 
+  * 1/rules/simple-patchsys.mk.in: revert the change for Debian bug #387103
+    that causes patches to be unapplied before cleaning the build directory;
+    the original issue was addressed more thoroughly in 0.4.49 with a fix
+    for Debian bug #424080, and unpatch-before-clean breaks some relevant
+    use cases that involve patching of build systems.
   * Relax python-central build-dependency to >= 0.5.6 in python-distutils
     class (instruction in python policy is only really needed for
     transition of python-central itself, which has now completed).
