--- /lib/cryptsetup/cryptdisks.functions	2012-11-07 15:28:47.000000000 +0000
+++ /lib/cryptsetup/cryptdisks.functions	2013-07-11 00:31:04.766286868 +0100
@@ -38,6 +38,7 @@
 	TMPFS=""
 	MAKESWAP=""
 	USELUKS=""
+	LUKSHEADER=""
 	KEYSCRIPT=""
 	IGNORE=""
 	CRYPTTAB_OPTIONS=""
@@ -147,6 +148,10 @@
 		luks)
 			USELUKS="yes"
 			;;
+		header)
+			PARAMS="$PARAMS --header=$VALUE"
+			LUKSHEADER="$VALUE"
+			;;
 		noearly)
 			if [ "$INITSTATE" = "early" ]; then
 				IGNORE="yes"
@@ -251,7 +256,7 @@
 	tried=0
 	keyscriptarg=""
 
-	if ! cryptsetup isLuks "$src" >/dev/null 2>&1; then
+	if ! cryptsetup isLuks "${LUKSHEADER:-$src}" >/dev/null 2>&1; then
 		log_warning_msg "$dst: device '$src' is not a LUKS partition, skipping"
 		return 1
 	fi
