? 186819.patch
? ChangeLog
Index: examples/rules
===================================================================
RCS file: /home/cvs/repository/joey-cvs/public/packages/debhelper/examples/rules,v
retrieving revision 1.18
diff -u -r1.18 rules
--- examples/rules	9 Dec 2002 17:21:33 -0000	1.18
+++ examples/rules	31 Mar 2003 04:58:04 -0000
@@ -50,6 +50,7 @@
 #	dh_installdebconf	
 #	dh_installlogrotate
 #	dh_installemacsen
+#	dh_installcatalogs
 #	dh_installpam
 #	dh_installmime
 #	dh_installinit
Index: examples/rules.indep
===================================================================
RCS file: /home/cvs/repository/joey-cvs/public/packages/debhelper/examples/rules.indep,v
retrieving revision 1.18
diff -u -r1.18 rules.indep
--- examples/rules.indep	9 Dec 2002 17:21:33 -0000	1.18
+++ examples/rules.indep	31 Mar 2003 04:58:04 -0000
@@ -47,6 +47,7 @@
 #	dh_installdebconf
 #	dh_installlogrotate
 #	dh_installemacsen
+#	dh_installcatalogs
 #	dh_installpam
 #	dh_installmime
 #	dh_installinit
Index: examples/rules.multi
===================================================================
RCS file: /home/cvs/repository/joey-cvs/public/packages/debhelper/examples/rules.multi,v
retrieving revision 1.23
diff -u -r1.23 rules.multi
--- examples/rules.multi	9 Dec 2002 17:21:33 -0000	1.23
+++ examples/rules.multi	31 Mar 2003 04:58:04 -0000
@@ -52,6 +52,7 @@
 #	dh_installdebconf -i
 #	dh_installlogrotate -i
 #	dh_installemacsen -i
+#	dh_installcatalogs -i
 #	dh_installpam -i
 #	dh_installmime -i
 #	dh_installinit -i
@@ -80,6 +81,7 @@
 #	dh_installdebconf -a
 #	dh_installlogrotate -a
 #	dh_installemacsen -a
+#	dh_installcatalogs -a
 #	dh_installpam -a
 #	dh_installmime -a
 #	dh_installinit -a
Index: examples/rules.multi2
===================================================================
RCS file: /home/cvs/repository/joey-cvs/public/packages/debhelper/examples/rules.multi2,v
retrieving revision 1.25
diff -u -r1.25 rules.multi2
--- examples/rules.multi2	19 Dec 2002 02:27:35 -0000	1.25
+++ examples/rules.multi2	31 Mar 2003 04:58:04 -0000
@@ -57,6 +57,7 @@
 #	dh_installdebconf
 #	dh_installlogrotate
 #	dh_installemacsen
+#	dh_installcatalogs
 #	dh_installpam
 #	dh_installmime
 #	dh_installinit
