--- dh_cligacpolicy.orig	2012-12-03 23:47:23.000000000 +0700
+++ dh_cligacpolicy	2012-12-03 23:48:54.772722367 +0700
@@ -87,7 +87,8 @@
   open(FILE, 'debian/control');
   my @filedata = <FILE>;
   close(FILE);
-  if (!($filedata[0] =~ /Build-Depends(-Indep)?: .*cli-common-dev \(>= 0\.5\.7\)/)) {
+  if ($filedata[0] =~ m/Build-Depends(?:\-Indep)?\:(?:.*\n\s+)*.*cli\-common\-dev\s*\(>=\s*([^\)]+)\)/ &&
+      system("dpkg", "--compare-versions", $1, ">=", "0.5.7") != 0) {
       warning("Warning! No Build-Depends(-Indep) on cli-common-dev (>= 0.5.7)!");
   }
 }

