--- /usr/share/docker.io/contrib/mkimage-debootstrap.sh	2014-02-05 01:08:57.000000000 +0100
+++ mkimage-debootstrap.sh	2014-03-05 17:14:25.566531185 +0100
@@ -87,7 +87,11 @@
 
 # some rudimentary detection for whether we need to "sudo" our docker calls
 docker=''
-if docker version > /dev/null 2>&1; then
+if docker.io version > /dev/null 2>&1; then
+	docker='docker.io'
+elif sudo docker.io version > /dev/null 2>&1; then
+	docker='sudo docker.io'
+elif docker version > /dev/null 2>&1; then
 	docker='docker'
 elif sudo docker version > /dev/null 2>&1; then
 	docker='sudo docker'
