diff -urN cron-3.0pl1/do_command.c cron-3.0pl1-fixed//do_command.c
--- cron-3.0pl1/do_command.c	2010-05-02 14:08:07.000000000 +0200
+++ cron-3.0pl1-fixed//do_command.c	2010-05-10 12:17:36.000000000 +0200
@@ -517,7 +517,7 @@
 	if (mailto == NULL)
 		mailto = usernm;
 	else if (!*mailto)
-		mailto = NULL;
+		goto mail_finished;
 
 	/* Don't send mail if MAILCMD is not available */
 	if ((statret = stat(MAILCMD, &mcsb)) != 0) {
