diff -Nru exim4-4.80/debian/rules exim4-4.80/debian/rules
--- exim4-4.80/debian/rules	2012-06-03 16:28:53.000000000 +0200
+++ exim4-4.80/debian/rules	2012-09-14 17:11:25.000000000 +0200
@@ -22,6 +22,8 @@
 endif
 
 CFLAGS := $(CFLAGS) $(shell getconf LFS_CFLAGS) -D_LARGEFILE_SOURCE -fno-strict-aliasing -Wall
+# The build system ignores CPPFLAGS, append them to CFLAGS.
+CFLAGS += $(CPPFLAGS)
 export CFLAGS
 
 LC_ALL=C
