Description: Add missing $(FE) to Makefiles.
Author: Simon Ruderich <simon@ruderich.org>
Last-Update: 2012-09-14

Index: exim4-4.80/src/transports/Makefile
===================================================================
--- exim4-4.80.orig/src/transports/Makefile	2012-09-14 17:07:28.110489005 +0200
+++ exim4-4.80/src/transports/Makefile	2012-09-14 17:07:34.374489244 +0200
@@ -7,7 +7,7 @@
 transports.a:    $(OBJ)
 		 @$(RM_COMMAND) -f transports.a
 		 @echo "$(AR) transports.a"
-		 @$(AR) transports.a $(OBJ)
+		 $(FE)$(AR) transports.a $(OBJ)
 		 $(RANLIB) $@
 
 .SUFFIXES:       .o .c
Index: exim4-4.80/src/routers/Makefile
===================================================================
--- exim4-4.80.orig/src/routers/Makefile	2012-09-14 17:07:28.110489005 +0200
+++ exim4-4.80/src/routers/Makefile	2012-09-14 17:07:34.374489244 +0200
@@ -14,7 +14,7 @@
 routers.a:       $(OBJ)
 		 @$(RM_COMMAND) -f routers.a
 		 @echo "$(AR) routers.a"
-		 @$(AR) routers.a $(OBJ)
+		 $(FE)$(AR) routers.a $(OBJ)
 		 $(RANLIB) $@
 
 .SUFFIXES:       .o .c
Index: exim4-4.80/src/lookups/Makefile
===================================================================
--- exim4-4.80.orig/src/lookups/Makefile	2012-09-14 17:07:28.110489005 +0200
+++ exim4-4.80/src/lookups/Makefile	2012-09-14 17:07:34.374489244 +0200
@@ -14,7 +14,7 @@
 lookups.a:       $(OBJ)
 		 @$(RM_COMMAND) -f lookups.a
 		 @echo "$(AR) lookups.a"
-		 @$(AR) lookups.a $(OBJ)
+		 $(FE)$(AR) lookups.a $(OBJ)
 		 $(RANLIB) $@
 
 .SUFFIXES:       .o .c .so
