Index: cairo-1.10.2/configure.ac
===================================================================
--- cairo-1.10.2.orig/configure.ac	2010-12-31 12:09:19.000000000 +0100
+++ cairo-1.10.2/configure.ac	2010-12-31 12:49:01.000000000 +0100
@@ -709,7 +709,7 @@
 dnl that are known to work.
 
 case $host in
-*-linux*|*-*bsd*|*-solaris*|*-*-darwin*|*-dragonfly*)
+*-linux*|*-*bsd*|*-solaris*|*-*-darwin*|*-dragonfly*|*-*-gnu*)
 	have_ld_preload="yes"
 	;;
 *)
