Index: /home/elmig/debian/debian-installer/packages/kernel/linux-kernel-di-powerpc-2.6/modules/powerpc-powerpc64/nic-extra-modules
===================================================================
--- /home/elmig/debian/debian-installer/packages/kernel/linux-kernel-di-powerpc-2.6/modules/powerpc-powerpc64/nic-extra-modules	(revisão 65306)
+++ /home/elmig/debian/debian-installer/packages/kernel/linux-kernel-di-powerpc-2.6/modules/powerpc-powerpc64/nic-extra-modules	(cópia de trabalho)
@@ -21,3 +21,4 @@
 p54usb
 yellowfin
 ps3_gelic
+spidernet
