diff -Nru emacsen-common-1.4.19/debian/changelog emacsen-common-1.4.19+nmu1/debian/changelog
--- emacsen-common-1.4.19/debian/changelog	2009-05-18 00:42:24.000000000 +0200
+++ emacsen-common-1.4.19+nmu1/debian/changelog	2010-11-28 23:08:26.000000000 +0100
@@ -1,3 +1,11 @@
+emacsen-common (1.4.19+nmu1) unstable; urgency=high
+
+  * Non-maintainer upload.
+  * Don't call install-info in the postinst (Closes: #604164). Setting
+    urgency to high due to RC bug fix.
+
+ -- Mehdi Dogguy <mehdi@debian.org>  Sun, 28 Nov 2010 23:04:54 +0100
+
 emacsen-common (1.4.19) unstable; urgency=low
 
   * Move the sample package install/remove scripts from
diff -Nru emacsen-common-1.4.19/debian/postinst emacsen-common-1.4.19+nmu1/debian/postinst
--- emacsen-common-1.4.19/debian/postinst	2009-02-13 07:18:08.000000000 +0100
+++ emacsen-common-1.4.19+nmu1/debian/postinst	2010-11-28 23:08:57.000000000 +0100
@@ -39,7 +39,4 @@
 
 /usr/lib/emacsen-common/emacs-package-install emacsen-common
 
-# Clean up dead emacs info entry...
-install-info --remove --quiet emacs
-
 exit 0
