--- advi-1.4.0/debian/control	2003-12-14 00:31:37.000000000 +0100
+++ advi-1.4.0.old/debian/control	2003-12-14 00:25:36.000000000 +0100
@@ -2,7 +2,7 @@
 Section: tex
 Priority: optional
 Maintainer: Sven Luther <luther@debian.org>
-Build-Depends: debhelper (>> 3.0.0), tetex-bin, ocaml, libncurses5-dev, xlibs-dev, libcamlimages-ocaml-dev, tetex-extra, zlib1g-dev, gs
+Build-Depends: debhelper (>> 3.0.0), tetex-bin, ocaml-3.06-1, libncurses5-dev, xlibs-dev, libcamlimages-ocaml-dev, tetex-extra, zlib1g-dev, gs
 Standards-Version: 3.5.8
 
 Package: advi
