diff -u exiv2-0.18.2/debian/control exiv2-0.18.2/debian/control
--- exiv2-0.18.2/debian/control
+++ exiv2-0.18.2/debian/control
@@ -67,0 +68,6 @@
+
+Package: libexiv2-dbg
+Depends: ${misc:Depends}
+Section: debug
+Description: EXIF/IPTC metadata manipulation library - debug
+ debug package
diff -u exiv2-0.18.2/debian/changelog exiv2-0.18.2/debian/changelog
--- exiv2-0.18.2/debian/changelog
+++ exiv2-0.18.2/debian/changelog
@@ -1,3 +1,9 @@
+exiv2 (0.18.2-2) unstable; urgency=low
+
+  * Add -dbg package.
+
+ -- Michal Čihař <nijel@debian.org>  Thu, 19 Nov 2009 11:49:14 +0100
+
 exiv2 (0.18.2-1) unstable; urgency=low
 
   * New upstream release
diff -u exiv2-0.18.2/debian/rules exiv2-0.18.2/debian/rules
--- exiv2-0.18.2/debian/rules
+++ exiv2-0.18.2/debian/rules
@@ -7,6 +7,9 @@
 %:
 	dh --with quilt $@
 
+override_dh_strip:
+	dh_strip -a --dbg-package=libexiv2-dbg
+
 override_dh_auto_configure:
 	dh_auto_configure -- --disable-rpath
 
only in patch2:
unchanged:
