diff -rN -u old-aptitude/po/Makevars new-aptitude/po/Makevars
--- old-aptitude/po/Makevars	2006-02-05 18:49:58.000000000 +0100
+++ new-aptitude/po/Makevars	2006-02-05 18:50:01.000000000 +0100
@@ -18,8 +18,6 @@
 # or entity, or to disclaim their copyright.  The empty string stands for
 # the public domain; in this case the translators are expected to disclaim
 # their copyright.
-#
-# (is that correct for aptitude?)
 COPYRIGHT_HOLDER = Daniel Burrows <dburrows@debian.org>
 
 # This is the email address or URL to which the translators shall report
