diff -rN -u old-aptitude/src/ui.cc new-aptitude/src/ui.cc
--- old-aptitude/src/ui.cc	2006-02-05 18:50:00.000000000 +0100
+++ new-aptitude/src/ui.cc	2006-02-05 18:50:02.000000000 +0100
@@ -881,15 +881,17 @@
 {
   char buf[512];
 
-  snprintf(buf, 512, HELPDIR "/%s", _("README")); // README can be translated..
+  snprintf(buf, 512, HELPDIR "/%s", _("README")); // README can be translated...
 
-  const char *encoding=P_("Encoding of README|UTF-8");
+  const char *encoding=P_("Encoding of README|ISO_8859-1");
 
   // Deal with missing localized docs.
   if(access(buf, R_OK)!=0)
     {
+    // encoding is latin1=ISO_8859-1, right?
+    // cannot open README in a German locale, why?
       strncpy(buf, HELPDIR "/README", 512);
-      encoding="UTF-8";
+      encoding="ISO_8859-1";
     }
 
   vs_table_ref t      = vs_table::create();
