--- emacspeak-29.0/debian/changelog
+++ emacspeak-29.0/debian/changelog
@@ -1,3 +1,13 @@
+emacspeak (29.0-1.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * debian/templates:
+     - Default shared/emacspeak/port to “none” (Closes: #522629). This
+       way, an empty string is no longer the default, allowing for a
+       noninteractive installation.
+
+ -- Cyril Brulebois <kibi@debian.org>  Thu, 16 Apr 2009 05:47:30 +0200
+
 emacspeak (29.0-1) unstable; urgency=low
 
   * New upstream release
--- emacspeak-29.0/debian/templates
+++ emacspeak-29.0/debian/templates
@@ -37,6 +37,7 @@
  '/dev/ttyUSB0'.
  .
  If you use a software method to generate speech, please enter 'none'.
+Default: none
 
 Template: shared/emacspeak/invalidport
 Type: error
