--- ca-certificates-java.postinst.orig	2009-10-04 12:46:17.000000000 +0200
+++ ca-certificates-java.postinst	2009-10-04 12:56:46.000000000 +0200
@@ -63,7 +63,7 @@
 		      echo "  added certificate $pem"
 		  elif grep -q 'Signature not available' $log; then
 		      echo "  ignored import, signature not available: ${line#+*}"
-		      cat $log
+		      sed -e 's/^/   -> /' $log
 		  else
 		      echo >&2 "  error adding ${line#+*}"
 		      errors=$(expr $errors + 1)
