--- ca-certificates-java.postinst.orig	2009-10-04 12:46:17.000000000 +0200
+++ ca-certificates-java.postinst	2009-10-04 12:46:22.000000000 +0200
@@ -62,8 +62,8 @@
 		  then
 		      echo "  added certificate $pem"
 		  elif grep -q 'Signature not available' $log; then
+		      cat $log
 		      echo "  ignored import, signature not available: ${line#+*}"
-		      cat $log
 		  else
 		      echo >&2 "  error adding ${line#+*}"
 		      errors=$(expr $errors + 1)
