diff --git a/debian/bustle-pcap.install b/debian/bustle-pcap.install
new file mode 100644
index 0000000..242724b
--- /dev/null
+++ b/debian/bustle-pcap.install
@@ -0,0 +1 @@
+usr/bin/bustle-pcap
diff --git a/debian/bustle.install b/debian/bustle.install
new file mode 100644
index 0000000..f73f493
--- /dev/null
+++ b/debian/bustle.install
@@ -0,0 +1,2 @@
+usr/share/*
+usr/bin/bustle
diff --git a/debian/changelog b/debian/changelog
index 4c751f4..4870212 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,9 @@
+bustle (0.4.7-2+ted1) UNRELEASED; urgency=medium
+
+  * d/control: Split out bustle-pcap into its own binary package
+
+ -- Ted Gould <ted@gould.cx>  Fri, 05 Dec 2014 09:02:48 -0600
+
 bustle (0.4.7-2) unstable; urgency=medium
 
   * d/control: really build depend on intltool
diff --git a/debian/control b/debian/control
index e694cf2..710cb84 100644
--- a/debian/control
+++ b/debian/control
@@ -29,6 +29,7 @@ Package: bustle
 Architecture: any
 Depends: ${misc:Depends}, ${shlibs:Depends}
 Suggests: graphviz
+Recommends: bustle-pcap
 Description: D-Bus activity visualiser
  Bustle is a tool to chart and provide timing information of D-Bus
  calls for profiling and debugging purposes. It is intended to replace
@@ -39,4 +40,16 @@ Description: D-Bus activity visualiser
  data in Graphviz format.
  .
  This package contains the graphical visualizer for traces generated
- with the bustle-pcap tool.
+ with the bustle-pcap tool in the bustle-pcap package.
+
+Package: bustle-pcap
+Architecture: any
+Depends: ${misc:Depends}, ${shlibs:Depends}
+Conflicts: bustle (<= 0.4.7-2)
+Description: D-Bus traffic capture tool for the pcap format
+ Bustle is a tool to chart and provide timing information of D-Bus
+ calls for profiling and debugging purposes. It is intended to replace
+ reading the cryptic output of dbus-monitor.
+ .
+ This package contains the capture tool which will capture the D-Bus
+ traffic into a pcap file that can be visualized using Bustle.
diff --git a/debian/rules b/debian/rules
index 34881fb..6c84f82 100755
--- a/debian/rules
+++ b/debian/rules
@@ -21,5 +21,5 @@ override_dh_auto_build:
 	./setup build
 
 override_dh_install:
-	./setup copy --destdir debian/bustle
+	./setup copy --destdir debian/tmp
 	dh_install
