diff -u ax25-tools-0.0.10-rc2+cvs20120204/hdlcutil/fl/Makefile.in ax25-tools-0.0.10-rc2+cvs20120204/hdlcutil/fl/Makefile.in
--- ax25-tools-0.0.10-rc2+cvs20120204/hdlcutil/fl/Makefile.in
+++ ax25-tools-0.0.10-rc2+cvs20120204/hdlcutil/fl/Makefile.in
@@ -50,16 +50,16 @@
 am_xfhdlcchpar_OBJECTS = xfhdlcchpar.$(OBJEXT) \
 	xfhdlcchpar_main.$(OBJEXT)
 xfhdlcchpar_OBJECTS = $(am_xfhdlcchpar_OBJECTS)
-xfhdlcchpar_LDADD = $(LDADD)
+xfhdlcchpar_LDADD = $(LDADD) ../hdrvcomm.o $(X_LIBS) -lX11 $(FLTK_LIB)
 am_xfhdlcst_OBJECTS = xfhdlcst.$(OBJEXT) xfhdlcst_main.$(OBJEXT)
 xfhdlcst_OBJECTS = $(am_xfhdlcst_OBJECTS)
-xfhdlcst_LDADD = $(LDADD)
+xfhdlcst_LDADD = $(LDADD) ../hdrvcomm.o $(X_LIBS) -lX11 $(FLTK_LIB)
 am_xfsmdiag_OBJECTS = xfsmdiag.$(OBJEXT) xfsmdiag_main.$(OBJEXT)
 xfsmdiag_OBJECTS = $(am_xfsmdiag_OBJECTS)
-xfsmdiag_LDADD = $(LDADD)
+xfsmdiag_LDADD = $(LDADD) ../hdrvcomm.o $(X_LIBS) -lX11 $(FLTK_LIB)
 am_xfsmmixer_OBJECTS = xfsmmixer.$(OBJEXT) xfsmmixer_main.$(OBJEXT)
 xfsmmixer_OBJECTS = $(am_xfsmmixer_OBJECTS)
-xfsmmixer_LDADD = $(LDADD)
+xfsmmixer_LDADD = $(LDADD) ../hdrvcomm.o $(X_LIBS) -lX11 $(FLTK_LIB)
 DEFAULT_INCLUDES = -I.@am__isrc@ -I$(top_builddir)
 depcomp = $(SHELL) $(top_srcdir)/depcomp
 am__depfiles_maybe = depfiles
@@ -184,8 +184,6 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 AM_CPPFLAGS = $(X_CFLAGS) -I$(srcdir)/..
-LDDADD = ../hdrvcomm.o
-AM_LDFLAGS = $(X_LIBS) -lX11 $(FLTK_LIB) $(LDDADD)
 xfsmmixer_SOURCES = xfsmmixer.cxx xfsmmixer_main.cxx xfsmmixer.h
 xfhdlcchpar_SOURCES = xfhdlcchpar.cxx xfhdlcchpar_main.cxx xfhdlcchpar.h
 xfhdlcst_SOURCES = xfhdlcst.cxx xfhdlcst_main.cxx xfhdlcst.h
--- ax25-tools-0.0.10-rc2+cvs20120204.orig/hdlcutil/fl/Makefile.am
+++ ax25-tools-0.0.10-rc2+cvs20120204/hdlcutil/fl/Makefile.am
@@ -4,12 +4,15 @@
 endif
 
 AM_CPPFLAGS = $(X_CFLAGS) -I$(srcdir)/..
-LDDADD = ../hdrvcomm.o
-AM_LDFLAGS = $(X_LIBS) -lX11 $(FLTK_LIB) $(LDDADD)
 
 xfsmmixer_SOURCES = xfsmmixer.cxx xfsmmixer_main.cxx xfsmmixer.h
 xfhdlcchpar_SOURCES = xfhdlcchpar.cxx xfhdlcchpar_main.cxx xfhdlcchpar.h
 xfhdlcst_SOURCES = xfhdlcst.cxx xfhdlcst_main.cxx xfhdlcst.h
 xfsmdiag_SOURCES = xfsmdiag.cxx xfsmdiag_main.cxx xfsmdiag.h xfsmdiag_main.h
 
+xfsmmixer_LDADD = ../hdrvcomm.o $(X_LIBS) -lX11 $(FLTK_LIB)
+xfhdlcchpar_LDADD = ../hdrvcomm.o $(X_LIBS) -lX11 $(FLTK_LIB)
+xfhdlcst_LDADD = ../hdrvcomm.o $(X_LIBS) -lX11 $(FLTK_LIB)
+xfsmdiag_LDADD = ../hdrvcomm.o $(X_LIBS) -lX11 $(FLTK_LIB)
+
 hdrvcomm.o: ../hdrvcomm.o
