---

 src/server.c |   19 ++++++++++---------
 1 files changed, 10 insertions(+), 9 deletions(-)

diff -puN src/server.c~cvs-bug-fix src/server.c
--- cvs-1.12.5/src/server.c~cvs-bug-fix	2004-03-15 17:13:06.000000000 +0900
+++ cvs-1.12.5-hirofumi/src/server.c	2004-03-15 17:13:25.000000000 +0900
@@ -2753,6 +2753,7 @@ error  \n");
 	    TRACE (TRACE_DATA, "CVS_SERVER_SLEEP not set.");
 
 	exitstatus = (*command) (argument_count, argument_vector);
+	fflush (NULL);
 
 	/* Output any partial lines.  If the client doesn't support
 	   "MT", we go ahead and just tack on a newline since the
@@ -2772,16 +2773,16 @@ error  \n");
 	 * the parent.
 	 */
 	buf_free (protocol);
-    close (STDIN_FILENO);
-    close (STDERR_FILENO);
-    close (STDOUT_FILENO);
-    close (protocol_pipe[1]);
+	close (STDIN_FILENO);
+	close (STDERR_FILENO);
+	close (STDOUT_FILENO);
+	close (protocol_pipe[1]);
 #ifdef SERVER_FLOWCONTROL
-    if (unset_nonblock_fd (flowcontrol_pipe[0]) == 0)
-    {
-        char junk;
-        while (read (flowcontrol_pipe[0], &junk, 1) != 0);
-    }
+	if (unset_nonblock_fd (flowcontrol_pipe[0]) == 0)
+	{
+		char junk;
+		while (read (flowcontrol_pipe[0], &junk, 1) != 0);
+	}
 #endif
 
 	exit (exitstatus);

_
