From fe83538be797cb4440eaa86ed93042b09eb0ccaa Mon Sep 17 00:00:00 2001
From: Osamu Aoki <osamu@debian.org>
Date: Sun, 3 Feb 2019 21:36:32 +0900
Subject: [PATCH 1/7] Fix file path to the correct one

Signed-off-by: Osamu Aoki <osamu@debian.org>
---
 doc/POS | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/POS b/doc/POS
index ab07c1a..286d654 100644
--- a/doc/POS
+++ b/doc/POS
@@ -23,7 +23,7 @@ wtype_tは
 であるときに対象側は基準側に含まれる。
 
 
-src-worddic/wtype.hに CC_??として、活用のしかたを分類してある。
+anthy/wtype.hに CC_??として、活用のしかたを分類してある。
 活用表は最後の数文字を見て、語幹と活用形がわかればよいので、
 上一段と下一段は統合して扱う(本当はcannadicの都合)。
  CC_NONE 無活用
-- 
2.20.1

