From 520a8ee7767ffb0abc6e0c8b41e49e4530655fc8 Mon Sep 17 00:00:00 2001
From: Balint Reczey <balint@balintreczey.hu>
Date: Mon, 8 Feb 2016 14:36:47 +0100
Subject: [PATCH] Fix adding hardened1-linux-<cpu>

---
 debian/patches/add-gnuhardened1-linux | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/debian/patches/add-gnuhardened1-linux b/debian/patches/add-gnuhardened1-linux
index c7007f4..9eeca8b 100644
--- a/debian/patches/add-gnuhardened1-linux
+++ b/debian/patches/add-gnuhardened1-linux
@@ -1,17 +1,15 @@
 Author: Balint Reczey <balint@balintreczey.hu>
 Description: add support for hardened1-linux-amd64
 Debian-bug: 812786
-Upstream-commit: 9f3933227cc52a995c2d08595d99cb7ee1501c07
+Upstream-commit: 9f3933227cc52a995c2d08595d99cb7ee1501c07 + fix
 
-diff --git a/deb/architecture.ml b/deb/architecture.ml
-index 1fd7d0f..b7c1c27 100644
 --- a/deb/architecture.ml
 +++ b/deb/architecture.ml
-@@ -52,6 +52,7 @@ let triplettable = ref [
+@@ -52,6 +52,7 @@
    (("gnuabi64","linux","mips64"),    "mips64");             (* line 15 *)
    (("gnuspe","linux","powerpc"),     "powerpcspe");
    (("gnux32","linux","amd64"),       "x32");
-+  (("gnuhardened1","linux","<cpu>"), "<cpu>");
++  (("gnuhardened1","linux","<cpu>"), "hardened1-linux-<cpu>");
    (("gnu","linux","<cpu>"),          "<cpu>");
    (("gnu","kfreebsd","<cpu>"),       "kfreebsd-<cpu>");
    (("gnu","knetbsd","<cpu>"),        "knetbsd-<cpu>");      (* line 20 *)
-- 
2.1.4

