diff --git a/debian/changelog b/debian/changelog
index 82ce863..13249d7 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,10 @@
+apt-setup (1:0.154) UNRELEASED; urgency=medium
+
+  * Update apt sources lines for security from <dist>/updates to
+    <dist>-security.
+
+ -- Holger Wansing <hwansing@mailbox.org>  Sat, 12 Oct 2019 21:41:47 +0200
+
 apt-setup (1:0.153) unstable; urgency=medium
 
   * Team upload.
diff --git a/generators/91security b/generators/91security
index e9fa4c9..8ac9827 100755
--- a/generators/91security
+++ b/generators/91security
@@ -29,7 +29,7 @@ for dist in contrib non-free; do
 done
 
 # Don't test mirror if no network selected in netcfg
-echo "deb http://$host/debian-security $codename/updates $dists" >> $file
+echo "deb http://$host/debian-security $codename-security $dists" >> $file
 if db_get netcfg/dhcp_options && \
    [ "$RET" = "Do not configure the network at this time" ]; then
 	CODE=9
@@ -52,6 +52,6 @@ if [ "$RET" = false ]; then
 	deb_src="# deb-src"
 fi
 
-echo "$deb_src http://$host/debian-security $codename/updates $dists" >> $file
+echo "$deb_src http://$host/debian-security $codename-security $dists" >> $file
 
 exit $CODE
