diff --git a/debian/emacs-mozc.postinst b/debian/emacs-mozc.postinst
new file mode 100644
index 0000000..d00fa35
--- /dev/null
+++ b/debian/emacs-mozc.postinst
@@ -0,0 +1,10 @@
+#!/bin/sh
+set -e
+if [ ! -e /var/lib/emacsen-common/state/package/installed/emacsen-common ]; then
+    # Fallback to emacsen-common 1.x
+    if [ "$1" = "configure" ] && [ -x /usr/lib/emacsen-common/emacs-package-install ]; then
+	/usr/lib/emacsen-common/emacs-package-install emacs-mozc
+    fi
+else
+#DEBHELPER#
+fi
diff --git a/debian/emacs-mozc.prerm b/debian/emacs-mozc.prerm
new file mode 100644
index 0000000..7ce0460
--- /dev/null
+++ b/debian/emacs-mozc.prerm
@@ -0,0 +1,10 @@
+#!/bin/sh
+set -e
+if [ ! -e /var/lib/emacsen-common/state/package/installed/emacsen-common ]; then
+    # Fallback to emacsen-common 1.x
+    if [ -x /usr/lib/emacsen-common/emacs-package-remove ]; then
+	/usr/lib/emacsen-common/emacs-package-remove emacs-mozc
+    fi
+else
+#DEBHELPER#
+fi
