From 74e7b3205855a03606470f8cf0ef6e5048f21440 Mon Sep 17 00:00:00 2001
From: Marcin Szewczyk <marcin.szewczyk@wodny.org>
Date: Wed, 12 Nov 2014 21:11:30 +0100
Subject: [PATCH 1/2] acpi-support: require dbus

---
 init.d/acpi-support   | 4 ++--

diff --git a/init.d/acpi-support b/init.d/acpi-support
index a704582..f06b895 100755
--- a/init.d/acpi-support
+++ b/init.d/acpi-support
@@ -4,8 +4,8 @@
 
 ### BEGIN INIT INFO
 # Provides:          acpi-support
-# Required-Start:    $local_fs $remote_fs $syslog
-# Required-Stop:     $local_fs $remote_fs $syslog
+# Required-Start:    $local_fs $remote_fs $syslog dbus
+# Required-Stop:     $local_fs $remote_fs $syslog dbus
 # Default-Start:     2 3 4 5
 # Default-Stop:      1
 # Short-Description: Start some power management scripts
-- 
2.1.1

