--- apt_0.8.8	2010-10-28 22:48:42.000000000 +0300
+++ /etc/cron.daily/apt	2010-12-07 12:34:43.122702879 +0200
@@ -415,7 +415,13 @@
 
 # sleep random amount of time to avoid hitting the 
 # mirrors at the same time
-random_sleep
+if pidof -n anacron > /dev/null; then
+  # if anacron is running and the current time is 7:35AM
+  [ "$(date +%H:%M)" = "07:35" ] && random_sleep
+else
+  # if anacron is not running
+  random_sleep
+fi
 
 # include default system language so that "apt-get update" will
 # fetch the right translated package descriptions
