--- cron.daily/apt.orig	2016-02-04 13:09:56.774148517 +0100
+++ cron.daily/apt	2016-02-04 13:10:11.221364868 +0100
@@ -422,7 +422,15 @@
 
 # sleep random amount of time to avoid hitting the 
 # mirrors at the same time
-random_sleep
+# However do only sleep if this job has been started by cron
+# and not by anacron
+if which pstree > /dev/null; then
+  if pstree -s $$ | grep -q -- '--cron--' ; then
+    random_sleep
+  fi
+else
+  random_sleep
+fi  
 check_power || exit 0
 
 # include default system language so that "apt-get update" will
