diff -Nru debian-edu-config-1.423/debian/changelog debian-edu-config-1.423+nmu1/debian/changelog
--- debian-edu-config-1.423/debian/changelog	2008-07-26 01:24:37.000000000 +0200
+++ debian-edu-config-1.423+nmu1/debian/changelog	2008-08-23 19:26:07.000000000 +0200
@@ -1,3 +1,11 @@
+debian-edu-config (1.423+nmu1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Check innetgr and debian-edu-fsautoresize in cron.hourly to give no output
+    when they are not present (Closes: #493338)
+
+ -- Filippo Giunchedi <filippo@debian.org>  Sat, 23 Aug 2008 19:16:33 +0200
+
 debian-edu-config (1.423) unstable; urgency=low
 
   [ Petter Reinholdtsen ]
diff -Nru debian-edu-config-1.423/debian/debian-edu-config.cron.hourly debian-edu-config-1.423+nmu1/debian/debian-edu-config.cron.hourly
--- debian-edu-config-1.423/debian/debian-edu-config.cron.hourly	2008-07-12 13:27:48.000000000 +0200
+++ debian-edu-config-1.423+nmu1/debian/debian-edu-config.cron.hourly	2008-08-23 19:16:31.000000000 +0200
@@ -3,6 +3,9 @@
 PATH=/bin:/usr/bin:/sbin:/usr/sbin
 export PATH
 
+[ -x /usr/bin/innetgr ] || exit 0
+[ -x /usr/sbin/debian-edu-fsautoresize ] || exit 0 
+
 # Automatically extend full LVM volumes if the host is a member of the
 # fsautoresize-hosts netgroup.
 hostname=$(uname -n)
