--- rules.orig	2004-07-19 22:03:31.000000000 +0200
+++ rules	2004-07-19 08:15:00.000000000 +0200
@@ -89,6 +89,8 @@
 	mv $(CURDIR)/debian/tmp/usr/bin/cas $(CURDIR)/debian/amule-utils/usr/bin
 	mv $(CURDIR)/debian/tmp/usr/bin/amulecmd $(CURDIR)/debian/amule-utils/usr/bin
 	mv $(CURDIR)/debian/tmp/usr/bin/amulecmdDLG $(CURDIR)/debian/amule-utils/usr/bin
+	mv $(CURDIR)/debian/tmp/usr/bin/alc $(CURDIR)/debian/amule-utils/usr/bin
+	mv $(CURDIR)/debian/tmp/usr/bin/alcc $(CURDIR)/debian/amule-utils/usr/bin
 	mv $(CURDIR)/debian/tmp/usr/share/pixmaps/stat.png $(CURDIR)/debian/amule-utils/usr/share/pixmaps/stat.png
 
 
