--- control.orig	2004-07-19 22:03:21.000000000 +0200
+++ control	2004-07-19 22:08:07.000000000 +0200
@@ -2,7 +2,7 @@
 Section: x11
 Priority: optional
 Maintainer: Julien Delange <julien@gunnm.org>
-Build-Depends: debhelper(>= 4.0.0), libglib1.2-dev, libgtk1.2-dev, xlibs-dev, zlib1g-dev, libwxgtk2.4-dev, libwxbase2.4-dev, autoconf, libcurl2-dev, libcrypto++-dev
+Build-Depends: debhelper(>= 4.0.0), libglib1.2-dev, libgtk1.2-dev, xlibs-dev, zlib1g-dev, libwxgtk2.4-dev, libwxbase2.4-dev, autoconf, libcurl2-dev, libcrypto++-dev, libgd2-xpm-dev
 Standards-Version: 3.6.1.0
 
 Package: amule
@@ -19,7 +19,7 @@
 Package: amule-utils
 Architecture: any
 Replaces: astats
-Depends: ${shlibs:Depends}, ${misc:Depends}, amule (>=1.2.6+2.0.0rc3-1)
+Depends: ${shlibs:Depends}, ${misc:Depends}, amule (>=1.2.6+2.0.0rc5-1)
 Description: Utilities for amule
  amule-utils is a set of tools to control amule.
  .
