From 605ae83783f598ec316ec1bde6df48962719f3b0 Mon Sep 17 00:00:00 2001
From: Michael Biebl <biebl@debian.org>
Date: Sat, 1 Mar 2014 08:14:32 +0100
Subject: [PATCH 4/4] Release version 4.101-4.1 to unstable

---
 debian/changelog | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/changelog b/debian/changelog
index 97bc72b..ca7fd1b 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,4 +1,4 @@
-bluez (4.101-4.1) UNRELEASED; urgency=medium
+bluez (4.101-4.1) unstable; urgency=medium
 
   * Non-maintainer upload.
   * Stop passing static start and stop priorities to update-rc.d. Those are
@@ -14,7 +14,7 @@ bluez (4.101-4.1) UNRELEASED; urgency=medium
       the correct order, by pointing dh_systemd_enable and dh_systemd_start at
       the uninstalled service file. See #715504 for details.
 
- -- Michael Biebl <biebl@debian.org>  Sat, 01 Mar 2014 07:36:31 +0100
+ -- Michael Biebl <biebl@debian.org>  Sat, 01 Mar 2014 08:14:10 +0100
 
 bluez (4.101-4) unstable; urgency=low
 
-- 
1.9.0

