--- /home/milan/Bureau/AdduserCommon.pm	2010-11-21 12:45:31.000000000 +0100
+++ /usr/share/perl5/Debian/AdduserCommon.pm	2010-11-21 12:44:51.697032827 +0100
@@ -207,8 +207,8 @@
   $configref->{"dir_mode"} = "0755";
   $configref->{"setgid_home"} = "no";
   $configref->{"no_del_paths"} = "^/$ ^/lost+found/.* ^/media/.* ^/mnt/.* ^/etc/.* ^/bin/.* ^/boot/.* ^/dev/.* ^/lib/.* ^/proc/.* ^/root/.* ^/sbin/.* ^/tmp/.* ^/sys/.* ^/srv/.* ^/opt/.* ^/initrd/.* ^/usr/.* ^/var/.*";
-  $configref->{"name_regex"} = "^[a-z][-a-z0-9_]*\$";
-  $configref->{"name_regex_system"} = "^[A-Za-z][-A-Za-z0-9_]*\$";
+  $configref->{"name_regex"} = "^[a-z][-a-z0-9_.]*\$";
+  $configref->{"name_regex_system"} = "^[A-Za-z][-A-Za-z0-9_.]*\$";
   $configref->{"exclude_fstypes"} = "(proc|sysfs|usbfs|devpts|tmpfs)";
   $configref->{"skel_ignore_regex"} = "dpkg-(old|new|dist)\$";
   $configref->{"extra_groups"} = "dialout cdrom floppy audio video plugdev users";
