diff -ru checksecurity-2.0.14.pristine/debian/control checksecurity-2.0.14/debian/control
--- checksecurity-2.0.14.pristine/debian/control	2010-10-27 22:44:37.000000000 +0100
+++ checksecurity-2.0.14/debian/control	2012-09-18 20:47:50.992411365 +0100
@@ -12,19 +12,17 @@
 Suggests: apt-watch | cron-apt, lockfile-progs
 Conflicts: lockfile-progs (<< 0.1.7)
 Replaces: cron
-Provides:
 Description: basic system security checks
- Checksecurity does some very basic system security checks, such as
- looking for changes in which programs have setuid permissions, and that
- remote filesystems are not allowed to have runnable setuid programs.
+ Checksecurity can do some very basic system security checks:
  .
- Note that these are not to be considered in any way complete, and
- you should not rely on checksecurity to actually provide any useful
- information concerning the security or vulnerability of your system.
+  * check-diskfree - scans for mounted filesystems nearing capacity;
+  * check-passwd - scans for empty or duplicate system accounts;
+  * check-setuid - scans for insecurely mounted remote file systems,
+    and tracks changes in setuid programs;
+  * check-sockets - tracks changes in open ports.
  .
- The lockfile-progs package is only a "Suggests" because of the poor
- way that dselect handles "Recommends", but I do strongly suggest that
- you install it; it prevents /etc/cron.daily/standard from running multiple
- times if something gets jammed.
+ Be aware that this is no substitute for a full security auditing and
+ integrity checking system.
  .
- Checksecurity was previously part of the cron package.
+ Installing the suggested package lockfile-progs can help to prevent
+ the cron jobs from running multiple times if something gets jammed.
