diff -ru bsdmainutils-8.0.1.pristine/debian/control bsdmainutils-8.0.1/debian/control
--- bsdmainutils-8.0.1.pristine/debian/control	2009-11-03 14:34:42.000000000 +0000
+++ bsdmainutils-8.0.1/debian/control	2009-11-23 14:36:04.000000000 +0000
@@ -15,8 +15,8 @@
  This package contains lots of small programs many people expect to find when
  they use a BSD-style Unix system.
  .
- Included are: banner, ncal, cal, calendar, col, colcrt, colrm, column, from,
- hexdump, look, lorder, ul, write.
+ Included are: bsd-write, calendar, col, colcrt, colrm, column, from, hexdump,
+ look, lorder, ncal, printerbanner, ul.
  .
  This package used to contain whois and vacation, which are now distributed in
  their own packages. Also here was tsort, which is now in the "coreutils"
