--- tempfile.c	2009-05-02 09:18:48.000000000 +0200
+++ tempfile.c.orig	2009-05-02 09:17:19.000000000 +0200
@@ -131,8 +131,7 @@
       if ((fd = open(filename, O_RDWR | O_CREAT | O_EXCL, mode)) < 0) {
 	if (errno == EEXIST) {
 	  free(name);
-          if (name != filename)
-	    free(filename);
+	  free(filename);
 	  continue;
 	}
 	syserror("open");
@@ -145,7 +144,6 @@
     syserror("close");
   puts(filename);
   free(filename);
-  if (name != filename)
-    free(name);
+  free(name);
   exit(0);
 }
