remove stray space in regex

Index: fai-6.0.3/lib/get-config-dir-svn
===================================================================
--- fai-6.0.3.orig/lib/get-config-dir-svn
+++ fai-6.0.3/lib/get-config-dir-svn
@@ -52,7 +52,7 @@ user=
 [ -n "$username" ] && user="--username $username"
 
 if [ -d "$FAI/.svn" ] ; then
-    if [ $(svn info $FAI | awk '/ ^URL:/ {print $2}') == "$svnurl" ]; then
+    if [ $(svn info $FAI | awk '/^URL:/ {print $2}') == "$svnurl" ]; then
         echo "Updating SVN in $FAI"
         cd $FAI
         svn up $user $pass | grep -v 'Updated to revision' > $LOGDIR/getconf.log
