diff --git a/config/di-sources.list b/config/di-sources.list
index caeb5f8..209ef4e 100644
--- a/config/di-sources.list
+++ b/config/di-sources.list
@@ -48,14 +48,12 @@ squeeze	sparc	http://ftp.debian.org/dists/squeeze/main/installer-sparc/current/i
 squeeze-gtk	amd64	http://ftp.debian.org/dists/squeeze/main/installer-amd64/current/images/	netboot/gtk/netboot.tar.gz
 squeeze-gtk	i386	http://ftp.debian.org/dists/squeeze/main/installer-i386/current/images/	netboot/gtk/netboot.tar.gz
 
-#Debian/Jessie (not released yet, As of writing this file)
-#jessie	amd64	http://ftp.debian.org/dists/jessie/main/installer-amd64/current/images/	netboot/netboot.tar.gz
-#jessie	i386	http://ftp.debian.org/dists/jessie/main/installer-i386/current/images/	netboot/netboot.tar.gz
-#jessie	ia64	http://ftp.debian.org/dists/jessie/main/installer-ia64/current/images/	netboot/netboot.tar.gz
-#jessie	sparc	http://ftp.debian.org/dists/jessie/main/installer-sparc/current/images/	netboot/boot.img
+#Debian/Jessie
+jessie	amd64	http://ftp.debian.org/dists/jessie/main/installer-amd64/current/images/	netboot/netboot.tar.gz
+jessie	i386	http://ftp.debian.org/dists/jessie/main/installer-i386/current/images/	netboot/netboot.tar.gz
 ##Graphical Installer (GTK)
-#jessie-gtk	amd64	http://ftp.debian.org/dists/jessie/main/installer-amd64/current/images/	netboot/gtk/netboot.tar.gz
-#jessie-gtk	i386	http://ftp.debian.org/dists/jessie/main/installer-i386/current/images/	netboot/gtk/netboot.tar.gz
+jessie-gtk	amd64	http://ftp.debian.org/dists/jessie/main/installer-amd64/current/images/	netboot/gtk/netboot.tar.gz
+jessie-gtk	i386	http://ftp.debian.org/dists/jessie/main/installer-i386/current/images/	netboot/gtk/netboot.tar.gz
 
 # DEVELOPMENT (( http://www.debian.org/devel/debian-installer/ ))
 
