From f46c89ee73812074fe4b4f5c29ddc444b90bcd32 Mon Sep 17 00:00:00 2001
From: Xiyue Deng <manphiz@gmail.com>
Date: Wed, 11 Sep 2024 15:25:35 -0700
Subject: [PATCH 2/2] Update d/changelog

---
 debian/changelog | 7 +++++++
 1 file changed, 7 insertions(+)

diff --git a/debian/changelog b/debian/changelog
index 414af7e..a40535b 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,10 @@
+aegisub (3.2.2+dfsg-7.2) UNRELEASED; urgency=medium
+
+  * Non-maintainer upload.
+  * Add patch to suppress startup warnings on newer wxWidgets versions.
+
+ -- Xiyue Deng <manphiz@gmail.com>  Wed, 11 Sep 2024 15:24:39 -0700
+
 aegisub (3.2.2+dfsg-7.1) unstable; urgency=medium
 
   * Non-maintainer upload.
-- 
2.39.2

