diff -Nru console-common-0.7.85/debian/changelog console-common-0.7.85+nmu1/debian/changelog
--- console-common-0.7.85/debian/changelog	2009-11-08 20:35:36.000000000 +0200
+++ console-common-0.7.85+nmu1/debian/changelog	2011-09-19 06:26:32.000000000 +0300
@@ -1,3 +1,9 @@
+console-common (0.7.85+nmu1) unstable; urgency=low
+
+  * Add init.d status support
+
+ -- Peter Eisentraut <petere@debian.org>  Mon, 19 Sep 2011 06:26:15 +0300
+
 console-common (0.7.85) unstable; urgency=low
 
   * Update Standards-Version to 3.8.3 (checked)
diff -Nru console-common-0.7.85/debian/keymap.sh console-common-0.7.85+nmu1/debian/keymap.sh
--- console-common-0.7.85/debian/keymap.sh	2009-10-31 21:41:53.000000000 +0200
+++ console-common-0.7.85+nmu1/debian/keymap.sh	2011-09-19 06:27:49.000000000 +0300
@@ -134,8 +134,12 @@
     stop)
 	;;
 
+    status)
+	exit 0
+	;;
+
     *)
-	log_warning_msg "Usage: $0 {start|stop|restart|reload|force-reload}"
+	log_warning_msg "Usage: $0 {start|stop|restart|reload|force-reload|status}"
 	;;
   esac
 
