--- project-history.sgml.orig	2010-09-22 16:57:00.000000000 +0200
+++ project-history.sgml	2011-01-24 22:37:44.000000000 +0100
@@ -337,20 +337,20 @@
 providing support for newer ARM processors and deprecating the old ARM
 port (<em>arm</em>).
 The <url id="http://wiki.debian.org/M68k" name="m68k"> port was not 
-included in this relase, although it was still provided in the 
+included in this release, although it was still provided in the 
 <em>unstable</em> distribution. This release did not feature the
 <url id="http://www.debian.org/ports/kfreebsd-gnu/" name="FreeBSD port">,
 although much work on the port had been done to make it qualify
 it did not meet yet the
 <url id="http://release.debian.org/lenny/arch_qualify.html" name="qualification requirements"> for this release.
 This release added support for Marvell's Orion platform which is used in many
-storage devices. and also provided supported for now supports several Netbooks,
+storage devices and also provided supported several Netbooks,
 in particular the Eee PC by Asus. <em>Lenny</em> also contained the build tools
 for Emdebian which allowed Debian source packages to be cross-built and shrunk
 to suit embedded ARM systems.
 It was also the first release to provide free versions of Sun's Java
 technology, making it possible to provide Java applications in the
-<em>main</em> repository.
+<em>main</em> section.
 
 
 <!--
@@ -649,7 +649,7 @@
 as papers analysing later releases.</p></footnote>
 thoroughly by a group of interested people in
 an article called <url
-id="http://www.upgrade-cepis.org/issues/2001/6/up2-6Gonzalez.pdf"
+id="http://pascal.case.unibz.it/retrieve/3246/counting-potatoes.html"
 name="Counting potatoes"> quoting from this article:
  
 <p><em> "[...] we use David A. Wheeler's sloccount system to determine
@@ -829,7 +829,7 @@
 id="http://www.debian.org/News/2009/20090214" name="released"> February 14th,
 2009 for one more architecture than its predecessor, <em>etch</em>. This
 included the port for newer ARM processors. As with the previous release,
-support for the m68k arquitecture was still available in the <em>unstable</em>.
+support for the m68k architecture was still available in <em>unstable</em>.
 There were around 23,000 binary packages (built from over 12,000 source
 packages) maintained by more than more than one thousand and ten Debian
 developers.
@@ -861,7 +861,7 @@
 
 <p>The ninth <em>Debconf</em> was held in Mar de Plata, Argentina, from
 August 10th to 16th, 2008 with over <url id="https://gallery.debconf.org/v/debconf8/karora/OfficialPhoto.jpg.html" name="two hundred"> participants.
-<url id="http://ftp.acc.umu.se/pub/debian-meetings/2008/debconf7/" name="Videos"> and <url id="https://gallery.debconf.org/v/debconf8/" name="pictures">
+<url id="http://ftp.acc.umu.se/pub/debian-meetings/2008/debconf8/" name="Videos"> and <url id="https://gallery.debconf.org/v/debconf8/" name="pictures">
 from this
 conference are available online.
 <!-- TODO: Add more information based on
