Description: Use build flags from the environment (dpkg-buildflags).
Author: Simon Ruderich <simon@ruderich.org>
Last-Update: 2012-03-10

--- audacity-2.0.0~rc8.orig/lib-src/portmixer/Makefile.in
+++ audacity-2.0.0~rc8/lib-src/portmixer/Makefile.in
@@ -9,9 +9,12 @@ AR       = @AR@
 RANLIB   = @RANLIB@
 
 DEFS    += @DEFS@
+CFLAGS  += @CFLAGS@
 CFLAGS  += @cflags@
 CFLAGS  += @include@
 
+CPPFLAGS = @CPPFLAGS@
+
 all : $(LIBRARY)
 
 tests:
@@ -22,7 +25,7 @@ $(LIBRARY): $(OBJECTS)
 	$(RANLIB) $(LIBRARY)
 
 %.o : src/%.c
-	$(CC) $(CFLAGS) $(DEFS) -c $(<) -o $@
+	$(CC) $(CFLAGS) $(CPPFLAGS) $(DEFS) -c $(<) -o $@
 
 clean : 
 	-rm -f $(LIBRARY)
