diff -u dsh-0.25.10/debian/changelog dsh-0.25.10/debian/changelog
--- dsh-0.25.10/debian/changelog
+++ dsh-0.25.10/debian/changelog
@@ -1,3 +1,14 @@
+dsh (0.25.10-1.1) unstable; urgency=medium
+
+  [ Aurelien Jarno ]
+  * Non-maintainer upload.
+  
+  [ Logan Rosen ]
+  * Use autotools-dev to update config.{sub,guess} for new arches (Closes:
+    #735633).
+
+ -- Aurelien Jarno <aurel32@debian.org>  Sat, 04 Oct 2014 21:55:24 +0200
+
 dsh (0.25.10-1) unstable; urgency=low
 
   * New upstream version
diff -u dsh-0.25.10/debian/control dsh-0.25.10/debian/control
--- dsh-0.25.10/debian/control
+++ dsh-0.25.10/debian/control
@@ -2,7 +2,7 @@
 Section: net
 Priority: optional
 Maintainer: Junichi Uekawa <dancer@debian.org>
-Build-Depends: debhelper (>> 3.0.0), libdshconfig1-dev (>= 0.20.11-1), po4a
+Build-Depends: debhelper (>> 3.0.0), libdshconfig1-dev (>= 0.20.11-1), po4a, autotools-dev
 Standards-Version: 3.7.2
 
 Package: dsh
diff -u dsh-0.25.10/debian/rules dsh-0.25.10/debian/rules
--- dsh-0.25.10/debian/rules
+++ dsh-0.25.10/debian/rules
@@ -25,6 +25,7 @@
 configure: configure-stamp
 configure-stamp:
 	dh_testdir
+	dh_autotools-dev_updateconfig
 	INSTALL_PROGRAM='${INSTALL_PROGRAM}' CFLAGS='${CFLAGS}' \
 	 ./configure --prefix=/usr --sysconfdir=/etc/dsh --mandir=/usr/share/man
 	touch configure-stamp
@@ -42,6 +43,7 @@
 	rm -f build-stamp configure-stamp
 	-$(MAKE) clean
 	-$(MAKE) distclean
+	dh_autotools-dev_restoreconfig
 	dh_clean
 
 install: build
