--- insighttoolkit-3.18.0.orig/Code/IO/itkNiftiImageIO.cxx
+++ insighttoolkit-3.18.0/Code/IO/itkNiftiImageIO.cxx
@@ -457,6 +457,10 @@
     _size[4] = numComponents;
     }
 
+  // Free memory if any was occupied already
+  if (this->m_NiftiImage != NULL)
+      nifti_image_free(this->m_NiftiImage);
+
   //
   // allocate nifti image...
   this->m_NiftiImage = nifti_image_read(this->GetFileName(),false);
