diff -u anacron-2.3/gregor.c anacron-2.3.mod/gregor.c
--- anacron-2.3/gregor.c	2000-06-23 02:00:14.000000000 +0200
+++ anacron-2.3.mod/gregor.c	2016-05-24 21:08:45.523927309 +0200
@@ -65,7 +65,7 @@
 {
     int dn;
     int i;
-    const int isleap; /* save three calls to leap() */
+    int isleap; /* save three calls to leap() */
 
     /* Some validity checks */
 
diff -u anacron-2.3/readtab.c anacron-2.3.mod/readtab.c
--- anacron-2.3/readtab.c	2000-06-23 02:00:14.000000000 +0200
+++ anacron-2.3.mod/readtab.c	2016-05-24 21:13:25.147931252 +0200
@@ -86,7 +86,7 @@
     int c;
 
     if (feof(tab)) return NULL;
-    while ((c = getc(tab)) != EOF && c != '\n')
+    while ((c = getc(tab)) != EOF && c != '\n' && c != '\r')
 	obstack_1grow(&input_o, c);
     if (ferror(tab)) die_e("Error reading %s", anacrontab);
     obstack_1grow(&input_o, '\0');
@@ -181,7 +181,7 @@
     char *ident;
     char *command;
 
-    /* an empty line? */
+    /* an empty line or a comment ? */
     r = match_rx("^[ \t]*($|#)", line, 0);
     if (r == -1) goto reg_err;
     if (r)
Gemeinsame Unterverzeichnisse: anacron-2.3/teststuff und anacron-2.3.mod/teststuff.
