diff -Nurad debsecan-0.4.10/README debsecan-0.4.10.new/README
--- debsecan-0.4.10/README	2007-08-19 21:17:44.000000000 +0200
+++ debsecan-0.4.10.new/README	2008-02-11 18:40:09.000000000 +0100
@@ -19,7 +19,7 @@
 
 A web interface to the database is available at:
 
-  <http://idssi.enyo.de/tracker/>
+  <http://security-tracker.debian.net/tracker/>
 
 FAQ
 ---
diff -Nurad debsecan-0.4.10/src/debsecan debsecan-0.4.10.new/src/debsecan
--- debsecan-0.4.10/src/debsecan	2007-09-02 17:57:39.000000000 +0200
+++ debsecan-0.4.10.new/src/debsecan	2008-02-11 18:40:35.000000000 +0100
@@ -1010,7 +1010,7 @@
             return ' '.join(result)     # should not be reachedg
 
         def write_url(bug):
-            w("  <http://idssi.enyo.de/tracker/%s>" % bug)
+            w("  <http://security-tracker.debian.net/tracker/%s>" % bug)
 
         def scan(fix_status, new_status):
             have_obsolete = False
