diff -Nur ca-certificates-20070303.orig/debian/templates ca-certificates-20070303/debian/templates
--- ca-certificates-20070303.orig/debian/templates	2007-07-27 11:03:13.000000000 +0200
+++ ca-certificates-20070303/debian/templates	2008-03-24 20:38:14.329437981 +0100
@@ -22,14 +22,14 @@
 
 Template: ca-certificates/new_crts
 Type: multiselect
-_Choices: ${new_crts}
+Choices: ${new_crts}
 _Description: New certificates to activate:
  During upgrades, new certificates will be added. Please choose
  those you trust.
 
 Template: ca-certificates/enable_crts
 Type: multiselect
-_Choices: ${enable_crts}
+Choices: ${enable_crts}
 _Description: Certificates to activate:
  This package installs common CA (Certificate Authority) certificates in
  /usr/share/ca-certificates.
