--- a/dpkg/config
+++ b/dpkg/config
@@ -3,7 +3,7 @@
 ###
 
 dbc_go(){
-	local f db happy default_dbtype host newhost port oldhosts authmethod_admin do_config _preconf_list _s _t STATE iformat ifile install_question migrating userdb_prio
+	local f db happy default_dbtype host newhost port oldhosts authmethod_admin do_config _preconf_list _s _t STATE iformat ifile install_question migrating userdb_prio supported_dbtypes
 	. /usr/share/dbconfig-common/dpkg/common
 	_dbc_debug "(config) dbc_go() $@"
 
@@ -116,8 +116,9 @@ dbc_go(){
 			dbc_dbtype=$dbc_hardcoded_dbtype
 		# else if the package supports multiple dbtypes, help them pick one
 		elif [ "$dbc_dbtypes" ]; then
-		# loop through all available dbtypes
-			for db in $dbc_all_supported_dbtypes; do
+		# loop through the listed dbtypes, fall back to all those available
+			supported_dbtypes=$(echo $dbc_dbtypes | sed 's/,/ /g')
+			for db in $supported_dbtypes $dbc_all_supported_dbtypes; do
 				# if we're already happy, we're already done
 				if [ "$happy" ]; then
 					true
