diff -ru developers-reference-3.4.1.pristine/l10n.dbk developers-reference-3.4.1/l10n.dbk
--- developers-reference-3.4.1.pristine/l10n.dbk	2008-06-09 21:17:46.000000000 +0100
+++ developers-reference-3.4.1/l10n.dbk	2009-06-25 23:53:45.000000000 +0100
@@ -4,7 +4,7 @@
   <!ENTITY % commondata SYSTEM "common.ent" > %commondata;
 ]>
 <chapter id="l10n">
-<title>Internationalizing, translating, being internationalized and being translated</title>
+<title>Internationalization and Translations</title>
 <para>
 Debian supports an ever-increasing number of natural languages.  Even if you
 are a native English speaker and do not speak any other language, it is part of
