diff -Nru devscripts-2.10.7~/scripts/debsign.sh devscripts-2.10.7/scripts/debsign.sh
--- devscripts-2.10.7~/scripts/debsign.sh	2007-08-10 03:18:57.000000000 -0700
+++ devscripts-2.10.7/scripts/debsign.sh	2007-09-25 17:14:28.000000000 -0700
@@ -159,8 +159,11 @@
     then
 	return 1
     else
-	printf "The .$2 file is already signed.\nWould you like to use the current signature? [Yn]"
-	read response
+        response="n"
+        if [ -z "$DEBSIGN_ALWAYS_RESIGN" ]; then
+		printf "The .$2 file is already signed.\nWould you like to use the current signature? [Yn]"
+		read response
+	fi
 	case $response in
 	[Nn]*)
 	    sed -e '1,/^$/d; /^$/,$d' "$1" > "$1.unsigned"
@@ -179,7 +182,8 @@
 DEFAULT_DEBSIGN_SIGNLIKE=
 DEFAULT_DEBSIGN_MAINT=
 DEFAULT_DEBSIGN_KEYID=
-VARS="DEBSIGN_PROGRAM DEBSIGN_SIGNLIKE DEBSIGN_MAINT DEBSIGN_KEYID"
+DEFAULT_DEBSIGN_ALWAYS_RESIGN=
+VARS="DEBSIGN_PROGRAM DEBSIGN_SIGNLIKE DEBSIGN_MAINT DEBSIGN_KEYID DEBSIGN_ALWAYS_RESIGN"
 
 if [ "$1" = "--no-conf" -o "$1" = "--noconf" ]; then
     shift
