--- asterisk-1.2.16~dfsg/debian/asterisk.init	2007-03-07 10:36:42.000000000 +0100
+++ asterisk-1.2.16~dfsg/debian/asterisk.init.new	2007-03-07 10:39:11.000000000 +0100
@@ -162,6 +162,9 @@ case "$1" in
 	echo "Reloading $DESC configuration files."
 	asterisk_rx 'extensions reload'
 	;;
+  restart-convenient)
+        asterisk_rx 'restart when convenient'
+        ;;
   restart|force-reload)
 	$0 stop
 	$0 start
@@ -182,7 +185,7 @@ case "$1" in
 	;; 	 
   *)
 	N=/etc/init.d/$NAME
-	echo "Usage: $N {start|stop|restart|reload|status|debug|logger-reload|extensions-reload|force-reload}" >&2
+	echo "Usage: $N {start|stop|restart|reload|status|debug|logger-reload|extensions-reload|restart-convenient|force-reload}" >&2
 	exit 1
 	;;
 esac
