--- src/text/ptbl/xp/pd_Document.cpp.orig	2012-10-26 17:51:28.000000000 +0300
+++ src/text/ptbl/xp/pd_Document.cpp	2012-10-26 18:27:32.000000000 +0300
@@ -213,8 +213,13 @@
 	if(strcmp(name, "Unknown") == 0)
 		name = g_get_user_name();
 	gchar *utf8name = g_locale_to_utf8(name, -1, NULL, NULL, NULL);
-	m_sUserName = utf8name;
-	g_free(utf8name);
+	if (utf8name != NULL)
+	{
+		m_sUserName = utf8name;
+		g_free(utf8name);
+	}
+	else
+		m_sUserName = "Unknown";
 }
 
 PD_Document::~PD_Document()
