--- initramfs-cryptroot-hook	2006-12-28 10:01:59.000000000 +0200
+++ /home/vinsci/src/linux/boot/cryptsetup/cryptsetup-1.0.4/debian/initramfs-cryptroot-hook	2006-12-28 11:44:13.000000000 +0200
@@ -222,13 +222,13 @@
 }
 
 get_device_modules() {
-	local node value cipher blockcipher ivhash
+	local node value cipher chainmode ivhash
 	node="$1"
 
 	# Check the ciphers used by the active root mapping
 	value=$(dmsetup table "$node" | cut -d " " -f4)
 	cipher=$(echo "$value" | cut -d ":" -f1 | cut -d "-" -f1)
-	blockcipher=$(echo "$value" | cut -d ":" -f1 | cut -d "-" -f2)
+	chainmode=$(echo "$value" | cut -d "-" -s -f2)
 	ivhash=$(echo "$value" | cut -d ":" -s -f2)
 
 	if [ -n "$cipher" ]; then
@@ -237,11 +237,11 @@
 		return 1
 	fi
 
-	if [ -n "$blockcipher" ]; then
-		echo "$blockcipher"
+	if [ -n "$chainmode" -a "$chainmode" != "plain" ]; then
+		echo "$chainmode"
 	fi
 
-	if [ -n "$ivhash" ]; then
+	if [ -n "$ivhash" -a "$ivhash" != "plain" ]; then
 		echo "$ivhash"
 	fi
 	return 0
