--- /usr/share/initramfs-tools/scripts/local.old	2006-10-17 09:26:59.000000000 +0200
+++ /usr/share/initramfs-tools/scripts/local	2006-12-11 17:48:00.000000000 +0100
@@ -7,6 +7,10 @@
 	run_scripts /scripts/local-top
 	[ "$quiet" != "y" ] && log_end_msg
 
+	if [ -r /conf/param.conf ]; then
+		. /conf/param.conf
+	fi
+
 	# If the root device hasn't shown up yet, give it a little while
 	# to deal with removable devices
 	if [ ! -e "${ROOT}" ]; then
