--- /dev/shm/cryptdisks.functions	2014-03-02 14:10:00.000000000 +0100
+++ /lib/cryptsetup/cryptdisks.functions	2014-03-03 10:05:12.000000000 +0100
@@ -451,9 +451,9 @@
 	while [ "$tried" -lt "$TRIES" ]; do
 		export CRYPTTAB_TRIED="$tried"
 		if [ -n "$KEYSCRIPT" ]; then
-			$KEYSCRIPT "$keyscriptarg" | cryptsetup $PLAINPARAMS $PARAMS open --type plain "${dst}_unformatted" "$src"
+			$KEYSCRIPT "$keyscriptarg" | cryptsetup $PLAINPARAMS $PARAMS open --type plain "$src" "${dst}_unformatted"
 		else
-			cryptsetup $PLAINPARAMS $PARAMS open --type plain "${dst}_unformatted" "$src"
+			cryptsetup $PLAINPARAMS $PARAMS open --type plain "$src" "${dst}_unformatted"
 		fi
 
 		if [ -z "$CHECK" ] || "$CHECK" "/dev/mapper/${dst}_unformatted" $CHECKARGS; then
