--- bwbasic-2.20pl2.orig/bwx_tty.c	2012-09-03 01:59:03.612080665 +0200
+++ bwbasic-2.20pl2/bwx_tty.c	2012-09-03 01:59:23.872081181 +0200
@@ -65,8 +65,8 @@
 
 ***************************************************************/
 
+int
 #if ANSI_C
-void
 main( int argc, char **argv )
 #else
 main( argc, argv )
@@ -92,7 +92,7 @@
 
       bwx_terminate();			/* allow ^D (Unix) exit with grace */
 
-
+      return 0;
    }
 
 /***************************************************************
