diff -ur ascii-3.8/nametable ascii/nametable
--- ascii-3.8/nametable	2005-03-02 17:29:17.000000000 +0100
+++ ascii/nametable	2006-03-31 10:05:03.000000000 +0200
@@ -24,18 +24,18 @@
 %%
 Mnemonics: "BEL",
 ISO names: "Bell",
-C escape:  "\a" 
+C escape:  "\\a",
 Synonyms:  "Alert",
 %%
 Mnemonics: "BS",
 ISO names: "Backspace",
 C escape:  "\\b",
 %%
-Mnemonics: "HT"  "TAB",
-ISO names: "Character Tabulation", "Horizontal Tab" 
+Mnemonics: "HT", "TAB",
+ISO names: "Character Tabulation", "Horizontal Tab",
 C escape:  "\\t",
 %%
-Mnemonics: "LF"  "NL",
+Mnemonics: "LF", "NL",
 ISO name: "Line Feed", "Newline",
 C escape:  "\\n",
 %%
@@ -135,7 +135,7 @@
 %%
 Mnemonics: "&",
 ISO names: "Ampersand",
-Synonyms: "Amper", "And", 
+Synonyms: "Amper", "And",
 XML name:  "&amp;",
 %%
 Mnemonics: "'",
@@ -224,7 +224,7 @@
 ISO names: "Less-than Sign",
 Synonyms:  "Left Angle Bracket", "Read From", "In",
 	"From", "Comesfrom", "Left Funnel", "Left Broket", "Crunch",
-	"Suck", 
+	"Suck",
 XML name:  "&lt;",
 Comment:   "# See `>' for matching names.",
 %%
@@ -236,7 +236,7 @@
 ISO names: "Greater-than sign",
 Synonyms: "Right Angle"  "Write To", "Into", "Toward",
 	"Out", "To", "Gozinta", "Right Funnel",
-	"Right Broket", "Zap", "Blow", 
+	"Right Broket", "Zap", "Blow",
 XML name:  "&gt;",
 Comments: "# See `<' for matching names.",
 %%
@@ -327,7 +327,7 @@
 Mnemonics: "Z",
 ISO names: "Majuscule Z", "Capital Z", "Uppercase Z",
 %%
-Mnemonics: "["		
+Mnemonics: "[",
 ISO names: "Left Square Bracket",
 Synonyms:  "Bracket", "Bra", "Square",
 Comments:  "# See `]' for matching names.",
@@ -439,16 +439,16 @@
 Mnemonics: "z",
 ISO names: "Miniscule z", "Small z", "Lowercase z",
 %%
-Mnemonics: "{"		
+Mnemonics: "{",
 ISO names:  "Left Curly Bracket",
 Synonyms: "Left Brace", "Brace", "Open Brace""Curly", "Leftit", "Embrace",
 Comments: "# See `}' for matching names.",
 %%
-Mnemonics: "|"		
+Mnemonics: "|",
 ISO names: "Vertical Line",
 Synonyms: "Pipe", "Bar", "Or", "V-Bar", "Spike", "Gozinta", "Thru",
 %%
-Mnemonics: "}"		
+Mnemonics: "}",
 ISO names: "Right Curly Bracket",
 Synonyms: "Right Brace", "Unbrace", "Close Brace",
 	"Uncurly", "Rytit", "Bracelet",
