Index: debian/amavisd-new.init
===================================================================
RCS file: /cvsroot/amavisd-new/amavisd-new/debian/amavisd-new.init,v
retrieving revision 1.9
diff -u -r1.9 amavisd-new.init
--- debian/amavisd-new.init	31 Dec 2003 09:48:15 -0000	1.9
+++ debian/amavisd-new.init	5 Jan 2004 23:50:36 -0000
@@ -10,6 +10,7 @@
 
 PATH=/sbin:/bin:/usr/sbin:/usr/bin
 DAEMON=/usr/sbin/amavisd-new
+DAEMON2=/usr/bin/perl
 NAME=amavisd
 DESC=amavisd
 PARAMS=""
@@ -69,7 +70,7 @@
   stop)
 	echo -n "Stopping $DESC: "
 	if start-stop-daemon --stop --quiet --pidfile /var/run/amavis/$NAME.pid \
-		-- $PARAMS ; then
+		--startas $DAEMON2 -- $PARAMS ; then
 		echo "amavisd-new."
 	else
 		echo "(failed)."
@@ -79,7 +80,7 @@
   reload)
 	echo "Reloading $DESC configuration files."
 	start-stop-daemon --stop --signal 1 --quiet --pidfile \
-		/var/run/amavis/$NAME.pid
+		/var/run/amavis/$NAME.pid --startas $DAEMON2
   	;;
   restart|force-reload)
 	$0 stop && {
@@ -88,7 +89,7 @@
 		while [ $i -gt 0 ] ; do
 			if start-stop-daemon --start --test \
 			   --pidfile /var/run/amavis/$NAME.pid \
-			   >/dev/null 2>&1 ; then
+			   --startas $DAEMON2 >/dev/null 2>&1 ; then
 			     break
 			fi
 			sleep 2s
