diff -u dejagnu-1.4.4.git20080407/debian/changelog dejagnu-1.4.4.git20080407/debian/changelog
--- dejagnu-1.4.4.git20080407/debian/changelog
+++ dejagnu-1.4.4.git20080407/debian/changelog
@@ -1,3 +1,10 @@
+dejagnu (1.4.4.git20080407-1.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Avoid shipping /usr/share/info/dir.gz (Closes: #546551)
+
+ -- Stefano Zacchiroli <zack@debian.org>  Sun, 04 Oct 2009 15:30:39 +0200
+
 dejagnu (1.4.4.git20080407-1) unstable; urgency=low
 
   * New upstream snapshot, with most Debian patches merged.
diff -u dejagnu-1.4.4.git20080407/debian/rules dejagnu-1.4.4.git20080407/debian/rules
--- dejagnu-1.4.4.git20080407/debian/rules
+++ dejagnu-1.4.4.git20080407/debian/rules
@@ -43,6 +43,7 @@
 	# Make install already installs this, but redo it to get the right
 	# postinst bits.
 	dh_installinfo doc/dejagnu.info
+	-rm -f debian/dejagnu/usr/share/info/dir	# Closes: #546551
 
 post-patches::
 	# Nothing in Debian is quite capable of regenerating this (neither
