--- /usr/share/docbook-utils/frontends/docbook	2012-05-09 19:31:08.000000000 +0200
+++ /tmp/docbook	2012-09-11 09:30:11.271357601 +0200
@@ -6,7 +6,7 @@
 # Give the location of the centralized catalog
 # First try the one with a version matching the version that is indicated in the document
   centralized-catalog)
-	if [ -n "$SGML_NORM" ] && [ "$SGML_XML" == "sgml" ]; then
+	if [ -n "$SGML_NORM" ] && [ "$SGML_XML" = "sgml" ]; then
 	  SGML_NORM_OPTS='-d'
 	else
 	  SGML_NORM=cat
