From fa216270692e1e2234dbc5dce479decbfc0643c1 Mon Sep 17 00:00:00 2001
From: =?utf-8?q?Stephan=20S=C3=BCrken?= <absurd@olurdix.de>
Date: Mon, 15 Feb 2016 18:00:07 +0100
Subject: [PATCH] dput/uploader.py: Make -u/--unchecked switch work.

---
 dput/uploader.py | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/dput/uploader.py b/dput/uploader.py
index 4589f48..19cb7f1 100644
--- a/dput/uploader.py
+++ b/dput/uploader.py
@@ -287,6 +287,9 @@ def invoke_dput(changes, args):
             full_upload_log = args.full_upload_log
         _write_upload_log(tmp_logfile.name, full_upload_log)
 
+    if "unchecked" in args and args.unchecked:
+        profile['allow_unsigned_uploads'] = True
+
     if args.delayed:
         make_delayed_upload(profile, args.delayed)
 
-- 
2.7.0

