--- Makefile.orig	2011-07-24 19:34:16.000000000 -0400
+++ Makefile	2011-07-24 19:33:02.000000000 -0400
@@ -588,7 +588,8 @@
 
 # Get the list of udebs to install.
 # HACK Alert: pkg-lists/ is still sorted by TYPE instead of a dir hierarchy.
-UDEBS = $(shell set -e; get-packages udeb update >&2; pkg-list $(TYPE) "$(DRIVER_FOR)" $(KERNEL_FLAVOUR) $(KERNELMAJOR) "$(SUBARCH)" $(KERNELIMAGEVERSION)) $(EXTRAS)
+KERNELIMAGEVERSION = $(shell set -e; get-packages udeb update >&2; grep Kernel-Version apt.udeb/state/lists/*_debian_dists_$(USE_UDEBS_FROM)_main_debian-installer_binary-$(ARCH)_Packages | head -1 | cut -d' ' -f2)
+UDEBS = $(shell set -e; pkg-list $(TYPE) "$(DRIVER_FOR)" $(KERNEL_FLAVOUR) $(KERNELMAJOR) "$(SUBARCH)" $(KERNELIMAGEVERSION)) $(EXTRAS)
 
 # Get all required udebs and put them in UDEBDIR.
 $(STAMPS)get_udebs-$(targetstring)-stamp: sources.list.udeb
