diff -u bootp-2.4.3/debian/postrm bootp-2.4.3/debian/postrm
--- bootp-2.4.3/debian/postrm
+++ bootp-2.4.3/debian/postrm
@@ -2,7 +2,9 @@
 # $Id: postrm,v 1.1 1999/03/14 04:26:30 herbert Exp $
 
 if [ "$1" = remove -o "$1" = purge ]; then
-	update-inetd --remove "bootps		dgram	udp	wait	root	/usr/sbin/bootpd	bootpd -i -t 120"
+	if command -v update-inetd >/dev/null 2>&1; then
+		update-inetd --remove "bootps		dgram	udp	wait	root	/usr/sbin/bootpd	bootpd -i -t 120"
+	fi
 fi
 
 #DEBHELPER#
