diff -ru bind9-9.7.0.dfsg.P1.pristine/debian/control bind9-9.7.0.dfsg.P1/debian/control
--- bind9-9.7.0.dfsg.P1.pristine/debian/control	2010-04-27 19:45:57.000000000 +0100
+++ bind9-9.7.0.dfsg.P1/debian/control	2010-04-27 19:44:18.000000000 +0100
@@ -5,6 +5,7 @@
 Uploaders: Bdale Garbee <bdale@gag.com>
 Build-Depends: libkrb5-dev, debhelper (>= 5), libssl-dev, libtool, bison, libdb-dev (>>4.6), libldap2-dev, libxml2-dev, libcap2-dev [!kfreebsd-i386 !kfreebsd-amd64 !hurd-i386], hardening-wrapper, libgeoip-dev (>= 1.4.6.dfsg-5)
 Build-Conflicts: libdb4.2-dev
+Homepage: http://www.isc.org/software/bind
 Standards-Version: 3.7.3.0
 XS-Vcs-Browser: http://git.debian.org/?p=users/lamont/bind9.git
 XS-Vcs-Git: git://git.debian.org/~lamont/bind9.git
@@ -43,11 +44,12 @@
 Conflicts: host, dnsutils (<< 1:9.0.0)
 Replaces: dnsutils (<< 1:9.0.0)
 Provides: host
-Description: Version of 'host' bundled with BIND 9.X
- This package provides the 'host' program in the form that is bundled with
- the BIND 9.X sources.  This version differs from the one provided in the
- package called host, which is from NIKHEF, and has a similar but different
- set of features/options.
+Description: DNS lookup utility         
+ This package provides "host", a simple utility for performing DNS
+ lookups. It is normally used to convert names to IP addresses and vice
+ versa.
+ .
+ This is the version bundled with the BIND 9.X sources.
 
 Package: libbind-dev
 Section: libdevel
