diff -ru e2fsprogs-1.40.8.pristine/debian/control e2fsprogs-1.40.8/debian/control
--- e2fsprogs-1.40.8.pristine/debian/control	2008-02-29 14:56:44.000000000 +0000
+++ e2fsprogs-1.40.8/debian/control	2008-05-26 02:08:44.000000000 +0100
@@ -213,9 +213,10 @@
 Conflicts: dump (<< 0.4b4-4), quota (<< 1.55-8.1), initscripts (<< 2.85-4), sysvinit (<< 2.85-4)
 Replaces: hurd (<= 20040301-1), libblkid1 (<< 1.38+1.39-WIP-2005.12.10-2), libuuid1 (<< 1.38+1.39-WIP-2005.12.10-2)
 Architecture: any
-Description: ext2 file system utilities and libraries
- EXT2 stands for "Extended Filesystem", version 2. It's the main
- filesystem type used for hard disks on Debian and other Linux systems.
+Description: file system utilities
+ The ext2, ext3 and ext4 file systems are successors of the original ext
+ ("extended") file system. They are the main file system types used for
+ hard disks on Debian and other Linux systems.
  .
- This package contains programs for creating, checking, and maintaining EXT2
- filesystems, and the generic `fsck' wrapper.
+ This package contains programs for creating, checking, and maintaining
+ ext-based file systems, and the generic fsck wrapper.
