diff -Nwru dillo-0.7.3/debian/dillo.1 dillo-0.7.3-he/debian/dillo.1
--- dillo-0.7.3/debian/dillo.1	1970-01-01 01:00:00.000000000 +0100
+++ dillo-0.7.3-he/debian/dillo.1	2003-10-23 12:41:39.000000000 +0200
@@ -0,0 +1,38 @@
+.TH DILLO 1 
+.SH NAME
+dillo \- GTK-based web browser
+.SH SYNOPSIS
+.B dillo
+[url]
+.SH DESCRIPTION
+This manual page documents briefly the
+.BR dillo
+command.
+This manual page was written for the Debian GNU/Linux distribution
+because the original program does not have a manual page.
+.PP
+.B dillo
+is based on
+.B gzilla
+and is a free browser which uses the GTK.  It should be considered
+"early alpha" software.
+.SH OPTIONS
+There are no options.  Pass a URL on the command line to open the
+browser at that location.
+.SH EXAMPLES
+.I /usr/share/doc/dillo/dillorc
+is a sample
+.IR ~/.dillo/dillorc .
+.SH FILES
+.I ~/.dillo/dillorc
+.br
+.I ~/.dillo/bookmarks.html
+.SH AUTHOR
+.B dillo
+was created by the Dillo project.  More information can be found on
+the project home page:
+.IP
+http://dillo.sourceforge.net
+.PP
+This man page was created by Ron Farrer <rbf@debian.org> for the
+Debian GNU/Linux distribution.
diff -Nwru dillo-0.7.3/debian/dillo.1x dillo-0.7.3-he/debian/dillo.1x
--- dillo-0.7.3/debian/dillo.1x	2003-10-23 11:55:23.000000000 +0200
+++ dillo-0.7.3-he/debian/dillo.1x	1970-01-01 01:00:00.000000000 +0100
@@ -1,38 +0,0 @@
-.TH DILLO 1 
-.SH NAME
-dillo \- GTK-based web browser
-.SH SYNOPSIS
-.B dillo
-[url]
-.SH DESCRIPTION
-This manual page documents briefly the
-.BR dillo
-command.
-This manual page was written for the Debian GNU/Linux distribution
-because the original program does not have a manual page.
-.PP
-.B dillo
-is based on
-.B gzilla
-and is a free browser which uses the GTK.  It should be considered
-"early alpha" software.
-.SH OPTIONS
-There are no options.  Pass a URL on the command line to open the
-browser at that location.
-.SH EXAMPLES
-.I /usr/share/doc/dillo/dillorc
-is a sample
-.IR ~/.dillo/dillorc .
-.SH FILES
-.I ~/.dillo/dillorc
-.br
-.I ~/.dillo/bookmarks.html
-.SH AUTHOR
-.B dillo
-was created by the Dillo project.  More information can be found on
-the project home page:
-.IP
-http://dillo.sourceforge.net
-.PP
-This man page was created by Ron Farrer <rbf@debian.org> for the
-Debian GNU/Linux distribution.
diff -Nwru dillo-0.7.3/debian/rules dillo-0.7.3-he/debian/rules
--- dillo-0.7.3/debian/rules	2003-10-23 11:55:23.000000000 +0200
+++ dillo-0.7.3-he/debian/rules	2003-10-23 12:47:09.000000000 +0200
@@ -30,7 +30,7 @@
 	dh_installchangelogs
 	dh_installmenu
 #	dh_installcron
-	dh_installmanpages -pdillo
+	dh_installman debian/dillo.1
 	install debian/dillo.desktop debian/tmp/usr/share/gnome/apps/Internet/
 	dh_movefiles
 	dh_strip
