diff -wru dillo-0.7.3/debian/changelog dillo-0.7.3-he/debian/changelog
--- dillo-0.7.3/debian/changelog	2003-10-23 11:55:23.000000000 +0200
+++ dillo-0.7.3-he/debian/changelog	2003-10-23 13:12:18.000000000 +0200
@@ -1,3 +1,13 @@
+dillo (0.7.3-1+he) unstable; urgency=low
+
+  * Bumped Standards-Version number to 3.6.1 (no changes)
+  * Fixed some lintian warnings:
+    - Now installs the Changelog vie dh_installchangelogs to the right place.
+    - Added a pointer to the GPL license to the copyright file.
+    - Corrected permissions for the dillo.desktop file.
+
+ -- Marc Brockschmidt <marc@dch-faq.de>  Thu, 23 Oct 2003 12:58:12 +0200
+
 dillo (0.7.3-1) unstable; urgency=low
 
   * New upstream release, closes: #205199
diff -wru dillo-0.7.3/debian/control dillo-0.7.3-he/debian/control
--- dillo-0.7.3/debian/control	2003-10-23 11:55:23.000000000 +0200
+++ dillo-0.7.3-he/debian/control	2003-10-23 12:58:09.000000000 +0200
@@ -2,7 +2,7 @@
 Section: web
 Priority: extra
 Maintainer: Phil Blundell <pb@debian.org>
-Standards-Version: 3.1.1.1
+Standards-Version: 3.6.1
 Build-Depends: debhelper, libgtk1.2-dev, libglib1.2-dev, libpng2-dev, zlib1g-dev, libjpeg62-dev, automake, autoconf
 
 Package: dillo
diff -wru dillo-0.7.3/debian/copyright dillo-0.7.3-he/debian/copyright
--- dillo-0.7.3/debian/copyright	2003-10-23 11:55:23.000000000 +0200
+++ dillo-0.7.3-he/debian/copyright	2003-10-23 13:06:59.000000000 +0200
@@ -20,3 +20,5 @@
 /* |   provided "as is" without express or implied warranty.           | */
 /* +-------------------------------------------------------------------+ */
 
+On Debian systems the complete text of the GPL license can be found at:
+        /usr/share/common-licenses/GPL
diff -wru dillo-0.7.3/debian/docs dillo-0.7.3-he/debian/docs
--- dillo-0.7.3/debian/docs	2003-10-23 11:55:23.000000000 +0200
+++ dillo-0.7.3-he/debian/docs	2003-10-23 13:03:16.000000000 +0200
@@ -1,4 +1,3 @@
-ChangeLog 
 ChangeLog.old
 README 
 AUTHORS
diff -wru dillo-0.7.3/debian/rules dillo-0.7.3-he/debian/rules
--- dillo-0.7.3/debian/rules	2003-10-23 11:55:23.000000000 +0200
+++ dillo-0.7.3-he/debian/rules	2003-10-23 13:02:45.000000000 +0200
@@ -27,11 +27,11 @@
 	dh_installdirs
 	dh_installdocs 
 	dh_installexamples
-	dh_installchangelogs
+	dh_installchangelogs ChangeLog
 	dh_installmenu
 #	dh_installcron
 	dh_installmanpages -pdillo
-	install debian/dillo.desktop debian/tmp/usr/share/gnome/apps/Internet/
+	install -m644 debian/dillo.desktop debian/tmp/usr/share/gnome/apps/Internet/
 	dh_movefiles
 	dh_strip
 	dh_compress
