--- /tmp/control	2011-03-05 00:07:51.362561002 +0100
+++ debian/control	2011-03-05 00:48:11.098561002 +0100
@@ -111,7 +111,7 @@
 
 Package: kppp
 Section: net
-Architecture: linux-any
+Architecture: armel powerpcspe lpia i386 ia64 alpha amd64 armeb arm avr32 hppa m32r m68k mips mipsel powerpc ppc64 s390 s390x sh3 sh3eb sh4 sh4eb sparc sparc64
 Depends: ${shlibs:Depends}, ${misc:Depends}, ppp
 Suggests: khelpcenter4
 Description: modem dialer for KDE
