diff -u docbook-simple-1.1/debian/control docbook-simple-1.1/debian/control
--- docbook-simple-1.1/debian/control
+++ docbook-simple-1.1/debian/control
@@ -13,8 +13,7 @@
 
 Package: docbook-simple
 Architecture: all
-Pre-Depends: xml-core (>= 0.12)
-Depends: ${misc:Depends}, sgml-data (>= 2.0.2), docbook-xml (>= 4.2-7)
+Depends: ${misc:Depends}, sgml-data (>= 2.0.2), docbook-xml (>= 4.2-7), xml-core (>= 0.12)
 Suggests: docbook-xsl
 Replaces: docbook-xml-simple
 Description: simplified DocBook XML Doctype and css stylesheets
diff -u docbook-simple-1.1/debian/changelog docbook-simple-1.1/debian/changelog
--- docbook-simple-1.1/debian/changelog
+++ docbook-simple-1.1/debian/changelog
@@ -1,3 +1,12 @@
+docbook-simple (1.1-4.2) unstable; urgency=high
+
+  [ Helmut Grohne ]
+  * Non-maintainer upload.
+  * Demote Pre-Depends on xml-core to Depends. It was used to fix upgrades
+    from etch to lenny, but this is no longer necessary.  Closes: #706073.
+
+ -- Julien Cristau <jcristau@debian.org>  Wed, 24 Apr 2013 16:18:01 +0200
+
 docbook-simple (1.1-4.1) unstable; urgency=low
 
   * Non-maintainer upload.
