Index: fai-guide.sgml
===================================================================
--- fai-guide.sgml	(Revision 4585)
+++ fai-guide.sgml	(Arbeitskopie)
@@ -476,6 +476,10 @@
 		  subversion repository. Also supported are svn+file,
 		  svn+http, svn+ssh, svn+https and checkouts without a user
 		  name.</p></item>
+
+		  <tag><var>git://host/path</var></tag>
+		  <item> <p>The config space checked out from a
+		  git repository, host can be empty. Also supported is git+http.</p></item>
     </taglist>
 
 	  If <var>FAI_CONFIG_SRC</var> is undefined in &fc, then the
@@ -1502,7 +1506,7 @@
 are defined. The DNS resolver configuration file is created. 
 <p>
 The location of the configuration space is defined by the variable
-<var>$FAI_CONFIG_SRC</var>. You can use NFS, cvs or svn to access the
+<var>$FAI_CONFIG_SRC</var>. You can use NFS, cvs, svn or git to access the
 configuration space. See section <ref id="isetup"> for how to set the variable.
 <!--MT: there is no info about that at id="isetup"-->
 <!--MT: config is also mounted/checked out in this task-->
