diff -Naurp cryptsetup-1.0.5.orig/debian/initramfs/cryptopensc-hook cryptsetup-1.0.5/debian/initramfs/cryptopensc-hook
--- cryptsetup-1.0.5.orig/debian/initramfs/cryptopensc-hook	2007-08-07 11:09:12.000000000 +0000
+++ cryptsetup-1.0.5/debian/initramfs/cryptopensc-hook	2007-08-07 11:09:00.000000000 +0000
@@ -27,6 +27,10 @@ fi
 
 # Install cryptroot key files into initramfs
 keys=$(sed 's/^\(.*,\|\)key=//; s/,.*//' ${DESTDIR}/conf/conf.d/cryptroot)
+
+if [ "${keys}" != "none" ]
+then
+
 if [ -z "${keys}" ] ; then
     echo $0: Missing key files in ${DESTDIR}/conf/conf.d/cryptroot >&2
     cat ${DESTDIR}/conf/conf.d/cryptroot >&2
@@ -39,6 +43,8 @@ for key in ${keys} ; do
     cp ${key} ${DESTDIR}/${key}
 done
 
+fi
+
 # Install directories needed by smartcard reading daemon, command, and
 # key-script
 for dir in etc/opensc usr/lib/pcsc var/run tmp ; do
