--- a/src/modplug/archive/arch_gzip.cxx
+++ b/src/modplug/archive/arch_gzip.cxx
@@ -13,6 +13,8 @@
 
 #include "arch_gzip.h"
 
+#include <cstring>
+
 	
 arch_Gzip::arch_Gzip(const string& aFileName)
 {
--- a/src/modplug/archive/arch_rar.cxx
+++ b/src/modplug/archive/arch_rar.cxx
@@ -15,6 +15,7 @@
 #include "arch_rar.h"
 #include <iostream>
 #include <vector>
+#include <cstring>
 	
 arch_Rar::arch_Rar(const string& aFileName)
 {
