--- /usr/bin/cdbs-edit-patch	2009-04-27 16:42:29.000000000 +0200
+++ cdbs-edit-patch	2009-04-27 17:45:00.000000000 +0200
@@ -63,9 +63,9 @@
 
 debian/rules apply-patches
 
-deb_tar_srcdir=$(sed -nr 's/^[[:space:]]*DEB_TAR_SRCDIR[[:space:]]*:?=[[:space:]]*//p' debian/rules)
-if [ -n "$deb_tar_srcdir" ]; then
-	deb_srcdir=build-tree/$deb_tar_srcdir
+deb_tar_srcdir=build-tree/$(sed -nr 's/^[[:space:]]*DEB_TAR_SRCDIR[[:space:]]*:?=[[:space:]]*//p' debian/rules)
+if ! [ -d "$deb_tar_srcdir" ]; then
+        deb_srcdir=build-tree/$(ls build-tree | head -n 1)
 fi
 
 # create new source dir
