--- debian/rules.orig	2006-03-09 20:24:11.496337170 -0500
+++ debian/rules	2006-03-09 15:45:54.415031435 -0500
@@ -56,6 +56,8 @@
 	dh_clean -k
 	dh_installdirs
 	$(MAKE) -f debian/Makefile.debian install DESTDIR=`pwd`/debian/antlr
+	install -m755 lib/python/antlr/antlr.py debian/antlr/usr/lib/python2.3/site-packages
+	install -m755 lib/python/antlr/antlr.py debian/antlr/usr/lib/python2.4/site-packages
 
 install: build
 	dh_testdir
