--- dnssec-trigger-0.12.orig/01-dnssec-trigger.in	2014-03-25 06:00:55.000000000 -0700
+++ dnssec-trigger-0.12/01-dnssec-trigger.in	2014-07-01 16:58:20.630472054 -0700
@@ -84,7 +84,7 @@
     if [ -r "$conn_zones_file" ]; then
         for domain in `cat $conn_zones_file`; do
             # Remove forward zone from unbound
-            if [ "$validate_forward_zones" == "no" ]; then
+            if [ "$validate_forward_zones" = "no" ]; then
             	unbound-control forward_remove +i $domain &> /dev/null
 	    else
             	unbound-control forward_remove $domain &> /dev/null
@@ -99,11 +99,11 @@
         rm -f $conn_zones_file &> /dev/null
     fi
 
-    if [ "$action" == "vpn-up" ] || [ "$action" == "up" ]; then
+    if [ "$action" = "vpn-up" -o "$action" = "up" ]; then
         if [ -n "$domains" ]; then
             for domain in $domains; do
                 # Add forward zone into unbound
-                if [ "$validate_forward_zones" == "no" ]; then
+                if [ "$validate_forward_zones" = "no" ]; then
                     unbound-control forward_add +i $domain $nameservers &> /dev/null
                 else
                     unbound-control forward_add $domain $nameservers &> /dev/null
@@ -112,6 +112,7 @@
                 unbound-control flush_requestlist &> /dev/null
 
                 # Create zone info file
+                mkdir -p $(dirname $conn_zones_file)
                 echo $domain >> $conn_zones_file
 
                 logger "dnssec-trigger-hook(networkmanager) $ifname $action added forward DNS zone $domain $nameservers"
