Index: pkgs.dbk
===================================================================
--- pkgs.dbk	(revision 10160)
+++ pkgs.dbk	(working copy)
@@ -1153,8 +1153,9 @@
 --compare-versions</literal>.  Be careful not to re-use a version number that
 you have already used for a previous upload, or one that conflicts with a
 binNMU. The convention is to append
-<literal>+</literal><replaceable>codename</replaceable><literal>1</literal>, e.g.
-<literal>1:2.4.3-4+lenny1</literal>, of course increasing 1 for any subsequent
+<literal>+deb</literal><replaceable>X</replaceable><literal>u1</literal> (where
+<replaceable>X</replaceable> is the major release number), e.g.
+<literal>1:2.4.3-4+deb7u1</literal>, of course increasing 1 for any subsequent
 uploads.
 </para>
 </listitem>
