--- /dev/null	2007-10-02 23:46:34.736041737 +0100
+++ debootstrap-1.0.3/README	2007-10-16 18:08:07.000000000 +0100
@@ -0,0 +1,9 @@
+				DEBOOTSTRAP MAN PAGE GENERATION
+				===============================
+
+The man page has been converted to XML with doclifter.
+After editing the XML, generate the manpage with:
+
+xsltproc --nonet http://docbook.sourceforge.net/release/xsl/current/manpages/docbook.xsl debootstrap.8.xml
+
+(Requires docbook-xsl).
