From a81ffeadac9e5a3ad6f12574c7810118abba0c40 Mon Sep 17 00:00:00 2001
From: Didier Raboud <odyx@debian.org>
Date: Mon, 5 Oct 2015 11:06:12 +0200
Subject: [PATCH] Implement point 2 of the Technical Committee decision on menu
 systems

---
 policy.sgml | 8 ++++++--
 1 file changed, 6 insertions(+), 2 deletions(-)

diff --git a/policy.sgml b/policy.sgml
index ee1e9f4..34336ee 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -8192,14 +8192,18 @@ Reloading <var>description</var> configuration...done.
 	</p>
 
         <p>
-	  Packages can, to be compatible with Debian additions to some window
-	  managers that do not support the FreeDesktop standard, also provide a
+	  Applications that are <em>not</em> registered in the desktop menu can optionally provide a
 	  <em>Debian menu</em> file, following the <em>Debian menu policy</em>,
 	  which can be found in the <tt>menu-policy</tt> files in the
 	  <tt>debian-policy</tt> package.  It is also available from the Debian
 	  web mirrors at <tt><url name="/doc/packaging-manuals/menu-policy/"
 	  id="http://www.debian.org/doc/packaging-manuals/menu-policy/"></tt>.
 	</p>
+
+        <p>
+          Applications that are registered in the desktop menu must not also
+          provide a <em>Debian menu</em> file for the same application.
+        </p>
       </sect>
 
       <sect id="mime">
-- 
2.6.0

