Index: scripts/debcheckout.pl
===================================================================
--- scripts/debcheckout.pl	(revision 2018)
+++ scripts/debcheckout.pl	(working copy)
@@ -914,6 +914,10 @@
 	    $rc = $? >> 8;
 	    print STDERR "TopGit population failed\n" if $rc != 0;
 	}
+	system("cd $wcdir && git config user.name \"$ENV{'DEBFULLNAME'}\"")
+	    if (defined($ENV{'DEBFULLNAME'}));
+	system("cd $wcdir && git config user.email \"$ENV{'DEBEMAIL'}\"")
+	    if (defined($ENV{'DEBEMAIL'}));
 	if (length $git_track) {
 	    my @heads;
 	    if ($git_track eq '*') {
