--- /etc/udev/rules.d/65_dmsetup.rules.orig	2008-10-01 19:05:55.000000000 +0200
+++ /etc/udev/rules.d/65_dmsetup.rules	2008-10-10 15:51:44.000000000 +0200
@@ -16,6 +16,9 @@
 ENV{DM_STATE_ACTIVE}!="?*",			GOTO="device_mapper_end"
 ENV{DM_TARGET_TYPES}=="|*error*",		GOTO="device_mapper_end"
 
+# Suppress creation of /dev/dm-* nodes, dmsetup creates /dev/mapper/name nodes.
+KERNEL=="dm-*",					NAME="mapper/$env{DM_NAME}", SYMLINK+="%k"
+
 IMPORT{program}="vol_id --export $tempnode"
 
 OPTIONS+="link_priority=-100"
