From 9f92e37e0b22a6d260af61fab6b6c7928d8cfbfa Mon Sep 17 00:00:00 2001
From: Andy Beverley <andy@andybev.com>
Date: Sat, 27 Dec 2014 23:51:28 +0000
Subject: [PATCH 5/6] Use correct location for deb when using install option

---
 lib/DhMakePerl/Command/make.pm |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/DhMakePerl/Command/make.pm b/lib/DhMakePerl/Command/make.pm
index 1b44c62..64a067f 100644
--- a/lib/DhMakePerl/Command/make.pm
+++ b/lib/DhMakePerl/Command/make.pm
@@ -438,7 +438,7 @@ sub install_package {
     $debname = sprintf( "%s_%s-1_%s.deb", $self->pkgname, $self->version,
         $archspec );
 
-    my $deb = $self->main_dir . "/$debname";
+    my $deb = $self->main_dir . "/../$debname";
     system("dpkg -i $deb") == 0
         || die "Cannot install package $deb\n";
 }
-- 
1.7.10.4

