Index: boinc-5.10.25/clientgui/ProjectProcessingPage.cpp
===================================================================
--- boinc-5.10.25.orig/clientgui/ProjectProcessingPage.cpp
+++ boinc-5.10.25/clientgui/ProjectProcessingPage.cpp
@@ -399,6 +399,9 @@
                 ai->email_addr = (const char*)pWAP->m_AccountInfoPage->GetAccountEmailAddress().mb_str();
                 ai->passwd = (const char*)pWAP->m_AccountInfoPage->GetAccountPassword().mb_str();
                 ai->user_name = (const char*)::wxGetUserName().mb_str();
+                if (ai->user_name.empty()) {
+                    ai->user_name = (const char*)::wxGetUserId().mb_str();
+                }
 
                 if (pWAP->m_AccountInfoPage->m_pAccountCreateCtrl->GetValue()) {
                     pDoc->rpc.create_account(*ai);
