diff -ur hal-0.5.7.1/debian/control hal-modified/debian/control
--- hal-0.5.7.1/debian/control	2006-09-02 11:35:27.000000000 +0100
+++ hal-modified/debian/control	2006-09-02 12:11:09.211997500 +0100
@@ -10,6 +10,7 @@
 Package: hal
 Architecture: any
 Depends: ${shlibs:Depends}, adduser, pciutils, usbutils, udev (>= 0.065), dbus (>= 0.60-1), lsb-base
+Recommends: acpid
 Suggests: hal-device-manager
 Description: Hardware Abstraction Layer
  HAL provides an abstract view on hardware.
diff -ur hal-0.5.7.1/debian/rules hal-modified/debian/rules
--- hal-0.5.7.1/debian/rules	2006-09-02 11:35:27.000000000 +0100
+++ hal-modified/debian/rules	2006-09-02 12:10:48.286689750 +0100
@@ -12,6 +12,7 @@
 DEB_CONFIGURE_EXTRA_FLAGS := --enable-hotplug-map  \
                              --with-pid-file=/var/run/hal/hald.pid \
                              --enable-pcmcia-support \
+                             --disable-acpi-proc \
                              --with-hwdata=/usr/share/misc
 
 binary-predeb/hal-device-manager::
