--- debian-emacs-policy	2006-11-20 22:45:46.000000000 -0500
+++ debian-emacs-policy	2006-11-20 22:52:09.000000000 -0500
@@ -236,6 +236,17 @@
      file.  emacsen-common will make sure that the install/remove
      scripts are ordered to respect inter-add-on package dependencies.
 
+  E) If an Emacs add-on package compiles any of its Emacs Lisp
+     sources, it should install the compiled bytecode files to
+     /usr/share/<flavour>/site-lisp/<package-name>.  It should also
+     create a symlink for each Emacs Lisp source file in
+     /usr/share/<emacs>/site-lisp/<package-name> and store the symlink
+     in /usr/share/<flavour>/site-lisp/<package-name>.  If any
+     byte-compiled Emacs Lisp code is stored in a subdirectory,
+     similar treatment should be used.  This ensures that Emacs will
+     be able to locate the source code for the add-on package when
+     using M-x find-function and similar functionality.
+
 7) Mandatory binary symlink
 
    Each emacsen main package must have a symlink
@@ -277,6 +288,14 @@
    (fboundp 'debian-pkg-add-load-path-item) before calling this
    function, or add a dependency on emacsen-common (>= 1.4.14).
 
+   If an Emacs add-on package compiles its Emacs Lisp sources, it
+   should only add /usr/share/<flavour>/site-lisp/<package-name> (and
+   its subdirectories of compiled code, if applicable) to the load
+   path, rather than /usr/share/<emacs>/site-lisp/<package-name>.  If
+   a subdirectory of /usr/share/<emacs>/site-lisp/<package-name>
+   contains uncompiled Emacs Lisp code, it may also be added to the
+   load path.
+
 10) Usage of autoload instead of load in the site-start.d files.
 
    It's been suggested, and is probably a good idea that maintainers
