diff --git a/debian/control b/debian/control
index fd75500..d785781 100644
--- a/debian/control
+++ b/debian/control
@@ -91,6 +91,7 @@ Build-Depends:
  libcups2-dev (>= 1.5.0),
  libevent-dev (>= 1.4.13),
  libgcrypt20-dev,
+ libpipewire-0.3-dev,
  fonts-ipafont-gothic,
  fonts-ipafont-mincho,
 
diff --git a/debian/rules b/debian/rules
index 917c6f9..a110b72 100755
--- a/debian/rules
+++ b/debian/rules
@@ -96,6 +96,8 @@ defines+=use_gio=true \
          proprietary_codecs=true \
          ffmpeg_branding=\"Chrome\" \
          fieldtrial_testing_like_official_build=true \
+         rtc_use_pipewire=true \
+         rtc_pipewire_version=0.3 \
 
 # handle parallel build options
 njobs=1
