diff -u eric-3.9.5/debian/control eric-3.9.5/debian/control
--- eric-3.9.5/debian/control
+++ eric-3.9.5/debian/control
@@ -3,7 +3,8 @@
 Priority: optional
 Maintainer: Torsten Marek <shlomme@debian.org>
 Build-Depends: debhelper(>= 5.0.38), dpatch
-Build-Depends-Indep: python (>= 2.3), python-qt3, python-qt-dev, sip4, qt3-dev-tools, python-central (>= 0.5.6)
+Build-Depends-Indep: python (>= 2.3), python-qt3, python-qt-dev, sip4,
+ qt3-dev-tools, python-central (>= 0.5.6), python-sip4-dev
 XS-Python-Version: current
 Standards-Version: 3.7.2
 
diff -u eric-3.9.5/debian/changelog eric-3.9.5/debian/changelog
--- eric-3.9.5/debian/changelog
+++ eric-3.9.5/debian/changelog
@@ -1,3 +1,12 @@
+eric (3.9.5-1.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * FTBFS: ImportError: No module named sipconfig (Closes: #453198)
+    - debian/control: sipconfig module was moved from python-sip4 to
+      python-sip4-dev, so add build-depends-indep on python-sip4-dev.
+
+ -- LI Daobing <lidaobing@gmail.com>  Sat, 15 Dec 2007 10:47:27 +0800
+
 eric (3.9.5-1) unstable; urgency=low
 
   * New upstream release
