Description: Don't stop processing packages after proper snippet
Author: Jean-Michel Vourgère <jmv_deb@nirgal.com>
Forwarded: not-needed
Last-Update: 2012-01-04

--- debian/extra/debhelper/dh_bash-completion	2012-06-17 21:08:08.000000000 +0200
+++ /usr/bin/dh_bash-completion	2012-07-09 23:20:33.000000000 +0200
@@ -38,7 +38,7 @@
 my $srcdir = '.';
 $srcdir = $dh{SOURCEDIR}."/" if defined $dh{SOURCEDIR};
 
-foreach my $package (@{$dh{DOPACKAGES}}) {
+PKG: foreach my $package (@{$dh{DOPACKAGES}}) {
 	next if is_udeb($package);
 
 	my $tmp = tmpdir($package);
@@ -80,7 +80,7 @@
 					warning "file-list parsing failed, installing as proper snippet";
 
 					doit("install", "-p", "-m644", $completions, "$bc_dir/$package");
-					exit 0
+					next PKG
 				}
 				push @filelist, @found;
 			}
