diff --git a/debian/powerbtn.sh b/debian/powerbtn.sh
index fc5bb98..1b2c673 100644
--- a/debian/powerbtn.sh
+++ b/debian/powerbtn.sh
@@ -10,7 +10,7 @@ test -f /var/lock/acpisleep && exit 0
 # them handle policy This is effectively the same as 'acpi-support's
 # '/usr/share/acpi-support/policy-funcs' file.
 
-if pidof gnome-power-manager kpowersave > /dev/null ||
+if pidof gnome-power-manager kpowersave xfce4-power-manager > /dev/null ||
   (pidof dcopserver > /dev/null && test -x /usr/bin/dcop && /usr/bin/dcop kded kded loadedModules | grep -q klaptopdaemon) ; then
     exit
 fi
