--- Packages-arch-specific.txt	2009-08-01 03:43:47.000000000 +0200
+++ Packages-arch-specific.txt_NEW	2009-08-01 04:02:41.000000000 +0200
@@ -37,6 +37,8 @@
 atari-bootstrap: m68k                                                 # m68k specific
 atitvout: i386							      # i386 specific
 %athcool: i386 kfreebsd-i386					      # Athlon powersave util
+%autofs: !kfreebsd-amd64 !kfreebsd-i386 !hurd-i386		      # Linux-specific
+%autofs5: !kfreebsd-amd64 !kfreebsd-i386 !hurd-i386		      # Linux-specific
 autorun4linuxcd: i386 amd64					      # MS Windows specific
 %avifile: i386 amd64						      # i386 Win32 DLLs needed/executed
 %avrprog: amd64 i386 kfreebsd-amd64 kfreebsd-i386 hurd-i386	      # inb/outb (ab)use
