diff -u chkrootkit-0.47/chkrootkit chkrootkit-0.47-new/chkrootkit
--- chkrootkit-0.47/chkrootkit  2007-08-17 15:42:02.000000000 +0100
+++ chkrootkit-0.47-new/chkrootkit      2007-08-17 15:41:13.000000000 
+0100
@@ -705,7 +705,7 @@
    if [ "${QUIET}" != "t" ]; then printn \
       "Searching for suspicious files and dirs, it may take a while... 
"; fi

-   files=`${find} ${DIR} -name ".[A-Za-z]*" -o -name "...*" -o -name 
".. *"`
+   files=`${find} ${DIR} \( -name ".[A-Za-z]*" -o -name "...*" -o -name 
".. *" \) -and -not -empty`
    dirs=`${find} ${DIR} -type d -name ".*"`
    if [ "${files}" = "" -a "${dirs}" = "" ]
       then

