? build-arch-stamp
? debian
? local.diff
Index: alsa-oss.c
===================================================================
RCS file: /cvsroot/alsa/alsa-oss/alsa-oss.c,v
retrieving revision 1.41
diff -p -u -r1.41 alsa-oss.c
--- alsa-oss.c	1 Dec 2003 14:06:23 -0000	1.41
+++ alsa-oss.c	27 Jan 2004 17:01:31 -0000
@@ -250,7 +250,9 @@ static int oss_dsp_hw_params(oss_dsp_t *
 			if (err < 0)
 				return err;
 			if (dsp->maxfrags > 0) {
-				unsigned int periods_max = dsp->maxfrags;
+				unsigned int periods_max = periods_min > dsp->maxfrags
+							 ? periods_min
+							 : dsp->maxfrags;
 				err = snd_pcm_hw_params_set_periods_max(pcm, hw,
 									&periods_max, 0);
 				if (err < 0)
