From 45cc7d295cb818a2c236687d9d395d4712e500ce Mon Sep 17 00:00:00 2001
From: Ian Jackson <ijackson@chiark.greenend.org.uk>
Date: Mon, 17 Oct 2016 17:31:38 +0100
Subject: [PATCH 1/4] changelog: Finalise 2.2

Signed-off-by: Ian Jackson <ijackson@chiark.greenend.org.uk>
---
 debian/changelog | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/changelog b/debian/changelog
index 203a864..b8811e0 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -5,7 +5,7 @@ dgit (2.2) unstable; urgency=low
   * Detect SIGPIPE (and SIGCHLD) being blocked or ignored.
     Closes:#841085.
 
- --
+ -- Ian Jackson <ijackson@chiark.greenend.org.uk>  Mon, 17 Oct 2016 17:31:18 +0100
 
 dgit (2.1) unstable; urgency=low
 
-- 
2.9.3

