--- /root/local-top_lvm2.ok	2010-02-10 08:11:32.000000000 +0100
+++ /usr/share/initramfs-tools/scripts/local-top/lvm2	2010-02-10 18:16:08.000000000 +0100
@@ -35,6 +35,11 @@
 		lvm vgchange -aly --ignorelockingfailure
 		exit 0
 		;;
+	# Test to activate swap located in LVMÂ for hibernation resume :
+	UUID=*)
+		lvm vgchange -aly --ignorelockingfailure
+		exit 0
+		;;
 	esac
 
 	# Make sure that we have a d-m path
