--- /etc/init.d/amavis	2004/12/01 01:55:56	1.1
+++ /etc/init.d/amavis	2005/04/16 03:25:17
@@ -16,11 +16,16 @@
 DESC=amavisd
 PARAMS=""

+if [ -r "/etc/default/${DAEMONNAME}" ]; then
+    . "/etc/default/${DAEMONNAME}"
+fi
+PIDFILE=${PIDFILE:-"/var/run/amavis/${NAME}.pid"}
+
 test -f ${DAEMON} || exit 0

 set -e

-START="--start --quiet --pidfile /var/run/amavis/${NAME}.pid --name ${DAEMONNAME} --startas ${DAEMON} -- ${PARAMS}"
+START="--start --quiet --pidfile ${PIDFILE} --name ${DAEMONNAME} --startas ${DAEMON} -- ${PARAMS}"

 createdir() {
 # $1 = user
@@ -71,7 +76,7 @@
 	;;
   stop)
 	echo -n "Stopping $DESC: "
-	if start-stop-daemon --stop --quiet --pidfile /var/run/amavis/$NAME.pid \
+	if start-stop-daemon --stop --quiet --pidfile "${PIDFILE}" \
 		--name $DAEMONNAME --retry 10 >/dev/null ; then
 		echo "amavisd-new."
 		cleanup
@@ -87,8 +92,8 @@
 	;;
   reload)
 	echo "Reloading $DESC configuration files."
-	start-stop-daemon --stop --signal 1 --quiet --pidfile \
-		/var/run/amavis/$NAME.pid --name $DAEMONNAME
+	start-stop-daemon --stop --signal 1 --quiet --pidfile "${PIDFILE}" \
+		--name $DAEMONNAME
   	;;
   restart|force-reload)
 	$0 stop
