diff -Nru dput-0.9.4ubuntu2/debian/changelog dput-0.9.4ubuntu3/debian/changelog
--- dput-0.9.4ubuntu2/debian/changelog	2009-06-19 11:16:04.000000000 +0200
+++ dput-0.9.4ubuntu3/debian/changelog	2010-01-11 23:08:15.000000000 +0100
@@ -1,3 +1,9 @@
+dput (0.9.4ubuntu3) lucid; urgency=low
+
+  * Compute md5sum.
+
+ -- Loïc Minier <loic.minier@ubuntu.com>  Mon, 11 Jan 2010 23:08:08 +0100
+
 dput (0.9.4ubuntu2) karmic; urgency=low
 
   * Don't incorrectly assume that HTTPResonse.reason is an integer when
diff -Nru dput-0.9.4ubuntu2/debian/rules dput-0.9.4ubuntu3/debian/rules
--- dput-0.9.4ubuntu2/debian/rules	2009-03-29 12:53:31.000000000 +0200
+++ dput-0.9.4ubuntu3/debian/rules	2010-01-11 23:08:46.000000000 +0100
@@ -55,6 +55,7 @@
 	gzip -9 $(TMPDIR)/usr/share/man/man1/*
 	gzip -9 $(TMPDIR)/usr/share/man/man5/*
 	gzip -9 $(TMPDIR)/usr/share/doc/$(PACKAGE)/changelog
+	cd $(TMPDIR); find -type f \! -regex '.*/DEBIAN/.*' -printf '%P\0' | xargs -r0 md5sum >DEBIAN/md5sums
 	install --mode=0644 debian/conffiles $(TMPDIR)/DEBIAN
 	install --mode=0755 debian/prerm $(TMPDIR)/DEBIAN
 	install --mode=0755 debian/postinst $(TMPDIR)/DEBIAN
