*** alsa-utils.orig	2010-11-18 13:30:54.523870664 +0100
--- alsa-utils	2010-11-18 13:28:54.951868973 +0100
*************** sanify_levels_on_card()
*** 206,211 ****
--- 206,219 ----
  	# On MacBookPro5,3 and later models (See Bug#597791)
  	unmute_and_set_level "Front Speaker" "80%"
  
+ 	# On Intel 82801H (See Bug#603550)
+ 	unmute_and_set_level "Master" "100%"
+ 	unmute_and_set_level "PCM" "100%"
+ 	unmute_and_set_level "Speaker" "80%"
+ 	unmute_and_set_level "Capture" "100%"
+ 	unmute_and_set_level "Mic Boost" "0%"
+ 	unmute_and_set_level "Front Mic Boost" "0%"
+ 	unmute_and_set_level "Digital" "50%"
  	return 0
  }
  
