diff -ru5 aptitude-orig/Makefile.am aptitude-fixed/Makefile.am
--- aptitude-orig/Makefile.am	2004-06-13 17:03:31.000000000 +0200
+++ aptitude-fixed/Makefile.am	2004-06-13 17:12:32.000000000 +0200
@@ -1,20 +1,22 @@
 MAINTAINERCLEANFILES=Makefile.in aclocal.m4 config.h.in
 
 SUBDIRS = src po intl doc
-EXTRA_DIST = FAQ README.i18n aptitude-hackers-guide.txt help.txt aclocal.m4 aptitude.1 \
-	aptitude.cs.1 aptitude.fi.1 aptitude.fr.1 aptitude.gl.1 aptitude.it.1 aptitude.pl.1 help-cs.txt \
-	help-fr.txt help-fi.txt help-gl.txt help-it.txt help-pl.txt help-pt_BR.txt \
-	help-zh_CN.txt function_groups function_pkgs aptitude-defaults .todo
+TLMANPAGES = aptitude.1 aptitude.??.1
+TLHELPTXTS = help.txt help-??.txt help-??_??.txt
+EXTRA_DIST = FAQ README.i18n aptitude-hackers-guide.txt aclocal.m4 \
+	function_groups function_pkgs aptitude-defaults .todo \
+	$(TLMANPAGES) $(TLHELPTXTS)
 
 pkgdatadir=$(datadir)/$(PACKAGE)
 
-pkgdata_DATA=help.txt help-cs.txt help-fi.txt help-fr.txt help-gl.txt help-it.txt help-pl.txt help-pt_BR.txt help-zh_CN.txt \
-	COPYING README FAQ NEWS function_groups function_pkgs aptitude-defaults
+pkgdata_DATA = $(TLHELPTXTS) COPYING README FAQ NEWS \
+	function_groups function_pkgs aptitude-defaults
 
 install-data-local:
 	[ -d $(DESTDIR)$(STATEDIR) ] || mkdir $(DESTDIR)$(STATEDIR)
 # FIXME: this really ought to use the autoconf directory variables
 
 uninstall-local:
 	rm -fr $(DESTDIR)$(STATEDIR)
 # Is -r a good idea here?
+
diff -ru5 aptitude-orig/src/mine/Makefile.am aptitude-fixed/src/mine/Makefile.am
--- aptitude-orig/src/mine/Makefile.am	2004-06-13 16:58:17.000000000 +0200
+++ aptitude-fixed/src/mine/Makefile.am	2004-06-13 17:14:17.000000000 +0200
@@ -2,10 +2,11 @@
 
 localedir = $(datadir)/locale
 INCLUDES = -Wall @WERROR@ -I.. -I. -I$(top_srcdir)/lib -I../../../intl
 DEFS = -DLOCALEDIR=\"$(localedir)\" @DEFS@
 LDADD = @INTLLIBS@
+SWEEPERHELPTXTS = mine-help.txt mine-help-??.txt
 
 noinst_LIBRARIES = libcmine.a
 
 libcmine_a_SOURCES=	\
 	cmine.cc	\
@@ -13,9 +14,8 @@
 	board.cc	\
 	board.h
 
 pkgdatadir=$(datadir)/$(PACKAGE)
 
-pkgdata_DATA=mine-help.txt mine-help-fi.txt mine-help-it.txt
-
-EXTRA_DIST=mine-help.txt mine-help-fi.txt mine-help-it.txt
+pkgdata_DATA = $(SWEEPERHELPTXTS)
+EXTRA_DIST = $(SWEEPERHELPTXTS)
 
