diff -ruN easytag-1.99.5.orig/src/easytag.c easytag-1.99.5/src/easytag.c
--- easytag-1.99.5.orig/src/easytag.c	2005-06-06 01:04:44.000000000 +0200
+++ easytag-1.99.5/src/easytag.c	2005-06-12 00:12:34.000000000 +0200
@@ -468,6 +468,7 @@
     GtkWidget *Icon;
     GtkWidget *VBox;
     GtkWidget *hbox;
+    GList *focusable_widgets_list = NULL;
     //GtkWidget *ScrollWindow;
     //GtkTextBuffer *TextBuffer;
     GtkEntryCompletion *completion;
@@ -860,7 +862,41 @@
     g_signal_connect_swapped(G_OBJECT(OrigArtistEntry), "activate",G_CALLBACK(gtk_widget_grab_focus),G_OBJECT(CopyrightEntry));
     g_signal_connect_swapped(G_OBJECT(CopyrightEntry),  "activate",G_CALLBACK(gtk_widget_grab_focus),G_OBJECT(URLEntry));
     g_signal_connect_swapped(G_OBJECT(URLEntry),        "activate",G_CALLBACK(gtk_widget_grab_focus),G_OBJECT(EncodedByEntry));
-    g_signal_connect_swapped(G_OBJECT(URLEntry),        "activate",G_CALLBACK(gtk_widget_grab_focus),G_OBJECT(TitleEntry));
+    g_signal_connect_swapped(G_OBJECT(EncodedByEntry),  "activate",G_CALLBACK(gtk_widget_grab_focus),G_OBJECT(TitleEntry));
+
+    // Set focus chain
+    focusable_widgets_list = g_list_append(focusable_widgets_list,TitleEntry);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,TitleMButton);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,ArtistEntry);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,ArtistMButton);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,AlbumEntry);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,AlbumMButton);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,DiscNumberEntry);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,DiscNumberMButton);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,YearEntry);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,YearMButton);
+    //focusable_widgets_list = g_list_append(focusable_widgets_list,TrackMButtonSequence); // Doesn't work as focus disabled for this widget to have enought space to display icon
+    focusable_widgets_list = g_list_append(focusable_widgets_list,TrackEntryCombo);
+    //focusable_widgets_list = g_list_append(focusable_widgets_list,TrackMButtonNbrFiles);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,TrackTotalEntry);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,TrackMButton);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,GenreCombo);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,GenreMButton);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,CommentEntry);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,CommentMButton);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,ComposerEntry);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,ComposerMButton);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,OrigArtistEntry);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,OrigArtistMButton);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,CopyrightEntry);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,CopyrightMButton);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,URLEntry);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,URLMButton);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,EncodedByEntry);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,EncodedByMButton);
+    focusable_widgets_list = g_list_append(focusable_widgets_list,TitleEntry); // To loop to the beginning
+    gtk_container_set_focus_chain(GTK_CONTAINER(Table),focusable_widgets_list);
+
 
 
     /*
