diff --git a/plugins/translated-desc.py b/plugins/translated-desc.py
index fe80772..79b5f6a 100644
--- a/plugins/translated-desc.py
+++ b/plugins/translated-desc.py
@@ -1,7 +1,7 @@
 import apt
 import xapian
 import re
-import os, os.path
+import os, os.path, urllib
 from debian_bundle import deb822
 
 APTLISTDIR="/var/lib/apt/lists"
@@ -31,7 +31,7 @@ class Indexer:
 
         # Read the translated descriptions
         self.descs = dict()
-        desckey = "Description-"+self.lang
+        desckey = "Description-"+urllib.unquote(self.lang)
         for pkg in deb822.Deb822.iter_paragraphs(open(file)):
             self.descs[pkg["Package"]] = pkg[desckey]
 
