diff -urN cuetools-1.3.1~/src/lib/cd.c cuetools-1.3.1/src/lib/cd.c
--- cuetools-1.3.1~/src/lib/cd.c	2012-02-05 23:35:20.000000000 -0500
+++ cuetools-1.3.1/src/lib/cd.c	2012-02-05 23:36:51.000000000 -0500
@@ -125,7 +125,7 @@
 
 Track *cd_add_track (Cd *cd)
 {
-	if (MAXTRACK - 1 > cd->ntrack)
+	if (MAXTRACK > cd->ntrack)
 		cd->ntrack++;
 	else
 		fprintf(stderr, "too many tracks\n");
@@ -261,7 +261,7 @@
 
 void track_add_index (Track *track, long index)
 {
-	if (MAXTRACK - 1 > track->nindex)
+	if (MAXINDEX > track->nindex)
 		track->nindex++;
 	else
 		fprintf(stderr, "too many indexes\n");
