Index: broadcom-sta-5.60.48.36-3/amd64/src/wl/sys/wl_iw.h
===================================================================
--- broadcom-sta-5.60.48.36-3.orig/amd64/src/wl/sys/wl_iw.h	2011-02-04 17:37:23.000000000 +0100
+++ broadcom-sta-5.60.48.36-3/amd64/src/wl/sys/wl_iw.h	2011-02-04 17:37:31.000000000 +0100
@@ -16,6 +16,7 @@
 #define _wl_iw_h_
 
 #include <linux/wireless.h>
+#include <linux/semaphore.h>
 
 #include <typedefs.h>
 #include <proto/ethernet.h>
Index: broadcom-sta-5.60.48.36-3/i386/src/wl/sys/wl_iw.h
===================================================================
--- broadcom-sta-5.60.48.36-3.orig/i386/src/wl/sys/wl_iw.h	2011-02-04 17:37:06.000000000 +0100
+++ broadcom-sta-5.60.48.36-3/i386/src/wl/sys/wl_iw.h	2011-02-04 17:37:20.000000000 +0100
@@ -16,6 +16,7 @@
 #define _wl_iw_h_
 
 #include <linux/wireless.h>
+#include <linux/semaphore.h>
 
 #include <typedefs.h>
 #include <proto/ethernet.h>
