From 0c1d6a90bc380d970b4d0004eb2bac667ecb70d5 Mon Sep 17 00:00:00 2001
From: Dandan Zhang <zhangdandan@loongson.cn>
Date: Thu, 20 Apr 2023 17:59:59 +0800
Subject: [PATCH] Add binutils loongarch64 package to the control file

---
 debian/control | 26 ++++++++++++++++++++++++++
 1 file changed, 26 insertions(+)

diff --git a/debian/control b/debian/control
index e5ab9d8e..a17acb8e 100644
--- a/debian/control
+++ b/debian/control
@@ -1108,3 +1108,29 @@ Multi-Arch: foreign
 Depends: binutils-mipsisa64r6el-linux-gnuabi64 (= ${binary:Version})
 Description: GNU binary utilities, for mipsisa64r6el-linux-gnuabi64 target (debug symbols)
  This package provides debug symbols for binutils-mipsisa64r6el-linux-gnuabi64.
+
+Package: binutils-loongarch64-linux-gnu
+Priority: optional
+Architecture: loongarch64 riscv64 amd64 i386 x32 arm64 ppc64el
+Multi-Arch: allowed
+Depends: binutils-common (= ${binary:Version}),
+  ${shlibs:Depends}, ${extraDepends}
+Suggests: binutils-doc (= ${source:Version})
+Provides: 
+Breaks: binutils (<< 2.29-6), binutils-dev (<< 2.38.50.20220609-2)
+Replaces: binutils (<< 2.29-6), binutils-dev (<< 2.38.50.20220609-2)
+Description: GNU binary utilities, for loongarch64-linux-gnu target
+ This package provides GNU assembler, linker and binary utilities
+ for the loongarch64-linux-gnu target.
+ .
+ You don't need this package unless you plan to cross-compile programs
+ for loongarch64-linux-gnu and loongarch64-linux-gnu is not your native platform.
+
+Package: binutils-loongarch64-linux-gnu-dbg
+Section: debug
+Priority: optional
+Architecture: loongarch64 riscv64 amd64 i386 x32 arm64 ppc64el
+Multi-Arch: foreign
+Depends: binutils-loongarch64-linux-gnu (= ${binary:Version})
+Description: GNU binary utilities, for loongarch64-linux-gnu target (debug symbols)
+ This package provides debug symbols for binutils-loongarch64-linux-gnu.
-- 
2.39.1

