From 05561bf0e918d66eac808b4ff52ed3e70409ef97 Mon Sep 17 00:00:00 2001
From: Dandan Zhang <zhangdandan@loongson.cn>
Date: Thu, 11 May 2023 19:24:21 +0800
Subject: [PATCH] Add loongarch64 support

---
 debian/rules | 7 +++++--
 1 file changed, 5 insertions(+), 2 deletions(-)

diff --git a/debian/rules b/debian/rules
index 2e1eac2c..5f5a8b27 100755
--- a/debian/rules
+++ b/debian/rules
@@ -118,7 +118,7 @@ install_script = install -m 755
 install_binary = install -m 755 -s --strip-program="$(STRIP)"
 
 NATIVE_ARCHS ?= amd64 i386 arm64 armhf armel ppc64el s390x
-NATIVE_ARCHS += alpha arc hppa ia64 m68k powerpc ppc64 \
+NATIVE_ARCHS += alpha arc hppa ia64 loong64 m68k powerpc ppc64 \
 	riscv64 sh4 sparc64 x32
 NATIVE_ARCHS += hurd-i386 kfreebsd-amd64 kfreebsd-i386
 #NATIVE_ARCHS += nios2 or1k s390 sparc
@@ -139,7 +139,7 @@ ifeq ($(DEB_SOURCE),binutils)
     # DEB_HOST_ARCH is filtered-out later anyway, do not test here.
     CROSS_ARCHS ?= amd64 i386 x32 \
                    s390x ppc64el arm64 armhf armel \
-                   alpha arc hppa m68k \
+                   alpha arc hppa loong64 m68k \
                    powerpc ppc64 sh4 sparc64 \
                    ia64 riscv64 \
                    kfreebsd-amd64 kfreebsd-i386 hurd-i386
@@ -228,6 +228,8 @@ HOST_ARCHS_kfreebsd-amd64 = amd64 i386 x32
 HOST_ARCHS_kfreebsd-i386 = amd64 i386 x32
 HOST_ARCHS_hurd-i386 = amd64 i386 x32
 
+HOST_ARCHS_loong64 = amd64 i386 x32 arm64 ppc64el
+
 # Map a Debian architecture alias to a GNU type or a multiarch path component.
 run_dpkg_arch = $(or $(dpkg_arch_$1),$(eval \
   dpkg_arch_$1 := $(shell dpkg-architecture -f -a$1))$(dpkg_arch_$1))
@@ -740,6 +742,7 @@ multiarch_targets = \
 	hppa-linux-gnu \
 	i686-linux-gnu \
 	ia64-linux-gnu \
+	loongarch64-linux-gnu \
 	m32r-linux-gnu \
 	m68k-linux-gnu \
 	m68k-rtems \
-- 
2.39.1

