From 487d89ae4f50d12c23fd931b90567c8e197cc38c Mon Sep 17 00:00:00 2001
From: Josh Triplett <josh@joshtriplett.org>
Date: Sat, 28 Apr 2012 15:30:25 -0700
Subject: [PATCH 2/3] Move mail-transport-agent from Recommends to Suggests.
 (Closes: 599667)

---
 debian/changelog |    1 +
 debian/control   |    4 ++--
 2 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/debian/changelog b/debian/changelog
index eff252a..b1e955a 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -2,6 +2,7 @@ apt-listchanges (2.85.10) UNRELEASED; urgency=low
 
   [ Josh Triplett ]
   * Avoid throwing an exception when sendmail does not exist. (Closes: 666086)
+  * Move mail-transport-agent from Recommends to Suggests. (Closes: 599667)
 
  -- Sandro Tosi <morph@debian.org>  Sat, 28 Apr 2012 14:58:35 -0700
 
diff --git a/debian/control b/debian/control
index a8c4854..4c33361 100644
--- a/debian/control
+++ b/debian/control
@@ -14,8 +14,8 @@ Package: apt-listchanges
 Architecture: all
 Depends: ${python:Depends}, apt (>= 0.5.3), python-apt (>= 0.7.93),
  ucf (>= 0.28), debianutils (>= 2.0.2), ${misc:Depends}
-Recommends: exim4 | mail-transport-agent
-Suggests: x-terminal-emulator, www-browser, python-glade2, python-gtk2
+Suggests: x-terminal-emulator, www-browser, python-glade2, python-gtk2,
+ exim4 | mail-transport-agent
 Description: package change history notification tool
  The tool apt-listchanges can compare a new version of a
  package with the one currently installed and show what has been
-- 
1.7.10

