From 4cc831dbb17656d459aaeccbba74f889ad348bd9 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Picca=20Fr=C3=A9d=C3=A9ric-Emmanuel?= <picca@synchrotron-soleil.fr>
Date: Sat, 20 Nov 2010 15:45:06 -0500
Subject: [PATCH] * do not compress javascript files under /usr/share/doc

---
 dh_compress |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/dh_compress b/dh_compress
index 14dfdb3..a2c504d 100755
--- a/dh_compress
+++ b/dh_compress
@@ -116,6 +116,7 @@ foreach my $package (@{$dh{DOPACKAGES}}) {
 				! -iname "*-gz"  ! -iname "*-z" ! -iname "*_z" \\
 				! -iname "*.jar" ! -iname "*.zip" ! -iname "*.css" \\
 				! -iname "*.svg" ! -iname "*.svgz" \\
+				! -iname "*.js" \\
 				! -name "index.sgml" \\
 				! -name "copyright" 2>/dev/null || true;
 			find usr/share/fonts/X11 -type f -name "*.pcf" 2>/dev/null || true;
-- 
1.7.2.3

