diff -urN devscripts-2.10.59.old/scripts/deb-reversion.sh devscripts-2.10.59/scripts/deb-reversion.sh
--- devscripts-2.10.59.old/scripts/deb-reversion.sh	2009-12-06 14:22:58.000000000 +0100
+++ devscripts-2.10.59/scripts/deb-reversion.sh	2009-12-06 14:23:52.000000000 +0100
@@ -188,7 +188,7 @@
 {
   cd ..
   dpkg-deb -b package >/dev/null
-  dpkg-name package.deb | sed -e 's,.*to `\(.*\).,\1,'
+  dpkg-name package.deb | sed -e "s,.*to '\(.*\).,\1,"
 }
 
 [ -z "${OLD_VERSION:-}" ] && OLD_VERSION="$(get_version $DEB)"
