diff -uNrp cyrus-sasl2-2.1.23.dfsg1~/debian/control cyrus-sasl2-2.1.23.dfsg1/debian/control
--- cyrus-sasl2-2.1.23.dfsg1~/debian/control	2009-08-20 17:26:34.000000000 -0700
+++ cyrus-sasl2-2.1.23.dfsg1/debian/control	2009-08-20 17:27:44.000000000 -0700
@@ -5,7 +5,7 @@ Maintainer: Ubuntu Core Developers <ubun
 Maintainer: Debian Cyrus SASL Team <pkg-cyrus-sasl2-debian-devel@lists.alioth.debian.org>
 Uploaders: Fabian Fagerholm <fabbe@debian.org>, Roberto C. Sanchez <roberto@connexer.com>
 Standards-Version: 3.8.0.0
-Build-Depends: debhelper (>= 5.0.0), dpatch, autotools-dev, automake1.9, autoconf, libtool, libdb4.7-dev, libpam0g-dev (>= 0.76-22), libssl-dev (>= 0.9.7e-3), libmysqlclient-dev (>= 5.0.20), libopie-dev (>= 2.32-10), libpq-dev (>= 8.1.3-4), libkrb5-dev, libsqlite0-dev (>= 2.8.16), libldap2-dev (>= 2.1.30-8), chrpath, groff-base, debconf (>= 0.5) | debconf-2.0, po-debconf, docbook-to-man
+Build-Depends: debhelper (>= 5.0.0), dpatch, autotools-dev, automake1.9, autoconf, libtool, libdb4.7-dev, libpam0g-dev (>= 0.76-22), libssl-dev (>= 0.9.7e-3), libmysqlclient-dev (>= 5.0.20), libopie-dev (>= 2.32-10), libpq-dev (>= 8.1.3-4), libkrb5-dev, libsqlite0-dev (>= 2.8.16), libldap2-dev (>= 2.1.30-8), chrpath, groff-base, debconf (>= 0.5) | debconf-2.0, po-debconf, docbook-to-man, hardening-wrapper
 Build-Conflicts: heimdal-dev
 XS-Debian-Vcs-Browser: http://svn.debian.org/wsvn/pkg-cyrus-sasl2/
 XS-Debian-Vcs-Svn: svn://svn.debian.org/svn/pkg-cyrus-sasl2/
diff -uNrp cyrus-sasl2-2.1.23.dfsg1~/debian/rules cyrus-sasl2-2.1.23.dfsg1/debian/rules
--- cyrus-sasl2-2.1.23.dfsg1~/debian/rules	2009-08-20 17:26:34.000000000 -0700
+++ cyrus-sasl2-2.1.23.dfsg1/debian/rules	2009-08-20 17:27:58.000000000 -0700
@@ -18,6 +18,7 @@
 # Debhelper control.
 export DH_ALWAYS_EXCLUDE=CVS
 #export DH_VERBOSE=1 # uncomment this to turn on verbose mode
+export DEB_BUILD_HARDENING=1
 
 # These are used for cross-compiling and for saving the configure script
 # from having to guess our platform (since we know it already).
