From 1140ba0f2a9edb23e907366179deb5791ed40e16 Mon Sep 17 00:00:00 2001
From: Jeremy Bicha <jbicha@ubuntu.com>
Date: Sun, 15 Jan 2017 09:10:57 -0500
Subject: [PATCH] Update Homepage and Vcs fields

Closes: #851484
---
 debian/control | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/debian/control b/debian/control
index 12bfc00..29e87eb 100644
--- a/debian/control
+++ b/debian/control
@@ -3,9 +3,9 @@ Section: sound
 Priority: optional
 Maintainer: Debian Accessibility Team <debian-accessibility@lists.debian.org>
 Uploaders: Samuel Thibault <sthibault@debian.org>
-Vcs-Browser: http://git.debian.org/?p=pkg-a11y/espeak.git;a=summary
-Vcs-Git: git://git.debian.org/git/pkg-a11y/espeak.git
-Homepage: http://espeak.sourceforge.net/
+Vcs-Browser: https://anonscm.debian.org/git/pkg-a11y/espeak-ng.git/
+Vcs-Git: https://anonscm.debian.org/git/pkg-a11y/espeak-ng.git/
+Homepage: https://github.com/espeak-ng/espeak-ng
 Build-Depends: debhelper (>= 9.20150628), dh-autoreconf, dh-exec,
  libpcaudio-dev, libsonic-dev,
  ruby-ronn, ruby-kramdown
-- 
2.10.2

