diff -ruN x/bind9-9.2.2+9.2.3rc3/lib/isccfg/Makefile.in bind9-9.2.2+9.2.3rc3/lib/isccfg/Makefile.in
--- x/bind9-9.2.2+9.2.3rc3/lib/isccfg/Makefile.in	Tue Sep 23 11:32:30 2003
+++ bind9-9.2.2+9.2.3rc3/lib/isccfg/Makefile.in	Tue Sep 23 11:25:33 2003
@@ -36,7 +36,7 @@
 ISCDEPLIBS =	../../lib/isc/libisc.@A@
 ISCCFGDEPLIBS =	libisccfg.@A@
 
-LIBS =		@LIBS@ -L../isc -lisc -L../dns -ldns
+LIBS =		@LIBS@ -L../dns -L../isc -lisc -ldns
 
 SUBDIRS =	include
 
