--- /usr/share/polkit-1/actions/org.freedesktop.udisks.policy.orig
+++ /usr/share/polkit-1/actions/org.freedesktop.udisks.policy
@@ -71,9 +71,9 @@
     <message xml:lang="da">Autorisering er påkrævet for at afmontere enheder monteret af en anden bruger</message>
     <message xml:lang="de">Zugriffsrechte werden benötigt um ein Gerät auszuhängen, das ein anderer Benutzer eingehängt hat</message>
     <defaults>
-      <allow_any>no</allow_any>
-      <allow_inactive>no</allow_inactive>
-      <allow_active>auth_admin</allow_active>
+      <allow_any>yes</allow_any>
+      <allow_inactive>yes</allow_inactive>
+      <allow_active>yes</allow_active>
     </defaults>
   </action>
 
@@ -112,8 +112,8 @@
     <message xml:lang="da">Autorisering er påkrævet for at skubbe medie ud af en enhed</message>
     <message xml:lang="de">Zugriffsrechte werden benötigt um das Medium aus dem Gerät auszuwerfen</message>
     <defaults>
-      <allow_any>no</allow_any>
-      <allow_inactive>no</allow_inactive>
+      <allow_any>yes</allow_any>
+      <allow_inactive>yes</allow_inactive>
       <allow_active>yes</allow_active>
     </defaults>
   </action>
@@ -124,8 +124,8 @@
     <message>Authentication is required to detach the drive</message>
     <message xml:lang="de">Zugriffsrechte werden benötigt um das Laufwerk zu trennen</message>
     <defaults>
-      <allow_any>no</allow_any>
-      <allow_inactive>no</allow_inactive>
+      <allow_any>yes</allow_any>
+      <allow_inactive>yes</allow_inactive>
       <allow_active>yes</allow_active>
     </defaults>
   </action>