--- /usr/bin/nmudiff	2008-07-26 22:59:06.000000000 +0200
+++ /tmp/nmudiff	2008-08-24 23:57:21.000000000 +0200
@@ -193,6 +193,10 @@
 fi
 
 if [ "$NMUDIFF_MUTT" = no ]; then
+    if [ -z "$FROM" ]; then
+	: ${FROMNAME:="$DEBFULLNAME"}
+	: ${FROMNAME:="$NAME"}
+    fi
     : ${FROM:="$DEBEMAIL"}
     : ${FROM:="$EMAIL"}
     if [ -z "$FROM" ]; then
@@ -200,6 +204,9 @@
 	echo "or EMAIL environment variable or using --from command line option." >&2
 	exit 1
     fi
+    if [ -n "$FROMNAME" ]; then
+	FROM="$FROMNAME <$FROM>"
+    fi
 fi
 
 if ! [ -f debian/changelog ]; then
