=== modified file 'debian/tests/control'
--- debian/tests/control	2013-02-02 11:14:27 +0000
+++ debian/tests/control	2013-04-03 12:23:51 +0000
@@ -3,9 +3,9 @@
 	 python-nose,
 	 cython,
          graphviz,
-         libc-bin (>= 2.13-1~),
-         python-all (>= 2.6.6-3~),
-         python-all-dev (>= 2.6.6-3~),
+         libc-bin,
+         python-all,
+         python-all-dev,
          python-argparse,
          python-configobj,
          python-decorator,
@@ -16,11 +16,11 @@
          python-rpy2,
          python-qt4,
          python-simplegeneric,
-         python-sphinx (>= 1.0.7+dfsg-1~),
+         python-sphinx,
          python-tk,
          python-tornado,
          python-wxgtk2.8,
-         python-zmq (>= 2.1.4),
+         python-zmq,
          xauth,
          xvfb
 
@@ -29,9 +29,9 @@
 	 python3-nose,
 	 cython3,
          graphviz,
-         libc-bin (>= 2.13-1~),
-         python3-all (>= 2.6.6-3~),
-         python3-all-dev (>= 2.6.6-3~),
+         libc-bin,
+         python3-all,
+         python3-all-dev,
          python3-decorator,
          python3-matplotlib,
          python3-pygments,
@@ -52,10 +52,10 @@
 Depends: ipython3, cython3, python3-dev, python-pexpect
 
 Tests: tools2
-Depends: ipython
+Depends: ipython, python-lxml
 
 Tests: tools3
-Depends: ipython3
+Depends: ipython3, python3-lxml
 
 Tests: incomplete-install
-Depends: ipython ipython3
+Depends: ipython, ipython3

