--- debian/ifupdown.sh.old	2019-01-16 10:27:42.000000000 +0100
+++ debian/ifupdown.sh	2020-02-07 13:10:50.822365528 +0100
@@ -31,6 +31,12 @@
   if [ "$IF_BRIDGE_HW" ]; then
     ip link set dev $IFACE address $IF_BRIDGE_HW
   fi
+# Activate VLAN filtering on VLAN aware bridges
+  if [ "$IF_BRIDGE_VLAN_AWARE" = "yes" ]; then
+    ip link set dev $IFACE type bridge vlan_filtering 1
+  else
+    ip link set dev $IFACE type bridge vlan_filtering 0
+  fi
 # Wait for the ports to become available
   if [ "$IF_BRIDGE_WAITPORT" ]
   then
