From cf5d2cd81bd9e39cfd6312a85351ca40dea88f85 Mon Sep 17 00:00:00 2001
From: Arto Jantunen <viiru@debian.org>
Date: Wed, 2 Aug 2023 11:37:00 +0300
Subject: [PATCH 2/2] Take emacs-bin-common contents from pgtk

---
 debian/rules | 14 +++++++-------
 1 file changed, 7 insertions(+), 7 deletions(-)

diff --git a/debian/rules b/debian/rules
index b42d2634a42d..1d76f18c9ece 100755
--- a/debian/rules
+++ b/debian/rules
@@ -399,7 +399,7 @@ endef
 
 override_dh_auto_install: $(autogen_install_files)
 	rm -rf \
-	  $(install_dir_gtk) $(install_dir_nox) $(install_dir_lucid) \
+	  $(install_dir_pgtk) $(install_dir_nox) $(install_dir_lucid) \
 	  $(pkgdir_common)/* \
 	  $(pkgdir_bin_common)/* \
 	  $(pkgdir_gtk)/* \
@@ -408,13 +408,13 @@ override_dh_auto_install: $(autogen_install_files)
 	  $(pkgdir_lucid)/* \
 	  $(pkgdir_el)/*
 
-	$(call emacs_inst,build-gtk,$(install_dir_gtk))
+	$(call emacs_inst,build-pgtk,$(install_dir_pgtk))
 
         ##################################################
         # emacs-common
         ifneq (,$(findstring emacs-common, $(shell dh_listpackages)))
 	  install -d $(pkgdir_common)
-	  cp -a $(install_dir_gtk)/* $(pkgdir_common)
+	  cp -a $(install_dir_pgtk)/* $(pkgdir_common)
 
 	  rm -r $(pkgdir_common)/usr/bin
 	  rm \
@@ -488,8 +488,8 @@ override_dh_auto_install: $(autogen_install_files)
         ifneq (,$(findstring emacs-bin-common, $(shell dh_listpackages)))
 	  # Move common binaries to emacs-bin-common.
 	  install -d $(pkgdir_bin_common)/usr
-	  cp -a $(install_dir_gtk)/usr/bin $(pkgdir_bin_common)/usr
-	  cp -a $(install_dir_gtk)/usr/libexec $(pkgdir_bin_common)/usr
+	  cp -a $(install_dir_pgtk)/usr/bin $(pkgdir_bin_common)/usr
+	  cp -a $(install_dir_pgtk)/usr/libexec $(pkgdir_bin_common)/usr
 
 	  # Make sure there's just one.
 	  test -f $(pkgdir_bin_common)/usr/bin/emacs-*
@@ -517,6 +517,7 @@ override_dh_auto_install: $(autogen_install_files)
         ##################################################
         # emacs-gtk
         ifneq (,$(findstring emacs, $(shell dh_listpackages)))
+	  $(call emacs_inst,build-gtk,$(install_dir_gtk))
 	  $(call install_common_binpkg_bits,\
 	    $(install_dir_gtk),$(pkgdir_gtk),emacs-gtk,gtk)
 
@@ -530,8 +531,7 @@ override_dh_auto_install: $(autogen_install_files)
 
         ##################################################
         # emacs-pgtk
-        ifneq (,$(findstring emacs-pgtk, $(shell dh_listpackages)))
-	  $(call emacs_inst,build-pgtk,$(install_dir_pgtk))
+        ifneq (,$(findstring emacs, $(shell dh_listpackages)))
 	  $(call install_common_binpkg_bits,\
 	    $(install_dir_pgtk),$(pkgdir_pgtk),emacs-pgtk,pgtk)
 
-- 
2.40.1

