--- a/build-rdeps
+++ b/build-rdeps
@@ -155,6 +155,8 @@
 my $opt_hostarch;
 my $opt_without_ceve;
 my $opt_quiet;
+my $opt_noarchall;
+my $opt_noarchany;
 
 sub version {
     print <<"EOT";
@@ -190,6 +192,8 @@
    --exclude-component COMPONENT  Ignore the specified component (can be given multiple times)
    --host-arch                    Set the host architecture (requires dose-extra >= 4.0)
    --build-arch                   Set the build architecture (requires dose-extra >= 4.0)
+   --no-arch-all                  Ignore Build-Depends-Indep
+   --no-arch-any                  Ignore Build-Depends-Arch
    --old                          Use the old simple reverse dependency resolution
 
 EOT
@@ -336,6 +340,8 @@
                 "--deb-host-arch=$opt_hostarch",
                 "deb://$hostarch_file");
         }
+        push(@ceve_cmd, "--deb-drop-b-d-indep") if ($opt_noarchall);
+        push(@ceve_cmd, "--deb-drop-b-d-arch") if ($opt_noarchany);
         my %sources;
         print STDERR 'DEBUG: executing: ' . join(' ', @ceve_cmd)
           if ($opt_debug);
@@ -415,6 +421,8 @@
     "origin=s"            => \$opt_origin,
     "host-arch=s"         => \$opt_hostarch,
     "build-arch=s"        => \$opt_buildarch,
+    "no-arch-all"         => \$opt_noarchall,
+    "no-arch-any"         => \$opt_noarchany,
     #   "profiles=s" => \$opt_profiles, # FIXME: add build profile support
     #                                            once dose-ceve has a
     #                                            --deb-profiles option
