--- /tmp/dh_compress	2008-06-06 12:11:03.703221059 +0200
+++ /usr/bin/dh_compress	2008-06-06 12:25:30.000000000 +0200
@@ -102,6 +102,7 @@
 				! -iname "*.tgz" ! -iname "*.z" ! -iname "*.bz2" \\
 				! -iname "*-gz"  ! -iname "*-z" ! -iname "*_z" \\
 				! -iname "*.jar" ! -iname "*.zip" ! -iname "*.css" \\
+				! -name "index.sgml" \\
 				! -name "copyright" 2>/dev/null || true;
 			find usr/X11R6/lib/X11/fonts usr/share/fonts/X11 -type f -name "*.pcf" 2>/dev/null || true;
 		`);
