--- debian-security-support-2014.05.16.pristine/debian/control	2014-05-16 20:10:28.000000000 +0100
+++ debian-security-support-2014.05.16/debian/control	2014-05-18 15:22:20.022375030 +0100
@@ -17,11 +17,14 @@
 Depends: ${misc:Depends},
     adduser,
     gettext-base,
-Description: Identify installed packages with ended/limited security support
- For some Debian packages, maintaining security support is not
- feasible for the planned life cycle. For other packages, security
- support is limited, it does not cover the full feature set.
+Description: security support coverage checker
+ For some packages, it is not feasible to maintain full security
+ support for all use cases through the full distribution release
+ cycle.
  .
- This package provides a program to identify installed packages
- where support had to be ended prematurely or is limited, and alerts
- the administrator in that case.
+ This package provides a program to identify installed packages for
+ which support has had to be limited or prematurely ended, and to
+ alert the administrator.
+ .
+ New versions of this package with updated checklists are provided
+ via security updates.
