--- debian-security-support-2014.05.16.pristine/check-support-status	2014-04-28 19:50:42.000000000 +0100
+++ debian-security-support-2014.05.16/check-support-status	2014-05-17 17:12:11.753387884 +0100
@@ -202,21 +202,22 @@
     case "$TYPE" in
         ended)
             gettext \
-"Security support has ended for one or more packages
+"Ended security support for one or more packages
 
-Unfortunately, security support for some packages needed to be stopped
-before the end of the regular security maintenance life cycle.
+Unfortunately, it has been necessary to end security support for some
+packages before the end of the regular security maintenance life cycle.
 
-The following packages found on your system are affected by this:"
+The following packages found on this system are affected by this:"
             echo
             ;;
         limited)
             gettext \
-"Security support is limited for one or more packages
+"Limited security support for one or more packages
 
-Unfortunately, security support for some packages had to be limited.
+Unfortunately, it has been necessary to limit security support for some
+packages.
 
-The following packages found on your system are affected by this:"
+The following packages found on this system are affected by this:"
             echo
             ;;
     esac
