From 2753269f15ce92781479bae63fc8ab1ace95f0a1 Mon Sep 17 00:00:00 2001
From: Tarek Soliman <tarek@bashasoliman.com>
Date: Sun, 29 Aug 2010 20:31:27 -0500
Subject: [PATCH] checkrestart: added exception for /drm

---
 checkrestart |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/checkrestart b/checkrestart
index 63aa7c4..fd70e30 100755
--- a/checkrestart
+++ b/checkrestart
@@ -268,6 +268,9 @@ def isdeletedFile (f):
     # Or about files under /tmp
     if f.startswith('/tmp/'):
         return 0
+    # Or about files under /drm
+    if f.startswith('/drm'):
+        return 0
     # Or about files under /var/tmp
     if f.startswith('/var/tmp/'):
         return 0
-- 
1.7.1

