diff --git a/dglob.pod b/dglob.pod
index 6a483b3..e304b21 100644
--- a/dglob.pod
+++ b/dglob.pod
@@ -24,3 +24,3 @@ this behavior (see L<"OPTIONS">).
 If you use dglob with the B<-f> option, all files in the matched packages
-are listed instead of their names. If you do not use de B<-a> switch,
+are listed instead of their names. If you do not use the B<-a> switch,
 only existing, plain (i.e. no symlinks, directories or other special ones)
