diff -Naur debian/control debian-new/control
--- debian/control	2010-06-10 14:55:31.000000000 +0000
+++ debian-new/control	2010-06-10 14:40:51.000000000 +0000
@@ -5,7 +5,7 @@
 Uploaders: Miriam Ruiz <little_miry@yahoo.es>, Barry deFreese <bdefreese@debian.org>,
  Bas Wijnen <wijnen@debian.org>
 Build-Depends: debhelper (>= 5.0.37.2), automake, autoconf (>= 2.53),
- gettext, quilt, libsdl1.2-dev, libsdl-image1.2-dev, libsdl-mixer1.2-dev, cvs
+ gettext, quilt, libsdl1.2-dev, libsdl-image1.2-dev, libsdl-mixer1.2-dev, autopoint
 Standards-Version: 3.8.3
 Homepage: http://www.losersjuegos.com.ar/juegos/ceferino/ceferino.php
 Vcs-Svn: svn://svn.debian.org/svn/pkg-games/packages/trunk/ceferino/
