From bdea538d86d91f2e36cd4875344ef140f05d1f33 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Sat, 15 Oct 2016 03:38:14 +0200
Subject: [PATCH] Remove -Zxz which is the default now

---
 lib/Debian/Debhelper/Sequence/golang.pm | 5 -----
 1 file changed, 5 deletions(-)

diff --git a/lib/Debian/Debhelper/Sequence/golang.pm b/lib/Debian/Debhelper/Sequence/golang.pm
index 2307c09..72e3781 100644
--- a/lib/Debian/Debhelper/Sequence/golang.pm
+++ b/lib/Debian/Debhelper/Sequence/golang.pm
@@ -5,9 +5,4 @@ use Debian::Debhelper::Dh_Lib;
 
 insert_before('dh_gencontrol', 'dh_golang');
 
-# XXX: -u is deprecated, but we cannot use “-- -Zxz” because additional command
-# options will be appended (“-O--buildsystem=golang”), resulting in
-# “dh_builddeb -- -Zxz -O--buildsystem=golang”, which fails.
-add_command_options('dh_builddeb', '-u-Zxz');
-
 1
-- 
2.9.3

