*** awesome-3.4.6_old/awesomerc.lua.in	2010-10-16 11:44:20.000000000 +0200
--- awesome-3.4.6/awesomerc.lua.in	2010-10-16 11:43:20.187203020 +0200
***************
*** 56,62 ****
  -- {{{ Menu
  -- Create a laucher widget and a main menu
  myawesomemenu = {
!    { "manual", terminal .. " -e man awesome" },
     { "edit config", editor_cmd .. " " .. awful.util.getdir("config") .. "/rc.lua" },
     { "restart", awesome.restart },
     { "quit", awesome.quit }
--- 56,62 ----
  -- {{{ Menu
  -- Create a laucher widget and a main menu
  myawesomemenu = {
!    { "manual", terminal .. " -e 'man awesome'" },
     { "edit config", editor_cmd .. " " .. awful.util.getdir("config") .. "/rc.lua" },
     { "restart", awesome.restart },
     { "quit", awesome.quit }
