From f1ced2ced6c8f6f17c06112f5d28d48ea4b2c40f Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lo=C3=AFc=20Minier?= <lool@debian.org>
Date: Mon, 17 Oct 2011 17:20:23 +0200
Subject: [PATCH] Drop lcd4linux; built on all arches anyway

---
 Packages-arch-specific |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/Packages-arch-specific b/Packages-arch-specific
index 9358f9c..63b6f08 100644
--- a/Packages-arch-specific
+++ b/Packages-arch-specific
@@ -103,7 +103,6 @@ kon2: i386                                                            # Hardcode
 %ia32-libs: amd64 ia64						      # ia32 compat libs for amd64,ia64
 %ia32-libs-gtk: amd64 ia64					      # ia32 compat libs for amd64,ia64
 %ia32-libs-core: ia64						      # ia32 compat libs, ia64-only bundle
-%lcd4linux: amd64 armel i386					      # <sys/io.h>
 %lcdproc: !s390 !s390x						      # this hardware cannot be attached
 %ldc: !s390 !s390x						      # needs porting
 %libacpi: amd64 i386 ia64 lpia					      # acpi is i386/ia64 specific
-- 
1.7.5.4

