From 194953bfc726feccb8c3e86c8ac92e5b0f44e334 Mon Sep 17 00:00:00 2001
From: Martin Zobel-Helas <zobel@debian.org>
Date: Wed, 28 Dec 2011 13:51:01 +0100
Subject: [PATCH 1/2] add DSA meeting 2010 in Munich

---
 data/events/conferences |    5 +++++
 1 files changed, 5 insertions(+), 0 deletions(-)

diff --git a/data/events/conferences b/data/events/conferences
index 5858f41..3776d7e 100644
--- a/data/events/conferences
+++ b/data/events/conferences
@@ -162,6 +162,11 @@ Start-Date: Aug 1 2010
 End-Date: Aug 7 2010
 Source: http://debconf10.debconf.org/
 
+Title: DSA meeting
+Start-Date: Sep 3 2010
+End-Date: Sep 5 2010
+Source: http://lists.debian.org/debian-project/2010/09/msg00027.html
+
 Title: FOSDEM 2011
 Start-Date: Feb 5 2011
 End-Date: Feb 6 2011
-- 
1.7.7.3

