--- apt-show-versions~	2022-10-15 09:52:39.000000000 +0000
+++ apt-show-versions	2023-07-11 10:10:03.205293265 +0000
@@ -267,7 +267,10 @@
         my ($href, $release) = &parse_file ($_);
         foreach my $pkg (keys %$href) {
 	    foreach my $arch (keys %{$href->{$pkg}}) {
-                $apackages->{$pkg}{$arch}{$release} = $href->{$pkg}{$arch};
+                $apackages->{$pkg}{$arch}{$release} = $href->{$pkg}{$arch} if
+                  (!defined $apackages->{$pkg}{$arch}{$release} ||
+                   $vs->compare($apackages->{$pkg}{$arch}{$release}{$VERS},
+                                $href->{$pkg}{$arch}{$VERS}) < 0);
             }
 	}
     }
