From 1355ccd20a42c33c8349d3efb07b0f3879d43431 Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Fri, 7 Apr 2017 08:18:17 +0800
Subject: [PATCH 1/2] Support zero delay for the various APT::Periodic
 activities

---
 debian/apt.systemd.daily | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/debian/apt.systemd.daily b/debian/apt.systemd.daily
index 03d41bae5..5e9ee4a2d 100755
--- a/debian/apt.systemd.daily
+++ b/debian/apt.systemd.daily
@@ -90,6 +90,12 @@ check_stamp()
         return 0
     fi
 
+    if [ $interval = always ]; then
+	debug_echo "check_stamp: ignoring time stamp file, interval set to always"
+	# treat as enough time has passed
+        return 0
+    fi
+
     # compare midnight today to midnight the day the stamp was updated
     stamp_file="$stamp"
     stamp=$(date --date=$(date -r $stamp_file --iso-8601) +%s 2>/dev/null)
-- 
2.11.0

