diff --git a/dbconfig-generate-include b/dbconfig-generate-include
index 039bc04..9ede85a 100755
--- a/dbconfig-generate-include
+++ b/dbconfig-generate-include
@@ -405,7 +405,7 @@ esac
 
 if [ "$outputfile" ]; then
 	if [ "$do_ucf" ]; then
-		ucf "$tmpout" "$outputfile"
+		ucf --debconf-ok "$tmpout" "$outputfile" >&2
 		rm -f "$tmpout"
 	else
 		mv "$tmpout" "$outputfile"
diff --git a/debian/changelog b/debian/changelog
index 0e1751a..587e282 100644
diff --git a/debian/control b/debian/control
index 1636b39..46e4078 100644
--- a/debian/control
+++ b/debian/control
@@ -7,7 +7,7 @@ Standards-Version: 3.7.2
 
 Package: dbconfig-common
 Architecture: all
-Depends: ucf, ${misc:Depends}
+Depends: ucf (>= 0.28), ${misc:Depends}
 Suggests: virtual-mysql-client | mysql-client | postgresql-client
 Description: common framework for packaging database applications
  dbconfig-common presents a policy and implementation for
