diff -Nru approx-5.1/debian/changelog approx-5.1/debian/changelog
--- approx-5.1/debian/changelog	2012-03-07 07:54:57.000000000 +0100
+++ approx-5.1/debian/changelog	2012-03-27 18:04:54.000000000 +0200
@@ -1,3 +1,12 @@
+approx (5.1-1.2) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Fix "should depend on system-log-daemon":
+    add dependency on "rsyslog | system-log-daemon".
+    (Closes: #656791)
+
+ -- gregor herrmann <gregoa@debian.org>  Tue, 27 Mar 2012 18:04:42 +0200
+
 approx (5.1-1.1) unstable; urgency=low
 
   * Non-maintainer upload.
diff -Nru approx-5.1/debian/control approx-5.1/debian/control
--- approx-5.1/debian/control	2012-03-07 07:53:20.000000000 +0100
+++ approx-5.1/debian/control	2012-03-27 18:04:14.000000000 +0200
@@ -26,7 +26,8 @@
  adduser,
  curl,
  openbsd-inetd | inet-superserver,
- update-inetd
+ update-inetd,
+ rsyslog | system-log-daemon
 Suggests: libconfig-model-approx-perl
 Description: caching proxy server for Debian archive files
  Approx is an HTTP-based proxy server for Debian-style package archives.
