From c7610d30e75913b7b18f350515097398f559b33a Mon Sep 17 00:00:00 2001
From: David Bremner <bremner@debian.org>
Date: Sun, 11 Sep 2022 18:53:02 -0300
Subject: [PATCH] try disabling native compilation for emacsen-common.install

---
 emacsen-common.install | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/emacsen-common.install b/emacsen-common.install
index 4c7562f..71e2c73 100755
--- a/emacsen-common.install
+++ b/emacsen-common.install
@@ -14,5 +14,5 @@ rm -f \
 (cd "/usr/share/$flavor/site-lisp"
  ln -s ../../emacsen-common/debian-startup.el .)
 
-"$flavor" --no-init-file --no-site-file -batch -f batch-byte-compile \
+"$flavor" --no-init-file --no-site-file -batch -f batch-byte-compile --eval='(setq native-comp-speed -1)' \
           "/usr/share/$flavor/site-lisp/debian-startup.el"
-- 
2.35.2

