--- /usr/bin/debmirror~	2013-06-23 18:34:26.000000000 +0200
+++ /usr/bin/debmirror	2015-09-11 12:22:44.329419806 +0200
@@ -1029,7 +1029,7 @@
     foreach my $arch (@arches) {
       add_bytes("dists/$dist/$section/binary-$arch/Packages");
       add_bytes("dists/$dist/$section/binary-$arch/Packages.gz");
-      add_bytes("dists/$dist/$section/binary-$arch/Packages.bz2");
+      add_bytes("dists/$dist/$section/binary-$arch/Packages.xz");
       add_bytes("dists/$dist/$section/binary-$arch/Release");
       add_bytes("dists/$dist/$section/binary-$arch/Packages.diff/Index") unless ($diff_mode eq "none");
     }
@@ -1037,7 +1037,7 @@
     if ($do_source && $section !~ /debian-installer/) {
       add_bytes("dists/$dist/$section/source/Sources");
       add_bytes("dists/$dist/$section/source/Sources.gz");
-      add_bytes("dists/$dist/$section/source/Sources.bz2");
+      add_bytes("dists/$dist/$section/source/Sources.xz");
       add_bytes("dists/$dist/$section/source/Release");
       add_bytes("dists/$dist/$section/source/Sources.diff/Index") unless ($diff_mode eq "none");
     }
@@ -2180,7 +2180,7 @@
 	if (check_lists("$tempdir/$subdir/$file")) {
 	  if (! $slow_cpu) {
 	    system_redirect_io("gzip $gzip_options", "$tempdir/$subdir/$file", "$tempdir/$subdir/$file.gz");
-	    system_redirect_io("bzip2", "$tempdir/$subdir/$file", "$tempdir/$subdir/$file.bz2");
+	    system_redirect_io("xz", "$tempdir/$subdir/$file", "$tempdir/$subdir/$file.xz");
 	  }
 	}
       }
@@ -2190,7 +2190,7 @@
       if (check_lists("$tempdir/$subdir/$file")) {
 	if (! $slow_cpu) {
 	  system_redirect_io("gzip $gzip_options", "$tempdir/$subdir/$file", "$tempdir/$subdir/$file.gz");
-	  system_redirect_io("bzip2", "$tempdir/$subdir/$file", "$tempdir/$subdir/$file.bz2");
+	  system_redirect_io("xz", "$tempdir/$subdir/$file", "$tempdir/$subdir/$file.xz");
 	}
       }
     }
@@ -2203,7 +2203,7 @@
       if (remote_get("$subdir/$file.gz")) {
 	system_redirect_io("gzip -d", "$tempdir/$subdir/$file.gz", "$tempdir/$subdir/$file");
 	if (! $slow_cpu) {
-	  system_redirect_io("bzip2", "$tempdir/$subdir/$file", "$tempdir/$subdir/$file.bz2");
+	  system_redirect_io("xz", "$tempdir/$subdir/$file", "$tempdir/$subdir/$file.xz");
 	}
       } else {
 	push (@errlog,"$subdir/$file.gz failed checksum verification\n");
@@ -2230,7 +2230,7 @@
     if (!check_lists("$tempdir/$subdir/$file")) {
       if (remote_get("$subdir/$file")) {
 	if (! $slow_cpu) {
-	  system_redirect_io("bzip2", "$tempdir/$subdir/$file", "$tempdir/$subdir/$file.bz2");
+	  system_redirect_io("xz", "$tempdir/$subdir/$file", "$tempdir/$subdir/$file.xz");
 	}
       } else {
 	push (@errlog,"$subdir/$file failed checksum verification\n");
@@ -2240,13 +2240,13 @@
       $bytes_gotten += $file_lists{"$tempdir/$subdir/$file"}{size};
     }
   }
-  if (exists $file_lists{"$tempdir/$subdir/$file.bz2"}) {
-    if (!check_lists("$tempdir/$subdir/$file.bz2")) {
-      if (!remote_get("$subdir/$file.bz2")) {
-	push (@errlog,"$subdir/$file.bz2 failed checksum verification, removing\n");
+  if (exists $file_lists{"$tempdir/$subdir/$file.xz"}) {
+    if (!check_lists("$tempdir/$subdir/$file.xz")) {
+      if (!remote_get("$subdir/$file.xz")) {
+	push (@errlog,"$subdir/$file.xz failed checksum verification, removing\n");
       }
     } else {
-      $bytes_gotten += $file_lists{"$tempdir/$subdir/$file.bz2"}{size};
+      $bytes_gotten += $file_lists{"$tempdir/$subdir/$file.xz"}{size};
     }
   }
   if (exists $file_lists{"$tempdir/$subdir/Release"}) {
@@ -2266,12 +2266,12 @@
     die "get_index called with unknown type $file\n";
   }
   $files{"$subdir/$file.gz"}=1;
-  $files{"$subdir/$file.bz2"}=1;
+  $files{"$subdir/$file.xz"}=1;
   # Uncompressed files are no longer kept on the mirrors
   $files{"$subdir/$file"}=1 unless exists $file_lists{"$tempdir/$subdir/$file.gz"};
   $files{"$subdir/Release"}=1;
   $files{"$tempdir/$subdir/$file.gz"}=1;
-  $files{"$tempdir/$subdir/$file.bz2"}=1;
+  $files{"$tempdir/$subdir/$file.xz"}=1;
   $files{"$tempdir/$subdir/$file"}=1;
   $files{"$tempdir/$subdir/Release"}=1;
 }
@@ -2370,10 +2370,10 @@
   link("$tempdir/dists/$dist/$section/$archdir/$file.gz",
        "$mirrordir/dists/$dist/$next/$section/$archdir/$file.gz")
     or die "Error while linking $tempdir/dists/$dist/$section/$archdir/$file.gz: $!\n";
-  unlink("$mirrordir/dists/$dist/$next/$section/$archdir/$file.bz2");
-  link("$tempdir/dists/$dist/$section/$archdir/$file.bz2",
-       "$mirrordir/dists/$dist/$next/$section/$archdir/$file.bz2")
-    or die "Error while linking $tempdir/dists/$dist/$section/$archdir/$file.bz2: $!\n";
+  unlink("$mirrordir/dists/$dist/$next/$section/$archdir/$file.xz");
+  link("$tempdir/dists/$dist/$section/$archdir/$file.xz",
+       "$mirrordir/dists/$dist/$next/$section/$archdir/$file.xz")
+    or die "Error while linking $tempdir/dists/$dist/$section/$archdir/$file.xz: $!\n";
 }
 
 sub i18n_from_release {
@@ -2394,7 +2394,7 @@
     next if substr($path, 0, length($compdir)) ne $compdir;
 
     my $filename = substr($path, length($compdir)+1, length($path)-length($compdir)-1);
-    next if $filename !~ /bz2$/;
+    next if $filename !~ /xz$/;
 
     my ($sha1, $size) = ($file_lists{$path}{SHA1}, $file_lists{$path}{size});
     if(!(defined($include) && ($subdir."/".$filename)=~/$include/o)) {
