diff --git a/utils/start-stop-daemon.c b/utils/start-stop-daemon.c
index 07b886b10..0b1449402 100644
--- a/utils/start-stop-daemon.c
+++ b/utils/start-stop-daemon.c
@@ -621,7 +621,7 @@ daemonize(void)
 		fatal("unable to do first fork");
 	else if (pid) { /* First Parent. */
 		if (notify_await) {
-			/* Wait for a readinnes notification from the second
+			/* Wait for a readiness notification from the second
 			 * child, so that we can safely exit when the service
 			 * is up. */
 			wait_for_notify(notify_fd);
@@ -731,7 +731,7 @@ usage(void)
 "                                  scheduler (default prio is 4)\n"
 "  -k, --umask <mask>            change the umask to <mask> before starting\n"
 "  -b, --background              force the process to detach\n"
-"      --notify-await            wait for a readinnes notification\n"
+"      --notify-await            wait for a readiness notification\n"
 "      --notify-timeout <int>    timeout after <int> seconds of notify wait\n"
 "  -C, --no-close                do not close any file descriptor\n"
 "  -m, --make-pidfile            create the pidfile before starting\n"
@@ -1124,7 +1124,7 @@ parse_options(int argc, char * const *argv)
 		{ "iosched",	  1, NULL, 'I'},
 		{ "umask",	  1, NULL, 'k'},
 		{ "background",	  0, NULL, 'b'},
-		{ "notify-away",  0, NULL, OPT_NOTIFY_AWAIT},
+		{ "notify-await", 0, NULL, OPT_NOTIFY_AWAIT},
 		{ "notify-timeout", 0, NULL, OPT_NOTIFY_TIMEOUT},
 		{ "no-close",	  0, NULL, 'C'},
 		{ "make-pidfile", 0, NULL, 'm'},
