--- apache2-2.2.6/debian/apache2.2-common.apache2.init
+++ apache2-2.2.6/debian/apache2.2-common.apache2.init
@@ -72,7 +72,7 @@
 
     PIDS2=""
 
-    for PFILE in `grep ^PidFile /etc/apache2/* -r | awk '{print $2}'`; do
+    for PFILE in `find /etc/apache2/ -type f -name '*.conf' -print0 | xargs -0 grep ^PidFile | awk '{print $2}'`; do
 	[ -e $PFILE ] && PIDS2="$PIDS2 `cat $PFILE`"
     done
 
