diff -u elinks-0.11.3/debian/control elinks-0.11.3/debian/control
--- elinks-0.11.3/debian/control
+++ elinks-0.11.3/debian/control
@@ -27,8 +27,8 @@
 Package: elinks-data
 Architecture: all
 Recommends: elinks (= ${source:Version})
-Conflicts: elinks (<= 0.11.1-1.5), elinks-lite
-Replaces: elinks (<= 0.11.1-1.5)
+Conflicts: elinks (<< 0.11.3-1), elinks-lite
+Replaces: elinks (<< 0.11.3-1)
 Description: Data files for ELinks - An advanced text-mode WWW browser
  ELinks (Enhanced Links) is a lynx-like character mode browser, created
  from the Links browser with many patches and enhancements.  It includes
@@ -42,7 +42,7 @@
 Section: doc
 Architecture: all
 Suggests: elinks
-Conflicts: elinks (<= 0.11.1-1.5), elinks-lite (<= 0.11.1-1.5)
+Conflicts: elinks (<< 0.11.3-1), elinks-lite (<< 0.11.3-1)
 Description: Documentation for ELinks - An advanced text-mode WWW browser
  ELinks (Enhanced Links) is a lynx-like character mode browser, created
  from the Links browser with many patches and enhancements.  It includes
