--- python-distutils.mk.old	2006-04-28 21:16:24.000000000 +0200
+++ python-distutils.mk	2006-04-28 21:16:41.000000000 +0200
@@ -47,7 +47,7 @@
 DEB_PYTHON_REAL_LIB_PACKAGES := $(strip $(filter $(patsubst %,python%%,$(DEB_PYTHON_VERSIONS)),$(DEB_ALL_PACKAGES)))
 # If no versioned python library packages found, grab all simpler ones
 ifeq (,$(DEB_PYTHON_REAL_LIB_PACKAGES))
-DEB_PYTHON_SIMPLE_PACKAGES := $(strip $(filter python-%,$(DEB_ALL_PACKAGES)))
+DEB_PYTHON_SIMPLE_PACKAGES := $(strip $(filter python-%,$(filter-out python-%-doc,$(DEB_ALL_PACKAGES))))
 endif
 
 cdbs_python_ver = $(filter-out -%,$(subst -, -,$(patsubst python%,%,$(cdbs_curpkg))))
