--- parse_diagram.c.orig	Thu Feb 14 22:45:04 2002
+++ parse_diagram.c	Thu Feb 14 22:44:07 2002
@@ -461,7 +461,9 @@
             id = xmlGetProp(attribute->xmlChildrenNode, "to");
         } else {
             attrname = xmlGetProp(attribute, "name");
-            if ( ! strcmp("text", attrname) ) {
+            if ( ! strcmp("text", attrname) &&
+                    attribute->xmlChildrenNode != NULL &&
+		    attribute->xmlChildrenNode->xmlChildrenNode != NULL ) {
                 name = attribute->xmlChildrenNode->xmlChildrenNode->content;
             }
             free(attrname);
@@ -470,7 +472,7 @@
     }
     implementator = find(classlist, id);
     free(id);
-    if (implementator != NULL && strlen(name) > 2) {
+    if (implementator != NULL && name != NULL && strlen(name) > 2) {
         interface = (umlclasslist) my_malloc (sizeof (umlclassnode));
         interface->key = (umlclass *) my_malloc (sizeof(umlclass));
         interface->parents = NULL;
