diff --git a/policy.sgml b/policy.sgml
index 8100da0..0482309 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -803,7 +803,7 @@
 		This contains all packages that conflict with others
 		with required, important, standard or optional
 		priorities, or are only likely to be useful if you
-		already know what they are or have specialized
+		have specialized
 		requirements (such as packages containing only detached
 		debugging symbols).
 	    </item>
