diff -u debian-edu-artwork-0.0.33/debian/changelog debian-edu-artwork-0.0.33/debian/changelog
--- debian-edu-artwork-0.0.33/debian/changelog
+++ debian-edu-artwork-0.0.33/debian/changelog
@@ -1,3 +1,12 @@
+debian-edu-artwork (0.0.33-2.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * debian/debian-edu-artwork.prerm
+    - fix "leaves diversion after upgrade from squeeze"
+      (Closes: #657474)
+
+ -- Hideki Yamane <henrich@debian.org>  Fri, 27 Jan 2012 23:51:20 +0900
+
 debian-edu-artwork (0.0.33-2) unstable; urgency=low
 
   [ Andreas B. Mundt ]
diff -u debian-edu-artwork-0.0.33/debian/debian-edu-artwork.prerm debian-edu-artwork-0.0.33/debian/debian-edu-artwork.prerm
--- debian-edu-artwork-0.0.33/debian/debian-edu-artwork.prerm
+++ debian-edu-artwork-0.0.33/debian/debian-edu-artwork.prerm
@@ -5,7 +5,13 @@
-#DEBHELPER#
-
 case "$1" in
   remove)
-    /usr/share/debian-edu-artwork/update-artwork remove
+    # avoid puring problem with version in squeeze 
+    if dpkg-divert --listpackage /usr/share/desktop-base/grub_background.sh.orig >/dev/null; then
+      dpkg-divert --package debian-edu-artwork \
+        --rename --remove /usr/share/desktop-base/grub_background.sh
+    fi
+
+    /usr/share/debian-edu-artwork/update-artwork $1
     ;;
 esac
+
+#DEBHELPER#
