From f338fa13e8a63ac6bf522ef12d4425f7199c2d3f Mon Sep 17 00:00:00 2001
From: Rafael Laboissiere <rafael@laboissiere.net>
Date: Sun, 15 Nov 2015 23:06:01 +0100
Subject: [PATCH] data/events/bugs: Update the bugs list

---
 data/events/bugs | 16 ++++++++++++++++
 1 file changed, 16 insertions(+)

diff --git a/data/events/bugs b/data/events/bugs
index bd3c684..e6770f4 100644
--- a/data/events/bugs
+++ b/data/events/bugs
@@ -133,3 +133,19 @@ Source: https://bugs.debian.org/750000
 Title: Debian Bug #760000 reported by René Mayorga
 Date: Aug 30 2014
 Source: https://bugs.debian.org/760000
+
+Title: Debian Bug #770000 reported by Martin Pitt
+Date: Nov 18 2014
+Source: https://bugs.debian.org/770000
+
+Title: Debian Bug #780000 reported by Thorsten Glaser
+Date: Mar 07 2015
+Source: https://bugs.debian.org/780000
+
+Title: Debian Bug #790000 reported by Salvatore Bonaccorso
+Date: Jun 26 2015
+Source: https://bugs.debian.org/790000
+
+Title: Debian Bug #800000 reported by Chris Lamb
+Date: Sep 25 2015
+Source: https://bugs.debian.org/800000
-- 
2.6.2

