diff --git a/checkrestart b/checkrestart
--- a/checkrestart
+++ b/checkrestart
@@ -145,7 +145,7 @@
                 diverted = m.group(3)
                 continue
             if not diverted:
-                raise 'Weird error while handling diversion'
+                raise Exception('Weird error while handling diversion')
             packagename, program = m.group(1), diverted
         else:
             packagename, program = line[:-1].split(': ')
