--- tftpd.c	2009-07-17 21:18:52.000000000 +0000
+++ tftpd.c.orig	2009-07-17 21:18:39.000000000 +0000
@@ -1123,7 +1123,7 @@
             " output messages\n"
             "  --trace                    : log all sent and received packets\n"
             "  --no-timeout               : disable 'timeout' from RFC2349\n"
-            "  --no-tsize                 : disable 'tsize' from RFC2349\n"
+            "  --no-tisize                : disable 'tsize' from RFC2349\n"
             "  --no-blksize               : disable 'blksize' from RFC2348\n"
             "  --no-multicast             : disable 'multicast' from RFC2090\n"
             "  --logfile <file>           : logfile to log logs to ;-)\n"
