--- wrap-and-sort.orig	2015-03-17 23:50:58.287803996 +0900
+++ wrap-and-sort	2015-03-17 23:53:29.259804726 +0900
@@ -117,7 +117,8 @@
     def open(self, filename):
         assert os.path.isfile(filename), "%s does not exist." % (filename)
         self.filename = filename
-        self.content = [l.strip() for l in open(filename).readlines()]
+        line_iterator = (l.strip() for l in open(filename).readlines())
+        self.content = [l for l in line_iterator if l !='']
 
     def save(self, filename=None):
         if filename:
