--- pkgtools.sgml.orig	2010-08-28 12:33:38.974272743 +0900
+++ pkgtools.sgml	2010-08-28 12:42:20.613280775 +0900
@@ -109,6 +109,10 @@
 <p>Note that you must be logged in as root to perform any commands that
 modify the system packages.
 
+<p>Note that <prgn/apt-get/ now installs recommended packages as default and is
+the preferred program for package management from console to perform system
+installation and major system upgrades for its robustness.
+
 <p>The apt tool suite also includes the <prgn/apt-cache/ tool to query
 the package lists. You can use it to find packages providing specific
 functionality through simple text or regular expression queries and
@@ -151,9 +155,7 @@
 either program:
 
 <list>
-       <item><prgn/aptitude/ offers access to all versions of a package.
-
-       <item><prgn/aptitude/ logs all its actions in <file>/var/log/aptitude</file>.
+       <item><prgn/aptitude/ offers easy access to all versions of a package.
 
        <item><prgn/aptitude/ makes it easy to keep track of obsolete
        software by listing it under "Obsolete and Locally Created
@@ -164,15 +166,6 @@
        display. Users familiar with <prgn/mutt/ will pick up quickly,
        as <prgn/mutt/ was the inspiration for the expression syntax.
 
-       <item><prgn/aptitude/ tracks which packages have been installed
-       due to dependencies and removes them automatically when the
-       packages that needed them are removed from the system.
-
-       <item><prgn/aptitude/ can automatically install
-       <em>Recommended:</em> packages<footnote>Although this can also lead to
-       systems with more packages installed than they actually need to
-       work.</footnote>.
-
        <item><prgn/aptitude/ can be used to install the predefined tasks
        available. For more information see <ref id="tasksel">.
 
@@ -189,9 +182,8 @@
 example, to install the <var>foo</var> package, you can run
 <tt>aptitude install <var>foo</var></tt>.
 
-<p>Note that <prgn/aptitude/ is the preferred program for package management
-from console both for package installations and package or system
-upgrades.
+<p>Note that <prgn/aptitude/ is the preferred program for daily package 
+management from console.
 
 <p>For more informations, read the manual page <manref
 name="aptitude" section="8"> and install the <package/aptitude-doc/
