--- /usr/share/initramfs/hooks/cryptroot
+++ /usr/share/initramfs/hooks/cryptroot
@@ -26,6 +26,7 @@
 	grep -s '^[^#]' /etc/fstab | \
 	while read device mount type options dump pass; do
 		if [ "$mount" = "/" ]; then
+			device=$(canonical_device "$device") || return 0
 			echo "$device"
 			return
 		fi
@@ -91,6 +92,8 @@
 			fi
 		fi
 
+		device=$(canonical_device "$device") || return 0
+
 		# Weed out duplicates
 		dupe=0
 		for opt in $devices; do
@@ -354,7 +357,7 @@
 	fi
 
 	# Check that it is a node under /dev/mapper/
-	nodes=$(canonical_device "$nodes") || return 0
+	# nodes=$(canonical_device "$nodes") || return 0
 
 	# Can we find this node in crypttab
 	if ! node_is_in_crypttab "$nodes"; then
