diff -Nru autokey-0.90.1/debian/changelog autokey-0.90.1/debian/changelog
--- autokey-0.90.1/debian/changelog	2012-06-03 01:06:19.000000000 -0400
+++ autokey-0.90.1/debian/changelog	2012-09-30 12:43:46.000000000 -0400
@@ -1,3 +1,15 @@
+autokey (0.90.1-1.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Update dependencies for autokey-gtk. It has been
+    ported to GTK+3 and gobject introspection (Closes: #688128).
+   - Drop depends on python-gtk2, python-gtksourceview2, python-glade2,
+     and python-notify.
+   - Add depends on python-gi, gir1.2-gtk-3.0, gir1.2-gtksource-3.0,
+     gir1.2-glib-2.0, and gir1.2-notify-0.7.
+
+ -- Andrew Starr-Bochicchio <a.starr.b@gmail.com>  Sun, 30 Sep 2012 12:43:14 -0400
+
 autokey (0.90.1-1) unstable; urgency=low
 
   * New upstream version.
diff -Nru autokey-0.90.1/debian/control autokey-0.90.1/debian/control
--- autokey-0.90.1/debian/control	2012-03-26 14:19:28.000000000 -0400
+++ autokey-0.90.1/debian/control	2012-09-30 12:41:06.000000000 -0400
@@ -42,7 +42,7 @@
 Package: autokey-gtk
 Section: gnome
 Architecture: all
-Depends: ${python:Depends}, ${misc:Depends}, python-gtk2, python-gtksourceview2, python-glade2, python-notify, python-dbus, zenity, autokey-common
+Depends: ${python:Depends}, ${misc:Depends}, python-gi, gir1.2-gtk-3.0, gir1.2-gtksource-3.0, gir1.2-glib-2.0, gir1.2-notify-0.7, python-dbus, zenity, autokey-common
 Replaces: autokey (<<0.61.4-0~0)
 Breaks: autokey (<<0.61.4-0~0)
 Description: desktop automation utility - GTK+ version
