--- atril-1.16.1_orig/debian/rules	2016-06-02 22:00:43.000000000 +0000
+++ atril-1.16.1/debian/rules	2016-12-02 08:58:00.000000000 +0000
@@ -44,8 +44,5 @@ override_dh_strip:
 	dh_strip -plibatrildocument3 --dbgsym-migration='libatrildocument3-dbg (<< 1.14.1-2~)'
 	dh_strip -plibatrilview3 --dbgsym-migration='libatrilview3-dbg (<< 1.14.1-2~)'
 
-override_dh_builddeb:
-	dh_builddeb -- -Zxz -z9
-
 get-orig-source:
 	uscan --noconf --force-download --rename --download-current-version --destdir=..
