diff -Nru efl-1.8.6/debian/changelog efl-1.8.6/debian/changelog
--- efl-1.8.6/debian/changelog	2014-05-04 13:54:55.000000000 +0200
+++ efl-1.8.6/debian/changelog	2014-09-20 14:50:26.000000000 +0200
@@ -1,3 +1,12 @@
+efl (1.8.6-2.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * GnuTLS v3 transition: Change (build-)depends to libgnutls28-dev, add b-d
+    on libgcrypt-dev. Closes: #753025
+  * efl-doc built with doxygen 1.8.7-3 to work around #762272.
+
+ -- Andreas Metzler <ametzler@debian.org>  Sat, 20 Sep 2014 14:47:56 +0200
+
 efl (1.8.6-2) unstable; urgency=medium
 
   * patches/04_eina_suite_lockup: cherry-pick f5c125 to fix eina suite lockup
diff -Nru efl-1.8.6/debian/control efl-1.8.6/debian/control
--- efl-1.8.6/debian/control	2014-05-04 13:54:55.000000000 +0200
+++ efl-1.8.6/debian/control	2014-09-20 14:46:09.000000000 +0200
@@ -15,9 +15,9 @@
  libxrandr-dev, libxext-dev, libxp-dev, libxcomposite-dev, libxi-dev,
  libxdamage-dev, x11proto-xext-dev, libxtst-dev, libglib2.0-dev,
  liblua5.1-0-dev, libdbus-1-dev, libsndfile-dev,
- libgnutls-dev, libcurl4-gnutls-dev, libc-ares-dev,
+ libgnutls28-dev, libcurl4-gnutls-dev, libc-ares-dev,
  libudev-dev [linux-any], libmount-dev [linux-any], libblkid-dev [linux-any],
- libgstreamer0.10-dev, libgstreamer-plugins-base0.10-dev
+ libgstreamer0.10-dev, libgstreamer-plugins-base0.10-dev, libgcrypt-dev
 Build-Depends-Indep: doxygen
 Standards-Version: 3.9.5
 Vcs-Git: git://anonscm.debian.org/pkg-e/libs/efl.git
@@ -72,7 +72,7 @@
 Architecture: any
 Depends: ${misc:Depends}, libeet1 (= ${binary:Version}), pkg-config,
  libeina-dev (>= ${source:Upstream-Version}),
- libjpeg-dev, libgnutls-dev
+ libjpeg-dev, libgnutls28-dev
 Recommends: efl-doc
 Description: Enlightenment DR17 file chunk reading/writing library development files
  Eet is a tiny library designed to write an arbitrary set of chunks of data to a
@@ -318,7 +318,7 @@
          libecore-x1 (= ${binary:Version}),
          libeina-dev (>= ${source:Upstream-Version}),
          libevas-dev (>= ${source:Upstream-Version}),
-         libgnutls-dev, libcurl4-gnutls-dev, libxcursor-dev, libxss-dev,
+         libgnutls28-dev, libcurl4-gnutls-dev, libxcursor-dev, libxss-dev,
          libxrender-dev, libxinerama-dev, libxrandr-dev, libxext-dev, libxi-dev,
          libxp-dev, libxcomposite-dev, libxdamage-dev, x11proto-xext-dev, libxtst-dev,
          libglib2.0-dev, libc-ares-dev, libdbus-1-dev
