diff -Nru dpkg.orig/src/help.c dpkg/src/help.c
--- dpkg.orig/src/help.c	2008-08-26 14:52:08.000000000 +0100
+++ dpkg/src/help.c	2008-11-01 01:50:50.000000000 +0000
@@ -153,6 +153,7 @@
 
   if (*instdir) {
     if (chroot(instdir)) ohshite(_("failed to chroot to `%.250s'"),instdir);
+    if (chdir("/")) ohshite(_("failed to chdir to `%.255s'"),"/");
   }
   if (f_debug & dbg_scripts) {
     fprintf(stderr,"D0%05o: fork/exec %s (",dbg_scripts,path);
