Index: checkbashisms.pl
===================================================================
--- checkbashisms.pl	(revision 1230)
+++ checkbashisms.pl	(working copy)
@@ -179,6 +179,7 @@
 		'(?:^|\s+)local\s+\w+=' =>     q<local foo=bar>,
 		'(?:^|\s+)\s*\(?\w*[^\(\w\s]+\S*?\s*[^\"]\(\)' => q<function names should only contain [a-z0-9_]>,
 		'(?:^|\s+)(push|pod)d\b' =>    q<(push|pod)d>,
+		'(?:^|\s+)export\s+-[nf]+\b' =>    q<export -[nf]>,
 	    );
 
 	    my %string_bashisms = (
