diff -u apbs-0.5.1/debian/control apbs-0.5.1/debian/control
--- apbs-0.5.1/debian/control
+++ apbs-0.5.1/debian/control
@@ -2,7 +2,7 @@
 Section: science
 Priority: optional
 Maintainer: Michael Banck <mbanck@debian.org>
-Build-Depends: debhelper (>= 4.1.0), autotools-dev, cdbs, dpatch, libmaloc-dev, refblas3-dev, bc
+Build-Depends: debhelper (>= 4.1.0), autotools-dev, cdbs, dpatch, libmaloc-dev, libblas-dev, bc, gfortran
 Standards-Version: 3.7.2
 
 Package: apbs
diff -u apbs-0.5.1/debian/changelog apbs-0.5.1/debian/changelog
--- apbs-0.5.1/debian/changelog
+++ apbs-0.5.1/debian/changelog
@@ -1,3 +1,11 @@
+apbs (0.5.1-1.1) unstable; urgency=low
+
+  * Non-maintainer upload
+  * Build depend on libblas-dev instead of refblas3-dev (closes: #462263)
+  * Add gfortran to Build Depends: to fix FTBFS in clean chroot
+
+ -- Colin Tuckley <colint@debian.org>  Mon, 28 Jan 2008 11:05:21 +0000
+
 apbs (0.5.1-1) unstable; urgency=low
 
   * New upstream release.
