--- debian/patches/05-help-dialog.dpatch.orig	2006-10-17 05:17:34.000000000 -0500
+++ debian/patches/05-help-dialog.dpatch	2006-10-17 05:21:27.000000000 -0500
@@ -57,7 +57,7 @@
 +				"Please do not bother the original authors of dvdisaster\n"
 +				"(www.dvdisaster.org) about issues with this version,\n"
 +				"but submit bugreports against the [debian package] instead.\n"),
-+				"http://packages.qa.debian.org/dvdisaster/");
++				"http://packages.qa.debian.org/dvdisaster");
  
  #else
     lang = g_getenv("LANG");
