--- debian/patches/05-help-dialog.dpatch~	2006-10-17 05:55:10.000000000 -0500
+++ debian/patches/05-help-dialog.dpatch	2006-10-17 05:53:59.000000000 -0500
@@ -53,7 +53,7 @@
 -			     "([www.dvdisaster.org]) about issues with this version.\n"),
 -		             "http://www.dvdisaster.org");
 +   AboutTextWithLink(vbox, _("\nThis version is <b>not the original</b>. It has been patched\n"
-+   				"for Debian to support encrypted DVD-ROMs.\n\n"
++   				"for Debian to support DVD-ROMs (with & without encryption).\n\n"
 +				"Please do not bother the original authors of dvdisaster\n"
 +				"(www.dvdisaster.org) about issues with this version,\n"
 +				"but submit bugreports against the [debian package] instead.\n"),
