From ac81ec40589303c9fb9893c8916dfdbcc9fa064a Mon Sep 17 00:00:00 2001
From: Davide G. M. Salvetti <salve@debian.org>
Date: Tue, 27 Dec 2011 07:45:50 +0100
Subject: [PATCH] dh_installdebconf: Fix doc typo

---
 dh_installdebconf |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/dh_installdebconf b/dh_installdebconf
index 5645884..5a20ed7 100755
--- a/dh_installdebconf
+++ b/dh_installdebconf
@@ -28,7 +28,7 @@ Note that if you use debconf, your package probably needs to depend on it
 
 Note that for your config script to be called by B<dpkg>, your F<postinst>
 needs to source debconf's confmodule. B<dh_installdebconf> does not
-install this statement into the F<postinst> automatically as it it too
+install this statement into the F<postinst> automatically as it is too
 hard to do it right.
 
 =head1 FILES
-- 
1.7.2.5

