Fixing IMDB links
Index: elinks-0.12~pre5/src/protocol/rewrite/rewrite.c
===================================================================
--- elinks-0.12~pre5.orig/src/protocol/rewrite/rewrite.c	2010-06-07 17:50:17.761937374 -0700
+++ elinks-0.12~pre5/src/protocol/rewrite/rewrite.c	2010-06-07 17:50:35.671993219 -0700
@@ -198,7 +198,7 @@
 	INIT_OPT_SMART_PREFIX("dict", "http://dictionary.reference.com/search?q=%s"),
 	INIT_OPT_SMART_PREFIX("thes", "http://thesaurus.reference.com/search?q=%s"),
 	INIT_OPT_SMART_PREFIX("a", "http://acronymfinder.com/af-query.asp?String=exact&Acronym=%s"),
-	INIT_OPT_SMART_PREFIX("imdb", "http://imdb.com/Find?%s"),
+	INIT_OPT_SMART_PREFIX("imdb", "http://www.imdb.com/find?s=all&q=%s"),
 	INIT_OPT_SMART_PREFIX("mw", "http://www.m-w.com/cgi-bin/dictionary?book=Dictionary&va=%s"),
 	INIT_OPT_SMART_PREFIX("mwt", "http://www.m-w.com/cgi-bin/thesaurus?book=Thesaurus&va=%s"),
 	INIT_OPT_SMART_PREFIX("whatis", "http://uptime.netcraft.com/up/graph/?host=%s"),
