diff --new-file -u ./apple-brightness-down.sh /etc/acpi/apple-brightness-down.sh
--- ./apple-brightness-down.sh	1970-01-01 03:00:00.000000000 +0300
+++ /etc/acpi/apple-brightness-down.sh	2013-05-15 02:20:18.828364559 +0400
@@ -0,0 +1,23 @@
+#!/bin/bash
+# TODO: Change above to /bin/sh
+
+test -f /usr/share/acpi-support/key-constants || exit 0
+
+MAX=$(cat /sys/class/backlight/gmux_backlight/max_brightness)
+BRIGHTNESS=$(cat /sys/class/backlight/gmux_backlight/brightness)
+
+STEP=$(( $MAX / 10 ))
+if [ "$STEP" -le 0 ]; then
+    STEP=1
+fi
+
+if [ "$BRIGHTNESS" -le 0 ]; then
+    exit 0
+fi
+
+BRIGHTNESS=$(( $BRIGHTNESS - $STEP ))
+if [ "$BRIGHTNESS" -le 0 ]; then
+    BRIGHTNESS=0
+fi
+
+echo $BRIGHTNESS >  /sys/class/backlight/gmux_backlight/brightness
diff --new-file -u ./apple-brightness-up.sh /etc/acpi/apple-brightness-up.sh
--- ./apple-brightness-up.sh	1970-01-01 03:00:00.000000000 +0300
+++ /etc/acpi/apple-brightness-up.sh	2013-05-15 02:19:03.374180984 +0400
@@ -0,0 +1,23 @@
+#!/bin/bash
+# TODO: Change above to /bin/sh
+
+test -f /usr/share/acpi-support/key-constants || exit 0
+
+MAX=$(cat /sys/class/backlight/gmux_backlight/max_brightness)
+BRIGHTNESS=$(cat /sys/class/backlight/gmux_backlight/brightness)
+
+STEP=$(( $MAX / 10 ))
+if [ "$STEP" -le 0 ]; then
+    STEP=1
+fi
+
+if [ "$BRIGHTNESS" -ge "$MAX" ]; then
+    exit 0
+fi
+
+BRIGHTNESS=$(( $BRIGHTNESS + $STEP ))
+if [ "$BRIGHTNESS" -gt "$MAX" ]; then
+    BRIGHTNESS="$MAX"
+fi
+
+echo $BRIGHTNESS >  /sys/class/backlight/gmux_backlight/brightness
diff --new-file -u ./apple-kbd-brightness-down.sh /etc/acpi/apple-kbd-brightness-down.sh
--- ./apple-kbd-brightness-down.sh	1970-01-01 03:00:00.000000000 +0300
+++ /etc/acpi/apple-kbd-brightness-down.sh	2013-09-04 04:23:52.109288861 +0400
@@ -0,0 +1,23 @@
+#!/bin/bash
+# TODO: Change above to /bin/sh
+
+test -f /usr/share/acpi-support/key-constants || exit 0
+
+MAX=$(cat /sys/class/leds/smc::kbd_backlight/max_brightness)
+BRIGHTNESS=$(cat /sys/class/leds/smc::kbd_backlight/brightness)
+
+STEP=$(( $MAX / 10 ))
+if [ "$STEP" -le 0 ]; then
+    STEP=1
+fi
+
+if [ "$BRIGHTNESS" -le 0 ]; then
+    exit 0
+fi
+
+BRIGHTNESS=$(( $BRIGHTNESS - $STEP ))
+if [ "$BRIGHTNESS" -le 0 ]; then
+    BRIGHTNESS=0
+fi
+
+echo $BRIGHTNESS >  /sys/class/leds/smc::kbd_backlight/brightness
diff --new-file -u ./apple-kbd-brightness-up.sh /etc/acpi/apple-kbd-brightness-up.sh
--- ./apple-kbd-brightness-up.sh	1970-01-01 03:00:00.000000000 +0300
+++ /etc/acpi/apple-kbd-brightness-up.sh	2013-09-04 04:24:29.028393391 +0400
@@ -0,0 +1,23 @@
+#!/bin/bash
+# TODO: Change above to /bin/sh
+
+test -f /usr/share/acpi-support/key-constants || exit 0
+
+MAX=$(cat /sys/class/leds/smc::kbd_backlight/max_brightness)
+BRIGHTNESS=$(cat /sys/class/leds/smc::kbd_backlight/brightness)
+
+STEP=$(( $MAX / 10 ))
+if [ "$STEP" -le 0 ]; then
+    STEP=1
+fi
+
+if [ "$BRIGHTNESS" -ge "$MAX" ]; then
+    exit 0
+fi
+
+BRIGHTNESS=$(( $BRIGHTNESS + $STEP ))
+if [ "$BRIGHTNESS" -gt "$MAX" ]; then
+    BRIGHTNESS="$MAX"
+fi
+
+echo $BRIGHTNESS >  /sys/class/leds/smc::kbd_backlight/brightness
