--- /usr/share/bash-completion/completions/cc.orig	2012-03-25 00:12:28.247673641 +0100
+++ /usr/share/bash-completion/completions/cc	2012-03-25 00:09:39.657678592 +0100
@@ -47,8 +47,8 @@
 } &&
 complete -F _gcc gcc g++ g77 gcj gpc &&
 {
-    cc  --version 2>/dev/null | grep -q GCC && complete -F _gcc cc  || :
-    c++ --version 2>/dev/null | grep -q GCC && complete -F _gcc c++ || :
+    cc  --version 2>/dev/null | grep -q 'GCC\|Debian' && complete -F _gcc cc  || :
+    c++ --version 2>/dev/null | grep -q 'GCC\|Debian' && complete -F _gcc c++ || :
 }
 
 # ex: ts=4 sw=4 et filetype=sh
