--- /usr/sbin/apt-listbugs	2006-11-03 07:54:09.000000000 +0530
+++ /tmp/apt-listbugs	2006-11-19 20:36:49.000000000 +0530
@@ -1073,7 +1073,7 @@
         config.frontend.puts " Exception: " + exception.class.to_s if $DEBUG
         config.frontend.puts " W: #{$!}"
         config.frontend.puts $intl._("Error retrieving bug reports")
-        config.frontend.puts $intl._("It appears that your network correction is down. Check it and try again") if exception.kind_of? SocketError
+        config.frontend.puts $intl._("It appears that your network connection is down. Check it and try again") if exception.kind_of? SocketError
         retrycount -= 1
         retry if config.frontend.yes_or_no?($intl._("Retry downloading bug information?")) && retrycount > 0
         raise $intl._("Exiting with error") if config.frontend.yes_or_no?($intl._("Abort the installation"))
