diff --git a/debian/control b/debian/control
index 14d1130..72aec7d 100644
--- a/debian/control
+++ b/debian/control
@@ -23,7 +23,7 @@ Breaks: exim4-daemon-light (<<${Upstream-Version}),
  exim4-daemon-custom (<<${Upstream-Version})
 Conflicts: exim, exim-tls
 Replaces: exim, exim-tls, exim4-daemon-light, exim4-daemon-heavy, exim4-daemon-custom
-Depends: ${shlibs:Depends}, ${misc:Depends}, cron | fcron, exim4-config (>=4.82) | exim4-config-2, adduser, netbase, lsb-base (>= 3.0-6)
+Depends: ${shlibs:Depends}, ${misc:Depends}, cron | anacron | fcron, exim4-config (>=4.82) | exim4-config-2, adduser, netbase, lsb-base (>= 3.0-6)
 # psmisc just for exiwhat.
 Recommends: psmisc, mailx, perl-modules
 Suggests: mail-reader, eximon4, exim4-doc-html|exim4-doc-info, 
