diff -rup debian-installer.orig/installer/build/pkg-lists/cd_drivers/common debian-installer/installer/build/pkg-lists/cd_drivers/common
--- debian-installer.orig/installer/build/pkg-lists/cd_drivers/common	2013-11-29 23:57:37.780913000 +0100
+++ debian-installer/installer/build/pkg-lists/cd_drivers/common	2013-11-30 12:37:18.882104469 +0100
@@ -3,3 +3,11 @@ cdrom-retriever
 disk-detect
 cdrom-detect
 file-preseed
+
+# This is for cdrom-detect to be able to loopmount an ISO.
+loop-modules-${kernel:Version}
+
+# This is to enable loopmount/usb-hdd from common non-fat file systems:
+ext4-modules-${kernel:Version}
+ntfs-modules-${kernel:Version}
+udf-modules-${kernel:Version}
diff -rup debian-installer.orig/installer/build/pkg-lists/cdrom/common debian-installer/installer/build/pkg-lists/cdrom/common
--- debian-installer.orig/installer/build/pkg-lists/cdrom/common	2013-11-29 23:57:37.780913000 +0100
+++ debian-installer/installer/build/pkg-lists/cdrom/common	2013-11-30 12:37:30.274111670 +0100
@@ -31,3 +31,11 @@ save-logs
 mountmedia
 
 libfribidi0-udeb
+
+# This is for cdrom-detect to be able to loopmount an ISO.
+loop-modules-${kernel:Version}
+
+# This is to enble loopmount/usb-hdd from common non-fat file systems:
+ext4-modules-${kernel:Version}
+ntfs-modules-${kernel:Version}
+udf-modules-${kernel:Version}
diff -rup debian-installer.orig/installer/build/pkg-lists/cdrom-apus/common debian-installer/installer/build/pkg-lists/cdrom-apus/common
--- debian-installer.orig/installer/build/pkg-lists/cdrom-apus/common	2013-11-29 23:57:37.780913000 +0100
+++ debian-installer/installer/build/pkg-lists/cdrom-apus/common	2013-11-30 12:37:40.694118260 +0100
@@ -24,3 +24,11 @@ save-logs
 mountmedia
 
 libfribidi0-udeb
+
+# This is for cdrom-detect to be able to loopmount an ISO.
+loop-modules-${kernel:Version}
+
+# This is to enable loopmount/usb-hdd from common non-fat file systems:
+ext4-modules-${kernel:Version}
+ntfs-modules-${kernel:Version}
+udf-modules-${kernel:Version}
