--- archive/coturn-4.5.1.1/debian/coturn.postinst	2019-05-26 16:11:04.000000000 +0200
+++ coturn-4.5.1.1/debian/coturn.postinst	2020-05-18 14:58:15.794185507 +0200
@@ -43,6 +43,7 @@
         echo "I: Creating $TURNDB_SQLITE from $TURNDB_SCHEMA" >&2
         mkdir -p "$(dirname "$TURNDB_SQLITE")"
         sqlite3 "$TURNDB_SQLITE" < "$TURNDB_SCHEMA"
+        chown "$TURNSERVER_USER:$TURNSERVER_GROUP" "$TURNDB_SQLITE"
     fi
 }
 
