--- dnssec-trigger-0.12.orig/dnssec-triggerd.service	2014-03-28 02:29:13.000000000 -0700
+++ dnssec-trigger-0.12/dnssec-triggerd.service	2014-07-01 16:55:50.220899471 -0700
@@ -8,9 +8,9 @@
 Type=simple
 Restart=always
 ExecStart=/usr/sbin/dnssec-triggerd -d
-ExecStartPre=-/usr/libexec/dnssec-trigger-script --prepare
-ExecStartPost=-/usr/libexec/dnssec-trigger-script --update
-ExecStopPost=-/usr/libexec/dnssec-trigger-script --cleanup
+ExecStartPre=-/usr/lib/x86_64-linux-gnu/dnssec-trigger-script --prepare
+ExecStartPost=-/usr/lib/x86_64-linux-gnu/dnssec-trigger-script --update
+ExecStopPost=-/usr/lib/x86_64-linux-gnu/dnssec-trigger-script --cleanup
 
 [Install]
 WantedBy=multi-user.target
