diff --git a/Debian/Debhelper/Buildsystem/makefile.pm b/Debian/Debhelper/Buildsystem/makefile.pm
index dcc5da8..2c68a83 100644
--- a/Debian/Debhelper/Buildsystem/makefile.pm
+++ b/Debian/Debhelper/Buildsystem/makefile.pm
@@ -61,7 +61,7 @@ sub exists_make_target {
 
 	return defined $output
 		&& length $output
-		&& $output !~ /\*\*\* No rule to make target `\Q$target\E'/;
+		&& $output !~ /\*\*\* No rule to make target (`|')\Q$target\E'/;
 }
 
 sub do_make {
