--- /usr/sbin/deluser.old	2008-03-23 13:52:29.000000000 +0100
+++ /usr/sbin/deluser	2008-03-23 13:56:42.000000000 +0100
@@ -198,17 +198,11 @@
 if($action eq "deluser") {
     &invalidate_nscd();
     
-    my($dummy1,$dummy2,$uid);
-
-
-
-
-
     # Don't allow a non-system user to be deleted when --system is given
     # Also, "user does not exist" is only a warning with --system, but an
     # error without --system.
     if( $config{"system"} ) {
-	if( ($dummy1,$dummy2,$uid) = getpwnam($user) ) {
+	if( my ($dummy1,$dummy2,$uid) = getpwnam($user) ) {
 	    if ( ($uid < $config{"first_system_uid"} ||
 		$uid > $config{"last_system_uid" } ) ) {
 		printf (gtx("The user `%s' is not a system user. Exiting.\n"), $user) if $verbose;
@@ -225,7 +219,7 @@
     }
     
     # Warn in any case if you want to remove the root account 
-    if ($uid == 0) {
+    if ($pw_uid == 0) {
         printf (gtx("WARNING: You are just about to delete the root account (uid 0)\n"));
         printf (gtx("Usually this is never required as it may render the whole system unusable\n"));
         printf (gtx("Press immediately Ctrl+C if you want to abort\n"));
