diff -ru acpi-support-0.131.orig/debian/acpi-support.init acpi-support-0.131/debian/acpi-support.init
--- acpi-support-0.131.orig/debian/acpi-support.init	2010-01-06 17:38:38.000000000 +0800
+++ acpi-support-0.131/debian/acpi-support.init	2010-01-06 19:05:12.000000000 +0800
@@ -24,23 +24,23 @@
 
 case "$1" in
   start)
-    log_begin_msg "Starting acpi_fakekey daemon..."
+    log_action_begin_msg "Starting acpi_fakekey daemon"
     if [ ! -d /sys/devices/virtual/misc/uinput ]; then
 	 if ! modprobe -q uinput; then
 		if ! find /lib/modules/`uname -r` -name "uinput\.*" 2>/dev/null|grep -q uinput; then
-			log_failure_msg "No suitable uinput module for running kernel found"
+			log_action_end_msg 1 "No suitable uinput module for running kernel found"
 		else
-			log_failure_msg "Modprobe uinput failed. Please use 'dmesg' to find out why"
+			log_action_end_msg 1 "Modprobe uinput failed. Please use 'dmesg' to find out why"
 		fi
 	fi
     fi
 
     if [ -d /sys/devices/virtual/misc/uinput ]; then
 	acpi_fakekeyd
+	log_action_end_msg 0
     fi
 
-    log_end_msg 0
-    log_begin_msg "Checking battery state..."
+    log_action_begin_msg "Checking battery state"
     if [ x"`echo /proc/acpi/ac_adapter/*`" != x"/proc/acpi/ac_adapter/*" ]; then 
 	/etc/acpi/power.sh
     fi
@@ -50,14 +50,14 @@
   	  . $SCRIPT
   	fi
     done
-    log_end_msg 0
+    log_action_end_msg 0
     ;;
   stop)
-    log_begin_msg "Disabling power management..."
+    log_action_begin_msg "Disabling power management"
     if [ x"`echo /proc/acpi/ac_adapter/*`" != x"/proc/acpi/ac_adapter/*" ]; then
 	/etc/acpi/power.sh false
     fi
-    log_end_msg 0
+    log_action_end_msg 0
     ;;
   restart|force-reload)
     # Doesn't make sense for that package
