--- /usr/share/docker.io/contrib/mkimage-debootstrap.sh	2014-03-26 01:00:37.000000000 +0300
+++ /tmp/mkimage-debootstrap.sh	2014-04-08 12:19:39.408053169 +0300
@@ -95,6 +95,12 @@
 	docker='sudo docker'
 elif command -v docker > /dev/null 2>&1; then
 	docker='docker'
+elif docker.io version > /dev/null 2>&1; then
+	docker='docker.io'
+elif sudo docker.io version > /dev/null 2>&1; then
+	docker='sudo docker.io'
+elif command -v docker.io > /dev/null 2>&1; then
+	docker='docker.io'
 else
 	echo >&2 "warning: either docker isn't installed, or your current user cannot run it;"
 	echo >&2 "         this script is not likely to work as expected"
