diff -upNr dnsdist-1.0.0~alpha2-orig/debian/rules dnsdist-1.0.0~alpha2/debian/rules
--- dnsdist-1.0.0~alpha2-orig/debian/rules	2016-02-24 22:57:58.000000000 +0000
+++ dnsdist-1.0.0~alpha2/debian/rules	2016-03-02 12:16:48.728183792 +0000
@@ -7,6 +7,13 @@
 DPKG_EXPORT_BUILDFLAGS = 1
 include /usr/share/dpkg/default.mk
 
+DEB_HOST_ARCH := $(shell dpkg-architecture -qDEB_HOST_ARCH)
+
+ifeq ($(DEB_HOST_ARCH),$(filter $(DEB_HOST_ARCH),mips mipsel))
+        LDFLAGS += -latomic
+        export LDFLAGS
+endif
+
 %:
 	dh $@ \
 	  --with autoreconf \
