--- toc2cue.cc.old	2002-10-06 20:53:56.000000000 +0200
+++ toc2cue.cc	2003-12-28 11:04:51.000000000 +0100
@@ -285,7 +285,7 @@
     exit(1);
   }
 
-  out << "FILE \"" << binFileName << "\" BINARY" << "\n";
+  out << "FILE \"" << binFileName << "\" BINARY" << std::endl;
 
   long offset = 0;
 
@@ -319,7 +319,7 @@
       break;
     }
     
-    out << "\n";
+    out << std::endl;
 
     const SubTrack *strun;
     SubTrackIterator stitr(trun);
@@ -327,17 +327,17 @@
 
     for (strun = stitr.first(); strun != NULL; strun = stitr.next()) {
       if (strun->TrackData::type() == TrackData::ZERODATA) {
-	out << "    PREGAP " << trun->start().str() << "\n";
+	out << "    PREGAP " << trun->start().str() << std::endl;
 	pregap = 1;
       }
       else {
 	if (!pregap && trun->start().lba() != 0) {
-	  out << "    INDEX 00 " << Msf(offset).str() << "\n";
+	  out << "    INDEX 00 " << Msf(offset).str() << std::endl;
 	  out << "    INDEX 01 " 
-	      << Msf(offset + trun->start().lba()).str() << "\n";
+	      << Msf(offset + trun->start().lba()).str() << std::endl;
 	}
 	else {
-	  out << "    INDEX 01 " << Msf(offset).str() << "\n";
+	  out << "    INDEX 01 " << Msf(offset).str() << std::endl;
 	}
 
 	offset += trun->length().lba();
