--- control.old	2012-06-14 20:35:52.000000000 +0200
+++ control	2012-08-06 15:09:37.000000000 +0200
@@ -11,13 +11,7 @@
 Depends: ${shlibs:Depends}, ${misc:Depends}, gdb, dpkg (>= 1.15.4) | install-info
 Homepage: http://cgdb.sourceforge.net/
 Description: curses-based interface to the GNU Debugger (GDB)
- CGDB is a curses  fontend to the GNU Debugger (GDB). The goal of CGDB is
- to be lightweight and responsive; not encumbered with unnecessary features.
- .
- The interface is designed to deliver the familiar GDB text interface, with a
- split screen showing the source as it executes. The UI is modeled on the
- classic Unix text editor, vi. Those familiar with vi should feel right at home
- using CGDB.
+ cgdb is a lightweight curses (terminal-based) interface to the GNU Debugger (GDB). In addition to the standard gdb console, cgdb provides a split screen view that displays the source code as it executes. The keyboard interface is modeled after vim, so vim users should feel at home using cgdb.
  .
  Some features offered by CGDB are:
    * Syntax-highlighted source window
