diff --git a/doc/users-and-groups.sgml b/doc/users-and-groups.sgml
index 5c99b70..5e9559e 100644
--- a/doc/users-and-groups.sgml
+++ b/doc/users-and-groups.sgml
@@ -159,10 +159,6 @@
 	    files</ulink>, but this was obsoleted long ago in favour of bin
 	    (now itself legacy; see above).
 	  </para>
-	  <para>
-	    I'm told that <filename>/var/spool/cups</filename> is owned by
-	    group sys, dunno why.
-	  </para>
 	</listitem>
       </varlistentry>
 
