diff -u dasher-4.11/debian/control dasher-4.11/debian/control
--- dasher-4.11/debian/control
+++ dasher-4.11/debian/control
@@ -37,8 +37,7 @@
 Depends: ${misc:Depends},
          ${shlibs:Depends},
          at-spi,
-         dasher-data (>= ${gnome:Version}),
-         dasher-data (<< ${gnome:NextVersion})
+         dasher-data (= ${source:Version})
 Description: A graphical predictive text input system
  Dasher is an information-efficient text-entry interface, driven by natural
  continuous pointing gestures. Dasher is a competitive text-entry system
diff -u dasher-4.11/debian/changelog dasher-4.11/debian/changelog
--- dasher-4.11/debian/changelog
+++ dasher-4.11/debian/changelog
@@ -1,3 +1,11 @@
+dasher (4.11-1.2) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Use source:Version instead of gnome:Version.
+    Make the package binNMU safe. (Closes: #629307)
+
+ -- Salvatore Bonaccorso <carnil@debian.org>  Sat, 07 Jul 2012 18:00:29 +0200
+
 dasher (4.11-1.1) unstable; urgency=low
 
   * Non-maintainer upload.
diff -u dasher-4.11/debian/control.in dasher-4.11/debian/control.in
--- dasher-4.11/debian/control.in
+++ dasher-4.11/debian/control.in
@@ -32,8 +32,7 @@
 Depends: ${misc:Depends},
          ${shlibs:Depends},
          at-spi,
-         dasher-data (>= ${gnome:Version}),
-         dasher-data (<< ${gnome:NextVersion})
+         dasher-data (= ${source:Version})
 Description: A graphical predictive text input system
  Dasher is an information-efficient text-entry interface, driven by natural
  continuous pointing gestures. Dasher is a competitive text-entry system
