--- exim4-base.cron.daily.orig	2007-11-21 15:29:05.000000000 +0000
+++ exim4-base.cron.daily	2007-11-26 11:48:18.000000000 +0000
@@ -54,17 +54,22 @@
   fi
 }
 
-if [ "$E4BCD_WATCH_PANICLOG" = "yes" ]; then
+if [ "$E4BCD_WATCH_PANICLOG" != "no" ]; then
   if [ -s "/var/log/exim4/paniclog" ]; then
-    if [ -x "/usr/local/lib/exim4/nonzero_paniclog_hook" ]; then
-      /usr/local/lib/exim4/nonzero_paniclog_hook
-    fi
-    if [ -z "$E4BCD_PANICLOG_NOISE" ] || grep -vq "$E4BCD_PANICLOG_NOISE" /var/log/exim4/paniclog; then
-      log_this "ALERT: exim paniclog /var/log/exim4/paniclog has non-zero size, mail system possibly broken"
-      if ! printf "Subject: exim paniclog on %s has non-zero size\nTo: root\n\nexim paniclog /var/log/exim4/paniclog on %s has non-zero size, mail system might be broken" "$(hostname --fqdn)" "$(hostname --fqdn)" | exim4 root; then
-        log_this "PANIC: sending out e-mail warning has failed, exim has non-zero return code"
+    if [ "$E4BCD_WATCH_PANICLOG" != "once" ] || \
+       [ ! -e "/var/lib/exim4/paniclog.timestamp" ] || \
+       [ "/var/log/exim4/paniclog" -nt "/var/lib/exim4/paniclog.timestamp" ]; then
+      if [ -x "/usr/local/lib/exim4/nonzero_paniclog_hook" ]; then
+        /usr/local/lib/exim4/nonzero_paniclog_hook
+      fi
+      if [ -z "$E4BCD_PANICLOG_NOISE" ] || grep -vq "$E4BCD_PANICLOG_NOISE" /var/log/exim4/paniclog; then
+        log_this "ALERT: exim paniclog /var/log/exim4/paniclog has non-zero size, mail system possibly broken"
+        if ! printf "Subject: exim paniclog on %s has non-zero size\nTo: root\n\nexim paniclog /var/log/exim4/paniclog on %s has non-zero size, mail system might be broken" "$(hostname --fqdn)" "$(hostname --fqdn)" | exim4 root; then
+          log_this "PANIC: sending out e-mail warning has failed, exim has non-zero return code"
+        fi
       fi
     fi
+    touch /var/lib/exim4/paniclog.timestamp
   fi
 fi
 
