--- mainwindow-old.c	2010-04-16 09:53:13.000000000 -0700
+++ mainwindow.c	2010-10-24 02:28:16.000000000 -0700
@@ -643,8 +643,8 @@
 	{"Message/Mark/UnwatchThread",		NULL, N_("Unwatch thread"), NULL, NULL, G_CALLBACK(unwatch_thread_cb) },
 	/* separation */
 
-	{"Message/Mark/MarkSpam",		NULL, N_("Mark as spam"), NULL, NULL, G_CALLBACK(mark_as_spam_cb) },
-	{"Message/Mark/MarkHam",		NULL, N_("Mark as ham"), NULL, NULL, G_CALLBACK(mark_as_ham_cb) },
+	{"Message/Mark/MarkSpam",		NULL, N_("Mark as _spam"), NULL, NULL, G_CALLBACK(mark_as_spam_cb) },
+	{"Message/Mark/MarkHam",		NULL, N_("Mark as _ham"), NULL, NULL, G_CALLBACK(mark_as_ham_cb) },
 	/* separation */
 
 	{"Message/Mark/Lock",			NULL, N_("Lock"), NULL, NULL, G_CALLBACK(lock_msgs_cb) },
