diff --git a/finish-install.d/10clock-setup b/finish-install.d/10clock-setup
index 1357609..646ba34 100755
--- a/finish-install.d/10clock-setup
+++ b/finish-install.d/10clock-setup
@@ -27,6 +27,11 @@ os_needs_local_clock () {
 		case $shortname in
 		MS-DOS*|Windows*|FreeDOS*) # keep in sync with os-prober
 			return 0
+		Solaris*)
+			case "$(udpkg --print-architecture)" in
+			*i386*|*amd64*)
+				return 0
+			;;
 		;;
 		esac
 	done
