--- /usr/sbin/popularity-contest.orig	2012-02-15 20:21:29.593742437 +0100
+++ /usr/sbin/popularity-contest	2012-02-15 20:22:10.274234367 +0100
@@ -158,7 +158,7 @@
 }
 
 # Read dpkg database of installed packages
-open PACKAGES, "dpkg-query --show --showformat='\${status} \${package}\\n'|";
+open PACKAGES, "dpkg-query --show --showformat='\${status} \${package}:\${architecture}\\n'|";
 my @pkglist = ();
 while (<PACKAGES>)
 {
