diff --git a/dput/methods/http.py b/dput/methods/http.py
index ef966bc..d9e1e3a 100644
--- a/dput/methods/http.py
+++ b/dput/methods/http.py
@@ -50,6 +50,7 @@ class AuthHandlerHackAround:
         self.headers = {}
         # data
         self.url = url
+        self.full_url = url
         self.resp_headers = resp_headers
         self.authhandlers = []
         self.timeout = {}
@@ -68,6 +69,9 @@ class AuthHandlerHackAround:
     def add_unredirected_header(self, k, v):
         self.headers[k] = v
 
+    def get_header(self, k, default=None):
+        return self.headers.get(k, default)
+
     def get_full_url(self):
         return self.url
 
