--- ejabberd.dpkg-dist	2011-05-22 20:20:27.000000000 +0400
+++ ejabberd	2012-08-10 20:50:12.000000000 +0400
@@ -20,6 +20,7 @@
 EJABBERDRUN=/var/run/ejabberd
 EJABBERDUSER=ejabberd
 NAME=ejabberd
+CONFIG=/etc/ejabberd/ejabberd.cfg
 
 test -f $EJABBERD || exit 0
 test -f $EJABBERDCTL || exit 0
@@ -116,6 +117,10 @@
 	    echo -n " already stopped"
 	fi
     ;;
+    reload)
+    	echo -n "Reloading jabber server: $NAME"
+	ctl "load_config $CONFIG"
+    ;;
     restart|force-reload)
 	echo -n "Restarting jabber server: $NAME"
 	if ctl status ; then
@@ -134,7 +139,7 @@
 	fi
     ;;
     *)
-	echo "Usage: $0 {start|stop|restart|force-reload|live}" >&2
+	echo "Usage: $0 {start|stop|reload|restart|force-reload|live}" >&2
 	exit 1
     ;;
 esac
