--- debian/control.orig	2018-01-29 16:53:39.000000000 +0100
+++ debian/control	2018-08-01 16:39:24.085282837 +0200
@@ -23,7 +23,7 @@
 
 Package: dunst
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}
+Depends: ${shlibs:Depends}, ${misc:Depends}, default-dbus-session-bus | dbus-session-bus
 Provides: notification-daemon
 Description: dmenu-ish notification-daemon
  Dunst is a highly configurable and lightweight notification-daemon: The
