Description: Fix Hungarian templates translations syntax errors.
Bug-Debian: http://bugs.debian.org/737709
Origin: vendor
Forwarded: https://cups.org/str.php?L4362
Author: Didier Raboud <odyx@debian.org>
Last-Update: 2014-02-05

--- a/templates/hu/printers-header.tmpl
+++ b/templates/hu/printers-header.tmpl
@@ -1 +1 @@
-<P ALIGN="CENTER">{total=0?Nincsenek nyomtatók:{total} nyomtatóból {#printer_name} megjelenítve.</P>
+<P ALIGN="CENTER">{total=0?Nincsenek nyomtatók:{total} nyomtatóból {#printer_name} megjelenítve}.</P>
--- a/templates/hu/classes-header.tmpl
+++ b/templates/hu/classes-header.tmpl
@@ -1 +1 @@
-<P ALIGN="CENTER">{total=0?Nincsenek osztályok:{total} osztályból {#printer_name} megjelenítve.</P>
+<P ALIGN="CENTER">{total=0?Nincsenek osztályok:{total} osztályból {#printer_name} megjelenítve}.</P>
--- a/templates/hu/jobs-header.tmpl
+++ b/templates/hu/jobs-header.tmpl
@@ -4,4 +4,4 @@
 
 <P ALIGN="CENTER">{total=0?Nincsenek feladatok:{total}
 {?which_jobs=?aktív:{which_jobs=all?:befejezett}} feladatból {#job_id}
-megjelenítve.</P>
+megjelenítve}.</P>
