diff -ur bsdmainutils-5.20010615/usr.bin/calendar/io.c bsdmainutils-5.20010615.new/usr.bin/calendar/io.c
--- bsdmainutils-5.20010615/usr.bin/calendar/io.c	Wed Feb  6 20:49:34 2002
+++ bsdmainutils-5.20010615.new/usr.bin/calendar/io.c	Wed Feb  6 20:45:12 2002
@@ -51,7 +51,9 @@
 #include <unistd.h>
 #include <err.h>
 #include <errno.h>
+#ifdef GLIBC
 #include <langinfo.h>
+#endif
 #include <locale.h>
 #include <string.h>
 #include <sys/uio.h>
diff -ur bsdmainutils-5.20010615/usr.bin/ncal/ncal.c bsdmainutils-5.20010615.new/usr.bin/ncal/ncal.c
--- bsdmainutils-5.20010615/usr.bin/ncal/ncal.c	Wed Feb  6 20:49:35 2002
+++ bsdmainutils-5.20010615.new/usr.bin/ncal/ncal.c	Wed Feb  6 20:45:52 2002
@@ -32,7 +32,9 @@
 #include <calendar.h>
 #include <ctype.h>
 #include <err.h>
+#ifdef GLIBC
 #include <langinfo.h>
+#endif  
 #include <locale.h>
 #include <stdio.h>
 #include <stdlib.h>
