diff -Nru --exclude changelog emacs-goodies-el-35.4/elisp/dpkg-dev-el/debian-changelog-mode.el emacs-goodies-el-35.4+nmu1/elisp/dpkg-dev-el/debian-changelog-mode.el
--- emacs-goodies-el-35.4/elisp/dpkg-dev-el/debian-changelog-mode.el	2011-10-27 07:41:58.000000000 -0400
+++ emacs-goodies-el-35.4+nmu1/elisp/dpkg-dev-el/debian-changelog-mode.el	2013-05-14 23:25:08.000000000 -0400
@@ -1453,6 +1453,7 @@
      (3 font-lock-string-face nil t)
      (4 debian-changelog-warning-face nil t))
    '(debian-changelog-fontify-stable . debian-changelog-warning-face)
+   '(debian-changelog-fontify-backports . debian-changelog-warning-face)
    '(debian-changelog-fontify-frozen . font-lock-type-face)
    '(debian-changelog-fontify-unstable . font-lock-string-face)
    '(debian-changelog-fontify-experimental . debian-changelog-warning-face)
@@ -1593,6 +1594,12 @@
     (store-match-data
      (list (match-beginning 1)(match-end 1)))
     t))
+
+(defun debian-changelog-fontify-backports (limit)
+  (when (re-search-forward "^\\sw.* (.+).* \\([a-z][a-z]*-backports\\)" limit t)
+    (store-match-data
+     (list (match-beginning 1)(match-end 1)))
+    t))
 
 ;;
 ;; browse-url interfaces, by Peter Galbraith, Feb 23 2001
