diff --git a/tools/boot/wheezy/boot-hppa b/tools/boot/wheezy/boot-hppa
index 9ce037a..315976d 100755
--- a/tools/boot/wheezy/boot-hppa
+++ b/tools/boot/wheezy/boot-hppa
@@ -36,7 +36,6 @@ if [ -z "$PALODEB" ] ; then
 fi
 (dpkg --fsys-tarfile "$MIRROR/$PALODEB" | \
     tar xf - -C . ./usr/share/palo/iplboot)
-mv boot/delo.2nd $CDROOT/boot/delo.2nd
 mv usr/share/palo/iplboot $CDROOT/install/iplboot
 
 if [ -n "$ARCHIVE_EXTRACTED_SOURCES" ]; then
