diff -u ddccontrol-db-20061014/debian/control ddccontrol-db-20061014/debian/control
--- ddccontrol-db-20061014/debian/control
+++ ddccontrol-db-20061014/debian/control
@@ -9,7 +9,7 @@
 Package: ddccontrol-db
 Architecture: all
 Depends: 
-Recommends: ddccontrol
+Recommends: ddccontrol [i386 amd64 ia64 alpha arm armel]
 Description: monitor database for ddccontrol
  Database of monitors used by ddccontrol.  This is mostly useless without
  ddccontrol.
diff -u ddccontrol-db-20061014/debian/changelog ddccontrol-db-20061014/debian/changelog
--- ddccontrol-db-20061014/debian/changelog
+++ ddccontrol-db-20061014/debian/changelog
@@ -1,3 +1,11 @@
+ddccontrol-db (20061014-1.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * debian/control: Add arches to the Recommends field as per release goal.
+    (Closes: #436649).
+
+ -- Amaya Rodrigo Sastre <amaya@debian.org>  Mon, 31 Mar 2008 01:22:12 +0200
+
 ddccontrol-db (20061014-1) unstable; urgency=low
 
   * New upstream release
