diff -Naur covered.orig/covered-0.7.10/debian/changelog covered/covered-0.7.10/debian/changelog
--- covered.orig/covered-0.7.10/debian/changelog	2014-08-07 13:17:06.008385292 -0500
+++ covered/covered-0.7.10/debian/changelog	2014-08-07 13:24:29.600393017 -0500
@@ -1,3 +1,11 @@
+covered (0.7.10-2) unstable; urgency=low
+
+  * Fix FTBFS with clang
+    - Fixed undefined reference error in
+      src/lxt2_read.h
+
+ -- Arthur Marble <arthur@info9.net>  Thu, 07 Aug 2014 13:24:29 -0500
+
 covered (0.7.10-1) unstable; urgency=low
 
   * New upstream release.
diff -Naur covered.orig/covered-0.7.10/debian/patches/clang-ftbfs.diff covered/covered-0.7.10/debian/patches/clang-ftbfs.diff 
--- covered.orig/covered-0.7.10/debian/patches/clang-ftbfs.diff	1969-12-31 18:00:00.000000000 -0600
+++ covered/covered-0.7.10/debian/patches/clang-ftbfs.diff	2014-08-07 13:19:49.684388142 -0500
@@ -0,0 +1,16 @@
+--- a/src/lxt2_read.h
++++ b/src/lxt2_read.h
+@@ -47,10 +47,10 @@ typedef long off_t;
+ #include <zlib.h>
+ #endif
+ 
+-#ifdef __GNUC__
+-#define _LXT2_RD_INLINE inline
+-#else
++#ifdef __clang__
+ #define _LXT2_RD_INLINE
++#else
++#define _LXT2_RD_INLINE inline
+ #endif
+ 
+ #define LXT2_RDLOAD "LXTLOAD | "
diff -Naur covered.orig/covered-0.7.10/debian/patches/series covered/covered-0.7.10/debian/patches/series
--- covered.orig/covered-0.7.10/debian/patches/series	2014-08-07 13:17:06.008385292 -0500
+++ covered/covered-0.7.10/debian/patches/series	2014-08-07 13:18:00.456386240 -0500
@@ -1,2 +1,3 @@
 outdated_config.diff
 manpage.diff
+clang-ftbfs.diff
