From e497311d359e0e112bed0cb56134e6c77f48f947 Mon Sep 17 00:00:00 2001
From: Riku Voipio <riku.voipio@linaro.org>
Date: Tue, 7 Feb 2012 02:41:25 +0200
Subject: [PATCH] Add multi-arch foreign field


Signed-off-by: Riku Voipio <riku.voipio@linaro.org>
---
 debian/control |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/debian/control b/debian/control
index 073a781..898c907 100644
--- a/debian/control
+++ b/debian/control
@@ -27,6 +27,7 @@ Vcs-Browser: http://git.debian.org/?p=collab-maint/cdbs.git;a=summary
 
 Package: cdbs
 Architecture: all
+Multi-Arch: foreign
 Recommends: autotools-dev
 Suggests: devscripts
 Description: common build system for Debian packages
-- 
1.7.5.4

