diff --git a/vbi.c b/vbi.c
index 0a030df..d0e6753 100644
--- a/vbi.c
+++ b/vbi.c
@@ -1077,6 +1077,10 @@ static int vbi_dvb_open(struct vbi *vbi, const char *vbi_name, const char *progn
 				break;
 			}
 	}
+	if (!progp) {
+		error("No Teletext service found\n");
+		goto outerr;
+	}
 
 	printf("Using: Service ID 0x%04x Type 0x%02x Provider Name \"%s\" Name \"%s\"\n"
 	       "  PMT PID 0x%04x TXT: PID 0x%04x lang %.3s type 0x%02x magazine %1u page %3u\n", 
