--- /usr/bin/dh_icons	2008-08-23 04:31:05.000000000 +0200
+++ /tmp/dh_icons	2009-03-25 15:18:30.976131601 +0100
@@ -46,6 +46,8 @@
 		opendir(DIRHANDLE, $icondir);
 		while (my $subdir = readdir(DIRHANDLE)) {
 			next if $subdir =~ /^\./;
+			next if $subdir eq "gnome";
+			next if $subdir eq "hicolor";
 			my $needs_cache = 0;
 			find sub {
 				$needs_cache = 1 if -f and (/\.png$/ or /\.svg$/ or /\.xpm$/ or /\.icon$/);
