From 1cd63a7a20d4f1ff71fd68987f6efcbf3a5924d0 Mon Sep 17 00:00:00 2001
From: Charles Plessy <plessy@debian.org>
Date: Mon, 24 Nov 2014 08:53:10 +0900
Subject: [PATCH] Clarify that fields with no value are not valid.

Closes: #666726
---
 policy.sgml | 1 +
 1 file changed, 1 insertion(+)

diff --git a/policy.sgml b/policy.sgml
index 7bb703b..daf0f12 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -2544,6 +2544,7 @@ endif
 	  space, and colon (i.e., characters in the ranges 33-57 and
 	  59-126, inclusive).  Field names must not begin with the comment
 	  character, <tt>#</tt>, nor with the hyphen character, <tt>-</tt>.
+	  Fields with no value are invalid.
 	</p>
 
 	<p>
-- 
2.1.1

