diff -Nru beep-1.4.9/debian/beep.install beep-1.4.9/debian/beep.install
--- beep-1.4.9/debian/beep.install	2020-05-16 22:34:14.000000000 +0200
+++ beep-1.4.9/debian/beep.install	2023-12-15 00:57:56.000000000 +0100
@@ -1,4 +1,4 @@
 usr/bin
 usr/share/doc/beep
 usr/share/man
-debian/70-pcspkr-beep.rules lib/udev/rules.d/
+debian/70-pcspkr-beep.rules ${env:deb_udevdir}/rules.d/
diff -Nru beep-1.4.9/debian/beep-udeb.install beep-1.4.9/debian/beep-udeb.install
--- beep-1.4.9/debian/beep-udeb.install	2020-05-16 22:34:14.000000000 +0200
+++ beep-1.4.9/debian/beep-udeb.install	2023-12-15 00:57:56.000000000 +0100
@@ -1,2 +1,2 @@
 usr/bin
-debian/70-pcspkr-beep.rules lib/udev/rules.d/
+debian/70-pcspkr-beep.rules ${env:deb_udevdir}/rules.d/
diff -Nru beep-1.4.9/debian/changelog beep-1.4.9/debian/changelog
--- beep-1.4.9/debian/changelog	2020-05-16 22:53:59.000000000 +0200
+++ beep-1.4.9/debian/changelog	2023-12-15 00:58:02.000000000 +0100
@@ -1,3 +1,10 @@
+beep (1.4.9-1.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Delegate placement of udev files to pkg-config data
+
+ -- Chris Hofstaedtler <zeha@debian.org>  Fri, 15 Dec 2023 00:58:02 +0100
+
 beep (1.4.9-1) unstable; urgency=medium
 
   [ Rhonda D'Vine ]
diff -Nru beep-1.4.9/debian/control beep-1.4.9/debian/control
--- beep-1.4.9/debian/control	2020-05-16 22:46:17.000000000 +0200
+++ beep-1.4.9/debian/control	2023-12-15 00:57:56.000000000 +0100
@@ -2,7 +2,7 @@
 Section: sound
 Priority: optional
 Maintainer: Rhonda D'Vine <rhonda@debian.org>
-Build-Depends: debhelper-compat (= 13)
+Build-Depends: debhelper-compat (= 13), pkgconf, systemd-dev
 Standards-Version: 4.5.0
 Homepage: https://github.com/spkr-beep/beep
 Vcs-Browser: https://salsa.debian.org/rhonda/beep
diff -Nru beep-1.4.9/debian/rules beep-1.4.9/debian/rules
--- beep-1.4.9/debian/rules	2020-05-16 22:34:14.000000000 +0200
+++ beep-1.4.9/debian/rules	2023-12-15 00:57:56.000000000 +0100
@@ -4,6 +4,8 @@
 export DEB_BUILD_MAINT_OPTIONS=hardening=+all
 -include /usr/share/dpkg/buildtools.mk
 
+export deb_udevdir = $(shell pkg-config --variable=udevdir udev | sed s,^/,,)
+
 %:
 	dh $@
 
