diff -urN bitlbee-1.2/debian/control bitlbee-1.2-with-dev/debian/control
--- bitlbee-1.2/debian/control	2008-03-31 00:43:33.000000000 +0200
+++ bitlbee-1.2-with-dev/debian/control	2008-03-30 23:59:48.000000000 +0200
@@ -11,3 +11,12 @@
 Description: An IRC to other chat networks gateway
  This program can be used as an IRC server which forwards everything you
  say to people on other chat networks: Jabber, ICQ, AIM, MSN and Yahoo.
+
+Package: bitlbee-dev
+Architecture: all
+Depends: bitlbee
+Description: An IRC to other chat networks gateway
+ This program can be used as an IRC server which forwards everything you
+ say to people on other chat networks: Jabber, ICQ, AIM, MSN and Yahoo.
+ .
+ This package holds development stuff for compiling modules.
diff -urN bitlbee-1.2/debian/rules bitlbee-1.2-with-dev/debian/rules
--- bitlbee-1.2/debian/rules	2008-03-31 00:43:33.000000000 +0200
+++ bitlbee-1.2-with-dev/debian/rules	2008-03-31 00:38:23.000000000 +0200
@@ -20,7 +20,7 @@
 
 clean:
 	if [ "`whoami`" != "root" -o ! -d debian ]; then exit 1; fi
-	rm -rf build-arch-stamp debian/bitlbee debian/*.substvars debian/files
+	rm -rf build-arch-stamp debian/bitlbee debian/*.substvars debian/files debian/bitlbee-dev
 	-$(MAKE) distclean
 #	-$(MAKE) -C doc/ clean
 				
@@ -34,6 +34,13 @@
 	cp doc/user-guide/user-guide.txt debian/bitlbee/usr/share/doc/bitlbee/
 	cp doc/user-guide/user-guide.html debian/bitlbee/usr/share/doc/bitlbee/
 
+install-indep: install-arch
+	if [ "`whoami`" != "root" -o ! -d debian ]; then exit 1; fi
+	mkdir -p debian/bitlbee-dev/DEBIAN/
+	$(MAKE) install-dev DESTDIR=`pwd`/debian/bitlbee-dev
+
+	mkdir -p debian/bitlbee-dev/usr/share/doc/bitlbee-dev/
+
 binary-arch: build-arch install-arch
 	if [ "`whoami`" != "root" -o ! -d debian ]; then exit 1; fi
 
@@ -76,11 +83,29 @@
 
 	dpkg --build debian/bitlbee ..
 
+binary-indep: install-indep
+	if [ "`whoami`" != "root" -o ! -d debian ]; then exit 1; fi
+
+	chown -R root.root debian/bitlbee-dev/
+	find debian/bitlbee-dev/usr/share/ -type d -exec chmod 755 {} \;
+	find debian/bitlbee-dev/usr/share/ -type f -exec chmod 644 {} \;
+
+	cp debian/changelog debian/bitlbee-dev/usr/share/doc/bitlbee-dev/changelog.Debian
+	gzip -9 debian/bitlbee-dev/usr/share/doc/bitlbee-dev/changelog.Debian
+	cp debian/copyright debian/bitlbee-dev/usr/share/doc/bitlbee-dev/copyright
+
+	cd debian/bitlbee-dev; \
+		find usr -type f -exec md5sum {} \; > DEBIAN/md5sums
+
+	dpkg-gencontrol -ldebian/changelog -isp -pbitlbee-dev -Pdebian/bitlbee-dev
+
+	dpkg --build debian/bitlbee-dev ..
+
 debug-build:
 	BITLBEE_VERSION=\"`date +%Y%m%d`-`hostname`-debug\" debian/rules clean binary DEBUG=1
 
-binary: binary-arch
+binary: binary-arch binary-indep
 build: build-arch
-install: install-arch
+install: install-arch install-indep
 
-.PHONY: build-arch build clean binary-arch binary install-arch install
+.PHONY: build-arch build clean binary-arch binary install-arch install binary-indep install-indep
