--- a/dh_perl	2016-04-03 10:17:54.000000000 +0000
+++ b/dh_perl	2016-05-21 16:46:07.000000000 +0000
@@ -126,10 +126,15 @@
 				unless $version;
 			$version = ">= $version";
 		}
-		
+
+		my $perlarch = $perl;
+		$perlarch .= ':any'
+			unless $deps & ( XS_MODULE | ARCHDEP_MODULE )
+				or $dh{V_FLAG_SET};
+
 		# no need to depend on an un-versioned perl-base -- it's
 		# essential
-		addsubstvar($package, "perl:Depends", $perl, $version)
+		addsubstvar($package, "perl:Depends", $perlarch, $version)
 			unless $perl eq 'perl-base' && ! length($version);
 
 		# add perlapi-<ver> for XS modules and other modules
