--- a/dh_perl	2016-04-03 10:17:54.000000000 +0000
+++ b/dh_perl	2016-05-18 19:02:27.000000000 +0000
@@ -126,10 +126,14 @@
 				unless $version;
 			$version = ">= $version";
 		}
-		
+
+		# unversioned dependency should be architecture independent
+		my $perlarch = $perl;
+		$perlarch .= ':any' unless length($version);
+
 		# no need to depend on an un-versioned perl-base -- it's
 		# essential
-		addsubstvar($package, "perl:Depends", $perl, $version)
+		addsubstvar($package, "perl:Depends", $perlarch, $version)
 			unless $perl eq 'perl-base' && ! length($version);
 
 		# add perlapi-<ver> for XS modules and other modules
