Index: restore/restore.c
===================================================================
RCS file: /cvsroot/dump/dump/restore/restore.c,v
retrieving revision 1.39
diff -u -r1.39 restore.c
--- restore/restore.c	22 Mar 2010 16:08:10 -0000	1.39
+++ restore/restore.c	1 Aug 2014 14:03:56 -0000
@@ -593,7 +593,7 @@
 					if (np->e_type == LEAF) {
 						removeleaf(np);
 						freeentry(np);
-					} else {
+					} else if (np->e_ino != 0){
 						np->e_flags |= TMPNAME;
 						deleteino(np->e_ino);
 						np->e_next = removelist;
