From 88605d1adc515e052c7f7b29ff45b79925bb3343 Mon Sep 17 00:00:00 2001
From: Ian Jackson <ian@davenant.greenend.org.uk>
Date: Mon, 19 Nov 2007 21:19:36 +0100
Subject: [PATCH] * src/processarc.c (process_archive): Fix incorrect sizeof in a memset call.

---
 src/processarc.c |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/src/processarc.c b/src/processarc.c
index cbad9ba..ac48208 100644
--- a/src/processarc.c
+++ b/src/processarc.c
@@ -680,7 +680,7 @@ void process_archive(const char *filename) {
 	      ohshite(_("unable to stat other new file `%.250s'"),
 		      cfile->namenode->name);
 	    memset(cfile->namenode->filestat, 0,
-		   sizeof(cfile->namenode->filestat));
+		   sizeof(*cfile->namenode->filestat));
 	    continue;
 	  }
 	}
-- 
1.5.3.8

