diff -ur 0ad-0.0.24b-2/debian/changelog 0ad-0.0.25/debian/changelog
--- 0ad-0.0.24b-2/debian/changelog	2021-08-15 17:50:57.379685891 -0500
+++ 0ad-0.0.25/debian/changelog	2021-08-16 12:17:49.089508040 -0500
@@ -1,3 +1,16 @@
+0ad (0.0.25-1.1) UNRELEASED; urgency=medium
+
+  * Non-maintainer upload.
+  * New upstream release. 
+  * d/control: Update required versions of dependencies.
+  * d/rules: Clean up build files for libnvtt and spidermonkey in 
+    dh_auto_clean.
+  * d/rules: Exclude libmozjs78-ps-release.so from dh_dwz in order to work 
+    around a crash in dwz.
+  * d/watch: Update URL for releases.
+
+ -- David W. Kennedy <dave_k AT reasoned DOT us>  Sun, 15 Aug 2021 15:07:54 -0500
+
 0ad (0.0.24b-2) UNRELEASED; urgency=medium
 
   [ Pino Toscano ]
diff -ur 0ad-0.0.24b-2/debian/control 0ad-0.0.25/debian/control
--- 0ad-0.0.24b-2/debian/control	2021-08-15 17:50:57.379685891 -0500
+++ 0ad-0.0.25/debian/control	2021-08-16 12:40:19.474742410 -0500
@@ -7,32 +7,35 @@
  Ludovic Rousseau <rousseau@debian.org>
 Build-Depends:
  autoconf,
+ automake,
  cargo,
  cmake,
  debhelper-compat (= 12),
  dpkg-dev (>= 1.15.5),
- libboost-dev,
- libboost-filesystem-dev,
- libcurl4-gnutls-dev | libcurl4-dev,
+ libboost-dev (>= 1.57.0.1),
+ libboost-filesystem-dev (>= 1.57.0.1),
+ libcurl4-gnutls-dev (>= 7.32.0) | libcurl4-dev (>= 7.32.0),
  libenet-dev (>= 1.3),
- libfmt-dev,
+ libfmt-dev (>= 4.0.0),
  libgloox-dev (>= 1.0.10),
- libicu-dev,
+ libicu-dev (>= 67.1-4~),
  libminiupnpc-dev (>= 1.6),
  libnspr4-dev,
  libogg-dev,
  libopenal-dev,
  libpng-dev,
- libsdl2-dev (>= 2.0.2),
+ libsdl2-dev (>= 2.0.5),
  libsodium-dev (>= 1.0.14),
  libvorbis-dev,
  libwxgtk3.0-gtk3-dev,
  libxcursor-dev,
  libxml2-dev,
+ llvm,
  pkg-config,
  python3,
- rustc,
- zlib1g-dev
+ rustc (>= 1.41),
+ tzdata,
+ zlib1g-dev (>= 1:1.2.3)
 Standards-Version: 4.5.0
 Homepage: http://play0ad.com/
 Vcs-Git: https://salsa.debian.org/games-team/0ad.git
diff -ur 0ad-0.0.24b-2/debian/rules 0ad-0.0.25/debian/rules
--- 0ad-0.0.24b-2/debian/rules	2021-08-15 17:50:57.379685891 -0500
+++ 0ad-0.0.25/debian/rules	2021-08-16 12:19:35.940339723 -0500
@@ -24,10 +24,12 @@
 	find binaries/system/ -type f ! -name readme.txt -delete
 	rm -f libraries/fcollada/lib/*.a
 	rm -f build/premake/.*.tmp
-	rm -f libraries/source/spidermonkey/lib/*
+	rm -rf libraries/source/spidermonkey/lib
 	rm -f libraries/source/cxxtest-4.4/python/cxxtest/*.pyc
 	rm -f libraries/source/fcollada/lib/*
 	rm -rf libraries/source/spidermonkey/include-unix-*
+	rm -rf libraries/source/spidermonkey/mozjs-78.6.0
+	rm -f libraries/source/nvtt/lib/*.so
 	rm -f source/ps/tests/stub_impl_hack.cpp
 	dh_auto_clean
 
@@ -59,3 +61,6 @@
 
 override_dh_makeshlibs:
 	dh_makeshlibs -Xusr/lib/games/0ad
+
+override_dh_dwz:
+	dh_dwz -Xlibmozjs78-ps-release.so
diff -ur 0ad-0.0.24b-2/debian/watch 0ad-0.0.25/debian/watch
--- 0ad-0.0.24b-2/debian/watch	2021-08-15 17:50:57.379685891 -0500
+++ 0ad-0.0.25/debian/watch	2021-08-15 17:50:57.647682959 -0500
@@ -1,2 +1,2 @@
 version=3
-http://sf.net/zero-ad/0ad-([\d\.]+)-.*-unix-build\.tar\.xz
+https://releases.wildfiregames.com/0ad-([\d\.]+)-.*-unix-build\.tar\.gz
