--- a/subnet-change	2014-09-14 18:13:25.000000000 +0200
+++ b/subnet-change	2015-10-01 11:13:35.060203336 +0200
@@ -117,7 +117,7 @@
 replace_exports_ip("/etc/exports", $oldsubnet, $newsubnet);
 replace_interfaces_ip("/etc/network/interfaces", $oldsubnet, $newsubnet);
 replace_ips("/etc/samba/smb-debian-edu.conf", $oldsubnet, $newsubnet);
-replace_ips("/etc/squid/squid.conf", $oldsubnet, $newsubnet);
+replace_ips("/etc/squid3/squid.conf", $oldsubnet, $newsubnet);
 change_muninnode("/etc/munin/debian-edu-munin-node.conf", $oldsubnet,
                  $newsubnet);
 change_hostallow("/etc/hosts.allow", $oldsubnet, $newsubnet);
