diff -u cryptsetup-1.0.3/debian/cryptdisks.functions cryptsetup-1.0.3/debian/cryptdisks.functions
--- cryptsetup-1.0.3/debian/cryptdisks.functions
+++ cryptsetup-1.0.3/debian/cryptdisks.functions
@@ -311,6 +311,9 @@
 		$CRYPTCMD $PARAMS create $dst $src <&1
 		RESULT=$?
 		if [ $RESULT = 0 ] ; then
+			do_swap
+			do_tmp
+
 			if [ "$CHECK" != ""  ] && ! $CHECK $MAPPER/$dst $CHECKARGS; then
 				echo -e "\n - the check for '$MAPPER/$dst' failed - maybe the password is wrong" >&2
 				echo " - removing the crypto device $dst" >&2
@@ -380,14 +383,12 @@
 			parse_opts
 			check_key
 			lo_setup
-
+		
 			if test "x$USELUKS" = "xyes" ; then
 				do_luks
 			else
 				do_noluks
 			fi
-			do_swap
-			do_tmp
 			
 			clean_keys
