From cd1f4e8c865ab26588582ad955f2ecc244abefbc Mon Sep 17 00:00:00 2001
From: Roland Mas <lolando@debian.org>
Date: Fri, 16 Dec 2011 11:37:03 +0100
Subject: [PATCH 2/2] Insert dh_installzabbix in the default sequence

---
 dh |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/dh b/dh
index 27733ac..ffc2f85 100755
--- a/dh
+++ b/dh
@@ -374,6 +374,7 @@ my @i = qw{
 	dh_installmodules
 	dh_installlogcheck
 	dh_installlogrotate
+	dh_installzabbix
 	dh_installpam
 	dh_installppp
 	dh_installudev
-- 
1.7.7.3

