Index: scripts/build-rdeps.pl
===================================================================
--- scripts/build-rdeps.pl	(revision 2077)
+++ scripts/build-rdeps.pl	(working copy)
@@ -188,7 +188,8 @@
     $filename =~ s/\//_/g;
     # -> ftp.debian.org_..._main_source_Sources
     $filename = "${base}_${filename}";
-    if (test_for_valid_component($filename) == 0) {
+    if (test_for_valid_component($filename) == 0
+	and -e "$sources_path/$filename") {
 	push(@source_files, $filename);
 	print STDERR "DEBUG: Added source file: $_\n" if ($opt_debug);
     }
@@ -305,17 +306,23 @@
 		findreversebuilddeps($package, "$sources_path/$source_file");
 	}
 
-	if ($source_file =~ /contrib/) {
+	elsif ($source_file =~ /contrib/) {
 		print "Reverse Build-depends in contrib:\n";
 		print "---------------------------------\n\n";
 		findreversebuilddeps($package, "$sources_path/$source_file");
 	}
 
-	if ($source_file =~ /non-free/) {
+	elsif ($source_file =~ /non-free/) {
 		print "Reverse Build-depends in non-free:\n";
 		print "----------------------------------\n\n";
 		findreversebuilddeps($package, "$sources_path/$source_file");
 	}
+
+	else {
+		print "Reverse Build-depends in UNKNOWN:\n";
+		print "----------------------------------\n\n";
+		findreversebuilddeps($package, "$sources_path/$source_file");
+	}
 }
 
 =head1 LICENSE
