diff --git a/debian/doomsday-compat b/debian/doomsday-compat
index a0fb1f4..05bda09 100755
--- a/debian/doomsday-compat
+++ b/debian/doomsday-compat
@@ -6,7 +6,7 @@
 # - Doomsday expects the -iwad parameter to give a directory containing
 #   IWAD files and loads one depending on the additional -game parameter.
 
-# Copyright © 2013 Fabian Greffrath <fabian+debian@greffrath.com>
+# Copyright © 2013-2015 Fabian Greffrath <fabian@debian.org>
 # This package is free software; you can redistribute it and/or modify
 # it under the terms of the GNU General Public License as published by
 # the Free Software Foundation; either version 2 of the License, or
@@ -29,7 +29,7 @@ next_is_iwad=0
 next_is_pwad=0
 
 iwaddir=""
-gamemode=""
+gamemode="doom2"
 
 for arg in "$@"
 do
@@ -142,6 +142,9 @@ fi
 if [ -n "$iwaddir" -a -n "$gamemode" ]
 then
 	eval $DOOMSDAY -iwad $iwaddir -game $gamemode $cmdline
+elif [ -n "$gamemode" ]
+then
+	eval $DOOMSDAY -game $gamemode $cmdline
 else
 	eval $DOOMSDAY "$@"
 fi
