diff -u cdpr-2.2.0/debian/rules cdpr-2.2.0/debian/rules
--- cdpr-2.2.0/debian/rules
+++ cdpr-2.2.0/debian/rules
@@ -24,6 +24,8 @@
 	dh_testdir
 	dh_testroot # we really need this here
 	dh_install --sourcedir=debian/cdpr
+	dh_installdocs
+	dh_installchangelogs
 	dh_strip
 	dh_compress
 	dh_fixperms
@@ -35,15 +37,4 @@
 
 binary-indep: install
-	dh_testdir
-	dh_testroot # we really need this here
-	dh_install --sourcedir=debian/cdpr
-	dh_installdocs
-	dh_installchangelogs
-	dh_compress
-	dh_fixperms
-	dh_installdeb
-	dh_gencontrol
-	dh_md5sums
-	dh_builddeb
 
 .PHONY: build clean install binary binary-arch binary-indep
diff -u cdpr-2.2.0/debian/changelog cdpr-2.2.0/debian/changelog
--- cdpr-2.2.0/debian/changelog
+++ cdpr-2.2.0/debian/changelog
@@ -1,3 +1,11 @@
+cdpr (2.2.0-2.1) unstable; urgency=high
+
+  * Non-maintainer upload.
+  * Make sure all content gets installed when building only
+    the binary-deb target, closes: #393550.
+
+ -- Thijs Kinkhorst <thijs@debian.org>  Thu, 19 Oct 2006 13:00:56 +0200
+
 cdpr (2.2.0-2) unstable; urgency=low
 
   * Initial release, closes: Bug#338153
