From d5732aa83ca363be2c27757f242c4ed2a9d55061 Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Mon, 19 Dec 2016 21:32:51 +0000
Subject: [PATCH] dgit-sponsorship(7): Update in light of fixed #844129

Signed-off-by: Sean Whitton <spwhitton@spwhitton.name>
---
 dgit-sponsorship.7.pod | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/dgit-sponsorship.7.pod b/dgit-sponsorship.7.pod
index ed37ff5..3fc59d2 100644
--- a/dgit-sponsorship.7.pod
+++ b/dgit-sponsorship.7.pod
@@ -245,7 +245,15 @@ or similar, to to the build, and then
 C<dgit -wgf [--quilt=...] push>
 to do the upload.
 
-(If you switched to the quilt-cache dgit view,
+(It is possible to upload from
+the quilt-cache dgit view,
+but this will cause the debian/1.2.3-1 tag to be
+placed on this branch
+rather than the sponsee's working branch.
+Since this might be confusing,
+it is a good idea to switch back to the sponsee's view,
+after reviewing and before pushing.
+If you do want to upload from the quilt-cache dgit view,
 B<do not> pass the --quilt or --gbp or --dpm option again.)
 
 If this was the first upload done with dgit,
-- 
2.10.2

