=== modified file 'doc/apt-ftparchive.1.xml'
--- doc/apt-ftparchive.1.xml	2009-12-12 15:32:18 +0000
+++ doc/apt-ftparchive.1.xml	2010-05-09 13:36:59 +0000
@@ -197,7 +197,7 @@
       Sets the default compression schemes to use 
       for the Package index files. It is a string that contains a space 
       separated list of at least one of: '.' (no compression), 'gzip' and 
-      'bzip2'. The default for all compression schemes is '. gzip'.</para></listitem>
+      'bzip2'. The default for all compression schemes is '. gzip bzip2'.</para></listitem>
       </varlistentry>
 
       <varlistentry><term>Packages::Extensions</term>

=== modified file 'ftparchive/apt-ftparchive.cc'
--- ftparchive/apt-ftparchive.cc	2009-12-18 16:08:45 +0000
+++ ftparchive/apt-ftparchive.cc	2010-05-09 13:24:37 +0000
@@ -118,11 +118,11 @@
       DeLinkLimit = 0;
    
    PkgCompress = Block.Find("Packages::Compress",
-			    Setup.Find("Default::Packages::Compress",". gzip").c_str());
-   CntCompress = Block.Find("Contents::Compress",
+			    Setup.Find("Default::Packages::Compress",". gzip bzip2").c_str());
+   CntCompress = Block.Find("aontents::Compress",
 			    Setup.Find("Default::Contents::Compress",". gzip").c_str());
    SrcCompress = Block.Find("Sources::Compress",
-			    Setup.Find("Default::Sources::Compress",". gzip").c_str());
+			    Setup.Find("Default::Sources::Compress",". gzip bzip2").c_str());
    
    SrcExt = Block.Find("Sources::Extensions",
 		       Setup.Find("Default::Sources::Extensions",".dsc").c_str());
