--- a/usr/lib/dracut/modules.d/95resume/module-setup.sh	2013-05-14 17:17:10.332675254 +0200
+++ b/usr/lib/dracut/modules.d/95resume/module-setup.sh	2013-05-14 17:09:32.448655680 +0200
@@ -5,7 +5,8 @@
 install() {
     local _bin
     # Optional uswsusp support
-    for _bin in /usr/sbin/resume /usr/lib/suspend/resume
+    for _bin in /usr/lib/uswsusp/resume /usr/lib/klibc/bin/resume \
+        /usr/sbin/resume /usr/lib/suspend/resume
     do
         [[ -x "${_bin}" ]] && {
             inst "${_bin}" /usr/sbin/resume
