=== modified file 'debian/changelog'
--- debian/changelog	2012-05-31 14:21:07 +0000
+++ debian/changelog	2013-05-22 16:33:04 +0000
@@ -1,3 +1,9 @@
+abi-compliance-checker (1.97.7-2) unstable; urgency=low
+
+  * Added DEP-8 self-test. 
+
+ -- James Hunt <james.hunt@ubuntu.com>  Wed, 22 May 2013 17:32:23 +0100
+
 abi-compliance-checker (1.97.7-1) unstable; urgency=low
 
   * New upstream

=== modified file 'debian/control'
--- debian/control	2012-05-31 14:21:07 +0000
+++ debian/control	2013-05-22 15:01:44 +0000
@@ -9,6 +9,7 @@
 Homepage: http://ispras.linux-foundation.org/index.php/ABI_compliance_checker
 Vcs-Svn: svn://svn.debian.org/collab-maint/deb-maint/abi-compliance-checker/trunk
 Vcs-Browser: http://anonscm.debian.org/viewvc/collab-maint/deb-maint/abi-compliance-checker/trunk/
+XS-Testsuite: autopkgtest
 
 Package: abi-compliance-checker
 Architecture: all

=== added directory 'debian/tests'
=== added file 'debian/tests/control'
--- debian/tests/control	1970-01-01 00:00:00 +0000
+++ debian/tests/control	2013-05-22 15:02:49 +0000
@@ -0,0 +1,2 @@
+Tests: self_test
+Depends: @

=== added file 'debian/tests/self_test'
--- debian/tests/self_test	1970-01-01 00:00:00 +0000
+++ debian/tests/self_test	2013-05-22 15:05:37 +0000
@@ -0,0 +1,3 @@
+#!/bin/sh -e
+
+abi-compliance-checker -test

