--- postinst.old	2010-08-11 21:41:18.221497943 +0200
+++ postinst	2010-08-11 21:43:12.478148710 +0200
@@ -421,7 +421,7 @@
             echo "please check them and consider removing them." >/dev/stderr
             mv /usr/share/dokuwiki/lib/plugins /usr/share/dokuwiki/lib/plugins.old
         fi
-        ln -s /var/lib/dokuwiki/plugins /usr/share/dokuwiki/lib/
+        ln -s /var/lib/dokuwiki/lib/plugins /usr/share/dokuwiki/lib/
         echo "[done]" >/dev/stderr
     fi
     # Possible cases for /var/lib/dokuwiki/plugins:
