--- /usr/share/awstats/tools/buildstatic.sh.orig	2014-04-10 04:21:35.169385745 +0200
+++ /usr/share/awstats/tools/buildstatic.sh	2014-04-10 04:37:37.942061858 +0200
@@ -30,9 +30,16 @@
 
 cd /etc/awstats
 
-for c in `/bin/ls -1 awstats.*.conf 2>/dev/null | \
-          /bin/sed 's/^awstats\.\(.*\)\.conf/\1/'` \
-         `[ -f /etc/awstats/awstats.conf ] && echo awstats`
+CONFIGS=`/bin/ls -1 awstats.*.conf 2>/dev/null | \
+	/bin/sed 's/^awstats\.\(.*\)\.conf/\1/'`
+
+if [ -f /etc/awstats/awstats.conf ] && (
+	[ "${AWSTATS_PROCESS_AWSTATS_CONF:-yes}" = "yes" ] || [ -z "$CONFIGS" ])
+then
+	CONFIGS="${CONFIGS+"$CONFIGS "}awstats"
+fi
+
+for c in $CONFIGS
 do
   mkdir -p /var/cache/awstats/$c/$YEAR/$MONTH/
 
