--- /usr/share/awstats/tools/update.sh.orig	2014-04-10 04:21:41.013276300 +0200
+++ /usr/share/awstats/tools/update.sh	2014-04-10 04:27:28.786853048 +0200
@@ -24,9 +24,16 @@
 
 cd /etc/awstats
 
-for c in `/bin/ls -1 awstats.*.conf 2>/dev/null | \
-          /bin/sed 's/^awstats\.\(.*\)\.conf/\1/'` \
-         `[ -f /etc/awstats/awstats.conf ] && echo awstats`
+CONFIGS=`/bin/ls -1 awstats.*.conf 2>/dev/null | \
+	/bin/sed 's/^awstats\.\(.*\)\.conf/\1/'`
+
+if [ -f /etc/awstats/awstats.conf ] && (
+	[ "${AWSTATS_PROCESS_AWSTATS_CONF:-yes}" = "yes" ] || [ -z "$CONFIGS" ])
+then
+	CONFIGS="${CONFIGS+"$CONFIGS "}awstats"
+fi
+
+for c in $CONFIGS
 do
   if ! nice -n $AWSTATS_NICE $AWSTATS \
 	  -config=$c \
