=== modified file 'scripts/process.in'
--- scripts/process.in	2007-08-28 21:16:46 +0000
+++ scripts/process.in	2008-01-29 19:55:42 +0000
@@ -154,7 +154,7 @@
     print DEBUG ">$fn|$fv|\n";
     $fn = lc $fn;
     # Don't lc owner or forwarded
-    $fv = lc $fv unless $fh =~ /^(?:owner|forwarded|usertags|version|source-version)$/;
+    $fv = lc $fv unless $fn =~ /^(?:owner|forwarded|usertags|version|source-version)$/;
     $pheader{$fn} = $fv;
     print DEBUG ">$fn~$fv<\n";
 }

