diff -ur asciidoc-8.6.6/debian/control asciidoc-8.6.6.new/debian/control
--- asciidoc-8.6.6/debian/control	2012-03-06 19:14:19.000000000 +0000
+++ asciidoc-8.6.6.new/debian/control	2012-03-05 18:07:24.000000000 +0000
@@ -14,6 +14,7 @@
 Depends: python (>= 2.4), ${misc:Depends}, ${python:Depends}
 Recommends: docbook-utils, xmlto, dblatex, libxml2-utils
 Suggests: vim-addon-manager, source-highlight
+Multi-Arch: foreign
 Description: Highly configurable text format for writing documentation
  AsciiDoc is a text document format for writing articles, books, manuals and
  UNIX man pages. AsciiDoc files can be translated to HTML (with or without
