diff -urN dnsmasq-2.45.orig/src/dbus.c dnsmasq-2.45/src/dbus.c
--- dnsmasq-2.45.orig/src/dbus.c	2008-07-20 21:18:16.000000000 +0200
+++ dnsmasq-2.45/src/dbus.c	2008-09-16 19:14:41.000000000 +0200
@@ -283,7 +283,10 @@
   daemon->dbus = connection; 
   
   if ((message = dbus_message_new_signal(DNSMASQ_PATH, DNSMASQ_SERVICE, "Up")))
-    dbus_connection_send(connection, message, NULL);
+    {
+      dbus_connection_send(connection, message, NULL);
+      dbus_message_unref(message);
+    }
 
   return NULL;
 }
