Index: debian/control
===================================================================
RCS file: /cvsroot/pkg-goodies-el/emacs-goodies-el/debian/control,v
retrieving revision 1.77
diff -u -r1.77 control
--- debian/control	3 Mar 2009 01:38:40 -0000	1.77
+++ debian/control	27 May 2009 05:49:46 -0000
@@ -5,7 +5,7 @@
 Build-Depends: debhelper (>= 4.0.0), dpatch (>=1.5)
 Build-Depends-Indep: texinfo
 Standards-Version: 3.7.2
-Vcs-Cvs: :pserver:anonymous@cvs.alioth.debian.org:/cvsroot/pkg-goodies-el co emacs-goodies-el
+Vcs-Cvs: :pserver:anonymous@cvs.alioth.debian.org:/cvsroot/pkg-goodies-el emacs-goodies-el
 Vcs-Browser: http://alioth.debian.org/plugins/scmcvs/cvsweb.php/?cvsroot=pkg-goodies-el
 
 Package: emacs-goodies-el
