Index: clisp-2.44.1/modules/berkeley-db/bdb.c
===================================================================
--- clisp-2.44.1.orig/modules/berkeley-db/bdb.c	2008-08-10 21:19:09.000000000 +0200
+++ clisp-2.44.1/modules/berkeley-db/bdb.c	2008-08-10 21:20:02.000000000 +0200
@@ -700,13 +700,13 @@
     set_flags(popSTACK(),&flags_on,&flags_off,DB_NOPANIC);
     set_flags(popSTACK(),&flags_on,&flags_off,DB_NOMMAP);
     set_flags(popSTACK(),&flags_on,&flags_off,DB_NOLOCKING);
-    set_flags(popSTACK(),&flags_on,&flags_off,DB_DIRECT_LOG);
+    set_flags(popSTACK(),&flags_on,&flags_off,DB_LOG_DIRECT);
 #  if defined(DB_LOG_INMEMORY)
     set_flags(popSTACK(),&flags_on,&flags_off,DB_LOG_INMEMORY);
 #  else
     skipSTACK(1);               /* skip :LOG-INMEMORY */
 #  endif
-    set_flags(popSTACK(),&flags_on,&flags_off,DB_LOG_AUTOREMOVE);
+    set_flags(popSTACK(),&flags_on,&flags_off,DB_LOG_AUTO_REMOVE);
 #  if defined(DB_DSYNC_LOG)
     set_flags(popSTACK(),&flags_on,&flags_off,DB_DSYNC_LOG);
 #  else
@@ -936,11 +936,11 @@
 #if defined(DB_LOG_INMEMORY)
   if (flags & DB_LOG_INMEMORY) { pushSTACK(`:LOG-INMEMORY`); count++; }
 #endif
-  if (flags & DB_LOG_AUTOREMOVE) { pushSTACK(`:LOG-AUTOREMOVE`); count++; }
+  if (flags & DB_LOG_AUTO_REMOVE) { pushSTACK(`:LOG-AUTOREMOVE`); count++; }
 #if defined(DB_DSYNC_LOG)
   if (flags & DB_DSYNC_LOG) { pushSTACK(`:DSYNC-LOG`); count++; }
 #endif
-  if (flags & DB_DIRECT_LOG) { pushSTACK(`:DIRECT-LOG`); count++; }
+  if (flags & DB_LOG_DIRECT) { pushSTACK(`:DIRECT-LOG`); count++; }
   if (flags & DB_CDB_ALLDB) { pushSTACK(`:CDB-ALLDB`); count++; }
   if (flags & DB_AUTO_COMMIT) { pushSTACK(`:AUTO-COMMIT`); count++; }
   SYSCALL(dbe->get_encrypt_flags,(dbe,&flags));
@@ -1101,13 +1101,13 @@
     VALUES_IF(dbe_get_flags_num(dbe) & DB_LOG_INMEMORY);
 #endif
   } else if (eq(what,`:LOG-AUTOREMOVE`)) {
-    VALUES_IF(dbe_get_flags_num(dbe) & DB_LOG_AUTOREMOVE);
+    VALUES_IF(dbe_get_flags_num(dbe) & DB_LOG_AUTO_REMOVE);
 #if defined(DB_DSYNC_LOG)
   } else if (eq(what,`:DSYNC-LOG`)) {
     VALUES_IF(dbe_get_flags_num(dbe) & DB_DSYNC_LOG);
 #endif
   } else if (eq(what,`:DIRECT-LOG`)) {
-    VALUES_IF(dbe_get_flags_num(dbe) & DB_DIRECT_LOG);
+    VALUES_IF(dbe_get_flags_num(dbe) & DB_LOG_DIRECT);
   } else if (eq(what,`:CDB-ALLDB`)) {
     VALUES_IF(dbe_get_flags_num(dbe) & DB_CDB_ALLDB);
   } else if (eq(what,`:AUTO-COMMIT`)) {
