diff --git a/scripts/bts.pl b/scripts/bts.pl
index e431c6f..50c0b95 100755
--- a/scripts/bts.pl
+++ b/scripts/bts.pl
@@ -538,18 +538,6 @@ if (@ARGV and $ARGV[0] =~ /^--no-?conf$/) {
     $config_vars{'BTS_INTERACTIVE'} =~ /^(yes|no|force)$/
 	or $config_vars{'BTS_INTERACTIVE'} = 'no';
 
-    if (!length $config_vars{'BTS_SMTP_HOST'}
-        and $config_vars{'BTS_SENDMAIL_COMMAND'} ne '/usr/sbin/sendmail') {
-	my $cmd = (split ' ', $config_vars{'BTS_SENDMAIL_COMMAND'})[0];
-	unless ($cmd =~ /^~?[A-Za-z0-9_\-\+\.\/]*$/) {
-	    warn "BTS_SENDMAIL_COMMAND contained funny characters: $cmd\nReverting to default value /usr/sbin/sendmail\n";
-	    $config_vars{'BTS_SENDMAIL_COMMAND'}='/usr/sbin/sendmail';
-	} elsif (system("command -v $cmd >/dev/null 2>&1") != 0) {
-	    warn "BTS_SENDMAIL_COMMAND $cmd could not be executed.\nReverting to default value /usr/sbin/sendmail\n";
-	    $config_vars{'BTS_SENDMAIL_COMMAND'}='/usr/sbin/sendmail';
-	}
-    }
-
     foreach my $var (sort keys %config_vars) {
 	if ($config_vars{$var} ne $config_default{$var}) {
 	    $modified_conf_msg .= "  $var=$config_vars{$var}\n";
@@ -674,20 +662,6 @@ $smtppass = $opt_smtppass if $opt_smtppass;
 $smtphelo = $opt_smtphelo if $opt_smtphelo;
 
 if ($opt_sendmail) {
-    if ($opt_sendmail ne '/usr/sbin/sendmail'
-	and $opt_sendmail ne $sendmailcmd) {
-	my $cmd = (split ' ', $opt_sendmail)[0];
-	unless ($cmd =~ /^~?[A-Za-z0-9_\-\+\.\/]*$/) {
-	    warn "--sendmail command contained funny characters: $cmd\nReverting to default value $sendmailcmd\n";
-	    undef $opt_sendmail;
-	} elsif (system("command -v $cmd >/dev/null 2>&1") != 0) {
-	    warn "--sendmail command $cmd could not be executed.\nReverting to default value $sendmailcmd\n";
-	    undef $opt_sendmail;
-	}
-    }
-}
-
-if ($opt_sendmail) {
     $sendmailcmd = $opt_sendmail;
     $smtphost = '';
 } else {
