From 213b7472e597f110e8bf8e5e774a54a63be630bf Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Sun, 29 Jul 2018 15:00:43 +0800
Subject: [PATCH 4/5] dgit-maint-gbp(7): update references to --*-dirty

Signed-off-by: Sean Whitton <spwhitton@spwhitton.name>
---
 dgit-maint-gbp.7.pod | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/dgit-maint-gbp.7.pod b/dgit-maint-gbp.7.pod
index e6f6863..b8abd66 100644
--- a/dgit-maint-gbp.7.pod
+++ b/dgit-maint-gbp.7.pod
@@ -55,15 +55,15 @@ branch, never the dgit patches-applied branch.
 
 =head1 BUILDING
 
-You can perform builds like this:
+You can perform test builds like this:
 
 =over 4
 
-    % dgit [--allow-dirty] gbp-build [OPTIONS]
+    % dgit [--include-dirty] gbp-build [OPTIONS]
 
 =back
 
-where I<--allow-dirty> is needed for testing uncommitted changes, and
+where I<--include-dirty> is needed for testing uncommitted changes, and
 I<OPTIONS> are any further options to be passed on to
 gbp-buildpackage(1).
 
-- 
2.11.0

