From 51652e5e281a7429a99b615079c58138bf71ed4c Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Sun, 29 Jul 2018 15:00:59 +0800
Subject: [PATCH 5/5] dgit-sponsorship(7): add references to push-source

Signed-off-by: Sean Whitton <spwhitton@spwhitton.name>
---
 dgit-sponsorship.7.pod | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/dgit-sponsorship.7.pod b/dgit-sponsorship.7.pod
index 9eecec0..3fbdac8 100644
--- a/dgit-sponsorship.7.pod
+++ b/dgit-sponsorship.7.pod
@@ -30,7 +30,7 @@ This section is addressed to the sponsee:
 =head2 General
 
 You should prepare the package as if you were going
-to upload it with C<dgit push> yourself.
+to upload it with C<dgit push-source> or C<dgit push> yourself.
 
 For a straightforward NMU, consult L<dgit-nmu-simple(7)>.
 
@@ -220,7 +220,7 @@ C<dgit fetch sid> will get you an up-to-date
 C<refs/remotes/dgit/dgit/sid>
 showing what's in the archive already.
 
-C<dgit -wgf --damp-run push>
+C<dgit -wgf --damp-run push-source>
 will check that dgit can build an appropriate source package.
 
 There is no need to run debdiff.
@@ -234,6 +234,8 @@ When you have completed your source review,
 and use
 C<dgit -wgf [--quilt=...] sbuild -A -C>
 or similar, to to the build, and then
+C<dgit -wgf [--quilt=...] push-source>
+or
 C<dgit -wgf [--quilt=...] push>
 to do the upload.
 
-- 
2.11.0

