diff -u acpitool-0.4.7/src/acpitool.cpp acpitool-0.4.7/src/acpitool.cpp
--- acpitool-0.4.7/src/acpitool.cpp
+++ acpitool-0.4.7/src/acpitool.cpp
@@ -32,6 +32,7 @@
 #include <iostream>
 #include <iomanip>
 #include <fstream>
+#include <cstring>
 #include <dirent.h>
 #include "acpitool.h"
 #include "thinkpad.h"
diff -u acpitool-0.4.7/debian/changelog acpitool-0.4.7/debian/changelog
--- acpitool-0.4.7/debian/changelog
+++ acpitool-0.4.7/debian/changelog
@@ -1,3 +1,10 @@
+acpitool (0.4.7-1.2) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Fix FTBFS with recent GCC 4.3 (Closes: #417069).
+
+ -- Luk Claes <luk@debian.org>  Fri, 14 Mar 2008 21:34:50 +0000
+
 acpitool (0.4.7-1.1) unstable; urgency=medium
 
   * Non-maintainer upload.
