From 55858cc4b8a5a126ccb8ad06a739772db54fcd93 Mon Sep 17 00:00:00 2001
From: Nicholas D Steeves <nsteeves@gmail.com>
Date: Sun, 29 Jan 2017 16:01:27 -0700
Subject: [PATCH 2/2] Update changelog for 3.10~1.gbp9c5495 release

Signed-off-by: Nicholas D Steeves <nsteeves@gmail.com>
---
 debian/changelog | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/debian/changelog b/debian/changelog
index 5b67460..2e424ed 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,11 @@
+dgit (3.10~1.gbp9c5495) UNRELEASED; urgency=medium
+
+  ** SNAPSHOT build @9c54952a8814f6bab39d82b0d5caff199807a08c **
+
+  * Fix typos in documentation.
+
+ -- Nicholas D Steeves <nsteeves@gmail.com>  Sun, 29 Jan 2017 16:01:26 -0700
+
 dgit (3.9) unstable; urgency=medium
 
   Improvements:
-- 
2.11.0

