Index: blender-2.70a/intern/atomic/atomic_ops.h
===================================================================
--- blender-2.70a.orig/intern/atomic/atomic_ops.h       2014-06-21 16:13:42.000000000 +0800
+++ blender-2.70a/intern/atomic/atomic_ops.h    2014-06-22 12:06:18.897812794 +0800
@@ -61,7 +61,7 @@
 #  endif
 #endif

-#if defined(_M_X64) || defined(__amd64__) || defined(__x86_64__) || defined(__s390x__) || defined(__powerpc64__) || defined(__aarch64__) || (defined(__sparc__) && defined(__arch64__))
+#if defined(_M_X64) || defined(__amd64__) || defined(__x86_64__) || defined(__s390x__) || defined(__powerpc64__) || defined(__aarch64__) || (defined(__sparc__) && defined(__arch64__)) || defined(__mips64)
 #  define LG_SIZEOF_PTR 3
 #  define LG_SIZEOF_INT 2
 #else
