Index: dpkg-1.14.25/dpkg-deb/build.c
===================================================================
--- dpkg-1.14.25.orig/dpkg-deb/build.c	2009-03-24 11:45:24.000000000 +0000
+++ dpkg-1.14.25/dpkg-deb/build.c	2009-03-24 11:47:13.000000000 +0000
@@ -69,6 +69,9 @@
 static int known_arbitrary_field(const struct arbitraryfield *field) {
   const char **known;
 
+  /* always accept fields starting with x- */
+  if (strncasecmp(field->name, "x-", 2) == 0)
+    return 1;
   for (known= arbitrary_fields; *known; known++)
     if (strcasecmp(field->name, *known) == 0)
       return 1;
