diff -Nru cdbs-0.4.61/1/class/python-distutils.mk.in cdbs-0.4.61+nmu1/1/class/python-distutils.mk.in
--- cdbs-0.4.61/1/class/python-distutils.mk.in	2009-07-10 10:59:44.000000000 +0200
+++ cdbs-0.4.61+nmu1/1/class/python-distutils.mk.in	2009-09-19 12:47:22.000000000 +0200
@@ -38,7 +38,7 @@
 DEB_PYTHON_SETUP_CMD = setup.py
 DEB_PYTHON_CLEAN_ARGS = -a
 DEB_PYTHON_BUILD_ARGS = --build-base="$(CURDIR)/$(DEB_BUILDDIR)/build"
-DEB_PYTHON_INSTALL_ARGS_ALL = --no-compile -O0
+DEB_PYTHON_INSTALL_ARGS_ALL = --prefix=/usr --no-compile -O0
 
 #DEB_PYTHON_DESTDIR = $(DEB_DESTDIR)
 
@@ -140,11 +140,13 @@
 
 # install stage
 $(patsubst %,install/%,$(cdbs_python_indep_packages)) :: install/%: python-install-py
-	cd $(DEB_SRCDIR) && $(call cdbs_python_binary,python$(cdbs_python_compile_version)) $(DEB_PYTHON_SETUP_CMD) install --root=$(cdbs_python_destdir) $(DEB_PYTHON_INSTALL_ARGS_ALL)
+	cd $(DEB_SRCDIR) && $(call cdbs_python_binary,python$(cdbs_python_compile_version)) $(DEB_PYTHON_SETUP_CMD) install --root=$(cdbs_python_destdir) \
+		--install-lib=/usr/lib/python$(firstword $(strip $(sort $(shell pyversions -vr))))/site-packages/ $(DEB_PYTHON_INSTALL_ARGS_ALL)
 
 $(patsubst %,install/%,$(cdbs_python_arch_packages)) :: install/%: $(addprefix python-install-, $(cdbs_python_build_versions))
 	for buildver in $(cdbs_python_build_versions); do \
-		cd $(CURDIR) && cd $(DEB_SRCDIR) && $(call cdbs_python_binary,python$$buildver) $(DEB_PYTHON_SETUP_CMD) install --root=$(cdbs_python_destdir) $(DEB_PYTHON_INSTALL_ARGS_ALL); \
+		cd $(CURDIR) && cd $(DEB_SRCDIR) && $(call cdbs_python_binary,python$$buildver) $(DEB_PYTHON_SETUP_CMD) install --root=$(cdbs_python_destdir) \
+		--install-lib=/usr/lib/python$$buildver/site-packages/ $(DEB_PYTHON_INSTALL_ARGS_ALL); \
 	done
 
 # Deprecated targets.  You should use above targets instead.
diff -Nru cdbs-0.4.61/debian/changelog cdbs-0.4.61+nmu1/debian/changelog
--- cdbs-0.4.61/debian/changelog	2009-08-27 21:13:42.000000000 +0200
+++ cdbs-0.4.61+nmu1/debian/changelog	2009-09-19 12:27:30.000000000 +0200
@@ -1,3 +1,13 @@
+cdbs (0.4.61+nmu1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * python-distutils.mk changes for Python >= 2.6:
+    - hardcode "site-packages" using --install-lib (Python >=2.6 is using
+      "dist-packages" by default)
+    - add --prefix=/usr (Python >= 2.6 is using /usr/local by default)
+
+ -- Piotr Ożarowski <piotr@debian.org>  Fri, 18 Sep 2009 22:54:47 +0200
+
 cdbs (0.4.61) unstable; urgency=low
 
   * Brown paperbag release: Fix excess closing paranthesis in
