diff -Naur ddccontrol.orig/ddccontrol-0.4.2/debian/changelog ddccontrol/ddccontrol-0.4.2/debian/changelog
--- ddccontrol.orig/ddccontrol-0.4.2/debian/changelog	2014-08-06 13:58:56.874924408 -0500
+++ ddccontrol/ddccontrol-0.4.2/debian/changelog	2014-08-06 14:05:49.606931596 -0500
@@ -1,3 +1,11 @@
+ddccontrol (0.4.2-12) unstable; urgency=low
+
+  * Fix FTBFS with clang
+    - Fixed undefined reference error in
+      src/ddcpci/i2c-algo-bit.c
+
+ -- Arthur Marble <arthur@info9.net>  Wed, 06 Aug 2014 14:05:49 -0500
+
 ddccontrol (0.4.2-11) unstable; urgency=low
 
   * QA upload.
diff -Naur ddccontrol.orig/ddccontrol-0.4.2/debian/patches/clang-ftbfs.diff ddccontrol/ddccontrol-0.4.2/debian/patches/clang-ftbfs.diff 
--- ddccontrol.orig/ddccontrol-0.4.2/debian/patches/clang-ftbfs.diff	1969-12-31 18:00:00.000000000 -0600
+++ ddccontrol/ddccontrol-0.4.2/debian/patches/clang-ftbfs.diff	2014-08-06 14:03:42.214929378 -0500
@@ -0,0 +1,11 @@
+--- a/src/ddcpci/i2c-algo-bit.c
++++ b/src/ddcpci/i2c-algo-bit.c
+@@ -378,7 +378,7 @@ int sendbytes(struct i2c_algo_bit_data *
+ 	return wrcount;
+ }
+ 
+-inline int readbytes(struct i2c_algo_bit_data *adap, struct i2c_msg *msg)
++static inline int readbytes(struct i2c_algo_bit_data *adap, struct i2c_msg *msg)
+ {
+ 	int inval;
+ 	int rdcount=0;   	/* counts bytes read */
diff -Naur ddccontrol.orig/ddccontrol-0.4.2/debian/patches/series ddccontrol/ddccontrol-0.4.2/debian/patches/series 
--- ddccontrol.orig/ddccontrol-0.4.2/debian/patches/series	2014-08-06 13:58:56.874924408 -0500
+++ ddccontrol/ddccontrol-0.4.2/debian/patches/series	2014-08-06 14:01:52.226927462 -0500
@@ -1,2 +1,3 @@
 0002_cerrors.patch
 0003_buffer_overrun.patch
+clang-ftbfs.diff
