Index: dovecot-2.1.7/doc/example-config/Makefile.in
===================================================================
--- dovecot-2.1.7.orig/doc/example-config/Makefile.in	2012-05-29 19:04:48.000000000 +0100
+++ dovecot-2.1.7/doc/example-config/Makefile.in	2013-01-26 15:39:38.000257535 +0000
@@ -296,7 +296,7 @@
 SUBDIRS = conf.d
 pkgsysconfdir = $(sysconfdir)/dovecot
 nodist_pkgsysconf_DATA = README
-exampledir = $(docdir)/example-config
+exampledir = $(docdir)/dovecot-core/example-config
 example_DATA = \
 	dovecot.conf \
 	dovecot-db.conf.ext \
