diff -Naur debian-cd.orig/tools/make_disc_trees.pl debian-cd/tools/make_disc_trees.pl
--- debian-cd.orig/tools/make_disc_trees.pl	2009-07-17 14:10:10.000000000 -0400
+++ debian-cd/tools/make_disc_trees.pl	2009-07-17 14:10:49.000000000 -0400
@@ -449,11 +449,15 @@
 	$filename = $File::Find::name;
 
 	if ((-d "$_") && ($filename =~ m/\/main\/binary-[^\/]*$/)) {
-		if ((-f "$_/Packages") && (! -d "../local/$_/")) {
-			mkdir "../local/$_/" || die "Error creating directory local/$_: $!\n";
+		if (-f "$_/Packages") {
+		    if (! -d "../local/$_/") {
+			mkdir "../local/$_/" || die "Error creating directory local/$_: $!\n"; 
+		    }
+		    if ( ! -f "../local/$_/Packages" ) {
 			open(LPFILE, ">../local/$_/Packages") or die "Error creating local/$_/Packages: $!\n";
 			close LPFILE;
 			print "  Created empty Packages file for local/$_\n";
+		    }
 		}
 	}
 }
@@ -735,8 +739,8 @@
     }	
 
     $pdir = "$dir/dists/$codename/$dist";
-    if ($section and $section eq "debian-installer") {
-        $pdir = "$dir/dists/$codename/$dist/debian-installer";
+    if ($section and (($section eq "debian-installer") or ($section eq "local/debian-installer"))) {
+	$pdir = "$dir/dists/$codename/$dist/debian-installer";
     }
     return $pdir;
 }
