diff -ru debtags-1.7.2.old/bash-completion debtags-1.7.2/bash-completion
--- debtags-1.7.2.old/bash-completion	2007-06-09 20:37:41.000000000 +0100
+++ debtags-1.7.2/bash-completion	2007-07-06 15:54:56.000000000 +0100
@@ -10,10 +10,10 @@
 	COMPREPLY=()
 	cur=${COMP_WORDS[COMP_CWORD]}
 	prev=${COMP_WORDS[COMP_CWORD-1]}
-	options='update implications check tagshow tagsearch show \
-	     	 related cat grep mkpatch install maintainers \
-	         submit score tag search todo facetcoll\
-			 todoreport stats'
+	options='cat check diff dumpavail grep help install \
+		maintainers mkpatch related score search selfcheck \
+		show smartsearch stats submit tag tagcat tagsearch \
+		tagshow todo update vocfilter'
 	
 	for (( i=0; i < ${#COMP_WORDS[@]}-1; i++ )); do
 		case ${COMP_WORDS[i]} in
