--- debian/debian-edu-profile-udeb.templates.orig	2008-06-23 22:15:24.367206962 +0200
+++ debian/debian-edu-profile-udeb.templates	2008-06-23 22:15:49.692206584 +0200
@@ -11,7 +11,7 @@
  .
   - Main-Server: reserved for the Debian Edu server. It does not
                  include any GUI (Graphical User Interface). There should only
-		 be one such server on a Debian Edu network.
+                 be one such server on a Debian Edu network.
   - Workstation: for normal machines on the Debian Edu network. 
   - Thin-Client-Server:
                  includes 'Workstation' and requires two network cards.
@@ -29,7 +29,7 @@
  .
   - Main-Server: reserved for the Debian Edu server. It does not
                  include any GUI (Graphical User Interface). There should only
-		 be one such server on a Debian Edu network.
+                 be one such server on a Debian Edu network.
   - Workstation: for normal machines on the Debian Edu network. 
   - Thin-Client-Server:
                  includes 'Workstation' and requires two network cards.
