=== modified file 'cmdline/apt-get.cc'
--- cmdline/apt-get.cc	2012-01-29 12:24:17 +0000
+++ cmdline/apt-get.cc	2012-02-27 22:44:55 +0000
@@ -2863,13 +2863,14 @@
 	       if ((BADVER(Ver)) == false)
 	       {
 		  string forbidden;
-		  if (Ver->MultiArch == pkgCache::Version::None || Ver->MultiArch == pkgCache::Version::All)
+		  if (Ver->MultiArch == pkgCache::Version::None)
 		  {
 		     if (colon == string::npos)
 		     {
 			Pkg = Ver.ParentPkg().Group().FindPkg(hostArch);
 		     }
 		  }
+		  else if (Ver->MultiArch == pkgCache::Version::All);
 		  else if (Ver->MultiArch == pkgCache::Version::Same)
 		  {
 		     if (colon == string::npos)

