diff -urpN bip-0.8.6.orig/src/irc.h bip-0.8.6/src/irc.h
--- bip-0.8.6.orig/src/irc.h	2010-09-12 16:47:57.000000000 +0000
+++ bip-0.8.6/src/irc.h	2010-09-19 23:48:43.000000000 +0000
@@ -58,7 +58,7 @@ struct channel {
 #define IRC_TYPE_LOGING_CLIENT (2)
 #define IRC_TYPE_TRUST_CLIENT (3)
 
-struct user {
+struct bipuser {
 	/** client connection static data **/
 
 	char *name;
@@ -139,7 +139,7 @@ struct link {
 	hash_t chan_infos;		/* channels we want */
 	list_t chan_infos_order;	/* for order only */
 
-	struct user *user;
+	struct bipuser *user;
 
 	/** server connection static data **/
 	/* server list */
diff -urpN bip-0.8.6.orig/src/log.c bip-0.8.6/src/log.c
--- bip-0.8.6.orig/src/log.c	2010-09-12 16:47:57.000000000 +0000
+++ bip-0.8.6/src/log.c	2010-09-19 23:48:05.000000000 +0000
@@ -1195,7 +1195,7 @@ void log_flush_all(void)
 	}
 }
 
-log_t *log_new(struct user *user, const char *network)
+log_t *log_new(struct bipuser *user, const char *network)
 {
 	log_t *logdata;
 
diff -urpN bip-0.8.6.orig/src/log.h bip-0.8.6/src/log.h
--- bip-0.8.6.orig/src/log.h	2010-09-12 16:47:57.000000000 +0000
+++ bip-0.8.6/src/log.h	2010-09-19 23:47:46.000000000 +0000
@@ -59,10 +59,10 @@ typedef struct log
 	int backlogging;
 	int lastfile_seeked;
 
-	struct user *user;
+	struct bipuser *user;
 } log_t;
 
-log_t *log_new(struct user *user, const char *network);
+log_t *log_new(struct bipuser *user, const char *network);
 void logdata_free(log_t *logdata);
 
 void log_join(log_t *logdata, const char *ircmask, const char *channel);
