--- abcde.svn.freedb2	2006-10-04 16:44:57.457570805 -0500
+++ abcde.svn.concatenate	2006-10-05 15:20:32.214134344 -0500
@@ -3212,7 +3212,7 @@
 		X) CUE2DISCID="$OPTARG" ;;
 		w) COMMENT="$OPTARG" ;;
 		W) if echo $OPTARG | grep "[[:digit:]]" > /dev/null 2>&1 ; then 
-		     STARTTRACKNUMBER="${OPTARG}01" ; STARTTRACKNUMBERTAG="y" ; COMMENT="CD${OPTARG}"
+		     STARTTRACKNUMBER="${OPTARG}$(printf %02d ${STARTTRACKNUMBER:-01})" ; STARTTRACKNUMBERTAG="y" ; COMMENT="CD${OPTARG}"
 		   else
 		     log error "argument of -W must be integer"
 			 exit 1
