--- /usr/bin/dupload.orig	2008-05-06 23:22:41.000000000 +0300
+++ /usr/bin/dupload	2008-05-06 23:25:01.000000000 +0300
@@ -70,6 +70,7 @@
     %dir, %changes, %log, %announce,    # ...
     %extra,
     $suspicious_but_proceed,
+    $allow_unreleased,
 );
 
 ### Prototypes
@@ -154,6 +155,7 @@
   $dinstall_runs = $nick->{dinstall_runs};
   $nonus = $nick->{nonus};
   $passive = $nick->{passive}; 
+  $allow_unreleased = $nick->{allow_unreleased};
   if ($passive and ($method ne "ftp")) { 
       fatal ("Passive mode is only for FTP ($host)");
   }
@@ -315,7 +317,7 @@
 			/\Wstable/i and $mailto{$mailto}++;
 			/\Wunstable/i and $mailto{$mailtx}++;
 			/\Wexperimental/i and $mailto{$mailtx}++;
-			/\WUNRELEASED/ and fatal "distribution: UNRELEASED";
+			!$allow_unreleased and /\WUNRELEASED/ and fatal "distribution: UNRELEASED";
 			$field = undef;
 			next;
 		};
