--- ../../rules.orig	2010-02-17 11:50:28.000000000 +1100
+++ rules	2010-02-17 11:53:31.000000000 +1100
@@ -9,5 +9,12 @@
 	debian/desktop-file-validate.1 \
 	debian/desktop-file-install.8
 
+# misc/Makefile only installs desktop-entry-mode.el if have an emacs at
+# build time, so explicit dh_install to ensure always.
+#
+# "rm" of anything misc/Makefile does install since it includes an emacs
+# flavour-specific .elc file.
+#
 binary-install/desktop-file-utils::
 	rm -rf debian/desktop-file-utils/usr/share/emacs
+	dh_install misc/desktop-entry-mode.el /usr/share/emacs/site-lisp
