diff --git a/pdf.utf-8~monospace b/pdf.utf-8
index 1280f25..c1c3d68 100644
--- a/pdf.utf-8~monospace
+++ b/pdf.utf-8
@@ -28,5 +28,5 @@ charset utf8
 # printing.
 #
 
-0000 04FF ltor single monospace monospace:bold monospace:oblique monospace:bold:oblique
-0500 05FF rtol single monospace
+0000 04FF ltor single FreeMono FreeMono:bold FreeMono:oblique FreeMono:bold:oblique
+0500 05FF rtol single FreeMono
