diff --git a/scripts/dpkg-divert.pl b/scripts/dpkg-divert.pl
index 012be90..7168c17 100755
--- a/scripts/dpkg-divert.pl
+++ b/scripts/dpkg-divert.pl
@@ -270,6 +270,10 @@ sub checkrename {
         quit(sprintf(_g("cannot stat old name \`%s': %s"), $rsrc, $!));
     (@sdest = lstat($rdest)) || $! == ENOENT ||
         quit(sprintf(_g("cannot stat new name \`%s': %s"), $rdest, $!));
+    if ($rsrc eq $rdest) {
+        quit(sprintf(_g("will not divert %s to itself"), $rsrc));
+    }
+ 
     # Unfortunately we have to check for write access in both
     # places, just having +w is not enough, since people do
     # mount things RO, and we need to fail before we start
