--- pkg_columnizer.cc.orig	2011-04-29 22:09:59.214666709 +0900
+++ pkg_columnizer.cc	2011-04-29 22:10:46.090737891 +0900
@@ -53,8 +53,8 @@
   {1, false, false},    // stateflag
   {1, false, false},    // actionflag
   {40, true, true},     // description
-  {10, false, false},   // currver
-  {10, false, false},   // candver
+  {15, false, false},   // currver
+  {15, false, false},   // candver
   {11, false, false},   // longstate
   {10, false, false},   // longaction
   {35, true, true},     // maintainer
@@ -65,7 +65,7 @@
   {1, false, false},    // autoset
   {1, false, false},    // tagged
   {10, true, true},     // archive
-  {7, false, false},    // sizechange
+  {8, false, false},    // sizechange
   {strlen(PACKAGE), false, false},  // progname
   {strlen(VERSION), false, false},  // progver
   {12, false, true},    // brokencount
