From 18c297db3557315a9ab844d1f6c4bdaceabbe987 Mon Sep 17 00:00:00 2001
From: Shengjing Zhu <i@zhsj.me>
Date: Sat, 25 Aug 2018 23:57:58 +0800
Subject: [PATCH] Fix index out of range when list godeps sources

Signed-off-by: Shengjing Zhu <i@zhsj.me>
---
 script/dh_golang | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/script/dh_golang b/script/dh_golang
index 73ded48..4dc66ab 100755
--- a/script/dh_golang
+++ b/script/dh_golang
@@ -77,7 +77,7 @@ my $tmpl = '{{ range .Deps }}{{.}}
 my @godeps = exec_single(qq{go list -f '$tmpl'}, @targets);
 
 my $gofiletmpl = '\
-{{ .Dir }}/{{ index (or .GoFiles .CgoFiles .TestGoFiles .XTestGoFiles .IgnoredGoFiles) 0 }}';
+{{ if $x := (or .GoFiles .CgoFiles .TestGoFiles .XTestGoFiles .IgnoredGoFiles) }}{{ .Dir }}/{{ index $x 0 }}{{ end }}';
 my @gofiles = exec_chunked(qq{go list -f '$gofiletmpl'}, uniq(@godeps));
 
 my @realpath;
-- 
2.18.0

