Index: b/contrib/apt-cacher-ng.service
===================================================================
--- a/contrib/apt-cacher-ng.service	2013-05-12 17:58:14.000000000 +0200
+++ b/contrib/apt-cacher-ng.service	2013-09-14 14:39:10.647397966 +0200
@@ -7,6 +7,10 @@
 <!DOCTYPE service-group SYSTEM "avahi-service.dtd">
 <service-group>
  <name replace-wildcards="yes">apt-cacher-ng proxy on %h</name>
+ <service protocol="ipv6">
+  <type>_apt_proxy._tcp</type>
+  <port>3142</port>
+ </service>
  <service protocol="ipv4">
   <type>_apt_proxy._tcp</type>
   <port>3142</port>
