--- FindQt4.cmake.orig	2008-06-22 12:31:58.000000000 +0000
+++ FindQt4.cmake	2008-06-22 12:37:41.000000000 +0000
@@ -944,13 +944,13 @@
     )
 
   FIND_PROGRAM(QT_LUPDATE_EXECUTABLE
-    NAMES lupdate
+    NAMES lupdate-qt4 lupdate
     PATHS ${QT_BINARY_DIR}
     NO_DEFAULT_PATH
     )
 
   FIND_PROGRAM(QT_LRELEASE_EXECUTABLE
-    NAMES lrelease
+    NAMES lrelease-qt4 lrelease
     PATHS ${QT_BINARY_DIR}
     NO_DEFAULT_PATH
     )
