diff -Nru debsums-2.0.52+nmu1/debian/changelog debsums-2.0.52+nmu2/debian/changelog
--- debsums-2.0.52+nmu1/debian/changelog	2013-08-03 15:05:07.000000000 +0200
+++ debsums-2.0.52+nmu2/debian/changelog	2014-05-29 19:50:49.000000000 +0200
@@ -1,3 +1,11 @@
+debsums (2.0.52+nmu2) unstable; urgency=medium
+
+  * Non-maintainer upload
+  * Suppress reporting conffiles which were moved to a new package as
+    modified in the old package. (Closes: #744398)
+
+ -- Axel Beckert <abe@debian.org>  Thu, 29 May 2014 19:50:48 +0200
+
 debsums (2.0.52+nmu1) unstable; urgency=low
 
   * Non-maintainer upload
diff -Nru debsums-2.0.52+nmu1/debsums debsums-2.0.52+nmu2/debsums
--- debsums-2.0.52+nmu1/debsums	2013-07-30 00:20:54.000000000 +0200
+++ debsums-2.0.52+nmu2/debsums	2014-05-29 17:23:58.000000000 +0200
@@ -462,6 +462,13 @@
 	    return 0;
 	}
 
+	my $correct_package = `dpkg-query "--admindir=$DPKG" -S "$path" | awk -F: '{print \$1}'`;
+	chomp($correct_package);
+	if ($pack ne $correct_package) {
+	    #print "$pack != $correct_package\n";
+	    return 0;
+	}
+
 	if ($changed)
 	{
 	    print $root, "/", $path, "\n";
