diff -Naur backuppc-3.1.0.ori/lib/BackupPC/CGI/EditConfig.pm backuppc-3.1.0/lib/BackupPC/CGI/EditConfig.pm
--- backuppc-3.1.0.ori/lib/BackupPC/CGI/EditConfig.pm	2009-10-05 08:04:01.000000000 -0400
+++ backuppc-3.1.0/lib/BackupPC/CGI/EditConfig.pm	2009-10-05 10:48:48.000000000 -0400
@@ -368,6 +368,9 @@
         ErrorExit(eval("qq{$Lang->{Only_privileged_users_can_edit_config_files}}"));
     }
 
+    # Debian: ClientNameAlias is too dangerous, disable it
+    $bpc->{Conf}{CgiUserConfigEdit}{ClientNameAlias} = 0;
+
     if ( defined($In{menu}) || $In{saveAction} eq "Save" ) {
 	$errors = errorCheck();
 	if ( %$errors ) {
@@ -430,9 +433,6 @@
         # available per-host settings.
         #
         
-        # Debian: ClientNameAlias is too dangerous, disable it
-        $bpc->{Conf}{CgiUserConfigEdit}{ClientNameAlias} = 0;
-        
         foreach my $m ( keys(%ConfigMenu) ) {
             my $enabled = 0;
             my $text = -1;
