diff -u -r -N nmu-kangaroo/debian/changelog qa-kangaroo/debian/changelog
--- nmu-kangaroo/debian/changelog	2018-11-10 06:02:59.000000000 -0800
+++ qa-kangaroo/debian/changelog	2019-02-03 09:39:39.272895685 -0800
@@ -1,9 +1,9 @@
-cowsay (3.03+dfsg2-5.1) unstable; urgency=medium
+cowsay (3.03+dfsg2-6) unstable; urgency=medium
 
-  * Non-maintainer upload.
-  * Add kangaroo cow. 
+  * QA upload.
+  * Add kangaroo cow (closes: #700576).
 
- -- Paul Hardy <unifoundry@unifoundry.com>  Sat, 10 Nov 2018 05:40:41 -0800
+ -- Paul Hardy <unifoundry@unifoundry.com>  Sun, 03 Feb 2019 09:39:28 -0800
 
 cowsay (3.03+dfsg2-5) unstable; urgency=medium
 
diff -u -r -N nmu-kangaroo/debian/copyright qa-kangaroo/debian/copyright
--- nmu-kangaroo/debian/copyright	2018-11-10 06:20:40.000000000 -0800
+++ qa-kangaroo/debian/copyright	2019-02-03 09:35:54.219604693 -0800
@@ -46,7 +46,7 @@
 Files: cows/kangaroo.cow
 Comment: Adapted from http://sumaleth.com/files/row_ascii_collection_1.2.html.
 Copyright: Copyright (C) 1994 Rowan Crawford.
-License: GPL2+
+License: GPL-2+
  This package is free software; you can redistribute it and/or modify
  it under the terms of the GNU General Public License as published by
  the Free Software Foundation; either version 2 of the License, or
