--- /usr/bin/adt-run.orig	2011-11-09 08:18:48.101899155 +0100
+++ /usr/bin/adt-run	2011-11-09 08:47:09.653981505 +0100
@@ -838,6 +838,9 @@
 		tb._opened(pl)
 	tb.modified = False
  def prepare2(tb, deps_new):
+	if os.getuid() != 0:
+	    pstderr("warning: running as normal user, skipping binary package install")
+	    return
 	tb._debug('prepare2, deps_new=%s' % deps_new, 1)
 	binaries.publish()
 	tb._install_deps(deps_new)
@@ -1149,7 +1152,7 @@
 	rc = testbed.execute('testchmod-'+t.what, ['chmod','+x','--',tf])
 	if rc: bomb('failed to chmod +x %s' % tf)
 
-	if 'needs-root' not in t.restriction_names and opts.user is not None:
+	if 'needs-root' not in t.restriction_names and opts.user:
 		tfl = ['su',opts.user,'-c',tf]
 		tmpdir = '%s%s-tmpdir' % (testbed.scratch.read(True), t.what)
 		script = 'rm -rf -- "$1"; mkdir -- "$1"'
