diff --git a/init.d/davmail b/init.d/davmail
index 63c3b1c..a9e3f26 100755
--- a/init.d/davmail
+++ b/init.d/davmail
@@ -19,23 +19,23 @@ PATH=/sbin:/usr/sbin:/bin:/usr/bin
 DESC="Davmail Exchange gateway"
 NAME=davmail
 DAEMON=/usr/bin/$NAME
-DAEMON_USER=$NAME
+DAEMON_USER=_$NAME
 HOME=/var/lib/$DAEMON_USER
 PIDFILE=/var/run/$NAME.pid
-LOGFILE=/var/log/$NAME.log
+LOGFILE=/var/log/$NAME/$NAME.log
 SCRIPTNAME=/etc/init.d/$NAME
 
 
 # Exit if the package is not installed
 [ -x "$DAEMON" ] || exit 0
 
-DAEMON_ARGS="/etc/davmail.properties"
+DAEMON_ARGS="-server /etc/davmail/davmail.properties"
 
 # Create logfiles if they do not exist
 if [ ! -r "$LOGFILE" ]
 then
     touch $LOGFILE
-    chown $NAME:adm $LOGFILE
+    chown $DAEMON_USER:adm $LOGFILE
 fi
 
 # Load the VERBOSE setting and other rcS variables
