From ee8b56ba78cc0a1419e0fb6022dc7aff0dff68d8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Sandro=20Knau=C3=9F?= <mail@sandroknauss.de>
Date: Wed, 30 Sep 2015 14:50:20 +0200
Subject: [PATCH] verify not decrypt

---
 scripts/uscan.pl | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/uscan.pl b/scripts/uscan.pl
index 99f90d2..336f117 100755
--- a/scripts/uscan.pl
+++ b/scripts/uscan.pl
@@ -3084,7 +3084,7 @@ EOF
 	print "-- Verifying OpenPGP self signature of $sigfile_base and extract $newfile_base\n" if $verbose;
 	system($havegpg, '--homedir', $gpghome,
 	       '--no-options', '-q', '--batch', '--no-default-keyring',
-	       '--keyring', $keyring, '--trust-model', 'always', '--decrypt', '-o',
+	       '--keyring', $keyring, '--trust-model', 'always', '--verify', '-o',
 	       "$destdir/$newfile_base", "$destdir/$sigfile_base") >> 8 == 0
 		    or uscan_die("$progname: OpenPGP signature did not verify.\n");
 	$previousfile_base = undef;
-- 
2.1.4

