Description: Fix amcheck -M

--- amanda-3.3.1.orig/server-src/amcheck.c
+++ amanda-3.3.1/server-src/amcheck.c
@@ -448,8 +448,8 @@ main(
 	}
 	if(mailto) {
 	    a = (char **) g_new0(char *, 2);
-	    a[1] = stralloc(mailto);
-	    a[2] = NULL;
+	    a[0] = stralloc(mailto);
+	    a[1] = NULL;
 	} else {
 	    /* (note that validate_mailto doesn't allow any quotes, so this
 	     * is really just splitting regular old strings) */
