diff -ruN apt-0.7.9.orig/cmdline/apt-get.cc apt-0.7.9/cmdline/apt-get.cc
--- apt-0.7.9.orig/cmdline/apt-get.cc	2007-10-31 18:58:23.000000000 -0400
+++ apt-0.7.9/cmdline/apt-get.cc	2007-10-31 18:59:12.000000000 -0400
@@ -2553,8 +2553,11 @@
 	 _error->Discard();
       
       // Now we check the state of the packages,
-      if (Cache->BrokenCount() != 0)
-         return _error->Error(_("Build-dependencies for %s could not be satisfied."),*I);
+      if (Cache->BrokenCount() != 0) 
+      {
+	  ShowBroken(cout, Cache, false);
+	  return _error->Error(_("Build-dependencies for %s could not be satisfied."),*I);
+      }
    }
   
    if (InstallPackages(Cache, false, true) == false)
