commit 05df5d56020aadf72a66aff594599fea4c7b4802
Author: José Manuel Santamaría Lema <panfaust@gmail.com>
Date:   Fri Nov 9 14:59:20 2012 +0100

    Add missing newline in the debconf passtrough frontend output.

diff --git a/apt-listchanges/apt_listchanges.py b/apt-listchanges/apt_listchanges.py
index fe9f9c3..cc879e5 100644
--- a/apt-listchanges/apt_listchanges.py
+++ b/apt-listchanges/apt_listchanges.py
@@ -223,7 +223,7 @@ Description: ${title}
 
 Template: apt-listchanges/news
 Type: note
-Description: ${packages} packages''')
+Description: ${packages} packages\n''')
         for line in text.split('\n'):
             if line.strip():
                 tmp.write('  ' + line + '\n')
diff --git a/debian/changelog b/debian/changelog
index 0fcd095..0d000cd 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,8 +1,12 @@
 apt-listchanges (2.85.12~exp2) UNRELEASED; urgency=low
 
+  [ Sandro Tosi ]
   * Update the program Spanish PO file; thanks to Javier Fernandez-Sanguino;
     Closes: #686426
 
+  [ José Manuel Santamaría Lema ]
+  * Add missing newline in the debconf passtrough frontend output.
+
  -- Sandro Tosi <morph@debian.org>  Sat, 01 Sep 2012 22:21:15 +0200
 
 apt-listchanges (2.85.12~exp1) experimental; urgency=low
