--- README.Debian.orig	2011-11-11 12:50:04.000000000 +0000
+++ README.Debian	2011-11-11 12:54:29.571864691 +0000
@@ -1,17 +1,17 @@
 Handling of obsolete configuration files/scripts
 ------------------------------------------------
 
-Since dpkg doesn't not remove conffiles that are no more provided by the
-current version of the package, we have to disable them manually so that they
-don't create troubles.
+As dpkg doesn't remove conffiles that are no longer provided by the
+current version of the package, we have to disable them manually so
+that they don't create trouble.
 
 Older versions of this package renamed all these files by adding a ".obsolete"
-extension. Newer version only rename those files that were changed. Unchanged
-files are simply deleted. If you find out such files in
+extension. Newer versions only rename those files that were changed. Unchanged
+files are simply deleted. If you find such files in
 /etc/acpi/{events,suspend.d,resume.d}, you can safely remove them. 
 
 This renaming is required because otherwise they would be actively used
-while they are no more supposed to be there.
+even though they are supposed to have been removed.
 
 You can use the following command to find the files that have been renamed:
 $ dpkg -s acpi-support | grep obsolete$
@@ -27,13 +27,13 @@
 keyboard the old approach of acpi_fakekey to just send the needed event through
 an existing keyboard device does not always work. As a workaround acpi_fakekey
 now sends the generated key press event through the user level input driver and
-thus needs CONFIG_INPUT_UINPUT defined in the kernel and, if build as a module,
+thus needs CONFIG_INPUT_UINPUT defined in the kernel and, if built as a module,
 the uinput module to be loaded.
 
 The "acpi-support" method for suspending.
 -----------------------------------------
 
-Older version of acpi-support had their own scripts to control suspend and
+Older versions of acpi-support had their own scripts to control suspend and
 resume. This has been deprecated for quite some time and finally has been
 removed in 0.122-1. The scripts are stil available under
 /usr/share/doc/acpi-support/examples and can easily be copied back to their
