diff -ur e2fsprogs-1.41.7.pristine/debian/control e2fsprogs-1.41.7/debian/control
--- e2fsprogs-1.41.7.pristine/debian/control	2009-06-01 12:48:42.000000000 +0100
+++ e2fsprogs-1.41.7/debian/control	2009-07-02 13:20:32.000000000 +0100
@@ -194,12 +194,15 @@
 Replaces: e2fsprogs (<< 1.34-1)
 Provides: libext2fs2, libe2p2
 Architecture: any
-Description: ext2 filesystem libraries
- The ext2fs and e2p libraries are used by programs that directly access 
- EXT2 filesystems from usermode programs.   The EXT2 filesystem is very often
- used as the default filesystem on Linux systems.   Various system programs
- that use libext2fs include e2fsck, mke2fs, tune2fs, etc.  Programs that use
- libe2p include dumpe2fs, chattr, and lsattr.
+Description: ext2/ext3/ext4 file system libraries
+ The ext2, ext3 and ext4 file systems are successors of the original ext
+ ("extended") file system. They are the main file system types used for
+ hard disks on Debian and other Linux systems.
+ .
+ This package provides the ext2fs and e2p libraries, for userspace software
+ that directly accesses extended file systems. Programs that use libext2fs
+ include e2fsck, mke2fs, and tune2fs. Programs that use libe2p include
+ dumpe2fs, chattr, and lsattr.
 
 Package: e2fslibs-dev
 Section: libdevel
@@ -209,12 +212,10 @@
 Provides: ext2fs-dev, e2p-dev
 Replaces: libkrb5-dev (<< 1.3)
 Architecture: any
-Description: ext2 filesystem libraries - headers and static libraries
- The ext2fs and e2p libraries are used by programs that directly access 
- EXT2 filesystems from usermode programs.   The EXT2 filesystem is very often
- used as the default filesystem on Linux systems.   Various system programs
- that use libext2fs include e2fsck, mke2fs, tune2fs, etc.  Programs that use
- libe2p include dumpe2fs, chattr, and lsattr.
+Description: ext2/ext3/ext4 file system libraries - headers and static libraries
+ The ext2, ext3 and ext4 file systems are successors of the original ext
+ ("extended") file system. They are the main file system types used for
+ hard disks on Debian and other Linux systems.
  .
  This package contains the development environment for the ext2fs and e2p 
  libraries.
@@ -239,7 +240,7 @@
 Priority: extra
 Depends: e2fsprogs (= ${binary:Version})
 Architecture: any
-Description: Debugging information for e2fsprogs
+Description: debugging information for e2fsprogs
  This package includes the debug information useful for debugging e2fsprogs
  and its libraries, contained in the e2fsprogs and e2fsck-static packages.
  The debug information is used for execution tracing and core
@@ -250,7 +251,7 @@
 Priority: extra
 Depends: uuid-runtime (= ${binary:Version})
 Architecture: any
-Description: Debugging information for uuid-runtime
+Description: debugging information for uuid-runtime
  This package includes the debug information useful for debugging the
  uuid runtime programs, contained in the uuid-runtime package.
  The debugging information is used for execution tracing and core
@@ -261,7 +262,7 @@
 Priority: extra
 Depends: e2fslibs (= ${binary:Version})
 Architecture: any
-Description: Debugging information for e2fslibs
+Description: debugging information for e2fslibs
  This package includes the debug information useful for debugging the
  ext2fs and e2p libraries, contained in the e2fslibs package.  The debug
  information is used for execution tracing and core dump analysis.
@@ -271,7 +272,7 @@
 Priority: extra
 Depends: libcomerr2 (= ${binary:Version})
 Architecture: any
-Description: Debugging information for libcomerr2
+Description: debugging information for libcomerr2
  This package includes the debug information useful for debugging the
  com_err library, contained in the libcomerr2 package.  The debugging
  information is used for execution tracing and core dump analysis.
@@ -281,7 +282,7 @@
 Priority: extra
 Depends: libss2 (= ${binary:Version})
 Architecture: any
-Description: Debugging information for libss2
+Description: debugging information for libss2
  This package includes the debug information useful for debugging the
  ss library, contained in the libss2 package.  The debug information
  is used for execution tracing and core dump analysis.
@@ -291,7 +292,7 @@
 Priority: extra
 Depends: libblkid1 (= ${binary:Version})
 Architecture: any
-Description: Debugging information for libblkid1
+Description: debugging information for libblkid1
  This package includes the debug information useful for debugging the
  blkid library, contained in the libblkid1 package.  The debug
  information is used for execution tracing and core dump analysis.
@@ -301,7 +302,7 @@
 Priority: extra
 Depends: libuuid1 (= ${binary:Version})
 Architecture: any
-Description: Debugging information for libuuid1
+Description: debugging information for libuuid1
  This package includes the debug information useful for debugging the
  UUID library, contained in the libuuid1 package.  The debug
  information is used for execution tracing and core dump analysis.
