From 318a187538ad69414a5d02c16910cb8ab6b9d414 Mon Sep 17 00:00:00 2001
From: Konstantin Manna <Konstantin.Manna@rwth-aachen.de>
Date: Sun, 15 Jun 2014 03:18:47 +0200
Subject: [PATCH] Fix #709236

---
 apt-pkg/cdrom.cc |    1 -
 1 file changed, 1 deletion(-)

diff --git a/apt-pkg/cdrom.cc b/apt-pkg/cdrom.cc
index a5ad6a9..93661fb 100644
--- a/apt-pkg/cdrom.cc
+++ b/apt-pkg/cdrom.cc
@@ -496,7 +496,6 @@ bool pkgCdrom::WriteSourceList(string Name,vector<string> &List,bool Source)
       if (F.fail() && !F.eof())
 	 return _error->Error(_("Line %u too long in source list %s."),
 			      CurLine,File.c_str());
-      _strtabexpand(Buffer,sizeof(Buffer));
       _strstrip(Buffer);
             
       // Comment or blank
-- 
1.7.10.4

