diff -NurbBp apt-1.1.4/doc/examples/configure-index apt-1.1.4-doc/doc/examples/configure-index
--- apt-1.1.4/doc/examples/configure-index	2015-12-07 15:31:55.000000000 +0100
+++ apt-1.1.4-doc/doc/examples/configure-index	2015-12-08 19:49:42.552159198 +0100
@@ -107,6 +107,9 @@ APT
   // be installed by default
   Install-Recommends "true";
   Install-Suggests "false";
+  // reverse Recommends or Suggests prevent autoremoval
+  AutoRemove::RecommendsImportant "true";
+  AutoRemove::SuggestsImportant "true";
 
   // consider dependencies of packages in this section manual
   Never-MarkAuto-Sections {"metapackages"; "universe/metapackages"; };
