diff --git a/pkgs.dbk b/pkgs.dbk
index af8bcf0..6959ac8 100644
--- a/pkgs.dbk
+++ b/pkgs.dbk
@@ -1923,8 +1923,15 @@ Before doing an NMU, consider the following questions:
 <itemizedlist>
 <listitem>
 <para>
-Does your NMU really fix bugs? Fixing cosmetic issues or changing the
-packaging style in NMUs is discouraged.
+Will the NMU help the maintainer?
+</para>
+</listitem>
+<listitem>
+<para>
+Does your NMU really fix bugs? ("Bugs" includes wishlist bugs for packaging
+a new upstream version, but care should be taken to minimize the impact to
+the maintianer.) Fixing cosmetic issues or changing the packaging style
+(dh, cdbs, etc) in NMUs is discouraged.
 </para>
 </listitem>
 <listitem>
