diff --git a/pkgs.dbk b/pkgs.dbk
index af8bcf0..c4f3d1d 100644
--- a/pkgs.dbk
+++ b/pkgs.dbk
@@ -1923,8 +1923,20 @@ Before doing an NMU, consider the following questions:
 <itemizedlist>
 <listitem>
 <para>
-Does your NMU really fix bugs? Fixing cosmetic issues or changing the
-packaging style in NMUs is discouraged.
+Have you geared the NMU towards helping the maintainer? As there might
+be disagreement on the notion of whether the maintainer actually needs
+help on not, the DELAYED queue exists to give time to the maintainer to
+react and has the beneficial side-effect of allowing for independent
+reviews of the NMU diff.
+</para>
+</listitem>
+<listitem>
+<para>
+Does your NMU really fix bugs? ("Bugs" means any kind of bugs, e.g.
+wishlist bugs for packaging a new upstream version, but care should be
+taken to minimize the impact to the maintainer.) Fixing cosmetic issues
+or changing the packaging style (e.g. switching from cdbs to dh) in NMUs
+is discouraged.
 </para>
 </listitem>
 <listitem>
