Index: control
===================================================================
--- control	(revision 20667)
+++ control	(working copy)
@@ -54,7 +54,7 @@
 
 Package: digikam
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, ${perl:Depends}, digikam-private-libs (= ${binary:Version}), libqt5sql5-sqlite, digikam-data (= ${source:Version})
+Depends: ${shlibs:Depends}, ${misc:Depends}, ${perl:Depends}, digikam-private-libs (= ${binary:Version}), libqt5sql5-sqlite, libqt5sql5-mysql, digikam-data (= ${source:Version})
 Recommends: www-browser, ffmpegthumbs
 Suggests: digikam-doc, systemsettings
 Conflicts: ${digikam:Conflicts}
