diff -Nru automysqlbackup-2.6+debian.2/debian/changelog automysqlbackup-2.6+debian.2/debian/changelog
--- automysqlbackup-2.6+debian.2/debian/changelog	2012-05-25 13:59:17.000000000 +0200
+++ automysqlbackup-2.6+debian.2/debian/changelog	2013-01-30 22:53:57.000000000 +0100
@@ -1,3 +1,10 @@
+automysqlbackup (2.6+debian.2-1.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * cron.daily: Do not fail after the package was removed.  (Closes: #633450) 
+
+ -- Andreas Beckmann <anbe@debian.org>  Wed, 30 Jan 2013 22:53:57 +0100
+
 automysqlbackup (2.6+debian.2-1) unstable; urgency=low
 
   * Doesn't backup the performance_schema and information_schema databases by
diff -Nru automysqlbackup-2.6+debian.2/debian/cron.daily automysqlbackup-2.6+debian.2/debian/cron.daily
--- automysqlbackup-2.6+debian.2/debian/cron.daily	2012-05-25 13:59:17.000000000 +0200
+++ automysqlbackup-2.6+debian.2/debian/cron.daily	2013-01-26 05:32:26.000000000 +0100
@@ -1,2 +1,3 @@
 #!/bin/sh
-test -x /usr/sbin/automysqlbackup && /usr/sbin/automysqlbackup
+test -x /usr/sbin/automysqlbackup || exit 0
+/usr/sbin/automysqlbackup
