Description: Fix build with gcc 6
Bug-Debian: https://bugs.debian.org/811640
Origin: upstream, https://git.gnome.org/browse/dasher/commit/?id=a271b6753a9ab154d68ce96f1a16dd932bfd2a22

--- dasher-4.11+git20130508.adc653.orig/Src/Gtk2/DasherAppSettings.cpp
+++ dasher-4.11+git20130508.adc653/Src/Gtk2/DasherAppSettings.cpp
@@ -395,7 +395,7 @@ const gchar *dasher_app_settings_get_str
     if(pPrivate->pDasherWidget)
       return gtk_dasher_control_get_parameter_string(pPrivate->pDasherWidget, iParameter);
     else
-      return false;
+      return nullptr;
   }
   else
     return app_stringparamtable[ iParameter - FIRST_APP_SP ].value;
