--- equivs-2.0.7/usr/bin/equivs-build	2006-02-28 10:33:25.000000000 +0200
+++ usr/bin/equivs-build	2007-02-10 21:53:38.000000000 +0200
@@ -154,6 +154,10 @@
 
   $control->{'Architecture'} = $specific_arch ? 'any' : 'all';
 
+  # Fix Source: entry, part I: remove any silly predefied "equivs-dummy"
+
+  undef $control->{'Source'} if ($control->{'Source'} eq 'equivs-dummy');
+
   open($in, $file) or
     die "Cannot open control file $file: $!\n";
 
@@ -161,8 +165,8 @@
     die "error: empty control file\n";
   close $in;
 
-  # Fix Source: entry
-  $control->{'Source'} = $control->{'Package'};
+  # If no Source entry was specified, copy Package:
+  $control->{'Source'} ||= $control->{'Package'};
 
   # remove trailing whitespace
 #  foreach my $key (keys %$control) {
@@ -251,7 +255,7 @@
   open OUT, '>', "$builddir/debian/changelog" or
     die "Couldn't write changelog: $!\n";
   print OUT <<EOINPUT;
-$control->{Package} ($version) unstable; urgency=low
+$control->{Source} ($version) unstable; urgency=low
 
   * First version
 
