  * equivs-build: Don't override Source: if it is present

--- equivs-2.0.6/usr/bin/equivs-build	2004-05-29 04:29:30.000000000 +0300
+++ equivs-2.0.6-0.1era1/usr/bin/equivs-build	2007-02-04 00:55:42.000000000 +0200
@@ -206,14 +206,18 @@
 
   $control{"Maintainer"} = "$fullname <$username\@$systemname>";
 
+  # Fix Source: entry, part I: remove any silly predefied "equivs-dummy"
+
+  undef $control{"Source"} if ($control{"Source"} eq "equivs-dummy");
+
   open(IN, $file) or 
         die "$file: cannot open control file for reading: $!";
 
   read_control_file_section(\%control) or die "error: empty control file";
 
-  # Fix Source: entry
+  # If no Source entry was specified, copy Package:
 
-  $control{"Source"} = $control{"Package"};
+  $control{"Source"} ||= $control{"Package"};
 
   # remove trailing whitespace
   
@@ -331,7 +335,7 @@
   my $version = $control{"Version"} || "1.0";
   
   print OUT <<EOINPUT;
-$control{Package} ($version) unstable; urgency=low
+$control{Source} ($version) unstable; urgency=low
 
   * First version
 
