diff --recursive -uN apt-0.8.7/apt-pkg/deb/dpkgpm.cc apt-0.8.7.lio/apt-pkg/deb/dpkgpm.cc
--- apt-0.8.7/apt-pkg/deb/dpkgpm.cc	2010-09-06 18:08:43.000000000 +0200
+++ apt-0.8.7.lio/apt-pkg/deb/dpkgpm.cc	2010-10-28 06:22:33.000000000 +0200
@@ -669,7 +669,7 @@
 				   _config->Find("Dir::Log::History"));
    if (!history_name.empty())
    {
-      history_out = fopen(history_name.c_str(),"a");
+      history_out = fopen(history_name.c_str(),"ae");
       if (history_out == NULL)
 	 return _error->WarningE("OpenLog", _("Could not open file '%s'"), history_name.c_str());
       chmod(history_name.c_str(), 0644);
