*** ding.orig	Sun Dec  6 14:55:41 2009
--- ding	Tue Mar 16 14:23:46 2010
***************
*** 3341,3347 ****
                          set cmd "$t tag configure $tag -elide \[expr \[set history_fold($num)($foldcount)\]\]"
                          checkbutton $t.fold$foldcount -image minus -selectimage plus \
                             -variable history_fold($num)($foldcount) -indicatoron 0 \
!                            -bd 0 -width 8 -height 8 -selectcolor {} \
                             -command $cmd
                          $t window create end -window $t.fold$foldcount
                          $t insert end " [lindex $l2 $i]\n" $bgtag
--- 3341,3347 ----
                          set cmd "$t tag configure $tag -elide \[expr \[set history_fold($num)($foldcount)\]\]"
                          checkbutton $t.fold$foldcount -image minus -selectimage plus \
                             -variable history_fold($num)($foldcount) -indicatoron 0 \
!                            -bd 0 -width 8 -height 8 \
                             -command $cmd
                          $t window create end -window $t.fold$foldcount
                          $t insert end " [lindex $l2 $i]\n" $bgtag
