--- tls-gnu.c.org	2007-10-21 23:51:28.000000000 +0300
+++ tls-gnu.c	2007-10-21 23:58:12.000000000 +0300
@@ -17,6 +17,7 @@
 
 /* Heading stuff for GnuTLS */
 
+#include <gcrypt.h>
 #include <gnutls/gnutls.h>
 #include <gnutls/x509.h>
 
@@ -444,6 +445,10 @@
 
 initialized = (host == NULL)? INITIALIZED_SERVER : INITIALIZED_CLIENT;
 
+/* To use /dev/urandom even for key generation
+ */
+gcry_control (GCRYCTL_ENABLE_QUICK_RANDOM, 0);
+
 rc = gnutls_global_init();
 if (rc < 0) return tls_error(US"tls-init", host, rc);
 
