=== modified file 'Packages-arch-specific'
--- Packages-arch-specific	2009-07-30 14:46:03 +0000
+++ Packages-arch-specific	2009-08-26 01:36:19 +0000
@@ -280,7 +280,6 @@
 powerpc-ibm-utils: powerpc					      # powerpc specific
 powerpc-utils: powerpc						      # powerpc specific
 %prctl: hppa ia64 alpha						      # ANAIS based on syscall availability
-%prelink: alpha amd64 i386 lpia powerpc				      # No upstream support for others
 primaxscan: i386 kfreebsd-i386 amd64 kfreebsd-amd64		      # [?]
 %proll: sparc							      # Build-deps sparc-utils
 %ps3-utils: powerpc						      # powerpc specific

