--- init-d-script.orig	2018-01-11 01:04:11.340881210 +0300
+++ init-d-script	2018-01-11 01:10:20.426711408 +0300
@@ -1,6 +1,11 @@
 #!/bin/sh
 # See init-d-script(5) for instructions on how to use this library.
 #=============================================================================
+
+# Shift arguments early to fix #826214
+__init_d_script_name="$1"
+shift
+
 # Define LSB log_* functions.
 # Depend on lsb-base (>= 3.2-14) to ensure that this file is present
 # and status_of_proc is working.
@@ -153,12 +158,10 @@
     set -x
 fi
 
-SCRIPTNAME=$1
-scriptbasename="$(basename $1)"
+SCRIPTNAME="$__init_d_script_name"
+scriptbasename="$(basename "$__init_d_script_name")"
 if [ "$scriptbasename" != "init-d-script" ] ; then
-    script="$1"
-    shift
-    . $script
+    . "$__init_d_script_name"
 else
     exit 0
 fi
