diff --git a/debian/apache2.preinst b/debian/apache2.preinst
index c3e49c9..b8d8805 100644
--- a/debian/apache2.preinst
+++ b/debian/apache2.preinst
@@ -105,9 +105,8 @@ case "$1" in
 
 	# 1) an apache2-mpm package exists
 	if [ -d "/etc/apache2/" ] ; then
-		mpm=$(dpkg-query -f '${Package}\t${Status}\n'  -W 'apache2-mpm-*' 2>/dev/null) || true
-		if [ $? -eq 0 ] ; then
-			mpm=$(echo "$mpm" | grep "install ok" | cut -f1)
+		mpm=$(dpkg-query -f '${Package}\t${Status}\n'  -W 'apache2-mpm-*' 2>/dev/null | grep "install ok" | cut -f1)
+		if [ -n "$mpm" ] ; then
 			if [ ! -f /etc/apache2/.apache2_mpm_selected ] ; then
 				echo "# automatically created during upgrade" >> /etc/apache2/.apache2_mpm_selected
 				echo "# it can be safely removed anytime" >> /etc/apache2/.apache2_mpm_selected
