diff -au autolog-0.40.orig/autolog.c autolog-0.40/autolog.c
--- autolog-0.40.orig/autolog.c	2010-10-29 16:15:11.000000000 +0200
+++ autolog-0.40/autolog.c	2010-10-29 16:14:27.000000000 +0200
@@ -1082,6 +1082,18 @@
 		}
 	    }
 	}
+	/* Read to the end of line to avoid parsing the rest of the command
+	 * line in next round and getting a segfault as ps_pid will be null!!
+	 */
+	i=strlen(iline);
+	while(iline[i-1] != '\n'){
+	    if(!fgets(iline, LINELEN, ps)) {
+	      /* end of file, exit*/
+	      fclose(ps);
+	      return;
+	    }
+	    i=strlen(iline);
+	}
     }
     fclose(ps);
 }
Les sous-répertoires autolog-0.40.orig/debian et autolog-0.40/debian sont identiques.
Les sous-répertoires autolog-0.40.orig/.pc et autolog-0.40/.pc sont identiques.
