Index: src/modules/frontend/gtk/gtk.c
===================================================================
--- src/modules/frontend/gtk/gtk.c	(revision 40817)
+++ src/modules/frontend/gtk/gtk.c	(working copy)
@@ -49,9 +49,13 @@
 #include <debian-installer/slist.h>
 #include <gdk/gdkkeysyms.h>
 
+#if GTK_CHECK_VERSION(2,10,0)
 #ifdef GDK_WINDOWING_DIRECTFB
 #include <directfb.h>
 #endif
+#else
+#include <directfb.h>
+#endif
 
 #define WINDOW_WIDTH 800
 #define WINDOW_HEIGHT 600
@@ -1543,10 +1547,15 @@
      * for dfb to support automatic keymap change detection and reloading
      * (See also bug #381979)
      */
+
+    #if GTK_CHECK_VERSION(2,10,0)
     #ifdef GDK_WINDOWING_DIRECTFB
     dfb_input_device_reload_keymap( dfb_input_device_at( DIDID_KEYBOARD ) );
     #endif
-    
+    #else
+    dfb_input_device_reload_keymap( dfb_input_device_at( DIDID_KEYBOARD ) );
+    #endif
+
     gtk_rc_reparse_all();
 
     questionbox = gtk_vbox_new(FALSE, 0);
