From 72cd863bf36e43077449fa2cfeede1167575d5a4 Mon Sep 17 00:00:00 2001
From: Peter Pentchev <roam@ringlet.net>
Date: Mon, 7 Sep 2009 15:04:55 +0300
Subject: [PATCH] Only invoke dh_installcatalogs once during "dh install".

---
 dh |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/dh b/dh
index 710117e..cd2f9f0 100755
--- a/dh
+++ b/dh
@@ -253,7 +253,6 @@ $sequences{install} = [@{$sequences{build}}, qw{
 	dh_installcatalogs
 	dh_installcron
 	dh_installdebconf
-	dh_installcatalogs
 	dh_installemacsen
 	dh_installifupdown
 	dh_installinfo
-- 
1.6.3.3

