--- /usr/share/acpi-support/policy-funcs	2011-12-13 15:00:19.000000000 +0100
+++ policy-funcs	2012-01-11 16:49:10.711114956 +0100
@@ -21,7 +21,8 @@
           pidof dcopserver > /dev/null &&
           test -x /usr/bin/dcop &&
           /usr/bin/dcop --user $XUSER kded kded loadedModules | grep -q klaptopdaemon; } ||
-        PowerDevilRunning
+        PowerDevilRunning ||
+	GSDWithSuspend
 }
 
 PowerDevilRunning() {
@@ -36,4 +37,18 @@
 	done
 	
 	return 1
+}
+
+GSDWithSuspend() {
+	test -x /usr/bin/dbus-send || return 1
+
+	local DBUS_SESS	
+	for p in $(pidof gnome-settings-daemon); do
+		test -r /proc/$p/environ || continue
+		DBUS_SESS=$(grep -a -z "DBUS_SESSION_BUS_ADDRESS=" /proc/$p/environ || :)
+		test "$DBUS_SESS" != "" || continue
+		! su $(ps -o user= $p) -s /bin/sh -c "$DBUS_SESS dbus-send --system --print-reply --dest="org.freedesktop.UPower" --type=method_call --reply-timeout=6000 /org/freedesktop/UPower org.freedesktop.DBus.Properties.Get string:org.freedesktop.UPower string:CanSuspend" | grep -q true || return 0
+	done
+	
+	return 1
 }
