diff --git a/policy.sgml b/policy.sgml
index 9cd182b..6395dbb 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -7773,13 +7773,8 @@ test -f <var>program-executed-later-in-script</var> || exit 0
 	      <example compact="compact">/etc/init.d/&lt;package&gt;
 	      &lt;action&gt;</example> in their <prgn>postinst</prgn>
 	      and <prgn>prerm</prgn> scripts to:
-	      <example compact="compact">
-	if which invoke-rc.d >/dev/null 2>&1; then
-		invoke-rc.d <var>package</var> &lt;action&gt;
-	else
-		/etc/init.d/<var>package</var> &lt;action&gt;
-	fi
-	      </example>
+	      <example compact="compact">invoke-rc.d <var>package</var>
+	      &lt;action&gt;</example>
 	    </p>
 
 	    <p>
