Index: debian/config.proto
===================================================================
--- debian/config.proto	(revision 230)
+++ debian/config.proto	(working copy)
@@ -1367,6 +1367,10 @@
 		    db_input high console-setup/toggle || true
 		    db_input medium console-setup/switch || true
 		fi
+		if [ "$layout" = us ]; then
+		    # US keyboards don't use AltGr.
+		    db_default console-setup/altgr 'No AltGr key'
+		fi
 		db_input medium console-setup/altgr || true
 		db_input medium console-setup/compose || true
 		db_endblock
