diff -u brltty-3.10~r3724/debian/changelog brltty-3.10~r3724/debian/changelog
--- brltty-3.10~r3724/debian/changelog
+++ brltty-3.10~r3724/debian/changelog
@@ -1,3 +1,11 @@
+brltty (3.10~r3724-1.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Apply patch from Samuel Thibault to fix FTBFS in unstable.
+    (Closes: #516240)
+
+ -- Adeodato Simó <dato@net.com.org.es>  Sat, 14 Mar 2009 11:09:53 +0100
+
 brltty (3.10~r3724-1) unstable; urgency=low
 
   * New subversion snapshot, fixing FTBFS (Closes: Bug#482205).
only in patch2:
unchanged:
--- brltty-3.10~r3724.orig/configure.ac
+++ brltty-3.10~r3724/configure.ac
@@ -688,7 +688,7 @@
 case "${host_os}"
 in
    linux*|gnu*|kfreebsd*)
-      brltty_cv_prog_cc_sysflags="-D_POSIX_C_SOURCE=2 -D_BSD_SOURCE -D_XOPEN_SOURCE=500 -D_XOPEN_SOURCE_EXTENDED"
+      brltty_cv_prog_cc_sysflags="-D_POSIX_C_SOURCE=2 -D_BSD_SOURCE -D_XOPEN_SOURCE=500 -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE"
       ;;
    solaris*)
       brltty_cv_prog_cc_sysflags="-D_XOPEN_SOURCE=500 -D__EXTENSIONS__"
only in patch2:
unchanged:
--- brltty-3.10~r3724.orig/configure
+++ brltty-3.10~r3724/configure
@@ -14856,7 +14856,7 @@
   case "${host_os}"
 in
    linux*|gnu*|kfreebsd*)
-      brltty_cv_prog_cc_sysflags="-D_POSIX_C_SOURCE=2 -D_BSD_SOURCE -D_XOPEN_SOURCE=500 -D_XOPEN_SOURCE_EXTENDED"
+      brltty_cv_prog_cc_sysflags="-D_POSIX_C_SOURCE=2 -D_BSD_SOURCE -D_XOPEN_SOURCE=500 -D_XOPEN_SOURCE_EXTENDED -D_GNU_SOURCE"
       ;;
    solaris*)
       brltty_cv_prog_cc_sysflags="-D_XOPEN_SOURCE=500 -D__EXTENSIONS__"
