=== modified file 'policy.sgml'
--- policy.sgml	2010-08-18 20:55:34 +0000
+++ policy.sgml	2010-08-28 00:43:43 +0000
@@ -783,9 +783,16 @@
 
 	<p>
 	  Packages must not depend on packages with lower priority
-	  values (excluding build-time dependencies).  In order to
+	  values (excluding build-time dependencies). In order to
 	  ensure this, the priorities of one or more packages may need
-	  to be adjusted.
+	  to be adjusted.<footnote>
+	    The Priority field of an existing Debian package does not
+	    determine the priority of that package;
+	    see <ref id="f-Priority">. For this reason, the package
+	    maintainer cannot fix this directly, and it is not
+	    recommended to file bugs against packages whose source
+	    declares an incorrect Priority field.
+	  </footnote>
 	</p>
       </sect>
 
@@ -2842,6 +2849,13 @@
 	    It also gives the default for the same field in the binary
 	    packages.
 	  </p>
+
+	  <p>
+	    Once a package is in Debian, this field no longer
+	    determines the priority of the package in the archive.
+	    Instead, the Debian ftpmaster team maintains priority
+	    values in the “override file”.
+	  </p>
 	</sect1>
 
 	<sect1 id="f-Package">

