diff -aur debian-archive-keyring-2009.01.31/debian/rules debian-archive-keyring-2009.01.31-debsums/debian/rules
--- debian-archive-keyring-2009.01.31/debian/rules	2007-05-11 21:00:48.000000000 +0200
+++ debian-archive-keyring-2009.01.31-debsums/debian/rules	2009-06-28 19:24:19.000000000 +0200
@@ -39,6 +39,7 @@
 	gzip -9v debian/tmp/usr/share/doc/debian-archive-keyring/*
 	$(install_file) debian/copyright debian/tmp/usr/share/doc/debian-archive-keyring/
 
+	dh_md5sums
 	dpkg-gencontrol -pdebian-archive-keyring -isp
 	chown -R root.root debian/tmp
 	chmod -R go=rX debian/tmp
