diff -aur debian-archive-keyring-2009.01.31/debian/rules debian-archive-keyring-2009.01.31-debsums/debian/rules
--- debian-archive-keyring-2009.01.31/debian/rules	2007-05-11 21:00:48.000000000 +0200
+++ debian-archive-keyring-2009.01.31-debsums/debian/rules	2009-06-28 13:14:25.000000000 +0200
@@ -39,6 +39,7 @@
 	gzip -9v debian/tmp/usr/share/doc/debian-archive-keyring/*
 	$(install_file) debian/copyright debian/tmp/usr/share/doc/debian-archive-keyring/
 
+	dh_md5sums
 	dpkg-gencontrol -pdebian-archive-keyring -isp
 	chown -R root.root debian/tmp
 	chmod -R go=rX debian/tmp
@@ -51,6 +52,7 @@
 	$(install_file) keyrings/debian-archive-keyring.gpg debian/debian-archive-keyring-udeb/usr/share/keyrings/
 	install -m 755 debian/debian-archive-keyring-udeb.postinst debian/debian-archive-keyring-udeb/DEBIAN/postinst
 
+	dh_md5sums
 	dpkg-gencontrol -ndebian-archive-keyring-udeb_$(VERSION)_all.udeb -pdebian-archive-keyring-udeb -Tdebian/debian-archive-keyring-udeb.substvars -Pdebian/debian-archive-keyring-udeb -isp
 	chown -R root.root debian/debian-archive-keyring-udeb
 	chmod -R go=rX debian/debian-archive-keyring-udeb
