Index: project-history/Makefile
===================================================================
--- project-history/Makefile	(révision 7505)
+++ project-history/Makefile	(copie de travail)
@@ -49,8 +49,13 @@
 	if [ ! -f $(MANUAL).$$lang.html/index.$$lang.html -o $$i -nt $(MANUAL).$$lang.html/index.$$lang.html ]; then \
 	    sgmlfile=$$nosuffix.sgml; \
 	    [ $$sgmlfile != $$i ] && make $$sgmlfile; \
-	    echo debiandoc2html -c -l $$lang $$sgmlfile; \
-            debiandoc2html -c -l $$lang $$sgmlfile; \
+	    if [ $$lang != fr ]; then \
+	      echo debiandoc2html -c -l $$lang $$sgmlfile; \
+	      debiandoc2html -c -l $$lang $$sgmlfile; \
+	    else \
+	      echo debiandoc2html -C -l $$lang.UTF-8 $$sgmlfile; \
+	      debiandoc2html -C -l $$lang.UTF-8 $$sgmlfile; \
+	    fi; \
           fi; \
         done
 
@@ -62,7 +67,13 @@
 	mv $(MANUAL).txt $(MANUAL).en.txt
 
 $(MANUAL).%.txt: $(MANUAL).%.sgml
-	debiandoc2text -l $* $<
+	@if [ $* != fr ]; then \
+	  echo debiandoc2text -l $* $<; \
+	  debiandoc2text -l $* $<; \
+	else \
+	  echo debiandoc2text -l $*.UTF-8 $<; \
+	  debiandoc2text -l $*.UTF-8 $<; \
+	fi
 
 # generating PostScript
 ps: $(MANUAL).en.ps $(patsubst %.sgml,%.ps,$(sources_noncjk))
@@ -72,7 +83,13 @@
 	mv $(MANUAL).ps $(MANUAL).en.ps
 
 $(MANUAL).%.ps: $(MANUAL).%.sgml
-	debiandoc2latexps -l $* $<
+	@if [ $* != fr ]; then \
+	  echo debiandoc2latexps -l $* $<; \
+	  debiandoc2latexps -l $* $<; \
+	else \
+	  echo debiandoc2latexps -l $*.UTF-8 $<; \
+	  debiandoc2latexps -l $*.UTF-8 $<; \
+	fi
 
 # generating Portable Document Format
 pdf: $(MANUAL).en.pdf $(patsubst %.sgml,%.pdf,$(sources_noncjk))
@@ -82,7 +99,13 @@
 	mv $(MANUAL).pdf $(MANUAL).en.pdf
 
 $(MANUAL).%.pdf: $(MANUAL).%.sgml
-	debiandoc2latexpdf -l $* $<
+	@if [ $* != fr ]; then \
+	  echo debiandoc2latexpdf -l $* $<; \
+	  debiandoc2latexpdf -l $* $<; \
+	else \
+	  echo debiandoc2latexpdf -l $*.UTF-8 $<; \
+	  debiandoc2latexpdf -l $*.UTF-8 $<; \
+	fi
 
 # publishing to the DDP web pages
 publish: all
