Index: debian/changelog
===================================================================
--- debian/changelog	(revision 22423)
+++ debian/changelog	(working copy)
@@ -71,6 +71,10 @@
   * PCI: quirks: Fix backport of quirk_io()
   * [x86] mm/fault: Fix semaphore imbalance introduced in 3.2.67
 
+  [ Cyril Brulebois ]
+  * udeb: scsi-core-modules: Add scsi_transport_sas and
+    scsi_transport_spi as optional (Closes: #774620).
+
  -- Ben Hutchings <ben@decadent.org.uk>  Mon, 23 Feb 2015 03:42:59 +0000
 
 linux (3.2.65-1+deb7u2) wheezy-security; urgency=medium
Index: debian/installer/modules/scsi-core-modules
===================================================================
--- debian/installer/modules/scsi-core-modules	(revision 22423)
+++ debian/installer/modules/scsi-core-modules	(working copy)
@@ -1,3 +1,5 @@
 scsi_dh
 scsi_mod
 sd_mod
+scsi_transport_sas ?
+scsi_transport_spi ?
