--- debian/changelog	(revision 22243)
+++ debian/changelog	(working copy)
@@ -1,3 +1,10 @@
+linux (3.2.65-2) UNRELEASED; urgency=low
+
+  * udeb: scsi-core-modules: Add scsi_transport_sas and
+    scsi_transport_spi as optional (Closes: #774620).
+
+ -- Cyril Brulebois <kibi@debian.org>  Tue, 06 Jan 2015 18:14:11 +0000
+
 linux (3.2.65-1) wheezy; urgency=medium
 
   * New upstream stable update:
--- debian/installer/modules/scsi-core-modules	(revision 22243)
+++ debian/installer/modules/scsi-core-modules	(working copy)
@@ -1,3 +1,5 @@
 scsi_dh
 scsi_mod
 sd_mod
+scsi_transport_sas ?
+scsi_transport_spi ?
