From 29fc4d45f8b93975653e4f4abb7010378e578645 Mon Sep 17 00:00:00 2001
From: Mattia Rizzolo <mattia@debian.org>
Date: Sat, 24 Feb 2018 00:18:03 +0100
Subject: [PATCH] Change the default ftp upload queue for mentors.debian.net

Signed-off-by: Mattia Rizzolo <mattia@debian.org>
---
 dupload.conf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dupload.conf b/dupload.conf
index 6fc7fd1..8b68565 100644
--- a/dupload.conf
+++ b/dupload.conf
@@ -125,7 +125,7 @@ $cfg{'delayed'} = {
 $cfg{'mentors'} = {
     fqdn => 'mentors.debian.net',
     passive => 1,
-    incoming => '/',
+    incoming => '/pub/UploadQueue/',
     dinstall_runs => 1,
 };
 
-- 
2.16.1

