diff -Nru cdbs-0.4.61+nmu1/1/class/python-distutils.mk.in cdbs-0.4.61+nmu2/1/class/python-distutils.mk.in
--- cdbs-0.4.61+nmu1/1/class/python-distutils.mk.in	2009-09-19 12:47:22.000000000 +0200
+++ cdbs-0.4.61+nmu2/1/class/python-distutils.mk.in	2009-10-07 21:03:43.000000000 +0200
@@ -141,7 +141,7 @@
 # install stage
 $(patsubst %,install/%,$(cdbs_python_indep_packages)) :: install/%: python-install-py
 	cd $(DEB_SRCDIR) && $(call cdbs_python_binary,python$(cdbs_python_compile_version)) $(DEB_PYTHON_SETUP_CMD) install --root=$(cdbs_python_destdir) \
-		--install-lib=/usr/lib/python$(firstword $(strip $(sort $(shell pyversions -vr))))/site-packages/ $(DEB_PYTHON_INSTALL_ARGS_ALL)
+		--install-lib=/usr/lib/python$(if $(call cdbs_streq,$(cdbs_python_compile_version),),$(cdbs_python_current_version),$(cdbs_python_compile_version))/site-packages/ $(DEB_PYTHON_INSTALL_ARGS_ALL)
 
 $(patsubst %,install/%,$(cdbs_python_arch_packages)) :: install/%: $(addprefix python-install-, $(cdbs_python_build_versions))
 	for buildver in $(cdbs_python_build_versions); do \
diff -Nru cdbs-0.4.61+nmu1/debian/changelog cdbs-0.4.61+nmu2/debian/changelog
--- cdbs-0.4.61+nmu1/debian/changelog	2009-09-21 00:04:42.000000000 +0200
+++ cdbs-0.4.61+nmu2/debian/changelog	2009-10-07 19:35:59.000000000 +0200
@@ -1,3 +1,12 @@
+cdbs (0.4.61+nmu2) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Set the right install-lib path when default Python is used to install
+    files in architecture independent packages (closes: #548457) (bug
+    introduced in 0.4.61+nmu1)
+
+ -- Piotr Ożarowski <piotr@debian.org>  Wed, 07 Oct 2009 19:28:45 +0200
+
 cdbs (0.4.61+nmu1) unstable; urgency=low
 
   * Non-maintainer upload.
