--- /etc/profile	2014-10-22 10:02:30.000000000 -0400
+++ profile	2015-01-24 11:51:39.272067172 -0500
@@ -8,8 +8,8 @@
 fi
 export PATH
 
-if [ "$PS1" ]; then
-  if [ "$BASH" ] && [ "$BASH" != "/bin/sh" ]; then
+if [ "${PS1-}" ]; then
+  if [ "${BASH-}" ] && [ "$BASH" != "/bin/sh" ]; then
     # The file bash.bashrc already sets the default PS1.
     # PS1='\h:\w\$ '
     if [ -f /etc/bash.bashrc ]; then
