diff -ur OLD/debian/control NEW/debian/control
--- OLD/debian/control	2013-05-29 10:47:53.000000000 +0200
+++ NEW/debian/control	2013-07-07 15:42:34.435268017 +0200
@@ -7,7 +7,8 @@
  Vincent Cheng <Vincentc1208@gmail.com>,
  Yunqiang Su <wzssyqa@gmail.com>
 Build-Depends: debhelper (>= 9), libbsd-dev (>= 0.2.0), pkg-config,
- libx11-dev, libglib2.0-dev, help2man
+ libx11-dev, libglib2.0-dev, help2man,
+ dh-systemd (>= 1.4)
 Standards-Version: 3.9.4
 Homepage: https://launchpad.net/~bumblebee
 Vcs-Git: git://anonscm.debian.org/pkg-nvidia/bumblebee.git
diff -ur OLD/debian/rules NEW/debian/rules
--- OLD/debian/rules	2013-05-29 10:39:36.000000000 +0200
+++ NEW/debian/rules	2013-07-07 15:42:34.435268017 +0200
@@ -5,7 +5,7 @@
 #export DH_VERBOSE=1
 
 %:
-	dh $@ 
+	dh $@  --with=systemd
 
 override_dh_auto_configure:
 # Ubuntu and Debian's packaging of nvidia's proprietary driver differ greatly
