--- decode_tds.c.orig	2013-06-18 10:35:34.000000000 +0200
+++ decode_tds.c	2013-06-18 10:37:41.000000000 +0200
@@ -140,15 +140,15 @@
 	
 	obuf[0] = '\0';
 
-    if (th->size != 8) {
-           /* wrong header length */
-           return (strlen(obuf));
-    }
-
 	for (th = (struct tds_hdr *)buf;
 	     len > sizeof(*th) && len >= ntohs(th->size);
 	     buf += ntohs(th->size), len -= ntohs(th->size)) {
 		
+		if (th->size != 8) {
+			/* wrong header length */
+			break;
+		}
+
 		if (th->type == 2) {
 			/* Version 4.x, 5.0 */
 			if (len < sizeof(*th) + sizeof(*tl))
