--- src/main/print-escp2-data.c.orig	2006-01-11 04:54:04.000000000 +0200
+++ src/main/print-escp2-data.c	2006-02-16 01:02:33.000000000 +0200
@@ -879,7 +879,7 @@
     360, 14400, -1, 1440, 720, 90, 90, 0, 1, 0, 0, 0, 0, 8,
     INCH(17 / 2), INCH(44), INCH(2), INCH(2),
     9, 9, 0, 30, 9, 9, 0, 30, 9, 9, 0, 0, 9, 9, 0, 0, -1, -1, 0, 0,
-    1, 7, 0, 0,
+    1, 15, 0, 0,
     sp700_dotsizes, sp700_densities, &stpi_escp2_simple_drops,
     stpi_escp2_g3_reslist, &stpi_escp2_photo_gen1_inkgroup,
     standard_bits, g3_base_res, &default_input_slot_list,
