diff --git a/deluge/core/rpcserver.py b/deluge/core/rpcserver.py
index 649c2ce..65601df 100644
--- a/deluge/core/rpcserver.py
+++ b/deluge/core/rpcserver.py
@@ -131,7 +131,8 @@ def getContext(self):
         SSL transport.
         """
         ssl_dir = deluge.configmanager.get_config_dir("ssl")
-        ctx = SSL.Context(SSL.SSLv3_METHOD)
+        ctx = SSL.Context(SSL.SSLv23_METHOD)
+        ctx.set_options(SSL.OP_NO_SSLv2 & SSL.OP_NO_SSLv3)
         ctx.use_certificate_file(os.path.join(ssl_dir, "daemon.cert"))
         ctx.use_privatekey_file(os.path.join(ssl_dir, "daemon.pkey"))
         return ctx
