From aec5f1bb67728385021b00cb9f9cc8f265611d2a Mon Sep 17 00:00:00 2001
From: Dominic Hargreaves <dom@earth.li>
Date: Thu, 17 Dec 2015 10:36:17 +0000
Subject: [PATCH] Depend on perl5 instead of perl5, since the latter has been
 removed

---
 debian/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index a5a1d3d..f1c88ba 100644
--- a/debian/control
+++ b/debian/control
@@ -10,7 +10,7 @@ Vcs-Browser: http://anonscm.debian.org/gitweb/?p=collab-maint/ddclient.git
 
 Package: ddclient
 Architecture: all
-Depends: perl5, ${misc:Depends}, lsb-base (>= 3.1), initscripts (>= 2.86)
+Depends: perl, ${misc:Depends}, lsb-base (>= 3.1), initscripts (>= 2.86)
 Recommends: libio-socket-ssl-perl
 Provides: dyndns-client
 Description: address updating utility for dynamic DNS services
-- 
2.1.4

