From d5fe11dda3dea29fd4d8a1dfd1e74cba8d12c324 Mon Sep 17 00:00:00 2001
From: Steve Langasek <vorlon@debian.org>
Date: Thu, 18 Jun 2009 15:46:14 -0700
Subject: [PATCH] Breaks are supported in stable, so are now allowed in the archive

---
 policy.sgml |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/policy.sgml b/policy.sgml
index 144cbfb..c82ced0 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -4532,8 +4532,7 @@ Build-Depends: foo [!i386] | bar [!amd64]
 	  <prgn>dpkg</prgn> from upgrading or installing the package
 	  which declared such a conflict until the upgrade or removal
 	  of the conflicted-with package had been completed.  Instead,
-	  <tt>Breaks</tt> may be used (once <tt>Breaks</tt> is supported
-	  by the stable release of Debian).
+	  <tt>Breaks</tt> may be used.
 	</p>
       </sect>
 
-- 
1.6.3.1

