--- cups-1.3.7/cgi-bin/ipp-var.c.str3436	2010-10-29 18:52:20.000000000 +0100
+++ cups-1.3.7/cgi-bin/ipp-var.c	2010-10-29 19:46:50.000000000 +0100
@@ -1353,7 +1353,12 @@
     else
     {
       if (dest)
+      {
+        snprintf(url, sizeof(url), "/%s/%s", section, dest);
+	cgiSetVariable("PRINTER_NAME", dest);
+	cgiSetVariable("PRINTER_URI_SUPPORTED", url);
         snprintf(url, sizeof(url), "/%s/%s?", section, dest);
+      }
       else
         strlcpy(url, "/jobs/?", sizeof(url));
 
