diff --git a/devel/a/chrony/debian/postinst b/devel/b/chrony/debian/postinst
index bb13102..9b332a1 100644
--- a/devel/a/chrony/debian/postinst
+++ b/devel/b/chrony/debian/postinst
@@ -17,15 +17,6 @@ case "$1" in
         # look for real-time clock
         if [ -f /etc/adjtime ]; then
             TZONE=$(grep -E "UTC|LOCAL" /etc/adjtime)
-
-        elif [ -f /etc/default/rcS ]; then
-            .  /etc/default/rcS
-            case "$UTC" in
-                no|"") TZONE="LOCAL"
-                ;;
-                yes)  TZONE="UTC"
-                ;;
-            esac
         fi
 
         # set rtconutc for chrony
