From 852884736019ea8c786e95864a882b3eb6985d7d Mon Sep 17 00:00:00 2001
From: Petter Reinholdtsen <pere@hungry.com>
Date: Sat, 18 Dec 2021 09:13:41 +0100
Subject: [PATCH 2/3] Added a d/watch file to track new upstream versions.

---
 debian/watch | 5 +++++
 1 file changed, 5 insertions(+)
 create mode 100644 debian/watch

diff --git a/debian/watch b/debian/watch
new file mode 100644
index 0000000..a36f436
--- /dev/null
+++ b/debian/watch
@@ -0,0 +1,5 @@
+version=4
+# qa.debian.org runs a redirector which allows a simpler form of URL
+# for SourceForge based projects. The format below will automatically
+# be rewritten to use the redirector.
+http://sf.net/dxf2gcode/dxf2gcode-(\d\S+)\.(?:zip|tgz|tbz|txz|(?:tar\.(?:gz|bz2|xz)))
-- 
2.30.2

