diff -Nurp birthday-1.6.2/bdcal.c birthday-1.6.2jbglaw/bdcal.c
--- birthday-1.6.2/bdcal.c	2005-12-09 19:06:59.000000000 +0100
+++ birthday-1.6.2jbglaw/bdcal.c	2011-01-25 16:34:16.000000000 +0100
@@ -57,7 +57,7 @@ void do_cal(struct event *evl, const str
     btime = *localtime( &now );
   }
 
-  for (i=j=0; i < iCTotal; i++)
+  for (i=0; i < iCTotal; i++)
     {
       if (iCWeeks != 0 && i!=0 && (i%(iCWeeks*7))==0) printf("\x0c");
       strftime(buf, iCWidth+1, "---%A-%B-%d-%Y", &btime);
@@ -78,13 +78,18 @@ void do_cal(struct event *evl, const str
 	}
       else
 	printf("%s\n", buf);
-      
+
       lines=1;
       chars=0;
-      /* assuming the events are ordered by date, this will process all the events for the current day, if any, and
-	 leave j pointing to the first event which is not for today */
-      for (; evl[j].text != NULL && delta(&(evl[j].date), today) <= i; j++)
-	if (delta(&(evl[j].date), today) == i)
+
+      for (j = 0; evl[j].text != NULL; j++)
+	if (delta(&(evl[j].date), today) == i /* Today's events.  */
+            || (evl[j].enddate.year != 0      /* Events that end somewhen... */
+                && evl[j].enddate.month != 0
+                && evl[j].enddate.day != 0
+                && delta(&(evl[j].enddate), today) >= i
+                && (delta(&(evl[j].date), today) >= delta(&(evl[j].enddate), today) /* ...and either started in the past. */
+                    || delta(&(evl[j].date), today) <= i)))                         /* ...or in the timespan of this calendar output. */
 	  {
 	    chars+=strlen(evl[j].text)+4;
 	    if (chars > iCWidth)
diff -Nurp birthday-1.6.2/test/ongoing-events.t birthday-1.6.2jbglaw/test/ongoing-events.t
--- birthday-1.6.2/test/ongoing-events.t	1970-01-01 01:00:00.000000000 +0100
+++ birthday-1.6.2jbglaw/test/ongoing-events.t	2011-01-25 16:48:04.000000000 +0100
@@ -0,0 +1,47 @@
+# This tests the output of events that last longer than a day.
+# Old `birthday' versions only printed those on their first
+# occurence in calendar (-c) mode, and only if they didn't start
+# in the past.
+*args
+-T 25/1/2011 -c  -d 6
+*file
+First event, starting in the past, already finished=	21/1/2011 ev to 23/1/2011
+Second event, starting in the past, ongoing=		21/1/2011 ev to 26/1/2011
+Third event, starting and ending in the future=		28/1/2011 ev to 30/1/2011
+*output
+---Tuesday-January-25-2011------------------------------------------------------
+Second event, starting in the past, ongoing    
+
+
+
+
+---Wednesday-January-26-2011----------------------------------------------------
+Second event, starting in the past, ongoing    
+
+
+
+
+---Thursday-January-27-2011-----------------------------------------------------
+
+
+
+
+
+---Friday-January-28-2011-------------------------------------------------------
+Third event, starting and ending in the future    
+
+
+
+
+------SSaattuurrddaayy--JJaannuuaarryy--2299--22001111----------------------------------------------------------------------------------------------------------
+Third event, starting and ending in the future    
+
+
+
+
+------SSuunnddaayy--JJaannuuaarryy--3300--22001111--------------------------------------------------------------------------------------------------------------
+Third event, starting and ending in the future    
+
+
+
+
