--- src/parser_rfc822.c~	2013-07-13 10:10:25.000000000 +0000
+++ src/parser_rfc822.c	2018-08-05 00:15:58.034730797 +0000
@@ -35,7 +35,7 @@
 #include <sys/types.h>
 #include <unistd.h>
 
-#define READSIZE 16384
+#define READSIZE 65536
 
 int di_parser_rfc822_read (char *begin, size_t size, di_parser_info *info, di_parser_read_entry_new entry_new, di_parser_read_entry_finish entry_finish, void *user_data)
 {
