diff -rauN ../orig/developers-reference-3.3.8/common.ent ./developers-reference-3.3.8/common.ent
--- ../orig/developers-reference-3.3.8/common.ent	2006-11-11 19:13:05.000000000 +0100
+++ ./developers-reference-3.3.8/common.ent	2007-08-05 01:15:22.000000000 +0200
@@ -111,7 +111,7 @@
 <!ENTITY url-dmup "http://&www-debian-org;/devel/dmup">
 <!ENTITY url-worldmap "http://&www-debian-org;/devel/developers.loc">
 
-<!ENTITY url-i18n-doc-check "http://cvs.debian.org/boot-floppies/documentation/doc-check?rev=HEAD&amp;content-type=text/vnd.viewcvs-markup">
+<!ENTITY url-i18n-doc-check "http://svn.debian.org/wsvn/d-i/trunk/manual/scripts/doc-check?op=file">
 
 <!ENTITY url-eg-desc-upstream-info "http://&packages-host;/unstable/web/wml">
 
diff -rauN ../orig/developers-reference-3.3.8/developers-reference.sgml ./developers-reference-3.3.8/developers-reference.sgml
--- ../orig/developers-reference-3.3.8/developers-reference.sgml	2006-11-12 12:05:22.000000000 +0100
+++ ./developers-reference-3.3.8/developers-reference.sgml	2007-08-05 01:59:26.000000000 +0200
@@ -4577,6 +4577,12 @@
 
       <sect id="bpp-i18n">
         <heading>Internationalization</heading>
+        <p>
+This section contains global information for developers to make translators'
+life easier.  More information for translators and developers interrested
+in internationalization are available in the
+<url id="http://people.debian.org/~jfs/debconf6/html/"
+name="Internationalisation and localisation in Debian"> documentation.
 
 	<sect1 id="bpp-i18n-debconf">
           <heading>Handling debconf translations</heading>
@@ -4614,25 +4620,30 @@
 labor.  There's no way to eliminate all that work, but you can make things
 easier for translators.
           <p>
-If you maintain documentation of any size, its easier for translators
+If you maintain documentation of any size, it is easier for translators
 if they have access to a source control system.  That lets translators
 see the differences between two versions of the documentation, so, for
 instance, they can see what needs to be retranslated.  It is
 recommended that the translated documentation maintain a note about
 what source control revision the translation is based on.  An
 interesting system is provided by <url id="&url-i18n-doc-check;"
-name="doc-check"> in the <package>boot-floppies</package> package,
+name="doc-check"> in the <package>d-i</package> package,
 which shows an overview of the translation status for any given
 language, using structured comments for the current revision of the
 file to be translated and, for a translated file, the revision of the
 original file the translation is based on.  You might wish to adapt
-and provide that in your CVS area.
+and provide that in your Subversion area.
           <p>
 If you maintain XML or SGML documentation, we suggest that you isolate
 any language-independent information and define those as entities in a
 separate file which is included by all the different
 translations. This makes it much easier, for instance, to keep URLs
 up to date across multiple files.
+          <p>
+Some tools (e.g. po4a, poxml, or the translate-toolkit) are specialized in
+extracting the translatable material from different formats.  They produce
+PO files, a format quite common to translators, which permits to see what
+needs to be retranslated when the translated document is updated.
         </sect1>
       </sect>
 
