From a4d4e22c8f14e8ef362bf95c0aae6d847edcb181 Mon Sep 17 00:00:00 2001
From: Rafael Laboissiere <rafael@debian.org>
Date: Mon, 22 Aug 2016 09:13:25 +0200
Subject: [PATCH] Avoid deprecated option --same-arch of debhelper

---
 1/rules/buildvars.mk.in | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/1/rules/buildvars.mk.in b/1/rules/buildvars.mk.in
index d1995e0..61fd441 100644
--- a/1/rules/buildvars.mk.in
+++ b/1/rules/buildvars.mk.in
@@ -48,7 +48,7 @@ ifdef _cdbs_rules_debhelper
 	dh_listpackages -i))
   DEB_ARCH_PACKAGES := $(filter-out $(DEB_INDEP_PACKAGES),\
 	$(strip $(shell \
-		dh_listpackages -s)))
+		dh_listpackages -a)))
 else
   DEB_INDEP_PACKAGES := $(strip $(shell \
 	$(_cdbs_scripts_path)/list-packages indep))
-- 
2.8.1

