From 60ee2166d2a3b0ff501553069439cd7ae8bdc57c Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Sun, 2 Jun 2013 10:16:36 +0800
Subject: [PATCH 1/8] Bug #710000 was reported.

---
 data/events/bugs | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/data/events/bugs b/data/events/bugs
index 666231c..202c4d0 100644
--- a/data/events/bugs
+++ b/data/events/bugs
@@ -109,3 +109,7 @@ Source: http://bugs.debian.org/690000
 Title: Debian Bug #700000 reported by Cédric Boutillier
 Date: Feb 7 2013
 Source: http://bugs.debian.org/700000
+
+Title: Debian Bug #710000 reported by Rene Engelhard
+Date: May 9 2013
+Source: http://bugs.debian.org/710000
-- 
1.8.3

