From ccf4dee1dc357592f35f7da62c0e19e95ec9562b Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Sun, 2 Jun 2013 10:16:56 +0800
Subject: [PATCH 2/8] Add DebConf13 and some miniconfs.

---
 data/events/conferences | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/data/events/conferences b/data/events/conferences
index 5678cc8..6ffc270 100644
--- a/data/events/conferences
+++ b/data/events/conferences
@@ -206,3 +206,18 @@ Title: Mini-DebConf, Paris, France
 Start-Date: Nov 24 2012
 End-Date: Nov 25 2012
 Source: http://fr2012.mini.debconf.org/
+
+Title: Mini-DebConf, Calicut, India
+Start-Date: Feb 22 2013
+End-Date: Feb 24 2013
+Source: http://wiki.debian.org/DebianIndia/MiniDebConf2013/Calicut
+
+Title: Debian/Ubuntu Community Conference Italia 2013
+Start-Date: Jun 1 2013
+End-Date: Jun 2 2013
+Source: http://www.debian.org/events/2013/0601-ducc-it
+
+Title: DebConf13, Le Camp, Switzerland
+Start-Date: Aug 11 2013
+End-Date: Aug 17 2013
+Source: http://debconf13.debconf.org/
-- 
1.8.3

