From 4bef02b8b182029e085a863576220f47a2ae017d Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Sun, 2 Jun 2013 10:17:16 +0800
Subject: [PATCH 3/8] Third RC of wheezy's debian-installer released.

---
 data/events/d-i | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/data/events/d-i b/data/events/d-i
index b46e364..c792f3e 100644
--- a/data/events/d-i
+++ b/data/events/d-i
@@ -109,3 +109,7 @@ Source: http://www.debian.org/devel/debian-installer/News/2013/20130217
 Title: Second RC of wheezy's debian-installer released
 Date: Apr 27 2013
 Source: http://www.debian.org/devel/debian-installer/News/2013/20130427
+
+Title: Third RC of wheezy's debian-installer released
+Date: May 2 2013
+Source: http://www.debian.org/devel/debian-installer/News/2013/20130502
-- 
1.8.3

