From d9629d8b32d9e378b51e6c17d32a6ae3c8a12477 Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Sun, 2 Jun 2013 10:17:35 +0800
Subject: [PATCH 4/8] Debian Live wheezy images released.

---
 data/events/debian-live | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/data/events/debian-live b/data/events/debian-live
index 55842fd..38a4a92 100644
--- a/data/events/debian-live
+++ b/data/events/debian-live
@@ -21,3 +21,7 @@ Source: http://cdimage.debian.org/cdimage/archive/5.0.0-live/
 Title: Debian Live Squeeze alpha2 images released
 Date: Jul 21 2010
 Source: http://cdimage.debian.org/cdimage/squeeze_live_alpha2/
+
+Title: Debian Live wheezy images released
+Date: May 6 2013
+Source: http://lists.debian.org/debian-live/2013/05/msg00014.html
-- 
1.8.3

