From e9109463ef8597fd564031cff5a0409e4177a5f9 Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Sun, 2 Jun 2013 10:18:09 +0800
Subject: [PATCH 6/8] DMD was announced last year.

---
 data/events/services | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/data/events/services b/data/events/services
index d6a0d8f..0a04c7b 100644
--- a/data/events/services
+++ b/data/events/services
@@ -94,6 +94,10 @@ Title: Derivatives patches service available
 Date: Mar 1 2012
 Source: http://lists.debian.org/debian-devel-announce/2012/03/msg00000.html
 
+Title: Debian Maintainer Dashboard announced
+Date: Aug 18 2012
+Source: http://lists.debian.org/debian-devel-announce/2012/08/msg00003.html
+
 Title: Debian Url ChecKer service available
 Date: Dec 31 2012
 Source: http://lists.debian.org/debian-devel-announce/2012/12/msg00002.html
-- 
1.8.3

