From f9a090618ddcbf81c33df826bf556d0b55669e94 Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Sun, 2 Jun 2013 10:18:29 +0800
Subject: [PATCH 7/8] Ray Dassen passed away.

---
 data/events/tragic | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/data/events/tragic b/data/events/tragic
index 58db8c7..3825466 100644
--- a/data/events/tragic
+++ b/data/events/tragic
@@ -31,3 +31,7 @@ Source: http://www.debian.org/News/2008/20081229
 Title: Frans Pop passed away
 Date: Aug 31 2010
 Source: http://www.debian.org/News/2010/20100831
+
+Title: Ray Dassen passed away
+Date: May 23 2013
+Source: http://www.debian.org/News/2013/20130523
-- 
1.8.3

