From 4bc77722740ddc7c1c5bcffbcd1ac576da1062cb Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Sun, 2 Jun 2013 10:18:45 +0800
Subject: [PATCH 8/8] Lucas Nussbaum elected as DPL.

---
 data/events/voting | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/data/events/voting b/data/events/voting
index 4c90504..0e73cf2 100644
--- a/data/events/voting
+++ b/data/events/voting
@@ -204,3 +204,7 @@ Title: DPL 2013 elections
 Start-Date: Mar 03 2013
 End-Date: Apr 13 2013
 Source: http://www.debian.org/vote/2013/vote_001
+
+Title: Lucas Nussbaum elected as DPL
+Date: April 14 2013
+Source: http://www.debian.org/vote/2013/vote_001
-- 
1.8.3

