--- src/auth/userdb-passwd.c.old	2005-04-09 21:09:45.093469928 +0300
+++ src/auth/userdb-passwd.c	2005-04-09 21:11:18.282291066 +0300
@@ -24,6 +24,13 @@
 		return;
 	}
 
+	if (strcasecmp(pw->pw_name, user) != 0) {
+		/* try to catch broken NSS implementations (nss_ldap) */
+		i_fatal("BROKEN NSS IMPLEMENTATION: "
+			"getpwnam() lookup returned different user than was "
+			"requested (%s != %s).", pw->pw_name, user);
+	}
+
 	memset(&data, 0, sizeof(data));
 	data.uid = pw->pw_uid;
 	data.gid = pw->pw_gid;
