From f5269c40f5e14481aa3098fac96260c11fa975e4 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Piotr=20O=C5=BCarowski?= <piotr@debian.org>
Date: Tue, 19 May 2015 21:17:50 +0200
Subject: [PATCH] uscan: use pypi.debian.net redirector instead of
 pypi.python.org

---
 scripts/uscan.pl | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/scripts/uscan.pl b/scripts/uscan.pl
index 841f8b5..441e73e 100755
--- a/scripts/uscan.pl
+++ b/scripts/uscan.pl
@@ -882,6 +882,9 @@ sub process_watchline ($$$$$$)
 	    $base =~ s%^http://sf\.net/%https://qa.debian.org/watch/sf.php/%;
 	    $filepattern .= '(?:\?.*)?';
 	}
+	# Handle pypi.python.org addresses specially
+	$base =~ s%^https?://pypi\.python\.org/packages/source/./%http://pypi.debian.net/%;
+
 	if ($base =~ m%^(\w+://[^/]+)%) {
 	    $site = $1;
 	} else {
-- 
2.1.4

