diff -u udev-0.125/debian/udev-udeb.dirs udev-0.125/debian/udev-udeb.dirs
--- udev-0.125/debian/udev-udeb.dirs
+++ udev-0.125/debian/udev-udeb.dirs
@@ -2,5 +2,5 @@
 /etc/udev/
 /etc/udev/rules.d/
-/lib/debian-installer-startup.d/
+/lib/debian-installer/
 /lib/udev/
 /sbin/
diff -u udev-0.125/debian/changelog udev-0.125/debian/changelog
--- udev-0.125/debian/changelog
+++ udev-0.125/debian/changelog
@@ -1,3 +1,12 @@
+udev (0.125-6~~lunar1) UNRELEASED; urgency=low
+
+  [ Jérémy Bobbio ]
+  * Move /lib/debian-installer-startup.d/S02udev to
+    /lib/debian-installer/start-udev in udev-udeb as udev will now be started
+    before busybox init in the debian-installer.
+
+ -- Jérémy Bobbio <lunar@debian.org>  Fri, 15 Aug 2008 20:44:35 +0200
+
 udev (0.125-5) unstable; urgency=medium
 
   * Modified debian/rules to work with the etch version of findutils.
diff -u udev-0.125/debian/rules udev-0.125/debian/rules
--- udev-0.125/debian/rules
+++ udev-0.125/debian/rules
@@ -126,7 +126,7 @@
 		$(DU)/lib/udev/hotplug.functions
 	install --mode=644 hotplug/modprobe.d/* $(DU)/etc/modprobe.d/
 	install --mode=755 extra/udev.startup \
-		$(DU)/lib/debian-installer-startup.d/S02udev
+		$(DU)/lib/debian-installer/start-udev
 	mkdir -p $(DU)/usr/lib/base-installer.d/
 	install --mode=755 extra/udev.base-installer \
 		$(DU)/usr/lib/base-installer.d/05udev
