diff -urN apt-0.7.6.orig/cmdline/apt-get.cc apt-0.7.6/cmdline/apt-get.cc
--- apt-0.7.6.orig/cmdline/apt-get.cc	2007-10-17 10:26:24.000000000 +0530
+++ apt-0.7.6/cmdline/apt-get.cc	2007-10-17 10:28:58.000000000 +0530
@@ -2568,10 +2568,10 @@
    cout << 
       "         (__) \n"
       "         (oo) \n"
-      "   /------\\/ \n"
-      "  / |    ||   \n" 
-      " *  /\\---/\\ \n"
-      "    ~~   ~~   \n"
+      "   /-----(__) \n"
+      "  / |    || \n"
+      " *  /\---/\ \n"
+      "    ~~   ~~ \n"
       "....\"Have you mooed today?\"...\n";
 			    
    return true;
