--- /usr/sbin/checkrestart	2012-06-16 04:38:23.000000000 +0200
+++ ./checkrestart	2014-05-04 19:45:54.000000000 +0200
@@ -286,11 +286,16 @@
             process.links.append(data)
         elif field == 'n':
             # Remove the previous entry to check if this is something we should use
-            if data.startswith('/SYSV'):
+            if data.find('SYSV') >= 0:
                 # If we find SYSV we discard the previous descriptor
                 last = process.descriptors.pop()
-            elif data.startswith('/'):
+            elif data.startswith('/') or data.startswith('(deleted)/') or data.startswith(' (deleted)/'):
                 last = process.descriptors.pop()
+                if data.startswith('(deleted)'):
+                    data = data[9:] + ' (deleted)'
+                elif data.startswith(' (deleted)'):
+                    data = data[10:] + ' (deleted)'
+
                 # Add it to the list of deleted files if the previous descriptor
                 # was DEL or lsof marks it as deleted
                 if re.compile("DEL").search(last) or re.compile("deleted").search(data) or re.compile("\(path inode=[0-9]+\)$").search(data):
