diff --git a/dbconfig-generate-include b/dbconfig-generate-include
index de15752..db62708 100755
--- a/dbconfig-generate-include
+++ b/dbconfig-generate-include
@@ -385,16 +385,14 @@ EOF
 	# the best available method to connect to the local db): expand it to
 	# "localhost" if needed
 	if [ -z "$dbc_dbserver" ] ; then
-		dbc_dballow="localhost"
-	else
-		dbc_dballow="$dbc_dbserver"
+		dbc_dbserver="localhost"
 	fi
 	cat << EOF > "$sedtmp"
 s/^\(.*\)_DBC_DBUSER_/${comment_dbuser}\1`sed_rhs_escape $dbc_dbuser`/g
 s/^\(.*\)_DBC_DBPASS_/${comment_dbpass}\1`sed_rhs_escape $dbc_dbpass`/g
 s/^\(.*\)_DBC_BASEPATH_/${comment_basepath}\1`sed_rhs_escape $dbc_basepath`/g
 s/^\(.*\)_DBC_DBNAME_/${comment_dbname}\1`sed_rhs_escape $dbc_dbname`/g
-s/^\(.*\)_DBC_DBSERVER_/${comment_dbserver}\1`sed_rhs_escape $dbc_dballow`/g
+s/^\(.*\)_DBC_DBSERVER_/${comment_dbserver}\1`sed_rhs_escape $dbc_dbserver`/g
 s/^\(.*\)_DBC_DBPORT_/${comment_dbport}\1`sed_rhs_escape $dbc_dbport`/g
 s/^\(.*\)_DBC_DBTYPE_/${comment_dbtype}\1`sed_rhs_escape $dbc_dbtype`/g
 EOF
