diff -urN apt-cacher-0.7.orig/apt-cacher.conf apt-cacher-0.7/apt-cacher.conf
--- apt-cacher-0.7.orig/apt-cacher.conf	2003-05-26 07:22:08.000000000 +0200
+++ apt-cacher-0.7/apt-cacher.conf	2004-06-20 14:39:48.000000000 +0200
@@ -58,3 +58,7 @@
 # Leave this off unless you need it, or your error log will get very
 # big. Acceptable values are 0 or 1.
 debug=0
+
+# Limit sets the max bandwidth to use for wget. See 'man wget'
+# for syntax. Set to 0 or negative to disable.
+limit=0
diff -urN apt-cacher-0.7.orig/apt-cacher.pl apt-cacher-0.7/apt-cacher.pl
--- apt-cacher-0.7.orig/apt-cacher.pl	2003-12-30 12:49:53.000000000 +0100
+++ apt-cacher-0.7/apt-cacher.pl	2004-06-20 14:39:50.000000000 +0200
@@ -639,7 +639,8 @@
 						$ENV{'http_proxy'} = "http://$config{http_proxy}";
 					}
 					# had to remove the -c option from wget because resuming is incompatible with the -s option
-					exec($fn, '-s', '-nv', '-o', "$private_dir/$new_filename.err", '-O', $unique_filename, $url);
+					my $limit = ($config{limit} > 0 ? "--limit-rate=$config{limit}" : "");
+					exec($fn, '-s', '-nv', $limit, '-o', "$private_dir/$new_filename.err", '-O', $unique_filename, $url);
 				};
 			}
 		}
