diff --git a/policy.sgml b/policy.sgml
index 6e04c81..ed10580 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -3125,7 +3125,7 @@ Package: libc6
 	          </footnote>
 	          and the characters <tt>.</tt> <tt>+</tt> <tt>-</tt>
 	          <tt>:</tt> <tt>~</tt> (full stop, plus, hyphen, colon,
-	          tilde) and should start with a digit.  If there is no
+	          tilde) and must start with a digit.  If there is no
 	          <var>debian_revision</var> then hyphens are not allowed;
 	          if there is no <var>epoch</var> then colons are not
 	          allowed.
