--- /tmp/psaccs_atop	2021-01-20 13:33:03.000000000 -0500
+++ /etc/logrotate.d/psaccs_atop	2021-02-09 08:34:57.633250717 -0500
@@ -10,7 +10,7 @@
     postrotate
 	# check if process accounting is installed
 	#
- 	if [ -e /etc/logrotate.d/psacct ] || grep -q 'savelog' /etc/cron.daily/acct
+	if [ -e /etc/logrotate.d/psacct ] || ([ -e /etc/cron.daily/acct ] && grep -q 'savelog' /etc/cron.daily/acct)
 	then
 	    # check if process accounting is actually in use
 	    #
@@ -18,9 +18,12 @@
             then
                 ACCTFILE=`awk '$2 == "{" {print $1}' /etc/logrotate.d/psacct`
             fi
-            if grep -q 'savelog' /etc/cron.daily/acct
+            if [ -e /etc/cron.daily/acct]
             then
-                ACCTFILE=`sed -n "/savelog[^\/]\+\(\/[[:alnum:]\/]\+\).*/{s//\1/;p;q;}" /etc/cron.daily/acct`
+                if grep -q 'savelog' /etc/cron.daily/acct
+                then
+                    ACCTFILE=`sed -n "/savelog[^\/]\+\(\/[[:alnum:]\/]\+\).*/{s//\1/;p;q;}" /etc/cron.daily/acct`
+                fi
             fi
 
 	    if [ -f "$ACCTFILE" ]
