--- /tmp/psaccs_atop	2021-01-20 13:33:03.000000000 -0500
+++ /etc/logrotate.d/psaccs_atop	2021-02-10 11:51:01.165928555 -0500
@@ -10,7 +10,7 @@
     postrotate
 	# check if process accounting is installed
 	#
- 	if [ -e /etc/logrotate.d/psacct ] || grep -q 'savelog' /etc/cron.daily/acct
+	if [ -e /etc/logrotate.d/psacct ] || ([ -e /etc/cron.daily/acct ] && grep -q 'savelog' /etc/cron.daily/acct)
 	then
 	    # check if process accounting is actually in use
 	    #
@@ -18,9 +18,12 @@
             then
                 ACCTFILE=`awk '$2 == "{" {print $1}' /etc/logrotate.d/psacct`
             fi
-            if grep -q 'savelog' /etc/cron.daily/acct
+            if [ -e /etc/cron.daily/acct ]
             then
-                ACCTFILE=`sed -n "/savelog[^\/]\+\(\/[[:alnum:]\/]\+\).*/{s//\1/;p;q;}" /etc/cron.daily/acct`
+                if grep -q 'savelog' /etc/cron.daily/acct
+                then
+                    ACCTFILE=`sed -n "/savelog[^\/]\+\(\/[[:alnum:]\/]\+\).*/{s//\1/;p;q;}" /etc/cron.daily/acct`
+                fi
             fi
 
 	    if [ -f "$ACCTFILE" ]
--- /tmp/psaccu_atop	2021-02-11 20:29:08.973973635 -0500
+++ /etc/logrotate.d/psaccu_atop	2021-02-11 20:29:34.093874797 -0500
@@ -8,7 +8,7 @@
     ifempty
     create 0600 root root
     postrotate
-	if [ -e /etc/logrotate.d/psacct ] || grep -q 'savelog.*/var/log/account/pacct' /etc/cron.daily/acct
+	if [ -e /etc/logrotate.d/psacct ] || ([ -e /etc/cron.daily/acct ] && grep -q 'savelog.*/var/log/account/pacct' /etc/cron.daily/acct)
 	then
 	    # if the atop daemon does not run, restart it after
 	    # accounting file is rotated
