diff --git a/debian/dmsetup-udeb.install b/debian/dmsetup-udeb.install
index 365b8c4d3..c7ae7f637 100644
--- a/debian/dmsetup-udeb.install
+++ b/debian/dmsetup-udeb.install
@@ -1 +1,2 @@
+lib/udev/rules.d/*-dm*.rules
 sbin/dmsetup
diff --git a/debian/rules b/debian/rules
index f67c52ddf..0fdd7650b 100755
--- a/debian/rules
+++ b/debian/rules
@@ -69,6 +69,9 @@ define CONFARGS.udeb
 	--with-optimisation="$(CFLAGS_OPT_UDEB)"
 	--with-lvm1=none
 	--with-pool=none
+	--with-udev-prefix=/
+	--enable-udev_rules
+	--enable-udev_sync
 	--disable-app-machineid
 	--disable-readline
 	--disable-selinux
