--- debian/rules	2013-01-22 10:07:44.960879845 -0300
+++ debian/rules.new	2013-01-22 10:07:25.064516522 -0300
@@ -1,8 +1,12 @@
 #!/usr/bin/make -f
 
+DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
+PREFIX = `pwd`/debian/caps/usr
+DEST = $(PREFIX)/lib/$(DEB_HOST_MULTIARCH)/ladspa
+
 include /usr/share/cdbs/1/rules/debhelper.mk
 include /usr/share/cdbs/1/class/makefile.mk
 
 DEB_INSTALL_DOCS_ALL := caps.html
-DEB_MAKE_INSTALL_TARGET := PREFIX=$(PREFIX) install
+DEB_MAKE_INSTALL_TARGET := PREFIX=$(PREFIX) DEST=$(DEST) install
 CFLAGS += -fPIC -DPIC
