diff -Naur kdeutils-3.3.2.orig/ark/arch.cpp kdeutils-3.3.2/ark/arch.cpp
--- kdeutils-3.3.2.orig/ark/arch.cpp	Sun Jan 30 21:26:45 2005
+++ kdeutils-3.3.2/ark/arch.cpp	Sun Jan 30 20:48:53 2005
@@ -334,7 +334,7 @@
 
 	pos = strpos + len;
 
-	columns[curCol->colRef] = line.mid(strpos, len);
+	columns[curCol->colRef] = QString::fromLocal8Bit( line.mid(strpos, len) );
   }
 
 
diff -Naur kdeutils-3.3.2.orig/ark/arch.h kdeutils-3.3.2/ark/arch.h
--- kdeutils-3.3.2.orig/ark/arch.h	Sun Jan 30 21:26:45 2005
+++ kdeutils-3.3.2/ark/arch.h	Sun Jan 30 21:16:17 2005
@@ -134,7 +134,7 @@
 
   QString getUtility() { return m_archiver_program; }
 
-  void appendShellOutputData( const char * data ) { m_lastShellOutput.append( data ); }
+  void appendShellOutputData( const char * data ) { m_lastShellOutput.append( QString::fromLocal8Bit( data ) ); }
   void clearShellOutput() { m_lastShellOutput.truncate( 0 ); }
   const QString& getLastShellOutput() const { return m_lastShellOutput; }
 
diff -Naur kdeutils-3.3.2.orig/ark/rar.cpp kdeutils-3.3.2/ark/rar.cpp
--- kdeutils-3.3.2.orig/ark/rar.cpp	Sun Jan 30 21:26:45 2005
+++ kdeutils-3.3.2/ark/rar.cpp	Sun Jan 30 21:02:10 2005
@@ -77,7 +77,7 @@
 {
   if ( m_isFirstLine )
   {
-    m_fileName = line;
+    m_fileName = QString::fromLocal8Bit( line );
     m_fileName.remove( 0, 1 );
     m_isFirstLine = false;
     return true;
