diff -u aptitude-0.4.6.1/debian/rules aptitude-0.4.6.1/debian/rules
--- aptitude-0.4.6.1/debian/rules
+++ aptitude-0.4.6.1/debian/rules
@@ -21,7 +21,7 @@
 	rm -f build-stamp
 	rm -fr debian/html-docs
 
-	-$(MAKE) distclean
+	[ ! -f Makefile ] || $(MAKE) distclean
 
 	dh_clean
 
diff -u aptitude-0.4.6.1/debian/menu aptitude-0.4.6.1/debian/menu
--- aptitude-0.4.6.1/debian/menu
+++ aptitude-0.4.6.1/debian/menu
@@ -1,3 +1,3 @@
-?package(aptitude):needs="text" section="Apps/System" \
+?package(aptitude):needs="text" section="Applications/System/Administration" \
   title="Aptitude" command="/usr/bin/aptitude" hints="apt" \
   longtitle="Terminal-based software package manager"
diff -u aptitude-0.4.6.1/debian/changelog aptitude-0.4.6.1/debian/changelog
--- aptitude-0.4.6.1/debian/changelog
+++ aptitude-0.4.6.1/debian/changelog
@@ -1,3 +1,14 @@
+aptitude (0.4.6.1-1.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Removed â*.gmoâ removal from the âdistcleanâ target in po/Makefile.in.in
+    to fix package content change on double-build (Closes: #441696).
+  * Moved the menu entry from the âApps/Systemâ section to the
+    âApplications/System/Administrationâ one.
+  * No longer ignore âmake distcleanâ errors, per lintian.
+
+ -- Cyril Brulebois <cyril.brulebois@enst-bretagne.fr>  Sun, 30 Sep 2007 00:14:43 +0200
+
 aptitude (0.4.6.1-1) unstable; urgency=low
 
   * New upstream release (includes scripts I missed in the last release).
only in patch2:
unchanged:
--- aptitude-0.4.6.1.orig/po/Makefile.in.in
+++ aptitude-0.4.6.1/po/Makefile.in.in
@@ -292,7 +292,7 @@
 clean: mostlyclean
 
 distclean: clean
-	rm -f Makefile Makefile.in POTFILES *.mo *.gmo
+	rm -f Makefile Makefile.in POTFILES *.mo
 
 maintainer-clean: distclean
 	@echo "This command is intended for maintainers to use;"
