Index: Makefile
===================================================================
--- Makefile	(revision 902)
+++ Makefile	(working copy)
@@ -192,14 +192,14 @@
 $(ADIR):
 	$(Q)mkdir -p $(ADIR)
 	$(Q)mkdir -p $(ADIR)/apt-ftparchive-db
-# Make sure unstable/sid points to testing/sarge, as there is no build
-# rule for unstable/sid. TODO: switch it to point to etch.
+# Make sure unstable/sid points to testing/etch, as there is no build
+# rule for unstable/sid.
 unstable-map:
 	$(Q)if [ ! -d $(BASEDIR)/data/sid ] ; then \
-		ln -s sarge $(BASEDIR)/data/sid ; \
+		ln -s etch $(BASEDIR)/data/sid ; \
 	fi
 	$(Q)if [ ! -d $(BASEDIR)/tools/boot/sid ] ; then \
-		ln -s sarge $(BASEDIR)/tools/boot/sid ; \
+		ln -s etch $(BASEDIR)/tools/boot/sid ; \
 	fi
 
 ## CLEANINGS ##
Index: CONF.sh
===================================================================
--- CONF.sh	(revision 902)
+++ CONF.sh	(working copy)
@@ -40,7 +40,6 @@
 unset BASE_INCLUDE      || true
 unset BASE_EXCLUDE      || true
 unset INSTALLER_CD      || true
-unset DI_CODENAME       || true
 unset MAXCDS            || true
 unset SPLASHPNG         || true
 unset OMIT_MANUAL	    || true
@@ -50,8 +49,8 @@
 # Where I am (hoping I'm in the debian-cd dir)
 export BASEDIR=`pwd`
 
-# Building sarge cd set ...
-export CODENAME=sarge
+# Building etch cd set ...
+export CODENAME=etch
 
 # By default use Debian installer packages from $CODENAME
 if [ ! "$DI_CODENAME" ]
