--- cstream.c.orig      2008-12-19 21:38:37.000000000 +0000
+++ cstream.c   2011-03-30 12:27:43.000000000 +0000
@@ -569,11 +569,16 @@
   }

   if (state->using_o_direct) {
+#ifdef O_DIRECT
     flags |= O_DIRECT;
     if (o->v > 1) {
       fprintf(stderr, "Using O_DIRECT on output file with blocksize %d\n"
              , state->b);
     }
+#else
+    fprintf(stderr, "Trying to use O_DIRECT but this OS doesn't have it\n");
+    exit(2);
+#endif

     if (!o->user_specified_blocksize) {
 #ifdef HAVE_SYS_STATVFS_H
