--- checkrestart      2018-07-20 17:27:39.000000000 +0000
+++ checkrestart        2019-05-31 12:07:56.136939223 +0000
@@ -566,6 +566,9 @@
     # Skip memfd files
     if f.startswith('/memfd:'):
         return 0
+    # Skip sssd cache
+    if f.startswith('/var/lib/sss/mc/'):
+        return 0
     # Skip, if asked to, files that do not belong to any package
     if onlyPackageFiles:
         # Remove some lsof information from the file to ensure that it is
