diff -Nur org/apt-0.7.6/methods/https.cc new/apt-0.7.6/methods/https.cc
--- org/apt-0.7.6/methods/https.cc	2008-12-15 15:58:05.000000000 +0200
+++ new/apt-0.7.6/methods/https.cc	2008-12-15 15:59:35.000000000 +0200
@@ -130,6 +130,7 @@
    curl_easy_setopt(curl, CURLOPT_NOPROGRESS, false);
    curl_easy_setopt(curl, CURLOPT_FAILONERROR, true);
    curl_easy_setopt(curl, CURLOPT_FILETIME, true);
+   curl_easy_setopt(curl, CURLOPT_NETRC, CURL_NETRC_OPTIONAL);
 
    // FIXME: https: offer various options of verification
    bool peer_verify = _config->FindB("Acquire::https::Verify-Peer", false);
