diff --git a/dh-autoreconf.pod b/dh-autoreconf.pod
index b3647d3..e7683ae 100644
--- a/dh-autoreconf.pod
+++ b/dh-autoreconf.pod
@@ -4,8 +4,9 @@ dh-autoreconf - debhelper add-on to run autoreconf during build
 
 =head1 DESCRIPTION
 
-The dh-autoreconf package provides a sequence addon for debhelper 7 which
-can be used in the following way:
+The dh-autoreconf package provides a sequence addon for debhelper 7 and is
+enabled by default since compatibility level 10. For earlier compatibility
+levels it can be enabled in the following way:
 
     #!/usr/bin/make -f
     %:
@@ -34,9 +35,6 @@ You can add support for -Wl,--as-needed to ltmain.sh (at least for those
 ltmain.sh scripts changed during autoreconf) by passing the argument
 B<--as-needed> to dh_autoreconf, as demonstrated in the following example:
 
-    #!/usr/bin/make -f
-    %:
-        dh $@ --with autoreconf
     override_dh_autoreconf:
         dh_autoreconf --as-needed
 
@@ -63,10 +61,11 @@ Or, if you use CDBS:
 
 =head1 CAVEATS
 
-dh_autoreconf is mostly a superset of the autotools-dev debhelper addons, so
-you do not need --with=autotools_dev if you use --with=autoreconf, as long
-as your autoreconf updates the config.guess and config.sub files. If it does
-not, feel free to use both together.
+dh_autoreconf is mostly a superset of the dh_update_autotools_config debhelper
+command included in debhelper since version 9.20160115. When using the dh
+sequencer, dh_update_autotools_config is run before dh_autoreconf and updates
+the config.guess and config.sub files. This is required in cases where
+autoreconf does not update config.guess and config.sub itself.
 
 From time to time, there might be a short breakage for those using
 automatic ltmain.sh patching, when the patch no longer applies to
