From b7b9123fa4527e1e29781df96e2f61529bc3d4f4 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Wed, 2 Jul 2014 17:52:34 +0200
Subject: [PATCH 02/11] po: Remove fuzzy from file msgid header

---
 doc/po/pt_BR.po | 1 -
 1 file changed, 1 deletion(-)

diff --git a/doc/po/pt_BR.po b/doc/po/pt_BR.po
index b996fa6..c3921a7 100644
--- a/doc/po/pt_BR.po
+++ b/doc/po/pt_BR.po
@@ -5,7 +5,6 @@
 # Translators:
 # André Luís Lopes <andrelop@ig.com.br>
 #
-#, fuzzy
 msgid ""
 msgstr ""
 "Project-Id-Version: apt\n"
-- 
2.0.1

