From b850304d55df5d4948cef43dcd3d5439adf9a7e4 Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Fri, 13 Oct 2017 14:08:51 -0700
Subject: [PATCH] dgit-maint-merge(7): Include instructions to clone existing
 repo

Signed-off-by: Sean Whitton <spwhitton@spwhitton.name>
---
 dgit-maint-merge.7.pod | 11 ++++++++++-
 1 file changed, 10 insertions(+), 1 deletion(-)

diff --git a/dgit-maint-merge.7.pod b/dgit-maint-merge.7.pod
index 3da1b78..ffece64 100644
--- a/dgit-maint-merge.7.pod
+++ b/dgit-maint-merge.7.pod
@@ -188,7 +188,16 @@ this workflow.  It should be skipped when debianising a new package.
 
 =head2 Existing git history using another workflow
 
-First, dump any existing patch queue:
+First, if you don't already have the git history locally, clone it,
+and obtain the corresponding orig.tar from the archive:
+
+=over 4
+
+    % git clone git.debian.org:collab-maint/foo
+    % cd foo
+    % origtargz
+
+Now dump any existing patch queue:
 
 =over 4
 
-- 
2.14.2

