--- scripts/checkbashisms.pl	2016-10-01 16:15:56.659665607 +0100
+++ /home/adam/.cache/evolution/tmp/evolution-adam-UUToFC/checkbashisms.pl	2017-02-24 20:10:51.680837263 +0000
@@ -665,8 +665,10 @@
 	qr'\$\{(?:\w+|@|\*)(/.+?){1,2}\}' =>      q<${parm/?/pat[/str]}>,
 	qr'\$\{\#?\w+\[.+\](?:[/,:#%^].+?)?\}' => q<bash arrays, ${name[0|*|@]}>,
 	qr'\$\{?RANDOM\}?\b' =>          q<$RANDOM>,
-	qr'\$\{?(OS|MACH)TYPE\}?\b'   => q<$(OS|MACH)TYPE>,
-	qr'\$\{?HOST(TYPE|NAME)\}?\b' => q<$HOST(TYPE|NAME)>,
+	qr'\$\{?OSTYPE\}?\b'   => q<$OSTYPE>,
+	qr'\$\{?MACHTYPE\}?\b'   => q<$MACHTYPE should be "$(gcc -dumpmachine">,
+	qr'\$\{?HOSTTYPE\}?\b' => q<$HOSTTYPE should be "$(uname -m)">,
+	qr'\$\{?HOSTNAME\}?\b' => q<$HOSTNAME should be "$(uname -n)">,
 	qr'\$\{?DIRSTACK\}?\b'        => q<$DIRSTACK>,
 	qr'\$\{?EUID\}?\b'	      => q<$EUID should be "$(id -u)">,
 	qr'\$\{?UID\}?\b'	       => q<$UID should be "$(id -ru)">,
