diff -Nru cvise-2.4.0/debian/changelog cvise-2.4.0/debian/changelog
--- cvise-2.4.0/debian/changelog	2021-11-02 16:15:02.000000000 +0200
+++ cvise-2.4.0/debian/changelog	2021-12-29 22:41:49.000000000 +0200
@@ -1,3 +1,12 @@
+cvise (2.4.0-2.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Remove stale LLVM 9 build dependencies on armel/armhf.
+    (Closes: #1000892)
+  * Recommend colordiff. (Closes: #1001375)
+
+ -- Adrian Bunk <bunk@debian.org>  Wed, 29 Dec 2021 22:41:49 +0200
+
 cvise (2.4.0-2) unstable; urgency=medium
 
   * (Build-)depend on python3-chardet.
diff -Nru cvise-2.4.0/debian/control cvise-2.4.0/debian/control
--- cvise-2.4.0/debian/control	2021-11-02 16:14:45.000000000 +0200
+++ cvise-2.4.0/debian/control	2021-12-29 22:41:49.000000000 +0200
@@ -15,9 +15,6 @@
   python3-pytest <!nocheck>,
   python3-pytest-flake8 <!nocheck>,
   llvm-13-dev, libclang-13-dev, clang-13, clang-format-13,
-#  llvm-12-dev, libclang-12-dev, clang-12, clang-format-12,
-#  llvm-11-dev [!armel !armhf], libclang-11-dev [!armel !armhf], clang-11 [!armel !armhf], clang-format-11 [!armel !armhf],
-  llvm-9-dev [armel armhf], libclang-9-dev [armel armhf], clang-9 [armel armhf], clang-format-9 [armel armhf],
   unifdef,
 Standards-Version: 4.6.0
 Homepage: https://github.com/marxin/cvise
@@ -26,13 +23,12 @@
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, ${python3:Depends},
   clang-format-13,
-#  clang-format-11 [!armel !armhf],
-#  clang-format-9 [armel armhf],
   python3,
   python3-chardet,
   python3-pebble,
   python3-psutil,
   unifdef,
+Recommends: colordiff
 Description: super-parallel Python port of the C-Reduce project
  C-Vise is a tool that takes a large C, C++ or OpenCL program that has
  a property of interest (such as triggering a compiler bug) and
diff -Nru cvise-2.4.0/debian/rules cvise-2.4.0/debian/rules
--- cvise-2.4.0/debian/rules	2021-11-02 14:45:41.000000000 +0200
+++ cvise-2.4.0/debian/rules	2021-12-29 22:41:45.000000000 +0200
@@ -3,11 +3,6 @@
 
 DEB_HOST_ARCH ?= $(shell dpkg-architecture -qDEB_HOST_ARCH)
 
-ifneq (,$(filter $(DEB_HOST_ARCH), armel armhf))
-  CLANG_V=9
-else
-  CLANG_V=11
-endif
 CLANG_V=13
 V=$(shell dpkg-parsechangelog -S Version | sed 's/-.*$$//')
 
