--- debarchiver.pl.orig	2018-06-22 21:07:41.000000000 +0200
+++ debarchiver.pl	2018-12-23 07:11:18.543228784 +0100
@@ -3190,6 +3190,15 @@
  * A file that is part of the Changes file is bigger than specified.
  * Verify signatures is enabled and signature do not match.
 
+head  EXAMPLES
+
+Suppose you have just uploaded package to repository e.g. with dput(1),
+and you don't want to wait for the cron process to pick them up. You
+can force immediate handling of incoming queue with this command. The
+second option allows overwriting existing archive files.
+ 
+ # debarchiver --scandetect --addoverride
+
 =head1 FILES
 
 B</etc/debarchiver.conf>
