From 4306db0c0e39f5c11e2dbe93d522c5effcfeb4b2 Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas@debian.org>
Date: Wed, 10 Apr 2024 00:16:20 +0200
Subject: [PATCH 05/10] scripts/t/mk/architecture.mk: test variable override

---
 scripts/t/mk/architecture.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/scripts/t/mk/architecture.mk b/scripts/t/mk/architecture.mk
index 538f80b48..56963ff1d 100644
--- a/scripts/t/mk/architecture.mk
+++ b/scripts/t/mk/architecture.mk
@@ -1,3 +1,6 @@
+DEB_BUILD_ARCH      := overridden
+TEST_DEB_BUILD_ARCH := overridden
+
 include $(srcdir)/mk/architecture.mk
 
 vars := \
-- 
2.39.2

