From de56dbe7d873fa825f10399d3cedd418f4f58cc7 Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas@debian.org>
Date: Fri, 8 Mar 2024 00:40:50 +0100
Subject: [PATCH 06/11] scripts/buildflags.mk: add missing GCJFLAGS

---
 scripts/mk/buildflags.mk | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/scripts/mk/buildflags.mk b/scripts/mk/buildflags.mk
index d3ce7aba3..700d2a1f7 100644
--- a/scripts/mk/buildflags.mk
+++ b/scripts/mk/buildflags.mk
@@ -10,6 +10,7 @@
 #   DFLAGS: flags for the host D compiler.
 #   FFLAGS: flags for the host Fortran 77 compiler.
 #   FCFLAGS: flags for the host Fortran 9x compiler.
+#   GCJFLAGS: flags for the host java compiler.
 #   LDFLAGS: flags for the host linker.
 #
 # And the following variables for the build tools (since dpkg 1.22.1):
@@ -23,6 +24,7 @@
 #   DFLAGS_FOR_BUILD: flags for the build D compiler.
 #   FFLAGS_FOR_BUILD: flags for the build Fortran 77 compiler.
 #   FCFLAGS_FOR_BUILD: flags for the build Fortran 9x compiler.
+#   GCJFLAGS_FOR_BUILD: flags for the host java compiler.
 #   LDFLAGS_FOR_BUILD: flags for the build linker.
 #
 # You can also export them in the environment by setting
@@ -47,6 +49,7 @@ DPKG_BUILDFLAGS_LIST := $(foreach var,\
   DFLAGS \
   FFLAGS \
   FCFLAGS \
+  GCJFLAGS \
   LDFLAGS \
   ,$(var) $(var)_FOR_BUILD)
 
-- 
2.39.2

