From b734bd99b2467170a9b7a99f002eba6d8d9ba521 Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas@debian.org>
Date: Wed, 10 Apr 2024 00:21:58 +0200
Subject: [PATCH 07/10] scripts/t/mk/buildtools.mk: add missing test for CPP

---
 scripts/t/mk/buildtools.mk | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/t/mk/buildtools.mk b/scripts/t/mk/buildtools.mk
index de2b52b59..3b758f8ba 100644
--- a/scripts/t/mk/buildtools.mk
+++ b/scripts/t/mk/buildtools.mk
@@ -6,6 +6,7 @@ tools := \
   AR \
   AS \
   CC \
+  CPP \
   CXX \
   F77 \
   FC \
-- 
2.39.2

