From d68d0e338febb0c54fdb90313b222a46345127f5 Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas@debian.org>
Date: Wed, 10 Apr 2024 00:23:19 +0200
Subject: [PATCH 08/10] scripts/t/mk/buildtools.mk: test override of a tool

---
 scripts/t/mk/buildtools.mk | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/scripts/t/mk/buildtools.mk b/scripts/t/mk/buildtools.mk
index 3b758f8ba..321e948ec 100644
--- a/scripts/t/mk/buildtools.mk
+++ b/scripts/t/mk/buildtools.mk
@@ -1,3 +1,7 @@
+AR                := overridden
+TEST_AR           := overridden
+TEST_AR_FOR_BUILD := overridden
+
 DPKG_EXPORT_BUILDTOOLS := 1
 
 include $(srcdir)/mk/buildtools.mk
-- 
2.39.2

