From ca50a267435a14fddd8679771b1e3c1178160ee1 Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas@debian.org>
Date: Tue, 2 Jul 2024 10:44:20 +0200
Subject: scripts/mk: replace more GNU sed constructs with POSIX ones

This hopefully fixes tests with BSD sed.

diff --git a/scripts/mk/pkg-info.mk b/scripts/mk/pkg-info.mk
index cfd456af1..ddda4f736 100644
--- a/scripts/mk/pkg-info.mk
+++ b/scripts/mk/pkg-info.mk
@@ -21,7 +21,7 @@ dpkg_pkg_info_mk_included = yes
 dpkg_parsechangelog_run = $(eval $(shell dpkg-parsechangelog | sed -n '\
   s/^Distribution: \(.*\)/$$(eval DEB_DISTRIBUTION:=\1)/p;\
   s/^Source: \(.*\)/$$(eval DEB_SOURCE:=\1)/p;\
-  s/^Version: \([0-9]*:\)\?\([^-]*\)\(\(.*\)-[^-]*\)\?$$/\
+  s/^Version: \([0-9]*:\)\{0,1\}\([^-]*\)\(\(.*\)-[^-]*\)\{0,1\}$$/\
     $$(eval DEB_VERSION:=\1\2\3)\
     $$(eval DEB_VERSION_EPOCH_UPSTREAM:=\1\2\4)\
     $$(eval DEB_VERSION_UPSTREAM_REVISION:=\2\3)\
