diff -Nru chiark-utils-4.4.2/debian/control chiark-utils-4.4.2+nmu1/debian/control
--- chiark-utils-4.4.2/debian/control	2014-10-26 10:20:47.000000000 -0400
+++ chiark-utils-4.4.2+nmu1/debian/control	2016-04-01 13:50:19.000000000 -0400
@@ -126,7 +126,7 @@
  general security risk to the system.
  .
  Unlike sudo it does not pretend that the called account can be any
- more secure than the calling account. so there is never a need for a
+ more secure than the calling account, so there is never a need for a
  password.  If you wanted to restrict which commands and functions the
  called user can perform, use userv, not really or sudo.
  .
