From 68b667376ad4d0e5c91b07123d5c52bfc2f0d4e9 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Sun, 8 Jan 2017 17:05:06 +0100
Subject: [PATCH] Fix DebianDoc-SGML tag

---
 upgrading-checklist.sgml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/upgrading-checklist.sgml b/upgrading-checklist.sgml
index 236ca7b..f0665a7 100644
--- a/upgrading-checklist.sgml
+++ b/upgrading-checklist.sgml
@@ -64,7 +64,7 @@ Not Released Yet
   <item>New <tt>Build-Depends-Arch</tt> and <tt>Build-Conflicts-Arch</tt>
   fields are now supported.</item>
 <tag>9.3.3.2</tag>
-  <item>Packages must not call <path>/etc/init.d</path> scripts directly
+  <item>Packages must not call <file>/etc/init.d</file> scripts directly
   even as a fallback, and instead must always use <prgn>invoke-rc.d</prgn>
   (which is essential and shouldn't require any conditional).</item>
 <tag>10.1</tag>
-- 
2.11.0

