--- cpu-1.4.3/debian/postrm
+++ cpu-1.4.3/debian/postrm
@@ -5,7 +5,7 @@
 
 set -e
 # Source debconf library.
-. /usr/share/debconf/confmodule
+[ -e /usr/share/debconf/confmodule ] && . /usr/share/debconf/confmodule
 
 
 # summary of how this script can be called:
@@ -41,9 +41,10 @@
 
 # Automatically added by dh_installdebconf
 if [ "$1" = purge ] && [ -e /usr/share/debconf/confmodule ]; then
-	. /usr/share/debconf/confmodule
 	db_purge
-	ucf --purge /etc/cpu/cpu.conf
+        if [ -x /usr/bin/ucf ]; then 
+                ucf --purge /etc/cpu/cpu.conf
+        fi
 fi
 # End automatically added section
 #DEBHELPER#
