From ce09340e7c51f23c3e2b08b5e069355aa8eecdad Mon Sep 17 00:00:00 2001
From: LaMont Jones <lamont@canonical.com>
Date: Thu, 23 Apr 2015 08:16:06 -0600
Subject: [PATCH] add m_sql and m_regex_{posix,pcre} modules

---
 debian/control | 2 +-
 debian/rules   | 3 +++
 2 files changed, 4 insertions(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index a5cf6e6..44b1655 100644
--- a/debian/control
+++ b/debian/control
@@ -2,7 +2,7 @@ Source: anope
 Section: net
 Priority: optional
 Maintainer: Dominic Hargreaves <dom@earth.li>
-Build-Depends: debhelper (>= 9), cmake
+Build-Depends: debhelper (>= 9), cmake, libpcre3-dev, libmysqlclient-dev
 Standards-Version: 3.9.6
 Homepage: http://www.anope.org/
 Vcs-Git: git://anonscm.debian.org/users/dom/anope.git
diff --git a/debian/rules b/debian/rules
index 6d0fc61..69276bd 100755
--- a/debian/rules
+++ b/debian/rules
@@ -22,6 +22,9 @@ include /usr/share/dpkg/default.mk
 	dh $@ 
 
 override_dh_auto_configure:
+	ln -s extra/m_mysql.cpp modules
+	ln -s extra/m_regex_pcre.cpp modules
+	ln -s extra/m_regex_posix modules
 	dh_auto_configure -- \
 	-DINSTDIR=/usr \
 	-DBIN_DIR=sbin \
-- 
2.1.4

