From 0aa52e10af3b5b99950073156a5d2dd0063ca9b0 Mon Sep 17 00:00:00 2001
From: Cyril Brulebois <kibi@debian.org>
Date: Sun, 22 Aug 2021 15:52:23 +0200
Subject: [PATCH] Talk to me.

---
 build/Makefile | 1 +
 1 file changed, 1 insertion(+)

diff --git a/build/Makefile b/build/Makefile
index 3a54a8bcd..9b6a9732b 100644
--- a/build/Makefile
+++ b/build/Makefile
@@ -724,6 +724,7 @@ endif
 	initramfs) \
 		$(mkinitramfs) $(TEMP)/initrd; \
 		$(gzip) -v9f $(TEMP)/initrd; \
+		zcat firmware-sof-signed.cpio.gz >> $(TEMP)/initrd.gz; \
 	;; \
 	jffs2) \
 		$(mkjffs2) $(TEMP_INITRD); \
-- 
2.30.2

