diff -Nurp dfu-util-0.8/debian/changelog dfu-util-0.9/debian/changelog
--- dfu-util-0.8/debian/changelog	2014-10-04 16:48:11.000000000 +0200
+++ dfu-util-0.9/debian/changelog	2016-02-09 21:47:40.433422482 +0100
@@ -1,3 +1,10 @@
+dfu-util (0.9-1) unstable; urgency=medium
+
+  * New upstream release
+  * Updated upstream location in control/copyright/watch
+
+ -- Tormod Volden <debian.tormod@gmail.com>  Tue, 09 Feb 2016 21:20:40 +0100
+
 dfu-util (0.8-1) unstable; urgency=medium
 
   * New upstream release (Closes: #696866).
diff -Nurp dfu-util-0.8/debian/control dfu-util-0.9/debian/control
--- dfu-util-0.8/debian/control	2014-10-04 16:49:24.000000000 +0200
+++ dfu-util-0.9/debian/control	2016-02-09 21:23:46.106317588 +0100
@@ -4,7 +4,7 @@ Priority: extra
 Maintainer: Uwe Hermann <uwe@debian.org>
 Build-Depends: cdbs, debhelper (>= 9), autotools-dev, libusb-1.0-0-dev, pkg-config
 Standards-Version: 3.9.6
-Homepage: http://dfu-util.gnumonks.org/
+Homepage: http://dfu-util.sourceforge.net
 
 Package: dfu-util
 Architecture: any
diff -Nurp dfu-util-0.8/debian/copyright dfu-util-0.9/debian/copyright
--- dfu-util-0.8/debian/copyright	2014-10-04 16:36:43.000000000 +0200
+++ dfu-util-0.9/debian/copyright	2016-02-09 21:41:29.074231284 +0100
@@ -1,10 +1,10 @@
 Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: dfu-util
-Source: http://dfu-util.gnumonks.org
+Source: http://dfu-util.sourceforge.net
 
 Files: *
 Copyright: 2005-2006 Weston Schmidt <weston_schmidt@alumni.purdue.edu>
-           2011-2014 Tormod Volden <debian.tormod@gmail.com>
+           2011-2016 Tormod Volden <debian.tormod@gmail.com>
 License: GPL-2+
  This program is free software; you can redistribute it and/or modify
  it under the terms of the GNU General Public License as published by
diff -Nurp dfu-util-0.8/debian/dfu-util.install dfu-util-0.9/debian/dfu-util.install
--- dfu-util-0.8/debian/dfu-util.install	2014-10-04 16:27:01.000000000 +0200
+++ dfu-util-0.9/debian/dfu-util.install	2016-02-09 21:27:10.307695568 +0100
@@ -1 +1,2 @@
 dfuse-pack.py usr/share/doc/dfu-util/examples
+doc/40-dfuse.rules usr/share/doc/dfu-util/examples
