diff -Naur apertium-1.0.3.old/configure.ac apertium-1.0.3.new/configure.ac
--- apertium-1.0.3.old/configure.ac	2006-10-02 18:41:05.000000000 +0200
+++ apertium-1.0.3.new/configure.ac	2007-06-28 14:36:22.000000000 +0200
@@ -3,7 +3,7 @@
 
 AC_PREREQ(2.52)
 
-m4_define([required_lttoolbox_version], [1.0.0])
+m4_define([required_lttoolbox_version], [2.0.0])
 m4_define([required_libxml_version], [2.6.17])
 
 #m4_define([required_pkg_config_version], [0.15])
@@ -149,7 +149,7 @@
 fi
 
 PKG_CHECK_MODULES(APERTIUM, [dnl
-  lttoolbox-1.0 >= required_lttoolbox_version dnl
+  lttoolbox-2.0 >= required_lttoolbox_version dnl
   libxml-2.0 >= required_libxml_version])
   
 # Checks for header files.
