From c3c7963de6d88f3b9e91230eb1f6229311267903 Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Fri, 1 Nov 2013 11:43:05 +0800
Subject: [PATCH 2/6] jessie freeze date announced

---
 data/releases/freezes | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/data/releases/freezes b/data/releases/freezes
index 1bd29fe..d140e14 100644
--- a/data/releases/freezes
+++ b/data/releases/freezes
@@ -41,3 +41,7 @@ Source: http://lists.debian.org/debian-devel-announce/2010/08/msg00000.html
 Title: Debian 7.0 ("wheezy") frozen
 Date: June 30 2012
 Source: http://lists.debian.org/debian-devel-announce/2012/06/msg00009.html
+
+Title: Debian 8.0 ("jessie") to be frozen
+Date: November 5 2014
+Source: http://lists.debian.org/debian-devel-announce/2013/10/msg00004.html
-- 
1.8.4.rc3

