From a15812cca74d01d48c8d108e6875719a48d41d29 Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Fri, 1 Nov 2013 11:48:43 +0800
Subject: [PATCH 3/6] Add LiMux and Bitraf 2013 BSPs

---
 data/events/bsps | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/data/events/bsps b/data/events/bsps
index 4f52ae8..57a3e1e 100644
--- a/data/events/bsps
+++ b/data/events/bsps
@@ -133,3 +133,13 @@ Title: Bug squashing party (Bitraf, Oslo, Norway)
 Start-Date: Jul 13 2013
 End-Date: Jul 13 2013
 Source: http://wiki.nuug.no/aktiviteter/20130713-feilfiksingsfest
+
+Title: Bug squashing party (Bitraf, Oslo, Norway)
+Start-Date: Oct 12 2013
+End-Date: Oct 13 2013
+Source: http://lists.debian.org/debian-events-eu/2013/09/msg00000.html
+
+Title: Bug squashing party (LiMux, Munich, Germany)
+Start-Date: Nov 22 2013
+End-Date: Nov 24 2013
+Source: http://wiki.debian.org/BSP/2013/11/de/Munich
-- 
1.8.4.rc3

