From bdffd16e4caeb47dcdaa8144a2cf5ce5dc8592ad Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Fri, 1 Nov 2013 11:53:10 +0800
Subject: [PATCH 4/6] debian-policy 3.9.5.0 was released

---
 data/events/policy | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/data/events/policy b/data/events/policy
index f34af3b..029c096 100644
--- a/data/events/policy
+++ b/data/events/policy
@@ -65,3 +65,7 @@ Source: http://ftp-master.metadata.debian.org/changelogs/main/d/debian-policy/un
 Title: debian-policy 3.9.4.0
 Date: Sep 18 2012
 Source: http://ftp-master.metadata.debian.org/changelogs/main/d/debian-policy/unstable_changelog
+
+Title: debian-policy 3.9.5.0
+Date: Oct 28 2013
+Source: http://ftp-master.metadata.debian.org/changelogs/main/d/debian-policy/unstable_changelog
-- 
1.8.4.rc3

