From d59a2154b75e7d3c1fc62cd42b202a809bac0101 Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Fri, 1 Nov 2013 11:55:09 +0800
Subject: [PATCH 5/6] s390 architecture retired

---
 data/events/architectures | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/data/events/architectures b/data/events/architectures
index f141908..def5b02 100644
--- a/data/events/architectures
+++ b/data/events/architectures
@@ -13,3 +13,7 @@ Source: http://lists.debian.org/debian-devel-announce/2009/04/msg00001.html
 Title: armhf and s390x architectures added
 Date: Nov 24 2011
 Source: http://lists.debian.org/debian-devel-announce/2011/11/msg00002.html
+
+Title: s390 architecture retired
+Date: Oct 14 2013
+Source: http://lists.debian.org/debian-devel-announce/2013/10/msg00005.html
-- 
1.8.4.rc3

