From 8894b7254537b83e9d32c6a14818391cdee3ccb1 Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Fri, 1 Nov 2013 12:09:01 +0800
Subject: [PATCH 6/6] Two upcoming Debian Mini-DebConfs

---
 data/events/conferences | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/data/events/conferences b/data/events/conferences
index 6ffc270..9959954 100644
--- a/data/events/conferences
+++ b/data/events/conferences
@@ -221,3 +221,13 @@ Title: DebConf13, Le Camp, Switzerland
 Start-Date: Aug 11 2013
 End-Date: Aug 17 2013
 Source: http://debconf13.debconf.org/
+
+Title: Mini-DebConf, Wulai, Taiwan
+Start-Date: Nov 8 2013
+End-Date: Nov 11 2013
+Source: http://wiki.debian.org/DebianTaiwan/MiniDebConf2013
+
+Title: Mini-DebConf, Cambridge, United Kingdom
+Start-Date: Nov 14 2013
+End-Date: Nov 17 2013
+Source: http://wiki.debconf.org/wiki/Miniconf-UK/2013
-- 
1.8.4.rc3

