From 7d9ab53f8b47815a62a4ff4e5c19facd70c57a90 Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Tue, 18 Oct 2016 17:22:49 -0700
Subject: [PATCH 3/4] debian/copyright: add myself

Signed-off-by: Sean Whitton <spwhitton@spwhitton.name>
---
 debian/copyright | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/debian/copyright b/debian/copyright
index 493b700..88a5e6d 100644
--- a/debian/copyright
+++ b/debian/copyright
@@ -3,6 +3,8 @@ Integration between git and Debian-style archives
 
 Copyright (C)2013-2016 Ian Jackson
 
+dgit-maint-merge.7.pod is copyright (C)2016 Sean Whitton
+
 This program is free software: you can redistribute it and/or modify
 it under the terms of the GNU General Public License as published by
 the Free Software Foundation, either version 3 of the License, or
-- 
2.9.3

