--- autoconf-archive-20170928.orig/m4/ax_path_bdb.m4
+++ autoconf-archive-20170928/m4/ax_path_bdb.m4
@@ -179,7 +179,7 @@
       ]])
     ],[
       # Extract version from preprocessor output.
-      HEADER_VERSION=`eval "$ac_cpp conftest.$ac_ext" 2> /dev/null \
+      HEADER_VERSION=`eval "$ac_cpp -P conftest.$ac_ext" 2> /dev/null \
         | grep AX_PATH_BDB_STUFF | sed 's/[[^0-9,]]//g;s/,/./g;1q'`
     ],[])
 
