Description: make boost look into multiarch locations as well
Author: Dmitrijs Ledkovs <dmitrij.ledkov@ubuntu.com>

--- cmake-2.8.11.2.orig/Modules/FindBoost.cmake
+++ cmake-2.8.11.2/Modules/FindBoost.cmake
@@ -841,6 +841,7 @@ else()
   list(APPEND _boost_LIBRARY_SEARCH_DIRS
     ${Boost_INCLUDE_DIR}/lib
     ${Boost_INCLUDE_DIR}/../lib
+    ${Boost_INCLUDE_DIR}/../lib/${CMAKE_LIBRARY_ARCHITECTURE}
     ${Boost_INCLUDE_DIR}/stage/lib
     )
   if( Boost_NO_SYSTEM_PATHS )
