From eb036ae710c7a92b8cfad0e982e13e76a218aa74 Mon Sep 17 00:00:00 2001
From: Osamu Aoki <osamu@debian.org>
Date: Sun, 23 Jun 2013 19:31:49 +0900
Subject: [PATCH 1/2] fix quotation

---
 parameter.c |    3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/parameter.c b/parameter.c
index d38f953..6423de8 100644
--- a/parameter.c
+++ b/parameter.c
@@ -78,7 +78,8 @@ int load_config_file(const char* config, pbuilderconfig* pc)
   char* delim;
   int result;
 
-  asprintf(&s, "env bash -c 'set -e ; . %s; set ' 2>&1", config);
+  /* pass string: env bash -c "set -e ; . %s; set |sed \"s/='\(.*\)'$/=\1/\" " 2>&1 */
+  asprintf(&s, "env bash -c \"set -e ; . %s; set |sed \\\"s/='\\(.*\\)'$/=\\1/\\\" \" 2>&1", config);
   f=popen(s, "r");
   if( NULL == f )
     return -1;
-- 
1.7.10.4

