--- cowdancer-0.72/parameter.c	2012-05-29 06:44:29.000000000 +0900
+++ cowdancer-0.72+nmu1/parameter.c	2013-06-23 17:51:55.323160548 +0900
@@ -78,7 +78,7 @@
   char* delim;
   int result;
 
-  asprintf(&s, "env bash -c 'set -e ; . %s; set ' 2>&1", config);
+  asprintf(&s, "env bash -c 'set -e ; . %s; set | sed -e \"s/\([^=]*\)=\'\(.*\)\'$/\1=\2/\"' 2>&1", config);
   f=popen(s, "r");
   if( NULL == f )
     return -1;
