--- ax_check_gl.m4-orig	2016-10-10 21:34:37.000000000 +0100
+++ ax_check_gl.m4	2016-12-16 16:54:19.198552717 +0000
@@ -164,7 +164,7 @@
 # Link the example program
 AC_DEFUN([_AX_CHECK_GL_LINK],
 [dnl
- _AX_CHECK_GL_SAVE_FLAGS([[CFLAGS],[LIBS]])
+ _AX_CHECK_GL_SAVE_FLAGS([[CFLAGS],[LIBS],[LDFLAGS]])
  AC_LINK_IFELSE([_AX_CHECK_GL_PROGRAM],
                 [ax_check_gl_link_opengl="yes"],
                 [ax_check_gl_link_opengl="no"])
