diff -pur a/trackdb/TocParser.g b/trackdb/TocParser.g
--- a/trackdb/TocParser.g	2007-09-01 13:09:43.000000000 +0200
+++ b/trackdb/TocParser.g	2007-09-01 13:10:03.000000000 +0200
@@ -496,6 +496,15 @@ subTrack < [ TrackData::Mode trackType, 
 	  }
        >>
     )
+    << if ($st != NULL && $st->length() == 0) {
+         // try to determine length 
+         if ($st->determineLength() != 0) {
+	   message(-2, "%s:%d: Cannot determine length of track data specification.",
+		   filename_, $lineNr);
+	   error_ = 1;
+	 }
+       }
+    >> 
     ;
     // fail action
     << delete $st, $st = NULL;
