diff --git a/scripts/chdist.pl b/scripts/chdist.pl
index b473b954..52832ed5 100755
--- a/scripts/chdist.pl
+++ b/scripts/chdist.pl
@@ -707,72 +707,53 @@ sub parseFile {
 my $recursed = 0;
 MAIN:
 my $command = shift @ARGV;
-given ($command) {
-    when ('create') {
-        dist_create(@ARGV);
-    }
-    when ('apt') {
-        aptcmd('apt', @ARGV);
-    }
-    when ('apt-get') {
-        aptcmd('apt-get', @ARGV);
-    }
-    when ('apt-cache') {
-        aptcmd('apt-cache', @ARGV);
-    }
-    when ('apt-file') {
-        apt_file(@ARGV);
-    }
-    when ('apt-rdepends') {
-        aptcmd('apt-rdepends', @ARGV);
-    }
-    when ('aptitude') {
-        aptcmd('aptitude', @ARGV);
-    }
-    when ('bin2src') {
-        bin2src(@ARGV);
-    }
-    when ('src2bin') {
-        src2bin(@ARGV);
-    }
-    when ('compare-packages') {
-        dist_compare(@ARGV, 0, 'Sources');
-    }
-    when ('compare-bin-packages') {
-        dist_compare(@ARGV, 0, 'Packages');
-    }
-    when ('compare-versions') {
-        dist_compare(@ARGV, 1, 'Sources');
-    }
-    when ('compare-bin-versions') {
-        dist_compare(@ARGV, 1, 'Packages');
-    }
-    when ('grep-dctrl-packages') {
-        grep_file(@ARGV, 'Packages');
-    }
-    when ('grep-dctrl-sources') {
-        grep_file(@ARGV, 'Sources');
-    }
-    when ('compare-src-bin-packages') {
-        compare_src_bin(@ARGV, 0);
-    }
-    when ('compare-src-bin-versions') {
-        compare_src_bin(@ARGV, 1);
-    }
-    when ('list') {
-        list;
-    }
-    default {
-        my $dist = $command;
-        my $dir  = "$datadir/$dist";
-        if (-d $dir && !$recursed) {
-            splice @ARGV, 1, 0, $dist;
-            $recursed = 1;
-            goto MAIN;
-        } elsif ($dist && !$recursed) {
-            dist_check($dist);
-        } else {
-            usage(1);
-        }
+if ($command eq 'create') {
+    dist_create(@ARGV);
+} elsif ($command eq 'apt') {
+    aptcmd('apt', @ARGV);
+} elsif ($command eq 'apt-get') {
+    aptcmd('apt-get', @ARGV);
+} elsif ($command eq 'apt-cache') {
+    aptcmd('apt-cache', @ARGV);
+} elsif ($command eq 'apt-file') {
+    apt_file(@ARGV);
+} elsif ($command eq 'apt-rdepends') {
+    aptcmd('apt-rdepends', @ARGV);
+} elsif ($command eq 'aptitude') {
+    aptcmd('aptitude', @ARGV);
+} elsif ($command eq 'bin2src') {
+    bin2src(@ARGV);
+} elsif ($command eq 'src2bin') {
+    src2bin(@ARGV);
+} elsif ($command eq 'compare-packages') {
+    dist_compare(@ARGV, 0, 'Sources');
+} elsif ($command eq 'compare-bin-packages') {
+    dist_compare(@ARGV, 0, 'Packages');
+} elsif ($command eq 'compare-versions') {
+    dist_compare(@ARGV, 1, 'Sources');
+} elsif ($command eq 'compare-bin-versions') {
+    dist_compare(@ARGV, 1, 'Packages');
+} elsif ($command eq 'grep-dctrl-packages') {
+    grep_file(@ARGV, 'Packages');
+} elsif ($command eq 'grep-dctrl-sources') {
+    grep_file(@ARGV, 'Sources');
+} elsif ($command eq 'compare-src-bin-packages') {
+    compare_src_bin(@ARGV, 0);
+} elsif ($command eq 'compare-src-bin-versions') {
+    compare_src_bin(@ARGV, 1);
+} elsif ($command eq 'list') {
+    list;
+} else {
+    my $dist = $command;
+    my $dir  = "$datadir/$dist";
+    if (-d $dir && !$recursed) {
+        splice @ARGV, 1, 0, $dist;
+        $recursed = 1;
+        goto MAIN;
+    } elsif ($dist && !$recursed) {
+        dist_check($dist);
+    } else {
+        usage(1);
     }
 }
+
