--- argus-3.0.0/common/scanner.l	2008-02-05 03:37:42.000000000 +0000
+++ argus-3.0.2/common/scanner.l	2009-08-17 14:34:34.000000000 +0000
@@ -64,7 +64,11 @@
 #include <argus_filter.h>
 
 #include <tokdefs.h>
+
+#if (HAVE_SYSLOG_H)
 #include <syslog.h>
+#endif
+
 
 #if !defined(HAVE_STRTOF)
 float strtof (char *, char **);
@@ -85,7 +89,6 @@
 int argus_get_lineno (void);
 FILE *argus_get_in (void);
 FILE *argus_get_out (void);
-int argus_get_leng (void);
 char *argus_get_text(void);
 void argus_set_lineno (int);
 void argus_set_in (FILE *);
