diff -u bash-4.2+dfsg/debian/skel.bashrc bash-4.2+dfsg/debian/skel.bashrc
--- bash-4.2+dfsg/debian/skel.bashrc
+++ bash-4.2+dfsg/debian/skel.bashrc
@@ -37,7 +37,7 @@
 
 # set a fancy prompt (non-color, unless we know we "want" color)
 case "$TERM" in
-    xterm-color) color_prompt=yes;;
+    xterm-color|*-256color) color_prompt=yes;;
 esac
 
 # uncomment for a colored prompt, if the terminal has the capability; turned
diff -u bash-4.2+dfsg/debian/changelog bash-4.2+dfsg/debian/changelog
--- bash-4.2+dfsg/debian/changelog
+++ bash-4.2+dfsg/debian/changelog
@@ -1,3 +1,12 @@
+bash (4.2+dfsg-0.1+deb7u3.1) UNRELEASED; urgency=low
+
+  * Non-maintainer upload.
+  * add check for xterm-256color and similar to skel.bashrc
+  * add check to bash/etc/skel/.bashrc
+  * fix typo in case statement
+
+ -- jason <jason@dickson.st>  Fri, 24 Oct 2014 11:20:08 +1100
+
 bash (4.2+dfsg-0.1+deb7u3) wheezy-security; urgency=high
 
   * Non-maintainer upload by the Security Team.
