--- a/scripts/dpkg-gencontrol.pl	2016-07-04 11:00:33.000000000 +0000
+++ b/scripts/dpkg-gencontrol.pl	2016-07-22 20:38:47.000000000 +0000
@@ -298,7 +298,9 @@
     }
 }
 
-$fields->{'Built-For-Profiles'} = join ' ', get_build_profiles();
+my @build_profiles = get_build_profiles();
+$fields->{'Built-For-Profiles'} = join ' ', @build_profiles
+    unless any { $_ eq 'clean' } @build_profiles;
 
 for my $f (qw(Package Version Architecture)) {
     error(g_('missing information for output field %s'), $f)
