--- working/client.c	2013-01-14 14:21:48.000000000 +0100
+++ orig/client.c	2013-01-14 13:31:04.000000000 +0100
@@ -2589,11 +2589,6 @@
 #ifdef HAVE_GSSAPI
     else if (auth_type == CUPSD_AUTH_NEGOTIATE)
     {
-#  ifdef AF_LOCAL
-      if (_httpAddrFamily(con->http.hostaddr) == AF_LOCAL)
-        strlcpy(auth_str, "Basic realm=\"CUPS\"", sizeof(auth_str));
-      else
-#  endif /* AF_LOCAL */
       strlcpy(auth_str, "Negotiate", sizeof(auth_str));
     }
 #endif /* HAVE_GSSAPI */
