diff -ur a/debian/chromium.desktop b/debian/chromium.desktop
--- a/debian/chromium.desktop	2015-10-24 18:27:47.000000000 -0300
+++ b/debian/chromium.desktop	2015-11-25 11:52:54.311748272 -0300
@@ -135,5 +135,5 @@
 Icon=chromium.png
 Categories=Network;WebBrowser;
 MimeType=text/html;text/xml;application/xhtml_xml;application/x-mimearchive;x-scheme-handler/http;x-scheme-handler/https;
-StartupWMClass=Chromium
+StartupWMClass=chromium-browser
 StartupNotify=true
