diff -u bwbar-1.2.3/debian/changelog bwbar-1.2.3/debian/changelog
--- bwbar-1.2.3/debian/changelog
+++ bwbar-1.2.3/debian/changelog
@@ -1,3 +1,10 @@
+bwbar (1.2.3-1.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * libpng transition.
+
+ -- Nobuhiro Iwamatsu <iwamatsu@nigauri.org>  Mon, 30 May 2011 10:37:01 +0900
+
 bwbar (1.2.3-1) unstable; urgency=low
 
   * New upstream release
only in patch2:
unchanged:
--- bwbar-1.2.3.orig/bwbar.c
+++ bwbar-1.2.3/bwbar.c
@@ -30,6 +30,7 @@
 #include <alloca.h>
 #include <getopt.h>
 #include <png.h>
+#include <zlib.h>
 
 void skipline(FILE *f)
 {
@@ -79,7 +80,7 @@
   if ( !info_ptr )
     goto barf;
 
-  if ( setjmp(png_ptr->jmpbuf) ) {
+  if ( setjmp(png_jmpbuf((png_ptr))) ) {
     status = 1;
     goto barf;			/* libpng abort */
   }
