diff -uNr bzr-0.6/bzrlib/trace.py bzr-0.6-tanner/bzrlib/trace.py
--- bzr-0.6/bzrlib/trace.py	2005-10-28 05:57:06.000000000 +0000
+++ bzr-0.6-tanner/bzrlib/trace.py	2005-11-02 22:36:14.317884117 +0000
@@ -130,7 +130,11 @@
     # buffering=1 means line buffered
     try:
         tf = codecs.open(trace_fname, 'at', 'utf8', buffering=1)
+    except IOError:
+	# Could not open logfile, default to sys.stderr
+        tf = sys.stderr
 
+    try:
         if os.fstat(tf.fileno())[stat.ST_SIZE] == 0:
             tf.write("\nthis is a debug log for diagnosing/reporting problems in bzr\n")
             tf.write("you can delete or truncate this file, or include sections in\n")
