--- /tmp/bash_completion	2008-02-13 22:47:59.000000000 +0100
+++ /etc/bash_completion	2008-02-10 16:23:20.000000000 +0100
@@ -2026,14 +2026,14 @@
 	prev=${COMP_WORDS[COMP_CWORD-1]}
 
 	for (( i=0; i < ${#COMP_WORDS[@]}-1; i++ )); do
-		if [[ ${COMP_WORDS[i]} == @(install|remove|source|build-dep) ]]; then
+		if [[ ${COMP_WORDS[i]} == @(install|remove|autoremove|source|build-dep) ]]; then
 			special=${COMP_WORDS[i]}
 		fi
 	done
 
 	if [ -n "$special" ]; then
 		case $special in
-		remove)
+		remove|autoremove)
 			if [ -f /etc/debian_version ]; then
 				# Debian system
 				COMPREPLY=( $( _comp_dpkg_installed_packages \
@@ -2085,7 +2085,7 @@
 
 		COMPREPLY=( $( compgen -W 'update upgrade dselect-upgrade \
 				dist-upgrade install remove source build-dep \
-				check clean autoclean' -- $cur ) )
+				check clean autoclean autoremove' -- $cur ) )
 
 	fi
 

