diff -Nru dublin-traceroute-0.4.2/debian/changelog dublin-traceroute-0.4.2/debian/changelog
--- dublin-traceroute-0.4.2/debian/changelog	2017-10-28 21:26:04.000000000 +0300
+++ dublin-traceroute-0.4.2/debian/changelog	2018-09-27 20:15:03.000000000 +0300
@@ -1,3 +1,12 @@
+dublin-traceroute (0.4.2-1.1) unstable; urgency=high
+
+  * Non-maintainer upload.
+  * libdublintraceroute-dev: Add the missing dependencies on
+    libtins-dev and libjsoncpp-dev, thanks Ximin Luo.
+    (Closes: #894345)
+
+ -- Adrian Bunk <bunk@debian.org>  Thu, 27 Sep 2018 20:15:03 +0300
+
 dublin-traceroute (0.4.2-1) unstable; urgency=medium
 
   * New upstream release
diff -Nru dublin-traceroute-0.4.2/debian/control dublin-traceroute-0.4.2/debian/control
--- dublin-traceroute-0.4.2/debian/control	2017-10-28 21:26:04.000000000 +0300
+++ dublin-traceroute-0.4.2/debian/control	2018-09-27 20:15:03.000000000 +0300
@@ -38,7 +38,7 @@
 
 Package: libdublintraceroute-dev
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}, libdublintraceroute0 (= ${binary:Version})
+Depends: ${shlibs:Depends}, ${misc:Depends}, libdublintraceroute0 (= ${binary:Version}), libtins-dev, libjsoncpp-dev
 Section: libdevel
 Description: NAT-aware multipath tracerouting tool (dev)
  Dublin Traceroute uses the techniques invented by the authors of
