diff -Nru backuppc-3.1.0/debian/control backuppc-3.1.0/debian/control
--- backuppc-3.1.0/debian/control	2009-08-03 17:30:20.000000000 +0000
+++ backuppc-3.1.0/debian/control	2009-08-03 17:30:20.000000000 +0000
@@ -9,7 +9,7 @@
 
 Package: backuppc
 Architecture: all
-Depends: ${perl:Depends}, libdigest-md5-perl, libcompress-zlib-perl, libarchive-zip-perl, tar (>> 1.13), adduser (>=3.9), dpkg (>=1.8.3), apache2 | httpd, perl-suid, ${misc:Depends}, smbclient, samba-common | samba-tng-common, bzip2, exim4 | sendmail | postfix | mail-transport-agent, iputils-ping
+Depends: ${perl:Depends}, libdigest-md5-perl, libcompress-zlib-perl, libarchive-zip-perl, tar (>> 1.13), adduser (>=3.9), dpkg (>=1.8.3), apache2 | httpd, perl-suid, ${misc:Depends}, smbclient, samba-common | samba-tng-common, bzip2, default-mta | mail-transport-agent, iputils-ping
 Recommends: rsync, libfile-rsyncp-perl (>= 0.68), openssh-client | ssh-client, rrdtool
 Suggests: w3m | www-browser, par2
 Conflicts: libfile-rsyncp-perl (<< 0.68)
