diff -Nru exim4-4.72.orig/debian/control exim4-4.72/debian/control
--- exim4-4.72.orig/debian/control	2010-06-03 17:35:40.000000000 +0200
+++ exim4-4.72/debian/control	2010-07-25 03:31:57.000000000 +0200
@@ -8,7 +8,7 @@
 Build-Depends: debhelper (>= 7.0.15), po-debconf, docbook-xsl, xsltproc,
   lynx-cur | lynx, docbook-xml, libpcre3-dev, libldap2-dev, libpam0g-dev,
   libident-dev, libdb4.8-dev, libxmu-dev, libxt-dev, libxext-dev, libx11-dev,
-  libxaw7-dev, libpq-dev, libmysqlclient15-dev, libsqlite3-dev, libperl-dev,
+  libxaw7-dev, libpq-dev, libmysqlclient-dev, libsqlite3-dev, libperl-dev,
   libgnutls-dev, libsasl2-dev
 
 Package: exim4-base
