Index: debian/rules
===================================================================
--- debian/rules	(revision 243)
+++ debian/rules	(working copy)
@@ -69,14 +69,26 @@
 # Some convenience variables
 export TMPPKG := $(CURDIR)/debian/tmp
 
-AUTOTOOLS=rm -f acinclude.m4 aclocal.m4 config/config.sub \
-	config/config.guess config/ltmain.sh config/libtool.m4 && \
-	libtoolize --force && \
-	aclocal-1.9 -I $(CURDIR)/cmulocal -I $(CURDIR)/config && \
-	autoheader && \
-	autoconf && \
-	automake-1.9 --add-missing --include-deps && \
-	touch stamp-h.in
+AUTOFILES=acinclude.m4 aclocal.m4 config/config.sub config/config.guess \
+		  config/ltmain.sh config/libtool.m4
+BUILD_TMP_SUFFIX=.debian-build.tmp
+AUTOTOOLS=for i in $(AUTOFILES); do \
+		  	if [ -e $$i ]; then \
+				mv --verbose $$i `basename $$i`$(BUILD_TMP_SUFFIX) ; \
+			fi ; \
+		  done && \
+		  libtoolize --force && \
+		  aclocal-1.9 -I $(CURDIR)/cmulocal -I $(CURDIR)/config && \
+		  autoheader && \
+		  autoconf && \
+		  automake-1.9 --add-missing --include-deps && \
+		  touch stamp-h.in
+AUTOTOOLS_REVERSE=for i in $(AUTOFILES); do \
+				  	if [ -e `basename $$i`$(BUILD_TMP_SUFFIX) ]; then \
+						if [ -e $$i ]; then rm -fv $$i; fi ; \
+						mv --verbose `basename $$i`$(BUILD_TMP_SUFFIX) $$i ; \
+					fi ; \
+				  done
 
 
 ### The Makefile targets begin. ###
@@ -92,14 +104,10 @@
 	dh_testroot
 	rm -f build-stamp 
 
-ifneq "$(wildcard config.sub.old)" ""
-	mv config.sub.old config/config.sub
-endif
-ifneq "$(wildcard config.guess.old)" ""
-	mv config.guess.old config/config.guess
-endif
 	rm -f $(CURDIR)/sample/{sample-client,sample-server,config.h}
 	-$(MAKE) distclean
+	$(AUTOTOOLS_REVERSE)
+	(cd saslauthd && $(AUTOTOOLS_REVERSE) && cd ..)
 	-rm -f config.h config.log autom4ate.cache
 
 	dh_clean 
@@ -111,13 +119,6 @@
 config.status: patch configure
 	dh_testdir
 
-ifneq "$(wildcard /usr/share/misc/config.sub)" ""
-	mv -f config/config.sub config.sub.old
-endif
-ifneq "$(wildcard /usr/share/misc/config.guess)" ""
-	mv -f config/config.guess config.guess.old
-endif
-
 	# Avoid problems with timestamp skew and autotools.
 	touch configure
 	$(AUTOTOOLS)
