--- rules.orig	2013-06-01 11:12:17.000000000 +1000
+++ rules	2013-10-24 09:27:37.000000000 +1100
@@ -104,11 +104,13 @@
 # Install directories
 	$(INSTALL_DIR) 	\
 		debian/tmp/bin	\
+		debian/tmp/usr/lib/mime/packages \
 		debian/tmp/usr/share/man/man1	\
 		debian/tmp/usr/share/info
 # Install files
 	$(MAKE) -C obj install DESTDIR=$(CURDIR)/debian/tmp
 	rm -rf debian/tmp/usr/libexec
+	install -m 644 debian/cpio.mime debian/tmp/usr/lib/mime/packages/cpio
 	mv debian/tmp/bin/mt debian/tmp/bin/mt-gnu
 	mv debian/tmp/usr/share/man/man1/mt.1 \
 	  debian/tmp/usr/share/man/man1/mt-gnu.1
