diff -ru dbus-1.2.1~/dbus/dbus-sysdeps-util-unix.c dbus-1.2.1/dbus/dbus-sysdeps-util-unix.c
--- dbus-1.2.1~/dbus/dbus-sysdeps-util-unix.c	2009-01-04 00:33:12.000000000 +0000
+++ dbus-1.2.1/dbus/dbus-sysdeps-util-unix.c	2009-01-04 00:35:31.000000000 +0000
@@ -48,6 +48,7 @@
 #include <sys/capability.h>
 #include <libaudit.h>
 #endif /* HAVE_LIBAUDIT */
+#include <syslog.h>
 
 #ifdef HAVE_SYS_SYSLIMITS_H
 #include <sys/syslimits.h>
