diff -pruN 2.0.5-2/debian/cecilia.desktop 2.0.5-2ubuntu1/debian/cecilia.desktop
--- 2.0.5-2/debian/cecilia.desktop	1970-01-01 01:00:00.000000000 +0100
+++ 2.0.5-2ubuntu1/debian/cecilia.desktop	2006-06-27 19:31:18.000000000 +0100
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Name=Cecilia
+GenericName=
+Comment=Graphical editor for CSound
+Icon=cecilia
+Exec=cecilia
+Terminal=false
+Categories=AudioVideo;Audio;
diff -pruN 2.0.5-2/debian/rules 2.0.5-2ubuntu1/debian/rules
--- 2.0.5-2/debian/rules	2006-06-27 19:31:18.000000000 +0100
+++ 2.0.5-2ubuntu1/debian/rules	2006-06-27 19:31:18.000000000 +0100
@@ -51,6 +51,8 @@ install: build
 	dh_testroot
 	dh_clean -k
 	dh_installdirs
+	dh_install debian/cecilia.desktop usr/share/applications
+	dh_desktop
 
 	# Add here commands to install the package into debian/cecilia.
 	$(MAKE) install DESTDIR=$(CURDIR)/debian/cecilia
