--- control.old	2014-10-26 22:52:39.403617958 -0600
+++ control	2014-10-26 22:52:48.911617536 -0600
@@ -9,7 +9,7 @@
 
 Package: dblatex
 Architecture: all
-Depends: ${misc:Depends}, ${python:Depends}, docbook-xml, xsltproc, python-apt (>= 0.7.91), texlive (>= 2009), texlive-bibtex-extra (>= 2009), texlive-latex-extra (>= 2009), texlive-math-extra (>= 2009), texlive-extra-utils (>= 2009)
+Depends: ${misc:Depends}, ${python:Depends}, docbook-xml, xsltproc, python-apt (>= 0.7.91), texlive (>= 2009), texlive-bibtex-extra (>= 2009), texlive-latex-extra (>= 2009), texlive-math-extra (>= 2009), texlive-extra-utils (>= 2009), inkscape
 Recommends: libxml2-utils
 Suggests: docbook, ghostscript, graphicsmagick-imagemagick-compat | imagemagick, latex-cjk-all, lmodern, opensp, pdf-viewer, texlive-lang-cyrillic (>= 2009), texlive-xetex (>= 2009), transfig
 Description: Produces DVI, PostScript, PDF documents from DocBook sources
