diff --git a/lib/Devscripts/Salsa/merge_request.pm b/lib/Devscripts/Salsa/merge_request.pm
index 89152aa9..7682bb52 100644
--- a/lib/Devscripts/Salsa/merge_request.pm
+++ b/lib/Devscripts/Salsa/merge_request.pm
@@ -47,3 +47,3 @@ sub merge_request {
                 ds_warn
-                  "Current branch as no origin or isn't pushed, aborting";
+                  "Current branch has no origin or isn't pushed, aborting";
                 return 1;
