diff -ur bind9-9.8.4.dfsg.P1.orig/lib/dns/dnssec.c bind9-9.8.4.dfsg.P1/lib/dns/dnssec.c
--- bind9-9.8.4.dfsg.P1.orig/lib/dns/dnssec.c	2012-10-26 06:52:55.000000000 +0200
+++ bind9-9.8.4.dfsg.P1/lib/dns/dnssec.c	2013-01-08 14:29:19.980398778 +0100
@@ -552,7 +552,7 @@
 		char namebuf[DNS_NAME_FORMATSIZE];
 		dns_name_format(&sig.signer, namebuf, sizeof(namebuf));
 		isc_log_write(dns_lctx, DNS_LOGCATEGORY_GENERAL,
-			      DNS_LOGMODULE_DNSSEC, ISC_LOG_INFO,
+			      DNS_LOGMODULE_DNSSEC, ISC_LOG_DEBUG(1),
 			      "sucessfully validated after lower casing "
 			      "signer '%s'", namebuf);
 		inc_stat(dns_dnssecstats_downcase);
diff -ur bind9-9.8.4.dfsg.P1.orig/lib/dns/opensslrsa_link.c bind9-9.8.4.dfsg.P1/lib/dns/opensslrsa_link.c
--- bind9-9.8.4.dfsg.P1.orig/lib/dns/opensslrsa_link.c	2012-10-26 06:52:55.000000000 +0200
+++ bind9-9.8.4.dfsg.P1/lib/dns/opensslrsa_link.c	2013-01-08 14:26:58.996397527 +0100
@@ -633,8 +633,7 @@
 #endif
 #endif
 	if (status != 1)
-		return (dst__openssl_toresult2("RSA_verify",
-					       DST_R_VERIFYFAILURE));
+		return (dst__openssl_toresult(DST_R_VERIFYFAILURE));
 
 	return (ISC_R_SUCCESS);
 }
