diff -ur bind9-9.8.4.dfsg.P1.orig/lib/dns/opensslrsa_link.c bind9-9.8.4.dfsg.P1/lib/dns/opensslrsa_link.c
--- bind9-9.8.4.dfsg.P1.orig/lib/dns/opensslrsa_link.c	2012-10-26 06:52:55.000000000 +0200
+++ bind9-9.8.4.dfsg.P1/lib/dns/opensslrsa_link.c	2013-01-08 14:26:58.996397527 +0100
@@ -633,8 +633,7 @@
 #endif
 #endif
 	if (status != 1)
-		return (dst__openssl_toresult2("RSA_verify",
-					       DST_R_VERIFYFAILURE));
+		return (dst__openssl_toresult(DST_R_VERIFYFAILURE));
 
 	return (ISC_R_SUCCESS);
 }
