Index: src/modules/frontend/newt/newt.c
===================================================================
--- src/modules/frontend/newt/newt.c	(révision 58334)
+++ src/modules/frontend/newt/newt.c	(copie de travail)
@@ -777,13 +777,15 @@
         t_height = newtTextboxGetNumLines(textbox);
         newtTextboxSetHeight(textbox, t_height);
         newtFormAddComponent(form, textbox);
-        b_height = 0; // A <Go Back> button is not necessary
         select_list_top = 1+t_height+1;
     } else {
         t_height = 0;
-        b_height = 1;
         select_list_top = 1; // No description. Only insert a blank line.
     }
+    if (obj->methods.can_go_back(obj, q))
+        b_height = 1;
+    else
+        b_height = 0;
     free(full_description);
     win_height  = t_height + sel_height + b_height;
     //    3 == First blank line + blanks before and after select
