diff -Nru --exclude changelog bbpager-0.4.7/src/Baseresource.cxx bbpager-0.4.7/src/Baseresource.cxx
--- bbpager-0.4.7/src/Baseresource.cxx	2006-12-31 21:54:38.000000000 +0530
+++ bbpager-0.4.7/src/Baseresource.cxx	2008-03-26 09:20:28.000000000 +0530
@@ -22,6 +22,7 @@
 #include <string>
 #include "Baseresource.h"
 #include <stdio.h>
+#include <cstring>
 #include "Menu.hh"
 
 BaseResource::BaseResource(bt::Application &_app, unsigned int _screen, const std::string &blackbox_rc_filename, const std::string &filename) : 
