diff --git a/project-history.sgml b/project-history.sgml
--- a/project-history.sgml
+++ b/project-history.sgml
@@ -190,7 +190,10 @@
 Stefano Zacchiroli led Debian from April 2010 until April 2013.
 
 <p>
-Lucas Nussbaum was elected in April 2013 and is our current leader.
+Lucas Nussbaum led Debian from April 2013 until April 2015.
+
+<p>
+Neil McGovern was elected in April 2015 and is our current leader.
 
 </chapt>
 
