diff -u cron-3.0pl1/debian/changelog cron-3.0pl1/debian/changelog
--- cron-3.0pl1/debian/changelog
+++ cron-3.0pl1/debian/changelog
@@ -1,3 +1,10 @@
+cron (3.0pl1-113.1) UNRELEASED; urgency=low
+
+  * Non-maintainer upload.
+  * Don't include information in NEWS.Debian that isn't relevant.
+
+ -- Steve Langasek <vorlon@debian.org>  Fri, 30 Jul 2010 13:38:24 -0700
+
 cron (3.0pl1-113) unstable; urgency=medium
 
   [ Christian Kastner / Javier Fernandez-Sanguino ]
diff -u cron-3.0pl1/debian/NEWS cron-3.0pl1/debian/NEWS
--- cron-3.0pl1/debian/NEWS
+++ cron-3.0pl1/debian/NEWS
@@ -2,21 +2,13 @@
 
     In the past, long-running jobs caused certain MTAs to time out,
     resulting in no output being sent. Justin Pryzby's patch to fix this
-    has been included. It may be worth to note that Debian's default MTA
-    exim was not susceptible to this.
+    has been included.
 
-    Tasks generating a lot of output no longer die with when no MTA is
-    installed.
-
-    crontab(1) now refuses any crontabs missing a newline before EOF. In
-    the past, this has been the source of much confusion -- especially
-    because the upstream code supposed to detect this was severely broken.
+    crontab(1) now refuses any crontabs missing a newline before EOF.
 
     cron(8) now recovers from broken symlinks in /etc/cron.d. This fix
     may cause power management issues (disk spin-ups), feedback welcome.
 
-    Christian Kastner came on board as a maintainer.
-
  -- Christian Kastner <debian@kvr.at>  Thu, 29 Apr 2010 13:25:09 +0200
 
 cron (3.0pl1-74) unstable; urgency=low
