diff --git a/init.d/dbus b/init.d/dbus
index 1f9b20d..3eb671c 100755
--- a/init.d/dbus
+++ b/init.d/dbus
@@ -86,6 +86,9 @@ dependent_services()
   for i in $services ; do
     service=$(basename $i)
     service=${service#S??}
+    # skip services which can survive a dbus restart
+    [ "$service" = "powersaved" ] && continue;
+    [ "$service" = "network-manager" ] && continue;
     invoke-rc.d $service $action || true
   done
 
