--- catdoc-0.94.3~git20160113.dbc9ec6+dfsg.pristine/debian/control	2016-01-13 22:44:37.000000000 +0000
+++ catdoc-0.94.3~git20160113.dbc9ec6+dfsg/debian/control	2016-02-19 00:30:43.299037609 +0000
@@ -12,16 +12,16 @@
 Depends: ${shlibs:Depends}, ${misc:Depends}
 Suggests: tk | wish
 Homepage: http://www.wagner.pp.ru/~vitus/software/catdoc/
-Description: Convert Word, Excel, and PowerPoint files to plain text
- The catdoc program reads one or more Microsoft word files and outputs text,
- contained insinde them to standard output.
+Description: text extractor for MS-Office files
+ The catdoc program reads one or more Microsoft Word files and outputs
+ their contents to standard output as text.
  .
- It is now accompanied by xls2csv, a program which converts Excel spreadsheet
- into comma-separated value file, and catppt a utility to extract textual
- information from Powerpoint files.
+ It is accompanied by xls2csv, a program which converts Excel spreadsheets
+ into comma-separated-values format, and catppt, a utility to extract textual
+ information from PowerPoint files.
  .
- It doesn't try to preserve Word formatting, it's goal is to extract plain text
- and allow you to read it and, probably, reformat with TeX.
+ It doesn't try to preserve Word formatting; its goal is to extract plain
+ text and allow you to read it (and, probably, reformat it with TeX).
  .
  This package suggests tk because it also includes wordview, an
  optional Tk-based GUI for catdoc.  The MIME config provided in this
