diff -ur debroster-1.12.orig/dr.sh debroster-1.12/dr.sh
--- debroster-1.12.orig/dr.sh	2003-11-11 16:45:38.000000000 +0100
+++ debroster-1.12/dr.sh	2006-05-31 12:31:16.000000000 +0200
@@ -22,6 +22,6 @@
 scroll "${names[@]}"
 }
 
-</var/lib/dpkg/available sed -ne 's/^Maintainer: //p' | sort -u -k1.1,2.0 | build_arg_list
+</var/lib/dpkg/available sed -ne 's/^Maintainer: //p' | sort -u | build_arg_list
 
 exit 0
