--- /usr/share/debhelper/dh_make/debiank/rules	2007-01-07 23:27:42.000000000 +0100
+++ dh_make/debiank/rules	2007-08-28 21:48:56.000000000 +0200
@@ -125,7 +125,7 @@
 	rm -f build-arch-stamp build-indep-stamp #CONFIGURE_STAMP#
 
 	# Add here commands to clean up after the build process.
-	-#CLEAN#
+	#CLEAN#
 
 	dh_clean
 
--- /usr/share/debhelper/dh_make/debianl/rules	2006-06-02 13:03:28.000000000 +0200
+++ dh_make/debianl/rules	2007-08-28 21:49:01.000000000 +0200
@@ -46,7 +46,7 @@
 	rm -f build-stamp #CONFIGURE_STAMP#
 
 	# Add here commands to clean up after the build process.
-	-#CLEAN#
+	#CLEAN#
 
 	dh_clean#PRESERVE#
 
--- /usr/share/debhelper/dh_make/debianm/rules	2006-06-02 13:03:27.000000000 +0200
+++ dh_make/debianm/rules	2007-08-28 21:49:13.000000000 +0200
@@ -51,7 +51,7 @@
 	rm -f build-arch-stamp build-indep-stamp #CONFIGURE-STAMP#
 
 	# Add here commands to clean up after the build process.
-	-#CLEAN#
+	#CLEAN#
 
 	dh_clean#PRESERVE#
 
--- /usr/share/debhelper/dh_make/debians/rules	2006-06-02 13:03:28.000000000 +0200
+++ dh_make/debians/rules	2007-08-28 21:48:21.000000000 +0200
@@ -39,7 +39,7 @@
 	rm -f build-stamp #CONFIGURE_STAMP#
 
 	# Add here commands to clean up after the build process.
-	-#CLEAN#
+	#CLEAN#
 
 	dh_clean#PRESERVE#
 
