--- courier-pop	2004-10-14 21:21:46.000000000 +0200
+++ courier-pop.neu	2004-10-16 12:10:58.000000000 +0200
@@ -57,7 +57,7 @@
 
     /usr/bin/env - /bin/sh -c " set -a; \
 		. ${sysconfdir}/pop3d; \
-		if [ "SSLCONFIG" ]; then . ${sysconfdir}/pop3d-ssl; fi; \
+		if [ "$SSLCONFIG" ]; then . ${sysconfdir}/pop3d-ssl; fi; \
 		POP3_STARTTLS=$POP3_STARTTLS; export POP3_STARTTLS; \
 		TLS_PROTOCOL=$TLS_STARTTLS_PROTOCOL; \
 	    $TCPD -pid=$PIDFILE -stderrlogger=${sbindir}/courierlogger \
