Index: resources.dbk
===================================================================
--- resources.dbk	(revision 7057)
+++ resources.dbk	(working copy)
@@ -529,29 +530,28 @@
 popular, the kernel was ported to other architectures, too.
 </para>
 <para>
-The Linux 2.0 kernel supports Intel x86, DEC Alpha, SPARC, Motorola 680x0 (like
-Atari, Amiga and Macintoshes), MIPS, and PowerPC.  The Linux 2.2 kernel
-supports even more architectures, including ARM and UltraSPARC.  Since Linux
-supports these platforms, Debian decided that it should, too.  Therefore,
-Debian has ports underway; in fact, we also have ports underway to non-Linux
-kernels.  Aside from <literal>i386</literal> (our name for Intel x86), there
-is <literal>m68k</literal>, <literal>alpha</literal>,
-<literal>powerpc</literal>, <literal>sparc</literal>,
-<literal>hurd-i386</literal>, <literal>arm</literal>,
-<literal>ia64</literal>, <literal>hppa</literal>,
-<literal>s390</literal>, <literal>mips</literal>,
-<literal>mipsel</literal> and <literal>sh</literal> as of this writing.
+As the Linux kernel began to support more architectures, Debian decided that
+it should, too.  Therefore, Debian has ports underway; in fact, we also have
+ports underway to non-Linux kernels, including <literal>hurd</literal> and
+<literal>kfreebsd</literal>. Aside from <literal>i386</literal> (our
+name for 32bit Intel x86), there are a dozen other architectures Debian runs
+on, including <literal>m68k</literal>, <literal>amd64</literal>,
+<literal>armel</literal>, <literal>powerpc</literal>,
+<literal>sparc</literal>, <literal>s390</literal>,
+<literal>mipsel</literal> and <literal>sh</literal> at this writing.
 </para>
 <para>
-&debian-formal; 1.3 is only available as <literal>i386</literal>.  Debian
+&debian-formal; 1.3 was only available as <literal>i386</literal>.  Debian
 2.0 shipped for <literal>i386</literal> and <literal>m68k</literal>
-architectures.  Debian 2.1 ships for the <literal>i386</literal>,
+architectures.  Debian 2.1 shipped for the <literal>i386</literal>,
 <literal>m68k</literal>, <literal>alpha</literal>, and
-<literal>sparc</literal> architectures.  Debian 2.2 added support for the
-<literal>powerpc</literal> and <literal>arm</literal> architectures.
-Debian 3.0 added support of five new architectures: <literal>ia64</literal>,
-<literal>hppa</literal>, <literal>s390</literal>, <literal>mips</literal>
-and <literal>mipsel</literal>.
+<literal>sparc</literal> architectures.  Since then Debian has grown hugely.
+Debian 5 supports a total of twelve architectures, including
+<literal>m68k</literal>, <literal>amd64</literal>,
+<literal>armel</literal>, <literal>powerpc</literal>,
+<literal>sparc</literal>, <literal>s390</literal>,
+<literal>mipsel</literal> and <literal>sh</literal>
+
 </para>
 <para>
 Information for developers and users about the specific ports are available at
Index: common.ent
===================================================================
--- common.ent	(revision 7057)
+++ common.ent	(working copy)
@@ -168,27 +168,27 @@
 
 <!ENTITY sample-dist-dirtree "<screen>dists/stable/main/
 dists/stable/main/binary-i386/
-dists/stable/main/binary-m68k/
-dists/stable/main/binary-alpha/
+dists/stable/main/binary-mipsel/
+dists/stable/main/binary-amd64/
      ...
 dists/stable/main/source/
      ...
 dists/stable/main/disks-i386/
-dists/stable/main/disks-m68k/
-dists/stable/main/disks-alpha/
+dists/stable/main/disks-mipsel/
+dists/stable/main/disks-amd64/
      ...
 
 dists/stable/contrib/
 dists/stable/contrib/binary-i386/
-dists/stable/contrib/binary-m68k/
-dists/stable/contrib/binary-alpha/
+dists/stable/contrib/binary-mipsel/
+dists/stable/contrib/binary-amd64/
      ...
 dists/stable/contrib/source/
 
 dists/stable/non-free/
 dists/stable/non-free/binary-i386/
-dists/stable/non-free/binary-m68k/
-dists/stable/non-free/binary-alpha/
+dists/stable/non-free/binary-mipsel/
+dists/stable/non-free/binary-amd64/
      ...
 dists/stable/non-free/source/
 
