--- clementine-1.2.3+dfsg.orig/debian/rules	2015-06-30 11:27:53.631504257 +1000
+++ clementine-1.2.3+dfsg/debian/rules	2015-06-30 11:15:50.581931068 +1000
@@ -25,6 +25,11 @@

 override_dh_auto_clean:
 	rm -f debian/clementine.1
+	rm -f dist/Info.plist
+	rm -f dist/clementine.spec
+	rm -f dist/maketarball.sh
+	rm -f dist/windows/clementine.nsi
+	rm -f src/translations/translations.pot
 	dh_auto_clean

 override_dh_installchangelogs:
