diff --git a/dh_installdocs b/dh_installdocs
index d7bb24f..935d724 100755
--- a/dh_installdocs
+++ b/dh_installdocs
@@ -27,7 +27,7 @@ documentation into F<usr/share/doc/package> in package build directories.
 
 List documentation files to be installed into I<package>.
 
-In compat 11 (or later), these will be installed info
+In compat 11 (or later), these will be installed into
 F</usr/share/doc/mainpackage>.  Previously it would be
 F</usr/share/doc/package>.
 
