diff -Nru angband-3.3.2/debian/changelog angband-3.3.2/debian/changelog
--- angband-3.3.2/debian/changelog	2011-11-04 21:36:57.000000000 +0100
+++ angband-3.3.2/debian/changelog	2012-08-22 19:12:23.000000000 +0200
@@ -1,3 +1,14 @@
+angband (1:3.3.2-2.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Fix "removes files that were installed by another package:
+    /usr/share/angband/*":
+    don't remove /usr/share/angband from angband's postrm; the directory
+    belongs to angband-data.
+    (Closes: #684180)
+
+ -- gregor herrmann <gregoa@debian.org>  Wed, 22 Aug 2012 19:11:55 +0200
+
 angband (1:3.3.2-2) unstable; urgency=low
 
   * Fixed conflict between angband-data and older versions of angband
diff -Nru angband-3.3.2/debian/postrm angband-3.3.2/debian/postrm
--- angband-3.3.2/debian/postrm	2011-10-25 21:36:59.000000000 +0200
+++ angband-3.3.2/debian/postrm	2012-08-22 19:11:52.000000000 +0200
@@ -122,10 +122,6 @@
 	    rm -rf /etc/angband/
 	fi
 
-	if [ -d /usr/share/angband ]; then
-	    rm -rf /usr/share/angband
-        fi
-
     ;;
   disappear)
     if test "$2" != overwriter; then
