From 01898a9449456ba7c17ad578a915e4688f81979c Mon Sep 17 00:00:00 2001
From: Stefano Rivera <stefanor@debian.org>
Date: Tue, 26 Jul 2016 16:44:27 -0700
Subject: [PATCH 1/2] Wrap long line

---
 support/ruby/generate | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/support/ruby/generate b/support/ruby/generate
index 4bff179..a5898d0 100755
--- a/support/ruby/generate
+++ b/support/ruby/generate
@@ -4,7 +4,12 @@ set -e
 
 # remove debhelper and gem2deb
 if [ -f debian/control ]; then
-  builddeps=$(grep-dctrl -n -s Build-Depends -F Build-Depends -r . debian/control | grep -v '^\s*#' |sed -e 's/,\s*/\n/g; s/^\s*//' | sed '/^\s*$/ d; /debhelper/ d; /gem2deb/ d; s/\n/, /' | tr '\n' ', ')
+  builddeps=$(
+    grep-dctrl -n -s Build-Depends -F Build-Depends -r . debian/control \
+    | grep -v '^\s*#' \
+    | sed -e 's/,\s*/\n/g; s/^\s*//' \
+    | sed '/^\s*$/ d; /debhelper/ d; /gem2deb/ d; s/\n/, /' \
+    | tr '\n' ', ')
 else
   builddeps=''
 fi
-- 
2.8.1

