From aff87aeff2b501b9126a30778ecf3f37e6601dbf Mon Sep 17 00:00:00 2001
From: Stefano Rivera <stefanor@debian.org>
Date: Tue, 26 Jul 2016 16:45:11 -0700
Subject: [PATCH 2/2] Strip build profile restritions

---
 support/ruby/generate | 1 +
 1 file changed, 1 insertion(+)

diff --git a/support/ruby/generate b/support/ruby/generate
index a5898d0..ab6067a 100755
--- a/support/ruby/generate
+++ b/support/ruby/generate
@@ -8,6 +8,7 @@ if [ -f debian/control ]; then
     grep-dctrl -n -s Build-Depends -F Build-Depends -r . debian/control \
     | grep -v '^\s*#' \
     | sed -e 's/,\s*/\n/g; s/^\s*//' \
+    | sed -e 's/\s*<[^)]*>\s*$//' \
     | sed '/^\s*$/ d; /debhelper/ d; /gem2deb/ d; s/\n/, /' \
     | tr '\n' ', ')
 else
-- 
2.8.1

