diff -ur debootstrap-1.0.10/scripts/debian/sid debootstrap-1.0.10.new/scripts/debian/sid
--- debootstrap-1.0.10/scripts/debian/sid	2008-06-02 10:43:20.000000000 +0000
+++ debootstrap-1.0.10.new/scripts/debian/sid	2008-12-26 22:03:24.000000000 +0000
@@ -24,7 +24,7 @@
     elif doing_variant buildd; then
       base="$(get_debs Build-Essential: yes)"
     elif doing_variant minbase; then
-      base="apt"
+      base="apt debian-archive-keyring gnupg gpgv libbz2-1.0 libreadline5 readline-common libusb-0.1-4"
     fi
 
     if doing_variant fakechroot; then 
