From 74b118609c5715d6ec913c3a610cbaedc936693d Mon Sep 17 00:00:00 2001
From: Cyril Brulebois <kibi@debian.org>
Date: Fri, 19 Jan 2018 02:23:44 +0100
Subject: [PATCH 2/2] Enable parallel building.

---
 debian/changelog | 1 +
 debian/rules     | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/debian/changelog b/debian/changelog
index 0afc8f8..0ffbc97 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,6 +1,7 @@
 pcre2 (10.22-6) UNRELEASED; urgency=medium
 
   * Add a libpcre2-8-0-udeb, needed by vte2.91 (Closes: #887674)
+  * Enable parallel building.
 
  -- Cyril Brulebois <kibi@debian.org>  Fri, 19 Jan 2018 01:00:12 +0000
 
diff --git a/debian/rules b/debian/rules
index 07bd4b3..614ac4a 100755
--- a/debian/rules
+++ b/debian/rules
@@ -20,7 +20,7 @@ deb_maint_conf_args +=--disable-jit
 endif
 
 %:
-	dh $@
+	dh $@ --parallel
 
 override_dh_auto_configure:
 	dh_auto_configure -- $(deb_maint_conf_args)
-- 
2.11.0

