From 282a804ebf108c49c3f205cb23590a326b2dda2b Mon Sep 17 00:00:00 2001
From: Ansgar Burchardt <ansgar@debian.org>
Date: Thu, 8 Sep 2016 17:40:40 +0200
Subject: [PATCH 6/6] Add changelog entry for merged-/usr support.

---
 debian/changelog | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/debian/changelog b/debian/changelog
index 96a1dc9..9b24038 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -2,6 +2,8 @@ debootstrap (1.0.83) UNRELEASED; urgency=medium
 
   * functions: Validate that the requested suite is listed in the
     Release file's Suite or Codename field. (Closes: #837075)
+  * Add support for merged-/usr, enabled by default for stretch and later.
+    (Closes: #810301)
 
  -- Ansgar Burchardt <ansgar@debian.org>  Thu, 08 Sep 2016 17:26:53 +0200
 
-- 
2.9.3

