--- /home/global/erik/compiling/ori/caja-extensions-1.8.0/sendto/plugins/emailclient/emailclient.c	2014-03-01 13:13:18.000000000 +0100
+++ emailclient.c	2016-06-22 11:37:48.000000000 +0200
@@ -95,23 +95,20 @@
 		/* Find what the default mailer is */
 		if (strstr (mail_cmd, "balsa"))
 			type = MAILER_BALSA;
-		else if (strstr (mail_cmd, "thunder") || strstr (mail_cmd, "seamonkey")) {
-			char **strv;
-
+		else if (strstr (mail_cmd, "thunder") || strstr (mail_cmd, "seamonkey")
+			 || strstr(mail_cmd, "icedove") )
 			type = MAILER_THUNDERBIRD;
-
-			/* Thunderbird sucks, see
-			 * https://bugzilla.gnome.org/show_bug.cgi?id=614222 */
-			strv = g_strsplit (mail_cmd, " ", -1);
-			g_free (mail_cmd);
-			mail_cmd = g_strdup_printf ("%s %%s", strv[0]);
-			g_strfreev (strv);
-		} else if (strstr (mail_cmd, "sylpheed") || strstr (mail_cmd, "claws"))
+		else if (strstr (mail_cmd, "sylpheed") || strstr (mail_cmd, "claws"))
 			type = MAILER_SYLPHEED;
 		else if (strstr (mail_cmd, "anjal"))
 			type = MAILER_EVO;
 	}
-
+	
+	char **strv = g_strsplit (mail_cmd, " ", -1);
+	g_free (mail_cmd);
+	mail_cmd = g_strdup_printf ("%s %%s", strv[0]);
+	g_strfreev (strv);
+	
 	if (mail_cmd == NULL)
 		return FALSE;
 
