diff -u emms-player-simple.el.orig emms-player-simple.el
--- emms-player-simple.el.orig	2007-05-09 19:21:45.000000000 +0400
+++ emms-player-simple.el	2008-02-22 03:45:35.000000000 +0300
@@ -98,7 +98,8 @@
        "Return non-nil when we can play this track."
        (and (executable-find ,command-name)
             (memq (emms-track-type track) ,types)
-            (string-match ,regex (emms-track-name track)))))))
+            (string-match (emms-player-get ,player-name 'regex)
+                          (emms-track-name track)))))))
 
 ;; Global variables
 (defvar emms-player-simple-process-name "emms-player-simple-process"
@@ -179,7 +180,7 @@
                              ".stm" ".stx" ".ult" ".apun" ".xm" ".mod")
   "mikmod" "-q" "-p" "1" "-X")
 (define-emms-simple-player alsaplayer '(file url)
-  (emms-player-simple-regexp ".ogg" ".mp3" ".wav" ".flac" ".pls" ".m3u" "http://")
+  (emms-player-simple-regexp "ogg" "mp3" "wav" "flac" "pls" "m3u")
   "alsaplayer" "--quiet" "--nosave" "\"--interface text\"")
 
 (provide 'emms-player-simple)
