--- unpacked/usr/lib/python3/dist-packages/azure/cli/command_modules/monitor/actions.py	2020-05-29 09:56:48.000000000 +0200
+++ /usr/lib/python3/dist-packages/azure/cli/command_modules/monitor/actions.py	2020-07-10 16:07:49.530265759 +0200
@@ -4,7 +4,6 @@
 # --------------------------------------------------------------------------------------------
 
 import argparse
-import antlr4
 
 from azure.cli.command_modules.monitor.util import (
     get_aggregation_map, get_operator_map, get_autoscale_operator_map,
@@ -92,6 +91,7 @@
 class MetricAlertConditionAction(argparse._AppendAction):
 
     def __call__(self, parser, namespace, values, option_string=None):
+        import antlr4
         from azure.cli.command_modules.monitor.grammar import (
             MetricAlertConditionLexer, MetricAlertConditionParser, MetricAlertConditionValidator)
 
