=== modified file 'methods/rred.cc'
--- methods/rred.cc	2006-05-17 15:58:24 +0000
+++ methods/rred.cc	2009-09-08 14:06:29 +0000
@@ -174,7 +174,7 @@
          hash);
    
    /* read the rest from infile */
-   if (result > 0) {
+   if (result >= 0) {
       while (fgets(buffer, BUF_SIZE, in_file) != NULL) {
          written = fwrite(buffer, 1, strlen(buffer), out_file);
          hash->Add((unsigned char*)buffer, written);

