diff -Nru ntfs-3g-2013.1.13/debian/control ntfs-3g-2013.1.13/debian/control
--- ntfs-3g-2013.1.13/debian/control	2013-02-14 16:23:20.000000000 +0000
+++ ntfs-3g-2013.1.13/debian/control	2013-02-16 03:58:04.000000000 +0000
@@ -14,6 +14,7 @@
 Architecture: linux-any kfreebsd-any
 Pre-Depends: multiarch-support, fuse [linux-any], fuse4bsd [kfreebsd-any]
 Depends: ${misc:Depends}, ${shlibs:Depends}
+Provides: ${ntfs:Provides}
 Description: read/write NTFS driver for FUSE
  NTFS-3G uses FUSE (Filesystem in Userspace) to provide support for the NTFS
  filesystem used by Microsoft Windows.
@@ -46,4 +47,5 @@
 Section: debian-installer
 Architecture: linux-any
 Depends: ${misc:Depends}, ${shlibs:Depends}, fuse-udeb
+Provides: ${ntfs-udeb:Provides}
 Description: read/write NTFS driver for FUSE
diff -Nru ntfs-3g-2013.1.13/debian/rules ntfs-3g-2013.1.13/debian/rules
--- ntfs-3g-2013.1.13/debian/rules	2013-02-14 16:23:38.000000000 +0000
+++ ntfs-3g-2013.1.13/debian/rules	2013-02-16 05:42:50.000000000 +0000
@@ -11,6 +11,9 @@
 CONFIGURE_FLAGS = --enable-posix-acls
 endif
 
+# Or shall one use objdump?
+SONAMEMAJ=$(shell find debian/ntfs-3g -name 'libntfs-3g.so*' -type f | cut -d. -f 3)
+
 upstream:
 	lynx -dump http://jp-andre.pagesperso-orange.fr/changelog.html > debian/local/changelog
 
@@ -79,14 +82,13 @@
 
 override_dh_link:
 	rm -rf debian/ntfs-3g-dev/usr/share/doc
-
-	# correcting symlink target
-	dh_link -pntfs-3g-dev lib/$(DEB_HOST_MULTIARCH)/$$(basename $$(readlink debian/tmp/usr/lib/$(DEB_HOST_MULTIARCH)/libntfs-3g.so)) usr/lib/$(DEB_HOST_MULTIARCH)/libntfs-3g.so
-
-	dh_link --remaining-packages
+	dh_link
 
 override_dh_makeshlibs:
-	dh_makeshlibs --add-udeb=ntfs-3g-udeb
+	dh_makeshlibs --add-udeb=ntfs-3g-udeb -V'libntfs-3g$(SONAMEMAJ)'
 
 override_dh_strip:
 	dh_strip --dbg-package=ntfs-3g-dbg
+
+override_dh_gencontrol:
+	dh_gencontrol -- -Vntfs:Provides="libntfs-3g$(SONAMEMAJ)" -Vntfs-udeb:Provides="libntfs-3g-udeb$(SONAMEMAJ)"
