--- /etc/apparmor.d/usr.sbin.cupsd.orig	2014-09-30 13:04:05.000000000 +0200
+++ /etc/apparmor.d/usr.sbin.cupsd	2014-10-01 21:03:01.191242269 +0200
@@ -141,7 +141,6 @@
   # silence noise
   deny /etc/udev/udev.conf r,
 
-  signal (receive, send) peer=third_party,
   profile third_party {
     # third party backends, filters, and drivers get relatively no restrictions
     # as they often need high privileges, are unpredictable or otherwise beyond
@@ -150,10 +149,6 @@
     capability,
     audit deny capability mac_admin,
     network,
-    dbus,
-    signal,
-    ptrace,
-    unix,
   }
 
   # Site-specific additions and overrides. See local/README for details.
