--- bbdb-2.35.cvs20060204.orig/lisp/bbdb.el
+++ bbdb-2.35.cvs20060204/lisp/bbdb.el
@@ -2452,7 +2452,8 @@
     (insert-before-markers (format ";; -*-coding: %s;-*-\n"
                                    bbdb-file-coding-system)))
   (setq bbdb-modified-p nil
-        bbdb-changed-records nil)
+        bbdb-changed-records nil
+	buffer-file-coding-system bbdb-file-coding-system)
   (let ((buf (get-buffer bbdb-buffer-name)))
     (when buf
       (with-current-buffer buf
