diff -r 2fc5e36dc340 -r 9277b3f02cab src/pkg_grouppolicy.cc
--- a/src/pkg_grouppolicy.cc	Mon Apr 07 09:11:07 2008 -0700
+++ b/src/pkg_grouppolicy.cc	Mon Apr 07 18:28:19 2008 -0700
@@ -262,7 +262,8 @@ void pkg_grouppolicy_section::add_packag
       section=_("virtual");
       may_passthrough = true;
     }
-  else if(!pkg.VersionList().Section())
+  else if(!pkg.VersionList().Section() ||
+	  (*pkg.VersionList().Section()) == '\0')
     {
       section=_("Unknown");
       may_passthrough = true;
