commit 0e50f2ba851d216c801029a27c7fda9e6ff15160
Author: Jan Wagner <waja@cyconet.org>
Date:   Sun Apr 25 23:53:28 2021 +0200

    Fix #986286

diff --git a/debian/changelog b/debian/changelog
index 719be1d..11ec0b5 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,12 @@
+diaspora-installer (0.7.14.0+debian2+nmu2) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Adjust diaspora-download.sh to modify Gemfile.lock so mimemagic (0.3.10)
+    is used (beside nokogiri (~> 1) and rake), upstream change 79133df
+    (Closes: #986286)
+
+ -- Jan Wagner <waja@cyconet.org>  Sun, 25 Apr 2021 23:33:26 +0200
+
 diaspora-installer (0.7.14.0+debian2+nmu1) unstable; urgency=medium
 
   * Non-maintainer upload
diff --git a/diaspora-download.sh b/diaspora-download.sh
index 2318c4a..ffa1ff2 100755
--- a/diaspora-download.sh
+++ b/diaspora-download.sh
@@ -43,7 +43,7 @@ cp -r  ${diaspora_cache}/${diaspora_archive}/public/* ${diaspora_user_home}/publ
 chown -R ${diaspora_user}: ${diaspora_user_home}/public
 chown -R ${diaspora_user}: ${diaspora_user_home}/app-assets
 # Remove BUNDLED WITH from Gemfile.lock to allow using packaged bundler
-head -n -3 ${diaspora_cache}/${diaspora_archive}/Gemfile.lock > ${diaspora_home}/Gemfile.lock
+head -n -3 ${diaspora_cache}/${diaspora_archive}/Gemfile.lock | sed "s/    mimemagic (0.3.5)/    mimemagic (0.3.10)\n      nokogiri (~> 1)\n      rake/" > ${diaspora_home}/Gemfile.lock
 
 echo "Copying source tarball to ${diaspora_user_home}/public..."
 cp -f ${diaspora_cache}/diaspora-${diaspora_version}.tar.gz ${diaspora_user_home}/public/source.tar.gz
