Index: approx-5.7/approx.ml
===================================================================
--- approx-5.7.orig/approx.ml
+++ approx-5.7/approx.ml
@@ -272,7 +272,7 @@ let with_pair rex str proc =
   | [| a; b |] -> proc (a, b)
   | _ -> assert false
 
-let status_re = Pcre.regexp "^HTTP/\\d+\\.\\d+\\s+(\\d{3})(?:\\s+(.*?)\\s*)?$"
+let status_re = Pcre.regexp "^HTTP/\\d+(?:\\.\\d+)?\\s+(\\d{3})(?:\\s+(.*?)\\s*)?$"
 let header_re = Pcre.regexp "^(.*?):\\s*(.*?)\\s*$"
 
 let process_header resp str =
