--- cryptdisks.functions.orig	2008-11-29 13:17:46.000000000 +0100
+++ cryptdisks.functions	2008-11-29 13:25:08.000000000 +0100
@@ -294,9 +294,9 @@
 		key="-"
 	elif [ -z "$key" ]; then
 		# no keyscript, no key => password
-		keyscriptarg="Enter passphrase to unlock the disk $src ($dst): "
+		keyscriptarg="Unlocking $src ($dst): "
 		key="-"
-		KEYSCRIPT="/lib/cryptsetup/askpass"
+		KEYSCRIPT="INTERACTIVE"
 	elif [ "$key" != "${key%/dev/*}" ]; then
 		# no keyscript, device key => special treatment
 		keyscriptarg=""
@@ -313,8 +313,13 @@
 
 	while [ "$tried" -lt "$TRIES" ]; do
 		if [ -n "$KEYSCRIPT" ]; then
-			if "$KEYSCRIPT" "$keyscriptarg" | cryptsetup $PARAMS luksOpen "$src" "$dst"; then
-				break
+			if [ "$KEYSCRIPT" = "INTERACTIVE" ] ; then
+				echo "$keyscriptarg"
+				if cryptsetup $PARAMS luksOpen "$src" "$dst"; then
+					break
+				fi
+			elif "$KEYSCRIPT" "$keyscriptarg" | cryptsetup $PARAMS luksOpen "$src" "$dst"; then
+					break
 			fi
 		else
 			if cryptsetup $PARAMS luksOpen "$src" "$dst"; then
@@ -360,9 +365,9 @@
 		key="-"
 	elif [ -z "$key" ]; then
 		# no keyscript, no key => password
-		keyscriptarg="Enter passphrase to unlock the disk $src ($dst): "
+		keyscriptarg="Unlocking $src ($dst): "
 		key="-"
-		KEYSCRIPT="/lib/cryptsetup/askpass"
+		KEYSCRIPT="INTERACTIVE"
 	else
 		# no keyscript, key => file input
 		keyscriptarg=""
@@ -374,7 +379,12 @@
 
 	while [ "$tried" -lt "$TRIES" ]; do
 		if [ -n "$KEYSCRIPT" ]; then
-			"$KEYSCRIPT" "$keyscriptarg" | cryptsetup $PARAMS create "$dst" "$src"
+			if [ "$KEYSCRIPT" = "INTERACTIVE" ] ; then
+				echo "$keyscriptarg"
+				cryptsetup $PARAMS create "$dst" "$src"
+			else
+				"$KEYSCRIPT" "$keyscriptarg" | cryptsetup $PARAMS create "$dst" "$src"
+			fi
 		else
 			cryptsetup $PARAMS create "$dst" "$src"
 		fi
