diff -Nru /tmp/JJ0iLQmLt4/debian-cd-3.0.2/debian/control /tmp/jSfJrNPkVh/debian-cd-3.0.2/debian/control
--- /tmp/JJ0iLQmLt4/debian-cd-3.0.2/debian/control	2006-12-23 04:02:37.000000000 +0100
+++ /tmp/jSfJrNPkVh/debian-cd-3.0.2/debian/control	2007-08-12 10:07:28.000000000 +0200
@@ -4,13 +4,13 @@
 Maintainer: Debian CD Group <debian-cd@lists.debian.org>
 Uploaders: Raphael Hertzog <hertzog@debian.org>, Santiago Garcia Mantinan <manty@debian.org>, Steve McIntyre <93sam@debian.org>
 Build-Depends: debhelper (>= 4)
-Build-Depends-Indep: tofrodos | sysutils
+Build-Depends-Indep: tofrodos
 Standards-Version: 3.6.1.0
 XS-Vcs-Svn: svn://svn.debian.org/debian-cd/trunk
 
 Package: debian-cd
 Architecture: all
-Depends: perl, cpp, libdigest-md5-perl, tofrodos | sysutils, apt (>= 0.3.11.1), make, mkisofs | genisoimage, lynx, grep-dctrl, bc, libcompress-zlib-perl
+Depends: perl, cpp, libdigest-md5-perl, tofrodos, apt (>= 0.3.11.1), make, mkisofs | genisoimage, lynx, grep-dctrl, bc, libcompress-zlib-perl
 Recommends: hfsutils, netpbm, syslinux
 Description: Tools for building (Official) Debian CD set
  Debian-cd is the official tool for building Debian CD set since the potato
