--- apt-listchanges.orig/apt-listchanges.py	2007-06-18 22:20:05.000000000 +0100
+++ apt-listchanges/apt-listchanges.py	2007-07-04 04:06:40.000000000 +0100
@@ -34,7 +34,10 @@
 sys.path += ['./apt-listchanges', '/usr/share/apt-listchanges']
 import apt_listchanges, DebianFiles, ALCConfig
 
-locale.setlocale(locale.LC_ALL, '')
+try:
+    locale.setlocale(locale.LC_ALL, '')
+except locale.Error:
+    sys.stderr.write("Can't set locale; make sure $LC_* and $LANG are correct!\n")
 def _(x):
     try:
         return gettext.translation('apt-listchanges').lgettext(x)
