Index: debian/changelog
===================================================================
--- debian/changelog	(revision 281)
+++ debian/changelog	(working copy)
@@ -7,6 +7,8 @@
     files.  Closes: bug#489430, thanks to Martin Koeppe.
   * Drop bogus phrase about ordered relation between debhelper.mk and
     perl.mk in documentation.
+  * Run dh_pycentral/dh_pysupport before dh_installinit.  Closes:
+    bug#494288, thanks to Guido Günther.
 
  -- Jonas Smedegaard <dr@jones.dk>  Thu, 17 Apr 2008 21:14:56 +0200
 
Index: 1/class/python-distutils.mk.in
===================================================================
--- 1/class/python-distutils.mk.in	(revision 281)
+++ 1/class/python-distutils.mk.in	(working copy)
@@ -140,7 +140,8 @@
 
 DEB_DH_PYTHON_ARGS = $(addprefix -V ,$(DEB_PYTHON_COMPILE_VERSION)) $(DEB_DH_PYTHON_ARGS_ALL) $(DEB_DH_PYTHON_ARGS_$(cdbs_curpkg))
 
-$(patsubst %,binary-install/%,$(DEB_PACKAGES)) :: binary-install/%:
+$(patsubst %,binary-install/%,$(DEB_PACKAGES)) :: binary-install/%: binary-install-python:/%
+$(patsubst %,binary-install-python/%,$(DEB_PACKAGES)) :: binary-install-python:/%
 	dh_python -p$(cdbs_curpkg) $(DEB_DH_PYTHON_ARGS)
 endif
 
@@ -235,7 +236,8 @@
 
 DEB_DH_PYTHON_ARGS = $(DEB_DH_PYTHON_ARGS_ALL) $(DEB_DH_PYTHON_ARGS_$(cdbs_curpkg)) $(DEB_PYTHON_PRIVATE_MODULES_DIRS)
 
-$(patsubst %,binary-install/%,$(DEB_PACKAGES)) :: binary-install/%:
+$(patsubst %,binary-install/%,$(DEB_PACKAGES)) :: binary-install/%: binary-install-python/%
+$(patsubst %,binary-install-python/%,$(DEB_PACKAGES)) :: binary-install-python/%:
 ifeq (pysupport, $(DEB_PYTHON_SYSTEM))
 	dh_pysupport -p$(cdbs_curpkg) $(DEB_PYTHON_PRIVATE_MODULES_DIRS)
 else
