Index: debian-copyright.el
===================================================================
RCS file: /cvsroot/pkg-goodies-el/emacs-goodies-el/elisp/dpkg-dev-el/debian-copyright.el,v
retrieving revision 1.4
diff -u -u -r1.4 debian-copyright.el
--- debian-copyright.el	4 Nov 2003 02:07:56 -0000	1.4
+++ debian-copyright.el	2 Jul 2010 19:35:29 -0000
@@ -21,8 +21,6 @@
 (require 'debian-changelog-mode)
 
 ;;; Code:
-(add-to-list 'auto-mode-alist '("debian/copyright$" . debian-copyright-mode))
-(add-to-list 'auto-mode-alist '("^/usr/share/doc/.*/copyright" . debian-copyright-mode))
 
 (defgroup debian-copyright nil "Debian copyright mode"
   :group 'tools
@@ -84,11 +82,13 @@
           ))
   (run-hooks 'debian-copyright-mode-hook))
 
-(add-to-list 'auto-mode-alist '("debian/.*copyright$" . debian-copyright-mode))
+
+;;;###autoload
+(add-to-list 'auto-mode-alist
+             '("debian/.*copyright\\'" . debian-copyright-mode))
+;;;###autoload
 (add-to-list 'auto-mode-alist
-             '("^/usr/share/doc/.*/copyright" . debian-copyright-mode))
-;;;###autoload(add-to-list 'auto-mode-alist '("debian/.*copyright$" . debian-copyright-mode))
-;;;###autoload(add-to-list 'auto-mode-alist '("^/usr/share/doc/.*/copyright" . debian-copyright-mode))
+             '("\\`/usr/share/doc/.*/copyright" . debian-copyright-mode))
 
 (run-hooks 'debian-copyright-mode-load-hook)
 
