Index: tools/start_new_disc
===================================================================
--- tools/start_new_disc	(revision 1372)
+++ tools/start_new_disc	(working copy)
@@ -124,7 +124,7 @@
 	echo "  Adding .disk/base_components"
 	echo "main" > $CDDIR/.disk/base_components
 	if [ -n "$LOCAL" ] ; then
-		echo "local" > $CDDIR/.disk/base_components
+		echo "local" >> $CDDIR/.disk/base_components
 	fi
 
 	echo "  Adding .disk/cd_type"
