--- postinst.old	2010-08-11 21:23:04.942155020 +0200
+++ postinst	2010-08-11 21:23:52.402147957 +0200
@@ -137,8 +137,8 @@
     fi
     db_get dokuwiki/system/writeplugins
     if [ "$RET" = "true" ]; then
-        if ! dpkg-statoverride --list /var/lib/dokuwiki/plugins >/dev/null 2>&1; then
-            dpkg-statoverride --update --add root www-data 0775 /var/lib/dokuwiki/plugins
+        if ! dpkg-statoverride --list /var/lib/dokuwiki/lib/plugins >/dev/null 2>&1; then
+            dpkg-statoverride --update --add root www-data 0775 /var/lib/dokuwiki/lib/plugins
         fi
     fi
 }
