--- ddclient.old	2014-04-23 17:02:21.000000000 +0200
+++ ddclient	2014-04-23 17:11:24.000000000 +0200
@@ -1783,14 +1783,21 @@
 ## load_sha1_support
 ######################################################################
 sub load_sha1_support {
-    my $sha1_loaded = eval {require Digest::SHA1};
-    unless ($sha1_loaded) {
-        fatal(<<"EOM");
+    my $sha1_loaded = eval {require Digest::SHA};
+    if ($sha1_loaded) {
+        import  Digest::SHA (qw/sha1_hex/);
+    } else {
+        $sha1_loaded = eval {require Digest::SHA1};
+        if ($sha1_loaded) {
+            import  Digest::SHA1 (qw/sha1_hex/);
+        } else {
+            fatal(<<"EOM");
 Error loading the Perl module Digest::SHA1 needed for freedns update.
-On Debian, the package libdigest-sha1-perl must be installed.
+On Debian Squeeze, the package libdigest-sha1-perl must be installed.
+Newer perl versions provide this as part of Digest:SHA.
 EOM
+        }
     }
-    import  Digest::SHA1 (qw/sha1_hex/);
 }
 ######################################################################
 ## geturl
