--- epiphany-browser-2.29.91.orig/debian/control.in
+++ epiphany-browser-2.29.91/debian/control.in
@@ -78,7 +78,8 @@
  and reliance on external applications for performing external tasks
  (such as reading email). Simplicity should not mean less powerful.
  Standards compliance is achieved on the HTML side by using the
- WebKitGTK+ rendering engine; and on the user interface side by
+ WebKitGTK+ rendering engine (basically the same engine used by Apple
+ Safari and Google Chrome); and on the user interface side by
  closely following the GNOME Human Interface Guidelines (HIG) and by
  close integration with the GNOME desktop.
 
