--- debian-startup.el.old	2007-12-07 15:34:55.000000000 +1100
+++ debian-startup.el	2007-12-07 15:37:15.000000000 +1100
@@ -107,14 +107,11 @@
                    files)))
          
          ;; Finally sort them, and delete duplicates
-         (base-names (debian-unique-strings (sort stripped-names 'string<)))
-         
-         (old-load-path load-path))
+         (base-names (debian-unique-strings (sort stripped-names 'string<))))
 
     ;; Set a new load path with the directories specified in the
     ;; proper order, and first.
-    (let ((new-path (append paths load-path)))
-      (setq load-path new-path)
+      (setq load-path (append paths load-path))
       ;; Now load the files.  "load" will make sure we get the byte
       ;; compiled one first, if any, and will respect load-path's
       ;; ordering.
@@ -124,16 +121,10 @@
              (load file nil)
            (error (message "Error while loading %s" file))))
        base-names)
-      ;; restore the old load-path -- including any new paths added by
-      ;; files loaded in directory traversal.
-      (let ((add-on-package-paths 
-             (delq nil (mapcar
-                        (lambda (item)
-                          (if (not (member item new-path))
-                              item
-                            nil))
-                        load-path))))      
-        (setq load-path (append add-on-package-paths old-load-path))))))
+      ;; remove the `paths' we added to load-path -- but leave anything the
+      ;; files added
+      (dolist (item paths)
+        (setq load-path (remq item load-path)))))
 
 (defun debian-startup (flavor)
 
