diff --git a/developers-reference/tools.dbk b/developers-reference/tools.dbk
index 70514dc..22af3a3 100644
--- a/developers-reference/tools.dbk
+++ b/developers-reference/tools.dbk
@@ -212,17 +212,17 @@ packages.
 The following packages help with the package building process, general driving
 <command>dpkg-buildpackage</command> as well as handling supporting tasks.
 </para>
-<section id="cvs-buildpackage">
-<title><systemitem role="package">cvs-buildpackage</systemitem></title>
+<section id="git-buildpackage">
+<title><systemitem role="package">git-buildpackage</systemitem></title>
 <para>
-<systemitem role="package">cvs-buildpackage</systemitem> provides the
-capability to inject or import Debian source packages into a CVS repository,
-build a Debian package from the CVS repository, and helps in integrating
+<systemitem role="package">git-buildpackage</systemitem> provides the
+capability to inject or import Debian source packages into a Git repository,
+build a Debian package from the Git repository, and helps in integrating
 upstream changes into the repository.
 </para>
 <para>
-These utilities provide an infrastructure to facilitate the use of CVS by
-Debian maintainers.  This allows one to keep separate CVS branches of a package
+These utilities provide an infrastructure to facilitate the use of Git by
+Debian maintainers.  This allows one to keep separate Git branches of a package
 for <literal>stable</literal>, <literal>unstable</literal> and possibly
 <literal>experimental</literal> distributions, along with the other benefits
 of a version control system.
@@ -254,9 +254,8 @@ package's build-dependencies are correct, and to be sure that unnecessary and
 wrong build dependencies will not exist in the resulting package.
 </para>
 <para>
-A related package is <systemitem role="package">pbuilder-uml</systemitem>,
-which goes even further by doing the build within a User Mode Linux
-environment.
+A related package is <systemitem role="package">cowbuilder</systemitem>,
+which speeds up the build process using COW filesystem on any standard Linux filesystem.
 </para>
 </section>
 
