=== modified file 'apt-pkg/contrib/configuration.cc'
--- apt-pkg/contrib/configuration.cc	2007-12-27 17:04:42 +0000
+++ apt-pkg/contrib/configuration.cc	2008-04-01 04:27:56 +0000
@@ -512,7 +512,8 @@
       CurLine++;
       // This should be made to work instead, but this is better than looping
       if (F.fail() && !F.eof())
-         return _error->Error(_("Line %d too long (max %lu)"), CurLine, sizeof(Buffer));
+         return _error->Error(_("%s: Line %d too long (max %lu)"),
+			      FName.c_str(), CurLine, sizeof(Buffer));
 
       _strtabexpand(Buffer,sizeof(Buffer));
       _strstrip(Buffer);

