diff -aur bzip2-1.0.6.orig/debian/rules bzip2-1.0.6/debian/rules
--- bzip2-1.0.6.orig/debian/rules	2011-12-04 09:39:46.000000000 +0100
+++ bzip2-1.0.6/debian/rules	2012-01-08 23:54:52.000000000 +0100
@@ -25,11 +25,10 @@
         DEBSTRIP=strip
 endif
 
-DEBCFLAGS:=
+DEBCFLAGS:=`dpkg-buildflags --get CFLAGS`
+DEBCFLAGS +=`dpkg-buildflags --get CPPFLAGS`
 WGET=wget
-ifneq (,$(findstring debug,$(DEB_BUILD_OPTIONS)))
-DEBCFLAGS += -g
-endif
+
 ifneq (,$(findstring nostrip,$(DEB_BUILD_OPTIONS)))
 DEBSTRIP:=:
 endif
Nur in bzip2-1.0.6/debian: rules~.
diff -aur bzip2-1.0.6.orig/Makefile bzip2-1.0.6/Makefile
--- bzip2-1.0.6.orig/Makefile	2011-12-26 13:38:19.000000000 +0100
+++ bzip2-1.0.6/Makefile	2012-01-08 23:56:06.000000000 +0100
@@ -20,10 +20,10 @@
 CC=gcc
 AR=ar
 RANLIB=ranlib
-LDFLAGS=
+LDFLAGS= `dpkg-buildflags --get LDFLAGS`
 
 BIGFILES=-D_FILE_OFFSET_BITS=64
-CFLAGS=-Wall -Winline -O2 -g $(BIGFILES) $(DEBCFLAGS)
+CFLAGS=-Wall -Winline $(BIGFILES) $(DEBCFLAGS)
 
 # Where you want it installed when you do 'make install'
 PREFIX=/usr/local
Nur in bzip2-1.0.6: Makefile~.
