--- autolog-0.40/autolog.c.orig	2007-05-11 08:31:55.000000000 +0200
+++ autolog-0.40/autolog.c	2007-05-11 08:32:50.000000000 +0200
@@ -483,8 +483,8 @@
 	    idle=c_arr[c_idx].idle;
 	
 /*.. Maybe it is necessary to reduce the max. sleeptime to shortest session. */
-	    if (0< c_arr[c_idx].hard && 0<idle)	 /* but not to zero seconds. */
-		if (2*60*idle < sleep_max)  sleep_max=2*60*idle;
+	    if (0<idle)				 /* but not to zero seconds. */
+		if (60*idle < sleep_max)  sleep_max=60*idle;
 	    c_idx++;
 	}
         fclose(f);