From 6885bbfe670ab48bdbc65a127331a7f380b71a59 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?=C5=81ukasz=20Stelmach?= <steelman@post.pl>
Date: Sun, 15 Nov 2015 21:54:08 +0100
Subject: [PATCH] Rename $FLAVOUR to $KERNEL_FLAVOUR
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Signed-off-by: Łukasz Stelmach <steelman@post.pl>
---
 library.sh | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/library.sh b/library.sh
index d73d880..cfb0f6f 100644
--- a/library.sh
+++ b/library.sh
@@ -349,12 +349,12 @@ kernel_update_list () {
 	cut -d" " -f1 | grep -v "linux-image-2.6" | uniq > "$KERNEL_LIST.unfiltered"
 	kernels=`sort -r "$KERNEL_LIST.unfiltered" | tr '\n' ' ' | sed -e 's/ $//'`
 	for candidate in $kernels; do
-		if [ -n "$FLAVOUR" ]; then
-			if arch_check_usable_kernel "$candidate" "$FLAVOUR"; then
+		if [ -n "$KERNEL_FLAVOUR" ]; then
+			if arch_check_usable_kernel "$candidate" "$KERNEL_FLAVOUR"; then
 				echo "$candidate"
-				info "kernel $candidate usable on $FLAVOUR"
+				info "kernel $candidate usable on $KERNEL_FLAVOUR"
 			else
-				info "kernel $candidate not usable on $FLAVOUR"
+				info "kernel $candidate not usable on $KERNEL_FLAVOUR"
 			fi
 		else
 			info "could not determine kernel flavour"
@@ -404,8 +404,8 @@ pick_kernel () {
 		# Unset seen flag in case we had an incorrect preseeded value.
 		db_fset base-installer/kernel/image seen false || true
 
-		if [ -n "$FLAVOUR" ]; then
-			arch_kernel="$(arch_get_kernel "$FLAVOUR")"
+		if [ -n "$KERNEL_FLAVOUR" ]; then
+			arch_kernel="$(arch_get_kernel "$KERNEL_FLAVOUR")"
 
 			# Hack to support selection of meta packages with a postfix
 			# added to the normal name (for updated kernels in stable).
-- 
2.4.6

