--- a/debian/exim4-config.templates
+++ b/debian/exim4-config.templates
@@ -1,13 +1,13 @@
 Template: exim4/dc_eximconfig_configtype
 Type: select
+__Default: "not on a network; no configuration at this time"
 # Translators beware! the following six strings form a single
 # Choices menu. - Every one of these strings has to fit in a standard
 # 80 characters console, as the fancy screen setup takes up some space
 # try to keep below ~71 characters.
 # DO NOT USE commas (,) in Choices translations otherwise
 # this will break the choices shown to users
-__Choices: internet site; mail is sent and received directly using SMTP, mail sent by smarthost; received via SMTP or fetchmail, mail sent by smarthost; no local mail, local delivery only; not on a network, no configuration at this time
-Default: local delivery only; not on a network
+__Choices: internet site; mail is sent and received directly using SMTP, mail sent by smarthost; received via SMTP or fetchmail, mail sent by smarthost; no local mail; local delivery only; not on a network; no configuration at this time
 _Description: General type of mail configuration:
  Please select the mail server configuration type that best meets your needs.
  .
@@ -29,6 +29,7 @@
 
 Template: exim4/mailname
 Type: string
+Default: empty
 _Description: System mail name:
  The 'mail name' is the domain name used to 'qualify' mail addresses
  without a domain name.
@@ -44,7 +45,7 @@
 
 Template: exim4/dc_other_hostnames
 Type: string
-Default: 
+Default: empty 
 _Description: Other destinations for which mail is accepted:
  Please enter a semicolon-separated list of recipient domains for
  which this machine should consider itself the final destination.
@@ -59,7 +60,7 @@
 
 Template: exim4/dc_relay_domains
 Type: string
-Default: 
+Default:empty 
 _Description: Domains to relay mail for:
  Please enter a semicolon-separated list of recipient domains for
  which this system will relay mail, for example as a fallback MX or
@@ -71,7 +72,7 @@
 
 Template: exim4/dc_relay_nets
 Type: string
-Default: 
+Default: empty 
 _Description: Machines to relay mail for:
  Please enter a semicolon-separated list of IP address ranges for
  which this system will unconditionally relay mail, functioning as a
@@ -92,6 +93,7 @@
 
 Template: exim4/dc_smarthost
 Type: string
+Default: empty
 _Description: IP address or host name of the outgoing smarthost:
  Please enter the IP address or the host name of a mail server that
  this system should use as outgoing smarthost. If the smarthost only
@@ -105,6 +107,7 @@
 
 Template: exim4/dc_postmaster
 Type: string
+Default: empty
 _Description: Root and postmaster mail recipient:
  Mail for the 'postmaster', 'root', and other system accounts needs to
  be redirected to the user account of the actual system administrator.
@@ -121,7 +124,7 @@
 
 Template: exim4/dc_local_interfaces
 Type: string
-Default: notset
+Default: empty
 _Description: IP-addresses to listen on for incoming SMTP connections:
  Please enter a semicolon-separated list of IP addresses. The Exim SMTP
  listener daemon will listen on all IP addresses listed here.
@@ -160,6 +163,7 @@
 
 Template: exim4/use_split_config
 Type: boolean
+Default: false
 _Description: Split configuration into small files?
  The Debian exim4 packages can either use 'unsplit configuration', a
  single monolithic file (/etc/exim4/exim4.conf.template) or 'split
@@ -176,6 +180,7 @@
 
 Template: exim4/hide_mailname
 Type: boolean
+Default: false
 _Description: Hide local mail name in outgoing mail?
  The headers of outgoing mail can be rewritten to make it appear to have been
  generated on a different system. If this option is chosen,
