diff -ur /mnt/donnees/1/debian/trans/dev-ref//resources.dbk dev-ref/resources.dbk
--- /mnt/donnees/1/debian/trans/dev-ref//resources.dbk	2010-06-20 09:22:53.112984447 -0400
+++ dev-ref/resources.dbk	2010-06-20 15:49:08.000000000 -0400
@@ -873,7 +873,7 @@
 maintainer and closes all the bugs marked as fixed by the upload, and the
 auto-builders may start recompiling it.  The package is now publicly accessible
 at <ulink url="&url-incoming;"></ulink> until it is really
-installed in the Debian archive.  This happens only once a day (and is also
+installed in the Debian archive.  This happens four times a day (and is also
 called the `dinstall run' for historical reasons); the package is then removed
 from incoming and installed in the pool along with all the other packages.
 Once all the other updates (generating new <filename>Packages</filename> and
