--- apsfilter-7.2.6/bin/setup_dvips.sh.orig	2008-08-23 09:58:46.000000000 +0000
+++ apsfilter-7.2.6/bin/setup_dvips.sh	2008-08-23 09:59:12.000000000 +0000
@@ -24,11 +24,11 @@
 	mkdir $TMPDIR || exit 1
 	cd $TMPDIR || return
 	rm -f *gf 2> /dev/null
-	mf '\mode='$MODE'; input logo10' >& /dev/null
+	mf '\mode='$MODE'; input logo10' > /dev/null 2>&1
 	RES_X=`echo logo10*gf | sed 's/logo10.//;s/gf//'`
 	RES=RES_X
 	rm -f *gf 2> /dev/null
-	mf '\mode='$MODE'; landscape; input logo10' >& /dev/null
+	mf '\mode='$MODE'; landscape; input logo10' > /dev/null 2>&1
 	RES_Y=`echo logo10*gf | sed 's/logo10.//;s/gf//'`
 	cd $TEMPDIR
 	rm -rf $TMPDIR
