diff --git a/devel/a/chrony/debian/chrony.conf b/devel/b/chrony/debian/chrony.conf
index e8d9200..69e484b 100644
--- a/devel/a/chrony/debian/chrony.conf
+++ b/devel/b/chrony/debian/chrony.conf
@@ -90,5 +90,5 @@ logchange 0.5
 # If the last line of this file reads 'rtconutc' chrony will assume that
 # the CMOS clock is on UTC (GMT).  If it reads '# rtconutc' or is absent
 # chrony will assume local time.  The line (if any) was written by the
-# chrony postinst based on what it found in /etc/default/rcS.  You may
+# chrony postinst script based on what it found in /etc/adjtime.  You may
 # change it if necessary. 
