--- prerm.orig	2007-08-13 16:34:17.000000000 +0200
+++ /usr/share/dbconfig-common/dpkg/prerm	2007-08-13 16:34:54.000000000 +0200
@@ -65,7 +65,7 @@
 				$dbc_sqlfile_cmd $rmsql || dbc_remove_error "running maintainer removal sql hook"
 			else
 				tsubstfile=`dbc_mktemp`
-				/usr/sbin/dbconfig-generate-include -f template -o template_infile=$dbc_sqlfile_adm $dbc_packageconfig > $tsubstfile
+				/usr/sbin/dbconfig-generate-include -f template -o template_infile=$rmsql $dbc_packageconfig > $tsubstfile
 				$dbc_sqlfile_cmd $tsubstfile || dbc_remove_error "running maintainer removal sql hook"
 				rm -f $tsubstfile
 			fi
