diff -Nru exiv2-0.20/debian/changelog exiv2-0.20/debian/changelog
--- exiv2-0.20/debian/changelog	2010-07-09 11:46:57.000000000 +0200
+++ exiv2-0.20/debian/changelog	2011-05-29 19:50:20.000000000 +0200
@@ -1,3 +1,11 @@
+exiv2 (0.20-2.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Remove references to other libraries from dependency_libs field
+    (Closes: #618747)
+
+ -- Luk Claes <luk@debian.org>  Sun, 29 May 2011 19:49:33 +0200
+
 exiv2 (0.20-2) unstable; urgency=low
 
   * Upload to unstable - cleared with debian-release
diff -Nru exiv2-0.20/debian/rules exiv2-0.20/debian/rules
--- exiv2-0.20/debian/rules	2010-05-30 07:19:07.000000000 +0200
+++ exiv2-0.20/debian/rules	2011-05-29 19:49:29.000000000 +0200
@@ -17,6 +17,11 @@
 	[ ! -f Makefile ] || $(MAKE) doc
 	dh_install --list-missing
 	find $(CURDIR)/debian -type f -name exiv2 | xargs /usr/bin/chrpath -d
+	# Drop content of dependency_libs to avoid having to recompile exiv2
+	# if the path of the libs that appear there changes
+	for file in $(CURDIR)/debian/libexiv2-dev/usr/lib/*.la; do \
+		sed -i "/dependency_libs/ s/'.*'/''/" $$file ; \
+	done
 
 override_dh_installdocs:
 	dh_installdocs -A -Xcmd.txt -XMakefile -Xtemplates -XChangeLog
