From 199ea60ca6e9a879f043fba7a588e71ad241468d Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Sat, 28 Aug 2010 09:00:17 +0200
Subject: [PATCH] Add FHS exception for GNU/Hurd top-level directories

---
 policy.sgml |    9 +++++++++
 1 files changed, 9 insertions(+), 0 deletions(-)

diff --git a/policy.sgml b/policy.sgml
index 9037de8..ffa60d1 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -6040,6 +6040,15 @@ install -m644 debian/shlibs.<var>package</var> debian/<var>package</var>/DEBIAN/
                   to get access to kernel information.</footnote>
                 </p>
               </item>
+              <item>
+                <p>
+                  The following directories in the root filesystem are
+                  additionally allowed on GNU/Hurd systems: <file>/hurd</file>
+                  and <file>/servers</file>. <footnote>These directories
+                  are used to store translators and as a set of standard
+                  names for mount points respectively.</footnote>
+                </p>
+              </item>
             </enumlist>
 
           </p>
-- 
1.7.1

