diff -ur ca-certificates-20090814.orig/debian/config.in ca-certificates-20090814/debian/config.in
--- ca-certificates-20090814.orig/debian/config.in	2009-12-10 13:09:58.000000000 +0100
+++ ca-certificates-20090814/debian/config.in	2009-12-10 13:26:38.000000000 +0100
@@ -47,7 +47,8 @@
 db_version 2.0
 db_capb multiselect
 
-db_title "ca-certificates configuration"
+db_settitle ca-certificates/title
+
 db_input medium ca-certificates/trust_new_crts || true
 db_go
 
@@ -142,7 +143,6 @@
 if test "$trust_new" = "ask" && test "$CERTS_NEW" != "" && test "$new_seen" = "false"; then
   # New certificates added
   db_fset ca-certificates/new_crts seen false
-  db_title "ca-certificates configuration"
   db_input critical ca-certificates/new_crts || true
   db_go
   
@@ -213,7 +213,6 @@
   if test "$seen" != true; then
    db_fset ca-certificates/enable_crts seen false
   fi
-  db_title "ca-certificates configuration"
   db_input $PRIO ca-certificates/enable_crts || true
   db_go
 
diff -ur ca-certificates-20090814.orig/debian/templates ca-certificates-20090814/debian/templates
--- ca-certificates-20090814.orig/debian/templates	2009-12-10 13:09:58.000000000 +0100
+++ ca-certificates-20090814/debian/templates	2009-12-10 13:27:30.000000000 +0100
@@ -7,6 +7,10 @@
 # Even minor modifications require translation updates and such
 # changes should be coordinated with translators and reviewers.
 
+Template: ca-certificates/title
+Type: title
+_Description: ca-certificates configuration
+
 Template: ca-certificates/trust_new_crts
 Type: select
 _Choices: yes, no, ask
