--- bash_completion.orig	2008-09-07 12:57:43.000000000 +0900
+++ bash_completion	2008-09-07 13:01:43.000000000 +0900
@@ -4055,7 +4055,7 @@
 			# functions and aliases. Thus we need to retrieve the program
 			# names manually.
 			IFS=":"
-			local path_array=( $(echo "$PATH") )
+			local path_array=( $(echo "$PATH" | sed 's/::\+/:/g;s/^:\|:$//g') )
 			IFS=$'\n'
 			COMPREPLY=( $( compgen -d -W '$(find "${path_array[@]}" . \
 				-mindepth 1 -maxdepth 1 -not -type d -executable -printf "%f\\n")' \
