diff -aur acpid-2.0.14.orig/debian/rules acpid-2.0.14/debian/rules
--- acpid-2.0.14.orig/debian/rules	2011-12-29 00:08:30.000000000 +0100
+++ acpid-2.0.14/debian/rules	2011-12-29 00:24:36.000000000 +0100
@@ -2,8 +2,11 @@
 
 cflags := -W -Wall -Werror -Wundef -Wshadow -D_GNU_SOURCE \
 	$(shell dpkg-buildflags --get CFLAGS) \
+	$(shell dpkg-buildflags --get CPPFLAGS) \
 	-fno-strict-aliasing
 
+
+
 build: build-arch build-indep
 build-arch: build-stamp
 build-indep: build-stamp
Nur in acpid-2.0.14/debian: rules~.
diff -aur acpid-2.0.14.orig/Makefile acpid-2.0.14/Makefile
--- acpid-2.0.14.orig/Makefile	2011-11-15 22:34:09.000000000 +0100
+++ acpid-2.0.14/Makefile	2011-12-29 00:24:16.000000000 +0100
@@ -33,6 +33,9 @@
 
 CFLAGS = -W -Wall -Werror -Wundef -Wshadow -D_GNU_SOURCE $(OPT) \
 	-fno-strict-aliasing -g $(DEFS)
+
+LDFLAGS = `dpkg-buildflags --get LDFLAGS`
+
 DEFS = -DVERSION="\"$(VERSION)\""
 
 all: $(PROGS)
Nur in acpid-2.0.14: Makefile~.
