--- setup_harddisks.orig	2007-06-22 13:12:56.000000000 +0200
+++ setup_harddisks	2007-06-22 13:13:53.000000000 +0200
@@ -744,7 +744,11 @@
             ($part eq $BootPartition) && ($line .= ", bootable");
             $sfdiskTables{$disk} .= "$line\n";
 	}
-#	print $sfdiskTables{$disk};
+        if($verbose) {
+            print("\n-[sfdisk input]-".("-"x50)."\n");
+            print $sfdiskTables{$disk};
+            print(("-"x66)."\n");
+        }
 	$filename = "$ENV{LOGDIR}/partition." . (($disk=~ m#/#) ? join('_', split('/', $disk)) : $disk);
 	if(($test != 1) && ($filename)){
 	    open(FILE, ">$filename") || die "unable to write temporary file $filename\n";
