Index: src/etpan/nntp-thread.c
===================================================================
RCS file: //claws/src/etpan/Attic/nntp-thread.c,v
retrieving revision 1.1.2.13
diff -u -r1.1.2.13 nntp-thread.c
--- src/etpan/nntp-thread.c	16 Feb 2011 07:16:19 -0000	1.1.2.13
+++ src/etpan/nntp-thread.c	9 May 2011 15:50:09 -0000
@@ -592,10 +592,13 @@
 	mailstream_debug = 0;
 #endif
 
-	r = newsnntp_authinfo_username(param->nntp, param->login);
-	if (r == NEWSNNTP_NO_ERROR || 
-	    r == NEWSNNTP_WARNING_REQUEST_AUTHORIZATION_PASSWORD) {
-		r = newsnntp_authinfo_password(param->nntp, param->password);
+	r = newsnntp_mode_reader(param->nntp);
+	if (r == NEWSNNTP_NO_ERROR) {
+		r = newsnntp_authinfo_username(param->nntp, param->login);
+		if (r == NEWSNNTP_NO_ERROR || 
+		    r == NEWSNNTP_WARNING_REQUEST_AUTHORIZATION_PASSWORD) {
+			r = newsnntp_authinfo_password(param->nntp, param->password);
+		}
 	}
 	
 
