Index: emacs-goodies-el.emacsen-install.in
===================================================================
RCS file: /cvsroot/pkg-goodies-el/emacs-goodies-el/debian/emacs-goodies-el.emacsen-install.in,v
retrieving revision 1.25
diff -u -u -r1.25 emacs-goodies-el.emacsen-install.in
--- emacs-goodies-el.emacsen-install.in	22 Jun 2010 21:55:02 -0000	1.25
+++ emacs-goodies-el.emacsen-install.in	28 Jul 2010 22:41:20 -0000
@@ -17,7 +17,10 @@
 # INCLUDED_xemacs21=""
 
 EXCLUDED_emacs20="cwebm.el tabbar.el session.el csv-mode.el maplev.el"
-# EXCLUDED_emacs21="whitespace.el"
+
+# pod-mode.el -- "\s" and defface min-colors no good in emacs21
+EXCLUDED_emacs21="pod-mode.el"
+
 EXCLUDED_xemacs21="csv-mode.el minibuf-electric.el pp-c-l.el tabbar.el todoo.el rfcview.el upstart-mode.el "
 #EXCLUDED_emacs_snapshot="cua.el cfengine.el ibuffer.el ido.el newsticker.el table.el wdired.el "
 #EXCLUDED_emacs22="cua.el cfengine.el ibuffer.el ido.el newsticker.el table.el wdired.el "
