diff -u ecryptfs-utils-50/debian/control ecryptfs-utils-50/debian/control
--- ecryptfs-utils-50/debian/control
+++ ecryptfs-utils-50/debian/control
@@ -1,8 +1,8 @@
 Source: ecryptfs-utils
 Section: misc
 Priority: optional
 Maintainer: Daniel Baumann <daniel@debian.org>
-Build-Depends: debhelper (>= 7), dpatch, autotools-dev, autoconf, automake, libtool, libgcrypt11-dev, libgpg-error-dev, libgpgme11-dev, libgtk2.0-dev, libkeyutils-dev, libopencryptoki-dev [i386], libpam0g-dev, libpkcs11-helper1-dev, libssl-dev, libtspi-dev [i386]
+Build-Depends: debhelper (>= 7), dpatch, autotools-dev, autoconf, automake, libtool, libgcrypt11-dev, libgpg-error-dev, libgpgme11-dev, libkeyutils-dev, libpam0g-dev, libpkcs11-helper1-dev, libssl-dev
 Standards-Version: 3.8.0
 Homepage: http://ecryptfs.sourceforge.net/
 Vcs-Browser: http://git.debian.net/?p=ecryptfs-utils.git
@@ -42,7 +43,7 @@
 Package: libecryptfs-dev
 Section: libdevel
 Architecture: any
-Depends: libecryptfs0 (= ${binary:Version}), libgcrypt11-dev, libgpg-error-dev, libgpgme11-dev, libgtk2.0-dev, libkeyutils-dev, libopencryptoki-dev [i386], libpam0g-dev, libpkcs11-helper1-dev, libssl-dev, libtspi-dev [i386]
+Depends: libecryptfs0 (= ${binary:Version}), libgcrypt11-dev, libgpg-error-dev, libgpgme11-dev, libkeyutils-dev, libpam0g-dev, libpkcs11-helper1-dev, libssl-dev
 Description: ecryptfs cryptographic filesystem (development)
  eCryptfs is a POSIX-compliant enterprise-class stacked cryptographic filesystem
  for Linux.
