From 4cad5da523ce6fca51ceb5f5cd2b19286b738c22 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Antoine=20Beaupr=C3=A9?= <anarcat@debian.org>
Date: Mon, 23 Oct 2017 17:01:47 -0400
Subject: [PATCH 3/3] add missing changelog line template for --lts

---
 scripts/debchange.pl | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/scripts/debchange.pl b/scripts/debchange.pl
index d01ca9a8..27915460 100755
--- a/scripts/debchange.pl
+++ b/scripts/debchange.pl
@@ -1273,6 +1273,9 @@ ()
 		print O "  * Non-maintainer upload by the Security Team.\n";
 	    }
 	    $line = 1;
+        } elsif ($opt_lts && ! $opt_news) {
+            print O "  * Non-maintainer upload by the LTS Security Team.\n";
+            $line = 1;
 	} elsif ($opt_team && ! $opt_news) {
 	    print O "  * Team upload.\n";
 	    $line = 1;
-- 
2.11.0

