diff --git a/scripts/debcommit.pl b/scripts/debcommit.pl
index 815b786..4420366 100755
--- a/scripts/debcommit.pl
+++ b/scripts/debcommit.pl
@@ -452,10 +452,13 @@ sub getprog {
 	}
     }
 
-    # .bzr or .git may be in a parent directory, rather than the current
+    # .svn, .bzr or .git may be in a parent directory, rather than the current
     # directory, if multiple packages are kept in one repository.
     my $dir=getcwd();
     while ($dir=~s/[^\/]*\/?$// && length $dir) {
+	if (-d "$dir/.svn") {
+	    return "svn";
+	}
 	if (-d "$dir/.bzr") {
 	    return "bzr";
 	}
