=== modified file 'methods/connect.cc'
--- methods/connect.cc	2008-02-08 10:31:11 +0000
+++ methods/connect.cc	2008-11-08 15:09:14 +0000
@@ -158,6 +158,7 @@
       struct addrinfo Hints;
       memset(&Hints,0,sizeof(Hints));
       Hints.ai_socktype = SOCK_STREAM;
+      Hints.ai_flags = AI_DEFAULT;
       Hints.ai_protocol = 0;
       
       // if we couldn't resolve the host before, we don't try now

=== modified file 'methods/ftp.cc'
--- methods/ftp.cc	2007-10-02 12:36:42 +0000
+++ methods/ftp.cc	2008-11-08 15:00:55 +0000
@@ -503,7 +503,7 @@
       memset(&Hints,0,sizeof(Hints));
       Hints.ai_socktype = SOCK_STREAM;
       Hints.ai_family = ((struct sockaddr *)&PeerAddr)->sa_family;
-      Hints.ai_flags |= AI_NUMERICHOST;
+      Hints.ai_flags = AI_DEFAULT|AI_NUMERICHOST;
       
       // Get a new passive address.
       char Port[100];
@@ -517,7 +517,7 @@
    memset(&Hints,0,sizeof(Hints));
    Hints.ai_socktype = SOCK_STREAM;
    Hints.ai_family = AF_INET;
-   Hints.ai_flags |= AI_NUMERICHOST;
+   Hints.ai_flags = AI_DEFAULT|AI_NUMERICHOST;
    
    // Get a new passive address.
    char Port[100];
@@ -584,7 +584,7 @@
    struct addrinfo Hints;
    memset(&Hints,0,sizeof(Hints));
    Hints.ai_socktype = SOCK_STREAM;
-   Hints.ai_flags |= AI_NUMERICHOST;
+   Hints.ai_flags = AI_DEFAULT|AI_NUMERICHOST;
    
    /* The RFC defined case, connect to the old IP/protocol using the
       new port. */
@@ -715,7 +715,7 @@
    struct addrinfo Hints;
    memset(&Hints,0,sizeof(Hints));
    Hints.ai_socktype = SOCK_STREAM;
-   Hints.ai_flags |= AI_PASSIVE;
+   Hints.ai_flags = AI_DEFAULT|AI_PASSIVE;
    Hints.ai_family = ((struct sockaddr *)&ServerAddr)->sa_family;
    int Res;
    if ((Res = getaddrinfo(0,"0",&Hints,&BindAddr)) != 0)

=== modified file 'methods/rfc2553emu.h'
--- methods/rfc2553emu.h	2004-09-20 16:56:08 +0000
+++ methods/rfc2553emu.h	2008-11-08 14:58:34 +0000
@@ -110,4 +110,8 @@
 #define AI_NUMERICHOST 0
 #endif
 
+#ifndef AI_DEFAULT
+#define  AI_DEFAULT  (AI_V4MAPPED | AI_ADDRCONFIG)
+#endif
+
 #endif

