From f08112549963bca5dfffdfbe32924143612a59f8 Mon Sep 17 00:00:00 2001
From: Dieter Adriaenssens <dcadriaenssens-guest@users.alioth.debian.org>
Date: Mon, 8 Aug 2016 21:59:35 +0200
Subject: [PATCH] add suite::mysql (closes #830626)

---
 debian-packages | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/debian-packages b/debian-packages
index d2cec47..600911e 100644
--- a/debian-packages
+++ b/debian-packages
@@ -1721,6 +1721,10 @@ Tag: suite::mozilla
 Description: Mozilla
  Mozilla Browser and extensions
 
+Tag: suite::mysql
+Description: MySQL
+ MySQL and variants like MariaDB and Percona
+
 Tag: suite::netscape
 Description: Netscape Navigator
  The pre-6.0 versions of netscape browser
-- 
2.5.0

