diff --git a/src/fatlabel.c b/src/fatlabel.c
index 8397a97..1598de7 100644
--- a/src/fatlabel.c
+++ b/src/fatlabel.c
@@ -119,8 +119,8 @@ int main(int argc, char *argv[])
 	    /* don't know if here should be more strict !uppercase(label[i]) */
 	    if (islower(label[i])) {
 		fprintf(stderr,
-			"fatlabel: labels cannot contain lower case characters\n");
-		exit(1);
+			"fatlabel: warning: lowercase labels work perfectly under Linux and MacOS X\n");
+		break;
 	    }
 	rw = 1;
     }
