diff --git a/src/reason_fragment.cc b/src/reason_fragment.cc
index 67782e3..d7f2cd9 100644
--- a/src/reason_fragment.cc
+++ b/src/reason_fragment.cc
@@ -42,6 +42,7 @@ cw::fragment *depname_frag(pkgCache::DepIterator dep)
                                                             cw::style_attrs_on(
     case pkgCache::Dep::Replaces: return cw::text_fragment(_("replaces"));
     case pkgCache::Dep::Obsoletes: return cw::text_fragment(_("obsoletes"));
+    case pkgCache::Dep::Enhances: return cw::text_fragment(_("enhances"));
     }
 
   // Untranslated (internal error that will only happen if things go
