--- /bin/setupcon.old	2010-01-14 22:15:57.000000000 +0100
+++ /bin/setupcon	2010-02-06 21:20:05.000000000 +0100
@@ -390,6 +390,7 @@
 	if [ "$save_only" != yes ]; then
 	    if [ "$KMAP" -a -f "$KMAP" ]; then
 		eval loadkeys "$KMAP" $verbose
+		udevadm trigger --action=change --subsystem-match=input
 	    else		    
 		if \
 		    [ "$VARIANT" = '' ] && [ "$CONFIG" != "$USER_CONFIG" ] \
@@ -398,14 +399,17 @@
 		           -ot /etc/default/keyboard ]
 		then
 		    eval loadkeys /etc/console-setup/cached.kmap.gz $verbose
+		    udevadm trigger --action=change --subsystem-match=input
 		else
 		    if [ -d /usr/share/X11/xkb ]; then
 			ckbcomp $acm_option $rules_option \
 			    -model "$XKBMODEL" \
 			    "$XKBLAYOUT" "$XKBVARIANT" "$XKBOPTIONS" \
 			    | eval loadkeys $verbose
+			udevadm trigger --action=change --subsystem-match=input
 		    elif [ -f /etc/console-setup/cached.kmap.gz ]; then
 			eval loadkeys /etc/console-setup/cached.kmap.gz $verbose
+			udevadm trigger --action=change --subsystem-match=input
 		    fi
 		fi
 	    fi
