--- chkrootkit.original	2006-12-10 19:15:58.000000000 +0100
+++ chkrootkit	2006-12-10 19:16:41.000000000 +0100
@@ -1098,7 +1098,7 @@
         echo "Warning: \`${files}' file size is zero"
       files1=`${find} ${ROOTDIR}${HOME} ${findargs} -name '.*history' \( -links 2 -o -type l \)`
       [ ! -z "${files1}" ] && \
-        echo "Warning: \`${files}' is linked to another file"
+        echo "Warning: \`${files1}' is linked to another file"
    fi
    if [ -z "${files}" -a -z "${files1}" ]; then
       if [ "${QUIET}" != "t" ]; then echo "nothing found"; fi
