--- ./autoconf.m4f.orig	2007-01-18 04:57:38.000000000 +0100
+++ ./autoconf.m4f	2007-12-07 21:06:05.000000000 +0100
@@ -2171,7 +2171,7 @@
 T15,312
 AC_LANG(Erlang)ac_ext=erl
 ac_compile='$ERLC $ERLCFLAGS -b beam conftest.$ac_ext >&AS_MESSAGE_LOG_FD'
-ac_link='$ERLC $ERLCFLAGS -b beam conftest.$ac_ext >&AS_MESSAGE_LOG_FD ; echo "#!/bin/sh" > conftest$ac_exeext ; echo "\"$ERL\" -run conftest start -run init stop -noshell" >> conftest$ac_exeext ; chmod +x conftest$ac_exeext'
+ac_link='$ERLC $ERLCFLAGS -b beam conftest.$ac_ext >&AS_MESSAGE_LOG_FD&& echo "#!/bin/sh" > conftest$ac_exeext&& echo "\"$ERL\" -run conftest start -run init stop -noshell" >> conftest$ac_exeext&& chmod +x conftest$ac_exeext'
 
 T13,515
 AC_CHECK_FILE_m4_defun_pro([AC_CHECK_FILE])AC_DIAGNOSE([cross],
--- ./erlang.m4.orig	2007-01-18 04:57:38.000000000 +0100
+++ ./erlang.m4	2007-12-07 21:01:22.000000000 +0100
@@ -112,7 +112,7 @@
 m4_define([AC_LANG(Erlang)],
 [ac_ext=erl
 ac_compile='$ERLC $ERLCFLAGS -b beam conftest.$ac_ext >&AS_MESSAGE_LOG_FD'
-ac_link='$ERLC $ERLCFLAGS -b beam conftest.$ac_ext >&AS_MESSAGE_LOG_FD ; echo "#!/bin/sh" > conftest$ac_exeext ; echo "\"$ERL\" -run conftest start -run init stop -noshell" >> conftest$ac_exeext ; chmod +x conftest$ac_exeext'
+ac_link='$ERLC $ERLCFLAGS -b beam conftest.$ac_ext >&AS_MESSAGE_LOG_FD && echo "#!/bin/sh" > conftest$ac_exeext && echo "\"$ERL\" -run conftest start -run init stop -noshell" >> conftest$ac_exeext && chmod +x conftest$ac_exeext'
 ])
 
 # AC_LANG_ERLANG
