Index: debian/aide.conf.d/31_aide_apt-file
===================================================================
--- debian/aide.conf.d/31_aide_apt-file	(revision 782)
+++ debian/aide.conf.d/31_aide_apt-file	(working copy)
@@ -7,7 +7,7 @@
     HOST="$(echo $uri | sed 's|.*//\([^/[:space:]]\+\).*|\1|')"
     HOSTPATH="$(echo $uri | sed 's|.*//[^/[:space:]]\+/\?||;s|/$||;s|/|_|g;s|^\(.\+\)$|_\1|')"
     if [ "$PROTOCOL" = "http" ] || [ "$PROTOCOL" = "ftp" ]; then
-        echo "/var/cache/apt/apt-file/"${HOST//\./\\\.}${HOSTPATH}"_dists_"${dist}"_Contents-@@{ARCH}\.(gz|IndexDiff)$ VarFile"
+        echo "/var/cache/apt/apt-file/"${HOST//\./\\\.}${HOSTPATH}"_dists_"${dist//\//_}"_Contents-@@{ARCH}\.(gz|IndexDiff)$ VarFile"
     fi
 done < $SOURCESLIST
 
