--- atanks.cpp.orig	2012-10-22 12:36:55.000000000 -0300
+++ atanks.cpp	2012-10-22 12:37:52.000000000 -0300
@@ -1008,6 +1008,7 @@
 int options (GLOBALDATA *global, ENVIRONMENT *env, MENUDESC *menu)
 {
   MENUENTRY *opts;
+  char my_pointer[32];
   BUTTON *reset_button = NULL;
   int selected_index = 0, my_key = 0;
   int numEntries;
@@ -1315,8 +1316,7 @@
             {
               int midX = opts[z].x;
               int midY = opts[z].y;
-              char my_pointer[2];
-  
+
               if (z == selected_index)
                   strcpy(my_pointer, "*");
               else
