diff --git a/debian/ekeyd.postrm b/../ekeyd-767671/debian/ekeyd.postrm
index 484db5c..4efc368 100644
--- a/debian/ekeyd.postrm
+++ b/../ekeyd-767671/debian/ekeyd.postrm
@@ -1,9 +1,5 @@
 #!/bin/sh -e
 
-if test -x /sbin/udevcontrol; then
-    udevcontrol --reload_rules 2>/dev/null || udevcontrol reload_rules 2>/dev/null
-elif test -x /sbin/udevadm; then
-    udevadm control --reload-rules 2>/dev/null || udevadm control --reload_rules 2>/dev/null
-fi
+invoke-rc.d --quiet udev reload || true
 
 #DEBHELPER#
