diff -urpN clevis-12.orig/debian/control clevis-12/debian/control
--- clevis-12.orig/debian/control	2020-02-23 16:05:51.000000000 -0700
+++ clevis-12/debian/control	2020-03-18 08:23:56.589606841 -0600
@@ -63,20 +63,20 @@ Architecture: all
 Depends: ${misc:Depends},
     clevis-systemd,
     dracut-network,
-Description: Dracut integration for clevis
+Description: Clevis integration for dracut
  Clevis is a plugable framework for automated decryption. This package
- provides integration for the dracut initramfs to automatically unlock
- LUKSv1 block devices in early boot.
+ provides integration for dracut to automatically unlock LUKSv1 block
+ devices in early boot.
 
 Package: clevis-initramfs
 Architecture: all
 Depends: ${misc:Depends},
     clevis-luks,
     initramfs-tools,
-Description: Dracut integration for initramfs
+Description: Clevis initramfs integration
  Clevis is a plugable framework for automated decryption. This package
- provides integration for the initramfs-tools initrd to automatically
- unlock LUKSv1 block devices in early boot.
+ provides integration for initramfs-tools to automatically unlock LUKSv1
+ block devices in early boot.
 
 Package: clevis-luks
 Architecture: all
