diff -Naur ddclient-3.7.3/debian/config ddclient-3.7.3.new/debian/config
--- ddclient-3.7.3/debian/config	2009-10-13 14:03:45.000000000 +1300
+++ ddclient-3.7.3.new/debian/config	2009-10-13 13:55:11.000000000 +1300
@@ -77,7 +77,7 @@
       protocol=dslreports1
       ;;
     www.zoneedit.com)
-      server=www.zoneedit.com
+      server=dynamic.zoneedit.com
       protocol=zoneedit1
       ;;
   esac
