--- apertium-3.0.7+1.old/debian/rules	2008-12-01 01:12:29.000000000 -0500
+++ apertium-3.0.7+1/debian/rules	2008-12-01 14:09:34.000000000 -0500
@@ -31,10 +31,14 @@
 config.status: configure
 	dh_testdir
 	# Add here commands to configure the package.
-	autoconf
+	# Modified bootstrapping from upstream, declares everything obsolete.
+	libtoolize --force --copy
+	aclocal $(ACLOCAL_FLAGS)
+	autoheader -f
+	automake --include-deps --add-missing -f --gnu --copy
+	autoconf -f
 	CXXFLAGS="$(CXXFLAGS)" CFLAGS="$(CFLAGS) -Wl,-z,defs" ./configure --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE) --prefix=/usr --mandir=\$${prefix}/share/man --infodir=\$${prefix}/share/info
 
-
 build: build-stamp
 build-stamp:  config.status
 	dh_testdir
