--- cmake-extras-1.5.orig/src/GMock/GMockConfig.cmake
+++ cmake-extras-1.5/src/GMock/GMockConfig.cmake
@@ -119,7 +119,8 @@
                                            "${findgmock_gmock_main_lib}"
                                            "${findgmock_gmock_lib}"
                           INSTALL_COMMAND ""
-                          CMAKE_ARGS "-DCMAKE_CXX_FLAGS=${findgmock_cxx_flags}")
+                          CMAKE_ARGS "-DCMAKE_CXX_COMPILER=${CMAKE_CXX_COMPILER}"
+                                     "-DCMAKE_CXX_FLAGS=${findgmock_cxx_flags}")
 
 add_library(gtest INTERFACE)
 target_include_directories(gtest INTERFACE ${GTEST_INCLUDE_DIRS})
