--- /usr/share/cups/templates/de/set-printer-options-header.tmpl.orig	2009-10-08 22:42:16.000000000 +0200
+++ /usr/share/cups/templates/de/set-printer-options-header.tmpl	2009-10-08 22:42:52.000000000 +0200
@@ -1,6 +1,6 @@
 <DIV CLASS="indent">
 
-<H2 CLASS="title">Standardeinstellungen f%uuml;r {printer_name} festlegen</H2>
+<H2 CLASS="title">Standardeinstellungen f&uuml;r {printer_name} festlegen</H2>
 
 <FORM METHOD="POST" ACTION="/admin">
 <INPUT TYPE="HIDDEN" NAME="PRINTER_NAME" VALUE="{printer_name}">
