--- project-history.sgml	2016-01-30 10:22:18.219568638 +0100
+++ project-history_workingcopy.sgml	2016-01-30 10:24:34.293468856 +0100
@@ -880,24 +880,31 @@
 <sect>The 5.x Releases
 
 <!-- TODO: Add more info about lenny and other Debian events here -->
 
 <p>Debian 5.0 (<em>lenny</em>) was <url
 id="http://www.debian.org/News/2009/20090214" name="released"> February 14th,
 2009 for one more architecture than its predecessor, <em>etch</em>. This
 included the port for newer ARM processors. As with the previous release,
 support for the m68k architecture was still available in <em>unstable</em>.
 There were around 23,000 binary packages (built from over 12,000 source
 packages) maintained by more than one thousand and ten Debian
 developers.
+
+<p>With the release of Debian lenny, the naming scheme for point releases was
+<url name="changed" id="https://lists.debian.org/debian-devel-announce/2009/02/msg00003.html">:
+point releases will use a true micro version number, so the first
+point release of Debian lenny will be 5.0.1. In the past point releases were
+named by an <em>r</em> plus the number appended to major and minor number, e.g. 4.0r1.
+
 <!-- NOTE: Developer count based on leader's vote in March -->
 
 <!-- More information:
 
   Release goals  (See https://release.debian.org/lenny/goals.txt and
     https://wiki.debian.org/LennyReleaseGoals)
      - debmake (old packaging tool) removed
      - i18n improvements: all packages using debconf  and package descriptions
         available in apt
      - standard Python is 2.5
 
   Reuse Whats new from Release Notes:
@@ -1043,24 +1050,30 @@
 
 
 
 <sect>The 7.x Releases
 <p>Debian 7.0 (<em>wheezy</em>) was released May 4th, 2013. 
 This new version of Debian included various interesting features such as
 <url id="http://www.debian.org/News/2011/20110726b" name="multiarch support">, 
 several <url id="http://www.debian.org/News/2012/20120425" name="specific tools to
 deploy private clouds">, an improved installer, and a complete set of 
 multimedia codecs and front-ends which removed the need for third-party repositories.
 
 
+<p>After the release of Debian wheezy, the naming scheme for point releases was
+<url name="changed once again" id="https://lists.debian.org/debian-release/2013/05/msg01020.html">:
+point releases will be named by the minor version number, e.g. 7.1.
+In the past point releases were named by the micro number appended to major and
+minor number, e.g. 6.0.1.
+
 <p>
 During the Debian Conference DebConf11, in July 2011, the "multiarch support"
 was introduced. This feature was a release goal for this release.
 Multiarch is a radical rethinking of the filesystem hierarchy with respect to
 library and header paths, to make programs and libraries of different hardware
 architectures easily installable in parallel on the very same system.  This allows
 users to install packages from 
 multiple architectures on the same machine. This is useful in various ways,
 but the most common is installing both 64 and 32-bit software on the same
 machine and having dependencies correctly resolved automatically. This feature
 is described extensively in the <url
 id="https://wiki.debian.org/Multiarch/HOWTO" name="Multiarch manual">.
