--- cron/debian/standard.daily	2009-08-14 00:15:16.000000000 +0200
+++ cron.new/debian/standard.daily	2009-08-14 00:46:23.000000000 +0200
@@ -51,12 +51,6 @@
 	fi
 fi
 
-if cd $bak ; then
-    if ! cmp -s dpkg.status.0 /var/lib/dpkg/status ; then
-	    cp -p /var/lib/dpkg/status dpkg.status
-	    savelog -c 7 dpkg.status >/dev/null
-    fi
-fi
 #
 # Check to see if any files are in lost+found directories and warn admin
 #
