diff --git a/build/boot/kfreebsd/grub-kfreebsd-cdrom.cfg b/build/boot/kfreebsd/grub-kfreebsd-cdrom.cfg
index 97a7746..503c55c 100644
--- a/build/boot/kfreebsd/grub-kfreebsd-cdrom.cfg
+++ b/build/boot/kfreebsd/grub-kfreebsd-cdrom.cfg
@@ -59,11 +59,6 @@ function menuentry_set {
 
 # Default kernel
 if test -e /boot/kernel/kfreebsd-9.gz ; then
-	# When both options are available, add an empty line in-between.
-	if test -e /boot/kernel/kfreebsd.gz ; then
-		menuentry "" { true }
-	fi
-
 	menuentry_set /boot/kernel/kfreebsd-9.gz /boot/mfsroot.gz " (using kernel of FreeBSD 9)"
 
 	if test -e /boot/gtk/mfsroot.gz ; then
@@ -74,6 +69,11 @@ fi
 
 # Alternate kernel
 if test -e /boot/kernel/kfreebsd.gz ; then
+	# When both options are available, add an empty line in-between.
+	if test -e /boot/kernel/kfreebsd-9.gz ; then
+		menuentry "" { true }
+	fi
+
 	menuentry_set /boot/kernel/kfreebsd.gz /boot/mfsroot.gz ""
 
 	if test -e /boot/gtk/mfsroot.gz ; then
