diff -urN dpkg-1.15.8.11/ostable dpkg-1.15.8.11.armhf//ostable
--- dpkg-1.15.8.11/ostable	2011-04-26 06:47:27.000000000 +0000
+++ dpkg-1.15.8.11.armhf//ostable	2011-08-29 10:14:18.498156859 +0000
@@ -16,6 +16,7 @@
 # <Debian name>		<GNU name>		<config.guess regex>
 uclibceabi-linux	linux-uclibceabi	linux[^-]*-uclibceabi
 uclibc-linux		linux-uclibc		linux[^-]*-uclibc
+gnueabihf-linux		linux-gnueabihf		linux[^-]*-gnueabihf
 gnueabi-linux		linux-gnueabi		linux[^-]*-gnueabi
 gnuspe-linux		linux-gnuspe		linux[^-]*-gnuspe
 gnulp-linux		linux-gnulp		linux[^-]*-gnulp
diff -urN dpkg-1.15.8.11/triplettable dpkg-1.15.8.11.armhf//triplettable
--- dpkg-1.15.8.11/triplettable	2011-04-26 06:47:31.000000000 +0000
+++ dpkg-1.15.8.11.armhf//triplettable	2011-08-29 10:14:17.278156859 +0000
@@ -5,6 +5,7 @@
 # <Debian triplet>	<Debian arch>
 uclibceabi-linux-arm	uclibc-linux-armel
 uclibc-linux-<cpu>	uclibc-linux-<cpu>
+gnueabihf-linux-arm	armhf
 gnueabi-linux-arm	armel
 gnuspe-linux-powerpc	powerpcspe
 gnulp-linux-i386	lpia
