diff -Nru bash-4.4/debian/changelog bash-4.4/debian/changelog
--- bash-4.4/debian/changelog	2017-05-15 12:45:32.000000000 -0700
+++ bash-4.4/debian/changelog	2017-07-26 18:02:45.000000000 -0700
@@ -1,3 +1,11 @@
+bash (4.4-6) UNRELEASED; urgency=medium
+
+  * Non-maintainer upload
+  * Add $HOME/.local/bin to skel.profile
+    - Closes: #839155, LP: #1705571
+
+ -- gdahlman <gdahlman@hotmail.com>  Wed, 26 Jul 2017 18:02:45 -0700
+
 bash (4.4-5) unstable; urgency=medium
 
   * Apply upstream patch 012.
diff -Nru bash-4.4/debian/skel.profile bash-4.4/debian/skel.profile
--- bash-4.4/debian/skel.profile	2013-10-23 05:41:22.000000000 -0700
+++ bash-4.4/debian/skel.profile	2017-07-26 18:02:45.000000000 -0700
@@ -20,3 +20,8 @@
 if [ -d "$HOME/bin" ] ; then
     PATH="$HOME/bin:$PATH"
 fi
+
+# set PATH so it includes user's pip private bin if it exists
+if [ -d "$HOME/.local/bin" ] ; then
+    PATH="$HOME/.local/bin:$PATH"
+fi
