--- init.d.lsb.ex.orig	2007-10-16 21:20:29.000000000 +0200
+++ init.d.lsb.ex	2007-10-16 21:21:13.000000000 +0200
@@ -54,7 +54,7 @@
 # at /etc/default/$NAME
 DAEMON_OPTS=""          # Additional options given to the server 
 
-DODTIME=10              # Time to wait for the server to die, in seconds
+DIETIME=10              # Time to wait for the server to die, in seconds
                         # If this value is set too low you might not
                         # let some servers to die gracefully and
                         # 'restart' will not work
