diff -urNp base-passwd-3.5.37.orig/doc/Makefile.am base-passwd-3.5.37/doc/Makefile.am
--- base-passwd-3.5.37.orig/doc/Makefile.am	2014-01-08 14:06:26.000000000 -0200
+++ base-passwd-3.5.37/doc/Makefile.am	2015-07-15 13:14:40.887240123 -0300
@@ -6,4 +6,4 @@ users-and-groups.html: users-and-groups.
 	sgmltools -b onehtml $<
 
 users-and-groups.txt: users-and-groups.sgml
-	sgmltools -b txt $<
+	LC_ALL=C.UTF-8 sgmltools -b txt $<
