From eb4f407d6c1541fda298dcfe8bcaee1fdebd5677 Mon Sep 17 00:00:00 2001
From: Xiyue Deng <manphiz@gmail.com>
Date: Fri, 19 Apr 2024 14:24:40 -0700
Subject: [PATCH 2/2] Add more comments to describe the purpose of renaming
 source files

---
 dh_elpa_test | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/dh_elpa_test b/dh_elpa_test
index b9f1152..c0e99e0 100755
--- a/dh_elpa_test
+++ b/dh_elpa_test
@@ -268,6 +268,10 @@ if ($autopkgtest) {
         exit 0;
     }
 
+    # Compile a list of files to be renamed during autopkgtest.  This usually
+    # renames source *.el file outside the test directories so that during
+    # autopkgtest we are testing the installed package instead of relying on
+    # source files from the source directory.
     my $rule = File::Find::Rule->new;
     $rule
       ->or(File::Find::Rule
-- 
2.39.2

