diff -u claws-mail-3.5.0/debian/changelog claws-mail-3.5.0/debian/changelog
--- claws-mail-3.5.0/debian/changelog
+++ claws-mail-3.5.0/debian/changelog
@@ -1,3 +1,10 @@
+claws-mail (3.5.0-2.1) testing-proposed-updates; urgency=high
+
+  * Non-maintainer upload.
+  * Disable building of dillo stuff
+
+ -- Neil McGovern <neilm@debian.org>  Sat, 03 Jan 2009 14:44:10 +0000
+
 claws-mail (3.5.0-2) unstable; urgency=low
 
   * debian/control
diff -u claws-mail-3.5.0/debian/control claws-mail-3.5.0/debian/control
--- claws-mail-3.5.0/debian/control
+++ claws-mail-3.5.0/debian/control
@@ -51,8 +51,7 @@
 
 Package: claws-mail-plugins
 Architecture: all
-Depends: claws-mail-dillo-viewer, 
- claws-mail-spamassassin, claws-mail-trayicon, claws-mail-pgpmime, 
+Depends:  claws-mail-spamassassin, claws-mail-trayicon, claws-mail-pgpmime,
  claws-mail-pgpinline, claws-mail-bogofilter
 Conflicts: sylpheed-claws-gtk2-plugins
 Replaces: sylpheed-claws-gtk2-plugins
@@ -67,17 +66,6 @@
  .
  See the claws-mail-extra-plugins package for more plugins.
 
-Package: claws-mail-dillo-viewer
-Architecture: any
-Depends: ${shlibs:Depends}, claws-mail (= ${binary:Version}), 
- dillo (>= 0.7.2)
-Conflicts: sylpheed-claws-gtk2-dillo-viewer
-Replaces: sylpheed-claws-gtk2-dillo-viewer
-Provides: sylpheed-claws-gtk2-dillo-viewer
-Description: HTML viewer plugin for Claws Mail using Dillo 
- This is an HTML viewer plugin that uses the Dillo browser to render the 
- HTML message parts in the Claws Mail message view window.
-
 Package: claws-mail-spamassassin
 Architecture: any
 Depends: ${shlibs:Depends}, claws-mail (= ${binary:Version}), 
diff -u claws-mail-3.5.0/debian/rules claws-mail-3.5.0/debian/rules
--- claws-mail-3.5.0/debian/rules
+++ claws-mail-3.5.0/debian/rules
@@ -32,7 +32,8 @@
 		--enable-jpilot \
 		--enable-ldap \
 		--enable-spamassassin-plugin=yes \
-		--with-config-dir=.claws-mail
+		--with-config-dir=.claws-mail \
+		--disable-dillo-viewer-plugin
 
 build: build-stamp
 
@@ -86,10 +87,10 @@
 	$(CURDIR)/debian/inst-manual.sh $(CURDIR)/manual/es/dist \
 		$(CURDIR)/debian/claws-mail-doc/usr/share/doc/claws-mail/es
 	# claws-mail-dillo-viewer
-	mkdir -m 755 -p \
-	     $(CURDIR)/debian/claws-mail-dillo-viewer/usr/lib/claws-mail/plugins
-	mv $(PLUGINSDIR)/dillo_viewer.so \
-	   $(CURDIR)/debian/claws-mail-dillo-viewer/usr/lib/claws-mail/plugins
+	# mkdir -m 755 -p \
+	#     $(CURDIR)/debian/claws-mail-dillo-viewer/usr/lib/claws-mail/plugins
+	# mv $(PLUGINSDIR)/dillo_viewer.so \
+	#   $(CURDIR)/debian/claws-mail-dillo-viewer/usr/lib/claws-mail/plugins
 	# claws-mail-spamassassin
 	mkdir -m 755 -p \
 	     $(CURDIR)/debian/claws-mail-spamassassin/usr/lib/claws-mail/plugins
