From ee071f976008c71b8133e02a48c7363304bd561d Mon Sep 17 00:00:00 2001
From: Arthur de Jong <arthur@arthurdejong.org>
Date: Sun, 27 Oct 2013 22:45:09 +0100
Subject: [PATCH] Explicitly set permissions of testtmp directory

---
 runner/adt-run | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/runner/adt-run b/runner/adt-run
index cc77c75..826764b 100755
--- a/runner/adt-run
+++ b/runner/adt-run
@@ -1413,7 +1413,7 @@ class Test:
         testtmp = '%s%s-testtmp' % (testbed.scratch.read(True), self.what)
 
         script = 'buildtree="$1"; shift\n'
-        script += 'rm -rf -- "$@"; mkdir -- "$@"\n'
+        script += 'rm -rf -- "$@"; mkdir -- "$@"; chmod 755 "$@"\n'
 
         if 'needs-root' not in self.restriction_names and opts.user is not None:
             tfl = ['su', opts.user, '-c', tf]
-- 
1.8.4.rc3

