--- dphys-swapfile.orig	2008-09-14 13:30:03.875846950 -0400
+++ dphys-swapfile	2008-09-14 13:30:40.845846980 -0400
@@ -25,9 +25,6 @@
 
 # this is what we want, 2 times RAM size
 SWAPFACTOR=2
-# this is a (outdated?) kernel limit (in MBytes), do not overrun it
-MAXSWAP=2048
-
 
 # what we are
 NAME=dphys-swapfile
@@ -59,13 +56,6 @@
     # announce end resulting config
     echo -n "want ${CONF_SWAPFILE}=${CONF_SWAPSIZE}MByte"
 
-    # check for legitimate swap size and restrict to it
-    if [ "${CONF_SWAPSIZE}" -gt "${MAXSWAP}" ] ; then
-      echo -n ", limiting to kernel limit: ${MAXSWAP}MBytes"
-      CONF_SWAPSIZE="${MAXSWAP}"
-    fi
-
-
     # we will be later starting, and in between possible deleting/rebuilding
     #   so deactivate any allready running swapfile, to avoid errors
     "$0" swapoff
