diff --git a/debian/bluez.postinst b/debian/bluez.postinst
index 3d35802..65664d6 100644
--- a/debian/bluez.postinst
+++ b/debian/bluez.postinst
@@ -9,12 +9,6 @@ case "$1" in
             # maybe a (medium/low debconf?) notice is best suited here
         fi
 
-        # use MAKEDEV instead of the original bluez script below as per policy 10.6
-        if [ -x /dev/MAKEDEV ]; then
-            echo "Creating device nodes ..."
-            cd /dev && ./MAKEDEV bluetooth 1>/dev/null 2>/dev/null
-        fi
-
         # create bluetooth group if not already present
         if ! getent group bluetooth > /dev/null; then
             addgroup --quiet --system bluetooth
@@ -25,11 +19,6 @@ case "$1" in
             invoke-rc.d dbus force-reload || true
         fi
 
-        #reload udev rules
-        if [ -x /etc/init.d/udev ]; then
-            invoke-rc.d udev reload || true
-        fi
-
         ;;
     abort-upgrade|abort-remove|abort-deconfigure)
     ;;
