diff --git a/completions/apt-get b/completions/apt-get
index a6c902e..d35e0b0 100644
--- a/completions/apt-get
+++ b/completions/apt-get
@@ -7,7 +7,7 @@ _apt_get()
 
     local special i
     for (( i=0; i < ${#words[@]}-1; i++ )); do
-        if [[ ${words[i]} == @(install|remove|autoremove|purge|source|build-dep|download) ]]; then
+        if [[ ${words[i]} == @(install|remove|autoremove|purge|source|build-dep|download|changelog) ]]; then
             special=${words[i]}
         fi
     done
