diff --git a/src/cacheman.cc b/src/cacheman.cc
index 43d8f12..940be40 100644
--- a/src/cacheman.cc
+++ b/src/cacheman.cc
@@ -1700,7 +1700,7 @@ bool cacheman::ParseAndProcessMetaFile(std::function<void(const tRemoteFileInfo&
 	{
 	case EIDX_PACKAGES:
 		LOG("filetype: Packages file");
-		static const string sMD5sum("MD5sum"), sFilename("Filename"), sSize("Size");
+		static const string sMD5sum("MD5sum"), sSHA256("SHA256"), sFilename("Filename"), sSize("Size");
 
 		UrlUnescape(sPkgBaseDir);
 
@@ -1728,6 +1728,8 @@ bool cacheman::ParseAndProcessMetaFile(std::function<void(const tRemoteFileInfo&
 				// not looking for data we already have
 				if(key==sMD5sum)
 					info.fpr.SetCs(val, CSTYPE_MD5);
+				else if(key==sSHA256)
+					info.fpr.SetCs(val, CSTYPE_SHA256);
 				else if(key==sSize)
 					info.fpr.size=atoofft(val.c_str());
 				else if(key==sFilename)
