diff --git a/cron.daily/chkrootkit b/cron.daily/chkrootkit
index 7a6b7b9..152d503 100755
--- a/cron.daily/chkrootkit
+++ b/cron.daily/chkrootkit
@@ -4,6 +4,7 @@ set -e
 
 CHKROOTKIT=/usr/sbin/chkrootkit
 CF=/etc/chkrootkit.conf
+IGNOREF=/etc/chkrootkit.ignore
 LOG_DIR=/var/log/chkrootkit
 
 if [ ! -x $CHKROOTKIT ]; then
@@ -16,7 +17,7 @@ fi
 
 if [ "$RUN_DAILY" = "true" ]; then
     if [ "$DIFF_MODE" = "true" ]; then
-				eval $CHKROOTKIT $RUN_DAILY_OPTS > $LOG_DIR/log.today.raw 2>&1
+				eval $CHKROOTKIT $RUN_DAILY_OPTS  | egrep -v -f $IGNOREF > $LOG_DIR/log.today.raw 2>&1
 				# the sed expression replaces the messages about /sbin/dhclient3 /usr/sbin/dhcpd3
 				# with a message that is the same whatever order eth0 and eth1 were scanned
 				sed -r -e 's,eth(0|1)(:[0-9])?: PACKET SNIFFER\((/sbin/dhclient3|/usr/sbin/dhcpd3)\[[0-9]+\]\),eth\[0|1\]: PACKET SNIFFER\([dhclient3|dhcpd3]{PID}\),' \
@@ -46,6 +47,6 @@ if [ "$RUN_DAILY" = "true" ]; then
 						echo "# (note that unedited output is in $LOG_DIR/log.today.raw)"
         fi
     else
-        eval $CHKROOTKIT $RUN_DAILY_OPTS
+        eval $CHKROOTKIT $RUN_DAILY_OPTS | egrep -v -f $IGNOREF || true
     fi
 fi
