--- activemq-admin.sh.orig	2012-03-05 15:44:24.075038859 -0500
+++ activemq-admin.sh.fixed	2012-03-05 15:44:49.647039321 -0500
@@ -129,7 +129,7 @@
 # Set default classpath
 ACTIVEMQ_CLASSPATH="${ACTIVEMQ_BASE}/conf;"$ACTIVEMQ_CLASSPATH
 
-if [ $1 = "start" ] ; then
+if [ "$1" = "start" ] ; then
     if [ -z "$ACTIVEMQ_OPTS" ] ; then
         ACTIVEMQ_OPTS="-Xmx512M -Dorg.apache.activemq.UseDedicatedTaskRunner=true -Djava.util.logging.config.file=logging.properties"
     fi
