diff -Nru dctrl-tools-2.22.2/debian/changelog dctrl-tools-2.22.2multiarch1/debian/changelog
--- dctrl-tools-2.22.2/debian/changelog	2012-06-07 09:48:29.000000000 +0100
+++ dctrl-tools-2.22.2multiarch1/debian/changelog	2012-11-16 21:25:59.000000000 +0000
@@ -1,3 +1,9 @@
+dctrl-tools (2.22.2multiarch1) unstable; urgency=low
+
+  * Set Multi-Arch:foreign to allow cross-dependency satisfaction
+
+ -- Wookey <wookey@wookware.org>  Tue, 13 Nov 2012 04:16:08 +0000
+
 dctrl-tools (2.22.2) unstable; urgency=low
 
   * Closes: #674836 (Builds with wrong CFLAGS), reported by me:
diff -Nru dctrl-tools-2.22.2/debian/control dctrl-tools-2.22.2multiarch1/debian/control
--- dctrl-tools-2.22.2/debian/control	2012-06-03 19:09:05.000000000 +0100
+++ dctrl-tools-2.22.2multiarch1/debian/control	2012-11-16 21:25:30.000000000 +0000
@@ -15,6 +15,7 @@
 Conflicts: grep-dctrl
 Replaces: grep-dctrl
 Provides: grep-dctrl
+Multi-Arch: foreign
 Description: Command-line tools to process Debian package information
  Debian package information is generally stored in files having a
  special file format, dubbed the Debian control file format (the dctrl
