From 4a72a98eac684fb5965808f7744e0267829b05c8 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?David=20Pr=C3=A9vot?= <taffit@debian.org>
Date: Tue, 7 May 2013 19:18:35 -0400
Subject: [PATCH 2/2] who-permits-upload.1: wfix

---
 scripts/who-permits-upload.pl |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/scripts/who-permits-upload.pl b/scripts/who-permits-upload.pl
index 207e567..bfb489b 100755
--- a/scripts/who-permits-upload.pl
+++ b/scripts/who-permits-upload.pl
@@ -50,8 +50,8 @@ B<who-permits-upload> [B<-h>] [B<-s> I<keyring>] [B<-d> I<dm_url>] [B<-s> I<sear
 =head1 DESCRIPTION
 
 B<who-permits-upload> looks up the given Debian Maintainer (DM) upload permissions
-from Debian's ftpmaster and parses them in a human readable way. The tool can
-search by DM name, sponsor (the person granted the permission) and by package.
+from ftp-master.debian.org and parses them in a human readable way. The tool can
+search by DM name, sponsor (the person who granted the permission) and by package.
 
 =head1 OPTIONS
 
@@ -78,7 +78,7 @@ Separate keyrings with a colon ":".
 
 =item B<--search=>I<search_type>, B<-s> I<search_type>
 
-Modify the look-up behavior instead of using the default. This influences the
+Modify the look-up behavior. This influences the
 interpretation of the I<query> argument. Supported search types are:
 
 =over 4
-- 
1.7.10.4

