diff -Naur etc/bash.bashrc etc2/bash.bashrc
--- etc/bash.bashrc	2009-03-21 17:38:35.000000000 -0700
+++ etc2/bash.bashrc	2009-04-11 17:50:36.803210958 -0700
@@ -29,7 +29,7 @@
 #esac
 
 # enable bash completion in interactive shells
-#if [ -f /etc/bash_completion ] && [ $(set -o | grep -q '^posix[[:space:]]*off$' ]; then
+#if [ -f /etc/bash_completion ] && shopt -oq posix; then
 #    . /etc/bash_completion
 #fi
 
