From 5ee2cdc9baa5ab5349012c86860e5f6200a0ff50 Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Mon, 1 Jul 2013 12:41:06 +0800
Subject: [PATCH 2/3] New look for Debian's websites (Closes: #655659)

---
 data/events/services | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/data/events/services b/data/events/services
index 0a04c7b..2de5bea 100644
--- a/data/events/services
+++ b/data/events/services
@@ -90,6 +90,10 @@ Title: Archive snapshot service available
 Date: Apr 12 2010
 Source: http://lists.debian.org/debian-announce/2010/msg00002.html
 
+Title: New look for Debian's websites
+Date: Feb 2 2011
+Source: http://www.debian.org/News/2011/20110205b
+
 Title: Derivatives patches service available
 Date: Mar 1 2012
 Source: http://lists.debian.org/debian-devel-announce/2012/03/msg00000.html
-- 
1.8.3.1

