From da4eb57f01a95c473763a1c395f987f6461be52d Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Mon, 1 Jul 2013 12:43:58 +0800
Subject: [PATCH 3/3] Debian won two Linux New Media Awards (Closes: #655640)

---
 data/events/other_projects | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/data/events/other_projects b/data/events/other_projects
index 7c16a51..19a8495 100644
--- a/data/events/other_projects
+++ b/data/events/other_projects
@@ -58,3 +58,7 @@ Source: https://lists.ubuntu.com/archives/ubuntu-announce/2004-October/000003.ht
 Title: First Linux Mint Debian Edition
 Date: Sep 7th, 2010
 Source: http://blog.linuxmint.com/?p=1527
+
+Title: Debian wins two Linux New Media Awards
+Date: Mar 4 2011
+Source: http://www.debian.org/News/2011/20110304
-- 
1.8.3.1

