diff -aur asterisk-1.8.8.0~dfsg.orig/debian/rules asterisk-1.8.8.0~dfsg/debian/rules
--- asterisk-1.8.8.0~dfsg.orig/debian/rules	2011-11-11 17:13:52.000000000 +0100
+++ asterisk-1.8.8.0~dfsg/debian/rules	2012-01-01 20:48:59.000000000 +0100
@@ -16,6 +16,11 @@
 
 export PROC := $(shell dpkg-architecture -qDEB_BUILD_GNU_CPU)
 
+CFLAGS = `dpkg-buildflags --get CFLAGS`
+LDFLAGS = `dpkg-buildflags --get LDFLAGS`
+CFLAGS += `dpkg-buildflags --get CPPFLAGS`
+
+
 ifneq (,$(filter noopt,$(DEB_BUILD_OPTIONS)))
 	BUILDFLAGS += OPTIMIZE=-O0
 else
@@ -66,6 +71,7 @@
 	chmod 755 $(CURDIR)/debian/dummyprogs/fetch
 	
 	$(FETCH_ENV) ./configure			\
+		CFLAGS="$(CFLAGS)" LDFLAGS="$(LDFLAGS)" \
 		--host=$(DEB_HOST_GNU_TYPE)		\
 		--build=$(DEB_BUILD_GNU_TYPE)		\
 		--prefix=/usr				\
Nur in asterisk-1.8.8.0~dfsg/debian: rules~.
