--- a/dkms	2016-07-06 01:12:00.000000000 +0200
+++ b/dkms	2016-07-26 22:30:26.486793435 +0200
@@ -2960,7 +2960,7 @@
 		die 7 $"There was a problem creating your ${create_type}."
 	    echo $""
 	    echo $"DKMS: mk${create_type} completed."
-	    invoke_command "mv '$temp_dir/${debian_package}-dkms_${module_version}_${debian_build_arch}.deb' '$deb_basedir'" "Moving built files to $deb_basedir"
+	    invoke_command "mv '$temp_dir/${debian_package}-dkms_${module_version}_all.deb' '$deb_basedir'" "Moving built files to $deb_basedir"
 	    ;;
 	bmdeb)
             export KVER="$kernelver"
