diff -u a/dkms b/dkms
--- a/dkms	2016-07-26 20:03:17.901049440 +0200
+++ b/dkms	2016-07-26 20:03:04.276981881 +0200
@@ -2887,7 +2887,11 @@
     make_common_test "mk${create_type}"
 
     debian_package=${module//_/-}
-    debian_build_arch=$(dpkg-architecture -qDEB_BUILD_ARCH)
+    if [[ $source_only ]]; then
+        debian_build_arch='all'
+    else
+        debian_build_arch=$(dpkg-architecture -qDEB_BUILD_ARCH)
+    fi
 
     # Read the conf file
     read_conf_or_die "$kernelver" "$arch"
