diff --git a/lib/testdesc.py b/lib/testdesc.py
index 3e696a2..39e1ecb 100644
--- a/lib/testdesc.py
+++ b/lib/testdesc.py
@@ -678,7 +678,7 @@ def parse_debian_source(srcdir, testbed_caps, testbed_arch, control_path=None,
                 raise InvalidControl('*', 'missing "Tests" or "Test-Command"'
                                      ' field')
         except Unsupported as u:
-            if testname is None or n == testname:
+            if testname is None:
                 u.report()
                 some_skipped = True
 
