From bbb5672096c7549109bb0786b841bd8c81c0b7de Mon Sep 17 00:00:00 2001
From: Dmitry Smirnov <onlyjob@member.fsf.org>
Date: Fri, 12 Oct 2012 10:48:39 +1100
Subject: [PATCH] d/watch update

---
 debian/watch |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/watch b/debian/watch
index 6ebb74d..5a2271d 100644
--- a/debian/watch
+++ b/debian/watch
@@ -1,2 +1,2 @@
-version=2
-http://freedesktop.org/software/desktop-file-utils/releases/desktop-file-utils-(.*)\.tar\.(gz|bz2)
+version=3
+http://freedesktop.org/software/desktop-file-utils/releases/desktop-file-utils-(.*)\.tar\.(gz|bz2|xz)
-- 
1.7.10.4

