diff -Nru /tmp/la3uTQwXjB/dput-0.9.2.28/debian/changelog /tmp/NreT9ewU9N/dput-0.9.2.28/debian/changelog
--- /tmp/la3uTQwXjB/dput-0.9.2.28/debian/changelog	2007-07-15 06:03:54.000000000 -0700
+++ /tmp/NreT9ewU9N/dput-0.9.2.28/debian/changelog	2007-12-20 17:40:05.000000000 -0800
@@ -1,3 +1,11 @@
+dput (0.9.2.28-0.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Add [security-master], [security-master-unembargoed], [ubuntu], and [revu]
+    stanzas.
+
+ -- Steve Langasek <vorlon@debian.org>  Thu, 20 Dec 2007 17:39:40 -0800
+
 dput (0.9.2.28) unstable; urgency=low
 
   * Add support for guessing distribution. Thanks to Christoph Berg
diff -Nru /tmp/la3uTQwXjB/dput-0.9.2.28/dput.cf /tmp/NreT9ewU9N/dput-0.9.2.28/dput.cf
--- /tmp/la3uTQwXjB/dput-0.9.2.28/dput.cf	2007-07-15 05:57:38.000000000 -0700
+++ /tmp/NreT9ewU9N/dput-0.9.2.28/dput.cf	2007-12-20 17:39:37.000000000 -0800
@@ -26,6 +26,16 @@
 # post_upload_command	= /path/to/some/script
 # pre_upload_command	= /path/to/some/script
 
+[security-master]
+fqdn			= security-master.debian.org
+incoming		= /pub/SecurityUploadQueue
+login			= anonymous
+
+[security-master-unembargoed]
+fqdn			= security-master.debian.org
+incoming		= /pub/OpenSecurityUploadQueue
+login			= anonymous
+
 [jp]
 fqdn			= hp.debian.or.jp
 incoming		= /pub/Incoming/upload
@@ -52,6 +62,16 @@
 method			= ftp
 login			= anonymous
 
+[ubuntu]
+fqdn			= upload.ubuntu.com
+incoming		= /
+login			= anonymous
+
+[revu]
+fqdn			= revu.tauware.de
+incoming		= /incoming
+login			= anonymous
+
 [local]
 method			= local
 incoming		= ~/public_html/debian/mini-dinstall/incoming
