--- /usr/bin/dlocate.distrib	2005-11-25 16:02:24.000000000 +0000
+++ /usr/bin/dlocate	2005-05-10 13:07:17.000000000 +0100
@@ -88,6 +88,7 @@
 		;;
 	"-S")
 		$LOCATE -d $DLOCATEDB $PKG | grep ":.*$PKG.*"
+		result=$?
 		;;
 	"-s")
 		if [ -e $DPKG_INFO/$PKG.list ] ; then 
@@ -151,6 +152,7 @@
 		;;
 	*)
 		$LOCATE -d $DLOCATEDB "$PKG"
+		result=$?
 		;;
 esac
 
@@ -158,4 +160,4 @@
 PKG=$1
 done
 
-
+test -n "$result" && exit $result
