From a2321a8f177fbd4c85ff18c9809c496a462bab71 Mon Sep 17 00:00:00 2001
From: James Clarke <jrtc27@debian.org>
Date: Sat, 27 May 2017 20:14:09 +0100
Subject: [PATCH] debsign: Fix signing a dsc directly

---
 scripts/debsign.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/debsign.sh b/scripts/debsign.sh
index 0717bead..924b03eb 100755
--- a/scripts/debsign.sh
+++ b/scripts/debsign.sh
@@ -725,7 +725,7 @@ for valid format" >&2;
 	report_signed
 
     else
-	ensure_local_copy "$remotehost" "$remotehost" "$remotedsc" dsc
+	ensure_local_copy "$remotehost" "$remotedsc" "$dsc" dsc
 	signas="$(guess_signas "$dsc")"
 	maybesign_dsc "$signas" "$remotehost" "$dsc"
 	report_signed
-- 
2.11.0

