diff -r 7d1a4a4d43db -r c7a2375be38c src/ui.cc
--- a/src/ui.cc	Sun Mar 30 14:24:28 2008 +0200
+++ b/src/ui.cc	Sun Mar 30 09:29:37 2008 -0700
@@ -2372,7 +2372,7 @@ cw::menu_info help_menu_info[]={
 	       sigc::ptr_fun(do_help_faq)),
 
   cw::menu_info(cw::menu_info::MENU_ITEM, N_("^News"), NULL,
-		ssprintf(N_("View the important changes made in each version of %s"), PACKAGE).c_str(),
+		N_("View the important changes made in each version of " PACKAGE),
 	       sigc::ptr_fun(do_help_news)),
 
   cw::menu_info(cw::menu_info::MENU_ITEM, N_("^License"), NULL,
