--- debian-policy-3.7.2.2/debian/rules	2006-10-03 01:36:50.000000000 +0300
+++ debian-policy-3.7.2.3/debian/rules	2007-06-29 14:07:49.000000000 +0300
@@ -90,7 +90,7 @@
 		$(SGML_FILES:=.html.tar.gz) \
 		$(SGML_FILES:=.txt.gz) \
 		policy.ps.gz policy.pdf.gz
-	tar -zxf $(FHS_ARCHIVE)
+	mkdir fhs && cd fhs && tar -zxf ../$(FHS_ARCHIVE) && cd ..
 	# Need to use a patched tmac.m macro file if we're using a pre-1.16
 	# groff; version 1.16 is apparently fixed
 	if dpkg --compare-versions "$(shell dpkg -s groff | sed -n -e 's/Version: //p')" lt 1.16; then \
