--- inutils-2.13.90.0.18.orig/gas/dwarf2dbg.c	2003-04-14 23:52:23.000000000 +0200
+++ inutils-2.13.90.0.18/gas/dwarf2dbg.c	2003-04-15 00:54:49.000000000 +0200
@@ -332,7 +332,8 @@
       return last_used;
 
   for (i = 1; i < files_in_use; ++i)
-    if (strcmp (filename, files[i].filename) == 0)
+    if (files && files[i].filename &&
+	strcmp (filename, files[i].filename) == 0)
       return i;
 
   if (i >= files_allocated)
@@ -350,7 +351,12 @@
   files[i].dir = 0;
   files_in_use = i + 1;
   last_used = i;
-
+  
+  if (!files[1].filename)
+    {
+      files[1].filename = xstrdup (filename);
+      files[1].dir = 0;
+    }
   return i;
 }
 
