diff -ru etherape-0.9.0.orig/glade/etherape.glade2 etherape-0.9.0/glade/etherape.glade2
--- etherape-0.9.0.orig/glade/etherape.glade2	2003-08-21 11:59:21.000000000 -0400
+++ etherape-0.9.0/glade/etherape.glade2	2003-08-21 12:44:46.000000000 -0400
@@ -299,6 +299,17 @@
 		      </child>
 
 		      <child>
+			<widget class="GtkCheckMenuItem" id="full_screen">
+			  <property name="visible">True</property>
+			  <property name="tooltip" translatable="yes">Activate full screen mode</property>
+			  <property name="label" translatable="yes">_Full Screen</property>
+			  <property name="use_underline">True</property>
+			  <property name="active">False</property>
+			  <signal name="activate" handler="on_full_screen_activate" last_modification_time="Thu, 21 Aug 2003 14:23:04 GMT"/>
+			</widget>
+		      </child>
+
+		      <child>
 			<widget class="GtkCheckMenuItem" id="toolbar_check">
 			  <property name="visible">True</property>
 			  <property name="tooltip" translatable="yes">Show or hide the toolbar</property>
Only in etherape-0.9.0/glade: etherape.glade2p
diff -ru etherape-0.9.0.orig/src/menus.c etherape-0.9.0/src/menus.c
--- etherape-0.9.0.orig/src/menus.c	2003-08-21 11:59:21.000000000 -0400
+++ etherape-0.9.0/src/menus.c	2003-08-21 12:46:29.000000000 -0400
@@ -315,6 +315,14 @@
 /* View menu */
 
 
+void
+on_full_screen_activate (GtkCheckMenuItem * menuitem, gpointer user_data)
+{
+	if (menuitem->active)
+		gtk_window_fullscreen((GtkWindow *)app1);
+	else
+		gtk_window_unfullscreen((GtkWindow *)app1);
+} /* on_full_screen_activate */
 
 void
 on_toolbar_check_activate (GtkCheckMenuItem * menuitem, gpointer user_data)
diff -ru etherape-0.9.0.orig/src/menus.h etherape-0.9.0/src/menus.h
--- etherape-0.9.0.orig/src/menus.h	2003-08-21 11:59:21.000000000 -0400
+++ etherape-0.9.0/src/menus.h	2003-08-21 12:00:22.000000000 -0400
@@ -46,6 +46,7 @@
 void on_start_menuitem_activate (GtkMenuItem * menuitem, gpointer user_data);
 void on_stop_menuitem_activate (GtkMenuItem * menuitem, gpointer user_data);
 
+void on_full_screen_activate (GtkCheckMenuItem * menuitem, gpointer user_data);
 void on_toolbar_check_activate (GtkCheckMenuItem * menuitem,
 				gpointer user_data);
 void on_legend_check_activate (GtkCheckMenuItem * menuitem,
