From 9743d0d6927da3c711a956c34bed4bdef8fe52ff Mon Sep 17 00:00:00 2001
From: Romain Porte <microjoe@microjoe.org>
Date: Mon, 25 Oct 2021 12:46:15 +0200
Subject: [PATCH] Fix qemu deprecation warnings. Closes #996705

---
 lib/autopkgtest_qemu.py | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/lib/autopkgtest_qemu.py b/lib/autopkgtest_qemu.py
index 0933103..6733ede 100644
--- a/lib/autopkgtest_qemu.py
+++ b/lib/autopkgtest_qemu.py
@@ -351,7 +351,7 @@ class Qemu:
             '-object', 'rng-random,filename=/dev/urandom,id=rng0',
             '-device', 'virtio-rng-pci,rng=rng0,id=rng-device0',
             '-monitor',
-            'unix:%s,server,nowait' % self.get_socket_path('monitor'),
+            'unix:%s,server=on,wait=off' % self.get_socket_path('monitor'),
             '-virtfs',
             (
                 'local,id=autopkgtest,path=%s,security_model=none,'
@@ -365,12 +365,12 @@ class Qemu:
                 # the VM on ppc64le, so identify it as such
                 argv.extend([
                     '-serial',
-                    'unix:%s,server,nowait' % self.get_socket_path('hvc0'),
+                    'unix:%s,server=on,wait=off' % self.get_socket_path('hvc0'),
                 ])
             else:
                 argv.extend([
                     '-chardev',
-                    'socket,path=%s,server,nowait,id=%s' % (
+                    'socket,path=%s,server=on,wait=off,id=%s' % (
                         self.get_socket_path(hvc), hvc,
                     ),
                     '-device', 'virtio-serial',
@@ -382,9 +382,9 @@ class Qemu:
             self.consoles.add('ttyS1')
             argv.extend([
                 '-serial',
-                'unix:%s,server,nowait' % self.get_socket_path('ttyS0'),
+                'unix:%s,server=on,wait=off' % self.get_socket_path('ttyS0'),
                 '-serial',
-                'unix:%s,server,nowait' % self.get_socket_path('ttyS1'),
+                'unix:%s,server=on,wait=off' % self.get_socket_path('ttyS1'),
             ])
         elif self.qemu_architecture == 'ppc64le':
             # There are no emulated hardware serial ports in this qemu
@@ -396,7 +396,7 @@ class Qemu:
             self.consoles.add('ttyS0')
             argv.extend([
                 '-serial',
-                'unix:%s,server,nowait' % self.get_socket_path('ttyS0'),
+                'unix:%s,server=on,wait=off' % self.get_socket_path('ttyS0'),
             ])
 
         for i, image in enumerate(self.images):
-- 
2.33.0

