diff -ur debiandoc-sgml-1.2.25.orig/debian/control debiandoc-sgml-1.2.25/debian/control
--- debiandoc-sgml-1.2.25.orig/debian/control	2011-03-17 11:13:29.000000000 +0000
+++ debiandoc-sgml-1.2.25/debian/control	2012-03-22 14:53:27.000000000 +0000
@@ -13,6 +13,7 @@
 Depends: ${perl:Depends}, ${misc:Depends}, libhtml-parser-perl, libroman-perl, libtext-format-perl, sgml-data, sgmlspl, sp
 Recommends: ghostscript, texlive, texlive-latex-extra, texinfo 
 Suggests: debiandoc-sgml-doc, latex-cjk-all, texlive-lang-all
+Multi-Arch: foreign
 Description: DebianDoc SGML DTD and formatting tools
  This is an SGML-based documentation formatting package used for the
  Debian manuals.  It reads markup files and produces HTML, LaTeX, DVI 
