From 2980bbe7e90dec0ea6825b441be33f176d0ffa96 Mon Sep 17 00:00:00 2001
From: Hideki Yamane <henrich@debian.org>
Date: Sun, 17 Dec 2017 14:39:45 +0900
Subject: [PATCH 1/2] fix typo

unncessary -> unn"e"cessary
---
 debhelper.pod | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debhelper.pod b/debhelper.pod
index 0e45a4d2..dd73c3f9 100644
--- a/debhelper.pod
+++ b/debhelper.pod
@@ -250,7 +250,7 @@ If neither option is specified, debhelper currently defaults to
 B<--parallel> in compat 10 (or later) and B<--no-parallel> otherwise.
 
 As an optimization, B<dh> will try to avoid passing these options to
-subprocesses, if they are unncessary and the only options passed.
+subprocesses, if they are unnecessary and the only options passed.
 Notably this happens when B<DEB_BUILD_OPTIONS> does not have a
 I<parallel> parameter (or its value is 1).
 
-- 
2.15.1

