Index: initramfs-cryptroot-script
===================================================================
--- initramfs-cryptroot-script	(revision 193)
+++ initramfs-cryptroot-script	(working copy)
@@ -56,9 +56,7 @@
 	# Start by parsing some options, all options are relevant to regular cryptsetup
 	# but only crypttarget and cryptsource is relevant to luks which picks up the 
 	# rest of the parameters by reading the partition header
-	cryptcipher=aes-cbc-essiv:sha256
-	cryptsize=256
-	crypthash=sha256
+	cryptsetupopts=""
 	crypttarget=cryptroot
 	cryptsource=$ROOT
 	cryptlvm=""
@@ -68,13 +66,13 @@
 		for x in $cryptopts; do
 			case $x in
 			hash=*)
-				crypthash=${x#hash=}
+				cryptsetupopts="$cryptsetupopts -h ${x#hash=}"
 				;;
 			size=*)
-				cryptsize=${x#size=}
+				cryptsetupopts="$cryptsetupopts -s ${x#size=}"
 				;;
 			cipher=*)
-				cryptcipher=${x#cipher=}
+				cryptsetupopts="$cryptsetupopts -c ${x#cipher=}"
 				;;
 			target=*)
 				crypttarget=${x#target=}
@@ -121,7 +119,7 @@
 	cryptcreate="/sbin/cryptsetup luksOpen $cryptsource $crypttarget"
 	cryptremove=""
 else
-	cryptcreate="/sbin/cryptsetup -c $cryptcipher -s $cryptsize -h $crypthash create $crypttarget $cryptsource"
+	cryptcreate="/sbin/cryptsetup $cryptsetupopts create $crypttarget $cryptsource"
 	cryptremove="/sbin/cryptsetup remove $crypttarget"
 fi
 NEWROOT="/dev/mapper/$crypttarget"
