--- dput.cf.orig	2013-03-25 07:58:27.814707146 +0900
+++ dput.cf	2013-03-25 07:59:21.350862311 +0900
@@ -75,12 +75,6 @@
 # Please be sure to know what you are doing before taking it out.
 pre_upload_command	= /usr/share/dput/helper/security-warning
 
-[jp]
-fqdn			= hp.debian.or.jp
-method			= ftp
-incoming		= /pub/Incoming/upload
-login			= anonymous
-
 [backports]
 fqdn			= backports-master.debian.org
 method			= ftp
