diff -urN origs/base-passwd-3.5.22//debian/rules patched/base-passwd-3.5.22//debian/rules
--- origs/base-passwd-3.5.22//debian/rules	2009-09-07 00:05:37.000000000 +0100
+++ patched/base-passwd-3.5.22//debian/rules	2011-06-26 20:01:59.171524996 +0100
@@ -13,8 +13,10 @@
 
 ifeq ($(DEB_BUILD_GNU_TYPE),$(DEB_HOST_GNU_TYPE))
   confflags += --build=$(DEB_HOST_GNU_TYPE)
+  STRIP = strip
 else
   confflags += --build=$(DEB_BUILD_GNU_TYPE) --host=$(DEB_HOST_GNU_TYPE)
+  STRIP = $(DEB_HOST_GNU_TYPE)-strip
 endif
 
 .PHONY: all build
@@ -58,7 +60,7 @@
 	install -p -m 755 -o root -g root update-passwd debian/tmp/usr/sbin/
 
 ifeq (,$(filter nostrip,$(DEB_BUILD_OPTIONS)))
-	strip --strip-unneeded --remove-section=.comment \
+	$(STRIP) --strip-unneeded --remove-section=.comment \
 		--remove-section=.note debian/tmp/usr/sbin/update-passwd
 endif
 
