Index: new-maintainer.dbk
===================================================================
--- new-maintainer.dbk	(revisione 8928)
+++ new-maintainer.dbk	(copia locale)
@@ -155,7 +155,7 @@
 </para>
 <para>
 You need a version 4 key for use in Debian Development.  Your key length must
-be at least 1024 bits; there is no reason to use a smaller key, and doing so
+be greater than 1024 bits; there is no reason to use a smaller key, and doing so
 would be much less secure.<footnote><para> Version 4 keys are keys conforming
 to the OpenPGP standard as defined in RFC 2440.  Version 4 is the key type that
 has always been created when using GnuPG.  PGP versions since 5.x also could
