diff -Nru bip-0.9.3/debian/bip.install bip-0.9.3/debian/bip.install
--- bip-0.9.3/debian/bip.install	2022-03-21 17:19:31.000000000 +0100
+++ bip-0.9.3/debian/bip.install	2024-07-08 16:24:58.000000000 +0200
@@ -1,6 +1,6 @@
 samples/bip.conf /etc/bip/
 scripts/bipgenconfig /usr/bin
-systemd/bip-config.service /lib/systemd/system
-systemd/bip.service /lib/systemd/system
+systemd/bip-config.service /usr/lib/systemd/system
+systemd/bip.service /usr/lib/systemd/system
 systemd/bip_env.sh /usr/lib/systemd/scripts/
-debian/bip-config.conf /lib/systemd/system/bip-config.service.d/
+debian/bip-config.conf /usr/lib/systemd/system/bip-config.service.d/
diff -Nru bip-0.9.3/debian/changelog bip-0.9.3/debian/changelog
--- bip-0.9.3/debian/changelog	2022-03-21 17:19:31.000000000 +0100
+++ bip-0.9.3/debian/changelog	2024-07-08 16:25:12.000000000 +0200
@@ -1,3 +1,10 @@
+bip (0.9.3-1.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Install systemd units into /usr. (Closes: #1073686)
+
+ -- Chris Hofstaedtler <zeha@debian.org>  Mon, 08 Jul 2024 16:25:12 +0200
+
 bip (0.9.3-1) unstable; urgency=medium
 
   [ Pierre-Louis Bonicoli ]
