From b21a596e11a2f5daf9ef4886a81911519ac4337e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Santiago=20Ruano=20Rinc=C3=B3n?= <santiagorr@riseup.net>
Date: Wed, 10 Jan 2024 22:11:21 -0500
Subject: [PATCH] autopkgtest-build-qemu pass MIRROR to setup-testbed

---
 tools/autopkgtest-build-qemu | 1 +
 1 file changed, 1 insertion(+)

diff --git a/tools/autopkgtest-build-qemu b/tools/autopkgtest-build-qemu
index c3938b0..a7d135c 100755
--- a/tools/autopkgtest-build-qemu
+++ b/tools/autopkgtest-build-qemu
@@ -587,6 +587,7 @@ class BuildQemu:
                     'shell': (
                         'export AUTOPKGTEST_BUILD_QEMU=1; ' +
                         'export RELEASE=' + shlex.quote(release) + '; ' +
+                        'export MIRROR=' + shlex.quote(mirror) + '; ' +
                         shlex.quote(s) + ' "$ROOT"'
                     ),
                     'root-fs': 'root',
-- 
2.43.0

