From f09d06ad3f76acaed755bd037fc0a757cea7e9a2 Mon Sep 17 00:00:00 2001
From: Simon McVittie <smcv@debian.org>
Date: Sat, 21 May 2016 15:27:06 +0100
Subject: [PATCH 3/7] Move from Section: web to Section: admin

---
 debian/changelog | 1 +
 debian/control   | 2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/debian/changelog b/debian/changelog
index 3fa6392..2d464a3 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -3,6 +3,7 @@ bubblewrap (0~git160513-3) UNRELEASED; urgency=medium
   * debian/copyright: correct package name and source
   * debian/control: make the whole package Linux-only. Like Flatpak, this
     package is inherently non-portable.
+  * Move from Section: web to Section: admin
 
  -- Simon McVittie <smcv@debian.org>  Sat, 21 May 2016 15:10:56 +0100
 
diff --git a/debian/control b/debian/control
index 0198270..e0ad3d5 100644
--- a/debian/control
+++ b/debian/control
@@ -1,5 +1,5 @@
 Source: bubblewrap
-Section: web
+Section: admin
 Priority: extra
 Maintainer: Laszlo Boszormenyi (GCS) <gcs@debian.org>
 Build-Depends: debhelper (>= 9), dh-autoreconf, pkg-config, libselinux1-dev (>= 2.1.9), libcap-dev, bash-completion, xsltproc, docbook-xsl
-- 
2.8.1

