--- etc.bash_completion.d.dlocate-completion.orig	2020-01-05 12:41:29.337112572 +0000
+++ /etc/bash_completion.d/dlocate-completion	2020-01-05 12:40:02.324835122 +0000
@@ -19,9 +19,13 @@
 		return 0
 		;;
 	*)
-		COMPREPLY=( $( compgen -W '-h -H -V -S -L -l -s -ls -du \
-			-conf -lsconf -md5sum -md5check -man --help \
+		COMPREPLY=( $( compgen -W '-h -H -K -k -V -S -L -l -s \
+		  -f -p -E -F -G -P -w -i -- -v -c \
+		  --ls --du --conf --lsconf --md5sum --md5check --man --lsman --lsbin --lsdir --help \
+			--columns --verbose --debug --version \
+			--filename-only --package-only --extended-regexp --fixed-strings --basic-regexp --perl-regexp --word-regexp --ignore-case \
 			--version' -- $cur ) )
+		_filedir
 		return 0
 		;;
 	esac
