diff -Nru doxygen-1.7.6.1/debian/control doxygen-1.7.6.1/debian/control
--- doxygen-1.7.6.1/debian/control	2012-01-07 13:58:17.000000000 +0200
+++ doxygen-1.7.6.1/debian/control	2012-02-07 04:06:34.000000000 +0200
@@ -8,6 +8,7 @@
 
 Package: doxygen
 Architecture: any
+Multi-Arch: foreign
 Depends: ${shlibs:Depends}, ${misc:Depends}
 Recommends: doxygen-latex
 Suggests: doxygen-doc, doxygen-gui, graphviz
@@ -24,6 +25,7 @@
 
 Package: doxygen-latex
 Architecture: all
+Multi-Arch: foreign
 Depends: doxygen (>= ${source:Version}), ${misc:Depends}, texlive-extra-utils, texlive-latex-extra, latex-xcolor, texlive-font-utils, ghostscript, texlive-fonts-recommended
 Description: Documentation system for C, C++, Java, Python and other languages
  Doxygen is a documentation system for C, C++, Java, Objective-C, Python, IDL
