Index: po4a/po4a.cfg
===================================================================
--- po4a/po4a.cfg	(révision 0)
+++ po4a/po4a.cfg	(révision 0)
@@ -0,0 +1,5 @@
+[po_directory] po4a/po/
+
+[type: sgml] debian-faq.sgml $lang:$lang/debian-faq.sgml \
+		add_$lang:@po4a/add_$lang/$lang.add \
+		opt:ignore-inclusion=faqdynamic
Index: po4a/po/debian-faq.pot
===================================================================
--- po4a/po/debian-faq.pot	(révision 0)
+++ po4a/po/debian-faq.pot	(révision 0)
@@ -0,0 +1 @@
+
Index: Makefile
===================================================================
--- Makefile	(révision 7525)
+++ Makefile	(copie de travail)
@@ -12,7 +12,7 @@
 # this can and will be overridden by a higher level makefile
 PUBLISHDIR := /org/www.debian.org/www/doc/manuals
 
-sources := $(manual).sgml faqstatic.ent faqdynamic.ent $(wildcard *.sgml)
+sources := $(manual).sgml faqstatic.ent faqdynamic.ent $(wildcard *.sgml) update-po
 
 langs ?= ru pt_BR it pl es fr de uk zh_CN
 
@@ -43,6 +43,9 @@
           echo "<!entity docversion \"CVS\">" >> faqdynamic.ent; \
         fi
 
+update-po:
+	po4a --force --rm-backups po4a/po4a.cfg
+
 validate:
 	nsgmls -ges -wall $(manual).sgml
 
Index: fr/Makefile
===================================================================
--- fr/Makefile	(révision 7525)
+++ fr/Makefile	(copie de travail)
@@ -58,7 +58,7 @@
 
 clean distclean:
 	rm -rf $(manual)*.$(currentlang).html
-	rm -f $(manual)*.{txt,ps,dvi,pdf,info*,aux,log,man,tex,toc,sasp*}
+	rm -f $(manual)*
 	rm -f *~ .*~ core tsa*
 
 .PHONY: all publish clean distclean validate
Index: debian/control
===================================================================
--- debian/control	(révision 7525)
+++ debian/control	(copie de travail)
@@ -7,8 +7,8 @@
 Build-Depends-Indep: debiandoc-sgml, texlive (>= 2007-11), texlive-latex-extra,
  texlive-lang-cyrillic, latex-cjk-chinese,
  latex-cjk-chinese-arphic-bkai00mp, latex-cjk-chinese-arphic-bsmi00lp,
- latex-cjk-chinese-arphic-gbsn00lp, latex-cjk-chinese-arphic-gkai00mp
-
+ latex-cjk-chinese-arphic-gbsn00lp, latex-cjk-chinese-arphic-gkai00mp,
+ po4a (>= 0.39)
 Standards-Version: 3.7.3
 Vcs-Browser: http://svn.debian.org/wsvn/ddp/manuals/trunk/debian-faq/
 Vcs-Svn: svn://svn.debian.org/ddp/manuals/trunk/debian-faq/
