diff --git a/functions b/functions
index c1e938e..ff733ce 100644
--- a/functions
+++ b/functions
@@ -1007,9 +1007,6 @@ setup_proc () {
 	    hurd*)
 		;;
 	    *)
-		umount_on_exit /dev/pts
-		umount_on_exit /dev/shm
-		umount_on_exit /proc/bus/usb
 		umount_on_exit /proc
 		umount "$TARGET/proc" 2>/dev/null || true
 		in_target mount -t proc proc /proc
@@ -1022,7 +1019,6 @@ setup_proc () {
 		on_exit clear_mtab
 		;;
 	esac
-	umount_on_exit /lib/init/rw
 }
 
 setup_proc_fakechroot () {
