diff -ur audacious-2.4.4/src/audacious/plugin-registry.c audacious-2.4.4.modified/src/audacious/plugin-registry.c
--- audacious-2.4.4/src/audacious/plugin-registry.c	2011-02-22 18:29:02.000000000 +0100
+++ audacious-2.4.4.modified/src/audacious/plugin-registry.c	2012-01-24 00:20:27.000000000 +0100
@@ -156,9 +156,13 @@
 
 static FILE * open_registry_file (const gchar * mode)
 {
-    gchar path[PATH_MAX];
-    snprintf (path, sizeof path, "%s/" FILENAME, aud_paths[BMP_PATH_USER_DIR]);
-    return fopen (path, mode);
+    FILE *file;
+    gchar *path = NULL;
+
+    path = g_strdup_printf ("%s/" FILENAME, aud_paths[BMP_PATH_USER_DIR]);
+    file = fopen (path, mode);
+    g_free (path);
+    return file;
 }
 
 static void input_plugin_save (PluginHandle * plugin, FILE * handle)
