--- cryptroot.orig	2009-10-19 13:55:35.000000000 +0200
+++ cryptroot	2009-11-16 15:53:51.000000000 +0100
@@ -316,7 +316,7 @@
 				return 1
 			fi
 
-			NEWROOT="/dev/mapper/$cryptlvm"
+			NEWROOT=${cmdline_root=/dev/mapper/$cryptlvm}
 			if [ "$cryptrootdev" = "yes" ]; then
 				# required for lilo to find the root device
 				echo "ROOT=$NEWROOT" >> /conf/param.conf
@@ -344,18 +344,29 @@
 #
 
 # Do we have any kernel boot arguments?
-found=''
+cmdline_cryptopts=''
+cmdline_root=''
 for opt in $(cat /proc/cmdline); do
 	case $opt in
 	cryptopts=*)
-		found=yes
-		setup_mapping "${opt#cryptopts=}"
+      	        opt="${opt#cryptopts=}"
+		if [ -n "$opt" ]; then
+		    if [ -n "cmdline_cryptopts" ]; then
+			cmdline_cryptopts="$cmdline_cryptopts,$opt"
+		    else
+			cmdline_cryptopts="$opt"
+		    fi
+		fi
 		;;
+	root=*)
+	        cmdline_root="${opt#root=}"
+	        ;;
 	esac
 done
 
-if [ -n "$found" ]; then
-	exit 0
+if [ -n "$cmdline_cryptopts" ]; then
+    setup_mapping "$cmdline_cryptopts"
+    exit 0
 fi
 
 # Do we have any settings from the /conf/conf.d/cryptroot file?
