--- cryptroot.orig	2009-09-25 12:06:15.000000000 +0200
+++ cryptroot	2009-09-25 15:45:47.000000000 +0200
@@ -298,7 +298,7 @@
 				return 1
 			fi
 
-			NEWROOT="/dev/mapper/$cryptlvm"
+			NEWROOT=${cmdline_root=/dev/mapper/$cryptlvm}
 			if [ "$cryptrootdev" = "yes" ]; then
 				# required for lilo to find the root device
 				echo "ROOT=$NEWROOT" >> /conf/param.conf
@@ -326,18 +326,22 @@
 #
 
 # Do we have any kernel boot arguments?
-found=''
+cmdline_cryptopts=''
+cmdline_root=''
 for opt in $(cat /proc/cmdline); do
 	case $opt in
 	cryptopts=*)
-		found=yes
-		setup_mapping "${opt#cryptopts=}"
+	        cmdline_cryptopts="${opt#cryptopts=}"
 		;;
+	root=*)
+	        cmdline_root="${opt#root=}"
+	        ;;
 	esac
 done
 
-if [ -n "$found" ]; then
-	exit 0
+if [ -n "$cmdline_cryptopts" ]; then
+    setup_mapping "$cmdline_cryptopts"
+    exit 0
 fi
 
 # Do we have any settings from the /conf/conf.d/cryptroot file?
