diff --git a/debian/initramfs/cryptroot-script b/debian/initramfs/cryptroot-script
index f83e52e..9de28a4 100644
--- a/debian/initramfs/cryptroot-script
+++ b/debian/initramfs/cryptroot-script
@@ -326,7 +326,7 @@ setup_mapping()
 				return 1
 			fi
 
-			NEWROOT="/dev/mapper/$cryptlvm"
+			NEWROOT=${cmdline_root=/dev/mapper/$cryptlvm}
 			if [ "$cryptrootdev" = "yes" ]; then
 				# required for lilo to find the root device
 				echo "ROOT=$NEWROOT" >> /conf/param.conf
@@ -354,18 +354,35 @@ setup_mapping()
 #
 
 # Do we have any kernel boot arguments?
-found=''
+cmdline_cryptopts=''
+unset cmdline_root
 for opt in $(cat /proc/cmdline); do
 	case $opt in
 	cryptopts=*)
-		found=yes
-		setup_mapping "${opt#cryptopts=}"
+      	        opt="${opt#cryptopts=}"
+		if [ -n "$opt" ]; then
+		    if [ -n "$cmdline_cryptopts" ]; then
+			cmdline_cryptopts="$cmdline_cryptopts,$opt"
+		    else
+			cmdline_cryptopts="$opt"
+		    fi
+		fi
 		;;
+        root=*)
+	        opt="${opt#root=}"
+		case $opt in
+		    /*) # Absolute path given. Not lilo major/minor number.
+			cmdline_root=$opt
+			;;
+		    *) # lilo major/minor number (See #398957). Ignore
+		esac
+	        ;;
 	esac
 done
 
-if [ -n "$found" ]; then
-	exit 0
+if [ -n "$cmdline_cryptopts" ]; then
+    setup_mapping "$cmdline_cryptopts"
+    exit 0
 fi
 
 # Do we have any settings from the /conf/conf.d/cryptroot file?
