--- /usr/bin/dh_pycentral	2006-08-13 21:23:30.000000000 +0200
+++ dh_pycentral	2006-10-01 12:48:13.000000000 +0200
@@ -52,8 +52,8 @@
 foreach my $package (@{$dh{DOPACKAGES}}) {
 	my $tmp = tmpdir($package);
 
-	# Move *.py files if needed
-	doit("pycentral debhelper $package $tmp");
+	# Move *.py files if needed and generate dependencies
+	doit("pycentral debhelper --substvars=file --provides $package $tmp");
 
 	# Check that we have *.py files!
 	my $found = 0;
@@ -64,7 +64,6 @@
 	}, $tmp;
 
 	if ($found or -d "$tmp/usr/share/pycentral") {
-                addsubstvar($package, "python:Depends", "python-central", ">= 0.5");
                 if (! $dh{NOSCRIPTS}) {
                         autoscript($package,"postinst","postinst-pycentral","s%#PACKAGE#%$package%");
                         autoscript($package,"prerm","prerm-pycentral","s%#PACKAGE#%$package%");
