--- /usr/share/colormake/colormake.pl.bak	2014-07-09 13:39:56.000000000 +0100
+++ /usr/share/colormake/colormake.pl	2014-07-09 13:41:33.000000000 +0100
@@ -88,7 +88,7 @@
 	{
 		$in = 'make';
 	}
-	elsif ($thisline =~ s/^(\s*(libtool:\s*)?((compile|link):\s*)?(([[:ascii:]]+-)?g?cc|(g|c)\+\+).*)$/$col_gcc$1$col_norm/)
+	elsif ($thisline =~ s/^(\s*(libtool:\s*)?((compile|link):\s*)?(([[:ascii:]]+-)?(g?cc|(g|c)\+\+)).*)$/$col_gcc$1$col_norm/)
 	{
 		$in = 'gcc';
 	}
