--- src/Makefile.am	2002-11-19 00:10:53.000000000 +0800
+++ src/Makefile.am	2008-01-29 22:50:40.173117520 +0800
@@ -27,7 +27,8 @@
 INCLUDES =  \
 	-I$(top_srcdir)/intl \
 	$(GNOME_INCLUDEDIR) \
-	$(GDK_PIXBUF_CFLAGS)
+	$(GDK_PIXBUF_CFLAGS) \
+	@GNOME_CFLAGS@
 
 bin_PROGRAMS = mathwar
 
@@ -47,4 +48,5 @@
 	$(GNOME_LIBDIR) \
 	$(GNOMEUI_LIBS) \
 	$(INTLLIBS) \
-	$(GDK_PIXBUF_LIBS)
+	$(GDK_PIXBUF_LIBS) \
+	@GNOME_LIBS@
