=== modified file 'NtfsConfig/Fstab/FstabHandler.py'
--- NtfsConfig/Fstab/FstabHandler.py	2010-07-30 23:24:29 +0000
+++ NtfsConfig/Fstab/FstabHandler.py	2012-09-08 20:27:52 +0000
@@ -516,7 +516,7 @@
             name is default to the date of saving '''
     
         if not name :
-            name = time.time()
+            name = str(time.time())
         if self.logchanges and self.lastsave_has_changed() :
             logging.debug("Save log as %s", name)
             self._logconf.add_section(name)



