diff --git a/debian/patches/50Fix_cve.dpatch b/debian/patches/50Fix_cve.dpatch
index 3d44ddb..45b9db7 100644
--- a/debian/patches/50Fix_cve.dpatch
+++ b/debian/patches/50Fix_cve.dpatch
@@ -19,4 +19,4 @@ diff -urNad '--exclude=CVS' '--exclude=.svn' '--exclude=.git' '--exclude=.arch'
 -                python $APPNAME.pyc "$@"
 -        fi
 -    fi
-+%python% %prefix%/share/catfish/bin/catfish.py "$@"
++python %prefix%/share/catfish/catfish.py "$@"
