--- a/kiss/kissattach.c
+++ b/kiss/kissattach.c
@@ -1,5 +1,6 @@
+#define _DEFAULT_SOURCE
+#define _XOPEN_SOURCE 500
 #include <stdio.h>
-#define __USE_XOPEN
 #include <stdlib.h>
 #include <string.h>
 #include <unistd.h>
--- a/kiss/kissnetd.c
+++ b/kiss/kissnetd.c
@@ -9,8 +9,9 @@
  * F1OAT 960804 - Frederic RIBLE
  */
  
+#define _DEFAULT_SOURCE
+#define _XOPEN_SOURCE 500
 #include <stdio.h>
-#define __USE_XOPEN
 #include <stdlib.h>
 #include <sys/time.h>
 #include <sys/types.h>
--- a/kiss/mkiss.c
+++ b/kiss/mkiss.c
@@ -37,8 +37,9 @@
  * 1.08 xx/xx/99 Tom Mazouch - Adjustable poll interval
  */
 
+#define _DEFAULT_SOURCE
+#define _XOPEN_SOURCE 500
 #include <stdio.h>
-#define __USE_XOPEN
 #include <string.h>
 #include <errno.h>
 #include <stdlib.h>
--- a/kiss/net2kiss.c
+++ b/kiss/net2kiss.c
@@ -30,8 +30,9 @@
 
 /*****************************************************************************/
 
+#define _DEFAULT_SOURCE
+#define _XOPEN_SOURCE 500
 #include <stdio.h>
-#define __USE_XOPEN
 #include <stdlib.h>
 #include <sys/types.h>
 #include <sys/time.h>
--- a/6pack/m6pack.c
+++ b/6pack/m6pack.c
@@ -33,8 +33,9 @@
  *********************************************************************
  */
 
+#define _DEFAULT_SOURCE
+#define _XOPEN_SOURCE 500
 #include <stdio.h>
-#define __USE_XOPEN
 #include <string.h>
 #include <errno.h>
 #include <stdlib.h>
