From 90fd20ad500252a9f79b94023d7783861b0e5c43 Mon Sep 17 00:00:00 2001
From: intrigeri <intrigeri@debian.org>
Date: Tue, 22 Jan 2013 16:04:00 +0100
Subject: [PATCH 2/2] cairo (1.12.2-2+deb7u1)

---
 debian/changelog |   12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/debian/changelog b/debian/changelog
index f27991c..5efde1d 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,15 @@
+cairo (1.12.2-2+deb7u1) UNRELEASED; urgency=low
+
+  * Fix Evince crash when printing certain PDF files (Closes: #682673)
+    Cherry-picked from upstream:
+      - 5dd0f2e cff subsetting: widths can be floating point
+      - 49c8e1b cff: use correct size for buffer
+      - 77106a0 cff: convert '.' to locale specific decimal point before
+        using sscanf
+      - 2f1d6b2 cff-subsetting: Ignore charset for non cid fonts
+
+ -- intrigeri <intrigeri@debian.org>  Tue, 22 Jan 2013 16:02:56 +0100
+
 cairo (1.12.2-2) unstable; urgency=low
 
   * debian/libcairo2-udeb.install:
-- 
1.7.10.4

