From f611692db8fc8c517e143d9ce696b495d428fcf8 Mon Sep 17 00:00:00 2001
From: Mathieu Parent <math.parent@gmail.com>
Date: Sat, 5 Nov 2016 22:16:27 +0100
Subject: [PATCH 5/5] Release version 23+nmu3

---
 debian/changelog | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/debian/changelog b/debian/changelog
index 459ae3b..dfa4a92 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,11 @@
+dh-lua (23+nmu3) unstable; urgency=medium
+
+  * Non-maintainer upload
+  * Add support for luasandbox (i.e. lua embedded in libluasandbox0)
+    (Closes: #844692)
+
+ -- Mathieu Parent <sathieu@debian.org>  Wed, 30 Nov 2016 07:40:58 +0100
+
 dh-lua (23+nmu2) unstable; urgency=medium
 
   * Non-maintainer upload.
-- 
2.10.2

