diff -ur debian-policy-3.7.2.2.old/policy.sgml debian-policy-3.7.2.2/policy.sgml
--- debian-policy-3.7.2.2.old/policy.sgml	2006-10-03 00:36:50.000000000 +0200
+++ debian-policy-3.7.2.2/policy.sgml	2007-06-30 10:55:06.000000000 +0200
@@ -8625,15 +8625,14 @@
 
 	<p>
 	  Packages distributed under the UCB BSD license, the Artistic
-	  license, the GNU GPL, and the GNU LGPL, should refer to the
-	  corresponding files under
+	  license, the GNU GPL or LGPL (any version as published by the Free
+	  Software Foundation that Debian considers free), should refer to
+	  the corresponding files under
 	  <file>/usr/share/common-licenses</file>,<footnote>
             <p>
               For example,
               <file>/usr/share/common-licenses/Artistic</file>,
               <file>/usr/share/common-licenses/BSD</file>,
-              <file>/usr/share/common-licenses/GPL</file>,
-              <file>/usr/share/common-licenses/LGPL</file>,
               <file>/usr/share/common-licenses/GFDL</file>,
               <file>/usr/share/common-licenses/GPL-2</file>, and
               <file>/usr/share/common-licenses/LGPL-2.1</file>, and so
@@ -8642,7 +8641,20 @@
               <file>/usr/share/common-licenses/GFDL</file>. 
             </p>
           </footnote> rather than quoting them in the copyright
-	  file. 
+	  file.  Packages under a fixed, definite version of the GPL or LGPL
+	  should refer to the versioned GPL or LGPL file in
+	  <file>/usr/share/common-licenses</file>.  Packages that are licensed under a fixed
+	  version of the GPL or LGPL, but giving the licensee the option to
+	  adhere to terms of any later version of the license, should refer to
+	  the unversioned GPL or LGPL file in <file>/usr/share/common-licenses</file>, making
+	  it clear which versions may be used.<footnote>
+	    <p>
+	      For example, the copyright file might read:
+	      <example>
+  This program is free software. It is under GPL version 2 or later. On Debian
+  systems, the latest version of the GPL is in /usr/share/common-licenses/GPL.
+	      </example>
+	    </p></footnote>
 	</p>
 
 	<p>
