diff -ru orig/analog-5.22/src/Makefile analog-5.22/src/Makefile
--- orig/analog-5.22/src/Makefile	Thu Mar 28 12:45:38 2002
+++ analog-5.22/src/Makefile	Thu Mar 28 12:21:29 2002
@@ -26,7 +26,7 @@
 #
 OS = UNIX         # Operating system: UNIX, DOS, WIN32, MAC, OS2, OSX, VMS
 #                   RISCOS, BEOS, NEXTSTEP, MPEIX, BS2000, AS400, OS390
-LIBS = -lm -lpng -ljpeg -lz # extra libraries needed; almost all platforms (but not OS X or BeOS)
+LIBS = -lm -lgd -lpng -ljpeg -lz # extra libraries needed; almost all platforms (but not OS X or BeOS)
 #               need -lm LAST
 #             if you defined HAVE_GD above you also need -lgd -lpng -ljpeg -lz
 #             if you defined HAVE_PCRE above you also need -lpcre
@@ -46,16 +46,14 @@
 OBJS = alias.o analog.o cache.o dates.o globals.o hash.o init.o init2.o \
 	input.o macinput.o macstuff.o output.o output2.o process.o \
 	settings.o sort.o tree.o utils.o win32.o
-SUBDIRS = libgd pcre
-SUBDIROBJS = libgd/gdfontf.o libgd/gd.o  libgd/gd_io.o libgd/gd_io_file.o libgd/gd_png.o \
-	libgd/gdfonts.o libgd/gdtables.o \
-	pcre/pcre.o
+SUBDIRS = pcre
+SUBDIROBJS = pcre/pcre.o
 #	libpng/pngwtran.o libpng/pngwutil.o pcre/pcre.o \
 #	libpng/png.o libpng/pngerror.o libpng/pngmem.o libpng/pngset.o \
 #	libpng/pngtrans.o libpng/pngwio.o libpng/pngwrite.o \
 #	zlib/adler32.o zlib/crc32.o  zlib/deflate.o zlib/trees.o zlib/zutil.o
 HEADERS = anlghead.h anlghea2.h anlghea3.h anlghea4.h macdir.h \
-	pcre/pcre.h libgd/gd.h libgd/gdfontf.h libgd/gdfonts.h
+	pcre/pcre.h
 ALLCFLAGS = $(CFLAGS) $(DEFS) -D$(OS)
 ALLOBJS = $(OBJS) $(SUBDIROBJS)
 
@@ -67,8 +65,8 @@
 
 # There doesn't seem to be a good way to write all these rules in a generic
 # form that works for all "make" programs
-libgd: ALWAYS
-	cd libgd && $(MAKE) 'CC=$(CC)' 'ALLCFLAGS=$(ALLCFLAGS)'
+#libgd: ALWAYS
+#	cd libgd && $(MAKE) 'CC=$(CC)' 'ALLCFLAGS=$(ALLCFLAGS)'
 
 #libpng: ALWAYS
 #	cd libpng && $(MAKE) 'CC=$(CC)' 'ALLCFLAGS=$(ALLCFLAGS)'
diff -ru orig/analog-5.22/src/output2.c analog-5.22/src/output2.c
--- orig/analog-5.22/src/output2.c	Wed Mar 20 01:38:52 2002
+++ analog-5.22/src/output2.c	Thu Mar 28 12:31:18 2002
@@ -2398,7 +2398,7 @@
   if (strcaseeq(lngstr[charset_], "ISO-8859-2"))
     font = gdFontSmall;
   else {
-    font = gdFontFixed;
+    font = gdFontSmall;
     if (!strcaseeq(lngstr[charset_], "ISO-8859-1") &&
 	!strcaseeq(lngstr[charset_], "US-ASCII"))
       normalchart = FALSE;
