--- evolution-data-server-1.12.0/libedataserver/e-iconv.c	2007-09-17 02:51:22.000000000 +0800
+++ evolution-data-server-1.12.0.changed/libedataserver/e-iconv.c	2007-10-19 12:03:24.000000000 +0800
@@ -135,15 +135,16 @@
 	{ "euckr-0",        "EUC-KR"     },
 	{ "5601",           "EUC-KR"     },
 	{ "zh_TW-euc",      "EUC-TW"     },
-	{ "zh_CN.euc",      "gb2312"     },
+        { "zh_CN.euc",      "gb18030"     },
 	{ "zh_TW-big5",     "BIG5"       },
-	{ "euc-cn",         "gb2312"     },
+        { "euc-cn",         "gb18030"     },
 	{ "big5-0",         "BIG5"       },
 	{ "big5.eten-0",    "BIG5"       },
 	{ "big5hkscs-0",    "BIG5HKSCS"  },
-	{ "gb2312-0",       "gb2312"     },
-	{ "gb2312.1980-0",  "gb2312"     },
-	{ "gb-2312",        "gb2312"     },
+        { "gb2312-0",       "gb18030"     },
+        { "gb2312.1980-0",  "gb18030"     },
+        { "gb-2312",        "gb18030"     },
+        { "gb2312",         "gb18030"     },
 	{ "gb18030-0",      "gb18030"    },
 	{ "gbk-0",          "GBK"        },
 
@@ -593,8 +594,8 @@
 } cjkr_lang_map[] = {
 	{ "Big5",        "zh" },
 	{ "BIG5HKSCS",   "zh" },
-	{ "gb2312",      "zh" },
 	{ "gb18030",     "zh" },
+        { "gb2312",      "zh" },
 	{ "gbk",         "zh" },
 	{ "euc-tw",      "zh" },
 	{ "iso-2022-jp", "ja" },
