=== modified file 'apt-pkg/depcache.cc'
--- apt-pkg/depcache.cc	2007-07-26 08:25:10 +0000
+++ apt-pkg/depcache.cc	2007-12-30 19:47:41 +0000
@@ -972,13 +972,14 @@
 	 {
 	    if(_config->FindB("Debug::pkgDepCache::AutoInstall",false) == true)
 	       std::clog << "Installing " << InstPkg.Name() 
-			 << " as dep of " << Pkg.Name() 
+			 << " as " << Start.DepType() << " of " << Pkg.Name() 
 			 << std::endl;
  	    // now check if we should consider it a automatic dependency or not
  	    if(Pkg.Section() && ConfigValueInSubTree("APT::Never-MarkAuto-Sections", Pkg.Section()))
  	    {
  	       if(_config->FindB("Debug::pkgDepCache::AutoInstall",false) == true)
- 		  std::clog << "Setting NOT as auto-installed (direct dep of pkg in APT::Never-MarkAuto-Section)" << std::endl;
+ 		  std::clog << "Setting NOT as auto-installed (direct "
+			    << Start.DepType() << " of pkg in APT::Never-MarkAuto-Section)" << std::endl;
  	       MarkInstall(InstPkg,true,Depth + 1, true);
  	    }
  	    else 

