Index: debian/control
===================================================================
--- debian/control	(révision 250)
+++ debian/control	(copie de travail)
@@ -5,7 +5,7 @@
 Uploaders: Loïc Minier <lool@dooz.org>,
            Fathi Boudra <fabo@debian.org>,
            Yves-Alexis Perez <corsac@debian.org>
-Build-Depends: cdbs, debhelper (>= 6)
+Build-Depends: cdbs (>= 0.4.93), debhelper (>= 8.1.3)
 # libusplash-dev, imagemagick, librsvg2-bin
 Standards-Version: 3.9.1
 Homepage: http://www.debian.org/devel/debian-desktop
Index: debian/gsettings-override
===================================================================
--- debian/gsettings-override	(révision 0)
+++ debian/gsettings-override	(révision 0)
@@ -0,0 +1,3 @@
+[org.gnome.desktop.background]
+picture-options='zoom'
+picture-uri='file:///usr/share/images/desktop-base/desktop-background'
Index: debian/changelog
===================================================================
--- debian/changelog	(révision 250)
+++ debian/changelog	(copie de travail)
@@ -1,3 +1,11 @@
+desktop-base (6.0.7) UNRELEASED; urgency=low
+
+  * Build-depend on recent cdbs+debhelper to have dh_installgsettings.
+  * debian/gsettings-override: set the default background using 
+    GSettings.
+
+ -- Josselin Mouette <joss@debian.org>  Sun, 17 Apr 2011 11:24:02 +0200
+
 desktop-base (6.0.6) unstable; urgency=low
 
   * plymouth/spacefun:
Index: debian/source.lintian-overrides
===================================================================
--- debian/source.lintian-overrides	(révision 250)
+++ debian/source.lintian-overrides	(copie de travail)
@@ -1,2 +1,2 @@
-# we ignore ${Misc:Depends} because it would bring gconf
+# we ignore ${Misc:Depends} because it would bring gconf and dconf
 desktop-base source: debhelper-but-no-misc-depends desktop-base
