diff --git a/src/makemake.in b/src/makemake.in
index f2f6db977..f8427d761 100644
--- a/src/makemake.in
+++ b/src/makemake.in
@@ -3997,6 +3997,8 @@ for f in $TXT_FILES ; do
   line="${HERE}${g}"
   test $f = clisp.1 -o $f = clisp-link.1 && \
     line=$line" | \$(GREP) -v ${ARGQ1}^ *\$\$${ARGQ1}"
+  test "$f" = clisp.1 && \
+    line=$line" | sed -e 's/\(.\)\.PP/\1\n.PP/p'"
   # *-1.html is for chunked impnotes and does not depend on user hyperspec
   test \( $f = clisp.html -o $f = clisp-link.html \) \
        -a "${HSDEFAULT}" != "${hyperspec}" && \
