--- /tmp/dpkg-buildpackage.pl	2013-07-28 11:56:47.000000000 +0200
+++ scripts/dpkg-buildpackage.pl	2013-08-31 16:25:18.904471927 +0200
@@ -531,6 +531,7 @@
     print STDERR " signfile $file\n";
     my $qfile = quotemeta($file);
 
+    $signkey = $ENV{DEBKEY} if(!defined $signkey && defined $ENV{DEBKEY});
     system("(cat ../$qfile ; echo '') | " .
            "$signcommand --utf8-strings --local-user " .
            quotemeta($signkey || $maintainer) .
