--- debfoster.c.orig	2008-04-19 17:10:40.000000000 +0200
+++ debfoster.c	2009-06-04 21:11:17.000000000 +0200
@@ -439,8 +439,8 @@
 		} else
 			count++;
 	}
-	if (!show_set(&dependents, _("The following %d packages on keeper list rely on %s:"), count, name))
-		printf(_("Packages on keeper list do not rely on %s.\n"), name);
+	if (!show_set(&dependents, _("The following %d packages on the keeper list rely on %s:"), count, name))
+		printf(_("Packages on the keeper list do not rely on %s.\n"), name);
 	printf(_("Packages kept by default rules %s %s.\n"), others ? _("rely on") : _("do not rely on"), name);
 }
 
@@ -570,7 +570,7 @@
 				printf(_("-k, --keeperfile FILE          Specify keeper file\n"));
 				printf(_("-n, --no-keeperfile            Don't read keeper file\n"));
 				printf(_("-i, --ignore-default-rules     Ignore default rules\n"));
-				printf(_("-a, --show-keepers             Show packages on keeper list\n"));
+				printf(_("-a, --show-keepers             Show packages on the keeper list\n"));
 				printf(_("-s, --show-orphans             Show orphaned packages\n"));
 				printf(_("-d, --show-depends PACKAGE     Show all depends of PACKAGE\n"));
 				printf(_("-e, --show-dependents PACKAGE  Show dependents of PACKAGE\n"));
@@ -692,7 +692,7 @@
 	/* Print information */
 	switch(show_option) {
 	case 'a': /* --show-keepers option */
-		if(!show_set(keepers, _("The following packages are on keeper list:")))
+		if(!show_set(keepers, _("The following packages are on the keeper list:")))
 			printf(_("The keeper list is empty.\n"));
 		show_set(nokeepers, _("The following packages are marked as non-keepers:"));
 		return 0;
