diff -u anacron-2.3/debian/init.d anacron-2.3/debian/init.d
--- anacron-2.3/debian/init.d
+++ anacron-2.3/debian/init.d
@@ -45,10 +45,10 @@
     log_end_msg 0
     ;;
   status)
-    exit 4
+    status_of_proc /usr/sbin/anacron anacron && exit 0 || exit $?
     ;;
   *)
-    echo "Usage: /etc/init.d/anacron {start|stop}"
+    echo "Usage: /etc/init.d/anacron {start|stop|status}"
     exit 2
     ;;
 esac
diff -u anacron-2.3/debian/control anacron-2.3/debian/control
--- anacron-2.3/debian/control
+++ anacron-2.3/debian/control
@@ -7,7 +7,7 @@
 
 Package: anacron
 Architecture: any
-Depends: ${shlibs:Depends}, debianutils (>= 1.7), ${misc:Depends}, lsb-base (>= 3.0-10)
+Depends: ${shlibs:Depends}, debianutils (>= 1.7), ${misc:Depends}, lsb-base (>= 3.2-13)
 Replaces: pe
 Recommends: cron (>= 3.0pl1-43), exim4 | mail-transport-agent, sysklogd | system-log-daemon
 Suggests: powermgmt-base
