diff -urN base-files-6.1.original/debian/changelog base-files-6.1/debian/changelog
--- base-files-6.1.original/debian/changelog	2011-04-01 09:20:54.288929294 +0100
+++ base-files-6.1/debian/changelog	2011-04-02 15:45:34.678407227 +0100
@@ -1,3 +1,9 @@
+base-files (6.1run0) unstable; urgency=low
+
+  * Add /run.
+
+ -- Roger Leigh <rleigh@debian.org>  Fri, 01 Apr 2011 09:26:35 +0100
+
 base-files (6.1) unstable; urgency=medium
 
   * Changed issue, issue.net and debian_version to read "wheezy/sid".
diff -urN base-files-6.1.original/debian/directory-list base-files-6.1/debian/directory-list
--- base-files-6.1.original/debian/directory-list	2011-04-01 09:20:54.288929294 +0100
+++ base-files-6.1/debian/directory-list	2011-04-02 15:39:40.784019553 +0100
@@ -10,6 +10,7 @@
 mnt
 proc
 root
+run
 sbin
 tmp
 usr
