diff -urN base-files-6.1.original/debian/1777-dirs base-files-6.1/debian/1777-dirs
--- base-files-6.1.original/debian/1777-dirs	2011-04-01 09:20:54.288929294 +0100
+++ base-files-6.1/debian/1777-dirs	2011-04-01 09:27:54.101365766 +0100
@@ -1,3 +1,2 @@
 tmp
-var/lock
 var/tmp
diff -urN base-files-6.1.original/debian/changelog base-files-6.1/debian/changelog
--- base-files-6.1.original/debian/changelog	2011-04-01 09:20:54.288929294 +0100
+++ base-files-6.1/debian/changelog	2011-04-01 10:25:53.877078068 +0100
@@ -1,3 +1,15 @@
+base-files (6.1run0) unstable; urgency=low
+
+  * Add /run.
+  * Remove /var/run and /var/lock.  These locations will be migrated
+    to be symbolic links to /run and /run/lock, respectively, by
+    initscripts.
+  * Create /var/run and /var/lock in postinst.  While base-files no
+    longer owns these directories, they should be present until adoped
+    by initscripts.
+
+ -- Roger Leigh <rleigh@debian.org>  Fri, 01 Apr 2011 09:26:35 +0100
+
 base-files (6.1) unstable; urgency=medium
 
   * Changed issue, issue.net and debian_version to read "wheezy/sid".
diff -urN base-files-6.1.original/debian/directory-list base-files-6.1/debian/directory-list
--- base-files-6.1.original/debian/directory-list	2011-04-01 09:20:54.288929294 +0100
+++ base-files-6.1/debian/directory-list	2011-04-01 09:21:26.049156094 +0100
@@ -10,6 +10,7 @@
 mnt
 proc
 root
+run
 sbin
 tmp
 usr
@@ -32,8 +33,6 @@
 var/lib/dpkg
 var/lib/misc
 var/local
-var/lock
 var/log
-var/run
 var/spool
 var/tmp
diff -urN base-files-6.1.original/debian/postinst.in base-files-6.1/debian/postinst.in
--- base-files-6.1.original/debian/postinst.in	2011-04-01 09:20:54.288929294 +0100
+++ base-files-6.1/debian/postinst.in	2011-04-01 10:27:39.774537564 +0100
@@ -44,6 +44,12 @@
   if [ ! -L /var/spool/mail ]; then
     ln -s ../mail /var/spool/mail
   fi
+  if [ ! -L /var/run ]; then
+    install_directory var/run   755 root
+  fi
+  if [ ! -L /var/lock ]; then
+    install_directory var/lock 1777 root
+  fi
 
   install_local_dir /usr/local
   install_local_dir /usr/local/share
