diff -urN base-files-6.1.original/debian/changelog base-files-6.2/debian/changelog
--- base-files-6.1.original/debian/changelog	2011-03-30 16:23:23.562397543 +0100
+++ base-files-6.2/debian/changelog	2011-03-30 16:35:15.875030171 +0100
@@ -1,3 +1,13 @@
+base-files (6.2) unstable; urgency=low
+
+  * Add /run.  /run is a replacement for /var/run, /var/lock and
+    /lib/init/rw, plus existing (ab)use of the writable tmpfs on /dev
+    and writing into /etc.  It will be used to mount a tmpfs prior to
+    /var being mounted, and serve as a unified location for transient
+    writable data which should not be preserved across reboots.
+
+ -- Roger Leigh <rleigh@debian.org>  Wed, 30 Mar 2011 16:29:02 +0100
+
 base-files (6.1) unstable; urgency=medium
 
   * Changed issue, issue.net and debian_version to read "wheezy/sid".
diff -urN base-files-6.1.original/debian/directory-list base-files-6.2/debian/directory-list
--- base-files-6.1.original/debian/directory-list	2011-03-30 16:23:23.562397543 +0100
+++ base-files-6.2/debian/directory-list	2011-03-30 16:24:02.178847205 +0100
@@ -10,6 +10,7 @@
 mnt
 proc
 root
+run
 sbin
 tmp
 usr
