--- debian/rules.org	2015-10-09 12:54:59.000000000 +0200
+++ debian/rules	2015-10-30 16:45:06.317724651 +0100
@@ -16,6 +16,13 @@
 export DEB_BUILD_PROFILES += noudeb
 endif
 
+# Linking systemd with the gold linker on sparc/sparc64 currently breaks
+# linking of other binaries against systemd's shared libraries due to a
+# limitation in gold on these architectures (binutils PR target/19019).
+ifneq (,$(findstring $(DEB_BUILD_ARCH), sparc sparc64))
+    LD=ld
+endif
+
 CONFFLAGS = \
 	--with-rootprefix= \
 	--with-rootlibdir=/lib/$(DEB_HOST_MULTIARCH) \
