diff -u collectd-5.1.0/debian/collectd-core.collectd.init.d collectd-5.1.0/debian/collectd-core.collectd.init.d
--- collectd-5.1.0/debian/collectd-core.collectd.init.d
+++ collectd-5.1.0/debian/collectd-core.collectd.init.d
@@ -50,12 +50,12 @@
 	. /etc/default/$NAME
 fi
 
-if test "$DISABLE" != 0 -a "$1" == "start"; then
+if test "$DISABLE" != 0 -a \( "$1" == "start" -o "$1" == "restart" \); then
 	log_warning_msg "Not starting $DESC, disabled by /etc/default/$NAME."
 	exit 0
 fi
 
-if test ! -e "$CONFIGFILE" -a "$1" == "start"; then
+if test ! -e "$CONFIGFILE" -a \( "$1" == "start" -o "$1" == "restart" \); then
 	log_warning_msg "Not starting $DESC, no configuration ($CONFIGFILE) found."
 	exit 0
 fi
diff -u collectd-5.1.0/debian/changelog collectd-5.1.0/debian/changelog
--- collectd-5.1.0/debian/changelog
+++ collectd-5.1.0/debian/changelog
@@ -1,3 +1,11 @@
+collectd (5.1.0-2.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Do not try to restart collectd when it's disabled (or missing config)
+    Closes: #681216
+
+ -- Evgeni Golov <evgeni@debian.org>  Wed, 11 Jul 2012 18:42:47 +0200
+
 collectd (5.1.0-2) unstable; urgency=low
 
   * debian/collectd-core.postinst:
