--- a/etc/init.d/dirmngr	2008-02-18 10:47:52.000000000 +0100
+++ b/etc/init.d/dirmngr	2008-08-17 12:25:24.000000000 +0200
@@ -56,7 +56,7 @@
 	;;
     stop)
 	log_daemon_msg "Stopping DirMngr" $NAME
-	killproc $DAEMON
+	killproc -p "$PIDFILE" "$DAEMON"
 	log_end_msg $?
 	;;
     status)
@@ -81,7 +81,7 @@
 	;;
     reload|force-reload)
 	log_daemon_msg "Reloading DirMngr configuration" $NAME
-	killproc $DAEMON HUP
+	killproc -p "$PIDFILE" "$DAEMON" -HUP
 	log_end_msg $?
 	;;
     *)
