diff --git a/policy.sgml b/policy.sgml
index 5760a3f..03d187e 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -1778,7 +1778,7 @@ zope.
 	    <item>yyyy is the four-digit year (e.g. 2010)</item>
 	    <item>hh is the two-digit hour (00-23)</item>
 	    <item>mm is the two-digit minutes (00-59)</item>
-	    <item>ss is the two-digit seconds (00-60)</item>
+	    <item>ss is the two-digit seconds (00-59)</item>
 	    <item>
 	      +zzzz or -zzzz is the the time zone offset from Coordinated
 	      Universal Time (UTC).  "+" indicates that the time is ahead
