diff -Nru activemq-5.5.0+dfsg/debian/changelog activemq-5.5.0+dfsg/debian/changelog
--- activemq-5.5.0+dfsg/debian/changelog	2012-04-01 20:26:30.000000000 +0200
+++ activemq-5.5.0+dfsg/debian/changelog	2012-04-29 22:24:06.000000000 +0200
@@ -1,3 +1,12 @@
+activemq (5.5.0+dfsg-7) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Install the activemq-{core,console,run} and kahadb jar files to
+    /usr/share/java.  (Closes: #668943)
+    - Add the necessary --java-lib flag to d/libactivemq-java.poms
+
+ -- Ulrich Dangel <mru@spamt.net>  Sun, 29 Apr 2012 20:49:33 +0200
+
 activemq (5.5.0+dfsg-6) unstable; urgency=low
 
   * d/patches/activemq-admin.patch: Fix activemq-admin "unexpected operator"
diff -Nru activemq-5.5.0+dfsg/debian/libactivemq-java.poms activemq-5.5.0+dfsg/debian/libactivemq-java.poms
--- activemq-5.5.0+dfsg/debian/libactivemq-java.poms	2012-04-01 20:26:30.000000000 +0200
+++ activemq-5.5.0+dfsg/debian/libactivemq-java.poms	2012-04-29 21:59:53.000000000 +0200
@@ -24,16 +24,16 @@
 #     temporarily for certain artifacts such as Javadoc jars.
 #
 pom.xml --no-parent --has-package-version
-activemq-core/pom.xml --has-package-version
-kahadb/pom.xml --has-package-version
+activemq-core/pom.xml --has-package-version --java-lib
+kahadb/pom.xml --has-package-version --java-lib
 activemq-jmdns_1.0/pom.xml --has-package-version
 activemq-jaas/pom.xml --has-package-version
 activemq-ra/pom.xml --has-package-version
 activemq-pool/pom.xml --has-package-version
 activemq-spring/pom.xml --has-package-version
 activemq-fileserver/pom.xml --has-package-version
-activemq-console/pom.xml --has-package-version
-activemq-run/pom.xml --has-package-version
+activemq-console/pom.xml --has-package-version --java-lib
+activemq-run/pom.xml --has-package-version --java-lib
 
 activemq-optional/pom.xml --ignore-pom
 activemq-karaf/pom.xml --ignore-pom
