Description: Fix build against libcap2 2.43
Author: Shengjing Zhu <zhsj@debian.org>

Bug: https://github.com/moby/moby/issues/41398
Forwarded: no
Last-Update: 2020-08-29

--- docker.io-19.03.12+dfsg1.orig/engine/pkg/archive/archive_unix_test.go
+++ docker.io-19.03.12+dfsg1/engine/pkg/archive/archive_unix_test.go
@@ -264,7 +264,7 @@ func TestTarUntarWithXattr(t *testing.T)
 		}
 		out, err := exec.Command("getcap", filepath.Join(origin, "2")).CombinedOutput()
 		assert.NilError(t, err, string(out))
-		assert.Check(t, is.Contains(string(out), "= cap_block_suspend+ep"), "untar should have kept the 'security.capability' xattr")
+		assert.Check(t, is.Contains(string(out), "cap_block_suspend=ep"), "untar should have kept the 'security.capability' xattr")
 	}
 }
 
