diff -Nru backuppc-4.4.0/debian/changelog backuppc-4.4.0/debian/changelog
--- backuppc-4.4.0/debian/changelog	2023-01-15 11:24:36.000000000 +0100
+++ backuppc-4.4.0/debian/changelog	2024-07-08 01:06:51.000000000 +0200
@@ -1,3 +1,10 @@
+backuppc (4.4.0-8.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Install systemd units into /usr. (Closes: #1073701)
+
+ -- Chris Hofstaedtler <zeha@debian.org>  Mon, 08 Jul 2024 01:06:51 +0200
+
 backuppc (4.4.0-8) unstable; urgency=medium
 
   [ Axel Beckert ]
diff -Nru backuppc-4.4.0/debian/install backuppc-4.4.0/debian/install
--- backuppc-4.4.0/debian/install	2022-12-18 03:29:49.000000000 +0100
+++ backuppc-4.4.0/debian/install	2024-07-08 01:06:51.000000000 +0200
@@ -1 +1 @@
-systemd/backuppc.service lib/systemd/system/
+systemd/backuppc.service usr/lib/systemd/system/
