--- developers-reference.sgml	2007-06-16 21:30:59.101774412 +0200
+++ developers-reference.sgml.new	2007-06-16 21:33:53.093000638 +0200
@@ -4420,6 +4420,8 @@
 because that means that in the classical dialog interface,
 people will need to scroll, and lot of people just don't do that.
 	<p>
+The extended description should <strong>never</strong> include a question.
+	<p>
 For specific rules depending on templates type (string, boolean,
 etc.), please read below.
 
@@ -4472,8 +4474,6 @@
     question is rather long (remember that translations are often longer
     than original versions)
 
-<item> The extended description should <strong>not</strong> include a question. 
-
 <item> Again, please avoid referring to specific interface widgets. A common
     mistake for such templates is "if you answer Yes"-type
     constructions.
