diff --git a/contrib/mplayer b/contrib/mplayer
index 157a7d8..248f1e0 100644
--- a/contrib/mplayer
+++ b/contrib/mplayer
@@ -223,5 +223,5 @@ _mplayer()
                 sed -ne '1,/^[[:space:]]*Name/d' \
                     -e "s/^[[:space:]]*/-/" -e "s/[[:space:]:].*//" \
-                    -e "/^-\(Total\|.*\*\)\?$/!p" )' -- "$cur" ) )
+                    -e "/^-\(Total\|.*\*\)\?$/!p" ) -shuffle' -- "$cur" ) )
             ;;
         *)
