--- apt-cacher-ng-0.8.0~pre2/source/acfg_defaults.cc.orig	2014-08-31 00:33:53.000000000 -0400
+++ apt-cacher-ng-0.8.0~pre2/source/acfg_defaults.cc	2014-09-01 00:57:05.425780431 -0400
@@ -44,8 +44,8 @@
 		"|packages\\.[a-zA-Z][a-zA-Z]\\.gz|info\\.txt|license\\.tar\\.gz|license\\.zip" //opensuse
 		"|" ALXPATTERN // Arch Linux
 		"|metalink\\?repo|.*prestodelta\\.xml\\.gz|repodata/.*\\.(xml|sqlite)" COMPOPT // CentOS
-		"|\\.treeinfo|vmlinuz|(initrd|product|squashfs|updates)\\.img" // Fedora
-		")$" // end of only-filename paterns
+		"|\\.?treeinfo|vmlinuz|(initrd|product|squashfs|updates)\\.img" // Fedora
+		")$" // end of only-filename patterns
 		"|/dists/.*/installer-[^/]+/[^0-9][^/]+/images/.*"); // d-i stuff but not containing a date (year number) in the revision directory (like "current", "beta", ...)
 
 //string wfilepat( VPATPREFIX  "(Release|Release\\.gpg|release|meta-release|Translation[^/]*\\.bz2)$");
