diff -rub bsdmainutils-orig/usr.bin/calendar/calendars/calendar.dutch bsdmainutils/usr.bin/calendar/calendars/calendar.dutch
--- bsdmainutils-orig/usr.bin/calendar/calendars/calendar.dutch	2013-05-06 16:46:00.000000000 +0200
+++ bsdmainutils/usr.bin/calendar/calendars/calendar.dutch	2014-06-02 17:14:44.000000000 +0200
@@ -13,7 +13,7 @@
 jan/01	Nieuwjaar
 jan/06	Driekoningen
 apr/01	1 april
-apr/30	Koninginnedag
+apr/27	Koningsdag
 mei/01	Dag van de Arbeid
 mei/04	Dodenherdenking
 mei/05	Bevrijdingsdag
@@ -60,20 +60,20 @@
  * Het koningshuis
  */
 jan/19	Prinses Margriet (1943)
-jan/31	Koningin Beatrix (1938)
-feb/17	Prins Willem III (1817 - 1890)
+jan/31	Prinses Beatrix (1938, koningin 1980 - 2013)
+feb/17	Prins Willem III (1817 - 1890, koning 1849 - 1890)
 feb/18	Prinses Christina (1947)
 apr/10	Prinses Ariane (2007)
 apr/19	Prins Hendrik (1876 - 1934)
-apr/27	Kroonprins Willem Alexander (1967)
-apr/30	Koningin Juliana (1909 - 2004)
+apr/27	Koning Willem Alexander (1967, koning sinds 2013)
+apr/30	Prinses Juliana (1909 - 2004, koningin 1948 - 1980)
 apr/30	Mr. Pieter van Vollenhoven (1939)
 mei/17	Prinses Maxima (1971)
 jun/26	Prinses Alexia (2005)
 jun/29	Prins Bernhard (1911 - 2004)
 aug/05	Prinses Irene (1939)
-aug/31	Prinses Wilhelmina (1880 - 1962)
+aug/31	Prinses Wilhelmina (1880 - 1962, koningin 1890 - 1948)
 sep/06	Prins Claus (1925 - 2002)
-sep/25	Prins Johan Friso (1968)
+sep/25	Prins Johan Friso (1968 - 2013)
 okt/11	Prins Constantijn (1969)
-dec/07	Prinses Catharina-Amalia (2003)
+dec/07	Kroonprinses Catharina-Amalia (2003)
