diff -u bitlbee-3.5.1/debian/bitlbee-common.postinst bitlbee-3.5.1/debian/bitlbee-common.postinst
--- bitlbee-3.5.1/debian/bitlbee-common.postinst
+++ bitlbee-3.5.1/debian/bitlbee-common.postinst
@@ -34,7 +34,9 @@
 	adduser --system --group --disabled-login --disabled-password --home /var/lib/bitlbee/ bitlbee
 fi
 
-chmod 700 /var/lib/bitlbee/
+if [ -d /var/lib/bitlbee ]; then
+    chmod 700 /var/lib/bitlbee/
+fi
 
 ## Can't do this in packaging phase: Don't know the UID yet. Access to
 ## the file should be limited, now that it stores passwords. Added
diff -u bitlbee-3.5.1/debian/changelog bitlbee-3.5.1/debian/changelog
--- bitlbee-3.5.1/debian/changelog
+++ bitlbee-3.5.1/debian/changelog
@@ -1,3 +1,23 @@
+bitlbee (3.5.1-1.2) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Comment out bitlbee-plugin-skype and skyped in d/control (Closes: #921895).
+    These binary packages are not built by d/rules by default, and as such
+    they are not currently in the archive.  Commenting them out in order
+    to avoid my NMU hitting binNEW; this caused the -1.1 source-only
+    upload to fail.
+    Thank you to Mattia Rizzolo for suggesting this fix.
+
+ -- Sean Whitton <spwhitton@spwhitton.name>  Sat, 09 Feb 2019 17:03:38 -0700
+
+bitlbee (3.5.1-1.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Add existence check to chmod call in bitlbee-common.postinst
+    (Closes: #816200).
+
+ -- Sean Whitton <spwhitton@spwhitton.name>  Fri, 25 Jan 2019 16:50:34 -0700
+
 bitlbee (3.5.1-1) unstable; urgency=medium
 
   * Crash bug fix. (Closes: #853282)
diff -u bitlbee-3.5.1/debian/control bitlbee-3.5.1/debian/control
--- bitlbee-3.5.1/debian/control
+++ bitlbee-3.5.1/debian/control
@@ -72,22 +72,22 @@
-Package: bitlbee-plugin-skype
-Architecture: any
-Depends: ${misc:Depends}, ${shlibs:Depends}, bitlbee (= ${binary:Version}) | bitlbee-libpurple (= ${binary:Version}), bitlbee-common (= ${source:Version})
-Recommends: skyped
-Description: IRC to other chat networks gateway (Skype plugin)
- This program can be used as an IRC server which forwards everything you
- say to people on other chat networks: Jabber (which includes Google
- Talk), ICQ, AIM, MSN and Twitter.
- .
- This package contains a plugin that adds support for the Skype IM network.
- You need to download and install the Skype client for this to work.
+# Package: bitlbee-plugin-skype
+# Architecture: any
+# Depends: ${misc:Depends}, ${shlibs:Depends}, bitlbee (= ${binary:Version}) | bitlbee-libpurple (= ${binary:Version}), bitlbee-common (= ${source:Version})
+# Recommends: skyped
+# Description: IRC to other chat networks gateway (Skype plugin)
+#  This program can be used as an IRC server which forwards everything you
+#  say to people on other chat networks: Jabber (which includes Google
+#  Talk), ICQ, AIM, MSN and Twitter.
+#  .
+#  This package contains a plugin that adds support for the Skype IM network.
+#  You need to download and install the Skype client for this to work.
 
-Package: skyped
-Architecture: all
-Depends: ${misc:Depends}, ${shlibs:Depends}, python (>= 2.5), python-gnutls, python-skype (>=0.9.28.7)
-Recommends: skype
-Description: Daemon to control Skype remotely
- Daemon to control the GUI Skype client. Currently required to control Skype
- from the BitlBee IRC2IM gateway. Skyped and Skype can run on a different
- host than the BitlBee server, the communication is encrypted.
- .
- You need to download and install the Skype client for this to work.
+# Package: skyped
+# Architecture: all
+# Depends: ${misc:Depends}, ${shlibs:Depends}, python (>= 2.5), python-gnutls, python-skype (>=0.9.28.7)
+# Recommends: skype
+# Description: Daemon to control Skype remotely
+#  Daemon to control the GUI Skype client. Currently required to control Skype
+#  from the BitlBee IRC2IM gateway. Skyped and Skype can run on a different
+#  host than the BitlBee server, the communication is encrypted.
+#  .
+#  You need to download and install the Skype client for this to work.
