--- camel/providers/imap/camel-imap-folder.c.orig	2008-10-23 11:26:24.314157759 +0200
+++ camel/providers/imap/camel-imap-folder.c	2008-10-23 11:27:13.086188561 +0200
@@ -1811,13 +1811,13 @@
 				handle_copyuid (response, source, destination);
 			if (response)
 				handle_copyuid_copy_user_tags (response, source, destination);
-			camel_imap_response_free (store, response);
 
-			if (!camel_exception_is_set(ex) && delete_originals) {
+			if (response && !camel_exception_is_set(ex) && delete_originals) {
 				for (i=last;i<uid;i++)
 					camel_folder_delete_message(source, uids->pdata[i]);
 				last = uid;
 			}
+			camel_imap_response_free (store, response);
 		}
 		g_free (uidset);
 	}
