From 4ba75eee566c3a6af6c75c2119d938e3fb7796b9 Mon Sep 17 00:00:00 2001
From: Dominic Hargreaves <dom@earth.li>
Date: Sun, 24 Jul 2016 19:12:52 +0100
Subject: [PATCH 3/4] releasing package debhelper version 9.20150101+deb8u1

---
 debian/changelog | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/changelog b/debian/changelog
index c5d2a6d..dcaa609 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,4 +1,4 @@
-debhelper (9.20150101+deb8u1) UNRELEASED; urgency=medium
+debhelper (9.20150101+deb8u1) jessie-security; urgency=high
 
   * Non-maintainer upload.
   * Invoke Makefile.PL and Build.PL with perl -I. as part of the fixes for
@@ -6,7 +6,7 @@ debhelper (9.20150101+deb8u1) UNRELEASED; urgency=medium
   * Export PERL_USE_UNSAFE_INC to fix a further set of packages which
     fail to build with . removed from @INC
 
- -- Dominic Hargreaves <dom@earth.li>  Thu, 07 Jul 2016 16:53:02 +0200
+ -- Dominic Hargreaves <dom@earth.li>  Sun, 24 Jul 2016 19:12:37 +0100
 
 debhelper (9.20150101) unstable; urgency=medium
 
-- 
2.1.4

