--- isolinux/text-orig.cfg	2008-11-11 19:10:16.000000000 +0100
+++ isolinux/text.cfg	2008-11-11 19:34:03.000000000 +0100
@@ -1,6 +1,14 @@
 default install
+
+label menu
+	kernel vesamenu.c32
+
 label install
+	kernel ifcpu64.c32
+	append amd64-install -- i386-install
+
+label i386-install
 	menu label ^Install
 	menu default
 	kernel /install.386/vmlinuz
-	append vga=normal initrd=/install.386/initrd.gz -- quiet 
+	append vga=normal initrd=/install.386/initrd.gz -- quiet
