--- alsa-tools-1.0.13/debian/control.orig	2007-01-13 16:12:20.000000000 +0100
+++ alsa-tools-1.0.13/debian/control	2007-01-13 16:13:29.000000000 +0100
@@ -7,7 +7,7 @@
 Standards-Version: 3.7.2
 
 Package: alsa-tools
-Architecture: i386 ia64 alpha amd64 arm armeb hppa m32r m68k mips mipsel powerpc ppc64 s390 s390x sh3 sh3eb sh4 sh4eb sparc
+Architecture: i386 ia64 alpha amd64 arm armeb armel hppa m32r m68k mips mipsel powerpc ppc64 s390 s390x sh3 sh3eb sh4 sh4eb sparc
 Depends: ${shlibs:Depends}, ${misc:Depends}
 Provides: as10k1
 Conflicts: as10k1
@@ -21,7 +21,7 @@
   us428control - Controller utility for Tascam US-X2Y
 
 Package: alsa-tools-gui
-Architecture: i386 ia64 alpha amd64 arm armeb hppa m32r m68k mips mipsel powerpc ppc64 s390 s390x sh3 sh3eb sh4 sh4eb sparc
+Architecture: i386 ia64 alpha amd64 arm armeb armel hppa m32r m68k mips mipsel powerpc ppc64 s390 s390x sh3 sh3eb sh4 sh4eb sparc
 Depends: ${shlibs:Depends}, ${misc:Depends}
 Description: GUI based ALSA utilities for specific hardware
  A collection of GUI based ALSA utilities for specific sound hardware:
@@ -35,7 +35,7 @@
 
 Package: alsa-firmware-loaders
 Section: contrib/sound
-Architecture: i386 ia64 alpha amd64 arm armeb hppa m32r m68k mips mipsel powerpc ppc64 s390 s390x sh3 sh3eb sh4 sh4eb sparc
+Architecture: i386 ia64 alpha amd64 arm armeb armel hppa m32r m68k mips mipsel powerpc ppc64 s390 s390x sh3 sh3eb sh4 sh4eb sparc
 Depends: ${shlibs:Depends}, ${misc:Depends}
 Description: ALSA software loaders for specific hardware
  A collection of software loaders for specific hardware:
@@ -50,7 +50,7 @@
 
 Package: ld10k1
 Section: sound
-Architecture: i386 ia64 alpha amd64 arm armeb hppa m32r m68k mips mipsel powerpc ppc64 s390 s390x sh3 sh3eb sh4 sh4eb sparc
+Architecture: i386 ia64 alpha amd64 arm armeb armel hppa m32r m68k mips mipsel powerpc ppc64 s390 s390x sh3 sh3eb sh4 sh4eb sparc
 Depends: ${shlibs:Depends}, ${misc:Depends}, lsb-base (>= 3.0-3)
 Suggests: alsa-tools
 Description: ALSA emu10k1/2 patch loader
@@ -61,7 +61,7 @@
 
 Package: liblo10k1-0
 Section: libs
-Architecture: i386 ia64 alpha amd64 arm armeb hppa m32r m68k mips mipsel powerpc ppc64 s390 s390x sh3 sh3eb sh4 sh4eb sparc
+Architecture: i386 ia64 alpha amd64 arm armeb armel hppa m32r m68k mips mipsel powerpc ppc64 s390 s390x sh3 sh3eb sh4 sh4eb sparc
 Depends: ${shlibs:Depends}, ${misc:Depends}
 Description: ALSA emu10k1/2 patch-loader library
  This is the runtime library for the ld10k1 patch loader for use with
@@ -72,7 +72,7 @@
 
 Package: liblo10k1-dev
 Section: devel
-Architecture: i386 ia64 alpha amd64 arm armeb hppa m32r m68k mips mipsel powerpc ppc64 s390 s390x sh3 sh3eb sh4 sh4eb sparc
+Architecture: i386 ia64 alpha amd64 arm armeb armel  hppa m32r m68k mips mipsel powerpc ppc64 s390 s390x sh3 sh3eb sh4 sh4eb sparc
 Depends: liblo10k1-0 (= ${Source-Version}), libasound2-dev
 Description: ALSA emu10k1/2 patch-loader library development files
  These are the development files for the ld10k1 patch loader for use with
@@ -82,7 +82,7 @@
  Homepage: http://ld10k1.sourceforge.net/
 
 Package: qlo10k1
-Architecture: i386 ia64 alpha amd64 arm armeb hppa m32r m68k mips mipsel powerpc ppc64 s390 s390x sh3 sh3eb sh4 sh4eb sparc
+Architecture: i386 ia64 alpha amd64 arm armeb armel hppa m32r m68k mips mipsel powerpc ppc64 s390 s390x sh3 sh3eb sh4 sh4eb sparc
 Recommends: ld10k1
 Depends: ${shlibs:Depends}, ${misc:Depends}
 Description: ALSA ld10k1 utility
