diff -urN base-files-7.1.original/share/nsswitch.conf base-files-7.1/share/nsswitch.conf
--- base-files-7.1.original/share/nsswitch.conf	2013-01-27 11:59:40.147903916 +0000
+++ base-files-7.1/share/nsswitch.conf	2013-01-27 12:04:44.534176087 +0000
@@ -7,6 +7,7 @@
 passwd:         compat
 group:          compat
 shadow:         compat
+gshadow:	files
 
 hosts:          files dns
 networks:       files
