diff -Nur debootstrap-1.0.10/debootstrap debootstrap-1.0.10.1/debootstrap
--- debootstrap-1.0.10/debootstrap	2008-01-20 09:33:33.000000000 +0100
+++ debootstrap-1.0.10.1/debootstrap	2008-10-01 08:20:11.000000000 +0200
@@ -203,16 +203,20 @@
 			error 1 NEEDARG "option requires an argument %s" "$1"
 		fi
 		;;
-	    --unpack-tarball)
-		if [ -n "$2" ] ; then
-			if [ ! -f "$2" ] ; then
-				error 1 NOTARBALL "%s: No such file or directory" "$2"
-			fi
+	    --unpack-tarball|--unpack-tarball=?*)
+		if [ "$1" = "--unpack-tarball" -a -n "$2" ] ; then
 			UNPACK_TARBALL="$2"
-			shift 2
+                        shift 2
+                elif [ "$1" != "${1#--unpack-tarball=}" ]; then
+                        UNPACK_TARBALL="${1#--unpack-tarball=}"
+                        shift
 		else
 			error 1 NEEDARG "option requires an argument %s" "$1"
 		fi
+
+		if [ ! -f "$UNPACK_TARBALL" ] ; then
+			error 1 NOTARBALL "%s: No such file or directory" "$UNPACK_TARBALL"
+		fi
 		;;
 	    --include|--include=?*)
 		if [ "$1" = "--include" -a -n "$2" ]; then
