From 5b729cc5272d2fbd2fae1dff6b4ff4c4557ff91b Mon Sep 17 00:00:00 2001
From: Robert Edmonds <edmonds@debian.org>
Date: Sun, 22 Mar 2015 00:12:45 -0400
Subject: [PATCH] HttpsMethod::Fetch(): Zero the FetchResult object when
 leaving due to 404

---
 methods/https.cc | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/methods/https.cc b/methods/https.cc
index 3a5981b..f2b00dd 100644
--- a/methods/https.cc
+++ b/methods/https.cc
@@ -402,6 +402,8 @@ bool HttpsMethod::Fetch(FetchItem *Itm)
       _error->Error("%s", err);
       // unlink, no need keep 401/404 page content in partial/
       unlink(File->Name().c_str());
+      Res.Size = 0;
+      Res.LastModified = 0;
       return false;
    }
 
-- 
2.1.4

