commit a44f2ed63a086a968ccfb1f7db4e3584fc2eb260
Author: Ivan Mincik <ivan.mincik@gmail.com>
Date:   Sat Jun 30 15:45:52 2012 +0200

    dev

diff --git a/debian/rules b/debian/rules
index c672d8a..7de459a 100755
--- a/debian/rules
+++ b/debian/rules
@@ -21,3 +21,6 @@ override_dh_installchangelogs:
 
 override_dh_compress:
 	dh_compress -X.js -X.txt
+
+override_dh_installdeb:
+	sed -i 's/\/usr\/bin\/python/\/usr\/bin\/env python/' debian/fabric/usr/bin/fab
