From 0e064e9aa81b7e3f680b5260ac572514a14c88be Mon Sep 17 00:00:00 2001
From: Roger Leigh <rleigh@debian.org>
Date: Sat, 27 Nov 2010 11:43:32 +0000
Subject: [PATCH 6/6] dh: Use "architecture independent" rather than "binary independent" for consistency

Signed-off-by: Roger Leigh <rleigh@debian.org>
---
 dh |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/dh b/dh
index a94a14c..9031747 100755
--- a/dh
+++ b/dh
@@ -22,7 +22,7 @@ B<install>, B<binary-arch>, B<binary-indep>, and B<binary>.
 
 Commands in the B<build-indep>, B<install-indep> and B<binary-indep>
 sequences are passed the B<-i> option to ensure they only work on
-binary independent packages, and commands in the B<build-arch>,
+architecture independent packages, and commands in the B<build-arch>,
 B<install-arch> and B<binary-arch> sequences are passed the B<-a>
 option to ensure they only work on architecture dependent packages.
 
-- 
1.7.2.3

