From 8b97ede36d368e28c0b916b430a956bea7fe9490 Mon Sep 17 00:00:00 2001
From: Jan Dittberner <jan@dittberner.info>
Date: Fri, 19 Feb 2010 17:54:34 +0100
Subject: [PATCH 2/2] springlobby Depends on arch-specific spring

---
 Packages-arch-specific |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/Packages-arch-specific b/Packages-arch-specific
index 0395baa..704d53c 100644
--- a/Packages-arch-specific
+++ b/Packages-arch-specific
@@ -330,6 +330,7 @@ silo: sparc							      # sparc lilo
 %spicctrl: i386							      # For Sony VAIO laptops
 %splay: i386 alpha arm armel m68k powerpc			      # [?] broken on sparc (see BTS]
 %spring: i386 amd64 kfreebsd-i386 kfreebsd-amd64 hurd-i386	      # x86 specific
+%springlobby: i386 amd64 kfreebsd-i386 kfreebsd-amd64 hurd-i386	      # Depends on spring
 %srm-reader: alpha						      # Alpha specific udeb
 %stalin: i386 amd64 ia64 sparc kfreebsd-i386 kfreebsd-amd64	      # [ANAIS]
 subterfugue: i386						      # i386 specific process tracing etc.
-- 
1.6.6.1

