Index: debiandoc-sgml-1.2.29/tools/bin/template
===================================================================
--- debiandoc-sgml-1.2.29.orig/tools/bin/template
+++ debiandoc-sgml-1.2.29/tools/bin/template
@@ -155,10 +155,17 @@ then
 fi
 
 @@@end-latexpdf-active@@@
-@@@start-latexdvi-latexps-latexpdf-active@@@
+@@@start-latexdvi-latexps-active@@@
 ## ----------------------------------------------------------------------
 ## check for presence of used latex styles
-if ! kpsewhich \
+if ! kpsewhich -progname latex \
+@@@end-latexdvi-latexps-active@@@
+@@@start-latexpdf-active@@@
+## ----------------------------------------------------------------------
+## check for presence of used pdflatex styles
+if ! kpsewhich -progname pdflatex \
+@@@end-latexpdf-active@@@
+@@@start-latexdvi-latexps-latexpdf-active@@@
     fancyhdr.sty \
     helvet.sty \
     hyperref.sty \
@@ -174,10 +181,17 @@ then
 fi
 
 @@@end-latexdvi-latexps-latexpdf-active@@@
-@@@start-latexdvi-latexps-latexpdf-active@@@
+@@@start-latexdvi-latexps-active@@@
 ## ----------------------------------------------------------------------
 ## check for presence of used latex styles
-if ! kpsewhich \
+if ! kpsewhich -progname latex \
+@@@end-latexdvi-latexps-active@@@
+@@@start-latexpdf-active@@@
+## ----------------------------------------------------------------------
+## check for presence of used pdflatex styles
+if ! kpsewhich -progname pdflatex \
+@@@end-latexpdf-active@@@
+@@@start-latexdvi-latexps-latexpdf-active@@@
     footmisc.sty \
     paralist.sty \
     vmargin.sty \
@@ -189,10 +203,17 @@ then
 fi
 
 @@@end-latexdvi-latexps-latexpdf-active@@@
-@@@start-latexdvi-latexps-latexpdf-active@@@
+@@@start-latexdvi-latexps-active@@@
 ## ----------------------------------------------------------------------
 ## check for presence of used latex styles
-if ! kpsewhich \
+if ! kpsewhich -progname latex \
+@@@end-latexdvi-latexps-active@@@
+@@@start-latexpdf-active@@@
+## ----------------------------------------------------------------------
+## check for presence of used pdflatex styles
+if ! kpsewhich -progname pdflatex \
+@@@end-latexpdf-active@@@
+@@@start-latexdvi-latexps-latexpdf-active@@@
     wasysym.sty \
     >/dev/null 2>&1
 then
