diff -urN battery-stats-0.3.3.0/debian/logrotate battery-stats-0.3.3/debian/logrotate
--- battery-stats-0.3.3.0/debian/logrotate	2007-09-01 22:01:13.000000000 +0530
+++ battery-stats-0.3.3/debian/logrotate	2007-09-01 22:03:46.000000000 +0530
@@ -7,6 +7,6 @@
     nomail
     weekly
     postrotate 
-	/etc/init.d/battery-stats restart > /dev/null
+        if which invoke-rc.d >/dev/null 2>&1; then invoke-rc.d battery-stats restart > /dev/null else /etc/init.d/package /etc/init.d/battery-stats restart > /dev/null fi
     endscript
 }
