--- pgsql-old	2016-05-17 00:53:25.925836858 +0200
+++ pgsql	2016-05-17 00:58:35.639317340 +0200
@@ -97,7 +97,8 @@
    if [ "$usevserver" = "yes" ]; then
       	cmdprefix="$VSERVER $vsname exec "
    fi
-   execstr="${cmdprefix} su - $PGSQLUSER -c 'psql -AtU $PGSQLUSER -c \"SELECT datname FROM pg_database WHERE NOT datistemplate\"'"
+   # fix for Bug#824508: backupninja: Postgresql 9.4 backup databases = all crashes
+   execstr="${cmdprefix} su -c 'psql -AtU $PGSQLUSER -c \"SELECT datname FROM pg_database WHERE NOT datistemplate\"' -m $PGSQLUSER"
    debug execstr
    dblist=""
    for db in $(eval $execstr 2>&1); do
