--- pgsql   2016-07-14 12:36:12.000000000 +0200
+++ pgsql.new   2016-07-14 12:38:39.601526817 +0200
@@ -97,8 +97,9 @@
    if [ "$usevserver" = "yes" ]; then
        cmdprefix="$VSERVER $vsname exec "
    fi
-   execstr="${cmdprefix} su - $PGSQLUSER -c 'psql -AtU $PGSQLUSER -c \"SELECT datname FROM pg_database WHERE NOT datistemplate\"'"
-   debug execstr
+   # fix for Bug#824508: backupninja: Postgresql 9.4 backup databases = all crashes
+   execstr="${cmdprefix} su -c 'psql -AtU $PGSQLUSER -c \"SELECT datname FROM pg_database WHERE NOT datistemplate\"'"
+   debug "$execstr"
    dblist=""
    for db in $(eval $execstr 2>&1); do
       dblist="$dblist $db"
