diff -uN emms-3.0+20110425+1.git298e022.orig/debian/changelog emms-3.0+20110425+1.git298e022/debian/changelog
--- emms-3.0+20110425+1.git298e022.orig/debian/changelog	2013-02-27 11:45:23.285936262 +0900
+++ emms-3.0+20110425+1.git298e022/debian/changelog	2013-02-27 11:47:03.242937535 +0900
@@ -1,3 +1,13 @@
+emms (3.0+20110425+1.git298e022-4) unstable; urgency=low
+
+  * debian/control:
+    + Require emacsen-common >= 2.0.5 because of #693472.
+  * debian/emacsen-install, debian/emacsen-remove:
+    + As emacs22 is not supported since squeeze, it must be ignored.
+      Closes: #699743.
+
+ -- Arnaud Fontaine <arnau@debian.org>  Wed, 27 Feb 2013 11:25:39 +0900
+
 emms (3.0+20110425+1.git298e022-3) unstable; urgency=low
 
   * debian/control:
diff -uN emms-3.0+20110425+1.git298e022.orig/debian/control emms-3.0+20110425+1.git298e022/debian/control
--- emms-3.0+20110425+1.git298e022.orig/debian/control	2013-02-27 11:45:29.866002178 +0900
+++ emms-3.0+20110425+1.git298e022/debian/control	2013-02-27 11:32:40.614292813 +0900
@@ -14,7 +14,7 @@
 Package: emms
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends},
-         emacsen-common,
+         emacsen-common (>= 2.0.5),
          w3m-el | w3m-el-snapshot,
          emacs23 | xemacs21 | emacs-snapshot
 Recommends: vorbis-tools | mpg321 | mplayer | vlc | mpd (>= 0.12.0) | xine-ui
diff -uN emms-3.0+20110425+1.git298e022.orig/debian/emacsen-install emms-3.0+20110425+1.git298e022/debian/emacsen-install
--- emms-3.0+20110425+1.git298e022.orig/debian/emacsen-install	2012-06-07 17:32:35.903138994 +0900
+++ emms-3.0+20110425+1.git298e022/debian/emacsen-install	2013-02-27 11:43:52.157023326 +0900
@@ -8,7 +8,7 @@
 FLAVOR=$1
 PACKAGE=emms
 
-if [ ${FLAVOR} = emacs ] || [ ${FLAVOR} = emacs21 ]; then
+if [ ${FLAVOR} = emacs ] || [ ${FLAVOR} = emacs21 ] || [ ${FLAVOR} = emacs22 ]; then
     exit 0;
 fi
 
diff -uN emms-3.0+20110425+1.git298e022.orig/debian/emacsen-remove emms-3.0+20110425+1.git298e022/debian/emacsen-remove
--- emms-3.0+20110425+1.git298e022.orig/debian/emacsen-remove	2011-11-05 16:19:31.000000000 +0900
+++ emms-3.0+20110425+1.git298e022/debian/emacsen-remove	2013-02-27 11:44:09.965201736 +0900
@@ -4,7 +4,7 @@
 FLAVOR=$1
 PACKAGE=emms
 
-if [ ${FLAVOR} != emacs ] && [ ${FLAVOR} != emacs21 ]; then
+if [ ${FLAVOR} != emacs ] && [ ${FLAVOR} != emacs21 ] && [ ${FLAVOR} != emacs22 ]; then
     echo remove/${PACKAGE}: purging byte-compiled files for ${FLAVOR}
     rm -rf /usr/share/${FLAVOR}/site-lisp/${PACKAGE}
 fi
