diff -u backuppc-3.2.1/debian/changelog backuppc-3.2.1/debian/changelog
--- backuppc-3.2.1/debian/changelog
+++ backuppc-3.2.1/debian/changelog
@@ -1,3 +1,11 @@
+backuppc (3.2.1-4.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Do not ship /etc/backuppc/config.pl as a conffile; it is handled
+    by ucf already (Closes: #706315)
+
+ -- Jonathan Wiltshire <jmw@tiger-computing.co.uk>  Wed, 05 Jun 2013 09:58:59 +0100
+
 backuppc (3.2.1-4) unstable; urgency=low
 
   * init.d status support. Closes: #651539
diff -u backuppc-3.2.1/debian/rules backuppc-3.2.1/debian/rules
--- backuppc-3.2.1/debian/rules
+++ backuppc-3.2.1/debian/rules
@@ -73,6 +73,7 @@
 	(cd debian/backuppc/usr/share/backuppc/cgi-bin; ln -s ../image; ln -s /usr/lib/backuppc/cgi-bin/index.cgi )
 	patch --no-backup-if-mismatch -p0 < debian/config.pl.diff
 	install --mode=644 debian/backuppc/etc/backuppc/config.pl debian/backuppc/usr/share/backuppc/conf
+	rm -rf debian/backuppc/etc/backuppc/config.pl
 
 # Build architecture-independent files here.
 binary-indep: build install
