--- /tmp/0sP5oNcslC/console-setup-1.13/debian/config.proto	2006-11-13 16:55:11.000000000 +0000
+++ /tmp/7b6qVlOkaE/console-setup-1.13ubuntu14/debian/config.proto	2007-04-13 08:42:16.000000000 +0100
@@ -718,6 +718,10 @@
     *_CA*)
 	default_layout=ca  # Canada
 	;;
+    fr_CH*)
+	default_layout=ch  # Switzerland
+	default_variant=fr # French
+	;;
     *_CH*)
 	default_layout=ch  # Switzerland
 	layout_priority=critical
