diff --git i/data/no-pie-compile.specs w/data/no-pie-compile.specs
index f85b394..2277b97 100644
--- i/data/no-pie-compile.specs
+++ w/data/no-pie-compile.specs
@@ -1,2 +1,2 @@
-*cc1_options:
+*self_spec:
 + %{!r:%{!fpie:%{!fPIE:%{!fpic:%{!fPIC:%{!fno-pic:-fno-PIE}}}}}}
diff --git i/data/no-pie-link.specs w/data/no-pie-link.specs
index 15243a0..54db649 100644
--- i/data/no-pie-link.specs
+++ w/data/no-pie-link.specs
@@ -1,2 +1,2 @@
 *self_spec:
-+ %{!shared:%{!r:-fno-PIE -no-pie}}
++ %{!shared:%{!r:%{!fPIE:%{!pie:-fno-PIE -no-pie}}}}
diff --git i/data/pie-compile.specs w/data/pie-compile.specs
index fc54bcb..74d8215 100644
--- i/data/pie-compile.specs
+++ w/data/pie-compile.specs
@@ -1,2 +1,2 @@
-*cc1_options:
-+ %{!r:%{!fpie:%{!fPIE:%{!fpic:%{!fPIC:%{!fno-pic:-fPIE}}}}}}
+*self_spec:
++ %{!r:%{!fpie:%{!fPIE:%{!fpic:%{!fPIC:%{!fno-pic:%{!fno-PIE:%{!no-pie:-fPIE}}}}}}}}
diff --git i/data/pie-link.specs w/data/pie-link.specs
index a5e0fe4..35d26e1 100644
--- i/data/pie-link.specs
+++ w/data/pie-link.specs
@@ -1,2 +1,2 @@
 *self_spec:
-+ %{!shared:%{!r:-fPIE -pie}}
++ %{!shared:%{!r:%{!fno-PIE:%{!no-pie:-fPIE -pie}}}}
