Index: debian/changelog
===================================================================
--- debian/changelog	(Revision 1933)
+++ debian/changelog	(Arbeitskopie)
@@ -35,8 +35,10 @@
     have a copyright (Closes: #535337)
   * nmudiff: Use 'bts select' to test, weither a check already has a patch
     tag set or not and only set it, if not (Closes: #519678)
+  * bts: Add a test weither the title would actually change something.
+    (Closes: #537875)
 
- -- Patrick Schoenfeld <schoenfeld@debian.org>  Tue, 21 Jul 2009 15:01:22 +0200
+ -- Patrick Schoenfeld <schoenfeld@debian.org>  Tue, 21 Jul 2009 17:25:40 +0200
 
 devscripts (2.10.52) unstable; urgency=low
 
Index: scripts/bts.pl
===================================================================
--- scripts/bts.pl	(Revision 1933)
+++ scripts/bts.pl	(Arbeitskopie)
@@ -1190,6 +1190,16 @@
     if (! length $title) {
 	die "bts retitle: set title of $bug to what?\n";
     }
+   
+    my $bug_status = Devscripts::Debbugs::status( map {[bug => $_, indicatesource => 1]} ($bug) );
+    if (not $bug_status) {
+	warn "bts retitle: Unable to retrieve the bug from SOAP server. Does it exist?";
+    } else {
+	my $old_title = $bug_status->{$bug}->{subject};
+	if ($title eq $old_title) {
+	    die "bts retitle: title of $bug has not changed.\n";
+	}
+    }
     mailbts("retitle $bug to $title", "retitle $bug $title");
 }
 
