diff --git a/debhelper.pod b/debhelper.pod
index b03a8bbd..5c0a7e4e 100644
--- a/debhelper.pod
+++ b/debhelper.pod
@@ -595,6 +595,11 @@ B<dh_systemd_start> instead.
 
 =item -
 
+B<dh_installinit> no longer installs F<upstart> files.  Please make sure to
+remove existing files on upgrades (e.g. by using the B<maintscript> mechanism).
+
+=item -
+
 The B<-s> (B<--same-arch>) option is removed.  Please use B<-a> (B<--arch>) instead.
 
 =item -
diff --git a/dh_installinit b/dh_installinit
index 02282e0d..cbd691cf 100755
--- a/dh_installinit
+++ b/dh_installinit
@@ -50,7 +50,7 @@ build directory.
 =item debian/I<package>.upstart
 
 If this exists, it is installed into etc/init/I<package>.conf in the package
-build directory.
+build directory. Only used in compat levels 10 and below.
 
 =item debian/I<package>.service
 
@@ -255,7 +255,7 @@ foreach my $package (@{$dh{DOPACKAGES}}) {
 		install_file($tmpfile, "$path/$script.conf");
 	}
 
-	my $job=pkgfile($package,"upstart");
+	my $job=pkgfile($package,"upstart") if compat(10);
 	if ($job ne '' && ! $dh{ONLYSCRIPTS}) {
 		install_dir("$tmp/etc/init");
 		install_file($job, "$tmp/etc/init/$jobfile.conf");
