From 134a64303027dc8a6445a2714c343cc6387569c9 Mon Sep 17 00:00:00 2001
From: James Cowgill <jcowgill@debian.org>
Date: Thu, 15 Mar 2018 14:55:36 +0000
Subject: [PATCH] Install db symlink into correct directory

Closes: #857401
---
 debian/chise-db.install | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/chise-db.install b/debian/chise-db.install
index 55ad050..6b50135 100644
--- a/debian/chise-db.install
+++ b/debian/chise-db.install
@@ -1,4 +1,4 @@
 debian/tmp/usr/lib/xemacs-21.4.15/etc/chise-db/character/by_feature/* /usr/lib/xemacs-21.4.15/etc/chise-db/by_feature
 debian/tmp/usr/lib/xemacs-21.4.15/etc/chise-db/character/feature/* /usr/lib/xemacs-21.4.15/etc/chise-db/feature
 debian/tmp/usr/lib/xemacs-21.4.15/etc/chise-db/feature/property/* /usr/lib/xemacs-21.4.15/etc/chise-db/feature/property
-debian/tmp/usr/share/chise/0.3/db /usr/share/chise/0.3/db
+debian/tmp/usr/share/chise/0.3/db /usr/share/chise/0.3
-- 
2.16.2

