From a54f075aa23b51b0ec904a0aecc3402bf2b3c106 Mon Sep 17 00:00:00 2001
From: Andrej Shadura <andrew.shadura@collabora.co.uk>
Date: Wed, 31 Aug 2022 13:02:10 +0200
Subject: Print dialog requires access to libproxy’s pxgsettings

Closes: #1002725
---
 debian/apparmor-profile | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/debian/apparmor-profile b/debian/apparmor-profile
index 95c28421cab9..904f0ead7e3c 100644
--- a/debian/apparmor-profile
+++ b/debian/apparmor-profile
@@ -68,6 +68,9 @@
   /usr/bin/krusader Cx -> sanitized_helper, # KDE
   /usr/bin/thunar Cx -> sanitized_helper,   # XFCE
 
+  # Print Dialog
+  /usr/lib/@{multiarch}/libproxy/*/pxgsettings Cx -> sanitized_helper,
+
   # For Xubuntu to launch the browser
   #include <abstractions/exo-open>
 
-- 
2.35.1

