--- cups-filters-1.0.4.orig/Makefile
+++ cups-filters-1.0.4/Makefile
@@ -23,6 +23,8 @@ include Makedefs
 DIRS	=	cupsfilters backend filter pdftoopvp pdftopdf $(PHPDIR)
 
 
+Makedefs:
+	$(CURDIR)/configure
 #
 # Make all targets...
 #
--- cups-filters-1.0.4.orig/filter/Makefile
+++ cups-filters-1.0.4/filter/Makefile
@@ -101,6 +101,7 @@ all:	$(TARGETS)
 #
 
 clean:
+	$(MAKE) -C fontembed $@
 	$(RM) $(OBJS) $(TARGETS)
 	$(RM) */*.d
 	$(RM) */*.o
--- cups-filters-1.0.4.orig/pdftoopvp/Makefile
+++ cups-filters-1.0.4/pdftoopvp/Makefile
@@ -39,6 +39,7 @@ framedhelp:
 #
 
 clean:
+	$(MAKE) -C oprs $@
 	$(RM) $(OBJS)
 	$(RM) pdftoopvp
 
