--- cups-filters-1.0.4.orig/Makefile
+++ cups-filters-1.0.4/Makefile
@@ -13,7 +13,9 @@
 #   file is missing or damaged, see the license at "http://www.cups.org/".
 #
 
+ifeq ($(findstring $(MAKECMDGOALS), clean distclean maintainer-clean),)
 include Makedefs
+endif
 
 
 #
@@ -44,6 +46,7 @@ all:
 #
 
 clean:
+	touch Makedefs
 	for dir in $(DIRS); do\
 		echo Cleaning in $$dir... ;\
 		(cd $$dir; $(MAKE) $(MFLAGS) clean) || exit 1;\
--- cups-filters-1.0.4.orig/filter/Makefile
+++ cups-filters-1.0.4/filter/Makefile
@@ -101,6 +101,7 @@ all:	$(TARGETS)
 #
 
 clean:
+	$(MAKE) -C fontembed $@
 	$(RM) $(OBJS) $(TARGETS)
 	$(RM) */*.d
 	$(RM) */*.o
--- cups-filters-1.0.4.orig/pdftoopvp/Makefile
+++ cups-filters-1.0.4/pdftoopvp/Makefile
@@ -39,6 +39,7 @@ framedhelp:
 #
 
 clean:
+	$(MAKE) -C oprs $@
 	$(RM) $(OBJS)
 	$(RM) pdftoopvp
 
