diff -u dbus-1.2.16/debian/libdbus-1-3.install dbus-1.2.16/debian/libdbus-1-3.install
--- dbus-1.2.16/debian/libdbus-1-3.install
+++ dbus-1.2.16/debian/libdbus-1-3.install
@@ -1 +1 @@
-debian/tmp/usr/lib/libdbus-1*.so.* lib/
+usr/lib/libdbus-1*.so.* lib/
diff -u dbus-1.2.16/debian/dbus-x11.install dbus-1.2.16/debian/dbus-x11.install
--- dbus-1.2.16/debian/dbus-x11.install
+++ dbus-1.2.16/debian/dbus-x11.install
@@ -1,2 +1,2 @@
-debian/tmp/usr/bin/dbus-launch
-debian/tmp/usr/share/man/man1/dbus-launch.1
+usr/bin/dbus-launch
+usr/share/man/man1/dbus-launch.1
diff -u dbus-1.2.16/debian/changelog dbus-1.2.16/debian/changelog
--- dbus-1.2.16/debian/changelog
+++ dbus-1.2.16/debian/changelog
@@ -1,3 +1,9 @@
+dbus (1.2.16-1jak1) unstable; urgency=low
+
+  * Correct installation by removing debian/tmp from path. (Closes: #537125)
+
+ -- Julian Andres Klode <jak@debian.org>  Wed, 15 Jul 2009 12:57:11 +0200
+
 dbus (1.2.16-1) unstable; urgency=low
 
   * New upstream release.
diff -u dbus-1.2.16/debian/libdbus-1-dev.install dbus-1.2.16/debian/libdbus-1-dev.install
--- dbus-1.2.16/debian/libdbus-1-dev.install
+++ dbus-1.2.16/debian/libdbus-1-dev.install
@@ -1,4 +1,4 @@
-debian/tmp/usr/include/dbus*/dbus/dbus*.h
-debian/tmp/usr/lib/dbus*/include/dbus/dbus*.h
-debian/tmp/usr/lib/pkgconfig/dbus-1.pc
-debian/tmp/usr/lib/libdbus-1*.a
+usr/include/dbus*/dbus/dbus*.h
+usr/lib/dbus*/include/dbus/dbus*.h
+usr/lib/pkgconfig/dbus-1.pc
+usr/lib/libdbus-1*.a
diff -u dbus-1.2.16/debian/dbus.install dbus-1.2.16/debian/dbus.install
--- dbus-1.2.16/debian/dbus.install
+++ dbus-1.2.16/debian/dbus.install
@@ -1,14 +1,14 @@
-debian/tmp/etc/dbus-1/
-debian/tmp/usr/bin/dbus-daemon
-debian/tmp/usr/bin/dbus-cleanup-sockets
-debian/tmp/usr/bin/dbus-send
-debian/tmp/usr/bin/dbus-uuidgen
-debian/tmp/usr/bin/dbus-monitor
-debian/tmp/usr/lib/dbus-1.0/dbus-daemon-launch-helper
-debian/tmp/usr/share/man/man1/dbus-daemon.1
-debian/tmp/usr/share/man/man1/dbus-cleanup-sockets.1
-debian/tmp/usr/share/dbus-1/services
-debian/tmp/usr/share/dbus-1/system-services
-debian/tmp/usr/share/man/man1/dbus-send.1
-debian/tmp/usr/share/man/man1/dbus-uuidgen.1
-debian/tmp/usr/share/man/man1/dbus-monitor.1
+etc/dbus-1/
+usr/bin/dbus-daemon
+usr/bin/dbus-cleanup-sockets
+usr/bin/dbus-send
+usr/bin/dbus-uuidgen
+usr/bin/dbus-monitor
+usr/lib/dbus-1.0/dbus-daemon-launch-helper
+usr/share/man/man1/dbus-daemon.1
+usr/share/man/man1/dbus-cleanup-sockets.1
+usr/share/dbus-1/services
+usr/share/dbus-1/system-services
+usr/share/man/man1/dbus-send.1
+usr/share/man/man1/dbus-uuidgen.1
+usr/share/man/man1/dbus-monitor.1
