diff --git a/po4a/Makefile b/po4a/Makefile
index 290ce5f..989c787 100644
--- a/po4a/Makefile
+++ b/po4a/Makefile
@@ -4,15 +4,17 @@ GEN_TR_MAN1S := $(patsubst %.1,fr/%.fr.1,$(GEN_MAN1S))
 all: translate $(GEN_TR_MAN1S)
 
 translate:
+	$(MAKE) -C ../scripts/ devscripts.1
 	po4a --previous --no-backups devscripts-po4a.conf
 	touch translate
 clean:
+	$(MAKE) -C ../scripts/ devscripts.1
 	po4a --previous --rm-translations --no-backups devscripts-po4a.conf
-	rm -f $(GEN_TR_MAN1S) translate
+	rm -f $(GEN_TR_MAN1S) ../scripts/devscripts.1 translate
 
 fr/%.fr.1: fr/%.fr.pl translate
 	podchecker $<
-	pod2man --center=" " --release="Debian Utilities" $< > $@
+	pod2man --utf8 --center=" " --release="Utilitaires Debian" $< > $@
 fr/%.fr.1: fr/%.fr.dbk translate
 	xsltproc --nonet -o $@ \
 	  /usr/share/sgml/docbook/stylesheet/xsl/nwalsh/manpages/docbook.xsl $<
diff --git a/po4a/devscripts-po4a.conf b/po4a/devscripts-po4a.conf
index 27eae1d..eae9d1e 100644
--- a/po4a/devscripts-po4a.conf
+++ b/po4a/devscripts-po4a.conf
@@ -62,7 +62,7 @@
 	fr:fr/debuild.fr.1 add_fr:add_fr/translator_man.add
 [type:pod] ../scripts/desktop2menu.pl \
 	fr:fr/desktop2menu.fr.pl add_fr:add_fr/translator_pod.add
-[type:man] ../scripts/devscripts.1.in \
+[type:man] ../scripts/devscripts.1 \
 	fr:fr/devscripts.fr.1 add_fr:add_fr/translator_man.add
 [type:pod] ../scripts/dget.pl \
 	fr:fr/dget.fr.pl add_fr:add_fr/translator_pod.add
