--- 50dictionaries-common.el~	2008-02-25 13:45:13.000000000 +0100
+++ 50dictionaries-common.el	2008-05-21 07:57:26.295827281 +0200
@@ -28,7 +28,8 @@
     (if (not (file-exists-p "/usr/share/emacs/site-lisp/dictionaries-common/debian-ispell.el"))
 	(message "Info: Package dictionaries-common removed but not purged.")
       (load "debian-ispell" t)
-      (load debian-dict-entries t))
+      (let ((coding-system-for-read 'raw-text))
+	(load debian-dict-entries t)))
     ))
 
 ;;; Previous code for loading ispell.el and refreshing spell-checking
