--- support/htdigest.c	2012-02-03 16:18:17.000000000 +0100
+++ /home/nirgal/htdigest.c	2012-02-03 16:16:42.000000000 +0100
@@ -212,7 +212,7 @@
     if (argc == 5) {
         if (strcmp(argv[1], "-c"))
             usage();
-        rv = apr_file_open(&f, argv[2], APR_WRITE | APR_CREATE,
+        rv = apr_file_open(&f, argv[2], APR_WRITE | APR_CREATE | APR_TRUNCATE,
                            APR_OS_DEFAULT, cntxt);
         if (rv != APR_SUCCESS) {
             char errmsg[120];
