--- /usr/share/apt-cacher/apt-cacher-cleanup.pl	2012-02-18 17:01:43.000000000 +0100
+++ /usr/local/bin/apt-cacher-cleanup-2.pl	2012-04-18 10:24:15.123830145 +0200
@@ -188,7 +188,7 @@
 	return;
     }
     my ($basename,$type) = ($name =~ /(^.+?)(\.(?:bz2|gz))?$/);
-    (my $release = $basename) =~ s/(?:main|contrib|non-free).*$/Release/;
+    (my $release = $basename) =~ s/(?:main|contrib|non-free).*$/InRelease/;
     (my $diffindex = $basename) .= '.diff_Index';
 
     for ($release, $diffindex) {
