diff -ur dhcpdump-1.8/debian/rules dhcpdump-1.8.new/debian/rules
--- dhcpdump-1.8/debian/rules	2011-04-11 16:09:46.000000000 +0200
+++ dhcpdump-1.8.new/debian/rules	2011-04-11 17:00:08.000000000 +0200
@@ -36,8 +36,14 @@
 STRIP = -s
 endif
 
+ARCH = $(shell dpkg-architecture -qDEB_HOST_ARCH)
+ifneq (,$(findstring $(ARCH), hurd-i386)) 
+  EXTRAFLAG="-D_BSD_SOURCE"
+endif
+
+
 build:
-	$(MAKE) CFLAGS="$(CFLAGS) -DHAVE_STRSEP"
+	$(MAKE) CFLAGS="$(CFLAGS) $(EXTRAFLAG) -DHAVE_STRSEP"
 	touch stamp-build
 	cat debian/copyright LICENSE > copyright.txt
