--- acpid-2.0.1.orig/connection_list.c
+++ acpid-2.0.1/connection_list.c
@@ -32,7 +32,7 @@
 /*---------------------------------------------------------------*/
 /* private objects */
 
-#define MAX_CONNECTIONS 10
+#define MAX_CONNECTIONS 20
 
 static struct connection connection_list[MAX_CONNECTIONS];
 
@@ -92,8 +92,15 @@
 			break;
 		}
 	}
+
+	/* prepare for recalculation of highestfd */
+	highestfd = -2;
 	
-	/* ??? might be nice to re-evaluate highestfd */
+	/* recalculate highestfd */
+	for (i = 0; i < nconnections; ++i) {
+		highestfd = max(highestfd, connection_list[i].fd);
+	}
+
 }
 
 /*---------------------------------------------------------------*/

