--- ant/core/trunk/src/main/org/apache/tools/ant/taskdefs/Copy.java	2008/08/29 14:01:32	690263
+++ ant/core/trunk/src/main/org/apache/tools/ant/taskdefs/Copy.java	2008/08/29 14:02:42	690264
@@ -73,6 +73,8 @@
     protected File destFile = null; // the destination file
     protected File destDir = null;  // the destination directory
     protected Vector rcs = new Vector();
+    // here to provide API backwards compatibility
+    protected Vector filesets = rcs;
 
     private boolean enableMultipleMappings = false;
     protected boolean filtering = false;
