diff --git a/lib/Debian/Pkg/DebSrc.pm b/lib/Debian/Pkg/DebSrc.pm
index b32267de1253a1311340af77426f58254f4bd1fb..6f0def45b7f60d707c18cb537253d2cbe4ccf909 100644
--- a/lib/Debian/Pkg/DebSrc.pm
+++ b/lib/Debian/Pkg/DebSrc.pm
@@ -231,8 +231,7 @@ sub get_control {
         my $self = shift;
 
         my $fullpath = File::Spec->catfile($self->{pkgtmpdir}, "debian/control");
-        my $control = Dpkg::Control::Info->new();
-        $control->load($fullpath);
+        my $control = Dpkg::Control::Info->new($fullpath);
         return $control;
 }
 
