diff --git a/debian/ddclient.templates b/debian/ddclient.templates
index bb62242..9fa7cc3 100644
--- a/debian/ddclient.templates
+++ b/debian/ddclient.templates
@@ -25,7 +25,7 @@ _Description: Dynamic DNS server:
 
 Template: ddclient/protocol
 Type: select
-Choices: dyndns2, dslreports1, easydns, hammernode1, zoneedit1, dyndns1
+Choices: changeip, cloudflare, concont, dnspark, dslreports1, dtdns, duckdns, dyndns1, dyndns2, easydns, freedns, googledomains, hammernode1, namecheap, noip, nsupdate, sitelutions, zoneedit1
 Default: dyndns2
 _Description: Dynamic DNS update protocol:
  Please select the dynamic DNS update protocol used by your dynamic DNS service
