diff -Naur duplicity-0.6.13.orig/duplicity duplicity-0.6.13/duplicity
--- duplicity-0.6.13.orig/duplicity	2011-04-02 14:48:22.000000000 -0300
+++ duplicity-0.6.13/duplicity	2011-05-04 16:28:33.000000000 -0300
@@ -1141,7 +1141,8 @@
     check_resources(action)
 
     # check archive synch with remote, fix if needed
-    sync_archive()
+    if action not in ["collection-status"]:
+      sync_archive()    
 
     # get current collection status
     col_stats = collections.CollectionsStatus(globals.backend,
