--- autopostgresqlbackup.old	2013-09-24 19:25:00.000000000 +0200
+++ autopostgresqlbackup	2013-10-17 02:28:17.000000000 +0200
@@ -99,14 +99,6 @@
 # Command run after backups (uncomment to use)
 #POSTBACKUP="/etc/postgresql-backup-post"
 
-# ===============================
-# === Debian specific options ===
-#================================
-
-if [ -f /etc/default/autopostgresqlbackup ]; then
-	. /etc/default/autopostgresqlbackup
-fi
-
 #=====================================================================
 # Options documentation
 #=====================================================================
@@ -269,6 +261,14 @@
 BACKUPFILES=""
 OPT=""						# OPT string for use with pg_dump ( see man pg_dump )
 
+# ===============================
+# === Debian specific options ===
+#================================
+
+if [ -f /etc/default/autopostgresqlbackup ]; then
+	. /etc/default/autopostgresqlbackup
+fi
+
 # Add --compress pg_dump option to $OPT
 if [ "$COMMCOMP" -gt 0 ];
 	then
