diff -up awesome_PKG/rc.lua awesome_PATCHED/rc.lua
--- awesome_PKG/rc.lua	2008-11-17 11:19:22.000000000 +0100
+++ awesome_PATCHED/rc.lua	2008-11-17 11:19:35.000000000 +0100
@@ -102,8 +102,8 @@ myawesomemenu = {
 }
 
 mymainmenu = awful.menu.new({ items = { { "awesome", myawesomemenu, beautiful.awesome_icon },
-                                        { "open terminal", terminal }
-                                        { "Debian", debian.menu.Debian_menu.Debian },
+                                        { "open terminal", terminal },
+                                        { "Debian", debian.menu.Debian_menu.Debian }
                                       }
                             })
 
