diff -urN debsecan-0.4.16+nmu1/debian/debsecan.config debsecan-0.4.16+nmu2/debian/debsecan.config
--- debsecan-0.4.16+nmu1/debian/debsecan.config	2011-10-06 15:02:56.000000000 +0200
+++ debsecan-0.4.16+nmu2/debian/debsecan.config	2013-08-21 07:47:44.970399819 +0200
@@ -10,7 +10,7 @@
 
 normalize_suite () {
     case "$SUITE" in
-	sarge|etch|lenny|squeeze|wheezy|sid)
+	sarge|etch|lenny|squeeze|wheezy|jessie|sid)
 	    ;;
 	*)
 	    SUITE=GENERIC
diff -urN debsecan-0.4.16+nmu1/debian/debsecan.templates debsecan-0.4.16+nmu2/debian/debsecan.templates
--- debsecan-0.4.16+nmu1/debian/debsecan.templates	2011-10-06 15:02:56.000000000 +0200
+++ debsecan-0.4.16+nmu2/debian/debsecan.templates	2013-08-21 07:48:08.342094074 +0200
@@ -16,7 +16,7 @@
 
 Template: debsecan/suite
 Type: select
-Choices: GENERIC, sarge, etch, lenny, squeeze, wheezy, sid
+Choices: GENERIC, sarge, etch, lenny, squeeze, wheezy, jessie, sid
 Default: GENERIC
 _Description: Main suite from which packages are installed:
  To present more useful data, debsecan needs to know
diff -urN debsecan-0.4.16+nmu1/src/debsecan debsecan-0.4.16+nmu2/src/debsecan
--- debsecan-0.4.16+nmu1/src/debsecan	2011-10-06 15:02:56.000000000 +0200
+++ debsecan-0.4.16+nmu2/src/debsecan	2013-08-21 07:45:28.152189248 +0200
@@ -270,7 +270,7 @@
                       default='/etc/default/debsecan')
     parser.add_option("--suite", type="choice",
                       choices=['woody', 'sarge', 'etch', 'lenny', 'squeeze', 'wheezy',
-                               'sid'],
+                               'jessie', 'sid'],
                       help="set the Debian suite of this installation")
     parser.add_option("--source", metavar="URL",
                       help="sets the URL for the vulnerability information")
