diff --git a/debsecan b/debsecan.new
index 7b869b7..559d49d 100755
--- a/debsecan
+++ b/debsecan.new
@@ -270,7 +270,7 @@ def parse_cli():
                       default='/etc/default/debsecan')
     parser.add_option("--suite", type="choice",
                       choices=['woody', 'sarge', 'etch', 'lenny', 'squeeze', 'wheezy',
-                               'sid'],
+                               'jessie', 'sid'],
                       help="set the Debian suite of this installation")
     parser.add_option("--source", metavar="URL",
                       help="sets the URL for the vulnerability information")
