Index: debian/control
===================================================================
--- debian/control	(revision 2014)
+++ debian/control	(working copy)
@@ -33,7 +33,6 @@
                libgconf2-dev,
                libnss3-dev,
                libgtk-3-dev (>= 3.0),
-               libgnome2-dev,
                libsoup2.4-dev (>= 2.31.2),
                libgnome-keyring-dev (>= 2.20.1),
                gtk-doc-tools,
@@ -42,14 +41,16 @@
                libgweather-3-dev (>= 2.90.0),
                libical-dev,
                libicu-dev,
-               libgdata-dev (>= 0.7.0),
+               libgdata-dev (>= 0.9.1),
                libgirepository1.0-dev (>= 0.9.12),
                gobject-introspection (>= 0.9.12-4~),
                gir1.2-glib-2.0,
                gir1.2-freedesktop,
                gperf,
                gnome-pkg-tools,
-               liboauth-dev (>= 0.9.4)
+               liboauth-dev (>= 0.9.4),
+               libgoa-1.0-dev,
+               valac-0.14
 
 Package: evolution-data-server
 Architecture: any
Index: debian/libecal1.2-dev.install
===================================================================
--- debian/libecal1.2-dev.install	(revision 2014)
+++ debian/libecal1.2-dev.install	(working copy)
@@ -3,3 +3,5 @@
 debian/tmp/usr/include/evolution-data-server-*/libecal
 debian/tmp/usr/share/gtk-doc/html/libecal
 debian/tmp/usr/share/gir-1.0/ECalendar-1.2.gir
+debian/tmp/usr/share/vala/vapi/libecalendar-1.2.deps
+debian/tmp/usr/share/vala/vapi/libecalendar-1.2.vapi
Index: debian/libedataserver1.2-dev.install
===================================================================
--- debian/libedataserver1.2-dev.install	(revision 2014)
+++ debian/libedataserver1.2-dev.install	(working copy)
@@ -3,3 +3,4 @@
 debian/tmp/usr/include/evolution-data-server-*/libedataserver
 debian/tmp/usr/share/gtk-doc/html/libedataserver
 debian/tmp/usr/share/gir-1.0/EDataServer-1.2.gir
+debian/tmp/usr/share/vala/vapi/libedataserver-1.2.vapi
Index: debian/changelog
===================================================================
--- debian/changelog	(revision 2014)
+++ debian/changelog	(working copy)
@@ -1,5 +1,6 @@
-evolution-data-server3 (3.2.0-1) UNRELEASED; urgency=low
+evolution-data-server3 (3.2.1-1) UNRELEASED; urgency=low
 
+  [ Yves-Alexis Perez ]
   * New upstream release.
     - correctly use secure connection (if used in imap) when storing mail in
       sent folder. This fixes CVE-2011-3355.                    closes: #641052
@@ -17,8 +18,14 @@
     - use --disable-goa since libgoa is not in Debian.
   * debian/compat bumped to 8.
 
- -- Yves-Alexis Perez <corsac@debian.org>  Fri, 30 Sep 2011 23:31:08 +0200
+  [ Sjoerd Simons ]
+  * New upstream release
+  * enable goa
+  * Update libgdata b-d for goa
+  * Enable vala bindings
 
+ -- Sjoerd Simons <sjoerd@debian.org>  Sun, 06 Nov 2011 20:23:39 +0000
+
 evolution-data-server3 (3.0.3-2) unstable; urgency=low
 
   * debian/control:
Index: debian/libebook1.2-dev.install
===================================================================
--- debian/libebook1.2-dev.install	(revision 2014)
+++ debian/libebook1.2-dev.install	(working copy)
@@ -3,3 +3,5 @@
 debian/tmp/usr/include/evolution-data-server-*/libebook
 debian/tmp/usr/share/gtk-doc/html/libebook
 debian/tmp/usr/share/gir-1.0/EBook-1.2.gir
+debian/tmp/usr/share/vala/vapi/libebook-1.2.vapi
+debian/tmp/usr/share/vala/vapi/libebook-1.2.deps
Index: debian/rules
===================================================================
--- debian/rules	(revision 2014)
+++ debian/rules	(working copy)
@@ -30,7 +30,9 @@
 	--with-libdb=/usr \
 	--enable-gtk-doc \
 	--enable-gnome-keyring=yes \
-	--disable-goa
+	--enable-goa \
+	--enable-gojb \
+	--enable-vala-bindings
 
 DEB_FIXPERMS_EXCLUDE_evolution-data-server += usr/lib/evolution/camel-lock-helper*
 
