diff -uprN ./asylum_orig/asylum.c ./asylum-0.3.2/asylum.c
--- ./asylum_orig/asylum.c	2009-08-03 00:21:07.000000000 +0200
+++ ./asylum-0.3.2/asylum.c	2015-05-12 11:09:27.000000000 +0200
@@ -665,7 +665,7 @@ char idpermitstring[] = "You are now per
 
 void loadconfig()
 {
-    char keyword[12];
+    char keyword[13];
 
     FILE* r0 = find_config(0x40); // read access
     if (r0 != NULL)
diff -uprN ./asylum_orig/file.c ./asylum-0.3.2/file.c
--- ./asylum_orig/file.c	2008-12-10 05:12:29.000000000 +0100
+++ ./asylum-0.3.2/file.c	2015-05-12 14:18:36.000000000 +0200
@@ -142,7 +142,7 @@ int loadvitalfile(char** spaceptr, char*
     strcat(fullname, r1);
     int r4 = swi_osfile(15, fullname, 0, 0);
     if (r4 <= 0) fatalfile();
-    *spaceptr = (char*)malloc(r4);
+    *spaceptr = (char*)malloc(r4+1);
     if (swi_osfile(14, fullname, *spaceptr, 0)) fatalfile();
     return r4;
 }
@@ -362,7 +362,11 @@ int swi_blitz_hammerop(int op, char* nam
         fclose(f); return op;
     }                            // file is not Hammered
 
-    if (op == 0) return 0x24000; // hack: should return length
+    if (op == 0)
+    {
+        fclose(f);
+        return 0x024000; // hack: should return length
+    }
     char a[524288];
     int p = 0;
     char c;
