--- apt-cacher~	2008-05-19 00:54:01.000000000 +0300
+++ apt-cacher	2008-05-19 00:54:01.000000000 +0300
@@ -643,7 +643,7 @@
         $do_import=0;
         $is_index_file=0;
 
-        if ($filename =~ /(\.deb|\.rpm|\.dsc|\.tar\.gz|\.diff\.gz|\.udeb)$/) {
+        if ($filename =~ /(\.deb|\.rpm|\.dsc|\.tar\.gz|\.diff\.gz|\.udeb|\.jigdo|\.template)$/) {
             # We must be fetching a .deb or a .rpm, so let's cache it.
             # Place the file in the cache with just its basename
             $new_filename = $filename;
