diff -Nru clamav-0.67-6/debian/control clamav-0.67/debian/control
--- clamav-0.67-6/debian/control	Tue Mar  2 08:19:57 2004
+++ clamav-0.67/debian/control	Thu Mar  4 20:29:32 2004
@@ -3,7 +3,7 @@
 Priority: optional
 Maintainer: Stephen Gran <sgran@debian.org>
 Uploaders: Thomas Lamy <Thomas.Lamy@in-online.net>, Magnus Ekdahl <magnus@debian.org>
-Build-Depends: debhelper (>=4), zlib1g-dev (>=1:1.1.4), libbz2-dev, libmilter-dev, libgmp3-dev, gcc (>=3.0)
+Build-Depends: debhelper (>=4), zlib1g-dev (>=1:1.1.4), libbz2-dev, libmilter-dev, libgmp3-dev, gcc (>=3.0)|gcc-3.0
 Standards-Version: 3.6.1
 
 Package: clamav-base
diff -Nru clamav-0.67-6/debian/rules clamav-0.67/debian/rules
--- clamav-0.67-6/debian/rules	Tue Mar  2 08:19:57 2004
+++ clamav-0.67/debian/rules	Thu Mar  4 21:18:03 2004
@@ -13,6 +13,18 @@
 DEB_HOST_GNU_TYPE   ?= $(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)
 DEB_BUILD_GNU_TYPE  ?= $(shell dpkg-architecture -qDEB_BUILD_GNU_TYPE)
 
+# Attempt to find a version of gcc that will compile clamav successfully
+# We use the first of ($CC, gcc, gcc-3.0) which has a version >= 3.0
+CC := $(shell sh -c 'for f in "$$CC" "gcc" "gcc-3.0"; do \
+  ver=`$$f -v 2>&1 | egrep "^gcc version " | cut -d" " -f3`; \
+  `dpkg --compare-versions $${ver} ge 3.0 >/dev/null 2>&1`; \
+  [ x$$? = x0 ] && echo $$f; done | head -n1')
+
+ifeq ($(strip $(CC)),)
+  $(error gcc >= 3.0 is required to build this package - if you \
+    are using woody, install the `gcc-3.0' package)
+endif
+
 CFLAGS = -Wall -g
 
 ifneq (,$(findstring noopt,$(DEB_BUILD_OPTIONS)))
@@ -38,7 +50,7 @@
 config.status: configure
 	dh_testdir
 	# Add here commands to configure the package.
-	./configure --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE) --prefix=/usr --mandir=\$${prefix}/share/man --infodir=\$${prefix}/share/info --disable-clamav --with-dbdir=/var/lib/clamav/ --sysconfdir=/etc/clamav --enable-milter --disable-clamuko
+	CC=$(CC) ./configure --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE) --prefix=/usr --mandir=\$${prefix}/share/man --infodir=\$${prefix}/share/info --disable-clamav --with-dbdir=/var/lib/clamav/ --sysconfdir=/etc/clamav --enable-milter --disable-clamuko
 
 build: build-stamp
 build-stamp: build-arch build-indep
