diff --git a/debmirror b/debmirror
index 83c1417..48582ed 100755
--- a/debmirror
+++ b/debmirror
@@ -1597,7 +1597,7 @@ sub http_get {
   my $ret=1;
 
   print "$url => " if ($debug);
-  print_percent "Getting: $file" if $progress or $verbose;
+  print_percent "Getting: $file... " if $progress or $verbose;
   print "\t #" if $progress;
   if (! $do_dry_run) {
     unlink($file) if (-f $file);
@@ -1605,7 +1605,7 @@ sub http_get {
     print $ret->status_line . "\n" if ($debug);
     if ($ret->is_error) {
       $files{$file} = -1;
-      warn "$file failed " . $ret->status_line . "\n" if ($progress or $verbose);
+      warn "failed " . $ret->status_line . "\n" if ($progress or $verbose);
       push (@errlog,"Download of $file failed: ".$ret->status_line."\n");
       $num_errors++;
     } elsif ($progress || $verbose) {
