--- smazat/init	2008-02-09 19:45:04.000000000 +0100
+++ a/init	2008-02-09 20:31:20.000000000 +0100
@@ -209,14 +209,14 @@
 
 	echo -n "Loading.."
 	# pipe_progress adds dots to the above line while there is
-	# activity. But we must be sure to catch errors from the zcat.
+	# activity. But we must be sure to catch errors from the micro-bunzip.
 	# Hard to do in a pipeline..
-	echo 0 > /tmp/zcat_failure
+	echo 0 > /tmp/bunzip_failure
 	cd mnt
-	(zcat ../floppy/initrd.gz || echo 1 > /tmp/zcat_failure ) | cpio -i -V || abort "failed to extract initrd (may be out of space on ram disk)"
+	(micro-bunzip <../floppy/initrd.bz2 || echo 1 > /tmp/bunzip_failure ) | cpio -i -V || abort "failed to extract initrd (may be out of space on ram disk)"
 	cd ..
 
-	if [ "`cat /tmp/zcat_failure`" = 0 ]; then
+	if [ "`cat /tmp/bunzip_failure`" = 0 ]; then
 		LOADED=1
 	else
 		echo "install media seems to be bad!" >&2
