--- S15lowmem-orig 2008-02-09 19:11:45.000000000 +0100
+++ S15lowmem      2007-09-29 18:44:51.000000000 +0200
@@ -4,7 +4,11 @@
 if [ "$ram" = "" ]; then
        echo "Cannot determine system memory, skipping lowmem probe" >&2
 else
-       ram=$(expr "$ram" / 1024) # convert to megabytes
+       swap=$(grep ^SwapTotal: /proc/meminfo | { read x y z; echo $y; }) || true
+       if [ "$swap" = "" ]; then
+               swap=0
+       fi
+       ram=$(expr \( "$ram" + "$swap" \) / 1024) # convert to megabytes

        # Set level1 to the minimum amount of memory that will support an
        # install not in lowmem mode. (This is the max memory footprint of
