From 56432ee843171e75596da9f5237025c80c61b5d8 Mon Sep 17 00:00:00 2001
From: Osamu Aoki <osamu@debian.org>
Date: Fri, 4 Sep 2015 02:47:20 +0000
Subject: [PATCH 1/4] s/dsfg/dfsg/g

---
 scripts/uscan.pl | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/scripts/uscan.pl b/scripts/uscan.pl
index 8dd5713..714bff0 100644
--- a/scripts/uscan.pl
+++ b/scripts/uscan.pl
@@ -180,14 +180,14 @@ is determined to be B<2.03+dfsg1> without the epoch and the Debian revision.
 If the B<dversionmangle> rule exists, the last upstream version is updated by
 applying this rule to it.  For example, if the last upstream version is
 B<2.03+dfsg1> indicating the source tarball is repackaged, the suffix B<+dfsg1>
-is removed by the string substitution B<s/\+dsfg\d*$//> to make the
+is removed by the string substitution B<s/\+dfsg\d*$//> to make the
 (dversionmangled) last upstream version B<2.03> and it is compared to the
 candidate upstream tarballs such as B<2.03>, B<2.04>, ... .  Thus, set this
 rule as:
 
 =over
 
-=item  B<opts="dversionmangle=s/\+dsfg\d*$//">
+=item  B<opts="dversionmangle=s/\+dfsg\d*$//">
 
 =back
 
@@ -348,7 +348,7 @@ The removal of files is required if files are not DFSG-compliant.  For such
 case, B<+dfsg1> is used as I<suffix>.
 
 So the combined per-site options are set as
-B<opts="dversionmangle=s/\+dsfg\d*$// ,repacksuffix=+dfsg1">, instead.
+B<opts="dversionmangle=s/\+dfsg\d*$// ,repacksuffix=+dfsg1">, instead.
 
 For example, the repacked upstream tarball may be:
 
-- 
2.1.4

