From 203bb5c2c62b03dde75905e35187c943decd4818 Mon Sep 17 00:00:00 2001
From: Osamu Aoki <osamu@debian.org>
Date: Tue, 1 Sep 2015 13:31:54 +0000
Subject: [PATCH 5/6] POD+po4a: uscan.pl

---
 po4a/devscripts-po4a.conf | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/po4a/devscripts-po4a.conf b/po4a/devscripts-po4a.conf
index ff6c490..f732ec8 100644
--- a/po4a/devscripts-po4a.conf
+++ b/po4a/devscripts-po4a.conf
@@ -120,8 +120,8 @@
 	$lang:$lang/tagpending.$lang.pl add_$lang:?add_$lang/translator_pod.add
 [type:pod] ../scripts/transition-check.pl \
 	$lang:$lang/transition-check.$lang.pl add_$lang:?add_$lang/translator_pod.add
-[type:man] ../scripts/uscan.1 \
-	$lang:$lang/uscan.$lang.1 add_$lang:?add_$lang/translator_man.add
+[type:pod] ../scripts/uscan.pl \
+	$lang:$lang/uscan.$lang.pl add_$lang:?add_$lang/translator_pod.add
 [type:man] ../scripts/uupdate.1 \
 	$lang:$lang/uupdate.$lang.1 add_$lang:?add_$lang/translator_man.add
 [type:man] ../doc/what-patch.1 \
-- 
2.1.4

