diff --git a/scripts/t/200_Dpkg_Shlibs.t b/scripts/t/200_Dpkg_Shlibs.t
index ba084a6..cc3307b 100644
--- a/scripts/t/200_Dpkg_Shlibs.t
+++ b/scripts/t/200_Dpkg_Shlibs.t
@@ -159,7 +159,8 @@ use File::Temp;
 use File::Basename qw(basename);
 
 sub save_load_test {
-    my ($symfile, $comment) = @_;
+    my $symfile = shift;
+    my $comment = shift;
 
     my $save_file = new File::Temp;
     $symfile->save($save_file->filename, @_);
