From 2138b8ce9337f85087ded3922cdcc68c7d03319f Mon Sep 17 00:00:00 2001
From: =?utf-8?q?J=C3=B6rg=20Sommer?= <joerg@alea.gnuu.de>
Date: Thu, 21 Aug 2008 14:46:27 +0200
Subject: [PATCH 4/5] Replace %7E by ~ in URLs

---
 getbuildlog |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/getbuildlog b/getbuildlog
index fc70317..d1e7508 100755
--- a/getbuildlog
+++ b/getbuildlog
@@ -82,7 +82,7 @@ getbuildlog() {
     # matches only one href. This is required because grep is greedy.
     sed -i -e "s/href=\"/\nhref=\"/g" $ALL_LOGS
     # Quick-and-dirty unescaping
-    sed -i -e "s/%2B/\+/g" -e "s/%3A/:/g" $ALL_LOGS
+    sed -i -e "s/%2B/\+/g" -e "s/%3A/:/g" -e "s/%7E/~/g" $ALL_LOGS
 
     for match in `grep -E -o "$PATTERN" $ALL_LOGS`; do
         ver=${match##*ver=}
-- 
1.5.6.3

