From 6df8755b398d69672cd1ad25444f919d804774dc Mon Sep 17 00:00:00 2001
From: Simon McVittie <smcv@debian.org>
Date: Wed, 25 May 2016 07:23:53 +0100
Subject: [PATCH 09/11] Run basic and dev tests in addition to userns

---
 debian/tests/control | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/debian/tests/control b/debian/tests/control
index 9774a97..e3bb376 100644
--- a/debian/tests/control
+++ b/debian/tests/control
@@ -1,4 +1,6 @@
 Tests:
+ basic
+ dev
  userns
 Depends:
  bats,
-- 
2.8.1

