From 59d714ecadab4f2855b27df924e543be1922d063 Mon Sep 17 00:00:00 2001
From: Simon McVittie <smcv@debian.org>
Date: Wed, 25 May 2016 07:29:09 +0100
Subject: [PATCH 11/11] Close bug

---
 debian/changelog | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/changelog b/debian/changelog
index 7435098..b7ffbc7 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -8,7 +8,7 @@ bubblewrap (0~git160513-3) UNRELEASED; urgency=medium
     depended on by gnome-software (via Flatpak) in future
   * debian/gbp.conf: add DEP-14-style git-buildpackage configuration
   * Add patches from upstream bug 71 to make --dev coexist with
-    --unshare-user
+    --unshare-user (Closes: #824968)
   * Add some simple autopkgtests, including one for bug 71
 
  -- Simon McVittie <smcv@debian.org>  Sat, 21 May 2016 15:10:56 +0100
-- 
2.8.1

