--- /usr/bin/apt-file	2011-02-08 07:17:16.000000000 +0800
+++ apt-file.mod	2011-05-17 23:00:47.000000000 +0800
@@ -184,6 +184,8 @@
             }
 
             foreach ( @{ ( unique $fetch) } ) {
+                for my $comp (@extra) {
+                    debug "comp is $comp";
                 if ( !defined $Conf->{"${scheme}"} ) {
                     warningx "Don't know how to handle $scheme";
                     next;
@@ -203,7 +205,7 @@
                 my $cdrom = $Conf->{cdrom_mount};
                 foreach my $var (
                     qw/host port user passwd path dist pkg
-                    cache arch uri cdrom/
+                    cache arch uri cdrom comp/  
                     )
                 {
                     map {
@@ -226,6 +228,7 @@
                     $hash->{$_} = eval "\$$_";
                 }
                 push @$ret, $hash;
+                }
             }
         }
         close SOURCE;
