diff --git a/debian/control b/debian/control
index f5ced8b..b23dbaa 100644
--- a/debian/control
+++ b/debian/control
@@ -9,8 +9,8 @@ Vcs-Git: git://git.debian.org/d-i/base-installer.git
 
 Package: base-installer
 XC-Package-Type: udeb
-Architecture: all
-Depends: ${shlibs:Depends}, ${misc:Depends}, di-utils-mapdevfs, di-utils (>= 1.48), rootskel (>= 1.19)
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends}, di-utils-mapdevfs, di-utils (>= 1.48), rootskel (>= 1.19), nullfs-modules [kfreebsd-any]
 Description: base system installation framework
 
 Package: bootstrap-base
