--- /usr/bin/uscan	2011-02-19 18:14:43.000000000 +0100
+++ uscan	2011-02-25 20:29:01.023208309 +0100
@@ -1369,7 +1369,7 @@
 
 	my $newfile_base_gz = "$1.tar.gz";
 	my $tempdir = tempdir ( "uscanXXXX", TMPDIR => 1, CLEANUP => 1 );
-	system("unzip -q -d $tempdir $destdir/$newfile_base; GZIP=-9 tar -C $tempdir -czf $destdir/$newfile_base_gz .") == 0 
+	system("unzip -q -d $tempdir $destdir/$newfile_base; cd $tempdir; GZIP=-9 tar --owner=root --group=root --mode=a+rX -czf $destdir/$newfile_base_gz *") == 0 
 	  or die("Repacking from zip to tar.gz failed\n");
 	unlink "$destdir/$newfile_base";
 	$newfile_base = $newfile_base_gz;
