From cf7791e084f46e586095c1973117711d5ccbedf2 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Sat, 22 Feb 2014 19:24:32 +0100
Subject: [PATCH] debian: Add debDebFile::ExtractTarMember to the symbols file

---
 debian/libapt-inst1.5.symbols | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/libapt-inst1.5.symbols b/debian/libapt-inst1.5.symbols
index 0307953..35cce91 100644
--- a/debian/libapt-inst1.5.symbols
+++ b/debian/libapt-inst1.5.symbols
@@ -9,6 +9,7 @@ libapt-inst.so.1.5 libapt-inst1.5 #MINVER#
  (c++)"debDebFile::CheckMember(char const*)@Base" 0.8.0
  (c++)"debDebFile::ControlExtract::DoItem(pkgDirStream::Item&, int&)@Base" 0.8.0
  (c++)"debDebFile::ControlExtract::~ControlExtract()@Base" 0.8.0
+ (c++)"debDebFile::ExtractTarMember(pkgDirStream&, char const*)@Base" 0.9.15.4
  (c++)"debDebFile::ExtractArchive(pkgDirStream&)@Base" 0.8.0
  (c++)"debDebFile::MemControlExtract::TakeControl(void const*, unsigned long)@Base" 0.8.0
  (c++)"debDebFile::MemControlExtract::Read(debDebFile&)@Base" 0.8.0
-- 
1.9.0.279.gdc9e3eb

