diff --git a/src/consolation.h b/src/consolation.h
index c8278e0..ee9fd4e 100644
--- a/src/consolation.h
+++ b/src/consolation.h
@@ -20,10 +20,10 @@
 extern int nodaemon;
 
 enum current_button {
-  BUTTON_RELEASED,
   BUTTON_LEFT,
   BUTTON_MIDDLE,
-  BUTTON_RIGHT
+  BUTTON_RIGHT,
+  BUTTON_RELEASED
 };
 
 /* global state */
diff --git a/src/selection.c b/src/selection.c
index 4f65863..0d4c1e9 100644
--- a/src/selection.c
+++ b/src/selection.c
@@ -78,7 +78,7 @@ linux_selection(int xs, int ys, int xe, int ye, int sel_mode)
 void
 report_pointer(int x, int y, enum current_button button)
 {
-  linux_selection(x, y, x, y, TIOCL_SELMOUSEREPORT + ((button+3) % 4) );
+  linux_selection(x, y, x, y, TIOCL_SELMOUSEREPORT + button );
 }
 
 void
