--- black-box-1.4.7.orig/src/blackbox.c
+++ black-box-1.4.7/src/blackbox.c
@@ -646,7 +555,7 @@
 
 void generate_field()
 {
-  int x,y,crystal_num;
+  int x,y;
   
   for (x=0;x<=10;x++){
     for (y=0;y<=8;y++){
@@ -654,9 +563,9 @@
       light[x][y]=0;
     }
   }
-  while ((crystal_num>difficulty*3)||(crystal_num<(difficulty-1)*3))
+  for (;;)
   {
-    crystal_num=0;
+    int crystal_num=0;
     srand(level_num);
     for (x=0;x<=9;x++){
       for (y=0;y<=7;y++){
@@ -669,14 +578,17 @@
         if (real[x][y]==1) {crystal_num++;}
       }
     }  
-    sprintf(text,"Black-Box: Level #%d",level_num);
     if (LevelGiven) {
 	LevelGiven=0;
 	break;
     }
-    level_num=(((unsigned)time(NULL)+SDL_GetTicks())%100000);
+    if (((difficulty-1)*3<=crystal_num)&&(crystal_num<=difficulty*3)) {
+        break;
+    }
+    level_num=(level_num+1)%100000;
 //    printf("%d\n",crystal_num); 
   }
+  sprintf(text,"Black-Box: Level #%d",level_num);
   for (x=0;x<=10;x++)
   {
     yline[x]=0;
