Index: src/modules/frontend/newt/newt.c
===================================================================
--- src/modules/frontend/newt/newt.c	(revision 46471)
+++ src/modules/frontend/newt/newt.c	(working copy)
@@ -183,11 +183,11 @@
             if (asprintf(&buf, "[%s] %s", sigils[i][1], title) == -1)
                 buf = NULL;
     }
-    if (buf != NULL) {
-        newtCenteredWindow(width, height, buf);
+    if (buf != NULL) {		/* Make sure borders surrounding the title are always balanced */
+        newtCenteredWindow(width - (((width - strlen(buf))%2)?1:0), height, buf);
         free(buf);
     } else {
-        newtCenteredWindow(width, height, title);
+        newtCenteredWindow(width - (((width - strlen(title))%2)?1:0), height, title);
     }
 }
 
