--- writer.cc.original	2015-05-28 23:34:13.335067263 +0000
+++ writer.cc	2015-05-28 23:35:07.396137053 +0000
@@ -1025,6 +1025,7 @@ ReleaseWriter::ReleaseWriter(string cons
    DoMD5 = _config->FindB("APT::FTPArchive::Release::MD5",DoMD5);
    DoSHA1 = _config->FindB("APT::FTPArchive::Release::SHA1",DoSHA1);
    DoSHA256 = _config->FindB("APT::FTPArchive::Release::SHA256",DoSHA256);
+   DoSHA512 = _config->FindB("APT::FTPArchive::Release::SHA512",DoSHA512);
 }
 									/*}}}*/
 // ReleaseWriter::DoPackage - Process a single package			/*{{{*/
