Index: tools/boot/etch/boot-powerpc
===================================================================
--- tools/boot/etch/boot-powerpc	(revision 1386)
+++ tools/boot/etch/boot-powerpc	(working copy)
@@ -75,6 +75,9 @@
  > boot.msg
 cat $BASEDIR/data/$DI_CODENAME/yaboot/yaboot.conf \
  | sed "s/CODENAME/${CODENAME}/g" > yaboot.conf
+if [ -n "$KERNEL_PARAMS" ]; then
+    sed -ie "s|append=\"|append=\"$KERNEL_PARAMS |g" yaboot.conf
+fi
 cp $BASEDIR/data/$DI_CODENAME/yaboot/ofboot.b ofboot.b
 
 for subarch in powerpc powerpc64 prep
@@ -123,5 +126,8 @@
 cp $BASEDIR/data/$DI_CODENAME/chrp/bootinfo.txt ppc
 mkdir etc
 cp $INSTALLDIR/yaboot.conf etc
+if [ -n "$KERNEL_PARAMS" ]; then
+    sed -ie "s|append=\"|append=\"$KERNEL_PARAMS |g" etc/yaboot.conf
+fi
 
 exit 0
Index: tools/boot/etch/boot-alpha
===================================================================
--- tools/boot/etch/boot-alpha	(revision 1386)
+++ tools/boot/etch/boot-alpha	(working copy)
@@ -80,9 +80,9 @@
     RSIZE=$(expr $(zcat boot$N/boot/initrd.gz | wc --bytes) / 1024)
     OPTS="ramdisk_size=$RSIZE initrd=/boot/initrd.gz root=/dev/ram devfs=mount,dall"
 
-    echo "0:boot/vmlinuz $OPTS"               >  boot$N/etc/aboot.conf
-    echo "1:boot/vmlinuz $OPTS console=ttyS0" >> boot$N/etc/aboot.conf
-    echo "2:boot/vmlinuz $OPTS console=ttyS1" >> boot$N/etc/aboot.conf
+    echo "0:boot/vmlinuz $OPTS $KERNEL_PARAMS"               >  boot$N/etc/aboot.conf
+    echo "1:boot/vmlinuz $OPTS console=ttyS0 $KERNEL_PARAMS" >> boot$N/etc/aboot.conf
+    echo "2:boot/vmlinuz $OPTS console=ttyS1 $KERNEL_PARAMS" >> boot$N/etc/aboot.conf
 
     mkdir -p boot$N/milo
 
Index: tools/boot/etch/boot-sparc
===================================================================
--- tools/boot/etch/boot-sparc	(revision 1386)
+++ tools/boot/etch/boot-sparc	(working copy)
@@ -46,6 +46,9 @@
 
 # Some custom etc files
 cp -f -p $BASEDIR/data/etch/sparc/silo.conf $inst/boot/
+if [ -n "$KERNEL_PARAMS" ]; then
+    sed -ie "s|append=\"|append=\"$KERNEL_PARAMS |g" $inst/boot/silo.conf
+fi
 cat $BASEDIR/data/etch/sparc/debian.txt \
  | sed "s/\${MEDIA_TYPE}/CDROM/" \
  | sed "s/\${DEBIAN_VERSION}/${CODENAME}/g" \
