--- ../../../../orig/adduser-3.104/doc/po4a/po4a.conf	2006-02-19 19:16:39.000000000 +0100
+++ po4a.conf	2007-08-04 00:27:30.000000000 +0200
@@ -2,6 +2,10 @@
 [po4a_paths] po/adduser.pot fr:po/fr.po es:po/es.po pt_BR:po/pt_BR.po \
 		ru:po/ru.po pl:po/pl.po sv:po/sv.po it:po/it.po
 
+# Force generation of the French and Russian manpages in the right encoding
+# (instead of the encoding of the PO file)
+[po4a_alias:man] man opt_fr:"-L ISO-8859-1" opt_ru:"-L KOI8-R"
+
 # Then list the documents to translate, their format, their translations
 # (as well as the addendums to apply to the translations)
 [type:man] ../adduser.8 \
@@ -10,7 +14,7 @@
 		pl:../adduser.8.pl add_pl:translator_polish.add \
 		sv:../adduser.8.sv add_sv:translator_swedish.add \
 		pt_BR:../adduser.8.pt_BR add_pt_BR:translator_pt_BR.add \
-		ru:../adduser.8.ru add_ru:translator_russian.add opt_ru:"-L KOI8-R" \
+		ru:../adduser.8.ru add_ru:translator_russian.add \
 		it:../adduser.8.it add_it:translator_italian.add
 [type:man] ../adduser.conf.5 \
 		fr:../adduser.conf.5.fr add_fr:translator_french2.add \
@@ -18,7 +22,7 @@
 		pl:../adduser.conf.5.pl add_pl:translator_polish.add \
 		sv:../adduser.conf.5.sv add_sv:translator_swedish.add \
 		pt_BR:../adduser.conf.5.pt_BR add_pt_BR:translator_pt_BR.add \
-		ru:../adduser.conf.5.ru add_ru:translator_russian.add opt_ru:"-L KOI8-R" \
+		ru:../adduser.conf.5.ru add_ru:translator_russian.add \
 		it:../adduser.conf.5.it add_it:translator_italian.add
 [type:man] ../deluser.8 \
 		fr:../deluser.8.fr add_fr:translator_french.add \
@@ -26,7 +30,7 @@
 		pl:../deluser.8.pl add_pl:translator_polish.add \
 		sv:../deluser.8.sv add_sv:translator_swedish.add \
 		pt_BR:../deluser.8.pt_BR add_pt_BR:translator_pt_BR.add \
-		ru:../deluser.8.ru add_ru:translator_russian.add opt_ru:"-L KOI8-R" \
+		ru:../deluser.8.ru add_ru:translator_russian.add \
 		it:../deluser.8.it add_it:translator_italian.add
 [type:man] ../deluser.conf.5 \
 		fr:../deluser.conf.5.fr add_fr:translator_french.add \
@@ -34,5 +38,5 @@
 		pl:../deluser.conf.5.pl add_pl:translator_polish.add \
 		sv:../deluser.conf.5.sv add_sv:translator_swedish.add \
 		pt_BR:../deluser.conf.5.pt_BR add_pt_BR:translator_pt_BR.add \
-		ru:../deluser.conf.5.ru add_ru:translator_russian.add opt_ru:"-L KOI8-R" \
+		ru:../deluser.conf.5.ru add_ru:translator_russian.add \
 		it:../deluser.conf.5.it add_it:translator_italian.add
