diff --git a/scripts/Dpkg/Shlibs/SymbolFile.pm b/scripts/Dpkg/Shlibs/SymbolFile.pm
index d03ce2598..b36d542ce 100644
--- a/scripts/Dpkg/Shlibs/SymbolFile.pm
+++ b/scripts/Dpkg/Shlibs/SymbolFile.pm
@@ -204,7 +204,7 @@ sub parse {
     my ($self, $fh, $file, %opts) = @_;
     my $state = $opts{state} //= {};
 
-    if (defined $state) {
+    if (exists $state->{seen}) {
 	return if exists $state->{seen}{$file}; # Avoid include loops
     } else {
 	$self->{file} = $file;
