Index: deja-dup-18.1.1/widgets/ConfigLocation.c
===================================================================
--- deja-dup-18.1.1.orig/widgets/ConfigLocation.c	2011-06-17 10:35:27.858149183 -0400
+++ deja-dup-18.1.1/widgets/ConfigLocation.c	2011-06-17 10:36:07.258149600 -0400
@@ -538,7 +538,8 @@
 						}
 						gtk_tree_model_get ((GtkTreeModel*) self->priv->store, &iter, DEJA_DUP_CONFIG_LOCATION_COL_UUID, &uuid, -1);
 						if (g_strcmp0 (vol_uuid, uuid) == 0) {
-							result = FALSE;
+							self->priv->index_vol_saved = i;
+							result = TRUE;
 							_g_free0 (uuid);
 							_g_free0 (vol_uuid);
 							_g_object_unref0 (fsettings);
