--- akregator/src/librss/article.cpp.orig	2006-11-12 14:46:12.000000000 +0100
+++ akregator/src/librss/article.cpp	2006-11-12 14:46:33.000000000 +0100
@@ -139,7 +139,7 @@
         if (time != 0)
 		d->pubDate.setTime_t(time);
 
-	if (!(elemText = extractNode(node, QString::fromLatin1("wfw:comment"))).isNull()) {
+	if (!(elemText = extractNode(node, QString::fromLatin1("comment"))).isNull()) {
 		d->commentsLink = elemText;
 	}
 
