Index: debian/aide.conf.d/31_aide_munin-nodes
===================================================================
--- debian/aide.conf.d/31_aide_munin-nodes	(revision 793)
+++ debian/aide.conf.d/31_aide_munin-nodes	(working copy)
@@ -3,6 +3,9 @@
 # generate aide exclude patterns for all nodes listed in $MUNINCONF
 
 MUNINCONF=/etc/munin/munin.conf
+
+[ -e $MUNINCONF ] || exit 0
+
 HOSTS=$(grep '^\[[[:alnum:]:.]\+\]' $MUNINCONF | tr -d '[]')
 
 escape_dots()
