--- ./debian/acpi-support/usr/share/doc/acpi-support/examples/acpi/sleep.sh.orig	2010-12-14 10:15:51.000000000 +0100
+++ ./debian/acpi-support/usr/share/doc/acpi-support/examples/acpi/sleep.sh	2010-12-14 13:14:55.000000000 +0100
@@ -11,9 +11,5 @@ test -f /usr/share/acpi-support/key-cons
 
 . /usr/share/acpi-support/policy-funcs
 
-if [ `CheckPolicy` != 0 ]; then
   # No power management daemons are running. Divert to our own implementation.
-  /usr/share/acpi-support/suspendorhibernate suspend
-fi
-
-
+CheckPolicy || /usr/share/acpi-support/suspendorhibernate suspend
--- ./power.sh.orig	2010-12-14 11:37:17.000000000 +0100
+++ ./power.sh	2010-12-14 11:45:03.000000000 +0100
@@ -4,7 +4,7 @@ test -f /usr/share/acpi-support/key-cons
 
 . /usr/share/acpi-support/policy-funcs
 
-if [ -z "$*" ] && ( CheckPolicy || CheckUPowerPolicy ); then
+if [ -z "$*" ] && { CheckPolicy || CheckUPowerPolicy; }; then
     exit;
 fi
 
