Index: disk-detect.sh
===================================================================
--- disk-detect.sh	(revision 55893)
+++ disk-detect.sh	(working copy)
@@ -180,8 +180,10 @@
 
 		if [ "$(dmraid -c -s)" != "No RAID disks" ]; then
 			logger -t disk-detect "Serial ATA RAID disk(s) detected; enabling dmraid support"
-			# Activate devices
-			log-output -t disk-detect dmraid -ay
+			# Activate only those arrays which have all disks present.
+			for dev in $(dmraid -r -c); do
+				log-output -t disk-detect dmraid-activate $dev
+			done
 		else
 			logger -t disk-detect "No Serial ATA RAID disks detected"
 		fi
