diff --git a/devtools/blend-gen-control b/devtools/blend-gen-control
index 1aba552..cde3237 100755
--- a/devtools/blend-gen-control
+++ b/devtools/blend-gen-control
@@ -566,9 +566,14 @@ sub load_task {
         my $header;
         for $header (qw(Depends Recommends Suggests)) {
             if (m/^$header:\s+(.+)$/ && $1 !~ /^\s*$/) {
+		my $pkgs = $1;
+		while (<TASKFILE>) {
+		    last if (m/^\S+/ || m/^\s*$/);
+		    $pkgs .= $_;
+		}
                 $taskinfo{$curpkg}{$header} = ()
                     if (! exists $taskinfo{$curpkg}{$header});
-                my ($pkglist, $missinglist) = process_pkglist($1);
+                my ($pkglist, $missinglist) = process_pkglist($pkgs);
                 push(@{$taskinfo{$curpkg}{$header}}, @{$pkglist});
 
 		$haspackages += $#{$taskinfo{$curpkg}{$header}} + 1;
