diff -ruN old/cryptsetup-1.0.6/lib/libdevmapper.c new/cryptsetup-1.0.6/lib/libdevmapper.c
--- old/cryptsetup-1.0.6/lib/libdevmapper.c	2007-10-25 09:11:19.000000000 +0200
+++ new/cryptsetup-1.0.6/lib/libdevmapper.c	2009-02-10 12:18:43.000000000 +0100
@@ -22,7 +22,8 @@
 
 static void run_udevsettle(void)
 {
-	system(UDEVSETTLE);
+	if(!access(UDEVSETTLE, F_OK|X_OK))
+		system(UDEVSETTLE);
 }
 
 static void set_dm_error(int level, const char *file, int line,
