Index: courier-0.76.3/courier/configure.ac
===================================================================
--- courier-0.76.3.orig/courier/configure.ac
+++ courier-0.76.3/courier/configure.ac
@@ -112,6 +112,9 @@ AC_ARG_WITH(transport,
 
 AC_SUBST(transport)
 
+AC_ARG_WITH(piddir, [  --with-piddir         Directory where PID files are located ], piddir="$withval", piddir=/var/run)
+AC_SUBST(piddir)
+
 AC_MSG_CHECKING(for modules to compile)
 
 dirlist=""
