From 3df4cd09ec8231c468347a7478e9e4940139b97e Mon Sep 17 00:00:00 2001
From: Elimar Riesebieter <riesebie@lxtec.de>
Date: Tue, 12 Feb 2019 10:31:40 +0100
Subject: [PATCH] Fix initscript to run non-root pid file.

---
 debian/amavisd-new.amavis.init | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/amavisd-new.amavis.init b/debian/amavisd-new.amavis.init
index 7890ce7..c07c05c 100644
--- a/debian/amavisd-new.amavis.init
+++ b/debian/amavisd-new.amavis.init
@@ -42,7 +42,7 @@ test -f ${DAEMON} || exit 0
 set -e
 
 START="--start --quiet --pidfile $PIDFILE --startas ${DAEMON}"
-STOP="--stop --quiet --pidfile $PIDFILE"
+STOP="--stop --quiet --pidfile $PIDFILE --user amavis"
 PARAMS=
 
 check_noncompatible_upgrade() {
-- 
2.20.1

