=== modified file 'doc/apt.ent'
--- doc/apt.ent	2008-02-19 19:37:41 +0000
+++ doc/apt.ent	2008-10-04 10:22:55 +0000
@@ -259,6 +259,8 @@
       <term><option>--option</option></term>
      <listitem><para>Set a Configuration Option; This will set an arbitrary
       configuration option. The syntax is <option>-o Foo::Bar=bar</option>.
+      <option>-o</option> and <option>--option</option> can be used multiple
+      times to set different options.
      </para>
      </listitem>
      </varlistentry>

