diff -urN sourcenav-5.0.old/debian/changelog sourcenav-5.0/debian/changelog
--- sourcenav-5.0.old/debian/changelog	2003-03-15 02:11:26.000000000 -0800
+++ sourcenav-5.0/debian/changelog	2003-03-15 01:48:37.000000000 -0800
@@ -1,3 +1,11 @@
+sourcenav (5.0-4.1) unstable; urgency=low
+
+  * NMU
+  * Update Build-Depends from xlib6g-dev to xlibs-dev.  Closes: #170162.
+  * Fix parsers/fortran/fortran.y.  Closes: #166267.
+
+ -- Daniel Schepler <schepler@debian.org>  Sat, 15 Mar 2003 01:48:37 -0800
+
 sourcenav (5.0-4) unstable; urgency=low
 
   * Remove tclsh Provides: lines (closes: #134418)
diff -urN sourcenav-5.0.old/debian/control sourcenav-5.0/debian/control
--- sourcenav-5.0.old/debian/control	2003-03-15 02:11:26.000000000 -0800
+++ sourcenav-5.0/debian/control	2003-03-15 01:43:42.000000000 -0800
@@ -3,7 +3,7 @@
 Priority: optional
 Maintainer: Eray Ozkural <erayo@cs.bilkent.edu.tr>
 Standards-Version: 3.1.1
-Build-Depends: debhelper (>= 1.1.17), gettext, flex, bison, xlib6g-dev (>= 3.3.6-4)
+Build-Depends: debhelper (>= 1.1.17), gettext, flex, bison, xlibs-dev
 
 Package: sourcenav
 Architecture: any
diff -urN sourcenav-5.0.old/snavigator/parsers/fortran/fortran.y sourcenav-5.0/snavigator/parsers/fortran/fortran.y
--- sourcenav-5.0.old/snavigator/parsers/fortran/fortran.y	2000-04-19 17:38:40.000000000 -0700
+++ sourcenav-5.0/snavigator/parsers/fortran/fortran.y	2003-03-15 01:45:20.000000000 -0800
@@ -2011,6 +2011,7 @@
 			    }
 			    $$.next_token = append_token($1.next_token,&($3));
 			}
+		;
 
 /* 74 not present: type checking not done at this level */
 
