diff -urN aspell-fr-0.50-3.old/debian/changelog aspell-fr-0.50-3/debian/changelog
--- aspell-fr-0.50-3.old/debian/changelog	2005-02-11 14:11:12.000000000 -0800
+++ aspell-fr-0.50-3/debian/changelog	2005-02-11 14:12:11.000000000 -0800
@@ -1,3 +1,11 @@
+aspell-fr (0.50-3-2.1) unstable; urgency=low
+
+  * Non-maintainer upload
+  * debian/control: provide aspell6-dictionary, and build-depend on aspell
+    (>> 0.60).  This transitions the package to the new Aspell 0.60.
+
+ -- Brian Nelson <pyro@debian.org>  Fri, 11 Feb 2005 14:12:11 -0800
+
 aspell-fr (0.50-3-2) unstable; urgency=low
 
   * Removing the uploader (Jerome Marant <jerome@debian.org>)
diff -urN aspell-fr-0.50-3.old/debian/control aspell-fr-0.50-3/debian/control
--- aspell-fr-0.50-3.old/debian/control	2005-02-11 14:11:12.000000000 -0800
+++ aspell-fr-0.50-3/debian/control	2005-02-11 14:11:43.000000000 -0800
@@ -3,14 +3,14 @@
 Priority: optional
 Maintainer: Remi Vanicat <vanicat@debian.org>
 Standards-Version: 3.5.8
-Build-Depends: debhelper (>> 4.0.0), aspell-bin (>> 0.50.0)
+Build-Depends: debhelper (>> 4.0.0), aspell-bin (>> 0.60)
 
 Package: aspell-fr
 Architecture: any
 Suggests: aspell-bin
 Section: text
 Depends: libaspell15
-Provides: aspell-dictionary
+Provides: aspell6-dictionary
 Description: French dictionary for aspell
  This package contains all the required files to add support
  for French language to aspell spell checker.
