Index: at-3.2.4/atd.c
===================================================================
--- at-3.2.4.orig/atd.c
+++ at-3.2.4/atd.c
@@ -804,7 +804,7 @@ void atd_setalarm(time_t next)
 {
     timeout.it_value.tv_sec = next;
     timer_settime(timer, TIMER_ABSTIME, &timeout, NULL);
-    pause();
+    sleep(next - now);
 }
 #else
 void timer_setup()
@@ -818,7 +818,7 @@ time_t atd_gettime()
 
 void atd_setalarm(time_t next)
 {
-    sleep(next - atd_gettime());
+    sleep(next - now);
 }
 #endif
 /* Global functions */
@@ -953,7 +953,7 @@ main(int argc, char *argv[])
     daemon_setup();
 
     do {
-	now = time(NULL);
+	now = atd_gettime();
 	next_invocation = run_loop();
 	if ((next_invocation > now) && (!hupped)) {
     	    atd_setalarm(next_invocation);
