--- /lib/cryptsetup/cryptdisks.functions
+++ /lib/cryptsetup/cryptdisks.functions
@@ -750,7 +750,7 @@
 		# will just be a no-op, but we don't want to defer to the
 		# other job entirely because this is the fallback for fixing
 		# up any ordering-dependent decrypting.
-		invoke-rc.d --quiet cryptdisks-udev status && while status cryptdisks-udev DEVNAME="$dev_match" 2>&1 | grep -q 'start'; do
+		[ -x /sbin/status ] && while /sbin/status cryptdisks-udev DEVNAME="$dev_match" 2>&1 | grep -q "\<start"; do
 			sleep 1
 		done
 		handle_crypttab_line_start "$dst" "$src" "$key" "$opts" <&3 || log_action_end_msg $?
