diff -rupN djvulibre-3.5.22/debian/control djvulibre-3.5.22.new/debian/control
--- djvulibre-3.5.22/debian/control	2010-08-14 16:29:04.000000000 +0200
+++ djvulibre-3.5.22.new/debian/control	2010-08-14 16:28:48.594379503 +0200
@@ -2,7 +2,15 @@ Source: djvulibre
 Section: web
 Priority: optional
 Maintainer: Barak A. Pearlmutter <bap@debian.org>
-Build-Depends: debhelper (>= 7), libx11-dev, libxext-dev, libxt-dev, libqt3-mt-dev (>= 3:3.3.4-4), libjpeg-dev, libtiff4-dev | libtiff3g-dev | libtiff-dev, imagemagick
+Build-Depends: autotools-dev (>= 20100122.1~),
+               debhelper (>= 7),
+               libx11-dev,
+               libxext-dev,
+               libxt-dev,
+               libqt3-mt-dev (>= 3:3.3.4-4),
+               libjpeg-dev,
+               libtiff4-dev | libtiff3g-dev | libtiff-dev,
+               imagemagick
 Vcs-Cvs: :ext:@djvu.cvs.sourceforge.net:/cvsroot/djvu
 VCS-Browser: http://djvu.cvs.sourceforge.net/djvu/djvulibre-3.5/
 Homepage: http://djvu.sourceforge.net/
@@ -95,8 +103,8 @@ Package: djvulibre-plugin
 Section: web
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, djview4 | djview3
-Enhances: mozilla-browser, mozilla, mozilla-firefox, iceweasel, iceape-browser, konqueror, galeon
-Recommends: mozilla-browser | mozilla | mozilla-firefox | iceweasel | iceape-browser | konqueror | galeon
+Enhances: mozilla-browser, mozilla, firefox, iceweasel, iceape-browser, konqueror, galeon
+Recommends: mozilla-browser | mozilla | firefox | iceweasel | iceape-browser | konqueror | galeon
 Suggests: mime-support
 Conflicts: djvulibre-bin (<< 3.5.19)
 Description: Browser plugin for the DjVu image format
diff -rupN djvulibre-3.5.22/debian/rules djvulibre-3.5.22.new/debian/rules
--- djvulibre-3.5.22/debian/rules	2010-08-14 16:29:04.000000000 +0200
+++ djvulibre-3.5.22.new/debian/rules	2010-08-14 16:28:37.805628883 +0200
@@ -47,6 +47,7 @@ testlibver:
 
 config.status: configure
 	dh_testdir
+	dh_autotools-dev_updateconfig
 	dh_auto_configure -- --disable-desktopfiles --enable-static --enable-shared
 
 build: build-stamp
@@ -63,12 +64,8 @@ endif
 	touch build-stamp
 
 clean:
+	dh_autotools-dev_restoreconfig
 	dh $@
-	@echo push in latest config support files
-	-test -r /usr/share/misc/config.sub && \
-	  cp -f /usr/share/misc/config.sub config/config.sub
-	-test -r /usr/share/misc/config.guess && \
-	  cp -f /usr/share/misc/config.guess config/config.guess
 
 plugsubdir=netscape/plugins-libc6
 plug=$(plugsubdir)/nsdejavu.so
