diff --git a/config/example-config.php b/config/example-config.php
index 22fb8e9..26bde5d 100644
--- a/config/example-config.php
+++ b/config/example-config.php
@@ -206,9 +206,9 @@ $c->schedule_private_key = 'PRIVATE-KEY-BASE-64-DATA';
 * The login mecanism is made in 2 places:
 *  - for the web interface in: index.php that calls DAViCalSession.php that extends
 *    Session.php (from AWL libraries)
-*  - for the caldav client in: caldav.php that calls BasicAuthSession.php
-* Both Session.php and BasicAuthSession.php check against the
-* authenticate_hook['call'], although for BasicAuthSession.php this will be for
+*  - for the caldav client in: caldav.php that calls HTTPAuthSession.php
+* Both Session.php and HTTPAuthSession.php check against the
+* authenticate_hook['call'], although for HTTPAuthSession.php this will be for
 * each page.  For Session.php this will only occur during login.
 *
 * $c->authenticate_hook['config'] should be set up with any configuration data
