diff --git a/src/m3u/.m3u.c.swp b/src/m3u/.m3u.c.swp
index 52fb8b1..ad60cf2 100644
Binary files a/src/m3u/.m3u.c.swp and b/src/m3u/.m3u.c.swp differ
diff --git a/src/m3u/m3u.c b/src/m3u/m3u.c
index 210f49e..729c024 100644
--- a/src/m3u/m3u.c
+++ b/src/m3u/m3u.c
@@ -106,6 +106,8 @@ static void playlist_load_m3u (const gchar * path, gint at)
 
 NEXT:
         parse = next;
+        if(parse == NULL) 
+            break;
     }
 
     aud_playlist_entry_insert_batch (aud_playlist_get_active (), at, add, NULL);
