--- dvgrab-1.8/debian/control.orig	2007-01-14 12:23:40.000000000 +0100
+++ dvgrab-1.8/debian/control	2007-01-14 12:23:44.000000000 +0100
@@ -6,7 +6,7 @@
 Standards-Version: 3.7.2
 
 Package: dvgrab
-Architecture: alpha amd64 arm hppa i386 ia64 mips mipsel powerpc ppc64 s390 sh sparc
+Architecture: alpha amd64 arm armeb armel hppa i386 ia64 mips mipsel powerpc ppc64 s390 sh sparc
 Depends: ${shlibs:Depends}
 Description: grab digital video data via IEEE1394 links
  dvgrab receives audio and video data from a digital camcorder via an
