diff -u cron-3.0pl1/debian/cron.pam cron-3.0pl1/debian/cron.pam
--- cron-3.0pl1/debian/cron.pam
+++ cron-3.0pl1/debian/cron.pam
@@ -12,7 +12,9 @@
-@include common-account
-@include common-session-noninteractive 
-
 # Sets up user limits, please define limits for cron tasks
 # through /etc/security/limits.conf
 session    required   pam_limits.so
 
+# Sets the loginuid process attribute
+session    required     pam_loginuid.so
+
+@include common-account
+@include common-session-noninteractive 
