--- a/cgi/pkgindex.cgi
+++ b/cgi/pkgindex.cgi
@@ -10,6 +10,7 @@ use Debbugs::CGI qw(:util :url :html);
 use Debbugs::Common qw(getmaintainers);
 use Debbugs::Bugs qw(count_bugs);
 use Debbugs::Status qw(:status);
+use Debbugs::Packages qw(getpkgsrc);
 
 nice(5);
 
@@ -103,7 +104,7 @@ if ($indexon eq "pkg") {
            } 
        } keys %count;
   }
-  %count = countbugs(function => sub {my %d=@_;
+  %count = count_bugs(function => sub {my %d=@_;
                           return map {
                             $pkgsrc->{$_} || $_
                           } splitpackages($d{"pkg"});
