diff -ur e2fsprogs-1.41.14-orig/misc/filefrag.c e2fsprogs-1.41.14/misc/filefrag.c
--- e2fsprogs-1.41.14-orig/misc/filefrag.c	2010-12-22 18:49:20.000000000 +0300
+++ e2fsprogs-1.41.14/misc/filefrag.c	2011-06-23 20:00:27.408000000 +0400
@@ -183,9 +183,6 @@
 
 	memset(fiemap, 0, sizeof(struct fiemap));
 
-	if (!verbose)
-		count = 0;
-
 	if (sync_file)
 		flags |= FIEMAP_FLAG_SYNC;
 
@@ -213,11 +210,6 @@
 			fiemap_header_printed = 1;
 		}
 
-		if (!verbose) {
-			*num_extents = fiemap->fm_mapped_extents;
-			goto out;
-		}
-
 		/* If 0 extents are returned, then more ioctls are not needed */
 		if (fiemap->fm_mapped_extents == 0)
 			break;
@@ -234,7 +226,9 @@
 				tot_extents++;
 			else
 				last_blk = 0;
-			print_extent_info(&fm_ext[i], n, last_blk, blk_shift);
+
+			if (verbose)
+				print_extent_info(&fm_ext[i], n, last_blk, blk_shift);
 
 			last_blk = phy_blk + ext_len - 1;
 			if (fm_ext[i].fe_flags & FIEMAP_EXTENT_LAST)
