diff -u blueman-1.21/debian/rules blueman-1.21/debian/rules
--- blueman-1.21/debian/rules
+++ blueman-1.21/debian/rules
@@ -19,4 +19,4 @@
 install/blueman::
-	rm -f $(CURDIR)/debian/blueman/usr/lib/python2.5/site-packages/_blueman.la
-	rm -f $(CURDIR)/debian/blueman/usr/lib/python2.5/site-packages/_blueman.a
+	rm -f $(CURDIR)/debian/blueman/usr/lib/python*/*-packages/_blueman.la
+	rm -f $(CURDIR)/debian/blueman/usr/lib/python*/*-packages/_blueman.a
 
diff -u blueman-1.21/debian/control blueman-1.21/debian/control
--- blueman-1.21/debian/control
+++ blueman-1.21/debian/control
@@ -7,7 +7,7 @@
 Build-Depends: debhelper (>= 7), autotools-dev, cdbs, libglib2.0-dev, libtool,
                libgtk2.0-dev (>= 2.14), libstartup-notification0-dev,
                python-gobject-dev, libbluetooth-dev (>= 4.0), intltool,
-               python2.5-dev | python2.6-dev, python-pyrex, python-gtk2-dev,
+               python (>= 2.5), python-pyrex, python-gtk2-dev,
                python-notify, python-dbus, python-central
 
 Package: blueman
@@ -18,7 +18,7 @@
          librsvg2-common, hicolor-icon-theme
 Recommends: python-gconf, policykit-1, libpulse-mainloop-glib0
 Homepage: http://blueman-project.org/
-XB-Python-Version: >= 2.5
+XB-Python-Version: ${python:Versions}
 Description: A Graphical bluetooth manager
  Blueman is a GTK+ bluetooth management utility for GNOME using bluez D-Bus
  backend.
