=== modified file 'control'
--- control	2008-08-02 23:46:24 +0000
+++ control	2008-08-02 23:47:42 +0000
@@ -15,7 +15,7 @@
 Depends: ${shlibs:Depends}
 Suggests: libasound2-plugins (>= 1.0.9)
 Conflicts: libasound2-plugins (<< 1.0.9)
-Description: ALSA library
+Description: user-space library for ALSA application programming
  This package contains the ALSA library and its standard
  plugins.
  .
@@ -29,7 +29,7 @@
 Suggests: libasound2-doc
 Conflicts: libasound-dev, alsa-headers
 Replaces: libasound2 (<< 0.9.0rc7-2), alsa-headers
-Description: ALSA library development files
+Description: user-space library development files for ALSA application programming
  This package contains files required for developing software
  that makes use of libasound2, the ALSA library.
  .
@@ -41,7 +41,7 @@
 Depends: libasound2 (= ${binary:Version}), ${shlibs:Depends}, ${bilib:depends}
 Suggests: libasound2-plugins (>= 1.0.9)
 Conflicts: libasound2-plugins (<< 1.0.9)
-Description: ALSA library (64 bit)
+Description: user-space library (64-bit) for ALSA application programming on multi-arch systems
  This package contains the ALSA library (64 bit) and its standard
  plugins.
  .
@@ -52,7 +52,7 @@
 Section: libdevel
 Depends: lib64asound2 (= ${binary:Version}), libasound2-dev (= ${binary:Version}), ${bidev:depends}
 Suggests: libasound2-doc
-Description: ALSA library development files (64 bit)
+Description: user-space library development files (64-bit) for ALSA application programming on multi-arch systems
  This package contains files required for developing software
  that makes use of lib64asound2, the ALSA library (64 bit).
  .
@@ -65,7 +65,7 @@
 Suggests: libasound2-plugins (>= 1.0.9)
 Conflicts: libasound2-plugins (<< 1.0.9), ia32-libs (<< 1.9)
 Replaces: ia32-libs (<< 1.9)
-Description: ALSA library (32 bit)
+Description: user-space library (32-bit) for ALSA application programming on multi-arch systems
  This package contains the ALSA library (32 bit) and its standard
  plugins.
  .
@@ -76,7 +76,7 @@
 Section: libdevel
 Depends: lib32asound2 (= ${binary:Version}), libasound2-dev (= ${binary:Version}), ${bidev:depends}
 Suggests: libasound2-doc
-Description: ALSA library development files (32 bit)
+Description: user-space library development files (32-bit) for ALSA application programming on multi-arch systems
  This package contains files required for developing software
  that makes use of lib32asound2, the ALSA library (32 bit).
  .
@@ -86,7 +86,7 @@
 Architecture: all
 Section: doc
 Suggests: libasound2-dev
-Description: ALSA library developer documentation
+Description: developer documentation for user-space ALSA application programming
  This package contains documentation for the ALSA library.
  .
  ALSA is the Advanced Linux Sound Architecture.

