Index: acpid-1.0.6/debian/acpid.init.d
===================================================================
--- acpid-1.0.6.orig/debian/acpid.init.d	2007-11-02 20:45:02.000000000 +0000
+++ acpid-1.0.6/debian/acpid.init.d	2007-11-02 20:47:59.000000000 +0000
@@ -36,6 +36,9 @@
 # As the name says. If the kernel supports modules, it'll try to load
 # the ones listed in "MODULES".
 load_modules() {
+        PRINTK=`cat /proc/sys/kernel/printk`
+        [ "$VERBOSE" = no ] && echo "0 0 0 0" > /proc/sys/kernel/printk
+
         LIST=`/sbin/lsmod|awk '!/Module/ {print $1}'`
 
 	# Get list of available modules
@@ -70,6 +73,7 @@
                 done
                 log_end_msg $STATUS
         fi
+        echo "$PRINTK" > /proc/sys/kernel/printk
 }
 
 case "$1" in
