diff -Nru clinfo-0.0.20130513/debian/changelog clinfo-0.0.20130513/debian/changelog
--- clinfo-0.0.20130513/debian/changelog	2013-05-13 19:53:04.000000000 +0200
+++ clinfo-0.0.20130513/debian/changelog	2014-09-15 18:03:02.000000000 +0200
@@ -1,3 +1,13 @@
+clinfo (0.0.20130513-1.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+
+  [ Andreas Boll ]
+  * Drop unused CL_DEVICE_BUILT_IN_KERNELS query, fixes clinfo output on
+    OpenCL 1.1 drivers (Closes: #721103).
+
+ -- Julien Cristau <jcristau@debian.org>  Mon, 15 Sep 2014 18:02:53 +0200
+
 clinfo (0.0.20130513-1) unstable; urgency=low
 
   * New upstream release
diff -Nru clinfo-0.0.20130513/debian/patches/fix-opencl-1.1.diff clinfo-0.0.20130513/debian/patches/fix-opencl-1.1.diff
--- clinfo-0.0.20130513/debian/patches/fix-opencl-1.1.diff	1970-01-01 01:00:00.000000000 +0100
+++ clinfo-0.0.20130513/debian/patches/fix-opencl-1.1.diff	2014-09-15 18:02:35.000000000 +0200
@@ -0,0 +1,12 @@
+--- a/device.cpp
++++ b/device.cpp
+@@ -49,7 +49,8 @@
+ {
+         query(address_bits, CL_DEVICE_ADDRESS_BITS);
+         query(available, CL_DEVICE_AVAILABLE);
+-        query(built_in_kernels, CL_DEVICE_BUILT_IN_KERNELS);
++        // FIXME check for OpenCL 1.2
++        // query(built_in_kernels, CL_DEVICE_BUILT_IN_KERNELS);
+         query(compiler_available, CL_DEVICE_COMPILER_AVAILABLE);
+         try
+         {
diff -Nru clinfo-0.0.20130513/debian/patches/series clinfo-0.0.20130513/debian/patches/series
--- clinfo-0.0.20130513/debian/patches/series	1970-01-01 01:00:00.000000000 +0100
+++ clinfo-0.0.20130513/debian/patches/series	2014-09-15 18:02:35.000000000 +0200
@@ -0,0 +1 @@
+fix-opencl-1.1.diff
