--- a/device.cpp
+++ b/device.cpp
@@ -49,7 +49,8 @@
 {
         query(address_bits, CL_DEVICE_ADDRESS_BITS);
         query(available, CL_DEVICE_AVAILABLE);
-        query(built_in_kernels, CL_DEVICE_BUILT_IN_KERNELS);
+        // FIXME check for OpenCL 1.2
+        // query(built_in_kernels, CL_DEVICE_BUILT_IN_KERNELS);
         query(compiler_available, CL_DEVICE_COMPILER_AVAILABLE);
         try
         {
