From 903a802b449d840cbf43eb3ffd3147e433bfaa1b Mon Sep 17 00:00:00 2001
From: Niels Thykier <niels@thykier.net>
Date: Sat, 20 Jan 2018 09:38:35 +0000
Subject: [PATCH] dh: Always run commands with sequence defined options

Signed-off-by: Niels Thykier <niels@thykier.net>
---
 dh | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/dh b/dh
index 47160317..787462aa 100755
--- a/dh
+++ b/dh
@@ -998,6 +998,9 @@ sub can_skip {
 	return 0 if $user_specified_options ||
 		(exists $ENV{DH_OPTIONS} && length $ENV{DH_OPTIONS});
 
+	return 0 if exists($command_opts{$command})
+		and @{$command_opts{$command}};
+
 	if (! defined $skipinfo{$command}) {
 		$skipinfo{$command}=[extract_skipinfo($command)];
 	}
-- 
2.15.1

