--- ecl-0.9j-20080306.orig/debian/rules	2008-04-28 18:05:40.000000000 +0200
+++ ecl-0.9j-20080306/debian/rules	2008-04-28 20:16:17.000000000 +0200
@@ -50,18 +50,27 @@
 	   --enable-boehm=system \
 	   --with-x 
 	   #--enable-boehm=system
-build: build-stamp
-build-stamp:  config.status
-	dh_testdir
+
+build: build-arch build-indep
+
+build-arch: build-arch-stamp
+build-arch-stamp:  config.status
+	dh_testdir -a
 
 	$(MAKE)
 
-	touch build-stamp
+	touch build-arch-stamp
+
+build-indep: build-indep-stamp
+build-indep-stamp: config.status
+	dh_testdir -i
+	$(MAKE) -C build/doc
+	touch build-indep-stamp
 
 clean:
 	dh_testdir
 	dh_testroot
-	rm -f build-stamp 
+	rm -f build-*-stamp 
 	-test -r /usr/share/misc/config.sub && \
            cp -f /usr/share/misc/config.sub src/gc/config.sub
 	-test -r /usr/share/misc/config.guess && \
@@ -79,11 +88,11 @@
 	
 	dh_clean
 
-install: build
-	dh_testdir
-	dh_testroot
-	dh_clean -k
-	dh_installdirs
+install: build-arch
+	dh_testdir -a
+	dh_testroot -a
+	dh_clean -k -a
+	dh_installdirs -a
 
 	# Add here commands to install the package into debian/ecl
 	$(MAKE) install prefix=$(CURDIR)/debian/ecl/usr 
@@ -108,34 +117,44 @@
 
 
 
-# Build architecture-dependent files here.
-binary-arch: build install
-# We have nothing to do by default.
-
 # Build architecture-independent files here.
-binary-indep: build install
-	dh_testdir
-	dh_testroot
+binary-indep: build-indep
+	dh_testdir -i
+	dh_testroot -i
+	dh_installdirs -i
+	dh_installdocs -i
+	dh_installchangelogs -i
+	dh_installinfo -v -i
+	dh_compress -i
+	dh_fixperms -i
+	dh_installdeb -i
+	dh_gencontrol -i
+	dh_md5sums -i
+	dh_builddeb -i
+
+# Build architecture-dependent files here.
+binary-arch: build-arch install
+	dh_testdir -a
+	dh_testroot -a
 #	dh_installdebconf	
 	ls -l $(CURDIR)/debian/ecl-doc/usr/share/info/	
-	dh_installdocs
-	dh_installexamples
-	dh_installmenu
-	dh_installman
+	dh_installdocs -a 
+	dh_installexamples -a
+	dh_installmenu -a
+	dh_installman -a
 #	dh_undocumented
-	dh_installchangelogs 
-	dh_link
-	dh_strip
-	dh_installinfo -v
-	dh_compress
-	dh_fixperms
-	dh_lisp ecl
-	dh_makeshlibs
-	dh_installdeb
-	dh_shlibdeps
-	dh_gencontrol
-	dh_md5sums
-	dh_builddeb
+	dh_installchangelogs -a
+	dh_link -a
+	dh_strip -a
+	dh_compress -a
+	dh_fixperms -a 
+	dh_lisp -a ecl
+	dh_makeshlibs -a
+	dh_installdeb -a
+	dh_shlibdeps -a
+	dh_gencontrol -a
+	dh_md5sums -a
+	dh_builddeb -a
 
 binary: binary-indep binary-arch
-.PHONY: build clean binary-indep binary-arch binary install 
+.PHONY: build build-arch build-indep clean binary-indep binary-arch binary install 
