Index: scripts/debian/sid
===================================================================
--- scripts/debian/sid	(revision 59820)
+++ scripts/debian/sid	(working copy)
@@ -171,9 +171,6 @@
     setup_available $required $base
     done_predeps=
     while predep=$(get_next_predep); do
-      # We have to resolve dependencies of pre-dependencies manually because
-      # dpkg --predep-package doesn't handle this.
-      predep=$(without "$(without "$(resolve_deps $predep)" "$required")" "$done_predeps")
       # XXX: progress is tricky due to how dpkg_progress works
       # -- cjwatson 2009-07-29
       p; smallyes '' | in_target dpkg --force-overwrite --force-confold --skip-same-version --install $(debfor $predep)
