diff -ruN easytag-1.99.10.orig/src/et_core.h easytag-1.99.10/src/et_core.h
--- easytag-1.99.10.orig/src/et_core.h	2005-09-08 00:04:05.000000000 +0200
+++ easytag-1.99.10/src/et_core.h	2005-11-29 23:11:09.986322520 +0100
@@ -358,6 +358,7 @@
 ET_File   *ET_File_Item_New       (void);
 File_Name *ET_File_Name_Item_New  (void);
 File_Tag  *ET_File_Tag_Item_New   (void);
+gboolean   ET_Free_File_Tag_Item  (File_Tag *FileTag);
 gboolean   ET_Free_File_List_Item (ET_File *ETFile);
 
 gboolean ET_Copy_File_Tag_Item         (ET_File *ETFile, File_Tag *FileTag);
diff -ruN easytag-1.99.10.orig/src/flac_tag.c easytag-1.99.10/src/flac_tag.c
--- easytag-1.99.10.orig/src/flac_tag.c	2005-11-28 22:49:17.000000000 +0100
+++ easytag-1.99.10/src/flac_tag.c	2005-11-29 23:11:35.567433600 +0100
@@ -39,6 +39,7 @@
 #include "id3_tag.h"
 #include "misc.h"
 #include "setting.h"
+#include "picture.h"
 #include "charset.h"
 
 
