--- /sbin/cryptdisks_start	2010-11-04 21:58:23.000000000 +0100
+++ /tmp/cryptdisks_start	2011-02-01 15:10:55.239624458 +0100
@@ -30,7 +30,7 @@
 mount_fs
 
 count=0
-tablen="$(egrep -v "^[[:space:]]*(#|$)" "$TABFILE" | /usr/bin/wc -l)"
+tablen="$(egrep -vc "^[[:space:]]*(#|$)" "$TABFILE")"
 egrep -v "^[[:space:]]*(#|$)" "$TABFILE" | while read dst src key opts; do
 	count=$(( $count + 1 ))
 	if [ "$1" = "$dst" ]; then
--- /sbin/cryptdisks_stop	2010-11-04 21:58:23.000000000 +0100
+++ /tmp/cryptdisks_stop	2011-02-01 15:11:10.345561093 +0100
@@ -30,7 +30,7 @@
 log_action_begin_msg "Stopping crypto disk"
 
 count=0
-tablen="$(egrep -v "^[[:space:]]*(#|$)" "$TABFILE" | /usr/bin/wc -l)"
+tablen="$(egrep -vc "^[[:space:]]*(#|$)" "$TABFILE")"
 egrep -v "^[[:space:]]*(#|$)" "$TABFILE" | while read dst src key opts; do
 	count=$(( $count + 1 ))
 	if [ "$1" = "$dst" ]; then
