diff -pruN acpid-1.0.6/debian/acpid.init.d acpid-1.0.6-ubuntu/debian/acpid.init.d
--- acpid-1.0.6/debian/acpid.init.d	2008-08-16 15:21:46.000000000 +0200
+++ acpid-1.0.6-ubuntu/debian/acpid.init.d	2008-08-16 15:16:43.000000000 +0200
@@ -6,7 +6,7 @@
 # X-Start-Before:    kdm gdm xdm hal
 # X-Stop-After:      kdm gdm xdm hal
 # Default-Start:     2 3 4 5
-# Default-Stop:      0 1 6
+# Default-Stop:      1
 # Short-Description: Start the Advanced Configuration and Power Interface daemon
 # Description:       Provide a socket for X11, hald and others to multiplex
 #                    kernel ACPI events.
diff -pruN acpid-1.0.6/debian/rules acpid-1.0.6-ubuntu/debian/rules
--- acpid-1.0.6/debian/rules	2008-08-16 15:21:46.000000000 +0200
+++ acpid-1.0.6-ubuntu/debian/rules	2008-08-16 15:16:43.000000000 +0200
@@ -36,7 +38,7 @@ binary-arch: build install
 	dh_testroot
 	dh_installchangelogs Changelog
 	dh_installdocs
-	dh_installinit -- defaults 12 88
+	dh_installinit -- start 10 2 3 4 5 . stop 21 1 .
 	dh_install
 	dh_installman
 	dh_link
