--- tr_q-latin5.kmap_old	2003-07-16 01:21:30.000000000 +0200
+++ tr_q-latin5.kmap	2003-07-16 01:21:30.000000000 +0200
@@ -1,12 +1,12 @@
 # turkish "Q" keyboard, copyed from LyX turkish keyboard description
 #
 # Pablo Saratxaga <srtxg@f2219.n293.z2.fidonet.org>
+# Sligthly modified by Recai Oktas <roktas@omu.edu.tr>
 
 charset "iso-8859-9"
 keymaps 0-2,4-6,8-9,12
 include "linux-with-alt-and-altgr"
 strings as usual
-include "euro"
 
 keycode   1 = Escape           Escape
 	alt     keycode   1 = Meta_Escape
@@ -55,6 +55,7 @@
 keycode  15 = Tab              Tab
 	alt     keycode  15 = Meta_Tab
 keycode  16 = q
+	altgr keycode 16 =  at
 keycode  17 = w
 keycode  18 = e
 keycode  19 = r
@@ -89,11 +90,13 @@
 keycode  40 = +i               +Idotabove
 	control keycode  40 = Control_g
 	alt     keycode  40 = Meta_apostrophe
-keycode  41 = backslash        quotedbl		asciitilde
+#keycode  41 = backslash        quotedbl		asciitilde
+keycode 41 =  quotedbl  eacute
 	control keycode  41 = nul
 	alt     keycode  41 = Meta_grave
 keycode  42 = Shift
-keycode  43 = comma            semicolon        bar
+#keycode  43 = comma            semicolon        bar
+keycode 43 =  comma semicolon grave
 	control keycode  43 = Control_backslash
 	alt     keycode  43 = Meta_backslash
 	alt shift keycode 43 = Meta_bar
