From 7540e093bf5e007cf1aca92659963ec67de8af61 Mon Sep 17 00:00:00 2001
From: Osamu Aoki <osamu@debian.org>
Date: Sat, 5 Sep 2015 01:39:52 +0900
Subject: [PATCH 1/2] print "adding suffix" if so

---
 scripts/mk-origtargz.pl | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/scripts/mk-origtargz.pl b/scripts/mk-origtargz.pl
index ef2c75e..dde2019 100755
--- a/scripts/mk-origtargz.pl
+++ b/scripts/mk-origtargz.pl
@@ -531,7 +531,9 @@ if ($same_name) {
     }
 }
 
-if ($deletecount) {
+if ($deletecount and $suffix) {
+    print ", deleting ${deletecount} files from it and adding suffix";
+} elsif ($deletecount) {
     print ", deleting ${deletecount} files from it";
 }
 if ($zipfile_deleted) {
-- 
2.1.4

