From 09d41dc3674ff428a36caa48b2c1f0ad9d1889f9 Mon Sep 17 00:00:00 2001
From: Osamu Aoki <osamu@debian.org>
Date: Wed, 26 Aug 2015 09:31:40 +0000
Subject: [PATCH 2/2] repacksuffix needs to be taken care

---
 scripts/uscan.pl | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/scripts/uscan.pl b/scripts/uscan.pl
index 33f3ad4..ef21e2c 100755
--- a/scripts/uscan.pl
+++ b/scripts/uscan.pl
@@ -1522,6 +1522,11 @@ EOF
 	$target = basename($path);
     }
 
+    if (defined $options{'repacksuffix'}) {
+	# assume it was repacked (if wasn't, remove opts=repacksuffix in watchfile)
+	$newversion = $newversion . $options{'repacksuffix'}
+    }
+
     if ($dehs) {
 	my $msg = "Successfully downloaded updated package $newfile_base\n";
 	if (defined $mk_origtargz_out) {
-- 
2.1.4

