From 2f42d5adb778c8e7e2e16ccae3b198d445008598 Mon Sep 17 00:00:00 2001
From: Osamu Aoki <osamu@debian.org>
Date: Sat, 5 Sep 2015 02:11:56 +0900
Subject: [PATCH 2/2] change by adding suffix

---
 scripts/uscan.pl | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/scripts/uscan.pl b/scripts/uscan.pl
index 33f3ad4..3f8aa25 100755
--- a/scripts/uscan.pl
+++ b/scripts/uscan.pl
@@ -1517,7 +1517,9 @@ EOF
 	      to_string => \$mk_origtargz_out,
 	      wait_child => 1);
 	chomp($mk_origtargz_out);
-	$path = $1 if $mk_origtargz_out =~ /Successfully .* (?:to|as) ([^,]+)\.$/;
+	$path = $1 if $mk_origtargz_out =~ /Successfully .* (?:to|as) ([^,]+)(?:,.*)?\.$/;
+	$newversion = $newversion . $options{'repacksuffix'}
+		if $mk_origtargz_out =~ /Successfully .* adding suffix\.$/;
 	$path = $1 if $mk_origtargz_out =~ /Leaving (.*) where it is/;
 	$target = basename($path);
     }
-- 
2.1.4

