diff -Nru devscripts-2.10.26ubuntu13/scripts/uscan.pl devscripts-2.10.26ubuntu14/scripts/uscan.pl
--- devscripts-2.10.26ubuntu13/scripts/uscan.pl	2008-04-13 17:47:02.000000000 +0100
+++ devscripts-2.10.26ubuntu14/scripts/uscan.pl	2008-10-11 18:00:09.000000000 +0100
@@ -1062,19 +1062,21 @@
 
     # We use dpkg's rules to determine whether our current version
     # is newer or older than the remote version.
-    if (system("dpkg --compare-versions '$mangled_lastversion' gt '$newversion'") == 0) {
-        if ($verbose) {
-	    print " => remote site does not even have current version\n";
-	} elsif ($dehs) {
-	    $dehs_tags{'status'} = "Debian version newer than remote site";
-	} else {
-	    print "$pkg: remote site does not even have current version\n";
-	}
-        return 0;
-    } else {
-	# There's a newer upstream version available, which may already
-	# be on our system or may not be
-	$found++;
+    if (! defined $download_version){
+        if (system("dpkg --compare-versions '$mangled_lastversion' gt '$newversion'") == 0) {
+	    if ($verbose) {
+		print " => remote site does not even have current version\n";
+	    } elsif ($dehs) {
+		$dehs_tags{'status'} = "Debian version newer than remote site";
+	    } else {
+		print "$pkg: remote site does not even have current version\n";
+	    }
+	    return 0;
+        } else {
+	    # There's a newer upstream version available, which may already
+	    # be on our system or may not be
+	    $found++;
+        }
     }
 
     if (defined $pkg_dir) {
