diff -ru catfish-0.3.2.orig/catfish.py catfish-0.3.2/catfish.py
--- catfish-0.3.2.orig/catfish.py	2010-07-20 23:14:37.263707525 +0100
+++ catfish-0.3.2/catfish.py	2010-07-20 23:16:52.632678804 +0100
@@ -690,7 +690,9 @@
                         messages.append([_('No files were found.'), None])
                     status = _('No files found for "%s".') % keywords
                 else:
-                    status = _('%s files found for "%s".') % (len(listmodel), keywords)
+                    status = gettext.ngettext('%(n)s file found for "%(q)s".', \
+                               '%(n)s files found for "%(q)s".',len(listmodel)) \
+                               % {"n":len(listmodel), "q":keywords}
             for message, action in messages:
                 icon = [None, self.get_icon_pixbuf(status_icon)][message == messages[0][0]]
                 listmodel.append([icon, message, -1, None, action])
diff -ru catfish-0.3.2.orig/po/messages.pot catfish-0.3.2/po/messages.pot
--- catfish-0.3.2.orig/po/messages.pot	2010-07-20 23:14:37.268678479 +0100
+++ catfish-0.3.2/po/messages.pot	2010-07-20 23:16:23.063753273 +0100
@@ -86,8 +86,10 @@
 msgid "No files were found."
 msgstr ""
 
-msgid "%s files found for \"%s\"."
-msgstr ""
+msgid "%(n)s file found for \"%(q)s\"."
+msgid_plural "%(n)s files found for \"%(q)s\"."
+msgstr[0] ""
+msgstr[1] ""
 
 msgid "Fatal error, search was aborted."
 msgstr ""
