--- emacs24-24.4+1/debian/control~	2015-03-07 19:33:35.000000000 +0000
+++ emacs24-24.4+1/debian/control	2015-06-29 05:19:58.505117088 +0000
@@ -98,7 +98,7 @@
 Package: emacs24-common
 Architecture: all
 Depends: emacsen-common (>= 2.0.8), dpkg (>= 1.15.4) | install-info, ${shlibs:Depends}, ${misc:Depends}
-Suggests: emacs24-el, emacs24-common-non-dfsg
+Suggests: emacs24-el, emacs24-common-non-dfsg, ncurses-term
 Conflicts: emacs24-el (<< ${Source-Version}), cedet, eieio, speedbar, gnus-bonus-el
 Breaks: apel (<< 10.8+0.20120427-4)
 Description: GNU Emacs editor's shared, architecture independent infrastructure
