commit 3c8d91146b9acfa5ab42e4c2496038185b86f95d
Author: Antonio Russo <aerusso@aerusso.net>
Date:   Mon Dec 26 12:31:48 2022 -0700

    Do not override disabled tcp listener

diff --git a/src/acfg.cc b/src/acfg.cc
index a137ac2..2a64348 100644
--- a/src/acfg.cc
+++ b/src/acfg.cc
@@ -702,9 +702,6 @@ void PostProcConfig()
 {
 	remotedb::GetInstance().PostConfig();
 
-	if(!port) // heh?
-		port=ACNG_DEF_PORT;
-
 	if(connectPermPattern == "~~~")
 	   connectPermPattern="^(bugs\\.debian\\.org|changelogs\\.ubuntu\\.com):443$";
 
diff --git a/src/conserver.cc b/src/conserver.cc
index 6ca6639..afa7d26 100644
--- a/src/conserver.cc
+++ b/src/conserver.cc
@@ -207,6 +207,8 @@ std::string scratchBuf;
 unsigned setup_tcp_listeners(LPCSTR addi, uint16_t port)
 {
 	LOGSTARTFUNCxs(addi, port);
+	if(!port)
+		return 0;
 	USRDBG("Binding on host: " << addi << ", port: " << port);
 
 	auto hints = addrinfo();
