commit 4e71a6542351e688855bc05349c17d2633c5d36c
Author: Antonio Russo <aerusso@aerusso.net>
Date:   Wed Dec 28 11:06:15 2022 -0700

    UDS connections cannot be reused

diff --git a/src/acngtool.cc b/src/acngtool.cc
index 19373ff..14e330f 100644
--- a/src/acngtool.cc
+++ b/src/acngtool.cc
@@ -533,14 +533,6 @@ struct TUdsFactory : public ::acng::IDlConFactory
 					failed = true;
 					return;
 				}
-				// basic identification needed
-				tSS ids;
-				ids << "GET / HTTP/1.0\r\nX-Original-Source: localhost\r\n\r\n";
-				if (!ids.send(m_conFd))
-				{
-					failed = true;
-					return;
-				}
 			}
 		};
 		auto ret = make_shared<udsconnection>();
