commit 080a5aecd9fbc9161e677ffcdbc600580fee5540
Author: Antonio Russo <aerusso@aerusso.net>
Date:   Sat Dec 24 08:32:14 2022 -0700

    Send sentinel client name for unix domain sockets

diff --git a/src/conserver.cc b/src/conserver.cc
index b47a548..6ca6639 100644
--- a/src/conserver.cc
+++ b/src/conserver.cc
@@ -53,7 +53,8 @@ SHARED_PTR<tpool> g_tpool;
 void SetupConAndGo(unique_fd&& man_fd, const char *szClientName, const char *portName)
 {
 	LOGSTARTFUNCs;
-	string sClient(szClientName ? szClientName : "");
+	// szClientName is null exactly when this is a unix domain socket
+	string sClient(szClientName ? szClientName : "0");
 	USRDBG("Client name: " << sClient << ":" << portName);
 	try
 	{
