diff -Naru audacity-1.3.5/debian/control audacity-1.3.5+twolame/debian/control
--- audacity-1.3.5/debian/control	2008-05-06 18:05:38.000000000 +0200
+++ audacity-1.3.5+twolame/debian/control	2008-05-06 17:13:38.000000000 +0200
@@ -4,7 +4,7 @@
 Maintainer: Debian Multimedia Team <debian-multimedia@lists.debian.org>
 Uploaders: Matt Brubeck <mbrubeck@debian.org>, Free Ekanayaka <freee@debian.org>
 Standards-Version: 3.7.2
-Build-Depends: cdbs (>=0.4.4), debhelper (>= 5.0.0), libwxgtk2.6-dev, libvorbis-dev (>=1.0.0-3), libogg-dev (>=1.0.0-1), libmad0-dev, libid3tag0-dev, libsndfile1-dev, gettext, zip, libflac++-dev, libexpat1-dev, libjack-dev, libasound2-dev [!kfreebsd-i386 !kfreebsd-amd64 !hurd-i386], libsamplerate0-dev, libsoundtouch1-dev, libgtk2.0-dev
+Build-Depends: cdbs (>=0.4.4), debhelper (>= 5.0.0), libwxgtk2.6-dev, libvorbis-dev (>=1.0.0-3), libogg-dev (>=1.0.0-1), libmad0-dev, libid3tag0-dev, libsndfile1-dev, gettext, zip, libflac++-dev, libexpat1-dev, libjack-dev, libasound2-dev [!kfreebsd-i386 !kfreebsd-amd64 !hurd-i386], libsamplerate0-dev, libsoundtouch1-dev, libgtk2.0-dev, libtwolame-dev
 
 Package: audacity
 Architecture: any
diff -Naru audacity-1.3.5/debian/rules audacity-1.3.5+twolame/debian/rules
--- audacity-1.3.5/debian/rules	2008-05-06 18:05:38.000000000 +0200
+++ audacity-1.3.5+twolame/debian/rules	2008-05-06 17:12:27.000000000 +0200
@@ -29,7 +29,8 @@
 			     --with-expat=system \
 			     --with-portaudio=v19 \
 			     --enable-unicode \
-			     --without-portmixer
+			     --without-portmixer \
+			     --with-libtwolame
 
 DEB_COMPRESS_EXCLUDE := *.htb
 
