--- /usr/share/fai/setup-storage/Sizes.pm	2009-04-22 14:59:20.000000000 +0200
+++ /usr/share/fai/setup-storage/Sizes.pm_new	2009-04-22 14:56:18.000000000 +0200
@@ -158,8 +158,13 @@ sub estimate_size {
         $min_size = $s if ($s < $min_size);
       }
 
-      return $min_size * POSIX::floor($dev_count / 2)
-        if ($level eq "raid1");
+      #return $min_size * POSIX::floor($dev_count / 2)
+      # if ($level eq "raid1");
+      # Since raid1 is a mirrored the maximum size
+      # is always the size of the smallest partition
+      # so just min_size should be correct.
+      # Not 100% sure  
+      return $min_size if ($level eq "raid1");
       return $min_size * $dev_count if ($level eq "raid0");
       return $min_size * ($dev_count - 1) if ($level eq "raid5");
     } else {
