--- ddclient    2008-10-17 21:00:51.000000000 +0200
+++ ddclient    2009-08-22 11:37:18.000000000 +0200
@@ -1798,6 +1798,7 @@
 ######################################################################
 ## get_ip
 ######################################################################
+my $stored_ip;
 sub get_ip {
     my $use = lc shift;
     my ($ip, $arg, $reply, $url, $skip) = (undef, opt($use), '');
@@ -1850,6 +1851,14 @@
        $reply = geturl('', $url, opt('fw-login'), opt('fw-password')) || '';
        $arg   = $url;

+    } elsif ($use eq 'store') {
+        if (defined $stored_ip) {
+            $ip = $stored_ip;
+        } else {
+            return get_ip(opt('use'));
+        }
+    }
+
     } else {
        $url  = opt('fw')       || '';
        $skip = opt('fw-skip')  || '';
@@ -1879,6 +1888,7 @@
     }

     debug("get_ip: using %s, %s reports %s", $use, $arg, define($ip, "<undefined>"));
+    $stored_ip = $ip;
     return $ip;
 }

@@ -2325,7 +2335,11 @@

                my ($status, $ip) = split / /, lc $line;
                my $h = shift @hosts;
-
+                ## Some service providers breaks the convention of appending the IP.
+                if (!defined $ip) {
+                    $ip = get_ip('stored');
+                }
+
                $config{$h}{'status'} = $status;
                if ($status eq 'good') {
                    $config{$h}{'ip'}     = $ip;

