diff --git a/debootstrap b/debootstrap
index 194cfed..74b7365 100755
--- a/debootstrap
+++ b/debootstrap
@@ -606,7 +606,7 @@ if am_doing_phase second_stage; then
 	if [ -e "$TARGET/etc/apt/sources.list" ]; then
 		rm -f "$TARGET/etc/apt/sources.list"
 	fi
-	if [ "${MIRRORS#http://}" != "$MIRRORS" ]; then
+	if [ "${MIRRORS#http*://}" != "$MIRRORS" ]; then
 		setup_apt_sources "${MIRRORS%% *}"
 		mv_invalid_to "${MIRRORS%% *}"
 	else
