--- a/debian/rules
+++ b/debian/rules
@@ -489,14 +489,14 @@ else
 	)
   endif
 endif
-ifneq (,$(CROSS))
-	sed -e "s/__TARGET__/$$(echo -n $(CROSS) | sed s/_/-/g)/" \
-		< debian/strip.cross.in >> debian/strip.cross
-	chmod 755 debian/strip.cross
-endif
 	@mkdir -p stamps
 	touch $@
 
+debian/strip.cross: debian/strip.cross.in
+	sed -e "s/__TARGET__/$(CROSS)/" \
+		< debian/strip.cross.in >> debian/strip.cross
+	chmod 755 debian/strip.cross
+
 #######################
 # single-arch targets #
 #######################
@@ -884,7 +884,7 @@ ifeq ($(BACKPORT),true)
     install_stamps :=
 endif
 install: $(install_stamp)
-stamps/install: checkroot stamps/build $(install_stamps)
+stamps/install: checkroot stamps/build $(install_stamps) $(if $(CROSS),debian/strip.cross)
 	$(checkdir)
 
 	rm -fr $(d_bin) $(d_dev) $(d_mul) $(d_mdev) $(d_doc) $(d_src)
@@ -1161,7 +1161,7 @@ endif
 
 	touch $@
 
-stamps/install-hppa64: checkroot stamps/build-hppa64
+stamps/install-hppa64: checkroot stamps/build-hppa64 $(if $(CROSS),debian/strip.cross)
 	$(checkdir)
 
 	rm -fr $(d_hppa64)
@@ -1204,7 +1204,7 @@ stamps/install-hppa64: checkroot stamps/build-hppa64
 
 	touch $@
 
-stamps/install-static: checkroot stamps/build-static
+stamps/install-static: checkroot stamps/build-static $(if $(CROSS),debian/strip.cross)
 	$(checkdir)
 
 	rm -fr $(d_static) $(d_udeb)

