diff --git a/debian/scripts/suspend/cryptsetup-suspend-wrapper b/debian/scripts/suspend/cryptsetup-suspend-wrapper
index 52e09dd1..953196c0 100644
--- a/debian/scripts/suspend/cryptsetup-suspend-wrapper
+++ b/debian/scripts/suspend/cryptsetup-suspend-wrapper
@@ -46,6 +46,7 @@ read_config() {
 # Run all executable scripts in directory SYSTEM_SLEEP_PATH with arguments ARGS
 # mimic systemd behavior
 run_dir() {
+    [ -d "$SYSTEM_SLEEP_PATH" ] || return 0
     find "$SYSTEM_SLEEP_PATH" -type f -executable -execdir {} "$@" \;
 }
 
