Index: dh_make
===================================================================
--- dh_make	(revision 18431)
+++ dh_make	(working copy)
@@ -151,13 +151,13 @@
   -f, --file <file>         specify file to use as the original source archive
       --createorig          make a copy for the original source archive
   -s, --single              set package class to single
-  -i, --indep				set package class to arch-independent
+  -i, --indep               set package class to arch-independent
   -m, --multi               set package class to multiple binary
   -l, --library             set package class to library
   -k, --kmod                set package class to kernel module
       --kpatch              set package class to kernel patch
   -a, --addmissing          reprocess package and add missing files
-  -t, --templates <dir>      apply customizing templates in <dir>
+  -t, --templates <dir>     apply customizing templates in <dir>
   -d  --defaultless         skip the default debian and package class templates
   -o, --overlay <dir>       reprocess package using template in <dir>
   -p, --packagename <name>  force package name to be <name>
@@ -418,14 +418,14 @@
 {
 	# Print what we have found
 	print "Maintainer name : $username\n";
-	print "Email-Address	: $email \n";
-	print "Date				: $date\n";
-	print "Package Name	 : $package_name\n";
-	print "Version			: $version\n";
-	print "License			: $license";
+	print "Email-Address   : $email \n";
+	print "Date            : $date\n";
+	print "Package Name    : $package_name\n";
+	print "Version         : $version\n";
+	print "License         : $license";
 	print 'blank' if $license eq '';
 	print "\n";
-	print "Using dpatch	 : ",($dpatch?"yes":"no"),"\n";
+	print "Using dpatch    : ",($dpatch?"yes":"no"),"\n";
 	print "Type of Package : ";
 	if (exists $PACKAGE_TYPES{$package_type})
 	{
