--- /etc/cron.daily/apt-listbugs.old	2007-11-21 00:10:00.000000000 +0100
+++ /etc/cron.daily/apt-listbugs	2007-12-24 18:12:05.000000000 +0100
@@ -8,7 +8,7 @@
 
 	tmp=$(tempfile)
 
-	/usr/share/apt-listbugs/aptcleanup > "$tmp"
+	/usr/share/apt-listbugs/aptcleanup > "$tmp" || return 0
 	if ! diff -B "$tmp" /etc/apt/preferences > /dev/null; then
 		if test -f "/var/backups/apt-listbugs.preferences"; then
 			savelog -q /var/backups/apt-listbugs.preferences
