--- /usr/share/apt-listbugs/aptcleanup.old	2007-11-21 00:10:00.000000000 +0100
+++ /usr/share/apt-listbugs/aptcleanup	2007-12-24 18:04:51.000000000 +0100
@@ -10,10 +10,17 @@
 pinnedpkgs = buf.split(' ')
 $stderr.puts "Pinned: #{pinnedpkgs.sort.join(' ')}" if $DEBUG
 bugpkgs = []
-open("|/usr/sbin/apt-listbugs -q list #{buf} | tail -n 1 ") { |io|
-  buf = io.read.delete(' ').gsub(/\([^\)]+\)/,'').chomp
-  bugpkgs = buf.split(',')
+open("|/usr/sbin/apt-listbugs -y -q list #{buf}") { |io|
+  array = io.readlines()
+  if array.size != 0
+    buf = array[array.size-1].delete(' ').gsub(/\([^\)]+\)/,'').chomp
+    bugpkgs = buf.split(',')
+  end
 }
+if $?.exitstatus != 0
+  $stderr.puts "Error... exiting!" if $DEBUG
+  exit 1
+end
 $stderr.puts "Bogs: #{bugpkgs.sort.join(' ')}" if $DEBUG
 if (pinnedpkgs - bugpkgs).size > 0
   $stderr.puts "#{(pinnedpkgs - bugpkgs).join(', ')} has been fixed"
