diff -urN apt-0.9.2.original/apt-pkg/deb/debsrcrecords.cc apt-0.9.2/apt-pkg/deb/debsrcrecords.cc
--- apt-0.9.2.original/apt-pkg/deb/debsrcrecords.cc	2012-04-16 18:23:05.000000000 +0100
+++ apt-0.9.2/apt-pkg/deb/debsrcrecords.cc	2012-05-10 03:49:45.864895400 +0100
@@ -74,12 +74,14 @@
    BuildDepRec rec;
    const char *fields[] = {"Build-Depends", 
                            "Build-Depends-Indep",
+			   "Build-Depends-Stage1",
+			   "Build-Depends-Stage2",
 			   "Build-Conflicts",
 			   "Build-Conflicts-Indep"};
 
    BuildDeps.clear();
 
-   for (I = 0; I < 4; I++) 
+   for (I = 0; I < 6; I++) 
    {
       if (ArchOnly && (I == 1 || I == 3))
          continue;
diff -urN apt-0.9.2.original/apt-pkg/srcrecords.cc apt-0.9.2/apt-pkg/srcrecords.cc
--- apt-0.9.2.original/apt-pkg/srcrecords.cc	2012-04-16 18:23:05.000000000 +0100
+++ apt-0.9.2/apt-pkg/srcrecords.cc	2012-05-10 03:54:08.282196637 +0100
@@ -123,9 +123,11 @@
 {
    const char *fields[] = {"Build-Depends", 
                            "Build-Depends-Indep",
+			   "Build-Depends-Stage1",
+			   "Build-Depends-Stage2",
 			   "Build-Conflicts",
 			   "Build-Conflicts-Indep"};
-   if (Type < 4) 
+   if (Type < 6) 
       return fields[Type]; 
    else 
       return "";
diff -urN apt-0.9.2.original/apt-pkg/tagfile.cc apt-0.9.2/apt-pkg/tagfile.cc
--- apt-0.9.2.original/apt-pkg/tagfile.cc	2012-04-16 18:23:05.000000000 +0100
+++ apt-0.9.2/apt-pkg/tagfile.cc	2012-05-10 03:33:38.116096575 +0100
@@ -498,6 +498,8 @@
 				      "Original-Maintainer",
                                       "Build-Depends",
                                       "Build-Depends-Indep",
+                                      "Build-Depends-Stage1",
+                                      "Build-Depends-Stage2",
                                       "Build-Conflicts",
                                       "Build-Conflicts-Indep",
                                       "Architecture",
