Index: resources.dbk
===================================================================
--- resources.dbk	(revisione 8928)
+++ resources.dbk	(copia locale)
@@ -238,8 +238,9 @@
 If you have a problem with the operation of a Debian server, and you think that
 the system operators need to be notified of this problem, you can check
 the list of open issues in the DSA queue of our request tracker at <ulink
-url="&url-rt;" /> (you can login with user "guest" and password "readonly").
-To report a new problem, simply send a mail to &email-rt-dsa; and make
+url="&url-rt;" /> (you can login with user "debian" its password is available at
+<filename>master.debian.org:&file-debian-rt-password;</filename>.
+) To report a new problem, simply send a mail to &email-rt-dsa; and make
 sure to put the string "Debian RT" somewhere in the subject.
 </para>
 <para>
Index: common.ent
===================================================================
--- common.ent	(revisione 8928)
+++ common.ent	(copia locale)
@@ -166,6 +166,7 @@
 <!ENTITY file-lisp-controller "<filename>/usr/share/doc/common-lisp-controller/README.packaging</filename>">
 <!ENTITY file-debian-private-archive "~debian/archive/debian-private/">
 <!ENTITY file-debian-private-key "~debian/misc/irc-password">
+<!ENTITY file-debian-rt-password "~debian/misc/rt-password">
 <!ENTITY file-bpp-autotools "<filename>/usr/share/doc/autotools-dev/README.Debian.gz</filename>">
 
 <!ENTITY cron-bug-report '0 17 * * fri   echo "index maint <replaceable>address</replaceable>" | mail request@&bugs-host;'>
