diff --git a/debian/control b/debian/control
index d766846..8c7b42c 100644
--- a/debian/control
+++ b/debian/control
@@ -8,8 +8,12 @@ Build-Depends: debhelper (>= 8)
  , librxtx-java (>= 2.2pre2-3) , libjna-java, ant
  , libastyle-dev
  , libjssc-java , libjsch-java , libjmdns-java
+ , libbcpg-java, libsemver-java
  , libcommons-logging-java , libcommons-httpclient-java
- , libcommons-exec-java
+ , libcommons-exec-java, libcommons-lang3-java, libcommons-codec-java
+ , libcommons-net-java, libcommons-compress-java
+ , libjackson-json-java, libjackson2-databind-java, libjackson2-core-java
+ , libjackson2-annotations-java
  , unzip
 Standards-Version: 3.9.5
 Homepage: http://www.arduino.cc
diff --git a/debian/linkjars b/debian/linkjars
index 1a81e56..0b28cb7 100644
--- a/debian/linkjars
+++ b/debian/linkjars
@@ -1 +1,2 @@
 app/lib
+arduino-core/lib
