diff -u emacspeak-29.0/debian/config emacspeak-29.0/debian/config
--- emacspeak-29.0/debian/config
+++ emacspeak-29.0/debian/config
@@ -56,13 +56,15 @@
 my ($b, $p, $t, $device, $olddevice, 
     %blurb, %program, %tcl, %device, %db, $devicelist);
 ($ret, $database) = metaget ('shared/emacspeak/database', 'choices');
-foreach (split (/, */, $database)){ # debconf database is comma separated
-    ($b, $p, $t, $device)=split(/:/);
-    # load data into hashes
-    $blurb{$device}=$b;
-    $program{$device}=$p;
-    $tcl{$device}=$t;
-    $db{$device}="$b:$p:$t";	# local database is colon separated
+if ($database ne 'shared/emacspeak/database doesn\'t exist') {
+  foreach (split (/, */, $database)){ # debconf database is comma separated
+      ($b, $p, $t, $device)=split(/:/);
+      # load data into hashes
+      $blurb{$device}=$b;
+      $program{$device}=$p;
+      $tcl{$device}=$t;
+      $db{$device}="$b:$p:$t";	# local database is colon separated
+  }
 }
 
 # Add choices from unpacked .blurb files from other packages, in case
@@ -182,6 +184,7 @@
 while (!$valid) {
     input('high', 'shared/emacspeak/port'); go();
     $port=get('shared/emacspeak/port');
+    $port = "none" if $port eq 'shared/emacspeak/port doesn\'t exist';
     if (($port eq "none") || (-c $port)){
 	$valid=1;
     } else {
