diff -u emacspeak-ss-1.12.1/debian/templates emacspeak-ss-1.12.1/debian/templates
--- emacspeak-ss-1.12.1/debian/templates
+++ emacspeak-ss-1.12.1/debian/templates
@@ -31,6 +31,7 @@
 Type: string
 #flag:comment:3
 # Translators, please do not translate "none"
+Default: none
 _Description: Hardware port of the speech generation device:
  If a hardware device is used to generate speech, please enter the
  Unix device file associated with it, such as '/dev/ttyS0' or
diff -u emacspeak-ss-1.12.1/debian/changelog emacspeak-ss-1.12.1/debian/changelog
--- emacspeak-ss-1.12.1/debian/changelog
+++ emacspeak-ss-1.12.1/debian/changelog
@@ -1,3 +1,10 @@
+emacspeak-ss (1.12.1-1.1) unstable; urgency=low
+
+  * Non maintainer upload.
+  * Use a sane default for shared/emacspeak/port template (Closes: #604234).
+
+ -- Agustin Martin Domingo <agmartin@debian.org>  Sat, 01 Jan 2011 04:06:07 +0100
+
 emacspeak-ss (1.12.1-1) unstable; urgency=low
 
   * New upstream release: fix bashism in script testit (thanks to Raphael
