--- /etc/acpi/asus-touchpad.sh	2010-02-16 15:18:19.000000000 +0300
+++ /tmp/asus-touchpad.sh	2010-02-27 11:38:44.828377061 +0300
@@ -25,6 +25,12 @@
 
 if [ $TPSTATUS = 0 ]; then
    xinput set-int-prop $XINPUTNUM "Synaptics Off" 8 1
+   if [ -e /sys/class/leds/asus::touchpad/brightness ]; then
+      sudo echo 0 > /sys/class/leds/asus::touchpad/brightness
+   fi
 else
    xinput set-int-prop $XINPUTNUM "Synaptics Off" 8 0
+   if [ -e /sys/class/leds/asus::touchpad/brightness ]; then
+      sudo echo 1 > /sys/class/leds/asus::touchpad/brightness
+   fi
 fi
