diff -ur docbook-xsl-ns.old/fo/pagesetup.xsl docbook-xsl-ns-1.75.2+dfsg/fo/pagesetup.xsl
--- docbook-xsl-ns.old/fo/pagesetup.xsl	2010-08-27 23:16:53.000000000 +0000
+++ docbook-xsl-ns-1.75.2+dfsg/fo/pagesetup.xsl	2010-08-27 23:38:13.000000000 +0000
@@ -2447,6 +2447,7 @@
     <xsl:when test="$element = 'toc' and self::d:book">i</xsl:when>
     <xsl:when test="$element = 'preface'">i</xsl:when>
     <xsl:when test="$element = 'dedication'">i</xsl:when>
+    <xsl:when test="$element = 'acknowledgements'">i</xsl:when>
     <xsl:otherwise>1</xsl:otherwise>
   </xsl:choose>
 </xsl:template>
@@ -2465,6 +2466,7 @@
                               self::d:bookinfo or
                               self::d:info or
                               self::d:dedication or
+                              self::d:acknowledgements or
                               self::d:preface or
                               self::d:toc or
                               self::d:lot)][1]"/>
