diff -u asciidoc-8.6.7/debian/changelog asciidoc-8.6.7/debian/changelog
--- asciidoc-8.6.7/debian/changelog
+++ asciidoc-8.6.7/debian/changelog
@@ -1,3 +1,12 @@
+asciidoc (8.6.7-2) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Removing configuration files from version 7 to version 8 (Closes: #341613)
+
+
+ -- Joseph HERLANT <herlantj@gmail.com>  Thu, 07 Nov 2013 21:21:26 +0100
+
 asciidoc (8.6.7-1) unstable; urgency=low
 
   * [594ef01] Add homepage field (Closes: #661051)
only in patch2:
unchanged:
--- asciidoc-8.6.7.orig/debian/postrm
+++ asciidoc-8.6.7/debian/postrm
@@ -0,0 +1,74 @@
+#!/bin/sh
+set -e
+#DEBHELPER#
+
+# This if statement avoids to add the "Pre-Depends: dpkg (>= 1.15.7.2)" in
+# the debian/control file
+if dpkg-maintscript-helper supports rm_conffile 2>/dev/null; then
+  # Bug #3416613
+  dpkg-maintscript-helper rm_conffile \
+    /etc/asciidoc/css-embedded-stylesheet.conf 8.6.7-1~ -- "$@"
+  dpkg-maintscript-helper rm_conffile \
+    /etc/asciidoc/css-embedded.conf 8.6.7-1~ -- "$@"
+  dpkg-maintscript-helper rm_conffile \
+    /etc/asciidoc/css.conf 8.6.7-1~ -- "$@"
+  dpkg-maintscript-helper rm_conffile \
+    /etc/asciidoc/docbook-sgml.conf 8.6.7-1~ -- "$@"
+  dpkg-maintscript-helper rm_conffile \
+    /etc/asciidoc/html.conf 8.6.7-1~ -- "$@"
+  dpkg-maintscript-helper rm_conffile \
+    /etc/asciidoc/xhtml.conf 8.6.7-1~ -- "$@"
+fi
