Index: debian/changelog
===================================================================
diff --git a/packages/cherrytree/trunk/debian/changelog b/packages/cherrytree/trunk/debian/changelog
--- a/packages/cherrytree/trunk/debian/changelog	(revision 14528)
+++ b/packages/cherrytree/trunk/debian/changelog	(working copy)
@@ -1,3 +1,10 @@
+cherrytree (0.37.6-1.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Drop depends on python-appindicator (Closes: #886035)
+
+ -- Jeremy Bicha <jbicha@debian.org>  Mon, 01 Jan 2018 15:50:46 -0500
+
 cherrytree (0.37.6-1) unstable; urgency=medium
 
   * New upstream release.
Index: debian/control
===================================================================
diff --git a/packages/cherrytree/trunk/debian/control b/packages/cherrytree/trunk/debian/control
--- a/packages/cherrytree/trunk/debian/control	(revision 14528)
+++ b/packages/cherrytree/trunk/debian/control	(working copy)
@@ -19,7 +19,6 @@
 Architecture: all
 Depends:
  p7zip-full,
- python-appindicator,
  python-dbus,
  python-enchant,
  python-chardet,
