--- /usr/bin/dh	2015-09-25 11:24:41.227536351 +0200
+++ /tmp/dh	2015-09-25 11:24:37.743528671 +0200
@@ -647,6 +647,14 @@
 	$stoppoint=command_pos($dh{BEFORE}, @sequence) - 1;
 }
 
+# Print information about compiler flags to be later used when parsing the
+# build log (e.g. by blhc).
+if ($sequence eq 'build'
+        or $sequence eq 'build-arch'
+        or $sequence eq 'build-indep') {
+    doit('dpkg-buildflags', '--status');
+}
+
 # Now run the commands in the sequence.
 foreach my $i (0..$stoppoint) {
 	my $command=$sequence[$i];
