diff -u anacron-2.3/log.c anacron-2.3/log.c
--- anacron-2.3/log.c
+++ anacron-2.3/log.c
@@ -43,6 +43,7 @@
 #include <signal.h>
 #include <sys/types.h>
 #include <string.h>
+#include <stdlib.h> /* for exit() */
 #include "global.h"
 
 static char truncated[] = " (truncated)";
diff -u anacron-2.3/main.c anacron-2.3/main.c
--- anacron-2.3/main.c
+++ anacron-2.3/main.c
@@ -31,6 +31,7 @@
 #include <sys/types.h>
 #include <sys/stat.h>
 #include <string.h>
+#include <stdlib.h> /* for exit() */
 #include "global.h"
 #include "gregor.h"
 
diff -u anacron-2.3/debian/rules anacron-2.3/debian/rules
--- anacron-2.3/debian/rules
+++ anacron-2.3/debian/rules
@@ -1,5 +1,7 @@
 #!/usr/bin/make -f
 
+export DEB_BUILD_MAINT_OPTIONS = hardening=+all
+
 %:
 	dh $@
 
diff -u anacron-2.3/debian/compat anacron-2.3/debian/compat
--- anacron-2.3/debian/compat
+++ anacron-2.3/debian/compat
@@ -1 +1 @@
-7
+9
diff -u anacron-2.3/debian/control anacron-2.3/debian/control
--- anacron-2.3/debian/control
+++ anacron-2.3/debian/control
@@ -1,7 +1,7 @@
 Source: anacron
 Section: admin
 Priority: optional
-Build-Depends: debhelper (>= 7.0.50~)
+Build-Depends: debhelper (>= 9)
 Maintainer: Peter Eisentraut <petere@debian.org>
 Uploaders: Antonio Radici <antonio@dyne.org>
 Standards-Version: 3.9.2
--- anacron-2.3.orig/matchrx.c
+++ anacron-2.3/matchrx.c
@@ -26,6 +26,7 @@
 #include <regex.h>
 #include <stdarg.h>
 #include <stdlib.h>
+#include <string.h> /* for memset() */
 #include "matchrx.h"
 
 int
--- anacron-2.3.orig/Makefile
+++ anacron-2.3/Makefile
@@ -22,7 +22,7 @@
 PREFIX = 
 BINDIR = $(PREFIX)/usr/sbin
 MANDIR = $(PREFIX)/usr/man
-CFLAGS = -Wall -pedantic -O2
+CFLAGS += -Wall -pedantic
 #CFLAGS = -Wall -O2 -g -DDEBUG
 
 # If you change these, please update the man-pages too
