--- /usr/bin//mk-origtargz	2014-09-20 18:13:48.064356878 +0800
+++ /tmp/mk-origtargz	2014-09-20 18:13:21.171380905 +0800
@@ -164,6 +164,7 @@
 use Cwd 'abs_path';
 use File::Copy;
 use Dpkg::Control::Hash;
+use List::MoreUtils qw(uniq);
 
 
 sub decompress_archive($$);
@@ -428,6 +429,7 @@
     # ensure files are mentioned before the directory they live in
     # (otherwise tar complains)
     @to_delete = sort {$b cmp $a}  @to_delete;
+    @to_delete = uniq @to_delete;
 
     $deletecount = scalar(@to_delete);
 }
