diff -ruN orig/dillo-0.8.6/debian/rules dillo-0.8.6/debian/rules
--- orig/dillo-0.8.6/debian/rules	2007-12-27 18:59:20.000000000 +0100
+++ dillo-0.8.6/debian/rules	2007-12-27 19:27:32.000000000 +0100
@@ -23,6 +23,8 @@
 configure: config.status
 config.status: patch-stamp
 	dh_testdir
+	[ -f config.sub.d-r-orig ] || cp -p config.sub config.sub.d-r-orig
+	[ -f config.guess.d-r-orig ] || cp -p config.guess config.guess.d-r-orig
 ifneq "$(wildcard /usr/share/misc/config.sub)" ""
 	cp -f /usr/share/misc/config.sub config.sub
 endif
@@ -45,7 +47,8 @@
 	$(MAKE)
 	touch build-stamp
 
-clean: unpatch
+clean: clean-patched unpatch
+clean-patched:
 	dh_testdir
 	dh_testroot
 	rm -f build-stamp configure-stamp
@@ -65,6 +68,8 @@
 	$(MAKE) install DESTDIR=$(CURDIR)/debian/dillo
 	# Install lintian overrides
 	install -m644 -D debian/dillo.lintian-overrides $(CURDIR)/debian/dillo/usr/share/lintian/overrides/dillo
+	[ ! -f config.sub.d-r-orig ] || mv config.sub.d-r-orig config.sub
+	[ ! -f config.guess.d-r-orig ] || mv config.guess.d-r-orig config.guess
 
 
 binary-indep: build install
