diff -Nru cdbs-0.4.122/debian/changelog cdbs-0.4.122+nmu1/debian/changelog
--- cdbs-0.4.122/debian/changelog	2013-07-02 09:50:00.000000000 +0100
+++ cdbs-0.4.122+nmu1/debian/changelog	2014-04-16 09:42:01.000000000 +0100
@@ -1,3 +1,10 @@
+cdbs (0.4.122+nmu1) UNRELEASED; urgency=medium
+
+  * Non-maintainer upload.
+  * Depend on autotools-dev so that CDBS packages build on new architectures
+
+ -- Wookey <wookey@debian.org>  Wed, 16 Apr 2014 09:41:05 +0100
+
 cdbs (0.4.122) unstable; urgency=low
 
   * Update and improve references in comments to Perl Policy.
diff -Nru cdbs-0.4.122/debian/control cdbs-0.4.122+nmu1/debian/control
--- cdbs-0.4.122/debian/control	2013-07-02 09:06:10.000000000 +0100
+++ cdbs-0.4.122+nmu1/debian/control	2014-04-16 09:40:33.000000000 +0100
@@ -25,7 +25,7 @@
 Package: cdbs
 Architecture: all
 Multi-Arch: foreign
-Recommends: autotools-dev
+Depends: autotools-dev
 Suggests: devscripts
 Description: common build system for Debian packages
  This package contains the Common Debian Build System, an abstract build
diff -Nru cdbs-0.4.122/debian/control.in cdbs-0.4.122+nmu1/debian/control.in
--- cdbs-0.4.122/debian/control.in	2013-07-01 19:18:29.000000000 +0100
+++ cdbs-0.4.122+nmu1/debian/control.in	2014-04-16 09:40:50.000000000 +0100
@@ -23,7 +23,7 @@
 Package: cdbs
 Architecture: all
 Multi-Arch: foreign
-Recommends: autotools-dev
+Depends: autotools-dev
 Suggests: devscripts
 Description: common build system for Debian packages
  This package contains the Common Debian Build System, an abstract build
