diff --git a/lib/dh-exec-install-rename b/lib/dh-exec-install-rename
index 7d4d993..d8b0355 100755
--- a/lib/dh-exec-install-rename
+++ b/lib/dh-exec-install-rename
@@ -63,6 +63,8 @@ if (/([^\s]*)\s+=>\s+([^\s]*)/ || /^=>\s+([^\s]*)/) {

         if ($move_this) {
             move ($src, File::Spec->catfile ($debpath, $dstfile)) or
+		move (File::Spec->catfile ("debian/tmp", $src),
+		      File::Spec->catfile ($debpath, $dstfile)) or
                 die "Failed to move '$src': $!";
         } else {
             cp ($src, File::Spec->catfile ($debpath, $dstfile)) or
