*** sockd.c.orig	Sat Mar 10 17:30:22 2001
--- sockd.c	Sat Mar 10 17:15:03 2001
***************
*** 272,279 ****
  
  	socks_seteuid(NULL, config.uid.privileged);
  	if ((fp = fopen(SOCKD_PIDFILE, "w")) == NULL) {
! 		swarn("open(%s)", SOCKD_PIDFILE);
! 		errno = 0;
  	}
  	socks_seteuid(NULL, config.uid.unprivileged);
  
--- 272,282 ----
  
  	socks_seteuid(NULL, config.uid.privileged);
  	if ((fp = fopen(SOCKD_PIDFILE, "w")) == NULL) {
! 		socks_seteuid(NULL, 0);
! 		if ((fp = fopen(SOCKD_PIDFILE, "w")) == NULL) {
! 			swarn("open(%s)", SOCKD_PIDFILE);
! 			errno = 0;
! 		}
  	}
  	socks_seteuid(NULL, config.uid.unprivileged);
  
