diff -Naur cdbs-0.4.121/1/class/qmake.mk.in cdbs-0.4.121.mod/1/class/qmake.mk.in
--- cdbs-0.4.121/1/class/qmake.mk.in	2012-12-23 13:11:54.000000000 -0300
+++ cdbs-0.4.121.mod/1/class/qmake.mk.in	2013-04-14 17:28:07.893676624 -0300
@@ -36,6 +36,11 @@
 DEB_QMAKE_CONFIG_VAL ?= nostrip
 endif
 
+# Use Qt4 by default except QT_SELECT says otherwise.
+ifndef QT_SELECT
+export QT_SELECT=qt4
+endif
+
 common-configure-arch common-configure-indep:: common-configure-impl
 common-configure-impl:: $(DEB_BUILDDIR)/Makefile
 $(DEB_BUILDDIR)/Makefile:
