diff -Nru dump-0.4b47/debian/changelog dump-0.4b47/debian/changelog
--- dump-0.4b47/debian/changelog	2022-05-03 07:16:47.000000000 +0200
+++ dump-0.4b47/debian/changelog	2024-07-08 00:44:52.000000000 +0200
@@ -1,3 +1,12 @@
+dump (0.4b47-4.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Install files into /usr/sbin. (Closes: #1073616)
+  * Update Build-Depends: pkg-config to pkgconf, e2fslibs-dev to
+    libext2fs-dev.
+
+ -- Chris Hofstaedtler <zeha@debian.org>  Mon, 08 Jul 2024 00:44:52 +0200
+
 dump (0.4b47-4) unstable; urgency=medium
 
   * fix dumpdates logic for backing up by LABEL= or UUID=
@@ -53,7 +62,7 @@
   * applied patch by Michael Cree to make dump build on alpha again
     (alpha has no getpid syscall, which
     the #623422 patch for sparc uses) (closes: #900885)
-    
+
  -- Alexander Zangerl <az@debian.org>  Sat, 09 Jun 2018 11:23:46 +1000
 
 dump (0.4b46-4) unstable; urgency=medium
@@ -90,7 +99,7 @@
   * amend build depends for lzo2 compression (closes: #836692)
   * ensure that conditional compilation for compression-related
 	  code is primed suitably (closes: #836548, #836674)
-	
+
  -- Alexander Zangerl <az@debian.org>  Mon, 05 Sep 2016 18:06:43 +1000
 
 dump (0.4b45-4) unstable; urgency=medium
@@ -110,7 +119,7 @@
 
   * interim build with QFA disabled, as that triggers a
     segfault and renders dump completely broken (closes: #826398)
-	
+
  -- Alexander Zangerl <az@debian.org>  Wed, 08 Jun 2016 19:04:16 +1000
 
 dump (0.4b45-1) unstable; urgency=medium
diff -Nru dump-0.4b47/debian/control dump-0.4b47/debian/control
--- dump-0.4b47/debian/control	2022-05-03 07:16:47.000000000 +0200
+++ dump-0.4b47/debian/control	2024-07-08 00:44:52.000000000 +0200
@@ -2,7 +2,7 @@
 Section: utils
 Priority: optional
 Maintainer: Alexander Zangerl <az@debian.org>
-Build-Depends: debhelper (>= 10), autoconf, comerr-dev, e2fslibs-dev, libblkid-dev, libbz2-dev,  liblzo2-dev, libdevmapper-dev, libreadline-dev,  libselinux1-dev, pkg-config, uuid-dev, zlib1g-dev
+Build-Depends: debhelper (>= 10), autoconf, comerr-dev, libext2fs-dev, libblkid-dev, libbz2-dev,  liblzo2-dev, libdevmapper-dev, libreadline-dev, libselinux1-dev, pkgconf, uuid-dev, zlib1g-dev
 Standards-Version: 4.1.3.0
 Homepage: http://dump.sourceforge.net
 Vcs-Browser: https://salsa.debian.org/debian/dump
diff -Nru dump-0.4b47/debian/dirs dump-0.4b47/debian/dirs
--- dump-0.4b47/debian/dirs	2022-05-03 05:01:50.000000000 +0200
+++ dump-0.4b47/debian/dirs	2024-07-08 00:44:17.000000000 +0200
@@ -1,3 +1,2 @@
-sbin
 usr/sbin
 usr/share/man/man8
diff -Nru dump-0.4b47/debian/rules dump-0.4b47/debian/rules
--- dump-0.4b47/debian/rules	2022-05-03 05:01:50.000000000 +0200
+++ dump-0.4b47/debian/rules	2024-07-08 00:44:52.000000000 +0200
@@ -8,7 +8,6 @@
 	dh_testdir
 	EXT2FS_LIBS="-lext2fs -lcom_err" dh_auto_configure -- \
 		--prefix=/usr \
-		--sbindir=/sbin \
 		--with-ccopts="-O2 -g -Wall" \
 		--with-dumpdatespath=/var/lib/dumpdates \
 		--enable-lzo --enable-bzip2 --enable-zlib \
@@ -16,14 +15,14 @@
 
 override_dh_auto_install:
 	dh_auto_install -- \
-		SBINDIR=`pwd`/debian/dump/sbin \
+		SBINDIR=`pwd`/debian/dump/usr/sbin \
 		MANDIR=`pwd`/debian/dump/usr/share/man/man8
 
 	mv debian/dump/usr/share/man/man8/rmt.8 \
 		debian/dump/usr/share/man/man8/rmt-dump.8
-	mv debian/dump/sbin/rmt debian/dump/usr/sbin/rmt-dump
+	mv debian/dump/usr/sbin/rmt debian/dump/usr/sbin/rmt-dump
 
-	(cd debian/dump/sbin ; rm -f rdump rrestore ; \
+	(cd debian/dump/usr/sbin ; rm -f rdump rrestore ; \
 		ln -s dump rdump ; ln -s restore rrestore )
 	(cd debian/dump/usr/share/man/man8 ; rm -f rdump* rrestore* ; \
 		ln -s dump.8.gz rdump.8.gz ; ln -s restore.8.gz rrestore.8.gz )
