--- a/Tests/RunCMake/Configure/RunCMakeTest.cmake
+++ b/Tests/RunCMake/Configure/RunCMakeTest.cmake
@@ -15,6 +15,7 @@
 set(output "${RunCMake_TEST_BINARY_DIR}/CustomCMakeOutput.txt")
 file(WRITE "${input}" "1")
 file(WRITE "${depend}" "1")
+execute_process(COMMAND ${CMAKE_COMMAND} -E sleep 1) # handle 1s resolution
 run_cmake(RerunCMake)
 execute_process(COMMAND ${CMAKE_COMMAND} -E sleep 1) # handle 1s resolution
 file(WRITE "${input}" "2")
--- a/Tests/RunCMake/Configure/RerunCMake.cmake
+++ b/Tests/RunCMake/Configure/RerunCMake.cmake
@@ -9,3 +9,5 @@
 file(READ ${depend} content)
 file(WRITE ${output} "${content}")
 set_property(DIRECTORY APPEND PROPERTY CMAKE_CONFIGURE_DEPENDS RerunCMake.txt)
+
+execute_process(COMMAND ${CMAKE_COMMAND} -E sleep 1) # handle 1s resolution
