--- /usr/share/perl5/App/ClusterSSH.pm	2013-08-26 17:27:51.809221174 +0200
+++ /tmp/ClusterSSH.pm.fixed	2013-08-26 17:28:25.341589234 +0200
@@ -635,7 +635,7 @@ sub get_clusters() {
                 ;    # ignore blank lines & commented lines
             chomp $l;
             if ( $l =~ s/\\\s*$// ) {
-                $l .= <CLUSTER>;
+                $l .= <CLUSTERS>;
                 redo unless eof(CLUSTERS);
             }
             my @line = split( /\s/, $l );
@@ -695,7 +695,7 @@ sub get_clusters() {
                     next if ( $l =~ /^\s*$/ || $l =~ /^#/ );
                     chomp $l;
                     if ( $l =~ s/\\\s*$// ) {
-                        $l .= <CLUSTER>;
+                        $l .= <CLUSTERS>;
                         redo unless eof(CLUSTERS);
                     }
 
