diff -Naur etckeeper-0.34.old/etckeeper.conf etckeeper-0.34/etckeeper.conf
--- etckeeper-0.34.old/etckeeper.conf	2009-02-25 19:28:52.000000000 +0000
+++ etckeeper-0.34/etckeeper.conf	2009-03-25 19:36:02.000000000 +0000
@@ -32,3 +32,6 @@
 # The low-level package manager that's being used.
 # (dpkg, rpm, pacman-g2, etc)
 LOWLEVEL_PACKAGE_MANAGER=dpkg
+
+# Uncomment the below to always commit after an install (bzr only)
+#FORCE_COMMIT_AFTER_INSTALL=1
diff -Naur etckeeper-0.34.old/post-install.d/50vcs-commit etckeeper-0.34/post-install.d/50vcs-commit
--- etckeeper-0.34.old/post-install.d/50vcs-commit	2008-12-16 23:14:38.000000000 +0000
+++ etckeeper-0.34/post-install.d/50vcs-commit	2009-03-25 20:04:15.000000000 +0000
@@ -5,9 +5,13 @@
 NL="
 "
 
-if etckeeper unclean; then
+if etckeeper unclean || [ ! -z $FORCE_COMMIT_AFTER_INSTALL ]; then
 	message="committing changes in /etc after $HIGHLEVEL_PACKAGE_MANAGER run"
-
+	
+	if [ ! -z $FORCE_COMMIT_AFTER_INSTALL -a "$VCS" = bzr ]; then
+		export BZR_COMMIT_OPTIONS="$BZR_COMMIT_OPTIONS --unchanged"
+	fi
+	
 	if [ -e $pl.pre-install ]; then
 		diff="$(etckeeper list-installed | diff -U0 $pl.pre-install - | tail -n+4 | egrep '^[-+]')" || true
 		if [ -n "$diff" ]; then
