diff -ur apt-0.7.9.old/cmdline/apt-key apt-0.7.9/cmdline/apt-key
--- apt-0.7.9.old/cmdline/apt-key	2007-09-12 22:03:51.000000000 +0200
+++ apt-0.7.9/cmdline/apt-key	2000-01-01 01:31:34.000000000 +0100
@@ -5,7 +5,7 @@
 # We don't use a secret keyring, of course, but gpg panics and
 # implodes if there isn't one available
 
-GPG_CMD="gpg --no-options --no-default-keyring --secret-keyring /etc/apt/secring.gpg --trustdb-name /etc/apt/trustdb.gpg"
+GPG_CMD="gpg --ignore-time-conflict --no-options --no-default-keyring --secret-keyring /etc/apt/secring.gpg --trustdb-name /etc/apt/trustdb.gpg"
 GPG="$GPG_CMD --keyring /etc/apt/trusted.gpg"
 
 
@@ -21,7 +21,7 @@
     fi
 
     # add new keys
-    $GPG_CMD --quiet --batch --keyring $ARCHIVE_KEYRING --export | $GPG --ignore-time-conflict --import
+    $GPG_CMD --quiet --batch --keyring $ARCHIVE_KEYRING --export | $GPG --import
 
     # remove no-longer used keys
     keys=`$GPG_CMD --keyring $REMOVED_KEYS --with-colons --list-keys | grep ^pub | cut -d: -f5`
diff -ur apt-0.7.9.old/methods/gpgv.cc apt-0.7.9/methods/gpgv.cc
--- apt-0.7.9.old/methods/gpgv.cc	2007-10-02 14:34:38.000000000 +0200
+++ apt-0.7.9/methods/gpgv.cc	2000-01-01 01:32:31.000000000 +0100
@@ -83,6 +83,7 @@
       Args[i++] = gpgvpath.c_str();
       Args[i++] = "--status-fd";
       Args[i++] = "3";
+      Args[i++] = "--ignore-time-conflict";
       Args[i++] = "--keyring";
       Args[i++] = pubringpath.c_str();
 
