diff -Naur console-setup-1.45~/debian/console-setup.console-setup.init console-setup-1.45/debian/console-setup.console-setup.init
--- console-setup-1.45~/debian/console-setup.console-setup.init	2009-09-23 20:54:12.000000000 +0200
+++ console-setup-1.45/debian/console-setup.console-setup.init	2009-09-23 20:54:40.000000000 +0200
@@ -3,7 +3,7 @@
 # Provides:          console-setup
 # Required-Start:    $remote_fs
 # Required-Stop:
-# Should-Start:      console-screen
+# Should-Start:      console-screen kbd
 # Default-Start:     S
 # Default-Stop:
 # X-Interactive:     true
