--- aptitude-0.4.9.orig/src/cmdline/cmdline_action.cc   2007-11-16 06:07:58.000000000 +0100
+++ aptitude-0.4.9/src/cmdline/cmdline_action.cc        2010-08-05 13:35:25.000000000 +0200
@@ -85,6 +85,11 @@
   if(action==cmdline_install)
     ver=cmdline_find_ver(pkg, source, sourcestr);
 
+       if (!ver)
+               {
+               printf("Requested version not found in repositories: %s\n", sourcestr.c_str());
+               abort();
+               }
   switch(action)
     {
     case cmdline_installauto:

