diff -u dibbler-0.7.3/debian/changelog dibbler-0.7.3/debian/changelog
--- dibbler-0.7.3/debian/changelog
+++ dibbler-0.7.3/debian/changelog
@@ -1,3 +1,12 @@
+dibbler (0.7.3-1.2) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Use '$DAEMON stop 2>&1 > /dev/null || true' as proposed by Edward
+    Welbourne so solve problems when uninstalling / upgrading / purging the
+    package (Closes: #574624)
+
+ -- Alexander Reichle-Schmehl <tolimar@debian.org>  Thu, 15 Jul 2010 13:43:57 +0200
+
 dibbler (0.7.3-1.1) unstable; urgency=low
 
   * Non-maintainer upload.
diff -u dibbler-0.7.3/debian/dibbler-client.init dibbler-0.7.3/debian/dibbler-client.init
--- dibbler-0.7.3/debian/dibbler-client.init
+++ dibbler-0.7.3/debian/dibbler-client.init
@@ -71,7 +71,7 @@
 	#	just the same as "restart".
 	#
 	echo -n "Restarting $DESC: "
-	($DAEMON stop 2>&1 > /dev/null; true)
+	($DAEMON stop 2>&1 > /dev/null|| true)
 	sleep 1
 	$DAEMON start 2>&1 > /dev/null
 	echo "$NAME."
