--- dupload.conf	2007-09-23 11:34:01.000000000 +0900
+++ /etc/dupload.conf	2007-09-23 11:37:13.000000000 +0900
@@ -90,22 +90,12 @@
 };
 
 # Mentors upload queue, see
-# http://mentors.debian.net/signup.php
+# http://mentors.debian.net/cgi-bin/maintainer-intro
 $cfg{'mentors'} = {
 	fqdn    =>'mentors.debian.net',
-	method  =>'scpb',
-	login   =>'incoming',
-	incoming=>'~',
-# Change these to the user and domain part of your email address
-# and uncomment them
-#	visibleuser=>'hugo',
-#	visiblename=>'mydomain.tld',
-	mailtx  =>'incoming@mentors.debian.net',
-	preupload=> {
-		deb=>'chmod 0644 %1',
-		changes=>'chmod 0644 %1',
-		file=>'chmod 0644 %1',
-	},
+	incoming=>'/',
+	dinstall_runs => 1,
+	passive => 1,
 };
 
 # NOTE: Do _NOT_ upload a package to the security upload queue 
