diff -u bind9-9.4.0/debian/changelog bind9-9.4.0/debian/changelog
--- bind9-9.4.0/debian/changelog
+++ bind9-9.4.0/debian/changelog
@@ -1,3 +1,10 @@
+bind9 (1:9.4.0-2.1) UNRELEASED; urgency=low
+
+  * Non-maintainer upload.
+  * Register BIND ARM with doc-base (closes: #139115).
+
+ -- Sam Morris <sam@robots.org.uk>  Sat, 14 Apr 2007 13:04:02 +0100
+
 bind9 (1:9.4.0-2) unstable; urgency=low
 
   * upload to unstable
only in patch2:
unchanged:
--- bind9-9.4.0.orig/debian/bind9-doc.doc-base
+++ bind9-9.4.0/debian/bind9-doc.doc-base
@@ -0,0 +1,14 @@
+Document: bind9-arm
+Title: BIND 9 Administrator Reference Manual
+Author: Internet Systems Consortium, Inc.
+Abstract: The Berkeley Internet Name Domain (BIND) implements a domain name
+ server for a number of operating systems. This document provides basic
+ information about the installation and care of the Internet Software
+ Consortium (ISC) BIND version 9 software package for system administrators.
+ .
+ This version of the manual corresponds to BIND version 9.3.
+Section: Apps/Net
+
+Format: html
+Index: /usr/share/doc/bind9-doc/arm/Bv9ARM.html
+Files: /usr/share/doc/bind9-doc/arm/*.html
