diff -urN advi-1.10.0.orig//debian/control advi-1.10.0/debian/control
--- advi-1.10.0.orig//debian/control	2011-08-03 03:06:56.000000000 +0900
+++ advi-1.10.0/debian/control	2011-10-09 07:52:11.071777264 +0900
@@ -23,7 +23,9 @@
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, ${ocaml:Depends},
  texlive-base-bin, texlive-base, ghostscript-x (>= 7.05), perl
-Suggests: ttf-kochi-gothic, ttf-kochi-mincho, bzip2
+Suggests: fonts-ipafont-gothic | fonts-japanese-gothic,
+          fonts-ipafont-mincho | fonts-japanese-mincho,
+          bzip2
 Replaces: mldvi, activedvi
 Description: active DVI previewer and presenter
  Active-DVI is a DVI previewer and presenter written in Objective Caml
@@ -42,7 +44,9 @@
 Architecture: all
 Depends: ${misc:Depends}, texlive-latex-base, advi, imagemagick
 Conflicts: advi (<< 1.6.0-4)
-Suggests: ttf-kochi-gothic, ttf-kochi-mincho, ocaml-base-nox, mpg321, tk8.4, xterm
+Suggests: fonts-ipafont-gothic | fonts-japanese-gothic,
+          fonts-ipafont-mincho | fonts-japanese-mincho,
+          ocaml-base-nox, mpg321, tk8.4, xterm
 Description: example presentations for Active-DVI (advi)
  Active-DVI is a DVI previewer and presenter written in Objective Caml
  with some eye candy effects for presentation, support for interactive
