From 1706b83e641e21f41a0a2ec11bb7d82f25fccfda Mon Sep 17 00:00:00 2001
From: Mark Hindley <mark@hindley.org.uk>
Date: Sun, 4 Aug 2024 07:19:30 +0100
Subject: [PATCH] d/elogind.init: start before $x-display-manager.

Closes: 1077676
---
 debian/elogind.init | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/elogind.init b/debian/elogind.init
index 13a156f6a..2014cd06d 100644
--- a/debian/elogind.init
+++ b/debian/elogind.init
@@ -3,6 +3,7 @@
 # Provides: elogind
 # Required-Start: dbus $remote_fs $syslog
 # Required-Stop: dbus $remote_fs $syslog
+# X-Start-Before: $x-display-manager
 # Default-Start: 2 3 4 5
 # Default-Stop: 0 1 6  
 # Short-Description: start elogind at boot tim
-- 
2.39.2

