diff --git a/libiberty/configure b/libiberty/configure
index 9463f8f..c52dfeb 100755
--- a/libiberty/configure
+++ b/libiberty/configure
@@ -4860,6 +4860,7 @@ if [ "${shared}" = "yes" ]; then
 			  * ) PICFLAG=-fPIC ;;
 			esac ;;
     s390*-*-*)		PICFLAG=-fpic ;;
+    sh*-*-*)	PICFLAG=-fPIC ;;
   esac
 fi
 
diff --git a/libiberty/configure.ac b/libiberty/configure.ac
index c34fc10..90ea852 100644
--- a/libiberty/configure.ac
+++ b/libiberty/configure.ac
@@ -217,6 +217,7 @@ if [[ "${shared}" = "yes" ]]; then
 			  * ) PICFLAG=-fPIC ;;
 			esac ;;
     s390*-*-*)		PICFLAG=-fpic ;;
+    sh*-*-*)		PICFLAG=-fPIC ;;
   esac
 fi
 AC_SUBST(PICFLAG)
