--- suspendorhibernate.orig	2008-06-26 00:40:53.000000000 -0700
+++ suspendorhibernate	2008-07-02 01:32:07.000000000 -0700
@@ -73,14 +73,14 @@
 			# Call the power management daemon (which, if it is
 			# running, we probably don't know about, since we send
 			# keys if we detect one running that we know of).
-			if /usr/bin/dbus-send --session \
+			if ! /usr/bin/dbus-send --session \
 			  --dest=org.freedesktop.PowerManagement \
 			  --type=method_call \
 			  --print-reply \
 			  --reply-timeout=2000 \
 			  /org/freedesktop/PowerManagement \
 			  org.freedesktop.PowerManagement.$DBUS_METHOD \
-			  2&>1 | grep -vq " org.freedesktop.DBus.Error." ;
+			  2>&1 | grep -q " org.freedesktop.DBus.Error." ;
 			  then			  
 			  	# Not a DBUS error: other side does exist, and
 			  	# reports an error. That means we don't try
@@ -96,14 +96,14 @@
 	dbus-hal)
 		if [ -x /usr/bin/dbus-send ] ; then
 			# Call HAL directly.
-			if /usr/bin/dbus-send --system \
-			  --dest=org.freedesktop.Hal.Device.SystemPowerManagement \
+			if ! /usr/bin/dbus-send --system \
+			  --dest=org.freedesktop.Hal \
 			  --type=method_call \
 			  --print-reply \
 			  --reply-timeout=2000 \
-			  /org/freedesktop/Hal/Device/SystemPowerManagement \
+			  /org/freedesktop/Hal/devices/computer \
 			  org.freedesktop.Hal.Device.SystemPowerManagement.$DBUS_METHOD $DBUS_PARAMS \
-			  2&>1 | grep -vq " org.freedesktop.DBus.Error." ;
+			  2>&1 | grep -q " org.freedesktop.DBus.Error." ;
 			  then			  
 			  	# Not a DBUS error: other side does exist, and
 			  	# reports an error. That means we don't try
