diff --git a/dh_auto_build b/dh_auto_build
index 75ce51c..ba519b8 100755
--- a/dh_auto_build
+++ b/dh_auto_build
@@ -49,6 +49,9 @@ elsif (-e "Build.PL" && -e "Build") {
 	$ENV{MODULEBUILDRC} = "/dev/null";
 	doit("perl", "Build", @{$dh{U_PARAMS}});
 }
+elsif (-e "build.xml") {
+	doit("ant", @{$dh{U_PARAMS}});
+}
 
 =head1 SEE ALSO
 
diff --git a/dh_auto_clean b/dh_auto_clean
index 610155a..d720b68 100755
--- a/dh_auto_clean
+++ b/dh_auto_clean
@@ -63,6 +63,9 @@ elsif (-e "Build.PL" && -e "Build") {
 	$ENV{MODULEBUILDRC} = "/dev/null";
 	doit("perl", "Build", "--allow_mb_mismatch", 1, "distclean", @{$dh{U_PARAMS}});
 }
+elsif (-e "build.xml") {
+	doit("ant", "clean", @{$dh{U_PARAMS}});
+}
 
 =head1 SEE ALSO
 
