--- checkinstall.orig	2018-03-15 16:00:29.997552520 +0800
+++ /usr/bin/checkinstall	2018-03-15 15:12:18.232830708 +0800
@@ -1975,7 +1975,7 @@
 
  cat ${TMP_DIR}/libdirs | while read libdir; do
    (
-    echo "if ! egrep \"^/${libdir} *$\" /etc/ld.so.conf &> /dev/null; then"
+    echo "if ! egrep \"^/${libdir} *$\" /etc/ld.so.conf >/dev/null 2>&1; then"
     echo "   echo \"Adding \"/$libdir\" to /etc/ld.so.conf\"" 
     echo "   echo /$libdir >> /etc/ld.so.conf"
     echo fi

