diff -Nru dvhtool-1.0.1/debian/changelog dvhtool-1.0.1/debian/changelog
--- dvhtool-1.0.1/debian/changelog	2022-01-13 22:40:03.000000000 +0200
+++ dvhtool-1.0.1/debian/changelog	2023-02-09 23:23:54.000000000 +0200
@@ -1,3 +1,10 @@
+dvhtool (1.0.1-6.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Add the path to dvhtool in the autopkgtest.
+
+ -- Adrian Bunk <bunk@debian.org>  Thu, 09 Feb 2023 23:23:54 +0200
+
 dvhtool (1.0.1-6) unstable; urgency=medium
 
   [ Guido Günther ]
diff -Nru dvhtool-1.0.1/debian/tests/control dvhtool-1.0.1/debian/tests/control
--- dvhtool-1.0.1/debian/tests/control	2022-01-13 22:38:52.000000000 +0200
+++ dvhtool-1.0.1/debian/tests/control	2023-02-09 23:23:51.000000000 +0200
@@ -1,2 +1,2 @@
-Test-Command: dvhtool --print-all /usr/share/doc/dvhtool/examples/volhdr-1.dat
+Test-Command: /usr/sbin/dvhtool --print-all /usr/share/doc/dvhtool/examples/volhdr-1.dat
 Depends: @
