Index: resources.dbk
===================================================================
--- resources.dbk	(revisione 8928)
+++ resources.dbk	(copia locale)
@@ -534,10 +534,9 @@
 architectures.  Debian 2.1 shipped for the <literal>i386</literal>,
 <literal>m68k</literal>, <literal>alpha</literal>, and
 <literal>sparc</literal> architectures.  Since then Debian has grown hugely.
-Debian 5 supports a total of twelve architectures: <literal>alpha</literal>,
-<literal>amd64</literal>, <literal>arm</literal>,
-<literal>armel</literal>, <literal>hppa</literal>,
-<literal>i386</literal>, <literal>ia64</literal>, <literal>mips</literal>,
+Debian 6 supports a total of nine architectures: <literal>amd64</literal>,
+<literal>armel</literal>, <literal>i386</literal>,
+<literal>ia64</literal>, <literal>mips</literal>,
 <literal>mipsel</literal>, <literal>powerpc</literal>,
 <literal>s390</literal>, <literal>sparc</literal>.
 </para>
@@ -669,8 +668,8 @@
 <filename>proposed-updates</filename> directory.  Those packages in
 <filename>proposed-updates</filename> that pass muster are periodically moved
 as a batch into the stable distribution and the revision level of the stable
-distribution is incremented (e.g., â3.0â becomes â3.0r1â, â2.2r4â
-becomes â2.2r5â, and so forth).  Please refer to
+distribution is incremented (e.g., â6.0â becomes â6.0.1â, â5.0.7â
+becomes â5.0.8â, and so forth).  Please refer to
 <link linkend="upload-stable">uploads to the <literal>stable</literal>
 distribution</link> for details.
 </para>
@@ -763,7 +762,7 @@
 Debian 2.1, <literal>slink</literal>; Debian 2.2, <literal>potato</literal>;
 Debian 3.0, <literal>woody</literal>; Debian 3.1, <literal>sarge</literal>;
 Debian 4.0, <literal>etch</literal>; Debian 5.0, <literal>lenny</literal>
-and the next release will be called <literal>squeeze</literal>.
+and the next release will be called <literal>wheezy</literal>.
 There is also a ``pseudo-distribution'', called
 <literal>sid</literal>, which is the current <literal>unstable</literal>
 distribution; since packages are moved from <literal>unstable</literal> to
Index: common.ent
===================================================================
--- common.ent	(revisione 8928)
+++ common.ent	(copia locale)
@@ -10,10 +10,10 @@
 
 <!-- volatile information -->
 
-<!ENTITY number-of-pkgs "12000">
+<!ENTITY number-of-pkgs "15000">
 <!ENTITY number-of-maintainers "1000">
 
-<!ENTITY number-of-arches "12">
+<!ENTITY number-of-arches "11">
 
 <!-- standard information -->
 <!ENTITY fsf-addr "Free Software Foundation, Inc.,
