Index: apt-build-0.12.37/apt-build
===================================================================
--- apt-build-0.12.37.orig/apt-build	2010-08-06 21:41:42.000000000 +0200
+++ apt-build-0.12.37/apt-build	2010-08-06 21:44:26.000000000 +0200
@@ -288,11 +288,11 @@
     chdir "$src_name-$upver";
     
     # Add an entry in changelog 
-    system "debchange --append 'Built by apt-build'";
+    system "debchange 'Built by apt-build'";
     for (@{$conf->patch})
     {
 	$_ =~ s[.*/(.*)\n$][$1]gio; # basename + chomp
-		system "debchange --append 'Patched with $_'";
+		system "debchange 'Patched with $_'";
     }
 	
     # Patch if asked
@@ -310,7 +310,7 @@
 	$buildoptions = "Build options: ".
 	    $conf->Olevel." ".$conf->mtune." ".$conf->options;
 
-	system "debchange --append \"$buildoptions\"";
+	system "debchange \"$buildoptions\"";
 	
 	# Now build
 	$r = !system $conf->build_command;
