--- newt-0.52.2-old/entry.c.org	2009-02-26 23:18:30.726625269 +0100
+++ newt-0.52.2/entry.c	2009-02-26 23:24:47.424902210 +0100
@@ -184,13 +184,12 @@
     chptr = en->buf + en->firstChar;
 
     if (en->flags & NEWT_FLAG_PASSWORD) {
-	char *p;
-	tmpptr = alloca(strlen(chptr)+2);
-	strcpy(tmpptr, chptr);
-	for (p = tmpptr; *p; p++)
-	    *p = '*';
+	len = wstrlen(chptr, -1);
+	tmpptr = alloca(len+1);
+	memset(tmpptr, '*', len);
+	tmpptr[len] = '\0';
 	chptr = tmpptr;
-    }			
+    }
 
     len = wstrlen(chptr, -1);
 
