diff -ur dput-0.9.2.25.orig/debian/control dput-0.9.2.25/debian/control
--- dput-0.9.2.25.orig/debian/control	2006-06-19 22:44:27.000000000 +0200
+++ dput-0.9.2.25/debian/control	2006-07-08 14:18:48.000000000 +0200
@@ -9,7 +9,7 @@
 Package: dput
 Architecture: all
 Depends: python (>= 2.3), debconf | debconf-2.0, gnupg
-Suggests: ssh, lintian, rsync
+Suggests: ssh, lintian, rsync, yaclc
 Description: Debian package upload tool
  This script will allow you to put one or more Debian packages into the
  archive.
