From 63df78d9870382d892909a94fa3c3534c533676b Mon Sep 17 00:00:00 2001
From: Louis Bettens <louis@bettens.info>
Date: Sat, 30 Nov 2013 20:55:09 +0100
Subject: [PATCH 4/5] mass-bug: replace call to date -R with strftime()

---
 scripts/mass-bug.pl | 6 ++++--
 1 file changed, 4 insertions(+), 2 deletions(-)

diff --git a/scripts/mass-bug.pl b/scripts/mass-bug.pl
index ad0fa20..247de52 100755
--- a/scripts/mass-bug.pl
+++ b/scripts/mass-bug.pl
@@ -152,6 +152,9 @@ use strict;
 use Getopt::Long qw(:config gnu_getopt);
 use Text::Wrap;
 use File::Basename;
+use POSIX qw(locale_h strftime);
+
+setlocale(LC_TIME, "C"); # so that strftime is locale independent
 
 my $progname = basename($0);
 $Text::Wrap::columns=70;
@@ -322,8 +325,7 @@ sub mailbts {
     my ($subject, $body, $to, $from) = @_;
 
     if (defined $from) {
-	my $date = `date -R`;
-	chomp $date;
+	my $date = strftime "%a, %d %b %Y %T %z", localtime;
 
 	my $pid = open(MAIL, "|-");
 	if (! defined $pid) {
-- 
1.8.4.3

