diff -Nru devscripts-2.10.29/debian/changelog devscripts-2.10.29+nmu1/debian/changelog
--- devscripts-2.10.29/debian/changelog	2008-06-03 19:30:04.000000000 +0100
+++ devscripts-2.10.29+nmu1/debian/changelog	2008-06-17 01:26:07.000000000 +0100
@@ -1,3 +1,10 @@
+devscripts (2.10.29+nmu1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Proof of concept change to use strftime
+
+ -- Stephen Gran <sgran@debian.org>  Tue, 17 Jun 2008 01:25:57 +0100
+
 devscripts (2.10.29) unstable; urgency=low
 
   * checkbashisms:
diff -Nru devscripts-2.10.29/scripts/bts.pl devscripts-2.10.29+nmu1/scripts/bts.pl
--- devscripts-2.10.29/scripts/bts.pl	2008-05-11 21:55:09.000000000 +0100
+++ devscripts-2.10.29+nmu1/scripts/bts.pl	2008-06-17 01:32:08.000000000 +0100
@@ -40,6 +40,7 @@
 use Net::SMTP;
 use Cwd;
 use IO::Handle;
+use POSIX qw(strftime);
 use lib '/usr/share/devscripts';
 use Devscripts::DB_File_Lock;
 use Devscripts::Debbugs;
@@ -1962,6 +1963,7 @@
     my $fromaddress = $fromaddresses[0];
     # Message-ID algorithm from git-send-email
     my $msgid = sprintf("%s-%s", time(), int(rand(4200)))."-bts-$fromaddress";
+    my $date = strftime("%a, %d %b %Y %H:%M:%S %z", localtime(time));
     my $date = `date -R`;
     chomp $date;
 
diff -Nru devscripts-2.10.29/scripts/debchange.pl devscripts-2.10.29+nmu1/scripts/debchange.pl
--- devscripts-2.10.29/scripts/debchange.pl	2008-04-27 18:10:59.000000000 +0100
+++ devscripts-2.10.29+nmu1/scripts/debchange.pl	2008-06-17 01:25:28.000000000 +0100
@@ -38,6 +38,7 @@
 use File::Copy;
 use File::Basename;
 use Cwd;
+use POSIX qw(strftime tzset);
 use lib '/usr/share/devscripts';
 use Devscripts::Debbugs;
 
@@ -803,9 +804,12 @@
     }
 }
 
-# Get the date
-my $date_cmd = ($opt_tz ? "TZ=$opt_tz " : "") . "date -R";
-chomp(my $DATE=`$date_cmd`);
+if ($opt_tz) { 
+  $ENV{"TZ"} = $opt_tz; 
+  tzset();
+}
+
+my $DATE = strftime("%a, %d %b %Y %H:%M:%S %z", localtime(time));
 
 if ($opt_news && !$opt_i && !$opt_a) {
     if ($VERSION eq $changelog{'Version'}) {
diff -Nru devscripts-2.10.29/scripts/mass-bug.pl devscripts-2.10.29+nmu1/scripts/mass-bug.pl
--- devscripts-2.10.29/scripts/mass-bug.pl	2008-05-27 21:00:20.000000000 +0100
+++ devscripts-2.10.29+nmu1/scripts/mass-bug.pl	2008-06-17 01:18:31.000000000 +0100
@@ -148,6 +148,7 @@
 use Getopt::Long;
 use Text::Wrap;
 use File::Basename;
+use POSIX qw(strftime);
 
 my $progname = basename($0);
 $Text::Wrap::columns=70;
@@ -316,8 +317,7 @@
     my ($subject, $body, $to, $from) = @_;
 
     if (defined $from) {
-	my $date = `date -R`;
-	chomp $date;
+	my $date = strftime("%a, %d %b %Y %H:%M:%S %z", localtime(time));
 	
 	my $pid = open(MAIL, "|-");
 	if (! defined $pid) {
