--- pico/composer.c.orig	2009-06-28 20:46:47.000000000 +0300
+++ pico/composer.c	2009-06-28 20:49:32.000000000 +0300
@@ -794,7 +794,7 @@
 		    len += lmp->fname ? strlen(lmp->fname) : 0;
 
 		    if(len+3 > sizeof(buf)){
-			bfp = malloc(len+3);
+			/*bfp = malloc(len+3);*/
 			space = len+3;
 			if((bfp=malloc(len+3)) == NULL){
 			    emlwrite("\007Can't malloc space for filename",
@@ -1270,7 +1270,7 @@
 			len += strlen(lmp->size);
 
 			if(len+3 > sizeof(buf)){
-			    bfp = malloc(len+3);
+			    /*bfp = malloc(len+3);*/
 			    space = len+3;
 			    if((bfp=malloc(len+3)) == NULL){
 				emlwrite("\007Can't malloc space for filename",
