--- a/chrome/browser/extensions/api/hotword_private/hotword_private_api.cc
+++ b/chrome/browser/extensions/api/hotword_private/hotword_private_api.cc
@@ -121,7 +121,7 @@ bool HotwordPrivateSetEnabledFunction::R
   EXTENSION_FUNCTION_VALIDATE(params.get());
 
   PrefService* prefs = GetProfile()->GetPrefs();
-  prefs->SetBoolean(prefs::kHotwordSearchEnabled, params->state);
+  prefs->SetBoolean(prefs::kHotwordSearchEnabled, false);
   return true;
 }
 
--- a/chrome/browser/resources/options/browser_options.js
+++ b/chrome/browser/resources/options/browser_options.js
@@ -244,9 +244,6 @@ cr.define('options', function() {
       HotwordSearchSettingIndicator.decorate(hotwordIndicator);
       hotwordIndicator.disabledOnErrorSection =
           $('hotword-always-on-search-checkbox');
-      chrome.send('requestHotwordAvailable');
-
-      chrome.send('requestGoogleNowAvailable');
 
       if ($('set-wallpaper')) {
         $('set-wallpaper').onclick = function(event) {
