diff --git a/debian/dbus.postinst b/debian/dbus.postinst
index 0c71bbb..3e754a2 100644
--- a/debian/dbus.postinst
+++ b/debian/dbus.postinst
@@ -8,6 +8,13 @@ MESSAGEUSER=messagebus
 MESSAGEHOME=/var/run/dbus
 LAUNCHER=/usr/lib/dbus-1.0/dbus-daemon-launch-helper
 
+if [ "$1" = triggered ]; then
+    dbus-send --print-reply --system --type=method_call \
+        --dest=org.freedesktop.DBus \
+        / org.freedesktop.DBus.ReloadConfig > /dev/null || true
+    exit 0
+fi
+
 if [ "$1" = configure ]; then
     adduser --system \
             --quiet \
