diff -ur dash-0.5.5.1.pristine/debian/control dash-0.5.5.1/debian/control
--- dash-0.5.5.1.pristine/debian/control	2009-09-12 22:19:11.000000000 +0100
+++ dash-0.5.5.1/debian/control	2009-09-12 22:30:40.000000000 +0100
@@ -13,15 +13,13 @@
 Priority: required
 Pre-Depends: ${shlibs:Depends}
 Depends: debianutils (>= 2.15), dpkg (>= 1.15.0)
-Description: POSIX-compliant shell
- The Debian Almquist Shell (dash) is a lightweight POSIX-compliant shell
- derived from ash.
+Description: lightweight standard shell
+ The Debian Almquist Shell (dash) is a POSIX-compliant shell derived
+ from ash.
  .
- It can be usefully installed as /bin/sh (because it executes scripts
- faster than bash), or as the default shell for the superuser.  It
- depends on fewer libraries than bash, and is therefore less likely to
- be affected by an upgrade problem or a disk failure. It is also
- useful for checking the POSIX compliance of scripts.
+ Since it executes scripts faster than bash, and has fewer library
+ dependencies (making it more robust against software or hardware
+ failures), it is used as the default system shell on Debian systems.
 
 Package: ash
 Architecture: all
