Index: debmirror/debmirror
===================================================================
--- debmirror-2.15ubuntu1.orig/debmirror	2013-11-20 01:46:16.000000000 +0000
+++ debmirror-2.15ubuntu1/debmirror	2015-08-30 08:13:44.468167045 +0000
@@ -2420,6 +2420,9 @@
   foreach my $file (sort keys %i18n_get) {
     if (! check_i18n("$tempdir/$file", $i18n_get{$file}{size}, $i18n_get{$file}{sha1})) {
       remote_get("$file");
+      if (! check_file(filename => "$tempdir/$file", size => $i18n_get{$file}{size}, SHA1 => $i18n_get{$file}{sha1})) {
+        die "Checksum incorrect for upstream file: $file";
+      }
       if ($debmarshal) {
 	link_translation_into_snapshot($file,
 				       $i18n_get{$file}{dist},
