--- ./CommandNotFound.py	2013-03-26 14:53:50.611756032 +0400
+++ /usr/share/command-not-found/CommandNotFound/CommandNotFound.py	2013-03-26 14:22:39.019186821 +0400
@@ -90,7 +90,7 @@
             print _("Could not find the database of available applications, "
                     "run update-command-not-found as root to fix this")
         try:
-            self.user_can_sudo = grp.getgrnam("admin")[2] in posix.getgroups()
+            self.user_can_sudo = grp.getgrnam("sudo")[2] in posix.getgroups()
         except KeyError:
             self.user_can_sudo = False
 
