diff -ur djvulibre-3.5.10.old/debian/control djvulibre-3.5.10/debian/control
--- djvulibre-3.5.10.old/debian/control	2003-02-19 09:19:56.000000000 -0800
+++ djvulibre-3.5.10/debian/control	2003-02-19 09:12:59.000000000 -0800
@@ -1,7 +1,7 @@
 Source: djvulibre
 Priority: optional
 Maintainer: Barak Pearlmutter <bap@debian.org>
-Build-Depends: debhelper (>> 3.0.0), xlibs-dev, libqt3-mt-dev | libqt3-dev | libqt-dev, libjpeg62-dev | libjpeg-dev, chrpath
+Build-Depends: debhelper (>> 3.0.0), xlibs-dev, libqt3-mt-dev (>= 3:3.1.1-3), libjpeg62-dev | libjpeg-dev, chrpath
 Standards-Version: 3.5.8
 
 Package: libdjvulibre1
diff -ur djvulibre-3.5.10.old/debian/rules djvulibre-3.5.10/debian/rules
--- djvulibre-3.5.10.old/debian/rules	2003-02-19 09:19:56.000000000 -0800
+++ djvulibre-3.5.10/debian/rules	2003-02-19 09:10:35.000000000 -0800
@@ -30,7 +30,7 @@
 config.status: configure
 	dh_testdir
 	QT_CFLAGS=-I$(shell ls -d /usr/include/qt{3,} 2>/dev/null|head -1) \
-	QT_LIBS=-lqt					\
+	QT_LIBS=-lqt-mt					\
 	./configure --host=$(DEB_HOST_GNU_TYPE)		\
 		    --build=$(DEB_BUILD_GNU_TYPE)	\
 		    --prefix=/usr			\
