--- /usr/bin/debchange	2014-10-14 03:52:06.000000000 +0100
+++ ./debchange	2014-10-23 20:29:16.384017375 +0100
@@ -645,7 +645,8 @@
     fatal "No maintainer in changelog!"
 	unless exists $changelog->{Maintainer};
     $changelog->{Maintainer} = decode_utf8($changelog->{Maintainer});
-    ($MAINTAINER,$EMAIL) = ($changelog->{Maintainer} =~ /^([^<]+) <(.*)>/);
+    ($MAINTAINER,$EMAIL) = ($changelog->{Maintainer} =~ /^([^<]*) <(.*)>/);
+    $MAINTAINER ||= '';
     fatal "No distribution in changelog!"
 	unless exists $changelog->{Distribution};
     if ($vendor eq 'Ubuntu') {
@@ -1342,7 +1343,7 @@
 	    ($dist_indicator = $1) =~ s/[!:.,;]$//;
 	    chomp $dist_indicator;
 	}
-	elsif (/^ --\s+([^<]+)\s+/) {
+	elsif (/^ --\s+([^<]+)\s+/ || /^ --\s+<(.+?)>/) {
 	    $lastmaint=$1;
 	    # Remember where we are so we can skip back afterwards
 	    $savedline = $line;
