diff -ruNp dpkg-1.10.22.castrated/archtable dpkg-1.10.22/archtable
--- dpkg-1.10.22.castrated/archtable	2004-06-07 18:30:32.000000000 +0100
+++ dpkg-1.10.22/archtable	2004-06-07 18:32:14.000000000 +0100
@@ -63,3 +63,4 @@ i386-darwin			darwin-i386	darwin-i386
 s390x-linux-gnu			s390x		s390x
 s390x-ibm-linux-gnu		s390x		s390x
 s390x-unknown-linux-gnu		s390x		s390x
+x86_64-linux-gnu		x86-64		x86_64
diff -ruNp dpkg-1.10.22.castrated/scripts/dpkg-architecture.pl dpkg-1.10.22/scripts/dpkg-architecture.pl
--- dpkg-1.10.22.castrated/scripts/dpkg-architecture.pl	2004-06-07 18:30:40.000000000 +0100
+++ dpkg-1.10.22/scripts/dpkg-architecture.pl	2004-06-01 22:18:16.000000000 +0100
@@ -65,6 +65,7 @@ require 'controllib.pl';
 	    's390',		's390-linux',
 	    's390x',		's390x-linux',
 	    'ia64',		'ia64-linux',
+	    'x86-64',           'x86_64-linux',
 	    'openbsd-i386',	'i386-openbsd',
 	    'freebsd-i386',	'i386-freebsd',
 	    'kfreebsd-i386',	'i386-kfreebsd-gnu',
