diff -Nru dictconv-0.2/debian/changelog dictconv-0.2/debian/changelog
--- dictconv-0.2/debian/changelog	2022-04-05 12:54:21.000000000 +1000
+++ dictconv-0.2/debian/changelog	2022-04-05 12:51:27.000000000 +1000
@@ -1,3 +1,23 @@
+dictconv (0.2-7.2) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Switch to source format "3.0 (quilt)" (Closes: #1007290).
+  * Switch from 'cdbs' to the 'dh' build system.
+  * Use pkg-config to find libxml2 (Closes: #949092).
+  * debian/changelog: Trim trailing whitespace.
+  * debian/control:
+    - Use debhelper-compat v13.
+    - Build-Depend on pkg-config.
+    - Remove cdbs, autotools-dev and patchutils from the Build-Depends list.
+    - Raise Standards-Version to 4.6.0 from 3.8.0 (no changes needed).
+    - Declare Rules-Requires-Root: no.
+    - Fix grammatical errors in the package description.
+  * debian/copyright: Use DEP-5 format and update for 2022.
+  * debian/patches: Add DEP-3 metadata to fixes-for-gcc-4.3.patch.
+  * debian/watch: Use version 4.
+
+ -- Hugh McMaster <hugh.mcmaster@outlook.com>  Tue, 05 Apr 2022 12:51:27 +1000
+
 dictconv (0.2-7.1) unstable; urgency=low
 
   * Non-maintainer upload.
@@ -61,4 +81,3 @@
   * Initial release. (Closes: #435814)
 
  -- Francesco Namuri <francesco@namuri.it>  Sat, 21 Jul 2007 17:41:30 +0200
-
diff -Nru dictconv-0.2/debian/compat dictconv-0.2/debian/compat
--- dictconv-0.2/debian/compat	2022-04-05 12:54:21.000000000 +1000
+++ dictconv-0.2/debian/compat	1970-01-01 10:00:00.000000000 +1000
@@ -1 +0,0 @@
-7
diff -Nru dictconv-0.2/debian/control dictconv-0.2/debian/control
--- dictconv-0.2/debian/control	2022-04-05 12:54:21.000000000 +1000
+++ dictconv-0.2/debian/control	2022-04-05 12:50:53.000000000 +1000
@@ -2,16 +2,21 @@
 Section: utils
 Priority: optional
 Maintainer: Francesco Namuri <francesco@namuri.it>
-Build-Depends: cdbs, debhelper (>= 5), autotools-dev, libxml2-dev, zlib1g-dev, patchutils
-Standards-Version: 3.8.0
+Build-Depends:
+ debhelper-compat (= 13),
+ libxml2-dev,
+ pkg-config,
+ zlib1g-dev
+Standards-Version: 4.6.0
+Rules-Requires-Root: no
 Homepage: http://ktranslator.sourceforge.net/
 
 Package: dictconv
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}
 Recommends: dictzip
-Description: convert a dictionary file type in another dictionary file type
- Dictconv is a small program to convert a dictionary file type in another
+Description: convert a dictionary file type to another dictionary file type
+ Dictconv is a small program to convert a dictionary file type to another
  dictionary file type. Currently, it supports converting from Babylon
  glossaries, Freedict dictionaries, Sdictionary dictionaries and Stardict
  dictionaries to DICT dictionaries, plain text dictionaries and StarDict
diff -Nru dictconv-0.2/debian/copyright dictconv-0.2/debian/copyright
--- dictconv-0.2/debian/copyright	2022-04-05 12:54:21.000000000 +1000
+++ dictconv-0.2/debian/copyright	2022-04-05 12:22:24.000000000 +1000
@@ -1,37 +1,54 @@
-This package was debianized by Francesco Namuri <francesco@namuri.it> on
-Sat, 21 Jul 2007 17:41:30 +0200.
-
-It was downloaded from http://ktranslator.sourceforge.net/
-
-Upstream Author:
-
-    Raul Fernandes <rgfbr@yahoo.com.br>
-
-Copyright:
-
-    Copyright (C) 2007 Raul Fernandes
-    Copyright (C) 2007 Raul Fernandes and Karl Grill
-    Copyright (C) 1996-2001 Free Software Foundation, Inc.
-    Copyright (C) 1991 Massachusetts Institute of Technology
-
-License:
-
-    This package is free software; you can redistribute it and/or modify
-    it under the terms of the GNU General Public License as published by
-    the Free Software Foundation; either version 2 of the License, or
-    (at your option) any later version.
-
-    This package is distributed in the hope that it will be useful,
-    but WITHOUT ANY WARRANTY; without even the implied warranty of
-    MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
-    GNU General Public License for more details.
-
-    You should have received a copy of the GNU General Public License
-    along with this package; if not, write to the Free Software
-    Foundation, Inc., 51 Franklin St, Fifth Floor, Boston, MA  02110-1301 USA
-
-On Debian systems, the complete text of the GNU General
-Public License can be found in `/usr/share/common-licenses/GPL'.
-
-The Debian packaging is (C) 2007, Francesco Namuri <francesco@namuri.it> and
-is licensed under the GPL, see above.
+Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: KTranslator
+Upstream-Contact: Raul Fernandes <rgfbr@yahoo.com.br>
+Source: https://sourceforge.net/projects/ktranslator/
+
+Files: *
+Copyright: 2005-2007 Raul Fernandes
+License: GPL-2+
+
+Files: debian/*
+Copyright: 2007-2008 Franceso Namuri
+           2022 Adrian Bunk
+           2022 Hugh McMaster
+License: GPL-2+
+
+Files: install-sh
+Copyright: 1991 Massachusetts Institute of Technology
+License: MIT
+
+Files: src/babylon.cpp
+       src/babylon.h
+Copyright: 2007 Raul Fernandes and Karl Grill
+License: GPL-2+
+
+License: GPL-2+
+  This program is free software; you can redistribute it and/or modify
+  it under the terms of the GNU General Public License as published by
+  the Free Software Foundation; either version 2 of the License, or
+  (at your option) any later version.
+ .
+  This program is distributed in the hope that it will be useful,
+  but WITHOUT ANY WARRANTY; without even the implied warranty of
+  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+  GNU General Public License for more details.
+ .
+  You should have received a copy of the GNU General Public License
+  along with this program;  if not, write to the Free Software
+  Foundation, Inc., 51 Franklin St, Fifth Floor, Boston,
+  MA 02110-1301, USA.
+ .
+  On Debian GNU/Linux systems, the complete text of the GNU General
+  Public License version 2 can be found in
+  `/usr/share/common-licenses/GPL-2'.
+
+License: MIT
+  Permission to use, copy, modify, distribute, and sell this software and its
+  documentation for any purpose is hereby granted without fee, provided that
+  the above copyright notice appear in all copies and that both that
+  copyright notice and this permission notice appear in supporting
+  documentation, and that the name of M.I.T. not be used in advertising or
+  publicity pertaining to distribution of the software without specific,
+  written prior permission.  M.I.T. makes no representations about the
+  suitability of this software for any purpose.  It is provided "as is"
+  without express or implied warranty.
diff -Nru dictconv-0.2/debian/manpages dictconv-0.2/debian/manpages
--- dictconv-0.2/debian/manpages	1970-01-01 10:00:00.000000000 +1000
+++ dictconv-0.2/debian/manpages	2022-04-04 22:36:03.000000000 +1000
@@ -0,0 +1 @@
+debian/dictconv.1
diff -Nru dictconv-0.2/debian/patches/fixes-for-gcc-4.3.patch dictconv-0.2/debian/patches/fixes-for-gcc-4.3.patch
--- dictconv-0.2/debian/patches/fixes-for-gcc-4.3.patch	2022-04-05 12:54:21.000000000 +1000
+++ dictconv-0.2/debian/patches/fixes-for-gcc-4.3.patch	2022-04-04 23:06:12.000000000 +1000
@@ -1,6 +1,11 @@
-diff -Nur -x '*.orig' -x '*~' dictconv-0.2/src/babylon.cpp dictconv-0.2.new/src/babylon.cpp
---- dictconv-0.2/src/babylon.cpp	2007-03-02 16:43:00.000000000 +0100
-+++ dictconv-0.2.new/src/babylon.cpp	2008-06-19 14:45:55.000000000 +0200
+Description: Include standard headers to declare memcmp and memset
+Author: Francesco Namuri <francesco@namuri.it>
+Bug-Debian: https://bugs.debian.org/486954
+Forwarded: not-needed
+Last-Update: 2008-06-19
+
+--- a/src/babylon.cpp
++++ b/src/babylon.cpp
 @@ -22,6 +22,7 @@
  
  #include<stdlib.h>
@@ -9,9 +14,8 @@
  
  #include <iconv.h>
  
-diff -Nur -x '*.orig' -x '*~' dictconv-0.2/src/dictdbuilder.cpp dictconv-0.2.new/src/dictdbuilder.cpp
---- dictconv-0.2/src/dictdbuilder.cpp	2007-02-25 15:33:33.000000000 +0100
-+++ dictconv-0.2.new/src/dictdbuilder.cpp	2008-06-19 14:47:55.000000000 +0200
+--- a/src/dictdbuilder.cpp
++++ b/src/dictdbuilder.cpp
 @@ -19,7 +19,7 @@
   ***************************************************************************/
  
@@ -21,9 +25,8 @@
  #include <iostream>
  
  DictdBuilder::DictdBuilder( std::string filename )
-diff -Nur -x '*.orig' -x '*~' dictconv-0.2/src/dictdbuilder.h dictconv-0.2.new/src/dictdbuilder.h
---- dictconv-0.2/src/dictdbuilder.h	2007-02-25 15:33:33.000000000 +0100
-+++ dictconv-0.2.new/src/dictdbuilder.h	2008-06-19 14:51:19.000000000 +0200
+--- a/src/dictdbuilder.h
++++ b/src/dictdbuilder.h
 @@ -22,6 +22,8 @@
  
  #include "dictbuilder.h"
@@ -33,9 +36,8 @@
  #include <map>
  #include <fstream>
  
-diff -Nur -x '*.orig' -x '*~' dictconv-0.2/src/plaintextdictbuilder.h dictconv-0.2.new/src/plaintextdictbuilder.h
---- dictconv-0.2/src/plaintextdictbuilder.h	2007-02-19 18:22:48.000000000 +0100
-+++ dictconv-0.2.new/src/plaintextdictbuilder.h	2008-06-19 14:51:13.000000000 +0200
+--- a/src/plaintextdictbuilder.h
++++ b/src/plaintextdictbuilder.h
 @@ -23,6 +23,8 @@
  
  #include "dictbuilder.h"
@@ -45,9 +47,8 @@
  #include <fstream>
  #include <map>
  
-diff -Nur -x '*.orig' -x '*~' dictconv-0.2/src/sdict.h dictconv-0.2.new/src/sdict.h
---- dictconv-0.2/src/sdict.h	2007-02-17 15:48:04.000000000 +0100
-+++ dictconv-0.2.new/src/sdict.h	2008-06-19 14:52:26.000000000 +0200
+--- a/src/sdict.h
++++ b/src/sdict.h
 @@ -24,6 +24,8 @@
  #include <fstream>
  #include <vector>
@@ -57,9 +58,8 @@
  using namespace std;
  
  typedef unsigned char uchar;
-diff -Nur -x '*.orig' -x '*~' dictconv-0.2/src/stardictbuilder.h dictconv-0.2.new/src/stardictbuilder.h
---- dictconv-0.2/src/stardictbuilder.h	2007-02-17 15:48:04.000000000 +0100
-+++ dictconv-0.2.new/src/stardictbuilder.h	2008-06-19 14:57:33.000000000 +0200
+--- a/src/stardictbuilder.h
++++ b/src/stardictbuilder.h
 @@ -25,6 +25,8 @@
  #include <map>
  #include <fstream>
@@ -69,9 +69,8 @@
  class StarDictBuilder : public DictBuilder
  {
  
-diff -Nur -x '*.orig' -x '*~' dictconv-0.2/src/stardict.cpp dictconv-0.2.new/src/stardict.cpp
---- dictconv-0.2/src/stardict.cpp	2007-02-17 15:48:04.000000000 +0100
-+++ dictconv-0.2.new/src/stardict.cpp	2008-06-19 14:55:59.000000000 +0200
+--- a/src/stardict.cpp
++++ b/src/stardict.cpp
 @@ -21,6 +21,11 @@
  
  #include <zlib.h>
@@ -84,9 +83,8 @@
  #define CHUNK 0xffffL
  
  StarDict::StarDict( const char *filename )
-diff -Nur -x '*.orig' -x '*~' dictconv-0.2/src/stardict.h dictconv-0.2.new/src/stardict.h
---- dictconv-0.2/src/stardict.h	2007-02-17 15:48:04.000000000 +0100
-+++ dictconv-0.2.new/src/stardict.h	2008-06-19 14:53:42.000000000 +0200
+--- a/src/stardict.h
++++ b/src/stardict.h
 @@ -24,6 +24,8 @@
  #include <fstream>
  #include <vector>
diff -Nru dictconv-0.2/debian/patches/pkg-config.patch dictconv-0.2/debian/patches/pkg-config.patch
--- dictconv-0.2/debian/patches/pkg-config.patch	1970-01-01 10:00:00.000000000 +1000
+++ dictconv-0.2/debian/patches/pkg-config.patch	2022-04-04 22:51:34.000000000 +1000
@@ -0,0 +1,29 @@
+Description: Use pkg-config to find libxml2
+Author: Hugh McMaster <hugh.mcmaster@outlook.com>
+Bug-Debian: https://bugs.debian.org/949092
+Forwarded: not-needed
+Last-Update: 2022-04-04
+
+--- a/configure.in
++++ b/configure.in
+@@ -7,6 +7,8 @@
+ AC_PROG_CXX
+ AM_PROG_LIBTOOL
+ 
+-AM_PATH_XML2(2.5.0, , AC_MSG_ERROR([You must have libxml2 >= 2.5.0 installed]))
++PKG_PROG_PKG_CONFIG
++
++PKG_CHECK_MODULES([XML], [libxml-2.0])
+ 
+ AC_OUTPUT(Makefile src/Makefile)
+--- a/src/Makefile.am
++++ b/src/Makefile.am
+@@ -2,7 +2,7 @@
+ dictconv_SOURCES = dictconv.cpp babylon.cpp babylonreader.cpp dictdbuilder.cpp freedictreader.cpp plaintextdictbuilder.cpp sdict.cpp sdictreader.cpp stardict.cpp stardictreader.cpp stardictbuilder.cpp
+ 
+ # set the include path found by configure
+-INCLUDES= $(all_includes) @XML_CPPFLAGS@
++INCLUDES= $(all_includes) @XML_CFLAGS@
+ 
+ # the library search path.
+ dictconv_LDFLAGS = $(all_libraries) -lz @XML_LIBS@
diff -Nru dictconv-0.2/debian/patches/series dictconv-0.2/debian/patches/series
--- dictconv-0.2/debian/patches/series	1970-01-01 10:00:00.000000000 +1000
+++ dictconv-0.2/debian/patches/series	2022-04-04 22:44:31.000000000 +1000
@@ -0,0 +1,2 @@
+pkg-config.patch
+fixes-for-gcc-4.3.patch
diff -Nru dictconv-0.2/debian/rules dictconv-0.2/debian/rules
--- dictconv-0.2/debian/rules	2022-04-05 12:54:21.000000000 +1000
+++ dictconv-0.2/debian/rules	2022-04-04 22:25:00.000000000 +1000
@@ -1,5 +1,4 @@
 #!/usr/bin/make -f
-DEB_INSTALL_MANPAGES_dictconv := debian/dictconv.1
-include /usr/share/cdbs/1/rules/debhelper.mk
-include /usr/share/cdbs/1/class/autotools.mk
-include /usr/share/cdbs/1/rules/simple-patchsys.mk
+
+%:
+	dh $@
diff -Nru dictconv-0.2/debian/source/format dictconv-0.2/debian/source/format
--- dictconv-0.2/debian/source/format	1970-01-01 10:00:00.000000000 +1000
+++ dictconv-0.2/debian/source/format	2022-04-04 21:56:24.000000000 +1000
@@ -0,0 +1 @@
+3.0 (quilt)
diff -Nru dictconv-0.2/debian/watch dictconv-0.2/debian/watch
--- dictconv-0.2/debian/watch	2022-04-05 12:54:21.000000000 +1000
+++ dictconv-0.2/debian/watch	2022-04-04 22:26:40.000000000 +1000
@@ -1,2 +1,2 @@
-version=3
+version=4
 http://sf.net/ktranslator/dictconv-(.*)\.tar\.bz2
