Index: source/fileitem.cc
===================================================================
--- source/fileitem.cc	(Revision 4012)
+++ source/fileitem.cc	(Arbeitskopie)
@@ -781,7 +781,7 @@
 				if (it->second.second.get()->m_nTimeDlDone < extime)
 				{
 					// try to find its sibling which is in good state?
-					for (; it->first == p->m_sPathRel; ++it)
+					for (; it!=mapItems.end() && it->first == p->m_sPathRel; ++it)
 					{
 						if (it->second.second.get()->m_nTimeDlDone >= extime)
 						{
