--- a/debian/skel.bashrc	2012-07-01 05:45:23.000000000 -0700
+++ b/debian/skel.bashrc	2013-07-06 15:38:39.717449740 -0700
@@ -63,13 +63,14 @@
 fi
 unset color_prompt force_color_prompt
 
-# If this is an xterm set the title to user@host:dir
+# Set the terminal title on supported terminals
 case "$TERM" in
+screen*)
+    PS1="\[\ek${debian_chroot:+($debian_chroot)}\u@\h:\w\e\\\\\]$PS1"
+    ;&
 xterm*|rxvt*)
     PS1="\[\e]0;${debian_chroot:+($debian_chroot)}\u@\h: \w\a\]$PS1"
     ;;
-*)
-    ;;
 esac
 
 # enable color support of ls and also add handy aliases
