--- grub-kfreebsd-cdrom.cfg.orig	2011-12-10 20:45:22.000000000 +0000
+++ grub-kfreebsd-cdrom.cfg	2011-12-10 20:47:36.000000000 +0000
@@ -31,41 +31,43 @@ menuentry "" {
 }
 
 function menuentry_set {
-	menuentry "Default install$desc_suffix" {
+	menuentry "Default install$desc_suffix" "$1" {
 		echo "Loading ..."
 		kfreebsd $kernel_path
-		kfreebsd_module $mfsroot_path type=mfs_root
+		kfreebsd_module "$2" type=mfs_root
 	}
 
-	menuentry "Automated install$desc_suffix" {
+	menuentry "Automated install$desc_suffix" "$1" {
 		echo "Loading ..."
 		kfreebsd $kernel_path
-		kfreebsd_module $mfsroot_path type=mfs_root
+		kfreebsd_module "$2" type=mfs_root
 		set kFreeBSD.auto=true
 		set kFreeBSD.priority=critical
 	}
 
-	menuentry "Expert install$desc_suffix" {
+	menuentry "Expert install$desc_suffix" "$1" {
 		echo "Loading ..."
 		kfreebsd $kernel_path
-		kfreebsd_module $mfsroot_path type=mfs_root
+		kfreebsd_module "$2" type=mfs_root
 		set kFreeBSD.priority=low
 	}
 }
 
+insmod test
+
 # Default kernel (8.x)
 if test -e /boot/kernel/kfreebsd.gz ; then
 	set kernel_path="/boot/kernel/kfreebsd.gz"
 
 	set mfsroot_path="/boot/mfsroot.gz"
 	set desc_suffix=""
-	menuentry_set
+	menuentry_set $mfsroot_path
 
 	if test -e /boot/gtk/mfsroot.gz ; then
 		menuentry "" { true }
 		set mfsroot_path="/boot/gtk/mfsroot.gz"
 		set desc_suffix=" (graphical mode)"
-		menuentry_set
+		menuentry_set $mfsroot_path
 	fi
 fi
 
@@ -79,13 +81,13 @@ if test -e /boot/kernel/kfreebsd-9.gz ;
 	set kernel_path="/boot/kernel/kfreebsd-9.gz"
 	set mfsroot_path="/boot/mfsroot.gz"
 	set desc_suffix=" (using kernel of FreeBSD 9)"
-	menuentry_set
+	menuentry_set $mfsroot_path
 
 	if test -e /boot/gtk/mfsroot.gz ; then
 		menuentry "" { true }
 		set mfsroot_path="/boot/gtk/mfsroot.gz"
 		set desc_suffix=" (graphical mode with kernel of FreeBSD 9)"
-		menuentry_set
+		menuentry_set $mfsroot_path
 	fi
 fi
 
