diff -Nru assaultcube-1.1.0.4+dfsg2/debian/autoreconf assaultcube-1.1.0.4+dfsg2/debian/autoreconf
--- assaultcube-1.1.0.4+dfsg2/debian/autoreconf	1970-01-01 00:00:00.000000000 +0000
+++ assaultcube-1.1.0.4+dfsg2/debian/autoreconf	2014-10-29 12:03:10.000000000 +0000
@@ -0,0 +1 @@
+source/enet
\ No newline at end of file
diff -Nru assaultcube-1.1.0.4+dfsg2/debian/changelog assaultcube-1.1.0.4+dfsg2/debian/changelog
--- assaultcube-1.1.0.4+dfsg2/debian/changelog	2011-10-30 14:00:26.000000000 +0000
+++ assaultcube-1.1.0.4+dfsg2/debian/changelog	2014-10-29 11:32:05.000000000 +0000
@@ -1,3 +1,10 @@
+assaultcube (1.1.0.4+dfsg2-1.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * dh-autoreconf to support new architectures (Closes: 764152)
+
+ -- Wookey <wookey@debian.org>  Wed, 29 Oct 2014 05:01:52 +0000
+
 assaultcube (1.1.0.4+dfsg2-1) unstable; urgency=low
 
   * Correct license in lintian override comment
diff -Nru assaultcube-1.1.0.4+dfsg2/debian/control assaultcube-1.1.0.4+dfsg2/debian/control
--- assaultcube-1.1.0.4+dfsg2/debian/control	2011-10-30 13:51:20.000000000 +0000
+++ assaultcube-1.1.0.4+dfsg2/debian/control	2014-10-29 11:54:19.000000000 +0000
@@ -3,7 +3,7 @@
 Priority: optional
 Maintainer: Debian Games Team <pkg-games-devel@lists.alioth.debian.org>
 Uploaders: Martin Erik Werner <martinerikwerner@gmail.com>
-Build-Depends: debhelper (>= 7.0.50~), autotools-dev, libsdl-image1.2-dev,
+Build-Depends: debhelper (>= 7.0.50~), dh-autoreconf, libsdl-image1.2-dev,
  libsdl1.2-dev, libgl1-mesa-dev, zlib1g-dev, libopenal-dev, libvorbis-dev
 Standards-Version: 3.9.2
 Vcs-Svn: svn://svn.debian.org/svn/pkg-games/non-free/packages/trunk/assaultcube/
diff -Nru assaultcube-1.1.0.4+dfsg2/debian/patches/20-fix-clean-target.patch assaultcube-1.1.0.4+dfsg2/debian/patches/20-fix-clean-target.patch
--- assaultcube-1.1.0.4+dfsg2/debian/patches/20-fix-clean-target.patch	2011-10-30 13:50:05.000000000 +0000
+++ assaultcube-1.1.0.4+dfsg2/debian/patches/20-fix-clean-target.patch	2014-10-29 11:36:49.000000000 +0000
@@ -7,17 +7,30 @@
 Forwarded: https://sourceforge.net/tracker/?func=detail&aid=3430222&group_id=123597&atid=697091
 Last-Update: 2011-10-28
 
---- a/source/src/Makefile
-+++ b/source/src/Makefile
-@@ -128,9 +128,9 @@
+Index: assaultcube-1.1.0.4+dfsg2/source/src/Makefile
+===================================================================
+--- assaultcube-1.1.0.4+dfsg2.orig/source/src/Makefile	2014-10-29 05:01:42.000000000 +0000
++++ assaultcube-1.1.0.4+dfsg2/source/src/Makefile	2014-10-29 11:36:30.115453061 +0000
+@@ -127,10 +127,10 @@
+ libenet: ../enet/Makefile
  	-$(MAKE) -C ../enet all
  
- clean-enet: ../enet/Makefile
+-clean-enet: ../enet/Makefile
 -	$(MAKE) -C ../enet/ clean
-+	$(MAKE) -C ../enet/ distclean
++clean-enet:
++	-$(MAKE) -C ../enet/ distclean
  
 -clean:
 +clean:	clean-enet
  	-$(RM) $(CLIENT_PCH) $(CLIENT_OBJS) $(SERVER_OBJS) $(MASTER_OBJS) ac_client ac_server ac_master
  #	-$(MAKE) -C ../enet/ clean
  
+@@ -172,7 +172,7 @@
+ 
+ master: libenet $(MASTER_OBJS)
+ 	$(CXX) $(CXXFLAGS) -o ac_master $(MASTER_OBJS) $(SERVER_LIBS)  
+-	
++
+ client_install: client
+ 	install -D ac_client ../../bin_unix/$(PLATFORM_PREFIX)_client
+ #ifneq (,$(STRIP))
diff -Nru assaultcube-1.1.0.4+dfsg2/debian/rules assaultcube-1.1.0.4+dfsg2/debian/rules
--- assaultcube-1.1.0.4+dfsg2/debian/rules	2011-10-28 12:45:04.000000000 +0000
+++ assaultcube-1.1.0.4+dfsg2/debian/rules	2014-10-29 11:55:38.000000000 +0000
@@ -6,7 +6,7 @@
 	dh_auto_clean
 
 %:
-	dh $@ --sourcedirectory=source/src
+	dh $@ --with autoreconf --sourcedirectory=source/src
 
 VER=$(shell dpkg-parsechangelog | sed -rne 's,^Version: ([^-+]+).*,\1,p')
 DEB_VER=$(shell dpkg-parsechangelog | sed -rne 's,^Version: ([^-]+).*,\1,p')
