Index: pkgs.dbk
===================================================================
--- pkgs.dbk	(revision 9236)
+++ pkgs.dbk	(working copy)
@@ -2388,7 +2388,8 @@
 the urgency is sticky, meaning that the highest urgency uploaded since the
 previous <literal>testing</literal> transition is taken into account.  Those
 delays may be doubled during a freeze, or <literal>testing</literal>
-transitions may be switched off altogether;
+transitions may be switched off altogether. The Debian Release team can override
+the urgency;
 </para>
 </listitem>
 <listitem>
