From 21d4d6439944b8dcca45cbc6981f0ee4139ee353 Mon Sep 17 00:00:00 2001
From: Guo Yixuan <culu.gyx@gmail.com>
Date: Wed, 13 Aug 2014 19:25:01 -0400
Subject: [PATCH 1/2] the patch

---
 debian/askpass.c | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/askpass.c b/debian/askpass.c
index 9ba139b..4ff8796 100644
--- a/debian/askpass.c
+++ b/debian/askpass.c
@@ -477,6 +477,7 @@ main(int argc, char **argv, char **envp)
 		disable_method(NULL);
 		exit(EXIT_FAILURE);
 	}
+        disable_method(NULL);
 	exit(EXIT_SUCCESS);
 }
 
-- 
2.1.0

