Index: basic_defs.sgml
===================================================================
--- basic_defs.sgml	(Revision 10498)
+++ basic_defs.sgml	(Arbeitskopie)
@@ -141,7 +141,7 @@
    very strict about shipping truly free software.  The guidelines used to
    determine if a work is "free" are provided in <url
    id="http://www.debian.org/social_contract#guidelines" name="The Debian Free
-   Software">.
+   Software Guidelines (DFSG)">.
 
 <tag/The Debian package maintenance system:/
   <item>The entire system, or any individual component of it, can be
