From 5961a805b5128e93fb4889347142139ead142c23 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?R=C3=A9mi=20Vanicat?= <vanicat@debian.org>
Date: Sun, 30 Oct 2016 19:05:45 +0100
Subject: [PATCH] advising indent-region with correct prototype

---
 develock.el | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/develock.el b/develock.el
index d7a107f..890cba4 100644
--- a/develock.el
+++ b/develock.el
@@ -1791,7 +1791,7 @@ IGNORE-POINT-POS are ignored."
 
 (let (current-load-list)
   (defadvice indent-region (around remove-useless-whitespace
-				   (start end column) activate)
+				   (start end &optional column) activate)
     "Advised by Develock.
 If Develock is on, remove useless leading and trailing whitespace in
 Lisp modes, C modes and Java mode.  You can turn off this advice
-- 
2.10.2

