diff --git i/bin/blhc w/bin/blhc
index fba278f..d6e2690 100755
--- i/bin/blhc
+++ w/bin/blhc
@@ -53,7 +53,7 @@ my $cc_regex_normal = qr/
 my $warning_regex = qr/^(.+?):(\d+):\d+: warning: (.+?) \[(.+?)\]$/;
 # Regex to catch libtool commands and not lines which show commands executed
 # by libtool (e.g. libtool: link: ...).
-my $libtool_regex = qr/\blibtool\s.*--mode=/;
+my $libtool_regex = qr/\blibtool["']?\s.*--mode=/;
 my $libtool_link_regex = qr/\blibtool: link: /;
 
 # List of source file extensions which require preprocessing.
