From cc36b2f690d694ad0cfe1b1c0ca5df7ead3e57ca Mon Sep 17 00:00:00 2001
From: Mark Hindley <mark@hindley.org.uk>
Date: Wed, 30 Nov 2022 11:30:21 +0000
Subject: [PATCH] Add encoded underscores to *_files_regexp. Test fix for
 #1025131.

---
 lib/apt-cacher.pl | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/lib/apt-cacher.pl b/lib/apt-cacher.pl
index 712d0bc..dad985a 100755
--- a/lib/apt-cacher.pl
+++ b/lib/apt-cacher.pl
@@ -132,7 +132,7 @@ sub read_config {
 							  qw(vmlinuz
 							     linux
 							     initrd\.gz
-							     (?:%VALID_PACKAGE_NAME%_%VALID_VERSION%[_\.])?changelog
+							     (?:%VALID_PACKAGE_NAME%(?:_|%5f)%VALID_VERSION%[_\.])?changelog
 							     NEWS\.Debian
 							     %VALID_UBUNTU_RELEASE_NAMES%\.tar\.gz(?:\.gpg)?
 							     (?:by-hash/(?i:MD5SUM/[0-9a-f]{32}|SHA1/[0-9a-f]{40}|SHA256/[0-9a-f]{64}))
@@ -141,7 +141,7 @@ sub read_config {
 							   )
 							       ) . ')$',
 		  package_files_regexp => '(?:' . join('|',
-						       qw((?:^|/)%VALID_PACKAGE_NAME%_%VALID_VERSION%(?:_%VALID_ARCHS%\.(?:u|d)?deb|\.dsc|\.tar\.(?:gz|bz2|xz|lzma)(?:\.asc)?|\.diff\.gz)
+						       qw((?:^|/)%VALID_PACKAGE_NAME%(?:_|%5f)%VALID_VERSION%(?:(?:_|%5f)%VALID_ARCHS%\.(?:u|d)?deb|\.dsc|\.tar\.(?:gz|bz2|xz|lzma)(?:\.asc)?|\.diff\.gz)
 							  \.rpm
 							  index\.db-.+\.gz
 							  \.jigdo
-- 
2.35.1

