diff -Nru /tmp/huBvPfBcak/debiandoc-sgml-1.2.7/debian/control /tmp/jfJeVaDTwP/debiandoc-sgml-1.2.7.1/debian/control
--- /tmp/huBvPfBcak/debiandoc-sgml-1.2.7/debian/control	2007-08-25 07:37:33.000000000 +0200
+++ /tmp/jfJeVaDTwP/debiandoc-sgml-1.2.7.1/debian/control	2008-01-09 10:32:22.000000000 +0100
@@ -12,7 +12,7 @@
 Priority: optional
 Architecture: all
 Depends: ${perl:Depends}, libhtml-parser-perl, libi18n-langtags-perl, libroman-perl, libsgmls-perl, libtext-format-perl, liburi-perl, sgml-data, sgmlspl, sp, ${misc:Depends}
-Recommends: libpaperg, texlive-latex-recommended, texlive-latex-extra, texlive-fonts-recommended, texinfo, gs-gpl | gs 
+Recommends: libpaperg, texlive-latex-recommended, texlive-latex-base, texlive-latex-extra, texlive-fonts-recommended, texinfo, gs-gpl | gs 
 Suggests: debiandoc-sgml-doc, latex-cjk-all, hlatex, littex
 Description: DebianDoc SGML DTD and formatting tools
  This is an SGML-based documentation formatting package used for the
@@ -21,7 +21,8 @@
  Info (via Texinfo), DebianDoc SGML, DocBook XML, plain text (with
  overstrikes a la troff as well as without) files.
  .
- The LaTeX based output needs the libpaperg, tetex-bin and tetex-extra
+ The LaTeX based output needs the libpaperg, texlive-latex-recommended,
+ texlive-latex-base, texlive-latex-extra and texlive-fonts-recommended
  packages.  In addition to this, Chinese and Japanese need cjk-latex and 
  corresponding font packages while Korean needs hlatex and corresponding 
  font packages. The Texinfo based output needs the texinfo package. 
diff -Nru /tmp/huBvPfBcak/debiandoc-sgml-1.2.7/tools/bin/template /tmp/jfJeVaDTwP/debiandoc-sgml-1.2.7.1/tools/bin/template
--- /tmp/huBvPfBcak/debiandoc-sgml-1.2.7/tools/bin/template	2007-08-25 07:35:02.000000000 +0200
+++ /tmp/jfJeVaDTwP/debiandoc-sgml-1.2.7.1/tools/bin/template	2007-12-04 19:04:15.000000000 +0100
@@ -131,7 +131,7 @@
 if ! which latex >/dev/null 2>&1
 then
     echo >&2 "${script}: LaTeX typesetting system not found"
-    echo >&2 "${script}: please install the package 'tetex-bin'"
+    echo >&2 "${script}: please install the package 'texlive-latex-base'"
     exit 2
 fi
 
@@ -142,7 +142,7 @@
 if ! which pdflatex >/dev/null 2>&1
 then
     echo >&2 "${script}: LaTeX typesetting system not found"
-    echo >&2 "${script}: please install the package 'tetex-bin'"
+    echo >&2 "${script}: please install the package 'texlive-latex-base'"
     exit 2
 fi
 
@@ -152,20 +152,44 @@
 ## check for presence of used latex styles
 if ! kpsewhich \
     fancyhdr.sty \
-    footmisc.sty \
     helvet.sty \
     hyperref.sty \
-    paralist.sty \
     palatino.sty \
     pifont.sty \
     times.sty \
     url.sty \
+    >/dev/null 2>&1
+then
+    echo >&2 "${script}: one or more used LaTeX typesetting styles not found"
+    echo >&2 "${script}: please install the package 'texlive-latex-base'"
+    exit 2
+fi
+
+@@@end-latexdvi-latexps-latexpdf-active@@@
+@@@start-latexdvi-latexps-latexpdf-active@@@
+## ----------------------------------------------------------------------
+## check for presence of used latex styles
+if ! kpsewhich \
+    footmisc.sty \
+    paralist.sty \
     vmargin.sty \
+    >/dev/null 2>&1
+then
+    echo >&2 "${script}: one or more used LaTeX typesetting styles not found"
+    echo >&2 "${script}: please install the package 'texlive-latex-extra'"
+    exit 2
+fi
+
+@@@end-latexdvi-latexps-latexpdf-active@@@
+@@@start-latexdvi-latexps-latexpdf-active@@@
+## ----------------------------------------------------------------------
+## check for presence of used latex styles
+if ! kpsewhich \
     wasysym.sty \
     >/dev/null 2>&1
 then
     echo >&2 "${script}: one or more used LaTeX typesetting styles not found"
-    echo >&2 "${script}: please install the package 'tetex-extra'"
+    echo >&2 "${script}: please install the package 'texlive-fonts-recommended'"
     exit 2
 fi
 
