diff -u elinks-0.12~pre5/debian/rules elinks-0.12~pre5/debian/rules
--- elinks-0.12~pre5/debian/rules
+++ elinks-0.12~pre5/debian/rules
@@ -17,9 +17,14 @@
 DEB_HOST_GNU_TYPE   ?= $(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)
 DEB_BUILD_GNU_TYPE  ?= $(shell dpkg-architecture -qDEB_BUILD_GNU_TYPE)
 
+# Use hardening flags.
+export DEB_BUILD_MAINT_OPTIONS = hardening=+all
+
 CFLAGS_COMMON=-g
 CFLAGS_COMMON += `dpkg-buildflags --get CFLAGS`
 
+CPPFLAGS_COMMON = `dpkg-buildflags --get CPPFLAGS`
+
 LDFLAGS_COMMON="-Wl,-z,defs"
 LDFLAGS_COMMON += `dpkg-buildflags --get LDFLAGS`
 
@@ -129,11 +134,11 @@
 build-arch-stamp: patch-stamp save-stamp
 	mkdir $(CURDIR)/build-main && cd $(CURDIR)/build-main && \
 		$(CURDIR)/configure --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE) \
-		$(confopts_main) CFLAGS="$(CFLAGS_COMMON)" LDFLAGS="$(LDFLAGS_COMMON)"
+		$(confopts_main) CFLAGS="$(CFLAGS_COMMON)" CPPFLAGS="$(CPPFLAGS_COMMON)" LDFLAGS="$(LDFLAGS_COMMON)"
 	$(MAKE) -C $(CURDIR)/build-main
 	mkdir $(CURDIR)/build-lite && cd $(CURDIR)/build-lite && \
 		$(CURDIR)/configure --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE) \
-		$(confopts_lite) CFLAGS="$(CFLAGS_COMMON) $(CFLAGS_LITE)" LDFLAGS="$(LDFLAGS_COMMON)"
+		$(confopts_lite) CFLAGS="$(CFLAGS_COMMON) $(CFLAGS_LITE)" CPPFLAGS="$(CPPFLAGS_COMMON)" LDFLAGS="$(LDFLAGS_COMMON)"
 	$(MAKE) -C $(CURDIR)/build-lite
 	touch $@
 
