diff -u abcde-2.6/debian/changelog abcde-2.6/debian/changelog
--- abcde-2.6/debian/changelog
+++ abcde-2.6/debian/changelog
@@ -1,3 +1,13 @@
+abcde (2.6-1.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Invoke eyeD3 with --set-encoding/--set-text-frame instead of
+    --encoding/--text-frame since Debian's version of eyeD3 is pre 0.7. Also
+    version the Suggests on eyed3 (<< 0.7~). (Patches by Samuel Thibault)
+    Closes: #772778
+
+ -- Andreas Metzler <ametzler@debian.org>  Sun, 11 Jan 2015 17:52:39 +0100
+
 abcde (2.6-1) unstable; urgency=medium
 
   * New upstream release with lots of bug fixes:
diff -u abcde-2.6/debian/control abcde-2.6/debian/control
--- abcde-2.6/debian/control
+++ abcde-2.6/debian/control
@@ -12,7 +12,8 @@
 Architecture: all
 Depends: ${misc:Depends}, cd-discid, wget, cdparanoia | icedax, vorbis-tools (>= 1.0beta4-1) | lame | flac | bladeenc | speex | musepack-tools | opus-tools
 Recommends: vorbis-tools, libmusicbrainz-discid-perl, libwebservice-musicbrainz-perl, libdigest-sha-perl, bsd-mailx
-Suggests: eject, distmp3, id3 (>= 0.12), id3v2, eyed3, normalize-audio, vorbisgain, mkcue, mp3gain, atomicparsley
+Suggests: eject, distmp3, id3 (>= 0.12), id3v2, eyed3 (<< 0.7~),
+ normalize-audio, vorbisgain, mkcue, mp3gain, atomicparsley
 Description: A Better CD Encoder
  frontend program to cdparanoia, wget, cd-discid, id3, and your favorite
  Ogg/Vorbis, MP3, FLAC, Ogg/Speex and/or MPP/MP+(Musepack) encoder (defaults 
only in patch2:
unchanged:
--- abcde-2.6.orig/abcde
+++ abcde-2.6/abcde
@@ -835,8 +835,8 @@
 						-a "$TRACKARTIST" -t "$TRACKNAME" ${CDYEAR:+-Y "$CDYEAR"} \
 						-G "$GENREID" -n "${TRACKNUM:-$1}" \
 						${TRACKNUM:+-N "$TRACKS"} \
-						${ENCODING:+--encoding="$ENCODING"} \
-						${TPE2:+--text-frame=TPE2:"$TPE2"} \
+						${ENCODING:+--set-encoding="$ENCODING"} \
+						${TPE2:+--set-text-frame=TPE2:"$TPE2"} \
 						"$ABCDETEMPDIR/track$1.$OUTPUT"
 					;;
 				# FIXME # Still not activated...
@@ -3324,7 +3324,7 @@
 # FIXME # Older versions of eyeD3 (< 0.7.0) expect --set-encoding=utf16-LE
 # so perhaps some version sniffing would be useful. Or perhaps it might be
 # better to simply cut ties with the older eyeD3... Andrew.
-EYED3OPTS="--encoding utf16"
+EYED3OPTS="--set-encoding utf16"
 CDPARANOIAOPTS=
 CDDA2WAVOPTS=
 DAGRABOPTS=
