Index: augeas-0.7.2/tests/test-save.c
===================================================================
--- augeas-0.7.2.orig/tests/test-save.c	2014-01-15 11:36:05.631506034 +0100
+++ augeas-0.7.2/tests/test-save.c	2014-01-15 12:20:41.804406967 +0100
@@ -30,6 +30,10 @@
 #include <sys/types.h>
 #include <sys/wait.h>
 
+#define CuAssertPositive(tc, n) CuAssertTrue(tc, (n) > 0)
+#define CuAssertZero(tc, n) CuAssertTrue(tc, (n) == 0)
+#define CuAssertRetSuccess(tc, n) CuAssertTrue(tc, (n) == 0)
+
 const char *abs_top_srcdir;
 const char *abs_top_builddir;
 char *root = NULL, *src_root = NULL;
