diff -ur adduser-3.111.orig/debian/config adduser-3.111/debian/config
--- adduser-3.111.orig/debian/config	2009-12-10 13:09:43.000000000 +0100
+++ adduser-3.111/debian/config	2009-12-10 13:13:00.000000000 +0100
@@ -6,7 +6,7 @@
 
 db_version 2.0
 db_capb
-db_title Adduser
+db_settitle adduser/title
 
 # For testing
 #db_fset adduser/homedir-permission isdefault true
diff -ur adduser-3.111.orig/debian/templates adduser-3.111/debian/templates
--- adduser-3.111.orig/debian/templates	2009-12-10 13:09:43.000000000 +0100
+++ adduser-3.111/debian/templates	2009-12-10 13:12:12.000000000 +0100
@@ -7,6 +7,10 @@
 # Even minor modifications require translation updates and such
 # changes should be coordinated with translators and reviewers.
 
+Template: adduser/title
+Type: title
+Description: Adduser
+
 Template: adduser/homedir-permission
 Type: boolean
 Default: true
