Author: Francois Marier <francois@debian.org>
Description: Reinitialize variable in check_tcpd
Bug-Ubuntu: https://bugs.launchpad.net/ubuntu/+source/chkrootkit/+bug/1808882

--- a/chkrootkit	2019-01-13 14:30:39.608931525 -0800
+++ b/chkrootkit	2019-01-13 15:05:53.496917560 -0800
@@ -2588,6 +2588,7 @@
 chk_tcpd () {
     STATUS=${NOT_INFECTED}
     TCPD_INFECTED_LABEL="p1r0c4|hack|/dev/xmx|/dev/hdn0|/dev/xdta|/dev/tux"
+    CMD=
 
     [ -r ${ROOTDIR}etc/inetd.conf ] &&
     CMD=`${egrep} '^[^#].*tcpd' ${ROOTDIR}etc/inetd.conf | _head -1 | \
