diff --git a/bash_completion b/bash_completion
index 552f808..2a748d3 100644
--- a/bash_completion
+++ b/bash_completion
@@ -2473,7 +2473,7 @@ _aptitude()
 				       autoclean install reinstall remove \
 				       hold unhold purge markauto unmarkauto why why-not \
 				       dist-upgrade full-upgrade download search show \
-				       forbid-version changelog keep-all' -- $cur ) )
+				       forbid-version changelog keep-all build-dep' -- $cur ) )
 	fi
 
 
