Only in .: aaa.diff
diff -ur ../etalon/audacious-1.2.2/debian/rules ./audacious-1.2.2/debian/rules
--- ../etalon/audacious-1.2.2/debian/rules	2007-03-27 19:42:02.000000000 +0000
+++ ./audacious-1.2.2/debian/rules	2007-03-27 19:46:59.000000000 +0000
@@ -27,6 +27,7 @@
 		    --mandir=/usr/share/man \
 		    --disable-rpath \
 		    --enable-ipv6 \
+		    --enable-chardet \
 		    CFLAGS="$(CFLAGS)"
 
 build: build-stamp
diff -ur ../etalon/audacious-plugins-1.2.5/debian/rules ./audacious-plugins-1.2.5/debian/rules
--- ../etalon/audacious-plugins-1.2.5/debian/rules	2007-03-27 19:44:46.000000000 +0000
+++ ./audacious-plugins-1.2.5/debian/rules	2007-03-27 19:47:19.000000000 +0000
@@ -35,6 +35,7 @@
 		    --prefix=/usr \
 		    --mandir=/usr/share/man \
 		    --disable-rpath \
+		    --enable-chardet \
 		    CFLAGS="$(CFLAGS)"
 
 build: build-stamp
