diff -Nru apt-cacher-ng-0.8.0~rc3.orig/debian/control apt-cacher-ng-0.8.0~rc3/debian/control
--- apt-cacher-ng-0.8.0~rc3.orig/debian/control	2014-09-20 11:53:25.000000000 +0100
+++ apt-cacher-ng-0.8.0~rc3/debian/control	2014-10-01 22:14:41.631681461 +0100
@@ -2,7 +2,7 @@
 Section: net
 Priority: optional
 Maintainer: Eduard Bloch <blade@debian.org>
-Build-Depends: debhelper (>= 9), cmake (>= 2.6.2), libbz2-dev, zlib1g-dev, liblzma-dev, libfuse-dev [!hurd-i386], pkg-config, libwrap0-dev, lsb-base (>> 3.0-6), dh-systemd (>= 1.5), po-debconf, libssl-dev, libsystemd-daemon-dev
+Build-Depends: debhelper (>= 9), cmake (>= 2.6.2), libbz2-dev, zlib1g-dev, liblzma-dev, libfuse-dev [!hurd-i386], pkg-config, libwrap0-dev, lsb-base (>> 3.0-6), dh-systemd (>= 1.5), po-debconf, libssl-dev, libsystemd-daemon-dev [linux-any]
 Standards-Version: 3.9.5
 Homepage: http://www.unix-ag.uni-kl.de/~bloch/acng/
 
diff -Nru apt-cacher-ng-0.8.0~rc3.orig/debian/rules apt-cacher-ng-0.8.0~rc3/debian/rules
--- apt-cacher-ng-0.8.0~rc3.orig/debian/rules	2014-09-20 11:53:25.000000000 +0100
+++ apt-cacher-ng-0.8.0~rc3/debian/rules	2014-10-01 22:05:13.062740285 +0100
@@ -13,6 +13,8 @@
 export CXXFLAGS:=$(shell dpkg-buildflags --get CXXFLAGS) $(CPPFLAGS)
 export LDFLAGS:=$(shell dpkg-buildflags --get LDFLAGS)
 
+DEB_HOST_ARCH_OS  ?= $(shell dpkg-architecture -qDEB_HOST_ARCH_OS)
+
 %:
 	dh $@ --parallel --with systemd
 
@@ -22,8 +24,10 @@
 override_dh_install:
 	dh_install $(shell test -e build/acngfs || echo -Xacngfs)
 	sed -e s,^Type=simple,Type=notify, < systemd/apt-cacher-ng.service > debian/apt-cacher-ng.service
-# make sure it has sd_notify support enabled
+ifeq ($(DEB_BUILD_ARCH_OS),linux)
+	# on Linux, make sure it has sd_notify support enabled
 	grep -q libsystemd $(TGT)/usr/sbin/apt-cacher-ng
+endif
 	cp systemd/apt-cacher-ng.conf debian/apt-cacher-ng.tmpfile
 	install -m 755 scripts/expire-caller.pl $(TGT)/usr/lib/apt-cacher-ng
 	$(MAKE) -C dbgen package DBGENERATOR=dbgenerator
