--- autofs.old	Sat May 13 14:39:28 2000
+++ autofs	Sat May 13 14:40:18 2000
@@ -78,12 +78,17 @@
             then
         #   not needed in here
         #       map=`echo "$map" | sed -e 's/^auto_/auto./'`
+	    # Added by graeme@mathie.cx:
+		type=`echo $map | sed -e 's/^\(.*\):.*$/\1/g'`
+		if [ ! -z "$type" ]; then
+		    map=`echo $map | sed -e 's/^.*:\(.*\)$/\1/g'`
+		fi
                 filter_opts
                 options=`echo "$options" | sed -e 's/\(^\|[ \t]\)-/\1/g'`
                 test ! -z "$PRUNEREGEX" && \
                     options=`echo $options | \
                         sed -e "s#$PRUNEREGEX##g" -e 's#,\+#,#g' -e 's#,$##'`
-                echo "$DAEMON $mountoptions $dir yp $map $options $localoptions"
+                echo "$DAEMON $mountoptions $dir $type $map $options $localoptions"
             fi
         done
     )
