From ca6b08a31fbf767c2e468c93321ec8f4f9c511d7 Mon Sep 17 00:00:00 2001
From: Julian Wollrath <jwollrath@web.de>
Date: Fri, 31 Jan 2014 14:21:30 +0100
Subject: [PATCH] fix x-session-manger -> x-window-manager

---
 debian/55awesome-javaworkaround | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/55awesome-javaworkaround b/debian/55awesome-javaworkaround
index c7b8d6257177..d50dc6057e87 100644
--- a/debian/55awesome-javaworkaround
+++ b/debian/55awesome-javaworkaround
@@ -2,8 +2,8 @@
 # manager. Cf. bug #508650.
 
 BASESTARTUP=$(basename "$STARTUP" | cut -d\  -f1)
-if [ "$BASESTARTUP" = x-session-manager ]; then
-    BASESTARTUP=$(basename $(readlink /etc/alternatives/x-session-manager))
+if [ "$BASESTARTUP" = x-window-manager ]; then
+    BASESTARTUP=$(basename $(readlink /etc/alternatives/x-window-manager))
 fi
 if [ "$BASESTARTUP" = "awesome" ]; then
     _JAVA_AWT_WM_NONREPARENTING=1; export _JAVA_AWT_WM_NONREPARENTING
-- 
1.9.rc1

