diff -ruN busybox-1.1.3.orig/debian/control busybox-1.1.3/debian/control
--- busybox-1.1.3.orig/debian/control	2006-10-10 11:06:09.000000000 +0200
+++ busybox-1.1.3/debian/control	2006-10-10 12:00:19.000000000 +0200
@@ -21,9 +21,7 @@
  counterparts.
  .
  This package installs the BusyBox binary but does not install symlinks
- for any of the supported utilities. You can use /bin/busybox --install
- to install BusyBox to the current directory (you do not want to do this
- in / on your Debian system!).
+ for any of the supported utilities.
 
 Package: busybox-static
 Architecture: any
