diff -ruN cvs-1.12.13-old/src/parseinfo.c cvs-1.12.13/src/parseinfo.c
--- cvs-1.12.13-old/src/parseinfo.c   2006-08-19 00:05:38.000000000 +0100
+++ cvs-1.12.13/src/parseinfo.c       2006-08-19 00:05:41.000000000 +0100
@@ -374,6 +374,7 @@
 struct config *
 parse_config (const char *cvsroot, const char *path)
 {
+    bool gotone = false;
     const char *infopath;
     char *freeinfopath = NULL;
     FILE *fp_info;
@@ -615,7 +616,6 @@
 	{
 	    if (strcmp (p, "all") != 0)
 	    {
-		static bool gotone = false;
 		if (gotone)
 		    error (0, 0, "\
 %s [%u]: warning: duplicate LogHistory entry found.",
