diff -r 7501139c0432 -r fdea5abcb280 debian/anacrontab
--- a/debian/anacrontab	Mon May 05 15:24:25 2008 +0200
+++ b/debian/anacrontab	Mon May 05 15:34:55 2008 +0200
@@ -6,7 +6,7 @@
 PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
 
 # These replace cron's entries
-1	5	cron.daily	 nice run-parts --report /etc/cron.daily
-7	10	cron.weekly	 nice run-parts --report /etc/cron.weekly
-@monthly	15	cron.monthly nice run-parts --report /etc/cron.monthly
+1	5	cron.daily	 nice ionice -c 3 run-parts --report /etc/cron.daily
+7	10	cron.weekly	 nice ionice -c 3 run-parts --report /etc/cron.weekly
+@monthly	15	cron.monthly nice ionice -c 3 run-parts --report /etc/cron.monthly
 
