--- evolution-3.2.2-old/debian/rules	2011-11-24 19:13:27.000000000 +0000
+++ evolution-3.2.2/debian/rules	2012-03-07 18:34:23.156893611 +0000
@@ -8,7 +8,6 @@
 include /usr/share/cdbs/1/rules/utils.mk
 include /usr/share/gnome-pkg-tools/1/rules/gnome-get-source.mk
 include /usr/share/gnome-pkg-tools/1/rules/gnome-version.mk
-include /usr/share/hardening-includes/hardening.make
 
 ELIBDIR := usr/lib/evolution/3.2
 
@@ -17,8 +16,9 @@
 DEB_DH_MAKESHLIBS_ARGS_evolution += -V
 DEB_DH_MAKESHLIBS_ARGS_ALL += --noscripts -X $(ELIBDIR)/plugins
 
-export DEB_LDFLAGS_MAINT_APPEND=-Wl,-z,defs -Wl,--as-needed -Wl,-O1
-export DEB_BUILD_MAINT_OPTIONS=hardening=+pie,+bindnow
+export DEB_BUILD_MAINT_OPTIONS = hardening=+all
+DPKG_EXPORT_BUILDFLAGS = 1
+include /usr/share/dpkg/buildflags.mk
 
 DEB_REVISION := $(shell echo $(DEB_VERSION) | awk -F- '{ print $$NF }')
 
