--- Source/kwsys/ProcessUNIX.c	2006-12-04 14:04:05.000000000 -0200
+++ Source/kwsys/ProcessUNIX.c	2007-01-17 08:49:45.000000000 -0200
@@ -2328,7 +2328,13 @@
     struct sigaction newSigChldAction;
     memset(&newSigChldAction, 0, sizeof(struct sigaction));
     newSigChldAction.sa_sigaction = kwsysProcessesSignalHandler;
+    /* SA_SIGINFO isn't supported by Hurd. Checking is needed to avoid build failure */
+#ifdef SA_SIGINFO
     newSigChldAction.sa_flags = SA_NOCLDSTOP | SA_SIGINFO;
+#else
+    newSigChldAction.sa_flags = SA_NOCLDSTOP;
+#endif
+
 #ifdef SA_RESTART
     newSigChldAction.sa_flags |= SA_RESTART;
 #endif
