--- lid.sh	(revision 4358)
+++ lid.sh	(working copy)
@@ -41,6 +41,12 @@
 		su $user -c "xscreensaver-command -deactivate"
 	    fi
 	    su $user -c "xset dpms force on"
+	    
+	    # Press LEFT SHIFT. On some laptops / in some configurations forcing
+	    # the DPMS on is not enough, X needs to be convinced to unblank the
+	    # screen. The left shift key is probably the most harmless key we
+	    # could press. :-)
+	    acpi_fakekey 42
 	fi
     done
 fi