diff -u cyrus-sasl2-2.1.19.dfsg1/debian/control cyrus-sasl2-2.1.19.dfsg1/debian/control
--- cyrus-sasl2-2.1.19.dfsg1/debian/control
+++ cyrus-sasl2-2.1.19.dfsg1/debian/control
@@ -1,5 +1,5 @@
 Source: cyrus-sasl2
-Build-Depends: autotools-dev, debhelper (>= 4), libpam0g-dev, zlib1g-dev, libdb4.2-dev (>= 3.2.9-14), libopie-dev (>= 2.32-8), chrpath, heimdal-dev (>= 0.4e-16), libssl-dev (>= 0.9.6d-1), libmysqlclient15-dev, libpq-dev, libldap2-dev (>= 2.1.21), automake1.7 (>= 1.7.9), autoconf (>=2.57), dbs, groff-base, comerr-dev, libtool (>= 1.5.6)
+Build-Depends: autotools-dev, debhelper (>= 4), libpam0g-dev, zlib1g-dev, libdb4.3-dev, libopie-dev (>= 2.32-8), chrpath, libkrb5-dev, libssl-dev (>= 0.9.6d-1), libmysqlclient15-dev, libpq-dev, libldap2-dev (>= 2.1.21), automake1.7 (>= 1.7.9), autoconf (>=2.57), dbs, groff-base, comerr-dev, libtool (>= 1.5.6)
 Build-Conflicts: autoconf2.13, automake1.4
 Section: devel
 Priority: important
diff -u cyrus-sasl2-2.1.19.dfsg1/debian/changelog cyrus-sasl2-2.1.19.dfsg1/debian/changelog
--- cyrus-sasl2-2.1.19.dfsg1/debian/changelog
+++ cyrus-sasl2-2.1.19.dfsg1/debian/changelog
@@ -1,3 +1,11 @@
+cyrus-sasl2 (2.1.19.dfsg1-0.3) UNRELEASED; urgency=high
+
+  * Non-maintainer upload
+  * Use libkrb5-dev instead of heimdal-dev. Closes: #379846.
+  * Use libdb4.3-dev instead of libdb4.2-dev. Closes: #336485.
+
+ -- Peter Eisentraut <petere@debian.org>  Fri, 15 Sep 2006 18:30:19 +0200
+
 cyrus-sasl2 (2.1.19.dfsg1-0.2) unstable; urgency=high
 
   * Non-maintainer upload
diff -u cyrus-sasl2-2.1.19.dfsg1/debian/patches/13_libdb42_autotools.diff cyrus-sasl2-2.1.19.dfsg1/debian/patches/13_libdb42_autotools.diff
--- cyrus-sasl2-2.1.19.dfsg1/debian/patches/13_libdb42_autotools.diff
+++ cyrus-sasl2-2.1.19.dfsg1/debian/patches/13_libdb42_autotools.diff
@@ -15,7 +15,7 @@
 +#            AC_CHECK_LIB($dbname, db_create, BDB_LIBADD="$BDB_LIBADD -l$dbname";
 +#              dblib="berkeley"; break, dblib="no")
 +#          done
-+        BDB_LIBADD="$BDB_LIBADD -ldb-4.2";
++        BDB_LIBADD="$BDB_LIBADD -ldb-4.3";
 +	dblib="berkeley";
          if test "$dblib" = "no"; then
            AC_CHECK_LIB(db, db_open, BDB_LIBADD="$BDB_LIBADD -ldb";
