Index: xkeyboard-config-1.8/symbols/fi
===================================================================
--- xkeyboard-config-1.8.orig/symbols/fi
+++ xkeyboard-config-1.8/symbols/fi
@@ -184,6 +184,8 @@
     key <AE12> { [ dead_acute, dead_grave ] };
     key <AD12> { [ dead_diaeresis, dead_circumflex, dead_tilde, dead_caron ] };
 
+    include "latin(type2)"
+
     // End alphanumeric section, begin "Keypad"
     include "kpdl(comma)"
     // End "Keypad" section
