--- a/src/word.c	2018-10-05 07:12:10.281153971 +0000
+++ b/src/word.c	2018-10-05 07:12:03.937217631 +0000
@@ -37,6 +37,10 @@
 
 int word_cmp(const word_t *w1, const word_t *w2)
 {
+    if (! w1 && ! w2) return 0;
+    if (! w1 && w2) return -1;
+    if (w1 && ! w2) return 1;
+
     uint l = min(w1->leng, w2->leng);
     int r = memcmp((const char *)w1->u.text, (const char *)w2->u.text, l);
     if (r) return r;
