diff -u avahi-0.6.22/debian/rules avahi-0.6.22/debian/rules
--- avahi-0.6.22/debian/rules
+++ avahi-0.6.22/debian/rules
@@ -41,6 +41,13 @@
 pre-build::
 	-rm -f avahi-python/avahi/ServiceTypeDatabase.py 
 
+common-build-arch::
+	# create an up to date PO template
+	cd po; intltool-update -p --verbose
+
+clean::
+	rm -f po/*.pot
+
 binary-install/avahi-discover::
 	dh_pysupport -p$(cdbs_curpkg)
 	rm -f debian/tmp/usr/lib/python*/site-packages/avahi/*.py[co]
