Index: lvm2-2.03.02/daemons/dmeventd/plugins/raid/dmeventd_raid.c
===================================================================
--- lvm2-2.03.02.orig/daemons/dmeventd/plugins/raid/dmeventd_raid.c
+++ lvm2-2.03.02/daemons/dmeventd/plugins/raid/dmeventd_raid.c
@@ -83,7 +83,7 @@ static int _process_raid_event(struct ds
 					 "before initiating repair on RAID device %s.", device);
 			}
 
-			goto out; /* Not yet done syncing with accessible devices */
+			//goto out; /* Not yet done syncing with accessible devices */
 		}
 
 		if (state->failed)
