diff --git a/debian/corekeeper.cron.daily b/debian/corekeeper.cron.daily
index 9457927..4c6d4fc 100755
--- a/debian/corekeeper.cron.daily
+++ b/debian/corekeeper.cron.daily
@@ -7,7 +7,7 @@ fi
 cd /var/cache/corekeeper/
 find /var/crash -name '*.core' \( \( -mtime +7 -delete -fprint /dev/stderr \) -o \( -print \) \) > next 2> deleted
 sort current current next | uniq -u > new
-sort deleted deleted new new next | uniq -u > old
+sort deleted deleted current | uniq -u > old
 
 if [ -s new ] ; then
 	echo 'New core file(s):'
