From 1b860ec77fae7a203f3599aef265fa7b7c5117b6 Mon Sep 17 00:00:00 2001
From: Ansgar Burchardt <ansgar@debian.org>
Date: Thu, 16 May 2013 18:27:59 +0200
Subject: [PATCH 2/2] etc/99builddsourceslist: suites on security mirror need
 /updates

---
 etc/99builddsourceslist |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/etc/99builddsourceslist b/etc/99builddsourceslist
index 71c265b..4c4b639 100755
--- a/etc/99builddsourceslist
+++ b/etc/99builddsourceslist
@@ -71,7 +71,7 @@ if [ "$1" = "setup-start" ] || [ "$1" = "setup-recover" ]; then
             SUITES="${SUITE_BASE}"
             [ -n "${debian_mirror}" ] && domirror "${debian_mirror} ${SUITE_BASE} COMPONENT" ${APT_LIST}
             domirror "http://ftp.debian.org/debian ${SUITE_BASE} COMPONENT" ${APT_LIST}
-            [ -n "${security_mirror}" ] && domirror "${security_mirror} ${SUITE_BASE} COMPONENT" ${APT_LIST}
+            [ -n "${security_mirror}" ] && domirror "${security_mirror} ${SUITE_BASE}/updates COMPONENT" ${APT_LIST}
             domirror "http://security-master.debian.org/debian-security ${SUITE_BASE}/updates COMPONENT" ${APT_LIST}
             domirror "http://security-master.debian.org/debian-security-buildd buildd-${SUITE_BASE}/updates COMPONENT" ${APT_LIST}
 
-- 
1.7.9.5

