--- ../bash-completion-20080705/bash_completion	2008-07-05 22:55:03.000000000 +0900
+++ bash_completion	2008-09-12 21:40:06.000000000 +0900
@@ -3125,6 +3125,9 @@
 		return 0
 	fi
 
+	# Enable -o filenames option
+	compgen -f /non-existing-dir/ > /dev/null
+
 	# Use standard dir completion if no CDPATH or parameter starts with /,
 	# ./ or ../
 	if [ -z "${CDPATH:-}" ] || [[ "$cur" == ?(.)?(.)/* ]]; then
@@ -3159,9 +3162,9 @@
 	return 0
 }
 if shopt -q cdable_vars; then
-    complete -v -F _cd $nospace $filenames cd
+    complete -v -F _cd $nospace cd
 else
-    complete -F _cd $nospace $filenames cd
+    complete -F _cd $nospace cd
 fi
 
 _remove_comp_word()
