diff -Nru diffpdf-2.1.3/debian/changelog diffpdf-2.1.3/debian/changelog
--- diffpdf-2.1.3/debian/changelog	2013-11-04 17:16:33.000000000 +0100
+++ diffpdf-2.1.3/debian/changelog	2015-02-02 11:17:28.000000000 +0100
@@ -1,3 +1,10 @@
+diffpdf (2.1.3-1.1) UNRELEASED; urgency=medium
+
+  * Non-maintainer upload.
+  * add patch to not require the input file extension to be pdf
+
+ -- Johannes Schauer <j.schauer@email.de>  Mon, 02 Feb 2015 11:17:00 +0100
+
 diffpdf (2.1.3-1) unstable; urgency=low
 
   * New upstream version (Closes: #703440).
diff -Nru diffpdf-2.1.3/debian/patches/do-not-check-extension diffpdf-2.1.3/debian/patches/do-not-check-extension
--- diffpdf-2.1.3/debian/patches/do-not-check-extension	1970-01-01 01:00:00.000000000 +0100
+++ diffpdf-2.1.3/debian/patches/do-not-check-extension	2015-02-02 11:16:51.000000000 +0100
@@ -0,0 +1,17 @@
+Description: do not require the file extension to be pdf
+Author: Johannes Schauer <j.schauer@email.de>
+
+--- diffpdf-2.1.3.orig/main.cpp
++++ diffpdf-2.1.3/main.cpp
+@@ -93,9 +93,9 @@ int main(int argc, char *argv[])
+             debug = DebugShowTextsAndYX;
+         else if (optionsOK && arg == "--")
+             optionsOK = false;
+-        else if (filename1.isEmpty() && arg.toLower().endsWith(".pdf"))
++        else if (filename1.isEmpty())
+             filename1 = arg;
+-        else if (filename2.isEmpty() && arg.toLower().endsWith(".pdf"))
++        else if (filename2.isEmpty())
+             filename2 = arg;
+         else
+             out << "unrecognized argument '" << arg << "'\n";
diff -Nru diffpdf-2.1.3/debian/patches/series diffpdf-2.1.3/debian/patches/series
--- diffpdf-2.1.3/debian/patches/series	2013-11-04 17:16:33.000000000 +0100
+++ diffpdf-2.1.3/debian/patches/series	2015-02-02 11:16:19.000000000 +0100
@@ -1 +1,2 @@
 01-fix_manpage.patch
+do-not-check-extension
