diff --git a/debian/bootstrap-base.postinst b/debian/bootstrap-base.postinst
index ed2e0ab..7e12a5f 100755
--- a/debian/bootstrap-base.postinst
+++ b/debian/bootstrap-base.postinst
@@ -23,7 +23,9 @@ EXCLUDES="$(printf '%s' "$RET" | sed 's/ /,/g')" || true
 # Check if a specific script should be used
 db_get base-installer/debootstrap_script
 DEBOOTSTRAP_SCRIPT="$RET"
-if [ ! -e /usr/share/debootstrap/scripts/"$DEBOOTSTRAP_SCRIPT" ]; then
+if [ -e /usr/share/debootstrap/scripts/"$DEBOOTSTRAP_SCRIPT" ]; then
+	DEBOOTSTRAP_SCRIPT=/usr/share/debootstrap/scripts/"$DEBOOTSTRAP_SCRIPT"
+elif [ ! -e "$DEBOOTSTRAP_SCRIPT" ]; then
 	error "debootstrap script '$DEBOOTSTRAP_SCRIPT' doesn't exist"
 fi
 
