diff -u digitemp-3.3.2/debian/rules digitemp-3.3.2/debian/rules
--- digitemp-3.3.2/debian/rules
+++ digitemp-3.3.2/debian/rules
@@ -32,10 +32,13 @@
 build-stamp: configure-stamp 
 	dh_testdir
 
-	mkdir -p build-serial/{src,userial/ds9097,userial/ds9097u}
+	mkdir -p build-serial/src
+	mkdir -p build-serial/userial/ds9097
+	mkdir -p build-serial/userial/ds9097u
 	(cd build-serial; $(MAKE) -f ../Makefile srcdir=.. VPATH=.. all)
 	
-	mkdir -p build-usb/{src,userial/ds2490}       
+	mkdir -p build-usb/src
+	mkdir -p build-usb/userial/ds2490
 	(cd build-usb; $(MAKE) -f ../Makefile srcdir=.. VPATH=.. ds2490)
