diff -Nru clasp-3.3.3/debian/changelog clasp-3.3.3/debian/changelog
--- clasp-3.3.3/debian/changelog	2018-02-05 20:22:21.000000000 +0100
+++ clasp-3.3.3/debian/changelog	2018-06-22 12:39:30.000000000 +0200
@@ -1,3 +1,10 @@
+clasp (3.3.3-3.1) UNRELEASED; urgency=medium
+
+  * Non-maintainer upload.
+  * Build static library, with -fPIC
+
+ -- Julian Andres Klode <jak@debian.org>  Fri, 22 Jun 2018 12:39:30 +0200
+
 clasp (3.3.3-3) unstable; urgency=medium
 
   * debian/rules: check DEB_BUILD_PROFILES against nocheck
diff -Nru clasp-3.3.3/debian/control clasp-3.3.3/debian/control
--- clasp-3.3.3/debian/control	2018-02-03 13:39:46.000000000 +0100
+++ clasp-3.3.3/debian/control	2018-06-22 12:39:30.000000000 +0200
@@ -16,6 +16,7 @@
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}
 Recommends: gringo
+Provides: libclasp-dev (= ${binary:Version})
 Description: conflict-driven nogood learning answer set solver
  clasp is an answer set solver for (extended) normal logic
  programs. It combines the high-level modeling capacities of answer
diff -Nru clasp-3.3.3/debian/rules clasp-3.3.3/debian/rules
--- clasp-3.3.3/debian/rules	2018-02-05 20:22:15.000000000 +0100
+++ clasp-3.3.3/debian/rules	2018-05-25 12:35:59.000000000 +0200
@@ -13,7 +13,7 @@
 
 # use custom hardening config flags for compiling clasp
 CONFIG=hardening
-CONFIGPARMS = -DLIB_POTASSCO_BUILD_APP=OFF -DCLASP_BUILD_TESTS=ON -DLIB_POTASSCO_BUILD_TESTS=ON -DCLASP_BUILD_EXAMPLES=ON
+CONFIGPARMS = -DLIB_POTASSCO_BUILD_APP=OFF -DCLASP_BUILD_TESTS=ON -DLIB_POTASSCO_BUILD_TESTS=ON -DCLASP_BUILD_EXAMPLES=ON -DCLASP_INSTALL_LIB=ON
 TOPBUILDDIR = build/$(CONFIG)_mt
 
 #
@@ -22,8 +22,8 @@
 # add -O3 and NDEBUG and enable all hardening flags
 # link with libatomic on architectures without built-in atomic
 #
-DEB_CXXFLAGS_MAINT_APPEND = -O3 -DNDEBUG
-DEB_LDFLAGS_MAINT_APPEND =
+DEB_CXXFLAGS_MAINT_APPEND = -O3 -DNDEBUG -fPIC
+DEB_LDFLAGS_MAINT_APPEND = -fPIC
 ifeq ($(if $(arch),atomic), atomic)
 	DEB_LDFLAGS_MAINT_APPEND += -latomic
 endif
