diff -ur devscripts-2.10.61/scripts/deb-reversion.dbk devscripts/scripts/deb-reversion.dbk
--- devscripts-2.10.61/scripts/deb-reversion.dbk	2010-01-03 04:20:42.000000000 +0100
+++ devscripts/scripts/deb-reversion.dbk	2010-01-12 18:44:23.618261523 +0100
@@ -166,6 +166,18 @@
       </varlistentry>
 
       <varlistentry>
+	<term><option>-d</option></term>
+	<term><option>--update-dependencies</option></term>
+	<listitem>
+	  <para>
+            If the package depends on another package with exactly the
+            same version, update the dependency to match the new
+            version number.
+          </para>
+	</listitem>
+      </varlistentry>
+
+      <varlistentry>
 	<term><option>-s</option> <replaceable class="parameter">string</replaceable></term>
 	<term><option>--string</option> <replaceable class="parameter">string</replaceable></term>
 	<listitem>
@@ -277,7 +289,8 @@
       it works fine in most cases, it may just not in yours. Especially,
       please consider that it changes binary packages (only!) and hence can
       break strict versioned dependencies between binary packages generated
-      from the same source. </para>
+      from the same source (in which case <option>-d</option> may help).
+    </para>
 
     <para>
       You are using this tool at your own risk and I shall not shed a tear if
diff -ur devscripts-2.10.61/scripts/deb-reversion.sh devscripts/scripts/deb-reversion.sh
--- devscripts-2.10.61/scripts/deb-reversion.sh	2010-01-03 04:20:42.000000000 +0100
+++ devscripts/scripts/deb-reversion.sh	2010-01-12 18:19:54.628811670 +0100
@@ -39,6 +39,7 @@
 	-v ver|--new-version=ver & use this as new version number
 	-o old|--old-version=ver & calculate new version number based on this old one
 	-c|--calculate-only & only calculate (and print) the augmented version
+	-d|--update-dependencies & update strict versioned dependencies
 	-s str|--string=str & append this string instead of '$VERSTR' to
 	                    & calculate new version number
 	-k script|--hook=script & call this script before repacking
@@ -61,11 +62,12 @@
 }
 
 CURDIR="$(pwd)"
-SHORTOPTS=hVo:v:ck:Ds:b
-LONGOPTS=help,version,old-version:new-version:,calculate-only,hook:,debug,string:,force-bad-version
+SHORTOPTS=hVo:v:cdk:Ds:b
+LONGOPTS=help,version,old-version:new-version:,calculate-only,update-dependencies,hook:,debug,string:,force-bad-version
 set -- $(getopt -s bash -o $SHORTOPTS -l $LONGOPTS --n $PROGNAME -- "$@")
 
 CALCULATE=0
+DEPENDENCIES=0
 DPKGDEB_DEBUG=
 DEB=
 DCH_OPTIONS=
@@ -83,6 +85,7 @@
     -v|--new-version) OPT_STATE=SET_NEW_VERSION;;
     -o|--old-version) OPT_STATE=SET_OLD_VERSION;;
     -c|--calculate-only|--print-only) CALCULATE=1;;
+    -d|--update-dependencies) DEPENDENCIES=1;;
     -s|--string) OPT_STATE=SET_STRING;;
     -k|--hook) OPT_STATE=SET_HOOK;;
     -D|--debug) DPKGDEB_DEBUG=--debug;;
@@ -181,6 +184,10 @@
   call_hook
   gzip -9 -c debian/changelog >| $LOGFILE
   sed -i -e "s,^Version: .*,Version: $VERSION," DEBIAN/control
+  if [ $DEPENDENCIES -eq 1 ]; then
+    ESC_OLD_VERSION=$(echo "$OLD_VERSION" | sed -e 's,\.,\\&,g')
+    sed -i -e "/^Depends: / s,(= $ESC_OLD_VERSION),(= $VERSION),g" DEBIAN/control
+  fi
   rm -rf debian
 }
 
