diff -Nru dpkg-cross-2.6.7arm64/config/cross-config.arm64 dpkg-cross-2.6.7arm64fix1/config/cross-config.arm64
--- dpkg-cross-2.6.7arm64/config/cross-config.arm64	2012-10-24 22:41:43.000000000 +0100
+++ dpkg-cross-2.6.7arm64fix1/config/cross-config.arm64	2012-11-20 16:13:55.000000000 +0000
@@ -1,4 +1,7 @@
 # read in package-specific and global values.
+echo "Reading Cross Config Cache (/etc/dpkg-cross/cross-config.arm64)"
+foobar=`dirname $ac_site_file`
+echo "Reading $foobar/cross-config.cache"
 . `dirname $ac_site_file`/cross-config.cache
 # now ensure the real architecture-dependent values take priority
 ac_cv_c_bigendian=no
@@ -16,7 +19,7 @@
 ac_cv_sizeof_signed_char=1
 ac_cv_sizeof_float=4
 ac_cv_sizeof_double=8
-ac_cv_sizeof_long_double=8
+ac_cv_sizeof_long_double=16
 ac_cv_sizeof_void_p=8
 ac_cv_sizeof_char_p=8
 ac_cv_sizeof_off_t=8
diff -Nru dpkg-cross-2.6.7arm64/debian/changelog dpkg-cross-2.6.7arm64fix1/debian/changelog
--- dpkg-cross-2.6.7arm64/debian/changelog	2012-10-24 22:44:55.000000000 +0100
+++ dpkg-cross-2.6.7arm64fix1/debian/changelog	2012-11-20 16:16:22.000000000 +0000
@@ -1,3 +1,9 @@
+dpkg-cross (2.6.7arm64fix1) quantal; urgency=low
+
+  * Correct size of long double to 128 bits.
+
+ -- Wookey <wookey@wookware.org>  Tue, 20 Nov 2012 16:14:02 +0000
+
 dpkg-cross (2.6.7arm64) precise; urgency=low
 
   * Add arm64 support
