--- MainWindow.py	2011-10-14 04:19:55.000000000 -0300
+++ MainWindow.py	2012-04-22 21:31:26.000000000 -0300
@@ -259,7 +259,7 @@
 			self.item_store.append((show, icon, name, item))
 
 	#this is a little timeout callback to insert new items after
-	#gnome-desktop-item-edit has finished running
+	#exo-desktop-item-edit has finished running
 	def waitForNewItemProcess(self, process, parent, file_path):
 		if process.poll() != None:
 			if os.path.isfile(file_path):
@@ -269,7 +269,7 @@
 
 	def waitForNewMenuProcess(self, process, parent_id, file_path):
 		if process.poll() != None:
-			#hack for broken gnome-desktop-item-edit
+			#hack for broken exo-desktop-item-edit
 			broken_path = os.path.join(os.path.split(file_path)[0], '.directory')
 			if os.path.isfile(broken_path):
 				os.rename(broken_path, file_path)
@@ -295,7 +295,7 @@
 		else:
 			parent = menus[iter][2]
 		file_path = os.path.join(util.getUserDirectoryPath(), util.getUniqueFileId('alacarte-made', '.directory'))
-		process = subprocess.Popen(['gnome-desktop-item-edit', file_path], env=os.environ)
+		process = subprocess.Popen(['exo-desktop-item-edit', file_path], env=os.environ)
 		gobject.timeout_add(100, self.waitForNewMenuProcess, process, parent.menu_id, file_path)
 
 	def on_new_item_button_clicked(self, button):
@@ -308,7 +308,7 @@
 		else:
 			parent = menus[iter][2]
 		file_path = os.path.join(util.getUserItemPath(), util.getUniqueFileId('alacarte-made', '.desktop'))
-		process = subprocess.Popen(['gnome-desktop-item-edit', file_path], env=os.environ)
+		process = subprocess.Popen(['exo-desktop-item-edit', file_path], env=os.environ)
 		gobject.timeout_add(100, self.waitForNewItemProcess, process, parent, file_path)
 
 	def on_new_separator_button_clicked(self, button):
@@ -379,7 +379,7 @@
 			self.editor._MenuEditor__addUndo([item,])
 		if file_path not in self.edit_pool:
 			self.edit_pool.append(file_path)
-			process = subprocess.Popen(['gnome-desktop-item-edit', file_path], env=os.environ)
+			process = subprocess.Popen(['exo-desktop-item-edit', file_path], env=os.environ)
 			gobject.timeout_add(100, self.waitForEditProcess, process, file_path)
 
 	def on_menu_tree_cursor_changed(self, treeview):
