diff -ur bochs-2.0.2+20020419.orig/debian/bochs.install bochs-2.0.2+20020419/debian/bochs.install
--- bochs-2.0.2+20020419.orig/debian/bochs.install	Mon May  5 08:31:07 2003
+++ bochs-2.0.2+20020419/debian/bochs.install	Mon May  5 08:32:59 2003
@@ -4,6 +4,7 @@
 debian/tmp/usr/lib/bochs/plugins/libbx_biosdev.*
 debian/tmp/usr/lib/bochs/plugins/libbx_cmos.*
 debian/tmp/usr/lib/bochs/plugins/libbx_dma.*
+debian/tmp/usr/lib/bochs/plugins/libbx_extfpuirq.*
 debian/tmp/usr/lib/bochs/plugins/libbx_floppy.*
 debian/tmp/usr/lib/bochs/plugins/libbx_harddrv.*
 debian/tmp/usr/lib/bochs/plugins/libbx_keyboard.*
