--- cppcheck-1.61/debian/rules	2013-08-04 10:05:47.000000000 -0700
+++ cppcheck-1.63+dfsg/debian/rules	2014-02-11 11:34:06.399181350 -0800
@@ -14,7 +14,7 @@
 
 override_dh_auto_build:
 	sed -i.nocppflags -e 's/\$$(CXX) \$$(CXXFLAGS)/$$(CXX) $$(CPPFLAGS) $$(CXXFLAGS)/' Makefile
-	dh_auto_build -- HAVE_RULES=yes TINYXML=-ltinyxml2 INCLUDE_FOR_LIB=-Ilib INCLUDE_FOR_CLI=-Ilib INCLUDE_FOR_TEST="-Ilib -Icli"
+	dh_auto_build -- HAVE_RULES=yes INCLUDE_FOR_LIB="-Ilib -Iexternals/tinyxml" INCLUDE_FOR_CLI="-Ilib -Iexternals/tinyxml" INCLUDE_FOR_TEST="-Ilib -Icli -Iexternals/tinyxml"
 	$(XP) $(DB2MAN) man/cppcheck.1.xml
 
 override_dh_auto_clean:
@@ -22,10 +22,10 @@
 	[ ! -f Makefile.nocppflags ] || mv -f Makefile.nocppflags Makefile
 
 override_dh_auto_test:
-	dh_auto_test -- HAVE_RULES=yes TINYXML=-ltinyxml2 INCLUDE_FOR_LIB=-Ilib INCLUDE_FOR_CLI=-Ilib INCLUDE_FOR_TEST="-Ilib -Icli"
+	dh_auto_test -- HAVE_RULES=yes INCLUDE_FOR_LIB="-Ilib -Iexternals/tinyxml" INCLUDE_FOR_CLI="-Ilib -Iexternals/tinyxml" INCLUDE_FOR_TEST="-Ilib -Icli -Iexternals/tinyxml"
 
 override_dh_auto_install:
-	dh_auto_install -- HAVE_RULES=yes TINYXML=-ltinyxml2 INCLUDE_FOR_LIB=-Ilib INCLUDE_FOR_CLI=-Ilib INCLUDE_FOR_TEST="-Ilib -Icli"
+	dh_auto_install -- HAVE_RULES=yes INCLUDE_FOR_LIB="-Ilib -Iexternals/tinyxml" INCLUDE_FOR_CLI="-Ilib -Iexternals/tinyxml" INCLUDE_FOR_TEST="-Ilib -Icli -Iexternals/tinyxml"
 
 %:
 	dh $@
