--- /usr/share/backintime/common/config.py	2011-07-05 21:46:55.000000000 +0100
+++ /tmp/config.py	2012-02-15 11:36:19.000000000 +0000
@@ -820,7 +820,7 @@
 				profile=''
 				if '1' != profile_id:
 					profile = "--profile-id %s" % profile_id
-				cmd = "/usr/bin/backintime %s --backup-job >/dev/null 2>&1" % profile
+				cmd = "/usr/bin/backintime %s --backup-job >/dev/null" % profile
 				if self.is_run_ionice_from_cron_enabled():
 					cmd = 'ionice -c2 -n7 ' + cmd
 				if self.is_run_nice_from_cron_enabled( profile_id ):
