diff -r -u armadillo-9.200.6-a/include/armadillo_bits/fn_trace.hpp armadillo-9.200.6-b/include/armadillo_bits/fn_trace.hpp
--- armadillo-9.200.6-a/include/armadillo_bits/fn_trace.hpp	2016-06-17 02:19:16.000000000 +1000
+++ armadillo-9.200.6-b/include/armadillo_bits/fn_trace.hpp	2016-06-17 02:19:16.000000000 +1000
@@ -82,7 +82,7 @@
 template<typename T1, typename T2>
 arma_warn_unused
 inline
-typename T1::elem_type
+typename enable_if2< is_cx<typename T1::elem_type>::no, typename T1::elem_type>::result
 trace(const Glue<T1, T2, glue_times>& X)
   {
   arma_extra_debug_sigprint();
