=== modified file 'dir2ogg'
--- dir2ogg	2007-12-15 21:50:57 +0000
+++ dir2ogg	2008-02-16 21:10:19 +0000
@@ -177,6 +177,8 @@
             key = convert and (key in convert and convert[key] or key) or key
             if not key in self.accept:
                continue
+            if not convert: # Hack for FLAC, which uses Vorbis tags
+                pass
             elif hasattr(val, 'text'):
                 val = val.text
             elif isinstance(val, list):

