--- ./cups-1.7.1/debian/cups-daemon.logrotate	2014-03-06 11:28:01.943676596 +0100
+++ ./cups-1.7.1.old/debian/cups-daemon.logrotate	2014-01-30 16:21:09.000000000 +0100
@@ -5,14 +5,14 @@
 	sharedscripts
 	prerotate
 		if [ -e /var/run/cups/cupsd.pid ]; then
-			invoke-rc.d --quiet cups stop > /dev/null 2>&1
+			invoke-rc.d --quiet cups stop > /dev/null
 			touch /var/run/cups/cupsd.stopped
 		fi
 	endscript
 	postrotate
 		if [ -e /var/run/cups/cupsd.stopped ]; then
 			rm /var/run/cups/cupsd.stopped
-			invoke-rc.d --quiet cups start > /dev/null 2>&1
+			invoke-rc.d --quiet cups start > /dev/null
 			sleep 10
 		fi
 	endscript
