--- debian/approx.cron.weekly.orig	2015-11-30 11:06:55.784478675 +0100
+++ debian/approx.cron.weekly	2015-11-30 11:06:55.784478675 +0100
@@ -2,6 +2,12 @@
 
 # Garbage collect the approx(8) cache
 
+# only use ionice if available, kfreebsd and hurd lack it
+unset IONICE
+if [ -x /usr/bin/ionice ]; then
+    IONICE="ionice -c3"
+fi
+
 if [ -x /usr/sbin/approx-gc ]; then
-    nice -n19 ionice -c3 /usr/sbin/approx-gc --quiet
+    nice -n19 $IONICE /usr/sbin/approx-gc --quiet
 fi
