diff -Nru dh-autoreconf-5/debian/control dh-autoreconf-5ubuntu1/debian/control
--- dh-autoreconf-5/debian/control	2011-06-14 20:58:39.000000000 +0300
+++ dh-autoreconf-5ubuntu1/debian/control	2012-02-07 01:44:59.000000000 +0200
@@ -9,6 +9,7 @@
 
 Package: dh-autoreconf
 Architecture: all
+Multi-Arch: foreign
 Depends: ${perl:Depends}, ${misc:Depends}, debhelper, autoconf,
          automake | automaken, libtool
 Enhances: debhelper, cdbs
