From 8f37c02063c6c9778fabcfdcac6ac3ffddeb14fc Mon Sep 17 00:00:00 2001
From: Anders Kaseorg <andersk@mit.edu>
Date: Fri, 8 Jul 2011 00:45:40 -0400
Subject: [PATCH 5/6] Support foreign multiarch packages

Signed-off-by: Anders Kaseorg <andersk@mit.edu>
---
 debsums |   14 +++++++++-----
 1 files changed, 9 insertions(+), 5 deletions(-)

diff --git a/debsums b/debsums
index 7161dd6..6e74cae 100755
--- a/debsums
+++ b/debsums
@@ -287,9 +287,10 @@ sub is_replaced
 
     unless ($installed{$pack}{ReplacedBy})
     {
-	return 0 unless $replaced{$pack};
+	(my $name = $pack) =~ s/:[^:]*$//;
+	return 0 unless $replaced{$name};
 
-	while (my ($ver, $p) = each %{$replaced{$pack}})
+	while (my ($ver, $p) = each %{$replaced{$name}})
 	{
 	    next unless $ver eq 'all'
 		or dpkg_cmp $installed{$pack}{Version}, $ver;
@@ -441,7 +442,7 @@ for (@ARGV)
     my $conffiles;
 
     # looks like a package name
-    unless (/[^a-z\d+.-]/ or /\.deb$/)
+    unless (/[^a-z\d+.:-]/ or /\.deb$/)
     {
 	$pack = $_;
 	unless (exists $installed{$pack})
@@ -465,8 +466,11 @@ for (@ARGV)
 	    {
 		# look for <pack>_<ver>_<arch>.deb or <pack>_<ver>.deb
 		# where <ver> may or may not contain an epoch
-		if (($deb) = grep -f, map +(glob "$dir/${pack}_$_.deb"),
-		    map +("${_}_$arch", "${_}_all", $_), @v)
+		my ($debname, $debarch);
+		($debname, $debarch) = ($pack =~ /^(.*):([^:]*)$/)
+		    or ($debname, $debarch) = ($pack, $arch);
+		if (($deb) = grep -f, map +(glob "$dir/${debname}_$_.deb"),
+		    map +("${_}_$debarch", "${_}_all", $_), @v)
 		{
 		    $deb =~ s!^\./+!!;
 		    last;
-- 
1.7.6

