=== modified file 'debsums'
--- debsums	2010-11-17 17:16:07 +0000
+++ debsums	2011-06-27 09:24:06 +0000
@@ -271,9 +271,12 @@
 	}
     }
 
+    my $sumsfile;
     for my $p (@{$installed{$pack}{ReplacedBy} || []})
     {
-	open S, "$DPKG/info/$p.md5sums" or next;
+	$sumsfile = `dpkg-query -c $p md5sums`;
+	chomp($sumsfile);
+	open S, $sumsfile or next;
 	while (<S>)
 	{
 	    if ($_ eq "$sum  $path\n")
@@ -460,7 +463,8 @@
 	}
 	else
 	{
-	    $sums = "$DPKG/info/$pack.md5sums";
+	    $sums = `dpkg-query -c $pack md5sums`;
+	    chomp($sums);
 	    unless (-f $sums or $config)
 	    {
 		if ($missing)
@@ -626,7 +630,8 @@
 
 	if ($generate{keep})
 	{
-	    my $target = "$DPKG/info/$pack.md5sums";
+	    my $target = `dpkg-query -c $pack md5sums`;
+	    chomp($target);
 	    copy $sums, $target
 		or die "$self: can't copy sums to $target ($!)\n";
 	}

