diff -Nur cjk-latex-4.5.1.orig/debian/changelog cjk-latex-4.5.1/debian/changelog
--- cjk-latex-4.5.1.orig/debian/changelog	2005-12-11 14:50:58.459814408 +0100
+++ cjk-latex-4.5.1/debian/changelog	2005-12-11 14:44:22.003084992 +0100
@@ -1,3 +1,10 @@
+cjk-latex (4.5.1-4.1) unstable; urgency=low
+
+  * Do not mess around with the generated file texmf.cnf; instead depend
+    on tex-common (>= 0.13) which provides the font support we need.
+
+ -- Frank Küster <frank@debian.org>  Sun, 11 Dec 2005 14:31:05 +0100
+
 cjk-latex (4.5.1-4) unstable; urgency=low
 
   * CVS update as of 2003-02-11.
diff -Nur cjk-latex-4.5.1.orig/debian/control cjk-latex-4.5.1/debian/control
--- cjk-latex-4.5.1.orig/debian/control	2005-12-11 14:50:58.462813952 +0100
+++ cjk-latex-4.5.1/debian/control	2005-12-11 14:43:23.812931240 +0100
@@ -7,7 +7,7 @@
 
 Package: cjk-latex
 Architecture: any
-Depends: ${shlibs:Depends}, tetex-base (>= 1.0-1), tetex-bin (>= 1.0.6-1.1)
+Depends: ${shlibs:Depends}, tetex-base (>= 1.0-1), tetex-bin (>= 1.0.6-1.1), tex-common (>= 0.13)
 Recommends: freetype1-tools
 Suggests: tfm-arphic-bsmi00lp, tfm-arphic-bkai00mp, tfm-arphic-gbsn00lp, tfm-arphic-gkai00mp, hbf-jfs56, hbf-cns40-b5, hbf-kanji48
 Description: A LaTeX macro package for CJK (Chinese/Japanese/Korean)
diff -Nur cjk-latex-4.5.1.orig/debian/postinst cjk-latex-4.5.1/debian/postinst
--- cjk-latex-4.5.1.orig/debian/postinst	2005-12-11 14:50:58.463813800 +0100
+++ cjk-latex-4.5.1/debian/postinst	2005-12-11 14:44:50.159804520 +0100
@@ -18,10 +18,3 @@
 
 #DEBHELPER#
 
-# Fixes /etc/texmf/texmf.cnf.
-# Remove the following line when Debian's teTeX package comes with
-# the same setting.
-if ! /bin/grep -q '^T1FONTS.*;$TEXMF/fonts/hbf//' /etc/texmf/texmf.cnf; then
-    /usr/bin/perl -pi -e 's|^(T1FONTS.*)$|$1;\$TEXMF/fonts/hbf//|;' \
-	/etc/texmf/texmf.cnf
-fi
