diff -ruN cdebconf-old/src/modules/frontend/newt/newt.c cdebconf/src/modules/frontend/newt/newt.c
--- cdebconf-old/src/modules/frontend/newt/newt.c	Tue May  4 22:55:26 2004
+++ cdebconf/src/modules/frontend/newt/newt.c	Tue Jun  1 00:08:45 2004
@@ -270,7 +270,7 @@
     // buttons
     int extra = 3;
     int format_note = 0;
-    char *full_description;;
+    char *full_description;
     int ret;
 #ifdef HAVE_LIBTEXTWRAP
     int flags = 0;
