diff --git a/debmany/debmany b/debmany/debmany
index dfea6e9..7a6029a 100755
--- a/debmany/debmany
+++ b/debmany/debmany
@@ -279,11 +279,11 @@ else
   if [ -z "$file" ]
   then
     debug "Mode3: Determining the path of '$package' using 'apt-get -q2 --print-uris download'" # comment
-    aptdata=`apt-get -q2 --print-uris download "$package" 2>/dev/null | grep "$package"_`
+    aptdata=`cd / && apt-get -q2 --print-uris download "$package" 2>/dev/null | grep "$package"_`
     if [ -z "$aptdata" ]
     then
       errormsg "There was an error looking for package '$package'."
-      aptdata=`apt-get -q2 --print-uris download "apt"  2>/dev/null | grep "apt"_`
+      aptdata=`cd / && apt-get -q2 --print-uris download "apt"  2>/dev/null | grep "apt"_`
       if [ $? -ne 0 ] ||  [ -z "$aptdata" ] ; then
       # If looking for apt fails then there is something amiss
         error "Apt-get might not be working properly Possible failure when running 'apt-get -q2 --print-uris download \"$package\"'"
