diff --git a/Makefile b/Makefile
index 9ab6801..8767276 100644
--- a/Makefile
+++ b/Makefile
@@ -18,10 +18,10 @@ perl-policy.sgml: version.ent
 	nsgmls -wall -gues $<
 
 %.html/index.html: %.sgml
-	LANG=C debiandoc2html $<
+	debiandoc2html -l en.UTF-8 $<
 
 %-1.html: %.sgml
-	LANG=C debiandoc2html -1 -b $*-1d $< && \
+	debiandoc2html -l en.UTF-8 -1 -b $*-1d $< && \
         mv $*-1d.html/index.html $*-1.html && \
         rmdir $*-1d.html
 
@@ -29,19 +29,19 @@ perl-policy.sgml: version.ent
 	tar -czf $(<:/index.html=.tar.gz) $(<:/index.html=)
 
 %.txt: %.sgml
-	LANG=C debiandoc2text $<
+	debiandoc2text -l en.UTF-8 $<
 
 %.txt.gz: %.txt
 	gzip -cf9 $< > $@
 
 %.ps: %.sgml
-	LANG=C debiandoc2latexps $<
+	debiandoc2latexps -l en.UTF-8 $<
 
 %.ps.gz: %.ps
 	gzip -cf9 $< > $@
 
 %.pdf: %.sgml
-	LANG=C debiandoc2latexpdf $<
+	debiandoc2latexpdf -l en.UTF-8 $<
 
 %.pdf.gz: %.pdf
 	gzip -cf9 $< > $@
