From: =?UTF-8?q?Stefan=20B=C3=BChler?= <stbuehler@web.de>
Date: Fri, 10 May 2013 10:55:14 +0200
Subject: configure cgit make options

---
 cgit.conf |    9 +++++++++
 1 file changed, 9 insertions(+)
 create mode 100644 cgit.conf

diff --git a/cgit.conf b/cgit.conf
new file mode 100644
index 0000000..cf9463c
--- /dev/null
+++ b/cgit.conf
@@ -0,0 +1,9 @@
+CGIT_SCRIPT_PATH=/usr/lib/cgi-bin
+CGIT_DATA_PATH=/usr/share/cgit
+V=1
+NO_OPENSSL=1
+SHA1_HEADER=<block-sha1/sha1.h>
+
+export V
+export NO_OPENSSL
+export SHA1_HEADER
