From: =?UTF-8?q?Stefan=20B=C3=BChler?= <stbuehler@web.de>
Date: Wed, 29 May 2013 14:33:50 +0200
Subject: Force man2html/groff to use UTF-8 as input encoding

---
 filters/html-converters/man2html |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/filters/html-converters/man2html b/filters/html-converters/man2html
index 1b28437..4975da4 100755
--- a/filters/html-converters/man2html
+++ b/filters/html-converters/man2html
@@ -1,5 +1,5 @@
 #!/bin/sh
 echo "<div style=\"font-family: monospace\">"
-groff -mandoc -T html -P -r -P -l | egrep -v '(<html>|<head>|<meta|<title>|</title>|</head>|<body>|</body>|</html>|<!DOCTYPE|"http://www.w3.org)'
+groff -mandoc -T html -K UTF-8 -P -r -P -l | egrep -v '(<html>|<head>|<meta|<title>|</title>|</head>|<body>|</body>|</html>|<!DOCTYPE|"http://www.w3.org)'
 echo "</div>"
 
