From: =?UTF-8?q?Stefan=20B=C3=BChler?= <stbuehler@web.de>
Date: Wed, 29 May 2013 14:34:15 +0200
Subject: Use debian binary name rst2html

---
 filters/html-converters/rst2html |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/filters/html-converters/rst2html b/filters/html-converters/rst2html
index c51f5be..a1ba574 100755
--- a/filters/html-converters/rst2html
+++ b/filters/html-converters/rst2html
@@ -1,2 +1,2 @@
 #!/bin/sh
-rst2html.py --template="$(dirname $0)/resources/rst-template.txt"
+rst2html --template="$(dirname $0)/resources/rst-template.txt"
