=== modified file 'bzrlib/remote.py'
--- bzrlib/remote.py	2010-02-17 17:11:16 +0000
+++ bzrlib/remote.py	2010-05-17 20:32:17 +0000
@@ -1229,10 +1229,11 @@
         return self._real_repository.add_inventory(revid, inv, parents)
 
     def add_inventory_by_delta(self, basis_revision_id, delta, new_revision_id,
-                               parents):
+            parents, basis_inv=None, propagate_caches=False):
         self._ensure_real()
         return self._real_repository.add_inventory_by_delta(basis_revision_id,
-            delta, new_revision_id, parents)
+            delta, new_revision_id, parents, basis_inv=basis_inv,
+            propagate_caches=propagate_caches)
 
     def add_revision(self, rev_id, rev, inv=None, config=None):
         self._ensure_real()

=== modified file 'debian/changelog'
--- debian/changelog	2010-03-24 21:35:30 +0000
+++ debian/changelog	2010-05-17 20:48:26 +0000
@@ -1,3 +1,11 @@
+bzr (2.1.1-1.1) unstable; urgency=low
+
+  * Non-maintainer upload with Jelmer's permission.
+  * Backport fix from bzr 2.2 that makes bzr-git and bzr-svn operations
+    work reliably again.  Closes: #582033
+
+ -- Steve Langasek <vorlon@debian.org>  Mon, 17 May 2010 13:48:25 -0700
+
 bzr (2.1.1-1) unstable; urgency=low
 
   * Remove bob2 from uploaders. Closes: #573427

