From 97ed81cc7fd48c4955d15efd5a341d6af4e96759 Mon Sep 17 00:00:00 2001
From: Martin Stransky <stransky@anakreon.cz>
Date: Sun, 3 Jan 2016 15:28:37 +0100
Subject: [PATCH] Fixed crashes in animation module

---
 src/komat/Berusky3d.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/komat/Berusky3d.h b/src/komat/Berusky3d.h
index 2b8bcc4..aef18c7 100644
--- a/src/komat/Berusky3d.h
+++ b/src/komat/Berusky3d.h
@@ -181,8 +181,8 @@ typedef struct _GLOBALNI_KONT_ANIMACE
 
   int *p_flag;                  // kofoluv flag
   void *p_param;                // kofolova end funkce
-  int param;
-  int param2;
+  size_ptr param;
+  size_ptr param2;
   END_FUNKCE p_endfce;
   int konec;
 
-- 
2.24.0

