diff --git a/dh_install b/dh_install
index 5a3a7f5..3aa748a 100755
--- a/dh_install
+++ b/dh_install
@@ -153,7 +153,7 @@ foreach my $package (@{$dh{DOPACKAGES}}) {
 			my @found = glob "$srcdir/$glob";
 			if (! compat(6)) {
 				# Fall back to looking in debian/tmp.
-				if (! @found || ! -e $found[0]) {
+				if (! @found || ! (-e $found[0] || -l $found[0])) {
 					@found = glob "debian/tmp/$glob";
 				}
 			}
