Index: aptitude/src/main.cc
===================================================================
--- aptitude.orig/src/main.cc
+++ aptitude/src/main.cc
@@ -659,16 +659,7 @@ int main(int argc, char *argv[])
   //cw::util::transcode_mbtow_err=handle_mbtow_error;
   //cw::util::transcode_wtomb_err=handle_wtomb_error;
 
-  // Use the C++ classes if possible, otherwise if ::global() is not called the
-  // internal state of std::locale is not initialized properly
-  //
-  // it can throw an exception if the locale defined in the environment is not
-  // valid
-  try {
-    std::locale::global(std::locale(""));
-  } catch (...) {
-    setlocale(LC_ALL, "");
-  }
+  setlocale(LC_ALL, "");
   bindtextdomain(PACKAGE, LOCALEDIR);
   textdomain(PACKAGE);
 
