--- abcde.svn.mkcue	2006-09-29 10:14:19.407635831 -0500
+++ abcde.svn.freedb2	2006-10-04 16:44:57.457570805 -0500
@@ -2222,12 +2222,22 @@
 				RESPONSECODE=$(head -n 1 "$ABCDETEMPDIR/cddbstat" | cut -f1 -d' ')
 				case "$RESPONSECODE" in
 				210)    # 210 OK, status information follows (until terminating `.')
-					rc=0;
+					rc=0
 					;;
-				501|*)  # 501 Illegal CDDB protocol level: <n>. 
+				501)    # 501 Illegal CDDB protocol level: <n>. 
 					CDDBPROTO=`expr $CDDBPROTO - 1`
 					;;
-				esac 
+				*)	# Try a cddb query, since freedb2.org doesn't support the stat or ver commands
+					# FreeDB TESTCD disc-id is used for query
+					$CDDBTOOL query $CDDBURL $CDDBPROTO $CDDBUSER $CDDBHOST 03015501 1 296 344 > "$ABCDETEMPDIR/cddbstat"
+					RESPONSECODE=$(head -n 1 "$ABCDETEMPDIR/cddbstat" | cut -f1 -d' ')
+					case "$RESPONSECODE" in
+						2??)	# Server responded, everything seems OK
+							rc=0
+							;;
+					esac
+					;;
+				esac
 			done
 			if test $rc -eq 1; then
 				CDDBAVAIL="n" 
