--- functions.orig	2020-03-08 21:41:03.000000000 +0800
+++ functions	2020-03-08 21:41:33.000000000 +0800
@@ -1199,7 +1199,9 @@
 		umount_on_exit /dev/shm
 		umount_on_exit /proc
 		umount_on_exit /proc/bus/usb
-		umount "$TARGET/proc" 2>/dev/null || true
+		if [ ! -h "$TARGET/proc" ]; then
+			umount "$TARGET/proc" 2>/dev/null || true
+		fi
 
 		# some container environment are used at second-stage, it already treats /proc and so on
 		if [ -z "$(ls -A "$TARGET/proc")" ]; then
