diff -r 395496810479 src/main.cc
--- a/src/main.cc	Fri Mar 12 01:32:14 2010 -0800
+++ b/src/main.cc	Tue Apr 02 17:26:59 2013 +0400
@@ -216,7 +216,7 @@
            "                strong dependencies.\n"));
   printf(_(" -S fname       Read the aptitude extended status info from fname.\n"));
   printf(_(" -u             Download new package lists on startup.\n"));
-  printf(_("                  (terminal interface only)"));
+  printf(_("                  (terminal interface only)\n"));
   printf(_(" -i             Perform an install run on startup.\n"));
   printf(_("                  (terminal interface only)"));
   printf("\n");
