From ffa86f5a1138d42b36c752a00b969e4d2696cafe Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lo=C3=AFc=20Minier?= <lool@debian.org>
Date: Mon, 17 Oct 2011 15:51:26 +0200
Subject: [PATCH] Drop gnome-ppp; built on armhf, no getcontext use

---
 Packages-arch-specific |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/Packages-arch-specific b/Packages-arch-specific
index 9358f9c..21d13c1 100644
--- a/Packages-arch-specific
+++ b/Packages-arch-specific
@@ -72,7 +72,6 @@ fdflush: alpha i386                                                   # i386/alp
 %gmod: i386                                                           # i386 specific
 %gpmudmon-applet: powerpc					      # PMUD is powerpc APM
 %gnome-mount: !linux-any                                              # superseded on linux by gvfs/g-d-u (#591429)
-%gnome-ppp: !armel					# no working getcontext()
 %gnu-efi: amd64 i386 ia64 lpia kfreebsd-amd64			      # EFI specific
 %gnumach: hurd-i386 i386 kfreebsd-i386				      # hurd kernel
 %google-perftools: amd64 i386 ia64 powerpc      		      # not yet ported to other archs
-- 
1.7.5.4

