From 0043253972d0cf6521f32f475acffe05258476df Mon Sep 17 00:00:00 2001
From: Shawn Landden <shawnlandden@gmail.com>
Date: Sat, 16 Jun 2012 08:20:12 +0300
Subject: [PATCH 2/3] Fix arm when using embedded v8

---
 debian/rules |    3 +++
 1 files changed, 3 insertions(+), 0 deletions(-)

diff --git a/debian/rules b/debian/rules
index 8a64f68..e4d43c3 100755
--- a/debian/rules
+++ b/debian/rules
@@ -92,6 +92,7 @@ GYP_DEFINES += \
 ifeq (armel,$(DEB_HOST_ARCH))
 AVOID_GCC_44 := 0
 GYP_DEFINES += \
+	v8_use_arm_eabi_hardfloat=false \
 	arm_float_abi=soft \
 	arm_thumb=0 \
 	armv7=0 \
@@ -100,6 +101,8 @@ GYP_DEFINES += \
 endif
 ifeq (armhf,$(DEB_HOST_ARCH))
 GYP_DEFINES += \
+	v8_use_arm_eabi_hardfloat=true \
+	arm_fpu=vfpv3 \
 	arm_float_abi=hard \
 	arm_thumb=1 \
 	armv7=1 \
-- 
1.7.2.5

