Index: NEWS
===================================================================
--- NEWS	(revision 792)
+++ NEWS	(working copy)
@@ -1,6 +1,6 @@
 cryptsetup (2:1.1.2-2) unstable; urgency=low
 
-  Cryptdisks init scripts changed their behaviour for failures at starting and
+  cryptdisks init scripts changed their behaviour for failures at starting and
   stopping encrypted devices. cryptdisks init script now raises a warning for
   failures at starting encrypted devices, and cryptdisks-early warns about
   failures at stopping encrypted devices.
@@ -110,7 +110,7 @@
 cryptsetup (2:1.0.4+svn16-1) unstable; urgency=high
 
   The --key-file=- argument has changed. If a --hash parameter is passed, it
-  will now be honored. This means that the decrypt_derived keyscript will in
+  will now be honoured. This means that the decrypt_derived keyscript will in
   some situations create a different key than previously meaning that any swap
   partitions that rely on the script will have to be recreated. To emulate the
   old behaviour, make sure that you pass "--hash=plain" to cryptsetup.
@@ -140,7 +140,7 @@
 cryptsetup (2:1.0.3-4) unstable; urgency=low
 
    Up to now, the us keymap was loaded at the passphrase prompt in the boot
-   process and ascii characters were always used. With this upload this is
+   process and ASCII characters were always used. With this upload this is
    fixed, meaning that the correct keymap is loaded and the keyboard is
    (optionally) set to UTF8 mode before the passphrase prompt.
 
@@ -149,7 +149,7 @@
    correct keymap loaded.
 
    Additionally, all four fields are now mandatory in /etc/crypttab. An entry
-   which does not contain all fields will be ignored. It is recommented to
+   which does not contain all fields will be ignored. It is recommended to
    set cipher, size and hash anyway, as defaults may change in the future.
 
    If you didn't set any of these settings yet, then you should add
@@ -160,7 +160,7 @@
 
 cryptsetup (2:1.0.2+1.0.3-rc2-2) unstable; urgency=low
 
-   The crypttab 'retry' has been renamed to 'tries' to reflect upstreams
+   The crypttab 'retry' has been renamed to 'tries' to reflect upstream's
    functionality. Default is 3 tries now, even if the option is not given.
    See the crypttab.5 manpage for more information.
 
