Index: README.remote
===================================================================
--- README.remote	(revision 792)
+++ README.remote	(working copy)
@@ -16,7 +16,7 @@
 the default when the busybox package is installed) to have busybox installed
 into the initramfs, and should not contain DROPBEAR=n, which would disable
 installation of dropbear to initramfs. If set to DROPBEAR=y, dropbear will
-beinstalled in any case; if DROPBEAR isn't set at all, then dropbear will only
+be installed in any case; if DROPBEAR isn't set at all, then dropbear will only
 be installed in case of an existing cryptroot setup.
 
 The host keys used for the initramfs are dropbear_dss_host_key and
@@ -88,7 +88,7 @@
 	"echo -ne \"secret\" >/lib/cryptsetup/passfifo"
 
 This example assumes that you have an extra known_hosts file 
-"~/.ssh/known_hosts.initramfs" which hold's the cryptroot system's host-key,
+"~/.ssh/known_hosts.initramfs" which holds the cryptroot system's host-key,
 that you have a file "~/id_rsa.initramfs" which holds the authorized-key for
 the cryptroot system, that the cryptroot system's name is
 "initramfshost.example.com", and that the cryptroot passphrase is "secret"
