Index: README.initramfs
===================================================================
--- README.initramfs	(revision 792)
+++ README.initramfs	(working copy)
@@ -16,7 +16,7 @@
 The installation will then take care of all the details and perform the
 necessary configuration for you, meaning that you should not have to 
 read the rest of this document to get a machine with an encrypted
-root fs up and running.
+rootfs up and running.
 
 However, if you are not planning to perform a new installation from scratch,
 the following information might be useful to you.
@@ -24,7 +24,7 @@
 
 3. Requirements
 ---------------
-In order to boot from an encrypted root fs, you need an initramfs-image
+In order to boot from an encrypted rootfs, you need an initramfs-image
 which includes the necessary kernel modules and scripts to setup
 the root device after the kernel has been initialized, but before the rest 
 of the operating system is booted.
@@ -70,7 +70,7 @@
 
 NOTE: the initramfs scripts default to using the sha256 hash function while
 the plain cryptsetup binary defaults to using the ripemd160 hash function.
-In order to ensure that the crypto setup works in a consistant manner, you
+In order to ensure that the crypto setup works in a consistent manner, you
 should make sure that the hash function is specified in the /etc/crypttab file
 if you are using regular dm-crypt (with LUKS the hash function to use is stored
 in the LUKS header).
