diff --git a/virt-subproc/adt-virt-qemu b/virt-subproc/adt-virt-qemu
index 5ae01b0..10c2b33 100755
--- a/virt-subproc/adt-virt-qemu
+++ b/virt-subproc/adt-virt-qemu
@@ -440,8 +440,9 @@ def get_cpuflag():
                         adtlog.debug('Detected KVM capable AMD host CPU, enabling nested KVM')
                         return ['-cpu', 'kvm64,+svm,+lahf_lm']
                     return []
+            return []
     except IOError as e:
-        logging.warning('Cannot read /proc/cpuinfo to detect CPU flags: %s' % e)
+        adtlog.warning('Cannot read /proc/cpuinfo to detect CPU flags: %s' % e)
         # fetching CPU flags isn't critical (only used to enable nested KVM),
         # so don't fail here
         return []
