diff -r 76a30419f4ad src/ario-main.c
--- a/src/ario-main.c	Mon Apr 04 16:42:48 2011 +0200
+++ b/src/ario-main.c	Mon Apr 04 16:43:36 2011 +0200
@@ -25,6 +25,7 @@
 #include <curl/curl.h>
 #include <libxml/parser.h>
 #include <glib/gi18n.h>
+#include <gcrypt.h>
 #include "lib/ario-conf.h"
 #include "preferences/ario-preferences.h"
 #include "shell/ario-shell.h"
@@ -111,6 +112,9 @@
         gtk_set_locale ();
         gtk_init (&argc, &argv);
 
+        /* Initialisation of libgcrypt */
+        gcry_check_version (NULL);
+
         /* Register Ario icons */
         ario_util_init_stock_icons ();
 
