diff -Nru dbus-1.8.8/debian/changelog dbus-1.8.8/debian/changelog
--- dbus-1.8.8/debian/changelog	2014-09-15 14:00:49.000000000 +0200
+++ dbus-1.8.8/debian/changelog	2014-10-06 15:06:46.000000000 +0200
@@ -1,3 +1,10 @@
+dbus (1.8.8-1.1) UNRELEASED; urgency=medium
+
+  * Non-maintainer upload.
+  * Make Build-Profiles field comply with the spec
+
+ -- Johannes Schauer <j.schauer@email.de>  Mon, 06 Oct 2014 15:05:56 +0200
+
 dbus (1.8.8-1) unstable; urgency=medium
 
   [ Michael Biebl ]
diff -Nru dbus-1.8.8/debian/control dbus-1.8.8/debian/control
--- dbus-1.8.8/debian/control	2014-09-15 14:00:49.000000000 +0200
+++ dbus-1.8.8/debian/control	2014-10-06 15:07:23.000000000 +0200
@@ -150,7 +150,7 @@
  for use in the Debian installer.
 
 Package: dbus-1-doc
-Build-Profiles: !stage1
+Build-Profiles: <!stage1>
 Section: doc
 Architecture: all
 Depends: ${misc:Depends}
