--- atomix-2.14.0.dist/src/main.c	2006-01-05 12:03:56.000000000 -0600
+++ atomix-2.14.0/src/main.c	2008-04-28 15:14:53.000000000 -0500
@@ -167,16 +167,15 @@
       NULL
     };
 
-  dlg = gtk_about_dialog_new ();
-  gtk_about_dialog_set_name (GTK_ABOUT_DIALOG(dlg), "Atomix");
-  gtk_about_dialog_set_version (GTK_ABOUT_DIALOG(dlg), VERSION);
-  gtk_about_dialog_set_comments (GTK_ABOUT_DIALOG(dlg), _("A puzzle game about atoms and molecules"));
-  gtk_about_dialog_set_website (GTK_ABOUT_DIALOG(dlg), "http://www.gnome.org/projects/atomix");
-  gtk_about_dialog_set_authors (GTK_ABOUT_DIALOG(dlg), authors);
-  gtk_about_dialog_set_artists (GTK_ABOUT_DIALOG(dlg), artists);
-  gtk_about_dialog_set_translator_credits (GTK_ABOUT_DIALOG(dlg), _("translator-credits"));
-
-  gtk_widget_show (dlg);
+  gtk_show_about_dialog(GTK_WINDOW(app->mainwin),
+  					"program-name", _("Atomix"),
+  					"version", VERSION,
+  					"comments", _("A puzzle game about atoms and molecules"),
+  					"website", _("http://www.gnome.org/projects/atomix"),
+  					"authors", authors,
+  					"artists", artists,
+  					"translator_credits", _("translator-credits"),
+  					NULL);
 }
 
 static gboolean on_app_destroy_event (GtkWidget *widget, GdkEvent *event,
