From 4bb5ca67b77584d7b178a11012fac10cecd86dd5 Mon Sep 17 00:00:00 2001
From: Osamu Aoki <osamu@debian.org>
Date: Sun, 9 Feb 2014 08:45:07 +0900
Subject: [PATCH] MAKE_LIBRARY_ARCHITECTUR

---
 Debian/Debhelper/Buildsystem/cmake.pm | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/Debian/Debhelper/Buildsystem/cmake.pm b/Debian/Debhelper/Buildsystem/cmake.pm
index db971db..4a210bc 100644
--- a/Debian/Debhelper/Buildsystem/cmake.pm
+++ b/Debian/Debhelper/Buildsystem/cmake.pm
@@ -39,9 +39,10 @@ sub new {
 sub configure {
 	my $this=shift;
 	my @flags;
-
+        my $multiarch=dpkg_architecture_value("DEB_HOST_MULTIARCH");
 	# Standard set of cmake flags
 	push @flags, "-DCMAKE_INSTALL_PREFIX=/usr";
+	push @flags, "-DCMAKE_LIBRARY_ARCHITECTURE=$multiarch";
 	push @flags, "-DCMAKE_VERBOSE_MAKEFILE=ON";
 	push @flags, "-DCMAKE_BUILD_TYPE=None";
 
-- 
1.9.rc1

