diff -Nru exiv2-0.23/debian/changelog exiv2-0.23/debian/changelog
--- exiv2-0.23/debian/changelog	2012-05-05 00:32:30.000000000 +0100
+++ exiv2-0.23/debian/changelog	2014-07-14 14:50:16.000000000 +0100
@@ -1,3 +1,10 @@
+exiv2 (0.23-1.1) unreleased; urgency=low
+
+  * Non-maintainer upload.
+  * Ensure configurey is updated for new architectures (Closes: #727295)
+
+ -- Wookey <wookey@debian.org>  Wed, 18 Jun 2014 21:55:26 +0000
+
 exiv2 (0.23-1) unstable; urgency=low
 
   * New upstream release
diff -Nru exiv2-0.23/debian/rules exiv2-0.23/debian/rules
--- exiv2-0.23/debian/rules	2012-01-24 20:19:10.000000000 +0000
+++ exiv2-0.23/debian/rules	2014-07-14 16:27:40.000000000 +0100
@@ -1,7 +1,7 @@
 #!/usr/bin/make -f
 
 %:
-	dh $@ --parallel
+	dh $@ --parallel --with autotools-dev
 
 override_dh_auto_configure:
 	dh_auto_configure -- --disable-rpath  $(shell dpkg-buildflags --export=configure)
