From 1bdddc8ba25cac2947cde4e5fcbac24f820478f1 Mon Sep 17 00:00:00 2001
From: Niels Thykier <niels@thykier.net>
Date: Sun, 16 Nov 2014 21:26:37 +0100
Subject: [PATCH 2/3] t/size: Ignore empty lines when counting size of a tool

Signed-off-by: Niels Thykier <niels@thykier.net>
---
 t/size | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/t/size b/t/size
index d8b9896..bf7847d 100755
--- a/t/size
+++ b/t/size
@@ -18,7 +18,7 @@ foreach my $file (@progs) {
 	while (<IN>) {
 		$cutting=1 if /^=/;
 		$cutting=0 if /^=cut/;
-		next if $cutting || /^(=|\s*\#)/;
+		next if $cutting || /^(=|\s*\#)/ || /^\s*+$/;
 		$lines++;
 		$maxlength=length($_) if length($_) > $maxlength;
 	}
-- 
2.1.3

