diff --git a/dh_installdocs b/dh_installdocs
index 9c82b5b4..4e6ba5d5 100755
--- a/dh_installdocs
+++ b/dh_installdocs
@@ -200,7 +200,11 @@ foreach my $package (@{$dh{DOPACKAGES}}) {
 			# Policy says that if you make your documentation
 			# directory a symlink, then you have to depend on
 			# the target.
-			addsubstvar($package, 'misc:Depends', "$dh{LINK_DOC} (= \${binary:Version})");
+			addsubstvar($package,
+						'misc:Depends',
+						package_arch($dh{LINK_DOC}) eq 'all' ?
+							"$dh{LINK_DOC} (= \${source:Version})" :
+							"$dh{LINK_DOC} (= \${binary:Version})");
 		}
 	}
 	else {
