diff --git a/debian/changelog b/debian/changelog
index 7f5be63..6129e55 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,10 @@
+clementine (0.7.1+dfsg-2.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Fix FTBFS by qFuzzyCompare function on armel, armhf and sh4.
+
+ -- Nobuhiro Iwamatsu <iwamatsu@debian.org>  Sat, 30 Jul 2011 03:53:04 +0900
+
 clementine (0.7.1+dfsg-2) unstable; urgency=low
 
   [ Thomas Pierson ]
diff --git a/src/widgets/nowplayingwidget.cpp b/src/widgets/nowplayingwidget.cpp
index 92c8877..ef92458 100644
--- a/src/widgets/nowplayingwidget.cpp
+++ b/src/widgets/nowplayingwidget.cpp
@@ -341,7 +341,7 @@ void NowPlayingWidget::DrawContents(QPainter *p) {
 
 void NowPlayingWidget::FadePreviousTrack(qreal value) {
   previous_track_opacity_ = value;
-  if (qFuzzyCompare(previous_track_opacity_, 0.0)) {
+  if (qFuzzyCompare(previous_track_opacity_, qreal(0.0))) {
     previous_track_ = QPixmap();
   }
 
