From dab32843a4bf233982e1f7262cf43622ba57a5e4 Mon Sep 17 00:00:00 2001
From: Peter <netservice@niekom.de>
Date: Thu, 20 Jun 2019 09:04:15 +0200
Subject: [PATCH 1/1] crontab fixup

---
 .etckeeper | 2 ++
 crontab    | 2 +-
 2 files changed, 3 insertions(+), 1 deletion(-)

diff --git a/.etckeeper b/.etckeeper
index 13ab841d..4ad8ff1c 100755
--- a/.etckeeper
+++ b/.etckeeper
@@ -152,6 +152,7 @@ maybe chmod 0644 'NetworkManager/system-connections/CybayGuest'
 maybe chmod 0600 'NetworkManager/system-connections/Dhcp-Default'
 maybe chmod 0644 'NetworkManager/system-connections/Doerflwirt'
 maybe chmod 0644 'NetworkManager/system-connections/EasyBox-151100'
+maybe chmod 0600 'NetworkManager/system-connections/FREE_WIFI@BAHNHOF.nmconnection'
 maybe chmod 0600 'NetworkManager/system-connections/FRITZ!BoxGast.nmconnection'
 maybe chmod 0644 'NetworkManager/system-connections/GUSTA-SC.nmconnection'
 maybe chmod 0644 'NetworkManager/system-connections/Garni-Sonne.nmconnection'
@@ -173,6 +174,7 @@ maybe chmod 0644 'NetworkManager/system-connections/T3BOARD16EU'
 maybe chmod 0644 'NetworkManager/system-connections/T3BOARD17'
 maybe chmod 0644 'NetworkManager/system-connections/T3DD-legacy'
 maybe chmod 0644 'NetworkManager/system-connections/Telekom'
+maybe chmod 0600 'NetworkManager/system-connections/Telekom.nmconnection'
 maybe chmod 0644 'NetworkManager/system-connections/Telekom_ICE'
 maybe chmod 0644 'NetworkManager/system-connections/ToediMobile'
 maybe chmod 0600 'NetworkManager/system-connections/Uni-Hannover'
diff --git a/crontab b/crontab
index c89cc0be..d75da44a 100644
--- a/crontab
+++ b/crontab
@@ -15,7 +15,7 @@ PATH=/usr/local/sbin:/usr/local/bin:/sbin:/bin:/usr/sbin:/usr/bin
 # |  |  |  |  .---- day of week (0 - 6) (Sunday=0 or 7) OR sun,mon,tue,wed,thu,fri,sat
 # |  |  |  |  |
 # *  *  *  *  * user-name command to be executed
-17 *	* * *	root    cd / && run-parts --report /etc/cron.hourly
+17 *	* * *	root	cd / && run-parts --report /etc/cron.hourly
 25 6	* * *	root	test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.daily )
 47 6	* * 7	root	test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.weekly )
 52 6	1 * *	root	test -x /usr/sbin/anacron || ( cd / && run-parts --report /etc/cron.monthly )
-- 
2.20.1

