diff -u dhcping-1.2/debian/rules dhcping-1.2/debian/rules
--- dhcping-1.2/debian/rules
+++ dhcping-1.2/debian/rules
@@ -37,6 +37,7 @@
 endif
 
 build:
+	cp -f /usr/share/misc/config.{sub,guess} .
 	test -f config.cache || ./configure
 	$(MAKE) CFLAGS="$(CFLAGS)"
 	touch stamp-build
diff -u dhcping-1.2/debian/control dhcping-1.2/debian/control
--- dhcping-1.2/debian/control
+++ dhcping-1.2/debian/control
@@ -2,6 +2,7 @@
 Section: admin
 Priority: optional
 Maintainer: Martin Schulze <joey@debian.org>
+Build-Depends: autotools-dev
 Standards-Version: 3.6.2
 
 Package: dhcping
diff -u dhcping-1.2/debian/changelog dhcping-1.2/debian/changelog
--- dhcping-1.2/debian/changelog
+++ dhcping-1.2/debian/changelog
@@ -1,3 +1,10 @@
+dhcping (1.2-4.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Update config.{guess,sub} to support new architectures (Closes: #533687)
+
+ -- Chen Baozi <baozich@gmail.com>  Tue, 03 Jun 2014 11:04:13 +0800
+
 dhcping (1.2-4) unstable; urgency=low
 
   * Fixed typo in NMU-Disclaimer, thanks to Petter Reinholdtsen (See:
