--- cryptdisks_start.orig	2009-04-29 22:02:47.000000000 +0800
+++ /usr/sbin/cryptdisks_start	2009-04-29 22:03:33.000000000 +0800
@@ -27,13 +27,17 @@
 mount_fs
 
 found="no"
-egrep -v "^[[:space:]]*(#|$)" "$TABFILE" | while read dst src key opts; do
-	if [ "$1" = "$dst" ]; then
-		found="yes"
-		handle_crypttab_line_start "$dst" "$src" "$key" "$opts" <&3
-		exit 0
-	fi
-done 3<&1
+found=$(
+	egrep -v "^[[:space:]]*(#|$)" "$TABFILE" | while read dst src key opts; do
+		if [ "$1" = "$dst" ]; then
+			found="yes"
+			handle_crypttab_line_start "$dst" "$src" "$key" "$opts" <&3
+			echo yes
+			exit 0
+		fi
+	done 3<&1
+	echo no
+)
 
 if [ "$found" = "no" ]; then
 	device_msg "$1" "failed, not found in crypttab"
--- cryptdisks_stop.orig	2009-04-29 22:08:46.000000000 +0800
+++ /usr/sbin/cryptdisks_stop	2009-04-29 22:09:33.000000000 +0800
@@ -28,13 +28,17 @@
 log_action_begin_msg "Stopping crypto disk"
 
 found="no"
-egrep -v "^[[:space:]]*(#|$)" "$TABFILE" | while read dst src key opts; do
-	if [ "$1" = "$dst" ]; then
-		found="yes"
-		handle_crypttab_line_stop "$dst" "$src" "$key" "$opts" <&3
-		exit 0
-	fi
-done 3<&1
+found=$(
+	egrep -v "^[[:space:]]*(#|$)" "$TABFILE" | while read dst src key opts; do
+		if [ "$1" = "$dst" ]; then
+			found="yes"
+			handle_crypttab_line_stop "$dst" "$src" "$key" "$opts" <&3
+			echo yes
+			exit 0
+		fi
+	done 3<&1
+	echo no
+)
 
 if [ "$found" = "no" ]; then
 	device_msg "$1" "failed, not found in crypttab"
