diff -Naur bpython/cli.py bpython.new/cli.py
--- bpython/cli.py	2009-07-19 00:12:17.000000000 +0200
+++ bpython.new/cli.py	2009-07-19 00:26:11.000000000 +0200
@@ -338,7 +338,7 @@
         if not OPTS.arg_spec:
             return
 
-        pythonhist = os.path.expanduser('~/.pythonhist')
+        pythonhist = os.path.expanduser(OPTS.hist_file)
         if os.path.exists(pythonhist):
             with codecs.open(pythonhist, 'r', getpreferredencoding(),
                              'ignore') as hfile:
@@ -1961,7 +1961,7 @@
 
     repl.repl()
     if OPTS.hist_length:
-        histfilename = os.path.expanduser('~/.pythonhist')
+        histfilename = os.path.expanduser(OPTS.hist_file)
         with codecs.open(histfilename, 'w', getpreferredencoding(),
                          'ignore') as hfile:
             hfile.writelines(repl.rl_hist[-OPTS.hist_length:])
