Index: acm-6.0+20200416/src/acm/mouse.h
===================================================================
--- acm-6.0+20200416.orig/src/acm/mouse.h
+++ acm-6.0+20200416/src/acm/mouse.h
@@ -35,14 +35,16 @@
  * high precision). Ailerons and elevator are in thei neutral position when the
  * mouse pointer is exactly in the middle of the window.
  */
-enum {
+enum mouse_STICK_MODE {
 	/** Linear. */
 	mouse_FAST,
 	/** Quadratic. */
 	mouse_NORMAL,
 	/** Cubic. */
 	mouse_PRECISE
-} mouse_stick_mode;
+};
+
+EXTERN enum mouse_STICK_MODE mouse_stick_mode;
 
 /**
  * Get stick input from mouse and set c->pitchComm, c->rollComm, and c->steerComm
