diff --git a/build/boot/x86/menu.cfg b/build/boot/x86/menu.cfg
index 207934e8c..eaaa20967 100644
--- a/build/boot/x86/menu.cfg
+++ b/build/boot/x86/menu.cfg
@@ -16,6 +16,8 @@ menu begin advanced
 	include ${SYSDIR}adtxt.cfg
 	include ${SYSDIR}adspkgtk.cfg
 	include ${SYSDIR}adspk.cfg
+	include ${SYSDIR}addark.cfg
+	include ${SYSDIR}addarkgtk.cfg
 menu end
 include ${SYSDIR}x86menu.cfg
 label help
diff --git a/build/boot/x86/x86.cfg b/build/boot/x86/x86.cfg
index 8a2e19d3f..355e4ca9c 100644
--- a/build/boot/x86/x86.cfg
+++ b/build/boot/x86/x86.cfg
@@ -17,6 +17,8 @@ menu begin advanced
         include ${SYSDIR}x86adtxt.cfg
         include ${SYSDIR}x86asgtk.cfg
         include ${SYSDIR}x86adspk.cfg
+	include ${SYSDIR}x86addark.cfg
+	include ${SYSDIR}x86addarkgtk.cfg
 menu end
 label help
         menu label ^Help
diff --git a/build/util/syslinux-cfgs b/build/util/syslinux-cfgs
index 1041b2aa6..9d91cd4d4 100755
--- a/build/util/syslinux-cfgs
+++ b/build/util/syslinux-cfgs
@@ -38,8 +38,10 @@ create_standard_config() {
 	cp "$SRC"/syslinux.cfg "$SRC"/menu.cfg "$SRC"/stdmenu.cfg \
 	   "$SRC"/prompt.cfg "$SRC"/exithelp.cfg "$DST"/
 	cp "$SRC"/{,ad,rq}txt.cfg "$DST"/
+	cp "$SRC"/addark.cfg "$SRC"/rqdark.cfg "$DST"/
 	if [ -n "$INCLUDE_GTK" ]; then
 		cp "$SRC"/{ad,rq,}{,spk}gtk.cfg "$DST"/
+		cp "$SRC"/addarkgtk.cfg "$SRC"/rqdarkgtk.cfg "$DST"/
 	fi
 	if [ "$IS_PURE_GTK" = 1 ]; then
 		cp "$SRC"/{,ad,rq}spk.cfg "$DST"/
@@ -53,6 +55,7 @@ case $TYPE in
 		rm -f "$DST"/*gtk.cfg
 		rm -f "$DST"/*spkgtk.cfg
 		rm -f "$DST"/*spgtk.cfg
+		rm -f "$DST"/*darkgtk.cfg
 	fi
 	if [ "$IS_PURE_GTK" != 1 ]; then
 		rm -f "$DST"/*spk.cfg
