diff -Nru ca-certificates-20140325/sbin/update-ca-certificates ca-certificates-20140325selinux1/sbin/update-ca-certificates
--- ca-certificates-20140325/sbin/update-ca-certificates	2014-03-13 13:43:00.000000000 +0100
+++ ca-certificates-20140325selinux1/sbin/update-ca-certificates	2014-03-29 11:40:41.000000000 +0100
@@ -146,6 +146,8 @@
 
 chmod 0644 "$TEMPBUNDLE"
 mv -f "$TEMPBUNDLE" "$CERTBUNDLE"
+# Restore proper SELinux label after moving the file
+[ -x /sbin/restorecon ] && /sbin/restorecon "$CERTBUNDLE"
 
 echo "$ADDED_CNT added, $REMOVED_CNT removed; done."
 
