Description: Fix compiling with -Werror=format-security.
Author: Simon Ruderich <simon@ruderich.org>
Last-Update: 2012-07-11

--- ecryptfs-utils-98.orig/src/libecryptfs/module_mgr.c
+++ ecryptfs-utils-98/src/libecryptfs/module_mgr.c
@@ -813,7 +813,7 @@ int ecryptfs_process_decision_graph(stru
 	}
 	ad_cursor = ad_cursor->next;
 	ad_cursor->next = NULL;
-	if ((rc = asprintf(&ad_cursor->name,
+	if ((rc = asprintf(&ad_cursor->name, "%s",
 			   key_module_select_node.mnt_opt_names[0])) == -1) {
 		rc = -ENOMEM;
 		goto out_free_allowed_duplicates;
