Index: resources.dbk
===================================================================
--- resources.dbk	(revision 7003)
+++ resources.dbk	(working copy)
@@ -573,12 +573,22 @@
 <literal>binary</literal> packages.
 </para>
 <para>
-Source packages consist of either two or three files: a
-<filename>.dsc</filename> file, and either a <filename>.tar.gz</filename> file
-or both an <filename>.orig.tar.gz</filename> and a
-<filename>.diff.gz</filename> file.
+Source packages can be either version "1.0" or version 3.  Version
+3 has two variants: "3.0 (native)" and "3.0 (quilt)".
 </para>
 <para>
+Source packages contains files such as:
+a <filename>.dsc</filename> file,
+a <filename>.tar.{gz,bz2,lzma}</filename> file for native packages,
+an <filename>.orig.tar.{gz,bz2,lzma}</filename> file (non-native packages),
+multiple <filename>.orig-&lt;component&gt;.tar.{gz,bz2,lzma}</filename>
+files,
+a <filename>.debian.tar.{gz,bz2,lzma}</filename> file (only in version
+"3.0 (quilt)"),
+and a <filename>.diff.gz</filename> file (for version 1).
+Version 1 supports only <filename>.gz</filename> compression.
+</para>
+<para>
 If a package is developed specially for Debian and is not distributed outside
 of Debian, there is just one <filename>.tar.gz</filename> file which contains
 the sources of the program.  If a package is distributed elsewhere too, the
