diff -Nur davfs2-0.2.7/debian/changelog davfs2-0.2.8/debian/changelog
--- davfs2-0.2.7/debian/changelog	2006-03-14 10:25:08.000000000 +0100
+++ davfs2-0.2.8/debian/changelog	2006-03-15 10:26:28.000000000 +0100
@@ -1,3 +1,12 @@
+davfs2 (0.2.8-1) unstable; urgency=low
+
+  * New upstream version:
+    - fixes pidfile directory creation (closes: #348796);
+    - fixes iis[56] misbehave (closes: #340799).
+  * Build with neon 0.25 (closes: #356963).
+
+ -- Luciano Bello <luciano@linux.org.ar>  Tue, 14 Mar 2006 10:47:18 +0100
+
 davfs2 (0.2.7-1) unstable; urgency=low
 
   * New upstream version 0.2.7.
diff -Nur davfs2-0.2.7/debian/control davfs2-0.2.8/debian/control
--- davfs2-0.2.7/debian/control	2006-03-14 10:25:08.000000000 +0100
+++ davfs2-0.2.8/debian/control	2006-03-14 10:48:14.000000000 +0100
@@ -3,7 +3,7 @@
 Priority: extra
 Maintainer: Luciano Bello <luciano@linux.org.ar>
 Uploaders: Luk Claes <luk@debian.org>
-Build-Depends: debhelper (>= 4.0.0), libssl-dev, libxml2-dev, libneon24-dev
+Build-Depends: debhelper (>= 4.0.0), libssl-dev, libxml2-dev, libneon25-dev
 Standards-Version: 3.6.2
 
 Package: davfs2
