Index: pkgs.dbk
===================================================================
--- pkgs.dbk	(revision 8898)
+++ pkgs.dbk	(working copy)
@@ -1947,6 +1947,11 @@
 <itemizedlist>
 <listitem>
 <para>
+Upload fixing only release-critical bugs older than 7 days, with no maintainer activity on the bug for 7 days and no indication that a fix is in progress: 0 days
+</para>
+</listitem>
+<listitem>
+<para>
 Upload fixing only release-critical bugs older than 7 days: 2 days
 </para>
 </listitem>
