--- WebCore/platform/network/soup/DNSSoup.cpp-orig	2010-04-29 01:26:06.048725419 +1000
+++ WebCore/platform/network/soup/DNSSoup.cpp	2010-04-29 14:28:55.731471025 +1000
@@ -36,6 +36,8 @@
 void prefetchDNS(const String& hostname)
 {
 #ifdef HAVE_LIBSOUP_2_29_90
+    if (hostname.isEmpty())
+        return;
     String uri = "http://"+hostname;
     GOwnPtr<SoupURI> soupURI(soup_uri_new(uri.utf8().data()));
     // We may get invalid hostnames, so NULL-check here.
