diff --git a/dpkg-deb/build.c b/dpkg-deb/build.c
index 655aa55..b15ab90 100644
--- a/dpkg-deb/build.c
+++ b/dpkg-deb/build.c
@@ -422,7 +422,7 @@ typedef void filenames_feed_func(const char *dir, int fd_out);
 static void
 tarball_pack(const char *dir, filenames_feed_func *tar_filenames_feeder,
              time_t timestamp,
-             struct compress_params *tar_compress_params, int fd_out)
+             struct compress_params *tar_compress_params, int fd_out, bool ctrl)
 {
   int pipe_filenames[2], pipe_tarball[2];
   pid_t pid_tar, pid_comp;
@@ -448,6 +448,7 @@ tarball_pack(const char *dir, filenames_feed_func *tar_filenames_feeder,
 
     execlp(TAR, "tar", "-cf", "-", "--format=gnu",
                        "--mtime", mtime, "--clamp-mtime",
+                       ctrl ? "--mode=go=rX,u+rw,a-s" : "--mode=+",
                        "--null", "--no-unquote",
                        "--no-recursion", "-T", "-", NULL);
     ohshite(_("unable to execute %s (%s)"), "tar -cf", TAR);
@@ -572,7 +573,7 @@ do_build(const char *const *argv)
 
   /* Fork a tar to package the control-section of the package. */
   tarball_pack(ctrldir, control_treewalk_feed, timestamp,
-               &control_compress_params, gzfd);
+               &control_compress_params, gzfd, 1);
 
   free(ctrldir);
 
@@ -633,7 +634,7 @@ do_build(const char *const *argv)
   }
 
   /* Pack the directory into a tarball, feeding files from the callback. */
-  tarball_pack(dir, file_treewalk_feed, timestamp, &compress_params, gzfd);
+  tarball_pack(dir, file_treewalk_feed, timestamp, &compress_params, gzfd, 0);
 
   /* Okay, we have data.tar as well now, add it to the ar wrapper. */
   if (deb_format.major == 2) {
