--- installkernel.orig	2024-06-09 15:25:40.000000000 +0800
+++ installkernel	2024-07-04 17:22:48.201433536 +0800
@@ -63,11 +63,14 @@
   fi
 }
 
-if [ "$(basename $img)" = "vmlinux" ] ; then
-  img_dest=vmlinux
-else
-  img_dest=vmlinuz
-fi
+case "$(basename $img)" in
+  "vmlinux"|"Image")
+    img_dest=vmlinux
+    ;;
+  *)
+    img_dest=vmlinuz
+    ;;
+esac
 updatever $img_dest "$img"
 updatever System.map "$map"
 