diff -u alsamixergui-0.9.0rc2-1/debian/changelog alsamixergui-0.9.0rc2-1/debian/changelog
--- alsamixergui-0.9.0rc2-1/debian/changelog
+++ alsamixergui-0.9.0rc2-1/debian/changelog
@@ -1,3 +1,14 @@
+alsamixergui (0.9.0rc2-1-10.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Apply patch from Logan Rosen to fix FTBFS with glibc 2.32.
+    (Closes: #979448)
+  * debhelper compat 5 -> 7. (Closes: #965408)
+  * Architecture: linux-any (Closes: #638240)
+  * Remove the dead Homepage: link. (Closes: #608644)
+
+ -- Adrian Bunk <bunk@debian.org>  Sun, 07 Nov 2021 18:32:38 +0200
+
 alsamixergui (0.9.0rc2-1-10) unstable; urgency=medium
 
   * Add alsa-utils to Depends (closes: #498168)
diff -u alsamixergui-0.9.0rc2-1/debian/compat alsamixergui-0.9.0rc2-1/debian/compat
--- alsamixergui-0.9.0rc2-1/debian/compat
+++ alsamixergui-0.9.0rc2-1/debian/compat
@@ -1 +1 @@
-5
+7
diff -u alsamixergui-0.9.0rc2-1/debian/control alsamixergui-0.9.0rc2-1/debian/control
--- alsamixergui-0.9.0rc2-1/debian/control
+++ alsamixergui-0.9.0rc2-1/debian/control
@@ -4,10 +4,9 @@
 Maintainer: Paul Brossier <piem@debian.org>
 Build-Depends: debhelper (>> 4.1.0), cdbs, dh-buildinfo, docbook-to-man, libfltk1.1-dev (>= 1.1.6-6), libasound2-dev (>= 0.9.0)
 Standards-Version: 3.9.8
-Homepage: http://www.iua.upf.es/~mdeboer/projects/alsamixergui/
 
 Package: alsamixergui
-Architecture: any
+Architecture: linux-any
 Provides: x-audio-mixer
 Depends: ${shlibs:Depends}, ${misc:Depends}, alsa-utils
 Description: graphical soundcard mixer for ALSA soundcard driver
diff -u alsamixergui-0.9.0rc2-1/src/alsamixer.cxx alsamixergui-0.9.0rc2-1/src/alsamixer.cxx
--- alsamixergui-0.9.0rc2-1/src/alsamixer.cxx
+++ alsamixergui-0.9.0rc2-1/src/alsamixer.cxx
@@ -2086,7 +2086,7 @@
 mixer_signal_handler (int signal)
 {
   if (signal != SIGSEGV)
-    mixer_abort (ERR_SIGNAL, sys_siglist[signal], 0);
+    mixer_abort (ERR_SIGNAL, strsignal(signal), 0);
   else
     {
       fprintf (stderr, "\nSegmentation fault.\n");
