Index: debian/ant-optional.poms
===================================================================
--- debian/ant-optional.poms	(revision 14163)
+++ debian/ant-optional.poms	(working copy)
@@ -14,4 +14,5 @@
 src/etc/poms/ant-jsch/pom.xml
 src/etc/poms/ant-junit/pom.xml
 src/etc/poms/ant-swing/pom.xml
+src/etc/poms/ant-testutil/pom.xml
 
Index: debian/rules
===================================================================
--- debian/rules	(revision 14163)
+++ debian/rules	(working copy)
@@ -40,7 +40,7 @@
 	# remove -SNAPSHOT from the versions in the Maven poms
 	perl -p -i -e 's/-SNAPSHOT//g' `find src/etc/poms/ -name pom.xml`
 	# the rest
-	/bin/sh bootstrap/bin/ant -Dant.build.javac.source=1.4 -Dant.build.javac.target=1.4 -Dbuild.classpath=${CLASSPATH} -propertyfile debian/ant.properties jars
+	/bin/sh bootstrap/bin/ant -Dant.build.javac.source=1.4 -Dant.build.javac.target=1.4 -Dbuild.classpath=${CLASSPATH} -propertyfile debian/ant.properties jars test-jar
 	ln -sf /usr/share/java/xml-apis.jar $(DEB_BUILDDIR)/lib/xml-apis.jar
 	ln -sf /usr/share/java/xercesImpl.jar $(DEB_BUILDDIR)/lib/xercesImpl.jar
 	touch stamp-build-ant
@@ -86,7 +86,7 @@
 	mh_installjar -pant-optional -l src/etc/poms/ant-jsch/pom.xml build/lib/ant-jsch.jar /usr/share/ant/lib/ant-jsch.jar
 	mh_installjar -pant-optional -l src/etc/poms/ant-junit/pom.xml build/lib/ant-junit.jar /usr/share/ant/lib/ant-junit.jar
 	mh_installjar -pant-optional -l src/etc/poms/ant-swing/pom.xml build/lib/ant-swing.jar /usr/share/ant/lib/ant-swing.jar
-	#mh_installjar -pant-optional src/etc/poms/ant-testutil/pom.xml ant-testutil/build/ant-testutil-$(VERSION).jar
+	mh_installjar -pant-optional -l src/etc/poms/ant-testutil/pom.xml build/lib/ant-testutil.jar /usr/share/ant/lib/ant-testutil.jar
 	#mh_installjar -pant-optional src/etc/poms/ant-jai/pom.xml ant-jai/build/ant-jai-$(VERSION).jar
 	#mh_installjar -pant-optional src/etc/poms/ant-netrexx/pom.xml ant-netrexx/build/ant-netrexx-$(VERSION).jar
 	#mh_installjar -pant-optional src/etc/poms/ant-starteam/pom.xml ant-starteam/build/ant-starteam-$(VERSION).jar
