From 9de31b549cda1d94d312419badca7867124a3c19 Mon Sep 17 00:00:00 2001
From: Jeremy Bicha <jbicha@ubuntu.com>
Date: Sat, 21 Jan 2017 03:54:19 +0000
Subject: [PATCH] Build-Depend on rarian-compat

Closes: #852047
---
 debian/control | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index 377002e..fd80f13 100644
--- a/debian/control
+++ b/debian/control
@@ -6,8 +6,9 @@ Uploaders: Thibaut Paumard <thibaut@debian.org>
 Vcs-Git: https://anonscm.debian.org/git/pkg-a11y/dasher.git
 Vcs-Browser: https://anonscm.debian.org/gitweb/?p=pkg-a11y/dasher.git
 Build-Depends: debhelper (>= 9.0.0),
-	       gnome-common,
+               gnome-common,
                gnome-pkg-tools (>= 0.6),
+               rarian-compat,
                dpkg-dev (>= 1.13.19),
                intltool (>= 0.40.1),
                libglib2.0-dev (>= 2.16.0),
-- 
2.10.2

