=== modified file 'apt-pkg/acquire-item.cc'
--- apt-pkg/acquire-item.cc	2011-04-08 11:57:04 +0000
+++ apt-pkg/acquire-item.cc	2011-04-16 09:00:02 +0000
@@ -1504,7 +1504,7 @@ void pkgAcqMetaIndex::Failed(string Mess
 
    /* Always move the meta index, even if gpgv failed. This ensures
     * that PackageFile objects are correctly filled in */
-   {
+   if (FileExists(DestFile)) {
       string FinalFile = _config->FindDir("Dir::State::lists");
       FinalFile += URItoFileName(RealURI);
       /* InRelease files become Release files, otherwise

=== modified file 'debian/changelog'
--- debian/changelog	2011-04-15 12:28:28 +0000
+++ debian/changelog	2011-04-16 09:02:33 +0000
@@ -1,3 +1,10 @@
+apt (0.8.14.1) UNRELEASED; urgency=low
+
+  * apt-pkg/acquire-item.cc:
+    - Only try to rename existing Release files (Closes: #622912)
+
+ -- Julian Andres Klode <jak@debian.org>  Sat, 16 Apr 2011 11:00:19 +0200
+
 apt (0.8.14) unstable; urgency=low
 
   [ Julian Andres Klode ]

