diff -ru autofs-3.9.99-4.0.0pre10~/debian/scripts/vars.build autofs-3.9.99-4.0.0pre10/debian/scripts/vars.build
--- autofs-3.9.99-4.0.0pre10~/debian/scripts/vars.build	Sun May 19 18:02:04 2002
+++ autofs-3.9.99-4.0.0pre10/debian/scripts/vars.build	Sun May 19 18:02:32 2002
@@ -1,7 +1,7 @@
 #!/usr/bin/make -f
 
 sed_cmd=''
-cat $1 | while read; do
+cat $1 | while read REPLY; do
 	case "$REPLY" in
 		\#*|"")	continue;;
 		*)
