--- /usr/share/awstats/plugins/geoipfree.pm	2009-03-03 14:59:31.000000000 +0300
+++ geoipfree.pm	2009-03-03 14:52:57.000000000 +0300
@@ -51,7 +51,7 @@
 	# ENTER HERE CODE TO DO INIT PLUGIN ACTIONS
 	debug(" Plugin geoipfree: InitParams=$InitParams",1);
 	%TmpDomainLookup=();
-	$gi = Geo::IPfree::new();
+	$gi = Geo::IPfree->new();
 #	$gi->Faster; 	# Do not enable Faster as the Memoize module is rarely available
 	# ----->
 
