diff -Nru debarchiver-0.11.5/debian/changelog debarchiver-0.11.5+nmu1/debian/changelog
--- debarchiver-0.11.5/debian/changelog	2020-05-03 20:10:21.000000000 +0100
+++ debarchiver-0.11.5+nmu1/debian/changelog	2020-11-10 22:47:54.000000000 +0000
@@ -1,3 +1,10 @@
+debarchiver (0.11.5+nmu1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Add Build-Depends on libpod-parser-perl. Closes: #961154.
+
+ -- Dominic Hargreaves <dom@earth.li>  Tue, 10 Nov 2020 22:47:54 +0000
+
 debarchiver (0.11.5) unstable; urgency=medium
 
   * Added source format "3.0 (native)".
diff -Nru debarchiver-0.11.5/debian/control debarchiver-0.11.5+nmu1/debian/control
--- debarchiver-0.11.5/debian/control	2020-05-03 20:10:21.000000000 +0100
+++ debarchiver-0.11.5+nmu1/debian/control	2020-11-10 22:46:16.000000000 +0000
@@ -2,7 +2,7 @@
 Section: devel
 Priority: optional
 Maintainer: Ola Lundqvist <opal@debian.org>
-Build-Depends-Indep: perl (>= 5.6.0-16), po4a
+Build-Depends-Indep: perl (>= 5.6.0-16), po4a, libpod-parser-perl
 Build-Depends: debhelper-compat (= 12)
 Standards-Version: 4.3.0
 
