diff --git a/dpkg/prerm b/dpkg/prerm
index c400ce0..ed18130 100644
--- a/dpkg/prerm
+++ b/dpkg/prerm
@@ -30,7 +30,7 @@ dbc_go(){
         else
             need_admin_pw="yup"
             # Note: dpkg/config has the same code
-            if [ $dbc_dbtype = mysql ] && _dbc_mysql_get_debian_sys_maint ; then
+            if [ "$dbc_dbtype" = mysql ] && _dbc_mysql_get_debian_sys_maint ; then
                 db_set $dbc_package/$dbc_dbtype/admin-user "$dbc_dbadmin"
                 db_fset $dbc_package/$dbc_dbtype/admin-user seen true
                 db_set $dbc_package/$dbc_dbtype/admin-pass "$dbc_dbadmpass"
