only in patch2:
unchanged:
--- exiv2-0.15.orig/src/types.cpp
+++ exiv2-0.15/src/types.cpp
@@ -44,6 +44,7 @@
 #include <ctime>
 #include <cstdio>
 #include <cassert>
+#include <cstring>
 
 // *****************************************************************************
 // class member definitions
only in patch2:
unchanged:
--- exiv2-0.15.orig/src/basicio.cpp
+++ exiv2-0.15/src/basicio.cpp
@@ -45,6 +45,7 @@
 #include <string>
 #include <memory>
 #include <cassert>
+#include <cstring>
 #include <cstdio>                       // for remove, rename
 #include <cstdlib>                      // for alloc, realloc, free
 #include <sys/types.h>                  // for stat, chmod
@@ -427,7 +428,7 @@
             // Minimum size for 1st block is 32kB
             long size  = std::max(32768 * (1 + need / 32768), size_);
             byte* data = (byte*)std::malloc(size);
-            std::memcpy(data, data_, size_);
+            memcpy(data, data_, size_);
             data_ = data;
             sizeAlloced_ = size;
             isMalloced_ = true;
only in patch2:
unchanged:
--- exiv2-0.15.orig/src/iotest.cpp
+++ exiv2-0.15/src/iotest.cpp
@@ -34,6 +34,7 @@
 #include "futils.hpp"
 #include "basicio.hpp"
 #include <iostream>
+#include <cstring>
 
 using Exiv2::byte;
 using Exiv2::BasicIo;
only in patch2:
unchanged:
--- exiv2-0.15.orig/src/value.hpp
+++ exiv2-0.15/src/value.hpp
@@ -37,6 +37,7 @@
 
 // + standard includes
 #include <string>
+#include <cstring>
 #include <vector>
 #include <iostream>
 #include <sstream>
only in patch2:
unchanged:
--- exiv2-0.15.orig/src/makernote.cpp
+++ exiv2-0.15/src/makernote.cpp
@@ -39,6 +39,7 @@
 #include "error.hpp"
 
 // + standard includes
+#include <cstring>
 #include <string>
 #include <sstream>
 #include <iomanip>
