--- Makefile	2008-09-26 00:58:25.000000000 +0200
+++ Makefile	2008-10-16 20:59:35.000000000 +0200
@@ -15,8 +15,8 @@
 
 install:
 	# splashy theme
-	mkdir -p $(DESTDIR)/etc/splashy/themes/moreblue-orbit
-	$(INSTALL) $(wildcard splashy/moreblue-orbit/*) $(DESTDIR)/etc/splashy/themes/moreblue-orbit
+	mkdir -p $(DESTDIR)/usr/share/splashy/themes/moreblue-orbit
+	$(INSTALL) $(wildcard splashy/moreblue-orbit/*) $(DESTDIR)/usr/share/splashy/themes/moreblue-orbit
 	# usplash theme
 #	make -C usplash/moreblue-orbit install DESTDIR=$(CURDIR)/debian/usplash-theme-debian-desktop/usr/lib/usplash
 	# background files
