diff -Nru checksecurity-2.0.16/debian/changelog checksecurity-2.0.16+nmu1/debian/changelog
--- checksecurity-2.0.16/debian/changelog	2015-02-21 16:26:51.000000000 +0100
+++ checksecurity-2.0.16+nmu1/debian/changelog	2015-12-29 14:06:05.000000000 +0100
@@ -1,3 +1,10 @@
+checksecurity (2.0.16+nmu1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Use "find -perm /x" instead of "find -perm +x". Closes: #731944
+
+ -- Andreas Metzler <ametzler@debian.org>  Tue, 29 Dec 2015 14:06:01 +0100
+
 checksecurity (2.0.16) unstable; urgency=medium
 
   * plugins/check-setuid: Prevent error from find by putting
diff -Nru checksecurity-2.0.16/plugins/check-setuid checksecurity-2.0.16+nmu1/plugins/check-setuid
--- checksecurity-2.0.16/plugins/check-setuid	2015-02-21 15:32:25.000000000 +0100
+++ checksecurity-2.0.16+nmu1/plugins/check-setuid	2015-12-29 14:04:54.000000000 +0100
@@ -100,7 +100,7 @@
 find `mount | grep -vE "$CHECKSECURITY_FILTER" | cut -d ' ' -f 3` \
         -ignore_readdir_race  \
 	-xdev $PATHCHK \
-	\( -type f -perm +06000 -o \( \( -type b -o -type c \) \
+	\( -type f -perm /06000 -o \( \( -type b -o -type c \) \
 	$DEVCHK \) \) \
 	-printf "%8i %5m %3n %-10u %-10g %9s %t %h/%f\n" |
 	sort -k 12 >$TMPSETUID
