--- gnutls28-3.3.8.orig/lib/accelerated/x86/aes-gcm-padlock.c
+++ gnutls28-3.3.8/lib/accelerated/x86/aes-gcm-padlock.c
@@ -52,6 +52,9 @@ static void padlock_aes_encrypt(void *_c
        struct padlock_ctx *ctx = _ctx;
        struct padlock_cipher_data *pce;
 
+       if (length == 0)
+               return;
+
        pce = ALIGN16(&ctx->expanded_key);
 
        padlock_ecb_encrypt(dst, src, pce, length);
--- gnutls28-3.3.8.orig/lib/accelerated/x86/aes-padlock.c
+++ gnutls28-3.3.8/lib/accelerated/x86/aes-padlock.c
@@ -130,6 +130,9 @@ padlock_aes_cbc_encrypt(void *_ctx, cons
        struct padlock_ctx *ctx = _ctx;
        struct padlock_cipher_data *pce;
 
+       if (src_size == 0)
+               return 0;
+
        pce = ALIGN16(&ctx->expanded_key);
 
        padlock_cbc_encrypt(dst, src, pce, src_size);
@@ -145,6 +148,9 @@ padlock_aes_cbc_decrypt(void *_ctx, cons
        struct padlock_ctx *ctx = _ctx;
        struct padlock_cipher_data *pcd;
 
+       if (src_size == 0)
+               return 0;
+
        pcd = ALIGN16(&ctx->expanded_key);
 
        padlock_cbc_encrypt(dst, src, pcd, src_size);
