diff --git a/lib/transfer.c b/lib/transfer.c
index 1f69706..b1595b5 100644
--- a/lib/transfer.c
+++ b/lib/transfer.c
@@ -1010,8 +1010,10 @@ static CURLcode readwrite_http_headers(struct SessionHandle *data,
              Content-Length: headers if we are now receiving data
              using chunked Transfer-Encoding.
           */
-          if(k->chunk)
+          if(k->chunk) {
             k->size=-1;
+	    k->maxdownload=-1;
+	  }
 
         }
         if(-1 != k->size) {
