diff -Naur equivs-2.0.7.orig/usr/bin/equivs-build equivs-2.0.7/usr/bin/equivs-build
--- equivs-2.0.7.orig/usr/bin/equivs-build	2006-02-28 10:33:25.000000000 +0200
+++ equivs-2.0.7/usr/bin/equivs-build	2007-03-22 05:09:01.000000000 +0200
@@ -246,7 +246,7 @@
   my ($version, $date);
 
   $version = $control->{'Version'} || "1.0";
-  chomp ($date = qx(822-date));
+  chomp ($date = qx(date -R));
 
   open OUT, '>', "$builddir/debian/changelog" or
     die "Couldn't write changelog: $!\n";
