diff -Nru csh-20230828/debian/changelog csh-20230828/debian/changelog
--- csh-20230828/debian/changelog	2023-08-28 11:54:37.000000000 +0200
+++ csh-20230828/debian/changelog	2024-07-08 01:27:48.000000000 +0200
@@ -1,6 +1,14 @@
+csh (20230828-1.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Install programs into /usr/bin. (Closes: #1073599)
+  * Fix urgency of changelog entry 20110502-7.
+
+ -- Chris Hofstaedtler <zeha@debian.org>  Mon, 08 Jul 2024 01:27:48 +0200
+
 csh (20230828-1) unstable; urgency=medium
 
-  * New snapshot from upstream 
+  * New snapshot from upstream
     - Closes: #933057
   * Take maintainesrhip. Closes: #1049952
   * Patches updated:
@@ -15,7 +23,7 @@
 
  -- Alastair McKinstry <mckinstry@debian.org>  Mon, 28 Aug 2023 10:54:37 +0100
 
-csh (20110502-7) unstable; urgency=mediu
+csh (20110502-7) unstable; urgency=medium
 
   [ Lukas Märdian ]
   * Fix FTBFS with glibc 2.32: sys_siglist[] and sys_sigabbrev[] have been
diff -Nru csh-20230828/debian/menu csh-20230828/debian/menu
--- csh-20230828/debian/menu	2023-08-28 11:54:37.000000000 +0200
+++ csh-20230828/debian/menu	2024-07-08 01:27:48.000000000 +0200
@@ -3,4 +3,4 @@
  section="Applications/Shells"\
  longtitle="C shell"\
  title="Csh"\
- command="/bin/bsd-csh -l"
+ command="/usr/bin/bsd-csh -l"
diff -Nru csh-20230828/debian/rules csh-20230828/debian/rules
--- csh-20230828/debian/rules	2023-08-28 11:54:37.000000000 +0200
+++ csh-20230828/debian/rules	2024-07-08 01:27:35.000000000 +0200
@@ -1,6 +1,6 @@
 #!/usr/bin/make -f
 
- 
+
 include /usr/share/dpkg/buildtools.mk
 export CC
 
@@ -21,7 +21,7 @@
 	rm -f *.cat1
 
 override_dh_auto_install:
-	install -D csh debian/csh/bin/bsd-csh
+	install -D csh debian/csh/usr/bin/bsd-csh
 	install -D -m 644 USD.doc/paper.txt debian/csh/usr/share/doc/csh/csh.txt
 	install -D -m 644 USD.doc/paper.ps debian/csh/usr/share/doc/csh/csh.ps
 	install -D -m 644 -p csh.1 debian/csh/usr/share/man/man1/bsd-csh.1
