diff -ru coreutils-8.23.pristine/debian/control coreutils-8.23/debian/control
--- coreutils-8.23.pristine/debian/control	2014-10-09 18:44:05.000000000 +0100
+++ coreutils-8.23/debian/control	2014-10-09 18:45:21.834664154 +0100
@@ -22,11 +22,11 @@
  arch base64 basename cat chcon chgrp chmod chown chroot cksum comm cp
  csplit cut date dd df dir dircolors dirname du echo env expand expr
  factor false flock fmt fold groups head hostid id install join link ln
- logname ls md5sum mkdir mkfifo mknod mktemp mv nice nl nohup nproc od 
- paste pathchk pinky pr printenv printf ptx pwd readlink realpath rm rmdir
- runcon sha*sum seq shred sleep sort split stat stty sum sync tac tail tee 
- test timeout touch tr true truncate tsort tty uname unexpand uniq unlink 
- users vdir wc who whoami yes
+ logname ls md5sum mkdir mkfifo mknod mktemp mv nice nl nohup nproc numfmt
+ od paste pathchk pinky pr printenv printf ptx pwd readlink realpath rm
+ rmdir runcon sha*sum seq shred sleep sort split stat stty sum sync tac
+ tail tee test timeout touch tr true truncate tsort tty uname unexpand
+ uniq unlink users vdir wc who whoami yes
 Homepage: http://gnu.org/software/coreutils
 
 Package: mktemp
