diff -Nru dpkg-1.16.2~wipmultiarch/debian/changelog dpkg-1.16.2~wipmultiarch+kibi1/debian/changelog
--- dpkg-1.16.2~wipmultiarch/debian/changelog	2012-02-05 22:53:48.000000000 +0000
+++ dpkg-1.16.2~wipmultiarch+kibi1/debian/changelog	2012-02-06 03:07:28.000000000 +0000
@@ -1,3 +1,11 @@
+dpkg (1.16.2~wipmultiarch+kibi1) experimental; urgency=low
+
+  * When removing a foreign architecture fails due to some foreign
+    packages still being installed, mention the first one of them to
+    help users find out which packages to get rid of.
+
+ -- Cyril Brulebois <kibi@debian.org>  Mon, 06 Feb 2012 03:05:44 +0000
+
 dpkg (1.16.2~wipmultiarch) experimental; urgency=low
 
   This is a WIP release, command line interfaces *will* change.
diff -Nru dpkg-1.16.2~wipmultiarch/src/main.c dpkg-1.16.2~wipmultiarch+kibi1/src/main.c
--- dpkg-1.16.2~wipmultiarch/src/main.c	2012-02-05 22:53:04.000000000 +0000
+++ dpkg-1.16.2~wipmultiarch+kibi1/src/main.c	2012-02-06 02:24:14.000000000 +0000
@@ -537,8 +537,8 @@
         warning(_("removing architecture '%s' currently in use by database"),
                 arch->name);
       else
-        ohshit(_("cannot remove architecture '%s' currently in use by the database"),
-               arch->name);
+        ohshit(_("cannot remove architecture '%s' currently in use by the database (for at least '%s')"),
+               arch->name, pkg->set->name);
       break;
     }
   }
