From bd52c9c889087e995f2341793b6671fa94616828 Mon Sep 17 00:00:00 2001
From: Matt Kraai <kraai@ftbfs.org>
Date: Mon, 6 Jul 2015 10:26:36 -0700
Subject: [PATCH] Fix Vcs-Git link

---
 debian/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index 87aa7a8..b28549a 100644
--- a/debian/control
+++ b/debian/control
@@ -7,7 +7,7 @@ Build-Depends: libncursesw5-dev, libreadline6-dev | libreadline-dev,
   po-debconf, quilt (>= 0.40), autotools-dev
 Homepage: http://abook.sourceforge.net/
 Vcs-Browser: http://git.deb.at/w/pkg/abook.git
-Vcs-Git: git://git.deb.at/g/pkg/abook.git
+Vcs-Git: git://git.deb.at/pkg/abook.git
 
 Package: abook
 Architecture: any
-- 
2.1.4

