From f4b259baf5a26d0960b7295f4390b9bf9f61f755 Mon Sep 17 00:00:00 2001
From: Guilhem Moulin <guilhem@guilhem.org>
Date: Mon, 18 Jan 2016 17:23:23 +0100
Subject: [PATCH] Avoid calling apt-cdrom(8) on file:// mirrors.

---
 library.sh | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/library.sh b/library.sh
index d73d880..cdece40 100644
--- a/library.sh
+++ b/library.sh
@@ -804,7 +804,7 @@ install_kernel() {
 
 # Assumes the file protocol is only used for CD (image) installs
 configure_apt () {
-	if [ "$PROTOCOL" = file ]; then
+	if [ "$PROTOCOL" = file ] && [ -f /cdrom/.disk/base_installable ]; then
 		local tdir=/target/media$DIRECTORY
 		rm -f /var/lib/install-cd.id
 
-- 
2.7.0.rc3

