diff -u autogen-5.10/debian/changelog autogen-5.10/debian/changelog
--- autogen-5.10/debian/changelog
+++ autogen-5.10/debian/changelog
@@ -1,3 +1,12 @@
+autogen (1:5.10-1.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Set shlibs so that other packages get proper Depends (Closes: #562791)
+  * Don't let autoopts-config return an rpath.  This is not useful on any
+    Debian system.  (Closes: #562607)
+
+ -- Kurt Roeckx <kurt@roeckx.be>  Sat, 30 Jan 2010 20:13:56 +0100
+
 autogen (1:5.10-1) unstable; urgency=low
 
   * New upstream release.
diff -u autogen-5.10/debian/rules autogen-5.10/debian/rules
--- autogen-5.10/debian/rules
+++ autogen-5.10/debian/rules
@@ -72,7 +72,7 @@
 	dh_strip
 	dh_compress
 	dh_fixperms
-	dh_makeshlibs
+	dh_makeshlibs -V
 	dh_installdeb
 	dh_shlibdeps
 	dh_gencontrol
only in patch2:
unchanged:
--- autogen-5.10.orig/autoopts/autoopts-config.in
+++ autogen-5.10/autoopts/autoopts-config.in
@@ -18,7 +18,6 @@
  exec_prefix="@exec_prefix@"
       bindir="@bindir@"
       libdir="@libdir@"
-      ldopts="@AG_LDFLAGS@"
       exeext="@EXEEXT@"
      version="@AO_CURRENT@:@AO_REVISION@:@AO_AGE@"
       dotver="@AO_CURRENT@.@AO_REVISION@.@AO_AGE@"
@@ -29,10 +28,6 @@
       libsrc="${pkgdatadir}/libopts-${dotver}.tar.gz"
  static_libs="${libdir}/libopts.a"
       cflags="-I${includedir}"
-test -n "${ldopts}" && {
-    ldflags="${ldopts}${libdir} ${ldflags}"
-    libs=${ldflags}
-}
 test "${includedir}" = "/usr/include" && cflags=""
 optlist="\
     autogen     bindir      cflags      datadir     datarootdir dotver
