Index: pkgs.dbk
===================================================================
--- pkgs.dbk	(révision 8599)
+++ pkgs.dbk	(copie de travail)
@@ -2596,11 +2596,8 @@
 before or after this main run, depending on the exact type.
 </para>
 <para>
-If you want to see more details, you can look it up on
-<filename>merkel:/org/&ftp-debian-org;/testing/update_out/</filename> (or
-in <filename>merkel:~aba/testing/update_out</filename> to see a setup with
-a smaller packages file).  Via web, it's at <ulink
-url="http://&ftp-master-host;/testing/update_out_code/"></ulink>.
+If you want to see more details, you can look it up on <ulink
+url="http://&ftp-master-host;/testing/update_output/"></ulink>.
 </para>
 <para>
 The hints are available via <ulink
Index: resources.dbk
===================================================================
--- resources.dbk	(révision 8599)
+++ resources.dbk	(copie de travail)
@@ -260,9 +260,6 @@
 the Bug Tracking System (BTS).
 </para>
 <para>
-It is restricted; a mirror is available on <literal>merkel</literal>.
-</para>
-<para>
 If you plan on doing some statistical analysis or processing of Debian bugs,
 this would be the place to do it.  Please describe your plans on
 &email-debian-devel; before implementing anything, however, to
@@ -278,7 +275,7 @@
 end up on this server, see <xref linkend="upload"/>.
 </para>
 <para>
-It is restricted; a mirror is available on <literal>merkel</literal>.
+It is restricted; a mirror is available on <literal>&ftp-master-mirror;</literal>.
 </para>
 <para>
 Problems with the Debian FTP archive generally need to be reported as bugs
Index: common.ent
===================================================================
--- common.ent	(révision 8599)
+++ common.ent	(copie de travail)
@@ -37,7 +37,7 @@
 <!ENTITY ftp-upload-host "ftp.upload.debian.org">
 <!ENTITY ftp-eu-upload-host "ftp.eu.upload.debian.org">
 <!ENTITY ssh-upload-host "ssh.upload.debian.org">
-<!ENTITY ftp-master-mirror "merkel.debian.org">
+<!ENTITY ftp-master-mirror "ries.debian.org">
 <!ENTITY upload-queue "/pub/UploadQueue/">
 
 <!ENTITY url-debian-policy "http://&www-debian-org;/doc/debian-policy/">
