--- /usr/share/debootstrap/scripts/gutsy.org	2018-01-02 08:52:38.304877997 +0000
+++ /usr/share/debootstrap/scripts/gutsy	2018-01-02 08:53:03.438878000 +0000
@@ -62,7 +62,7 @@
 	case "$CODENAME" in
 		etch|etch-m68k|jessie|jessie-kfreebsd|lenny|squeeze|wheezy) ;;
 		*)
-			EXTRACT_DEB_TAR_OPTIONS="$EXTRACT_DEB_TAR_OPTIONS -k"
+			#EXTRACT_DEB_TAR_OPTIONS="$EXTRACT_DEB_TAR_OPTIONS -k"
 			setup_merged_usr
 			;;
 	esac
