diff -Nru cscope-15.7a/debian/changelog cscope-15.7a/debian/changelog
--- cscope-15.7a/debian/changelog       2011-09-25 20:34:34.000000000 -0700
+++ cscope-15.7a/debian/changelog       2011-12-13 08:34:31.000000000 -0800
@@ -1,3 +1,11 @@
+cscope (15.7a-3.5) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * debian/control
+    - (Suggests): Remove cbrowser (Closes: #651945).
+
+ -- Jari Aalto <jari.aalto@cante.net>  Tue, 13 Dec 2011 18:33:47 +0200
+
 cscope (15.7a-3.4) unstable; urgency=low
 
   * Non-maintainer upload.
diff -Nru cscope-15.7a/debian/control cscope-15.7a/debian/control
--- cscope-15.7a/debian/control 2011-09-25 20:32:01.000000000 -0700
+++ cscope-15.7a/debian/control 2011-12-13 08:34:31.000000000 -0800
@@ -11,7 +11,7 @@
 Package: cscope
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, ed
-Suggests: cbrowser, cscope-el
+Suggests: cscope-el
 Description: Interactively examine a C program source
  Cscope is an interactive text screen based source browsing tool.
  Although it is primarily designed to search C code (including lex

