Index: aspell/Makefile.am
===================================================================
--- aspell.orig/Makefile.am	2014-09-09 15:04:40.659234249 +0200
+++ aspell/Makefile.am	2014-09-09 15:04:40.655234208 +0200
@@ -258,8 +258,8 @@
 endif # COMPILE_IN_FILTERS
 
 noinst_DATA += $(static_optfiles) gen/filter.pot
-opt_DATA = $(dynamic_optfiles)
-filter_DATA = $(fltfiles)
+# opt_DATA = $(dynamic_optfiles)
+filter_DATA = $(fltfiles) $(dynamic_optfiles)
 
 # settings.h added as a dependency so it will get recreated if
 #   the COMPILE_IN_FILTERS option changes
