--- a/autopkgtest-build-qemu
+++ b/autopkgtest-build-qemu
@@ -539,6 +539,8 @@
             mkfs["options"] = "-O ^large_dir,^metadata_csum_seed"
             if release in ('trusty', 'jessie'):
                 mkfs["options"] += ",^metadata_csum"
+            if release in ('jessie', 'stretch', 'buster', 'bullseye'):
+                mkfs["options"] += ",^orphan_file"
         steps.append(mkfs)
         steps.append(dict(mount='root'))
 
