--- binutils-2.17.20070103cvs/debian/rules.orig	2007-01-11 20:31:46.000000000 +0000
+++ binutils-2.17.20070103cvs/debian/rules	2007-01-11 20:32:33.000000000 +0000
@@ -84,6 +86,12 @@
 ifeq ($(DEB_HOST_ARCH),mipsel)
        CONFARGS += --enable-targets=mips64el-linux-gnu
 endif
+ifeq ($(DEB_HOST_ARCH),armeb)
+	CONFARGS += --disable-werror
+endif
+ifeq ($(DEB_HOST_ARCH),armel)
+	CONFARGS += --disable-werror
+endif
 ifeq ($(DEB_HOST_ARCH),ia64)
 	CONFARGS += --disable-werror
 endif
@@ -175,7 +183,7 @@
 	mkdir builddir-multi
 	cd builddir-multi \
 	    && env CC="$(CC)" ../configure $(CONFARGS) \
-		--enable-targets=alpha-linux-gnu,arm-linux-gnu,hppa-linux-gnu,i486-gnu,i486-linux-gnu,ia64-linux-gnu,m68k-linux-gnu,m68k-rtems,mips-linux-gnu,mipsel-linux-gnu,mips64-linux-gnu,mips64el-linux-gnu,powerpc-linux-gnu,powerpc64-linux-gnu,s390-linux-gnu,s390x-linux-gnu,sh-linux-gnu,sparc-linux-gnu,sparc64-linux-gnu,x86_64-linux-gnu,m32r-linux-gnu
+		--enable-targets=alpha-linux-gnu,arm-linux-gnu,armeb-linux-gnu,armel-linux-gnu,hppa-linux-gnu,i486-gnu,i486-linux-gnu,ia64-linux-gnu,m68k-linux-gnu,m68k-rtems,mips-linux-gnu,mipsel-linux-gnu,mips64-linux-gnu,mips64el-linux-gnu,powerpc-linux-gnu,powerpc64-linux-gnu,s390-linux-gnu,s390x-linux-gnu,sh-linux-gnu,sparc-linux-gnu,sparc64-linux-gnu,x86_64-linux-gnu,m32r-linux-gnu
 	$(MAKE) -C builddir-multi configure-host
 	touch configure-multi-stamp
 
