From db50bd76f495828b1055c7c532c2cbeea23ec213 Mon Sep 17 00:00:00 2001
From: Simon Josefsson <simon@josefsson.org>
Date: Fri, 8 Nov 2024 10:20:27 +0100
Subject: [PATCH] Drop no longer used Build-Depends on golang-ed25519-dev.

---
 debian/control | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/control b/debian/control
index 37495fbe..4e11a906 100644
--- a/debian/control
+++ b/debian/control
@@ -4,7 +4,6 @@ Priority: optional
 Maintainer: Eric Dorland <eric@debian.org>
 Build-Depends: debhelper-compat (= 13),
                dh-golang,
-               golang-ed25519-dev,
                golang-github-aead-chacha20-dev,
                golang-github-aead-poly1305-dev,
                golang-github-burntsushi-toml-dev,
-- 
2.46.0

