--- debugfs/dump.c.orig	2005-09-06 09:40:13.000000000 +0000
+++ debugfs/dump.c	2007-02-27 01:50:14.000000000 +0000
@@ -5,6 +5,8 @@
  * under the terms of the GNU Public License.
  */
 
+#define _GNU_SOURCE /* for O_LARGEFILE */
+
 #include <stdio.h>
 #include <unistd.h>
 #include <stdlib.h>
@@ -256,7 +258,7 @@
 		rdump_symlink(ino, inode, fullname);
 	else if (LINUX_S_ISREG(inode->i_mode)) {
 		int fd;
-		fd = open(fullname, O_WRONLY | O_CREAT | O_TRUNC, S_IRWXU);
+		fd = open(fullname, O_WRONLY | O_CREAT | O_TRUNC | O_LARGEFILE, S_IRWXU);
 		if (fd == -1) {
 			com_err("rdump", errno, "while dumping %s", fullname);
 			goto errout;
