diff -rauN ../orig/debhelper-7.3.12/Makefile ./debhelper-7.3.12/Makefile
--- ../orig/debhelper-7.3.12/Makefile	2009-07-26 20:38:08.000000000 +0200
+++ ./debhelper-7.3.12/Makefile	2009-08-12 23:30:12.778629143 +0200
@@ -42,12 +42,12 @@
 		dir=man/$$lang; \
 		for file in $$dir/dh*.pod; do \
 			prog=`basename $$file | sed 's/.pod//'`; \
-			$(POD2MAN) $$file $$prog.$$lang.1; \
+			$(POD2MAN) --utf8 $$file $$prog.$$lang.1; \
 		done; \
 		if [ -e $$dir/debhelper.pod ]; then \
 			cat $$dir/debhelper.pod | \
 				$(MAKEMANLIST) `find $$dir -type f -maxdepth 1 -name "dh_*.pod" | sort` | \
-				$(POD2MAN) --name="debhelper" --section=7 > debhelper.$$lang.7; \
+				$(POD2MAN) --utf8 --name="debhelper" --section=7 > debhelper.$$lang.7; \
 		fi; \
 	done
 
