Index: debian/control
===================================================================
--- debian/control	(revision 20693)
+++ debian/control	(working copy)
@@ -65,7 +65,7 @@
    * GPG/OpenPGP mail signing and encryption
  .
  Support for Kerberos and SSL has been enabled in this package.
-Homepage: http://balsa.gnome.org/
+Homepage: http://pawsa.fedorapeople.org/balsa/
 
 Package: balsa-dbg
 Architecture: any
Index: debian/control.in
===================================================================
--- debian/control.in	(revision 20693)
+++ debian/control.in	(working copy)
@@ -65,7 +65,7 @@
    * GPG/OpenPGP mail signing and encryption
  .
  Support for Kerberos and SSL has been enabled in this package.
-Homepage: http://balsa.gnome.org/
+Homepage: http://pawsa.fedorapeople.org/balsa/
 
 Package: balsa-dbg
 Architecture: any
Index: debian/rules
===================================================================
--- debian/rules	(revision 20693)
+++ debian/rules	(working copy)
@@ -1,6 +1,6 @@
 #!/usr/bin/make -f
 
-GNOME_DOWNLOAD_URL = http://balsa.gnome.org/$(GNOME_TARBALL)
+GNOME_DOWNLOAD_URL = http://pawsa.fedorapeople.org/balsa/$(GNOME_TARBALL)
 TARBALL_EXT := tar.bz2
 include /usr/share/cdbs/1/class/gnome.mk
 include /usr/share/cdbs/1/rules/debhelper.mk
Index: debian/watch
===================================================================
--- debian/watch	(revision 20693)
+++ debian/watch	(working copy)
@@ -1,4 +1,4 @@
 version=2
-http://balsa.gnome.org/download.html \
-	http://balsa.gnome.org/balsa-(.*).tar.bz2 \
+http://pawsa.fedorapeople.org/balsa/download.html \
+	http://pawsa.fedorapeople.org/balsa/balsa-(.*).tar.bz2 \
 	debian uupdate
