Index: scripts/debchange.1
===================================================================
--- scripts/debchange.1	(révision 2013)
+++ scripts/debchange.1	(copie de travail)
@@ -204,8 +204,8 @@
 .TP
 \fB\-\-package\fR \fIpackage\fR
 This specifies the package name to be used in the new changelog; this
-may only be used in conjunction with the \fB\-\-create\fR and
-\fB\-\-increment\fR options.
+may only be used in conjunction with the \fB\-\-create\fR, \fB\-\-increment\fR and
+\fB\-\-newversion\fR options.
 .TP
 .BR \-\-nmu ", " \-n
 Increment the Debian release number for a non-maintainer upload by
Index: scripts/debchange.pl
===================================================================
--- scripts/debchange.pl	(révision 2013)
+++ scripts/debchange.pl	(copie de travail)
@@ -419,8 +419,8 @@
     if $opt_news && @closes;
 
 # hm, this can probably be used with more than just -i.
-fatal "--package can only be used with --create and --increment"
-    if $opt_package && ! ($opt_create || $opt_i);
+fatal "--package can only be used with --create, --increment and --newversion"
+    if $opt_package && ! ($opt_create || $opt_i || $opt_v);
 
 my $changelog_path = $opt_c || $ENV{'CHANGELOG'} || 'debian/changelog';
 my $real_changelog_path = $changelog_path;
