--- caja-1.8.1/debian/control.orig	2014-04-05 22:31:11.000000000 +0100
+++ caja-1.8.1/debian/control	2014-05-27 11:46:05.952689000 +0100
@@ -20,7 +20,7 @@
                gtk-doc-tools,
                libxt-dev,
                libx11-dev,
-               libselinux1-dev,
+               libselinux1-dev [linux-any],
                libxrender-dev,
                libxext-dev,
                x11proto-core-dev,
