Index: scripts/bts.pl
===================================================================
--- scripts/bts.pl	(revision 888)
+++ scripts/bts.pl	(working copy)
@@ -1918,11 +1918,9 @@
     $message   .= "Cc: $cc\n" if length $cc;
     $message   .= "Subject: $subject\n"
 	       .  "Date: $date\n"
-               .  "X-BTS-Version: $version\n"
+               .  "User-Agent: bts devscripts/$version\n"
                .  "Message-ID: <$msgid>\n"
                .  "\n"
-               .  "# Automatically generated email from bts,"
-                  . " devscripts version $version\n"
                .  "$body\n";
 
     if ($noaction) {
@@ -2031,7 +2029,6 @@
 	$SIG{'PIPE'} = sub { die "bts: pipe for mail broke\n"; };
 	if ($pid) {
 	    # parent
-	    print MAIL "# Automatically generated email from bts, devscripts version $version\n";
 	    print MAIL $body;
 	    close MAIL or die "bts: mail: $!\n";
 	}
@@ -2042,7 +2039,7 @@
 		    or die "bts: error running cat: $!\n";
 	    } else {
 		my @args;
-		@args = ("-s", $subject, "-a", "X-BTS-Version: $version", $btsemail);
+		@args = ("-s", $subject, "-a", "User-Agent: bts devscripts/$version", $btsemail);
 		push(@args, "-c", "$ccemail") if $ccemail;
 		push(@args, "-c", "$ccsecurity") if $ccsecurity;
 		exec("mail", @args) or die "bts: error running mail: $!\n";
