From 3c89ad8b44f5766672edcee757aa5a2ab7e36de9 Mon Sep 17 00:00:00 2001
From: Ben Wiederhake <BenWiederhake.GitHub@gmx>
Date: Mon, 21 Sep 2015 21:46:27 +0200
Subject: [PATCH 2/2] Beautify error messages.

---
 debian/changelog |  1 +
 dh_make          | 10 ++++++----
 2 files changed, 7 insertions(+), 4 deletions(-)

diff --git a/debian/changelog b/debian/changelog
index 4945ff0..5df7dd2 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -6,6 +6,7 @@ dh-make (1.20150913) UNRELEASED; urgency=medium
 
   [ Ben Wiederhake ]
   * Explain why version couldn't be parsed Closes: #799714
+  * Fix linebreaks (expect $pwd to be long)
 
  -- Craig Small <csmall@debian.org>  Mon, 07 Sep 2015 21:12:25 +1000
 
diff --git a/dh_make b/dh_make
index 26b77c2..06c3ece 100755
--- a/dh_make
+++ b/dh_make
@@ -414,8 +414,9 @@ since underscores are illegal in both package-name and version.
 Make sure that the version starts with a digit and contains only
 digits, lower and uppercase letters, dashes, or the signs plus, dot, tilde.
 
-Your current directory is $pwd, perhaps you could try going to
-directory where the sources are?
+Your current directory is:
+$pwd
+Perhaps you could try going to directory where the sources are?
 
 Please note that this change is necessary ONLY during the initial
 Debianization with dh_make.  When building the package, dpkg-source
@@ -457,8 +458,9 @@ needs to be in the format of <package>-<version>.  Alternatively use the
 -p flag using the format <name>_<version> to override it.
 The directory name you have specified is invalid!
 
-Your current directory is $pwd, perhaps you could try going to
-directory where the sources are?
+Your current directory is:
+$pwd
+Perhaps you could try going to directory where the sources are?
 
 Please note that this change is necessary ONLY during the initial
 Debianization with dh_make.  When building the package, dpkg-source
-- 
2.5.1

