diff --git a/scripts/debbisect b/scripts/debbisect
index df5e5022..6eabc518 100755
--- a/scripts/debbisect
+++ b/scripts/debbisect
@@ -754,5 +754,5 @@ devscripts as /usr/share/doc/devscripts/examples/debbisect_buildsrc.sh and
 either use it unmodified like this:
 
-    $ DEBIAN_BISECT_SRCPKG=mysrc {progname} "two years ago" yesterday \
+    $ DEBIAN_BISECT_SRCPKG=mysrc {progname} "two years ago" yesterday \\
     > /usr/share/doc/devscripts/examples/debbisect_buildsrc.sh
 
