Index: asciidoc/asciidoc.py
===================================================================
--- asciidoc.orig/asciidoc.py
+++ asciidoc/asciidoc.py
@@ -202,7 +202,7 @@
     else:
         assert os.path.isdir(directory)
         directory = os.path.realpath(directory)
-    fname = os.path.realpath(fname)
+    fname = os.path.normpath(fname)
     return os.path.commonprefix((directory, fname)) == directory
 
 def safe():
