--- debian/aumix-gtk.postinst
+++ debian/aumix-gtk.postinst
@@ -1,11 +1,6 @@
 #!/bin/sh
 # aumix-gtk postinst
 
-# remove init symlinks so old scripts cannot fuck up new settings
-# new links are provided below
-update-rc.d -f aumix-gtk remove >/dev/null 2>/dev/null || true
-update-rc.d -f aumix remove >/dev/null 2>/dev/null || true
-
 # do this manually instead of using dh_installinit.  See postrm.
 if [ -x "/etc/init.d/aumix" ]; then
         update-rc.d aumix start 21 1 2 3 4 5 . stop 19 0 1 6 . >/dev/null
--- debian/aumix.postinst
+++ debian/aumix.postinst
@@ -1,11 +1,6 @@
 #!/bin/sh
 # aumix postinst
 
-# remove init symlinks so old scripts cannot fuckup new settings
-# new links are provided below
-update-rc.d -f aumix-gtk remove >/dev/null 2>/dev/null || true
-update-rc.d -f aumix remove >/dev/null 2>/dev/null || true
-
 # do this manually instead of using dh_installinit.  See postrm.
 if [ -x "/etc/init.d/aumix" ]; then
         update-rc.d aumix start 21 1 2 3 4 5 . stop 19 0 1 6 . >/dev/null
