diff -ruN Alacarte.orig/MenuEditor.py Alacarte/MenuEditor.py
--- Alacarte.orig/MenuEditor.py	2009-12-21 22:59:55.000000000 +0100
+++ Alacarte/MenuEditor.py	2009-12-22 16:47:32.000000000 +0100
@@ -48,16 +48,16 @@
 			self.applications.dom = xml.dom.minidom.parse(self.applications.path)
 		self.__remove_whilespace_nodes(self.applications.dom)
 
-		self.settings = Menu() 	 
+		self.settings = Menu()
 		self.settings.tree = gmenu.lookup_tree('settings.menu', gmenu.FLAGS_SHOW_EMPTY|gmenu.FLAGS_INCLUDE_EXCLUDED|gmenu.FLAGS_INCLUDE_NODISPLAY|gmenu.FLAGS_SHOW_ALL_SEPARATORS)
-		self.settings.visible_tree = gmenu.lookup_tree('settings.menu') 	 
+		self.settings.visible_tree = gmenu.lookup_tree('settings.menu')
 		self.settings.tree.sort_key = gmenu.SORT_DISPLAY_NAME
 		self.settings.visible_tree.sort_key = gmenu.SORT_DISPLAY_NAME
-		self.settings.path = os.path.join(util.getUserMenuPath(), self.settings.tree.get_menu_file()) 	 
-		if not os.path.isfile(self.settings.path): 	 
-			self.settings.dom = xml.dom.minidom.parseString(util.getUserMenuXml(self.settings.tree)) 	 
-		else: 	 
-			self.settings.dom = xml.dom.minidom.parse(self.settings.path) 	 
+		self.settings.path = os.path.join(util.getUserMenuPath(), self.settings.tree.get_menu_file())
+		if not os.path.isfile(self.settings.path):
+			self.settings.dom = xml.dom.minidom.parseString(util.getUserMenuXml(self.settings.tree))
+		else:
+			self.settings.dom = xml.dom.minidom.parse(self.settings.path)
 		self.__remove_whilespace_nodes(self.settings.dom)
 
 		self.save(True)
@@ -478,7 +478,7 @@
 
 	def __getPath(self, menu, path=None):
 		if not path:
-                        path = menu.tree.root.get_menu_id()
+			path = menu.tree.root.get_menu_id()
 		if menu.get_parent():
 			path = self.__getPath(menu.get_parent(), path)
 			path += '/'
