diff -r -u ../at-3.1.12/at.c ./at.c
--- ../at-3.1.12/at.c	2010-08-26 16:12:11.000000000 +0200
+++ ./at.c	2010-08-26 15:48:48.000000000 +0200
@@ -355,6 +355,12 @@
     /* Get the userid to mail to, first by trying getlogin(), which reads
      * /var/run/utmp, then from LOGNAME, finally from getpwuid().
      */
+
+    /* New Feature: MAILTO overrides everything */
+    mailname = getenv("MAILTO");
+    if (mailname != NULL)
+        goto dontbother;
+
     mailname = getlogin();
     if (mailname == NULL)
 	mailname = getenv("LOGNAME");
@@ -364,6 +370,7 @@
 	    mailname = pass_entry->pw_name;
     }
 
+dontbother:
     if ((mailname == NULL) || (mailname[0] == '\0')
 	|| (strlen(mailname) > mailsize) ) {
 	panic("Cannot find username to mail output to");
@@ -374,6 +381,7 @@
 	    perr("Cannot open input file %.500s", atinput);
     }
 
+    fprintf(stderr, "Mail will be sent to %s\n", mailname);
     fprintf(fp, "#!/bin/sh\n# atrun uid=%d gid=%d\n# mail %s %d\n",
 	    real_uid, real_gid, mailname, send_mail);
 
