Index: debian/changelog
===================================================================
--- debian/changelog	(Revision 912)
+++ debian/changelog	(Arbeitskopie)
@@ -20,8 +20,12 @@
   [ Christoph Berg ]
   * debcommit: also look for \t in the diff for GNU-style changelogs.
 
- -- Christoph Berg <myon@debian.org>  Sun, 27 Jan 2008 13:26:30 +0100
+  [ Patrick Schoenfeld ]
+  * debcommit: Make commit work if the current working directory is the
+    debian - directory (Closes: #450878)
 
+ -- Patrick Schoenfeld <schoenfeld@in-medias-res.com>  Sun, 27 Jan 2008 19:52:55 +0100
+
 devscripts (2.10.13) unstable; urgency=low
 
   * The 'we sponsor Adam for DM' upload
Index: scripts/debcommit.pl
===================================================================
--- scripts/debcommit.pl	(Revision 912)
+++ scripts/debcommit.pl	(Arbeitskopie)
@@ -240,7 +240,13 @@
 
 my $prog=getprog();
 if (! -e $changelog) {
-    die "debcommit: cannot find $changelog\n";
+	my $dir=basename(getcwd());
+	if ($dir eq 'debian') {
+		$changelog = 'changelog';
+	}
+	if (! -e $changelog) {
+		die "debcommit: cannot find $changelog\n";
+	}
 }
 
 if ($release) {
