Index: scripts/debcommit.pl
===================================================================
--- scripts/debcommit.pl	(revision 1862)
+++ scripts/debcommit.pl	(working copy)
@@ -341,7 +341,7 @@
     }
     close C;
     
-    $version=`dpkg-parsechangelog | grep '^Version:' | cut -f 2 -d ' '`;
+    $version=`dpkg-parsechangelog -l\Q$changelog\E | grep '^Version:' | cut -f 2 -d ' '`;
     chomp $version;
 
     $message="releasing version $version" if ! defined $message;
