--- dh_auto_install.orig	2009-05-26 10:48:34.000000000 +0200
+++ dh_auto_install	2009-05-26 10:49:32.000000000 +0200
@@ -64,9 +64,9 @@
 	$ENV{MAKE}="make" unless exists $ENV{MAKE};
 	my @params="DESTDIR=$destdir";
 
-	# Special case for MakeMaker generated Makefiles.
+	# Special case for Makefiles that set their own PREFIX variable.
 	if (-e "Makefile" &&
-	    system('grep -q "generated automatically by MakeMaker" Makefile') == 0) {
+	    system('grep -q "^PREFIX[[:space:]]*\([:+?]\)\?=" Makefile') == 0) {
 		push @params, "PREFIX=/usr";
 	}
 
