From 8a539b90bbf4eaa507988dc820b627d11d6e0a5b Mon Sep 17 00:00:00 2001
From: Ken Sharp <kennybobs@o2.co.uk>
Date: Thu, 14 Jun 2012 01:38:32 +0100
Subject: [PATCH] Add .msi completion for Wine

---
 completions/wine |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/completions/wine b/completions/wine
index c02c97f..2e8b123 100644
--- a/completions/wine
+++ b/completions/wine
@@ -10,7 +10,7 @@ _wine()
             COMPREPLY=( $( compgen -W '--help --version' -- "$cur" ) )
             [[ $COMPREPLY ]] && return
         fi
-        _filedir '[eE][xX][eE]?(.[sS][oO])|[cC][oO][mM]|[sS][cC][rR]'
+        _filedir '[eE][xX][eE]?(.[sS][oO])|[cC][oO][mM]|[sS][cC][rR]|[mM][sS][iI]'
     else
         _filedir
     fi
-- 
1.7.4.1

