From 77b548921b6f53841f7afc6434092943356b03fc Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Wed, 15 Feb 2012 19:48:24 +0800
Subject: [PATCH] Also ignore mercurial and darcs VCS directories

---
 scripts/suspicious-source |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/scripts/suspicious-source b/scripts/suspicious-source
index 3cacd6e..843fb3d 100755
--- a/scripts/suspicious-source
+++ b/scripts/suspicious-source
@@ -107,7 +107,7 @@ def suspicious_source(whitelisted_mimetypes, whitelisted_extensions, directory,
                         print "%s (%s)" % (os.path.join(root, f), mimetype)
                     else:
                         print os.path.join(root, f)
-        for vcs_dir in (".bzr", "CVS", ".git", ".svn"):
+        for vcs_dir in (".bzr", "CVS", ".git", ".svn", ".hg", "_darcs"):
             if vcs_dir in dirs:
                 dirs.remove(vcs_dir)
 
-- 
1.7.9

