diff -Nru dovecot-2.1.4/debian/control dovecot-2.1.4/debian/control
--- dovecot-2.1.4/debian/control	2012-04-10 18:34:45.000000000 +0300
+++ dovecot-2.1.4/debian/control	2012-05-09 14:50:17.000000000 +0300
@@ -3,7 +3,7 @@
 Priority: optional
 Maintainer: Dovecot Maintainers <jaldhar-dovecot@debian.org>
 Uploaders: Jaldhar H. Vyas <jaldhar@debian.org>, Fabio Tranchitella <kobold@debian.org>, Joel Johnson <mrjoel@lixil.net>, Marco Nenciarini <mnencia@debian.org>
-Build-Depends: debhelper (>= 7.2.3~), dpkg-dev (>= 1.16.1), pkg-config, libssl-dev, libpam0g-dev, libldap2-dev, libpq-dev, libmysqlclient-dev, libsqlite3-dev, libsasl2-dev, zlib1g-dev, libkrb5-dev, drac-dev (>= 1.12-5), libbz2-dev, libdb-dev, libcurl4-gnutls-dev, libexpat-dev
+Build-Depends: debhelper (>= 7.2.3~), dpkg-dev (>= 1.16.1), pkg-config, libssl-dev, libpam0g-dev, libldap2-dev, libpq-dev, libmysqlclient-dev, libsqlite3-dev, libsasl2-dev, zlib1g-dev, libkrb5-dev, drac-dev (>= 1.12-5), libbz2-dev, libdb-dev, libcurl4-gnutls-dev, libexpat-dev, systemd
 Standards-Version: 3.9.3
 Homepage: http://dovecot.org/
 Vcs-Git: git://git.debian.org/git/collab-maint/dovecot.git
diff -Nru dovecot-2.1.4/debian/rules dovecot-2.1.4/debian/rules
--- dovecot-2.1.4/debian/rules	2012-04-10 18:34:45.000000000 +0300
+++ dovecot-2.1.4/debian/rules	2012-05-09 15:18:36.000000000 +0300
@@ -43,6 +43,7 @@
 	            --infodir=\$${prefix}/share/info \
 	            --with-moduledir=\$${prefix}/lib/dovecot/modules \
 	            --disable-rpath \
+	            --with-systemdsystemunitdir=auto \
 	            --disable-static
 	$(MAKE) dovecot-config
 	# Pigeonhole
