--- a/debian/libuv1.symbols	2018-07-20 17:18:26.000000000 +0200
+++ b/debian/libuv1.symbols	2018-07-24 13:48:35.000000000 +0200
@@ -1,4 +1,5 @@
 libuv.so.1 libuv1 #MINVER#
+ (arch=hurd-any)pthread_atfork@Base 1.22.0-1
  (arch=linux-any)uv__accept4@Base 1.11.0
  uv__accept@Base 1.11.0
  uv__async_close@Base 1.11.0
@@ -23,14 +24,14 @@
  (arch=linux-any)uv__eventfd@Base 1.11.0
  uv__fd_exists@Base 1.21.0
  uv__free@Base 1.11.0
- (arch=!hurd-any)uv__fs_event_close@Base 1.11.0
+ uv__fs_event_close@Base 1.11.0
  uv__fs_poll_close@Base 1.11.0
  uv__fs_scandir_cleanup@Base 1.11.0
  uv__getaddrinfo_translate_error@Base 1.11.0
  uv__getiovmax@Base 1.11.0
  uv__getpwuid_r@Base 1.11.0
  uv__handle_type@Base 1.11.0
- (arch=!hurd-any)uv__hrtime@Base 1.11.0
+ uv__hrtime@Base 1.11.0
  uv__idle_close@Base 1.11.0
  (arch=linux-any)uv__inotify_add_watch@Base 1.11.0
  (arch=linux-any)uv__inotify_fork@Base 1.18.0
@@ -38,12 +39,12 @@
  (arch=linux-any)uv__inotify_init@Base 1.11.0
  (arch=linux-any)uv__inotify_rm_watch@Base 1.11.0
  uv__io_active@Base 1.11.0
- (arch=!hurd-any)uv__io_check_fd@Base 1.11.0
+ uv__io_check_fd@Base 1.11.0
  uv__io_close@Base 1.11.0
  uv__io_feed@Base 1.11.0
- (arch=!hurd-any)uv__io_fork@Base 1.18.0
+ uv__io_fork@Base 1.18.0
  uv__io_init@Base 1.11.0
- (arch=!hurd-any)uv__io_poll@Base 1.11.0
+ uv__io_poll@Base 1.11.0
  uv__io_start@Base 1.11.0
  uv__io_stop@Base 1.11.0
  uv__loop_close@Base 1.11.0
@@ -59,9 +60,9 @@
  uv__open_file@Base 1.11.0
  (arch=linux-any)uv__pipe2@Base 1.11.0
  uv__pipe_close@Base 1.11.0
- (arch=!hurd-any)uv__platform_invalidate_fd@Base 1.11.0
- (arch=!hurd-any)uv__platform_loop_delete@Base 1.11.0
- (arch=!hurd-any)uv__platform_loop_init@Base 1.11.0
+ uv__platform_invalidate_fd@Base 1.11.0
+ uv__platform_loop_delete@Base 1.11.0
+ uv__platform_loop_init@Base 1.11.0
  uv__poll_close@Base 1.11.0
  (arch=linux-any)uv__preadv@Base 1.11.0
  uv__prepare_close@Base 1.11.0
@@ -148,9 +149,9 @@
  uv_fs_close@Base 1.4.2
  uv_fs_copyfile@Base 1.18.0
  uv_fs_event_getpath@Base 1.4.2
- (arch=!hurd-any)uv_fs_event_init@Base 1.4.2
- (arch=!hurd-any)uv_fs_event_start@Base 1.4.2
- (arch=!hurd-any)uv_fs_event_stop@Base 1.4.2
+ uv_fs_event_init@Base 1.4.2
+ uv_fs_event_start@Base 1.4.2
+ uv_fs_event_stop@Base 1.4.2
  uv_fs_fchmod@Base 1.4.2
  uv_fs_fchown@Base 1.4.2
  uv_fs_fdatasync@Base 1.4.2
