diff -ruNb ardour-2.4/debian/ardour-altivec.desktop ardour-2.4.new/debian/ardour-altivec.desktop
--- ardour-2.4/debian/ardour-altivec.desktop	2008-04-02 21:31:31.000000000 +0100
+++ ardour-2.4.new/debian/ardour-altivec.desktop	2008-04-03 23:04:52.000000000 +0100
@@ -1,10 +1,9 @@
 [Desktop Entry]
-Encoding=UTF-8
-
+Version=1.0
 Name=Ardour GTK2 [altivec]
 Comment=Ardour Digital Audio Workstation (GTK2 Interface) [altivec]
 Exec=/usr/bin/ardour2
 Terminal=false
 Type=Application
 Icon=/usr/share/ardour2/icons/ardour_icon_22px.png
-Categories=Application;AudioVideo;Audio;
+Categories=AudioVideo;Audio;
diff -ruNb ardour-2.4/debian/ardour.desktop ardour-2.4.new/debian/ardour.desktop
--- ardour-2.4/debian/ardour.desktop	2008-04-02 21:31:31.000000000 +0100
+++ ardour-2.4.new/debian/ardour.desktop	2008-04-03 23:05:12.000000000 +0100
@@ -1,10 +1,9 @@
 [Desktop Entry]
-Encoding=UTF-8
-
+Version=1.0
 Name=Ardour GTK2
 Comment=Ardour Digital Audio Workstation (GTK2 Interface)
 Exec=/usr/bin/ardour2
 Terminal=false
 Type=Application
 Icon=/usr/share/ardour2/icons/ardour_icon_22px.png
-Categories=Application;AudioVideo;Audio;
+Categories=AudioVideo;Audio;
diff -ruNb ardour-2.4/debian/ardour-i686.desktop ardour-2.4.new/debian/ardour-i686.desktop
--- ardour-2.4/debian/ardour-i686.desktop	2008-04-02 21:31:31.000000000 +0100
+++ ardour-2.4.new/debian/ardour-i686.desktop	2008-04-03 23:05:26.000000000 +0100
@@ -1,10 +1,9 @@
 [Desktop Entry]
-Encoding=UTF-8
-
+Version=1.0
 Name=Ardour GTK2 [i686]
 Comment=Ardour Digital Audio Workstation (GTK2 Interface) [i686]
 Exec=/usr/bin/ardour2
 Terminal=false
 Type=Application
 Icon=/usr/share/ardour2/icons/ardour_icon_22px.png
-Categories=Application;AudioVideo;Audio;
+Categories=AudioVideo;Audio;
diff -ruNb ardour-2.4/debian/ardour-opt.desktop ardour-2.4.new/debian/ardour-opt.desktop
--- ardour-2.4/debian/ardour-opt.desktop	2008-04-02 21:31:31.000000000 +0100
+++ ardour-2.4.new/debian/ardour-opt.desktop	2008-04-03 23:05:41.000000000 +0100
@@ -1,10 +1,9 @@
 [Desktop Entry]
-Encoding=UTF-8
-
+Version=1.0
 Name=Ardour GTK2 [@optarch@]
 Comment=Ardour Digital Audio Workstation (GTK2 Interface) [@optarch@]
 Exec=/usr/bin/ardour2
 Terminal=false
 Type=Application
 Icon=/usr/share/ardour2/icons/ardour_icon_22px.png
-Categories=Application;AudioVideo;Audio;
+Categories=AudioVideo;Audio;
