--- debian/control.old	2012-01-04 14:43:59.000000000 +0100
+++ debian/control	2012-01-04 14:44:10.000000000 +0100
@@ -24,7 +24,7 @@
 
 Package: doxygen-latex
 Architecture: all
-Depends: doxygen (>= ${source:Version}), ${misc:Depends}, texlive-extra-utils, texlive-latex-extra, latex-xcolor, texlive-font-utils
+Depends: doxygen (>= ${source:Version}), ${misc:Depends}, texlive-extra-utils, texlive-latex-extra, latex-xcolor, texlive-font-utils, ghostscript
 Description: Documentation system for C, C++, Java, Python and other languages
  Doxygen is a documentation system for C, C++, Java, Objective-C, Python, IDL
  and to some extent PHP, C#, and D.  It can generate an on-line class browser
