--- debian/init.d	2012-03-29 07:40:19.000000000 +0200
+++ debian/init.d.new	2012-12-05 19:56:08.062475662 +0100
@@ -32,7 +32,7 @@
 		mkdir -p /var/run/dirmngr || return 1
 		chown dirmngr:dirmngr /var/run/dirmngr || return 1
 
-		output=$(su -c ". /lib/lsb/init-functions && umask 027 && start_daemon -p $PIDFILE $DAEMON --daemon --sh" - dirmngr) || return 1
+		output=$(umask 027 && start_daemon -p $PIDFILE $DAEMON --daemon --sh) || return 1
 		eval "$output" || return 1
 		pid=$(echo "$DIRMNGR_INFO" | cut -d : -f 2) || return 1
 		echo "$pid" >$PIDFILE || return 1
