Index: apcupsd-3.14.10/autoconf/configure.in
===================================================================
--- apcupsd-3.14.10.orig/autoconf/configure.in	2012-05-22 23:02:44.000000000 +0100
+++ apcupsd-3.14.10/autoconf/configure.in	2012-05-22 23:02:58.465449378 +0100
@@ -349,6 +349,7 @@
    *-*-hpux*)    AC_DEFINE(HAVE_HPUX_OS)    ;;
    *-*-linux*)   AC_DEFINE(HAVE_LINUX_OS)   ;;
    *-*-freebsd*) AC_DEFINE(HAVE_FREEBSD_OS) ;;
+   *-*-kfreebsd*)AC_DEFINE(HAVE_FREEBSD_OS) ;;
    *-*-netbsd*)  AC_DEFINE(HAVE_NETBSD_OS)  ;;
    *-*-openbsd*) AC_DEFINE(HAVE_OPENBSD_OS) ;;
    *-*-bsdi*)    AC_DEFINE(HAVE_BSDI_OS)    ;;
Index: apcupsd-3.14.10/configure
===================================================================
--- apcupsd-3.14.10.orig/configure	2012-05-22 23:04:47.645447000 +0100
+++ apcupsd-3.14.10/configure	2012-05-22 23:05:25.409449524 +0100
@@ -9509,6 +9509,10 @@
 #define HAVE_FREEBSD_OS 1
 _ACEOF
  ;;
+   *-*-kfreebsd*) cat >>confdefs.h <<\_ACEOF
+#define HAVE_FREEBSD_OS 1
+_ACEOF
+ ;;
    *-*-netbsd*)  cat >>confdefs.h <<\_ACEOF
 #define HAVE_NETBSD_OS 1
 _ACEOF
