Index: source/dlcon.cc
===================================================================
--- source/dlcon.cc	(Revision 3689)
+++ source/dlcon.cc	(Arbeitskopie)
@@ -269,6 +269,9 @@
 					continue;
 				}
 				
+				if(!con.ptr) // failed above? will recheck the connection state
+					continue;
+
 				for(dljIter it=m_qToReceive.begin(); it!=m_qToReceive.end(); it++)
 				{
 					dbgline;
@@ -284,9 +287,6 @@
 						break;
 					}
 				}
-				
-				if(!con.ptr) // failed above? will recheck the connection state
-					continue;
 
 			} // (re)connect done
 
