diff -Naur base-files-4.0.4.orig/debian/postinst base-files-4.0.4/debian/postinst
--- base-files-4.0.4.orig/debian/postinst	2008-03-11 11:02:27.000000000 +0100
+++ base-files-4.0.4/debian/postinst	2008-08-06 12:40:45.000000000 +0200
@@ -57,6 +57,18 @@
   install_local_dir /usr/local/src
   install_local_dir /usr/local/etc
   ln -sf share/man /usr/local/man
+
+  if [ ! -f /var/log/wtmp ]; then
+     echo -n>/var/log/wtmp
+  fi
+  if [ ! -f /var/log/btmp ]; then
+     echo -n>/var/log/btmp
+  fi
+  if [ ! -f /var/log/lastlog ]; then
+     echo -n>/var/log/lastlog
+  fi
+  chown root:utmp /var/log/wtmp /var/log/btmp /var/log/lastlog
+  chmod 664 /var/log/wtmp /var/log/btmp /var/log/lastlog
 fi
 
 if [ "$1" = "configure" ] && [ "$2" != "" ]; then
@@ -86,17 +98,8 @@
 if [ ! -f /var/run/utmp ]; then
   echo -n>/var/run/utmp
 fi
-if [ ! -f /var/log/wtmp ]; then
-  echo -n>/var/log/wtmp
-fi
-if [ ! -f /var/log/btmp ]; then
-  echo -n>/var/log/btmp
-fi
-if [ ! -f /var/log/lastlog ]; then
-  echo -n>/var/log/lastlog
-fi
-chown root:utmp /var/run/utmp /var/log/wtmp /var/log/btmp /var/log/lastlog
-chmod 664 /var/run/utmp /var/log/wtmp /var/log/btmp /var/log/lastlog
+chown root:utmp /var/run/utmp
+chmod 664 /var/run/utmp
 
 if [ ! -d /var/lib/dpkg ]; then
   mkdir -m 755 -p /var/lib/dpkg
