diff -Naurp dput-0.9.6.1/dcut dput-0.9.6.1.new/dcut
--- dput-0.9.6.1/dcut	2010-01-11 08:10:15.000000000 +0100
+++ dput-0.9.6.1.new/dcut	2011-01-04 18:56:26.147957236 +0100
@@ -18,7 +18,7 @@
 # along with this program; if not, write to the Free Software
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 
-import sys, getopt, os, tempfile, string, time
+import sys, getopt, os, tempfile, string, time, subprocess
 
 sys.path.insert(0,'/usr/share/dput/helper')
 import dputhelper
@@ -203,11 +203,13 @@ def write_commands(commands, options, co
   f.write("Uploader: %s\n"%options['uploader'])
   f.write("Commands:\n %s\n\n"%('\n '.join(commands)))
   f.close()
-  debsign_cmdline = "debsign '-m%s'" % options['uploader']
-  if options['keyid']: debsign_cmdline += " '-k%s'" % options['keyid']
-  debsign_cmdline += " '%s'" % filename
+  debsign_cmdline = ['debsign']
+  debsign_cmdline.append('-m%s' % options['uploader'])
+  if options['keyid']: debsign_cmdline.append('-k%s' % options['keyid'])
+  debsign_cmdline.append('%s' % filename)
   if options['debug']: print "D: calling debsign:",debsign_cmdline
-  if os.system(debsign_cmdline):
+  debsign_prog = subprocess.Popen(debsign_cmdline)
+  if os.waitpid(debsign_prog.pid, 0)[1]:
     print >> sys.stderr, "Error: debsign failed."
     sys.exit(1)
   return filename
