diff --git a/common.ent b/common.ent
index 1bbed6a..561af11 100644
--- a/common.ent
+++ b/common.ent
@@ -48,6 +48,7 @@
 <!ENTITY bugs-host "bugs.debian.org">
 <!ENTITY pts-host "packages.qa.debian.org">
 <!ENTITY tracker-host "tracker.debian.org">
+<!ENTITY queued-log-host "usper.debian.org">
 <!ENTITY ftp-master-host "ftp-master.debian.org">
 <!ENTITY ftp-upload-host "ftp.upload.debian.org">
 <!ENTITY ftp-eu-upload-host "ftp.eu.upload.debian.org">
diff --git a/pkgs.dbk b/pkgs.dbk
index 17c42e3..8af374f 100644
--- a/pkgs.dbk
+++ b/pkgs.dbk
@@ -504,7 +504,7 @@ linkend="irc-channels">IRC</link> channel
 <literal>#debian-devel-changes</literal>. If your upload fails
 silently, it could be that your package is improperly signed, in which
 case you can find more explanations on
-<filename>ssh.debian.org:/srv/upload.debian.org/queued/run/log</filename>.
+<filename>&queued-log-host;:/srv/upload.debian.org/queued/run/log</filename>.
 </para>
 </section>
 
