diff -u cron-3.0pl1/debian/changelog cron-3.0pl1/debian/changelog
--- cron-3.0pl1/debian/changelog
+++ cron-3.0pl1/debian/changelog
@@ -1,3 +1,11 @@
+cron (3.0pl1-124.2) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * debian/cron.service: Set IgnoreSIGPIPE=false. (Closes: #756047)
+  * debian/cron.service: Add Documentation field.
+
+ -- Ansgar Burchardt <ansgar@debian.org>  Mon, 04 Aug 2014 21:44:00 +0200
+
 cron (3.0pl1-124.1) unstable; urgency=medium
 
   * Non-maintainer upload.
diff -u cron-3.0pl1/debian/cron.service cron-3.0pl1/debian/cron.service
--- cron-3.0pl1/debian/cron.service
+++ cron-3.0pl1/debian/cron.service
@@ -1,8 +1,10 @@
 [Unit]
 Description=Regular background program processing daemon
+Documentation=man:cron(8)
 
 [Service]
 ExecStart=/usr/sbin/cron -f
+IgnoreSIGPIPE=false
 
 [Install]
 WantedBy=multi-user.target
