--- cryptroot	2008-02-16 18:39:57.000000000 +0100
+++ cryptroot_helmut	2008-02-21 13:33:06.000000000 +0100
@@ -43,6 +43,7 @@
 	cryptlvm=""
 	cryptkeyscript=""
 	cryptkey="" # This is only used as an argument to an eventual keyscript
+	crypttries=3
 
 	local IFS=" ,"
 	for x in $cryptopts; do
@@ -78,6 +79,11 @@
 				cryptkey=${x#key=}
 			fi
 			;;
+		tries=*)
+			if [ "${x#tries=}" -ge 0 ]; then
+				crypttries=${x#tries=}
+			fi
+			;;
 		esac
 	done
 
@@ -179,7 +185,7 @@
 
 	# Try to get a satisfactory password three times
 	count=0
-	while [ $count -lt 3 ]; do
+	while [ $crypttries -le 0 -o $count -lt $crypttries ]; do
 		count=$(( $count + 1 ))
 
 		if [ -n "$cryptkeyscript" ]; then
