diff --git a/datefudge.c b/datefudge.c
index fe93ef8..00cb012 100644
--- a/datefudge.c
+++ b/datefudge.c
@@ -50,8 +50,6 @@ static void set_fudge(time_t *seconds)
         *seconds -= fudge;
 }
 
-#ifndef __GNU__
-
 time_t time(time_t *x) {
     static time_t (*libc_time)(time_t *) = NULL;
     time_t res;
@@ -64,8 +62,6 @@ time_t time(time_t *x) {
     return res;
 }
 
-#endif
-
 int __gettimeofday(struct timeval *x, struct timezone *y) {
     static int (*libc_gettimeofday)(struct timeval *, struct timezone *) = NULL;
     int res;
@@ -82,8 +78,6 @@ int gettimeofday(struct timeval *x, struct timezone *y) {
     return __gettimeofday(x,y);
 }
 
-#ifndef __GNU__
-
 int clock_gettime(clockid_t x, struct timespec *y) {
     static int (*libc_clock_gettime)(clockid_t, struct timespec*);
     int res;
@@ -95,5 +89,3 @@ int clock_gettime(clockid_t x, struct timespec *y) {
     set_fudge(&y->tv_sec);
     return 0;
 }
-
-#endif
