--- cups-filters-1.0.16.orig/filter/texttopdf.c
+++ cups-filters-1.0.16/filter/texttopdf.c
@@ -59,7 +59,7 @@ EMB_PARAMS *font_load(const char *font)
   } else {
     FcInit ();
     pattern = FcNameParse ((const FcChar8 *)font);
-    FcPatternAddInteger (pattern, FC_SPACING, FC_MONO); // guide fc, in case substitution becomes necessary
+/*    FcPatternAddInteger (pattern, FC_SPACING, FC_MONO); // Debian #670055 */
     FcConfigSubstitute (0, pattern, FcMatchPattern);
     FcDefaultSubstitute (pattern);
 
