--- cups-filters-1.0.16.orig/filter/texttopdf.c
+++ cups-filters-1.0.16/filter/texttopdf.c
@@ -75,6 +75,9 @@ EMB_PARAMS *font_load(const char *font)
       FcPatternGetString  (candidates->fonts[i], FC_FONTFORMAT, 0, &fontformat);
       FcPatternGetInteger (candidates->fonts[i], FC_SPACING,    0, &spacing);
 
+      fprintf(stderr,"### Candidate: %s\n",
+        FcPatternFormat (candidates->fonts[i], (const FcChar8 *)"%{file|cescape}:%{fontformat}:%{spacing}"));
+
       if ( (fontformat)&&(spacing == FC_MONO) ) {
         if (strcmp((const char *)fontformat, "TrueType") == 0) {
           fontname = FcPatternFormat (candidates->fonts[i], (const FcChar8 *)"%{file|cescape}/%{index}");
@@ -86,6 +89,9 @@ EMB_PARAMS *font_load(const char *font)
       }
     }
     FcFontSetDestroy (candidates);
+
+    if (fontname)
+      fprintf(stderr,"### Chosen: %s\n", fontname);
   }
 
   if (!fontname) {
