diff -ruN dpkg-cross-2.5.0/Debian/DpkgCross.pm dpkg-cross-2.5.0a/Debian/DpkgCross.pm
--- dpkg-cross-2.5.0/Debian/DpkgCross.pm	2009-03-10 05:25:02.000000000 +0900
+++ dpkg-cross-2.5.0a/Debian/DpkgCross.pm	2009-05-14 16:09:23.000000000 +0900
@@ -138,7 +138,12 @@
 	"m32r" => [ "m32r-linux-gnu-", "m32r-linux-" ],
 	"amd64" => [ "amd64-linux-gnu-", "amd64-linux-", "x86_64-linux-gnu-", "x86_64-linux-",
 			"x86-64-linux-gnu-", "x86-64-linux-" ],
-	"w32-i386" => [ "i586-mingw32msvc-" ]);
+	"w32-i386" => [ "i586-mingw32msvc-" ],
+	"sh3" => [ "sh3-linux-gnu-", "sh3-linux-" ],
+	"sh4" => [ "sh4-linux-gnu-", "sh4-linux-" ],
+	"sh3eb" => [ "sh3eb-linux-gnu-", "sh3eb-linux-" ],
+	"sh4eb" => [ "sh4eb-linux-gnu-", "sh4eb-linux-" ],
+	);
 
 =head1 MAKEFLAGS
 
diff -ruN dpkg-cross-2.5.0/debian/dpkg-cross.templates dpkg-cross-2.5.0a/debian/dpkg-cross.templates
--- dpkg-cross-2.5.0/debian/dpkg-cross.templates	2007-03-12 08:14:42.000000000 +0900
+++ dpkg-cross-2.5.0a/debian/dpkg-cross.templates	2009-05-14 16:16:01.000000000 +0900
@@ -2,7 +2,7 @@
 Type: select
 # Translators: "None" here refers to "architecture"
 #flag:translate:1
-__Choices: None, alpha, amd64, arm, armeb, armel, hppa, i386, ia64, m68k, mips, mipsel, powerpc, s390, sparc
+__Choices: None, alpha, amd64, arm, armeb, armel, hppa, i386, ia64, m68k, mips, mipsel, powerpc, s390, sh3 sh4 sh3eb sh4eb sparc
 Default: None
 _Description: Default cross-building architecture:
  If this machine is typically cross-building for one main architecture,
