diff -Nru doc-debian-6.3/doc/Makefile doc-debian-6.3/doc/Makefile
--- doc-debian-6.3/doc/Makefile	2015-02-18 00:39:57.000000000 +0100
+++ doc-debian-6.3/doc/Makefile	2016-06-06 11:30:59.000000000 +0200
@@ -40,7 +40,7 @@
 	wml -i bug-pkgreport-opts.inc -q $< >$@
 
 %.txt: %.html
-	lynx -dump -nolist $< >$@
+	lynx -dump -nolist -display_charset=UTF-8 -assume_charset=UTF-8 $< >$@
 
 # These rules are conditioned to the existence of $(WEBWML)
 # so that the package can be built regardless of its existence
