--- old/debian/control	2012-03-27 06:47:44.000000000 +1000
+++ new/debian/control	2014-10-13 19:42:19.454504062 +1000
@@ -6,7 +6,11 @@
  Joost van Baal <joostvb@debian.org>, Simon Paillard <spaillard@debian.org>
 Build-Depends: debhelper (>= 5)
 Build-Depends-Indep: debiandoc-sgml, texlive (>= 2007-11), texlive-latex-extra,
- texlive-lang-cyrillic, latex-cjk-chinese,
+ texlive-lang-chinese, texlive-lang-cyrillic, texlive-lang-english,
+ texlive-lang-french, texlive-lang-german, texlive-lang-italian,
+ texlive-lang-other, texlive-lang-polish, texlive-lang-portuguese,
+ texlive-lang-spanish,
+ latex-cjk-chinese,
  latex-cjk-chinese-arphic-bkai00mp, latex-cjk-chinese-arphic-bsmi00lp,
  latex-cjk-chinese-arphic-gbsn00lp, latex-cjk-chinese-arphic-gkai00mp,
  ghostscript, po4a
