From caa38d7b98eafeea99fcda02e81d7a43bf8e02a5 Mon Sep 17 00:00:00 2001
From: Xiyue Deng <manphiz@gmail.com>
Date: Tue, 1 Oct 2024 22:47:20 -0700
Subject: [PATCH 2/2] Rename to emacs-major-modes

---
 debian/control | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/control b/debian/control
index d471a4b..1c77505 100644
--- a/debian/control
+++ b/debian/control
@@ -42,7 +42,7 @@ Description: Miscellaneous add-ons for Emacs
  See /usr/share/doc/emacs-goodies-el/README.Debian.gz for a discussion
  of the disposition of previously bundled add-ons.
 
-Package: emacs-editing-modes
+Package: emacs-major-modes
 Architecture: all
 Depends: ${misc:Depends}
 Recommends:
@@ -90,6 +90,6 @@ Suggests:
  elpa-rainbow-mode,
 Description: Miscellaneous editing modes for Emacs
  This is a metapackage that recommends or suggests various useful
- Emacs editing modes including many programming modes, configuration
+ Emacs major modes including many programming modes, configuration
  file modes, etc.  The more well-known modes are recommended, and the
  lesser known ones are suggested.
-- 
2.45.2

