--- changelog.Debian.orig	2008-10-06 11:25:41.000000000 +0200
+++ changelog.Debian	2008-10-06 11:25:57.000000000 +0200
@@ -6,7 +6,7 @@
 
 devmapper (2:1.02.27-3) unstable; urgency=low
 
-  * Fix junp to undefined label in udev rules. (closes: #491107)
+  * Fix jump to undefined label in udev rules. (closes: #491107)
 
  -- Bastian Blank <waldi@debian.org>  Thu, 17 Jul 2008 11:14:48 +0200
 
