=== modified file 'debian/watch'
--- debian/watch	2008-04-04 10:02:18 +0000
+++ debian/watch	2012-11-13 22:12:40 +0000
@@ -1,3 +1,3 @@
 version=3
-opts=uversionmangle=s/\.(gz|bz2|lzma)$// \
-ftp://ftp.gnu.org/gnu/coreutils/coreutils-([\d+\.]+)\.tar\.(gz|bz2|lzma) debian uupdate
+opts=uversionmangle=s/\.(gz|bz2|xz|lzma)$// \
+ftp://ftp.gnu.org/gnu/coreutils/coreutils-([\d+\.]+)\.tar\.(gz|bz2|xz|lzma) debian uupdate

