diff -ruNP debconf-1.5.24/Debconf/AutoSelect.pm debconf-1.5.24,p/Debconf/AutoSelect.pm
--- debconf-1.5.24/Debconf/AutoSelect.pm	2005-08-01 19:51:57.000000000 +0200
+++ debconf-1.5.24,p/Debconf/AutoSelect.pm	2009-01-14 15:34:12.000000000 +0100
@@ -41,7 +41,8 @@
 
 );
 
-my $frontend;
+our $frontend;
+our $confmodule;
 my $type;
 
 =head1 METHODS
@@ -69,14 +70,19 @@
 
 	my $showfallback=0;
 	foreach $type ($starttype, @{$fallback{$starttype}}, 'Noninteractive') {
+		my $file = $type;
+		if ( index($type, '=') > 0) {
+			($file, $type) = split /=/, $type;
+			$type=ucfirst($type);
+		}
 		if (! $showfallback) {
-			debug user => "trying frontend $type";
+			debug user => "trying frontend $type (file $file)";
 		}
 		else {
 			warn(sprintf(gettext("falling back to frontend: %s"), $type));
 		}
 		$frontend=eval qq{
-			use Debconf::FrontEnd::$type;
+			use Debconf::FrontEnd::$file;
 			Debconf::FrontEnd::$type->new();
 		};
 		return $frontend if defined $frontend;
@@ -98,7 +104,7 @@
 =cut
 
 sub make_confmodule {
-	my $confmodule=Debconf::ConfModule->new(frontend => $frontend);
+	$confmodule=Debconf::ConfModule->new(frontend => $frontend);
 
 	$confmodule->startup(@_) if @_;
 	
