diff -Nru alsa-lib-1.0.25/debian/rules alsa-lib-1.0.25/debian/rules
--- alsa-lib-1.0.25/debian/rules	2012-02-11 19:44:21.000000000 +0100
+++ alsa-lib-1.0.25/debian/rules	2012-03-05 18:54:52.000000000 +0100
@@ -3,6 +3,9 @@
 # Uncomment this to turn on verbose mode.
 #export DH_VERBOSE=1
 
+# Enable verbose build.
+export V=1
+
 export DEB_BUILD_GNU_TYPE ?= $(shell dpkg-architecture -qDEB_BUILD_GNU_TYPE)
 export DEB_HOST_GNU_TYPE  ?= $(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)
 
@@ -18,13 +21,10 @@
 	'-Vbidev:Depends=libc6-dev-i386'
 endif
 
-CFLAGS = -g -Wall -D_GNU_SOURCE
+export DEB_CFLAGS_MAINT_APPEND = -Wall -D_GNU_SOURCE
 
-ifneq (,$(findstring noopt,$(DEB_BUILD_OPTIONS)))
-	CFLAGS += -O0
-else
-	CFLAGS += -O2
-endif
+DPKG_EXPORT_BUILDFLAGS = 1
+include /usr/share/dpkg/buildflags.mk
 
 autoreconf-stamp:
 	dh_testdir
