diff -Naur cyrus-sasl2-2.1.25.dfsg1/debian.orig/changelog cyrus-sasl2-2.1.25.dfsg1/debian/changelog
--- cyrus-sasl2-2.1.25.dfsg1/debian.orig/changelog	2014-02-10 18:18:36.591910550 -0600
+++ cyrus-sasl2-2.1.25.dfsg1/debian/changelog	2014-02-11 20:22:53.619972685 -0600
@@ -1,3 +1,10 @@
+cyrus-sasl2 (2.1.25.dfsg1-18) unstable; urgency=low
+
+  * Fix the FTBFS with clang:
+    - Fixed return void bug in lib/common.c
+
+ -- Arthur Marble <arthur@playtux.info9.net>  Mon, 10 Feb 2014 17:50:13 -0600
+
 cyrus-sasl2 (2.1.25.dfsg1-17) unstable; urgency=low
 
   * Add libsasl2-modules-db to the include/exclude lists in debian/rules.
diff -Naur cyrus-sasl2-2.1.25.dfsg1/debian.orig/patches/clang-ftbfs.diff cyrus-sasl2-2.1.25.dfsg1/debian/patches/clang-ftbfs.diff
--- cyrus-sasl2-2.1.25.dfsg1/debian.orig/patches/clang-ftbfs.diff	1969-12-31 18:00:00.000000000 -0600
+++ cyrus-sasl2-2.1.25.dfsg1/debian/patches/clang-ftbfs.diff	2014-02-10 18:19:10.099911133 -0600
@@ -0,0 +1,13 @@
+Index: cyrus-sasl2-2.1.25.dfsg1/lib/common.c
+===================================================================
+--- cyrus-sasl2-2.1.25.dfsg1.orig/lib/common.c	2014-02-10 17:45:24.000000000 -0600
++++ cyrus-sasl2-2.1.25.dfsg1/lib/common.c	2014-02-10 17:47:36.379878155 -0600
+@@ -841,7 +841,7 @@
+ 
+   if (!free_mutex || free_mutex == 0x1)
+     free_mutex = sasl_MUTEX_ALLOC();
+-  if (!free_mutex) return SASL_FAIL;
++  if (!free_mutex) return;
+ 
+   result = sasl_MUTEX_LOCK(free_mutex);
+   if (result!=SASL_OK) return;
diff -Naur cyrus-sasl2-2.1.25.dfsg1/debian.orig/patches/series cyrus-sasl2-2.1.25.dfsg1/debian/patches/series
--- cyrus-sasl2-2.1.25.dfsg1/debian.orig/patches/series	2014-02-10 18:18:36.591910550 -0600
+++ cyrus-sasl2-2.1.25.dfsg1/debian/patches/series	2014-02-10 18:19:10.099911133 -0600
@@ -25,3 +25,4 @@
 0038-send_imap_logout.patch
 0039-fix-canonuser-ldapdb-garbage-in-out-buffer.patch
 CVE-2013-4122.patch
+clang-ftbfs.diff
