diff -Nur doc-central-1.8.2.old/cgi/docconfig.py doc-central-1.8.2/cgi/docconfig.py
--- doc-central-1.8.2.old/cgi/docconfig.py	2005-01-27 23:11:11.000000000 +0100
+++ doc-central-1.8.2/cgi/docconfig.py	2007-12-03 19:31:13.000000000 +0100
@@ -18,7 +18,7 @@
 ## System defaults
 
 # Location of the doc-base registries. 
-docbasedirs		= [ "/usr/share/doc-base" ]
+docbasedirs		= [ "/var/lib/doc-base/documents" ]
 
 # Descriptions for the various documentation settings. These should probably
 # also be in some policy document.
diff -Nur doc-central-1.8.2.old/debian/control doc-central-1.8.2/debian/control
--- doc-central-1.8.2.old/debian/control	2005-12-13 02:08:27.000000000 +0100
+++ doc-central-1.8.2/debian/control	2007-12-03 19:40:42.000000000 +0100
@@ -7,7 +7,7 @@
 
 Package: doc-central
 Architecture: all
-Depends: apache | httpd, python, info2www
+Depends: apache | httpd, python, info2www, doc-base (>= 0.8.7)
 Recommends: w3m | www-browser
 Description: web-based documentation browser
  Doc-Central is a tool to browse the documentation installed on your
