From e5bed2a92f642a9068ac4261c01d111b50a0b292 Mon Sep 17 00:00:00 2001
From: Charles Plessy <plessy@debian.org>
Date: Mon, 23 Sep 2013 12:26:31 +0900
Subject: [PATCH] Policy: underline that Built-Using is only to be used when
 license requires.

Closes: #688251
---
 policy.sgml | 8 +++++++-
 1 file changed, 7 insertions(+), 1 deletion(-)

diff --git a/policy.sgml b/policy.sgml
index 113429b..e0b36a7 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -5563,7 +5563,13 @@ Replaces: mail-transport-agent
 	</p>
 
 	<p>
-	  A <tt>Built-Using</tt> field must list the corresponding source
+	  When the licensing terms of the incorporated parts require to provide
+	  their source <footnote>
+	    As of September 2013, the current practice in Debian is to consider
+	    that we can benefit from the GPL exceptions for the libc, GCC and
+	    similar packages.
+	  </footnote>,
+	  a <tt>Built-Using</tt> field must list the corresponding source
 	  package for any such binary package incorporated during the build
 	  <footnote>
 	    <tt>Build-Depends</tt> in the source package is not adequate since
-- 
1.8.4.rc3

