--- ./bin/ftar.orig	2012-04-11 05:06:25.000000000 -0700
+++ ./bin/ftar	2014-08-25 11:19:41.936082615 -0700
@@ -54,7 +54,7 @@
     fi
 
     echo "ftar: extracting $file to $target/$dir" | tr -s '/'
-    $catname $file | tar -C $target/$dir $vflag -xf -
+    $catname $file | tar -C $target/$dir $vflag --numeric-owner -xf -
     tardone=1
     # if option -1 is set, only one class will be used
     [ $single -eq 1 ] && exit 0
