diff -ur apt-0.7.6ubuntu1/methods/copy.cc apt-0.7.6ubuntu1andersk1/methods/copy.cc
--- apt-0.7.6ubuntu1/methods/copy.cc	2006-12-13 13:28:57.000000000 -0500
+++ apt-0.7.6ubuntu1andersk1/methods/copy.cc	2007-08-04 19:45:13.000000000 -0400
@@ -12,6 +12,7 @@
 #include <apt-pkg/fileutl.h>
 #include <apt-pkg/acquire-method.h>
 #include <apt-pkg/error.h>
+#include <apt-pkg/hashes.h>
 
 #include <sys/stat.h>
 #include <utime.h>
@@ -79,6 +80,10 @@
       return _error->Errno("utime",_("Failed to set modification time"));
    }
    
+   Hashes Hash;
+   FileFd Fd(Res.Filename, FileFd::ReadOnly);
+   Hash.AddFD(Fd.Fd(), Fd.Size());
+   Res.TakeHashes(Hash);
    URIDone(Res);
    return true;
 }
