diff -Nurd autopsy-2.08.old/configure autopsy-2.08/configure
--- autopsy-2.08.old/configure	2006-09-01 18:19:15.000000000 +0200
+++ autopsy-2.08/configure	2008-07-03 15:51:10.000000000 +0200
@@ -62,8 +62,8 @@
 for d in $dirs
   do if (test -x ${d}perl) then
     if (test -n "`${d}perl -v 2> /dev/null | awk '/This is perl/ {print $0}'`") then
-      ver=`${d}perl -v 2> /dev/null | awk '/This is perl,/ {print $4}'`;
-      if (test "$ver" '<' "v5.8.0") then 
+      ver=`${d}perl -MConfig -e 'print $Config{"version"};'`;
+      if (`${d}perl -MConfig -Mversion -e 'exit( version->new($Config{"version"}) >= version->new(5.8.0));'`) then 
         echo "old version of perl found: ${d}perl (version $ver) -- continuing";
       else	  
         echo "perl found: ${d}perl (version $ver)";
@@ -84,13 +84,13 @@
     read perlexe;
     if (test -x "$perlexe") then
       if (test -n "`$perlexe -v 2> /dev/null | awk '/This is perl/ {print $0}'`")       then
-        ver=`$perlexe -v 2> /dev/null | awk '/This is perl,/ {print $4}'`;
-        if (test "$ver" '<' "v5.8.0") then 
+        ver=`$perlexe -MConfig -e 'print $Config{"version"};'`;
+        if (`$perlexe -MConfig -Mversion -e 'exit( version->new($Config{"version"}) >= version->new(5.8.0));'`) then 
           echo "This version of Perl is too old, 5.8.0 or older needed";
         else	  
           echo "Correct version found";
-          echo "#!${d}perl -wT" > ./config.tmp;
-          echo "#!${d}perl" > ./config2.tmp;
+          echo "#!${perlexe} -wT" > ./config.tmp;
+          echo "#!${perlexe}" > ./config2.tmp;
           found=1; 
           break; 
         fi;
