diff -Nru developers-reference-3.4.3/developer-duties.dbk developers-reference-3.4.3/developer-duties.dbk
--- developers-reference-3.4.3/developer-duties.dbk	2009-06-24 18:01:42.000000000 +0200
+++ developers-reference-3.4.3/developer-duties.dbk	2009-09-23 22:40:03.000000000 +0200
@@ -202,6 +202,11 @@
 </listitem>
 <listitem>
 <para>
+Retire from the Debian teams you're member of.
+</para>
+</listitem>
+<listitem>
+<para>
 Send an gpg-signed email about why you are leaving the project to
 <email>debian-private@&lists-host;</email>.
 </para>
