--- a/userial/ds9097/linuxlnk.c
+++ b/userial/ds9097/linuxlnk.c
@@ -135,6 +135,7 @@
 	OWERROR(OWERROR_SYSTEM_RESOURCE_INIT_FAILED);
 	perror("owTouchReset: Error with tcsetattr 1");
 	close(fd[portnum]);
+	fd[portnum] = -1;
 	return FALSE;
      }
    
@@ -193,6 +194,7 @@
 	OWERROR(OWERROR_SYSTEM_RESOURCE_INIT_FAILED);
 	perror("Reset: Error with tcsetattr 2");
 	close(fd[portnum]);
+	fd[portnum] = -1;
 	return FALSE;
      }
 
--- a/userial/ds9097/linuxses.c
+++ b/userial/ds9097/linuxses.c
@@ -77,6 +77,7 @@
 	OWERROR(OWERROR_SYSTEM_RESOURCE_INIT_FAILED);
 	perror("owAcquire: failed to set attributes");
 	close(fd[portnum]);
+	fd[portnum] = -1;
 	return FALSE;
      }
    
@@ -106,6 +107,7 @@
 	OWERROR(OWERROR_SYSTEM_RESOURCE_INIT_FAILED);
 	perror("owAcquire: failed to set attributes");	
 	close(fd[portnum]);
+	fd[portnum] = -1;
 	return FALSE;
      }
       
@@ -124,7 +126,6 @@
 	/* We failed doing that */
 	OWERROR(OWERROR_SYSTEM_RESOURCE_INIT_FAILED);
 	perror("owAcquire: failed to set attributes");
-	close(fd[portnum]);
      }
    
    /* Close the port */
@@ -134,6 +135,7 @@
 	OWERROR(OWERROR_SYSTEM_RESOURCE_INIT_FAILED);
 	perror("owAcquire: failed to close port");
      }
+   fd[portnum] = -1;
 
    /* we should return an error condition here but MAXIMS API is 
     * badly designed */
