diff -Nru critterding-1.0-beta12.1/debian/changelog critterding-1.0-beta12.1/debian/changelog
--- critterding-1.0-beta12.1/debian/changelog	2011-10-23 14:19:10.000000000 +0000
+++ critterding-1.0-beta12.1/debian/changelog	2012-04-05 00:45:47.000000000 +0000
@@ -1,3 +1,10 @@
+critterding (1.0-beta12.1-1.2) UNRELEASED; urgency=low
+
+  * Non-maintainer upload.
+  * Fix FTBFS with gcc 4.7 due to missing <unistd.h> include (Closes: #667143).
+
+ -- Cyril Brulebois <kibi@debian.org>  Thu, 05 Apr 2012 00:45:14 +0000
+
 critterding (1.0-beta12.1-1.1) unstable; urgency=low
 
   * Non-maintainer upload.
diff -Nru critterding-1.0-beta12.1/debian/patches/20fix_ftbfs_gcc_4.7 critterding-1.0-beta12.1/debian/patches/20fix_ftbfs_gcc_4.7
--- critterding-1.0-beta12.1/debian/patches/20fix_ftbfs_gcc_4.7	1970-01-01 00:00:00.000000000 +0000
+++ critterding-1.0-beta12.1/debian/patches/20fix_ftbfs_gcc_4.7	2012-04-05 00:44:36.000000000 +0000
@@ -0,0 +1,15 @@
+--- a/src/scenes/evolution.cpp
++++ b/src/scenes/evolution.cpp
+@@ -1,10 +1,9 @@
+-#ifdef _WIN32
+-	#include <unistd.h>
+-#endif
++#include <unistd.h>
+ #include "evolution.h"
+ 
+ # include <math.h>
+ 
++
+ Evolution::Evolution()
+ {
+ 	cmd = Commands::Instance();
diff -Nru critterding-1.0-beta12.1/debian/patches/series critterding-1.0-beta12.1/debian/patches/series
--- critterding-1.0-beta12.1/debian/patches/series	2011-10-23 14:18:02.000000000 +0000
+++ critterding-1.0-beta12.1/debian/patches/series	2012-04-05 00:43:54.000000000 +0000
@@ -1,3 +1,4 @@
 00fontandiconpath
 10uninitialized_constant
 11const_cast
+20fix_ftbfs_gcc_4.7
