From f19f5aa4ce66af8f77404b8df69baadee1e50669 Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Wed, 1 May 2013 09:46:57 +0800
Subject: [PATCH 2/6] Add bug squashing parties for 2013

---
 data/events/bsps | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/data/events/bsps b/data/events/bsps
index b9455d5..e79bbfd 100644
--- a/data/events/bsps
+++ b/data/events/bsps
@@ -123,3 +123,8 @@ Title: Bug squashing party (IRILL, Paris, France)
 Start-Date: Feb 17 2012
 End-Date: Feb 19 2012
 Source: http://lists.debian.org/debian-events-eu/2011/12/msg00000.html
+
+Title: Bug squashing party (Cambridge, United Kingdom)
+Start-Date: Jan 25 2013
+End-Date: Jan 27 2013
+Source: http://lists.debian.org/debian-devel-announce/2013/01/msg00003.html
-- 
1.8.3.rc0

