From 76f7b308e3784489c59661e800a48acef98fa849 Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Wed, 1 May 2013 09:47:58 +0800
Subject: [PATCH 3/6] Add the 20th Anniversary of Debian

---
 data/events/anniversaries | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/data/events/anniversaries b/data/events/anniversaries
index 529a8e4..c1c23e6 100644
--- a/data/events/anniversaries
+++ b/data/events/anniversaries
@@ -9,3 +9,6 @@ Date: Aug 16 2003
 
 Title: 15th Anniversary of Debian
 Date: Aug 16 2008
+
+Title: 20th Anniversary of Debian
+Date: Aug 16 2013
-- 
1.8.3.rc0

