From 5e907f1158d43023ec05cda99ae2afef31f32c3f Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Wed, 1 May 2013 09:56:45 +0800
Subject: [PATCH 5/6] Add announcement of dedup.d.n

---
 data/events/services | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/data/events/services b/data/events/services
index 67b600f..d6a0d8f 100644
--- a/data/events/services
+++ b/data/events/services
@@ -97,3 +97,7 @@ Source: http://lists.debian.org/debian-devel-announce/2012/03/msg00000.html
 Title: Debian Url ChecKer service available
 Date: Dec 31 2012
 Source: http://lists.debian.org/debian-devel-announce/2012/12/msg00002.html
+
+Title: deduplication service available
+Date: Apr 20 2013
+Source: http://lists.debian.org/debian-devel-announce/2013/04/msg00007.html
-- 
1.8.3.rc0

