From ccfc0d491da087b489fb77e8b1106dc6e09d710d Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Wed, 1 May 2013 10:02:27 +0800
Subject: [PATCH 6/6] Add more d-i wheezy release dates

---
 data/events/d-i | 20 ++++++++++++++++++++
 1 file changed, 20 insertions(+)

diff --git a/data/events/d-i b/data/events/d-i
index 029f677..b46e364 100644
--- a/data/events/d-i
+++ b/data/events/d-i
@@ -89,3 +89,23 @@ Source: http://www.debian.org/devel/debian-installer/News/2012/20120513
 Title: First beta of wheezy's debian-installer released
 Date: Aug 4 2012
 Source: http://www.debian.org/devel/debian-installer/News/2012/20120804
+
+Title: Second beta of wheezy's debian-installer released
+Date: Sep 9 2012
+Source: http://www.debian.org/devel/debian-installer/News/2012/20120909
+
+Title: Third beta of wheezy's debian-installer released
+Date: Oct 18 2012
+Source: http://www.debian.org/devel/debian-installer/News/2012/20121018
+
+Title: Fourth beta of wheezy's debian-installer released
+Date: Nov 22 2012
+Source: http://www.debian.org/devel/debian-installer/News/2012/20121122
+
+Title: First RC of wheezy's debian-installer released
+Date: Feb 17 2013
+Source: http://www.debian.org/devel/debian-installer/News/2013/20130217
+
+Title: Second RC of wheezy's debian-installer released
+Date: Apr 27 2013
+Source: http://www.debian.org/devel/debian-installer/News/2013/20130427
-- 
1.8.3.rc0

