--- alsa-utils-1.0.13/debian/control.orig	2007-01-13 16:12:20.000000000 +0100
+++ alsa-utils-1.0.13/debian/control	2007-01-13 16:13:33.000000000 +0100
@@ -8,7 +8,7 @@
 XS-X-Vcs-Svn: svn://svn.debian.org/pkg-alsa/trunk/alsa-utils
 
 Package: alsa-utils
-Architecture: i386 ia64 alpha amd64 arm armeb hppa m32r m68k mips mipsel powerpc ppc64 s390 s390x sh3 sh3eb sh4 sh4eb sparc
+Architecture: i386 ia64 alpha amd64 arm armeb armel hppa m32r m68k mips mipsel powerpc ppc64 s390 s390x sh3 sh3eb sh4 sh4eb sparc
 Provides: audio-mixer
 Depends: ${shlibs:Depends}, whiptail | dialog, module-init-tools | modutils (>= 2.3.5-1), pciutils (>= 1:2.1.11-4), python, lsb-base (>= 3.0-9), linux-sound-base (>= 1.0.11-2)
 Recommends: alsa-base (>= 1.0.13-1)
