--- claws-mail-3.0.2-orig/src/compose.c 2007-11-01 03:27:03.000000000 -0300
+++ claws-mail-3.0.2/src/compose.c      2007-11-01 23:52:17.000000000 -0300
@@ -2409,8 +2412,6 @@
                gtk_text_buffer_insert(buffer, &iter, "\n", 1);
 
                compose->autowrap = prev_autowrap;
-               if (compose->autowrap)
-                       compose_wrap_all(compose);
        }
 
        if (attach) {
