diff -u dirdiff-2.1/debian/changelog dirdiff-2.1/debian/changelog
--- dirdiff-2.1/debian/changelog
+++ dirdiff-2.1/debian/changelog
@@ -1,3 +1,12 @@
+dirdiff (2.1-7.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * debian/control: Depend on tk8.5 (Closes: #771776)
+  * debian/patches/05_wish-tk8.5.patch: Replace wish8.6 with wish8.5 and
+    really use Tk 8.5. libfilecmp is still linked against libtcl8.5 after all.
+
+ -- Sebastian Ramacher <sramacher@debian.org>  Thu, 15 Jan 2015 23:57:44 +0100
+
 dirdiff (2.1-7) unstable; urgency=medium
 
   * patch in the correct rule in debian/rules 
diff -u dirdiff-2.1/debian/control dirdiff-2.1/debian/control
--- dirdiff-2.1/debian/control
+++ dirdiff-2.1/debian/control
@@ -9,7 +9,7 @@
 
 Package: dirdiff
 Architecture: any
-Depends: ${misc:Depends}, ${shlibs:Depends}
+Depends: ${misc:Depends}, ${shlibs:Depends}, tk8.5
 Description: Display and merge changes between two directory trees
  Dirdiff can handle up to 5 trees.  It displays a main window with a
  list of the files which are different between the trees, with colored
diff -u dirdiff-2.1/debian/patches/05_wish-tk8.5.patch dirdiff-2.1/debian/patches/05_wish-tk8.5.patch
--- dirdiff-2.1/debian/patches/05_wish-tk8.5.patch
+++ dirdiff-2.1/debian/patches/05_wish-tk8.5.patch
@@ -4,7 +4,7 @@
  #!/bin/sh
  # Tcl ignores the next line \
 -exec wish "$0" -- "${1+$@}"
-+exec wish8.6 "$0" -- "${1+$@}"
++exec wish8.5 "$0" -- "${1+$@}"
  
  # Copyright (C) 1999-2004 Paul Mackerras.  All rights reserved.
  # This program is free software; it may be used, copied, modified
