From 2e278271f550ddc237b77b41626f7f73a8dca3c8 Mon Sep 17 00:00:00 2001
From: Vagrant Cascadian <vagrant@reproducible-builds.org>
Date: Tue, 29 Dec 2020 07:36:26 +0000
Subject: [PATCH 3/6] debian/control: Add help2man to Build-Depends.

---
 debian/control | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/control b/debian/control
index 7ebb0b4..74edb74 100644
--- a/debian/control
+++ b/debian/control
@@ -15,6 +15,7 @@ Build-Depends: autotools-dev,
                graphicsmagick-imagemagick-compat,
                groff-base,
                gv,
+               help2man,
                html2ps,
                libpaper-dev,
                lpr,
-- 
2.20.1

