From 635b679d09b9eafabaa9a20747fce865d6ff50fb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?David=20Pr=C3=A9vot?= <taffit@debian.org>
Date: Mon, 9 Dec 2013 21:37:07 -0400
Subject: [PATCH] Policy: DEHS service is dead

---
 policy.sgml | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/policy.sgml b/policy.sgml
index dad8d23..4eb55d9 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -2367,8 +2367,7 @@ endif
           This is an optional, recommended configuration file for the
           <tt>uscan</tt> utility which defines how to automatically scan
           ftp or http sites for newly available updates of the
-          package. This is used
-          by <url id="http://dehs.alioth.debian.org/"> and other Debian QA
+          package. This is used Debian QA
           tools to help with quality control and maintenance of the
           distribution as a whole.
         </p>
-- 
1.8.5.1

