--- ap/aptitude-0.6.3/src/main.cc	2010-11-21 20:30:36.000000000 +0100
+++ aptitude/aptitude-0.6.3/src/main.cc	2010-11-21 20:29:47.000000000 +0100
@@ -203,6 +203,7 @@
   printf(_(" why-not      - Show the manually installed packages that lead to a conflict\n"
            "                with the given package, or why one or more packages would\n"
            "                lead to a conflict with the given package if installed.\n"));
+  printf(_(" versions     - Displays the versions of the packages listed on the command-line.\n");
   printf("\n");
   printf(_("  Options:\n"));
   printf(_(" -h             This help text.\n"));

