--- mac-usb-us.kmap.dist	2003-09-01 16:03:26.000000000 +0200
+++ mac-usb-us.kmap	2003-09-01 16:03:26.000000000 +0200
@@ -70,11 +70,5 @@
 keycode 101 = Pause # Break/Pause
 	control	keycode 101 = Break # Ctrl+Break/Pause
 
-# Apple portables fn key + Apple logo key
-keycode 116 = Compose
 # Apple keypad has equal
 keycode 117 = equal
-
-# Apple logo keys
-keycode 125 = VoidSymbol # left Command/Apple key
-keycode 126 = Compose # right Command/Apple key
