--- snmp.php.PG 2012-01-08 22:37:52.000000000 +0100
+++ snmp.php    2012-01-19 21:01:42.000000000 +0100
@@ -254,7 +254,6 @@
 
         /* do not attempt to poll invalid combinations */
         if (($version == 0) || (!is_numeric($version)) ||
-               (!is_numeric($max_oids)) ||
                 (!is_numeric($port)) ||
                 (!is_numeric($retries)) ||
                 (!is_numeric($timeout)) ||
@@ -344,7 +343,7 @@
                        $temp_array = exec_into_array(escapeshellcmd(read_config_option("path_snmpwalk")) . " -v$version -t $timeout -r $retries " . escapeshellarg($hostname) . ":$port $snmp_auth " . escapeshellarg($oid));
                }else {
                        if (file_exists($path_snmpbulkwalk) && ($version > 1)) {
-                               $temp_array = exec_into_array(escapeshellcmd($path_snmpbulkwalk) . " -O Qn $snmp_auth -v $version -t $timeout -r $retries -Cr$max_oids " . escapeshellarg($hostname) . ":$port " . escapeshellarg($oid));
+                               $temp_array = exec_into_array(escapeshellcmd($path_snmpbulkwalk) . " -O Qn $snmp_auth -v $version -t $timeout -r $retries -Cr50 " . escapeshellarg($hostname) . ":$port " . escapeshellarg($oid));
                        }else{
                                $temp_array = exec_into_array(escapeshellcmd(read_config_option("path_snmpwalk")) . " -O Qn $snmp_auth -v $version -t $timeout -r $retries " . escapeshellarg($hostname) . ":$port " . escapeshellarg($oid));
                        }
