--- dpkg-maintscript-helper.orig	2014-10-27 14:58:28.663122395 +0100
+++ dpkg-maintscript-helper	2014-10-27 15:00:49.463124673 +0100
@@ -210,8 +210,10 @@
 	ensure_package_owns_file "$PACKAGE" "$OLDCONFFILE" || return 0
 
 	echo "Preserving user changes to $NEWCONFFILE (renamed from $OLDCONFFILE)..."
-	mv -f "$NEWCONFFILE" "$NEWCONFFILE.dpkg-new"
-	mv -f "$OLDCONFFILE" "$NEWCONFFILE"
+	if [ -e "$NEWCONFFILE" ]; then
+	    mv -f "$NEWCONFFILE" "$NEWCONFFILE.dpkg-new"
+    	fi
+    	mv -f "$OLDCONFFILE" "$NEWCONFFILE"
 }
 
 abort_mv_conffile() {
