--- merge-lst-to-xml.orig	2009-09-08 22:11:42.777558468 +0200
+++ merge-lst-to-xml	2009-09-08 22:18:59.330558360 +0200
@@ -4,7 +4,8 @@
 import optparse
 import string
 
-from elementtree import ElementTree, XMLTreeBuilder
+from xml.etree import ElementTree
+from xml.etree.ElementTree import XMLTreeBuilder
 
 class LstParser:
     """Parser for discover 1 device lists.  Once initialized, the
@@ -116,14 +117,14 @@
 
         return False
 
-class TreeBuilderWithComments(XMLTreeBuilder.FancyTreeBuilder):
-    """This class extends ElementTree's FancyTreeBuilder to
+class TreeBuilderWithComments(XMLTreeBuilder):
+    """This class extends ElementTree's to
     parse comments, which no builder in ElementTree seems able
     to do by itself.
     """
 
     def __init__(self):
-        XMLTreeBuilder.FancyTreeBuilder.__init__(self)
+        XMLTreeBuilder.__init__(self)
         self._parser.CommentHandler = self._comment
 
     def _comment(self, data):
