diff --git a/debian/control b/debian/control
index 1994a1b..a4f66f8 100644
--- a/debian/control
+++ b/debian/control
@@ -10,7 +10,7 @@ Vcs-Browser: http://svn.debian.org/viewsvn/collab-maint/ext-maint/aqemu/trunk/
 
 
 Package: aqemu
-Architecture: amd64 arm armel hppa i386 powerpc sparc ppc64
+Architecture: amd64 arm armel hppa i386 ia64 mips mipsel powerpc sparc sparc64 kfreebsd-i386 kfreebsd-amd64 ppc64
 Depends: qemu, ${shlibs:Depends}, ${misc:Depends}
 Description: Qt4 front-end for QEMU and KVM
  aqemu is a Qt4 graphical interface used to manage QEMU and KVM virtual
