diff -Nru dpkg-1.18.10.orig/man/dpkg-scanpackages.1 dpkg-1.18.10/man/dpkg-scanpackages.1
--- dpkg-1.18.10.orig/man/dpkg-scanpackages.1	2016-07-05 03:55:13.000000000 +0200
+++ dpkg-1.18.10/man/dpkg-scanpackages.1	2016-08-11 00:04:27.000000000 +0200
@@ -24,7 +24,7 @@
 .
 .SH SYNOPSIS
 .B dpkg\-scanpackages
-.RI [ option "...] " binary-dir
+.RI [ option "...] " binary-path
 .RI [ override-file
 .RI [ path-prefix ]]
 .B >
@@ -57,7 +57,7 @@
 .B file://
 sources).
 .PP
-.I binary-dir
+.I binary-path
 is the name of the tree of the binary packages to process (for example,
 .BR contrib/binary\-i386 ).
 It is best to make this relative to the root of the Debian archive,
diff -Nru dpkg-1.18.10.orig/scripts/dpkg-scanpackages.pl dpkg-1.18.10/scripts/dpkg-scanpackages.pl
--- dpkg-1.18.10.orig/scripts/dpkg-scanpackages.pl	2016-07-05 03:55:14.000000000 +0200
+++ dpkg-1.18.10/scripts/dpkg-scanpackages.pl	2016-08-11 00:04:48.000000000 +0200
@@ -231,10 +231,10 @@
     }
 }
 
-my ($binarydir, $override, $pathprefix) = @ARGV;
+my ($binarypath, $override, $pathprefix) = @ARGV;
 
-if (not -d $binarydir) {
-    error(g_('binary directory %s not found'), $binarydir);
+if (not -e $binarypath) {
+    error(g_('binary path %s not found'), $binarypath);
 }
 if (defined $override and not -e $override) {
     error(g_('override file %s not found'), $override);
@@ -253,7 +253,7 @@
     push @archives, $File::Find::name if m/$find_filter/;
 };
 
-find({ follow => 1, follow_skip => 2, wanted => $scan_archives}, $binarydir);
+find({ follow => 1, follow_skip => 2, wanted => $scan_archives}, $binarypath);
 foreach my $fn (@archives) {
     process_deb($pathprefix, $fn);
 }
