diff -u autopsy-2.24/debian/changelog autopsy-2.24/debian/changelog
--- autopsy-2.24/debian/changelog
+++ autopsy-2.24/debian/changelog
@@ -1,3 +1,10 @@
+autopsy (2.24-1.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Use "find -perm /x" instead of "find -perm +x". Closes: #803030
+
+ -- Andreas Metzler <ametzler@debian.org>  Sun, 20 Dec 2015 14:02:36 +0100
+
 autopsy (2.24-1) unstable; urgency=low
 
   * New upstream release
only in patch2:
unchanged:
--- autopsy-2.24.orig/Makefile
+++ autopsy-2.24/Makefile
@@ -21,7 +21,7 @@
 	rm -f ./lib/*.bak
 	rm -f ./base/*.bak
 	find . -name ".DS_Store" | xargs rm -f
-	find . -type f -perm +g+x,o+x,u+x | xargs  chmod -x
+	find . -type f -perm /g+x,o+x,u+x | xargs  chmod -x
 	grep "curtskver=" ./configure
 	grep "VER" ./lib/define.pl
 	find . -name ".*" | grep -v perltidy
