--- Downloader.pm	2018-11-05 12:57:41.591465757 -0800
+++ Downloader.pm.patched	2018-11-05 12:58:07.300086945 -0800
@@ -103,9 +103,9 @@
         $request = HTTP::Request->new('GET', "$url");
         $response = $self->user_agent->request($request, $fname);
         if (!$response->is_success) {
-            uscan_warn(defined $pkg_dir ? "In directory $pkg_dir,  d" : "D")
+            uscan_warn(defined $pkg_dir ? "In directory $pkg_dir,  d" : "D"
               . "ownloading\n  $url failed: "
-              . $response->status_line;
+              . $response->status_line);
             return 0;
         }
     } else {    # elsif ($$optref{'mode'} eq 'git')
