diff -aur antiword-0.37.orig/debian/rules antiword-0.37/debian/rules
--- antiword-0.37.orig/debian/rules	2011-11-26 14:14:01.000000000 +0100
+++ antiword-0.37/debian/rules	2011-12-28 23:43:01.000000000 +0100
@@ -5,14 +5,12 @@
 # Uncomment this to turn on verbose mode.
 export DH_VERBOSE=1
 
-CFLAGS = -Wall -pedantic -D_FILE_OFFSET_BITS=64
-LDFLAGS=
 
-ifneq (,$(findstring noopt,$(DEB_BUILD_OPTIONS)))
-        CFLAGS += -O0
-else
-        CFLAGS += -O2
-endif
+CFLAGS = `dpkg-buildflags --get CFLAGS`
+CFLAGS += -Wall -pedantic -D_FILE_OFFSET_BITS=64
+CFLAGS += `dpkg-buildflags --get CPPFLAGS`
+
+LDFLAGS = `dpkg-buildflags --get LDFLAGS`
 
 ifneq (,$(findstring nostrip,$(DEB_BUILD_OPTIONS)))
         CFLAGS += -g -DDEBUG
Nur in antiword-0.37/debian: rules~.
Nur in antiword-0.37.orig/: kantiword.
