diff --git a/lib/autopkgtest_qemu.py b/lib/autopkgtest_qemu.py
index f958e0f..0cf307d 100644
--- a/lib/autopkgtest_qemu.py
+++ b/lib/autopkgtest_qemu.py
@@ -206,7 +206,7 @@ class Qemu:
             subdirectory of $TMPDIR)
         """
 
-        self.consoles = set(['hvc0', 'hvc1'])
+        self.consoles = set(['hvc0', 'hvc1', 'hvc2'])
         self.cpus = cpus
         self.images = []    # type: List[QemuImage]
         self.overlay_dir = overlay_dir
@@ -359,7 +359,7 @@ class Qemu:
             ) % self.shareddir,
         ])
 
-        for hvc in ('hvc0', 'hvc1'):
+        for hvc in self.consoles:
             if hvc == 'hvc0' and self.qemu_architecture == 'ppc64le':
                 # The first -serial argument shows up as /dev/hvc0 in
                 # the VM on ppc64le, so identify it as such
diff --git a/virt/autopkgtest-virt-qemu b/virt/autopkgtest-virt-qemu
index 2b5b3fe..a92cee5 100755
--- a/virt/autopkgtest-virt-qemu
+++ b/virt/autopkgtest-virt-qemu
@@ -151,7 +151,7 @@ def setup_shell() -> str:
     user = args.user
     password = args.password
 
-    for name in ('hvc1', 'ttyS1'):
+    for name in ('hvc1', 'ttyS1', 'hvc2'):
         # if the VM is already prepared to start a root shell on ttyS1, just use it
         if name not in qemu.consoles:
             continue
