Index: autopkgtest-5.27/lib/autopkgtest_qemu.py
===================================================================
--- autopkgtest-5.27.orig/lib/autopkgtest_qemu.py
+++ autopkgtest-5.27/lib/autopkgtest_qemu.py
@@ -368,12 +368,15 @@ class Qemu:
                     'unix:%s,server=on,wait=off' % self.get_socket_path('hvc0'),
                 ])
             else:
+                if hvc == 'hvc0' or self.qemu_architecture == 'ppc64le':
+                    argv.extend([
+                        '-device', 'virtio-serial',
+                    ])
                 argv.extend([
                     '-chardev',
                     'socket,path=%s,server=on,wait=off,id=%s' % (
                         self.get_socket_path(hvc), hvc,
                     ),
-                    '-device', 'virtio-serial',
                     '-device', 'virtconsole,chardev=%s' % hvc,
                 ])
 
