Index: debian/README.Debian
===================================================================
--- debian.orig/README.Debian	2004-04-03 16:39:40.000000000 +0100
+++ debian/README.Debian	2004-04-03 23:37:37.000000000 +0100
@@ -37,53 +37,6 @@
 
 -- Wichert Akkerman <wakkerma@debian.org>, Sun Dec 19 15:50:48 CET 1999
 
-Control of Keyword Expansion
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~
-
-I've copied support for this via CVSROOT/options from FreeBSD.
-The options file can include tag and tagexpand directives. The
-tag directive defines a local keyword (if you've ever seen
-$FreeBSD in a file, now you know what it is. So when do we start
-seeing $Debian? :) and the tagexpand directive defines which
-keywords (if any) should be expanded.
-
-This is more useful than using the normal -k options to control
-keyword expansion because you can simply turn off all expansion
-except for your local keyword. This is in fact what the FreeBSD
-project does.
-
-The tag directive looks like
-
-tag=<NEWKEYWORD>=<KEYWORD>
-
-where <NEWKEYWORD> is your new local keyword and <KEYWORD> is the
-standard keyword it mimics (one of Author, Date, CVSHeader,
-Header, Id, Locker, Log, Name, RCSfile, Revision, Source, or
-State) CVSHeader (a better Id which also shows the file's
-location in the repository) is the most common choice; support
-for it was also added by this patch.
-
-The tagexpand directive looks like
-
-tagexpand=<INCLUDE><KEYWORDS>
-
-where <INCLUDE> is either i (if you will be listing the keywords
-to expand) or e (if you will be listing the keywords not to
-expand) and <KEYWORDS> is a comma-separated list of keywords to
-include or exclude. In the absence of a tagexpand directive, all
-keywords are expanded according to the normal rules.
-
-tagexpand=iDate,iDebian
-
-The following example tells CVS to create a new $Foo keyword
-which looks like $CVSHeader and to expand only the $Date and $Foo
-keywords.
-
-tag=Foo=CVSHeader
-tagexpand=iFoo,Date
-
- -- Tollef Fog Heen <tfheen@debian.org>  Mon,  8 Apr 2002 12:33:01 +0200
-
 PAM Support
 ~~~~~~~~~~~
 
@@ -143,3 +96,15 @@
 accessed.
 
  -- Steve McIntyre <93sam@debian.org> Tue, 10 Feb 2004 19:54:50 +0000
+
+Control of Keyword Expansion
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Since version 1.12.2 CVS has supported, without external 
+patches, custom keyword expansion options.  Previously CVS required a 
+patch to implement this, and users may know the feature as the options 
+"tag" and "tagexpand" from the CVSROOT/options file.  CVS now uses a 
+similar method in CVSROOT/config.  For more information see the CVS 
+documentation(infobrowser "(CVS)Configuring keyword expansion").
+
+ -- James Rowe <Jay@jnrowe.ukfsn.org>  Sat,  03 Apr 2004 23:23:57 +0100
