diff --git a/debian/changelog b/debian/changelog
index 35139fb..15848b0 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,9 @@
+cups (1.4.4-7+squeeze3) stable-security; urgency=high
+
+  * Ship cups-files.conf's manpage in cups (Closes: #697543).
+
+ -- Didier Raboud <odyx@debian.org>  Mon, 07 Jan 2013 08:12:16 +0100
+
 cups (1.4.4-7+squeeze2) stable-security; urgency=high
 
   * Backport upstream configuration files split:
diff --git a/debian/cups.install b/debian/cups.install
index 1daa312..5ae5567 100644
--- a/debian/cups.install
+++ b/debian/cups.install
@@ -28,6 +28,8 @@ usr/share/man/man5/classes.conf.5.gz
 usr/share/man/*/man5/classes.conf.5.gz
 usr/share/man/man5/cupsd.conf.5.gz
 usr/share/man/*/man5/cupsd.conf.5.gz
+usr/share/man/man5/cups-files.conf.5.gz
+usr/share/man/*/man5/cups-files.conf.5.gz
 usr/share/man/man5/cups-snmp.conf.5.gz
 usr/share/man/*/man5/cups-snmp.conf.5.gz
 usr/share/man/man5/mailto.conf.5.gz
