Index: automake-1.15-1.15.1/m4/vala.m4
===================================================================
--- automake-1.15-1.15.1.orig/m4/vala.m4
+++ automake-1.15-1.15.1/m4/vala.m4
@@ -19,7 +19,7 @@
 # AM_PROG_VALAC([MINIMUM-VERSION], [ACTION-IF-FOUND], [ACTION-IF-NOT-FOUND])
 # --------------------------------------------------------------------------
 AC_DEFUN([AM_PROG_VALAC],
-  [AC_PATH_PROG([VALAC], [valac], [valac])
+  [AC_PATH_TOOL([VALAC], [valac], [valac])
    AS_IF([test "$VALAC" != valac && test -n "$1"],
       [AC_MSG_CHECKING([whether $VALAC is at least version $1])
        am__vala_version=`$VALAC --version | sed 's/Vala  *//'`
