diff -Nru bootcd-3.11/debian/bootcd.install bootcd-3.12/debian/bootcd.install
--- bootcd-3.11/debian/bootcd.install	2007-11-30 05:37:38.000000000 -0700
+++ bootcd-3.12/debian/bootcd.install	2008-09-10 19:31:59.000000000 -0600
@@ -3,7 +3,6 @@
 usr/share/bootcd/bootcd-check.lib
 usr/share/bootcd/bootcd2disk.conf
 usr/share/bootcd/default.txt
-usr/share/bootcd/bootcd-ia64.lib
 usr/share/bootcd/bootcd2disk
 usr/share/bootcd/bootcdflopcp
 usr/share/bootcd/S13bootcdflop.sh
diff -Nru bootcd-3.11/debian/changelog bootcd-3.12/debian/changelog
--- bootcd-3.11/debian/changelog	2008-07-29 03:08:01.000000000 -0600
+++ bootcd-3.12/debian/changelog	2008-09-10 19:32:51.000000000 -0600
@@ -1,3 +1,9 @@
+bootcd (3.12) UNRELEASED; urgency=low
+
+  * Remove bootcd-ia64.lib from bootcd binary to avoid conflict
+
+ -- dann frazier <dannf@debian.org>  Wed, 10 Sep 2008 19:32:05 -0600
+
 bootcd (3.11) unstable; urgency=low
 
   * the bootcd-ia64.lib is now in bootcd-ia64. This bug was reported by 
