--- calendar.debian.orig        2012-12-14 11:24:20.223598165 +0200
+++ calendar.debian     2012-12-14 11:26:49.219447168 +0200
@@ -45,6 +45,7 @@
 Jun  6 Debian GNU/Linux 3.1 alias ``sarge'' released, 2005
 Apr  8         Debian GNU/Linux 4.0 alias ``etch'' released, 2007
 Feb 14         Debian GNU/Linux 5.0 alias ``lenny'' released, 2009
+Feb  6  Debian GNU/Linux 6.0 alias ``Squeeze'' released, 2011
 
 /* Notably bug report numbers */
 Jun  7 Debian Bug#100000 reported by David B. Harris, 2001
