--- 75dbus_dbus-launch.orig	2008-04-26 13:12:00.000000000 +0200
+++ 75dbus_dbus-launch	2008-05-08 19:47:07.295064128 +0200
@@ -5,6 +5,7 @@
 
 STARTDBUS=
 DBUSLAUNCH=/usr/bin/dbus-launch
+DBUSOPTS=
 
 if grep -qs ^use-session-dbus "$OPTIONFILE"; then
   if [ -x "$DBUSLAUNCH" ]; then
@@ -12,6 +13,14 @@
   fi
 fi
 
+# If the user has a .dbus/session.conf in its home directory then use it
+# instead of the default session file. -- <garden@acheronte.it> 20080508
+
+DBUSUSERSESSIONCONF=$HOME/.dbus/session.conf
+if [ -f "$DBUSUSERSESSIONCONF" ]; then
+  DBUSOPTS="--config-file=$DBUSUSERSESSIONCONF"
+fi
+
 if [ -n "$STARTDBUS" ]; then
-  STARTUP="$DBUSLAUNCH --exit-with-session $STARTUP"
+  STARTUP="$DBUSLAUNCH $DBUSOPTS --exit-with-session $STARTUP"
 fi
