diff -Nru autopkgtest-1.2.0/debian/changelog autopkgtest-1.2.0+nmu1/debian/changelog
--- autopkgtest-1.2.0/debian/changelog	2008-08-27 23:34:48.000000000 +0200
+++ autopkgtest-1.2.0+nmu1/debian/changelog	2010-01-12 00:52:27.000000000 +0100
@@ -1,3 +1,15 @@
+autopkgtest (1.2.0+nmu1) unstable; urgency=low
+
+  * Non-maintainer upload, to drop Python 2.4; thanks to Luca Falavigna for the
+    report and to Jakub Wilk for the patch; Closes: #562408
+    - debian/control
+      + Depends on 'python (>= 2.4)'
+    - runner/adt-run, virt-subproc/adt-virt-*
+      + uses the default Python interpreter instead of hard-coding python2.4;
+      also Closes: #548985
+
+ -- Sandro Tosi <morph@debian.org>  Tue, 12 Jan 2010 00:52:25 +0100
+
 autopkgtest (1.2.0) unstable; urgency=medium
 
   Packaging changes:
diff -Nru autopkgtest-1.2.0/debian/control autopkgtest-1.2.0+nmu1/debian/control
--- autopkgtest-1.2.0/debian/control	2008-08-27 23:34:48.000000000 +0200
+++ autopkgtest-1.2.0+nmu1/debian/control	2010-01-12 00:48:14.000000000 +0100
@@ -7,7 +7,7 @@
 
 Package: autopkgtest
 Architecture: all
-Depends: python2.4
+Depends: python (>= 2.4)
 Conflicts: autodebtest (<< 0.5.3)
 Replaces: autodebtest (<< 0.5.3)
 Recommends: apt-utils
diff -Nru autopkgtest-1.2.0/runner/adt-run autopkgtest-1.2.0+nmu1/runner/adt-run
--- autopkgtest-1.2.0/runner/adt-run	2008-08-27 23:34:48.000000000 +0200
+++ autopkgtest-1.2.0+nmu1/runner/adt-run	2010-01-12 00:37:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.4
+#!/usr/bin/python
 #
 # adt-run is part of autopkgtest
 # autopkgtest is a tool for testing Debian binary packages
diff -Nru autopkgtest-1.2.0/virt-subproc/adt-virt-chroot autopkgtest-1.2.0+nmu1/virt-subproc/adt-virt-chroot
--- autopkgtest-1.2.0/virt-subproc/adt-virt-chroot	2008-08-27 23:34:48.000000000 +0200
+++ autopkgtest-1.2.0+nmu1/virt-subproc/adt-virt-chroot	2010-01-12 00:37:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.4
+#!/usr/bin/python
 #
 # adt-virt-chroot is part of autopkgtest
 # autopkgtest is a tool for testing Debian binary packages
diff -Nru autopkgtest-1.2.0/virt-subproc/adt-virt-null autopkgtest-1.2.0+nmu1/virt-subproc/adt-virt-null
--- autopkgtest-1.2.0/virt-subproc/adt-virt-null	2008-08-27 23:34:48.000000000 +0200
+++ autopkgtest-1.2.0+nmu1/virt-subproc/adt-virt-null	2010-01-12 00:37:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.4
+#!/usr/bin/python
 #
 # adt-virt-null is part of autopkgtest
 # autopkgtest is a tool for testing Debian binary packages
diff -Nru autopkgtest-1.2.0/virt-subproc/adt-virt-xenlvm autopkgtest-1.2.0+nmu1/virt-subproc/adt-virt-xenlvm
--- autopkgtest-1.2.0/virt-subproc/adt-virt-xenlvm	2008-08-27 23:34:48.000000000 +0200
+++ autopkgtest-1.2.0+nmu1/virt-subproc/adt-virt-xenlvm	2010-01-12 00:37:24.000000000 +0100
@@ -1,4 +1,4 @@
-#!/usr/bin/python2.4
+#!/usr/bin/python
 #
 # adt-virt-xenlvm is part of autopkgtest
 # autopkgtest is a tool for testing Debian binary packages
