=== modified file 'bzrlib/msgeditor.py'
--- bzrlib/msgeditor.py	
+++ bzrlib/msgeditor.py	
@@ -89,6 +89,7 @@
     """
     import tempfile
 
+    msgfilename = None
     try:
         tmp_fileno, msgfilename = tempfile.mkstemp(prefix='bzr_log.', dir=u'.')
         msgfile = os.close(tmp_fileno)
@@ -135,8 +136,11 @@
             return "".join(msg)
     finally:
         # delete the msg file in any case
-        try: os.unlink(msgfilename)
-        except IOError: pass
+        if msgfilename is not None:
+            try:
+                os.unlink(msgfilename)
+            except IOError, e:
+                mutter("failed to unlink %s: %s; ignored", msgfilename, e)
 
 
 def make_commit_message_template(working_tree, specific_files):

