From 2a389cbb44e58e573b06f84f0a601cc64ce69718 Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas.boulenguez@free.fr>
Date: Thu, 2 Jun 2016 23:58:43 +0200
Subject: [PATCH 01/20] Replace HTTP with HTTPS in VCS-* fields.

---
 debian/control | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/control b/debian/control
index 3530be7..a3c958a 100644
--- a/debian/control
+++ b/debian/control
@@ -5,8 +5,8 @@ Maintainer: Adrian-Ken Rueegsegger <ken@codelabs.ch>
 Build-Depends: debhelper (>= 9), gnat, gnat-4.9, libasis2014-dev, quilt
 Standards-Version: 3.9.6
 Homepage: http://home.datacomm.ch/t_wolf/tw/ada95/adabrowse/
-Vcs-Browser: http://git.codelabs.ch/?p=debian/adabrowse.git
-Vcs-Git: http://git.codelabs.ch/git/debian/adabrowse.git
+Vcs-Browser: https://git.codelabs.ch/?p=debian/adabrowse.git
+Vcs-Git: https://git.codelabs.ch/git/debian/adabrowse.git
 
 Package: adabrowse
 Architecture: any
-- 
2.8.1

