From 2dc26836ccc2ffe38e27a484b12072639ea6b6d3 Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas.boulenguez@free.fr>
Date: Fri, 3 Jun 2016 00:09:04 +0200
Subject: [PATCH 04/20] Require dpkg-dev providing buildflags.mk.

---
 debian/control | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index 3870b64..8636d0d 100644
--- a/debian/control
+++ b/debian/control
@@ -2,7 +2,9 @@ Source: adabrowse
 Section: devel
 Priority: optional
 Maintainer: Adrian-Ken Rueegsegger <ken@codelabs.ch>
-Build-Depends: debhelper (>= 9), gnat, gnat-4.9, libasis2014-dev
+Build-Depends: debhelper (>= 9), dpkg-dev (>= 1.16.1),
+# dpkg-dev 1.16.1 provides /usr/share/dpkg/default.mk
+ gnat, gnat-4.9, libasis2014-dev
 Standards-Version: 3.9.6
 Homepage: http://home.datacomm.ch/t_wolf/tw/ada95/adabrowse/
 Vcs-Browser: https://git.codelabs.ch/?p=debian/adabrowse.git
-- 
2.8.1

