From 9f3c2afcdad1905116556dd902e74c0fbcdd226a Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas.boulenguez@free.fr>
Date: Fri, 3 Jun 2016 00:36:45 +0200
Subject: [PATCH 08/20] Standards-Version: 3.9.8 (no changes).

---
 debian/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index 8636d0d..198955b 100644
--- a/debian/control
+++ b/debian/control
@@ -5,7 +5,7 @@ Maintainer: Adrian-Ken Rueegsegger <ken@codelabs.ch>
 Build-Depends: debhelper (>= 9), dpkg-dev (>= 1.16.1),
 # dpkg-dev 1.16.1 provides /usr/share/dpkg/default.mk
  gnat, gnat-4.9, libasis2014-dev
-Standards-Version: 3.9.6
+Standards-Version: 3.9.8
 Homepage: http://home.datacomm.ch/t_wolf/tw/ada95/adabrowse/
 Vcs-Browser: https://git.codelabs.ch/?p=debian/adabrowse.git
 Vcs-Git: https://git.codelabs.ch/git/debian/adabrowse.git
-- 
2.8.1

