From fb0a195e9d4d9cf5ce0d4986ac3ef15cab365388 Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas.boulenguez@free.fr>
Date: Fri, 3 Jun 2016 00:39:26 +0200
Subject: [PATCH 09/20] Build with gnat-6 and libasis2015-dev.

---
 debian/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index 198955b..38e43b7 100644
--- a/debian/control
+++ b/debian/control
@@ -4,7 +4,7 @@ Priority: optional
 Maintainer: Adrian-Ken Rueegsegger <ken@codelabs.ch>
 Build-Depends: debhelper (>= 9), dpkg-dev (>= 1.16.1),
 # dpkg-dev 1.16.1 provides /usr/share/dpkg/default.mk
- gnat, gnat-4.9, libasis2014-dev
+ gnat, gnat-6, libasis2015-dev
 Standards-Version: 3.9.8
 Homepage: http://home.datacomm.ch/t_wolf/tw/ada95/adabrowse/
 Vcs-Browser: https://git.codelabs.ch/?p=debian/adabrowse.git
-- 
2.8.1

