--- a/aoe-sancheck.c
+++ b/aoe-sancheck.c
@@ -513,8 +513,18 @@ ethlist(char **ifs, int nifs)
 		ifr.ifr_ifindex = i;
 		if (ioctl(s, SIOCGIFNAME, &ifr) < 0)
 			continue;
-		if (strncmp(ifr.ifr_name, "eth", 3))
+        // get interface flags
+		if (ioctl(s, SIOCGIFFLAGS, &ifr) < 0)
 			continue;
+        // only use interfaces that use arp protocol
+        if (ifr.ifr_flags & IFF_NOARP)
+            continue;
+        // only use interfaces that are up
+        if (!(ifr.ifr_flags & IFF_UP))
+            continue;
+        // skip loopback interfaces
+        if (ifr.ifr_flags & IFF_LOOPBACK)
+            continue;
 		inserteth(ifs, nifs, ifr.ifr_name);
 		n++;
 	}
