diff -u brltty-3.7.2/Programs/usb_linux.c brltty-3.7.2/Programs/usb_linux.c
--- brltty-3.7.2/Programs/usb_linux.c
+++ brltty-3.7.2/Programs/usb_linux.c
@@ -30,7 +30,6 @@
 #include <sys/stat.h>
 #include <sys/vfs.h>
 #include <sys/ioctl.h>
-#include <linux/compiler.h>
 #include <linux/usbdevice_fs.h>
 
 #ifndef USBDEVFS_DISCONNECT
diff -u brltty-3.7.2/debian/changelog brltty-3.7.2/debian/changelog
--- brltty-3.7.2/debian/changelog
+++ brltty-3.7.2/debian/changelog
@@ -1,3 +1,12 @@
+brltty (3.7.2-7.2) unstable; urgency=high
+
+  * Non-maintainer upload.
+  * High-urgency upload for RC bug fix.
+  * Fix FTBFS with linux-libc-dev (closes: #414068).
+  * libbluetooth2-dev build-dep doesn't apply to non-linux architectures.
+
+ -- Julien Cristau <jcristau@debian.org>  Wed, 18 Jul 2007 16:50:34 +0200
+
 brltty (3.7.2-7.1) unstable; urgency=high
 
   * Non-maintainer upload.
diff -u brltty-3.7.2/debian/control brltty-3.7.2/debian/control
--- brltty-3.7.2/debian/control
+++ brltty-3.7.2/debian/control
@@ -3,7 +3,8 @@
 Priority: extra
 Maintainer: Mario Lang <mlang@debian.org>
 Build-Depends: debhelper (>= 4.2), bison, doxygen, linuxdoc-tools, groff,
- flite1-dev, libncurses5-dev, libxaw7-dev, libatspi-dev, libbluetooth2-dev | libbluetooth-dev
+ flite1-dev, libncurses5-dev, libxaw7-dev, libatspi-dev,
+ libbluetooth2-dev [!hurd-i386 !kfreebsd-i386 !kfreebsd-amd64] | libbluetooth-dev [!hurd-i386 !kfreebsd-i386 !kfreebsd-amd64]
 Standards-Version: 3.7.2
 
 Package: brltty
