=== modified file 'debian/apparmor.postrm'
--- debian/apparmor.postrm	2014-03-19 20:29:27 +0000
+++ debian/apparmor.postrm	2014-11-10 16:50:43 +0000
@@ -30,6 +30,11 @@
         if [ -e "/etc/apparmor.d/tunables/xdg-user-dirs.d/site.local" ]; then
             rm -f /etc/apparmor.d/tunables/xdg-user-dirs.d/site.local
         fi
+        if [ -e "/var/lib/apparmor/profiles/.apparmor.md5sums" ]; then
+            rm -f "/var/lib/apparmor/profiles/.apparmor.md5sums"
+        fi
+	rmdir --ignore-fail-on-non-empty /var/lib/apparmor/profiles
+	rmdir --ignore-fail-on-non-empty /var/lib/apparmor
         ;;
     abort-install|abort-upgrade|remove|upgrade|failed-upgrade|disappear)
         ;;

