--- a/scheduler/job.c
+++ b/scheduler/job.c
@@ -705,8 +705,9 @@
   * Add decompression/raw filter as needed...
   */
 
-  if (job->compressions[job->current_file] &&
-      (!job->printer->remote || job->num_files == 1))
+  if ((!job->printer->raw && job->compressions[job->current_file]) ||
+      (!filters && !job->printer->remote &&
+       (job->num_files > 1 || !strncmp(job->printer->device_uri, "file:", 5))))
   {
    /*
     * Add gziptoany filter to the front of the list...
