--- /usr/share/perl5/Debian/Debhelper/Buildsystem/perl_makemaker.pm.orig	2013-04-10 17:12:06.000000000 +0200
+++ /usr/share/perl5/Debian/Debhelper/Buildsystem/perl_makemaker.pm	2013-04-10 17:10:53.000000000 +0200
@@ -54,7 +54,7 @@
 		push @flags, "LD=$Config{ld} $ENV{CFLAGS} $ENV{LDFLAGS}";
 	}
 
-	$this->doit_in_sourcedir("perl", "Makefile.PL", "INSTALLDIRS=vendor",
+	$this->doit_in_sourcedir("perl", "Makefile.PL", "INSTALLDIRS=vendor", "INSTALL_BASE=",
 		# if perl_build is not tested first, need to pass packlist
 		# option to handle fallthrough case
 		(compat(7) ? "create_packlist=0" : ()),
