--- blobwars-1.05.orig/src/items.cpp	2005-05-03 23:05:41.000000000 +0200
+++ blobwars-1.05/src/items.cpp	2007-03-22 19:00:24.000000000 +0100
@@ -191,7 +200,7 @@
 		item->health = 240;
 		item->dx = 0;
 		item->dy = -2;
-		item->x = game.checkPointX + Math::rrand(-3, 3);
+		item->x = game.checkPointX + Math::rrand(0, 6);
 		item->y = game.checkPointY;
 		item->flags = ENT_INANIMATE + ENT_BOUNCES + ENT_NOCOLLISIONS;
 	}
