diff -Nru chromium-browser-38.0.2125.101.orig/debian/chromium.install chromium-browser-38.0.2125.101/debian/chromium.install
--- chromium-browser-38.0.2125.101.orig/debian/chromium.install	2014-10-13 08:15:44.000000000 +0900
+++ chromium-browser-38.0.2125.101/debian/chromium.install	2014-10-18 17:25:39.767440130 +0900
@@ -15,3 +15,5 @@
 debian/chromium.xml usr/share/gnome-control-center/default-apps
 
 debian/chromium.desktop usr/share/applications
+
+debian/GOOGLE_API_KEY etc/chromium.d
diff -Nru chromium-browser-38.0.2125.101.orig/debian/GOOGLE_API_KEY chromium-browser-38.0.2125.101/debian/GOOGLE_API_KEY
--- chromium-browser-38.0.2125.101.orig/debian/GOOGLE_API_KEY	1970-01-01 09:00:00.000000000 +0900
+++ chromium-browser-38.0.2125.101/debian/GOOGLE_API_KEY	2014-10-18 17:23:24.559444031 +0900
@@ -0,0 +1,5 @@
+# Debian's Google API KEY
+export GOOGLE_API_KEY="AIzaSyCkfPOPZXDKNn8hhgu3JrA62wIgC93d44k"
+export GOOGLE_DEFAULT_CLIENT_ID="811574891467.apps.googleusercontent.com"
+export GOOGLE_DEFAULT_CLIENT_SECRET="kdloedMFGdGla2P1zacGjAQh"
+
