--- a/debian/rules
+++ b/debian/rules
@@ -42,6 +42,11 @@
          linux_link_libspeechd=1 \
          linux_link_gnome_keyring=1 \
 
+# Debian Google API Key
+defines+=google_api_key=AIzaSyCkfPOPZXDKNn8hhgu3JrA62wIgC93d44k \
+         google_default_client_id=811574891467.apps.googleusercontent.com \
+         google_default_client_secret=kdloedMFGdGla2P1zacGjAQh \
+
 # system libraries to use
 defines+=use_system_re2=1 \
          use_system_yasm=1 \
@@ -58,7 +63,7 @@
          use_system_libsrtp=1 \
          use_system_jsoncpp=1 \
          use_system_libevent=1 \
-         use_system_protobuf=1 \
+         use_system_protobuf=0 \
          use_system_harfbuzz=1 \
          use_system_xdg_utils=1 \
 
