From 81f104cdfbf196e1420d17de553e70f81b54aef8 Mon Sep 17 00:00:00 2001
From: Modestas Vainius <modestas@vainius.eu>
Date: Sat, 21 Nov 2009 17:16:45 +0200
Subject: [PATCH] Fix deep recursive in cmake::test().

---
 Debian/Debhelper/Buildsystem/cmake.pm |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Debian/Debhelper/Buildsystem/cmake.pm b/Debian/Debhelper/Buildsystem/cmake.pm
index 24f486f..d4a98f4 100644
--- a/Debian/Debhelper/Buildsystem/cmake.pm
+++ b/Debian/Debhelper/Buildsystem/cmake.pm
@@ -51,7 +51,7 @@ sub test {
 	my $this=shift;
 
 	$ENV{CTEST_OUTPUT_ON_FAILURE} = 1;
-	return $this->test(@_);
+	return $this->SUPER::test(@_);
 }
 
 1
-- 
1.6.5.3

