diff -uNrp debmirror-2.8~bpo60+1.orig/debmirror debmirror-2.8~bpo60+1/debmirror
--- debmirror-2.8~bpo60+1.orig/debmirror	2011-03-21 18:03:12.000000000 +0100
+++ debmirror-2.8~bpo60+1/debmirror	2011-07-28 13:46:36.059369695 +0200
@@ -572,6 +572,7 @@ our $allow_dist_rename=0;
 our $debmarshal=0;
 our $slow_cpu=0;
 our $check_gpg=1;
+our $new_mirror=0;
 my @errlog;
 my $HOME;
 ($HOME = $ENV{'HOME'}) or die "HOME not defined in environment!\n";
@@ -772,7 +773,10 @@ say("Debmarshal snapshots kept.") if $de
 
 # Set up mirror directory and resolve $mirrordir to a full path for
 # locking and rsync
-make_dir($mirrordir) if (! -d $mirrordir);
+if (! -d $mirrordir) {
+  make_dir($mirrordir);
+  $new_mirror = 1;
+}
 die "You need write permissions on $mirrordir" if (! -w $mirrordir);
 chdir($mirrordir) or die "chdir $mirrordir: $!";
 $mirrordir = cwd();
@@ -1187,7 +1191,7 @@ SOURCE:  foreach my $file (@source_files
   }
 
   # Sanity check to avoid completely nuking a mirror.
-  if ($empty_mirror) {
+  if ($empty_mirror && ! $new_mirror) {
     print "Errors:\n ".join(" ",@errlog) if (@errlog);
     die "No packages after parsing Packages and Sources files!\n";
   }
