From 78e134ede7646bacfaaafb71172fd4f86b890d0f Mon Sep 17 00:00:00 2001
From: Robert Bradshaw <robertwb@math.washington.edu>
Date: Thu, 3 Mar 2011 11:07:23 -0800
Subject: [PATCH] Fix compile time division.

---
 Cython/Compiler/ExprNodes.py |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Cython/Compiler/ExprNodes.py b/Cython/Compiler/ExprNodes.py
index c965251..1fe8538 100755
--- a/Cython/Compiler/ExprNodes.py
+++ b/Cython/Compiler/ExprNodes.py
@@ -5992,7 +5992,7 @@ class DivNode(NumBinopNode):
         operand2 = self.operand2.compile_time_value(denv)
         try:
             func = self.find_compile_time_binary_operator(
-                self, operand1, operand2)
+                operand1, operand2)
             return func(operand1, operand2)
         except Exception, e:
             self.compile_time_value_error(e)
-- 
1.7.7.rc1

