--- pkginfo.cc	2006-01-18 01:30:03.000000000 -0700
+++ pkginfo-bms.cc	2006-10-05 14:27:22.000000000 -0600
@@ -129,10 +129,9 @@
 void packagelist::itd_statuscontrol() {
   werase(infopad);
   if (!table[cursorline]->pkg->name) {
-    severalinfoblurb(_("currently installed control info"));
+    severalinfoblurb(_("control file information for Installed package"));
   } else {
-    whatinfovb(_("installed control info for "));
-    whatinfovb(table[cursorline]->pkg->name);
+    whatinfovb(_("Installed control file information"));
     varbuf vb;
     varbufrecord(&vb,table[cursorline]->pkg,&table[cursorline]->pkg->installed);
     vb.terminate();
@@ -145,10 +144,9 @@
 void packagelist::itd_availablecontrol() {
   werase(infopad);
   if (!table[cursorline]->pkg->name) {
-    severalinfoblurb(_("available version of control file info"));
+    severalinfoblurb(_("control file information for Available package"));
   } else {
-    whatinfovb(_("available version of control info for "));
-    whatinfovb(table[cursorline]->pkg->name);
+    whatinfovb(_("Available control file information"));
     varbuf vb;
     varbufrecord(&vb,table[cursorline]->pkg,&table[cursorline]->pkg->available);
     vb.terminate();
