diff -Nurd binutils-2.21.0.20110327.orig/debian/control binutils-2.21.0.20110327/debian/control
--- binutils-2.21.0.20110327.orig/debian/control	2011-03-28 22:33:16.000000000 +0900
+++ binutils-2.21.0.20110327/debian/control	2011-03-28 22:35:40.000000000 +0900
@@ -48,7 +48,7 @@
  requiring support for reading info from binaries from other architectures.
 
 Package: binutils-gold
-Architecture: amd64 armel armhf i386 powerpc sparc
+Architecture: amd64 armel armhf i386 powerpc ppc64 sparc
 Priority: extra
 Depends: ${shlibs:Depends}, binutils (= ${binary:Version})
 Description: The (experimental) GNU gold linker utility
diff -Nurd binutils-2.21.0.20110327.orig/debian/control.in binutils-2.21.0.20110327/debian/control.in
--- binutils-2.21.0.20110327.orig/debian/control.in	2011-03-28 22:33:16.000000000 +0900
+++ binutils-2.21.0.20110327/debian/control.in	2011-03-28 23:45:56.000000000 +0900
@@ -48,7 +48,7 @@
  requiring support for reading info from binaries from other architectures.
 
 Package: binutils-gold
-Architecture: amd64 armel armhf i386 powerpc sparc
+Architecture: amd64 armel armhf i386 powerpc ppc64 sparc
 Priority: extra
 Depends: ${shlibs:Depends}, binutils (= ${binary:Version})
 Description: The (experimental) GNU gold linker utility
