diff -Naur blobwars-0.92.orig/src/init.cpp blobwars-0.92/src/init.cpp
--- blobwars-0.92.orig/src/init.cpp	2004-09-25 03:04:29.000000000 -0700
+++ blobwars-0.92/src/init.cpp	2004-09-28 17:05:00.000000000 -0700
@@ -230,11 +230,11 @@
 	if (!engine.fullScreen)
 		graphics.screen = SDL_SetVideoMode(640, 480, 0, SDL_HWPALETTE);
 	else
-		graphics.screen = SDL_SetVideoMode(640, 480, 16, SDL_HWPALETTE | SDL_FULLSCREEN);
+		graphics.screen = SDL_SetVideoMode(640, 480, 0, SDL_HWPALETTE | SDL_FULLSCREEN);
 
 	if (graphics.screen == NULL)
 	{
-		printf("Couldn't set 640x480x16 video mode: %s\n", SDL_GetError());
+		printf("Couldn't set 640x480 video mode: %s\n", SDL_GetError());
 		exit(1);
 	}
 
