diff --exclude=.svn --exclude=CVS --exclude='~' -urN aplus-fsf-4.20.2.orig/debian/rules aplus-fsf-4.20.2/debian/rules
--- aplus-fsf-4.20.2.orig/debian/rules	2008-04-05 20:36:17.000000000 -0600
+++ aplus-fsf-4.20.2/debian/rules	2008-04-05 20:50:19.000000000 -0600
@@ -112,7 +112,7 @@
 xfonts-kapl-stamp:
 	dh_testdir
 	dh_testroot
-	rm -fr debian/xfonts-kapl{/,.*.debhelper}
+	rm -fr debian/xfonts-kapl/ debian/xfonts-kapl.*.debhelper
 	dh_installdirs
 # Alias file must be <package>.alias; dh_install cannot rename
 	cd $(DESTDIR)/usr/fonts/X11/pcf && cp -p Kapl.alias xfonts-kapl.alias
@@ -132,7 +132,7 @@
 aplus-fsf-el-stamp:
 	dh_testdir
 	dh_testroot
-	rm -fr debian/aplus-fsf-el{/,.*.debhelper}
+	rm -fr debian/aplus-fsf-el/ debian/aplus-fsf-el.*.debhelper
 	dh_installdirs
 	dh_install
 	dh_installdocs
@@ -165,7 +165,7 @@
 aplus-fsf-stamp:
 	dh_testdir
 	dh_testroot
-	rm -fr debian/aplus-fsf{/,.*.debhelper}
+	rm -fr debian/aplus-fsf/ debian/aplus-fsf.*.debhelper
 	dh_installdirs
 	dh_install
 	cp -p debian/aplus-fsf.lintian debian/aplus-fsf/usr/share/lintian/overrides/aplus-fsf
@@ -189,7 +189,7 @@
 aplus-fsf-dev-stamp:
 	dh_testdir
 	dh_testroot
-	rm -fr debian/aplus-fsf-dev{/,.*.debhelper}
+	rm -fr debian/aplus-fsf-dev/ debian/aplus-fsf-dev.*.debhelper
 	dh_installdirs
 	dh_install
 	dh_installdocs
