diff -ru debhelper.orig//dh_python debhelper/dh_python
--- debhelper.orig//dh_python	2011-12-06 07:54:07.000000000 +0100
+++ debhelper/dh_python	2011-12-06 08:49:04.000000000 +0100
@@ -44,8 +44,8 @@
 
 If your package installs Python modules in non-standard directories, you
 can make F<dh_python> check those directories by passing their names on the
-command line. By default, it will check F</usr/lib/site-python,
-/usr/lib/$PACKAGE>, F</usr/share/$PACKAGE>, F</usr/lib/games/$PACKAGE>,
+command line. By default, it will check F</usr/lib/site-python>,
+F</usr/lib/$PACKAGE>, F</usr/share/$PACKAGE>, F</usr/lib/games/$PACKAGE>,
 F</usr/share/games/$PACKAGE> and F</usr/lib/python?.?/site-packages>.
 
 Note: only F</usr/lib/site-python>, F</usr/lib/python?.?/site-packages> and the
