diff -ru debhelper.orig//debhelper.pod debhelper/debhelper.pod
--- debhelper.orig//debhelper.pod	2011-12-06 07:54:06.000000000 +0100
+++ debhelper/debhelper.pod	2011-12-06 08:42:56.000000000 +0100
@@ -220,7 +220,7 @@
 If this option is not specified, building will be done in source by default
 unless the build system requires or prefers out of source tree building.
 In such a case, the default build directory will be used even if
-B<--builddirectory> is not specified.
+B<--builddirectory> is specified.
 
 If the build system prefers out of source tree building but still
 allows in source building, the latter can be re-enabled by passing a build
@@ -419,7 +419,7 @@
 
 =item -
 
-B<dh_clean> will delete toplevel F<*-stamp> files.
+B<dh_clean> will delete top-level F<*-stamp> files.
 
 =item -
 
diff -ru debhelper.orig//dh_installinit debhelper/dh_installinit
--- debhelper.orig//dh_installinit	2011-12-06 07:54:07.000000000 +0100
+++ debhelper/dh_installinit	2011-12-06 08:44:37.000000000 +0100
@@ -21,7 +21,7 @@
 for non-upstart systems.
 
 It also automatically generates the F<postinst> and F<postrm> and F<prerm> commands
-needed to set up the symlinks in F</etc/rc*.d/> and to start and stop the init
+needed to set up the symlinks in F</etc/rc*.d/> to start and stop the init
 scripts.
 
 =head1 FILES
diff -ru debhelper.orig//dh_installman debhelper/dh_installman
--- debhelper.orig//dh_installman	2011-12-06 07:54:06.000000000 +0100
+++ debhelper/dh_installman	2011-12-06 08:46:58.000000000 +0100
@@ -40,7 +40,7 @@
 
 After the man page installation step, B<dh_installman> will check to see if
 any of the man pages in the temporary directories of any of the packages it
-is acting on contain F<.so> links. If so, it changes them to symlinks.
+is acting on contains F<.so> links. If so, it changes them to symlinks.
 
 Also, B<dh_installman> will use man to guess the character encoding of each
 manual page and convert it to UTF-8. If the guesswork fails for some
