diff -ur bsdmainutils-6.1.6/usr.bin/ncal/ncal.c bsdmainutils/usr.bin/ncal/ncal.c
--- bsdmainutils-6.1.6/usr.bin/ncal/ncal.c	2006-08-15 08:03:30.000000000 +0200
+++ bsdmainutils/usr.bin/ncal/ncal.c	2007-03-15 20:40:33.000000000 +0100
@@ -875,10 +875,9 @@
 		col = wcswidth(wcs, j + 1);
 		len = wcstombs(NULL, wcs, 0);
 		wds->names[i] = calloc(len + 2 - col + 1, sizeof(char));
-		wcstombs(wds->names[i], wcs, len);
+		wds->names[i][0] = ' ';
+		wcstombs(wds->names[i] + (col < 2), wcs, len);
 		free(wcs);
-		if (col < 2)
-			strcat(wds->names[i], " ");
 	}
 }
 
