diff -ur ace-of-penguins-1.3/lib/make-imglib.c ace-of-penguins-1.3-clang/lib/make-imglib.c
--- ace-of-penguins-1.3/lib/make-imglib.c	2014-03-14 14:09:28.000000000 +0100
+++ ace-of-penguins-1.3-clang/lib/make-imglib.c	2014-03-10 12:39:16.457216105 +0100
@@ -202,7 +202,7 @@
   char *rv;
   if (string) {
     next = string;
-    return;
+    return 0;
   }
   while (*next && !isgraph(*next)) next++;
   if (!*next) return 0;
