diff --git a/dh_make b/dh_make
index 04ea6d8..9c635ba 100755
--- a/dh_make
+++ b/dh_make
@@ -258,7 +258,7 @@ class License(Enum):
         matches = {'apache': cls.apache, 'artistic': cls.artistic,
                    'bsd': cls.bsd, 'gpl': cls.gpl2, 'gpl2': cls.gpl2,
                    'gpl3': cls.gpl3, 'lgpl': cls.lgpl2, 'lgpl2': cls.lgpl2,
-                   'lgpl3': cls.lgpl3}
+                   'lgpl3': cls.lgpl3, 'mit': cls.mit}
         if args.copyright == 'custom':
             if args.copyrightfile is None:
                 sys.stderr.write(
