--- debian/debfoster.bash_completion.orig	2015-09-28 11:19:22.000000000 +0200
+++ debian/debfoster.bash_completion	2015-09-28 11:23:01.323306735 +0200
@@ -23,7 +23,7 @@
             return 0
             ;;
         -@(d|-show-depends|e|-show-dependents|r|-show-related))
-            COMPREPLY=( $( _comp_dpkg_installed_packages $cur ) )
+            COMPREPLY=( $( _xfunc dpkg _comp_dpkg_installed_packages $cur ) )
             return 0
             ;;
     esac
@@ -33,7 +33,7 @@
     else
         # This is just an approximation.  Actually, debfoster can
         # install new packages, which won't appear in that list.
-        COMPREPLY=( $( _comp_dpkg_installed_packages $cur ) )
+        COMPREPLY=( $( _xfunc dpkg _comp_dpkg_installed_packages $cur ) )
     fi
 
     return 0
