From 7b374419038eb03af043cf57a519ba6b0959af99 Mon Sep 17 00:00:00 2001
From: Matt Taggart <taggart@debian.org>
Date: Tue, 9 Jun 2015 14:52:48 -0700
Subject: [PATCH 1/6] drop ia64 and sparc from recent releases

---
 config/di-sources.list | 6 ------
 1 file changed, 6 deletions(-)

diff --git a/config/di-sources.list b/config/di-sources.list
index e1f5ce5..d1b877c 100644
--- a/config/di-sources.list
+++ b/config/di-sources.list
@@ -15,8 +15,6 @@
 #Debian/Stable (( http://www.debian.org/distrib/netinst#netboot ))
 stable	amd64	http://ftp.debian.org/dists/stable/main/installer-amd64/current/images/	netboot/netboot.tar.gz
 stable	i386	http://ftp.debian.org/dists/stable/main/installer-i386/current/images/	netboot/netboot.tar.gz
-stable	ia64	http://ftp.debian.org/dists/stable/main/installer-ia64/current/images/	netboot/netboot.tar.gz
-stable	sparc	http://ftp.debian.org/dists/stable/main/installer-sparc/current/images/	netboot/boot.img
 #Graphical Installer (GTK)
 stable-gtk	amd64	http://ftp.debian.org/dists/stable/main/installer-amd64/current/images/	netboot/gtk/netboot.tar.gz
 stable-gtk	i386	http://ftp.debian.org/dists/stable/main/installer-i386/current/images/	netboot/gtk/netboot.tar.gz
@@ -51,8 +49,6 @@ squeeze-gtk	i386	http://ftp.debian.org/dists/squeeze/main/installer-i386/current
 #Debian/Jessie (not released yet, As of writing this file)
 jessie	amd64	http://ftp.debian.org/dists/jessie/main/installer-amd64/current/images/	netboot/netboot.tar.gz
 jessie	i386	http://ftp.debian.org/dists/jessie/main/installer-i386/current/images/	netboot/netboot.tar.gz
-#jessie	ia64	http://ftp.debian.org/dists/jessie/main/installer-ia64/current/images/	netboot/netboot.tar.gz
-#jessie	sparc	http://ftp.debian.org/dists/jessie/main/installer-sparc/current/images/	netboot/boot.img
 ##Graphical Installer (GTK)
 jessie-gtk	amd64	http://ftp.debian.org/dists/jessie/main/installer-amd64/current/images/	netboot/gtk/netboot.tar.gz
 jessie-gtk	i386	http://ftp.debian.org/dists/jessie/main/installer-i386/current/images/	netboot/gtk/netboot.tar.gz
@@ -62,8 +58,6 @@ jessie-gtk	i386	http://ftp.debian.org/dists/jessie/main/installer-i386/current/i
 #beta1 or beta2 or beta3 or rc1 or rc2
 testing	amd64	http://ftp.debian.org/dists/testing/main/installer-amd64/current/images/	netboot/netboot.tar.gz
 testing	i386	http://ftp.debian.org/dists/testing/main/installer-i386/current/images/	netboot/netboot.tar.gz
-testing	ia64	http://ftp.debian.org/dists/testing/main/installer-ia64/current/images/	netboot/netboot.tar.gz
-testing	sparc	http://ftp.debian.org/dists/testing/main/installer-sparc/current/images/	netboot/boot.img
 #Graphical Installer (GTK)
 testing-gtk	amd64	http://ftp.debian.org/dists/testing/main/installer-amd64/current/images/	netboot/gtk/netboot.tar.gz
 testing-gtk	i386	http://ftp.debian.org/dists/testing/main/installer-i386/current/images/	netboot/gtk/netboot.tar.gz
-- 
2.1.4
