From dc321f4c9f6c28e0203cb51be813096f256a5321 Mon Sep 17 00:00:00 2001
From: Matt Taggart <taggart@debian.org>
Date: Tue, 9 Jun 2015 14:56:03 -0700
Subject: [PATCH 3/6] switch squeeze entry to squeeze-lts, drop sparc/ia64

---
 config/di-sources.list | 8 +++-----
 1 file changed, 3 insertions(+), 5 deletions(-)

diff --git a/config/di-sources.list b/config/di-sources.list
index 8a42a09..8712a37 100644
--- a/config/di-sources.list
+++ b/config/di-sources.list
@@ -44,11 +44,9 @@ wheezy	sparc	http://ftp.debian.org/dists/wheezy/main/installer-sparc/current/ima
 wheezy-gtk	amd64	http://ftp.debian.org/dists/wheezy/main/installer-amd64/current/images/	netboot/gtk/netboot.tar.gz
 wheezy-gtk	i386	http://ftp.debian.org/dists/wheezy/main/installer-i386/current/images/	netboot/gtk/netboot.tar.gz
 
-#Debian/squeeze
-squeeze	amd64	http://ftp.debian.org/dists/squeeze/main/installer-amd64/current/images/	netboot/netboot.tar.gz
-squeeze	i386	http://ftp.debian.org/dists/squeeze/main/installer-i386/current/images/	netboot/netboot.tar.gz
-squeeze	ia64	http://ftp.debian.org/dists/squeeze/main/installer-ia64/current/images/	netboot/netboot.tar.gz
-squeeze	sparc	http://ftp.debian.org/dists/squeeze/main/installer-sparc/current/images/	netboot/boot.img
+#Debian/Squeeze-LTS
+squeeze	amd64	http://ftp.debian.org/dists/squeeze-lts/main/installer-amd64/current/images/	netboot/netboot.tar.gz
+squeeze	i386	http://ftp.debian.org/dists/squeeze-lts/main/installer-i386/current/images/	netboot/netboot.tar.gz
 #Graphical Installer (GTK)
 squeeze-gtk	amd64	http://ftp.debian.org/dists/squeeze/main/installer-amd64/current/images/	netboot/gtk/netboot.tar.gz
 squeeze-gtk	i386	http://ftp.debian.org/dists/squeeze/main/installer-i386/current/images/	netboot/gtk/netboot.tar.gz
-- 
2.1.4
