From 0883a4deebdfc48eeca673c1852bbeccb6ad5fd6 Mon Sep 17 00:00:00 2001
From: Matt Taggart <taggart@debian.org>
Date: Tue, 9 Jun 2015 14:57:00 -0700
Subject: [PATCH 4/6] drop lenny

---
 config/di-sources.list | 16 ----------------
 1 file changed, 16 deletions(-)

diff --git a/config/di-sources.list b/config/di-sources.list
index 8712a37..130595b 100644
--- a/config/di-sources.list
+++ b/config/di-sources.list
@@ -75,22 +75,6 @@ daily-gtk	amd64	http://d-i.debian.org/daily-images/amd64/daily/	netboot/gtk/netb
 daily-gtk	i386	http://d-i.debian.org/daily-images/i386/daily/	netboot/gtk/netboot.tar.gz
 
 
-# ########### Obsolete distribution (unsupported) ########### #
-# Old (discontinued) distribution may work with di-netboot-assistant, or not!
-
-#Debian/Lenny
-#      http://www.debian.org/releases/lenny/
-lenny	alpha	http://archive.debian.org/dists/lenny/main/installer-alpha/current/images/	netboot/boot.img
-lenny	amd64	http://archive.debian.org/dists/lenny/main/installer-amd64/current/images/	netboot/netboot.tar.gz
-lenny	hppa	http://archive.debian.org/dists/lenny/main/installer-hppa/current/images/	netboot/2.6/boot.img
-lenny	i386	http://archive.debian.org/dists/lenny/main/installer-i386/current/images/	netboot/netboot.tar.gz
-lenny	ia64	http://archive.debian.org/dists/lenny/main/installer-ia64/current/images/	netboot/netboot.tar.gz
-lenny	sparc	http://archive.debian.org/dists/lenny/main/installer-sparc/current/images/	netboot/boot.img
-#Graphical Installer (GTK)
-lenny-gtk	amd64	http://archive.debian.org/dists/lenny/main/installer-amd64/current/images/	netboot/gtk/netboot.tar.gz
-lenny-gtk	i386	http://archive.debian.org/dists/lenny/main/installer-i386/current/images/	netboot/gtk/netboot.tar.gz
-
-
 #Â ############ Some Debian Derivatives ################ #
 
 ## Ubuntu LTS Releases
-- 
2.1.4
