From 5c47b6bb156a3ad4fcbb0fe684ba29ebb611221e Mon Sep 17 00:00:00 2001
From: Matt Taggart <taggart@debian.org>
Date: Tue, 9 Jun 2015 15:13:10 -0700
Subject: [PATCH 5/6] update daily section to reflect current reality

---
 config/di-sources.list | 13 +++++++++----
 1 file changed, 9 insertions(+), 4 deletions(-)

diff --git a/config/di-sources.list b/config/di-sources.list
index 130595b..a275e97 100644
--- a/config/di-sources.list
+++ b/config/di-sources.list
@@ -63,12 +63,17 @@ testing-gtk	i386	http://ftp.debian.org/dists/testing/main/installer-i386/current
 
 #Daily netboot DI images. Read :
 #     http://d-i.debian.org/daily-images/build-logs.html
-#     http://wiki.debian.org/DebianInstaller/Today
-daily	alpha	http://d-i.debian.org/daily-images/alpha/daily/	netboot/boot.img
 daily	amd64	http://d-i.debian.org/daily-images/amd64/daily/	netboot/netboot.tar.gz
-daily	hppa	http://d-i.debian.org/daily-images/hppa/daily/	netboot/2.6/boot.img
+daily	arm64	http://d-i.debian.org/daily-images/arm64/daily/	netboot/netboot.tar.gz
+#armel has different variants, no default
+daily	armhf	http://d-i.debian.org/daily-images/armhf/daily/	netboot/netboot.tar.gz
+daily	hurd-i386	http://d-i.debian.org/daily-images/hurd-i386/daily/	netboot/netboot.tar.gz
 daily	i386	http://d-i.debian.org/daily-images/i386/daily/	netboot/netboot.tar.gz
-daily	ia64	http://d-i.debian.org/daily-images/ia64/daily/	netboot/netboot.tar.gz
+#kfreebsd-amd64/i386 path is non-standard, no default
+#mips/mipsel have different variants, no default
+#powerpc path is non-standard, no default
+daily	ppc64el	http://d-i.debian.org/daily-images/ppc64el/daily/	netboot/netboot.tar.gz
+#s390 has no netboot
 daily	sparc	http://d-i.debian.org/daily-images/sparc/daily/	netboot/boot.img
 #Graphical Installer (GTK)
 daily-gtk	amd64	http://d-i.debian.org/daily-images/amd64/daily/	netboot/gtk/netboot.tar.gz
-- 
2.1.4
