From a96d2d793bd28c4d6ee26b43631a81fbd486e104 Mon Sep 17 00:00:00 2001
From: Matt Taggart <taggart@debian.org>
Date: Tue, 9 Jun 2015 15:34:40 -0700
Subject: [PATCH 6/6] update ubuntu releases based on feedback from slangasek,
 doko

---
 config/di-sources.list | 30 +++++++++++-------------------
 1 file changed, 11 insertions(+), 19 deletions(-)

diff --git a/config/di-sources.list b/config/di-sources.list
index a275e97..5616ece 100644
--- a/config/di-sources.list
+++ b/config/di-sources.list
@@ -83,33 +83,25 @@ daily-gtk	i386	http://d-i.debian.org/daily-images/i386/daily/	netboot/gtk/netboo
 #Â ############ Some Debian Derivatives ################ #
 
 ## Ubuntu LTS Releases
-# Hardy Heron 8.04
-hardy	i386	http://archive.ubuntu.com/ubuntu/dists/hardy/main/installer-i386/current/images/	netboot/netboot.tar.gz
-hardy	amd64	http://archive.ubuntu.com/ubuntu/dists/hardy/main/installer-amd64/current/images/	netboot/netboot.tar.gz
-
-# Lucid Lynx 10.04
-lucid	i386	http://archive.ubuntu.com/ubuntu/dists/lucid/main/installer-i386/current/images/	netboot/netboot.tar.gz
-lucid	amd64	http://archive.ubuntu.com/ubuntu/dists/lucid/main/installer-amd64/current/images/	netboot/netboot.tar.gz
+# Trusty Tahr 14.04
+trusty	i386	http://archive.ubuntu.com/ubuntu/dists/trusty/main/installer-i386/current/images/	netboot/netboot.tar.gz
+trusty	amd64	http://archive.ubuntu.com/ubuntu/dists/trusty/main/installer-amd64/current/images/	netboot/netboot.tar.gz
 
 # Precise Pangolin 12.04
 precise	i386	http://archive.ubuntu.com/ubuntu/dists/precise/main/installer-i386/current/images/	netboot/netboot.tar.gz
 precise	amd64	http://archive.ubuntu.com/ubuntu/dists/precise/main/installer-amd64/current/images/	netboot/netboot.tar.gz
 
 ## Ubuntu Normal Releases
+# Vivid Vervet 15.04 
+vivid	i386	http://archive.ubuntu.com/ubuntu/dists/vivid/main/installer-i386/current/images/	netboot/netboot.tar.gz
+vivid	amd64	http://archive.ubuntu.com/ubuntu/dists/vivid/main/installer-amd64/current/images/	netboot/netboot.tar.gz
+
+# Utopic Unicorn 14.10
+utopic	i386	http://archive.ubuntu.com/ubuntu/dists/utopic/main/installer-i386/current/images/	netboot/netboot.tar.gz
+utopic	amd64	http://archive.ubuntu.com/ubuntu/dists/utopic/main/installer-amd64/current/images/	netboot/netboot.tar.gz
+
 # Saucy Salamander 13.10
 saucy	i386	http://archive.ubuntu.com/ubuntu/dists/saucy/main/installer-i386/current/images/	netboot/netboot.tar.gz
 saucy	amd64	http://archive.ubuntu.com/ubuntu/dists/saucy/main/installer-amd64/current/images/	netboot/netboot.tar.gz
 
-# Raring Ringtail 13.04
-raring	i386	http://archive.ubuntu.com/ubuntu/dists/raring/main/installer-i386/current/images/	netboot/netboot.tar.gz
-raring	amd64	http://archive.ubuntu.com/ubuntu/dists/raring/main/installer-amd64/current/images/	netboot/netboot.tar.gz
-
-# Quantal Quetzal 12.10
-quantal	i386	http://archive.ubuntu.com/ubuntu/dists/quantal/main/installer-i386/current/images/	netboot/netboot.tar.gz
-quantal	amd64	http://archive.ubuntu.com/ubuntu/dists/quantal/main/installer-amd64/current/images/	netboot/netboot.tar.gz
-
-# Oneiric Ocelot 11.10
-oneiric	i386	http://archive.ubuntu.com/ubuntu/dists/oneiric/main/installer-i386/current/images/	netboot/netboot.tar.gz
-oneiric	amd64	http://archive.ubuntu.com/ubuntu/dists/oneiric/main/installer-amd64/current/images/	netboot/netboot.tar.gz
-
 # vim: ft=disources
-- 
2.1.4
