--- dh_md5sums	2010-03-13 10:42:42.000000000 +0100
+++ dh_md5sums.orig	2010-03-13 10:49:26.000000000 +0100
@@ -75,7 +75,7 @@
 		$exclude.="! \\( $dh{EXCLUDE_FIND} \\) ";
 	}
 	
-	complex_doit("(cd $tmp >/dev/null ; find . -type f $exclude ! -regex '.*/DEBIAN/.*' -printf '%P\\0' | LC_ALL=C sort -z | xargs -r0 md5sum > DEBIAN/md5sums) >/dev/null");
+	complex_doit("(cd $tmp >/dev/null ; find . -type f $exclude ! -regex '.*/DEBIAN/.*' -printf '%P\\0' | xargs -r0 md5sum > DEBIAN/md5sums) >/dev/null");
 	# If the file's empty, no reason to waste inodes on it.
 	if (-z "$tmp/DEBIAN/md5sums") {
 		doit("rm","-f","$tmp/DEBIAN/md5sums");
