--- empathy.orig/debian/changelog	2009-08-03 21:30:00.000000000 +0200
+++ empathy-2.26.2/debian/changelog	2009-08-03 21:01:35.000000000 +0200
@@ -1,3 +1,10 @@
+empathy (2.26.2-1.1) UNRELEASED; urgency=low
+
+  * Non-maintainer upload.
+  * Adding empathy-dbg package.
+
+ -- Jan Beyer <jan@beathovn.de>  Mon, 03 Aug 2009 21:01:09 +0200
+
 empathy (2.26.2-1) unstable; urgency=low
 
   [ Jonny Lamb ]
--- empathy.orig/debian/control	2009-08-03 21:30:00.000000000 +0200
+++ empathy-2.26.2/debian/control	2009-08-03 20:59:54.000000000 +0200
@@ -57,6 +57,21 @@
  .
  This packet contains the empathy IM application and account manager.
 
+Package: empathy-dbg
+Architecture: any
+Section: libdevel
+Priority: extra
+Depends: empathy (= ${binary:Version})
+Description: High-level library and user-interface for Telepathy - debug symbols
+ Empathy consists of a rich set of reusable instant messaging widgets, and a
+ GNOME client using those widgets. It uses Telepathy and Nokia's Mission
+ Control, and reuses Gossip's UI. The main goal is to permit desktop
+ integration by providing libempathy and libempathy-gtk libraries.
+ libempathy-gtk is a set of powerful widgets that can be embeded into any
+ GNOME application.
+ .
+ This package contains the debugging symbols for empathy.
+
 Package: libempathy-common
 Architecture: all
 Section: misc
--- empathy.orig/debian/rules	2009-08-03 21:30:00.000000000 +0200
+++ empathy-2.26.2/debian/rules	2009-08-03 21:00:42.000000000 +0200
@@ -1,6 +1,7 @@
 #!/usr/bin/make -f
 
 DEB_DH_INSTALL_SOURCEDIR=debian/tmp
+DEB_DH_STRIP_ARGS := --dbg-package=empathy-dbg
 
 include /usr/share/cdbs/1/rules/debhelper.mk
 include /usr/share/cdbs/1/rules/simple-patchsys.mk
