diff -Naur bsdmainutils/usr.bin/calendar/calendars/ru_RU.KOI8-R/calendar.pagan bsdmainutils-minor-fix/usr.bin/calendar/calendars/ru_RU.KOI8-R/calendar.pagan
--- bsdmainutils/usr.bin/calendar/calendars/ru_RU.KOI8-R/calendar.pagan	2018-01-30 13:07:30.000000000 +0300
+++ bsdmainutils-minor-fix/usr.bin/calendar/calendars/ru_RU.KOI8-R/calendar.pagan	2020-07-27 01:35:59.344754522 +0300
@@ -33,7 +33,7 @@
 21 Á×Ç.	äÅÎØ óÔÒÉÂÏÇÁ
 28 Á×Ç.	õÓÐÅÎÉÅ úÌÁÔÏÇÏÒËÉ
 14 ÓÅÎÔ.	äÅÎØ ÷ÏÌÈÁ úÍÅÅ×ÉÞÁ
-22 ÓÅÎÔ.*	ðÏ×ÏÒÏÔ Ë ÚÉÍÅ (ÏÓÅÎÎÅÅ ÒÁ×ÎÏÄÅÎÓÔ×ÉÅ)
+22 ÓÅÎÔ.*	ïÓÅÎÎÅÅ ÒÁ×ÎÏÄÅÎÓÔ×ÉÅ
 10 ÎÏÑÂ.	äÅÎØ íÁËÏÛÉ
 21 ÎÏÑÂ.	äÅÎØ ó×ÁÒÏÇÁ É óÅÍÁÒÇÌÁ
  9 ÄÅË.	äÅÎØ äÁÖØÂÏÇÁ É íÁÒÅÎÙ
diff -Naur bsdmainutils/usr.bin/calendar/calendars/ru_RU.UTF-8/calendar.pagan bsdmainutils-minor-fix/usr.bin/calendar/calendars/ru_RU.UTF-8/calendar.pagan
--- bsdmainutils/usr.bin/calendar/calendars/ru_RU.UTF-8/calendar.pagan	2020-07-23 15:53:53.000000000 +0300
+++ bsdmainutils-minor-fix/usr.bin/calendar/calendars/ru_RU.UTF-8/calendar.pagan	2020-07-27 01:34:34.593052438 +0300
@@ -33,7 +33,7 @@
 21 Ð°Ð²Ð³.	ÐÐµÐ½Ñ Ð¡ÑÑÐ¸Ð±Ð¾Ð³Ð°
 28 Ð°Ð²Ð³.	Ð£ÑÐ¿ÐµÐ½Ð¸Ðµ ÐÐ»Ð°ÑÐ¾Ð³Ð¾ÑÐºÐ¸
 14 ÑÐµÐ½Ñ.	ÐÐµÐ½Ñ ÐÐ¾Ð»ÑÐ° ÐÐ¼ÐµÐµÐ²Ð¸ÑÐ°
-22 ÑÐµÐ½Ñ.*	ÐÐ¾Ð²Ð¾ÑÐ¾Ñ Ðº Ð·Ð¸Ð¼Ðµ (Ð¾ÑÐµÐ½Ð½ÐµÐµ ÑÐ°Ð²Ð½Ð¾Ð´ÐµÐ½ÑÑÐ²Ð¸Ðµ)
+22 ÑÐµÐ½Ñ.*	ÐÑÐµÐ½Ð½ÐµÐµ ÑÐ°Ð²Ð½Ð¾Ð´ÐµÐ½ÑÑÐ²Ð¸Ðµ
 10 Ð½Ð¾ÑÐ±.	ÐÐµÐ½Ñ ÐÐ°ÐºÐ¾ÑÐ¸
 21 Ð½Ð¾ÑÐ±.	ÐÐµÐ½Ñ Ð¡Ð²Ð°ÑÐ¾Ð³Ð° Ð¸ Ð¡ÐµÐ¼Ð°ÑÐ³Ð»Ð°
  9 Ð´ÐµÐº.	ÐÐµÐ½Ñ ÐÐ°Ð¶ÑÐ±Ð¾Ð³Ð° Ð¸ ÐÐ°ÑÐµÐ½Ñ
