diff --git a/scripts/debcheckout.pl b/scripts/debcheckout.pl
index 3477f8a..6b41278 100755
--- a/scripts/debcheckout.pl
+++ b/scripts/debcheckout.pl
@@ -100,8 +100,10 @@ Print a detailed help message and exit.
 =item B<-p>, B<--print>
 
 Only print a summary about package repository information, without
-checking it out; the output format is TAB-separated with two fields:
-repository type, repository URL. This action works offline, it only
+checking it out; the output format is TAB-separated with three fields:
+repository type, repository checkout URL, repository Web view URL.
+More TAB-separated fields may be added in the future.
+This action works offline, it only
 uses "static" information as known by APT's cache.
 
 Also see B<-d>. This option and B<-d> are mutually exclusive.
@@ -907,10 +909,11 @@ sub unpack_source($$$$$) {
 }
 
 # Print information about a repository and quit.
-sub print_repo($$) {
-    my ($repo_type, $repo_url) = @_;
+sub print_repo($$$) {
+    my ($repo_type, $repo_url, $browse_url) = @_;
 
-    print "$repo_type\t$repo_url\n";
+    $browse_url //= "";
+    print "$repo_type\t$repo_url\t$browse_url\n";
     exit(0);
 }
 
@@ -950,8 +953,8 @@ sub tg_info($) {
 }
 
 # Print details about a repository and quit.
-sub print_details($$) {
-    my ($repo_type, $repo_url) = @_;
+sub print_details($$$) {
+    my ($repo_type, $repo_url, $browse_url) = @_;
 
     print "type\t$repo_type\n";
     print "url\t$repo_url\n";
@@ -961,6 +964,7 @@ sub print_details($$) {
 	    print "$k\t$v\n";
 	}
     }
+    print "browse\t$browse_url\n" if $browse_url;
     exit(0);
 }
 
@@ -1076,14 +1080,15 @@ and it will not be possible to commit them directly.
 EOF
             exit(1);
 	}
-	$browse_url = find_browse($pkg, $version) if @files;
+	$browse_url = find_browse($pkg, $version)
+	    if @files or $print_mode or $details_mode;
     }
 
     $repo_url = munge_url($repo_type, $repo_url);
     $repo_url = set_auth($repo_type, $repo_url, $user, $dont_act)
 	if $auth and not @files;
-    print_repo($repo_type, $repo_url) if $print_mode;		# ... then quit
-    print_details($repo_type, $repo_url) if $details_mode;	# ... then quit
+    print_repo($repo_type, $repo_url, $browse_url) if $print_mode;	# ... then quit
+    print_details($repo_type, $repo_url, $browse_url) if $details_mode;	# ... then quit
     if (length $pkg) {
 	print "declared $repo_type repository at $repo_url\n";
 	$destdir = $pkg unless length $destdir;
