--- checkrestart.orig	2012-04-09 21:02:09.000000000 +0200
+++ checkrestart	2012-04-09 21:03:27.000000000 +0200
@@ -301,8 +301,12 @@
     dpkgProc.wait()
     if verbose:
         print "Running:%s" % dpkgQuery
+        if file_regexp:
+            print "Will look in lines using regular expression '%s'" % f
     for line in dpkgProc.stdout.readlines():
         line = line.strip()
+        if verbose:
+            print "Analysing output: %s" % line
         if line.find('no path found matching pattern ' + f) > 0:
             file_in_package = False
             break
