=== modified file 'apt-pkg/contrib/strutl.cc'
--- apt-pkg/contrib/strutl.cc	2008-10-03 18:57:25 +0000
+++ apt-pkg/contrib/strutl.cc	2008-11-08 16:25:33 +0000
@@ -696,9 +696,11 @@
       End += Res;
       
       // Look for the end of the message
-      for (char *I = Buffer; I + 1 < End; I++)
+      for (char *I = Buffer; I < End; I++)
       {
-	 if (I[0] != '\n' || I[1] != '\n')
+	 if (I == Buffer && I[0] == '\n' && PartialMessage[PartialMessage.length()-1] )
+	    PartialMessage = PartialMessage.substr(0, PartialMessage.length()-1);
+	 else if (I[0] != '\n' || I == End || I[1] != '\n')
 	    continue;
 	 
 	 // Pull the message out

