diff --git a/debdelta/debdelta b/debdelta/debdelta
index 7618e10..627d2c0 100755
--- a/debdelta/debdelta
+++ b/debdelta/debdelta
@@ -4300,7 +4300,7 @@ def delta_upgrade_(args):
           return e, None, None, None
     else: #use urllib2
       try:
-        req = urllib2.Request(uri, headers=headers)
+        req = urllib2.Request('http://'+urllib2.quote(uri[7:]), headers=headers)
         r = urllib2.urlopen(req)
         #print r.info(),dir(r),r.code
         return r, getattr(r,'code',None), getattr(r,'msg','(no message)'), r.info()
