diff -ru cryptsetup-1.0.4+svn26.orig/lib/setup.c cryptsetup-1.0.4+svn26/lib/setup.c
--- cryptsetup-1.0.4+svn26.orig/lib/setup.c	2009-01-30 17:47:37.000000000 +0100
+++ cryptsetup-1.0.4+svn26/lib/setup.c	2009-01-30 17:46:29.000000000 +0100
@@ -756,7 +756,7 @@
 			r = -EINVAL; goto out;
 		}
 		openedIndex = LUKS_open_any_key(device, password, passwordLen, &hdr, &mk, backend);
-		if(openedIndex < 0 || keyIndex == openedIndex) {
+		if(openedIndex < 0) {
 			printf("No remaining key available with this passphrase.\n");
 			r = -EPERM; goto out;
 		}
