--- debian/rules~	2013-12-31 17:24:55.000000000 +0100
+++ debian/rules	2014-01-18 01:23:48.390467684 +0100
@@ -2,7 +2,7 @@
 
 #export DH_VERBOSE=1
 
-ARCH=$(shell dpkg-architecture -qDEB_HOST_ARCH)
+ARCH=$(shell dpkg-architecture -qDEB_HOST_ARCH_CPU)
 MULTIARCH=$(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
 MAKE_VERSION=$(shell sed -n '/^MAKE_VERSION=/s,.*=[^0-9]*,,p' Makefile)
 CFLAGS=$(shell dpkg-buildflags --get CFLAGS) \
