diff -u a2ps-4.14/debian/changelog a2ps-4.14/debian/changelog
--- a2ps-4.14/debian/changelog
+++ a2ps-4.14/debian/changelog
@@ -1,3 +1,10 @@
+a2ps (1:4.14-1.2) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Fixed typo in package description. (Closes: #608647)
+
+ -- Benjamin Kerensa <bkerensa@ubuntu.com>  Sun, 22 Jan 2012 03:02:42 -0800
+
 a2ps (1:4.14-1.1) unstable; urgency=low
 
   * Non-maintainer upload.
diff -u a2ps-4.14/debian/control a2ps-4.14/debian/control
--- a2ps-4.14/debian/control
+++ a2ps-4.14/debian/control
@@ -26,7 +26,7 @@
  .
  Among the other most noticeable features of a2ps are:
   - various encodings (all the Latins and others),
-  - various fonts (automatic font down loading),
+  - various fonts (automatic font downloading),
   - various medias,
   - various printer interfaces,
   - various output styles,
