=== modified file 'profiles/apparmor.d/abstractions/nameservice'
--- profiles/apparmor.d/abstractions/nameservice	2017-06-24 18:42:47 +0000
+++ profiles/apparmor.d/abstractions/nameservice	2017-10-05 16:42:29 +0000
@@ -88,6 +88,11 @@
   network inet  dgram,
   network inet6 dgram,
 
+  # unix dgram/stream
+  # TODO: replace with more specific unix rules when support for unix rules arrives in the Kernel (probably in 4.15) and gives us detailed log messages
+  network unix dgram,
+  network unix stream,
+
   # TODO: adjust when support finer-grained netlink rules
   # Netlink raw needed for nscd
   network netlink raw,

