diff --git a/policy.sgml b/policy.sgml
index 404dc73..74f0a3b 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -8508,6 +8508,21 @@ fi
 	  renamed.  If a consensus cannot be reached, <em>both</em>
 	  programs must be renamed.
 	</p>
+
+	<p>
+	  To support merged /usr systems, packages must not install a
+	  file in <file>/usr/bin</file> with the same name as a file in
+	  <file>/bin</file> or a file in <file>/usr/sbin</file> with the
+	  same name as a file in <file>/sbin</file>.
+	  If such a compatibility symlink is needed then it must be
+	  managed in the maintainer scripts in a way that will not break
+	  when e.g. <file>/usr/bin</file> and <file>/bin</file> are the
+	  same directory.
+	  Packages must not install a file in <file>/usr/lib</file> (or
+	  one of its subdirectories) with the same name as a file in
+	  <file>/lib</file> (or the corresponding subdirectory).
+	</p>
+
 	<p>
           Binary executables must not be statically linked with the GNU C
           library, since this prevents the binary from benefiting from
