--- debmirror	2007-01-24 11:42:14.000000000 +0100
+++ /tmp/debmirror	2007-04-20 12:01:38.095482600 +0200
@@ -690,6 +690,7 @@
 foreach my $dist (@dists) {
   foreach my $section (@sections) {
     foreach my $arch (@arches) {
+      next if (($arch=~/amd64/) && ($dist=~/sarge/));
       add_bytes("dists/$dist/$section/binary-$arch/Packages");
       add_bytes("dists/$dist/$section/binary-$arch/Packages.gz");
       add_bytes("dists/$dist/$section/binary-$arch/Packages.bz2");
@@ -724,6 +725,7 @@
       next if $dist=~/experimental/;
       next if $dist=~/.*-proposed-updates/;
       next if $arch=~/source/;
+      next if (($arch=~/amd64/) && ($dist=~/sarge/));
       add_bytes("dists/$dist/Contents-$arch.gz");
     }
   }
@@ -741,7 +743,10 @@
     next if ($section =~ /debian-installer/ && $dist =~ /.*-proposed-updates/);
     next if ($section =~ /debian-installer/ && $dist =~ /.*breezy-updates/ );
     next if ($section =~ /debian-installer/ && $dist eq "breezy-security" );
+    next if ($section =~ /debian-installer/ && $dist eq "sarge");
+    next if ($section =~ /debian-installer/ && $dist eq "etch");
     foreach my $arch (@arches) {
+      next if (($arch=~/amd64/) && ($dist=~/sarge/));
       get_index("dists/$dist/$section/binary-$arch", "Packages");
     }
     get_index("dists/$dist/$section/source", "Sources") if ($do_source);
@@ -778,6 +783,7 @@
       next if $dist=~/experimental/;
       next if $dist=~/.*-proposed-updates/;
       next if $arch=~/source/;
+      next if (($arch=~/amd64/) && ($dist=~/sarge/));
       remote_get("dists/$dist/Contents-$arch.gz");
       $files{"dists/$dist/Contents-$arch.gz"}=1;
       $files{$tempdir."/"."dists/$dist/Contents-$arch.gz"}=1;
