diff -rc devscripts-2.14.6/scripts/grep-excuses.pl devscripts-2.14.6.new/scripts/grep-excuses.pl
*** devscripts-2.14.6/scripts/grep-excuses.pl	2014-08-17 01:29:57.395288804 +0200
--- devscripts-2.14.6.new/scripts/grep-excuses.pl	2014-08-18 23:39:11.928207102 +0200
***************
*** 48,57 ****
  
  my $url='http://ftp-master.debian.org/testing/update_excuses.html.gz';
  
- # No longer use these - see bug#309802
  my $cachedir = $ENV{'HOME'}."/.devscripts_cache/";
  my $cachefile = $cachedir . basename($url);
! unlink $cachefile if -f $cachefile;
  
  sub usage {
      print <<"EOF";
--- 48,56 ----
  
  my $url='http://ftp-master.debian.org/testing/update_excuses.html.gz';
  
  my $cachedir = $ENV{'HOME'}."/.devscripts_cache/";
  my $cachefile = $cachedir . basename($url);
! #unlink $cachefile if -f $cachefile;
  
  sub usage {
      print <<"EOF";
***************
*** 179,186 ****
      die "$progname: this program requires the wget package to be installed\n";
  }
  
! open EXCUSES, "wget -q -O - $url | zcat |" or
!     die "$progname: wget | zcat failed: $!\n";
  
  my $item='';
  my $mainlist=0;
--- 178,195 ----
      die "$progname: this program requires the wget package to be installed\n";
  }
  
! if (-d $cachedir or mkdir $cachedir) {
!     chdir $cachedir;
!     if (system("wget -q -N $url") != 0) {
! 	die "$progname: wget failed: $!\n";
!     }
!     open EXCUSES, "zcat $cachefile | " or
! 	die "$progname: zcat failed: $!\n";
! }
! else {
!     open EXCUSES, "wget -q -O - $url | zcat |" or
! 	die "$progname: wget | zcat failed: $!\n";
! }
  
  my $item='';
  my $mainlist=0;
