--- /usr/bin/nmudiff	2012-11-25 00:51:46.000000000 +0100
+++ nmudiff	2013-02-14 20:00:51.864737374 +0100
@@ -275,8 +275,7 @@
     fi
 fi
 
-OLDVERSION=$( sed -n "s/^[^ .][^ ]* (\(.*\)).*$/\1/p" debian/changelog | \
-  head -2 | tail -1 )
+OLDVERSION=$( dpkg-parsechangelog -o1 -c1 | grep ^Version: | cut -d" " -f2 )
 if [ -z "$OLDVERSION" ]; then
     echo "nmudiff: could not determine previous package version from changelog!" >&2
     exit 1
