From bf1402735235ebc6430914bbaf94c9968a4c09f9 Mon Sep 17 00:00:00 2001
From: Reto Gantenbein <reto.gantenbein@linuxmonk.ch>
Date: Wed, 19 Sep 2012 14:20:40 +0200
Subject: [PATCH] fix rsync cmd quoting (should resolve issue #4019)

---
 handlers/rsync.in |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/handlers/rsync.in b/handlers/rsync.in
index 1954fc1..a5b422c 100644
--- a/handlers/rsync.in
+++ b/handlers/rsync.in
@@ -1138,7 +1138,7 @@ for SECTION in $include; do
   info "Syncing $SECTION on $dest_path..."
   debug $nice $rsync ${rsync_options[@]} $filelist_flag $excludes $batch_option $orig $dest_path
   set_pipefail
-  $nice su -c "$rsync ${rsync_options[@]} --delete-excluded $filelist_flag $excludes $batch_option $orig $dest_path" | tee -a $log
+  $nice su -c "`echo $rsync ${rsync_options[@]} --delete-excluded $filelist_flag $excludes $batch_option $orig $dest_path`" | tee -a $log
 
   check_rsync_exit_status $?
   restore_pipefail
-- 
1.7.8.6

