diff --git a/apt-file.1.sgml b/apt-file.1.sgml
index 4411124..2b4b30f 100644
--- a/apt-file.1.sgml
+++ b/apt-file.1.sgml
@@ -143,7 +143,8 @@
 	  architecture from the one installed on your system.
 	  It determines how the <literal>$ARCH</literal> variable
 	  in sources.list is expanded (but it does not influence
-	  the search in any other way).
+	  the search in any other way). The "source" architecture
+	  is also supported.
 	  </para>
 	</listitem>
       </varlistentry>
