diff --git a/bin/fai-make-nfsroot b/bin/fai-make-nfsroot
index 8aebd6d..6d32b00 100755
--- a/bin/fai-make-nfsroot
+++ b/bin/fai-make-nfsroot
@@ -502,7 +502,7 @@ umount_dirs() {
 
     if [ -n "$FAI_DEBMIRROR" ]; then
         test -d $NFSROOT/$MNTPOINT && umount $NFSROOT/$MNTPOINT || true
-	rmdir $MNTPOINT || true
+        rmdir $NFSROOT/$MNTPOINT || true
     fi
     # show directories still mounted on nfsroot
     mount | grep " on $NFSROOT " || true
