From 4676d9f9d73ddc3945540ed5ea41d86ca01299bf Mon Sep 17 00:00:00 2001
From: Arnout Engelen <arnouten@bzzt.net>
Date: Thu, 26 Apr 2012 13:54:39 +0200
Subject: [PATCH] Update Vcs-* fields to point to Git instead of Bzr

---
 debian/control |    3 +--
 1 files changed, 1 insertions(+), 2 deletions(-)

diff --git a/debian/control b/debian/control
index a4550fa..732f98c 100644
--- a/debian/control
+++ b/debian/control
@@ -4,8 +4,7 @@ Priority: extra
 Maintainer: Debbugs developers <debian-debbugs@lists.debian.org>
 Uploaders: Colin Watson <cjwatson@debian.org>, Don Armstrong <don@debian.org>
 Standards-Version: 3.8.2
-Vcs-Bzr: http://bugs.debian.org/debbugs-source/mainline
-Vcs-Browser: http://bugs.debian.org/debbugs-source/mainline
+Vcs-Git: http://bugs.debian.org/debbugs-source/debbugs.git
 Build-Depends: debhelper (>= 5)
 Build-Depends-Indep: libparams-validate-perl,
  libmailtools-perl, libmime-tools-perl, libio-stringy-perl, libmldbm-perl,
-- 
1.7.9.1

