--- scsi-layer.c.orig	2006-07-09 11:13:36.000000000 -0500
+++ scsi-layer.c	2006-07-10 23:26:57.000000000 -0500
@@ -209,12 +209,16 @@ static int query_type(DeviceHandle *dh, 
       case  0: 
       {  int layer_type = buf[6] & 0x0f;
 
+	 if(layer_type == 0x01)
+            dh->typedescr = "DVD-ROM";
+	 else
 	 if(layer_type & 0x06) 
 	    dh->typedescr = "DVD-ROM (faked book type)";
 	 else
 	 {  dh->typedescr = "DVD-ROM";
 	    dh->subType = UNSUPPORTED;
 	 }
+	 }
 	 break;
       }
 
