--- scsi-layer.c.orig	2006-07-10 23:26:57.000000000 -0500
+++ scsi-layer.c	2006-07-10 23:31:00.000000000 -0500
@@ -1034,10 +1034,12 @@ DeviceHandle* OpenAndQueryDevice(char *d
       }
    }
 
+/*
    if(dh->mainType == DVD && query_copyright(dh))
    {  CloseDevice(dh);
       Stop(_("This software does not support encrypted media.\n"));
    }
+*/
 
    if(dh->sessions>1)
    {  int sessions = dh->sessions;
