diff -ru ace-of-penguins-1.2.pristine/debian/control ace-of-penguins-1.2/debian/control
--- ace-of-penguins-1.2.pristine/debian/control	2009-10-05 23:21:45.000000000 +0100
+++ ace-of-penguins-1.2/debian/control	2009-10-05 23:23:40.000000000 +0100
@@ -11,10 +11,10 @@
 Package: ace-of-penguins
 Architecture: any
 Depends: ${shlibs:Depends}
-Description: graphical solitaire-games with penguin-look
- The Ace of Penguins is a set of graphical solitaire games based on the
- ones available for Windows(tm) but with a number of enhancements.
+Description: penguin-themed solitaire games
+ The Ace of Penguins is a set of solitaire games inspired by the ones
+ available for MS Windows, but with a number of enhancements.
  .
- The package consists of the games Pegged, Minesweeper, Solitaire,
- Taipei (together with a level editor), Golf, Mastermind, Merlin and
- Freecell.
+ The package consists of the games Canfield, Freecell, Golf, Mastermind,
+ Merlin, Minesweeper, Pegged, Solitaire, Taipei (with a level editor),
+ and Thornq.
