--- state-funcs.orig	2008-10-08 21:22:01.000000000 +0200
+++ state-funcs	2008-10-08 21:58:14.000000000 +0200
@@ -65,7 +65,7 @@
             # For madwifi we need to check "operstate" instead.
 	    if [ -w $DEVICE/operstate ] ; then
 	        NET_IF=`echo $DEVICE | cut -d \/ -f 5`
-	        if [ "`cat $CONTROL`" = "up" ] ; then
+	        if [ "`cat $DEVICE/operstate`" = "up" ] ; then
 		    # It's powered on. Switch it off.
 		    ifdown $NET_IF
 	        else
