diff -r 872138ab2a66 -r bd548a8d1d45 src/cmdline/cmdline_prompt.cc
--- a/src/cmdline/cmdline_prompt.cc	Fri Nov 23 08:51:39 2007 -0800
+++ b/src/cmdline/cmdline_prompt.cc	Fri Nov 23 08:53:02 2007 -0800
@@ -467,6 +467,7 @@ static bool cmdline_show_preview(bool as
 				 bool showvers, bool showdeps, bool showsize,
 				 int verbose)
 {
+  const int quiet = aptcfg->FindI("Quiet", 0);
   bool all_empty=true;
 
   pkgvector lists[num_pkg_action_states];
@@ -545,7 +546,7 @@ static bool cmdline_show_preview(bool as
 	}
     }
 
-  if(!recommended.empty())
+  if(quiet == 0 && !recommended.empty())
     {
       printf(_("The following packages are RECOMMENDED but will NOT be installed:\n"));
       cmdline_show_instinfo(recommended, showvers, showdeps, showsize, false);
