diff -aur docbook-utils-0.6.14.orig/backends/txt docbook-utils-0.6.14/backends/txt
--- docbook-utils-0.6.14.orig/backends/txt	2004-02-11 14:58:03.000000000 +0100
+++ docbook-utils-0.6.14/backends/txt	2013-09-04 16:38:59.000000000 +0200
@@ -10,6 +10,10 @@
 then
   CONVERT=/usr/bin/links
   ARGS="-dump"
+elif [ -x /usr/bin/elinks ]
+then
+  CONVERT=/usr/bin/elinks
+  ARGS="-dump"
 elif [ -x /usr/bin/w3m ]
 then
   CONVERT=/usr/bin/w3m
Nur in docbook-utils-0.6.14/backends: txt~.
diff -aur docbook-utils-0.6.14.orig/debian/control docbook-utils-0.6.14/debian/control
--- docbook-utils-0.6.14.orig/debian/control	2012-05-09 18:38:19.000000000 +0200
+++ docbook-utils-0.6.14/debian/control	2013-09-04 16:39:10.000000000 +0200
@@ -12,7 +12,7 @@
 
 Package: docbook-utils
 Architecture: all
-Depends: docbook-dsssl, jadetex, lynx | links | links2| w3m, sgmlspl, sp, ${perl:Depends}, ${misc:Depends}
+Depends: docbook-dsssl, jadetex, lynx | elinks | links | links2| w3m, sgmlspl, sp, ${perl:Depends}, ${misc:Depends}
 Recommends: docbook-xml
 Replaces: docbook2man, docbook2texi, docbk-xml2x, docbook2x-doc, cygnus-stylesheets
 Description: Convert DocBook files to other formats (HTML, RTF, PS, man, PDF)
Nur in docbook-utils-0.6.14/debian: control~.
