diff -u chasen-2.3.3/debian/changelog chasen-2.3.3/debian/changelog
--- chasen-2.3.3/debian/changelog
+++ chasen-2.3.3/debian/changelog
@@ -1,3 +1,12 @@
+chasen (2.3.3-6.3) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Downgrade Recommends on ipadic to Suggests, as per release goal.
+    (Closes: #452474).
+  * Fix a typo in Description (Closes: #363196).
+
+ -- Amaya Rodrigo Sastre <amaya@debian.org>  Mon, 31 Mar 2008 01:07:55 +0200
+
 chasen (2.3.3-6.2) unstable; urgency=low
 
   * Non-maintainer upload.
diff -u chasen-2.3.3/debian/control chasen-2.3.3/debian/control
--- chasen-2.3.3/debian/control
+++ chasen-2.3.3/debian/control
@@ -12,7 +12,7 @@
 Conflicts: chasen-dev
 Description: a Japanese Morphological Analysis System (libraries and headers)
  You can use ChaSen library to put ChaSen's module into other programs.
- ChaSen is a morphological analysys system. It can segment and
+ ChaSen is a morphological analysis system. It can segment and
  tokenize Japanese text string, and can output with many additional
  informations (pronunciation, semantic information, and others).
 
@@ -21,10 +21,10 @@
 Depends: ${shlibs:Depends}
 Conflicts: libchasen0
 Replaces: libchasen0
-Recommends: ipadic | chasen-cannadic
+Suggests: ipadic | chasen-cannadic
 Description: a Japanese Morphological Analysis System (shared libraries)
  Shared libraries for ChaSen.
- ChaSen is a morphological analysys system. It can segment and
+ ChaSen is a morphological analysis system. It can segment and
  tokenize Japanese text string, and can output with many additional
  informations (pronunciation, semantic information, and others).
 
@@ -32,7 +32,7 @@
 Architecture: any
 Depends: ${shlibs:Depends}, ipadic (>= 2.6.3) | chasen-cannadic
 Description: a Japanese Morphological Analysis System
- ChaSen is a morphological analysys system. It can segment and
+ ChaSen is a morphological analysis system. It can segment and
  tokenize Japanese text string, and can output with many additional
  informations (pronunciation, semantic information, and others).
  It will print the result of such an operation to the standard output,
@@ -44,5 +44,5 @@
 Description: a Japanese Morphological Analysis System - utilities for dictionary
  Dictionary utilities for ChaSen.
- ChaSen is a morphological analysys system. It can segment and
+ ChaSen is a morphological analysis system. It can segment and
  tokenize Japanese text string, and can output with many additional
  informations (pronunciation, semantic information, and others).
