diff --git a/apparmor.d/abstractions/ubuntu-browsers b/apparmor.d/abstractions/ubuntu-browsers
index be0e070..8f57255 100644
--- a/apparmor.d/abstractions/ubuntu-browsers
+++ b/apparmor.d/abstractions/ubuntu-browsers
@@ -29,8 +29,8 @@
 
   # this should cover all firefox browsers and versions (including shiretoko
   # and abrowser)
-  /usr/bin/firefox Cxr -> sanitized_helper,
-  /usr/lib/firefox*/firefox*.sh Cx -> sanitized_helper,
+  /usr/bin/{firefox,iceweasel} Cxr -> sanitized_helper,
+  /usr/lib/{firefox*,iceweasel}/{firefox*.sh,iceweasel} Cx -> sanitized_helper,
 
   # some unpackaged, but popular browsers
   /usr/lib/icecat-*/icecat Cx -> sanitized_helper,
