=== modified file 'debian/control'
--- debian/control	2014-05-05 15:10:10 +0000
+++ debian/control	2014-06-02 18:35:02 +0000
@@ -3,29 +3,31 @@
 Priority: optional
 Maintainer: Kubuntu Developers <kubuntu-devel@lists.ubuntu.com>
 XSBC-Original-Maintainer: Scarlett Clark <scarlett@scarlettgatelyclark.com>
-Build-Depends: debhelper (>= 9.0.0), cmake, pkg-kde-tools,
- extra-cmake-modules (>= 0.0.13), qtbase5-dev, libkf5archive-dev (>= 4.99.0), libxslt1-dev, libxml2-dev, docbook-xml, docbook-xsl, libxml2-utils
+Build-Depends: cmake,
+               debhelper (>= 9.0.0),
+               docbook-xml,
+               docbook-xsl,
+               extra-cmake-modules (>= 0.0.13),
+               libkf5archive-dev (>= 4.99.0),
+               libxml2-dev,
+               libxml2-utils,
+               libxslt1-dev,
+               pkg-kde-tools,
+               qtbase5-dev
 Standards-Version: 3.9.5
 Homepage: https://projects.kde.org/projects/frameworks/kdoctools
 
-Package: kdoctools-dev
-Section: libdevel
-Architecture: any
-Depends: ${misc:Depends}, kdoctools5 (= ${binary:Version}),
- libkf5archive-dev (>= 4.99.0)
-Description: Documentation generation from docbook
- Tools to read documentation in Docbook.
- .
- Contains development files for kdoctools.
- 
 Package: kdoctools5
-Architecture: any
-Multi-Arch: allowed
 Section: doc
-Depends: ${shlibs:Depends}, ${misc:Depends}, libxml2-utils, 
- docbook-xml, docbook-xsl
+Architecture: any
+Depends: docbook-xml,
+         docbook-xsl,
+         libxml2-utils,
+         ${misc:Depends},
+         ${shlibs:Depends}
 Description: Documentation generation from docbook
  Tools to read documentation in Docbook.
+Multi-Arch: allowed
 
 Package: kdoctools5-dbg
 Priority: extra

