--- apertium-3.0.7+1.orig/apertium/apertium-header.sh	2008-02-05 07:49:07.000000000 +0100
+++ apertium-3.0.7+1/apertium/apertium-header.sh	2008-08-23 21:41:12.000000000 +0200
@@ -38,17 +38,17 @@
 
 function translate_odt
 {
-  INPUT_TMPDIR=/tmp/$$odtdir
+  INPUT_TMPDIR=`mktemp`
 
   locale_utf8
   test_zip
 
   if [[ $FICHERO == "" ]]
-  then FICHERO=/tmp/$$odtorig
+  then FICHERO=`mktemp`
        cat > $FICHERO
        BORRAFICHERO="true"
   fi
-  OTRASALIDA=/tmp/$$odtsalida.zip
+  OTRASALIDA=/tmp/$RANDOM-$RANDOM-$RANDOM-odtsalida.zip
   
   unzip -q -o -d $INPUT_TMPDIR $FICHERO
   find $INPUT_TMPDIR | grep content\\\.xml |\
@@ -78,17 +78,17 @@
 
 function translate_docx
 {
-  INPUT_TMPDIR=/tmp/$$docxdir
+  INPUT_TMPDIR=`mktemp`
 
   locale_utf8
   test_zip
   
   if [[ $FICHERO == "" ]]
-  then FICHERO=/tmp/$$docxorig
+  then FICHERO=`mktemp`
        cat > $FICHERO
        BORRAFICHERO="true"
   fi
-  OTRASALIDA=/tmp/$$docxsalida.zip
+  OTRASALIDA=/tmp/$RANDOM-$RANDOM-$RANDOM-docxsalida.zip
   
   if [[ $UWORDS == "no" ]]
   then OPCIONU="-u";
@@ -98,8 +98,9 @@
   unzip -q -o -d $INPUT_TMPDIR $FICHERO
   
   for i in $(find $INPUT_TMPDIR|grep "xlsx$");
-  do $APERTIUM_PATH/apertium -f xlsx -d $DIRECTORY $OPCIONU $PREFIJO <$i >/tmp/$$xlsxembed;
-     mv /tmp/$$xlsxembed $i;
+  TMP_XLSXEMBED=`mktemp`
+  do $APERTIUM_PATH/apertium -f xlsx -d $DIRECTORY $OPCIONU $PREFIJO <$i > $TMP_XLSXEMBED;
+     mv $TMP_XLSXEMBED $i;
   done;
   
   find $INPUT_TMPDIR | grep "xml" |\
@@ -130,17 +131,17 @@
 
 function translate_xlsx
 {
-  INPUT_TMPDIR=/tmp/$$xlsxdir
+  INPUT_TMPDIR=`mktemp`
 
   locale_utf8
   test_zip
   
   if [[ $FICHERO == "" ]]
-  then FICHERO=/tmp/$$xlsxorig
+  then FICHERO=`mktemp`
        cat > $FICHERO
        BORRAFICHERO="true"
   fi
-  OTRASALIDA=/tmp/$$xslxsalida.zip
+  OTRASALIDA=/tmp/$RANDOM-$RANDOM-$RANDOM-xslxsalida.zip
   
   unzip -q -o -d $INPUT_TMPDIR $FICHERO
   find $INPUT_TMPDIR | grep "sharedStrings.xml" |\
