--- apertium-3.0.7+1.orig/apertium/Makefile.in	2008-03-31 06:22:55.000000000 +0200
+++ apertium-3.0.7+1/apertium/Makefile.in	2008-08-23 19:37:13.000000000 +0200
@@ -1148,25 +1148,29 @@
 	@echo "Creating apertium-gen-deformat script"
 	@echo "#!$(BASH)" > $@
 	@cat deformat-header.sh >> $@
+	@echo "TMP_DEFORMAT=\`mktemp\`" >> $@
+	@echo "TMP_LEX=\`mktemp\`" >> $@
 	@echo "$(XMLLINT) --dtdvalid $(apertiumdir)/format.dtd --noout \$$FILE1 && \\" >> $@
 	@if [ `basename $(XSLTPROC)` == xsltproc ]; \
-	  then echo "$(XSLTPROC) --stringparam mode \$$MODE $(apertiumdir)/deformat.xsl \$$FILE1 >/tmp/\$$\$$.deformat.l && \\"; \
-          else echo "$(XSLTPROC) $(apertiumdir)/deformat.xsl \$$FILE1 \"\\\$$mode=\$$MODE\" >/tmp/\$$\$$.deformat.l && \\"; \
+	  then echo "$(XSLTPROC) --stringparam mode \$$MODE $(apertiumdir)/deformat.xsl \$$FILE1 \$$TMP_DEFORMAT && \\"; \
+          else echo "$(XSLTPROC) $(apertiumdir)/deformat.xsl \$$FILE1 \"\\\$$mode=\$$MODE\" > \$$TMP_DEFORMAT && \\"; \
           fi >> $@ 
-	@echo "$(FLEX) \$$FLEXOPTS -o/tmp/\$$\$$.lex.cc /tmp/\$$\$$.deformat.l && \\" >> $@
-	@echo "$(CXX) $(CXXFLAGS) -w $(APERTIUM_CFLAGS) -I $(apertiuminclude) -o \$$FILE2 /tmp/\$$\$$.lex.cc && \\" >> $@
-	@echo "rm /tmp/\$$\$$.deformat.l /tmp/\$$\$$.lex.cc" >> $@
+	@echo "$(FLEX) \$$FLEXOPTS -o \$$TMP_LEX \$$TMP_DEFORMAT && \\" >> $@
+	@echo "$(CXX) $(CXXFLAGS) -w $(APERTIUM_CFLAGS) -I $(apertiuminclude) -o \$$FILE2 \$$TMP_LEX && \\" >> $@
+	@echo "rm \$$TMP_DEFORMAT \$$TMP_LEX" >> $@
 	@chmod a+x $@
 
 apertium-gen-reformat: Makefile.am gen-header.sh
 	@echo "Creating apertium-gen-reformat script"
 	@echo "#!$(BASH)" > $@
 	@cat gen-header.sh >> $@
+	@echo "TMP_REFORMAT=\`mktemp\`" >> $@
+	@echo "TMP_LEX=\`mktemp\`" >> $@
 	@echo "$(XMLLINT) --dtdvalid $(apertiumdir)/format.dtd --noout \$$FILE1 && \\" >> $@
-	@echo "$(XSLTPROC) $(apertiumdir)/reformat.xsl \$$FILE1 >/tmp/\$$\$$.reformat.l && \\" >> $@
-	@echo "$(FLEX) \$$FLEXOPTS -o/tmp/\$$\$$.lex.cc /tmp/\$$\$$.reformat.l && \\" >> $@
-	@echo "$(CXX) $(CXXFLAGS) -w $(APERTIUM_CFLAGS) -I $(apertiuminclude) -o \$$FILE2 /tmp/\$$\$$.lex.cc &&\\" >> $@
-	@echo "rm /tmp/\$$\$$.reformat.l /tmp/\$$\$$.lex.cc" >> $@
+	@echo "$(XSLTPROC) $(apertiumdir)/reformat.xsl \$$FILE1 > \$$TMP_REFORMAT && \\" >> $@
+	@echo "$(FLEX) \$$FLEXOPTS -o \$$TMP_LEX \$$TMP_REFORMAT && \\" >> $@
+	@echo "$(CXX) $(CXXFLAGS) -w $(APERTIUM_CFLAGS) -I $(apertiuminclude) -o \$$FILE2 \$$TMP_LEX &&\\" >> $@
+	@echo "rm \$$TMP_REFORMAT \$$TMP_LEX" >> $@
 	@chmod a+x $@
 
 apertium-gen-modes: Makefile.am modes-header.sh
