diff --git a/policy.sgml b/policy.sgml
index ae7149f..7d5a108 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -718,25 +718,22 @@
 		install if you didn't know what it was and don't have
 		specialized requirements. This is a much larger system
 		and includes the X Window System, a full TeX
-		distribution, and many applications. Note that
-		optional packages should not conflict with each other.
+		distribution, and many applications.
 	    </item>
 	    <tag><tt>extra</tt></tag>
 	    <item>
-		This contains all packages that conflict with others
-		with required, important, standard or optional
-		priorities, or are only likely to be useful if you
-		already know what they are or have specialized
-		requirements.
+		This contains all packages that are only likely to be
+		useful if you already know what they are or have
+		specialized requirements.
 	    </item>
 	  </taglist>
 	</p>
 
 	<p>
-	  Packages must not depend on packages with lower priority
-	  values (excluding build-time dependencies).  In order to
-	  ensure this, the priorities of one or more packages may need
-	  to be adjusted.
+	  Packages with prority required, important, or standard must
+	  not depend on packages with lower priority values (excluding
+	  build-time dependencies). In order to ensure this, the
+	  priorities of one or more packages may need to be adjusted.
 	</p>
       </sect>
 
