diff -Nru dpkg-1.15.5.6ubuntu3/src/trigproc.c dpkg-1.15.5.6ubuntu4/src/trigproc.c
--- dpkg-1.15.5.6ubuntu3/src/trigproc.c	2010-03-16 11:29:49.000000000 +0100
+++ dpkg-1.15.5.6ubuntu4/src/trigproc.c	2010-03-18 10:22:21.000000000 +0100
@@ -103,6 +103,7 @@
 {
 	struct pkg_list *node;
 	struct pkginfo *pkg;
+	jmp_buf ejbuf;	
 
 	debug(dbg_triggers, "trigproc_run_deferred");
 	while ((node = remove_from_some_queue(&deferred))) {
@@ -110,8 +111,18 @@
 		free(node);
 		if (!pkg)
 			continue;
+
+		if (setjmp(ejbuf)) {
+		   error_unwind(ehflag_bombout);
+		}
+		push_error_handler(&ejbuf,print_error_perpackage, pkg->name);
+
 		pkg->clientdata->trigprocdeferred = NULL;
 		trigproc(pkg);
+
+		set_error_display(NULL, NULL);
+		error_unwind(ehflag_normaltidy);
+
 	}
 }
 
