diff --git a/debian/changelog b/debian/changelog
index 0f66d80..7d7146e 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,10 @@
+dsniff (2.4b1+debian-19.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Build-depend on libdb-dev (Closes: #621105)
+
+ -- Ondřej Surý <ondrej@debian.org>  Fri, 13 May 2011 11:24:37 +0200
+
 dsniff (2.4b1+debian-19) unstable; urgency=low
 
   * Standards-Version: 3.9.1 
diff --git a/debian/control b/debian/control
index 714b9c1..7648f48 100644
--- a/debian/control
+++ b/debian/control
@@ -3,7 +3,7 @@ Section: net
 Priority: extra
 Maintainer: Luciano Bello <luciano@debian.org>
 Standards-Version: 3.9.1
-Build-Depends: libdb4.6-dev, libpcap0.8-dev, libnids-dev, libssl-dev, libxmu-dev, libnet1-dev, debhelper (>= 8.0.0), dpatch
+Build-Depends: libdb-dev, libpcap0.8-dev, libnids-dev, libssl-dev, libxmu-dev, libnet1-dev, debhelper (>= 8.0.0), dpatch
 
 Package: dsniff
 Architecture: any
