diff -u audacious-2.1/debian/rules audacious-2.1/debian/rules
--- audacious-2.1/debian/rules
+++ audacious-2.1/debian/rules
@@ -52,6 +52,8 @@
 build: build-stamp
 build-stamp: config.status
 	dh_testdir
+	# Change build id from UNSUPPORTED VERSION to <vendor> package
+	echo "const char *svn_stamp = \"$(shell dpkg-vendor --query vendor) package\";" > src/audacious/build_stamp.c;
 	$(MAKE)
 	touch $@
 
