From 53db343e40e1bc4aeaca29ae1ee429aebbcaad08 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sun, 27 Dec 2009 00:11:59 +0100
Subject: [PATCH] Add debugging package

---
 debian/control |   13 +++++++++++++
 1 files changed, 13 insertions(+), 0 deletions(-)

diff --git a/debian/control b/debian/control
index d7494e2..f58bc8a 100644
--- a/debian/control
+++ b/debian/control
@@ -39,6 +39,19 @@ Description: efficient, featureful word processor with collaboration
  Additional plugins that require significant amounts of extra software to
  function are in the various abiword-plugin-* packages.
 
+Package: abiword-dbg
+Section: debug
+Architecture: any
+Priority: extra
+Depends: ${misc:Depends}, ${shlibs:Depends}, abiword (= ${binary:Version}) | libabiword-2.8 (= ${binary:Version})
+Recommends: abiword-plugin-grammar, abiword-plugin-mathview, abiword-docs, aspell-en | aspell-dictionary, poppler-utils, ttf-liberation
+Description: efficient, featureful word processor with collaboration
+ AbiWord is a full-featured, efficient word processing application.
+ It is suitable for a wide variety of word processing tasks, and
+ is extensible with a variety of plugins.
+ .
+ This packages contains the debugging symbols.
+
 Package: abiword-plugin-grammar
 Section: editors
 Architecture: any
-- 
1.6.5.7

