From 3197d7d6ccb437541877e28a0130c65dc9b5f846 Mon Sep 17 00:00:00 2001
From: Etienne Millon <etienne.millon@gmail.com>
Date: Mon, 19 Dec 2011 16:17:31 +0100
Subject: [PATCH] zsnes now builds on amd64

---
 Packages-arch-specific |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Packages-arch-specific b/Packages-arch-specific
index 797a947..1cdc248 100644
--- a/Packages-arch-specific
+++ b/Packages-arch-specific
@@ -284,7 +284,7 @@ yforth: i386 m68k sparc arm armel powerpc kfreebsd-i386 kfreebsd-amd64 # compile
 z88dk: !alpha                                                         # 32bit only (using -m32 on amd64 and ia64)
 %zipl-installer: s390						      # installer stuff
 zorroutils: m68k powerpc					      # zorro (Amiga) specific
-zsnes: i386							      # Mostly written in i386 assembler
+zsnes: i386 amd64						      # Mostly written in i386 assembler
 
 # xorg stuff
 %xf86-input-multitouch: !s390 !s390x
-- 
1.7.7.3

