diff --git a/parts/1ftpfiles b/parts/1ftpfiles
index bb8bdb0..8d38dce 100755
--- a/parts/1ftpfiles
+++ b/parts/1ftpfiles
@@ -94,6 +94,7 @@ getdeb all debian-refcard
 getdeb amd64 build-essential
 getdeb amd64 menu
 getdeb amd64 python3-dev
+getdeb amd64 live-manual-html
 
 # Files in $crondir/ftpfiles/doc are not used any more ???
 # This should make clear if someone tries and fails.
diff --git a/parts/7doc b/parts/7doc
index 9d7421f..c2c742d 100755
--- a/parts/7doc
+++ b/parts/7doc
@@ -620,6 +620,18 @@ for lang in $langlist ; do
 done
 fi
 
+unpack live-manual-html
+if [ ! -e $ftpdir/${dist}/live-manual-html.skip ]; then
+langlist=`file2lang live-manual .html usr/share/doc/live-manual/html`
+for lang in $langlist ; do
+	mvhtml live-manual/html usr/share/doc/live-manual/html YES $lang
+	mv $webdocdir/manuals/live-manual/html/live-manual.$lang.html $webdocdir/manuals/live-manual/html/index.$lang.html
+done
+mv usr/share/doc/live-manual/_sisu $webdocdir/manuals/live-manual/_sisu
+fi
+
+
+
 # Move broken code segment here for debug
 #echo
 #echo "7doc start debug (at `date`)"
