diff --git a/english/doc/devel-manuals.defs b/english/doc/devel-manuals.defs
index 4a789a21abc..7bc102528fb 100644
--- a/english/doc/devel-manuals.defs
+++ b/english/doc/devel-manuals.defs
@@ -83,3 +83,14 @@
             vcsrepo="https://salsa.debian.org/debian/dbconfig-common.git"
             />
 </define-tag>
+
+<define-tag inddpvcs-live-manual whitespace=delete>
+  <inddpvcs name="live-manual/html"
+            langs="en ca de es fr it ja pl pt_BR ro"
+            formats="html"
+            srctype="XML"
+            vcstype="git"
+            vcsweb="https://salsa.debian.org/live-team/live-manual"
+            vcsrepo="https://salsa.debian.org/live-team/live-manual.git"
+            />
+</define-tag>
diff --git a/english/doc/devel-manuals.wml b/english/doc/devel-manuals.wml
index ac0923d903e..1d113c81caf 100644
--- a/english/doc/devel-manuals.wml
+++ b/english/doc/devel-manuals.wml
@@ -260,3 +260,29 @@ package, and packaging the <code>gnujump</code> game and a Java library.
 </doctable>
 </div>
 
+<hr>
+
+<document "Debian Live Manual" "live-manual">
+
+<div class="centerblock">
+<p>
+  This document describes the build of Debian live images.
+
+  <p>FIXME:
+     debian-live team: please add some useful short description here!
+     FIXME
+  </p>
+
+<doctable>
+  <authors "Debian Live Team">
+  <maintainer "Debian Live Team">
+  <status>
+  ready
+  </status>
+  <availability>
+  <inpackage "live-manual">
+  <inddpvcs-live-manual>
+  </availability>
+</doctable>
+</div>
+
diff --git a/english/doc/index.wml b/english/doc/index.wml
index c98072d6de9..dad274a9b0d 100644
--- a/english/doc/index.wml
+++ b/english/doc/index.wml
@@ -104,6 +104,7 @@ name and/or links to the online versions.</p>
       <li><a href="devel-manuals#d-i-internals">Debian Installer internals</a></li>
       <li><a href="devel-manuals#dbconfig-common">Guide for database using package maintainers</a></li>
       <li><a href="devel-manuals#dbapp-policy">Policy for packages using databases</a></li>
+      <li><a href="devel-manuals#live-manual">Debian Live manual</a></li>
     </ul>
 
   </div>
