diff -ruN dvdisaster-0.63.2/debian/changelog dvdisaster-0.63.2+/debian/changelog
--- dvdisaster-0.63.2/debian/changelog	2005-10-26 09:46:18.000000000 +0200
+++ dvdisaster-0.63.2+/debian/changelog	2005-10-25 22:54:07.000000000 +0200
@@ -1,3 +1,9 @@
+dvdisaster (0.63.2-1.1) unstable; urgency=low
+
+  * Don't put a one megabyte buffer on the stack.
+
+ -- Robert Bihlmeyer <robbe@orcus.priv.at>  Tue, 25 Oct 2005 22:54:07 +0200
+
 dvdisaster (0.63.2-1) unstable; urgency=low
 
   * New upstream release:
diff -ruN dvdisaster-0.63.2/dvdisaster.c dvdisaster-0.63.2+/dvdisaster.c
--- dvdisaster-0.63.2/dvdisaster.c	2005-08-21 11:13:11.000000000 +0200
+++ dvdisaster-0.63.2+/dvdisaster.c	2005-10-25 22:51:17.000000000 +0200
@@ -215,7 +215,7 @@
 void ScanImage(ImageInfo *ii, EccInfo *ei, int mode, GtkWidget *pbar)
 {  EccHeader eh; 
    unsigned char buf[2048];
-   guint32 crcbuf[CRCBUFSIZE];
+   static guint32 crcbuf[CRCBUFSIZE];
    int crcidx = 0;
    struct MD5Context image_md5;
    gint64 eh_sectors = 0;
