From df6264a0572c0dbda8c49625c274d261f52a0c8d Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lo=C3=AFc=20Minier?= <lool@debian.org>
Date: Mon, 17 Oct 2011 17:52:13 +0200
Subject: [PATCH 1/9] Remove kexec-tools; Auto-not-for-us is better here

---
 Packages-arch-specific |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/Packages-arch-specific b/Packages-arch-specific
index 9358f9c..680ab2e 100644
--- a/Packages-arch-specific
+++ b/Packages-arch-specific
@@ -97,7 +97,6 @@ ikarus: i386							      # i386 assembly
 %isight-firmware-tools: i386 amd64				      # [ANAIS]
 %joystick: !hurd-i386 !kfreebsd-amd64 !kfreebsd-i386 !s390 !s390x     # Linux specific, analog joysticks
 %kbd-chooser: !s390 !s390x					      # s390 installs don't use a keyboard
-kexec-tools: armel i386 powerpc amd64 ppc64 ia64 s390 lpia            # [ANAIS] based on syscall availability
 kgb: !arm                                                             # alignment issues, no upstream support
 kon2: i386                                                            # Hardcoded i386 assembler
 %ia32-libs: amd64 ia64						      # ia32 compat libs for amd64,ia64
-- 
1.7.5.4

