From 778ba403adfed498d5c434b7136e9837f28ed8da Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lo=C3=AFc=20Minier?= <lool@debian.org>
Date: Mon, 17 Oct 2011 17:54:17 +0200
Subject: [PATCH 3/9] Add lcd4linux on armhf; built fine

---
 Packages-arch-specific |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Packages-arch-specific b/Packages-arch-specific
index 0e123a2..f9bbab8 100644
--- a/Packages-arch-specific
+++ b/Packages-arch-specific
@@ -102,7 +102,7 @@ kon2: i386                                                            # Hardcode
 %ia32-libs: amd64 ia64						      # ia32 compat libs for amd64,ia64
 %ia32-libs-gtk: amd64 ia64					      # ia32 compat libs for amd64,ia64
 %ia32-libs-core: ia64						      # ia32 compat libs, ia64-only bundle
-%lcd4linux: amd64 armel i386					      # <sys/io.h>
+%lcd4linux: amd64 armel armhf i386				      # <sys/io.h>
 %lcdproc: !s390 !s390x						      # this hardware cannot be attached
 %ldc: !s390 !s390x						      # needs porting
 %libacpi: amd64 i386 ia64 lpia					      # acpi is i386/ia64 specific
-- 
1.7.5.4

