From 3b0df39eae8ca6ae3564a48241aa26af5ab947bc Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lo=C3=AFc=20Minier?= <lool@debian.org>
Date: Mon, 17 Oct 2011 19:09:37 +0200
Subject: [PATCH 4/9] Remove nikwi; Auto-not-for-us better here

---
 Packages-arch-specific |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/Packages-arch-specific b/Packages-arch-specific
index f9bbab8..9fa593a 100644
--- a/Packages-arch-specific
+++ b/Packages-arch-specific
@@ -176,7 +176,6 @@ ndisgtk: i386 amd64 lpia                                              # depends
 %ndiswrapper: i386 amd64 lpia                                         # Windows DLL loader
 %newlib: i386 powerpc ppc64
 %nictools-nopci: i386						      # [?] ISA nictools
-nikwi: alpha amd64 arm armel kfreebsd-i386 kfreebsd-amd64 hurd-i386 ia64 mipsel # Only little endian support
 nvidia-xconfig: i386 amd64 ia64					      # i386 specific, but match what's in the archive
 %gpart: i386 hurd-i386 ia64 alpha arm armel mipsel amd64	      # little endian specific
 nspluginwrapper: amd64						      # amd64 specific
-- 
1.7.5.4

