From 9e053eff1961d89c5e80705028868f4a7bc18073 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lo=C3=AFc=20Minier?= <lool@debian.org>
Date: Mon, 17 Oct 2011 19:10:56 +0200
Subject: [PATCH 5/9] Add linux-wlan-ng on armhf; built fine

---
 Packages-arch-specific |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Packages-arch-specific b/Packages-arch-specific
index 9fa593a..95417e5 100644
--- a/Packages-arch-specific
+++ b/Packages-arch-specific
@@ -143,7 +143,7 @@ lilo-installer: i386 amd64 lpia					      # only useful if you have lilo
 %linux-modules-di-s390-2.6: s390
 %linux-modules-di-s390x-2.6: s390x
 %linux-modules-di-sparc-2.6: sparc
-%linux-wlan-ng: amd64 i386 powerpc arm armel alpha hppa lpia	      # ANAIS [?]
+%linux-wlan-ng: amd64 i386 powerpc arm armel armhf alpha hppa lpia	      # ANAIS [?]
 lkl: i386 kfreebsd-i386						      # i386 kb hardware specific
 lphdisk: i386							      # i386 laptop specific
 lsadb: powerpc							      # macintosh-only bus
-- 
1.7.5.4

