From 0322691ad29b389e753e65e6318ed98b8b8d9489 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lo=C3=AFc=20Minier?= <lool@debian.org>
Date: Mon, 17 Oct 2011 19:47:58 +0200
Subject: [PATCH 6/9] Remove qcontrol; Auto-not-for-us better here

---
 Packages-arch-specific |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/Packages-arch-specific b/Packages-arch-specific
index 95417e5..a6b739a 100644
--- a/Packages-arch-specific
+++ b/Packages-arch-specific
@@ -214,7 +214,6 @@ primaxscan: i386 kfreebsd-i386 amd64 kfreebsd-amd64		      # [?]
 %ps3-utils: powerpc						      # powerpc specific
 %psyco: i386 i386 hurd-i386 kfreebsd-i386			      # [ANAIS] no upstream support
 %purelibc: i386 amd64 powerpc ppc64				      # [ANAIS] no upstream support
-%qcontrol: arm armel						      # arm hardware specific
 %qemu-kvm: amd64 i386
 q-tools: ia64							      # ia64 perf utils
 qcam: alpha i386 arm armel					      # Attaches through the parallel port
-- 
1.7.5.4

