From 172109386eb150b25ea8f6b16d09f09781065b8a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lo=C3=AFc=20Minier?= <lool@debian.org>
Date: Mon, 17 Oct 2011 20:16:23 +0200
Subject: [PATCH 7/9] Add qcam on armhf; built fine

---
 Packages-arch-specific |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Packages-arch-specific b/Packages-arch-specific
index a6b739a..00baddb 100644
--- a/Packages-arch-specific
+++ b/Packages-arch-specific
@@ -216,7 +216,7 @@ primaxscan: i386 kfreebsd-i386 amd64 kfreebsd-amd64		      # [?]
 %purelibc: i386 amd64 powerpc ppc64				      # [ANAIS] no upstream support
 %qemu-kvm: amd64 i386
 q-tools: ia64							      # ia64 perf utils
-qcam: alpha i386 arm armel					      # Attaches through the parallel port
+qcam: alpha i386 arm armel armhf				      # Attaches through the parallel port
 qprof: ia64 amd64 i386 mips mipsel powerpc			      # Build-Depends on libpfm3-3.2
 quik: powerpc							      # powerpc boot loader
 quik-installer: powerpc						      # powerpc boot loader
-- 
1.7.5.4

