From 5be8dd2df8e6300e372a2aece7542b037811d463 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lo=C3=AFc=20Minier?= <lool@debian.org>
Date: Mon, 17 Oct 2011 20:21:53 +0200
Subject: [PATCH 8/9] Add splay on armhf; built fine

---
 Packages-arch-specific |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Packages-arch-specific b/Packages-arch-specific
index 00baddb..a669260 100644
--- a/Packages-arch-specific
+++ b/Packages-arch-specific
@@ -241,7 +241,7 @@ silo: sparc							      # sparc lilo
 %silo-installer: sparc						      # sparc lilo
 %slicer: !alpha !armel !mips !mipsel
 %sparc-utils: sparc                                                   # sparc specific
-%splay: i386 alpha arm armel m68k powerpc			      # [?] broken on sparc (see BTS]
+%splay: i386 alpha arm armel armhf m68k powerpc			      # [?] broken on sparc (see BTS]
 %spring: i386 amd64						      # x86 specific, needs java5 and hardware accellerated OpenGL
 %springlobby: i386 amd64					      # Depends on spring
 %stalin: i386 amd64 ia64 sparc kfreebsd-i386 kfreebsd-amd64	      # [ANAIS]
-- 
1.7.5.4

