From 772ecbee0969e02acf2c405ce1fa8fc1d8788b0e Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lo=C3=AFc=20Minier?= <lool@debian.org>
Date: Mon, 17 Oct 2011 20:30:00 +0200
Subject: [PATCH 9/9] Add tcc on armhf; built fine

---
 Packages-arch-specific |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Packages-arch-specific b/Packages-arch-specific
index a669260..4e485d4 100644
--- a/Packages-arch-specific
+++ b/Packages-arch-specific
@@ -251,7 +251,7 @@ spu-tools: powerpc ppc64					      # PowerPC specific
 svgatextmode: i386 amd64                                              # Hardcoded i386 assembler
 syslinux: i386 amd64 lpia                                             # i386 boot loader thingy
 %sysprof: i386 amd64                            		      # x86-specific kernel module, ANAIS
-%tcc: i386 kfreebsd-i386 hurd-i386 amd64 kfreebsd-amd64 armel	      # C compiler.
+%tcc: i386 kfreebsd-i386 hurd-i386 amd64 kfreebsd-amd64 armel armhf   # C compiler.
 tendra: i386							      # i386 specific (compiler)
 toshset: i386 amd64						      # i386 specific
 toshutils: i386 amd64                                                 # x86 specific
-- 
1.7.5.4

