--- asterisk-1.8.13.1~dfsg/main/Makefile.orig	2013-02-23 19:37:08.772815230 +0000
+++ asterisk-1.8.13.1~dfsg/main/Makefile	2013-02-23 19:37:27.236819819 +0000
@@ -34,7 +34,7 @@
 AST_LIBS += $(BKTR_LIB)
 AST_LIBS += $(LIBXML2_LIB) 
 
-ifneq ($(findstring $(OSARCH), linux-gnu uclinux linux-uclibc linux-gnueabi linux-gnueabihf kfreebsd-gnu),)
+ifneq ($(findstring $(OSARCH), linux-gnu linux-gnuspe uclinux linux-uclibc linux-gnueabi linux-gnueabihf kfreebsd-gnu),)
   ifneq ($(findstring LOADABLE_MODULES,$(MENUSELECT_CFLAGS)),)
   AST_LIBS+=-ldl
   endif
