=== modified file 'debdelta'
--- debdelta	2009-07-21 13:08:58 +0000
+++ debdelta	2009-07-21 13:11:59 +0000
@@ -93,7 +93,7 @@
 
 ####################################################################
 
-import sys , os , tempfile , string ,getopt , tarfile , shutil , time, md5, traceback, ConfigParser
+import sys , os , tempfile , string ,getopt , tarfile , shutil , time, hashlib, traceback, ConfigParser
 
 from stat    import ST_SIZE, ST_MTIME, ST_MODE, S_IMODE, S_IRUSR, S_IWUSR, S_IXUSR 
 from os.path import abspath
@@ -123,7 +123,7 @@
 f.close()
 
 f=os.popen('hostname -f')
-HOSTID=md5.new( f.read() ).hexdigest()
+HOSTID=hashlib.md5( f.read() ).hexdigest()
 f.close()
 
 

