--- autofs.orig	2012-06-14 15:38:44.000000000 +0200
+++ autofs.srp	2012-06-14 15:50:37.000000000 +0200
@@ -38,7 +38,7 @@
 fi
 
 start() {
-	log_action_begin_msg "Starting $prog" "$prog"
+	log_action_begin_msg "Starting $prog"
 
 	# Make sure autofs4 module is loaded
 	if ! grep -q autofs /proc/filesystems
@@ -68,7 +68,7 @@
 }
 
 stop() {
-	log_action_begin_msg $"Stopping $prog: "
+	log_action_begin_msg "Stopping $prog"
 	count=0
 	while [ -n "`pidof $prog`" -a $count -lt 15 ] ; do
 		start-stop-daemon --stop --exec $DAEMON --oknodo
@@ -93,11 +93,11 @@
 reload() {
 	pid=`pidof $prog`
 	if [ -z $pid ]; then
-		log_action_msg $"$prog not running"
+		log_action_msg "$prog not running"
 		RETVAL=1
 	else
 		kill -HUP $pid 2> /dev/null
-		log_action_msg $"Reloading maps"
+		log_action_msg "Reloading maps"
 		RETVAL=0
 	fi
 	return $RETVAL
