diff -u 9menu-1.8/debian/changelog 9menu-1.8/debian/changelog
--- 9menu-1.8/debian/changelog
+++ 9menu-1.8/debian/changelog
@@ -1,3 +1,11 @@
+9menu (1.8-1.2) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Add build-dep on libxext-dev, fixes FTBFS (Closes: #486973, #464593)
+  * Add build-dep on xutils-dev for imake (Closes: #485188)x	
+
+ -- Moritz Muehlenhoff <jmm@debian.org>  Fri, 27 Jun 2008 21:52:55 +0200
+
 9menu (1.8-1.1) unstable; urgency=high
 
   * Non-maintainer upload.
diff -u 9menu-1.8/debian/control 9menu-1.8/debian/control
--- 9menu-1.8/debian/control
+++ 9menu-1.8/debian/control
@@ -2,7 +2,7 @@
 Section: x11
 Priority: optional
 Maintainer: Andrew Stribblehill <ads@debian.org>
-Build-Depends: libx11-dev, x-dev, debhelper (>>4.0.0), xutils, dpatch
+Build-Depends: libx11-dev, x-dev, debhelper (>>4.0.0), xutils, dpatch, libxext-dev, xutils-dev
 Standards-Version: 3.6.2.0
 
 Package: 9menu
