--- clang-3.0/tools/clang/lib/Driver/ToolChains.cpp.orig	2011-12-14 17:34:01.000000000 +0100
+++ clang-3.0/tools/clang/lib/Driver/ToolChains.cpp	2011-12-14 17:24:45.000000000 +0100
@@ -1785,6 +1785,10 @@ static std::string getMultiarchTriple(co
     if (llvm::sys::fs::exists(SysRoot + "/lib/x86_64-unknown-linux-gnu"))
       return "x86_64-unknown-linux-gnu";
     return TargetTriple.str();
+  case llvm::Triple::ppc:
+    if (llvm::sys::fs::exists(SysRoot + "/lib/powerpc-linux-gnu"))
+      return "powerpc-linux-gnu";
+    return TargetTriple.str();
   }
 }
 
