--- bash_completion.sh.orig	2015-01-24 13:22:04.380023295 -0500
+++ bash_completion.sh	2015-01-24 13:23:23.758283088 -0500
@@ -1,9 +1,9 @@
 # Check for interactive bash and that we haven't already been sourced.
-if [ -n "$BASH_VERSION" -a -n "$PS1" -a -z "$BASH_COMPLETION_COMPAT_DIR" ]; then
+if [ -n "${BASH_VERSION-}" -a -n "${PS1-}" -a -z "${BASH_COMPLETION_COMPAT_DIR-}" ]; then
 
 # Check for recent enough version of bash.
 bash=${BASH_VERSION%.*}; bmajor=${bash%.*}; bminor=${bash#*.}
-if [ $bmajor -gt 4 ] || [ $bmajor -eq 4 -a $bminor -ge 1 ]; then
+if [ "$bmajor" -gt 4 ] || [ "$bmajor" -eq 4 -a "$bminor" -ge 1 ]; then
     [ -r "${XDG_CONFIG_HOME:-$HOME/.config}/bash_completion" ] && \
         . "${XDG_CONFIG_HOME:-$HOME/.config}/bash_completion"
     if shopt -q progcomp && [ -r /usr/share/bash-completion/bash_completion ]; then
