--- AfterStep-2.2.9/libASGTK/asgtklookedit.c.orig	2009-07-20 12:48:52.749249506 -0600
+++ AfterStep-2.2.9/libASGTK/asgtklookedit.c	2009-07-20 12:48:55.577201648 -0600
@@ -28,6 +28,7 @@
 #include "../libAfterStep/parser.h"
 #include "../libAfterStep/freestor.h"
 #include "../libAfterStep/mystyle.h"
+#include "../libAfterStep/session.h"
 #include "../libAfterConf/afterconf.h"
 
 #include <unistd.h>		   
