diff --git a/debian/changelog b/debian/changelog
index 051942b..219e9e8 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,10 @@
+bzip2 (1.0.5-4+multiarch) UNRELEASED; urgency=low
+
+  * Use multiarch paths, based on a patch from Goswin von Brederlow
+  * Set the shared library to Multi-Arch: same (closes: #528143)
+
+ -- Simon McVittie <smcv@debian.org>  Fri, 12 Feb 2010 20:39:17 +0000
+
 bzip2 (1.0.5-4) unstable; urgency=low
 
   [Jorge Ernesto Guevara Cuenca]
diff --git a/debian/control b/debian/control
index 41a1ffa..1cbbbd8 100644
--- a/debian/control
+++ b/debian/control
@@ -12,6 +12,7 @@ Vcs-Browser: http://git.debian.org/?p=collab-maint/bzip2.git
 Package: libbz2-1.0
 Priority: important
 Section: libs
+Multi-Arch: same
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}
 Description: high-quality block-sorting file compressor library - runtime
diff --git a/debian/rules b/debian/rules
index 5e11bf0..355f0ff 100755
--- a/debian/rules
+++ b/debian/rules
@@ -157,20 +157,20 @@ binary-arch: build
 	### Split
 
 	# Development package
-	install -d debian/tmp-dev/usr/lib
+	install -d debian/tmp-dev/usr/lib/$(DEB_HOST_GNU_TYPE)
 	rm debian/tmp/lib/libbz2.so
-	ln -s /lib/libbz2.so.1.0 debian/tmp-dev/usr/lib/libbz2.so
-	mv debian/tmp/lib/libbz2.a debian/tmp-dev/usr/lib/
+	ln -s /lib/$(DEB_HOST_GNU_TYPE)/libbz2.so.1.0 debian/tmp-dev/usr/lib/$(DEB_HOST_GNU_TYPE)/libbz2.so
+	mv debian/tmp/lib/libbz2.a debian/tmp-dev/usr/lib/$(DEB_HOST_GNU_TYPE)/
 	mv debian/tmp/include debian/tmp-dev/usr/include
 	
 
 	# Library package
-	install -d debian/tmp-lib/lib
+	install -d debian/tmp-lib/lib/$(DEB_HOST_GNU_TYPE)
 	for i in $$(ls debian/tmp/lib/libbz2.so*); do \
-	  (set -x; echo $$i; mv $$i debian/tmp-lib/lib/); \
+	  (set -x; echo $$i; mv $$i debian/tmp-lib/lib/$(DEB_HOST_GNU_TYPE)/); \
 	done
-	ln -s libbz2.so.1.0.4 debian/tmp-lib/lib/libbz2.so.1
-	chmod -x debian/tmp-lib/lib/*
+	ln -s libbz2.so.1.0.4 debian/tmp-lib/lib/$(DEB_HOST_GNU_TYPE)/libbz2.so.1
+	chmod -x debian/tmp-lib/lib/$(DEB_HOST_GNU_TYPE)/*
 
 	# Binary package
 	install -d debian/tmp-run
@@ -226,14 +226,14 @@ binary-arch: build
 	  fi; \
 	done
 	$(DEBSTRIP) --strip-unneeded -R .note -R .comment \
-	  debian/tmp-lib/lib/*.so*
-	$(DEBSTRIP) --strip-debug debian/tmp-dev/usr/lib/*.a
+	  debian/tmp-lib/lib/$(DEB_HOST_GNU_TYPE)/*.so*
+	$(DEBSTRIP) --strip-debug debian/tmp-dev/usr/lib/$(DEB_HOST_GNU_TYPE)/*.a
 
 	# Control files
 	install -d debian/tmp-{lib,dev,run}/DEBIAN
 	cp debian/shlibs debian/tmp-lib/DEBIAN/shlibs
 
-	dpkg-shlibdeps debian/tmp-lib/lib/*.so*
+	dpkg-shlibdeps debian/tmp-lib/lib/$(DEB_HOST_GNU_TYPE)/*.so*
 	dpkg-gencontrol -isp -Pdebian/tmp-lib -plibbz2-1.0
 	rm debian/substvars
 	dpkg-gencontrol -isp -Pdebian/tmp-dev -plibbz2-dev
