diff -u bzip2-1.0.5/debian/control bzip2-1.0.5/debian/control
--- bzip2-1.0.5/debian/control
+++ bzip2-1.0.5/debian/control
@@ -4,7 +4,7 @@
 Maintainer: Anibal Monsalve Salazar <anibal@debian.org>
 Uploaders: Santiago Ruano Rincón <santiago@debian.org>, Jorge Ernesto Guevara Cuenca <jguevara@debiancolombia.org>
 Standards-Version: 3.8.2
-Build-depends: texinfo, gcc-multilib [amd64 i386 kfreebsd-amd64 powerpc ppc64 s390 sparc] | gcc-4.1 (<< 4.1.2) [amd64 i386 kfreebsd-amd64 powerpc ppc64 s390 sparc]
+Build-depends: texinfo, gcc-multilib [amd64 i386 kfreebsd-amd64 powerpc ppc64 s390 sparc] | gcc-4.1 (<< 4.1.2) [amd64 i386 kfreebsd-amd64 powerpc ppc64 s390 sparc], dpkg-dev (>= 1.16.0)
 Homepage: http://www.bzip.org/
 Vcs-Git: git://git.debian.org/collab-maint/bzip2.git
 Vcs-Browser: http://git.debian.org/?p=collab-maint/bzip2.git
@@ -12,8 +12,10 @@
 Package: libbz2-1.0
 Priority: important
 Section: libs
+Multi-Arch: same
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}
+Pre-Depends: multiarch-support
 Description: high-quality block-sorting file compressor library - runtime
  This package contains libbzip2 which is used by the bzip2 compressor.
  .
@@ -34,6 +36,7 @@
 Package: libbz2-dev
 Priority: optional
 Section: libdevel
+Multi-Arch: same
 Architecture: any
 Depends: libbz2-1.0 (=${binary:Version}), libc6-dev | libc-dev
 Replaces: bzip2 (<<0.9.5d-3)
@@ -58,6 +61,7 @@
 Priority: optional
 Section: utils
 Architecture: any
+Multi-Arch: foreign
 Depends: libbz2-1.0 (=${binary:Version}), ${shlibs:Depends}
 Suggests: bzip2-doc
 Replaces: libbz2 (<<0.9.5d-3)
@@ -98,7 +102,7 @@
 Priority: optional
 Section: libs
 Architecture: amd64 ppc64
-Pre-Depends: libc6-i386 (>= 2.9-18)
+Pre-Depends: libc6-i386 (>= 2.9-18) [amd64]
 Depends: ${shlibs:Depends}, ${misc:Depends}
 Replaces: ia32-libs
 Description: high-quality block-sorting file compressor library - 32bit runtime
diff -u bzip2-1.0.5/debian/rules bzip2-1.0.5/debian/rules
--- bzip2-1.0.5/debian/rules
+++ bzip2-1.0.5/debian/rules
@@ -13,16 +13,19 @@
 DEB_HOST_GNU_TYPE	:= $(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)
 DEB_HOST_ARCH		:= $(shell dpkg-architecture -qDEB_HOST_ARCH)
 DEB_HOST_ARCH_OS	:= $(shell dpkg-architecture -qDEB_HOST_ARCH_OS)
+DEB_HOST_MULTIARCH      ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
+
 ifneq ($(DEB_HOST_GNU_TYPE),$(DEB_BUILD_GNU_TYPE))
         CROSS=CC=$(DEB_HOST_GNU_TYPE)-gcc
         CC=$(DEB_HOST_GNU_TYPE)-gcc
+        DEBSTRIP=$(DEB_HOST_GNU_TYPE)-strip
 else
         CROSS=CC=gcc
         CC=gcc
+        DEBSTRIP=strip
 endif
 
 DEBCFLAGS:=
-DEBSTRIP:=strip
 WGET=wget
 ifneq (,$(findstring debug,$(DEB_BUILD_OPTIONS)))
 DEBCFLAGS += -g
@@ -157,20 +160,20 @@
 	### Split
 
 	# Development package
-	install -d debian/tmp-dev/usr/lib
+	install -d debian/tmp-dev/usr/lib/$(DEB_HOST_MULTIARCH)
 	rm debian/tmp/lib/libbz2.so
-	ln -s /lib/libbz2.so.1.0 debian/tmp-dev/usr/lib/libbz2.so
-	mv debian/tmp/lib/libbz2.a debian/tmp-dev/usr/lib/
+	ln -s /lib/$(DEB_HOST_MULTIARCH)/libbz2.so.1.0 debian/tmp-dev/usr/lib/$(DEB_HOST_MULTIARCH)/libbz2.so
+	mv debian/tmp/lib/libbz2.a debian/tmp-dev/usr/lib/$(DEB_HOST_MULTIARCH)/
 	mv debian/tmp/include debian/tmp-dev/usr/include
 	
 
 	# Library package
-	install -d debian/tmp-lib/lib
+	install -d debian/tmp-lib/lib/$(DEB_HOST_MULTIARCH)
 	for i in $$(ls debian/tmp/lib/libbz2.so*); do \
-	  (set -x; echo $$i; mv $$i debian/tmp-lib/lib/); \
+	  (set -x; echo $$i; mv $$i debian/tmp-lib/lib/$(DEB_HOST_MULTIARCH)/); \
 	done
-	ln -s libbz2.so.1.0.4 debian/tmp-lib/lib/libbz2.so.1
-	chmod -x debian/tmp-lib/lib/*
+	ln -s libbz2.so.1.0.4 debian/tmp-lib/lib/$(DEB_HOST_MULTIARCH)/libbz2.so.1
+	chmod -x debian/tmp-lib/lib/$(DEB_HOST_MULTIARCH)/*
 
 	# Binary package
 	install -d debian/tmp-run
@@ -226,14 +229,14 @@
 	  fi; \
 	done
 	$(DEBSTRIP) --strip-unneeded -R .note -R .comment \
-	  debian/tmp-lib/lib/*.so*
-	$(DEBSTRIP) --strip-debug debian/tmp-dev/usr/lib/*.a
+	  debian/tmp-lib/lib/$(DEB_HOST_MULTIARCH)/*.so*
+	$(DEBSTRIP) --strip-debug debian/tmp-dev/usr/lib/$(DEB_HOST_MULTIARCH)/*.a
 
 	# Control files
 	install -d debian/tmp-{lib,dev,run}/DEBIAN
 	cp debian/shlibs debian/tmp-lib/DEBIAN/shlibs
 
-	dpkg-shlibdeps debian/tmp-lib/lib/*.so*
+	dpkg-shlibdeps debian/tmp-lib/lib/$(DEB_HOST_MULTIARCH)/*.so*
 	dpkg-gencontrol -isp -Pdebian/tmp-lib -plibbz2-1.0
 	rm debian/substvars
 	dpkg-gencontrol -isp -Pdebian/tmp-dev -plibbz2-dev
diff -u bzip2-1.0.5/debian/changelog bzip2-1.0.5/debian/changelog
--- bzip2-1.0.5/debian/changelog
+++ bzip2-1.0.5/debian/changelog
@@ -1,3 +1,12 @@
+bzip2 (1.0.5-7) unstable; urgency=low
+
+  * multiarchize, closes: #528143
+  * from Colin watson: 
+    lib32bz2-1.0 only pre-depends on libc6-i386 on amd64; that
+    pre-dependency is nonsensical on ppc64.
+
+ -- Riku Voipio <riku.voipio@linaro.org>  Mon, 15 Aug 2011 13:21:25 +0300
+
 bzip2 (1.0.5-6) unstable; urgency=high
 
   * Fix integer overflow 
