diff -u dmraid-1.0.0.rc16/debian/changelog dmraid-1.0.0.rc16/debian/changelog
--- dmraid-1.0.0.rc16/debian/changelog
+++ dmraid-1.0.0.rc16/debian/changelog
@@ -1,3 +1,11 @@
+dmraid (1.0.0.rc16-4.2) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Call udevadm settle before activating dmraid devices
+    (Closes: #677450)
+
+ -- Ulrich Dangel <uli@debian.org>  Mon, 10 Sep 2012 17:35:50 +0100
+
 dmraid (1.0.0.rc16-4.1) unstable; urgency=low
 
   * Non-maintainer upload.
diff -u dmraid-1.0.0.rc16/debian/initramfs/dmraid.initramfs-local-top/dmraid dmraid-1.0.0.rc16/debian/initramfs/dmraid.initramfs-local-top/dmraid
--- dmraid-1.0.0.rc16/debian/initramfs/dmraid.initramfs-local-top/dmraid
+++ dmraid-1.0.0.rc16/debian/initramfs/dmraid.initramfs-local-top/dmraid
@@ -16,6 +16,9 @@
 	;;
 esac
 
+. /scripts/functions
+wait_for_udev 30
+
 # Activate any dmraid arrays that were not identified by udev and vol_id.
 
 if devices=$(dmraid -r -c); then
