--- ./panabright.sh.orig	2009-04-19 18:24:27.000000000 +0000
+++ ./panabright.sh	2009-04-19 18:29:39.000000000 +0000
@@ -16,10 +16,20 @@
 fi
 
 # get brightness parameters
-
-BRIGHTNESS=$(( `cat /proc/acpi/pcc/$INTERFACE` ))
-MAXBRIGHT=$(( `cat /proc/acpi/pcc/"$INTERFACE"_max` ))
-MINBRIGHT=$(( `cat /proc/acpi/pcc/"$INTERFACE"_min` ))
+# The new in-kernel driver uses /sys/.../panasonic, while the old
+# pcc_acpi driver uses /proc/acpi/pcc
+if test -d "/sys/class/backlight/panasonic"
+then
+	BRIGHTNESS="`cat /sys/class/backlight/panasonic/brightness`"
+	MAXBRIGHT="`cat /sys/class/backlight/panasonic/max_brightness`"
+	MINBRIGHT="20"
+	BRIGHTNESS_ADJUST="/sys/class/backlight/panasonic/brightness"
+else
+	BRIGHTNESS=$(( `cat /proc/acpi/pcc/$INTERFACE` ))
+	MAXBRIGHT=$(( `cat /proc/acpi/pcc/"$INTERFACE"_max` ))
+	MINBRIGHT=$(( `cat /proc/acpi/pcc/"$INTERFACE"_min` ))
+	BRIGHTNESS_ADJUST="/proc/acpi/pcc/$INTERFACE"
+fi
 
 # adjust span so that there are 10 brightness increments
 
@@ -34,7 +44,7 @@
    else
       BRIGHTNESS=$(( $MINBRIGHT ))
    fi
-   echo $BRIGHTNESS > /proc/acpi/pcc/$INTERFACE
+   echo $BRIGHTNESS > "${BRIGHTNESS_ADJUST}"
 elif [ "x$1" = "xup" ]; then
    LIMIT=$(( $MAXBRIGHT - $SPAN ))
    if [ $BRIGHTNESS -lt $LIMIT ]; then
@@ -42,7 +52,7 @@
    else
       BRIGHTNESS=$(( $MAXBRIGHT ))
    fi
-   echo $BRIGHTNESS > /proc/acpi/pcc/$INTERFACE
+   echo $BRIGHTNESS > "${BRIGHTNESS_ADJUST}"
 else
    echo >&2 Unknown argument $1
 fi
