Index: best-pkging-practices.dbk
===================================================================
--- best-pkging-practices.dbk	(revision 10221)
+++ best-pkging-practices.dbk	(working copy)
@@ -412,10 +412,29 @@
 </screen>
 </section>
 
+<section id="s6.2.5.3">
+<title>Vcs-Upstream-Browser</title>
+<para>
+Value of this field should be a <literal>http://</literal> URL pointing to a
+web-browsable copy of the upstream Version Control System repository, if
+available.
+</para>
+<para>
+The information is meant to be useful for the final user, willing to browse the
+latest upstream work (e.g. when looking for an upstream fix for a bug).
+</para>
 </section>
 
+<section id="s6.2.5.4">
+<title>Vcs-Upstream-*</title>
+Value of this field should be the same as in <xref linkend="s6.2.5.2"/>, but
+for the upstream repository instead of the Debian packaging.
 </section>
 
+</section>
+
+</section>
+
 <section id="bpp-debian-changelog">
 <title>Best practices for <filename>debian/changelog</filename></title>
 <para>
