--- /usr/bin/dh_icons	2007-11-03 04:18:55.000000000 +0100
+++ dh_icons	2008-01-03 11:53:49.000000000 +0100
@@ -46,7 +46,7 @@
 			next if $subdir =~ /^\./;
 			my $needs_cache = 0;
 			find sub {
-				$needs_cache = 1 if -f and (/\.png$/ or /\.svg$/ or /\.jpg$/);
+				$needs_cache = 1 if -f and (/\.png$/ or /\.svg$/ or /\.xpm$/);
 			}, "$icondir/$subdir" ;
 			push @dirlist, "$baseicondir/$subdir" if $needs_cache;
 		}
