diff --git a/Packages-arch-specific b/Packages-arch-specific
index b86373c..46828dc 100644
--- a/Packages-arch-specific
+++ b/Packages-arch-specific
@@ -331,8 +331,8 @@ silo: sparc							      # sparc lilo
 %sparc-utils: sparc                                                   # sparc specific
 %spicctrl: i386							      # For Sony VAIO laptops
 %splay: i386 alpha arm armel m68k powerpc			      # [?] broken on sparc (see BTS]
-%spring: i386 amd64 kfreebsd-i386 kfreebsd-amd64 hurd-i386	      # x86 specific
-%springlobby: i386 amd64 kfreebsd-i386 kfreebsd-amd64 hurd-i386	      # Depends on spring
+%spring: i386 amd64						      # x86 specific, needs java5 and hardware accellerated OpenGL
+%springlobby: i386 amd64					      # Depends on spring
 %srm-reader: alpha						      # Alpha specific udeb
 %stalin: i386 amd64 ia64 sparc kfreebsd-i386 kfreebsd-amd64	      # [ANAIS]
 subterfugue: i386						      # i386 specific process tracing etc.
