diff --git a/debian/chromium.links b/debian/chromium.links
index fe0fba3..4b06380 100644
--- a/debian/chromium.links
+++ b/debian/chromium.links
@@ -1,2 +1,2 @@
-usr/share/icons/hicolor/32x32/chromium.xpm usr/share/pixmaps/chromium.xpm
-usr/share/icons/hicolor/48x48/chromium.png usr/share/pixmaps/chromium.png
+usr/share/icons/hicolor/32x32/apps/chromium.xpm usr/share/pixmaps/chromium.xpm
+usr/share/icons/hicolor/48x48/apps/chromium.png usr/share/pixmaps/chromium.png
diff --git a/debian/rules b/debian/rules
index f22616a..9ac2488 100755
--- a/debian/rules
+++ b/debian/rules
@@ -124,7 +124,7 @@ override_dh_auto_install:
 	for file in $(shell find chrome/app/theme/chromium -name product_logo_[0-9]* -a ! -name *mono*); do \
 	    ext=$$(echo $$file | cut -d\. -f2); \
 	    siz=$$(echo $$file | cut -d_ -f3 | sed s/\.$$ext//); \
-	    dst=debian/chromium/usr/share/icons/hicolor/$$siz\x$$siz; \
+	    dst=debian/chromium/usr/share/icons/hicolor/$$siz\x$$siz/apps; \
 	    mkdir -p $$dst; \
 	    cp $$file $$dst/chromium.$$ext; \
 	    done
