--- debian-el.old/debian-bug.el	2007-09-25 04:25:00.000000000 +0400
+++ debian-el.new/debian-bug.el	2009-03-28 16:59:26.000000000 +0300
@@ -1686,7 +1686,7 @@
                      debian-bug-download-directory))
           (status)
           (url (concat "http://bugs.debian.org/cgi-bin/bugreport.cgi?bug="
-                       bug-number "&mbox=yes")))
+                       bug-number "&mbox=yes&mboxmaint=yes")))
       (if (and (file-exists-p filename)
                (not (y-or-n-p "Bug file already exists.  Download again? ")))
           filename
