diff -u emacs-goodies-el-29.4/debian/README.Debian emacs-goodies-el-29.4/debian/README.Debian
--- emacs-goodies-el-29.4/debian/README.Debian
+++ emacs-goodies-el-29.4/debian/README.Debian
@@ -252,6 +252,10 @@
 protocols.el provides M-x protocols-lookup, to search for info in your
 /etc/protocols.
 
+quack.el enhances emacs support for Scheme programming. It integrates menus and
+commands for viewing Scheme-related reference materials, including SRFIs. It
+was originally designed for PLT Scheme but is not implementation-specific.
+
 rfcview.el formats IETF RFCs for improved readability.
 
 services.el provides M-x services-lookup, to search for info in your
diff -u emacs-goodies-el-29.4/debian/control emacs-goodies-el-29.4/debian/control
--- emacs-goodies-el-29.4/debian/control
+++ emacs-goodies-el-29.4/debian/control
@@ -72,6 +72,7 @@
   projects - create project-based meaningful buffer names;
   prot-buf - protect buffers from accidental killing;
   protocols - perform lookups in /etc/protocols;
+  quack - enhanced emacs support for Scheme programming
   rfcview - view IETF RFCs with readability-improved formatting;
   services - perform lookups in /etc/services;
   session - menu to restore files visited in previous editing session;
diff -u emacs-goodies-el-29.4/debian/emacs-goodies-el.copyright emacs-goodies-el-29.4/debian/emacs-goodies-el.copyright
--- emacs-goodies-el-29.4/debian/emacs-goodies-el.copyright
+++ emacs-goodies-el-29.4/debian/emacs-goodies-el.copyright
@@ -907,6 +907,20 @@
 | ;; General Public Licence, version 2. For details see the file COPYING.
 `----
 
+Author/copyright info for quack.el
+from http://www.neilvandyke.org/quack/quack.el
+,----
+| ;;; quack.el --- Enhanced emacs support for Scheme programming
+|
+| ;; Author:    Neil W. Van Dyke <neil@neilvandyke.org>
+| [...]
+| ;; It is free software; you can redistribute it and/or modify it
+| ;; under the terms of the GNU General Public License as published by
+| ;; the Free Software Foundation; either version 2, or (at your option)
+| ;; any later version.
+`----
+
+
 Author/copyright info for rfcview.el:
 ,----
 | ;;; rfcview.el -- view IETF RFCs with readability-improved formatting
diff -u emacs-goodies-el-29.4/debian/emacs-goodies-el.install emacs-goodies-el-29.4/debian/emacs-goodies-el.install
--- emacs-goodies-el-29.4/debian/emacs-goodies-el.install
+++ emacs-goodies-el-29.4/debian/emacs-goodies-el.install
@@ -62,6 +62,7 @@
 elisp/emacs-goodies-el/projects.el /usr/share/emacs/site-lisp/emacs-goodies-el/
 elisp/emacs-goodies-el/protbuf.el /usr/share/emacs/site-lisp/emacs-goodies-el/
 elisp/emacs-goodies-el/protocols.el /usr/share/emacs/site-lisp/emacs-goodies-el/
+elisp/emacs-goodies-el/quack.el /usr/share/emacs/site-lisp/emacs-goodies-el/
 elisp/emacs-goodies-el/rfcview.el /usr/share/emacs/site-lisp/emacs-goodies-el/
 elisp/emacs-goodies-el/services.el /usr/share/emacs/site-lisp/emacs-goodies-el/
 elisp/emacs-goodies-el/session.el /usr/share/emacs/site-lisp/emacs-goodies-el/
