--- dh_make.orig	2012-12-26 23:37:13.419564524 +0900
+++ dh_make	2012-12-26 23:39:34.451755551 +0900
@@ -531,6 +531,8 @@
 		print "Skipping creating ../$package_name\_$version.orig.tar.bz2 because it already exists\n";
 	} elsif ( -f "../$package_name\_$version.orig.tar.lzma") {
 		print "Skipping creating ../$package_name\_$version.orig.tar.lzma because it already exists\n";
+	} elsif ( -f "../$package_name\_$version.orig.tar.xz") {
+		print "Skipping creating ../$package_name\_$version.orig.tar.xz because it already exists\n";
 	} else {
 		if ($source_file)
 		{
@@ -543,6 +545,8 @@
 					system('cp', '-a', "$source_file", "../$package_name\_$version.orig.tar.bz2");
 				} elsif ($source_file =~ /lzma$/ ) {
 					system('cp', '-a', "$source_file", "../$package_name\_$version.orig.tar.lzma");
+				} elsif ($source_file =~ /xz$/ ) {
+					system('cp', '-a', "$source_file", "../$package_name\_$version.orig.tar.xz");
 				}
 			} else {
 				print "Source archive you specified ( $source_file ) was not found!\n";
