--- developers-reference.sgml	2003-10-20 03:09:41.000000000 +0200
+++ developers-reference.sgml.new	2004-10-29 23:22:32.000000000 +0200
@@ -286,9 +286,19 @@
 	<p>
 If you add signatures to your public key, or add user identities, you
 can update the Debian key ring by sending your key to the key server at
-<tt>&keyserver-host;</tt>.  If you need to add a completely new key,
-or remove an old key, send mail to &email-debian-keyring;.  The same
-key extraction routines discussed in <ref id="registering"> apply.
+<tt>&keyserver-host;</tt>.
+	<p>
+If you need to add a completely new key or remove an old key, you need
+to get the new key signed by another developer. After this, a mail
+signed by another developer listing your account name, the keyids
+of the old and of the new key and the reason should be send to 
+&email-debian-keyring;. If the old key is compromised or invalid, you
+also have to add the revocation certificate. If there is no real
+reason for a new key, the Keyring Maintainers will only accept it if
+it's more secure and connected to the old key.
+	<p>
+The same key extraction routines discussed in <ref id="registering">
+apply.
 	<p>
 You can find a more in-depth discussion of Debian key maintenance in
 the documentation of the <package>debian-keyring</package> package.
