--- /tmp/bash_completion.orig	2007-07-29 13:02:35.000000000 +0200
+++ /etc/bash_completion	2007-07-29 00:49:12.000000000 +0200
@@ -2509,7 +2509,7 @@
 	    fi
 	    # append any available aliases from config files
 	    if [ ${#config[@]} -gt 0 ] && [ -n "$aliases" ]; then
-		hosts=$( compgen -W "$( sed -ne 's/^[Hh][Oo][Ss][Tt]['"$'\t '"']*\([^*?]*\)$/\1/p' ${config[@]} )" -- $ocur )
+		hosts=$( compgen -W "$( sed -ne 's/^[Hh][Oo][Ss][Tt] ['"$'\t '"']*\([^*?]*\)$/\1/p' ${config[@]} )" -- $ocur )
 		COMPREPLY=( ${COMPREPLY[@]} $hosts )
 	    fi
 
