diff -Nru etckeeper-0.37/debian/cron.daily etckeeper-0.37ubuntu1/debian/cron.daily
--- etckeeper-0.37/debian/cron.daily	2009-02-13 19:52:49.000000000 +0100
+++ etckeeper-0.37ubuntu1/debian/cron.daily	2009-06-16 11:00:34.000000000 +0200
@@ -3,6 +3,10 @@
 if [ -x /usr/sbin/etckeeper ] && [ -e /etc/etckeeper/etckeeper.conf ]; then
 	. /etc/etckeeper/etckeeper.conf
 	if [ "$AVOID_DAILY_AUTOCOMMITS" != "1" ]; then
+		if [ "$VCS" = hg ]; then
+			hostname=`hostname -f 2>/dev/null || hostname`
+			export HGUSER=cron@$hostname
+		fi
 		if etckeeper unclean; then
 			etckeeper commit "daily autocommit" >/dev/null
 		fi
