diff --git a/debian/control b/debian/control
index 74e8a2a..4e2bf47 100644
--- a/debian/control
+++ b/debian/control
@@ -50,7 +50,8 @@ Depends: ${shlibs:Depends},
          gvfs,
          libbrasero-media3-1 (= ${binary:Version}),
          brasero-common (= ${source:Version})
-Recommends: yelp
+Recommends: yelp,
+            brasero-cdrkit
 Suggests: vcdimager,
           libdvdcss2,
           tracker
diff --git a/debian/control.in b/debian/control.in
index 4882589..f7b23c6 100644
--- a/debian/control.in
+++ b/debian/control.in
@@ -46,7 +46,8 @@ Depends: ${shlibs:Depends},
          gvfs,
          libbrasero-media3-1 (= ${binary:Version}),
          brasero-common (= ${source:Version})
-Recommends: yelp
+Recommends: yelp,
+            brasero-cdrkit
 Suggests: vcdimager,
           libdvdcss2,
           tracker
