--- unpacked/usr/bin/checkbashisms	2009-09-11 05:50:40.000000000 -0500
+++ /usr/bin/checkbashisms	2009-12-11 22:35:27.000000000 -0600
@@ -472,6 +472,8 @@
 	qr'\s\|\&' =>                    q<pipelining is not POSIX>,
 	qr'[^\\\$]\{([^\s\\\}]*?,)+[^\\\}\s]*\}' =>
 	                               q<brace expansion>,
+	qr'\{\d+\.\.\d+\}' =>
+	                               q<brace expansion, should be $(seq a b)>,
 	qr'(?:^|\s+)\w+\[\d+\]=' =>      q<bash arrays, H[0]>,
 	$LEADIN . qr'read\s+(?:-[a-qs-zA-Z\d-]+)' => q<read with option other than -r>,
 	$LEADIN . qr'read\s*(?:-\w+\s*)*(?:\".*?\"|[\'].*?[\'])?\s*(?:;|$)'
