Index: resources.dbk
===================================================================
--- resources.dbk	(revision 7057)
+++ resources.dbk	(working copy)
@@ -425,7 +426,7 @@
 <para>
 The &debian-formal; distribution consists of a lot of packages
 (<filename>.deb</filename>'s, currently around
-&number-of-pkgs;) and a few additional files (such as
+&number-of-pkgs; source packages) and a few additional files (such as
 documentation and installation disk images).
 </para>
 <para>
Index: common.ent
===================================================================
--- common.ent	(revision 7057)
+++ common.ent	(working copy)
@@ -10,8 +10,8 @@
 
 <!-- volatile information -->
 
-<!ENTITY number-of-pkgs "9000">
-<!ENTITY number-of-maintainers "900">
+<!ENTITY number-of-pkgs "12000">
+<!ENTITY number-of-maintainers "1000">
 
 <!ENTITY number-of-arches "12">
 
