--- /usr/sbin/extlinux-update.orig	2012-01-27 21:42:52.658542144 +0100
+++ /usr/sbin/extlinux-update	2012-01-27 21:40:15.458545094 +0100
@@ -148,7 +148,7 @@ _CONFIG="\
 "
 
 # Find linux versions
-_VERSIONS="$(cd /boot && ls vmlinuz-* | grep -v .dpkg-tmp | sed -e 's|vmlinuz-||g' | sort -r)"
+_VERSIONS="$(cd /boot && ls vmlinuz-* | grep -v .dpkg-tmp | sed -e 's|vmlinuz-||g' | sort -rn)"
 
 if [ "$(stat --printf %d /)" = "$(stat --printf %d /boot)" ]
 then

