--- aide_fixperms	2007-10-23 14:54:34.000000000 +0200
+++ aide_fixperms.new	2007-10-23 14:53:57.000000000 +0200
@@ -3,7 +3,7 @@
 PACKAGES="aide aide-config-zg2"
 
 for package in $PACKAGES; do
-  for file in $(find debian/$package/usr/share/$package/config/aide -type f -print 2>/dev/null); do
+  for file in $(find debian/aide-common/usr/share/$package/config/aide -type f -print 2>/dev/null); do
     if < $file head -n 1 | grep -q '^#!'; then
       chmod 755 $file
     fi
