--- ./debian/control.orig	2011-05-29 13:19:53.000000000 +0200
+++ ./debian/control	2011-05-29 13:20:30.000000000 +0200
@@ -7,7 +7,6 @@
 Package: base-files
 Provides: base
 Architecture: any
-Pre-Depends: awk
 Essential: yes
 Priority: required
 Replaces: base, miscutils, dpkg (<= 1.15.0)
--- ./debian/current-md5sums.orig	2011-05-29 13:21:48.000000000 +0200
+++ ./debian/current-md5sums	2011-05-29 13:23:33.000000000 +0200
@@ -1,3 +1,3 @@
 #!/bin/sh
-sed -e "s&#OSNAME#&GNU/Linux&" etc/motd | awk 'NR > 2' | md5sum
-sed -e "s&#OSNAME#&GNU/Hurd&"  etc/motd | awk 'NR > 2' | md5sum
+sed -e "s&#OSNAME#&GNU/Linux&" etc/motd | tail -n+3 | md5sum
+sed -e "s&#OSNAME#&GNU/Hurd&"  etc/motd | tail -n+3 | md5sum
--- ./debian/postinst.in.orig	2011-05-29 13:23:53.000000000 +0200
+++ ./debian/postinst.in	2011-05-29 13:30:15.000000000 +0200
@@ -104,12 +104,12 @@
 
 if [ "$1" = "configure" ] && [ "$2" != "" ]; then
   if [ -f /etc/motd ]; then
-    oldmd=`awk 'NR > 2' /etc/motd | md5sum | awk '{print $1}'`
-    newmd=`awk 'NR > 2' /usr/share/base-files/motd | md5sum | awk '{print $1}'`
+    oldmd=`tail -n+3 /etc/motd | md5sum | cut -d' ' -f1`
+    newmd=`tail -n+3 /usr/share/base-files/motd | md5sum | cut -d' ' -f1`
     if [ "$oldmd" != "$newmd" ]; then
       if grep -q "$oldmd" /usr/share/base-files/motd.md5sums; then
-        awk 'NR <= 2' /etc/motd > /etc/motd.new
-        awk 'NR > 2' /usr/share/base-files/motd >> /etc/motd.new
+        head -n2 /etc/motd > /etc/motd.new
+        tail -n+3 /usr/share/base-files/motd >> /etc/motd.new
         mv /etc/motd /etc/motd.old
         mv /etc/motd.new /etc/motd
       fi
