From 520effe3ab6cce664bb8b95d60a541849fd1c391 Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Thu, 22 Dec 2016 18:43:37 +0000
Subject: [PATCH] dgit-maint-merge(7): Use debian/source/patch-header

Signed-off-by: Sean Whitton <spwhitton@spwhitton.name>
---
 dgit-maint-merge.7.pod | 6 +++++-
 1 file changed, 5 insertions(+), 1 deletion(-)

diff --git a/dgit-maint-merge.7.pod b/dgit-maint-merge.7.pod
index dc5bfa3..245be4c 100644
--- a/dgit-maint-merge.7.pod
+++ b/dgit-maint-merge.7.pod
@@ -238,7 +238,7 @@ source:
 You don't need to create this file if you are using the version 1.0
 source package format.
 
-=head2 Sample text for README.source
+=head2 Sample text for debian/source/patch-header
 
 It is a good idea to explain how a user can obtain a break down of the
 changes to the upstream source:
@@ -263,6 +263,10 @@ See dgit(1), dgit(7) and dgit-maint-merge(7) for more information.
 
 =back
 
+Alternatively, this text could be added to README.source. However,
+this might distract from more important information present in the
+latter file.
+
 =head1 BUILDING AND UPLOADING
 
 Use B<dgit build>, B<dgit sbuild>, B<dgit build-source>, and B<dgit
-- 
2.10.2

