diff -wru apt-file-2.0.3/apt-file apt-file-2.0.3-he/apt-file
--- apt-file-2.0.3/apt-file	2003-05-16 15:14:31.000000000 +0200
+++ apt-file-2.0.3-he/apt-file	2003-10-21 14:55:01.000000000 +0200
@@ -89,10 +89,11 @@
 	my $line = $_;
 	debug "got $line";
 	$line =~ s/([^\/])\#.*$/$1/o;
+	$line =~ s/^(\S+\s+)\[\S+\]/$1/o;
 	$line =~ s/\s+/ /go;
 
 	# CDROM entry
-	if (@tmp = m/^([^\[]*)\[([^\]]*)\](.*)$/o) {
+	if (@tmp = $line =~ m/^([^\[]*)\[([^\]]*)\](.*)$/o) {
 	    $tmp[1] =~ s/ /_/g;
 	    $line = $tmp[0].'['.$tmp[1].']'.$tmp[2];
 	}
Only in apt-file-2.0.3-he: .apt-file.swp
