From 616b638fbf4b2a006c90d9954cce0956d7871eb9 Mon Sep 17 00:00:00 2001
From: Ingo Saitz <ingo@hannover.ccc.de>
Date: Wed, 11 May 2022 08:44:36 +0200
Subject: [PATCH 1/2] skip arch part of package, multiarch workaround

---
 src/vercmp.c | 10 +++++++++-
 1 file changed, 9 insertions(+), 1 deletion(-)

diff --git a/src/vercmp.c b/src/vercmp.c
index 76bd94e..3fba5ca 100644
--- a/src/vercmp.c
+++ b/src/vercmp.c
@@ -168,7 +168,7 @@ const char *parsedependency(const char *string, struct pkg_version *pv, enum dep
     /* Extract package name. */
     {
       const char *namestart = p;
-      while (*p && !cisspace(*p) && *p != '(' && *p != ',' && *p != '|') {
+      while (*p && !cisspace(*p) && *p != '(' && *p != ',' && *p != '|' && *p != ':') {
         p++;
       }
 
@@ -179,6 +179,14 @@ const char *parsedependency(const char *string, struct pkg_version *pv, enum dep
         pv->name = NULL;
     }
 
+    /* skip arch part */
+    if (*p == ':') {
+      p++;
+      while (*p && !cisspace(*p) && *p != '(' && *p != ',' && *p != '|') {
+        p++;
+      }
+    }
+
     /* skip whitespace after packagename */
     while (cisspace(*p)) p++;
     if (*p == '(') {                  /* if we have a versioned relation */
-- 
2.36.1

