diff -ur atftp-0.7.dfsg/debian/control testing/atftp-0.7.dfsg/debian/control
--- atftp-0.7.dfsg/debian/control	2007-08-18 17:30:42.000000000 -0400
+++ testing/atftp-0.7.dfsg/debian/control	2007-08-18 17:39:42.000000000 -0400
@@ -2,7 +2,7 @@
 Section: net
 Priority: extra
 Maintainer: Ludovic Drolez <ldrolez@debian.org>
-Build-Depends: libreadline5-dev, debhelper (>= 4.1.16), po-debconf, libwrap-dev, libpcre3-dev
+Build-Depends: libreadline5-dev, debhelper (>= 4.1.16), po-debconf, libwrap0-dev, libpcre3-dev
 Standards-Version: 3.7.2
 
 Package: atftp
