*** build.c.orig	Thu Nov 11 21:10:03 2004
--- build.c	Wed Nov 17 09:00:10 2004
***************
*** 286,294 ****
          strcat(controlfile, "/");
          strcat(controlfile, conffilename);
          if (lstat(controlfile,&controlstab)) {
!           if (errno == ENOENT)
              ohshit(_("conffile `%.250s' does not appear in package"),conffilename);
!           else
              ohshite(_("conffile `%.250s' is not stattable"),conffilename);
          } else if (!S_ISREG(controlstab.st_mode)) {
            fprintf(stderr, _("warning, conffile `%s'"
--- 286,296 ----
          strcat(controlfile, "/");
          strcat(controlfile, conffilename);
          if (lstat(controlfile,&controlstab)) {
!           if (errno == ENOENT){
! 	    if(n>1 && conffilename[n-2]==' ') fprintf(stderr, _("warning, "
! 	  	"conffile filename `%s' contains trailing white spaces\n"), conffilename);
              ohshit(_("conffile `%.250s' does not appear in package"),conffilename);
! 	  }else
              ohshite(_("conffile `%.250s' is not stattable"),conffilename);
          } else if (!S_ISREG(controlstab.st_mode)) {
            fprintf(stderr, _("warning, conffile `%s'"
