diff -Nur gtkboard-0.11pre0_old/debian/gtkboard.doc-base gtkboard-0.11pre0/debian/gtkboard.doc-base
--- gtkboard-0.11pre0_old/debian/gtkboard.doc-base	2007-05-08 20:58:00.000000000 +0200
+++ gtkboard-0.11pre0/debian/gtkboard.doc-base	2007-05-08 22:12:19.000000000 +0200
@@ -4,15 +4,7 @@
 Abstract: This manual describes what gtkboard is
  and how it can be used to play games.
 Section: Games/Board
+
 Format: HTML
 Index: /usr/share/doc/gtkboard/html/index.html
 Files: /usr/share/doc/gtkboard/html/*.html
-
-Document: gtkboard-dev
-Title: gtkboard Dev Manual
-Author: Arvind Narayanan
-Abstract: This manual describes how to add new games to gtkboard.
-Section: Games/Board
-Format: HTML
-Index: /usr/share/doc/gtkboard/html/devel/index.html
-Files: /usr/share/doc/gtkboard/html/devel/*.html
diff -Nur gtkboard-0.11pre0_old/debian/gtkboard.doc-base.gtkboard-dev gtkboard-0.11pre0/debian/gtkboard.doc-base.gtkboard-dev
--- gtkboard-0.11pre0_old/debian/gtkboard.doc-base.gtkboard-dev	1970-01-01 01:00:00.000000000 +0100
+++ gtkboard-0.11pre0/debian/gtkboard.doc-base.gtkboard-dev	2007-05-08 22:12:09.000000000 +0200
@@ -0,0 +1,9 @@
+Document: gtkboard-dev
+Title: gtkboard Dev Manual
+Author: Arvind Narayanan
+Abstract: This manual describes how to add new games to gtkboard.
+Section: Games/Board
+
+Format: HTML
+Index: /usr/share/doc/gtkboard/html/devel/index.html
+Files: /usr/share/doc/gtkboard/html/devel/*.html
