Index: backupninja-1.0~rc1/handlers/mysql.in
===================================================================
--- backupninja-1.0~rc1.orig/handlers/mysql.in	2012-05-15 17:21:41.000000000 +0200
+++ backupninja-1.0~rc1/handlers/mysql.in	2012-05-19 20:59:02.417874223 +0200
@@ -256,10 +256,11 @@
    for db in $databases
    do
       DUMP_BASE="$MYSQLDUMP $defaultsfile $sqldumpoptions"
-      if [ "$db" = "information_schema" ] || [ "$db" = "performance_schema" ]
-      then
-         DUMP_BASE="${DUMP_BASE} --skip-lock-tables"
-      fi
+
+      case "$db" in
+        information_schema) DUMP_BASE="${DUMP_BASE} --skip-lock-tables" ;;
+        performance_schema) DUMP_BASE="${DUMP_BASE} --skip-lock-tables --skip-events" ;;
+      esac
 
       # Dumping structure and data
       DUMP="$DUMP_BASE $ignore $db"
