--- passdev.c	2009-11-21 22:34:29.000000000 +0100
+++ passdev.c	2009-11-21 22:42:24.000000000 +0100
@@ -53,7 +53,7 @@
 	pid_t pid;
 	pid_t wpid;
 	int status;
-	char *fstypes[] = { "ext3", "ext2", "vfat", "reiserfs", "xfs", "isofs", "udf" };
+	char *fstypes[] = { "btrfs", "ext4", "ext3", "ext2", "xfs", "jfs", "reiserfs", "ntfs", "vfat", "udf", "iso9660", "ufs" };
 	int fsindex;
 
 	if (!device || !dir)
@@ -88,7 +88,7 @@
 			open("/dev/null", O_WRONLY, 0);
 			execl("/bin/mount", "/bin/mount", "-n", "-t",
 			      fstypes[fsindex],
-			      /*"ext3,ext2,vfat,reiserfs,xfs,isofs,udf",*/
+			      /*"btrfs,ext4,ext3,ext2,xfs,jfs,reiserfs,ntfs,vfat,udf,iso9660,ufs"*/
 			      "-o", "noatime,nodiratime,nodev,noexec,nosuid,ro",
 			      device, dir, (char *)NULL);
 
