From ab1fe743df8e8db7f843ed3d12c19af5f6d7c592 Mon Sep 17 00:00:00 2001
From: Osamu Aoki <osamu@debian.org>
Date: Mon, 22 Jun 2020 10:21:30 +0900
Subject: [PATCH 2/2] Append extra album name text to the album name

Signed-off-by: Osamu Aoki <osamu@debian.org>
---
 abcde | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/abcde b/abcde
index ff143ac..689867e 100755
--- a/abcde
+++ b/abcde
@@ -39,6 +39,8 @@ echo "-g     Use \"lame --nogap\" for MP3 encoding. Disables low disk and pipes
 echo "-G     Get album art by using the 'getalbumart' action"
 echo "-h     This help information"
 #echo "-i    Tag files while encoding, when possible (local only) -NWY-"
+echo "-I <extra_album_name>"
+echo "       Append extra album name text to the album name before munge"
 echo "-j <#> Number of encoder processes to run at once (localhost)"
 echo "-k     Keep the wav tracks for later use"
 echo "-l     Use low disk space algorithm"
@@ -4144,7 +4146,7 @@ case "$EXTRAVERBOSE" in
 esac
 
 # Parse command line options
-while getopts 1a:bBc:C:d:DefgGhj:klLmMnNo:pPQ:r:s:S:t:T:UvVxX:w:W:z opt ; do
+while getopts 1a:bBc:C:d:DefgGhI:j:klLmMnNo:pPQ:r:s:S:t:T:UvVxX:w:W:z opt ; do
 	case "$opt" in
 		1) ONETRACK=y ;;
 		a) ACTIONS="$OPTARG" ;;
@@ -4168,6 +4170,7 @@ while getopts 1a:bBc:C:d:DefgGhj:klLmMnNo:pPQ:r:s:S:t:T:UvVxX:w:W:z opt ; do
 		g) NOGAP=y ;;
 		G) GETALBUMART=y ;;
 		i) INLINETAG=y ;;
+		I) DALBUMTAIL="$OPTARG" ;;
 		j) MAXPROCS="$OPTARG" ;;
 		k) KEEPWAVS=y ;;
 		l) LOWDISK=y ;;
@@ -5202,6 +5205,8 @@ if [ "$DOCDDB" = "y" ]; then
 	do_cddbedit
 
 	eval "$($CDDBTOOL parse "$CDDBDATA")"
+	# Append extra album name text to the album name before munge
+	DALBUM="${DALBUM}${DALBUMTAIL}"
 fi
 
 # Before reading tracks, we set the speed of the device
-- 
2.20.1

