--- debspawn-0.4.1.orig/debspawn/osbase.py
+++ debspawn-0.4.1/debspawn/osbase.py
@@ -56,9 +56,9 @@ class OSBase:
 
     def _make_name(self):
         if not self._arch:
-            out, _, ret = safe_run(['dpkg-architecture', '-qDEB_HOST_ARCH'])
+            out, _, ret = safe_run(['dpkg', '--print-architecture'])
             if ret != 0:
-                raise Exception('Running dpkg-architecture failed: {}'.format(out))
+                raise Exception('Running dpkg --print-architecture failed: {}'.format(out))
 
             self._arch = out.strip()
         if self._variant:
--- debspawn-0.4.1.orig/tests/conftest.py
+++ debspawn-0.4.1/tests/conftest.py
@@ -70,12 +70,12 @@ def build_arch():
     '''
     from debspawn.utils.command import safe_run
 
-    out, _, ret = safe_run(['dpkg-architecture', '-q', 'DEB_BUILD_ARCH'])
+    out, _, ret = safe_run(['dpkg', '--print-architecture'])
     assert ret == 0
 
     arch = out.strip()
     if not arch:
-        arch = 'amd64'  # assume arm64 as default
+        arch = 'amd64'  # assume amd64 as default
 
     return arch
 
