--- a/ddclient	2013-11-01 08:27:21.000000000 +0100
+++ b/ddclient	2013-11-01 08:32:02.723299106 +0100
@@ -1783,14 +1783,14 @@
 ## load_sha1_support
 ######################################################################
 sub load_sha1_support {
-    my $sha1_loaded = eval {require Digest::SHA1};
+    my $sha1_loaded = eval {require Digest::SHA};
     unless ($sha1_loaded) {
         fatal(<<"EOM");
-Error loading the Perl module Digest::SHA1 needed for freedns update.
-On Debian, the package libdigest-sha1-perl must be installed.
+Error loading the Perl module Digest::SHA needed for freedns update.
+On Debian, the package libdigest-sha-perl must be installed.
 EOM
     }
-    import  Digest::SHA1 (qw/sha1_hex/);
+    import  Digest::SHA (qw/sha1_hex/);
 }
 ######################################################################
 ## geturl
