--- /usr/bin/mk-build-deps	2012-05-07 02:10:27.000000000 +0200
+++ /tmp/mk-build-deps	2012-05-31 12:42:25.390044862 +0200
@@ -373,6 +373,7 @@
     "Priority: optional\n".
     "Standards-Version: 3.7.3\n\n".
     "Package: $opts->{name}-$opts->{type}\n".
+    "Readme: /usr/share/devscripts/mk-build-deps/README.template\n".
     "Architecture: $arch\n".
     "Depends: build-essential, $opts->{depends}\n";
     print EQUIVS "Version: $opts->{version}\n" if $opts->{version};
