--- ./debian/acpi-support/usr/share/acpi-support/device-funcs.orig	2010-12-14 11:37:17.000000000 +0100
+++ ./debian/acpi-support/usr/share/acpi-support/device-funcs	2010-12-14 21:24:06.000000000 +0100
@@ -1,7 +1,7 @@
 
 DeviceConfig() {
-    manufacturer=`cat /sys/class/dmi/id/sys_vendor|sed -e 's/ *$//'`
-    model=`cat /sys/class/dmi/id/product_name|sed -e 's/ *$//'`
-    version=`cat /sys/class/dmi/id/product_version|sed -e 's/ *$//'`
-    bios_version=`cat /sys/class/dmi/id/bios_version|sed -e 's/ *$//'`
+    read manufacturer </sys/class/dmi/id/sys_vendor
+    read model </sys/class/dmi/id/product_name
+    read version </sys/class/dmi/id/product_version
+    read bios_version </sys/class/dmi/id/bios_version
 }
