--- a/brightd.c
+++ b/brightd.c
@@ -183,16 +183,16 @@
  */
 int isOnAC() /*{{{*/
 {
-	char line[255];
-	FILE *ac = fopen("/proc/acpi/ac_adapter/AC/state", "r");
+	int ac_state;
+	FILE *ac = fopen("/sys/class/power_supply/AC/online", "r");
 	if(!ac) {
 		/* Ignore this */
 		return 0;
 	}
-	fgets(line, 255, ac);
+	ac_state = fgetc(ac);
 	fclose(ac);
 
-	return strstr(line, "on-line") != NULL;
+	return ac_state;
 } /*}}}*/
 
 /*
