--- /usr/bin/dh_auto_configure.orig	2009-05-04 11:48:43.000000000 -0700
+++ /usr/bin/dh_auto_configure	2009-05-09 19:55:16.000000000 -0700
@@ -88,7 +88,7 @@
 	# If set to a true value then MakeMaker's prompt function will
 	# # always return the default without waiting for user input.
 	$ENV{PERL_MM_USE_DEFAULT}=1;
-	doit("perl", "Makefile.PL", "INSTALLDIRS=vendor", @{$dh{U_PARAMS}});
+	doit("perl", "Makefile.PL", "INSTALLDIRS=vendor", "create_packlist=0", @{$dh{U_PARAMS}});
 }
 elsif (-e "Build.PL") {
 	$ENV{PERL_MM_USE_DEFAULT}=1; # Module::Build can also use this.
