Index: english/doc/devel-manuals.wml
===================================================================
RCS file: /cvs/webwml/webwml/english/doc/devel-manuals.wml,v
retrieving revision 1.61
diff -u -r1.61 devel-manuals.wml
--- english/doc/devel-manuals.wml	18 Oct 2011 15:01:08 -0000	1.61
+++ english/doc/devel-manuals.wml	26 Dec 2011 23:30:44 -0000
@@ -48,8 +48,6 @@
     [<a href="packaging-manuals/menu-policy/menu-policy.txt.gz">plain text</a>] 
     <li><a href="packaging-manuals/perl-policy/">Perl policy</a>
     [<a href="packaging-manuals/perl-policy/perl-policy.txt.gz">plain text</a>] 
-    <li><a href="packaging-manuals/mime-policy/">MIME policy</a>
-    [<a href="packaging-manuals/mime-policy/mime-policy.txt.gz">plain text</a>] 
     <li><a href="packaging-manuals/debconf_specification.html">debconf specification</a>
     <li><a href="packaging-manuals/debian-emacs-policy">Emacsen policy</a>
     <li><a href="packaging-manuals/java-policy/">Java policy</a>
Index: english/devel/index.wml
===================================================================
RCS file: /cvs/webwml/webwml/english/devel/index.wml,v
retrieving revision 1.229
diff -u -r1.229 index.wml
--- english/devel/index.wml	29 Oct 2011 15:46:40 -0000	1.229
+++ english/devel/index.wml	26 Dec 2011 23:30:44 -0000
@@ -148,9 +148,6 @@
             have a menu entry registered.
             Check the <a href="$(DOC)/packaging-manuals/menu.html/">menu
             system documentation</a>, too.</li>
-        <li><a href="$(DOC)/packaging-manuals/mime-policy/">MIME support</a>
-        <br />MIME type handlers are used by other programs to view, edit or
-            display MIME types they don't support directly.</li>
         <li><a href="$(DOC)/packaging-manuals/debian-emacs-policy">Emacs policy</a>
         <br />The packages related to Emacs are expected to abide by their
             own sub-policy document.</li>
