--- a/dkms
+++ b/dkms
@@ -233,6 +233,9 @@
 do_depmod()
 {
     # $1 = kernel version
+    if [ "${current_os}" != "Linux" ] ; then
+        return
+    fi
     if [[ -f /boot/System.map-$1 ]]; then
 	/sbin/depmod -a "$1" -F "/boot/System.map-$1"
     else
@@ -377,7 +380,7 @@
     local orig_location="$1"
     [[ ${addon_modules_dir} ]] && echo "/${addon_modules_dir}" && return
 
-    if [ "$uname_s" = "GNU/kFreeBSD" ] ; then
+    if [ "$current_os" = "GNU/kFreeBSD" ] ; then
 	# Does not support subdirs, regardless of distribution
 	echo "" && return
     fi
--- a/kernel_postinst.d_dkms
+++ b/kernel_postinst.d_dkms
@@ -39,11 +39,6 @@
     /usr/lib/dkms/dkms_autoinstaller start $inst_kern >&2
 fi
 
-if [ ! -e /lib/modules/$inst_kern/build/include ] ; then
-    echo "dkms: WARNING: linux headers are missing, which may explain the above failures." >&2
-    echo "      please install the linux-headers-$inst_kern package to fix this." >&2
-fi
-
 if ! _check_kernel_dir $inst_kern ; then
     echo "dkms: WARNING: $kernel headers are missing, which may explain the above failures." >&2
     echo "      please install the $header_pkg package to fix this." >&2
