diff -Nru bash-completion-2.1/debian/changelog bash-completion-2.1/debian/changelog
--- bash-completion-2.1/debian/changelog	2014-03-22 21:57:44.000000000 +0100
+++ bash-completion-2.1/debian/changelog	2015-05-06 20:32:11.000000000 +0200
@@ -1,3 +1,11 @@
+bash-completion (1:2.1-4.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Remove dmesg and renice completions which will instead be shipped
+    from src:util-linux in their respective packages (Closes: #755986)
+
+ -- Andreas Henriksson <andreas@fatal.se>  Wed, 06 May 2015 20:32:10 +0200
+
 bash-completion (1:2.1-4) unstable; urgency=medium
 
   * Fix dont_return_from_sourced_script.patch: typo during the patch
diff -Nru bash-completion-2.1/debian/rules bash-completion-2.1/debian/rules
--- bash-completion-2.1/debian/rules	2014-03-22 21:57:44.000000000 +0100
+++ bash-completion-2.1/debian/rules	2015-05-06 20:30:45.000000000 +0200
@@ -5,7 +5,7 @@
 #export DH_VERBOSE=1
 
 VERSION=$(shell parsechangelog | grep ^Version | awk -F": " '{print $$2}' | cut -d"-" -f1)
-REMOVE=nmcli hwclock ionice mock modules rtcwake
+REMOVE=nmcli hwclock ionice mock modules rtcwake dmesg renice
 
 override_dh_auto_configure:
 	dh_auto_configure
