diff -Nru ca-certificates-20090814+nmu1/debian/changelog ca-certificates-20090814+nmu2/debian/changelog
--- ca-certificates-20090814+nmu1/debian/changelog	2010-07-31 14:36:56.000000000 -0300
+++ ca-certificates-20090814+nmu2/debian/changelog	2010-08-13 20:51:07.000000000 -0300
@@ -1,3 +1,11 @@
+ca-certificates (20090814+nmu2) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Fixes buggy shell functions included in the postinst script.
+    (Closes: #591607)
+
+ -- Maximiliano Curia <maxy@debian.org>  Fri, 13 Aug 2010 20:16:21 -0300
+
 ca-certificates (20090814+nmu1) unstable; urgency=low
 
   * Non-maintainer upload.
diff -Nru ca-certificates-20090814+nmu1/debian/postinst ca-certificates-20090814+nmu2/debian/postinst
--- ca-certificates-20090814+nmu1/debian/postinst	2010-07-31 14:36:01.000000000 -0300
+++ ca-certificates-20090814+nmu2/debian/postinst	2010-08-13 20:51:14.000000000 -0300
@@ -21,13 +21,13 @@
 #     `abort-remove' or `abort-deconfigure'.
 
 each_value() {
- echo "$l" |tr ',' '\n' | sed -e 's/^[[:space:]]*//' 
+ echo "$1" |tr ',' '\n' | sed -e 's/^[[:space:]]*//' 
 }
 
 memberp() {
  m="$1"
  l="$2"
- each_value "$1" | grep -q "^$m\$"
+ each_value "$l" | grep -q "^$m\$"
 }
 
 delca() {
