diff -Nru etckeeper-0.37/debian/postinst etckeeper-0.37ubuntu1/debian/postinst
--- etckeeper-0.37/debian/postinst	2009-05-27 21:26:53.000000000 +0200
+++ etckeeper-0.37ubuntu1/debian/postinst	2009-06-16 09:56:57.000000000 +0200
@@ -77,7 +77,11 @@
 		# Fresh install.
 		. /etc/etckeeper/etckeeper.conf || true
 		if [ -n "$VCS" ] && [ -x "`which $VCS 2>/dev/null`" ]; then
-			etckeeper init || echo "etckeeper init failed; run it by hand" >&2
+			if etckeeper init; then
+				etckeeper commit "Initial commit"
+			else
+				echo "etckeeper init failed; run it by hand" >&2
+			fi
 		else
 			echo "etckeeper init not ran as $VCS is not installed" >&2
 		fi
