diff -u dictconv-0.2/debian/control dictconv-0.2/debian/control
--- dictconv-0.2/debian/control
+++ dictconv-0.2/debian/control
@@ -11,7 +11,7 @@
 Depends: ${shlibs:Depends}, ${misc:Depends}
 Recommends: dictzip
 Description: convert a dictionary file type in another dictionary file type
- Dictconv is a small program to convert a dictionay file type in another
+ Dictconv is a small program to convert a dictionary file type in another
  dictionary file type. Currently, it supports converting from Babylon
  glossaries, Freedict dictionaries, Sdictionary dictionaries and Stardict
  dictionaries to DICT dictionaries, plain text dictionaries and StarDict
diff -u dictconv-0.2/debian/changelog dictconv-0.2/debian/changelog
--- dictconv-0.2/debian/changelog
+++ dictconv-0.2/debian/changelog
@@ -1,3 +1,11 @@
+dictconv (0.2-7.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * debian/compat: 5 -> 7. (Closes: #965491)
+  * Spelling fix in the package description. (Closes: #581630)
+
+ -- Adrian Bunk <bunk@debian.org>  Wed, 19 Jan 2022 16:23:08 +0200
+
 dictconv (0.2-7) unstable; urgency=low
 
   * debian/rules: added simple-patchsys include.
diff -u dictconv-0.2/debian/compat dictconv-0.2/debian/compat
--- dictconv-0.2/debian/compat
+++ dictconv-0.2/debian/compat
@@ -1 +1 @@
-5
+7
