From cfa33feb85b9352686d35515bdad14725357925e Mon Sep 17 00:00:00 2001
From: Peter Pentchev <roam@ringlet.net>
Date: Wed, 9 Mar 2011 15:05:40 +0200
Subject: [PATCH 04/15] Expand the long description of the alex4-data package.

---
 debian/changelog |    1 +
 debian/control   |    6 +++++-
 2 files changed, 6 insertions(+), 1 deletions(-)

diff --git a/debian/changelog b/debian/changelog
index 9226668..05bdf06 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -12,6 +12,7 @@ alex4 (1.1-4) UNRELEASED; urgency=low
       depends on
   * Convert to the 3.0 (quilt) source format.
   * Add misc:Depends to the alex4-data package just in case.
+  * Expand the alex4-data package's long description.
 
  -- Peter Pentchev <roam@ringlet.net>  Wed, 09 Mar 2011 14:14:04 +0200
 
diff --git a/debian/control b/debian/control
index 61c9dc5..c8704e0 100644
--- a/debian/control
+++ b/debian/control
@@ -24,5 +24,9 @@ Package: alex4-data
 Architecture: all
 Depends: ${misc:Depends}
 Description: Alex the Allegator 4 - game data
- This package contains data files required by the game Alex the Allegator 4.
+ Guide Alex the Allegator through the jungle in order to save his
+ girlfriend Lola from evil humans who want to make a pair of shoes out
+ of her. Plenty of classic platforming in four nice colors guaranteed!
+ .
+ This package contains the game data files.
  
-- 
1.7.2.3

