diff --git a/t/12_merge.t b/t/12_merge.t
index c654359..dd40db9 100644
--- a/t/12_merge.t
+++ b/t/12_merge.t
@@ -1,6 +1,6 @@
 # -*- mode: cperl;-*-
 
-use Test::More tests => 35;
+use Test::More tests => 38;
 
 use warnings;
 use strict;
@@ -141,7 +141,7 @@ send_message(to => 'control@bugs.something',
 			],
 	     body => <<'EOF') or fail 'message to control@bugs.something failed';
 debug 10
-clone 2 -1 -2 -3 -4 -5 -6
+clone 2 -1 -2 -3 -4 -5 -6 -7 -8
 retitle 2 foo
 owner 2 bar@baz.com
 submitter 2 fleb@bleh.com
@@ -158,6 +158,10 @@ fixed -4 1.2-3
 found -4 1.2-1
 found -5 1.2-5
 fixed -5 1.2-6
+block -7 by -1
+block -7 by -2
+block -8 by -2
+block -8 by -1
 thanks
 EOF
 	;
@@ -193,6 +197,12 @@ test_control_commands(forcemerge   => {command => 'forcemerge',
 				       status_value => '8',
 				       bug => '7',
 				      },
+		      merge        => {command => 'merge',
+				       value   => '9 10',
+				       status_key => 'mergedwith',
+				       status_value => '10',
+				       bug => '9',
+				      },
 		     );
 
 
