--- apt-cacher	2017-04-24 19:38:26.000000000 +0100
+++ apt-cacher.nick	2018-01-30 18:40:16.983772750 +0000
@@ -1148,8 +1148,8 @@
 <tr bgcolor="#cccccc">
 <td>
 Usage:
-<p>Edit /etc/apt/apt.conf to include the configuration
-<blockquote>Acquire::http::proxy=http://$hosturl</blockquote>
+<p>Add a configuration fragment under /etc/apt/apt.conf.d/ containing the following:
+<blockquote>Acquire::http::proxy "http://$hosturl";</blockquote>
 Alternatively, edit /etc/apt/sources.list so all your HTTP sources are prepended
 with the address of your apt-cacher machine and the port, like this:
 <blockquote>deb&nbsp;http://example.debian.org/debian&nbsp;unstable&nbsp;main&nbsp;contrib&nbsp;non-free</blockquote>
