diff -Npru netcfg-1.60.orig/netcfg-common.c netcfg-1.60/netcfg-common.c
--- netcfg-1.60.orig/netcfg-common.c	2011-02-05 20:42:06.000000000 -0500
+++ netcfg-1.60/netcfg-common.c	2011-02-06 21:44:10.611927865 -0500
@@ -1048,6 +1048,14 @@ int netcfg_detect_link(struct debconfcli
             break;
         }
         if (ethtool_lite (if_name) == 1) /* ethtool-lite's CONNECTED */ {
+            if (gateway.s_addr) {
+                char arping[256];
+                char s_gateway[INET_ADDRSTRLEN];
+
+                inet_ntop (AF_INET, &gateway, s_gateway, sizeof(s_gateway));
+                sprintf(arping, "arping -w 45 -f -I %s %s", if_name, s_gateway);
+                di_exec_shell_log(arping);
+            }
             debconf_progress_set(client, NETCFG_LINK_WAIT_TIME * 4);
             rv = 1;
             break;
