diff --git a/apt-private/private-install.h b/apt-private/private-install.h
index 439c897..54ffabf 100644
--- a/apt-private/private-install.h
+++ b/apt-private/private-install.h
@@ -106,6 +106,8 @@ struct TryToInstall {
       for (std::list<std::pair<pkgCache::VerIterator, pkgCache::VerIterator> >::const_iterator c = Changed.begin();
 	   c != Changed.end(); ++c)
       {
+         if (c->first == NULL)
+            continue;
 	 if (c->second.end() == true)
 	    ioprintf(out, _("Selected version '%s' (%s) for '%s'\n"),
 		     c->first.VerStr(), c->first.RelStr().c_str(), c->first.ParentPkg().FullName(true).c_str());
