diff -ru coreutils-7.4.pristine/debian/control coreutils-7.4/debian/control
--- coreutils-7.4.pristine/debian/control	2009-07-01 18:58:03.000000000 +0100
+++ coreutils-7.4/debian/control	2009-07-01 23:56:38.000000000 +0100
@@ -10,17 +10,20 @@
 Pre-Depends: ${shlibs:Depends}
 Essential: yes
 Replaces: mktemp
-Description: The GNU core utilities
- This package contains the essential basic system utilities.
+Description: GNU core utilities
+ This package contains the basic file, shell and text manipulation
+ utilities which are expected to exist on every operating system.
  .
  Specifically, this package includes: 
- basename cat chgrp chmod chown chroot cksum comm cp csplit cut date dd df dir
- dircolors dirname du echo env expand expr factor false fmt fold groups head
- hostid id install join link ln logname ls md5sum mkdir mkfifo mknod mktemp
- mv nice nl nohup od paste pathchk pinky pr printenv printf ptx pwd readlink 
- rm rmdir sha1sum seq shred sleep sort split stat stty sum sync tac tail tee 
- test touch tr true tsort tty uname unexpand uniq unlink users vdir wc who 
- whoami yes 
+ arch base64 basename cat chcon chgrp chmod chown chroot cksum comm cp
+ csplit cut date dd df dir dircolors dirname du echo env expand expr
+ factor false flock fmt fold groups head hostid id install join link ln
+ logname ls md5sum mkdir mkfifo mknod mktemp mv nice nl nohup od paste
+ pathchk pinky pr printenv printf ptx pwd readlink rm rmdir runcon
+ sha*sum seq shred sleep sort split stat stty sum sync tac tail tee test
+ touch tr true truncate tsort tty uname unexpand uniq unlink users vdir wc
+ who whoami yes
+Homepage: http://gnu.org/software/coreutils
 
 Package: mktemp
 Pre-Depends: coreutils (>= 7.4-1)
