diff -up -ru coreutils-8.13.orig/tests/misc/seq-long-double coreutils-8.13/tests/misc/seq-long-double
--- coreutils-8.13.orig/tests/misc/seq-long-double	2011-07-28 12:38:27.000000000 +0200
+++ coreutils-8.13/tests/misc/seq-long-double	2011-09-21 09:40:41.000000000 +0200
@@ -33,7 +33,7 @@ int foo[sizeof (long double) - sizeof (d
 "run this test only with glibc"
 #endif
 EOF
-$CC -c long.c \
+$CC $CFLAGS -c long.c \
   || skip_ \
      'this test runs only on systems with glibc and long double != double'
 
