Index: pkgs.dbk
===================================================================
--- pkgs.dbk	(revision 9223)
+++ pkgs.dbk	(working copy)
@@ -1309,7 +1309,10 @@
 <literal>testing</literal> directly.  Rather, they will be removed
 automatically after the package has been removed from
 <literal>unstable</literal> and no package in
-<literal>testing</literal> depends on it.
+<literal>testing</literal> depends on it. (Removals from
+<literal>testing</literal> are possible though by filing a removal bug report
+against the <systemitem role="package">&release-debian-org;</systemitem>
+pseudo-package. See the section <xref linkend="removals"/>.)
 </para>
 <para>
 There is one exception when an explicit removal request is not necessary: If a
Index: common.ent
===================================================================
--- common.ent	(revision 9223)
+++ common.ent	(working copy)
@@ -27,6 +27,7 @@
 <!ENTITY master-host "master.debian.org">
 <!ENTITY www-debian-org "www.debian.org">
 <!ENTITY ftp-debian-org "ftp.debian.org">
+<!ENTITY release-debian-org "release.debian.org">
 <!ENTITY lists-host "lists.debian.org">
 <!ENTITY archive-host "archive.debian.org">
 <!ENTITY keyserver-host "keyring.debian.org">
