diff -ru apt-0.7.13.pristine/debian/control apt-0.7.13/debian/control
--- apt-0.7.13.pristine/debian/control	2008-05-02 16:05:25.000000000 +0100
+++ apt-0.7.13/debian/control	2008-05-25 18:30:30.000000000 +0100
@@ -15,13 +15,14 @@
 Provides: ${libapt-pkg:provides}
 Suggests: aptitude | synaptic | gnome-apt | wajig, dpkg-dev, apt-doc, bzip2, lzma
 Section: admin
-Description: Advanced front-end for dpkg
- This is Debian's next generation front-end for the dpkg package manager.
- It provides the apt-get utility and APT dselect method that provides a 
- simpler, safer way to install and upgrade packages.
+Description: Debian's advanced package tool
+ This package provides command-line utilities for package management,
+ including apt-get, used for finding and fetching software upgrades,
+ and apt-cache, used for interrogating the package database. It depends
+ on dpkg as its back-end for installs, and can be used either directly
+ or via high-level interfaces such as aptitude.
  .
- APT features complete installation ordering, multiple source capability
- and several other unique features, see the Users Guide in apt-doc.
+ A User's Guide is available in the package apt-doc.
 
 Package: apt-doc
 Architecture: all
