diff -ruN cdbs-0.4.89/1/class/python-module.mk.in cdbs-0.4.90_pycompat/1/class/python-module.mk.in
--- cdbs-0.4.89/1/class/python-module.mk.in	2010-05-22 22:52:01.000000000 +0200
+++ cdbs-0.4.90_pycompat/1/class/python-module.mk.in	2010-11-23 20:32:11.390840052 +0100
@@ -37,7 +37,4 @@
 $(patsubst %,binary-install-python/%,$(DEB_PACKAGES)) :: binary-install-python/%: binary-install/%
 	dh_$(cdbs_python_system) -p$(cdbs_curpkg) $(DEB_PYTHON_PRIVATE_MODULES_DIRS) $(DEB_PYTHON_PRIVATE_MODULES_DIRS_$(cdbs_curpkg))
 
-clean::
-	$(if $(cdbs_python_pycompat_explicit)$(cdbs_python_use_xs_field)$(cdbs_python_selected_pyversions),,echo "$(cdbs_python_pycompat)" >debian/pycompat)
-
 endif
diff -ruN cdbs-0.4.89/1/class/python-vars.mk.in cdbs-0.4.90_pycompat/1/class/python-vars.mk.in
--- cdbs-0.4.89/1/class/python-vars.mk.in	2010-06-21 15:13:35.000000000 +0200
+++ cdbs-0.4.90_pycompat/1/class/python-vars.mk.in	2010-11-23 20:46:39.263023985 +0100
@@ -44,7 +44,6 @@
 
 # check python system
 cdbs_python_use_xs_field := $(shell grep -q "^XS-Python-Version:" debian/control && echo yes)
-cdbs_python_pycompat_explicit := $(shell if [ -e debian/pycompat ]; then cat debian/pycompat; fi)
 cdbs_python_selected_pyversions := $(shell if [ -e debian/pyversions ]; then cat debian/pyversions; fi)
 
 # TODO: expand recursively (drop colon) when no longer used for rules expansion
@@ -52,10 +51,6 @@
 cdbs_python_pycentral := $(filter pycentral,$(DEB_PYTHON_SYSTEM))
 cdbs_python_system = $(or $(cdbs_python_pysupport),$(cdbs_python_pycentral),$(error unsupported Python system: $(DEB_PYTHON_SYSTEM) (select either pysupport or pycentral)))
 
-cdbs_python_pycompat_default_pysupport = 2
-cdbs_python_pycompat_default_pycentral = 2
-cdbs_python_pycompat = $(or $(cdbs_python_pycompat_explicit),$(cdbs_python_pycompat_default_$(cdbs_python_system)))
-
 # warning pysupport compatibility mode
 $(if $(cdbs_python_pysupport),$(if $(cdbs_python_use_xs_field),$(warning WARNING:  Use of XS-Python-Version and XB-Python-Version fields in debian/control is deprecated with pysupport method; use debian/pyversions if you need to specify specific versions.)))
 
diff -ruN cdbs-0.4.89/debian/changelog cdbs-0.4.90_pycompat/debian/changelog
--- cdbs-0.4.89/debian/changelog	2010-09-16 11:11:03.000000000 +0200
+++ cdbs-0.4.90_pycompat/debian/changelog	2010-11-23 20:33:14.347654870 +0100
@@ -1,3 +1,9 @@
+cdbs (0.4.90) UNRELEASED; urgency=low
+
+  * No longer creates debian/pycompat files in clean rule. Closes: #424898
+
+ -- Piotr Ożarowski <piotr@debian.org>  Sat, 20 Nov 2010 22:04:42 +0100
+
 cdbs (0.4.89) unstable; urgency=low
 
   * Stop conflicting against ancient/bogus build-common.
