=== modified file 'cgi/pkgreport.cgi'
--- cgi/pkgreport.cgi	2007-09-07 12:18:44 +0000
+++ cgi/pkgreport.cgi	2007-10-09 22:56:16 +0000
@@ -1019,7 +1019,7 @@
 sub myurl {
      return html_escape(pkg_url(map {exists $param{$_}?($_,$param{$_}):()}
 			     qw(archive repeatmerged mindays maxdays),
-			     qw(version dist arch pkg src tag maint submitter)
+			     qw(version dist arch package src tag maint submitter)
 			    )
 		    );
 }

