diff -ru acpid-1.0.10.pristine/debian/control acpid-1.0.10/debian/control
--- acpid-1.0.10.pristine/debian/control	2009-10-06 15:05:46.000000000 +0100
+++ acpid-1.0.10/debian/control	2009-10-06 15:09:50.000000000 +0100
@@ -18,13 +18,13 @@
          lsb-base (>= 3.2-14),
          module-init-tools (>= 3.1-rel-2ubuntu2)
 Recommends: acpi-support-base
-Description: Utilities for using ACPI power management
+Description: Advanced Configuration and Power Interface event daemon
  Modern computers support the Advanced Configuration and Power Interface (ACPI)
  to allow intelligent power management on your system and to query battery and
  configuration status.
  .
  ACPID is a completely flexible, totally extensible daemon for delivering
- ACPI events. It listens on a file (/proc/acpi/event) and when an event
- occurs, executes programs to handle the event. The programs it executes
- are configured through a set of configuration files, which can be
- dropped into place by packages or by the admin.
+ ACPI events. It listens on a netlink interface (or on the deprecated file
+ /proc/acpi/event), and when an event occurs, executes programs to handle the
+ event. The programs it executes are configured through a set of configuration
+ files, which can be dropped into place by packages or by the admin.
