diff -ur old/dh_installdocs new/dh_installdocs
--- old/dh_installdocs	2008-01-12 09:30:11.000000000 +0100
+++ new/dh_installdocs	2008-01-12 09:38:24.000000000 +0100
@@ -137,8 +137,8 @@
 				my $dir = ($basename eq '.') ? $doc : "$doc/..";
 				my $pwd=`pwd`;
 				chomp $pwd;
-				$exclude='\\( -type f -or -type l \\)'.$exclude;
-				complex_doit("cd '$dir' && find '$basename' $exclude -exec cp --parents -dp {} $pwd/$tmp/usr/share/doc/$package \\;");
+				my $exclude2='\\( -type f -or -type l \\)'.$exclude;
+				complex_doit("cd '$dir' && find '$basename' $exclude2 -exec cp --parents -dp {} $pwd/$tmp/usr/share/doc/$package \\;");
 			}
 			else {
 				doit("cp", "-a", $doc, "$tmp/usr/share/doc/$package");
diff -ur old/dh_installexamples new/dh_installexamples
--- old/dh_installexamples	2008-01-12 09:30:34.000000000 +0100
+++ new/dh_installexamples	2008-01-12 09:38:33.000000000 +0100
@@ -90,8 +90,8 @@
 				my $dir = ($basename eq '.') ? $example : "$example/..";
 				my $pwd=`pwd`;
 				chomp $pwd;
-				$exclude = '-type f'.$exclude;
-				complex_doit("cd '$dir' && find '$basename' $exclude -exec cp --parents -dp {} $pwd/$tmp/usr/share/doc/$package/examples \\;");
+				my $exclude2 = '-type f'.$exclude;
+				complex_doit("cd '$dir' && find '$basename' $exclude2 -exec cp --parents -dp {} $pwd/$tmp/usr/share/doc/$package/examples \\;");
 			}
 			else {
 				doit("cp", "-a", $example, "$tmp/usr/share/doc/$package/examples");
