--- debian_cdrom-retriever.templates	2011-09-17 22:03:51.493323384 +0200
+++ debian_cdrom-retriever.templates.workingcopy	2011-09-17 22:05:11.825308586 +0200
@@ -5,4 +5,5 @@
 _Description: Failed to copy file from CD-ROM. Retry?
  There was a problem reading data from the CD-ROM. Please make sure it is in
  the drive. If retrying does not work, you should check the integrity of
- your CD-ROM.
+ your CD-ROM. You can use 'Check the CD-ROM(s) integrity' from the main menu
+ for that.
