diff -Naur debian-policy-3.7.2.2.orig/policy.sgml debian-policy-3.7.2.2/policy.sgml
--- debian-policy-3.7.2.2.orig/policy.sgml	2006-10-03 01:36:50.000000000 +0300
+++ debian-policy-3.7.2.2/policy.sgml	2007-03-28 04:39:46.000000000 +0300
@@ -6458,6 +6458,11 @@
 		not be stripped from the binary during installation,
 		so that debugging information may be included in the package.
 	    </item>
+	    <tag>nocheck</tag>
+	    <item>
+		This string means that any build-time test-suites should
+		not be run.
+	    </item>
 	  </taglist>
 	</p>
 
@@ -6482,6 +6487,15 @@
 ifeq (,$(findstring nostrip,$(DEB_BUILD_OPTIONS)))
 INSTALL_PROGRAM += -s
 endif
+ifeq (,$(findstring nocheck,$(DEB_BUILD_OPTIONS)))
+CHECKS := check
+endif
+
+check: build
+	...
+
+install: build $(CHECKS)
+	...
 	  </example>
 	</p>
 
