--- extract.c.orig	2016-03-07 22:44:03.421627595 +0900
+++ extract.c	2016-03-07 22:47:23.810190427 +0900
@@ -320,6 +320,7 @@
       command_init(&cmd, TAR, "tar");
       command_add_arg(&cmd, "tar");
 
+      command_add_arg(&cmd, "--sort=name");
       if ((taroption & DPKG_TAR_LIST) && (taroption & DPKG_TAR_EXTRACT))
         command_add_arg(&cmd, "-xv");
       else if (taroption & DPKG_TAR_EXTRACT)
