--- /usr/share/debootstrap/scripts/sid	2008-07-02 17:44:35.000000000 +0200
+++ sid	2008-09-12 20:37:49.000000000 +0200
@@ -50,10 +50,12 @@
         cp "$TARGET/usr/bin/md5sum.textutils" "$TARGET/usr/bin/md5sum"
     fi
 
-    if doing_variant fakechroot; then 
-        setup_devices_fakechroot
-    else
-        setup_devices
+    if [ "$ARCH" != "hurd-i386" ]; then
+        if doing_variant fakechroot; then 
+            setup_devices_fakechroot
+        else
+            setup_devices
+        fi
     fi
 
     x_feign_install () {
@@ -79,6 +81,14 @@
 }
 
 second_stage_install () {
+    if [ "$ARCH" = "hurd-i386" ]; then
+        if doing_variant fakechroot; then
+            setup_devices_fakechroot
+        else
+            setup_devices
+        fi
+    fi
+
     x_core_install () {
 	smallyes '' | in_target dpkg --force-depends --install $(debfor "$@")
     }
