diff --git a/scripts/dget.pl b/scripts/dget.pl
index b7ef84e..0e15763 100755
--- a/scripts/dget.pl
+++ b/scripts/dget.pl
@@ -312,7 +312,7 @@ sub apt_get {
     if (-f "/etc/apt/sources.list") {
 	$apt = new IO::File("/etc/apt/sources.list") or die "/etc/apt/sources.list: $!";
 	while (<$apt>) {
-	    if (/^\s*deb\s*($host_re\S+)/) {
+	    if (/^\s*deb\s*($host_re\S*)/) {
 		push @repositories, $1;
 	    }
 	}
@@ -325,7 +325,7 @@ sub apt_get {
 	$_ = "/etc/apt/sources.list.d/$_";
 	$apt = new IO::File("$_") or die "$_: $!";
 	while (<$apt>) {
-	    if (/^\s*deb\s*($host_re\S+)/) {
+	    if (/^\s*deb\s*($host_re\S*)/) {
 		push @repositories, $1;
 	    }
 	}
