From 0b7f6e4fb4833328df2c64324ed79602651b297c Mon Sep 17 00:00:00 2001
Message-Id: <0b7f6e4fb4833328df2c64324ed79602651b297c.1348334836.git.agx@sigxcpu.org>
From: =?UTF-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sat, 22 Sep 2012 19:15:17 +0200
Subject: [PATCH] Ship auvirt
Status: O
Content-Length: 439
Lines: 18

---
 debian/auditd.install |    1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/auditd.install b/debian/auditd.install
index e990e7c..1d365bd 100644
--- a/debian/auditd.install
+++ b/debian/auditd.install
@@ -11,6 +11,7 @@ sbin/autrace
 usr/bin/aulast
 usr/bin/aulastlog
 usr/bin/ausyscall
+usr/bin/auvirt
 usr/share/man/man5/audispd.conf.5
 usr/share/man/man5/auditd.conf.5
 usr/share/man/man5/ausearch-expression.5
-- 
1.7.10.4

