diff -aur audacity-2.0.3.orig/src/FFmpeg.h audacity-2.0.3/src/FFmpeg.h
--- audacity-2.0.3.orig/src/FFmpeg.h	2013-01-18 16:50:58.000000000 +0100
+++ audacity-2.0.3/src/FFmpeg.h	2013-07-23 15:36:11.900000000 +0200
@@ -353,7 +353,7 @@
    int                  m_initialchannels;               // number of channels allocated when we begin the importing. Assumes that number of channels doesn't change on the fly.
 
    int                  m_samplesize;                    // input sample size in bytes
-   SampleFormat         m_samplefmt;                     // input sample format
+   AVSampleFormat         m_samplefmt;                     // input sample format
 
    int                  m_osamplesize;                   // output sample size in bytes
    sampleFormat         m_osamplefmt;                    // output sample format
Nur in audacity-2.0.3/src: FFmpeg.h~.
