Index: debian-installer/installer/build/pkg-lists/cdrom/powerpc.cfg
===================================================================
--- debian-installer.orig/installer/build/pkg-lists/cdrom/powerpc.cfg	2014-10-31 10:46:54.285199730 +0100
+++ debian-installer/installer/build/pkg-lists/cdrom/powerpc.cfg	2014-10-31 13:01:23.162166973 +0100
@@ -18,6 +18,7 @@
 pcmcia-storage-modules-${kernel:Version}
 pcmcia-modules-${kernel:Version}
 pcmciautils-udeb
+virtio-modules-${kernel:Version}
 
 # Support for USB CD-ROMs
 usb-storage-modules-${kernel:Version}
Index: debian-installer/installer/build/pkg-lists/netboot/powerpc.cfg
===================================================================
--- debian-installer.orig/installer/build/pkg-lists/netboot/powerpc.cfg	2014-10-31 10:46:54.289199728 +0100
+++ debian-installer/installer/build/pkg-lists/netboot/powerpc.cfg	2014-10-31 12:57:39.126163007 +0100
@@ -9,6 +9,7 @@
 pcmciautils-udeb
 #fb-modules-${kernel:Version}
 usb-modules-${kernel:Version}
+virtio-modules-${kernel:Version}
 
 console-setup-udeb
 kbd-udeb
