--- /usr/share/doc/collectd/examples/collection.cgi	2008-12-07 17:26:19.000000000 +0200
+++ ./collection.cgi	2010-01-22 00:45:29.000000000 +0200
@@ -13,6 +13,7 @@
 
 our $Config = "/etc/collectd/collection.conf";
 our @DataDirs = ();
+our @dontshowtype = ();
 our $LibDir;
 
 our $ValidTimespan =
@@ -78,6 +79,10 @@
 			$value =~ s#/*$##;
 			$LibDir = $value;
 		}
+		elsif ($key eq 'dontshowtype')
+                {
+			push (@dontshowtype, $value);
+                }
 		else
 		{
 			print STDERR "Unknown key: $key\n";
@@ -239,6 +244,7 @@
       my $name = "$_";
       $name =~ s/\.rrd$//i;
       my ($type, $instance) = split (m/-/, $name, 2);
+      if ( grep (/^$type$/, @dontshowtype) )  {next}
       $types{$type} = [] if (!$types{$type});
       push (@{$types{$type}}, $instance) if (defined ($instance));
     }
