--- attr-2.4.43.debian/debian/rules 
+++ attr-2.4.43.emdebian/debian/rules 
@@ -3,6 +3,13 @@
 package = attr
 develop = lib$(package)1-dev
 library = lib$(package)1
+
+DEB_BUILD_GNU_TYPE  = $(shell dpkg-architecture -qDEB_BUILD_GNU_TYPE)
+DEB_HOST_GNU_TYPE   = $(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)
+ifneq ($(DEB_BUILD_GNU_TYPE),$(DEB_HOST_GNU_TYPE))
+LOCAL_CONFIGURE_OPTIONS = --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE)
+export LOCAL_CONFIGURE_OPTIONS
+endif
 
 dirme  = debian/$(package)
 dirdev = debian/$(develop)
@@ -28,6 +35,9 @@
 .census:
 	@echo "== dpkg-buildpackage: configure" 1>&2
 	$(checkdir)
+	libtoolize
+	aclocal -I m4
+	autoconf
 	$(options) $(MAKE) configure
 	touch .census
 
@@ -38,6 +48,7 @@
 	$(MAKE) distclean
 	-rm -rf $(dirme) $(dirdev) $(dirlib) debian/tmp
 	-rm -f debian/*substvars debian/files* debian/*.debhelper
+	${RM} config.guess config.sub ltmain.sh
 
 binary-indep:
 
