--- attr.old/debian/rules	2007-11-07 15:01:13.000000000 +0000
+++ attr-2.4.39/debian/rules	2007-11-07 15:02:12.000000000 +0000
@@ -4,6 +4,13 @@
 develop = lib$(package)1-dev
 library = lib$(package)1
 
+DEB_BUILD_GNU_TYPE  = $(shell dpkg-architecture -qDEB_BUILD_GNU_TYPE)
+DEB_HOST_GNU_TYPE   = $(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)
+ifneq ($(DEB_BUILD_GNU_TYPE),$(DEB_HOST_GNU_TYPE))
+LOCAL_CONFIGURE_OPTIONS = --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE)
+export LOCAL_CONFIGURE_OPTIONS
+endif
+
 dirme  = debian/$(package)
 dirdev = debian/$(develop)
 dirlib = debian/$(library)
@@ -29,6 +36,9 @@
 .census:
 	@echo "== dpkg-buildpackage: configure" 1>&2
 	$(checkdir)
+	libtoolize
+	aclocal -I m4
+	autoconf
 	$(options) $(MAKE) configure
 	touch .census
 
