From c2887156bb9f9a87bc89ad970adcb4cca59fd9a3 Mon Sep 17 00:00:00 2001
From: Mathieu Parent <math.parent@gmail.com>
Date: Wed, 22 Oct 2014 18:43:20 +0200
Subject: [PATCH 2/2] Changelog for previous commit

---
 debian/changelog | 8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/debian/changelog b/debian/changelog
index 22173de..9479f30 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,11 @@
+ckeditor (4.4.4+dfsg1-1.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Install complete core sources instead of broken closure-compiled one
+    (Closes: #753668, #756155)
+
+ -- Mathieu Parent <sathieu@debian.org>  Wed, 22 Oct 2014 18:43:11 +0200
+
 ckeditor (4.4.4+dfsg1-1) unstable; urgency=high
 
   * New upstream release.
-- 
2.1.1

