diff --git a/tools/dom-new-git-repo b/tools/dom-new-git-repo
index d62524c..44623af 100755
--- a/tools/dom-new-git-repo
+++ b/tools/dom-new-git-repo
@@ -23,7 +23,7 @@ REPODIR="/git/pkg-ocaml-maint/packages/$PACKAGE.git"
 initialize_alioth_repos () {
 echo "I: initialize (empty) remote repository on alioth.d.o"
 # see http://wiki.debian.org/Alioth/Git
-cat <<EOCMD | ssh $ALIOTH
+cat <<EOCMD | ssh $ALIOTH sh
 umask 002
 set -e
 
@@ -38,7 +38,7 @@ EOCMD
 
 setup_notifications () {
 echo "I: set up commit notification (mail and CIA)"
-cat <<EOCMD | ssh $ALIOTH
+cat <<EOCMD | ssh $ALIOTH sh
 umask 002
 
 cd "$REPODIR"
