diff -u brltty-3.8/debian/changelog brltty-3.8/debian/changelog
--- brltty-3.8/debian/changelog
+++ brltty-3.8/debian/changelog
@@ -1,3 +1,10 @@
+brltty (3.8-9.1) unstable; urgency=medium
+
+  * NMU
+  * Explicitly link with -ldl.
+
+ -- Joey Hess <joeyh@debian.org>  Wed, 19 Sep 2007 16:48:05 -0400
+
 brltty (3.8-8) unstable; urgency=low
 
   * Disable GPM in udeb build (Closes: Bug#440731, Bug#441782).
only in patch2:
unchanged:
--- brltty-3.8.orig/Programs/Makefile.in
+++ brltty-3.8/Programs/Makefile.in
@@ -169,7 +169,7 @@
 BRLTTY_OBJECTS = main.$O config.$O options.$O misc.$O $(SYSTEM_OBJECT) io_misc.$O queue.$O async.$O auth.$O cut.$O cmd.$O kbd.$O lock.$O charset.$O $(TBL_OBJECTS) $(CTB_OBJECTS) touch.$O serial.$O $(USB_OBJECTS) $(BLUETOOTH_OBJECTS) drivers.$O $(SCREEN_OBJECTS) $(SCREEN_DRIVER_OBJECTS) route.$O brl.$O $(BRAILLE_DRIVER_OBJECTS) $(SPEECH_OBJECT) $(SPEECH_DRIVER_OBJECTS) $(TUNE_OBJECTS) $(API_OBJECTS)
 
 brltty$X: $(BRLTTY_OBJECTS)
-	$(CC) $(LDFLAGS) -o $@ $(BRLTTY_OBJECTS) $(BRAILLE_DRIVER_LIBRARIES) $(SPEECH_DRIVER_LIBRARIES) $(SCREEN_DRIVER_LIBRARIES) $(LDLIBS)
+	$(CC) $(LDFLAGS) -o $@ $(BRLTTY_OBJECTS) $(BRAILLE_DRIVER_LIBRARIES) $(SPEECH_DRIVER_LIBRARIES) $(SCREEN_DRIVER_LIBRARIES) $(LDLIBS) -ldl
 
 main.$O:
 	$(CC) $(CFLAGS) -c $(SRC_DIR)/main.c
