--- apt-0.7.20.2/apt-pkg/contrib/strutl.cc	2009-02-07 16:09:35.000000000 +0100
+++ apt-0.7.20.2+524406/apt-pkg/contrib/strutl.cc	2009-04-18 11:41:48.000000000 +0200
@@ -45,8 +45,10 @@
   const char *inbuf;
   char *inptr, *outbuf, *outptr;
   size_t insize, outsize;
+  char tocode[strlen(codeset) + 11];
+  sprintf(tocode, "%s//TRANSLIT", codeset);
   
-  cd = iconv_open(codeset, "UTF-8");
+  cd = iconv_open(tocode, "UTF-8");
   if (cd == (iconv_t)(-1)) {
      // Something went wrong
      if (errno == EINVAL)
