From 87df467c805c156486e69a4e94e2fc35f3161245 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lo=C3=AFc=20Minier?= <loic.minier@ubuntu.com>
Date: Thu, 6 Jun 2013 16:06:09 +0200
Subject: [PATCH] debchange: Allow config overrides in environment

---
 scripts/debchange.pl | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/scripts/debchange.pl b/scripts/debchange.pl
index 98acd63..684e5b8 100755
--- a/scripts/debchange.pl
+++ b/scripts/debchange.pl
@@ -314,6 +314,11 @@ if (@ARGV and $ARGV[0] =~ /^--no-?conf$/) {
     my $shell_out = `/bin/bash -c '$shell_cmd'`;
     @config_vars{keys %config_vars} = split /\n/, $shell_out, -1;
 
+    # Allow overrides in the environment
+    foreach my $var (keys %config_vars) {
+        $config_vars{$var} = $ENV{$var} if $ENV{$var};
+    }
+
     # Check validity
     $config_vars{'DEBCHANGE_PRESERVE'} =~ /^(yes|no)$/
 	or $config_vars{'DEBCHANGE_PRESERVE'}='no';
-- 
1.8.3

