diff -Nru chrony-1.24.orig/rtc_linux.c chrony-1.24/rtc_linux.c
--- chrony-1.24.orig/rtc_linux.c
+++ chrony-1.24/rtc_linux.c
@@ -541,9 +541,13 @@
 int
 RTC_Linux_Initialise(void)
 {
-  int major, minor, patch;
   char *direc;
 
+  /* All kernel versions since 2.2.x have RTC support. */
+  /* This test is obsolete for some years (see chrony 1.26). */
+#if 0
+  int major, minor, patch;
+
   /* Check whether we can support the real time clock.
 
      Linux 1.2.x - haven't checked yet
@@ -597,6 +601,7 @@
         break; /* OK for all patch levels */
     } 
   }
+#endif
 
   /* Setup details depending on configuration options */
   setup_config();
