commit aa736e0b453df45d6cdbd429662751e14068caec
Author: Paul Wise <pabs@debian.org>
Date:   Sat Sep 27 10:51:21 2014 +0800

    Enable the timeless build flags by default.

diff --git a/man/dpkg-buildflags.1 b/man/dpkg-buildflags.1
index f484eaa..c86b01a 100644
--- a/man/dpkg-buildflags.1
+++ b/man/dpkg-buildflags.1
@@ -369,7 +369,7 @@ compilation. Except as noted below, these are enabled by default for
 architectures that support them.
 .TP
 .B timeless
-This setting (disabled by default) adds
+This setting (enabled by default) adds
 .B \-Wdate\-time
 to \fBCPPFLAGS\fP.
 This will cause warnings when the \fB__TIME__\fP, \fB__DATE__\fP and
diff --git a/scripts/Dpkg/Vendor/Debian.pm b/scripts/Dpkg/Vendor/Debian.pm
index 9b40e7a..db40b2c 100644
--- a/scripts/Dpkg/Vendor/Debian.pm
+++ b/scripts/Dpkg/Vendor/Debian.pm
@@ -157,7 +157,7 @@ sub _add_reproducible_flags {
 
     # Default feature states.
     my %use_feature = (
-        timeless => 0,
+        timeless => 1,
     );
 
     # Adjust features based on user or maintainer's desires.
