--- objectives.cpp	2004-10-02 15:30:31.000000000 +0200
+++ objectives.cpp.new	2004-10-09 16:54:45.000000000 +0200
@@ -53,7 +53,6 @@ void adjustObjectives()
 	int target = 0;
 
 	Objective *objective = (Objective*)map.objectiveList.getHead();
-	Entity *item, *enemy;
 	
 	bool previouslyCleared = gameData.stagePreviouslyCleared(map.name);
 
@@ -80,43 +79,14 @@ void adjustObjectives()
 		{
 			if (gameData.objectiveCompleted(map.name, objective->description))
 			{
-				objective->currentValue = objective->targetValue;
 				objective->completed = true;
 	
 				debug(("Objective %s has been completed\n", objective->description));
-	
-				item = (Entity*)map.itemList.getHead();
-				while (item->next != NULL)
-				{
-					item = (Entity*)item->next;
-	
-					if (strcmp(item->name, objective->target) == 0)
-					{
-						item->health = 0;
-						map.totalItems--;
-					}
-				}
-	
-				if (strcmp(objective->target, "Enemy"))
-				{
-					enemy = (Entity*)map.enemyList.getHead();
-					while (enemy->next != NULL)
-					{
-						enemy = (Entity*)enemy->next;
-	
-						if (strcmp(enemy->name, objective->target) == 0)
-						{
-							enemy->health = 0;
-						}
-					}
-				}
-			}
-			else
-			{
-				gameData.getObjectiveValues(map.name, objective->description, &current, &target);
-				objective->currentValue = current;
-				objective->targetValue = target;
 			}
+	
+			gameData.getObjectiveValues(map.name, objective->description, &current, &target);
+			objective->currentValue = current;
+			objective->targetValue = target;
 		}
 	}
 }
