diff -wru asterisk-0.5.0/debian/control asterisk-0.5.0-he/debian/control
--- asterisk-0.5.0/debian/control	2003-10-21 10:27:42.000000000 +0200
+++ asterisk-0.5.0-he/debian/control	2003-10-21 10:28:05.000000000 +0200
@@ -3,7 +3,7 @@
 Section: comm
 Maintainer: Mark Purcell <msp@debian.org>
 Standards-Version: 3.5.6
-Build-Depends: debhelper (>= 3), libgtk2.0-dev | libgtk-dev, libreadline4-dev | libreadline-dev, libgsm1-dev, libssl-dev, libzap-dev, libtonezone-dev, libmysqlclient-dev, doxygen, libnewt-dev, bison, libedit-dev, libasound2-dev, libgtk1.2-dev | libgtk-dev
+Build-Depends: debhelper (>= 4.0.4), libgtk2.0-dev | libgtk-dev, libreadline4-dev | libreadline-dev, libgsm1-dev, libssl-dev, libzap-dev, libtonezone-dev, libmysqlclient-dev, doxygen, libnewt-dev, bison, libedit-dev, libasound2-dev, libgtk1.2-dev | libgtk-dev
 
 Package: asterisk
 Architecture: any
