diff -x po -ur apt-0.7.19/debian/rules apt-0.7.19.new/debian/rules
--- apt-0.7.19/debian/rules	2008-11-24 09:32:23.000000000 +0000
+++ apt-0.7.19.new/debian/rules	2009-01-02 20:02:57.000000000 +0000
@@ -61,7 +61,7 @@
 ifneq ($(APT_DEBVER),$(APT_CONFVER))
 ifneq ($(APT_DEBVER),)
 .PHONY: configure.in
-configure.in:
+configure.in: debian/changelog
 	sed -e 's/$(APT_CONFVER)/$(APT_DEBVER)/' $@ > $@.$$$$ && mv $@.$$$$ $@
 endif
 else
@@ -269,7 +269,7 @@
 	dh_md5sums -p$@
 	dh_builddeb -p$@
 
-apt-utils: build debian/shlibs.local
+apt-utils: build build-doc debian/shlibs.local
 	dh_testdir -p$@
 	dh_testroot -p$@
 	dh_clean -p$@ -k
