diff -Nudr cvs-1.11.2.old/debian/cvs.templates cvs-1.11.2.new/debian/cvs.templates
--- cvs-1.11.2.old/debian/cvs.templates	2002-11-09 19:18:44.000000000 +0100
+++ cvs-1.11.2.new/debian/cvs.templates	2002-11-09 19:19:36.000000000 +0100
@@ -54,7 +54,7 @@
 Type: select
 Choices: yes, no, individual
 Default: no
-Description: Select how many previous history files should be kept?
+Description: Change the number of kept previous history files?
  When rotating history files in repositories, by default the previous
  7 are kept. If you want to change this number globally, select yes,
  or for individual repositories, select individual. Otherwise, select
diff -Nudr cvs-1.11.2.old/debian/cvs.templates.de cvs-1.11.2.new/debian/cvs.templates.de
--- cvs-1.11.2.old/debian/cvs.templates.de	2002-11-09 19:18:44.000000000 +0100
+++ cvs-1.11.2.new/debian/cvs.templates.de	2002-11-09 19:20:06.000000000 +0100
@@ -47,7 +47,7 @@
 Template: cvs/rotatekeep_nondefault
 Type: select
 Choices-de: Ja, Nein, Individuell
-Description-de: Wie viele alte Log-Dateien sollen aufbewahrt werden?
+Description-de: Ändere die Anzahl der aufbewahrten alten Logdateien?
  Wenn die Log-Dateien bzw. History-Dateien in den einzelnen Repositorys
  rotiert werden, werden standardmäßig die sieben neuesten aufbewahrt.
  Wenn Sie dies global ändern möchten, wählen Sie "Ja", um dies bei
