--- aspcud-1.8.0/debian/tests/control   2014-01-11 09:27:53.000000000 +0100
+++ aspcud-1.8.0.new/debian/tests/control       2014-01-13 07:26:51.322461599 +0100
@@ -1,2 +1,2 @@
 Tests: upstream
-Depends: cudf-tools
+Depends: @, cudf-tools
--- aspcud-1.8.0/debian/tests/upstream  2014-01-11 09:27:53.000000000 +0100
+++ aspcud-1.8.0.new/debian/tests/upstream      2014-01-13 07:31:37.990461599 +0100
@@ -2,7 +2,7 @@
 
 set -e
 
-indir=cudf
+indir=debian/tests/cudf
 outdir=$TMPDIR
 
 for infile in $indir/*.cudf
@@ -10,8 +10,8 @@
     cudf=$(basename $infile)
     outfile=$outdir/${cudf%.cudf}.sol
     checkfile=$outdir/${cudf%.cudf}.check
-    aspcud $infile $outfile trendy > /dev/null 2>&1
-    cudf-check -cudf $infile -sol $outfile > $checkfile 2> /dev/null
+    aspcud $infile $outfile trendy
+    cudf-check -cudf $infile -sol $outfile > $checkfile
     if [ `grep -c "is_solution: true" $checkfile` -ne 1 ]
     then
        echo "Test \"legacy\" with options \"$solver\":"

