diff --git i/man/dpkg-parsechangelog.pod w/man/dpkg-parsechangelog.pod
index 52026ed04..4a6ffb182 100644
--- i/man/dpkg-parsechangelog.pod
+++ w/man/dpkg-parsechangelog.pod
@@ -109,6 +109,10 @@ concatenated (space-separated) comments from all the versions requested.
 
 =item B<Maintainer:> I<author>
 
+The name and email address of the person who prepared these changes,
+they are B<not> necessarily those of the uploader or the usual package
+maintainer.
+
 =item B<Date:> I<date>
 
 The date of the entry as a string, as it appears in the changelog.
@@ -272,6 +276,13 @@ number itself.
 
 =back
 
+=head1 BUGS
+
+The B<Maintainer> field has a confusing name matching the field in
+the F<debian/control> file but not its exact semantics,
+where its meaning would be better represented by the B<Changed-By> field
+name used in the F<.changes> file.
+
 =head1 SEE ALSO
 
 B<deb-changelog>(5).
