From 1ec12e462282b5f031cd28aaaef5ded7ffeb039d Mon Sep 17 00:00:00 2001
From: Adam Borowski <kilobyte@angband.pl>
Date: Thu, 12 Feb 2015 07:39:01 +0100
Subject: [PATCH] Link x32 kernel selectors to amd64.

---
 kernel/x32.sh | 1 +
 1 file changed, 1 insertion(+)
 create mode 120000 kernel/x32.sh

diff --git a/kernel/x32.sh b/kernel/x32.sh
new file mode 120000
index 0000000..2f5ab2e
--- /dev/null
+++ b/kernel/x32.sh
@@ -0,0 +1 @@
+amd64.sh
\ No newline at end of file
-- 
2.1.4

