*** lib_udev_rules.d_97-hid2hci.rules	2019-04-27 13:11:06.009105863 +0300
--- /lib/udev/rules.d/97-hid2hci.rules	2019-04-27 13:54:48.747630419 +0300
*************** SUBSYSTEM!="usb*", GOTO="hid2hci_end"
*** 8,13 ****
--- 8,14 ----
  # Known supported devices: 413c:8154, 413c:8158, 413c:8162
  ATTR{bInterfaceClass}=="03", ATTR{bInterfaceSubClass}=="01", ATTR{bInterfaceProtocol}=="02", \
    ATTRS{bDeviceClass}=="00", ATTRS{idVendor}=="413c", ATTRS{bmAttributes}=="e0", \
+   DRIVER=="usbhid", \
    RUN+="hid2hci --method=dell --devpath=%p", ENV{HID2HCI_SWITCH}="1"
  
  # Logitech devices
