--- cdbs.old/1/class/makefile-vars.mk.in	2007-05-18 17:01:14.000000000 +0100
+++ cdbs-0.4.50/1/class/makefile-vars.mk.in	2007-11-24 11:43:20.000000000 +0000
@@ -25,7 +25,11 @@
 include $(_cdbs_class_path)/langcore.mk$(_cdbs_makefile_suffix)
 
 DEB_MAKE_MAKEFILE =
+ifneq ($(DEB_HOST_GNU_TYPE),$(DEB_BUILD_GNU_TYPE))
+DEB_MAKE_ENVVARS=CC=$(DEB_HOST_GNU_TYPE)-gcc
+else
 DEB_MAKE_ENVVARS = 
+endif
 DEB_MAKE_INVOKE = $(DEB_MAKE_ENVVARS) $(MAKE) $(if $(DEB_MAKE_MAKEFILE), -f $(DEB_MAKE_MAKEFILE),) -C $(DEB_BUILDDIR) CFLAGS=$(if $(CFLAGS_$(cdbs_curpkg)),"$(CFLAGS_$(cdbs_curpkg))","$(CFLAGS)") CXXFLAGS=$(if $(CXXFLAGS_$(cdbs_curpkg)),"$(CXXFLAGS_$(cdbs_curpkg))","$(CXXFLAGS)") CPPFLAGS=$(if $(CPPFLAGS_$(cdbs_curpkg)),"$(CPPFLAGS_$(cdbs_curpkg))","$(CPPFLAGS)") LDFLAGS=$(if $(LDFLAGS_$(cdbs_curpkg)),"$(LDFLAGS_$(cdbs_curpkg))","$(LDFLAGS)") 
 
 # This variable is deprecated.
