--- blobwars-1.05.orig/src/spawnPoints.cpp	2005-05-03 23:05:39.000000000 +0200
+++ blobwars-1.05/src/spawnPoints.cpp	2007-03-22 17:33:40.000000000 +0100
@@ -191,7 +191,19 @@
 							y = y >> BRICKSHIFT;
 							if (map.isSolid(x, y))
 							{
-								map.data[x][y] = MAP_AIR;
+								int waterLevel = (int)map.waterLevel;
+								if (waterLevel == -1 || y < waterLevel)
+								{
+									map.data[x][y] = MAP_AIR;
+								}
+								else if (y == waterLevel)
+								{
+									map.data[x][y] = MAP_WATERANIM;
+								}
+								else
+								{
+									map.data[x][y] = MAP_WATER;
+								}
 							}
 							break;
 						case HAZARD_POWERBULLETS:
