diff -u digitemp-3.3.2/debian/changelog digitemp-3.3.2/debian/changelog
--- digitemp-3.3.2/debian/changelog
+++ digitemp-3.3.2/debian/changelog
@@ -1,3 +1,10 @@
+digitemp (3.3.2-2.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Removed bashisms from debian/rules (use of {,}). Closes: #315599, #379489.
+
+ -- Lucas Nussbaum <lucas@lucas-nussbaum.net>  Sat, 19 Jan 2008 17:15:48 +0100
+
 digitemp (3.3.2-2) unstable; urgency=low
 
   * Added digitemp_DS2490. Patch created by Mikael Mikael Magnusson
diff -u digitemp-3.3.2/debian/rules digitemp-3.3.2/debian/rules
--- digitemp-3.3.2/debian/rules
+++ digitemp-3.3.2/debian/rules
@@ -32,10 +32,10 @@
 build-stamp: configure-stamp 
 	dh_testdir
 
-	mkdir -p build-serial/{src,userial/ds9097,userial/ds9097u}
+	mkdir -p build-serial/src build-serial/userial/ds9097 build-serial/userial/ds9097u
 	(cd build-serial; $(MAKE) -f ../Makefile srcdir=.. VPATH=.. all)
 	
-	mkdir -p build-usb/{src,userial/ds2490}       
+	mkdir -p build-usb/src build-usb/userial/ds2490
 	(cd build-usb; $(MAKE) -f ../Makefile srcdir=.. VPATH=.. ds2490)
 
 	#/usr/bin/docbook-to-man debian/digitemp.sgml > digitemp.1
