diff -u cutils-1.6/debian/changelog cutils-1.6/debian/changelog
--- cutils-1.6/debian/changelog
+++ cutils-1.6/debian/changelog
@@ -1,3 +1,12 @@
+cutils (1.6-3.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Rebuild with newer debhelper to get rid of install-info calls in
+    maintainer scripts (Closes: #708490).
+  * debian/cutils.prerm: Empty maintainer script to ensure clean upgraded.
+
+ -- Sebastian Ramacher <sramacher@debian.org>  Sat, 10 Aug 2013 21:06:20 +0200
+
 cutils (1.6-3) unstable; urgency=low
 
   * Change build-depends to use flex-old rather than flex; 
only in patch2:
unchanged:
--- cutils-1.6.orig/debian/cutils.prerm
+++ cutils-1.6/debian/cutils.prerm
@@ -0,0 +1,8 @@
+#!/bin/sh
+set -e
+
+# Dummy maintainer script to ensure clean upgrades from a version of cutils that
+# has an install-info call in its prerm script with dpkg >= 1.17 installed. This
+# can be removed after the jessie release.
+
+#DEBHELPER#
