From 1cbe248ca0b6cd6c0e3947c5e8d74ebcf77e7fd1 Mon Sep 17 00:00:00 2001
From: Jari Aalto <jari.aalto@cante.net>
Date: Thu, 22 Mar 2012 08:01:41 -0400
Subject: [PATCH] 665001
Organization: Private
Content-Type: text/plain; charset="utf-8"
Content-Transfer-Encoding: 8bit

Signed-off-by: Jari Aalto <jari.aalto@cante.net>
---
 debian/control |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/control b/debian/control
index d98a674..c287f8a 100644
--- a/debian/control
+++ b/debian/control
@@ -3,12 +3,12 @@ Section: misc
 Priority: optional
 Maintainer: Debbugs developers <debian-debbugs@lists.debian.org>
 Uploaders: Josip Rodin <jrodin@jagor.srce.hr>, Colin Watson <cjwatson@debian.org>
-Standards-Version: 3.2.1
+Standards-Version: 3.9.3.1
 Build-Depends-Indep: debhelper
 
 Package: debbugs
 Architecture: all
-Depends: perl5 | perl, exim | mail-transport-agent, libmailtools-perl, ed, libmime-perl, libio-stringy-perl
+Depends: perl5 | perl, exim | mail-transport-agent, libmailtools-perl, ed, libmime-tools-perl, libio-stringy-perl
 Recommends: httpd, links | lynx
 Description: The bug tracking system based on the active Debian BTS
  Debian has a bug tracking system which files details of bugs reported by
-- 
1.7.9.1

