--- cccc-3.pre63/cccc/cccc_tbl.h.orig	2003-07-13 12:33:48.000000000 +0000
+++ cccc-3.pre63/cccc/cccc_tbl.h	2003-07-13 12:34:13.000000000 +0000
@@ -20,7 +20,7 @@
 : public std::map<string,T*>
 {
   typedef std::map<string,T*> map_t;
-  map_t::iterator iter_;
+  typename map_t::iterator iter_;
   bool sorted;
 
  public:
--- cccc-3.pre63/cccc/cccc_tbl.cc.orig	2003-07-13 12:33:55.000000000 +0000
+++ cccc-3.pre63/cccc/cccc_tbl.cc	2003-07-13 12:35:06.000000000 +0000
@@ -45,7 +45,7 @@
 T* CCCC_Table<T>::find(string name)
 {
   T *retval=NULL;
-  map_t::iterator value_iterator=map_t::find(name);
+  typename map_t::iterator value_iterator=map_t::find(name);
   if(value_iterator!=end())
     {
       retval=(*value_iterator).second;
@@ -60,7 +60,7 @@
   T *retval=find(new_key);
   if(retval==NULL)
     {
-      map_t::value_type new_pair(new_key,new_item_ptr);
+      typename map_t::value_type new_pair(new_key,new_item_ptr);
       insert(new_pair);
       sorted=false;
       retval=new_item_ptr;
@@ -72,7 +72,7 @@
 bool CCCC_Table<T>::remove(T* old_item_ptr)
 {
   bool retval=false; 
-  map_t::iterator value_iterator=map_t::find(old_item_ptr->key());
+  typename map_t::iterator value_iterator=map_t::find(old_item_ptr->key());
   if(value_iterator!=map_t::end())
     {
       erase(value_iterator);
