diff -Nru dpkg-cross-2.5.3+adhoc/Debian/DpkgCross.pm dpkg-cross-2.5.3+adhoc+multiarch/Debian/DpkgCross.pm
--- dpkg-cross-2.5.3+adhoc/Debian/DpkgCross.pm	2009-09-04 16:43:39.000000000 +0200
+++ dpkg-cross-2.5.3+adhoc+multiarch/Debian/DpkgCross.pm	2009-09-07 12:52:54.000000000 +0200
@@ -404,6 +404,7 @@
 
 	$crosslib64 ||= $crosslib . "64";
 	$crosslib32 ||= $crosslib . "32";
+	$config{'crosstype'} = $deb_host_gnu_type;
 	$config{'crossbase'} = $crossbase;
 	$config{'crossprefix'} = $crossprefix;
 	$config{'crossdir'} = $crossdir;
diff -Nru dpkg-cross-2.5.3+adhoc/debian/changelog dpkg-cross-2.5.3+adhoc+multiarch/debian/changelog
--- dpkg-cross-2.5.3+adhoc/debian/changelog	2009-09-04 16:40:09.000000000 +0200
+++ dpkg-cross-2.5.3+adhoc+multiarch/debian/changelog	2009-09-07 12:29:12.000000000 +0200
@@ -1,3 +1,9 @@
+dpkg-cross (2.5.3+adhoc+multiarch) unstable; urgency=low
+
+  * Add multiarch support patch
+
+ -- Simon Richter <sjr@debian.org>  Mon, 07 Sep 2009 12:28:55 +0200
+
 dpkg-cross (2.5.3+adhoc) unstable; urgency=low
 
   * Add ad-hoc arch definition patch
diff -Nru dpkg-cross-2.5.3+adhoc/dpkg-cross dpkg-cross-2.5.3+adhoc+multiarch/dpkg-cross
--- dpkg-cross-2.5.3+adhoc/dpkg-cross	2009-09-04 16:41:18.000000000 +0200
+++ dpkg-cross-2.5.3+adhoc+multiarch/dpkg-cross	2009-09-07 12:52:02.000000000 +0200
@@ -215,7 +215,7 @@
 @keepdeps $dpkg_statfile $progname $debname $anyway $cross2cross
 $crossdir $crosslib $crosslib64 $crosslib32 $crossinc $data $len
 $retval $dpkg_cmd $mode $pkg @exlist $conffile $removedeps $keepdeps
-$DPKGCROSSVERSION $keep_temp $gnuarch);
+$DPKGCROSSVERSION $keep_temp $gnuarch $crosstype);
 
 $dpkg_statfile = "/var/lib/dpkg/status";
 &read_config;
@@ -721,6 +721,7 @@
 		return 1;
 	}
 	my $config = &get_config;
+	$crosstype = $$config{'crosstype'};
 	$crossinc = $$config{'crossinc'};
 	$crossdir = $$config{'crossdir'};
 	$crosslib = $$config{'crosslib'};
@@ -765,6 +766,10 @@
 			}
 			close(FROM);
 			close(TO);
+		} elsif (m:^/usr/include/$crosstype/:) {
+			link_file("$src$_", "$dst$_") or goto fail;
+		} elsif (m:^/usr/lib/$crosstype/:) {
+			link_file("$src$_", "$dst$_") or goto fail;
 		} elsif (m:^/usr(/X11R6)?/include/:) {
 			# regular file under /usr/include or /usr/X11R6/include
 			link_file("$src$_", "$dst$crossinc/$'") or goto fail;
