--- desmume-0.7.2-orig/debian/control	2007-08-02 16:23:30.000000000 +0530
+++ desmume-0.7.2/debian/control	2007-08-02 16:23:50.000000000 +0530
@@ -3,7 +3,7 @@
 Priority: extra
 Maintainer: Debian Games Team <pkg-games-devel@lists.alioth.debian.org>
 Uploaders: Reinhard Tartler <siretart@tauware.de>, Pascal Giard <pascal@debian.org>
-Build-Depends: debhelper (>= 5), libglade2-dev, libgtk2.0-dev, libsdl1.2-dev, libgtkglext1-dev
+Build-Depends: debhelper (>= 5), libglade2-dev, libgtk2.0-dev, libsdl1.2-dev, libgtkglext1-dev, tofrodos
 Standards-Version: 3.7.2
 
 Package: desmume
