--- policy.sgml.orig	2008-05-10 21:59:39.000000000 +0200
+++ policy.sgml	2008-05-10 22:00:39.000000000 +0200
@@ -8246,8 +8246,7 @@
 	    in the <em>X Toolkit Intrinsics - C Language
 	    Interface</em> manual is also permitted).  They must be
 	    registered as <tt>conffile</tt>s or handled as
-	    configuration files.  Packages must not provide the
-	    directory <file>/usr/X11R6/lib/X11/app-defaults/</file>.
+	    configuration files.
 	  </p>
 
 	  <p>
@@ -8263,12 +8262,6 @@
 		are stored in the X server and affect all connecting
 		clients.
 	    </footnote>
-	    <em>Important:</em> packages that install files into the
-	    <file>/etc/X11/Xresources/</file> directory must conflict with
-	    <tt>xbase (&lt;&lt; 3.3.2.3a-2)</tt>; if this is not done
-	    it is possible for the installing package to destroy a
-	    previously-existing <file>/etc/X11/Xresources</file> file
-	    which had been customized by the system administrator.
 	  </p>
 	</sect1>
 
