diff --git a/policy.sgml b/policy.sgml
index df586d1..8f02c12 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -1218,17 +1218,16 @@
 	  <heading>Prompting in maintainer scripts</heading>
 	  <p>
 	    Package maintainer scripts may prompt the user if
-	    necessary. Prompting should be done by communicating
+	    necessary. Prompting must be done by communicating
 	    through a program, such as <prgn>debconf</prgn>, which
 	    conforms to the Debian Configuration Management
-	    Specification, version 2 or higher.  Prompting the user by
-	    other means, such as by hand<footnote>
-                From the Jargon file: by hand 2. By extension,
-                writing code which does something in an explicit or
-                low-level way for which a presupplied library
-                (<em>debconf, in this instance</em>) routine ought
-                to have been available.
-            </footnote>, is now deprecated.
+	    Specification, version 2 or higher.
+	  </p>
+
+	  <p>
+	    Packages which are essential, or which are dependencies of
+	    essential packages, may fall back on another prompting method
+	    if no such interface is available when they are executed.
 	  </p>
 
 	  <p>
