diff -Nru abook-0.6.0~pre2/debian/changelog abook-0.6.0~pre2/debian/changelog
--- abook-0.6.0~pre2/debian/changelog	2015-07-07 17:39:52.000000000 +0200
+++ abook-0.6.0~pre2/debian/changelog	2015-07-26 08:29:38.000000000 +0200
@@ -1,3 +1,15 @@
+abook (0.6.0~pre2-4.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Quote argument to -x test when testing for update-menus.
+    Postinst switched from testing existence and execution permissions for
+    hardcoded /usr/bin/update-menus to testing the path returned by `which
+    update-menus`.
+    Fixes "abook: fails to install: abook.postinst: update-menus: not
+    found". (Closes: #791803)
+
+ -- Salvatore Bonaccorso <carnil@debian.org>  Sun, 26 Jul 2015 08:19:44 +0200
+
 abook (0.6.0~pre2-4) unstable; urgency=medium
 
   [ Rhonda D'Vine ]
diff -Nru abook-0.6.0~pre2/debian/postinst abook-0.6.0~pre2/debian/postinst
--- abook-0.6.0~pre2/debian/postinst	2015-07-06 17:35:05.000000000 +0200
+++ abook-0.6.0~pre2/debian/postinst	2015-07-26 08:29:38.000000000 +0200
@@ -54,6 +54,6 @@
 
 
 # menu entry
-if [ "$action" = 'configure' ] && [ -x `which update-menus` ]; then
+if [ "$action" = 'configure' ] && [ -x "`which update-menus`" ]; then
 	update-menus
 fi
