--- /usr/lib/cgi-bin/awstats.pl.orig	2006-11-04 21:17:24.000000000 +0000
+++ /usr/lib/cgi-bin/awstats.pl	2007-08-29 11:41:50.000000000 +0100
@@ -1199,7 +1199,7 @@
 			    # Correct relative include files
 				if ($FileConfig =~ /^(.*[\\\/])[^\\\/]*$/) { $includeFile = "$1$includeFile"; }
 			}
-			if ($level > 1) {
+			if ($level > 1 && $^V lt v5.6.0) {
 				warning("Warning: Perl versions before 5.6 cannot handle nested includes");
 				next;
 			}
