--- /usr/bin/dh-make-perl	2006-08-29 04:01:02.000000000 +0200
+++ /usr/local/bin/dh-make-perl	2006-09-12 10:46:32.624366278 +0200
@@ -160,9 +160,10 @@
     # $version is the version from the perl module itself
     $version, 
     # $pkgversion is the resulting version of the package: User's
-    # --version=s or "$version-1"
-    $pkgversion, 
+    # --version=s or "$version-1" $appversion is just the appendix.
+    $pkgversion,
     $desc, $longdesc, $copyright, $author);
+my $appversion = '-1';
 my ($extrasfields, $extrapfields);
 my (@docs, $changelog, @args);
 my ($cpanmodule, $cpanplusmodule, $cpanmirror, $build, $install, $dbflags, 
@@ -180,6 +181,7 @@
 	"desc=s" => \$desc,
 	"arch=s" => \$arch,
 	"version=s" => \$pkgversion,
+	"append-to-version=s" => \$appversion,
 	"help" => sub {die "\n"},
 	# disabled: see build_package()
 	"dbflags=s" => \$dbflags,
@@ -204,7 +206,7 @@
 $meta = process_meta("$maindir/META.yml") if (-f "$maindir/META.yml");
 ($pkgname, $version) = extract_basic();
 if (! defined $pkgversion) {
-	$pkgversion = $version . "-1";
+	$pkgversion = $version . $appversion;
 }
 move ($tarball, dirname($tarball) . "/${pkgname}_${version}.orig.tar.gz") if ($tarball && $tarball =~ /(?:\.tar\.gz|\.tgz)$/);
 my $module_build = (-f "$maindir/Build.PL") ? "Module-Build" : "MakeMaker";
@@ -894,6 +896,10 @@
 
 Specifies the version of the resulting package.
 
+=item B<--append-to-version> I<VERSION>
+
+Specifies a string to append to the CPAN version of the package. Defaults to '-1'.
+
 =item B<--cpan-mirror> I<MIRROR>
 
 Specifies a CPAN site to use as mirror.
