diff -u dpkg-cross-2.6.11+nmu1/dpkg-cross dpkg-cross-2.6.11+nmu1/dpkg-cross
--- dpkg-cross-2.6.11+nmu1/dpkg-cross	2014-11-30 08:53:56.000000000 +0100
+++ dpkg-cross-2.6.11+nmu1/dpkg-cross	2014-11-30 22:40:26.000000000 +0100
@@ -629,6 +629,14 @@
 		}
 		while (<FROM>) {
 			s:(^|[^-\w/])(/usr)?/lib/$multiarch:$1$crosslib/:g;
+			unless ($multiarch) {
+				s:(^|[^-\w/])(/usr)?/lib32/:$1$crosslib32/:g;
+				s:(^|[^-\w/])(/usr)?/lib64/:$1$crosslib64/:g;
+				s:(^|[^-\w/])(/usr)?/libhf/:$1$crosslibhf/:g;
+				s:(^|[^-\w/])(/usr)?/libn32/:$1$crosslibn32/:g;
+				s:(^|[^-\w/])(/usr)?/libsf/:$1$crosslibsf/:g;
+				s:(^|[^-\w/])(/usr)?/libx32/:$1$crosslibx32/:g;
+			}
 			print TO;
 		}
 		close(FROM);
