--- jabber-bookmarks.el.orig	2008-08-26 16:43:09.000000000 +0300
+++ jabber-bookmarks.el	2008-12-13 00:48:10.143646075 +0200
@@ -39,7 +39,8 @@
 immediately, and return nil if it is not in the cache."
   (if (null cont)
       (jabber-get-conference-data-internal
-       (jabber-get-bookmarks-from-cache jc)
+       (let ((rooms (jabber-get-bookmarks-from-cache jc)))
+	 (when (listp rooms) rooms))
        conference-jid
        key)
     (jabber-get-bookmarks 
@@ -94,7 +95,7 @@
 	 (if (eq (jabber-xml-node-name result) 'storage)
 	     (or (jabber-xml-node-children result) t)
 	   t)))
-    (puthash my-jid value jabber-bookmarks)
+    (when (listp value) (puthash my-jid value jabber-bookmarks))
     (funcall cont jc (when (listp value) value))))
 
 ;;;###autoload
