diff --git a/gas/config/tc-hppa.c b/gas/config/tc-hppa.c
index 5ee7f72..df24834 100644
--- a/gas/config/tc-hppa.c
+++ b/gas/config/tc-hppa.c
@@ -435,9 +435,6 @@ typedef struct label_symbol_struct
 #ifdef OBJ_SOM
     sd_chain_struct *lss_space;
 #endif
-#ifdef OBJ_ELF
-    segT lss_segment;
-#endif
     struct label_symbol_struct *lss_next;
   }
 label_symbol_struct;
@@ -1125,7 +1122,7 @@ pa_get_label (void)
       return label_chain;
 #endif
 #ifdef OBJ_ELF
-    if (now_seg == label_chain->lss_segment && label_chain->lss_label)
+    if (label_chain->lss_label)
       return label_chain;
 #endif
     }
@@ -1151,9 +1148,6 @@ pa_define_label (symbolS *symbol)
 #ifdef OBJ_SOM
       label_chain->lss_space = current_space;
 #endif
-#ifdef OBJ_ELF
-      label_chain->lss_segment = now_seg;
-#endif
       label_chain->lss_next = NULL;
 
       if (label_symbols_rootp)
@@ -1185,7 +1179,7 @@ pa_undefine_label (void)
 	  && current_space == label_chain->lss_space && label_chain->lss_label
 #endif
 #ifdef OBJ_ELF
-	  && now_seg == label_chain->lss_segment && label_chain->lss_label
+	  && label_chain->lss_label
 #endif
 	  )
 	{
