--- src/Makefile.orig	2012-09-04 02:17:36.335709989 +0200
+++ src/Makefile	2012-09-04 02:18:20.091711108 +0200
@@ -4,17 +4,18 @@
 #
 
 MAKE = make
+CC?=gcc
 
 all: amideco
 
 amideco: amifunc kernel
-	gcc amideco.c amifunc.o kernel.o -fpack-struct -o amideco
+	$(CC) amideco.c amifunc.o kernel.o -fpack-struct -o amideco
 
 amifunc:
-	gcc amifunc.c -c -o amifunc.o -fpack-struct
+	$(CC) amifunc.c -c -o amifunc.o -fpack-struct
 
 kernel:
-	gcc kernel.c -c -o kernel.o -fpack-struct
+	$(CC) kernel.c -c -o kernel.o -fpack-struct
 
 clean: 
 	rm -f *.o
