diff --git a/debian/changelog b/debian/changelog
index f386cd9..645393b 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -28,6 +28,12 @@ base-passwd (3.6.0) UNRELEASED; urgency=medium
   * Make it possible to build without debconf support or documentation
     generation.
 
+  [ Georges Khaznadar ]
+  * frozen the group id for crontab (= 101), in order to have this id
+    reproducible, whether one builds a system with debootstrap or with
+    mmdebstrap. (closes: #1012622)
+    
+
  -- Colin Watson <cjwatson@debian.org>  Mon, 04 Oct 2021 19:20:11 +0100
 
 base-passwd (3.5.52) unstable; urgency=medium
diff --git a/group.master b/group.master
index 3e7bf1c..8ee9dd7 100644
--- a/group.master
+++ b/group.master
@@ -35,4 +35,5 @@ plugdev:*:46:
 staff:*:50:
 games:*:60:
 users:*:100:
+crontab:*:101:
 nogroup:*:65534:
