Only in djvulibre-3.5.22.new/: arname
diff -aur djvulibre-3.5.22/debian/rules djvulibre-3.5.22.new/debian/rules
--- djvulibre-3.5.22/debian/rules	2010-01-23 16:03:14.000000000 +0100
+++ djvulibre-3.5.22.new/debian/rules	2010-01-23 16:01:19.225427357 +0100
@@ -147,3 +147,16 @@
 	dh $@ --after makeshlibs
 
 .PHONY: build clean binary-indep binary-arch binary install testlibver
+
+get-orig-source:
+	tmpdir=`mktemp -d` ; destdir=`pwd` ; \
+	uscan --force-download --rename --destdir $$tmpdir ; \
+	cd $$tmpdir ; tarname=`ls *.orig.tar.gz | head -n 1` ; \
+	tar xzf $$tarname ; \
+	rm */doc/djvu?spec.djvu ; \
+	tarname=`echo $$tarname | sed 's/.orig/+dfsg.orig/'` ; \
+	srcdir=`echo $$tarname | sed -e 's/.orig.tar.gz//' -e 's/_/-/'` ; \
+	mv djvulibre-*/ $$srcdir ; \
+	tar czf $$tarname $$srcdir ; \
+	mv $$tarname $$destdir ; \
+	rm -rf $$tmpdir
