Index: debian/control-udeb
===================================================================
--- debian/control-udeb	(revision 230)
+++ debian/control-udeb	(working copy)
@@ -7,7 +7,7 @@
  This is an experimental replacement of kbd-chooser.  It depends on
  loadkeys which does not exist in the installer currently.
 XC-Package-Type: udeb
-XB-Installer-Menu-Item: 12
+XB-Installer-Menu-Item: 1200
 
 Package: console-setup-amiga-ekmap
 Provides: console-setup-ekmap
