diff -ur origs/debianutils-4.2.2/debian/rules patched/debianutils-4.2.2/debian/rules
--- origs/debianutils-4.2.2/debian/rules	2012-03-24 14:52:38.000000000 +0000
+++ patched/debianutils-4.2.2/debian/rules	2012-03-27 15:44:03.000000000 +0000
@@ -12,8 +12,12 @@
 DEB_BUILD_ARCH_OS ?= $(shell dpkg-architecture -qDEB_BUILD_ARCH_OS)
 DEB_BUILD_GNU_TYPE = $(shell dpkg-architecture -qDEB_BUILD_GNU_TYPE)
 DEB_HOST_GNU_TYPE = $(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)
+
+STRIP=strip
+
 ifneq ($(DEB_BUILD_GNU_TYPE),$(DEB_HOST_GNU_TYPE))
-CONFARGS = --host=$(DEB_HOST_GNU_TYPE)
+CONFARGS = --build=$(DEB_BUILD_GNU_TYPE) --host=$(DEB_HOST_GNU_TYPE)
+STRIP=$(DEB_HOST_GNU_TYPE)-strip
 endif
 
 CFLAGS := $(shell dpkg-buildflags --get CFLAGS)
@@ -63,7 +67,7 @@
 	   debian/tmp/bin/
 
 ifeq (,$(findstring nostrip,$(DEB_BUILD_OPTIONS)))
-	strip --remove-section=.comment --remove-section=.note \
+	$(STRIP) --remove-section=.comment --remove-section=.note \
 		debian/tmp/bin/run-parts \
 	   	debian/tmp/bin/tempfile \
 	   	debian/tmp/usr/bin/ischroot
