diff -u coreutils-8.23/debian/rules coreutils-8.23/debian/rules
--- coreutils-8.23/debian/rules
+++ coreutils-8.23/debian/rules
@@ -105,10 +105,6 @@
 	# backward compatability
 	ln -s /usr/bin/md5sum $(d)/usr/bin/md5sum.textutils
 	ln -s /usr/share/man/man1/md5sum.1 $(d)/usr/share/man/man1/md5sum.textutils.1
-ifneq ($(DEB_HOST_ARCH_OS),hurd)
-		# touch used to be in /usr/bin, don't break scripts
-		ln -s /bin/touch $(d)/usr/bin/touch
-endif
 
 	# remove stuff provided by other packages
 ifeq ($(DEB_HOST_ARCH_OS),linux)
only in patch2:
unchanged:
--- coreutils-8.23.orig/debian/coreutils.postinst
+++ coreutils-8.23/debian/coreutils.postinst
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -e
+
+if [ "$1" = 'configure' -a ! -e '/usr/bin/touch' ]; then
+  ln -s /bin/touch /usr/bin/touch
+fi
+
+#DEBHELPER#
+
only in patch2:
unchanged:
--- coreutils-8.23.orig/debian/coreutils.postrm
+++ coreutils-8.23/debian/coreutils.postrm
@@ -0,0 +1,9 @@
+#!/bin/sh
+set -e
+
+if [ "$1" = 'remove' -a -L '/usr/bin/touch' ]; then
+  rm /usr/bin/touch
+fi
+
+#DEBHELPER#
+
