diff -Nru couriergrey-0.3.1/debian/changelog couriergrey-0.3.1/debian/changelog
--- couriergrey-0.3.1/debian/changelog	2012-02-22 06:14:26.000000000 +0000
+++ couriergrey-0.3.1/debian/changelog	2012-04-05 00:49:38.000000000 +0000
@@ -1,3 +1,10 @@
+couriergrey (0.3.1-3.1) UNRELEASED; urgency=low
+
+  * Non-maintainer upload.
+  * Fix FTBFS with gcc 4.7 due to missing <unistd.h> include (Closes: #667142).
+
+ -- Cyril Brulebois <kibi@debian.org>  Thu, 05 Apr 2012 00:49:13 +0000
+
 couriergrey (0.3.1-3) unstable; urgency=low
 
   * Typo at d/README.Debian 
diff -Nru couriergrey-0.3.1/debian/patches/fix-ftbfs-with-gcc-4.7 couriergrey-0.3.1/debian/patches/fix-ftbfs-with-gcc-4.7
--- couriergrey-0.3.1/debian/patches/fix-ftbfs-with-gcc-4.7	1970-01-01 00:00:00.000000000 +0000
+++ couriergrey-0.3.1/debian/patches/fix-ftbfs-with-gcc-4.7	2012-04-05 00:50:09.000000000 +0000
@@ -0,0 +1,14 @@
+Description: Fix FTBFS with gcc 4.7 due to missing <unistd.h> include.
+Author: Cyril Brulebois <kibi@debian.org>
+Bug-Debian: http://bugs.debian.org/667142
+
+--- couriergrey-0.3.1.orig/database.cc
++++ couriergrey-0.3.1/database.cc
+@@ -28,6 +28,7 @@
+ #include <cstdio>
+ #include <stdexcept>
+ #include <glibmm.h>
++#include <unistd.h>
+ 
+ namespace couriergrey {
+     database::database() : db(NULL) {
diff -Nru couriergrey-0.3.1/debian/patches/series couriergrey-0.3.1/debian/patches/series
--- couriergrey-0.3.1/debian/patches/series	1970-01-01 00:00:00.000000000 +0000
+++ couriergrey-0.3.1/debian/patches/series	2012-04-05 00:49:54.000000000 +0000
@@ -0,0 +1 @@
+fix-ftbfs-with-gcc-4.7
