From 25c61ff13ca959dd53380ad3ea8a01f7e6c49407 Mon Sep 17 00:00:00 2001
From: Dominic Hargreaves <dom@earth.li>
Date: Mon, 25 Jul 2016 09:34:18 +0100
Subject: [PATCH 2/2] releasing package cdbs version 0.4.130+deb8u1

---
 debian/changelog | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/changelog b/debian/changelog
index bc16d84..5bc4c42 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,10 +1,10 @@
-cdbs (0.4.130+deb8u1) UNRELEASED; urgency=medium
+cdbs (0.4.130+deb8u1) jessie-security; urgency=high
 
   * Non-maintainer upload.
   * Invoke Makefile.PL and Build.PL with perl -I. as part of the fixes
     for CVE-2016-1238
 
- -- Dominic Hargreaves <dom@earth.li>  Sat, 09 Jul 2016 11:24:14 +0200
+ -- Dominic Hargreaves <dom@earth.li>  Mon, 25 Jul 2016 09:34:18 +0100
 
 cdbs (0.4.130) unstable; urgency=medium
 
-- 
2.1.4

