From 88a7a29baae52a0e5b5164a538c89813d13802f7 Mon Sep 17 00:00:00 2001
From: Kumar Appaiah <akumar@debian.org>
Date: Wed, 15 Oct 2008 23:04:01 -0500
Subject: [PATCH] Fix wrapper for compilers with host name.

---
 Makefile |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Makefile b/Makefile
index 41f3cf6..9c033bf 100644
--- a/Makefile
+++ b/Makefile
@@ -28,7 +28,7 @@ install:	$(APT_BUILD_WRAPPER) $(APT_BUILD)
 	done
 ifneq ($(HOST_TYPE),)
 	for prog in gcc g++; do \
-		$(LN_S) $(APT_BUILD_WRAPPER) $(APT_BUILD_DIR)/$$prog-$(HOST_TYPE); \
+		$(LN_S) $(APT_BUILD_WRAPPER) $(APT_BUILD_DIR)/$(HOST_TYPE)-$$prog; \
 	done
 endif
 
-- 
1.5.6.5

