--- /usr/share/emacs/site-lisp/auctex/configure.orig	2004-09-06 20:57:01.000000000 +0200
+++ /usr/share/emacs/site-lisp/auctex/configure	2004-09-06 20:55:48.000000000 +0200
@@ -1562,9 +1562,7 @@
   rm -f ${OUTPUT}
 
   if test "$valid_build_dir" = "no"; then
-    { { echo "$as_me:$LINENO: error: Build directory inside load-path!  Aborting!" >&5
-echo "$as_me: error: Build directory inside load-path!  Aborting!" >&2;}
-   { (exit 1); exit 1; }; }
+    echo "$as_me:$LINENO: result: no, but who cares" >&5
   else
     echo "$as_me:$LINENO: result: yes" >&5
 echo "${ECHO_T}yes" >&6
