diff -ru autotalent-0.2.pristine/debian/control autotalent-0.2/debian/control
--- autotalent-0.2.pristine/debian/control	2014-06-18 13:12:29.000000000 +0100
+++ autotalent-0.2/debian/control	2015-05-13 16:32:58.398809002 +0100
@@ -30,9 +30,9 @@
  between major and minor or to change the musical mode.
  .
  Autotalent was designed from the ground up to process musical melodies,
- whether sung by voice or played on some kind of instrument. As long as
- one gives it a monophonic source (i.e. a melody in a single, 1-channel
- track), he can usually expects pretty good results. So this means that
+ whether sung or played on some kind of instrument. As long as you
+ give it a monophonic source (i.e. a melody in a single, 1-channel
+ track), you can usually expect pretty good results. So this means that
  if you're in the business of processing vocal harmonies, you should
  apply it to each monophonic vocal part separately. If in the spirit of
  experimentation you're daring enough to try tuning or pitch shifting
