diff -Naurp debian-installer.orig/debian/control debian-installer/debian/control
--- debian-installer.orig/debian/control	2014-08-02 00:09:06.000000000 +0000
+++ debian-installer/debian/control	2014-08-25 05:43:42.460472053 +0000
@@ -93,6 +93,7 @@ Build-Depends:
 #	still called debian-boot..
 	syslinux [i386 amd64],
 	isolinux [i386 amd64],
+	syslinux-utils [i386 amd64],
 	pxelinux [i386 amd64],
 	syslinux-common (>= 3:6) [i386 amd64],
 #		With isohybrid.
