diff -ur dbconfig-common-1.8.47+nmu1.orig/dpkg/common dbconfig-common-1.8.47+nmu1/dpkg/common
--- dbconfig-common-1.8.47+nmu1.orig/dpkg/common	2010-03-16 19:35:54.000000000 +0100
+++ dbconfig-common-1.8.47+nmu1/dpkg/common	2012-05-21 19:25:56.153589161 +0200
@@ -207,12 +207,14 @@
 	fi
 
 	# if dbserver is unset, that means localhost.
-	if [ -z "$dbc_dbserver" ] || [ "$dbc_dbserver" = "localhost" ]; then
-		# if the server is local, only allow connections from localhost
-		dbc_dballow="localhost"
-	else
-		# otherwise, only tell the remote database to allow from us
-		dbc_dballow=`hostname -f`
+	if [ -z "$dbc_dballow" ]; then
+		if [ -z "$dbc_dbserver" ] || [ "$dbc_dbserver" = "localhost" ]; then
+			# if the server is local, only allow connections from localhost
+			dbc_dballow="localhost"
+		else
+			# otherwise, only tell the remote database to allow from us
+			dbc_dballow=`hostname -f`
+		fi
 	fi
 }
 