Index: README-contrib
===================================================================
--- README-contrib	(révision 5385)
+++ README-contrib	(copie de travail)
@@ -51,15 +51,21 @@
 
 If you're interested in ongoing maintenance, once you've shown that
 you've mastered the style of the manual with a couple accepted
-patches, you can be given SVN access. If you have already
+patches, you can be given SVN access. If you already have
 access to the SVN server for other reasons, do not use it unless
 authorized to do so.
 
-Instructions on how to get the SVN version of this software, including
-how to get SVN access, can be found at
-<URL:http://svn.debian.org/>.
+This manual is a part of the Debian Documentation Project (DDP).
+Information about SVN access for DDP can be found at
+<http://www.debian.org/doc/cvs>.
 
+If you just want to check out the current SVN version of this
+manual to create patches against, you can use the following
+command to do so:
 
+svn co svn://svn.debian.org/ddp/manuals/trunk/developers-reference
+
+
 * Translators
 
 We have tried to keep language-independant bits of text in common.ent.
