diff -Nru eggdrop-1.6.20/debian/changelog eggdrop-1.6.20/debian/changelog
--- eggdrop-1.6.20/debian/changelog	2011-07-24 20:47:00.000000000 +0400
+++ eggdrop-1.6.20/debian/changelog	2013-09-30 11:43:58.000000000 +0400
@@ -1,3 +1,10 @@
+eggdrop (1.6.20-1.1) unstable; urgency=high
+
+  * Non-maintainer upload.
+  * Help configure script to find multiarchified Tcl library.
+
+ -- Sergei Golovan <sgolovan@debian.org>  Mon, 30 Sep 2013 11:43:55 +0400
+
 eggdrop (1.6.20-1) unstable; urgency=high
 
   * NMUs acknowledged; thanks guys!
diff -Nru eggdrop-1.6.20/debian/control eggdrop-1.6.20/debian/control
--- eggdrop-1.6.20/debian/control	2011-07-24 11:18:21.000000000 +0400
+++ eggdrop-1.6.20/debian/control	2013-09-30 11:45:31.000000000 +0400
@@ -1,6 +1,6 @@
 Source: eggdrop
 Section: net
-Build-Depends: tcl8.5-dev, libz-dev, debhelper (>= 8), cdbs
+Build-Depends: tcl8.5-dev (>= 8.5.15), libz-dev, debhelper (>= 8), cdbs, dpkg-dev (>= 1.16.0)
 Priority: extra
 Maintainer: Guilherme de Siqueira Pastore <gpastore@debian.org>
 Standards-Version: 3.9.2
diff -Nru eggdrop-1.6.20/debian/rules eggdrop-1.6.20/debian/rules
--- eggdrop-1.6.20/debian/rules	2011-07-24 11:12:53.000000000 +0400
+++ eggdrop-1.6.20/debian/rules	2013-09-30 11:49:18.000000000 +0400
@@ -1,12 +1,15 @@
 #!/usr/bin/make -f
 # -*- mode: makefile; coding: utf-8 -*-
 
+DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
+
 include /usr/share/cdbs/1/rules/debhelper.mk
 include /usr/share/cdbs/1/class/autotools.mk
 
 DEB_CONFIGURE_SCRIPT_ENV :=
 DEB_INSTALL_MANPAGES_eggdrop-data := doc/man1/eggdrop.1
 DEB_MAKE_INSTALL_TARGET := install prefix=$(DEB_DESTDIR)/usr
+DEB_CONFIGURE_EXTRA_FLAGS := --with-tcllib=/usr/lib/$(DEB_HOST_MULTIARCH)/libtcl8.5.so --with-tclinc=/usr/include/tcl8.5/tcl.h
 
 configure/eggdrop::
 	$(MAKE) config
