diff --git a/debian/rules b/debian/rules
index 752e8b0..48b1117 100755
--- a/debian/rules
+++ b/debian/rules
@@ -25,7 +25,7 @@ CFLAGS += $(CPPFLAGS)
 
 override_dh_auto_build:
        dh_auto_build -Smakefile -- -C build/gcc $(BUILD_TARGET) shared java
-       [ "$(BUILD_TARGET)" = "astyle" ] || mv bin/$(BUILD_TARGET) bin/astyle
+       [ "$(BUILD_TARGET)" = "astyle" ] || mv build/gcc/bin/$(BUILD_TARGET) build/gcc/bin/astyle
 
 override_dh_auto_clean:
        dh_auto_clean
