diff -ru dns2tcp-0.5.2.pristine/debian/control dns2tcp-0.5.2/debian/control
--- dns2tcp-0.5.2.pristine/debian/control	2010-06-18 00:34:28.000000000 +0100
+++ dns2tcp-0.5.2/debian/control	2012-06-28 14:46:29.307392873 +0100
@@ -9,8 +9,7 @@
 Package: dns2tcp
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}
-Description: TCP over DNS tunnel client and server
- dns2tcp is a set of tools allowing to encapsulate a TCP session in DNS
- packets. This type of encapsulation generates smaller packets compeared to
- ip-over-DNS tools, thus allows a better throughput. The client does not need
- root priledges.
+Description: TCP-over-DNS tunnel server and client
+ dns2tcp is a set of tools to encapsulate a TCP session in DNS packets.
+ This type of encapsulation generates smaller packets than IP-over-DNS,
+ improving throughput. The client does not need root privileges.
