--- /usr/bin/dpkg-maintscript-helper.orig	2014-02-14 09:49:53.000000000 +0100
+++ /usr/bin/dpkg-maintscript-helper	2014-02-14 09:51:47.000000000 +0100
@@ -394,9 +394,9 @@
 		package="$1"
 		file="$2"
 		if ! dpkg-query -L "$package" | grep -q -x "$file"; then
-			return 1
+			exit 1
 		fi
-		return 0
+		exit 0
 	' check-files-ownership "$PACKAGE" || \
 		error "directory '$PATHNAME' contains files not owned by" \
 		      "package $PACKAGE, cannot switch to symlink"
