--- changelog	2008-08-17 08:39:08.000000000 +0200
+++ changelog.orig	2008-08-17 08:38:53.000000000 +0200
@@ -1,6 +1,6 @@
 deborphan (1.7.26) unstable; urgency=high
 
-  * Urgency set to high as it solves an RC bug.
+  * Urgency set to high as is solves an RC bug.
   * Check whether get_pkg_info() correctly detected a line as status line
     before abortion.  (Closes: #493896)
   * Translation updates. Many thanks to:
