From 160eb9e209b5ed99fab7f19dff92f3dbd707e311 Mon Sep 17 00:00:00 2001
From: Satyam Zode <satyamzode@gmail.com>
Date: Sat, 18 Jun 2016 21:55:28 +0530
Subject: [PATCH 2/4] Added dependencies for argument completion

---
 debian/control | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/debian/control b/debian/control
index 19619d7..b24fbef 100644
--- a/debian/control
+++ b/debian/control
@@ -7,9 +7,11 @@ Uploaders:
  Mattia Rizzolo <mattia@debian.org>,
  Reiner Herrmann <reiner@reiner-h.de>,
 Build-Depends:
+ bash-completion,
  binutils-multiarch,
  debhelper (>= 9),
  dh-python,
+ python-argcomplete,
  python3-all,
  python3-debian,
  python3-docutils,
-- 
2.1.4

