From d7192fb92a4fca051bee2b13548878923c34995e Mon Sep 17 00:00:00 2001
From: Satyam Zode <satyamzode@gmail.com>
Date: Sat, 18 Jun 2016 22:05:19 +0530
Subject: [PATCH 3/4] Added debian/clean file

---
 debian/clean | 3 +++
 1 file changed, 3 insertions(+)
 create mode 100644 debian/clean

diff --git a/debian/clean b/debian/clean
new file mode 100644
index 0000000..6e1cd10
--- /dev/null
+++ b/debian/clean
@@ -0,0 +1,3 @@
+rm -f debian/diffoscope.1
+dh_clean -O--buildsystem=pybuild
+debian/diffoscope.bash-completion
-- 
2.1.4

