diff -uNra a/debian/control b/debian/control
--- a/debian/control	2024-04-08 03:42:37.811907563 +0000
+++ b/debian/control	2024-04-08 03:43:05.503973197 +0000
@@ -25,7 +25,7 @@
  for the current architecture.
 
 Package: default-d-compiler
-Architecture: amd64 arm64 i386 armhf s390x riscv64 ppc64el x32
+Architecture: amd64 arm64 i386 armhf s390x riscv64 ppc64el x32 loong64
 Depends: gdc (>= 4:12) [armhf s390x ppc64el x32],
          ldc (>= 1:1.36) [amd64 arm64 i386 riscv64],
          ${misc:Depends}
