--- awstats.pl.orig	2010-04-29 20:34:38.000000000 +0000
+++ awstats.pl	2010-04-29 20:34:08.000000000 +0000
@@ -1742,17 +1742,23 @@
 		if ( $searchdir && $searchdir !~ /[\\\/]$/ ) { $searchdir .= "/"; }
 		if ( open( CONFIG, "$searchdir$PROG.$SiteConfig.conf" ) ) {
 			$FileConfig = "$searchdir$PROG.$SiteConfig.conf";
 			$FileSuffix = ".$SiteConfig";
 			last;
 		}
+		elsif ( -e "$searchdir$PROG.$SiteConfig.conf" ) {
+			error("Couldn't open config file \"$searchdir$PROG.$SiteConfig.conf\": $!" );
+		}
 		if ( open( CONFIG, "$searchdir$PROG.conf" ) ) {
 			$FileConfig = "$searchdir$PROG.conf";
 			$FileSuffix = '';
 			last;
 		}
+		elsif ( -e "$searchdir$PROG.conf" ) {
+			error("Couldn't open config file \"$searchdir$PROG.conf\": $!" );
+		}
 	}
 	if ( !$FileConfig ) {
 		error(
 "Couldn't open config file \"$PROG.$SiteConfig.conf\" nor \"$PROG.conf\" after searching in path \""
 			  . join( ',', @PossibleConfigDir )
 			  . "\": $!" );
