--- libparams-classify-perl_0.013-5_i386.build	2012-03-06 18:41:33.000000000 +0000
+++ libparams-classify-perl_0.013-5_i386.build.AFTER	2012-03-06 18:41:04.000000000 +0000
@@ -22,9 +22,9 @@
 Creating new 'Build' script for 'Params-Classify' version '0.013'
    dh_auto_build
 Building Params-Classify
-cc -I/usr/lib/perl/5.14/CORE -DXS_VERSION="0.013" -DVERSION="0.013" -fPIC -c -D_REENTRANT -D_GNU_SOURCE -DDEBIAN -fstack-protector -fno-strict-aliasing -pipe -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -g -O2 -fstack-protector --param=ssp-buffer-size=4 -Wformat -Wformat-security -Werror=format-security -g -O2 -fstack-protector --param=ssp-buffer-size=4 -Wformat -Wformat-security -Werror=format-security -o lib/Params/Classify.o lib/Params/Classify.c
+cc -I/usr/lib/perl/5.14/CORE -DXS_VERSION="0.013" -DVERSION="0.013" -fPIC -c -D_REENTRANT -D_GNU_SOURCE -DDEBIAN -fstack-protector -fno-strict-aliasing -pipe -I/usr/local/include -D_LARGEFILE_SOURCE -D_FILE_OFFSET_BITS=64 -g -O2 -fstack-protector --param=ssp-buffer-size=4 -Wformat -Wformat-security -Werror=format-security -g -O2 -fstack-protector --param=ssp-buffer-size=4 -Wformat -Wformat-security -Werror=format-security -D_FORTIFY_SOURCE=2 -o lib/Params/Classify.o lib/Params/Classify.c
 ExtUtils::Mkbootstrap::Mkbootstrap('blib/arch/auto/Params/Classify/Classify.bs')
-cc -shared -L/usr/local/lib -fstack-protector -o blib/arch/auto/Params/Classify/Classify.so lib/Params/Classify.o
+cc -Wl,-z,relro -shared -L/usr/local/lib -fstack-protector -o blib/arch/auto/Params/Classify/Classify.so lib/Params/Classify.o
    dh_auto_test
 t/blessed.t ...... ok
 t/blessed_pp.t ... ok
@@ -40,7 +40,7 @@
 t/ref.t .......... ok
 t/ref_pp.t ....... ok
 All tests successful.
-Files=13, Tests=4749,  2 wallclock secs ( 0.54 usr  0.06 sys +  1.45 cusr  0.08 csys =  2.13 CPU)
+Files=13, Tests=4749,  2 wallclock secs ( 0.57 usr  0.02 sys +  1.44 cusr  0.08 csys =  2.11 CPU)
 Result: PASS
  fakeroot debian/rules binary
 dh binary
@@ -107,5 +107,5 @@
 gpg: Invalid passphrase; please try again ...
 gpg: Invalid passphrase; please try again ...
 gpg: skipped "Dominic Hargreaves <dom@earth.li>": bad passphrase
-gpg: /tmp/debsign.I1xJ7y5o/libparams-classify-perl_0.013-5.dsc: clearsign failed: bad passphrase
+gpg: /tmp/debsign.I4im40yg/libparams-classify-perl_0.013-5.dsc: clearsign failed: bad passphrase
 debsign: gpg error occurred!  Aborting....
