diff -ruN dpkg-1.16.2/cputable dpkg-1.16.3/cputable
--- dpkg-1.16.2/cputable	2012-03-19 07:40:07.000000000 +0100
+++ dpkg-1.16.3/cputable	2012-04-15 18:52:27.000000000 +0200
@@ -38,3 +38,4 @@
 sh4eb		sh4eb		sh4eb			32	big
 sparc		sparc		sparc			32	big
 sparc64		sparc64		sparc64			64	big
+x32		x32		x32			32	little
diff -ruN dpkg-1.16.2/ostable dpkg-1.16.3/ostable
--- dpkg-1.16.2/ostable	2012-03-19 07:40:08.000000000 +0100
+++ dpkg-1.16.3/ostable	2012-04-15 18:50:27.000000000 +0200
@@ -20,6 +20,7 @@
 gnueabi-linux		linux-gnueabi		linux[^-]*-gnueabi
 gnuspe-linux		linux-gnuspe		linux[^-]*-gnuspe
 gnulp-linux		linux-gnulp		linux[^-]*-gnulp
+gnux32-linux		linux-gnux32		linux[^-]*-gnux32
 gnu-linux		linux-gnu		linux[^-]*(-gnu.*)?
 gnu-kfreebsd		kfreebsd-gnu		kfreebsd[^-]*(-gnu.*)?
 gnu-knetbsd		knetbsd-gnu		knetbsd[^-]*(-gnu.*)?
diff -ruN dpkg-1.16.2/triplettable dpkg-1.16.3/triplettable
--- dpkg-1.16.2/triplettable	2012-03-19 07:40:10.000000000 +0100
+++ dpkg-1.16.3/triplettable	2012-04-15 18:50:49.000000000 +0200
@@ -9,6 +9,7 @@
 gnueabi-linux-arm	armel
 gnuspe-linux-powerpc	powerpcspe
 gnulp-linux-i386	lpia
+gnux32-linux-x86_64	x32
 gnu-linux-<cpu>		<cpu>
 gnu-kfreebsd-<cpu>	kfreebsd-<cpu>
 gnu-knetbsd-<cpu>	knetbsd-<cpu>
