diff --git a/pkgs.dbk b/pkgs.dbk
index 5a34b95..a21c24b 100644
--- a/pkgs.dbk
+++ b/pkgs.dbk
@@ -2741,11 +2741,11 @@ updates through <literal>unstable</literal> are possible, if the upload via
 <literal>unstable</literal> does not pull in any new dependencies.
 </para>
 <para>
-Version numbers are usually selected by adding the codename of the
-<literal>testing</literal> distribution and a running number, like
-<literal>1.2squeeze1</literal> for the first upload through
-<literal>testing-proposed-updates</literal> of package version
-<literal>1.2</literal>.
+Version numbers are usually selected by appending
+<literal>+deb</literal><replaceable>X</replaceable><literal>u1</literal>,
+where <replaceable>X</replaceable> is the major release number and
+<replaceable>Y</replaceable> is a counter starting at <literal>1</literal>.
+e.g. <literal>1:2.4.3-4+deb8u1</literal>.
 </para>
 <para>
 Please make sure you didn't miss any of these items in your upload:
