From 775e5e40b375eaec94f9dff21602c8e65805d667 Mon Sep 17 00:00:00 2001
From: Sven Joachim <svenjoac@gmx.de>
Date: Mon, 29 Oct 2018 12:40:24 +0100
Subject: [PATCH] Remove obsolete wheezy keys

---
 debian/debian-archive-keyring.maintscript | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/debian/debian-archive-keyring.maintscript b/debian/debian-archive-keyring.maintscript
index eb97225..2d9c1f4 100644
--- a/debian/debian-archive-keyring.maintscript
+++ b/debian/debian-archive-keyring.maintscript
@@ -1,2 +1,4 @@
 rm_conffile /etc/apt/trusted.gpg.d/debian-archive-squeeze-automatic.gpg 2017.5~~
 rm_conffile /etc/apt/trusted.gpg.d/debian-archive-squeeze-stable.gpg 2017.5~~
+rm_conffile /etc/apt/trusted.gpg.d/debian-archive-wheezy-automatic.gpg 2018.2~~
+rm_conffile /etc/apt/trusted.gpg.d/debian-archive-wheezy-stable.gpg 2018.2~~
-- 
2.19.1

