diff -Naur debhelper-7.0.17.orig/debian/control debhelper-7.0.17/debian/control
--- debhelper-7.0.17.orig/debian/control	2008-12-04 16:29:15.000251171 +0100
+++ debhelper-7.0.17/debian/control	2008-12-04 16:29:32.137274831 +0100
@@ -5,6 +5,7 @@
 Build-Depends: po4a (>= 0.24), man-db (>= 2.5.1), file (>= 3.23)
 Standards-Version: 3.8.0
 Vcs-Git: git://git.debian.org/git/debhelper/debhelper.git
+Vcs-Browser: http://git.debian.org/?p=debhelper/debhelper.git;a=summary
 Homepage: http://kitenet.net/~joey/code/debhelper/
 
 Package: debhelper
