diff -u acpitool-0.4.7/debian/control acpitool-0.4.7/debian/control
--- acpitool-0.4.7/debian/control
+++ acpitool-0.4.7/debian/control
@@ -8,7 +8,7 @@
 Package: acpitool
 Architecture: any
 Depends: ${shlibs:Depends}
-Recommends: acpid
+Recommends: acpid [amd64 i386 ia64]
 Description: a small, convenient command-line ACPI client
  AcpiTool is (yet another) Linux ACPI client. It's a small command-line
  application, intended to be a replacement for the apm tool. The primary 
diff -u acpitool-0.4.7/debian/changelog acpitool-0.4.7/debian/changelog
--- acpitool-0.4.7/debian/changelog
+++ acpitool-0.4.7/debian/changelog
@@ -1,3 +1,11 @@
+acpitool (0.4.7-1.4) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Recommend acpid only on supported archs [amd64 i386 ia64]. Thanks Robert
+    Lemmen. (Closes: #436637). 
+
+ -- Amaya Rodrigo Sastre <amaya@debian.org>  Sun, 30 Mar 2008 21:41:08 +0200
+
 acpitool (0.4.7-1.3) unstable; urgency=medium
 
   * Non-maintainer upload.
