--- chrony_0_20160617	2016-06-17 20:37:19.000000000 +0200
+++ chrony_20160813stretch	2016-08-13 17:50:40.316071778 +0200
@@ -27,6 +27,8 @@
 
 . /lib/lsb/init-functions
 
+[ -r "/etc/default/$NAME" ] && . "/etc/default/$NAME"
+
 putonline ()
 {   # Do we have a default route?  If so put chronyd online.
     if ip r 2>/dev/null | grep -q '^default'; then
@@ -45,7 +47,7 @@
 
 case "$1" in
     start)
-	start-stop-daemon --start --verbose --exec $DAEMON
+	start-stop-daemon --start --verbose --exec $DAEMON -- $OPTS
 	case "$?" in
 		0) # daemon successfully started
 			putonline
@@ -66,7 +68,7 @@
 	echo -n "Restarting $DESC: "
 	start-stop-daemon --stop --quiet --exec $DAEMON
 	sleep 1
-	start-stop-daemon --start --verbose --exec $DAEMON -- -r
+	start-stop-daemon --start --verbose --exec $DAEMON -- -r $OPTS
 	case "$?" in
 		0) # daemon successfully started
 			putonline
