Sólo en new/ccache-3.1.5/debian/: ccache.triggers
diff -ur orig/ccache-3.1.5/debian//ccache.triggers.in new/ccache-3.1.5/debian//ccache.triggers.in
--- orig/ccache-3.1.5/debian//ccache.triggers.in	2011-06-25 23:54:41.000000000 +0200
+++ new/ccache-3.1.5/debian//ccache.triggers.in	2011-07-05 02:04:11.000000000 +0200
@@ -1,2 +1,4 @@
+interest /usr/bin/cc
+interest /usr/bin/c++
 interest /usr/lib/gcc
 interest /usr/lib/%DEB_HOST_MULTIARCH%/gcc
diff -ur orig/ccache-3.1.5/debian//changelog new/ccache-3.1.5/debian//changelog
--- orig/ccache-3.1.5/debian//changelog	2011-06-25 23:54:41.000000000 +0200
+++ new/ccache-3.1.5/debian//changelog	2011-07-05 02:05:24.000000000 +0200
@@ -1,3 +1,9 @@
+ccache (3.1.5-2+santa1) UNRELEASED; urgency=low
+
+  * Add triggers for /usr/bin/cc and /usr/bin/c++.
+
+ -- José Manuel Santamaría Lema <panfaust@gmail.com>  Tue, 05 Jul 2011 02:04:30 +0200
+
 ccache (3.1.5-2) unstable; urgency=low
 
   * Update dpkg triggers and update-ccache-symlinks to find compilers at
