diff -ur orig/gcc-defaults-1.105/debian//changelog new/gcc-defaults-1.105+nmu0~santa2/debian//changelog
--- orig/gcc-defaults-1.105/debian//changelog	2011-06-16 23:53:08.000000000 +0200
+++ new/gcc-defaults-1.105+nmu0~santa2/debian//changelog	2011-07-05 23:27:05.324000747 +0200
@@ -1,3 +1,10 @@
+gcc-defaults (1.105+nmu0~santa2) UNRELEASED; urgency=low
+
+  * Activate triggers for /usr/bin/cc and /usr/bin/c++ in gcc and g++
+    postinst/prerm scripts respectively.
+
+ -- José Manuel Santamaría Lema <panfaust@gmail.com>  Tue, 05 Jul 2011 03:16:21 +0200
+
 gcc-defaults (1.105) unstable; urgency=low
 
   * Build libgcj-bc for multiarch. Closes: #630417.
diff -ur orig/gcc-defaults-1.105/debian//gcc.postinst.in new/gcc-defaults-1.105+nmu0~santa2/debian//gcc.postinst.in
--- orig/gcc-defaults-1.105/debian//gcc.postinst.in	2006-08-19 14:57:11.000000000 +0200
+++ new/gcc-defaults-1.105+nmu0~santa2/debian//gcc.postinst.in	2011-07-05 23:34:25.492000813 +0200
@@ -19,6 +19,8 @@
     --install /usr/bin/c99 c99 /usr/bin/c99-gcc 20 \
     --slave /usr/share/man/man1/c99.1.gz c99.1.gz /usr/share/man/man1/c99-gcc.1.gz
 
+dpkg-trigger /usr/bin/cc
+
 # remove f77 alternative left over from an old egcs version ...
 if [ -h /etc/alternatives/f77 -a ! -e /etc/alternatives/f77 ]; then
     rm -f /etc/alternatives/f77
diff -ur orig/gcc-defaults-1.105/debian//gcc.prerm new/gcc-defaults-1.105+nmu0~santa2/debian//gcc.prerm
--- orig/gcc-defaults-1.105/debian//gcc.prerm	2006-05-07 11:32:50.000000000 +0200
+++ new/gcc-defaults-1.105+nmu0~santa2/debian//gcc.prerm	2011-07-05 23:29:54.856000771 +0200
@@ -6,6 +6,8 @@
     update-alternatives --quiet --remove c99 /usr/bin/c99-gcc
 fi
 
+dpkg-trigger /usr/bin/cc
+
 #DEBHELPER#
 
 exit 0
diff -ur orig/gcc-defaults-1.105/debian//g++.postinst.in new/gcc-defaults-1.105+nmu0~santa2/debian//g++.postinst.in
--- orig/gcc-defaults-1.105/debian//g++.postinst.in	2006-08-19 15:09:21.000000000 +0200
+++ new/gcc-defaults-1.105+nmu0~santa2/debian//g++.postinst.in	2011-07-05 03:14:38.000000000 +0200
@@ -32,6 +32,8 @@
     --install /usr/bin/c++ c++ /usr/bin/g++ 20 \
     @GFDL@--slave /usr/share/man/man1/c++.1.gz c++.1.gz /usr/share/man/man1/g++.1.gz
 
+dpkg-trigger /usr/bin/c++
+
 #DEBHELPER#
 
 exit 0
diff -ur orig/gcc-defaults-1.105/debian//g++.prerm new/gcc-defaults-1.105+nmu0~santa2/debian//g++.prerm
--- orig/gcc-defaults-1.105/debian//g++.prerm	2001-02-27 02:28:32.000000000 +0100
+++ new/gcc-defaults-1.105+nmu0~santa2/debian//g++.prerm	2011-07-05 23:28:20.356000759 +0200
@@ -4,6 +4,8 @@
     update-alternatives --remove c++ /usr/bin/g++
 fi
 
+dpkg-trigger /usr/bin/c++
+
 #DEBHELPER#
 
 exit 0
diff -ur orig/gcc-defaults-1.105/debian//README.Debian new/gcc-defaults-1.105+nmu0~santa2/debian//README.Debian
--- orig/gcc-defaults-1.105/debian//README.Debian	2011-04-29 17:10:27.000000000 +0200
+++ new/gcc-defaults-1.105+nmu0~santa2/debian//README.Debian	2011-07-05 23:34:52.752000817 +0200
@@ -45,7 +45,7 @@
 architecture. To use the other compiler, simply set CC=gcc-4.1, or
 similar.
 
-The default compiler versions for Debian GNU/Linux on i386 are
+The default compiler versions for Debian GNU/Linux on amd64 are
 (minor version numbers omitted):
 
 	cpp		: cpp-4.6
