diff -Nru cryptsetup-1.0.6~pre1.orig/debian/initramfs/cryptroot-hook cryptsetup-1.0.6~pre1/debian/initramfs/cryptroot-hook
--- cryptsetup-1.0.6~pre1.orig/debian/initramfs/cryptroot-hook	2007-12-10 15:01:55.000000000 +0000
+++ cryptsetup-1.0.6~pre1/debian/initramfs/cryptroot-hook	2007-12-10 15:05:02.000000000 +0000
@@ -365,7 +365,7 @@
 
 # Unless MODULES = "dep", we always add a basic subset of modules/tools
 if [ "$MODULES" != "dep" ]; then
-	for mod in dm_mod dm_crypt aes sha256 cbc; do
+	for mod in dm_mod dm_crypt aes blowfish serpent twofish sha256 cbc; do
 		manual_add_modules $mod
 	done
 	copy_exec /sbin/cryptsetup /sbin
