--- orig/usr/sbin/update-command-not-found	2009-11-02 21:29:50.000000000 +0100
+++ patch/usr/sbin/update-command-not-found	2014-01-12 15:42:08.264747814 +0100
@@ -48,7 +48,11 @@
 		fname = os.path.basename(fname)
 		
 		for package in packages.split(','):
-			section, package = package.strip().rsplit('/', 1)
+
+			if len(package.strip().rsplit('/', 1)) >= 2:
+				section, package = package.strip().rsplit('/', 1)
+			else:
+				section = '_nosection'
 			if len(section.split('/')) == 2:
 				component, section = section.split('/')
 			else:
