--- dictionaries-common.templates.orig	2014-07-01 12:32:04.575647866 +0200
+++ dictionaries-common.templates	2014-07-01 12:35:32.687307259 +0200
@@ -21,18 +21,18 @@
 
 Template: dictionaries-common/invalid_debconf_value
 Type: error
-_Description: Invalid configuration setting for ${value}
+_Description: Invalid configuration setting for ${question}
  An invalid value has been found for a configuration setting for
- dictionaries-common. It does not correspond to any installed package
- in the system.
+ dictionaries-common. "${value}" does not correspond to any installed
+ package in the system.
  .
  That is usually caused by problems at some time during packages
- installation, where the package providing [${value}] was selected for
+ installation, where the package providing "${value}" was selected for
  installation but finally not installed because of errors in other
  packages.
  .
  To fix this error, reinstall (or install) the package that provides
- the missing value. Then, if you don't want this package on
+ the invalid value. Then, if you don't want this package on
  your system, remove it, which will also remove its configuration
  settings in the "debconf" database. A choices menu
  will be shown after this message in order to leave the system in a
