Index: accountsservice/src/daemon.c
===================================================================
--- accountsservice.orig/src/daemon.c	2011-04-28 19:12:31.452838301 +0200
+++ accountsservice/src/daemon.c	2011-04-28 19:14:46.413498112 +0200
@@ -418,7 +418,7 @@ reload_ck_history (Daemon *daemon)
         gint standard_out;
         GIOChannel *channel;
 
-        command = g_strdup ("ck-history --frequent --session-type=''");
+        command = g_strdup ("ck-history --frequent --session-type='x11'");
         g_debug ("running '%s'", command);
         error = NULL;
         if (! g_shell_parse_argv (command, NULL, &argv, &error)) {
