=== modified file 'apt-pkg/algorithms.cc'
--- apt-pkg/algorithms.cc	2008-10-03 18:15:06 +0000
+++ apt-pkg/algorithms.cc	2008-10-26 11:40:22 +0000
@@ -1342,7 +1342,12 @@
 
       (*I)->Finished();
 
-      _error->Warning(_("Failed to fetch %s  %s\n"),(*I)->DescURI().c_str(),
+	  ::URI uri((*I)->DescURI());
+	  uri.User.clear();
+	  uri.Password.clear();
+	  string descUri = string(uri);
+
+      _error->Warning(_("Failed to fetch %s  %s\n"), descUri.c_str(),
 	      (*I)->ErrorText.c_str());
 
       if ((*I)->Status == pkgAcquire::Item::StatTransientNetworkError) 

