Index: acpid-1.0.6/debian/acpid.default
===================================================================
--- acpid-1.0.6.orig/debian/acpid.default	2007-11-02 20:14:57.000000000 +0000
+++ acpid-1.0.6/debian/acpid.default	2007-11-02 20:15:01.000000000 +0000
@@ -6,4 +6,7 @@
 # MODULES may be uncommented to load "none", contain the string "all"
 # to load all acpi related modules or simply a space seperated list
 # of modules to be probed.
-MODULES="battery ac processor button fan thermal"
+#MODULES="battery ac processor button fan thermal"
+# using all allows us to drop extra modules in drivers/acpi and have them
+# be autoloaded.
+MODULES="all"
