diff -rup ../einstein-2.0.dfsg.2.orig/main.cpp ./main.cpp
--- ../einstein-2.0.dfsg.2.orig/main.cpp	2006-10-29 21:32:30.000000000 +0100
+++ ./main.cpp	2006-10-29 21:39:14.000000000 +0100
@@ -31,6 +31,12 @@ static void initScreen()
     SDL_SetColorKey(mouse, SDL_SRCCOLORKEY, SDL_MapRGB(mouse->format, 0, 0, 0));
     screen.setMouseImage(mouse);
     SDL_FreeSurface(mouse);
+    
+    SDL_Surface *icon = loadImage(L"icon.bmp");
+    SDL_SetColorKey(icon, SDL_SRCCOLORKEY, SDL_MapRGB(icon->format, 0, 0, 0));
+    SDL_WM_SetIcon(icon, NULL);
+    SDL_FreeSurface(icon);
+    
     SDL_WM_SetCaption("Einstein", NULL);
 
 #ifdef __APPLE__
diff -rup ../einstein-2.0.dfsg.2.orig/res/resources.descr ./res/resources.descr
--- ../einstein-2.0.dfsg.2.orig/res/resources.descr	2006-10-29 21:32:30.000000000 +0100
+++ ./res/resources.descr	2006-10-29 21:35:32.000000000 +0100
@@ -1,6 +1,7 @@
 priority = 100
 
 resources = {
+    { name = "icon.bmp" }
     { name = "cursor.bmp" }
     { name = "rain.bmp" }
     { name = "tile.bmp" }
