--- dh_compress.orig	2014-03-01 15:52:37.978492343 +0900
+++ dh_compress	2014-03-01 15:54:56.453046988 +0900
@@ -113,7 +113,7 @@
 			find usr/share/doc \\
 				\\( -type d -name _sources -prune -false \\) -o \\
 				-type f \\( -size +4k -or -name "changelog*" -or -name "NEWS*" \\) \\
-				\\( -name changelog.html -or ! -iname "*.htm*" \\) \\
+				\\( -name changelog.html -or ! -iname "*.htm*" -or ! -iname "*.xhtml"\\) \\
 				! -iname "*.gif" ! -iname "*.png" ! -iname "*.jpg" \\
 				! -iname "*.jpeg" ! -iname "*.gz" ! -iname "*.taz" \\
 				! -iname "*.tgz" ! -iname "*.z" ! -iname "*.bz2" \\
