diff -ruN docbook-xsl-1.73.2.dfsg.1.orig/manpages/other.xsl docbook-xsl-1.73.2.dfsg.1/manpages/other.xsl
--- docbook-xsl-1.73.2.dfsg.1.orig/manpages/other.xsl	2007-06-23 06:07:45.000000000 -0300
+++ docbook-xsl-1.73.2.dfsg.1/manpages/other.xsl	2008-12-09 16:27:16.000000000 -0200
@@ -243,15 +243,6 @@
   </xsl:call-template>
 </xsl:template>
 
-<xsl:template name="escape.apostrophe">
-  <xsl:param name="content"/>
-  <xsl:call-template name="string.subst">
-    <xsl:with-param name="string" select="$content"/>
-    <xsl:with-param name="target">'</xsl:with-param>
-    <xsl:with-param name="replacement">\'</xsl:with-param>
-  </xsl:call-template>
-</xsl:template>
-
 <xsl:template name="convert.nobreak-space">
   <xsl:param name="content"/>
   <xsl:call-template name="string.subst">
