diff --git a/debdelta/debdelta b/debdelta/debdelta
index 7618e10..eda1004 100755
--- a/debdelta/debdelta
+++ b/debdelta/debdelta
@@ -4575,7 +4575,7 @@ def delta_upgrade_(args):
       assert(a[0] == 'http')
 
       #delta name
-      delta_name=p.name+'_'+version_mangle(installed_version)+\
+      delta_name=p.name.split(':')[0]+'_'+version_mangle(installed_version)+\
                   '_'+ version_mangle(candidate_version)+'_'+\
                   arch+'.debdelta'
 
