From 540070e13ed47514742c6c434c543fadaf929578 Mon Sep 17 00:00:00 2001
From: Shirish Agarwal <shirishag75@gmail.com>
Date: Fri, 6 Jan 2017 02:25:28 +0530
Subject: [PATCH] debian-policy should have an Example Section in a manpage, if
 a manpage exists.

---
 policy.sgml | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/policy.sgml b/policy.sgml
index 06d094c..9cc2782 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -10661,6 +10661,10 @@ name ["<var>syshostname</var>"]:
 	</p>
 
 	<p>
+	   It is also suggested that all packages should also have an EXAMPLE section which describes with syntax how the package should be used. The Example section could be optional for library packages
+	</p> 
+
+	<p>
           If no manual page is available, this is considered as a bug
           and should be reported to the Debian Bug Tracking System (the
           maintainer of the package is allowed to write this bug report
-- 
2.11.0

