diff -Naur amideco.orig/amideco-0.31e/debian/changelog  amideco/amideco-0.31e/debian/changelog 
--- amideco.orig/amideco-0.31e/debian/changelog	2014-05-20 21:34:49.717661418 -0500
+++ amideco/amideco-0.31e/debian/changelog	2014-05-20 21:36:15.937659492 -0500
@@ -1,3 +1,11 @@
+amideco (0.31e-3.2) unstable; urgency=low
+
+  * Fix FTBFS with clang:
+    - Fixed "wrong main declaration" in
+      src/amideco.c
+
+ -- Arthur Marble <arthur@info9.net>  Tue, 20 May 2014 21:36:15 -0500
+
 amideco (0.31e-3.1) unstable; urgency=low
 
   * Non-maintainer upload.
diff -Naur amideco.orig/amideco-0.31e/patches/clang-ftbfs.diff  amideco/amideco-0.31e/patches/clang-ftbfs.diff 
--- amideco.orig/amideco-0.31e/patches/clang-ftbfs.diff	1969-12-31 18:00:00.000000000 -0600
+++ amideco/amideco-0.31e/patches/clang-ftbfs.diff	2014-05-20 21:36:20.933659380 -0500
@@ -0,0 +1,11 @@
+--- a/src/amideco.c
++++ b/src/amideco.c
+@@ -42,7 +42,7 @@
+ #include	"./amiver.h"
+ #include	"./amihelp.h"
+ 
+-int main(byte argc, byte *argv[])
++int main(int argc, char *argv[])
+ 	{
+ 
+ 		FILE *ptx, *pto;
diff -Naur amideco.orig/amideco-0.31e/patches/series  amideco/amideco-0.31e/patches/series 
--- amideco.orig/amideco-0.31e/patches/series	1969-12-31 18:00:00.000000000 -0600
+++ amideco/amideco-0.31e/patches/series	2014-05-20 21:35:08.505660998 -0500
@@ -0,0 +1 @@
+clang-ftbfs.diff
