--- /usr/share/initramfs-tools/scripts/local-top/cryptroot.orig	2013-01-01 18:44:21.012393587 -0500
+++ /usr/share/initramfs-tools/scripts/local-top/cryptroot	2013-01-01 18:33:45.173478481 -0500
@@ -293,7 +293,7 @@
 
 		#FSTYPE=''
 		#eval $(fstype < "$NEWROOT")
-		FSTYPE="$(blkid -s TYPE -o value "$NEWROOT")"
+		FSTYPE="$(/sbin/blkid -s TYPE -o value "$NEWROOT")"
 
 		# See if we need to setup lvm on the crypto device
 		#if [ "$FSTYPE" = "lvm" ] || [ "$FSTYPE" = "lvm2" ]; then
