Index: /vendors/cacti/cacti-0.8.6i-ipax/debian/patches/00list
===================================================================
--- /vendors/cacti/cacti-0.8.6i-ipax/debian/patches/00list (revision 299)
+++ /vendors/cacti/cacti-0.8.6i-ipax/debian/patches/00list (revision 302)
@@ -6,5 +6,4 @@
 07_official_import_template_argument_space_removal.dpatch
 07_official_poller_output_remainder.dpatch
-08_svn_timespan_breakage_fix.dpatch
 10_CVE-2007-6035.dpatch
 10_CVE-2008-0783+5.dpatch
