diff --git a/debian/tests/test-suite b/debian/tests/test-suite
index cff9edf2..21797e58 100644
--- a/debian/tests/test-suite
+++ b/debian/tests/test-suite
@@ -2,5 +2,9 @@

 set -e

+cp -a tests "$AUTOPKGTEST_TMP"
+ln -s /usr/share/asymptote "$AUTOPKGTEST_TMP"/base
+ln -s /usr/bin/asy "$AUTOPKGTEST_TMP"/asy
+
 export ASYMPTOTE_HOME=$(mktemp -d)
-make test
\ No newline at end of file
+make -C "$AUTOPKGTEST_TMP"/tests all
