diff -Nru debuerreotype-0.15/debian/changelog debuerreotype-0.15/debian/changelog
--- debuerreotype-0.15/debian/changelog	2022-08-24 00:27:02.000000000 +0300
+++ debuerreotype-0.15/debian/changelog	2023-02-07 11:26:21.000000000 +0200
@@ -1,3 +1,10 @@
+debuerreotype (0.15-1.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Disable the autopkgtest. (see #1022854)
+
+ -- Adrian Bunk <bunk@debian.org>  Tue, 07 Feb 2023 11:26:21 +0200
+
 debuerreotype (0.15-1) unstable; urgency=medium
 
   * Update to 0.15 upstream release
diff -Nru debuerreotype-0.15/debian/tests/control debuerreotype-0.15/debian/tests/control
--- debuerreotype-0.15/debian/tests/control	2021-03-27 00:47:05.000000000 +0200
+++ debuerreotype-0.15/debian/tests/control	2023-02-07 11:26:21.000000000 +0200
@@ -1,10 +1,10 @@
-Tests: stretch
-Depends: ca-certificates,
-         debian-archive-keyring,
-         debootstrap,
-         debuerreotype,
-         diffoscope,
-         gnupg,
-         wget,
-         xz-utils
-Restrictions: allow-stderr, needs-root
+#Tests: stretch
+#Depends: ca-certificates,
+#         debian-archive-keyring,
+#         debootstrap,
+#         debuerreotype,
+#         diffoscope,
+#         gnupg,
+#         wget,
+#         xz-utils
+#Restrictions: allow-stderr, needs-root
