Index: debian/console-keymaps-at.templates
===================================================================
--- debian/console-keymaps-at.templates	(revision 445)
+++ debian/console-keymaps-at.templates	(working copy)
@@ -1,6 +1,6 @@
 Template: console-keymaps-at/keymap
 Type: select
-Choices-C: us, by, be2-latin1, br-abnt2, br-latin1, uk, bg, cf, ca-multi, croat, cz-lat2, dk-latin1, nl, dvorak, et, fi-latin1, fr-latin9, de-latin1-nodeadkeys, gr, hebrew, hu, is-latin1, it, jp106, la-latin1, lv-latin4, lt, mk, no-latin1, pl, pt-latin1, ro, ru, sr-cy, sk-qwerty, slovene, es, se-latin1, fr_CH-latin1, sg-latin1, trfu, trqu, ua
+Choices-C: us, by, be2-latin1, br-abnt2, br-latin1, uk, bg, cf, ca-multi, croat, cz-lat2, dk-latin1, nl, dvorak, et, fi-latin1, fr-latin9, de-latin1-nodeadkeys, gr, hebrew, hu, is-latin1, it, jp106, la-latin1, lv-latin4, lt, mk, no-latin1, pl, pt-latin1, ro, ru, sr-cy, sk-qwerty, slovene, es, se-latin1, fr_CH-latin1, sg-latin1, th-tis, trfu, trqu, ua
 #flag:partial
-__Choices: American English, Belarusian, Belgian, Brazilian (ABNT2 layout), Brazilian (EUA layout), British English, Bulgarian, Canadian French, Canadian Multilingual, Croatian, Czech, Danish, Dutch, Dvorak, Estonian, Finnish, French, German, Greek, Hebrew, Hungarian, Icelandic, Italian, Japanese, Latin American, Latvian, Lithuanian, Macedonian, Norwegian, Polish, Portuguese, Romanian, Russian, Serbian (Cyrillic), Slovakian, Slovene, Spanish, Swedish, Swiss French, Swiss German, Turkish (F layout), Turkish (Q layout), Ukrainian
+__Choices: American English, Belarusian, Belgian, Brazilian (ABNT2 layout), Brazilian (EUA layout), British English, Bulgarian, Canadian French, Canadian Multilingual, Croatian, Czech, Danish, Dutch, Dvorak, Estonian, Finnish, French, German, Greek, Hebrew, Hungarian, Icelandic, Italian, Japanese, Latin American, Latvian, Lithuanian, Macedonian, Norwegian, Polish, Portuguese, Romanian, Russian, Serbian (Cyrillic), Slovakian, Slovene, Spanish, Swedish, Swiss French, Swiss German, Thai, Turkish (F layout), Turkish (Q layout), Ukrainian
 _Description: Keymap to use:
