--- tools/start_new_disc~	2013-04-01 01:26:54.000000000 +0000
+++ tools/start_new_disc	2013-04-17 12:25:22.000000000 +0000
@@ -39,6 +39,7 @@
 if [ "$DISKINFO_DISTRO"x = ""x ] ; then
     DISKINFO_DISTRO="Debian"
 fi
+export DISKINFO_DISTRO
 
 if [ $NUM_ARCHES = 1 ] ; then
     case $ARCHES in
--- tools/boot/wheezy/boot-x86~	2013-03-24 20:13:25.000000000 +0000
+++ tools/boot/wheezy/boot-x86	2013-04-17 12:58:44.000000000 +0000
@@ -419,7 +419,7 @@
     # Create grub menu entries to match the isolinux ones
     sed -i '/^menuentry/Q' $CDDIR/boot/grub/grub.cfg;
     $BASEDIR/tools/boot/$DI_CODENAME/parse_isolinux \
-        boot$N/isolinux $CDDIR $BASEDIR/data/$DI_CODENAME/grub-theme.in "Debian GNU/Linux $DEBVERSION" \
+        boot$N/isolinux $CDDIR $BASEDIR/data/$DI_CODENAME/grub-theme.in "$DISKINFO_DISTRO" "$DEBIAN_KERNEL" "$DEBVERSION" \
         >> $CDDIR/boot/grub/grub.cfg
 
     # Stuff the EFI boot files into a FAT filesystem, making it as
--- tools/boot/wheezy/parse_isolinux~	2012-12-29 03:52:48.000000000 +0000
+++ tools/boot/wheezy/parse_isolinux	2013-04-17 11:50:32.000000000 +0000
@@ -14,7 +14,9 @@
 my $isolinuxdir = shift or die "Need to know where the isolinux directory is!\n";
 my $outdir = shift or die "Need to know where to write output!\n";
 my $grub_theme = shift or die "Need input file location for base grub theme!\n";
-my $tl_desc = shift or die "Need a top-level description (e.g. Debian GNU/Linux 7.0)\n";
+my $tl_distro= shift or die "Need a top-level distro name (e.g. Debian)\n";
+my $tl_kernel= shift or die "Need a top-level kernel (e.g. GNU/Linux)\n";
+my $tl_version= shift or die "Need a top-level version (e.g. 7.0)\n";
 my $theme_dir = "$outdir/boot/grub/theme";
 my @cpp_and_opts = ('cpp',
 		    '-traditional',
@@ -92,7 +94,7 @@
     my $filename = shift;
     my @args;
     push(@args, @cpp_and_opts);
-    push(@args, "-DTITLE=\"$tl_desc\"");
+    push(@args, "-DTITLE=\"$tl_distro $tl_kernel $tl_version\"");
     for (my $i = 0; $i < $menudepth; $i++) {
 	push(@args, "-DMENU$i=\"" . $menu_title[$i] . "\"");
     }
@@ -115,7 +117,7 @@
 
 $menu{"number"} = "1";
 $menu{"label"} = "top";
-$menu{"title"} = "Debian GNU/Linux Installer menu";
+$menu{"title"} = "$tl_distro $tl_kernel Installer menu";
 $menu_title[$menudepth] = $menu{"title"};
 
 my %kernel;
