--- doc/aptitude-man.xsl.orig	2008-03-30 22:06:05.000000000 +0200
+++ doc/aptitude-man.xsl	2008-04-16 19:33:07.000000000 +0200
@@ -19,7 +19,10 @@
 
 
 <xsl:template match="literal">
-  <xsl:apply-templates mode="bold" select="."/>
+  <xsl:call-template name="bold">
+    <xsl:with-param name="node" select="."/>
+    <xsl:with-param name="context" select="."/>
+  </xsl:call-template>
 </xsl:template>
 
 <xsl:param name="preferred.mediaobject.role">text</xsl:param>
