diff -Nru apt-show-versions-0.22.11/debian/changelog apt-show-versions-0.22.11+nmu1/debian/changelog
--- apt-show-versions-0.22.11/debian/changelog	2019-02-16 11:10:23.000000000 +0000
+++ apt-show-versions-0.22.11+nmu1/debian/changelog	2020-11-08 17:23:26.000000000 +0000
@@ -1,3 +1,11 @@
+apt-show-versions (0.22.11+nmu1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Add build-dep on libpod-parser-perl, which is no longer shipped in
+    the core perl package (closes: #961152)
+
+ -- Dominic Hargreaves <dom@earth.li>  Sun, 08 Nov 2020 17:23:26 +0000
+
 apt-show-versions (0.22.11) unstable; urgency=medium
 
   * fix long standing bug handling incomplete entries in
diff -Nru apt-show-versions-0.22.11/debian/control apt-show-versions-0.22.11+nmu1/debian/control
--- apt-show-versions-0.22.11/debian/control	2019-02-16 11:10:23.000000000 +0000
+++ apt-show-versions-0.22.11+nmu1/debian/control	2020-11-08 17:23:26.000000000 +0000
@@ -3,7 +3,7 @@
 Priority: optional
 Maintainer: Christoph Martin <martin@uni-mainz.de>
 Uploaders: Andreas Hoenen <andreas@hoenen-terstappen.de>
-Build-Depends: debhelper (>= 9)
+Build-Depends: debhelper (>= 9), libpod-parser-perl
 Build-Depends-Indep: po4a
 Standards-Version: 3.9.8
 Vcs-Browser: https://salsa.debian.org/debian/apt-show-versions
