*** cg_print.c	2011-04-18 20:34:00.309575533 -0300
--- cg_print_modified.c	2011-04-18 20:32:49.685603137 -0300
***************
*** 94,101 ****
      {
        printf ("%6.6s %5.5s %7.7s %11.11s %7.7s/%-7.7s     %-8.8s\n",
  	      "", "", "", "", _("called"), _("total"), _("parents"));
!       printf ("%-6.6s %5.5s %7.7s %11.11s %7.7s+%-7.7s %-8.8s\t%5.5s\n",
! 	      _("index"), _("%time"), _("self"), _("descendants"),
  	      _("called"), _("self"), _("name"), _("index"));
        printf ("%6.6s %5.5s %7.7s %11.11s %7.7s/%-7.7s     %-8.8s\n",
  	      "", "", "", "", _("called"), _("total"), _("children"));
--- 94,102 ----
      {
        printf ("%6.6s %5.5s %7.7s %11.11s %7.7s/%-7.7s     %-8.8s\n",
  	      "", "", "", "", _("called"), _("total"), _("parents"));
!       printf ("%-6.6s %5.5s %7.7s %11.11s %7.7s+%-7.7s %-8.8s\t%5.5s\n", _("index"),
!       /* xgettext:no-c-format */
! 	      _("%time"), _("self"), _("descendants"),
  	      _("called"), _("self"), _("name"), _("index"));
        printf ("%6.6s %5.5s %7.7s %11.11s %7.7s/%-7.7s     %-8.8s\n",
  	      "", "", "", "", _("called"), _("total"), _("children"));
