diff -Nru dh-exec-0.1/t/Makefile.am dh-exec-0.1ubuntu1/t/Makefile.am
--- dh-exec-0.1/t/Makefile.am	2011-12-21 00:51:17.000000000 +0100
+++ dh-exec-0.1ubuntu1/t/Makefile.am	2012-01-30 11:59:36.000000000 +0100
@@ -18,7 +18,8 @@
 			DH_EXEC_LIBDIR="${top_builddir}/src" \
 			DH_EXEC_BINDIR="${top_builddir}/src" \
 			top_builddir="${top_builddir}" \
-			srcdir="${srcdir}"
+			srcdir="${srcdir}" \
+			NO_PKG_MANGLE=1
 
 check: ${TESTS}
 	$(AM_V_GEN) ${TESTS_ENVIRONMENT} prove -e '' -f -o ${PROVE_OPTIONS} $(addprefix ${srcdir}/,${TESTCASES})
