diff -aur coreutils-8.13.orig/debian/rules coreutils-8.13/debian/rules
--- coreutils-8.13.orig/debian/rules	2011-12-30 17:14:43.000000000 +0100
+++ coreutils-8.13/debian/rules	2011-12-30 17:20:56.000000000 +0100
@@ -20,12 +20,10 @@
   endif
 endif
 
-# implement no optimization build option
-ifneq (,$(findstring noopt,$(DEB_BUILD_OPTIONS)))
-CFLAGS = -g -O0
-else
-CFLAGS ?= -g -O2
-endif
+CFLAGS = `dpkg-buildflags --get CFLAGS`
+LDFLAGS = `dpkg-buildflags --get LDFLAGS`
+CPPFLAGS = `dpkg-buildflags --get CPPFLAGS`
+
 CFLAGS += -DSYSLOG_SUCCESS -DSYSLOG_FAILURE -DSYSLOG_NON_ROOT
 
 # Renesas SH(sh4) need -mieee option.
@@ -51,7 +49,7 @@
 configure-stamp: 
 	dh_testdir
 
-	CFLAGS="$(CFLAGS)" \
+	CFLAGS="$(CFLAGS)" CPPFLAGS="$(CFLAGS)" \
 		LDFLAGS='$(LDFLAGS)' ./configure \
 		--build=$(DEB_BUILD_GNU_TYPE) --host=$(DEB_HOST_GNU_TYPE) \
 		--prefix=/usr -v \
Nur in coreutils-8.13/debian: rules~.
