diff --git a/debian/dictionaries-common.templates b/debian/dictionaries-common.templates
index a0e80cc..f5f0a7a 100644
--- a/debian/dictionaries-common.templates
+++ b/debian/dictionaries-common.templates
@@ -24,8 +24,11 @@ Type: select
 Choices-C: ${choices}, Manual symlinks setting
 __Choices: ${echoices}, Manual symlinks setting
 _Description: System's default ispell dictionary:
- Because more than one ispell dictionary will be available in your system,
- please select the one you'd like applications to use by default.
+ When no specific dictionary is selected, ispell will choose its default
+ by using "/usr/lib/default.{aff,hash}" links. Selected dictionary will
+ be used for default symlink and as global emacs default when ispell is
+ used. Use "Manual symlinks setting" if you want to handle symlinks
+ yourself. No global emacs default for ispell in that case.
  .
  You can change the default ispell dictionary at any time by running
  "select-default-ispell".
@@ -35,8 +38,9 @@ Type: select
 Choices-C: ${choices}, Manual symlinks setting
 __Choices: ${echoices}, Manual symlinks setting
 _Description: System's default wordlist:
- Because more than one wordlist will be available in your system, please
- select the one you'd like applications to use by default.
+ Selected wordlist will be used as "/usr/share/dict/words" by linking
+ to it. Use "Manual symlinks setting" if you want to handle that symlink
+ yourself.
  .
  You can change the default wordlist at any time by running
  "select-default-wordlist".
