--- abcde.orig	2012-05-18 09:36:21.083695541 +0000
+++ abcde	2012-05-18 11:17:03.153289133 +0000
@@ -339,12 +339,12 @@
 
 new_checkexec ()
 {
-	if [ ! "$@" = "" ]; then
-		# Cut off any command-line option we added in
-		X=$(echo $@ | cut -d' ' -f2)
-		if [ "$(which $X)" = "" ]; then
+	local X
+	if [ "X$1" != 'X' ]; then
+		X=`command -v "$1" 2>/dev/null`
+		if [ "X$X" = 'X' ]; then
 			return 1
-		elif [ ! -x $(which $X) ]; then
+		elif [ `expr "X$X" : 'X/'` -eq 2 ] && [ ! -x "$X" ]; then
 			return 2
 		fi
 	fi
@@ -353,23 +353,21 @@
 
 checkexec ()
 {
-	if [ ! "$@" = "" ]; then
-		# Cut off any command-line option we added in
-		X=$(echo $@ | cut -d' ' -f2)
-		# Test for built-in abcde.function
-		[ "$X" != "${X#abcde.}" ] && type $X >/dev/null 2>&1 && return
-		if [ "$(which $X)" = "" ]; then
-			log error "$X is not in your path." >&2
+	local X
+	if [ "X$1" != 'X' ]; then
+		X=`command -v "$1" 2>/dev/null`
+		if [ "X$X" = 'X' ]; then
+			log error "$1 is not in your path." >&2
 			log info  "Define the full path to the executable if it exists on your system." >&2
 			if [ -e /etc/debian_release ] ; then
-				case $X in
+				case $1 in
 					oggenc) 	MISSING_PACKAGE=vorbis-tools ;;
-					lame|flac)	MISSING_PACKAGE=$X ;;
+					lame|flac)	MISSING_PACKAGE=$1 ;;
 				esac
 				log info "Hint: apt-get install $MISSING_PACKAGE" >&2
 			fi
 			exit 1
-		elif [ ! -x "$(which $X)" ]; then
+		elif [ `expr "X$X" : 'X/'` -eq 2 ] && [ ! -x "$X" ]; then
 			log error "$X is not executable." >&2
 			exit 1
 		fi
