diff -rupN dpkg-1.17.13/utils/start-stop-daemon.c dpkg-1.17.13-new/utils/start-stop-daemon.c
--- dpkg-1.17.13/utils/start-stop-daemon.c	2014-08-19 11:12:18.000000000 -0700
+++ dpkg-1.17.13-new/utils/start-stop-daemon.c	2014-09-01 14:47:12.000000000 -0700
@@ -1780,11 +1780,11 @@ do_start(int argc, char **argv)
 		set_proc_schedule(proc_sched);
 	if (io_sched)
 		set_io_schedule(io_sched);
-	if (umask_value >= 0)
-		umask(umask_value);
 	if (mpidfile && pidfile != NULL)
 		/* User wants _us_ to make the pidfile. */
 		write_pidfile(pidfile, getpid());
+	if (umask_value >= 0)
+		umask(umask_value);
 	if (changeroot != NULL) {
 		if (chdir(changeroot) < 0)
 			fatal("unable to chdir() to %s", changeroot);
