diff --git a/handlers/dup.in b/handlers/dup.in
index ffae48c..00fd786 100644
--- a/handlers/dup.in
+++ b/handlers/dup.in
@@ -163,6 +163,15 @@ if [ "$incremental" == "no" ]; then
    else
       execstr_command="full"
    fi
+else
+   if [ "$keep" != "yes" ]; then
+      fullifolderthan="30D"
+      if [ "`echo $keep | tr -d 0-9`" == "" ]; then
+         let "fullifolderthan = keep / 2"
+         fullifolderthan="${fullifolderthan}D"
+      fi
+      execstr_options="${execstr_options} --full-if-older-than $fullifolderthan"
+   fi
 fi
 
 ### Temporary directory
