--- dovecot-1.1.11/dovecot-example.conf~	2009-03-06 17:28:46.785662296 +0000
+++ dovecot-1.1.11/dovecot-example.conf	2009-03-06 17:29:23.082571853 +0000
@@ -1057,7 +1057,8 @@
 # referenced using URIs in format "proxy::<name>".
 
 dict {
-  #quota = mysql:/etc/dovecot-dict-quota.conf 
+  #expire = db:/var/lib/dovecot/expire.db
+  #quota = mysql:/etc/dovecot-dict-quota.conf
 }
 
 # Path to Berkeley DB's configuration file. See doc/dovecot-db-example.conf
@@ -1136,7 +1137,7 @@
   # you must set up:
   #   dovecot --exec-mail ext /usr/libexec/dovecot/expire-tool
   #expire = Trash 7 Spam 30
-  #expire_dict = db:/var/lib/dovecot/expire.db
+  #expire_dict = proxy::expire
 
   # Lazy expunge plugin. Currently works only with maildirs. When a user
   # expunges mails, the mails are moved to a mailbox in another namespace

