From 13d2d1df2dcd4881db05368d19906c28164cd64b Mon Sep 17 00:00:00 2001
From: Will Aoki <waoki@umnh.utah.edu>
Date: Wed, 30 Mar 2016 15:29:35 -0600
Subject: [PATCH 2/2] ignore .pc used by quilt

---
 .gitignore |    1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitignore b/.gitignore
index aa57078..61b43b8 100644
--- a/.gitignore
+++ b/.gitignore
@@ -10,3 +10,4 @@ config.status
 libtool
 src/bedup
 src/burp
+.pc
-- 
1.7.10.4

