From 8b02a8dcdd315b85fbc8246ae0265faed1828ab8 Mon Sep 17 00:00:00 2001
From: Eduard - Gabriel Munteanu <edgmnt@gmail.com>
Date: Thu, 26 Dec 2013 20:39:36 +0200
Subject: [PATCH] Abort if keyring is missing, don't just warn.

Signed-off-by: Eduard - Gabriel Munteanu <edgmnt@gmail.com>
---
 functions | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/functions b/functions
index a2e1431..69d8ea7 100644
--- a/functions
+++ b/functions
@@ -521,7 +521,7 @@ download_release_sig () {
 		 "$relsigdest" "$reldest" || true) | read_gpg_status
 		progress 100 100 DOWNRELSIG "Downloading Release file signature"
 	elif [ -z "$DISABLE_KEYRING" ] && [ -n "$KEYRING_WANTED" ]; then
-		warning KEYRING "Cannot check Release signature; keyring file not available %s" "$KEYRING_WANTED"
+		error 1 NOKEYRING "Cannot check Release signature; keyring file not available %s" "$KEYRING_WANTED"
 	fi
 }
 
-- 
1.8.3.2

