diff -ur dbf2mysql-1.14a/dbf2mysql.c dbf2mysql-1.14a-new/dbf2mysql.c
--- dbf2mysql-1.14a/dbf2mysql.c	2007-07-25 14:38:19.000000000 +0200
+++ dbf2mysql-1.14a-new/dbf2mysql.c	2007-07-25 14:35:01.000000000 +0200
@@ -221,7 +221,9 @@
 	if (fieldlow)
 	    strtolower(dbh->db_fields[i].db_name);
 
+        strcat(query, '`');
 	strcat(query, dbh->db_fields[i].db_name);
+        strcat(query, '`');
 	switch(dbh->db_fields[i].db_type) {
 	  case 'C':
 	    if (var_chars)
@@ -324,7 +326,7 @@
 /* Patched by GLC to fix quick mode Numeric fields */
 void do_inserts(MYSQL *SQLsock, char *table, dbhead *dbh)
 {
-    int		result, i, j, nc = 0, h;
+    int		result, i, j, nc = 0, h, first_done;
     field	*fields;
     char	*query, *vpos, *pos;
     char	str[257], *cvt = NULL, *s;
@@ -370,6 +372,7 @@
 
     for ( i = 0 ; i < dbh->db_nfields ; i++ )
     {
+        val_len += 32;
 	switch(dbh->db_fields[i].db_type)
 	{
 	  case 'M':
@@ -393,7 +396,24 @@
     }
     if (!quick)
     {
-	sprintf(query, "INSERT INTO %s VALUES (",table);
+        sprintf(query, "INSERT INTO %s (",table);
+        first_done = 0;
+        for ( i = 0; i < dbh->db_nfields; i++) {
+            if (!strlen(dbh->db_fields[i].db_name)) {
+                continue;
+                /* skip field if length of name == 0 */
+            }
+            if (first_done) {
+                strcat(query, ",\n\t");
+            }
+            first_done = 1;
+            if (fieldlow)
+                strtolower(dbh->db_fields[i].db_name);
+            strcat(query, '`');
+            strcat(query, dbh->db_fields[i].db_name);
+            strcat(query, '`');
+        }
+        strcat(query, ") VALUES (");
     }
     else
 	if (express)
