--- autopkgtest-normal/usr/bin/autopkgtest-virt-qemu	2017-04-30 19:09:57.000000000 +0200
+++ /mnt/extras/src/autopkgtest-4.4+nmu1/virt/autopkgtest-virt-qemu	2017-08-14 03:25:20.382311089 +0200
@@ -328,7 +328,7 @@
     fcntl.fcntl(fin, fcntl.F_SETFL,
                 fcntl.fcntl(fin, fcntl.F_GETFL) | os.O_NONBLOCK)
     count = 0
-    while running:
+    while True:
         try:
             block = os.read(fin, 1000000)
             if flagfile_on_eof and not block:
@@ -343,6 +343,8 @@
                 raise
             block = None
         if not block:
+            if not running:
+                return
             time.sleep(0.01)
             continue
         while True:
