--- rules	2008-11-06 15:56:28.000000000 +0100
+++ rules.new	2008-11-06 15:54:23.000000000 +0100
@@ -62,6 +62,9 @@
 	dh_installexamples
 	dh_link
 	dh_compress
+	ln -s dgrep.1.gz debian/debian-goodies/usr/share/man/man1/degrep.1.gz
+	ln -s dgrep.1.gz debian/debian-goodies/usr/share/man/man1/dfgrep.1.gz
+	ln -s dgrep.1.gz debian/debian-goodies/usr/share/man/man1/dzgrep.1.gz
 	dh_fixperms
 	dh_installdeb
 	dh_gencontrol
