Index: dpkg-dev-el.el
===================================================================
RCS file: /cvsroot/pkg-goodies-el/emacs-goodies-el/elisp/dpkg-dev-el/dpkg-dev-el.el,v
retrieving revision 1.2
diff -u -u -r1.2 dpkg-dev-el.el
--- dpkg-dev-el.el	23 Feb 2009 16:18:53 -0000	1.2
+++ dpkg-dev-el.el	2 Jul 2010 19:28:41 -0000
@@ -72,8 +72,12 @@
   (modify-coding-system-alist 'file "/debian/control\\'" 'utf-8)
 
 ;;; (modify-coding-system-alist 'file "/debian/changelog\\'" 'utf-8)
-;;; Instead use this.  See http://bugs.debian.org/457047 by Trent W. Buck
-  (modify-coding-system-alist 'file "/debian/\\([[:lower:][:digit:].+-]\\.\\)?changelog\\'" 'utf-8)
+;;; Instead use this for dh_installchangelog debian/packagename.changelog
+;;; files too.  See http://bugs.debian.org/457047 by Trent W. Buck
+;;; But not [:lower:][:digit:] since those forms are not available in xemacs21.
+;;; xemacs21 can have utf-8 at startup if you use mule-ucs with
+;;; DEB_MULEUCS_UNICODE=yes
+  (modify-coding-system-alist 'file "/debian/\\([a-z0-9.+-]+\\.\\)?changelog\\'" 'utf-8)
 
   ;; Handle Debian native package, from Kevin Ryde in bug #317597 and #416218
   (defun debian-changelog-coding-system (args)
