diff -Nru debootstrap-1.0.25/debootstrap debootstrap-1.0.25+nmu1/debootstrap
--- debootstrap-1.0.25/debootstrap	2010-09-20 13:22:26.000000000 -0400
+++ debootstrap-1.0.25+nmu1/debootstrap	2010-10-25 23:19:58.000000000 -0400
@@ -289,6 +289,9 @@
 		else
 			error 1 NEEDARG "option requires an argument %s" "$1"
 		fi
+               if [ "$VARIANT" = "fakechroot" ]; then
+                       export PATH=/usr/sbin:/sbin:$PATH
+               fi
 		;;
 	    --keyring|--keyring=?*)
 		if ! gpgv --version >/dev/null 2>&1; then
