--- apt-1.0.6/apt-private/private-cacheset.cc	2014-04-25 13:39:00.000000000 +0200
+++ apt-1.0.6-753297/apt-private/private-cacheset.cc	2014-07-21 17:07:18.836716832 +0200
@@ -55,9 +55,10 @@
       }
       else if (_config->FindB("APT::Cmd::Upgradable") == true)
       {
-         if(P.CurrentVer() && state.Upgradable())
+         pkgPolicy *policy = CacheFile.GetPolicy();
+         pkgCache::VerIterator candidateVer = policy->GetCandidateVer(P);
+         if(P.CurrentVer() && state.Upgradable() && candidateVer.CompareVer(P.CurrentVer()) != 0)
          {
-             pkgPolicy *policy = CacheFile.GetPolicy();
              output_set.insert(policy->GetCandidateVer(P));
          }
       }
