--- /usr/share/autopkgtest/setup-commands/setup-testbed~	2017-04-30 13:09:57.000000000 -0400
+++ /usr/share/autopkgtest/setup-commands/setup-testbed	2017-08-06 17:22:16.942392522 -0400
@@ -196,6 +196,11 @@
     fi
 fi
 
+if [ "$root" != "/" ]; then
+    chroot "$root" mv /etc/resolv.conf /etc/resolv.conf.backup
+    cat /etc/resolv.conf > "$root/etc/resolv.conf"
+fi
+
 if [ -z "${AUTOPKGTEST_IS_SETUP_COMMAND:-}" ]; then
     chroot "$root" apt-get update || (sleep 15; chroot "$root" apt-get update)
 fi
@@ -288,3 +293,8 @@
 
 # avoid cron interference with apt-get update
 echo 'APT::Periodic::Enable "0";' > "$root/etc/apt/apt.conf.d/02periodic"
+
+
+if [ "$root" != "/" ]; then
+    chroot "$root" mv /etc/resolv.conf.backup /etc/resolv.conf
+fi
