From a84787ffb4a584feff2c76c701a9c828553486fc Mon Sep 17 00:00:00 2001
From: Boris Pek <tehnick-8@yandex.ru>
Date: Fri, 29 Nov 2013 01:45:33 +0200
Subject: [PATCH] Use debhelper (>= 9) instead of debhelper (>= 8.0.0) by
 default.

---
 dh_make | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dh_make b/dh_make
index 4eb38c3..2a1b368 100755
--- a/dh_make
+++ b/dh_make
@@ -586,7 +586,7 @@ foreach $orig (@ORIG)
 }
 
 our ($CONFIG_STATUS, $CONFIGURE_STAMP, $PHONY_CONFIGURE, $CONFIGURE, $DPKG_ARCH, $INSTALL, $CLEAN, $MAKE, $CDBS_CLASS );
-our $BUILD_DEPS = 'debhelper (>= 8.0.0)';
+our $BUILD_DEPS = 'debhelper (>= 9)';
 our $DH7_ADDON = '';
 if ($rules_format eq "cdbs") {
 	$BUILD_DEPS = 'cdbs, '.$BUILD_DEPS;
-- 
1.8.4.4

