=== modified file 'debian/debhelper/postinst-apparmor'
--- debian/debhelper/postinst-apparmor	2012-01-12 22:55:48 +0000
+++ debian/debhelper/postinst-apparmor	2012-04-08 07:39:18 +0000
@@ -16,7 +16,7 @@
         }
 
         # Reload the profile, including any abstraction updates
-        if aa-status --enabled 2>/dev/null; then
+        if type aa-status >/dev/null && aa-status --enabled 2>/dev/null; then
             apparmor_parser -r -T -W "$APP_PROFILE" || true
         fi
     fi

