From f8c5247d985d39b197a89f0b57557b6445e33bdd Mon Sep 17 00:00:00 2001
From: Sven Joachim <svenjoac@gmx.de>
Date: Mon, 19 Dec 2016 17:44:18 +0100
Subject: [PATCH] pkg-details: Strip architecture qualifier

---
 pkgdetails.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/pkgdetails.c b/pkgdetails.c
index d588c80a..9444d2d2 100644
--- a/pkgdetails.c
+++ b/pkgdetails.c
@@ -54,7 +54,7 @@ static void outputdeps(char *deps) {
         if (!*pch) break;
 
         while (*pch && *pch != '(' && *pch != '|' && *pch != ','
-               && !isspace(*pch))
+               && *pch != ':' && !isspace(*pch))
         {
             fputc(*pch++, stdout);
         }
-- 
2.11.0

