diff -Nru dovecot-2.0.13/debian/changelog dovecot-2.0.13/debian/changelog
--- dovecot-2.0.13/debian/changelog	2011-05-21 23:58:58.000000000 +0200
+++ dovecot-2.0.13/debian/changelog	2011-06-18 12:31:42.000000000 +0200
@@ -1,3 +1,10 @@
+dovecot (1:2.0.13-1.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Don't ship .la files (Closes: #621297).
+
+ -- Luk Claes <luk@debian.org>  Sat, 18 Jun 2011 12:31:28 +0200
+
 dovecot (1:2.0.13-1) unstable; urgency=high
 
   [ Marco Nenciarini ]
diff -Nru dovecot-2.0.13/debian/rules dovecot-2.0.13/debian/rules
--- dovecot-2.0.13/debian/rules	2011-05-21 23:58:58.000000000 +0200
+++ dovecot-2.0.13/debian/rules	2011-06-18 12:31:23.000000000 +0200
@@ -74,6 +74,7 @@
 	dh_installdirs
 	$(MAKE) install DESTDIR=$(CURDIR)/debian/dovecot-common
 	$(MAKE) -C $(PIGEONHOLE_DIR) install DESTDIR=$(CURDIR)/debian/dovecot-common
+	rm `find $(CURDIR)/debian -name '*.la'`
 	cp $(CURDIR)/src/plugins/drac/drac.so $(CURDIR)/debian/dovecot-common/usr/lib/dovecot/modules
 	install -o root -g root -m 0644 $(CURDIR)/doc/example-config/*.conf \
 		$(CURDIR)/debian/dovecot-common/usr/share/dovecot/
