diff -Nru cgroupfs-mount-1.1/debian/control cgroupfs-mount-1.1+nmu1/debian/control
--- cgroupfs-mount-1.1/debian/control	2014-10-15 19:05:49.000000000 +0300
+++ cgroupfs-mount-1.1+nmu1/debian/control	2015-01-23 10:11:56.000000000 +0200
@@ -10,6 +10,7 @@
 
 Package: cgroupfs-mount
 Architecture: all
+Multi-Arch: foreign
 Depends: mountall (>> 2.48~), ${misc:Depends}
 Description: Light-weight package to set up cgroupfs mounts
  Control groups are a kernel mechanism for tracking and imposing
