diff --git a/contrib/munin-node b/contrib/munin-node
index af8225c..cfe1f27 100644
--- a/contrib/munin-node
+++ b/contrib/munin-node
@@ -30,6 +30,20 @@ _munin_run()
 } &&
 complete -F _munin_run munin-run
 
+have munindoc &&
+_munindoc()
+{
+    local cur prev
+
+    COMPREPLY=()
+    cur=`_get_cword`
+    prev=${COMP_WORDS[COMP_CWORD-1]}
+
+    COMPREPLY=( $( compgen -W '$( command ls /usr/share/munin/plugins )' \
+        -- "$cur" ) )
+} &&
+complete -F _munindoc munindoc
+
 have munin-update &&
 _munin_update()
 {
