--- tools/boot/squeeze/boot-x86~	2011-02-22 20:52:44.000000000 +0000
+++ tools/boot/squeeze/boot-x86	2013-04-25 15:20:38.000000000 +0000
@@ -177,7 +177,9 @@
 mkdir -p $CDDIR/$INSTALLDIR
 cp -lf cdrom/vmlinuz $CDDIR/$INSTALLDIR/
 cp -lf cdrom/initrd.gz $CDDIR/$INSTALLDIR/
-echo "\\tools\\loadlin.exe vmlinuz initrd=initrd.gz" | todos > $CDDIR/$INSTALLDIR/install.bat
+if [ -e $CDDIR/tools/loadlin.exe ]; then
+	echo "\\tools\\loadlin.exe vmlinuz initrd=initrd.gz" | todos > $CDDIR/$INSTALLDIR/install.bat
+fi
 
 mkdir -p $CDDIR/../syslinux
 SYSLINUXDEB=$(find_pkg_file syslinux)
@@ -245,8 +247,9 @@
 	if [ -e boot$N/isolinux/isolinux.cfg.withgtk ]; then
 		mv boot$N/isolinux/isolinux.cfg.withgtk boot$N/isolinux/isolinux.cfg
 	fi
-	echo "\\tools\\loadlin.exe \\$INSTALLDIR\\vmlinuz initrd=initrd.gz vga=788" | todos > $CDDIR/$INSTALLDIR/gtk/install.bat
-
+	if [ -e $CDDIR/tools/loadlin.exe ]; then
+		echo "\\tools\\loadlin.exe \\$INSTALLDIR\\vmlinuz initrd=initrd.gz vga=788" | todos > $CDDIR/$INSTALLDIR/gtk/install.bat
+	fi
 fi
 rm -f boot$N/isolinux/isolinux.cfg.with*
 
--- tools/boot/wheezy/boot-x86~	2013-03-24 20:13:25.000000000 +0000
+++ tools/boot/wheezy/boot-x86	2013-04-25 15:20:38.000000000 +0000
@@ -186,7 +186,9 @@
 mkdir -p $CDDIR/$INSTALLDIR
 cp -lf cdrom/vmlinuz $CDDIR/$INSTALLDIR/
 cp -lf cdrom/initrd.gz $CDDIR/$INSTALLDIR/
-echo "\\tools\\loadlin.exe vmlinuz initrd=initrd.gz" | todos > $CDDIR/$INSTALLDIR/install.bat
+if [ -e $CDDIR/tools/loadlin.exe ]; then
+	echo "\\tools\\loadlin.exe vmlinuz initrd=initrd.gz" | todos > $CDDIR/$INSTALLDIR/install.bat
+fi
 
 mkdir -p $CDDIR/../syslinux
 SYSLINUXDEB=$(find_pkg_file syslinux)
@@ -255,8 +257,9 @@
 	if [ -e boot$N/isolinux/isolinux.cfg.withgtk ]; then
 		mv boot$N/isolinux/isolinux.cfg.withgtk boot$N/isolinux/isolinux.cfg
 	fi
-	echo "\\tools\\loadlin.exe \\$INSTALLDIR\\vmlinuz initrd=initrd.gz vga=788" | todos > $CDDIR/$INSTALLDIR/gtk/install.bat
-
+	if [ -e $CDDIR/tools/loadlin.exe ]; then
+		echo "\\tools\\loadlin.exe \\$INSTALLDIR\\vmlinuz initrd=initrd.gz vga=788" | todos > $CDDIR/$INSTALLDIR/gtk/install.bat
+	fi
 fi
 rm -f boot$N/isolinux/isolinux.cfg.with*
 
