--- conntrack-0.9.7.orig/src/conntrack.c	2008-05-22 12:29:38.000000000 +0000
+++ conntrack-0.9.7/src/conntrack.c	2008-08-14 15:08:04.000000000 +0000
@@ -637,7 +637,7 @@
 		if (!(output_mask & _O_XML)) {
 			struct timeval tv;
 			gettimeofday(&tv, NULL);
-			printf("[%-8ld.%-6ld]\t", tv.tv_sec, tv.tv_usec);
+			printf("[%-8ld.%-6ld]\t", (long) tv.tv_sec, (long) tv.tv_usec);
 		} else
 			op_flags |= NFCT_OF_TIME;
 	}
