diff -u attr-2.4.43/Makefile attr-2.4.43/Makefile
--- attr-2.4.43/Makefile
+++ attr-2.4.43/Makefile
@@ -39,8 +39,8 @@
 		--exec-prefix=/ \
 		--sbindir=/bin \
 		--bindir=/usr/bin \
-		--libdir=/lib \
-		--libexecdir=/usr/lib \
+		--libdir=/lib/$(DEB_HOST_GNU_TYPE) \
+		--libexecdir=/usr/lib/$(DEB_HOST_GNU_TYPE) \
 		--includedir=/usr/include \
 		--mandir=/usr/share/man \
 		--datadir=/usr/share \
diff -u attr-2.4.43/debian/changelog attr-2.4.43/debian/changelog
--- attr-2.4.43/debian/changelog
+++ attr-2.4.43/debian/changelog
@@ -1,3 +1,10 @@
+attr (1:2.4.43-2a0.multi.1) unstable; urgency=low
+
+  * Convert to multiarch
+  * Add missing dh_clean
+
+ -- Goswin von Brederlow <goswin-v-b@web.de>  Mon, 11 May 2009 02:41:12 +0200
+
 attr (1:2.4.43-2) unstable; urgency=high
 
   * Remove --enable-lib64=yes in Makefile (closes: #514017)
only in patch2:
unchanged:
--- attr-2.4.43.orig/debian/rules
+++ attr-2.4.43/debian/rules
@@ -16,6 +16,9 @@
 	  INSTALL_USER=root INSTALL_GROUP=root ;
 checkdir = test -f debian/rules
 
+DEB_HOST_GNU_TYPE := $(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)
+export DEB_HOST_GNU_TYPE
+
 build: built
 built: config
 	@echo "== dpkg-buildpackage: build" 1>&2
@@ -38,6 +41,7 @@
 	$(MAKE) distclean
 	-rm -rf $(dirme) $(dirdev) $(dirlib) debian/tmp
 	-rm -f debian/*substvars debian/files* debian/*.debhelper
+	dh_clean
 
 binary-indep:
 
only in patch2:
unchanged:
--- attr-2.4.43.orig/debian/control
+++ attr-2.4.43/debian/control
@@ -11,6 +11,7 @@
 Depends: ${shlibs:Depends}, ${misc:Depends}
 Conflicts: xfsdump (<< 2.0.0)
 Architecture: any
+Multi-Arch: no
 Description: Utilities for manipulating filesystem extended attributes
  A set of tools for manipulating extended attributes on filesystem
  objects, in particular getfattr(1) and setfattr(1).
@@ -41,6 +42,7 @@
 Priority: required
 Conflicts: attr (<< 2.0.0)
 Architecture: any
+Multi-Arch: yes
 Description: Extended attribute shared library
  Contains the runtime environment required by programs that make use
  of extended attributes.
