--- /usr/share/initramfs-tools/scripts/local-top/cryptroot.orig	2013-01-01 18:15:47.148091165 -0500
+++ /usr/share/initramfs-tools/scripts/local-top/cryptroot	2013-01-01 18:33:45.173478481 -0500
@@ -323,6 +323,12 @@
 			continue
 		fi
 
+		# Finally, pass cryptkey into /lib/cryptsetup/passfifo if this
+		# is being run from an ssh session
+		if [ -n "$SSH_TTY" -a -p /lib/cryptsetup/passfifo ]; then
+			echo -ne "$cryptkey" >/lib/cryptsetup/passfifo
+		fi
+
 		message "cryptsetup: $crypttarget set up successfully"
 		break
 	done
