diff -Nru cfengine3-3.9.1/debian/changelog cfengine3-3.9.1/debian/changelog
--- cfengine3-3.9.1/debian/changelog	2016-12-01 21:55:30.000000000 +0100
+++ cfengine3-3.9.1/debian/changelog	2017-01-17 01:37:38.000000000 +0100
@@ -1,3 +1,11 @@
+cfengine3 (3.9.1-4.1) UNRELEASED; urgency=medium
+
+  * Non-maintainer upload.
+  * Switch Build-Depends to default-libmysqlclient-dev.  (Closes: #845829)
+  * cfengine3: Add dependency on lsb-base, thanks to lintian.
+
+ -- Andreas Beckmann <anbe@debian.org>  Tue, 17 Jan 2017 01:37:38 +0100
+
 cfengine3 (3.9.1-4) unstable; urgency=medium
 
   * Uploading the experimental package to unstable (Closes: 828263).
diff -Nru cfengine3-3.9.1/debian/control cfengine3-3.9.1/debian/control
--- cfengine3-3.9.1/debian/control	2016-12-01 21:55:30.000000000 +0100
+++ cfengine3-3.9.1/debian/control	2017-01-17 01:37:38.000000000 +0100
@@ -4,7 +4,7 @@
 Maintainer: Antonio Radici <antonio@debian.org>
 Build-Depends: debhelper (>= 7.0.50), autotools-dev, libssl-dev (>= 1.1),
  flex, bison, libpcre3-dev, dh-autoreconf, libvirt-dev, libacl1-dev,
- liblmdb-dev, libmysqlclient-dev, libxml2-dev, quilt, libpam0g-dev
+ liblmdb-dev, default-libmysqlclient-dev, libxml2-dev, quilt, libpam0g-dev
 Standards-Version: 3.9.8
 Homepage: http://www.cfengine.org/
 Vcs-Git: https://anonscm.debian.org/git/collab-maint/cfengine3.git
@@ -12,7 +12,7 @@
 
 Package: cfengine3
 Architecture: any
-Depends: ${shlibs:Depends}, ${misc:Depends}
+Depends: lsb-base (>= 3.0-6), ${shlibs:Depends}, ${misc:Depends}
 Recommends: python
 Description: tool for configuring and maintaining network machines
  Cfengine is a suite of programs for integrated autonomic management
