--- debmany.orig	2018-03-06 20:18:57.011490293 +0100
+++ debmany	2018-03-06 20:19:22.838676740 +0100
@@ -413,7 +413,12 @@ do
     else
       # other file (usr/share/doc)
       debug "Opening other file: "`printf "$othercmdline" "$PWD/$return"` # comment
-      eval $(printf "$othercmdline" "$PWD/$return")
+      if [[ "$return" =~ \.gz$ ]]
+      then
+              eval $(printf "gzip -dc < $PWD/$return | $othercmdline")
+      else
+              eval $(printf "$othercmdline" "$PWD/$return")
+      fi
     fi
   else
     cd "$curdir"
