diff -ru dhelp-0.6.21/debian/changelog dhelp-0.6.21+nmu1/debian/changelog
--- dhelp-0.6.21/debian/changelog	2012-06-21 15:32:47.000000000 -0500
+++ dhelp-0.6.21+nmu1/debian/changelog	2012-07-02 12:19:30.000000000 -0500
@@ -1,3 +1,11 @@
+dhelp (0.6.21+nmu1) unstable; urgency=low
+
+  * Non-Maintainer Upload
+  * Dropped the declaration of dependence on ruby-commandline, which was
+    already done on the code
+
+ -- Gunnar Wolf <gwolf@debian.org>  Mon, 02 Jul 2012 12:16:40 -0500
+
 dhelp (0.6.21) unstable; urgency=low
 
   * New maintainer: Georgios M. Zarkadas <gz@member.fsf.org> (Closes: #650441). 
diff -ru dhelp-0.6.21/debian/control dhelp-0.6.21+nmu1/debian/control
--- dhelp-0.6.21/debian/control	2012-06-12 13:59:55.000000000 -0500
+++ dhelp-0.6.21+nmu1/debian/control	2012-07-02 12:13:32.000000000 -0500
@@ -10,10 +10,9 @@
 
 Package: dhelp
 Depends: perl-modules, libtemplate-perl, libhtml-parser-perl,
- liburi-perl, liblocale-gettext-perl, libdata-page-perl,
- ruby1.8, ruby-bdb | libdb-ruby1.8, ruby-commandline | libcommandline-ruby1.8,
- ruby-debian, libgettext-ruby1.8, doc-base, swish++, pstotext, poppler-utils,
- ucf (>= 0.8), ${misc:Depends}
+ liburi-perl, liblocale-gettext-perl, libdata-page-perl, ruby1.8,
+ ruby-bdb | libdb-ruby1.8, ruby-debian, libgettext-ruby1.8, doc-base,
+ swish++, pstotext, poppler-utils, ucf (>= 0.8), ${misc:Depends}
 Recommends: www-browser | html2text
 Suggests: httpd-cgi, info2www, man2html, catdvi
 Architecture: all
