--- base-passwd-3.5.24.orig/debian/rules	2011-07-16 13:42:25.000000000 +0200
+++ base-passwd-3.5.24/debian/rules	2012-01-11 19:27:51.000000000 +0100
@@ -39,14 +39,13 @@
 	$(MAKE) -C man clean
 	rm -rf debian/tmp debian/files debian/substvars
 
-ifeq (,$(filter noopt,$(DEB_BUILD_OPTIONS)))
-CFLAGS := -g -O2 -Wall
-else
-CFLAGS := -g -Wall
-endif
+CFLAGS = `dpkg-buildflags --get CFLAGS`
+CFLAGS += -Wall
+LDFLAGS = `dpkg-buildflags --get LDFLAGS`
+CFLAGS += `dpkg-buildflags --get CPPFLAGS`
 
 Makefile: Makefile.in configure.ac
-	CFLAGS="$(CFLAGS)" ./configure $(confflags)
+	CFLAGS="$(CFLAGS)" LDFLAGS="$(LDFLAGS)" ./configure $(confflags)
 
 .PHONY: binary
 binary: binary-arch binary-indep
