From: Jakub Wilk <jwilk@jwilk.net>
Date: Mon, 13 Feb 2023 12:09:17 +0100
Subject: [PATCH] Expand tabs in init script

---
 debian/apt-cacher-ng.init | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/debian/apt-cacher-ng.init b/debian/apt-cacher-ng.init
index 27f42b9..126da9c 100644
--- a/debian/apt-cacher-ng.init
+++ b/debian/apt-cacher-ng.init
@@ -26,7 +26,7 @@ test -x $DAEMON || exit 0
 
 # Include apt-cacher-ng defaults if available
 if [ -f /etc/default/apt-cacher-ng ] ; then
-	. /etc/default/apt-cacher-ng
+  . /etc/default/apt-cacher-ng
 fi
 
 # our runtime state files directory, will be purged on startup!
@@ -44,7 +44,7 @@ do_start() {
 
 do_stop() {
 
-	if ! start-stop-daemon --stop --retry 15 --quiet --pidfile $PIDFILE \
+  if ! start-stop-daemon --stop --retry 15 --quiet --pidfile $PIDFILE \
      --exec $DAEMON
   then
      if ! test -e "$PIDFILE" && ! start-stop-daemon --stop \
@@ -55,7 +55,7 @@ do_stop() {
   fi
   rm -f $PIDFILE
   return 0
-	
+
 }
 
 case "$1" in
-- 
2.39.1

