--- dracut-functions.sh.old	2013-09-24 17:34:16.000000000 +0200
+++ dracut-functions.sh	2013-09-24 17:34:40.000000000 +0200
@@ -74,11 +74,11 @@
 if ! [[ $libdirs ]] ; then
     if [[ "$(ldd /bin/sh)" == */lib64/* ]] &>/dev/null \
         && [[ -d /lib64 ]]; then
-        libdirs+=" /lib64"
-        [[ -d /usr/lib64 ]] && libdirs+=" /usr/lib64"
+        libdirs+=" /lib64 "
+        [[ -d /usr/lib64 ]] && libdirs+=" /usr/lib64 "
     else
-        libdirs+=" /lib"
-        [[ -d /usr/lib ]] && libdirs+=" /usr/lib"
+        libdirs+=" /lib "
+        [[ -d /usr/lib ]] && libdirs+=" /usr/lib "
     fi
 
     libdirs+="$(ldconfig_paths)"
