--- /usr/bin/checkinstall	2007-08-13 21:14:02.000000000 +0100
+++ checkinstall	2007-11-06 20:48:06.000000000 +0000
@@ -155,7 +155,7 @@
    echog "--stripso=<yes|no>             Strip any ELF binary libraries (.so files)" 
    echog "--addso=<yes|no>               Search for any shared libs and add"
    echog "                               them to /etc/ld.so.conf"
-   echog "--reset-uids=<yes|no>          Reset perms for all files/dirs to 755 and"
+   echog "--reset-uids=<yes|no>          Reset perms for all files to o=g, dirs to 755 and"
    echog "                               the owner/group for all dirs to root.root"
    echog "--gzman=<yes|no>               Compress any man pages found inside the package"
    echog "--docdir=<path>                Where to put documentation files"
@@ -1642,6 +1642,7 @@
 	if [ $RESET_UIDS -gt 0 ]; then
 	   find "$BUILD_DIR" -exec chown root.root {} \;
 	   find "$BUILD_DIR" -type d -exec chmod 755 {} \;
+           find "$BUILD_DIR" -type f -exec chmod o=g {} \;
 	fi
 
 	##############################################################################
