diff -Nru dpkg-1.14.25/scripts/Dpkg/Source/Package.pm dpkg-1.14.25a0.mrvn.1/scripts/Dpkg/Source/Package.pm
--- dpkg-1.14.25/scripts/Dpkg/Source/Package.pm	2009-02-01 07:19:01.000000000 +0100
+++ dpkg-1.14.25a0.mrvn.1/scripts/Dpkg/Source/Package.pm	2009-04-27 15:26:14.000000000 +0200
@@ -46,7 +46,7 @@
 # File-names that should be ignored (never directories)
 (?:^|/)(?:DEADJOE|\.cvsignore|\.arch-inventory|\.bzrignore|\.gitignore)$|
 # File or directory names that should be ignored
-(?:^|/)(?:CVS|RCS|\.deps|\{arch\}|\.arch-ids|\.svn|\.hg|_darcs|\.git|
+(?:^|/)(?:CVS|RCS|\.deps|\{arch\}|\.arch-ids|\.svn|\.hg|\.hgtags|_darcs|\.git|
 \.shelf|_MTN|\.bzr(?:\.backup|tags)?)(?:$|/.*$)
 ';
 # Take out comments and newlines
