--- askpass.c.orig	2014-12-17 12:18:19.507662401 +0100
+++ askpass.c	2014-12-17 12:20:56.720441977 +0100
@@ -195,13 +195,13 @@
 {
 	debug("In systemd_read\n");
 	if (fifo_common_read(fd, &systemdbuf, &systemdused, &systemdsize)) {
-		*buf = systemdbuf;
-		*size = systemdused;
 		/* systemd likes to include the terminating newline */
-		if (systemdused > 1 && systemdbuf[systemdused - 1] == '\n') {
+		if (systemdused >= 1 && systemdbuf[systemdused - 1] == '\n') {
 			systemdbuf[systemdused - 1] = '\0';
 			systemdused--;
 		}
+		*buf = systemdbuf;
+		*size = systemdused;
 		return true;
 	}
 
