--- /usr/share/doc/crafty/setup_crafty.sh	2010-11-30 13:54:31.000000000 +0100
+++ setup_crafty.sh	2013-09-16 18:16:05.605315292 +0200
@@ -35,7 +35,7 @@
 		
 		echo "Copying crafty.hlp"
 		echo "To use it you have to run crafty in $CPATH"
-		zcat /usr/share/doc/crafty/crafty.hlp.gz > $CPATH/crafty.hlp
+		cp -v /usr/share/doc/crafty/crafty.hlp $CPATH
 		echo ""
 
 		echo "For setting enviroment variables"
