diff -u enlightenment-0.16.7.2/debian/changelog enlightenment-0.16.7.2/debian/changelog
--- enlightenment-0.16.7.2/debian/changelog
+++ enlightenment-0.16.7.2/debian/changelog
@@ -1,3 +1,10 @@
+enlightenment (1:0.16.7.2-5.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Deprecate Freetype 1 (Closes: #473676)
+
+ -- Moritz Muehlenhoff <jmm@debian.org>  Wed, 23 Apr 2008 21:40:55 +0200
+
 enlightenment (1:0.16.7.2-5) unstable; urgency=low
 
   * fixed cache file build problem up on NFS mounts. Reported and fixed
diff -u enlightenment-0.16.7.2/debian/control enlightenment-0.16.7.2/debian/control
--- enlightenment-0.16.7.2/debian/control
+++ enlightenment-0.16.7.2/debian/control
@@ -2,7 +2,7 @@
 Section: x11
 Priority: optional
 Maintainer: Laurence J. Lane <ljlane@debian.org>
-Build-Depends: libttf-dev, libesd0-dev, libimlib2-dev, gettext, libsm-dev, libx11-dev, libxext-dev, libxrandr-dev, libxinerama-dev, libxxf86vm-dev, libxt-dev, x-dev, xlibs-data, xlibs-static-dev, cdbs, debhelper (>=4.1.0)
+Build-Depends: libfreetype6-dev, libesd0-dev, libimlib2-dev, gettext, libsm-dev, libx11-dev, libxext-dev, libxrandr-dev, libxinerama-dev, libxxf86vm-dev, libxt-dev, x-dev, xlibs-data, xlibs-static-dev, cdbs, debhelper (>=4.1.0)
 Standards-Version: 3.7.2
 
 Package: enlightenment
