--- dh_compress.orig	2013-11-25 21:33:05.278237722 +0900
+++ dh_compress	2013-11-25 21:54:36.896223647 +0900
@@ -118,7 +118,9 @@
 				! -iname "*.jpeg" ! -iname "*.gz" ! -iname "*.taz" \\
 				! -iname "*.tgz" ! -iname "*.z" ! -iname "*.bz2" \\
 				! -iname "*-gz"  ! -iname "*-z" ! -iname "*_z" \\
-				! -iname "*.jar" ! -iname "*.zip" ! -iname "*.css" \\
+				! -iname "*.epub" ! -iname "*.jar" ! -iname "*.zip" \\
+				! -iname "*.odg" ! -iname "*.odp" ! -iname "*.odt" \\
+				! -iname ".htaccess" ! -iname "*.css" \\
 				! -iname "*.svg" ! -iname "*.svgz" ! -iname "*.js" \\
 				! -name "index.sgml" ! -name "objects.inv" \\
 				! -name "copyright" 2>/dev/null || true;
