--- /usr/sbin/ddclient	2009-01-05 08:11:28.000000000 +1100
+++ /tmp/ddclient	2009-02-07 17:01:57.000000000 +1100
@@ -2599,11 +2599,11 @@
 o 'zoneedit1'
 
 The 'zoneedit1' protocol is used by a DNS service offered by
-www.zoneedit.com.
+dynamic.zoneedit.com.
 
 Configuration variables applicable to the 'zoneedit1' protocol are:
   protocol=zoneedit1           ## 
-  server=fqdn.of.service       ## defaults to www.zoneedit.com
+  server=fqdn.of.service       ## defaults to dynamic.zoneedit.com
   login=service-login          ## login name and password  registered with the service
   password=service-password    ##
   your.domain.name             ## the host registered with the service.
@@ -2611,7 +2611,7 @@
 Example ${program}.conf file entries:
   ## single host update
   protocol=zoneedit1,                                     \\
-  server=www.zoneedit.com,                                \\
+  server=dynamic.zoneedit.com,                                \\
   login=my-zoneedit-login,                                \\
   password=my-zoneedit-password                           \\
   my.domain.name
