Index: generators/91security
===================================================================
--- generators/91security	(revision 56524)
+++ generators/91security	(working copy)
@@ -14,6 +14,12 @@
 host="$RET"
 [ "$host" ] || exit
 
+if db_get apt-setup/security_directory && [ "$RET" ]; then
+	directory="$RET"
+else
+	directory="/"
+fi
+
 if ! db_get mirror/codename || [ -z "$RET" ]; then
 	db_get cdrom/codename
 fi
@@ -29,7 +35,7 @@
 done
 
 # Don't test mirror if no network selected in netcfg
-echo "deb http://$host/ $codename/updates $dists" >> $file
+echo "deb http://$host$directory $codename/updates $dists" >> $file
 if db_get netcfg/dhcp_options && \
    [ "$RET" = "Do not configure the network at this time" ]; then
 	CODE=9
@@ -46,6 +52,6 @@
 	fi
 fi
 
-echo "deb-src http://$host/ $codename/updates $dists" >> $file
+echo "deb-src http://$host$directory $codename/updates $dists" >> $file
 
 exit $CODE
