diff -Nru cloud-init-0.7.2/debian/changelog cloud-init-0.7.2/debian/changelog
--- cloud-init-0.7.2/debian/changelog	2013-05-29 07:21:09.000000000 +0100
+++ cloud-init-0.7.2/debian/changelog	2013-06-21 14:39:06.000000000 +0100
@@ -1,3 +1,13 @@
+cloud-init (0.7.2-1.1) UNRELEASED; urgency=low
+
+  * Non-maintainer upload.
+  * Comply with debian policy 9.11.1 by checking if init is upstart and
+    exiting in the init.d script.
+  * Depend on lsb-base 4.1+Debian3 or higher to use init_is_upstart
+    function.
+
+ -- Dmitrijs Ledkovs <dmitrij.ledkov@ubuntu.com>  Fri, 21 Jun 2013 14:37:18 +0100
+
 cloud-init (0.7.2-1) unstable; urgency=low
 
   * New upstream release.
diff -Nru cloud-init-0.7.2/debian/cloud-init.init cloud-init-0.7.2/debian/cloud-init.init
--- cloud-init-0.7.2/debian/cloud-init.init	2013-05-27 14:58:26.000000000 +0100
+++ cloud-init-0.7.2/debian/cloud-init.init	2013-06-21 14:37:13.000000000 +0100
@@ -33,6 +33,17 @@
 # and status_of_proc is working.
 . /lib/lsb/init-functions
 
+if init_is_upstart; then
+    case "$1" in
+	stop)
+	    exit 0
+	    ;;
+	*)
+	    exit 1
+	    ;;
+    esac
+fi
+
 case "$1" in
     start)
 	log_daemon_msg "Starting $DESC" "$NAME"
diff -Nru cloud-init-0.7.2/debian/control cloud-init-0.7.2/debian/control
--- cloud-init-0.7.2/debian/control	2013-05-29 07:20:59.000000000 +0100
+++ cloud-init-0.7.2/debian/control	2013-06-21 14:38:41.000000000 +0100
@@ -26,7 +26,7 @@
 
 Package: cloud-init
 Architecture: all
-Depends: lsb-base,
+Depends: lsb-base (>= 4.1+Debian3),
          ifupdown,
          procps,
          python,
