diff --git a/lib/licenses/apache b/lib/licenses/apache
index ba89271..84ccdba 100644
--- a/lib/licenses/apache
+++ b/lib/licenses/apache
@@ -1,5 +1,6 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: #PACKAGE#
+Upstream-Contact: <preferred name and address to reach the upstream project>
 Source: <url://example.com>
 
 Files: *
diff --git a/lib/licenses/artistic b/lib/licenses/artistic
index 9387711..69ae13c 100644
--- a/lib/licenses/artistic
+++ b/lib/licenses/artistic
@@ -1,5 +1,6 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: #PACKAGE#
+Upstream-Contact: <preferred name and address to reach the upstream project>
 Source: <url://example.com>
 
 Files: *
diff --git a/lib/licenses/blank b/lib/licenses/blank
index 420f1ac..b67795d 100644
--- a/lib/licenses/blank
+++ b/lib/licenses/blank
@@ -1,5 +1,6 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: #PACKAGE#
+Upstream-Contact: <preferred name and address to reach the upstream project>
 Source: <url://example.com>
 
 Files: *
diff --git a/lib/licenses/bsd b/lib/licenses/bsd
index 89d0a96..98519f2 100644
--- a/lib/licenses/bsd
+++ b/lib/licenses/bsd
@@ -1,5 +1,6 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: #PACKAGE#
+Upstream-Contact: <preferred name and address to reach the upstream project>
 Source: <url://example.com>
 
 Files: *
diff --git a/lib/licenses/gpl2 b/lib/licenses/gpl2
index d05404d..e7dff4e 100644
--- a/lib/licenses/gpl2
+++ b/lib/licenses/gpl2
@@ -1,5 +1,6 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: #PACKAGE#
+Upstream-Contact: <preferred name and address to reach the upstream project>
 Source: <url://example.com>
 
 Files: *
diff --git a/lib/licenses/gpl3 b/lib/licenses/gpl3
index fbef1b3..3d759f8 100644
--- a/lib/licenses/gpl3
+++ b/lib/licenses/gpl3
@@ -1,5 +1,6 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: #PACKAGE#
+Upstream-Contact: <preferred name and address to reach the upstream project>
 Source: <url://example.com>
 
 Files: *
diff --git a/lib/licenses/isc b/lib/licenses/isc
index 4ad1efa..4b213e7 100644
--- a/lib/licenses/isc
+++ b/lib/licenses/isc
@@ -1,5 +1,6 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: #PACKAGE#
+Upstream-Contact: <preferred name and address to reach the upstream project>
 Source: <url://example.com>
 
 Files: *
diff --git a/lib/licenses/lgpl2 b/lib/licenses/lgpl2
index f067eb2..56f0dae 100644
--- a/lib/licenses/lgpl2
+++ b/lib/licenses/lgpl2
@@ -1,5 +1,6 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: #PACKAGE#
+Upstream-Contact: <preferred name and address to reach the upstream project>
 Source: <url://example.com>
 
 Files: *
diff --git a/lib/licenses/lgpl3 b/lib/licenses/lgpl3
index b033863..8e6d5bc 100644
--- a/lib/licenses/lgpl3
+++ b/lib/licenses/lgpl3
@@ -1,5 +1,6 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: #PACKAGE#
+Upstream-Contact: <preferred name and address to reach the upstream project>
 Source: <url://example.com>
 
 Files: *
diff --git a/lib/licenses/mit b/lib/licenses/mit
index 1867d3a..a4a8a65 100644
--- a/lib/licenses/mit
+++ b/lib/licenses/mit
@@ -1,5 +1,6 @@
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: #PACKAGE#
+Upstream-Contact: <preferred name and address to reach the upstream project>
 Source: <url://example.com>
 
 Files: *
