From 88d3556c27c98f87b865f4b67ea86a29bb3d82ad Mon Sep 17 00:00:00 2001
From: Francesco Poli (t1000) <frx@firenze.linux.it>
Date: Mon, 10 May 2010 22:48:51 +0200
Subject: [PATCH 1/8] rearrange todo list

---
 debian/TODO |    5 ++++-
 1 files changed, 4 insertions(+), 1 deletions(-)

diff --git a/debian/TODO b/debian/TODO
index e59e2c3..ba6f326 100644
--- a/debian/TODO
+++ b/debian/TODO
@@ -1,10 +1,13 @@
-Release goals for version 0.1.4
+Release goals for version 0.1.5
 -------------------------------
 * check the pbuilder tests and make them work
 * close #532678 - apt-listbugs: faster method to select bugs
 * close #539487 - apt-listbugs: does not work via proxy using windows style user names
 * close #536613 - apt-listbugs: lists no bugs and returns 0 for non existing packages [waiting on debbugs, #553661]
 * close #553694 - apt-listbugs: remove html from translations
+
+Release goals for version 0.1.6
+-------------------------------
 * close #257873 - apt-listbugs: can check not only designated packages but also its source packages
 * close #562499 - apt-listbugs: make ParseStep configurable
 * hunt further leaks in dpkg-ruby. (see bug #564117)
-- 
1.7.1

