From 465f1e02ebb07be3d5a4c002476ad0e31449c90e Mon Sep 17 00:00:00 2001
From: Francesco Poli (t1000) <frx@firenze.linux.it>
Date: Wed, 2 Jun 2010 15:46:38 +0200
Subject: [PATCH 5/8] fixed typo ("browswer")

---
 lib/apt-listbugs/logic.rb |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/lib/apt-listbugs/logic.rb b/lib/apt-listbugs/logic.rb
index 0b2fed9..ba5d020 100644
--- a/lib/apt-listbugs/logic.rb
+++ b/lib/apt-listbugs/logic.rb
@@ -612,15 +612,15 @@ Pin-Priority: #{pin_pri}
       tmp.puts "</body></html>"
       tmp.close
 
-      puts "Invoking www-browswer for #{tmp.path}" if $DEBUG
+      puts "Invoking browser for #{tmp.path}" if $DEBUG
       browsercommandline = ""
       browsercommandline << "su #{ENV["SUDO_USER"]} -c \"" if ENV["SUDO_USER"]
       browsercommandline << "#{@config.browser} #{tmp.path} < /dev/tty"
       browsercommandline << "\"" if ENV["SUDO_USER"]
       if system(browsercommandline)
-        puts "successfully invoked www-browswer" if $DEBUG
+        puts "successfully invoked browser" if $DEBUG
       else
-        $stderr.puts _("W: Failed to invoke www-browser.")
+        $stderr.puts _("W: Failed to invoke browser.")
         $stderr.puts " #{browsercommandline}"
       end
       clear_stdin
-- 
1.7.1

