From c6a7cdcc6194a870835c39de825628df1d5d4adf Mon Sep 17 00:00:00 2001
From: Francesco Poli (t1000) <frx@firenze.linux.it>
Date: Sun, 25 Jul 2010 19:17:18 +0200
Subject: [PATCH 8/8] add another rescue

---
 apt-listbugs     |   11 ++++++++---
 debian/changelog |    3 ++-
 2 files changed, 10 insertions(+), 4 deletions(-)

diff --git a/apt-listbugs b/apt-listbugs
index 5089c7b..605efe1 100755
--- a/apt-listbugs
+++ b/apt-listbugs
@@ -324,9 +324,14 @@ end
 exit 0 if pkgnames.size == 0
 
 # creating new packages database
-new_pkgs = Factory::PackageFactory.create(pkgnames) { |msg, val|
-  config.frontend.progress(msg, val) if config.quiet == false
-}
+begin
+  new_pkgs = Factory::PackageFactory.create(pkgnames) { |msg, val|
+    config.frontend.progress(msg, val) if config.quiet == false
+  }
+rescue
+  config.frontend.puts " ... E: #{$!}"
+  exit 1
+end
 
 Factory::PackageFactory.delete_ignore_pkgs(new_pkgs) if config.command == "apt"
 # exitting if no new packages is found
diff --git a/debian/changelog b/debian/changelog
index c38b205..68607eb 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -18,12 +18,13 @@ apt-listbugs (0.1.4) UNRELEASED; urgency=low
   * changed exit status for errors encountered when reading bug reports: the
     right exit status is 1 ("something is wrong", as documented in the man
     page), not 10 (which is reserved for warning APT not to proceed)
+  * added minimal handling of errors encountered when reading package fields
 
   [ Ryan Niebur ]
   * updated German translation, thanks Thomas Mueller! (Closes: #578305)
   * flush stdin after running the web browser (Closes: #578299)
 
- -- Francesco Poli (t1000) <frx@firenze.linux.it>  Sun, 25 Jul 2010 18:18:19 +0200
+ -- Francesco Poli (t1000) <frx@firenze.linux.it>  Sun, 25 Jul 2010 18:32:02 +0200
 
 apt-listbugs (0.1.3) unstable; urgency=low
 
-- 
1.7.1

