From 8a37e6fc7ec4889b456efa7e7811c243d1758c41 Mon Sep 17 00:00:00 2001
From: Joachim Breitner <mail@joachim-breitner.de>
Date: Sun, 19 Jun 2011 16:09:05 +0200
Subject: [PATCH] Make builder column known to DBFactory

---
 lib/debian/wannabuild/db.py |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/lib/debian/wannabuild/db.py b/lib/debian/wannabuild/db.py
index b3c53c9..769b5ed 100644
--- a/lib/debian/wannabuild/db.py
+++ b/lib/debian/wannabuild/db.py
@@ -51,6 +51,7 @@ class DBFactory(object):
             Column('version', String, primary_key=True),
             Column('timestamp', DateTime, primary_key=True),
             Column('result', String),
+            Column('builder', String),
             Column('build_time', Integer),
             Column('disk_space', Integer),
             schema=arch)
-- 
1.7.5.4

