diff -r 84cd0521e303 src/generic/apt/apt.cc
--- a/src/generic/apt/apt.cc	Tue Jul 03 07:06:43 2007 -0700
+++ b/src/generic/apt/apt.cc	Thu Jul 05 20:54:36 2007 -0700
@@ -288,6 +288,9 @@ void apt_load_cache(OpProgress *progress
   apt_source_list->ReadMainList();
 
   bool simulate = aptcfg->FindB(PACKAGE "::Simulate", false);
+
+  // Clear the error stack so that we don't get confused by old errors.
+  consume_errors();
 
   bool open_failed=!new_file->Open(*progress_bar, do_initselections,
 				   (getuid() == 0) && !simulate,
