diff -ur console-setup-1.200/Keyboard/ckb/symbols/us console-setup-1.200m/Keyboard/ckb/symbols/us
--- console-setup-1.200/Keyboard/ckb/symbols/us	2020-05-10 05:02:58.000000000 -0600
+++ console-setup-1.200m/Keyboard/ckb/symbols/us	2021-05-24 19:43:20.384032517 -0600
@@ -216,7 +216,7 @@
     key <AE06> { [	    6,	asciicircum, dead_circumflex, dead_circumflex ]	};
     key <AE07> { [	    7,	ampersand	]	};
     key <AE08> { [	    8,	asterisk	]	};
-    key <AE09> { [	    9,	parenleft,  dead_grave]	};
+    key <AE09> { [	    9,	parenleft,  dead_grave, dead_breve	] };
     key <AE10> { [	    0,	parenright	]	};
     key <AE11> { [ bracketleft,	braceleft	]	};
     key <AE12> { [ bracketright, braceright,  dead_tilde] };
