Index: apt-show-versions
===================================================================
--- apt-show-versions	(revision 2469)
+++ apt-show-versions	(revision 2470)
@@ -205,9 +205,9 @@
 # Get hash with all installed packages from cache or create new hash
 # and write cache if root
 # $ipackages structure example:
-# {'dblatex' => {'Version' => '0.2.8-6',
-#                'Status' => 'install ok installed',
-#                'Package' => 'dblatex'},
+# {'dblatex' => {'amd64' => {'Version' => '0.2.8-6',
+#                            'Status' => 'install ok installed',
+#                            'Package' => 'dblatex'}},
 #  ...}
 my $ipackages;
 
@@ -222,18 +222,18 @@
 
 # Get available packages list from cache if possible
 # $apackages structure example:
-# {'dblatex' => {'ftp.de.debian.org_debian_dists_stable'
-#                 => {'Version' => '0.2-2',
-#                     'Release' => 'ftp.de.debian.org_debian_dists_stable',
-#                     'Package' => 'dblatex'},
-#                'ftp.de.debian.org_debian_dists_testing'
-#                 => {'Version' => '0.2.8-2',
-#                     'Release' => 'ftp.de.debian.org_debian_dists_testing',
-#                     'Package' => 'dblatex'},
-#                'ftp.de.debian.org_debian_dists_unstable'
-#                 => {'Version' => '0.2.8-6',
-#                     'Release' => 'ftp.de.debian.org_debian_dists_unstable',
-#                     'Package' => 'dblatex'}},
+# {'dblatex' => {'amd64' => {'ftp.de.debian.org_debian_dists_stable'
+#                             => {'Version' => '0.2-2',
+#                                 'Release' => 'ftp.de.debian.org_debian_dists_stable',
+#                                 'Package' => 'dblatex'},
+#                            'ftp.de.debian.org_debian_dists_testing'
+#                             => {'Version' => '0.2.8-2',
+#                                 'Release' => 'ftp.de.debian.org_debian_dists_testing',
+#                                 'Package' => 'dblatex'},
+#                            'ftp.de.debian.org_debian_dists_unstable'
+#                             => {'Version' => '0.2.8-6',
+#                                 'Release' => 'ftp.de.debian.org_debian_dists_unstable',
+#                                 'Package' => 'dblatex'}}},
 #  ...}
 my $apackages;
 my $cache_file_corrupt;
