Only in dlint-1.4.0.orig/debian: README.Debian
diff -ur dlint-1.4.0.orig/debian/changelog dlint-1.4.0/debian/changelog
--- dlint-1.4.0.orig/debian/changelog	2006-06-02 19:05:52.000000000 +0200
+++ dlint-1.4.0/debian/changelog	2006-06-02 19:04:51.000000000 +0200
@@ -1,3 +1,14 @@
+dlint (1.4.0-4.1) unstable; urgency=low
+
+  * Non-Maintainer Upload
+
+  * Finish the /usr/doc transition (Closes: #322782).
+  * Change Build-Depends-Indep on debhelper to Build-Depends.
+  * Remove obsolete README.Debian about non-US.
+  * Update FSF address in copyright.
+
+ -- Thijs Kinkhorst <kink@squirrelmail.org>  Fri,  2 Jun 2006 18:58:23 +0200
+
 dlint (1.4.0-4) unstable; urgency=low
 
   * Moved from non-US to US main. 
diff -ur dlint-1.4.0.orig/debian/control dlint-1.4.0/debian/control
--- dlint-1.4.0.orig/debian/control	2006-06-02 19:05:52.000000000 +0200
+++ dlint-1.4.0/debian/control	2006-06-02 19:05:14.000000000 +0200
@@ -3,7 +3,7 @@
 Priority: extra
 Maintainer: Florian Hinzmann <fh@debian.org>
 Standards-Version: 3.5.6
-Build-Depends-Indep: debhelper (>= 3.0.5)
+Build-Depends: debhelper (>= 3.0.5)
 
 Package: dlint
 Architecture: all
diff -ur dlint-1.4.0.orig/debian/copyright dlint-1.4.0/debian/copyright
--- dlint-1.4.0.orig/debian/copyright	2006-06-02 19:05:52.000000000 +0200
+++ dlint-1.4.0/debian/copyright	2006-06-02 19:03:12.000000000 +0200
@@ -18,5 +18,5 @@
 
     A copy of the GNU General Public License can be found at
     /usr/share/common-licenses/GPL or ordered from 
-    Free Software Foundation, Inc., 59 Temple Place - Suite 330, Boston,
-    MA 02111-1307, USA.
+    Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
+    Boston, MA  02110-1301, USA
