--- ddclient-3.8.0/debian/ddclient.init~	2009-02-18 23:25:24.000000000 +0100
+++ ddclient-3.8.0/debian/ddclient.init	2009-06-30 10:53:52.000000000 +0200
@@ -29,8 +29,8 @@
   . /etc/default/ddclient
 fi
 
-OPTIONS="-daemon $daemon_interval -syslog"
 PIDFILE=/var/run/$NAME.pid
+OPTIONS="-daemon $daemon_interval -syslog -pid $PIDFILE"
 
 # Load the VERBOSE setting and other rcS variables
 . /lib/init/vars.sh
