diff -ru clog-1.1.0.pristine/debian/control clog-1.1.0/debian/control
--- clog-1.1.0.pristine/debian/control	2013-09-20 15:38:50.000000000 +0100
+++ clog-1.1.0/debian/control	2013-09-28 17:55:02.152452619 +0100
@@ -13,8 +13,7 @@
 Architecture: any
 Depends: ${shlibs:Depends},
          ${misc:Depends}
-Description: highlight (or hide) important information
- clog is useful for watching log files. It can highlight important
- information or hide it. It let colorize patterns in the input and
- lines in the input, suppress lines matching a pattern in the input,
- add timestamps to all lines and add datestamps to all lines.
+Description: colorizing log tail utility
+ Clog is useful for watching log files. It can spot patterns in an input
+ stream and colorize or suppress the lines. It can also decorate the lines
+ with time- or datestamps.
