--- control	2008-09-08 12:09:22.000000000 -0430
+++ control    	2008-09-08 12:13:10.000000000 -0430
@@ -9,7 +9,7 @@
 Package: duplicity
 Architecture: any
 XB-Python-Version: ${python:Versions}
-Depends: ${shlibs:Depends}, ${python:Depends}, gnupg, python-gnupginterface, python-pexpect
+Depends: ${shlibs:Depends}, ${python:Depends}, gnupg, python-gnupginterface, python-pexpect, python-boto (>= 0.9d)
 Suggests: python-boto, ncftp
 Description: encrypted bandwidth-efficient backup
  Duplicity backs directories by producing encrypted tar-format volumes
