diff -u bochs-2.3.7/debian/control bochs-2.3.7/debian/control
--- bochs-2.3.7/debian/control
+++ bochs-2.3.7/debian/control
@@ -6,7 +6,8 @@
  autoconf, libz-dev, libncurses5-dev, libreadline5-dev | libreadline-dev,
  libaa1-dev, libsvga1-dev [i386 amd64], libx11-dev, libxpm-dev, libice-dev,
  libsm-dev, libsdl1.2-dev, libwxgtk2.6-dev, libgtk2.0-dev,
- libasound2-dev [alpha amd64 arm armel hppa i386 ia64 m68k mips mipsel powerpc ppc64 s390]
+ libasound2-dev [alpha amd64 arm armel hppa i386 ia64 m68k mips mipsel powerpc ppc64 s390],
+ bison, flex
 Build-Depends-Indep: iasl, bcc (>= 0.16.3-2), bin86 (>= 0.16.3-2),
  docbook-utils
 Homepage: http://bochs.sourceforge.net/
diff -u bochs-2.3.7/debian/rules bochs-2.3.7/debian/rules
--- bochs-2.3.7/debian/rules
+++ bochs-2.3.7/debian/rules
@@ -98,7 +98,7 @@
 	  --enable-mmx \
 	  --enable-3dnow \
 	  --enable-sse=4 \
-	  --enable-gdb-stub \
+	  --enable-debugger \
 	  --enable-disasm \
 	  --enable-idle-hack \
 	  --enable-all-optimizations \
only in patch2:
unchanged:
--- bochs-2.3.7.orig/bx_debug/symbols.cc
+++ bochs-2.3.7/bx_debug/symbols.cc
@@ -84,6 +84,9 @@
    in config.h */
 #if BX_HAVE_HASH_MAP
 #include <hash_map>
+#  if (defined(__GNUC__) && (((__GNUC__ == 3) && ( __GNUC_MINOR__ > 0)) || __GNUC__ >= 4))
+using __gnu_cxx::hash_map;
+#  endif
 #elif BX_HAVE_HASH_MAP_H
 #include <hash_map.h>
 #endif
