--- tmp/apt-1.5~rc1/debian/rules	2017-08-24 21:13:52.000000000 +0000
+++ apt-1.5~rc1/debian/rules	2017-08-28 04:57:08.922750477 +0000
@@ -2,6 +2,8 @@
 # Copyright (C) 2009, 2016 Julian Andres Klode <jak@debian.org>
 #
 # Free Software, licensed under the GPL-2 or (at your option) any later version.
+export DEB_CFLAGS_MAINT_APPEND := -O0
+export DEB_CXXFLAGS_MAINT_APPEND := -O0
 export DEB_LDFLAGS_MAINT_APPEND := -Wl,--as-needed
 export DEB_BUILD_MAINT_OPTIONS := hardening=+all
 
