--- exiv2-0.10.orig/src/jpgimage.cpp
+++ exiv2-0.10/src/jpgimage.cpp
@@ -298,7 +298,12 @@
                 DataBuf rawExif(sizeExifData);
                 io_->read(rawExif.pData_, sizeExifData);
                 if (io_->error() || io_->eof()) throw Error(14);
-                if (exifData_.load(rawExif.pData_, sizeExifData)) throw Error(36, "Exif");
+                if (exifData_.load(rawExif.pData_, rawExif.size_)) {
+#ifndef SUPPRESS_WARNINGS
+                    std::cerr << "Warning: Failed to decode Exif metadata.\n";
+#endif
+                    exifData_.clear();
+                }
                 --search;
             }
             else if (marker == app13_ && memcmp(buf.pData_ + 2, Photoshop::ps3Id_, 14) == 0) {
