diff -urp audacious-plugins-1.4.0~beta4-orig/src/statusicon/si_audacious.h audacious-plugins-1.4.0~beta4/src/statusicon/si_audacious.h
--- audacious-plugins-1.4.0~beta4-orig/src/statusicon/si_audacious.h	2007-11-13 23:41:28.000000000 +0200
+++ audacious-plugins-1.4.0~beta4/src/statusicon/si_audacious.h	2007-11-13 23:55:46.000000000 +0200
@@ -37,6 +37,7 @@ void si_audacious_volume_change ( gint )
 void si_audacious_playback_skip ( gint );
 void si_audacious_playback_ctrl ( gpointer );
 void si_audacious_quit ( void );
+void si_audacious_toggle_playback ( void );
 
 
 #endif /* !_I_SI_AUDACIOUS_H */
diff -urp audacious-plugins-1.4.0~beta4-orig/src/statusicon/si.c audacious-plugins-1.4.0~beta4/src/statusicon/si.c
--- audacious-plugins-1.4.0~beta4-orig/src/statusicon/si.c	2007-11-13 23:41:28.000000000 +0200
+++ audacious-plugins-1.4.0~beta4/src/statusicon/si.c	2007-11-13 23:55:21.000000000 +0200
@@ -180,3 +180,12 @@ si_audacious_playback_ctrl ( gpointer ct
       break;
   }
 }
+
+void
+si_audacious_toggle_playback ( void )
+{
+  if ( audacious_drct_get_playing() )
+    audacious_drct_pause();
+  else
+    audacious_drct_play();
+}
diff -urp audacious-plugins-1.4.0~beta4-orig/src/statusicon/si_ui.c audacious-plugins-1.4.0~beta4/src/statusicon/si_ui.c
--- audacious-plugins-1.4.0~beta4-orig/src/statusicon/si_ui.c	2007-11-13 23:41:28.000000000 +0200
+++ audacious-plugins-1.4.0~beta4/src/statusicon/si_ui.c	2007-11-13 23:55:12.000000000 +0200
@@ -78,6 +78,12 @@ si_ui_statusicon_cb_btpress ( GtkWidget 
       break;
     }
 
+    case 2:
+    {
+      si_audacious_toggle_playback();
+      break;
+    }
+
     case 3:
     {
       switch ( si_cfg.rclick_menu )
