--- blobwars-1.05.orig/src/title.cpp	2007-02-17 00:07:08.000000000 +0100
+++ blobwars-1.05/src/title.cpp	2007-02-16 05:09:35.000000000 +0100
@@ -430,8 +430,7 @@
 		
 		debug(("Map Name = %s", game.mapName))
 	}
-		
-	if (engine.skill == -1)
+	else if (engine.skill == -1)
 	{
 		game.setMapName("data/practice");
 		game.setStageName("Training Mission");
@@ -439,8 +438,7 @@
 		game.skill = 1;
 		return SECTION_GAME;
 	}
-	
-	if (engine.skill == 3)
+	else if (engine.skill == 3)
 	{
 		game.skill = 3;
 		game.hasAquaLung = game.hasJetPack = true;
