diff --git a/autoscripts/postinst-systemd-restart b/autoscripts/postinst-systemd-restart
index a619055..048e968 100644
--- a/autoscripts/postinst-systemd-restart
+++ b/autoscripts/postinst-systemd-restart
@@ -1,6 +1,6 @@
 if [ "$1" = "configure" ] || [ "$1" = "abort-upgrade" ] || [ "$1" = "abort-deconfigure" ] || [ "$1" = "abort-remove" ] ; then
-		if [ -d /run/systemd/system ]; then
-			systemctl --system daemon-reload >/dev/null || true
+	if [ -d /run/systemd/system ]; then
+		systemctl --system daemon-reload >/dev/null || true
 		if [ -n "$2" ]; then
 			_dh_action=restart
 		else
