--- /etc/aide/aide.conf.d/31_aide_cron-apt.orig	2008-04-05 16:13:00.000000000 +0200
+++ /etc/aide/aide.conf.d/31_aide_cron-apt	2009-08-10 09:02:28.000000000 +0200
@@ -4,7 +4,7 @@
 !/tmp/cron-apt\.[a-zA-Z0-9]{6}$
 !/tmp/cron-apt\.[a-zA-Z0-9]{6}/initlog$
 /var/log/cron-apt/log$ Log
-/var/log/cron-apt/log\.1$ LowLog
-/var/log/cron-apt/log\.2\.gz$ LoSerMemberLog
-/var/log/cron-apt/log\.[0-9]+\.gz$ SerMemberLog
+/var/log/cron-apt/log\.1\.gz$ LoSerMemberLog
+/var/log/cron-apt/log\.[23]\.gz$ SerMemberLog
+/var/log/cron-apt/log\.4\.gz$ HiSerMemberLog
 /var/log/cron-apt$ VarDir
