diff -aur amule-2.3.1.orig/debian/rules amule-2.3.1/debian/rules
--- amule-2.3.1.orig/debian/rules	2011-11-12 19:32:36.000000000 +0100
+++ amule-2.3.1/debian/rules	2011-12-29 00:11:27.000000000 +0100
@@ -32,13 +32,13 @@
 
 # CFLAGS
 
-CFLAGS = -Wall -g
+CFLAGS = `dpkg-buildflags --get CFLAGS`
+CFLAGS += -Wall
 
-ifneq (,$(findstring noopt,$(DEB_BUILD_OPTIONS)))
-	CFLAGS += -O0
-else
-	CFLAGS += -O2
-endif
+CXXFLAGS = `dpkg-buildflags --get CXXFLAGS`
+
+LDFLAGS_DEF += `dpkg-buildflags --get LDFLAGS`
+CPPFLAGS_DEF = `dpkg-buildflags --get CPPFLAGS`
 
 ###
 
@@ -52,7 +52,7 @@
 
 	-mkdir $(objdir)
 	cd $(objdir) && \
-	env CFLAGS="$(CFLAGS)" CXXFLAGS="$(CFLAGS)" ../configure $(confflags)
+	env CFLAGS="$(CFLAGS)" CXXFLAGS="$(CFLAGS)" CPPFLAGS="$(CPPFLAGS_DEF)" LDFLAGS="$(LDFLAGS_DEF)" ../configure $(confflags)
 
 	touch $@
 
Nur in amule-2.3.1/debian: rules~.
