diff --git a/ostable b/ostable
index f79ad0b..0cc9b8d 100644
--- a/ostable
+++ b/ostable
@@ -14,6 +14,8 @@
 # system part of the output of the GNU config.guess script.
 #
 # <Debian name>	<GNU name>	<config.guess regex>
+uclibceabi-linux	linux-uclibceabi	linux[^-]*-uclibceabi
+uclibc-linux	linux-uclibc	linux[^-]*-uclibc
 gnueabi-linux	linux-gnueabi	linux[^-]*-gnueabi
 gnulp-linux	linux-gnulp	linux[^-]*-gnulp
 gnu-linux	linux-gnu	linux[^-]*(-gnu.*)?
diff --git a/triplettable b/triplettable
index 79ca1d0..c0439f2 100644
--- a/triplettable
+++ b/triplettable
@@ -3,6 +3,8 @@
 # Supported variables: <cpu>
 #
 # <Debian triplet>	<Debian arch>
+uclibceabi-linux-arm	uclibc-linux-armel
+uclibc-linux-<cpu>	uclibc-linux-<cpu>
 gnueabi-linux-arm	armel
 gnulp-linux-i386	lpia
 gnu-linux-<cpu>		<cpu>
