diff -Nur old/dpkg-1.13.25/scripts/controllib.pl new/dpkg-1.13.25/scripts/controllib.pl
--- old/dpkg-1.13.25/scripts/controllib.pl	2007-08-29 16:35:03.000000000 +0300
+++ new/dpkg-1.13.25/scripts/controllib.pl	2007-11-20 17:47:34.000000000 +0200
@@ -95,7 +95,7 @@
 	return $cpu;
     } elsif ($os =~ /^(none|gnu)-(.*)/) {
 	return "$2-$cpu";
-    } elsif ("$os-$cpu" eq "gnueabi-linux-arm") {
+    } elsif ($cpu eq "arm" && $os =~ /gnueabi/) {
 	return "armel";
     } else {
 	return "$os-$cpu";
diff -Nur old/dpkg-1.13.25/ostable new/dpkg-1.13.25/ostable
--- old/dpkg-1.13.25/ostable	2007-08-29 16:35:05.000000000 +0300
+++ new/dpkg-1.13.25/ostable	2007-11-20 17:57:11.000000000 +0200
@@ -13,7 +13,7 @@
 # system part of the output of the GNU config.guess script.
 #
 # <Debian name>	<GNU name>	<config.guess regex>
-gnueabi-linux	linux-gnueabi	linux[^-]*-gnueabi
+gnueabi-linux	linux-gnueabi	linux[^-]*-[^-]*gnueabi
 gnu-linux	linux-gnu	linux[^-]*(-gnu)?
 none-darwin	darwin		darwin[^-]*
 none-freebsd	freebsd		freebsd[^-]*
