From d8badfe86c67797701bb7d8281e7cb6206344e8e Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Thu, 14 Feb 2013 04:41:05 +0100
Subject: [PATCH] Update dak reference from old katie name

---
 policy.sgml | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/policy.sgml b/policy.sgml
index 10e626b..73b8d6a 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -1729,7 +1729,7 @@ zope.
 /closes:\s*(?:bug)?\#?\s?\d+(?:,\s*(?:bug)?\#?\s?\d+)*/i
 	      </example>
 	      Then all of the bug numbers listed will be closed by the
-	      archive maintenance script (<prgn>katie</prgn>) using the
+	      archive maintenance software (<prgn>dak</prgn>) using the
 	      <var>version</var> of the changelog entry.
 	  </footnote>
 	  This information is conveyed via the <tt>Closes</tt> field
-- 
1.8.1.3

