diff -ur OLD/debian/control NEW/debian/control
--- OLD/debian/control	2012-10-10 12:12:58.000000000 +0200
+++ NEW/debian/control	2013-07-06 16:57:42.465366094 +0200
@@ -22,6 +22,7 @@
  dh-autoreconf,
  gtk-doc-tools,
  autopoint,
+ dh-systemd (>= 1.4)
 Standards-Version: 3.9.2
 Section: graphics
 Homepage: http://www.freedesktop.org/software/colord/
diff -ur OLD/debian/rules NEW/debian/rules
--- OLD/debian/rules	2012-10-10 12:12:58.000000000 +0200
+++ NEW/debian/rules	2013-07-06 16:57:42.465366094 +0200
@@ -49,4 +49,4 @@
 	NOCONFIGURE=1 dh_autoreconf ./autogen.sh
 
 %:
-	dh $@ --parallel --with autoreconf
+	dh $@ --parallel --with autoreconf --with=systemd
