diff -Nru libtorrent-rasterbar-1.2.9/debian/changelog libtorrent-rasterbar-1.2.9/debian/changelog
--- libtorrent-rasterbar-1.2.9/debian/changelog	2020-11-25 13:34:56.000000000 +0500
+++ libtorrent-rasterbar-1.2.9/debian/changelog	2020-12-05 16:49:55.000000000 +0500
@@ -1,3 +1,11 @@
+libtorrent-rasterbar (1.2.9-0.2) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Fix building Python bindings with -std=c++14 (Closes: #975953), idea from
+    Holger Hoffstätte, https://bugs.gentoo.org/739654.
+
+ -- Andrey Rahmatullin <wrar@debian.org>  Sat, 05 Dec 2020 16:49:55 +0500
+
 libtorrent-rasterbar (1.2.9-0.1) unstable; urgency=medium
 
   * Non-maintainer upload.
diff -Nru libtorrent-rasterbar-1.2.9/debian/rules libtorrent-rasterbar-1.2.9/debian/rules
--- libtorrent-rasterbar-1.2.9/debian/rules	2020-11-25 13:30:22.000000000 +0500
+++ libtorrent-rasterbar-1.2.9/debian/rules	2020-12-05 16:36:55.000000000 +0500
@@ -33,6 +33,8 @@
 		$(CONFIGURE_ARGS)
 	mv build-py$*/bindings/python build/bindings/python$*
 	cp -r bindings/python/* build/bindings/python$*
+	sed s/-std=c++11//g < build/bindings/python$*/compile_cmd > build/bindings/python$*/compile_cmd.new && \
+		mv -f build/bindings/python$*/compile_cmd.new build/bindings/python$*/compile_cmd
 
 override_dh_auto_configure: override_dh_auto_configure-nopy $(ALLPY:%=override_dh_auto_configure-%)
 
