diff --git a/packages/debootstrap/debootstrap b/packages/debootstrap/debootstrap
index ac821b8..086f5ea 100755
--- a/packages/debootstrap/debootstrap
+++ b/packages/debootstrap/debootstrap
@@ -466,6 +466,7 @@ if am_doing_phase finddebs; then
 	work_out_debs
 
 	base=$(without "$base $additional" "$exclude")
+	required=$(without "$required" "$exclude")
 
 	if [ "$RESOLVE_DEPS" = true ]; then
 		requiredX=$(echo $(echo $required | tr ' ' '\n' | sort | uniq))
