=== modified file 'methods/http.cc'
--- methods/http.cc	2013-01-24 17:52:25 +0000
+++ methods/http.cc	2013-03-02 14:47:10 +0000
@@ -668,6 +668,11 @@
    // The HTTP server expects a hostname with a trailing :port
    char Buf[1000];
    string ProperHost = Uri.Host;
+   if (ProperHost.find(':') != string::npos)
+   {
+      // Host is an IP-literal.
+      ProperHost = '[' + ProperHost + ']';
+   }
    if (Uri.Port != 0)
    {
       sprintf(Buf,":%u",Uri.Port);

