diff -Naur dhelp-0.6.16.orig/apache-dhelp.conf dhelp-0.6.16/apache-dhelp.conf
--- dhelp-0.6.16.orig/apache-dhelp.conf	2009-04-28 18:34:24.168265535 +0200
+++ dhelp-0.6.16/apache-dhelp.conf	2009-04-28 18:34:36.196310611 +0200
@@ -1,13 +1,13 @@
 <Location /cgi-bin/dsearch>
   order deny,allow
   deny from all
-  allow from 127.0.0.0/255.0.0.0
+  allow from localhost
   Options Indexes FollowSymLinks MultiViews
 </Location>
 
 <Location /cgi-bin/dhelp_fetcher>
   order deny,allow
   deny from all
-  allow from 127.0.0.0/255.0.0.0
+  allow from localhost
   Options Indexes FollowSymLinks MultiViews
 </Location>
