--- debhelper-5.0.42/dh_installudev	2006-04-18 16:13:46.000000000 +0200
+++ debhelper-5.0.42.fixed/dh_installudev	2007-01-15 02:09:42.000000000 +0100
@@ -69,6 +69,10 @@
 	my $tmp=tmpdir($package);
 	my $rules_file=pkgfile($package,"udev");
 	my $filename=basename($rules_file);
+
+	if ($filename == "udev") {
+		$filename = $package . ".udev";
+	}
 	$filename=~s/\.udev$/.rules/;
 
 	if ($rules_file) {
