--- debian/control.orig	2017-02-06 09:41:40.992315257 +0900
+++ debian/control	2017-02-06 09:42:09.657890179 +0900
@@ -23,6 +23,7 @@
 Depends: ${misc:Depends}, ${shlibs:Depends}
 Replaces: btrfs-tools (<< 4.4.1-1)
 Breaks: btrfs-tools (<< 4.4.1-1)
+Suggests: duperemove
 Description: Checksumming Copy on Write Filesystem utilities
  Btrfs is a new copy on write filesystem for Linux aimed at implementing
  advanced features while focusing on fault tolerance, repair and easy
