--- debian/control.orig	2017-02-06 20:18:56.932886041 +0900
+++ debian/control	2017-02-06 20:19:17.017559518 +0900
@@ -12,6 +12,7 @@
 Architecture: linux-any
 Multi-Arch: foreign
 Depends: ${shlibs:Depends}, ${misc:Depends}
+Enhances: btrfs-progs
 Description: extent-based deduplicator for file systems
  Duperemove is a tool for finding duplicated extents and submitting them for
  deduplication.  When given a list of files it will hash their contents on a
