diff -ruN eggdrop-1.6.19/configure eggdrop-1.6.19-threads/configure
--- eggdrop-1.6.19/configure	2008-04-19 04:21:20.000000000 +0000
+++ eggdrop-1.6.19-threads/configure	2008-08-03 19:25:20.000000000 +0000
@@ -10034,7 +10034,6 @@
 fi
 
 
-  if test "$egg_cv_var_tcl_threaded" = "yes"; then
     if test "$enable_tcl_threads" = "no"; then
 
       cat << 'EOF' >&2
@@ -10056,7 +10055,6 @@
     if test ! "${ac_cv_lib_pthread-x}" = "x"; then
       LIBS="$ac_cv_lib_pthread $LIBS"
     fi
-  fi
 
 
   if test "$EGG_CYGWIN" = "yes"; then
diff -ruN eggdrop-1.6.19/debian/rules eggdrop-1.6.19-threads/debian/rules
--- eggdrop-1.6.19/debian/rules	2008-08-03 18:19:59.000000000 +0000
+++ eggdrop-1.6.19-threads/debian/rules	2008-08-03 19:04:59.000000000 +0000
@@ -5,6 +5,7 @@
 include /usr/share/cdbs/1/class/autotools.mk
 include /usr/share/cdbs/1/rules/simple-patchsys.mk
 
+DEB_CONFIGURE_EXTRA_FLAGS := --enable-tcl-threads
 DEB_CONFIGURE_SCRIPT_ENV :=
 DEB_INSTALL_MANPAGES_eggdrop-data := doc/man1/eggdrop.1
 DEB_MAKE_INSTALL_TARGET := install prefix=$(DEB_DESTDIR)/usr
