diff -urN dillo-0.8.5/dpi/file.c dillo-0.8.5FIX2/dpi/file.c
--- dillo-0.8.5/dpi/file.c	2005-06-14 08:12:21.000000000 -0700
+++ dillo-0.8.5FIX2/dpi/file.c	2007-04-26 22:30:18.000000000 -0700
@@ -271,9 +271,8 @@
          continue;              /* skip "." and ".." */
 
       if (HIDE_DOTFILES) {
-         /* Don't add hidden files or backup files to the list */
+         /* Don't add hidden files to the list */
          if (de->d_name[0] == '.' ||
-             de->d_name[0] == '#' ||
              (de->d_name[0] != '\0' &&
               de->d_name[strlen(de->d_name) - 1] == '~'))
             continue;
