diff -u dmraid-1.0.0.rc15/debian/initramfs/dmraid.initramfs-hook/dmraid dmraid-1.0.0.rc15/debian/initramfs/dmraid.initramfs-hook/dmraid
--- dmraid-1.0.0.rc15/debian/initramfs/dmraid.initramfs-hook/dmraid
+++ dmraid-1.0.0.rc15/debian/initramfs/dmraid.initramfs-hook/dmraid
@@ -23,7 +23,9 @@
 	force_load dm-raid45
 	copy_exec /sbin/dmraid sbin
 	copy_exec /sbin/dmraid-activate sbin
-	copy_exec /bin/grep bin
+	if [ "$BUSYBOX" = n ]; then
+		copy_exec /bin/grep bin
+	fi
 	if [ -f /etc/udev/rules.d/85_dmraid.rules ]; then
 		mkdir -p ${DESTDIR}/etc/udev/rules.d
 		cp -p /etc/udev/rules.d/85_dmraid.rules ${DESTDIR}/etc/udev/rules.d
