diff -rauN ../orig/adduser-3.110/debian/control ./adduser-3.110/debian/control
--- ../orig/adduser-3.110/debian/control	2008-08-04 01:11:44.000000000 +0200
+++ ./adduser-3.110/debian/control	2009-09-07 14:21:03.905389339 +0200
@@ -5,7 +5,7 @@
 Uploaders: Joerg Hoh <joerg@joerghoh.de>, Roland Bauerschmidt <rb@debian.org>, Stephen Gran <sgran@debian.org>
 Standards-Version: 3.8.0
 Build-Depends: po-debconf
-Build-Depends-Indep: gettext (>= 0.13), po4a (>= 0.23)
+Build-Depends-Indep: gettext (>= 0.13), po4a (>= 0.31)
 Vcs-Svn: svn://svn.debian.org/adduser/
 Vcs-Browser: http://svn.debian.org/wsvn/adduser/
 Homepage: http://alioth.debian.org/projects/adduser/
diff -rauN ../orig/adduser-3.110/debian/rules ./adduser-3.110/debian/rules
--- ../orig/adduser-3.110/debian/rules	2008-03-02 17:59:31.000000000 +0100
+++ ./adduser-3.110/debian/rules	2009-09-07 14:21:51.165389474 +0200
@@ -7,14 +7,14 @@
 	$(checkdir)
 	$(MAKE) -C po all adduser.pot
 	# generate man pages
-	cd doc/po4a && po4a po4a.conf
+	cd doc/po4a && po4a --previous po4a.conf
 	touch build
 
 clean:
 	$(checkdir)
 	$(MAKE) -C po update clean
 	debconf-updatepo
-	cd doc/po4a && po4a --rm-translations po4a.conf
+	cd doc/po4a && po4a --previous --rm-translations po4a.conf
 	-rm -f doc/po4a/po/*~
 	-rm -rf build *~ debian/tmp debian/*~ debian/files* debian/substvars
 
diff -rauN ../orig/adduser-3.110/doc/po4a/po/fr.po ./adduser-3.110/doc/po4a/po/fr.po
--- ../orig/adduser-3.110/doc/po4a/po/fr.po	2008-08-10 23:35:14.000000000 +0200
+++ ./adduser-3.110/doc/po4a/po/fr.po	2009-09-07 14:20:53.058062117 +0200
@@ -6,7 +6,7 @@
 msgstr ""
 "Project-Id-Version: adduser\n"
 "POT-Creation-Date: 2008-08-10 22:35+0200\n"
-"PO-Revision-Date: 2008-06-07 15:33+0200\n"
+"PO-Revision-Date: 2009-09-07 14:20+0200\n"
 "Last-Translator: Nicolas François <nicolas.francois@centraliens.net>\n"
 "Language-Team: Debian French Team <debian-l10n-french@lists.debian.org>\n"
 "MIME-Version: 1.0\n"
@@ -52,13 +52,12 @@
 
 # type: Plain text
 #: ../adduser.8:14
-#, fuzzy
 msgid ""
 "B<adduser> [options] [--home DIR] [--shell SHELL] [--no-create-home] [--uid "
 "ID] [--firstuid ID] [--lastuid ID] [--ingroup GROUP | --gid ID] [--disabled-"
 "password] [--disabled-login] [--gecos GECOS] [--add_extra_groups] user"
 msgstr ""
-"B<adduser> [options] [--home REP] [--shell|-s SHELL] [--no-create-home] [--"
+"B<adduser> [options] [--home REP] [--shell SHELL] [--no-create-home] [--"
 "uid ID] [--firstuid ID] [--lastuid ID] [--ingroup GROUPE | --gid ID] [--"
 "disabled-password] [--disabled-login] [--gecos GECOS] [--add_extra_groups] "
 "utilisateur"
