Index: doc/po4a/po4a.conf
===================================================================
--- doc/po4a/po4a.conf	(revision 398)
+++ doc/po4a/po4a.conf	(working copy)
@@ -9,22 +9,22 @@
 		es:../adduser.8.es add_es:translator_spanish.add \
 		pl:../adduser.8.pl add_pl:translator_polish.add \
 		pt_BR:../adduser.8.pt_BR add_pt_BR:translator_pt_BR.add \
-		ru:../adduser.8.ru add_ru:translator_russian.add
+		ru:../adduser.8.ru add_ru:translator_russian.add opt_ru:"-L KOI8-R"
 [type:man] ../adduser.conf.5 \
 		fr:../adduser.conf.5.fr add_fr:translator_french2.add \
 		es:../adduser.conf.5.es add_es:translator_spanish.add \
 		pl:../adduser.conf.5.pl add_pl:translator_polish.add \
 		pt_BR:../adduser.conf.5.pt_BR add_pt_BR:translator_pt_BR.add \
-		ru:../adduser.conf.5.ru add_ru:translator_russian.add
+		ru:../adduser.conf.5.ru add_ru:translator_russian.add opt_ru:"-L KOI8-R"
 [type:man] ../deluser.8 \
 		fr:../deluser.8.fr add_fr:translator_french.add \
 		es:../deluser.8.es add_es:translator_spanish.add \
 		pl:../deluser.8.pl add_pl:translator_polish.add \
 		pt_BR:../deluser.8.pt_BR add_pt_BR:translator_pt_BR.add \
-		ru:../deluser.8.ru add_ru:translator_russian.add
+		ru:../deluser.8.ru add_ru:translator_russian.add opt_ru:"-L KOI8-R"
 [type:man] ../deluser.conf.5 \
 		fr:../deluser.conf.5.fr add_fr:translator_french.add \
 		es:../deluser.conf.5.es add_es:translator_spanish.add \
 		pl:../deluser.conf.5.pl add_pl:translator_polish.add \
 		pt_BR:../deluser.conf.5.pt_BR add_pt_BR:translator_pt_BR.add \
-		ru:../deluser.conf.5.ru add_ru:translator_russian.add
+		ru:../deluser.conf.5.ru add_ru:translator_russian.add opt_ru:"-L KOI8-R"
