diff -uNrp debootstrap-1.0.79.orig/debootstrap debootstrap-1.0.79/debootstrap
--- debootstrap-1.0.79.orig/debootstrap	2015-11-09 13:01:25.000000000 +0900
+++ debootstrap-1.0.79/debootstrap	2016-03-06 19:22:04.970728229 +0900
@@ -686,7 +686,7 @@ if am_doing_phase second_stage; then
 			mv "$TARGET/debootstrap/debootstrap.log" "$TARGET/var/log/bootstrap.log"
 		fi
 	fi
-	sync
+	sync -f "$TARGET"
 
 	if [ "$KEEP_DEBOOTSTRAP_DIR" = true ]; then
 		if [ -x "$TARGET/debootstrap/debootstrap" ]; then
