Index: debian/control
===================================================================
--- debian/control	(revision 888)
+++ debian/control	(working copy)
@@ -79,7 +79,8 @@
     placing them  under debian/ or, if present, debian/patches
   - dscverify: verify the integrity of a Debian package from the
     .changes or .dsc files [gnupg, debian-keyring, libdigest-md5-perl]
-  - grep-excuses: grep the update_excuses.html file for your packages [wget]
+  - grep-excuses: grep the update_excuses.html file for your packages [wget, 
+    libterm-size-perl]
   - licensecheck: attempt to determine the license of source files
   - list-unreleased: searches for unreleased packages
   - manpage-alert: locate binaries without corresponding manpages [man-db]
