diff --git a/debian/copyright b/debian/copyright
index bfe7071..1717327 100644
--- a/debian/copyright
+++ b/debian/copyright
@@ -72,6 +72,24 @@ libgnomecanvasmm Development Team
 The Files under libs/libsndfile are Copyright 1999-2005 Erik de Castro
 Lopo <erikd@mega-nerd.com>.
 
+The files under libs/libsndfile/GSM610/ are Copyright 1992, 1993, 1994
+by Jutta Degener and Carsten Bormann, Technische Universitaet Berlin,
+according to this license:
+
+   Any use of this software is permitted provided that this notice is not
+   removed and that neither the authors nor the Technische Universitaet Berlin
+   are deemed to have made any representations as to the suitability of this
+   software for any purpose nor are held responsible for any defects of
+   this software.  THERE IS ABSOLUTELY NO WARRANTY FOR THIS SOFTWARE.
+   
+   As a matter of courtesy, the authors request to be informed about uses
+   this software has found, about bugs in this software, and about any
+   improvements that may be of general interest.
+   
+   Berlin, 28.11.1994
+   Jutta Degener
+   Carsten Bormann
+
 The Files under libs/midi++2 are Copyright (C) 1998 Paul Barton-Davis
 
 The Files under libs/sigc++2 are Copyright 2002, The libsigc++
