--- Packages-arch-specific	2009-07-21 00:31:30.000000000 +0800
+++ Packages-arch-specific.new	2009-07-21 16:53:37.000000000 +0800
@@ -248,7 +248,7 @@
 %open-vm-tools: i386 amd64					      # [ANAIS]
 %openhackware: powerpc						      # per Guillem Jover
 %openoffice.org: !hppa !alpha                                         # 491623
-%oprofile: amd64 i386 ia64 alpha armel hppa powerpc sparc	      # [ANAIS]
+%oprofile: amd64 i386 ia64 alpha arm armel hppa powerpc sparc mips mipsel s390 # [ANAIS]
 orpie: !hppa !sparc                                                   # #444494
 %partimage: !ia64 !alpha !amd64					      # 64-bit is br0ken
 %paxtest: i386							      # [ANAIS]
