diff -Nru equivs-2.0.9/debian/changelog equivs-2.0.9+multiarch1/debian/changelog
--- equivs-2.0.9/debian/changelog	2011-09-30 07:22:31.000000000 +0100
+++ equivs-2.0.9+multiarch1/debian/changelog	2012-11-11 02:14:05.000000000 +0000
@@ -1,3 +1,9 @@
+equivs (2.0.9+multiarch1) unstable; urgency=low
+
+  * Add Multi-Arch field support
+
+ -- Wookey <wookey@wookware.org>  Sun, 11 Nov 2012 02:13:18 +0000
+
 equivs (2.0.9) unstable; urgency=low
 
   * Upgrade to Policy 3.9.2 - no changes.
diff -Nru equivs-2.0.9/usr/bin/equivs-build equivs-2.0.9+multiarch1/usr/bin/equivs-build
--- equivs-2.0.9/usr/bin/equivs-build	2011-09-30 07:22:24.000000000 +0100
+++ equivs-2.0.9+multiarch1/usr/bin/equivs-build	2012-11-11 02:13:07.000000000 +0000
@@ -309,6 +309,7 @@
 			   "Breaks",
 			   "Provides",
 			   "Replaces",
+			   "Multi-Arch",
 			   "Description");
   close OUT;
 }
diff -Nru equivs-2.0.9/usr/share/equivs/template/debian/control.in equivs-2.0.9+multiarch1/usr/share/equivs/template/debian/control.in
--- equivs-2.0.9/usr/share/equivs/template/debian/control.in	2011-09-30 06:25:26.000000000 +0100
+++ equivs-2.0.9+multiarch1/usr/share/equivs/template/debian/control.in	2012-11-11 02:23:46.000000000 +0000
@@ -6,6 +6,7 @@
 
 Package: equivs-dummy
 Architecture: all
+Multi-Arch: foreign
 Description: Dummy package to fulfill package dependencies
  This is a dummy package that makes Debian's package management
  system believe that equivalents to packages on which other
diff -Nru equivs-2.0.9/usr/share/equivs/template.ctl equivs-2.0.9+multiarch1/usr/share/equivs/template.ctl
--- equivs-2.0.9/usr/share/equivs/template.ctl	2011-09-30 06:59:19.000000000 +0100
+++ equivs-2.0.9+multiarch1/usr/share/equivs/template.ctl	2012-11-11 02:23:07.000000000 +0000
@@ -16,6 +16,7 @@
 # Provides: <comma-separated list of packages>
 # Replaces: <comma-separated list of packages>
 # Architecture: all
+# Multi-Arch: <one of: foreign|same|allowed>
 # Copyright: <copyright file; defaults to GPL2>
 # Changelog: <changelog file; defaults to a generic changelog>
 # Readme: <README.Debian file; defaults to a generic one>
