--- collectd2html.pl.orig	2007-08-15 12:58:20.000000000 +0300
+++ collectd2html.pl	2007-08-01 01:31:50.000000000 +0300
@@ -1,4 +1,4 @@
-#!/usr/bin/perl
+#!/usr/bin/perl -w
 
 ################################################################################
 #
@@ -36,12 +36,20 @@
 use strict;
 use Fatal qw(open close);
 use File::Basename;
+use Getopt::Long qw(:config no_ignore_case bundling pass_through);
+
+my $DIR  = "/var/lib/collectd";
+my $HOST = "_UNDEFINED_";
+
+GetOptions (
+    "host-is=s"  => \$HOST,
+    "data-dir=s" => \$DIR
+);
 
-my $DIR = "/var/lib/collectd";
 my @COLORS = (0xff7777, 0x7777ff, 0x55ff55, 0xffcc77, 0xff77ff, 0x77ffff,
 	0xffff77, 0x55aaff);
 my @tmp = `/bin/hostname`; chomp(@tmp);
-my $HOST = $tmp[0];
+$HOST = $tmp[0] if ( $HOST =~ /_UNDEFINED_/ );
 my $IMG_DIR = "${HOST}.dir";
 my $HTML = "${HOST}.html";
 
