diff -r a7f3dfe362ea local.c
--- a/local.c	Tue Aug 02 13:20:54 2011 +0200
+++ b/local.c	Tue Aug 02 14:18:58 2011 +0200
@@ -106,7 +106,8 @@
 				}
 				p += written;
 			}
-			names[--nameslen] = '\0';		/* chop trailing space */
+			if(nameslen > 0) --nameslen;
+			names[nameslen] = '\0';		/* chop trailing space */
 		} else {
 			nameslen = (strlen(force_mda) + 3);	// 'force_mda'
 			names = (char *)xmalloc(nameslen + 1);	// 'force_mda'\0
