--- /usr/bin/dh_installcron	2010-08-07 21:11:29.000000000 +0530
+++ dh_installcron	2011-07-05 17:35:12.000000000 +0530
@@ -43,8 +43,8 @@
 
 =item B<--name=>I<name>
 
-Look for files named debian/package.name.cron.* and install them as
-etc/cron.*/name, instead of using the usual files and installing them
+Look for files named debian/I<package>.cron.* and install them as
+etc/cron.*/I<name>, instead of using the usual files and installing them
 as the package name.
 
 =back
