diff --git a/debian/control b/debian/control
index f512d4f..1166a0e 100644
--- a/debian/control
+++ b/debian/control
@@ -3,7 +3,7 @@ Section: devel
 Priority: optional
 Maintainer: Debian Java Maintainers <pkg-java-maintainers@lists.alioth.debian.org>
 Uploaders: Arnaud Vandyck <avdyk@debian.org>, John Leuner <jewel@debian.org>, Michael Koch <konqueror@gmx.de>, Barry Hawkins <barry@bytemason.org>
-Build-Depends: debhelper (>= 4), default-jdk-builddep, python, autotools-dev, python-all-dev (>= 2.3.5-11), sharutils
+Build-Depends: debhelper (>= 4), default-jdk-builddep, python, autotools-dev, python-all-dev (>= 2.3.5-11), sharutils, java-gcj-compat-dev
 Build-Depends-Indep: python-support (>= 0.3), nant, cli-common-dev (>= 0.4.4), mono-mcs (>= 1.0) | c-sharp-compiler
 Standards-Version: 3.7.3
 Vcs-Svn: svn://svn.debian.org/svn/pkg-java/trunk/antlr
