--- /usr/share/initramfs-tools/scripts/local-top/cryptroot.orig	2014-10-03 17:14:21.000000000 +0200
+++ /usr/share/initramfs-tools/scripts/local-top/cryptroot	2014-10-12 17:55:49.953232090 +0200
@@ -274,7 +274,7 @@
 	# Try to get a satisfactory password $crypttries times
 	count=0
 	while [ $crypttries -le 0 ] || [ $count -lt $crypttries ]; do
-		export CRYPTTAB_TRIED="$crypttries"
+		export CRYPTTAB_TRIED="$count"
 		count=$(( $count + 1 ))
 
 		if [ -z "$cryptkeyscript" ]; then
