diff --git a/1/class/autotools-vars.mk.in b/1/class/autotools-vars.mk.in
index 13fff0b..fd493ba 100644
--- a/1/class/autotools-vars.mk.in
+++ b/1/class/autotools-vars.mk.in
@@ -45,6 +45,7 @@ DEB_CONFIGURE_SCRIPT ?= $(CURDIR)/$(DEB_SRCDIR)/configure
 DEB_CONFIGURE_CROSSBUILD_ARGS ?= --build=$(DEB_BUILD_GNU_TYPE) $(if $(cdbs_crossbuild),--host=$(DEB_HOST_GNU_TYPE))
 
 DEB_CONFIGURE_PREFIX ?=/usr
+DEB_CONFIGURE_EXECDIR ?=$(DEB_CONFIGURE_PREFIX)
 DEB_CONFIGURE_INCLUDEDIR ?= "\$${prefix}/include"
 DEB_CONFIGURE_MANDIR ?= "\$${prefix}/share/man"
 DEB_CONFIGURE_INFODIR ?= "\$${prefix}/share/info"
@@ -54,7 +55,15 @@ DEB_CONFIGURE_LIBEXECDIR ?= "\$${prefix}/lib/$(DEB_SOURCE_PACKAGE)"
 # --srcdir=. is required because otherwise configure wants to analyse
 # $0 to see whether a VPATH build is needed.  This tells it with
 # absolute certainly that this is NOT a VPATH build.
-DEB_CONFIGURE_PATH_ARGS ?= --prefix=$(DEB_CONFIGURE_PREFIX) --includedir=$(DEB_CONFIGURE_INCLUDEDIR) --mandir=$(DEB_CONFIGURE_MANDIR) --infodir=$(DEB_CONFIGURE_INFODIR) --sysconfdir=$(DEB_CONFIGURE_SYSCONFDIR) --localstatedir=$(DEB_CONFIGURE_LOCALSTATEDIR) --libexecdir=$(DEB_CONFIGURE_LIBEXECDIR) $(if $(subst $(DEB_SRCDIR),,$(cdbs_make_curbuilddir)),,--srcdir=.)
+DEB_CONFIGURE_PATH_ARGS ?= --prefix=$(DEB_CONFIGURE_PREFIX) \
+						   --exec-prefix=$(DEB_CONFIGURE_EXECDIR) \
+						   --includedir=$(DEB_CONFIGURE_INCLUDEDIR) \
+						   --mandir=$(DEB_CONFIGURE_MANDIR) \
+						   --infodir=$(DEB_CONFIGURE_INFODIR) \
+						   --sysconfdir=$(DEB_CONFIGURE_SYSCONFDIR) \
+						   --localstatedir=$(DEB_CONFIGURE_LOCALSTATEDIR) \
+						   --libexecdir=$(DEB_CONFIGURE_LIBEXECDIR) \
+						   $(if $(subst $(DEB_SRCDIR),,$(cdbs_make_curbuilddir)),,--srcdir=.)
 
 DEB_CONFIGURE_NORMAL_ARGS ?= $(DEB_CONFIGURE_CROSSBUILD_ARGS) $(DEB_CONFIGURE_PATH_ARGS) --disable-maintainer-mode --disable-dependency-tracking --disable-silent-rules
 
