diff -aur bochs-2.4.6.orig/debian/rules bochs-2.4.6/debian/rules
--- bochs-2.4.6.orig/debian/rules	2011-10-31 16:42:24.000000000 +0100
+++ bochs-2.4.6/debian/rules	2011-12-29 01:14:30.000000000 +0100
@@ -19,13 +19,11 @@
   conf_gnu_type += --build $(DEB_BUILD_GNU_TYPE) --host $(DEB_HOST_GNU_TYPE)
 endif
 
-CFLAGS = -Wall -g
+CFLAGS = `dpkg-buildflags --get CFLAGS`
+CFLAGS += -Wall
 
-ifneq (,$(filter noopt,$(DEB_BUILD_OPTIONS)))
-  CFLAGS += -O0
-else
-  CFLAGS += -O2
-endif
+LDFLAGS_DEF = `dpkg-buildflags --get LDFLAGS`
+CPPFLAGS_DEF = `dpkg-buildflags --get CPPFLAGS`
 
 sb16 := dummy
 cdrom := cdrom
@@ -93,7 +91,7 @@
 	# Note: We disable docbook support here as a cheap way to avoid
 	# building it in build-arch, and only in build-indep. This works
 	# because configure only disables entering the directory.
-	CFLAGS="$(CFLAGS)" CXXFLAGS="$(CFLAGS)" ./configure \
+	CFLAGS="$(CFLAGS)" CXXFLAGS="$(CFLAGS)" CPPFLAGS="$(CPPFLAGS_DEF)" LDFLAGS="$(LDFLAGS_DEF)" ./configure \
 	  $(conf_gnu_type) \
 	  --prefix=/usr \
 	  --mandir=\$${prefix}/share/man \
Nur in bochs-2.4.6/debian: rules~.
