cvs diff: Diffing .
cvs diff: Diffing debian
Index: debian/control
===================================================================
RCS file: /cvs/pkg-goodies-el/emacs-goodies-el/debian/control,v
retrieving revision 1.106
diff -u -u -r1.106 control
--- debian/control	17 Aug 2011 03:16:21 -0000	1.106
+++ debian/control	31 Oct 2012 22:00:35 -0000
@@ -4,7 +4,7 @@
 Maintainer: Peter S Galbraith <psg@debian.org>
 Build-Depends: debhelper (>= 7), quilt (>= 0.46-7), texinfo
 Standards-Version: 3.9.0
-Vcs-Cvs: :pserver:anonymous@cvs.alioth.debian.org:/cvsroot/pkg-goodies-el emacs-goodies-el
+Vcs-Cvs: :pserver:anonymous@cvs.alioth.debian.org:/cvs/pkg-goodies-el emacs-goodies-el
 Vcs-Browser: http://alioth.debian.org/scm/viewvc.php/?root=pkg-goodies-el
 
 Package: emacs-goodies-el
cvs diff: Diffing debian/patches
cvs diff: Diffing debian/source
cvs diff: Diffing elisp
cvs diff: Diffing elisp/debian-el
cvs diff: Diffing elisp/devscripts-el
cvs diff: Diffing elisp/dpkg-dev-el
cvs diff: Diffing elisp/emacs-goodies-el
cvs diff: Diffing elisp/gnus-bonus-el
cvs diff: Diffing elisp/vm-bonus-el
