--- /tmp/debsums.orig	2014-12-15 00:29:26.374937420 +0300
+++ /usr/bin/debsums	2014-12-15 00:29:32.316997362 +0300
@@ -462,7 +462,7 @@
 	    return 0;
 	}
 
-	my $correct_package = `dpkg-query "--admindir=$DPKG" -S "$path" | awk -F: '{print \$1}'`;
+	my $correct_package = `dpkg-query "--admindir=$DPKG" -S "$path" | awk -F': ' '{print \$1}'`;
 	chomp($correct_package);
 	if ($pack ne $correct_package) {
 	    #print "$pack != $correct_package\n";
