diff -Nur devhelp-0.19.1.old/src/default-prefs-common.js devhelp-0.19.1/src/default-prefs-common.js
--- devhelp-0.19.1.old/src/default-prefs-common.js	2008-05-25 18:55:37.000000000 +0200
+++ devhelp-0.19.1/src/default-prefs-common.js	2008-07-18 22:23:40.000000000 +0200
@@ -53,8 +53,8 @@
 pref("network.protocol-handler.external-default", true);
 pref("network.protocol-handler.warn-external-default", false);
 pref("network.protocol-handler.external.ftp", true);
-pref("network.protocol-handler.external.http", true);
-pref("network.protocol-handler.external.https", true);
+pref("network.protocol-handler.external.http", false);
+pref("network.protocol-handler.external.https", false);
 pref("network.protocol-handler.external.ftp", false);
 pref("network.protocol-handler.external.news", true);
 pref("network.protocol-handler.external.mailto", true);
