Author: Dmitry Shachnev <mitya57@gmail.com>
Description: use system python-markdown
Forwarded: not-needed
Last-Update: 2013-04-04

Index: calibre-2.45.0+dfsg/src/calibre/library/comments.py
===================================================================
--- calibre-2.45.0+dfsg.orig/src/calibre/library/comments.py
+++ calibre-2.45.0+dfsg/src/calibre/library/comments.py
@@ -150,7 +150,7 @@ def sanitize_html(html):
     return serializer.render(stream)
 
 def sanitize_comments_html(html):
-    from calibre.ebooks.markdown import Markdown
+    from markdown import Markdown
     text = html2text(html)
     md = Markdown()
     html = md.convert(text)
Index: calibre-2.45.0+dfsg/src/calibre/linux.py
===================================================================
--- calibre-2.45.0+dfsg.orig/src/calibre/linux.py
+++ calibre-2.45.0+dfsg/src/calibre/linux.py
@@ -20,7 +20,7 @@ entry_points = {
              'ebook-meta         = calibre.ebooks.metadata.cli:main',
              'ebook-convert      = calibre.ebooks.conversion.cli:main',
              'ebook-polish       = calibre.ebooks.oeb.polish.main:main',
-             'markdown-calibre   = calibre.ebooks.markdown.__main__:run',
+             'markdown-calibre   = markdown.__main__:run',
              'web2disk           = calibre.web.fetch.simple:main',
              'calibre-server     = calibre.library.server.main:main',
              'lrf2lrs            = calibre.ebooks.lrf.lrfparser:main',
