diff -u acpi-1.7/debian/changelog acpi-1.7/debian/changelog
--- acpi-1.7/debian/changelog
+++ acpi-1.7/debian/changelog
@@ -1,3 +1,10 @@
+acpi (1.7-1.2) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Add arm64 architecture. (Closes: #790265) 
+
+ -- Emanuele Rocca <ema@debian.org>  Wed, 25 Jan 2023 15:05:11 +0100
+
 acpi (1.7-1.1) unstable; urgency=medium
 
   * Non-maintainer upload.
diff -u acpi-1.7/debian/control acpi-1.7/debian/control
--- acpi-1.7/debian/control
+++ acpi-1.7/debian/control
@@ -8,7 +8,7 @@
 Homepage: http://sourceforge.net/projects/acpiclient
 
 Package: acpi
-Architecture: i386 ia64 amd64 mips mipsel mipsn32 mipsn32el mips64 mips64el x32
+Architecture: i386 ia64 amd64 mips mipsel mipsn32 mipsn32el mips64 mips64el x32 arm64
 Depends: ${shlibs:Depends}, ${misc:Depends}
 Description: displays information on ACPI devices
  Attempts to replicate the functionality of the 'old' apm command on
