--- /usr/share/doc/debian-policy/policy.sgml	2006-10-11 08:44:02.684306000 +0100
+++ ./policy.sgml	2007-06-26 17:12:56.325983530 +0100
@@ -2105,6 +2105,18 @@
 	  the file to the list in <file>debian/files</file>.</p>
       </sect>
 
+    <sect id="embeddedfiles">
+      <heading>Embedding code provided in other packages</heading>
+      <p>
+      Should the upstream source ship with a convenience copy of an external
+      library, the Debian package should not embed or include this code.
+      Instead, the package should be modified to reference the required
+      files in the library package provided by Debian, and a Depends and/or
+      Build-Depends relationship declared as required.
+      Preferably, the convenience copy should not be compiled in the
+      build-process. 
+      </p>
+      </sect>
     </chapt>
 
 
