diff --git a/apt-pkg/deb/dpkgpm.cc b/apt-pkg/deb/dpkgpm.cc
index 959d064..bc86258 100644
--- a/apt-pkg/deb/dpkgpm.cc
+++ b/apt-pkg/deb/dpkgpm.cc
@@ -1239,9 +1239,9 @@ bool pkgDPkgPM::Go(int OutStatusFd)
       // if tcgetattr does not return zero there was a error
       // and we do not do any pty magic
       _error->PushToStack();
-      if (tcgetattr(STDOUT_FILENO, &tt) == 0)
+      if (tcgetattr(STDIN_FILENO, &tt) == 0)
       {
-	 ioctl(0, TIOCGWINSZ, (char *)&win);
+	 ioctl(STDIN_FILENO, TIOCGWINSZ, (char *)&win);
 	 if (openpty(&master, &slave, NULL, &tt, &win) < 0)
 	 {
 	    _error->Errno("openpty", _("Can not write log (%s)"), _("Is /dev/pts mounted?"));
