--- /usr/share/enscript/hl/changelog.st.original	2005-02-13 21:00:59.000000000 +0100
+++ /usr/share/enscript/hl/changelog.st	2005-11-25 02:18:59.000000000 +0100
@@ -26,7 +26,7 @@ state changelog extends HighlightEntry
   }
 
   /* File descriptions with function names. */
-  /(^\t\* )([^\(]+)(\()([^\)]+)(\):)/ {
+  /(^\t\* )([^\(]+)(\()([^\)]+)(\)[:\n])/ {
     language_print ($1);
 
     function_name_face (true);
@@ -54,7 +54,7 @@ state changelog extends HighlightEntry
   }
 
   /* Function name descriptions without file names. */
-  /(^\t\()([^\)]+)(\):)/ {
+  /(^\t\()([^\)]+)(\)[:\n])/ {
     language_print ($1);
 
     keyword_face (true);
