diff -Nru --exclude changelog /tmp/D2p43dtdH7/debram-1.0.3/src/utf8.h /tmp/s0K1SU5RtL/debram-1.0.3/src/utf8.h
--- a/src/utf8.h	2005-09-03 23:47:06.000000000 +0530
+++ b/src/utf8.h	2007-12-11 18:01:20.000000000 +0530
@@ -6,6 +6,7 @@
 #include "argp.h"
 #include "color.h"
 #include <ctype.h>
+#include <stddef.h>
 
 /* This buffer size permits a full line of maximum-width utf-8
  * characters, plus a newline, a terminating null, and as many as two
