--- ../apt-0.7.21/apt-pkg/deb/dpkgpm.cc	2009-04-14 13:20:29.000000000 +0100
+++ apt-pkg/deb/dpkgpm.cc	2009-04-21 21:31:35.000000000 +0100
@@ -524,7 +524,7 @@
    if (!logfile_name.empty())
    {
       term_out = fopen(logfile_name.c_str(),"a");
-      chmod(logfile_name.c_str(), 0600);
+      chmod(logfile_name.c_str(), 0644);
       // output current time
       char outstr[200];
       time_t t = time(NULL);
