--- /usr/bin/apt-show-versions	2009-06-19 16:48:49.000000000 +0100
+++ bin/apt-show-versions.clean	2011-04-18 16:54:55.000000000 +0100
@@ -563,7 +563,8 @@
     return $pkgfile if ($releasenames{$pkgfile});
     $pkgfile =~ s{.*/}{};
     return undef if ($pkgfile eq 'status');
-    foreach (keys %releasenames) {
+    # Sort according to length so we make the most specfic match
+    foreach (sort { length($b) <=> length($a) } keys %releasenames) {
         return $_ if ($_ eq substr($pkgfile, 0, length($_)));
     }
     # As package file has no release file, create a fallback %releasenames
