--- /usr/share/apt-cacher/apt-cacher.orig	2008-05-27 22:28:41.000000000 +0100
+++ /usr/share/apt-cacher/apt-cacher	2008-08-14 07:26:43.000000000 +0100
@@ -1250,9 +1250,11 @@
 	    unshift(@{$pathmap{$vhost}}, $hostcand);
 	}
 	else {
-	    # truncate cached_file to remove previous HTTP error
-	    truncate($$pkfdref, 0);
-	    sysseek($$pkfdref, 0, 0);
+	    if($pkfdref) {
+		# truncate cached_file to remove previous HTTP error
+		truncate($$pkfdref, 0);
+		sysseek($$pkfdref, 0, 0);
+	    }
 	    goto RETRY_ACTION;
 	}
     }
