--- clamav-0.98+dfsg.orig/libclamav/clamav.h
+++ clamav-0.98+dfsg/libclamav/clamav.h
@@ -22,7 +22,7 @@
 #define __CLAMAV_H
 
 /* Certain OSs already use 64bit variables in their stat struct */
-#define STAT64_BLACKLIST !defined(__FreeBSD__) && !defined(__APPLE__)
+#define STAT64_BLACKLIST !defined(__FreeBSD__) && !defined(__APPLE__) && !defined(__linux__) && !defined(__FreeBSD_kernel__) && !defined(__hurd__)
 
 #if defined(HAVE_STAT64) && STAT64_BLACKLIST
 
