--- i.orig/bin/list-devices	2007-11-06 20:55:13.000000000 +1100
+++ i/bin/list-devices	2007-11-06 20:57:27.000000000 +1100
@@ -50,7 +50,7 @@
 	esac
 	# Some USB sticks and CD drives are misdetected as floppy
 	# This allows to scan for those
-	if ! $match && [ "$TYPE" = maybe-usb-floppy ]; then
+	if ! $match && [ "$TYPE" = maybe-usb-floppy -o "$TYPE" = disk ]; then
 		if udevinfo -q env -p "$devpath" 2>/dev/null | \
 		   grep -q '^ID_BUS=usb' && \
 		   udevinfo -q env -p "$devpath" 2>/dev/null | \
