--- src/options.c.old	2003-07-21 18:03:46.000000000 +0200
+++ src/options.c	2005-01-16 03:12:21.000000000 +0100
@@ -10,7 +10,7 @@ FILE *OpenOptionsFile( char *mode )
     FILE *file;
     
     if ((char *)getenv("HOME") != NULL)
-	sprintf( filename, "%s/.bumprace", getenv("HOME"));
+	snprintf( filename, sizeof(filename), "%s/.bumprace", getenv("HOME"));
     else
 	sprintf( filename, "bumprace.dat");
 
@@ -52,7 +52,7 @@ void ReadOptions()
 
     fscanf(file, "%*s%*s%*s%*s\n");
     for (i=0;i<10;i++)
-	fscanf(file, "%[^\n]\n",&Player[i]);
+	fscanf(file, "%199[^\n]\n",&Player[i]);
     for (i=0;i<10;i++)
 	fscanf(file, "%d\n",&Hiscore[i]);
 //    for (i=0;i<10;i++)
