--- alsa-utils.orig	2007-02-22 00:26:34.000000000 +0100
+++ alsa-utils	2007-02-22 00:25:03.000000000 +0100
@@ -303,6 +303,9 @@
   start)
 	EXITSTATUS=0
 	TARGET_CARD="$2"
+	if [ ! -e /dev/snd ] ; then
+		/usr/share/alsa-base/snddevices
+	fi
 	case "$TARGET_CARD" in
 	  ""|all) TARGET_CARD=all ; log_action_begin_msg "Setting up ALSA" ;;
 	  *) log_action_begin_msg "Setting up ALSA card ${TARGET_CARD}" ;;
