--- dpkg-divert.pl-orig	2004-11-11 04:16:35.000000000 +0100
+++ dpkg-divert.pl	2005-06-13 00:26:01.000000000 +0200
@@ -219,16 +219,22 @@
     # same name as the diversions but with an extension that
     # (hopefully) wont overwrite anything. If it succeeds, we
     # assume a writable filesystem.
-    foreach $file ($rsrc,$rdest) {
-	if (open (TMP, ">> ${file}.dpkg-devert.tmp")) {
-		close TMP;
-		unlink ("${file}.dpkg-devert.tmp");
-	} elsif ($! == ENOENT) {
-		$dorename = !$dorename;
-	} else {
-		&quit("error checking \`$file': $!");
-	}
+    if (open (TMP, ">> ${rsrc}.dpkg-devert.tmp")) {
+        close TMP;
+        unlink ("${rsrc}.dpkg-devert.tmp");
+    } elsif ($! == ENOENT) {
+        $dorename = !$dorename;
+    } else {
+        &quit("error checking \`$rsrc': $!");
     }
+
+    if (open (TMP, ">> ${rdest}.dpkg-devert.tmp")) {
+        close TMP;
+        unlink ("${rdest}.dpkg-devert.tmp");
+    } else {
+        &quit("error checking \`$rdest': $!");
+    }
+
     if (@ssrc && @sdest &&
         !($ssrc[0] == $sdest[0] && $ssrc[1] == $sdest[1])) {
         &quit("rename involves overwriting \`$rdest' with\n".
