diff -aruN aboot-1.0~pre20040408.old/zip/misc.c aboot-1.0~pre20040408/zip/misc.c
--- aboot-1.0~pre20040408.old/zip/misc.c	2001-10-09 01:03:54.000000000 +0200
+++ aboot-1.0~pre20040408/zip/misc.c	2008-05-29 14:22:36.000000000 +0200
@@ -11,8 +11,6 @@
  */
 #include <linux/kernel.h>
 
-#include <asm/page.h>
-
 #include "aboot.h"
 #include "bootfs.h"
 #include "setjmp.h"
--- aboot-1.0~pre20040408.orig/disk.c
+++ aboot-1.0~pre20040408/disk.c
@@ -35,8 +35,8 @@
 
 #include <linux/elf.h>
 #include <asm/console.h>
-#include <asm/system.h>
-#include <asm/elf.h>
+#include "system.h"
+//#include <asm/elf.h>
 
 extern struct bootfs ext2fs;
 extern struct bootfs iso;
