From f680a1705a529d65f374667b44251fcf12b9abbb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Bastien=20ROUCARI=C3=88S?= <roucaries.bastien@gmail.com>
Date: Tue, 14 Mar 2017 21:57:38 +0100
Subject: [PATCH 3/7] Symplify a condition by using a function

---
 scripts/dpkg-maintscript-helper.sh | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/scripts/dpkg-maintscript-helper.sh b/scripts/dpkg-maintscript-helper.sh
index 4cf7f04eb..b4b3ac1b3 100755
--- a/scripts/dpkg-maintscript-helper.sh
+++ b/scripts/dpkg-maintscript-helper.sh
@@ -383,9 +383,7 @@ dir_to_symlink() {
 		   [ -d "${PATHNAME}.dpkg-backup" ] &&
 		   [ \( ! -h "$PATHNAME" -a -d "$PATHNAME" -a \
 		        -f "$PATHNAME/.dpkg-staging-dir" \) -o \
-		     \( -h "$PATHNAME" -a \
-		        \( "$(readlink "$PATHNAME")" = "$SYMLINK_TARGET" -o \
-		           "$(readlink -f "$PATHNAME")" = "$SYMLINK_TARGET" \) \) ] &&
+		     \( -h "$PATHNAME" -a symlink_match "$PATHNAME" "$SYMLINK_TARGET" \)] &&
 		   dpkg --compare-versions -- "$2" le-nl "$LASTVERSION"; then
 			abort_dir_to_symlink "$PATHNAME"
 		fi
-- 
2.11.0

