--- evolution-2.8.1.orig/widgets/table/e-tree.c
+++ evolution-2.8.1/widgets/table/e-tree.c
@@ -1850,10 +1850,12 @@
 	}
 
 	if (tree->priv) {
-		gtk_layout_set_hadjustment (GTK_LAYOUT(tree->priv->table_canvas),
-					    hadjustment);
-		gtk_layout_set_vadjustment (GTK_LAYOUT(tree->priv->table_canvas),
-					    vadjustment);
+		if (tree->priv->table_canvas) {
+			gtk_layout_set_hadjustment (GTK_LAYOUT(tree->priv->table_canvas),
+						    hadjustment);
+			gtk_layout_set_vadjustment (GTK_LAYOUT(tree->priv->table_canvas),
+						    vadjustment);
+		}
 
 		if (tree->priv->header_canvas != NULL)
 			gtk_layout_set_hadjustment (GTK_LAYOUT(tree->priv->header_canvas),
