diff -Nru dokuwiki-0.0.20120125/debian/rules dokuwiki-0.0.20120125/debian/rules
--- dokuwiki-0.0.20120125/debian/rules	2012-01-26 22:03:02.000000000 +0100
+++ dokuwiki-0.0.20120125/debian/rules	2012-03-21 13:54:22.000000000 +0100
@@ -21,6 +21,8 @@
 	# Remove Flash source (useless) and bytecode (violates Policy)
 	rm -rf $(DESTDIR)/usr/share/dokuwiki/lib/_fla
 	rm -rf $(DESTDIR)/usr/share/dokuwiki/lib/exe/multipleUpload.swf
+	# Create plugins.local.php 
+	touch $(DESTDIR)/etc/dokuwiki/plugins.local.php
 
 override_dh_fixperms:
 	dh_fixperms
