diff -u apf-firewall-9.7+rev1/debian/changelog apf-firewall-9.7+rev1/debian/changelog
--- apf-firewall-9.7+rev1/debian/changelog
+++ apf-firewall-9.7+rev1/debian/changelog
@@ -1,3 +1,11 @@
+apf-firewall (9.7+rev1-5.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Depend on iproute2 instead of transitional package iproute.
+    (Closes: #862847)
+
+ -- Luca Boccassi <bluca@debian.org>  Fri, 05 Jan 2018 22:54:06 +0000
+
 apf-firewall (9.7+rev1-5) unstable; urgency=medium
 
   * QA upload
diff -u apf-firewall-9.7+rev1/debian/control apf-firewall-9.7+rev1/debian/control
--- apf-firewall-9.7+rev1/debian/control
+++ apf-firewall-9.7+rev1/debian/control
@@ -8,7 +8,7 @@
 
 Package: apf-firewall
 Architecture: all
-Depends: ${misc:Depends}, iptables, lsb-base, wget, iproute
+Depends: ${misc:Depends}, iptables, lsb-base, wget, iproute2
 Description: easy iptables based firewall system
  Advanced Policy Firewall (APF) is an iptables(netfilter) based firewall
  system designed around the essential needs of today's Internet deployed
