diff -uNrp debmirror-2.4.4.original/debmirror debmirror-2.4.4/debmirror
--- debmirror-2.4.4.original/debmirror	2010-03-25 19:13:29.000000000 +0100
+++ debmirror-2.4.4/debmirror	2010-07-28 12:00:51.064616005 +0200
@@ -2418,6 +2418,9 @@ sub di_get_files {
 	link("$tdir/$image_dir/$file", "$image_dir/$file");
       }
       # Move the MD5SUMS file in place on mirror
+      if (-f "$image_dir/MD5SUMS") {
+        unlink "$image_dir/MD5SUMS";
+      }
       link("$tdir/$image_dir/MD5SUMS", "$image_dir/MD5SUMS");
     } elsif (! $do_dry_run) {
       say("Failed to download some files in $image_dir; not updating images.");
