--- emacs-goodies-el-29.2.orig/elisp/dpkg-dev-el/debian-changelog-mode.el
+++ emacs-goodies-el-29.2/elisp/dpkg-dev-el/debian-changelog-mode.el
@@ -421,7 +421,7 @@
   ;; XEmacs:
   (if (fboundp 'replace-in-string)
       (save-match-data ;; apparently XEmacs needs save-match-data
-        (apply 'replace-in-string args))
+        (apply 'replace-in-string (list (nth 2 args) (nth 0 args) (nth 1 args))))
     ;; Emacs:
     (apply 'replace-regexp-in-string args)))
 
