diff -Nru /tmp/UZPeXrAbyS/debhelper-4.9.8/autoscripts/postinst-gconf /tmp/2flMFRmv6Y/debhelper-4.9.8.1/autoscripts/postinst-gconf
--- /tmp/UZPeXrAbyS/debhelper-4.9.8/autoscripts/postinst-gconf	2005-05-13 22:46:36.000000000 +0200
+++ /tmp/2flMFRmv6Y/debhelper-4.9.8.1/autoscripts/postinst-gconf	2005-09-21 23:37:54.000000000 +0200
@@ -1,13 +1,3 @@
 if [ "$1" = "configure" ]; then
-	SCHEMA_LOCATION=/usr/share/gconf/schemas
-	SCHEMA_FILES="#SCHEMAS#"
-	for SCHEMA in $SCHEMA_FILES; do
-		if [ -e $SCHEMA_LOCATION/$SCHEMA ]; then
-			HOME=/root GCONF_CONFIG_SOURCE=`gconftool-2 --get-default-source` \
-				gconftool-2 \
-				--makefile-install-rule $SCHEMA_LOCATION/$SCHEMA > /dev/null
-		fi
-	done
-
-	kill -s HUP `pidof gconfd-2` >/dev/null 2>&1 || true
+	/usr/sbin/gconf-schemas --register #SCHEMAS#
 fi
diff -Nru /tmp/UZPeXrAbyS/debhelper-4.9.8/autoscripts/prerm-gconf /tmp/2flMFRmv6Y/debhelper-4.9.8.1/autoscripts/prerm-gconf
--- /tmp/UZPeXrAbyS/debhelper-4.9.8/autoscripts/prerm-gconf	2005-05-13 22:46:36.000000000 +0200
+++ /tmp/2flMFRmv6Y/debhelper-4.9.8.1/autoscripts/prerm-gconf	2005-09-21 23:38:55.000000000 +0200
@@ -1,11 +1,3 @@
 if [ "$1" = remove ] || [ "$1" = upgrade ]; then
-	SCHEMA_LOCATION=/usr/share/gconf/schemas
-	SCHEMA_FILES="#SCHEMAS#"
-	for SCHEMA in $SCHEMA_FILES; do
-		if [ -e $SCHEMA_LOCATION/$SCHEMA ]; then
-			HOME=/root GCONF_CONFIG_SOURCE=`gconftool-2 --get-default-source` \
-				gconftool-2 \
-					--makefile-uninstall-rule $SCHEMA_LOCATION/$SCHEMA > /dev/null
-		fi
-	done
+	/usr/sbin/gconf-schemas --unregister #SCHEMAS#
 fi
diff -Nru /tmp/UZPeXrAbyS/debhelper-4.9.8/debian/changelog /tmp/2flMFRmv6Y/debhelper-4.9.8.1/debian/changelog
--- /tmp/UZPeXrAbyS/debhelper-4.9.8/debian/changelog	2005-08-27 04:13:20.000000000 +0200
+++ /tmp/2flMFRmv6Y/debhelper-4.9.8.1/debian/changelog	2005-09-21 23:42:58.000000000 +0200
@@ -1,3 +1,12 @@
+debhelper (4.9.8.1) unstable; urgency=low
+
+  * NMU with maintainer approval.
+  * dh_gconf: delegate schema registration the gconf-schemas script, 
+    which moves schemas to /var/lib/gconf, and require gconf2 2.10.1-2,
+    where it can be found. Closes: #327209
+
+ -- Josselin Mouette <joss@debian.org>  Wed, 21 Sep 2005 23:39:01 +0200
+
 debhelper (4.9.8) unstable; urgency=low
 
   * Spelling patch from Kumar Appaiah. Closes: #324892
diff -Nru /tmp/UZPeXrAbyS/debhelper-4.9.8/dh_gconf /tmp/2flMFRmv6Y/debhelper-4.9.8.1/dh_gconf
--- /tmp/UZPeXrAbyS/debhelper-4.9.8/dh_gconf	2005-05-13 22:46:41.000000000 +0200
+++ /tmp/2flMFRmv6Y/debhelper-4.9.8.1/dh_gconf	2005-09-21 23:37:24.000000000 +0200
@@ -50,7 +50,7 @@
 			autoscript($package,"postinst","postinst-gconf","s%#SCHEMAS#%$schemas%");
 			autoscript($package,"prerm","prerm-gconf","s%#SCHEMAS#%$schemas%");
 			autoscript($package,"postrm","postrm-gconf","s%#SCHEMAS#%$schemas%");
-			addsubstvar($package, "misc:Depends", "gconf2 (>= 2.6.2-1)");
+			addsubstvar($package, "misc:Depends", "gconf2 (>= 2.10.1-2)");
 		}
 	}
 }
