diff -Nru cuneiform-1.1.0+dfsg/debian/compat cuneiform-1.1.0+dfsg/debian/compat
--- cuneiform-1.1.0+dfsg/debian/compat	2011-11-09 19:19:32.000000000 +0100
+++ cuneiform-1.1.0+dfsg/debian/compat	2012-04-30 21:38:47.000000000 +0200
@@ -1 +1 @@
-8
+9
diff -Nru cuneiform-1.1.0+dfsg/debian/control cuneiform-1.1.0+dfsg/debian/control
--- cuneiform-1.1.0+dfsg/debian/control	2012-04-24 00:12:22.000000000 +0200
+++ cuneiform-1.1.0+dfsg/debian/control	2012-04-30 21:39:13.000000000 +0200
@@ -2,7 +2,7 @@
 Section: non-free/graphics
 Priority: optional
 Maintainer: Jakub Wilk <jwilk@debian.org>
-Build-Depends: debhelper (>= 8.1.3), dpkg-dev (>= 1.16), cmake (>= 2.8.2),
+Build-Depends: debhelper (>= 9.20120417), dpkg-dev (>= 1.16), cmake (>= 2.8.2),
   libgraphicsmagick++1-dev, pkg-config
 Standards-Version: 3.9.3
 Homepage: http://launchpad.net/cuneiform-linux/
diff -Nru cuneiform-1.1.0+dfsg/debian/rules cuneiform-1.1.0+dfsg/debian/rules
--- cuneiform-1.1.0+dfsg/debian/rules	2012-04-23 23:54:34.000000000 +0200
+++ cuneiform-1.1.0+dfsg/debian/rules	2012-04-30 21:45:24.000000000 +0200
@@ -1,22 +1,26 @@
 #!/usr/bin/make -f
 
+export DEB_BUILD_MAINT_OPTIONS=hardening=+all,-pie
+export DEB_CFLAGS_MAINT_APPEND=-O0
+export DEB_CXXFLAGS_MAINT_APPEND=-O0
+export DEB_LDFLAGS_MAINT_APPEND=-Wl,--as-needed
+
 here = $(dir $(firstword $(MAKEFILE_LIST)))/..
 upstream_version = $(shell cd $(here) && dpkg-parsechangelog | sed -n -r -e '/^Version: ([0-9.]+)([+]dfsg).*/ { s//\1/; p; q; }')
 
 DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
 
-CFLAGS = $(shell dpkg-buildflags --get CFLAGS) $(shell dpkg-buildflags --get CPPFLAGS)
-CXXFLAGS = $(shell dpkg-buildflags --get CXXFLAGS) $(shell dpkg-buildflags --get CPPFLAGS)
-LDFLAGS = $(shell dpkg-buildflags --get LDFLAGS) -Wl,--as-needed
+DPKG_EXPORT_BUILDFLAGS = 1
+include /usr/share/dpkg/buildflags.mk
 
 cmake_options = \
 	-DCMAKE_INSTALL_PREFIX=/usr \
 	-DCMAKE_VERBOSE_MAKEFILE=ON \
 	-DCMAKE_BUILD_TYPE=relwithdebinfo \
-	-DCMAKE_C_FLAGS_RELWITHDEBINFO="$(CFLAGS)" \
-	-DCMAKE_CXX_FLAGS_RELWITHDEBINFO="$(CXXFLAGS)" \
-	-DCMAKE_SHARED_LINKER_FLAGS_RELWITHDEBINFO="$(LDFLAGS)" \
-	-DCMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO="$(LDFLAGS)" \
+	-DCMAKE_C_FLAGS="$(CFLAGS)" \
+	-DCMAKE_CXX_FLAGS="$(CXXFLAGS)" \
+	-DCMAKE_SHARED_LINKER_FLAGS="$(LDFLAGS)" \
+	-DCMAKE_EXE_LINKER_FLAGS="$(LDFLAGS)" \
 	-DCMAKE_INSTALL_RPATH=/usr/lib/$(DEB_HOST_MULTIARCH)/cuneiform
 
 parallel = -j$(or $(patsubst parallel=%,%,$(filter parallel=%,$(DEB_BUILD_OPTIONS))),1)
