diff -up -ru dictionaries-common-0.80.1.orig/debian/dictionaries-common.preinst dictionaries-common-0.80.1/debian/dictionaries-common.preinst
--- dictionaries-common-0.80.1.orig/debian/dictionaries-common.preinst	2005-02-14 16:54:58.000000000 +0100
+++ dictionaries-common-0.80.1/debian/dictionaries-common.preinst	2007-05-10 10:14:21.000000000 +0200
@@ -13,14 +13,14 @@ case "$1" in install|upgrade)
 	for i in $DICTS ; do
 	    /usr/sbin/update-alternatives --remove ispell-dictionary.hash $i
 	done
-	/usr/sbin/update-alternatives --auto ispell-dictionary.hash
+	/usr/sbin/update-alternatives --auto ispell-dictionary.hash || true
     # The same for wordlists
 	WORDS=`/usr/sbin/update-alternatives --display dictionary \
       | grep priority | cut -f1 -d" "`
 	for i in $WORDS ; do
 	    /usr/sbin/update-alternatives --remove dictionary $i
 	done
-	/usr/sbin/update-alternatives --auto dictionary
+	/usr/sbin/update-alternatives --auto dictionary || true
 	;;
 esac
 
