From ef142e9800abceda3382146e0dfc018c41c6813c Mon Sep 17 00:00:00 2001
From: Andreas Beckmann <anbe@debian.org>
Date: Wed, 10 May 2023 17:27:31 +0200
Subject: [PATCH] Add Breaks against known incompatible package elpa-cider

Closes: #1035781
---
 debian/control | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/control b/debian/control
index d4df3906221..a9a1b3d45e3 100644
--- a/debian/control
+++ b/debian/control
@@ -142,6 +142,7 @@ Breaks:
  apel (<< 10.8+0.20120427-4),
  edb (<< 1.32),
  egg (<< 4.2.0-2),
+ elpa-cider (<< 0.19.0+dfsg-4~),
  emacs (<< 1:25),
  emacs-gtk (<< 1:25),
  emacs-lucid (<< 1:25),
-- 
2.20.1

