diff -u cronolog-1.6.2/debian/rules cronolog-1.6.2/debian/rules
--- cronolog-1.6.2/debian/rules
+++ cronolog-1.6.2/debian/rules
@@ -38,6 +38,8 @@
 $(STAMP_DIR)/configure: setup
 	dh_testdir
 
+	cp -f /usr/share/misc/config.sub $(BUILD_TREE)/config.sub
+	cp -f /usr/share/misc/config.guess $(BUILD_TREE)/config.guess
 	cd $(BUILD_TREE) && CC="$(CC)" CFLAGS="$(CFLAGS)" ./configure --prefix=/usr
 
 build-depends := $(STAMP_DIR)/build
diff -u cronolog-1.6.2/debian/changelog cronolog-1.6.2/debian/changelog
--- cronolog-1.6.2/debian/changelog
+++ cronolog-1.6.2/debian/changelog
@@ -1,3 +1,10 @@
+cronolog (1.6.2-5.3) unstable; urgency=low
+
+  * Porter NMU.
+  * Update config.guess/config.sub at build-time (Closes: #331428).
+
+ -- Aurelien Jarno <aurel32@debian.org>  Sun, 10 Dec 2006 23:45:56 +0100
+
 cronolog (1.6.2-5.2) unstable; urgency=low
 
   * Non-maintainer upload.
diff -u cronolog-1.6.2/debian/control cronolog-1.6.2/debian/control
--- cronolog-1.6.2/debian/control
+++ cronolog-1.6.2/debian/control
@@ -3,7 +3,7 @@
 Priority: optional
 Maintainer: Joel Baker <fenton@debian.org>
 Standards-Version: 3.6.1
-Build-Depends: dbs, debhelper (>= 4), dejagnu
+Build-Depends: dbs, debhelper (>= 4), dejagnu, autotools-dev
 
 Package: cronolog
 Architecture: any
