From 2290e110fb67b7e0f56aeab5a4cf12dc57b7cfe8 Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs3@bonedaddy.net>
Date: Sat, 1 Nov 2014 11:50:58 +0800
Subject: [PATCH 1/7] Bug squashing party (Mozilla, Paris, France)

---
 data/events/bsps | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/data/events/bsps b/data/events/bsps
index aee31fe..fe2c086 100644
--- a/data/events/bsps
+++ b/data/events/bsps
@@ -168,3 +168,8 @@ Title: Bug squashing party (LiMux, Munich, Germany)
 Start-Date: Nov 21 2014
 End-Date: Nov 23 2014
 Source: https://wiki.debian.org/BSP/2014/11/de/Munich
+
+Title: Bug squashing party (Mozilla, Paris, France)
+Start-Date: Nov 14 2014
+End-Date: Nov 16 2014
+Source: https://wiki.debian.org/BSP/2014/11/fr/Paris
-- 
2.1.1

