From 4984bcd7290ff74ca6f278341d85c48561c26173 Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs3@bonedaddy.net>
Date: Sat, 1 Nov 2014 11:53:33 +0800
Subject: [PATCH 2/7] Add some more miniconfs

---
 data/events/conferences | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/data/events/conferences b/data/events/conferences
index b0bf792..f54d4bd 100644
--- a/data/events/conferences
+++ b/data/events/conferences
@@ -252,7 +252,22 @@ Start-Date: Aug 23 2014
 End-Date: Aug 31 2014
 Source: http://debconf14.debconf.org/
 
+Title: Debutsav, Kollam, India
+Start-Date: Oct 17 2014
+End-Date: Jan 18 2014
+Source: https://wiki.debian.org/DebianIndia/Debutsavam2014
+
 Title: Mini-DebConf, Auckland, New Zealand
 Start-Date: Jan 10 2015
 End-Date: Jan 11 2015
 Source: https://wiki.debconf.org/wiki/Miniconf-AU/2015
+
+Title: Mini-DebConf, Mumbai, India
+Start-Date: Jan 31 2015
+End-Date: Feb 1 2015
+Source: https://wiki.debian.org/DebianIndia/MiniDebConf2015/Mumbai
+
+Title: Mini-DebConf, Yuli Township, Taiwan
+Start-Date: Feb 6 2015
+End-Date: Feb 8 2015
+Source: https://wiki.debian.org/DebianTaiwan/MiniDebConf2015
-- 
2.1.1

