From 8a18566c6c63a81eb9c21169a33d8b33b66954eb Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs3@bonedaddy.net>
Date: Sat, 1 Nov 2014 11:53:42 +0800
Subject: [PATCH 3/7] More debian-installer news

---
 data/events/d-i | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/data/events/d-i b/data/events/d-i
index 0432684..68bf682 100644
--- a/data/events/d-i
+++ b/data/events/d-i
@@ -118,7 +118,7 @@ Title: First alpha of jessie's debian-installer released
 Date: Mar 19 2014
 Source: https://www.debian.org/devel/debian-installer/News/2014/20140319
 
-Title: First beta of jessies's debian-installer released
+Title: First beta of jessie's debian-installer released
 Date: Aug 13 2014
 Source: https://www.debian.org/devel/debian-installer/News/2014/20140813
 
-- 
2.1.1

