From 291660290b75b4eb9fed7868f2692712b6aad93d Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs3@bonedaddy.net>
Date: Sat, 1 Nov 2014 11:53:56 +0800
Subject: [PATCH 4/7] Peter Miller passed away

---
 data/events/tragic | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/data/events/tragic b/data/events/tragic
index 08966e9..362e28d 100644
--- a/data/events/tragic
+++ b/data/events/tragic
@@ -39,3 +39,7 @@ Source: https://www.debian.org/News/2011/20110423
 Title: Ray Dassen passed away
 Date: May 23 2013
 Source: https://www.debian.org/News/2013/20130523
+
+Title: Peter Miller passed away
+Date: Oct 20 2014
+Source: https://www.debian.org/News/2014/20141020
-- 
2.1.1

