From f0f1f038350696fad4551c9d46959688fe0c2a78 Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs3@bonedaddy.net>
Date: Sat, 1 Nov 2014 11:54:02 +0800
Subject: [PATCH 5/7] init system coupling General Resolution

---
 data/events/voting | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/data/events/voting b/data/events/voting
index 30a0d79..159ec7f 100644
--- a/data/events/voting
+++ b/data/events/voting
@@ -217,3 +217,7 @@ Source: https://www.debian.org/vote/2014/vote_001
 Title: Code of conduct General Resolution
 Date: Apr 28 2014
 Source: https://www.debian.org/vote/2014/vote_002
+
+Title: init system coupling General Resolution
+Date: Oct 16 2014
+Source: https://www.debian.org/vote/2014/vote_003
-- 
2.1.1

