From 65c21bea36673e04a1d4a7ecc1142e2c168c0777 Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs3@bonedaddy.net>
Date: Sat, 1 Nov 2014 11:54:11 +0800
Subject: [PATCH 6/7] Debian 7 ("wheezy") updated (r7)

---
 data/releases/point_releases | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/data/releases/point_releases b/data/releases/point_releases
index 27aa225..cc3c3f0 100644
--- a/data/releases/point_releases
+++ b/data/releases/point_releases
@@ -245,3 +245,7 @@ Source: https://lists.debian.org/debian-announce/2014/msg00005.html
 Title: Debian 6.0 ("squeeze") updated (r10)
 Date: Jul 19 2014
 Source: https://lists.debian.org/debian-announce/2014/msg00006.html
+
+Title: Debian 7 ("wheezy") updated (r7)
+Date: Oct 18 2014
+Source: https://lists.debian.org/debian-announce/2014/msg00008.html
-- 
2.1.1

