diff -u awesome-3.3.1/debian/changelog awesome-3.3.1/debian/changelog
--- awesome-3.3.1/debian/changelog
+++ awesome-3.3.1/debian/changelog
@@ -1,3 +1,10 @@
+awesome (3.3.1-1.1) unstable; urgency=low
+
+  * NMU to add dbus_connection_set_exit_on_disconnect to stop the bus
+    disconnecting and causing awesome to die. (Closes: #527872)
+
+ -- Daniel Silverstone <dsilvers@debian.org>  Fri, 24 Jul 2009 15:42:14 +0200
+
 awesome (3.3.1-1) unstable; urgency=low
 
   * New upstream release
only in patch2:
unchanged:
--- awesome-3.3.1.orig/dbus.c
+++ awesome-3.3.1/dbus.c
@@ -502,7 +502,9 @@
             a_dbus_cleanup_bus(dbus_connection, dbusio);
         }
     }
-
+    
+    dbus_connection_set_exit_on_disconnect(dbus_connection, FALSE);
+    
     return dbus_connection;
 }
 
