--- a/dh_strip
+++ b/dh_strip
@@ -167,6 +167,7 @@

 my $objcopy = cross_command("objcopy");
 my $strip = cross_command("strip");
+my $readelf = cross_command("readelf");
 $dh{ENABLE_DDEBS} = 0 if get_buildoption('noddebs');

 # I could just use `file $_[0]`, but this is safer
@@ -243,7 +244,7 @@
 	my ($debug_path, $debug_build_id);

 	if ($use_build_id &&
-	    `LC_ALL=C readelf -n $file`=~ /^\s+Build ID: ([0-9a-f]{2})([0-9a-f]+)$/m) {
+	    `LC_ALL=C $readelf -n $file`=~ /^\s+Build ID: ([0-9a-f]{2})([0-9a-f]+)$/m) {
 		$debug_path=$desttmp."/usr/lib/debug/.build-id/$1/$2.debug";
 		$debug_build_id="${1}${2}";
	}
