diff -Nurd base-files-7.10/debian/postinst.in base-files-7.10.new/debian/postinst.in
--- base-files-7.10/debian/postinst.in	2014-10-27 13:36:30.000000000 +0100
+++ base-files-7.10.new/debian/postinst.in	2014-11-05 05:35:03.801097773 +0100
@@ -1,12 +1,25 @@
 #!/bin/sh
 set -e
 
+# During debootstrap /etc/passwd and /etc/group may not exist yet.
+if [ -f /etc/group ]
+  then
+    STAFF=staff
+    MAIL=mail
+    UTMP=utmp
+  else
+    STAFF=50
+    MAIL=8
+    UTMP=43
+fi
+ROOT=0
+
 install_local_dir() {
   if [ ! -d $1 ]; then
     mkdir -p $1
   fi
   if [ -f /etc/staff-group-for-usr-local ]; then
-    chown root:staff $1 2> /dev/null || true
+    chown $ROOT:$STAFF $1 2> /dev/null || true
     chmod 2775 $1 2> /dev/null || true
   fi
 }
@@ -20,7 +33,7 @@
 install_directory() {
   if [ ! -d /$1 ]; then
     mkdir /$1
-    chown root:$3 /$1
+    chown $ROOT:$3 /$1
     chmod $2 /$1
   fi
 }
@@ -58,17 +71,17 @@
   install_from_default /usr/share/base-files/dot.bashrc    /root/.bashrc
   install_from_default /usr/share/base-files/profile       /etc/profile
   install_from_default /usr/share/base-files/motd          /etc/motd
-  install_directory mnt       755 root
-  install_directory srv       755 root
-  install_directory opt       755 root
-  install_directory etc/opt   755 root
-  install_directory var/opt   755 root
-  install_directory media     755 root
-  install_directory var/mail 2775 mail
+  install_directory mnt       755 $ROOT
+  install_directory srv       755 $ROOT
+  install_directory opt       755 $ROOT
+  install_directory etc/opt   755 $ROOT
+  install_directory var/opt   755 $ROOT
+  install_directory media     755 $ROOT
+  install_directory var/mail 2775 $MAIL
   if [ ! -L /var/spool/mail ]; then
     ln -s ../mail /var/spool/mail
   fi
-  install_directory run/lock 1777 root
+  install_directory run/lock 1777 $ROOT
   migrate_directory /var/run /run
   migrate_directory /var/lock /run/lock
 
@@ -93,13 +106,13 @@
   if [ ! -f /var/log/lastlog ]; then
     echo -n>/var/log/lastlog
   fi
-  chown root:utmp /var/log/wtmp /var/log/btmp /var/log/lastlog
+  chown $ROOT:$UTMP /var/log/wtmp /var/log/btmp /var/log/lastlog
   chmod 664 /var/log/wtmp /var/log/lastlog
   chmod 660 /var/log/btmp
   if [ ! -f /var/run/utmp ]; then
     echo -n>/var/run/utmp
   fi
-  chown root:utmp /var/run/utmp
+  chown $ROOT:$UTMP /var/run/utmp
   chmod 664 /var/run/utmp
 fi
 
@@ -120,7 +133,7 @@
   update_to_current_default profile
   update_to_current_default nsswitch.conf
   if dpkg --compare-versions "$2" lt-nl "7.7"; then
-    install_directory mnt 755 root
+    install_directory mnt 755 $ROOT
   fi
 fi
 
