diff -rup aspcud_2011.03.17.dfsg.orig/scripts/aspcud.sh aspcud_2011.03.17.dfsg/scripts/aspcud.sh
--- aspcud_2011.03.17.dfsg.orig/scripts/aspcud.sh	2011-03-14 17:52:27.000000000 +0100
+++ aspcud_2011.03.17.dfsg/scripts/aspcud.sh	2012-09-01 13:00:35.000000000 +0200
@@ -62,7 +62,11 @@ base="$(dirname "$(readlink -f "$0")")"
 PATH=".:$base:$base/../build/release/bin:$PATH"
 
 # default options
-clasp_opts_def=( "--opt-he=1" "--sat" "--restarts=32" "--heu=VSIDS" "--restart-o" "--opt-hi=2" )
+if clasp -v | grep -q "clasp 2.0"; then
+    clasp_opts_def=( "--opt-he=1" "--sat" "--restarts=32" "--heu=VSIDS" "--restart-o" "--opt-hi=2" )
+else
+    clasp_opts_def=( "--opt-he=1" "--sat" "--restarts=L,32" "--heu=VSIDS" "--restart-o" "--opt-hi=2" )
+fi
 gringo_opts_def=( "$(enc configuration.lp)" "$(enc optimize-define.lp)" )
 
 cudf_opts=( )
