diff -Nru /tmp/Bvfp0387UD/devscripts-2.9.22/debian/changelog /tmp/Otgj7dAS6w/devscripts-2.9.22/debian/changelog
--- /tmp/Bvfp0387UD/devscripts-2.9.22/debian/changelog	2006-10-11 21:16:54.000000000 +0200
+++ /tmp/Otgj7dAS6w/devscripts-2.9.22/debian/changelog	2006-10-18 18:33:22.000000000 +0200
@@ -1,3 +1,10 @@
+devscripts (2.9.22-0.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * adds --insecure to dch to allow insecure ssl downloads
+
+ -- Sune Vuorela <debian@pusling.com>  Wed, 18 Oct 2006 18:32:44 +0200
+
 devscripts (2.9.22) unstable; urgency=low
 
   [ Julian Gilbey ]
diff -Nru /tmp/Bvfp0387UD/devscripts-2.9.22/dget.pl /tmp/Otgj7dAS6w/devscripts-2.9.22/dget.pl
--- /tmp/Bvfp0387UD/devscripts-2.9.22/dget.pl	2006-10-01 02:30:34.000000000 +0200
+++ /tmp/Otgj7dAS6w/devscripts-2.9.22/dget.pl	2006-10-18 18:32:41.000000000 +0200
@@ -45,6 +45,7 @@
     my ($file, $url) = @_;
     my @cmd = ($wget);
     push @cmd, ($wget eq "wget" ? "-q" : "-s") if $opt->{quiet};
+    push @cmd, ($wget eq "wget" ? "--no-check-certificate" : "--insecure") if $opt->{insecure};
     push @cmd, ($wget eq "wget" ? "-O" : "-o");
     system @cmd, $file, $url;
     return $? >> 8;
@@ -222,6 +223,7 @@
     '--quiet'	=>  \$opt->{'quiet'},
     '-x'	=>  \$opt->{'unpack_source'},
     '--extract'	=>  \$opt->{'unpack_source'},
+    '--insecure'	=> \$opt->{'insecure'},
 )) {
     usage();
 }
@@ -296,6 +298,8 @@
 
 B<-x> run B<dpkg-source -x> on the downloaded source package.
 
+B<--insecure> allow ssl connections to untrusted hosts.
+
 =head1 BUGS
 
 B<dget> I<package> should be implemented in B<apt-get install -d>.
