diff -Nru ./build/pkg-lists/cdrom/powerpc.cfg ../debian-installer-20101121/build/pkg-lists/cdrom/powerpc.cfg
--- ./build/pkg-lists/cdrom/powerpc.cfg	2010-01-11 10:35:40.000000000 -0500
+++ ../debian-installer-20101121/build/pkg-lists/cdrom/powerpc.cfg	2010-11-21 17:22:13.000000000 -0500
@@ -2,6 +2,7 @@
 firewire-core-modules-${kernel:Version}
 
 ide-modules-${kernel:Version}
+pata-modules-${kernel:Version}
 console-keymaps-at
 console-keymaps-usb
 kbd-chooser
