diff -ruN cyrus-sasl-2.1.19-old/plugins/ntlm.c cyrus-sasl-2.1.19/plugins/ntlm.c
--- cyrus-sasl-2.1.19-old/plugins/ntlm.c	2004-06-30 14:50:04.000000000 +0000
+++ cyrus-sasl-2.1.19/plugins/ntlm.c	2005-10-20 14:05:26.000000000 +0000
@@ -74,6 +74,7 @@
 #endif /* WIN32 */
 
 #include <openssl/md4.h>
+#include <openssl/md5.h>
 #include <openssl/hmac.h>
 #include <openssl/des.h>
 #include <openssl/opensslv.h>
