--- initscripts/postinst~	2011-11-26 15:28:52.000000000 -0500
+++ initscripts/postinst	2011-11-26 15:55:07.000000000 -0500
@@ -226,7 +226,7 @@
 # sysadmin should, if they care, move the old locations to the new
 # locations and create compatibilty symlinks at their convenience
 # following the upgrade.
-if ischroot; then
+if ischroot --default-true; then
 	# Symlink /var/run from /run
         # Note var/run is relative
 	if compat_link /var/run /run; then
