--- 90-hdparm.sh~	2008-05-13 03:03:31.000000000 -0400
+++ 90-hdparm.sh	2008-05-18 21:27:30.000000000 -0400
@@ -20,7 +20,9 @@
   AC_POWER=$( /usr/bin/on_ac_power; echo $? )
   for dev in /dev/sd? /dev/hd? ; do
     if [ -b $dev ] ; then
-      if hdparm -i $dev | grep -q 'AdvancedPM=yes' ; then
+      # Check for APM support; discard errors since not all drives
+      # support HDIO_GET_IDENTITY (-i).
+      if hdparm -i $dev 2> /dev/null | grep -q 'AdvancedPM=yes' ; then
 	if [ $AC_POWER -eq 1 ] ; then
 	  hdparm -B 128 $dev
 	else
