diff --git a/src/latexgen.cpp b/src/latexgen.cpp
index 40ad877..10e50de 100644
--- a/src/latexgen.cpp
+++ b/src/latexgen.cpp
@@ -290,6 +290,7 @@ static void writeDefaultHeaderPart1(FTextStream &t)
 
   // Load required packages
   t << "% Packages required by doxygen\n"
+       "\\usepackage{fixltx2e}\n" // for \textsubscript
        "\\usepackage{calc}\n"
        "\\usepackage{doxygen}\n"
        "\\usepackage{graphicx}\n"
@@ -297,7 +298,6 @@ static void writeDefaultHeaderPart1(FTextStream &t)
        "\\usepackage{makeidx}\n"
        "\\usepackage{multicol}\n"
        "\\usepackage{multirow}\n"
-       "\\usepackage{fixltx2e}\n" // for \textsubscript
        "\\PassOptionsToPackage{warn}{textcomp}\n"
        "\\usepackage{textcomp}\n"
        "\\usepackage[nointegrals]{wasysym}\n"
