diff -ru cron-3.0pl1.pristine/debian/control cron-3.0pl1/debian/control
--- cron-3.0pl1.pristine/debian/control	2008-06-09 17:16:51.000000000 +0100
+++ cron-3.0pl1/debian/control	2008-06-09 17:25:23.000000000 +0100
@@ -13,24 +13,15 @@
 Suggests: anacron (>=2.0-1), logrotate, lockfile-progs, checksecurity
 Conflicts: suidmanager (<< 0.50), lockfile-progs (<< 0.1.7)
 Provides:
-Description: management of regular background processing
- cron is a background process (`daemon') that runs programs at regular
- intervals (for example, every minute, day, week or month); which
- processes are run and at what times are specified in the `crontab'.
+Description: process scheduling daemon
+ The cron daemon is a background process that runs particular programs
+ at particular times (such as every Friday at midnight, or once every
+ month), as specified in a crontab. By default, users may also create
+ crontabs of their own so that processes are run on their behalf.
  .
- Users may also install crontabs so that processes are run on
- their behalf, though this feature can be disabled or restricted to
- particular users.
+ A standard daily "cron job" is installed to perform system maintenance
+ tasks such as creating /var/backups/passwd.bak.
  .
- Output from the commands is usually mailed to the system administrator
- (or to the user in question); you should probably install a mail system
- as well so that you can receive these messages.
- .
- This cron package is configured by default to do various standard
- system maintenance tasks, such as ensuring that logfiles do not
- grow endlessly and overflow the disk.
- .
- The lockfile-progs package is only a "Suggests" because of the poor
- way that dselect handles "Recommends", but I do strongly suggest that
- you install it; it prevents /etc/cron.daily/standard from running multiple
- times if something gets jammed.
+ Output from the commands is usually sent to the system administrator
+ (or to the user in question); for these messages to be delivered a
+ mail system must be installed.
