Index: namecheck.pl
===================================================================
--- namecheck.pl	(révision 1882)
+++ namecheck.pl	(copie de travail)
@@ -91,18 +91,14 @@
 
 
 
-=begin doc
+#
+#  Load the list of sites, and patterns, to test.
+#
+#  By default these will come from the end of the script
+# itself.  A user may create the file ~/.namecheckrc with
+# their own patterns if they prefer.
+#
 
-  Load the list of sites, and patterns, to test.
-
-  By default these will come from the end of the script
- itself.  A user may create the file ~/.namecheckrc with
- their own patterns if they prefer.
-
-=end doc
-
-=cut
-
 sub loadPatterns
 {
     my $file  = $ENV{ 'HOME' } . "/.namecheckrc";
@@ -129,17 +125,11 @@
     return (@lines);
 }
 
+#
+#  Test the given name against the patterns we've loaded from our
+# own script, or the users configuration file.
+#
 
-
-=begin doc
-
-  Test the given name against the patterns we've loaded from our
- own script, or the users configuration file.
-
-=end doc
-
-=cut
-
 sub testSites
 {
     my (@patterns) = (@_);
