diff -urN bash-4.4.18/debian/bash-doc.doc-base.bashref bash-4.4.18-sebastien/debian/bash-doc.doc-base.bashref
--- bash-4.4.18/debian/bash-doc.doc-base.bashref	2013-10-23 14:41:22.000000000 +0200
+++ bash-4.4.18-sebastien/debian/bash-doc.doc-base.bashref	2018-03-20 14:25:24.042496979 +0100
@@ -6,5 +6,9 @@
  incorporates useful features from the Korn and C shells (ksh and csh).
 Section: Shells
 
+Format: html
+Index: /usr/share/doc/bash/bashref.html
+Files: /usr/share/doc/bash/bashref.html
+
 Format: pdf
 Files: /usr/share/doc/bash/bashref.pdf
