--- dovecot-core.postinst~	2012-06-22 07:52:45.000000000 +0200
+++ dovecot-core.postinst	2012-07-28 09:49:25.440570550 +0200
@@ -119,8 +119,8 @@
   SSL_KEY="/etc/ssl/private/dovecot.pem"
 
   # Generate new certs if needed
-  if [ -e $SSL_CERT ] && [ -e $SSL_KEY ]; then
-    echo "You already have ssl certs for dovecot."
+  if [ -e $SSL_CERT ] || [ -e $SSL_KEY ]; then
+    echo "Warning : You seem to already have ssl certs for dovecot."
   else
     echo "Creating generic self-signed certificate: $SSL_CERT"
     echo "This certificate will expire in 10 years."
