diff -ru dh-make-0.44.old/lib/debiank/control dh-make-0.44/lib/debiank/control
--- dh-make-0.44.old/lib/debiank/control	2007-01-07 23:10:24.000000000 +0000
+++ dh-make-0.44/lib/debiank/control	2008-02-12 17:23:53.000000000 +0000
@@ -13,7 +13,7 @@
 Package: #PACKAGE#-source
 Architecture: all
 Depends: module-assistant, debhelper (>= 5), make, bzip2
-Description: Source for the #PACKAGE# driver.
+Description: source for the #PACKAGE# driver
  This package provides the source code for the #PACKAGE# kernel modules.
  The #PACKAGE# package is also required in order to make use of these
  modules. Kernel source or headers are required to compile these modules.
diff -ru dh-make-0.44.old/lib/debianm/control dh-make-0.44/lib/debianm/control
--- dh-make-0.44.old/lib/debianm/control	2006-06-22 02:03:58.000000000 +0100
+++ dh-make-0.44/lib/debianm/control	2008-02-12 17:24:05.000000000 +0000
@@ -13,5 +13,5 @@
 
 Package: #PACKAGE#-doc
 Architecture: all
-Description: Documentation for #PACKAGE#
+Description: documentation for #PACKAGE#
  <insert long description, indented with spaces>
