diff -Nru emacs-goodies-el-36.3/debian/changelog emacs-goodies-el-36.3+nmu1/debian/changelog
--- emacs-goodies-el-36.3/debian/changelog	2016-11-21 13:20:03.000000000 -0700
+++ emacs-goodies-el-36.3+nmu1/debian/changelog	2017-01-27 22:15:49.000000000 -0700
@@ -1,3 +1,13 @@
+emacs-goodies-el (36.3+nmu1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * emacs-goodies-el:
+  - Bug fix: "fails to upgrade lenny -> squeeze -> wheezy -> jessie",
+    thanks to Andreas Beckmann (Closes: #851605).  Skip byte-compilation
+    for minibuffer-complete-cycle.el on emacs22.
+
+ -- Sean Whitton <spwhitton@spwhitton.name>  Fri, 27 Jan 2017 22:15:49 -0700
+
 emacs-goodies-el (36.3) unstable; urgency=high
 
   * emacs-goodies-el:
diff -Nru emacs-goodies-el-36.3/debian/emacs-goodies-el.emacsen-install.in emacs-goodies-el-36.3+nmu1/debian/emacs-goodies-el.emacsen-install.in
--- emacs-goodies-el-36.3/debian/emacs-goodies-el.emacsen-install.in	2016-11-21 13:17:53.000000000 -0700
+++ emacs-goodies-el-36.3+nmu1/debian/emacs-goodies-el.emacsen-install.in	2017-01-27 19:27:51.000000000 -0700
@@ -21,6 +21,7 @@
 EXCLUDED_xemacs21="csv-mode.el minibuf-electric.el pp-c-l.el tabbar.el todoo.el rfcview.el upstart-mode.el maplev.el button-lock.el"
 #EXCLUDED_emacs_snapshot="cua.el cfengine.el ibuffer.el ido.el newsticker.el table.el "
 #EXCLUDED_emacs22="cua.el cfengine.el ibuffer.el ido.el newsticker.el table.el "
+EXCLUDED_emacs22="minibuffer-complete-cycle.el"
 EXCLUDED_emacs23="minibuffer-complete-cycle.el"
 
 # Skip byte-compilation here if necessary:
