diff -ru abcde-2.1.4/abcde abcde-2.1.4-he/abcde
--- abcde-2.1.4/abcde	2003-05-18 21:01:13.000000000 +0200
+++ abcde-2.1.4-he/abcde	2003-07-24 09:28:18.000000000 +0200
@@ -525,7 +525,7 @@
 	do
 		# Shares some code with do_move since the filenames have to match
 		CDDBTRACKNUM=$(expr $UTRACKNUM - 1)
-		TRACKNAME=$(grep ^TTITLE$CDDBTRACKNUM= "$CDDBDATA" | head -1 | cut -f2 -d= | tr -d \[:cntrl:\])
+		TRACKNAME=$(grep ^TTITLE$CDDBTRACKNUM= "$CDDBDATA" | head -1 | cut -f2- -d= | tr -d \[:cntrl:\])
 		splitvarious
 		TRACKFILE=$(mungefilename "$TRACKNAME")
 		ARTISTFILE=$(mungefilename "$TRACKARTIST")
@@ -1093,7 +1093,7 @@
 	WAVDATA="$ABCDETEMPDIR/track$UTRACKNUM.wav"
 	OUTDATA="$ABCDETEMPDIR/track$UTRACKNUM.$OUTPUTTYPE"
 	if [ -r "$CDDBDATA" ]; then
-		TRACKNAME=$(grep ^TTITLE$CDDBTRACKNUM= "$CDDBDATA" | head -1 | cut -f2 -d= | tr -d \[:cntrl:\])
+		TRACKNAME=$(grep ^TTITLE$CDDBTRACKNUM= "$CDDBDATA" | head -1 | cut -f2- -d= | tr -d \[:cntrl:\])
 		echo "Grabbing track $UTRACKNUM: $TRACKNAME..." >&2
 	else
 		echo "Grabbing track $UTRACKNUM..." >&2
@@ -1607,7 +1607,7 @@
 	if [ -e "$CDDBDATA" ]; then
 		TRACKNUM=$(printf %0.${TRACKNUMPADDING}d $(expr ${UTRACKNUM} + 0))
 	        CDDBTRACKNUM=$(expr $UTRACKNUM - 1)
-		TRACKNAME=$(grep ^TTITLE$CDDBTRACKNUM= "$CDDBDATA" | head -1 | cut -f2 -d= | tr -d \[:cntrl:\])
+		TRACKNAME=$(grep ^TTITLE$CDDBTRACKNUM= "$CDDBDATA" | head -1 | cut -f2- -d= | tr -d \[:cntrl:\])
 		splitvarious
 	fi
 	# You can't tag a file before it's finished encoding -
