diff -Nru angband-3.3.2/debian/changelog angband-3.3.2/debian/changelog
--- angband-3.3.2/debian/changelog	2012-08-22 19:12:23.000000000 +0200
+++ angband-3.3.2/debian/changelog	2014-06-09 20:15:51.000000000 +0200
@@ -1,3 +1,10 @@
+angband (1:3.3.2-2.2) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * debian/control: Fixed typo in the description (Closes: #648058)
+
+ -- Domenico Iezzi <dodo.godlike@gmail.com>  Mon, 09 Jun 2014 20:13:18 +0200
+
 angband (1:3.3.2-2.1) unstable; urgency=low
 
   * Non-maintainer upload.
diff -Nru angband-3.3.2/debian/control angband-3.3.2/debian/control
--- angband-3.3.2/debian/control	2011-11-04 21:38:35.000000000 +0100
+++ angband-3.3.2/debian/control	2014-06-09 20:12:35.000000000 +0200
@@ -32,7 +32,7 @@
  defeat many foes and sift through a great deal of treasure.
  .
  Angband is a reference to Morgoth's "prison of iron" in the world of
- Middle-Earth, created by J.R.R. Tolkein.
+ Middle-Earth, created by J.R.R. Tolkien.
 
 Package: angband-data
 Architecture: all
