--- debian/dict-wn.postinst.old	2012-06-02 10:20:23.000000000 +0900
+++ debian/dict-wn.postinst	2012-06-02 10:48:17.000000000 +0900
@@ -9,7 +9,9 @@
 	if [ -x /usr/sbin/dictdconfig ]; then dictdconfig -w ;fi
 
 	if which invoke-rc.d >/dev/null 2>&1; then
-	    invoke-rc.d dictd restart
+            if invoke-rc.d --query dictd restart; [ $? = 104 ]; then
+	        invoke-rc.d dictd restart
+            fi
 	else
 	    [ -x /etc/init.d/dictd ] && /etc/init.d/dictd restart
 	fi
