commit 8019e48
Author: Tomas Pospisek <tpo_hp@sourcepole.ch>
Date:   Thu Sep 11 17:59:34 2014 +0200

    remove 'base' package
    
    Fixes #734053

diff --git bin/reportbug bin/reportbug
index 127cbda..80155b4 100755
--- bin/reportbug
+++ bin/reportbug
@@ -483,7 +483,7 @@ def get_package_name(bts='debian', mode=MODE_EXPERT):
                          'bug tracking system itself.'):
                 package = 'reportbug'
 
-        if package in ('general', 'project', 'debian-general', 'base'):
+        if package in ('general', 'project', 'debian-general'):
             ui.long_message(
                 "If you have a general problem, please do consider using "
 		'the available Debian support channels to narrow the problem '
diff --git reportbug/debbugs.py reportbug/debbugs.py
index ce21ba4..0e4abb6 100644
--- reportbug/debbugs.py
+++ reportbug/debbugs.py
@@ -160,7 +160,6 @@ def convert_severity(severity, type='debbugs'):
 
 # These packages are virtual in Debian; we don't look them up...
 debother = {
-    'base' : 'General bugs in the base system',
     'bugs.debian.org' : 'The bug tracking system, @bugs.debian.org',
     'buildd.debian.org' :  'Problems and requests related to the Debian Buildds',
     'buildd.emdebian.org' :  'Problems related to building packages for Emdebian',
