diff -u dnsmasq-2.46/debian/postinst dnsmasq-2.46/debian/postinst
--- dnsmasq-2.46/debian/postinst
+++ dnsmasq-2.46/debian/postinst
@@ -11,7 +11,7 @@
 
 fi
 
-update-rc.d dnsmasq defaults 15 85 >/dev/null
+update-rc.d dnsmasq start 15 2 3 4 5 . stop 85 1 . >/dev/null
 
 if [ "$1" = "configure" ] || [ "$1" = "abort-upgrade" ]; then
   if [ -e /var/run/dnsmasq.pid ]; then
diff -u dnsmasq-2.46/debian/init dnsmasq-2.46/debian/init
--- dnsmasq-2.46/debian/init
+++ dnsmasq-2.46/debian/init
@@ -4,7 +4,7 @@
 # Required-Start: $network $remote_fs $syslog
 # Required-Stop:  $network $remote_fs $syslog
 # Default-Start:  2 3 4 5
-# Default-Stop:   0 1 6
+# Default-Stop:   1
 # Description:    DHCP and DNS server
 ### END INIT INFO
 
