--- apt-cacher-ng-0.4.9.orig/conf/acng.conf
+++ apt-cacher-ng-0.4.9/conf/acng.conf
@@ -98,7 +98,7 @@
 # MaxConThreads: -1
 #
 #VfilePattern = (^|.*?/)(Index|Packages\.bz2|Packages\.gz|Packages|Release|Release\.gpg|Sources\.bz2|Sources\.gz|Sources|release|index\.db-.*\.gz|Contents-[^/]*\.gz|pkglist[^/]*\.bz2|rclist[^/]*\.bz2|/meta-release[^/]*|Translation[^/]*\.bz2)$
-#PfilePattern = .*(\.deb|\.rpm|\.dsc|\.tar\.gz\.gpg|\.tar\.gz|\.diff\.gz|\.diff\.bz2|\.jigdo|\.template|changelog|copyright|\.udeb|\.diff/.*\.gz|vmlinuz|initrd\.gz|ReleaseAnnouncement)$
+#PfilePattern = .*(\.deb|\.rpm|\.dsc|\.tar\.gz\.gpg|\.tar\.gz|\.diff\.gz|\.diff\.bz2|\.jigdo|\.template|changelog|copyright|\.udeb|\.diff/.*\.gz|vmlinuz|initrd\.gz|(Devel)?ReleaseAnnouncement(\\?.*)?)$
 # Whitelist for expiration, file types not to be removed even when being
 # unreferenced. Default: same as VfilePattern which is a safe bed. When and
 # only when the only used mirrors are official repositories (with working
--- apt-cacher-ng-0.4.9.orig/source/acfg_defaults.cc
+++ apt-cacher-ng-0.4.9/source/acfg_defaults.cc
@@ -28,7 +28,7 @@
 
 string pfilepat(".*(\\.d?deb|\\.rpm|\\.dsc|\\.tar\\."COMPRLIST"(\\.gpg)?"
 		"|\\.diff\\."COMPRLIST"|\\.jigdo|\\.template|changelog|copyright|"
-		"\\.udeb|\\.diff/.*\\.gz|vmlinuz|initrd\\."COMPRLIST"|ReleaseAnnouncement)$");
+		"\\.udeb|\\.diff/.*\\.gz|vmlinuz|initrd\\."COMPRLIST"|(Devel)?ReleaseAnnouncement(\\?.*)?)$");
 
 string vfilepat( VPATPREFIX "(Index|Packages\\.bz2|Packages\\.gz|Packages|Release|Release\\.gpg|"
 		"Sources\\.bz2|Sources\\.gz|Sources|release|index\\.db-.*\\.gz|Contents-[^/]*\\.gz|"
