Index: gtk.c
===================================================================
--- gtk.c       (revisione 42389)
+++ gtk.c       (copia locale)
@@ -1661,6 +1661,10 @@

     gdk_threads_enter();

+    gtk_widget_set_sensitive (data->button_prev, FALSE);
+    gtk_widget_set_sensitive (data->button_next, FALSE);
+    gtk_widget_set_sensitive (data->button_screenshot, FALSE);
+
     if (data->button_val == DC_OK)
     {
         call_setters(obj);
@@ -1679,9 +1683,6 @@
     else
         gtk_widget_destroy(questionbox_scroll);

-    gtk_widget_set_sensitive (data->button_prev, FALSE);
-    gtk_widget_set_sensitive (data->button_next, FALSE);
-
     gdk_threads_leave();

     if (data->button_val == DC_OK)
