--- console-setup.orig/debian/console-setup.postinst	2006-10-19 21:11:10.000000000 +0100
+++ console-setup/debian/console-setup.postinst	2007-04-01 12:18:35.000000000 +0100
@@ -2,6 +2,9 @@
 
 set -e
 
+# Source debconf library.
+. /usr/share/debconf/confmodule
+
 # The same as /usr/bin/which - in order to make this script work
 # in environments where "which" is not available
 which () {
@@ -31,9 +34,6 @@
 CONFIGDIR=/etc/console-setup
 CONFIGFILE=/etc/default/console-setup
 
-# Source debconf library.
-. /usr/share/debconf/confmodule
-
 if [ "$1" = "configure" ]; then
     
     db_get console-setup/codesetcode
