diff -aur courier-authlib-0.63.0.orig/debian/rules courier-authlib-0.63.0/debian/rules
--- courier-authlib-0.63.0.orig/debian/rules	2012-01-09 00:09:28.000000000 +0100
+++ courier-authlib-0.63.0/debian/rules	2012-01-09 00:11:24.000000000 +0100
@@ -29,8 +29,14 @@
 # Common configuration options
 NONSSL_CONFOPTS=--without-certdb --without-authpgsql
 
+CFLAGS = `dpkg-buildflags --get CFLAGS`
+LDFLAGS = `dpkg-buildflags --get LDFLAGS`
+LDFLAGS += -lcrypt
+CPPFLAGS = `dpkg-buildflags --get CPPFLAGS`
+
 COMMON_CONFOPTS=--prefix=/usr --mandir=\$${prefix}/share/man \
 		--with-authdaemonvar=/var/run/courier/authdaemon \
+		CFLAGS="$(CFLAGS)" CPPFLAGS="$(CPPFLAGS)" LDFLAGS="$(LDFLAGS)" \
 		--sysconfdir=/etc/courier \
 		--with-pkgconfdir=/etc/courier \
 		--libexecdir=\$${prefix}/lib/courier \
@@ -72,11 +78,11 @@
 # create a list of files that currently exists
 	if [ ! -f stamp-build ]; then \
 		if [ -d /usr/include/postgresql/8.0/libpq_fe.h ]; then \
-			LDFLAGS=-lcrypt ./configure $(COMMON_CONFOPTS) \
+			./configure $(COMMON_CONFOPTS) \
 			--with-postgresql-includes=/usr/include/postgresql/8.0 \
 			&& $(MAKE) && touch stamp-build; \
 		else \
-			LDFLAGS=-lcrypt ./configure $(COMMON_CONFOPTS) \
+			./configure $(COMMON_CONFOPTS) \
 			--with-postgresql-includes=/usr/include/postgresql \
 			&& $(MAKE) && touch stamp-build; \
 		fi \
Nur in courier-authlib-0.63.0/debian: rules~.
