Index: en/upgrading.dbk
===================================================================
--- en/upgrading.dbk	(revision 10039)
+++ en/upgrading.dbk	(working copy)
@@ -368,7 +368,10 @@
 </screen>
 <para>
 Replace <literal>hold</literal> with <literal>install</literal> to unset the
-<quote>hold</quote> state.
+<quote>hold</quote> state. Also take into account that starting with dpkg
+1.16.x, an up-to-date available database is needed for the command to be
+useful (see the <ulink url="&url-wiki;Teams/Dpkg/FAQ">dpkg FAQ</ulink>
+for more information).
 </para>
 <para>
 If there is anything you need to fix, it is best to make sure your
