--- a/debian/default
+++ b/debian/default
@@ -4,3 +4,7 @@
 # (Things like the locate updatedb cause a lot of I/O.)
 
 ANACRON_RUN_ON_BATTERY_POWER=no
+
+# Arguments/options to pass to anacron. If you want tasks to execute
+# in parallel when they are due to start, do not pass '-s'
+ANACRON_ARGS="-s"
--- a/debian/init.d
+++ b/debian/init.d
@@ -41,7 +41,7 @@
 
     # on_ac_power doesn't exist, on_ac_power returns 0 (ac power being used)
     # or on_ac_power returns 255 (undefined, desktop machine without APM)
-    start-stop-daemon --start --exec /usr/sbin/anacron -- -s
+    start-stop-daemon --start --exec /usr/sbin/anacron -- $ANACRON_ARGS
     log_end_msg 0
     ;;
   restart|force-reload|reload)
