--- developers-reference.sgml.old	2005-08-02 00:03:37.000000000 -0700
+++ developers-reference.sgml	2005-08-02 00:02:22.000000000 -0700
@@ -3741,10 +3741,11 @@
 the package manager (e.g., "this is the client for the foo server")?
 	    <p>
 Be careful to avoid spelling and grammar mistakes. Ensure that you
-spell-check it.  <prgn>ispell</prgn> has a special <tt>-g</tt> option
-for <file>debian/control</file> files:
+spell-check it.  Both <prgn>ispell</prgn> and <prgn>aspell</prgn> have
+modes for <file>debian/control</file> files:
 
 <example>ispell -d american -g debian/control</example>
+<example>aspell -d en -D -c debian/control</example>
 	    <p>
 Users usually expect these questions to be answered in the package
 description:
