--- a/usr/bin/battery-graph	2024-01-20 00:52:13.000000000 +0100
+++ b/usr/bin/battery-graph	2024-06-03 19:35:42.271192074 +0200
@@ -184,6 +184,10 @@
 
 
 (   
+    if [ -n "$title" ] ; then
+	echo "set title \"$title\""
+    fi
+    
     if $text ; then
 	echo set terminal dumb ${COLUMNS:-$(tput cols)} ${LINES:-$(tput lines)}
     fi
@@ -218,7 +222,7 @@
 		echo ", g(x -($TIME_LAST_DISCHARGE_BEGIN-$adjustment) ) title (B<0?sprintf(\"slope= (%.2f +/- %.2f) %/h\", B*3600, B_err*3600):\"\") lc rgb \"black\" lt 2 "
     fi
     
-)  | gnuplot -persist ${geometry:+-geometry} $geometry ${title:+-title} "${title}" ; rm -f $TMPFILENAME
+)  | gnuplot -persist ${geometry:+-geometry} $geometry ; rm -f $TMPFILENAME
 
 
 # TODO Have to decide if we want to clean up or leave the file for us to zoom in/out in the graph
