diff -urN asciijump-0.0.6-old/debian/changelog asciijump-0.0.6/debian/changelog
--- asciijump-0.0.6-old/debian/changelog	2005-06-20 13:59:11.109604000 +0000
+++ asciijump-0.0.6/debian/changelog	2005-06-20 14:00:05.905274008 +0000
@@ -1,3 +1,9 @@
+asciijump (0.0.6-1.3) unstable; urgency=low
+
+  * Build against libslang2. 
+
+ -- Alastair McKinstry <mckinstry@debian.org>  Mon, 20 Jun 2005 13:59:45 +0000
+
 asciijump (0.0.6-1.2) unstable; urgency=low
 
   * NMU
diff -urN asciijump-0.0.6-old/debian/control asciijump-0.0.6/debian/control
--- asciijump-0.0.6-old/debian/control	2005-06-20 13:59:11.109604000 +0000
+++ asciijump-0.0.6/debian/control	2005-06-20 13:59:42.840780344 +0000
@@ -2,7 +2,7 @@
 Section: games
 Priority: optional
 Maintainer: Rafal Zawadzki <bluszcz@ats.com.pl>
-Build-Depends: debhelper (>> 4.0.0), slang1-dev (>= 1.4.5-1 ), dpatch
+Build-Depends: debhelper (>> 4.0.0), libslang2-dev, dpatch
 Standards-Version: 3.6.1
 
 Package: asciijump
