--- bash_completion.orig	2008-09-22 11:43:38.000000000 +0200
+++ bash_completion	2008-09-22 11:45:19.000000000 +0200
@@ -4058,7 +4058,7 @@
 			local path_array=( $(echo "$PATH") )
 			IFS=$'\n'
 			COMPREPLY=( $( compgen -d -W '$(find "${path_array[@]}" . \
-				-mindepth 1 -maxdepth 1 -not -type d -executable -printf "%f\\n")' \
+				-mindepth 1 -maxdepth 1 -not -type d -executable -printf "%f\\n" 2>/dev/null)' \
 				-- "$cur" ) )
 		fi
 	elif [ $COMP_CWORD -eq 2 ]; then
