Index: adonthell-0.3.4.cvs.20050813/src/py_adonthell_wrap.cc
===================================================================
--- adonthell-0.3.4.cvs.20050813.orig/src/py_adonthell_wrap.cc	2008-04-20 22:54:27.000000000 +0100
+++ adonthell-0.3.4.cvs.20050813/src/py_adonthell_wrap.cc	2008-04-20 22:55:15.000000000 +0100
@@ -1174,7 +1174,7 @@
     obj = pyobj;
     if (PyCFunction_Check(obj)) {
       /* here we get the method pointer for callbacks */
-      char *doc = (((PyCFunctionObject *)obj) -> m_ml -> ml_doc);
+      const char *doc = (((PyCFunctionObject *)obj) -> m_ml -> ml_doc);
       c = doc ? strstr(doc, "swig_ptr: ") : 0;
       if (c) {
 	c = SWIG_UnpackVoidPtr(c + 10, &vptr, ty->name);
@@ -28592,11 +28592,11 @@
     swig_type_info **types_initial) {
         size_t i;
         for (i = 0; methods[i].ml_name; ++i) {
-            char *c = methods[i].ml_doc;
+            const char *c = methods[i].ml_doc;
             if (c && (c = strstr(c, "swig_ptr: "))) {
                 int j;
                 swig_const_info *ci = 0;
-                char *name = c + 10;
+                const char *name = c + 10;
                 for (j = 0; const_table[j].type; j++) {
                     if (strncmp(const_table[j].name, name, 
                     strlen(const_table[j].name)) == 0) {
