diff -ruNp awstats-7.1/wwwroot/cgi-bin/awstats.pl awstats-7.1.1/wwwroot/cgi-bin/awstats.pl
--- awstats-7.1/wwwroot/cgi-bin/awstats.pl	2012-12-19 18:30:47.000000000 +0100
+++ awstats-7.1.1/wwwroot/cgi-bin/awstats.pl	2013-03-08 21:16:52.000000000 +0100
@@ -7817,7 +7817,8 @@ sub DecodeEncodedString {
 # Return:		standardregex
 #------------------------------------------------------------------------------
 sub UnCompileRegex {
-	shift =~ /\(\?[-\w]*:(.*)\)/;
+	shift =~ /\(\?[-^\w]*:(.*)\)/;         # Works with all perl
+	# shift =~ /\(\?[-\w]*:(.*)\)/;        < perl 5.14
 	return $1;
 }
 
