From 253ed2021ea8e8aded96cbe55a7c40fae0bf3d66 Mon Sep 17 00:00:00 2001
From: "Paul R. Tagliamonte" <paultag@debian.org>
Date: Sat, 23 Feb 2013 20:43:00 -0500
Subject: [PATCH] Change desktop entry to display in the login manager.

---
 awesome.desktop |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/awesome.desktop b/awesome.desktop
index 1f7878c..6f830b9 100644
--- a/awesome.desktop
+++ b/awesome.desktop
@@ -2,7 +2,7 @@
 Encoding=UTF-8
 Name=awesome
 Comment=Highly configurable framework window manager
-NoDisplay=true
+NoDisplay=false
 TryExec=awesome
 Exec=awesome
 Type=Application
-- 
1.7.10.4

