diff -u apache2-2.2.6/debian/rules apache2-2.2.6/debian/rules
--- apache2-2.2.6/debian/rules
+++ apache2-2.2.6/debian/rules
@@ -102,7 +102,7 @@
 	dh_testdir
 	mkdir -p $(BUILD)/$*
 	cd $(BUILD)/$* ;\
-	CFLAGS="$(AP2_CONFLAGS)" $(CONFFLAGS) $(REALCURDIR)/configure --srcdir=$(REALCURDIR) $(AP2_COMMON_CONFARGS) $(AP2_CONFARGS) --with-mpm=$*  ;\
+	CFLAGS="$(AP2_CONFLAGS)" LDFLAGS="-Wl,--as-needed" $(CONFFLAGS) $(REALCURDIR)/configure --srcdir=$(REALCURDIR) $(AP2_COMMON_CONFARGS) $(AP2_CONFARGS) --with-mpm=$*  ;\
 	$(MAKE)
 	touch $@
 
diff -u apache2-2.2.6/debian/changelog apache2-2.2.6/debian/changelog
--- apache2-2.2.6/debian/changelog
+++ apache2-2.2.6/debian/changelog
@@ -1,3 +1,10 @@
+apache2 (2.2.6-3ubuntu1) hardy; urgency=low
+
+  * Build with LDFLAGS=-Wl,--as-needed to drop a lot of unnecessary
+    dependencies (including db4.5).
+
+ -- Martin Pitt <martin.pitt@ubuntu.com>  Thu, 03 Jan 2008 11:19:10 +0100
+
 apache2 (2.2.6-3) unstable; urgency=low
 
   * Allocate fewer bucket brigades in case of a flush bucket. This might help
