--- bluez-hcidump-1.35/parser/hci.c	2007-07-15 22:46:26.000000000 +0000
+++ bluez-hcidump-1.35/parser/hci.c	2007-07-15 22:50:45.000000000 +0000
@@ -775,7 +775,7 @@
 
 static inline void sniff_subrate_dump(int level, struct frame *frm)
 {
-	sniff_subrate_cp *cp = frm->ptr;
+	sniff_subrating_cp *cp = frm->ptr;
 
 	p_indent(level, frm);
 	printf("handle %d\n", btohs(cp->handle));
@@ -1205,7 +1205,7 @@
 		case OCF_WRITE_LINK_POLICY:
 			write_link_policy_dump(level + 1, frm);
 			return;
-		case OCF_SNIFF_SUBRATE:
+		case OCF_SNIFF_SUBRATING:
 			sniff_subrate_dump(level + 1, frm);
 			return;
 		}
@@ -1839,7 +1839,7 @@
 	case OGF_LINK_POLICY:
 		switch (ocf) {
 		case OCF_WRITE_LINK_POLICY:
-		case OCF_SNIFF_SUBRATE:
+		case OCF_SNIFF_SUBRATING:
 			generic_response_dump(level, frm);
 			return;
 		}
@@ -2446,7 +2446,7 @@
 
 static inline void sniff_subrate_event_dump(int level, struct frame *frm)
 {
-	evt_sniff_subrate *evt = frm->ptr;
+	evt_sniff_subrating *evt = frm->ptr;
 
 	p_indent(level, frm);
 	printf("status 0x%2.2x handle %d\n", evt->status, btohs(evt->handle));
@@ -2492,7 +2492,7 @@
 
 static inline void link_supervision_timeout_change_dump(int level, struct frame *frm)
 {
-	evt_link_supervision_timeout_change *evt = frm->ptr;
+	evt_link_supervision_timeout_changed *evt = frm->ptr;
 
 	p_indent(level, frm);
 	printf("handle %d timeout %d\n",
@@ -2668,13 +2668,13 @@
 	case EVT_SYNC_CONN_CHANGED:
 		sync_conn_changed_dump(level + 1, frm);
 		break;
-	case EVT_SNIFF_SUBRATE:
+	case EVT_SNIFF_SUBRATING:
 		sniff_subrate_event_dump(level + 1, frm);
 		break;
 	case EVT_EXTENDED_INQUIRY_RESULT:
 		extended_inq_result_dump(level + 1, frm);
 		break;
-	case EVT_LINK_SUPERVISION_TIMEOUT_CHANGE:
+	case EVT_LINK_SUPERVISION_TIMEOUT_CHANGED:
 		link_supervision_timeout_change_dump(level + 1, frm);
 		break;
 	default:
