diff --git debian/watch debian/watch
index 331129d..4a7e288 100644
--- debian/watch
+++ debian/watch
@@ -3,5 +3,6 @@
 # to check for upstream updates and more.
 # Site		Directory		Pattern			Version	Script
 version=2
-http://prdownloads.sourceforge.net/adc-archmbox/archmbox-(.*)\.tar\.gz debian  uupdate
+http://sourceforge.net/project/showfiles.php?group_id=65776&package_id=63642	\
+(?:http://downloads.sourceforge.net/adc-archmbox/archmbox-([0-9\.]+)\.tar\.gz)?\?.*	debian  uupdate
 
