From 7b0d8b96d6923f353a724ffad451d2a4013cbfad Mon Sep 17 00:00:00 2001
From: Josh Triplett <josh@joshtriplett.org>
Date: Sat, 28 Apr 2012 15:31:36 -0700
Subject: [PATCH 3/3] Change exim4 to default-mta in Suggests.

---
 debian/changelog |    1 +
 debian/control   |    2 +-
 2 files changed, 2 insertions(+), 1 deletion(-)

diff --git a/debian/changelog b/debian/changelog
index b1e955a..e2f6b9c 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -3,6 +3,7 @@ apt-listchanges (2.85.10) UNRELEASED; urgency=low
   [ Josh Triplett ]
   * Avoid throwing an exception when sendmail does not exist. (Closes: 666086)
   * Move mail-transport-agent from Recommends to Suggests. (Closes: 599667)
+  * Change exim4 to default-mta in Suggests.
 
  -- Sandro Tosi <morph@debian.org>  Sat, 28 Apr 2012 14:58:35 -0700
 
diff --git a/debian/control b/debian/control
index 4c33361..84131de 100644
--- a/debian/control
+++ b/debian/control
@@ -15,7 +15,7 @@ Architecture: all
 Depends: ${python:Depends}, apt (>= 0.5.3), python-apt (>= 0.7.93),
  ucf (>= 0.28), debianutils (>= 2.0.2), ${misc:Depends}
 Suggests: x-terminal-emulator, www-browser, python-glade2, python-gtk2,
- exim4 | mail-transport-agent
+ default-mta | mail-transport-agent
 Description: package change history notification tool
  The tool apt-listchanges can compare a new version of a
  package with the one currently installed and show what has been
-- 
1.7.10

