diff -aur dpkg-1.17.6.orig/scripts/Dpkg/Source/Patch.pm dpkg-1.17.6/scripts/Dpkg/Source/Patch.pm
--- dpkg-1.17.6.orig/scripts/Dpkg/Source/Patch.pm	2013-12-14 06:31:03.000000000 +0100
+++ dpkg-1.17.6/scripts/Dpkg/Source/Patch.pm	2014-02-09 02:10:03.155885592 +0100
@@ -227,12 +227,12 @@
                 push @diff_files, [$fn, 0, 0, "$old/$fn", '/dev/null',
                                    "$basedir.orig/$fn", '/dev/null'];
             } else {
-                warning(_g('ignoring deletion of file %s'), $fn);
+                warning(_g('ignoring deletion of file %s, use --include-removal to override'), $fn);
             }
         } elsif (-d _) {
-            warning(_g('ignoring deletion of directory %s'), $fn);
+            warning(_g('ignoring deletion of directory %s, use --include-removal to override'), $fn);
         } elsif (-l _) {
-            warning(_g('ignoring deletion of symlink %s'), $fn);
+            warning(_g('ignoring deletion of symlink %s, use --include-removal to override'), $fn);
         } else {
             $self->_fail_not_same_type("$old/$fn", "$new/$fn");
         }
Nur in dpkg-1.17.6/scripts/Dpkg/Source: Patch.pm~.
