diff --git a/debian/control b/debian/control
index 9ac5681..fc1dde8 100644
--- a/debian/control
+++ b/debian/control
@@ -9,7 +9,7 @@ Build-Depends-Indep: docbook-xsl (>= 1.71.0), dblatex (>= 0.2), xsltproc, libxml
  texlive-lang-cyrillic, texlive-lang-french, texlive-lang-german, texlive-xetex,
  xmlto, zip
 Build-Depends: debhelper (>= 8), dpkg-dev (>= 1.16.1~)
-Vcs-Git: git://anonscn.debian.org/collab-maint/developers-reference.git
+Vcs-Git: git://anonscm.debian.org/collab-maint/developers-reference.git
 Vcs-Browser: http://anonscm.debian.org/gitweb/?p=collab-maint/developers-reference.git
 
 Package: developers-reference
diff --git a/resources.dbk b/resources.dbk
index d8bd056..2751b46 100644
--- a/resources.dbk
+++ b/resources.dbk
@@ -764,8 +764,8 @@ Debian 1.3, <literal>bo</literal>; Debian 2.0, <literal>hamm</literal>;
 Debian 2.1, <literal>slink</literal>; Debian 2.2, <literal>potato</literal>;
 Debian 3.0, <literal>woody</literal>; Debian 3.1, <literal>sarge</literal>;
 Debian 4.0, <literal>etch</literal>; Debian 5.0, <literal>lenny</literal>;
-Debian 6.0, <literal>squeeze</literal>
-and the next release will be called <literal>wheezy</literal>.
+Debian 6.0, <literal>squeeze</literal>; Debian 7.0, <literal>wheezy</literal>;
+and the next release will be called <literal>jessie</literal>.
 There is also a ``pseudo-distribution'', called
 <literal>sid</literal>, which is the current <literal>unstable</literal>
 distribution; since packages are moved from <literal>unstable</literal> to
