Index: resources.dbk
===================================================================
--- resources.dbk	(revision 10361)
+++ resources.dbk	(working copy)
@@ -1588,6 +1588,16 @@
 <ulink url="http://&lists-host;/debian-devel-announce/2008/11/msg00004.html"></ulink>.
 </para>
 </section>
+<section id="steam">
+<title>Valve games on Steam</title>
+<para>
+Since January 2014, Valve has sponsored free subscribtions to all past and present
+Valve games on the Steam game distribution service for all interested Debian Developers.
+Since Steam and Valve games are not Free Software, please avoid using your Debian
+development machines for using Steam and playing games from Steam.
+See <ulink url="http://&lists-host;/debian-devel-announce/2014/01/msg00006.html"></ulink>.
+</para>
+</section>
 
 </section>
 
