diff -ru autojump-17.pristine/debian/control autojump-17/debian/control
--- autojump-17.pristine/debian/control	2011-10-02 20:30:40.000000000 +0100
+++ autojump-17/debian/control	2011-11-22 15:55:03.006036202 +0000
@@ -14,12 +14,15 @@
 Depends: ${misc:Depends}, python3
 Enhances: bash, zsh, zsh-beta
 Description: shell extension to jump to frequently used directories
- autojump provides is a faster way to navigate your filesystem, with a
+ autojump provides a faster way to navigate your filesystem, with a
  "cd command that learns".
  .
  It works by maintaining a database of the directories you use the most from
  the command line, and allows you to "jump" to frequently used directories by
  typing only a small pattern.
+ .
+ To use autojump, you need to configure your shell to source
+ /usr/share/autojump/autojump.sh on startup.
 
 Package: jumpapplet
 Architecture: all
