diff --git a/debian/control b/debian/control
index 3a79b79..342eef3 100644
--- a/debian/control
+++ b/debian/control
@@ -4,7 +4,7 @@ Priority: optional
 Maintainer: Daniel Hartwig <mandyke@gmail.com>
 Build-Depends: debhelper (>= 9), libpopt-dev, linuxdoc-tools, autoconf, libgtk2.0-dev, libgnomeui-dev, po-debconf, python-dev, python-support (>= 0.90), autotools-dev
 Standards-Version: 3.9.3
-Homepage: http://code.google.com/p/distcc/
+Homepage: https://github.com/distcc/distcc
 
 Package: distcc
 Architecture: any
diff --git a/debian/watch b/debian/watch
index 4b348e8..82b466e 100644
--- a/debian/watch
+++ b/debian/watch
@@ -1,4 +1,3 @@
-version=3
-opts="uversionmangle=s/(\d)((rc|prerelease)\d*)$/$1~$2/" \
-  http://code.google.com/p/distcc/downloads/list?can=1 \
-  .*/distcc-(\d[\d.]*(?:(?:rc|prerelease)\d*)?)\.(?:zip|tgz|tbz|txz|(?:tar\.(?:gz|bz2|xz)))
+version=4
+opts=filenamemangle=s/.+\/v?(\d\S+)\.tar\.gz/distcc-$1\.tar\.gz/ \
+  https://github.com/distcc/distcc/releases/ .*/v?([\d\.]+)\.tar\.gz
