From b80ba12236d45e80e4ce643049170364f47d92c0 Mon Sep 17 00:00:00 2001
From: Paul Fenwick <pjf@perltraining.com.au>
Date: Mon, 24 Nov 2008 20:40:45 +1100
Subject: [PATCH] Trivial test script in lieu of having Makefile.PL to do things for us.

---
 test |   12 ++++++++++++
 1 files changed, 12 insertions(+), 0 deletions(-)
 create mode 100755 test

diff --git a/test b/test
new file mode 100755
index 0000000..502487c
--- /dev/null
+++ b/test
@@ -0,0 +1,12 @@
+#!/usr/bin/perl -w
+use strict;
+use autodie;
+
+use FindBin qw($Bin);
+
+print "Running tests in $Bin/t\n";
+
+chdir($Bin);
+
+# Multi-arg exec avoids the shell.
+exec( qw(prove -Ilib t/) );
-- 
1.5.5.GIT

