--- /root/scratch/backup/apache2.~2006-03-14~	2006-01-16 12:15:48.000000000 +0200
+++ /etc/init.d/apache2	2006-03-14 23:09:45.000000000 +0200
@@ -37,6 +37,7 @@
 	# here, but only first-level includes are supported, not nested ones
 
 	for i in $AP_CONF `awk '$1 ~ /^\s*[Ii]nclude$/ && $2 ~ /^\// {print $2}' $AP_CONF`; do
+		[ -e "$i" ] || continue
 		PIDFILE=`grep -i ^PidFile $i | tail -n 1 | awk '{print $2}'`
 		if [ -e "$PIDFILE" ]; then
 			PID=`cat $PIDFILE`
