diff -Nru bash-5.2.21/debian/changelog bash-5.2.21/debian/changelog
--- bash-5.2.21/debian/changelog	2024-06-05 17:19:12.000000000 +0200
+++ bash-5.2.21/debian/changelog	2024-08-15 13:19:13.000000000 +0200
@@ -1,3 +1,10 @@
+bash (5.2.21-2.2) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Uncomment default aliases for grep and ls. Closes: #774779.
+
+ -- Baptiste Beauplat <lyknode@debian.org>  Thu, 15 Aug 2024 13:19:13 +0200
+
 bash (5.2.21-2.1) unstable; urgency=medium
 
   * Non-maintainer upload.
diff -Nru bash-5.2.21/debian/skel.bashrc bash-5.2.21/debian/skel.bashrc
--- bash-5.2.21/debian/skel.bashrc	2015-01-28 17:34:29.000000000 +0100
+++ bash-5.2.21/debian/skel.bashrc	2024-08-15 13:19:04.000000000 +0200
@@ -79,18 +79,18 @@
     #alias dir='dir --color=auto'
     #alias vdir='vdir --color=auto'
 
-    #alias grep='grep --color=auto'
-    #alias fgrep='fgrep --color=auto'
-    #alias egrep='egrep --color=auto'
+    alias grep='grep --color=auto'
+    alias fgrep='fgrep --color=auto'
+    alias egrep='egrep --color=auto'
 fi
 
 # colored GCC warnings and errors
 #export GCC_COLORS='error=01;31:warning=01;35:note=01;36:caret=01;32:locus=01:quote=01'
 
 # some more ls aliases
-#alias ll='ls -l'
-#alias la='ls -A'
-#alias l='ls -CF'
+alias ll='ls -l'
+alias la='ls -A'
+alias l='ls -CF'
 
 # Alias definitions.
 # You may want to put all your additions into a separate file like
