diff -Nru --exclude changelog bash-5.2.21/debian/skel.bashrc bash-5.2.21/debian/skel.bashrc
--- bash-5.2.21/debian/skel.bashrc	2015-01-28 17:34:29.000000000 +0100
+++ bash-5.2.21/debian/skel.bashrc	2024-07-25 20:51:34.000000000 +0200
@@ -79,18 +79,18 @@
     #alias dir='dir --color=auto'
     #alias vdir='vdir --color=auto'
 
-    #alias grep='grep --color=auto'
-    #alias fgrep='fgrep --color=auto'
-    #alias egrep='egrep --color=auto'
+    alias grep='grep --color=auto'
+    alias fgrep='fgrep --color=auto'
+    alias egrep='egrep --color=auto'
 fi
 
 # colored GCC warnings and errors
 #export GCC_COLORS='error=01;31:warning=01;35:note=01;36:caret=01;32:locus=01:quote=01'
 
 # some more ls aliases
-#alias ll='ls -l'
-#alias la='ls -A'
-#alias l='ls -CF'
+alias ll='ls -l'
+alias la='ls -A'
+alias l='ls -CF'
 
 # Alias definitions.
 # You may want to put all your additions into a separate file like
