--- bashhist.c	2005-10-01 04:30:52.000000000 +0200
+++ bashhist.c~	2007-03-11 14:21:56.000000000 +0100
@@ -270,6 +270,9 @@
      Note that the history file is automatically truncated to the
      size of HISTSIZE if the user does not explicitly set the size
      differently. */
+  set_if_not ("HISTSIZE", "500");
+  sv_histsize ("HISTSIZE");
+
   set_if_not ("HISTFILESIZE", get_string_value ("HISTSIZE"));
   sv_histsize ("HISTFILESIZE");
 

