diff -ru entagged-0.35.pristine/debian/control entagged-0.35/debian/control
--- entagged-0.35.pristine/debian/control	2011-09-25 22:18:45.000000000 +0100
+++ entagged-0.35/debian/control	2012-07-07 22:22:55.363392436 +0100
@@ -14,8 +14,8 @@
 Architecture: all
 Depends: ${misc:Depends}, openjdk-6-jre | java6-runtime, libhsqldb-java,
  squareness
-Description: Java Audio File Tagger
- Entagged is a audiofile tagger, using the freedb online database for
- the retrieval of the tags. It supports custom file renaming from tags
- (with any directory stucture) and vice versa. Supports: mp3, ogg,
- flac, mpc, ape, wma.
+Description: graphical audio file tagger with freedb support
+ Entagged is an audio file tagger written in Java, using the freedb online
+ database for the retrieval of the tags. It can be used to organize files
+ into complex directory structures according to their tags, or to tag them
+ from their filenames. Supports: APE, FLAC, MP3, MPC, OGG, WMA.
