diff -wru apt-file-2.0.3/apt-file.conf apt-file-2.0.3-he/apt-file.conf
--- apt-file-2.0.3/apt-file.conf	2003-05-16 13:04:32.000000000 +0200
+++ apt-file-2.0.3-he/apt-file.conf	2003-12-27 13:31:14.000000000 +0100
@@ -20,4 +20,5 @@
 ssh = scp -l <user> -P <port|22> "<host>:/<path>/dists/<dist>/Contents-<arch>.gz" "<cache>/<dest>"
 rsh = rcp -l <user> "<host>:/<path>/dists/<dist>/Contents-<arch>.gz" "<cache>/<dest>"
 file = cp "/<path>/dists/<dist>/Contents-<arch>.gz" "<cache>/<dest>"
+copy = cp "/<path>/dists/<dist>/Contents-<arch>.gz" "<cache>/<dest>"
 cdrom = echo "Put CDROM labeled <path> in the cdrom device" > /dev/stderr ; read ; mount "<cdrom>"; cp "<cdrom>/dists/<dist>/Contents-<arch>.gz" "<cache>/<dest>"; umount "<cdrom>"
