From f5f51aadd12c3053ad1bc7fee2253a3a5062aa5f Mon Sep 17 00:00:00 2001
From: Marco Amadori <amadorim@vdavda.com>
Date: Fri, 11 Feb 2011 15:45:27 +0100
Subject: [PATCH 5/6] Exit from the sysvinit script to use the upstart job.

---
 debian/changelog |    3 ++-
 debian/nodm.init |    6 ++++++
 2 files changed, 8 insertions(+), 1 deletions(-)

diff --git a/debian/changelog b/debian/changelog
index faa6817..c9711ab 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -6,8 +6,9 @@ nodm (0.7-2) UNRELEASED; urgency=low
 
   [ Marco Amadori ]
   * Update Standards-Version to 3.9.1 (no changes required).
+  * Exit from the sysvinit script to use the upstart job.
 
- -- Marco Amadori <marco.amadori@vdavda.com>  Fri, 11 Feb 2011 15:36:30 +0100
+ -- Marco Amadori <marco.amadori@vdavda.com>  Fri, 11 Feb 2011 15:45:07 +0100
 
 nodm (0.7-1.1) unstable; urgency=low
 
diff --git a/debian/nodm.init b/debian/nodm.init
index 1b8bb6e..15ca887 100644
--- a/debian/nodm.init
+++ b/debian/nodm.init
@@ -18,6 +18,12 @@ NAME=nodm
 PIDDIR=/var/run/
 PIDFILE=${PIDDIR}/${NAME}.pid
 
+if [ -e "/etc/init/${NAME}.conf" ] && /sbin/telinit --version >/dev/null 2>&1 | grep -qs upstart
+then
+	# An upstart job exists and upstart is in use, exit gracefully
+	exit 0
+fi
+
 NODM_ENABLED=no
 NODM_XINIT=/usr/bin/xinit
 NODM_FIRST_VT=7
-- 
1.7.2.3

