diff -ruN apt-0.7.14/debian/apt.cron.daily apt-0.7.14-patched/debian/apt.cron.daily
--- apt-0.7.14/debian/apt.cron.daily	2008-05-28 15:22:14.000000000 +0200
+++ apt-0.7.14-patched/debian/apt.cron.daily	2008-05-29 00:32:35.000000000 +0200
@@ -178,11 +178,15 @@
 UnattendedUpgradeInterval=0
 eval $(apt-config shell UnattendedUpgradeInterval APT::Periodic::Unattended-Upgrade)
 
-# check if we actually have to do anything
+# check if we actually have to do anything that requires locking the cache
 if [ $UpdateInterval -eq 0 ] &&
    [ $DownloadUpgradeableInterval -eq 0 ] &&
    [ $UnattendedUpgradeInterval -eq 0 ] &&
    [ $AutocleanInterval -eq 0 ]; then
+
+    # check cache size 
+    check_size_constraints
+
     exit 0
 fi
 
