diff -Nru debian-installer-launcher-21/debian/changelog debian-installer-launcher-21+nmu1/debian/changelog
--- debian-installer-launcher-21/debian/changelog	2015-07-25 17:28:50.000000000 +0100
+++ debian-installer-launcher-21+nmu1/debian/changelog	2015-08-16 14:20:33.000000000 +0100
@@ -1,3 +1,10 @@
+debian-installer-launcher (21+nmu1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Add sudo to Depends. (Closes: #784045)
+
+ -- Iain R. Learmonth <irl@debian.org>  Sun, 16 Aug 2015 14:20:28 +0100
+
 debian-installer-launcher (21) unstable; urgency=medium
 
   [ Updated translations ]
diff -Nru debian-installer-launcher-21/debian/control debian-installer-launcher-21+nmu1/debian/control
--- debian-installer-launcher-21/debian/control	2015-06-20 05:43:14.000000000 +0100
+++ debian-installer-launcher-21+nmu1/debian/control	2015-08-16 14:18:49.000000000 +0100
@@ -14,7 +14,7 @@
 Package: debian-installer-launcher
 Section: utils
 Architecture: all
-Depends: ${misc:Depends}, menu, psmisc, xterm | x-terminal-emulator
+Depends: ${misc:Depends}, menu, psmisc, xterm | x-terminal-emulator, sudo
 Suggests: kexec-tools
 Description: Debian Installer desktop launcher
  This is a desktop launcher for Debian Installer images. It is intended to be
