--- /usr/bin/dh_fixperms	2011-09-12 03:47:55.000000000 +0200
+++ /tmp/dh_fixperms	2011-09-12 12:21:07.000000000 +0200
@@ -105,10 +105,12 @@
 	}
 	
 	# ADA ali files should be mode 444 to avoid recompilation
-	if (-d "$tmp/usr/lib/ada") {
-		complex_doit("find $tmp/usr/lib/ada -type f",
-			"-name '*.ali' $find_options -print0",
-			"2>/dev/null | xargs -0r chmod uga-w");
+        foreach my $ali_dir ("$tmp/usr/lib/ada", "$tmp/usr/lib/`dpkg-architecture -qDEB_HOST_MULTIARCH`/ada") {
+		if (-d "$ali_dir") {
+			complex_doit("find $ali_dir -type f",
+				"-name '*.ali' $find_options -print0",
+				"2>/dev/null | xargs -0r chmod uga-w");
+		}
 	}
 
 	# Lintian overrides should never be executable, too.
