diff --git a/debian/ecryptfs-utils.postinst b/debian/ecryptfs-utils.postinst
index bc830c6..90680b0 100644
--- a/debian/ecryptfs-utils.postinst
+++ b/debian/ecryptfs-utils.postinst
@@ -26,6 +26,8 @@ case "${1}" in
 		;;
 esac
 
+pam-auth-update --package
+
 #DEBHELPER#
 
 exit 0
diff --git a/debian/ecryptfs-utils.prerm b/debian/ecryptfs-utils.prerm
new file mode 100644
index 0000000..80cd5ba
--- /dev/null
+++ b/debian/ecryptfs-utils.prerm
@@ -0,0 +1,9 @@
+#!/bin/sh
+
+set -e
+
+if [ "$1" = remove ]; then
+	pam-auth-update --package --remove ecryptfs-utils
+fi
+
+exit 0
