diff --git a/calypso/webdav.py b/calypso/webdav.py
index 2292841..4f9ef1a 100644
--- a/calypso/webdav.py
+++ b/calypso/webdav.py
@@ -506,10 +506,11 @@ class Collection(object):
                     if ve.contents.has_key('dtstart') and ve.contents.has_key('duration'):
                         del ve.contents['duration']
                     new_ics.vevent_list = [ve]
-                    new_item = Item(new_ics.serialize(), None, path)
+                    # serialize returns a str, Item expects unicode
+                    new_item = Item(new_ics.serialize().decode('utf8'), None, path)
                     self.import_item(new_item, path)
             else:
-                new_item = Item(new_ics.serialize(), None, path)
+                new_item = Item(new_ics.serialize().decode('utf8'), None, path)
                 self.import_item(new_item, path)
             return True
         except Exception, ex:
