From b3659c91f7fd96634ff27ebfd5ee30f09c7a6d0c Mon Sep 17 00:00:00 2001
From: Mattia Rizzolo <mattia@debian.org>
Date: Sat, 24 Feb 2018 00:14:37 +0100
Subject: [PATCH] Change the default ftp upload queue for ftp.debian.org

Signed-off-by: Mattia Rizzolo <mattia@debian.org>
---
 dput.cf | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dput.cf b/dput.cf
index 6cf429f..444d606 100644
--- a/dput.cf
+++ b/dput.cf
@@ -91,7 +91,7 @@ login			= anonymous
 [mentors]
 method			= ftp
 fqdn			= mentors.debian.net
-incoming		= .
+incoming		= /pub/UploadQueue/
 login			= anonymous
 
 [local]
-- 
2.16.1

