diff --git a/debian/changelog b/debian/changelog
index cbdd94ec944e213be8396fe82ad6ad64c70c30b3..7bfaf33b1094440ab9b8ec7aa261cfa4ffc63f77 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,9 @@
+debootstrap (1.0.117) UNRELEASED; urgency=medium
+
+  * Add (Ubuntu) focal as a symlink to gutsy.
+
+ -- Simon Quigley <tsimonq2@debian.org>  Thu, 17 Oct 2019 12:40:56 -0500
+
 debootstrap (1.0.116) unstable; urgency=medium
 
   [ Hideki Yamane ]
diff --git a/scripts/focal b/scripts/focal
new file mode 120000
index 0000000000000000000000000000000000000000..3840936c61bfb4e08050a799ca304939ae635b8a
--- /dev/null
+++ b/scripts/focal
@@ -0,0 +1 @@
+gutsy
\ No newline at end of file
