--- /usr/share/cdbs/1/rules/buildcore.mk	2005-03-10 13:35:06.000000000 +0100
+++ buildcore.mk	2005-04-03 10:07:28.000000000 +0200
@@ -211,6 +211,7 @@
 		-e "s/\[cpu: \([^]]*\)\]/\[\`type-handling \\\\\`echo \1 | tr ' ' ','\\\\\` any\`\]/g" \
 		-e "s/\[system: \([^]]*\)\]/\[\`type-handling any \\\\\`echo \1 | tr ' ' ','\\\\\`\`\]/g" \
 		\
+		-e "s/\"/\\\\\"/g" \
 		-e "s/^/echo \"/g" \
 		-e "s/\\$$/\\\\$$/g" \
 		-e "s/$$/\"/g" \
