From 41bd411b4104982f05d63b7e1738591ba357ab7a Mon Sep 17 00:00:00 2001
From: Bruno Maitre <bruno.maitre@gmail.com>
Date: Fri, 1 Sep 2017 17:25:09 +0200
Subject: [PATCH] dh_configpackage: Fix leading slashes check for the transform
 operation

---
 dh_configpackage | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/dh_configpackage b/dh_configpackage
index ba1b060..ab6f067 100755
--- a/dh_configpackage
+++ b/dh_configpackage
@@ -376,7 +376,7 @@ foreach my $package (@{$dh{DOPACKAGES}}) {
         @transformfiles = filedoublearray($transformfile) if $transformfile;
     }
 
-    foreach my $listref (\@displacefiles, \@hidefiles, \@undisplacefiles, \@unhidefiles, \@transformfiles) {
+    foreach my $listref (\@displacefiles, \@hidefiles, \@undisplacefiles, \@unhidefiles) {
         foreach my $file (@$listref) {
             $file =~ s|^/?|/|;
         }
@@ -392,6 +392,7 @@ foreach my $package (@{$dh{DOPACKAGES}}) {
 
     foreach my $line (@transformfiles) {
         my $file = shift @$line;
+        $file =~ s|^/?|/|;
         my $source;
         my $source_is_local = 0;
         if (@$line[0] =~ /^</) {
-- 
2.11.0

