From 6b706c3ee007d1f43ddadd02d2669929dfc94322 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lo=C3=AFc=20Minier?= <lool@dooz.org>
Date: Fri, 12 Feb 2010 22:28:00 +0100
Subject: [PATCH] valgrind has an ARM port now; add armel

---
 Packages-arch-specific |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Packages-arch-specific b/Packages-arch-specific
index f3d3570..80f28f5 100644
--- a/Packages-arch-specific
+++ b/Packages-arch-specific
@@ -28,7 +28,7 @@ acpid: i386 ia64 amd64 lpia					      # acpi is i386/ia64 specific
 acpidump: i386 ia64 amd64 lpia					      # acpi is i386/ia64 specific
 %acpitail: amd64 i386 ia64 lpia					      # acpi is i386/ia64 specific
 %afbinit: sparc							      # Sparc gfx card firmware loader
-%alleyoop: i386 amd64 powerpc lpia				      # [ANAIS] - Depends valgrind
+%alleyoop: i386 amd64 powerpc lpia armel			      # [ANAIS] - Depends valgrind
 %amd64-libs: i386						      # i386 only
 %apex: arm armeb armel
 %arcboot: mips							      # mips boot loader
@@ -357,7 +357,7 @@ toshutils: i386 amd64                                                 # x86 spec
 %usplash: amd64 armel i386 lpia powerpc sparc			      # ANAIS
 %uswsusp: i386 amd64 powerpc
 v86d: amd64 i386						      # x86 specific
-%valgrind: i386 amd64 powerpc lpia				      # Needs ported (asm, arch knwldge)
+%valgrind: i386 amd64 powerpc lpia armel			      # Needs ported (asm, arch knwldge)
 %vbetool: !hppa !ia64 !m68k !mips !mipsel !powerpc !sh4 !sparc	      # sys/io.h
 %vcs-tree: i386 sparc powerpc mips mipsel alpha amd64		      # [ANAIS]
 %vegastrike: !m68k						      # requested by Mike Furr <mfurr@debian.org>, see 207578
-- 
1.6.6.1

