diff --git configure.ac configure.ac
index 79cc95b..27d22b3 100644
--- configure.ac
+++ configure.ac
@@ -6,5 +6,5 @@ AC_PROG_CC
 AC_HEADER_STDC
 AC_CHECK_HEADERS(paths.h getopt.h)
 
-AC_CONFIG_FILES([Makefile po4a/Makefile po4a/fr/Makefile po4a/sl/Makefile po4a/de/Makefile po4a/es/Makefile])
+AC_CONFIG_FILES([Makefile po4a/Makefile po4a/fr/Makefile po4a/sl/Makefile po4a/de/Makefile po4a/es/Makefile po4a/ja/Makefile])
 AC_OUTPUT
diff --git debian/rules debian/rules
index 6256fd2..e5fa313 100755
--- debian/rules
+++ debian/rules
@@ -90,6 +90,7 @@ endif
 	     debian/tmp/usr/share/man/sl/man[18] \
 	     debian/tmp/usr/share/man/de/man[18] \
 	     debian/tmp/usr/share/man/es/man[18] \
+	     debian/tmp/usr/share/man/ja/man[18] \
              debian/tmp/usr/share/doc/$(package) -type f | xargs gzip -9
 	$(INSTALL_FILE) debian/copyright debian/tmp/usr/share/doc/$(package)
 	$(INSTALL_SCRIPT) debian/postinst debian/tmp/DEBIAN/
diff --git po4a/Makefile.am po4a/Makefile.am
index 5c4c216..792bbcd 100644
--- po4a/Makefile.am
+++ po4a/Makefile.am
@@ -1,6 +1,6 @@
 AUTOMAKE_OPTIONS = foreign
 
-SUBDIRS = fr sl de es
+SUBDIRS = fr sl de es ja
 
 #SUBDIRS = . fr
 
diff --git po4a/ja/Makefile.am po4a/ja/Makefile.am
new file mode 100644
index 0000000..2371a78
--- /dev/null
+++ po4a/ja/Makefile.am
@@ -0,0 +1,8 @@
+AUTOMAKE_OPTIONS=foreign
+
+manjaonedir = $(mandir)/ja/man1
+manjaone_DATA = tempfile.1 which.1
+# readlink.1 is now distributed by coreutils
+
+manjaeightdir = $(mandir)/ja/man8
+manjaeight_DATA = add-shell.8 installkernel.8 remove-shell.8 run-parts.8 savelog.8
diff --git po4a/po/ja.po po4a/po/ja.po
index 57a9d93..4ab2095 100644
--- po4a/po/ja.po
+++ po4a/po/ja.po
@@ -7,7 +7,7 @@
 msgid ""
 msgstr ""
 "Project-Id-Version: debianutils\n"
-"POT-Creation-Date: 2012-07-25 07:06+0900\n"
+"POT-Creation-Date: 2012-07-31 07:50+0900\n"
 "PO-Revision-Date: 2012-07-28 22:56+0900\n"
 "Last-Translator: KURASAWA Nozomu <nabetaro@debian.or.jp>\n"
 "Language-Team: Japanese <debian-japanese@lists.debian.org>\n"
@@ -331,7 +331,6 @@ msgstr "RUN-PARTS"
 #. type: TH
 #: ../run-parts.8:8 ../tempfile.1:2
 #, no-wrap
-#| msgid "7 Jan 2001"
 msgid "27 Jun 2012"
 msgstr "27 Jun 2012"
 
@@ -404,9 +403,6 @@ msgstr ""
 
 #. type: Plain text
 #: ../run-parts.8:46
-#| msgid ""
-#| "Files are run in the lexical sort order of their names unless the --"
-#| "reverse option is given, in which case they are run in the opposite order."
 msgid ""
 "Files are run in the lexical sort order (according to the C/POSIX locale "
 "character collation rules) of their names unless the --reverse option is "
@@ -1184,9 +1180,6 @@ msgstr ""
 
 #. type: Plain text
 #: ../tempfile.1:77
-#| msgid ""
-#| "Exclusive creation is not guaranteed when creating files on NFS "
-#| "partitions.  tempfile is deprecated; you should use mktemp(1)  instead."
 msgid ""
 "Exclusive creation is not guaranteed when creating files on NFS partitions.  "
 "tempfile cannot make temporary directories.  tempfile is deprecated; you "
