From 7ec840919a13b4c70abbf5cce53ef67d1dd30e1c Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Fr=C3=A9d=C3=A9ric=20P=C3=A9ters?= <fpeters@0d.be>
Date: Fri, 18 Mar 2011 11:29:49 +0100
Subject: [PATCH 1/2] do not consider system users returned from consolekit history

---
 src/daemon.c |    5 ++++-
 1 files changed, 4 insertions(+), 1 deletions(-)

diff --git a/src/daemon.c b/src/daemon.c
index 32e9c97..a32276b 100644
--- a/src/daemon.c
+++ b/src/daemon.c
@@ -336,6 +336,7 @@ process_ck_history_line (Daemon      *daemon,
         gchar *username;
         gulong frequency;
         User *user;
+        struct passwd *pw;
 
         frequency = 0;
         username = NULL;
@@ -344,7 +345,9 @@ process_ck_history_line (Daemon      *daemon,
                 return;
         }
 
-        if (user_is_excluded (daemon, username, daemon->priv->minimal_uid)) {
+        pw = getpwnam (username);
+
+        if (user_is_excluded (daemon, username, pw->pw_uid)) {
                 g_debug ("excluding user '%s'", username);
                 g_free (username);
                 return;
-- 
1.7.4.1

