diff -Nru coreutils-8.4/lib/fstatat.c coreutils-8.4/lib/fstatat.c
--- coreutils-8.4/lib/fstatat.c 2010-01-04 16:19:05.000000000 +0000
+++ coreutils-8.4/lib/fstatat.c 2010-03-18 19:21:08.000000000 +0000
@@ -107,4 +107,15 @@
 # undef AT_FUNC_POST_FILE_PARAM_DECLS
 # undef AT_FUNC_POST_FILE_ARGS

+#ifdef __FreeBSD_kernel__
+
+int __fxstatat(int version, int fd, const char *file, struct stat *st, int flag)
+{
+  typedef int (*tp)(int, const char *, struct stat *, int);
+  volatile tp f = fstatat;
+  return f(fd, file, st, flag);
+}
+
+#endif
+
 #endif /* !HAVE_FSTATAT */
