diff -ru avahi-0.6.16.orig/initscript/debian/avahi-daemon.in avahi-0.6.16/initscript/debian/avahi-daemon.in
--- avahi-0.6.16.orig/initscript/debian/avahi-daemon.in	2007-01-03 02:20:25.000000000 +0100
+++ avahi-0.6.16/initscript/debian/avahi-daemon.in	2007-01-03 02:25:46.000000000 +0100
@@ -75,6 +75,16 @@
 	return $1
     }
     
+    log_warning_msg () {
+	if log_use_fancy_output; then
+	    YELLOW=`$TPUT setaf 3`
+	    NORMAL=`$TPUT op`
+	    echo "$YELLOW*$NORMAL $@"
+	else
+	    echo "$@"
+	fi
+    }
+
 fi
 
 #set -e
@@ -93,6 +103,7 @@
 test -f /etc/default/avahi-daemon && . /etc/default/avahi-daemon
 
 if [ "$AVAHI_DAEMON_START" != "1" -a "$1" != "stop" ]; then
+    log_warning_msg "Not starting $DESC $NAME, disabled via /etc/default/$NAME"
     exit 0
 fi
 
diff -ru avahi-0.6.16.orig/initscript/debian/avahi-dnsconfd.in avahi-0.6.16/initscript/debian/avahi-dnsconfd.in
--- avahi-0.6.16.orig/initscript/debian/avahi-dnsconfd.in	2007-01-03 02:20:25.000000000 +0100
+++ avahi-0.6.16/initscript/debian/avahi-dnsconfd.in	2007-01-03 02:26:45.000000000 +0100
@@ -75,6 +75,16 @@
 	return $1
     }
     
+    log_warning_msg () {
+	if log_use_fancy_output; then
+	    YELLOW=`$TPUT setaf 3`
+	    NORMAL=`$TPUT op`
+	    echo "$YELLOW*$NORMAL $@"
+	else
+	    echo "$@"
+	fi
+    }
+
 fi
 
 #set -e
@@ -93,6 +103,7 @@
 test -f /etc/default/avahi-dnsconfd && . /etc/default/avahi-dnsconfd
 
 if [ "$AVAHI_DNSCONFD_START" != "1" -a "$1" != "stop" ]; then
+    log_warning_msg "Not starting $DESC $NAME, disabled via /etc/default/$NAME"
     exit 0
 fi
 
