diff -u aptitude-0.4.11.8/src/pkg_grouppolicy.cc aptitude-0.4.11.8/src/pkg_grouppolicy.cc
--- aptitude-0.4.11.8/src/pkg_grouppolicy.cc
+++ aptitude-0.4.11.8/src/pkg_grouppolicy.cc
@@ -1676,11 +1676,11 @@
 
   void add_package(const pkgCache::PkgIterator &pkg, pkg_subtree *root)
   {
-    string source;
-    if (!pkg.Source())
+    if (pkg.VersionList().end() || pkg.VersionList().FileList().end())
+      return;
+    string source=apt_package_records->Lookup(pkg.VersionList().FileList()).SourcePkg();
+    if (source.length()==0)
       source=pkg.Name();
-    else
-      source=pkg.Source();
 
     childmap::iterator found=children.find(source);
 
