--- di-sources.list	2011-05-10 18:00:32.000000000 -0700
+++ di-sources.list.new	2013-06-28 11:51:36.000000000 -0700
@@ -77,20 +77,34 @@
 
 #Â ############ Some Debian Derivatives ################ #
 
-#Ubuntu Dapper Drake 6.06
-dapper	i386	http://archive.ubuntu.com/ubuntu/dists/dapper/main/installer-i386/current/images/	netboot/netboot.tar.gz
-dapper	amd64	http://archive.ubuntu.com/ubuntu/dists/dapper/main/installer-amd64/current/images/	netboot/netboot.tar.gz
-
-#Ubuntu Hardy Heron 8.04
+## Ubuntu LTS Releases
+# Hardy Heron 8.04
 hardy	i386	http://archive.ubuntu.com/ubuntu/dists/hardy/main/installer-i386/current/images/	netboot/netboot.tar.gz
 hardy	amd64	http://archive.ubuntu.com/ubuntu/dists/hardy/main/installer-amd64/current/images/	netboot/netboot.tar.gz
 
-#Ubuntu Intrepid Ibex 8.10
-intrepid	i386	http://archive.ubuntu.com/ubuntu/dists/intrepid/main/installer-i386/current/images/	netboot/netboot.tar.gz
-intrepid	amd64	http://archive.ubuntu.com/ubuntu/dists/intrepid/main/installer-amd64/current/images/	netboot/netboot.tar.gz
-
-#Ubuntu Jaunty Jackalope 9.04
-jaunty	i386	http://archive.ubuntu.com/ubuntu/dists/jaunty/main/installer-i386/current/images/	netboot/netboot.tar.gz
-jaunty	amd64	http://archive.ubuntu.com/ubuntu/dists/jaunty/main/installer-amd64/current/images/	netboot/netboot.tar.gz
+# Lucid Lynx 10.04
+lucid	i386	http://archive.ubuntu.com/ubuntu/dists/lucid/main/installer-i386/current/images/	netboot/netboot.tar.gz
+lucid	amd64	http://archive.ubuntu.com/ubuntu/dists/lucid/main/installer-amd64/current/images/	netboot/netboot.tar.gz
+
+# Precise Pangolin 12.04
+precise	i386	http://archive.ubuntu.com/ubuntu/dists/precise/main/installer-i386/current/images/	netboot/netboot.tar.gz
+precise	amd64	http://archive.ubuntu.com/ubuntu/dists/precise/main/installer-amd64/current/images/	netboot/netboot.tar.gz
+
+## Ubuntu Normal Releases
+# Saucy Salamander 13.10
+saucy	i386	http://archive.ubuntu.com/ubuntu/dists/saucy/main/installer-i386/current/images/	netboot/netboot.tar.gz
+saucy	amd64	http://archive.ubuntu.com/ubuntu/dists/saucy/main/installer-amd64/current/images/	netboot/netboot.tar.gz
+
+# Raring Ringtail 13.04
+raring	i386	http://archive.ubuntu.com/ubuntu/dists/raring/main/installer-i386/current/images/	netboot/netboot.tar.gz
+raring	amd64	http://archive.ubuntu.com/ubuntu/dists/raring/main/installer-amd64/current/images/	netboot/netboot.tar.gz
+
+# Quantal Quetzal 12.10
+quantal	i386	http://archive.ubuntu.com/ubuntu/dists/quantal/main/installer-i386/current/images/	netboot/netboot.tar.gz
+quantal	amd64	http://archive.ubuntu.com/ubuntu/dists/quantal/main/installer-amd64/current/images/	netboot/netboot.tar.gz
+
+# Oneiric Ocelot 11.10
+oneiric	i386	http://archive.ubuntu.com/ubuntu/dists/oneiric/main/installer-i386/current/images/	netboot/netboot.tar.gz
+oneiric	amd64	http://archive.ubuntu.com/ubuntu/dists/oneiric/main/installer-amd64/current/images/	netboot/netboot.tar.gz
 
 # vim: ft=disources