diff -urN base-files-6.5.original/share/nsswitch.conf base-files-6.5/share/nsswitch.conf
--- base-files-6.5.original/share/nsswitch.conf	2011-12-07 23:56:30.693525167 +0000
+++ base-files-6.5/share/nsswitch.conf	2011-12-07 23:56:50.993778261 +0000
@@ -4,9 +4,9 @@
 # If you have the `glibc-doc-reference' and `info' packages installed, try:
 # `info libc "Name Service Switch"' for information about this file.
 
-passwd:         compat
-group:          compat
-shadow:         compat
+passwd:         files
+group:          files
+shadow:         files
 
 hosts:          files dns
 networks:       files
