From 5bcc05f4b4bae71ad0c98219052b9a9719a2a8fb Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lo=C3=AFc=20Minier?= <lool@dooz.org>
Date: Mon, 11 Jan 2010 23:34:29 +0100
Subject: [PATCH 2/3] Cleanup debian/tmp and debian/bogofilter-sqlite

---
 debian/rules |    3 ++-
 1 files changed, 2 insertions(+), 1 deletions(-)

diff --git a/debian/rules b/debian/rules
index e734e02..7eb4a7d 100755
--- a/debian/rules
+++ b/debian/rules
@@ -58,7 +58,8 @@ clean: checkroot
 		debian/files debian/bogofilter-bdb.substvars \
 		debian/bogofilter-sqlite.substvars
 	rm -rf debian/bogofilter debian/bogofilter-bdb \
-	       debian/bogofilter-common
+	       debian/bogofilter-common debian/bogofilter-sqlite \
+	       debian/tmp
 
 	-cd obj-db && $(MAKE) clean
 	rm -rf obj-db obj-sqlite
-- 
1.6.5

