From 2f94884f6b8bf9d46588906c17be7852b636567b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?David=20Pr=C3=A9vot?= <taffit@debian.org>
Date: Fri, 13 Mar 2015 13:30:37 -0400
Subject: [PATCH] Drop now useless restrictions for Provides

Closes: #761219
---
 policy.sgml | 22 +---------------------
 1 file changed, 1 insertion(+), 21 deletions(-)

diff --git a/policy.sgml b/policy.sgml
index 4adee0b..c6f6677 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -4746,7 +4746,7 @@ fi
 	</p>
 
 	<p>
-	  All of the fields except for <tt>Provides</tt> may restrict
+	  All of the fields may restrict
 	  their applicability to particular versions of each named
 	  package.  This is done in parentheses after each individual
 	  package name; the parentheses should contain a relation from
@@ -5328,26 +5328,6 @@ Provides: bar
 	</p>
 
 	<p>
-	  If a relationship field has a version number attached, only real
-	  packages will be considered to see whether the relationship is
-	  satisfied (or the prohibition violated, for a conflict or
-	  breakage).  In other words, if a version number is specified,
-	  this is a request to ignore all <tt>Provides</tt> for that
-	  package name and consider only real packages.  The package
-	  manager will assume that a package providing that virtual
-	  package is not of the "right" version.  A <tt>Provides</tt>
-	  field may not contain version numbers, and the version number of
-	  the concrete package which provides a particular virtual package
-	  will not be considered when considering a dependency on or
-	  conflict with the virtual package name.<footnote>
-	    It is possible that a future release of <prgn>dpkg</prgn> may
-	    add the ability to specify a version number for each virtual
-	    package it provides.  This feature is not yet present,
-	    however, and is expected to be used only infrequently.
-	  </footnote>
-	</p>
-
-	<p>
 	  To specify which of a set of real packages should be the default
 	  to satisfy a particular dependency on a virtual package, list
 	  the real package as an alternative before the virtual one.
-- 
2.1.4

