--- /etc/bash_completion.d/service	2010-06-16 18:04:40.000000000 +0100
+++ bash_completion.d/service	2010-06-17 12:21:29.000000000 +0100
@@ -32,9 +32,12 @@
     return 0
 } &&
 complete -F _service service
-[ -d /etc/init.d/ ] && complete -F _service -o default \
-    $(for i in /etc/init.d/*; do
-    complete -p ${i##*/} &>/dev/null || printf '%s\n' ${i##*/}; done)
+[ -d /etc/init.d/ ] && (
+    services=$(for i in /etc/init.d/*; do
+        complete -p ${i##*/} &>/dev/null || printf '%s\n' ${i##*/}; done)
+    if [[ "$services" != "" ]]; then
+        echo "[$services]"
+        complete -F _service -o default $services; fi)
 
 # Local variables:
 # mode: shell-script
