=== modified file 'apt-pkg/contrib/configuration.cc'
--- apt-pkg/contrib/configuration.cc	2008-04-02 03:55:09 +0000
+++ apt-pkg/contrib/configuration.cc	2008-10-04 15:42:44 +0000
@@ -505,6 +505,7 @@
    
    int CurLine = 0;
    bool InComment = false;
+   char Buffer[500];
    while (F.eof() == false)
    {
       // The raw input line.
@@ -513,16 +514,12 @@
       std::string Fragment;
 
       // Grab the next line of F and place it in Input.
-      do
-	{
-	  char *Buffer = new char[1024];
-
-	  F.clear();
-	  F.getline(Buffer,sizeof(Buffer) / 2);
-
-	  Input += Buffer;
-	}
-      while (F.fail() && !F.eof());
+      F.clear();
+      F.getline(Buffer,sizeof(Buffer));
+      if (F.fail() && !F.feof())
+        return _error->Error(_("Line %u too long in source list %s."),   
+        			CurLine,FName.c_str());      
+      Input += Buffer;
 
       // Expand tabs in the input line and remove leading and trailing
       // whitespace.

