From d830a757e896aeba13104033b0dd58bfb11c0b40 Mon Sep 17 00:00:00 2001
From: Matt Kraai <kraai@ftbfs.org>
Date: Sun, 4 Sep 2011 14:57:59 -0700
Subject: [PATCH] Use an initrd on mipsel/loongson-2f systems.

---
 debian/changelog |    3 +++
 library.sh       |    4 ++++
 2 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/debian/changelog b/debian/changelog
index bcd3da4..ef1fe93 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -11,6 +11,9 @@ base-installer (1.121) UNRELEASED; urgency=low
   [ Samuel Thibault ]
   * Propose kernels only of the same kind (closes: Bug#637432)
 
+  [ Matt Kraai ]
+  * Use an initrd on mipsel/loongson-2f systems.
+
  -- Colin Watson <cjwatson@ubuntu.com>  Mon, 10 Jan 2011 12:38:41 -0600
 
 base-installer (1.120) unstable; urgency=low
diff --git a/library.sh b/library.sh
index c033eb0..74ea784 100644
--- a/library.sh
+++ b/library.sh
@@ -499,6 +499,10 @@ install_kernel_linux () {
 		do_initrd=yes
 	fi
 
+	if [ `archdetect` = mipsel/loongson-2f ]; then
+	    do_initrd=yes
+	fi
+
 	if db_get base-installer/kernel/linux/link_in_boot ; then
 		if [ "$RET" = "true" ]; then
 			link_in_boot=yes
-- 
1.7.5.4

