diff -ur llvm-defaults-0.28/debian/rules llvm-defaults-0.28-mod/debian/rules
--- llvm-defaults-0.28/debian/rules	2015-10-19 09:33:36.000000000 -0300
+++ llvm-defaults-0.28-mod/debian/rules	2015-10-20 12:30:11.552847123 -0300
@@ -216,6 +216,13 @@
 			/usr/lib/llvm-$(PV_LLVM)/bin/$$bin \
 			/usr/bin/$$bin ; \
 	done
+	for bin in \
+		clang-tidy \
+		; do \
+		dh_link -pclang-tidy \
+			/usr/lib/llvm-$(PV_LLVM)/bin/$$bin \
+			/usr/bin/$$bin ; \
+	done
 	dh_link -pclang \
 			/usr/bin/scan-build-$(PV_LLVM) \
 			/usr/bin/scan-build
