diff -Nru dpkg-1.21.19/src/main/archives.c dpkg-1.21.19/src/main/archives.c
--- dpkg-1.21.19/src/main/archives.c	2023-01-02 02:42:41.000000000 +0100
+++ dpkg-1.21.19/src/main/archives.c	2023-01-24 23:39:50.000000000 +0100
@@ -764,7 +764,7 @@
      * However, it's possible that we were in the middle of some other
      * backup/restore operation and were rudely interrupted.
      * So, we see if we have .dpkg-tmp, and if so we restore it. */
-    if (rename(fnametmpvb.buf,fnamevb.buf)) {
+    if (access(fnametmpvb.buf, F_OK) || rename(fnametmpvb.buf,fnamevb.buf)) {
       if (errno != ENOENT && errno != ENOTDIR)
         ohshite(_("unable to clean up mess surrounding '%.255s' before "
                   "installing another version"), ti->name);
