From 477a66c57d03ad3128614719aa3e7b8884733548 Mon Sep 17 00:00:00 2001
From: Markus Koschany <apo@gambaru.de>
Date: Fri, 21 Mar 2014 12:16:31 +0100
Subject: [PATCH] fix 737829

---
 debian/changelog |  8 ++++++++
 debian/menu      | 10 ++++++----
 2 files changed, 14 insertions(+), 4 deletions(-)

diff --git a/debian/changelog b/debian/changelog
index c84c1c5..ac8e99a 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,11 @@
+berusky (1.4-1.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Update berusky's menu file and use the absolute icon path to display the
+    menu icon. (Closes: #737829)
+
+ -- Markus Koschany <apo@gambaru.de>  Fri, 21 Mar 2014 12:15:42 +0100
+
 berusky (1.4-1) unstable; urgency=low
 
   * New upstream release.
diff --git a/debian/menu b/debian/menu
index ed88f7d..483e155 100644
--- a/debian/menu
+++ b/debian/menu
@@ -1,4 +1,6 @@
-?package(berusky):needs="X11" section="Games/Puzzles"\
-  icon="berusky.xpm"\
-  longtitle="Berusky logic game based on Sokoban"\
-  title="Berusky" command="/usr/games/berusky"
+?package(berusky):needs="X11" \
+  section="Games/Puzzles" \
+  icon="/usr/share/pixmaps/berusky.xpm" \
+  longtitle="Berusky logic game based on Sokoban" \
+  title="Berusky" \
+  command="/usr/games/berusky"
-- 
1.9.1

