diff -u ed-1.10/debian/changelog ed-1.10/debian/changelog
--- ed-1.10/debian/changelog
+++ ed-1.10/debian/changelog
@@ -1,3 +1,11 @@
+ed (1.10-2.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * debian/rules: add build-arch target to ensure patches
+    get applied (Closes: #799702)
+
+ -- Jonathan Wiltshire <jmw@debian.org>  Sat, 20 Feb 2016 12:49:56 +0000
+
 ed (1.10-2) unstable; urgency=medium
 
   * enable DEB_BUILD_MAINT_OPTIONS = hardening=+all
diff -u ed-1.10/debian/rules ed-1.10/debian/rules
--- ed-1.10/debian/rules
+++ ed-1.10/debian/rules
@@ -22,6 +22,9 @@
 build: patch-stamp
 	dh build
 
+build-arch: patch-stamp
+	dh build
+
 clean: unpatch
 	dh clean
 	dh_clean Makefile config.status *.o ed red
