diff --git a/completions/wtf b/completions/wtf
index cc8352f..c354cbe 100644
--- a/completions/wtf
+++ b/completions/wtf
@@ -18,7 +18,7 @@ _wtf()
         fi
         shift
     done
-    [[ -z $db ]] && db=${ACRONYMDB:-/usr/share/misc/acronyms*}
+    [[ -z $db ]] && db=${ACRONYMDB:-/usr/share/games/bsdgames/acronyms*}
 
     COMPREPLY=( $( compgen -W "$( cut -f 1 -s $db ) -f" -- "${cur^^}" ) )
 } &&
