diff -Nur debiandoc-sgml-1.1.88.orig/debian/changelog debiandoc-sgml-1.1.88/debian/changelog
--- debiandoc-sgml-1.1.88.orig/debian/changelog	2005-05-31 00:40:43.000000000 +0200
+++ debiandoc-sgml-1.1.88/debian/changelog	2005-08-08 12:35:19.000000000 +0200
@@ -1,3 +1,10 @@
+debiandoc-sgml (1.1.88.1) unstable; urgency=low
+
+  * Fixed LaTeX code to produce DVI output even if pdfTeX is the engine
+    behind LaTeX, needed to cooperate with teTeX-3.0
+
+ -- Frank Küster <frank@debian.org>  Mon,  8 Aug 2005 12:35:19 +0200
+
 debiandoc-sgml (1.1.88) unstable; urgency=high
 
   * Locale: added Romanian locale (closes: Bug#310885)
diff -Nur debiandoc-sgml-1.1.88.orig/tools/lib/Format/LaTeX.pm debiandoc-sgml-1.1.88/tools/lib/Format/LaTeX.pm
--- debiandoc-sgml-1.1.88.orig/tools/lib/Format/LaTeX.pm	2005-02-13 17:04:24.000000000 +0100
+++ debiandoc-sgml-1.1.88/tools/lib/Format/LaTeX.pm	2005-08-08 12:40:23.583533849 +0200
@@ -122,14 +122,10 @@
     output( "\\usepackage{varioref}\n" );
     output( "\\vrefwarning\n" );
     output( "\n" );
-    output( "\\ifx\\pdfoutput\\undefined\n" );
-    output( "\\usepackage[hypertex,colorlinks=true]{hyperref}\n" );
-    output( "\\else\n" );
-    output( "\\usepackage[pdftex,colorlinks=true" );
+    output( "\\usepackage[colorlinks=true" );
     output( ",$locale{ 'pdfhyperref' }" )
     	if (length( $locale{ 'pdfhyperref' } ));
     output( "]{hyperref}\n" );
-    output( "\\fi\n" );
     output( "\n" );
     output( "\\parindent=0pt\n" );
     output( "\\setlength{\\parskip}{%\n" );
