From ffb5d38beab3775405e400b9371e943b4a415cf6 Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@debian.org>
Date: Sun, 14 Jun 2015 22:35:28 +0100
Subject: [PATCH] debcheckout: Handle Launchpad Git URLs

---
 scripts/debcheckout.pl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/debcheckout.pl b/scripts/debcheckout.pl
index 0cd20a3..1595d62 100755
--- a/scripts/debcheckout.pl
+++ b/scripts/debcheckout.pl
@@ -482,6 +482,7 @@ sub set_auth($$$$) {
 	    } else {
 		$url =~ s|^\w+://(git\.debian\.org)/(?:git/)?(.*)|git+ssh://$user$1/git/$2|;
 	    }
+	    $url =~ s[^\w+://(?:(git|code)\.)?(launchpad\.net/.*)][git+ssh://${user}git.$2];
 	}
 	# "hg ssh://" needs an extra slash so paths are not based in the user's $HOME
 	when ("hg") {
-- 
2.1.4

