diff -pruN debian-reference-1.10/debian/debian-reference-common.desktop debian-reference-1.10/debian/debian-reference-common.desktop
--- debian-reference-1.10/debian/debian-reference-common.desktop	1970-01-01 01:00:00.000000000 +0100
+++ debian-reference-1.10/debian/debian-reference-common.desktop	2006-07-01 22:09:00.000000000 +0100
@@ -0,0 +1,12 @@
+[Desktop Entry]
+Type=Application
+Version=1.0
+Encoding=UTF-8
+Name=Debian Reference
+Comment=Debian Reference Guide
+Name[en_CA]=Debian Reference
+Name[en_GB]=Debian Reference
+Exec=debian-reference
+Icon=
+Terminal=false
+Categories=System;
diff -pruN debian-reference-1.10/debian/rules debian-reference-1.10/debian/rules
--- debian-reference-1.10/debian/rules	2006-07-01 22:09:39.000000000 +0100
+++ debian-reference-1.10/debian/rules	2006-07-01 22:09:00.000000000 +0100
@@ -176,6 +176,10 @@ install: build copyfiles
 	    PUBLISHDIR=$(CURDIR)/debian/debian-reference-common/usr/share/doc/Debian \
 	    publish-examples
 
+	install -d $(CURDIR)/debian/debian-reference-common/usr/share/applications
+	install -m 644 debian/debian-reference-common.desktop \
+	    $(CURDIR)/debian/debian-reference-common/usr/share/applications/
+
 	install -m 755 --preserve-timestamps debian-reference \
 	    $(CURDIR)/debian/debian-reference-common/usr/bin/debian-reference
 
