--- -	2008-09-11 22:31:16.264879195 +0200
+++ /usr/bin/debsign	2008-09-11 22:28:47.000000000 +0200
@@ -491,7 +491,8 @@
               sub checkcommands {
                   chomp($line=$_[0]);
                   if ($line =~ m%^\s*mv(\s+[^\s/]+){2}\s*$%) { return 0; }
-                  if ($line =~ m%^\s*rm(\s+[^\s/]+)+\s*$%) { return 0; }
+                  if ($line =~ m%^\s*mv(\s+DELAYED/[0-9]+-day/[^\s/]+){2}\s*$%) { return 0; }
+                  if ($line =~ m%^\s*rm(\s+(?:DELAYED/[0-9]+-day/)?[^\s/]+)+\s*$%) { return 0; }
                   if ($line eq "") { return 0; }
                   die ".commands file has invalid Commands line: $line\n";
               }
