--- scripts/debsign.sh	2008-06-26 20:16:27.000000000 +0200
+++ -	2008-09-20 18:01:01.301561092 +0200
@@ -490,8 +490,9 @@
                   }
               sub checkcommands {
                   chomp($line=$_[0]);
-                  if ($line =~ m%^\s*mv(\s+[^\s/]+){2}\s*$%) { return 0; }
-                  if ($line =~ m%^\s*rm(\s+[^\s/]+)+\s*$%) { return 0; }
+                  if ($line =~ m%^\s*reschedule\s+[^\s/]+\.changes\s+[0-9]+-day\s*$%) { return 0; }
+                  if ($line =~ m%^\s*cancel\s+[^\s/]+\.changes\s*$%) { return 0; }
+                  if ($line =~ m%^\s*rm(\s+(?:DELAYED/[0-9]+-day/)?[^\s/]+)+\s*$%) { return 0; }
                   if ($line eq "") { return 0; }
                   die ".commands file has invalid Commands line: $line\n";
               }
