diff -Nru debian-reference-1.12/debian/changelog debian-reference-1.12+nmu1/debian/changelog
--- debian-reference-1.12/debian/changelog	2008-01-26 09:51:52.000000000 +0100
+++ debian-reference-1.12+nmu1/debian/changelog	2008-05-21 21:50:29.000000000 +0200
@@ -1,3 +1,10 @@
+debian-reference (1.12+nmu1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Deprecate Freetype 1 (Closes: #481789)
+
+ -- Moritz Muehlenhoff <jmm@debian.org>  Wed, 21 May 2008 21:50:11 +0200
+
 debian-reference (1.12) unstable; urgency=low
 
   * Update Build to the new texlive and ghostscript chain.
diff -Nru debian-reference-1.12/debian/control debian-reference-1.12+nmu1/debian/control
--- debian-reference-1.12/debian/control	2008-01-26 06:59:08.000000000 +0100
+++ debian-reference-1.12+nmu1/debian/control	2008-05-21 21:50:47.000000000 +0200
@@ -3,7 +3,7 @@
 Priority: optional
 Maintainer: Osamu Aoki <osamu@debian.org>
 Build-Depends: debhelper (>> 3.0.0)
-Build-Depends-Indep: debiandoc-sgml (>= 1.1.101), texlive-full, w3m (>= 0.3), latex-cjk-all, freetype1-tools, ghostscript | gs
+Build-Depends-Indep: debiandoc-sgml (>= 1.1.101), texlive-full, w3m (>= 0.3), latex-cjk-all, freetype2-demos, ghostscript | gs
 Standards-Version: 3.7.2
 
 Package: debian-reference
