--- install_packages.orig-3.2.4	2008-05-02 14:38:20.000000000 +0200
+++ install_packages	2008-05-02 15:08:25.000000000 +0200
@@ -274,8 +274,15 @@
 	# use packages on for a list of classes
 	$doit = 0; # assume no class is defined
 	@oclasses = split(/\s+/,$cllist);
-	# if a listed class is defined, add the packaes, otherwise skip these packages
-	foreach (@oclasses) { exists $classisdef{$_} and $doit = 1;}
+	# if all listed class are defined, add the packaes, otherwise skip these packages
+	foreach (@oclasses) {
+	  if (/^!/) {
+	    s/^!//;
+	    not exists $classisdef{$_} and $doit = 1 or $doit = 0 or last;
+	  }
+	  else {
+	    exists $classisdef{$_} and $doit = 1 or $doit = 0 or last;
+	  }
       }
       next;
     }
