--- emacs-29.1+1.orig/lisp/net/rcirc.el
+++ emacs-29.1+1/lisp/net/rcirc.el
@@ -859,6 +859,7 @@ If QUIET is non-nil, no not emit a messa
       (if (rcirc--connection-open-p process)
           (throw 'exit (or quiet (message "Server process is alive")))
         (delete-process process))
+      (setq rcirc-user-authenticated nil)
       (let ((conn-info rcirc-connection-info))
         (setf (nth 5 conn-info)
               (cl-remove-if-not #'rcirc-channel-p
