From 4428399dc197cd57a06a56640496443ff8ad90a3 Mon Sep 17 00:00:00 2001
From: Balint Reczey <balint@balintreczey.hu>
Date: Wed, 27 Jan 2016 04:35:01 +0100
Subject: [PATCH] Fix tests after adding hardened1-linux-<cpu>

---
 scripts/t/Dpkg_Arch.t | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/t/Dpkg_Arch.t b/scripts/t/Dpkg_Arch.t
index b57a0cf..dcfd193 100644
--- a/scripts/t/Dpkg_Arch.t
+++ b/scripts/t/Dpkg_Arch.t
@@ -98,7 +98,7 @@ is(gnutriplet_to_debarch(undef), undef, 'undef gnutriplet');
 is(gnutriplet_to_debarch('unknown-unknown-unknown'), undef, 'unknown gnutriplet');
 is(gnutriplet_to_debarch('x86_64-linux-gnu'), 'amd64', 'known gnutriplet');
 
-is(scalar get_valid_arches(), 417, 'expected amount of known architectures');
+is(scalar get_valid_arches(), 446, 'expected amount of known architectures');
 
 {
     local $ENV{CC} = 'false';
-- 
2.1.4

