diff -Nru broadcom-sta-5.100.82.38/debian/broadcom-sta-source.README.Debian broadcom-sta-5.100.82.38/debian/broadcom-sta-source.README.Debian
--- broadcom-sta-5.100.82.38/debian/broadcom-sta-source.README.Debian	2010-01-15 05:31:00.000000000 +1100
+++ broadcom-sta-5.100.82.38/debian/broadcom-sta-source.README.Debian	2011-03-31 01:29:40.000000000 +1100
@@ -21,7 +21,8 @@
 =======================
 
 Broadcom STA is a binary-only device driver to support the following IEEE
-802.11a/b/g/n wireless network cards: BCM4311-, BCM4312-, BCM4321-, and BCM4322.
+802.11a/b/g/n wireless network cards: BCM4311-, BCM4312-, BCM4313-,
+BCM4321-, BCM4322-, BCM43224-, BCM43225-, BCM43227-, and BCM43228.
 
 You also need the wireless-tools package to get the necessary user-space support
 programs.
diff -Nru broadcom-sta-5.100.82.38/debian/control broadcom-sta-5.100.82.38/debian/control
--- broadcom-sta-5.100.82.38/debian/control	2010-09-23 14:42:22.000000000 +1000
+++ broadcom-sta-5.100.82.38/debian/control	2011-03-31 01:29:07.000000000 +1100
@@ -14,7 +14,8 @@
 Description: Common files for the Broadcom STA Wireless driver
  Broadcom STA is a binary-only device driver to support the following IEEE
  802.11a/b/g/n wireless network cards: BCM4311-, BCM4312-, BCM4313-,
- BCM4321-, and BCM4322-based hardware.
+ BCM4321-, BCM4322-, BCM43224-, BCM43225-, BCM43227-, and BCM43228-based
+ hardware.
  .
  This package contains the common files.
 
@@ -26,7 +27,8 @@
 Description: Source for the Broadcom STA Wireless driver
  Broadcom STA is a binary-only device driver to support the following IEEE
  802.11a/b/g/n wireless network cards: BCM4311-, BCM4312-, BCM4313-,
- BCM4321-, and BCM4322-based hardware.
+ BCM4321-, BCM4322-, BCM43224-, BCM43225-, BCM43227-, and BCM43228-based
+ hardware.
  .
  This package provides the source code for the wl kernel modules. The
  wireless-tools package is also required in order to make use of these modules.
diff -Nru broadcom-sta-5.100.82.38/debian/control.modules.in broadcom-sta-5.100.82.38/debian/control.modules.in
--- broadcom-sta-5.100.82.38/debian/control.modules.in	2010-09-23 14:43:05.000000000 +1000
+++ broadcom-sta-5.100.82.38/debian/control.modules.in	2011-03-31 01:27:38.000000000 +1100
@@ -14,7 +14,8 @@
 Description: broadcom-sta modules for Linux (kernel _KVERS_)
  Broadcom STA is a binary-only device driver to support the following IEEE
  802.11a/b/g/n wireless network cards: BCM4311-, BCM4312-, BCM4313-,
- BCM4321-, and BCM4322-based hardware.
+ BCM4321-, BCM4322-, BCM43224-, BCM43225-, BCM43227-, and BCM43228-based
+ hardware.
  .
  This package contains the set of loadable kernel modules for the broadcom-sta
  wireless network driver. The wireless-tools package is also required in order
