--- evolution-data-server-2.30.3.orig/addressbook/backends/webdav/e-book-backend-webdav.c
+++ evolution-data-server-2.30.3/addressbook/backends/webdav/e-book-backend-webdav.c
@@ -1031,6 +1031,7 @@ e_book_backend_webdav_load_source(EBookB
 	const gchar               *use_ssl;
 	SoupSession               *session;
 	SoupURI                   *suri;
+    guint                     *port;
 
 	uri = e_source_get_uri(source);
 	if (uri == NULL) {
@@ -1063,11 +1064,14 @@ e_book_backend_webdav_load_source(EBookB
 	}
 
 	use_ssl = e_source_get_property (source, "use_ssl");
+    /* soup_uri_set_scheme reset the port if it was specified by the user. */
+    port = suri->port;
 	if (use_ssl != NULL && strcmp (use_ssl, "1") == 0) {
 		soup_uri_set_scheme (suri, "https");
 	} else {
-		soup_uri_set_scheme (suri, "https");
+		soup_uri_set_scheme (suri, "http");
 	}
+    soup_uri_set_port(suri, port);
 
 	/* append slash if missing */
 	if (!suri->path || !*suri->path || suri->path[strlen (suri->path) - 1] != '/') {
