diff -Nur djvulibre-3.5.16.orig/debian/changelog djvulibre-3.5.16/debian/changelog
--- djvulibre-3.5.16.orig/debian/changelog	2005-12-17 16:34:29.000000000 +0100
+++ djvulibre-3.5.16/debian/changelog	2005-12-17 16:38:36.000000000 +0100
@@ -1,3 +1,10 @@
+djvulibre (3.5.16-1ubuntu1) dapper; urgency=low
+
+  * debian/control:
+    - fix the -dev package to Depends on the lib itself and libjpeg62-dev
+
+ -- Sebastien Bacher <seb128@canonical.com>  Sat, 17 Dec 2005 16:36:59 +0100
+
 djvulibre (3.5.16-1) unstable; urgency=low
 
   * New upstream release
diff -Nur djvulibre-3.5.16.orig/debian/control djvulibre-3.5.16/debian/control
--- djvulibre-3.5.16.orig/debian/control	2005-12-17 16:34:29.000000000 +0100
+++ djvulibre-3.5.16/debian/control	2005-12-17 16:36:20.000000000 +0100
@@ -8,7 +8,7 @@
 Package: libdjvulibre-dev
 Section: devel
 Architecture: any
-Depends: ${shlibs:Depends}
+Depends: libdjvulibre15 (= ${Source-Version}), libjpeg62-dev
 Description: Development files for the DjVu image format
  DjVu image format static library and development files.
  .
