--- a/debian/copyright
+++ b/debian/copyright
@@ -1,9 +1,33 @@
-This package was debianized by Tollef Fog Heen <tfheen@debian.org> on
-Thu, 14 Feb 2002 04:03:22 +0100.
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: chrpath
+Upstream-Contact: Petter Reinholdtsen <pere@hungry.com>
+Source: https://alioth.debian.org/projects/chrpath/
 
-It was downloaded from https://alioth.debian.org/projects/chrpath/
-
-Upstream Author: Petter Reinholdtsen <pere@hungry.com>
-
-chrpath is under the GNU Public License, which can normally
-be found in /usr/share/common-licenses/GPL on Debian systems.
+Files: *
+Copyright: 2001-2014 chrpath authors
+           Geoffrey Keating <geoffk@ozemail.com.au>
+           Peeter Joot <peeterj@ca.ibm.com>
+           Petter Reinholdtsen <pere@hungry.com>
+           Javier Serrano Polo <jasp00@terra.es>
+           Darren Salt <linux@youmustbejoking.demon.co.uk>
+           David Hull <hull@paracel.com>
+           Bradford W. Johnson <bradford@math.umn.edu>
+           Thomas Anders <anders@hmi.de>
+           Tollef Fog Heen <tfheen@debian.org>
+           Eric S. Raymond <esr@thyrsus.com>
+License: GPL-3+
+ This program is free software: you can redistribute it and/or modify
+ it under the terms of the GNU General Public License as published by
+ the Free Software Foundation, either version 3 of the License, or (at
+ your option) any later version.
+ .
+ This program is distributed in the hope that it will be useful, but
+ WITHOUT ANY WARRANTY; without even the implied warranty of
+ MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ General Public License for more details.
+ .
+ You should have received a copy of the GNU General Public License
+ along with this program. If not, see <http://www.gnu.org/licenses/>.
+ .
+ On Debian systems, the complete text of the GNU General
+ Public License version 3 can be found in "/usr/share/common-licenses/GPL-3".
--- /dev/null
+++ b/debian/docs
@@ -0,0 +1 @@
+AUTHORS
--- /dev/null
+++ b/debian/install
@@ -0,0 +1 @@
+chrpath   /usr/bin
--- /dev/null
+++ b/debian/manpages
@@ -0,0 +1 @@
+chrpath.1
--- a/debian/rules
+++ b/debian/rules
@@ -3,8 +3,8 @@
 %:
 	dh $@ --with autotools_dev
 
+override_dh_installchangelogs:
+	dh_installchangelogs NEWS
+
 override_dh_auto_install:
-	dh_auto_install -- docdir="/usr/share/doc/chrpath"
-	rm debian/chrpath/usr/share/doc/chrpath/COPYING
-	rm debian/chrpath/usr/share/doc/chrpath/ChangeLog
-	rm debian/chrpath/usr/share/doc/chrpath/INSTALL
+
--- /dev/null
+++ b/debian/source/format
@@ -0,0 +1 @@
+3.0 (quilt)
--- a/debian/dirs
+++ /dev/null
@@ -1,2 +0,0 @@
-usr/bin
-usr/share/doc
