--- alsa-oss-1.0.12/debian/control.orig	2007-01-13 16:12:20.000000000 +0100
+++ alsa-oss-1.0.12/debian/control	2007-01-13 16:13:00.000000000 +0100
@@ -7,7 +7,7 @@
 Standards-Version: 3.7.2
 
 Package: alsa-oss
-Architecture: i386 ia64 alpha amd64 arm armeb hppa m32r m68k mips mipsel powerpc ppc64 s390 s390x sh3 sh3eb sh4 sh4eb sparc
+Architecture: i386 ia64 alpha amd64 arm armeb armel hppa m32r m68k mips mipsel powerpc ppc64 s390 s390x sh3 sh3eb sh4 sh4eb sparc
 Depends: ${shlibs:Depends}
 Description: ALSA wrapper for OSS applications
  This package contains a program loader, aoss, which wraps
