diff -ur debhelper-pristine/dh_shlibdeps debhelper-9.20120909/dh_shlibdeps
--- debhelper-pristine/dh_shlibdeps	2013-02-09 15:15:56.406329132 -0500
+++ debhelper-9.20120909/dh_shlibdeps	2013-02-09 15:16:57.252994702 -0500
@@ -129,7 +129,7 @@
 	if (defined($dh{EXCLUDE_FIND}) && $dh{EXCLUDE_FIND} ne '') {
 		$find_options="! \\( $dh{EXCLUDE_FIND} \\)";
 	}
-	foreach my $file (split(/\n/,`find $tmp -type f \\( -perm +111 -or -name "*.so*" -or -name "*.cmxs" \\) $find_options -print`)) {
+	foreach my $file (split(/\n/,`find $tmp -type f \\( -perm -111 -or -name "*.so*" -or -name "*.cmxs" \\) $find_options -print`)) {
 		# Prune directories that contain separated debug symbols.
 		next if $file=~m!^\Q$tmp\E/usr/lib/debug/(lib|lib64|usr|bin|sbin|opt|dev|emul)/!;
 		# TODO this is slow, optimize. Ie, file can run once on
