Index: cdebconf/src/modules/frontend/gtk/gtk.c
===================================================================
--- cdebconf/src/modules/frontend/gtk/gtk.c	(revisione 47245)
+++ cdebconf/src/modules/frontend/gtk/gtk.c	(copia locale)
@@ -49,13 +49,7 @@
 #include <debian-installer/slist.h>
 #include <gdk/gdkkeysyms.h>
 
-#if GTK_CHECK_VERSION(2,10,0)
-#ifdef GDK_WINDOWING_DIRECTFB
 #include <directfb.h>
-#endif
-#else
-#include <directfb.h>
-#endif
 
 #define WINDOW_WIDTH 800
 #define WINDOW_HEIGHT 600
@@ -1493,14 +1487,7 @@
      * for dfb to support automatic keymap change detection and reloading
      * (See also bug #381979)
      */
-
-    #if GTK_CHECK_VERSION(2,10,0)
-    #ifdef GDK_WINDOWING_DIRECTFB
     dfb_input_device_reload_keymap( dfb_input_device_at( DIDID_KEYBOARD ) );
-    #endif
-    #else
-    dfb_input_device_reload_keymap( dfb_input_device_at( DIDID_KEYBOARD ) );
-    #endif
 
     gtk_rc_reparse_all();
 
