--- a/debian/rules	2012-01-16 17:43:09.000000000 +0100
+++ b/debian/rules	2012-01-18 10:18:18.000000000 +0100
@@ -35,6 +35,7 @@
 override_dh_auto_clean:
 	cp debian/patches/series.in debian/patches/series
 	[ "$(DEB_HOST_ARCH_OS)" = "hurd" ] && echo "648153-fix_FTBFS4Hurd.patch" >>debian/patches/series || return 0
+	[ "$(DEB_HOST_ARCH_OS)" = "hurd" ] && dh_quilt_patch || return 0
 	dh_auto_clean
 
 #: configure options reference: http://www.abisource.com/wiki/Compiling_AbiWord
