--- ./scripts/debuild.pl	2010-01-02 21:20:42.000000000 -0600
+++ /usr/local/bin/debuild-hacked	2010-01-10 03:25:30.000000000 -0600
@@ -276,7 +276,7 @@
     foreach my $var ($dpkg_opts_var, $lintian_opts_var) {
 	$shell_cmd .= "eval set -- \$$var;\n";
 	$shell_cmd .= "echo \">>> $var BEGIN <<<\";\n";
-	$shell_cmd .= 'while [ $# -gt 0 ]; do printf "%s\n" $1; shift; done;' . "\n";
+	$shell_cmd .= 'while [ $# -gt 0 ]; do echo $1; shift; done;' . "\n";
 	$shell_cmd .= "echo \">>> $var END <<<\";\n";
     }
     # Not totally efficient, but never mind
