--- ddt.cgi	2017-04-15 18:11:39.932861451 +0200
+++ ddt.cgi-new	2017-04-15 18:13:45.586184068 +0200
@@ -135,4 +135,5 @@
     'alltranslatedpackages' => $LANGUAGE_NAME, 
     'alluntranslatedpackages' => $LANGUAGE_NAME, 
+    'something' => $LANGUAGE_NAME, 
     'package' => $PACKAGE_NAME, 
     'diff1' => qr/^\d+$/, 
@@ -834,4 +835,30 @@
 	print "<br>\n";
 	print "</BODY>\n";	
+} elsif (param('something')) {
+
+	my $language=param('something');
+
+	my $package;
+	my $description_id;
+
+	print "Content-type: text/html; charset=UTF-8\n";	
+	print "\n";	
+	print "<HTML>\n";	
+	print "<HEAD>\n";	
+	print "<meta http-equiv=\"content-type\" content=\"text/html;charset=UTF-8\">";
+	print "<TITLE>Debian Description Tracking  -- something -- </TITLE>\n";	
+	print "</HEAD>\n";	
+	print "<BODY>\n";	
+	print "<h3>Something</h3>\n";	
+
+	my $sth = $dbh->prepare("SELECT description_id,package FROM description_tb WHERE description_id in (SELECT description_id FROM active_tb) and description_id not in (SELECT description_id FROM translation_tb WHERE description_id in (SELECT description_id FROM active_tb) and language=?) and package in (SELECT package FROM description_tb WHERE description_id in (SELECT description_id FROM translation_tb WHERE language=?) GROUP BY package) ORDER BY package");
+	$sth->execute($language,$language);
+
+	print "<pre>";
+	while(($description_id,$package) = $sth->fetchrow_array) {
+		print "<a href=\"ddt.cgi?desc_id=$description_id&getuntrans=$language\">$package</a>";
+		print "\n";
+	}
+	print "</pre>";		
 } else {
 	print "Content-type: text/html; charset=UTF-8\n";	
