--- evolution-data-server-1.10.1/src/server.c.orig	2007-05-05 22:41:08.889680750 +0200
+++ evolution-data-server-1.10.1/src/server.c	2007-05-05 22:44:05.768735000 +0200
@@ -369,7 +369,7 @@
 		else if (!did_cals)
 			failed = "CALS";
 
-		g_error (G_STRLOC ": could not initialize Server service \"%s\"; terminating", failed);
+		g_warning (G_STRLOC ": could not initialize Server service \"%s\"; terminating", failed);
 
 		if (e_data_book_factory) {
 			bonobo_object_unref (BONOBO_OBJECT (e_data_book_factory));
