--- debmirror	2024-09-09 13:13:06.335485914 -0600
+++ debmirror.revised	2024-09-09 13:15:21.742485255 -0600
@@ -3301,7 +3301,7 @@
   my($routine, $routine_args, $operational_params) = @_;
   my @sects = ((map {"/$_"} @sections), "");
   foreach my $dist (keys %distset) {
-    next if $dist=~/\bexperimental\b|-proposed-updates\b/o;
+    next if $dist=~/\bexperimental\bproposed-updates\b/o;
     next unless exists $distset{$dist}{mirror};
     foreach my $arch (@arches, "all") {
       my %op_params = %$operational_params;
