Index: chromium-81.0.4044.92/third_party/widevine/cdm/widevine.gni
===================================================================
--- chromium-81.0.4044.92.orig/third_party/widevine/cdm/widevine.gni
+++ chromium-81.0.4044.92/third_party/widevine/cdm/widevine.gni
@@ -10,7 +10,8 @@ declare_args() {
   # on Android and Fuchsia platforms.
   # Can be optionally enabled in Chromium on non-Android platforms. Please see
   # //src/third_party/widevine/LICENSE file for details.
-  enable_widevine = is_chrome_branded || is_android || is_fuchsia
+  #enable_widevine = is_chrome_branded || is_android || is_fuchsia
+  enable_widevine = 1
 }
 
 # Widevine CDM is available as a library CDM on the following platforms and
