diff --git a/Debian/Debhelper/Buildsystem/perl_build.pm b/Debian/Debhelper/Buildsystem/perl_build.pm
index 671bca5..3ccefea 100644
--- a/Debian/Debhelper/Buildsystem/perl_build.pm
+++ b/Debian/Debhelper/Buildsystem/perl_build.pm
@@ -58,7 +58,7 @@ sub build {
 
 sub test {
 	my $this=shift;
-	$this->do_perl("Build", "test", @_);
+	$this->do_perl("Build", "test", "--verbose", 1, @_);
 }
 
 sub install {
diff --git a/Debian/Debhelper/Buildsystem/perl_makemaker.pm b/Debian/Debhelper/Buildsystem/perl_makemaker.pm
index 60cda3d..cbf3061 100644
--- a/Debian/Debhelper/Buildsystem/perl_makemaker.pm
+++ b/Debian/Debhelper/Buildsystem/perl_makemaker.pm
@@ -61,6 +61,12 @@ sub configure {
 		@flags, @_);
 }
 
+sub test {
+	my $this=shift;
+	# Make tests verbose
+	$this->SUPER::test("TEST_VERBOSE=1", @_);
+}
+
 sub install {
 	my $this=shift;
 	my $destdir=shift;
