--- scripts/Dpkg/Changelog.pm	2009-09-18 13:39:36 +0000
+++ scripts/Dpkg/Changelog.pm	2010-02-12 21:28:36 +0000
@@ -720,12 +720,14 @@
 sub find_closes {
     my $changes = shift;
     my @closes = ();
+    my %seen = ();
 
     while ($changes &&
 	   ($changes =~ /closes:\s*(?:bug)?\#?\s?\d+(?:,\s*(?:bug)?\#?\s?\d+)*/ig)) {
 	push(@closes, $& =~ /\#?\s?(\d+)/g);
     }
 
+    @closes = grep { ! $seen{$_} ++ } @closes;
     @closes = sort { $a <=> $b } @closes;
     return \@closes;
 }

--- scripts/Dpkg/Vendor/Ubuntu.pm	2009-05-21 06:45:45 +0000
+++ scripts/Dpkg/Vendor/Ubuntu.pm	2010-02-12 21:28:36 +0000
@@ -92,12 +92,14 @@
 sub find_launchpad_closes {
     my ($changes) = @_;
     my @closes = ();
+    my %seen = ();
 
     while ($changes &&
           ($changes =~ /lp:\s+\#\d+(?:,\s*\#\d+)*/ig)) {
        push(@closes, $& =~ /\#?\s?(\d+)/g);
     }
 
+    @closes = grep { ! $seen{$_} ++ } @closes;
     @closes = sort { $a <=> $b } @closes;
 
     return \@closes;
