--- scripts/Dpkg/Vendor/Ubuntu.pm	2009-05-21 06:45:45 +0000
+++ scripts/Dpkg/Vendor/Ubuntu.pm	2010-02-12 22:27:15 +0000
@@ -91,14 +91,14 @@
 
 sub find_launchpad_closes {
     my ($changes) = @_;
-    my @closes = ();
+    my %closes;
 
     while ($changes &&
           ($changes =~ /lp:\s+\#\d+(?:,\s*\#\d+)*/ig)) {
-       push(@closes, $& =~ /\#?\s?(\d+)/g);
+        $closes{$_} = 1 foreach($& =~ /\#?\s?(\d+)/g);
     }
 
-    @closes = sort { $a <=> $b } @closes;
+    my @closes = sort { $a <=> $b } keys %closes;
 
     return \@closes;
 }

