Description: Fix for deprecation warning when used with python 2.6
Bug-Debian: http://bugs.debian.org/517831
Bug-Ubuntu: https://bugs.launchpad.net/bugs/336601
Author: Michael Bienia <michael@vorlon.ping.de>
Last-Update: 2010-06-26

--- a/dput
+++ b/dput
@@ -19,7 +19,8 @@
 # Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 
 import os, sys, string, re, glob, signal
-import pwd, stat, md5, sha, rfc822
+import pwd, stat, rfc822
+from hashlib import md5, sha1
 
 # Now import our modules
 import ConfigParser
@@ -171,9 +172,9 @@
     if hash == 'md5':
         hash_type = md5
     else:
-        hash_type = sha
+        hash_type = sha1
 
-    check_obj = hash_type.new()
+    check_obj = hash_type()
 
     while 1:
         data = file_to_test.read(65536)
