ï»¿--- dh_fixperms~
+++ dh_fixperms
@@ -105,6 +105,22 @@
 			"-name '*.ali' $find_options -print0",
 			"2>/dev/null | xargs -0r chmod uga-w");
 	}
+
+	# Bug control files should not be executable (except for script)
+	if (-d "$tmp/usr/share/bug") {
+		complex_doit("find $tmp/usr/share/bug -type f",
+			"! -name 'script' $find_options -print0",
+			"2>/dev/null | xargs -0r chmod 644");
+		complex_doit("find $tmp/usr/share/bug -type f",
+			"-name 'script' $find_options -print0",
+			"2>/dev/null | xargs -0r chmod 755");
+	}
+
+	# Lintian overrides should never be executable, too.
+	if (-d "$tmp/usr/share/lintian") {
+		complex_doit("find $tmp/usr/share/lintian -type f $find_options -print0",
+			"2>/dev/null | xargs -0r chmod 644");
+	}
 }
 
 =head1 SEE ALSO
