From 5cb0a7ca277527a51c3449d1c61b232adc08058c Mon Sep 17 00:00:00 2001
From: Simon McVittie <smcv@debian.org>
Date: Fri, 7 May 2021 09:39:04 +0100
Subject: [PATCH] d/rules: Build udeb with debug

---
 debian/rules | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/debian/rules b/debian/rules
index 428e69767c..fe4cb92185 100755
--- a/debian/rules
+++ b/debian/rules
@@ -119,6 +119,8 @@ shared_configure_flags := $(configure_flags) \
 			--enable-introspection \
 			--enable-man
 shared_udeb_configure_flags := $(configure_flags) \
+			CPPFLAGS="$(CPPFLAGS) -DDEBIAN_INSTALLER" \
+			--enable-debug \
 			--disable-introspection \
 			--disable-xcomposite \
 			--disable-xdamage \
-- 
2.31.1

