--- /usr/lib/ruby/1.8/dhelp.rb	2007-12-06 12:06:00.000000000 -0500
+++ /tmp/1.8/dhelp.rb	2008-01-20 01:56:56.000000000 -0500
@@ -127,7 +127,7 @@
     end
 
     def files
-      @files.strip.split(/\s+/)
+      @files.to_s.strip.split(/\s+/)
     end
   end
 
@@ -384,6 +384,7 @@
       @pool.each do |doc|
         doc.formats.each do |format|
           next unless SUPPORTED_FORMATS.include? format.format.downcase
+	  puts "processing files for #{doc.document}'s format #{format.format}"
           format.files.each do |glob|
             # Dirnames can be globs too. There will also be duplicates
             # (different formats, same doc), hence the set



