From: Martin Dickopp <martin@zero-based.org>
Date: Sat, 30 Apr 2016 17:45:53 +0200
Subject: Link with OpenVDB library

---
 debian/control | 3 +++
 debian/rules   | 1 +
 2 files changed, 4 insertions(+)

diff --git a/debian/control b/debian/control
index 3f6d122..414a41d 100644
--- a/debian/control
+++ b/debian/control
@@ -14,6 +14,7 @@ Build-Depends: autotools-dev,
                libavfilter-dev,
                libboost-dev,
                libboost-filesystem-dev,
+               libboost-iostreams-dev,
                libboost-locale-dev,
                libboost-regex-dev,
                libboost-system-dev,
@@ -34,11 +35,13 @@ Build-Depends: autotools-dev,
                libopenexr-dev (>= 2.2.0),
                libopenimageio-dev,
                libopenjpeg-dev,
+               libopenvdb-dev,
                libpng-dev,
                libsdl-dev,
                libsndfile-dev,
                libspnav-dev,
                libswscale-dev,
+               libtbb-dev,
                libtiff-dev,
                libvorbis-dev,
                libxi-dev,
diff --git a/debian/rules b/debian/rules
index 18b64ba..4bcfdc4 100755
--- a/debian/rules
+++ b/debian/rules
@@ -37,6 +37,7 @@ override_dh_auto_configure:
 		-DWITH_JACK=ON \
 		-DWITH_MOD_OCEANSIM=ON \
 		-DWITH_OPENCOLORIO=ON \
+		-DWITH_OPENVDB=ON \
 		-DWITH_PLAYER=ON \
 		-DWITH_PYTHON_INSTALL=OFF \
 		-DWITH_SYSTEM_OPENJPEG=ON
