--- /usr/sbin/install-keymap~	2009-10-25 14:20:56.000000000 +0100
+++ /usr/sbin/install-keymap	2009-10-25 14:22:16.000000000 +0100
@@ -129,7 +129,7 @@ then
     backup
     rm -f ${CONFFILE}
 else
-    if [ "`dpkg --print-installation-architecture`" = "arm" ] && 
+    if [ "`dpkg --print-architecture`" = "arm" ] && 
 	grep -i '^Hardware.*: \(Acorn-\|RiscStation-\)' /proc/cpuinfo >/dev/null &&
 	find /usr/share/keymaps/i386/ -type f -name ${keymap} -o \
 		    -name ${keymap}.gz -o -name ${keymap}.kmap.gz
