--- ui.cc.orig	2009-11-19 08:24:24.000000000 +0100
+++ ui.cc	2009-11-19 08:27:25.000000000 +0100
@@ -491,9 +491,10 @@
       if(protocol == "su")
 	{
 	  std::ostringstream cmdbuf;
-	  cmdbuf << argv0 << "--no-gui -S "
-		 << statusname.get_name() << " "
-		 << args;
+	  cmdbuf << argv0
+	         << " --no-gui"
+	         << " -S " << statusname.get_name()
+	         << " " << args;
 	  execl(root_program.c_str(), root_program.c_str(), "-c", cmdbuf.str().c_str(), NULL);
 
 	  exit(1);
