From: Yuriy M. Kaminskiy <yumkam+debian@gmail.com>
Subject: [oneline] fix data:..;base64 protocol decoding

Index: elinks-0.12pre6/src/protocol/data.c
===================================================================
--- elinks-0.12pre6.orig/src/protocol/data.c
+++ elinks-0.12pre6/src/protocol/data.c
@@ -141,7 +141,7 @@ data_protocol_handler(struct connection
 	}
 
 	if (base64) {
-		unsigned char *decoded = base64_encode(data);
+		unsigned char *decoded = base64_decode(data);
 
 		if (!decoded) {
 			abort_connection(conn, connection_state(S_OUT_OF_MEM));

