--- cmdline_do_action.cc	2007-07-03 16:08:21.000000000 +0200
+++ cmdline_do_action_tmp.cc	2007-07-11 16:12:24.000000000 +0200
@@ -49,6 +49,10 @@
   else if(!strcasecmp(argv[0], "full-upgrade") ||
 	  !strcasecmp(argv[0], "dist-upgrade"))
     {
+      if(argc != 1){
+          fprintf(stderr, _("E: The %s command takes no arguments\n"), argv[0]);
+          return -1;
+      }
       default_action=cmdline_install;
       dist_upgrade=true;
     }
