 debian/control |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/debian/control b/debian/control
index 34fd05e..4fcb6cb 100644
--- a/debian/control
+++ b/debian/control
@@ -15,7 +15,7 @@ Depends: ${perl:Depends}, ${misc:Depends}, gcc, binutils
 Description: tool to compare ABI compatibility of shared C/C++ library versions
  abi-compliance-checker can be used to help reduce the possibility of
  an application crashing when the shared libraries it links against
- have changed.  Signatures and data type definations from two separate
+ have changed.  Signatures and data type definitions from two separate
  versions of a library are compared by examining the shared objects
  (.so) files themselves, and by analysing the header files (.h)
  provided for the two versions of the library and their dependencies.
-- 
tg: (f099df1..) t/debian/spelling (depends on: master)
