diff --git a/debian/changelog b/debian/changelog
index 896040d..c64b4c0 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -2,8 +2,9 @@ elinks (0.11.3-7) unstable; urgency=low
 
   * Use '<<' versioning rather than '<=' for Conflicts/Replaces.  Thanks Colin
     Watson <cjwatson@ubuntu.com> (Closes: #475520)
+  * Enable the FSP file transfer protocol (Closes: #478007)
 
- -- Y Giridhar Appaji Nag <giridhar@appaji.net>  Fri, 11 Apr 2008 17:11:14 +0530
+ -- Y Giridhar Appaji Nag <giridhar@appaji.net>  Sat, 26 Apr 2008 15:33:22 +0530
 
 elinks (0.11.3-6) unstable; urgency=low
 
diff --git a/debian/control b/debian/control
index 60c488d..edd5031 100644
--- a/debian/control
+++ b/debian/control
@@ -3,7 +3,7 @@ Section: web
 Priority: optional
 Maintainer: Y Giridhar Appaji Nag <giridhar@appaji.net>
 Uploaders: Moritz Muehlenhoff <jmm@debian.org>
-Build-Depends: debhelper (>= 5), libgnutls-dev, libidn11-dev, libgpmg1-dev [!hurd-i386 !kfreebsd-i386 !kfreebsd-amd64], liblua50-dev, liblualib50-dev, libbz2-dev, libexpat1-dev (>= 1.95.6), libperl-dev, ruby, ruby1.8-dev, autotools-dev, gettext, asciidoc, xmlto, docbook-utils, perl
+Build-Depends: debhelper (>= 5), libgnutls-dev, libidn11-dev, libfsplib-dev, libgpmg1-dev [!hurd-i386 !kfreebsd-i386 !kfreebsd-amd64], liblua50-dev, liblualib50-dev, libbz2-dev, libexpat1-dev (>= 1.95.6), libperl-dev, ruby, ruby1.8-dev, autotools-dev, gettext, asciidoc, xmlto, docbook-utils, perl
 Standards-Version: 3.7.3
 Homepage: http://elinks.cz/
 Vcs-Git: git://git.debian.org/git/collab-maint/elinks.git
diff --git a/debian/rules b/debian/rules
index a075573..b8359d0 100755
--- a/debian/rules
+++ b/debian/rules
@@ -48,6 +48,7 @@ confopts_main = $(confopts) \
 	--enable-leds \
 	--enable-html-highlight \
 	--enable-cgi \
+	--enable-fsp \
 	--disable-smb \
 	--with-perl \
 	--with-ruby \
@@ -75,7 +76,7 @@ confopts_lite = $(confopts) \
 	--without-zlib \
 	--without-bzlib \
 	--without-idn \
-	--without-lua \
+	--without-lua
 
 save-upstream: patch save-stamp
 patch: patch-stamp
