From 1e7e78b3671c7fd3e8ae6135699721b9aae5ff82 Mon Sep 17 00:00:00 2001
From: intrigeri <intrigeri@debian.org>
Date: Wed, 10 Jan 2018 09:30:46 +0000
Subject: [PATCH 3/3] AppArmor: document in README.Debian how to fully disable
 AppArmor confinement. (Closes: #884707)

---
 debian/README.Debian | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/debian/README.Debian b/debian/README.Debian
index 97eb499..9b1b2cb 100644
--- a/debian/README.Debian
+++ b/debian/README.Debian
@@ -282,5 +282,14 @@ APPARMOR PROFILES
   /etc/clamav/onupdateexecute.d, or /etc/clamav/virusevent.d directories,
   appropriate rules need to be added to the AppArmor profile.
 
+  If you prefer to fully disable AppArmor confinement for
+  clamav-daemon or freshclam, run respectively:
+
+    aa-disable /usr/sbin/clamd
+
+  or:
+
+    aa-disable /usr/bin/freshclam
+
   Please see https://wiki.debian.org/AppArmor for information and
   documentation on modifying AppArmor profiles.
-- 
2.15.1

