diff -Naur ctwm-3.7-orig/workmgr.c ctwm-3.7/workmgr.c
--- ctwm-3.7-orig/workmgr.c	2010-06-13 09:34:50.000000000 +0200
+++ ctwm-3.7/workmgr.c	2010-06-13 09:36:47.000000000 +0200
@@ -2735,7 +2735,7 @@
 			    winX = Scr->BorderLeft;
 			    newX = msw->x + XW + Scr->BorderLeft * mw->wwidth / vs->w;
 			}
-			if (((winX + w) > vs->x - Scr->BorderRight) &&
+			if (((winX + w) > vs->w - Scr->BorderRight) &&
 			    ((Scr->MoveOffResistance < 0) ||
 			     ((winX + w) < vs->w - Scr->BorderRight + Scr->MoveOffResistance))) {
 			    winX = vs->w - Scr->BorderRight - w;
