From f8f65cca2484fe35e2a80288180e22e22122ed8f Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Wed, 29 Aug 2012 13:59:14 +0800
Subject: [PATCH 01/11] Add wheezy freeze date

---
 data/releases/freezes |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/data/releases/freezes b/data/releases/freezes
index bf994f6..1bd29fe 100644
--- a/data/releases/freezes
+++ b/data/releases/freezes
@@ -37,3 +37,7 @@ Source: http://lists.debian.org/debian-devel-announce/2008/07/msg00007.html
 Title: Debian 6.0 ("squeeze") frozen
 Date: August 6 2010
 Source: http://lists.debian.org/debian-devel-announce/2010/08/msg00000.html
+
+Title: Debian 7.0 ("wheezy") frozen
+Date: June 30 2012
+Source: http://lists.debian.org/debian-devel-announce/2012/06/msg00009.html
-- 
1.7.10.4

