From 3ac6907a9724796eae9e01b90ea059fd0d462963 Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Wed, 29 Aug 2012 14:06:28 +0800
Subject: [PATCH 02/11] Add point releases since March 2011

---
 data/releases/point_releases |   24 ++++++++++++++++++++++++
 1 file changed, 24 insertions(+)

diff --git a/data/releases/point_releases b/data/releases/point_releases
index 3a14d90..9b1c8af 100644
--- a/data/releases/point_releases
+++ b/data/releases/point_releases
@@ -177,3 +177,27 @@ Source: http://lists.debian.org/debian-announce/2011/msg00000.html
 Title: Debian 6.0 ("squeeze") updated (r1)
 Date: Mar 19 2011
 Source: http://lists.debian.org/debian-announce/2011/msg00003.html
+
+Title: Debian 6.0 ("squeeze") updated (r2)
+Date: Jun 25 2011
+Source: http://lists.debian.org/debian-announce/2011/msg00004.html
+
+Title: Debian 5.0 ("lenny") updated (r9)
+Date: Oct 1 2011
+Source: http://lists.debian.org/debian-announce/2011/msg00006.html
+
+Title: Debian 6.0 ("squeeze") updated (r3)
+Date: Oct 8 2011
+Source: http://lists.debian.org/debian-announce/2011/msg00007.html
+
+Title: Debian 6.0 ("squeeze") updated (r4)
+Date: Jan 28 2012
+Source: http://lists.debian.org/debian-announce/2012/msg00000.html
+
+Title: Debian 5.0 ("lenny") updated (r10)
+Date: Mar 10 2012
+Source: http://lists.debian.org/debian-announce/2012/msg00007.html
+
+Title: Debian 6.0 ("squeeze") updated (r5)
+Date: May 12 2012
+Source: http://lists.debian.org/debian-announce/2012/msg00009.html
-- 
1.7.10.4

