From 79522901c2464765059258942130e1539a24cfc3 Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Wed, 29 Aug 2012 14:30:20 +0800
Subject: [PATCH 05/11] More bugs were reported since 2011

---
 data/events/bugs |   12 ++++++++++++
 1 file changed, 12 insertions(+)

diff --git a/data/events/bugs b/data/events/bugs
index 1c6c9c2..b5efae3 100644
--- a/data/events/bugs
+++ b/data/events/bugs
@@ -89,3 +89,15 @@ Source: http://bugs.debian.org/640000
 Title: Debian Bug #650000 reported by Sam Morris
 Date: Nov 25 2011
 Source: http://bugs.debian.org/650000
+
+Title: Debian Bug #660000 reported by Beatrice Torracca
+Date: Feb 15 2012
+Source: http://bugs.debian.org/660000
+
+Title: Debian Bug #670000 reported by Julien Cristau
+Date: Apr 22 2012
+Source: http://bugs.debian.org/670000
+
+Title: Debian Bug #680000 reported by Jan Dejemyr
+Date: Jul 2 2012
+Source: http://bugs.debian.org/680000
-- 
1.7.10.4

