From ef268d1048d6217a56c3ecb66877893a1b012183 Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Wed, 29 Aug 2012 14:34:53 +0800
Subject: [PATCH 06/11] More conferences happened since 2011

---
 data/events/conferences |   15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/data/events/conferences b/data/events/conferences
index 4240cf4..1fecb99 100644
--- a/data/events/conferences
+++ b/data/events/conferences
@@ -181,3 +181,18 @@ Title: DebConf11, Banja Luka, Bosnia and Herzegovina
 Start-Date: Jul 24 2011
 End-Date: Jul 30 2011
 Source: http://debconf11.debconf.org/
+
+Title: FOSDEM 2012
+Start-Date: Feb 4 2012
+End-Date: Feb 5 2012
+Source: http://www.fosdem.org/2012/
+
+Title: DebCamp12, Managua, Nicaragua
+Start-Date: Jul 1 2012
+End-Date: Jul 7 2012
+Source: http://debconf12.debconf.org/
+
+Title: DebConf12, Managua, Nicaragua
+Start-Date: Jul 8 2012
+End-Date: Jul 14 2012
+Source: http://debconf12.debconf.org/
-- 
1.7.10.4

