From 136201ed2cbd575046e62c9118ff0011856df653 Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Wed, 29 Aug 2012 14:44:37 +0800
Subject: [PATCH 07/11] More d-i updates since 2008

---
 data/events/d-i |   32 ++++++++++++++++++++++++++++++++
 1 file changed, 32 insertions(+)

diff --git a/data/events/d-i b/data/events/d-i
index e09a5c1..029f677 100644
--- a/data/events/d-i
+++ b/data/events/d-i
@@ -57,3 +57,35 @@ Source: http://www.debian.org/devel/debian-installer/News/2008/20080609
 Title: First RC of lenny's debian-installer released
 Date: Nov 12 2008
 Source: http://www.debian.org/devel/debian-installer/News/2008/20081112
+
+Title: Second RC of lenny's debian-installer released
+Date: Jan 31 2009
+Source: http://www.debian.org/devel/debian-installer/News/2009/20090131
+
+Title: First alpha of squeeze's debian-installer released
+Date: Feb 21 2010
+Source: http://www.debian.org/devel/debian-installer/News/2010/20100221
+
+Title: First beta of squeeze's debian-installer released
+Date: Oct 30 2010
+Source: http://www.debian.org/devel/debian-installer/News/2010/20101030
+
+Title: Second beta of squeeze's debian-installer released
+Date: Dec 8 2010
+Source: http://www.debian.org/devel/debian-installer/News/2010/20101208
+
+Title: First RC of squeeze's debian-installer released
+Date: Jan 12 2011
+Source: http://www.debian.org/devel/debian-installer/News/2011/20110112
+
+Title: Second RC of squeeze's debian-installer released
+Date: Jan 22 2011
+Source: http://www.debian.org/devel/debian-installer/News/2011/20110122
+
+Title: First alpha of wheezy's debian-installer released
+Date: May 13 2012
+Source: http://www.debian.org/devel/debian-installer/News/2012/20120513
+
+Title: First beta of wheezy's debian-installer released
+Date: Aug 4 2012
+Source: http://www.debian.org/devel/debian-installer/News/2012/20120804
-- 
1.7.10.4

