From 4c7354e814da8146681e19f3edc9956e348f79da Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Wed, 29 Aug 2012 14:49:07 +0800
Subject: [PATCH 08/11] Another version of policy was released

---
 data/events/policy |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/data/events/policy b/data/events/policy
index 7a41630..a2af1c0 100644
--- a/data/events/policy
+++ b/data/events/policy
@@ -57,3 +57,7 @@ Source: http://packages.debian.org/changelogs/pool/main/d/debian-policy/current/
 Title: debian-policy 3.9.2.0
 Date: Apr 6 2011
 Source: http://packages.debian.org/changelogs/pool/main/d/debian-policy/current/changelog
+
+Title: debian-policy 3.9.3.0
+Date: Feb 22 2012
+Source: http://packages.debian.org/changelogs/pool/main/d/debian-policy/current/changelog
-- 
1.7.10.4

