From 382066035426fdb8927859b82764ba4b0c5b98d9 Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Wed, 29 Aug 2012 14:53:15 +0800
Subject: [PATCH 09/11] Security support dropped for more old releases

---
 data/events/security |    8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/data/events/security b/data/events/security
index b3e48a2..39a918b 100644
--- a/data/events/security
+++ b/data/events/security
@@ -25,3 +25,11 @@ Source: http://www.debian.org/security/2008/dsa-1571
 Title: Kaminsky's DNS cache poisoning vulnerability publicly announced
 Date: Jul 08 2008
 Source: http://www.debian.org/security/2008/dsa-1603
+
+Title: Security dropped for Debian 4.0 ("etch")
+Date: Jan 21 2010
+Source: http://lists.debian.org/debian-announce/2010/msg00000.html
+
+Title: Security dropped for Debian 5.0 ("lenny")
+Date: Feb 9 2012
+Source: http://lists.debian.org/debian-announce/2012/msg00001.html
-- 
1.7.10.4

