From 8b52cb7b536ebcd7ace627f71b35f8078b5f9c2f Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Wed, 29 Aug 2012 14:58:14 +0800
Subject: [PATCH 10/11] Add derivs patches service

---
 data/events/services |    4 ++++
 1 file changed, 4 insertions(+)

diff --git a/data/events/services b/data/events/services
index e492be3..4d13f93 100644
--- a/data/events/services
+++ b/data/events/services
@@ -89,3 +89,7 @@ Source: http://lists.debian.org/debian-devel-announce/2009/10/msg00004.html
 Title: Archive snapshot service available
 Date: Apr 12 2010
 Source: http://lists.debian.org/debian-announce/2010/msg00002.html
+
+Title: Derivatives patches service available
+Date: Mar 1 2012
+Source: http://lists.debian.org/debian-devel-announce/2012/03/msg00000.html
-- 
1.7.10.4

