From b2139f7b7ae74303decfe20d97acca01aad476ce Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Wed, 29 Aug 2012 15:00:47 +0800
Subject: [PATCH 11/11] We voted again

---
 data/events/voting |    8 ++++++++
 1 file changed, 8 insertions(+)

diff --git a/data/events/voting b/data/events/voting
index ded83d9..3554e4f 100644
--- a/data/events/voting
+++ b/data/events/voting
@@ -181,3 +181,11 @@ Source: http://www.debian.org/vote/2010/vote_002
 Title: Stefano Zacchiroli re-elected as DPL
 Date: Apr 17 2011
 Source: http://www.debian.org/vote/2011/vote_001
+
+Title: Stefano Zacchiroli re-elected as DPL
+Date: April 14 2012
+Source: http://www.debian.org/vote/2012/vote_001
+
+Title: Debian welcomes diversity
+Date: Jun 2 2012
+Source: http://www.debian.org/vote/2012/vote_002
-- 
1.7.10.4

