From 52a1ecd72ba4205764e9c1441f464881c2608e6d Mon Sep 17 00:00:00 2001
From: Simon McVittie <smcv@debian.org>
Date: Thu, 28 Jun 2018 12:00:26 +0100
Subject: [PATCH 1/4] Add quirk for libfribidi-dev

Closes: #901771
---
 d-devlibdeps | 1 +
 1 file changed, 1 insertion(+)

diff --git a/d-devlibdeps b/d-devlibdeps
index 71fffa1..e805491 100755
--- a/d-devlibdeps
+++ b/d-devlibdeps
@@ -82,6 +82,7 @@ overridedevlibdeps() {
 		-e 's/libeXosip2-[0-9]\+-dev/libexosip2-dev/' \
 		-e 's/libf2clibf2c.so-dev/libf2c2-dev/' \
 		-e 's/libffado2-dev/libffado-dev/' \
+		-e 's/libfribidi0-dev/libfribidi-dev/' \
 		-e 's/libGenome-\([.0-9]\+\)-[0-9]\+-dev/libgenome-\1-dev/' \
 		-e 's/libGL1-dev/xlibmesa-gl-dev/' \
 		-e 's/libGLU1-dev/xlibmesa-glu-dev/' \
-- 
2.18.0

