Index: src/AudioIO.cpp
===================================================================
--- src/AudioIO.cpp	(revision 11191)
+++ src/AudioIO.cpp	(working copy)
@@ -342,7 +342,9 @@
    32000,
    44100,
    48000,
-   96000
+   96000,
+   192000,
+   384000
 };
 const int AudioIO::NumStandardRates = sizeof(AudioIO::StandardRates) /
                                       sizeof(AudioIO::StandardRates[0]);
@@ -360,7 +362,8 @@
    48000,
    88200,
    96000,
-   192000
+   192000,
+   384000
 };
 const int AudioIO::NumRatesToTry = sizeof(AudioIO::RatesToTry) /
                                       sizeof(AudioIO::RatesToTry[0]);
Index: src/import/ImportRaw.cpp
===================================================================
--- src/import/ImportRaw.cpp	(revision 11191)
+++ src/import/ImportRaw.cpp	(working copy)
@@ -453,10 +453,8 @@
       mPercent = 0.0;
    if (mPercent > 100.0)
       mPercent = 100.0;
-   if (mRate < 100.0)
-      mRate = 100.0;
-   if (mRate > 100000.0)
-      mRate = 100000.0;
+   if (mRate < 1.0)
+      mRate = 1.0;
 
    EndModal(true);
 }
