diff -Nru apparmor-2.9.0/debian/control apparmor-2.9.0/debian/control
--- apparmor-2.9.0/debian/control	2014-12-23 15:19:28.000000000 +0200
+++ apparmor-2.9.0/debian/control	2015-01-23 10:16:00.000000000 +0200
@@ -24,6 +24,7 @@
 
 Package: apparmor
 Architecture: any
+Multi-Arch: foreign
 Pre-Depends: ${misc:Pre-Depends}
 Depends: ${shlibs:Depends}, ${misc:Depends}, ${python3:Depends}, libapparmor-perl, lsb-base, debconf
 Breaks: libapache2-mod-apparmor (<< 2.5.1-0ubuntu3), apparmor-utils (<< 2.8.0-0ubuntu28), libvirt-bin (<< 1.2.6-0ubuntu6~), apparmor-easyprof-ubuntu (<< 1.2.22), rsyslog (<< 7.4.4-1ubuntu9~)
@@ -38,6 +39,7 @@
 
 Package: apparmor-utils
 Architecture: any
+Multi-Arch: foreign
 Depends: apparmor (>= 2.6.1-4ubuntu1), libapparmor-perl, python3-apparmor, ${shlibs:Depends}, ${misc:Depends}, ${perl:Depends}, ${python3:Depends}
 Suggests: apparmor-docs, vim-addon-manager
 Description: Utilities for controlling AppArmor
