--- src/wl/sys/wl_cfg80211_hybrid.c.orig	2019-05-07 16:52:35.903359637 +0200
+++ src/wl/sys/wl_cfg80211_hybrid.c	2019-05-10 13:40:54.327109866 +0200
@@ -457,7 +457,11 @@ wl_dev_ioctl(struct net_device *dev, u32
 	ifr.ifr_data = (caddr_t)&ioc;
 
 	fs = get_fs();
+#if LINUX_VERSION_CODE < KERNEL_VERSION(5, 1, 0)
 	set_fs(get_ds());
+#else
+	set_fs(KERNEL_DS);
+#endif
 #if defined(WL_USE_NETDEV_OPS)
 	err = dev->netdev_ops->ndo_do_ioctl(dev, &ifr, SIOCDEVPRIVATE);
 #else
