diff -Nru catfish-1.4.7.orig/catfish/CatfishSearchEngine.py catfish-1.4.7/catfish/CatfishSearchEngine.py
--- catfish-1.4.7.orig/catfish/CatfishSearchEngine.py	2019-01-20 16:26:25.000000000 +0100
+++ catfish-1.4.7/catfish/CatfishSearchEngine.py	2019-04-11 11:35:57.559838448 +0200
@@ -166,9 +166,9 @@
         # Path exclusions for efficiency
         exclude = []
         maybe_exclude = [
-            os.path.expanduser("~/.cache"),
-            os.path.expanduser("~/.gvfs"),
-            "/dev"
+            os.path.expanduser("~/.cache/"),
+            os.path.expanduser("~/.gvfs/"),
+            "/dev/"
         ]
         for maybe_path in maybe_exclude:
             if not path.startswith(maybe_path):
@@ -186,7 +186,7 @@
                 if isinstance(filename, str) and path in filename:
                     found_bad = False
                     for filepath in exclude:
-                        if filepath in filename:
+                        if filename.startswith(filepath):
                             if self.stop_time > 0:
                                 logger.debug("Engine is stopped")
                                 return
