diff --git a/test/test_package_lifecycle b/test/test_package_lifecycle
index b4a94f8a..759ce4dc 100755
--- a/test/test_package_lifecycle
+++ b/test/test_package_lifecycle
@@ -133,7 +133,7 @@ runCommandOutCmpFile() {
 test_debuild() {
   export GNUPGHOME=$GPGHOME
   cd ${WORKDIR}/package_lifecycle/test
-  COMMAND='debuild --no-conf --no-lintian --preserve-envvar=PATH --preserve-envvar=PERL5LIB --preserve-envvar=DEBFULLNAME --preserve-envvar=DEBEMAIL --preserve-envvar=GNUPGHOME --set-envvar=NO_PKG_MANGLE=1'
+  COMMAND='LC_MESSAGES=C debuild --no-conf --no-lintian --preserve-envvar=PATH --preserve-envvar=PERL5LIB --preserve-envvar=DEBFULLNAME --preserve-envvar=DEBEMAIL --preserve-envvar=GNUPGHOME --set-envvar=NO_PKG_MANGLE=1'
   cat > ${WORKDIR}/package_lifecycle/debuild.txt <<-EOS
 	 dpkg-buildpackage -us -uc -ui
 	dpkg-buildpackage: source package test
@@ -277,7 +277,7 @@ EOS
 test_debuild2() {
   export GNUPGHOME=$GPGHOME
   cd ${WORKDIR}/package_lifecycle/test
-  COMMAND='debuild --no-conf --no-lintian --preserve-envvar=PATH --preserve-envvar=PERL5LIB --preserve-envvar=DEBFULLNAME --preserve-envvar=DEBEMAIL --preserve-envvar=GNUPGHOME --set-envvar=NO_PKG_MANGLE=1'
+  COMMAND='LC_MESSAGES=C debuild --no-conf --no-lintian --preserve-envvar=PATH --preserve-envvar=PERL5LIB --preserve-envvar=DEBFULLNAME --preserve-envvar=DEBEMAIL --preserve-envvar=GNUPGHOME --set-envvar=NO_PKG_MANGLE=1'
   cat > ${WORKDIR}/package_lifecycle/debuild.txt <<-EOS
 	 dpkg-buildpackage -us -uc -ui
 	dpkg-buildpackage: source package test
@@ -310,7 +310,7 @@ EOS
 test_debuild_forcesign() {
   export GNUPGHOME=$GPGHOME
   cd ${WORKDIR}/package_lifecycle/test
-  COMMAND='debuild --no-conf --no-lintian --preserve-envvar=PATH --preserve-envvar=PERL5LIB --preserve-envvar=DEBFULLNAME --preserve-envvar=DEBEMAIL --preserve-envvar=GNUPGHOME --set-envvar=NO_PKG_MANGLE=1 --force-sign'
+  COMMAND='LC_MESSAGES=C debuild --no-conf --no-lintian --preserve-envvar=PATH --preserve-envvar=PERL5LIB --preserve-envvar=DEBFULLNAME --preserve-envvar=DEBEMAIL --preserve-envvar=GNUPGHOME --set-envvar=NO_PKG_MANGLE=1 --force-sign'
   cat > ${WORKDIR}/package_lifecycle/debuild.txt <<-EOS
 	 dpkg-buildpackage -us -uc -ui
 	dpkg-buildpackage: source package test
