diff -u backuppc-3.1.0/lib/BackupPC/Lib.pm backuppc-3.1.0/lib/BackupPC/Lib.pm
--- backuppc-3.1.0/lib/BackupPC/Lib.pm
+++ backuppc-3.1.0/lib/BackupPC/Lib.pm
@@ -488,7 +488,8 @@
 #    return if ( !opendir(my $fh, $path) );
     my ($fh);
     if ( !opendir($fh, $path) ) {
-       print "log ERROR: opendir ($path) failed\n";
+       print STDERR "ERROR: opendir ($path) failed: $!\n"
+           unless $!{ENOENT};
        return;
     }
 
diff -u backuppc-3.1.0/debian/changelog backuppc-3.1.0/debian/changelog
--- backuppc-3.1.0/debian/changelog
+++ backuppc-3.1.0/debian/changelog
@@ -1,3 +1,10 @@
+backuppc (3.1.0-9.1) stable; urgency=low
+
+  * Non-maintainer upload.
+  * Fix data corruption in tarballs due to logging to stdout.
+
+ -- Thijs Kinkhorst <thijs@debian.org>  Thu, 05 Jan 2012 10:24:29 +0100
+
 backuppc (3.1.0-9) unstable; urgency=low
 
   * chown/chmod only the needed dirs. Closes: #531948
