diff -Nru apt-xapian-index-0.41ubuntu2/data/org.debian.AptXapianIndex.conf apt-xapian-index-0.41ubuntu3/data/org.debian.AptXapianIndex.conf
--- apt-xapian-index-0.41ubuntu2/data/org.debian.AptXapianIndex.conf	2010-11-06 18:21:49.000000000 +0100
+++ apt-xapian-index-0.41ubuntu3/data/org.debian.AptXapianIndex.conf	2011-01-27 15:39:44.000000000 +0100
@@ -7,9 +7,12 @@
         </policy>
 
 	<policy context="default">
-    		<allow send_interface="org.debian.AptXapianIndex"/>
-		<allow receive_interface="org.debian.AptXapianIndex"
-   		       receive_sender="org.debian.AptXapianIndex"/>
+          <allow send_destination="org.debian.AptXapianIndex"
+                 send_interface="org.debian.AptXapianIndex"/>
+          <allow send_destination="org.debian.AptXapianIndex"
+                 send_interface="org.freedesktop.DBus.Introspectable"/>
+          <allow send_destination="org.debian.AptXapianIndex"
+                 send_interface="org.freedesktop.DBus.Properties"/>
 	</policy>
 
 </busconfig>
diff -Nru apt-xapian-index-0.41ubuntu2/debian/changelog apt-xapian-index-0.41ubuntu3/debian/changelog
