diff -u dhcp-probe-1.3.0/debian/changelog dhcp-probe-1.3.0/debian/changelog
--- dhcp-probe-1.3.0/debian/changelog
+++ dhcp-probe-1.3.0/debian/changelog
@@ -1,3 +1,11 @@
+dhcp-probe (1.3.0-4.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Really solve the dhcp-probe.postrm problem when ucf is removed before the
+    dhcp-probe purge (closes: #574210).
+
+ -- Johann Felix Soden <johfel@gmx.de>  Sun, 28 Mar 2010 11:41:46 +0200
+
 dhcp-probe (1.3.0-4) unstable; urgency=low
 
   * Fix dhcp-probe.postrm file problem when ucf is removed before 
diff -u dhcp-probe-1.3.0/debian/dhcp-probe.postrm dhcp-probe-1.3.0/debian/dhcp-probe.postrm
--- dhcp-probe-1.3.0/debian/dhcp-probe.postrm
+++ dhcp-probe-1.3.0/debian/dhcp-probe.postrm
@@ -4,7 +4,6 @@
 # see: dh_installdeb(1)
 
 set -e
-set -x
 
 # summary of how this script can be called:
 #        * <postrm> `remove'
@@ -24,7 +23,7 @@
 UPSTRCONF="/etc/dhcp_probe.cf"
 
 UCF_CONF="/etc/ucf.conf"
-UCF_BIN=`which ucf`
+UCF_BIN=`which ucf || true`
 
 
 case "$1" in
