Index: partman-reiserfs/debian/changelog
===================================================================
--- partman-reiserfs/debian/changelog	(revision 53486)
+++ partman-reiserfs/debian/changelog	(working copy)
@@ -1,3 +1,9 @@
+partman-reiserfs (43) UNRELEASED; urgency=low
+
+  * Default to mounting reiserfs with relatime (closes: #481653).
+
+ -- Colin Watson <cjwatson@debian.org>  Tue, 27 May 2008 10:30:56 +0100
+
 partman-reiserfs (42) unstable; urgency=low
 
   [ Updated translations ]
Index: partman-reiserfs/mountoptions/reiserfs_defaults
===================================================================
--- partman-reiserfs/mountoptions/reiserfs_defaults	(revision 0)
+++ partman-reiserfs/mountoptions/reiserfs_defaults	(revision 0)
@@ -0,0 +1 @@
+relatime
Index: partman-jfs/debian/changelog
===================================================================
--- partman-jfs/debian/changelog	(revision 53486)
+++ partman-jfs/debian/changelog	(working copy)
@@ -1,3 +1,9 @@
+partman-jfs (28) UNRELEASED; urgency=low
+
+  * Default to mounting jfs with relatime (closes: #481653).
+
+ -- Colin Watson <cjwatson@debian.org>  Tue, 27 May 2008 10:30:09 +0100
+
 partman-jfs (27) unstable; urgency=low
 
   [ Updated translations ]
Index: partman-jfs/mountoptions/jfs_defaults
===================================================================
--- partman-jfs/mountoptions/jfs_defaults	(revision 0)
+++ partman-jfs/mountoptions/jfs_defaults	(revision 0)
@@ -0,0 +1 @@
+relatime
Index: partman-ext3/debian/changelog
===================================================================
--- partman-ext3/debian/changelog	(revision 53486)
+++ partman-ext3/debian/changelog	(working copy)
@@ -1,3 +1,9 @@
+partman-ext3 (53) UNRELEASED; urgency=low
+
+  * Default to mounting ext3 with relatime (closes: #481653).
+
+ -- Colin Watson <cjwatson@debian.org>  Tue, 27 May 2008 10:28:58 +0100
+
 partman-ext3 (52) unstable; urgency=low
 
   [ Updated translations ]
Index: partman-ext3/mountoptions/ext3_defaults
===================================================================
--- partman-ext3/mountoptions/ext3_defaults	(revision 0)
+++ partman-ext3/mountoptions/ext3_defaults	(revision 0)
@@ -0,0 +1 @@
+relatime
Index: partman-xfs/debian/changelog
===================================================================
--- partman-xfs/debian/changelog	(revision 53486)
+++ partman-xfs/debian/changelog	(working copy)
@@ -1,3 +1,9 @@
+partman-xfs (42) UNRELEASED; urgency=low
+
+  * Default to mounting xfs with relatime (closes: #481653).
+
+ -- Colin Watson <cjwatson@debian.org>  Tue, 27 May 2008 10:31:56 +0100
+
 partman-xfs (41) unstable; urgency=low
 
   [ Updated translations ]
Index: partman-xfs/mountoptions/xfs_defaults
===================================================================
--- partman-xfs/mountoptions/xfs_defaults	(revision 0)
+++ partman-xfs/mountoptions/xfs_defaults	(revision 0)
@@ -0,0 +1 @@
+relatime
Index: partman-basicfilesystems/debian/changelog
===================================================================
--- partman-basicfilesystems/debian/changelog	(revision 53486)
+++ partman-basicfilesystems/debian/changelog	(working copy)
@@ -1,3 +1,9 @@
+partman-basicfilesystems (61) UNRELEASED; urgency=low
+
+  * Default to mounting ext2 with relatime (closes: #481653).
+
+ -- Colin Watson <cjwatson@debian.org>  Tue, 27 May 2008 10:24:44 +0100
+
 partman-basicfilesystems (60) unstable; urgency=low
 
   [ Frans Pop ]
Index: partman-basicfilesystems/mountoptions/ext2_defaults
===================================================================
--- partman-basicfilesystems/mountoptions/ext2_defaults	(revision 0)
+++ partman-basicfilesystems/mountoptions/ext2_defaults	(revision 0)
@@ -0,0 +1 @@
+relatime
