diff -u dmraid-1.0.0.rc14/debian/dmraid.postinst dmraid-1.0.0.rc14/debian/dmraid.postinst
--- dmraid-1.0.0.rc14/debian/dmraid.postinst
+++ dmraid-1.0.0.rc14/debian/dmraid.postinst
@@ -13,6 +13,9 @@
   	update-rc.d -f dmraid remove
 	rm -f /etc/init.d/dmraid
     fi
+
+    # Activate existing arrays now.
+    udevadm trigger --subsystem-match=block --action=add
   ;;
   abort-upgrade|abort-remove|abort-deconfigure)
     ;;
