From e0be5ef54b2a4a13cf3fac9bc078509cd0ae638f Mon Sep 17 00:00:00 2001
From: Rafael Laboissiere <rafael@laboissiere.net>
Date: Tue, 16 Aug 2016 11:51:26 +0200
Subject: [PATCH] Remove copyright statement and clarify licensing conditions

---
 update-bug-list.py | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/update-bug-list.py b/update-bug-list.py
index aaa039f..de877be 100755
--- a/update-bug-list.py
+++ b/update-bug-list.py
@@ -1,8 +1,8 @@
 #!/usr/bin/env python
 
-## Copyright (C) 2015 Rafael Laboissiere
+## Originally written by Rafael Laboissiere in 2015.
 ##
-## This program is in the public domain
+## This script is hereby placed into the public domain, no rights reserved.
 
 ## Use this script for updating file data/events/bugs with more recent
 ## bugs.  Launch the script from the top level directory, like this:
-- 
2.8.1

