diff -u dirvish-1.2.1/debian/rules dirvish-1.2.1/debian/rules
--- dirvish-1.2.1/debian/rules
+++ dirvish-1.2.1/debian/rules
@@ -41,7 +41,7 @@
 	dh_testroot
 	dh_clean -k
 	dh_installdirs
-	echo -e "\n`pwd`/debian/dirvish\n`pwd`/debian/dirvish/usr/sbin\n`pwd`/debian/dirvish/usr/share/man\n/etc/dirvish\ny\ny\nn" | sh ./install.sh
+	/bin/echo -e "\n`pwd`/debian/dirvish\n`pwd`/debian/dirvish/usr/sbin\n`pwd`/debian/dirvish/usr/share/man\n/etc/dirvish\ny\ny\nn" | sh ./install.sh
 	@echo ''
 	# I consider dirvish-locate more a user tool than an administrator's tool
 	mv debian/dirvish/usr/sbin/dirvish-locate debian/dirvish/usr/bin/.
diff -u dirvish-1.2.1/debian/changelog dirvish-1.2.1/debian/changelog
--- dirvish-1.2.1/debian/changelog
+++ dirvish-1.2.1/debian/changelog
@@ -1,3 +1,11 @@
+dirvish (1.2.1-1.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Use coreutils’ echo in debian/rules to allow the use of the -e option
+    (Closes: #459174).
+
+ -- Michael Schutte <m.schutte.jr@gmail.com>  Sat, 08 Mar 2008 19:10:13 +0100
+
 dirvish (1.2.1-1) unstable; urgency=low
 
   * Acknowledge NMU fixes.
