diff -Nru dput-ng-1.8/debian/changelog dput-ng-1.8+nmu1/debian/changelog
--- dput-ng-1.8/debian/changelog	2014-09-11 20:58:12.000000000 +0100
+++ dput-ng-1.8+nmu1/debian/changelog	2014-12-11 12:50:39.000000000 +0000
@@ -1,3 +1,10 @@
+dput-ng (1.8+nmu1) UNRELEASED; urgency=medium
+
+  * Non-maintainer upload.
+  * Fix manpage for dcut allow/deny syntax (Closes: #741966)
+
+ -- Wookey <wookey@debian.org>  Thu, 11 Dec 2014 12:50:00 +0000
+
 dput-ng (1.8) unstable; urgency=low
 
   * The "There is no sin except stupidity" release
diff -Nru dput-ng-1.8/docs/man/dcut.1.man dput-ng-1.8+nmu1/docs/man/dcut.1.man
--- dput-ng-1.8/docs/man/dcut.1.man	2014-01-31 02:50:03.000000000 +0000
+++ dput-ng-1.8+nmu1/docs/man/dcut.1.man	2014-12-11 13:10:11.000000000 +0000
@@ -162,15 +162,23 @@
 bypass any argument checking/translation. Please note, this will generate a
 commands file which will be uploaded literally as is. Use with caution.
 
-*--allow*=PACKAGES::
+*--allow*=PACKAGE
 
-Source package(s) where permissions to upload should be granted. Repeat the
-argument to apply permissions to more than one package at once.
+*--allow* PACKAGE LIST::
 
-*--deny*=PACKAGES::
+Source package(s) where permissions to upload should be granted. Give
+a space-separated list of packages to apply permissions to more than
+one package at once. If multiple '--allow' options are given, the last
+one takes precedence.
 
-Source package(s) where permissions to upload should be denied. Repeat the
-argument to apply permissions to more than one package at once.
+*--deny*=PACKAGE
+
+*--deny* PACKAGE LIST::
+
+Source package(s) where permissions to upload should be denied. Give a
+space-separated list of packages to apply permissions to more than one
+package at once. If multiple '--deny' options are given, the last
+one takes precedence.
 
 reschedule
 ~~~~~~~~~~
