diff -Nru /tmp/Z4InDfenC5/apt-listchanges-2.72.5/debian/changelog /tmp/NCc6IpT93w/apt-listchanges-2.72.5/debian/changelog
--- /tmp/Z4InDfenC5/apt-listchanges-2.72.5/debian/changelog	2007-02-18 23:39:36.000000000 +0100
+++ /tmp/NCc6IpT93w/apt-listchanges-2.72.5/debian/changelog	2007-04-02 00:46:25.000000000 +0200
@@ -1,3 +1,11 @@
+apt-listchanges (2.72.5-0.1) unstable; urgency=low
+
+  * Non-maintainer upload for NM T&S purposes.
+  * Unconditionally run ucf at postinst, patch from Frank Küster.
+    (Closes: #412925).
+
+ -- Yves-Alexis Perez <corsac@corsac.net>  Mon, 02 Apr 2007 00:44:16 +0200
+
 apt-listchanges (2.72.5) unstable; urgency=low
 
   * Fix a typo in the preinst: .pyo -> .pyc, thanks to Marc Brockschmidt.
diff -Nru /tmp/Z4InDfenC5/apt-listchanges-2.72.5/debian/postinst /tmp/NCc6IpT93w/apt-listchanges-2.72.5/debian/postinst
--- /tmp/Z4InDfenC5/apt-listchanges-2.72.5/debian/postinst	2006-10-07 22:00:15.000000000 +0200
+++ /tmp/NCc6IpT93w/apt-listchanges-2.72.5/debian/postinst	2007-04-02 00:27:51.000000000 +0200
@@ -31,11 +31,7 @@
     db_get apt-listchanges/which
     echo "which=$RET" >> $PREFERENCES.new
 
-    if [ -f $PREFERENCES ]; then
-        ucf --debconf-ok $PREFERENCES.new $PREFERENCES
-    else
-        mv $PREFERENCES.new $PREFERENCES
-    fi
+    ucf --debconf-ok $PREFERENCES.new $PREFERENCES
     rm -f $PREFERENCES.new
 
     ## Cleanup from older versions
