Index: common.ent
===================================================================
--- common.ent	(revisione 8928)
+++ common.ent	(copia locale)
@@ -103,7 +103,7 @@
 
 <!-- deprecated -->
 
-<!ENTITY url-mentors "http://people.debian.org/~mpalmer/debian-mentors_FAQ.html">
+<!ENTITY url-mentors "http://wiki.debian.org/DebianMentorsFaq">
 
 <!ENTITY url-rules-files "http://arch.debian.org/arch/private/srivasta/">
 
