--- ../orig/debsums-2.0.32/man/po4a.cfg	2006-02-20 15:40:49.000000000 +0100
+++ debsums-2.0.32/man/po4a.cfg	2007-09-12 20:23:10.000000000 +0200
@@ -1,15 +1,19 @@
 [po4a_langs] fr pt_BR ru sv
 [po4a_paths] po/debsums.pot $lang:po/$lang.po
 
+[po4a_alias:man] man opt:"-o groff_code=verbatim -o untranslated=Id" \
+                     opt_fr:"-L ISO-8859-1" \
+                     opt_pt_BR:"-L ISO-8859-1" \
+                     opt_ru:"-L KOI8-R" \
+                     opt_sv:"-L ISO-8859-1"
+
 [type: man] debsums.1 $lang:$lang/debsums.$lang.1 \
 	add_fr:fr/addendum.fr \
-	add_pt_BR:fr/addendum.pt_BR \
-	add_ru:ru/addendum.ru opt_ru:"-L KOI8-R" \
-	opt:"-o groff_code=verbatim -o untranslated=Id"
+	add_pt_BR:pt_BR/addendum.pt_BR \
+	add_ru:ru/addendum.ru
 
 [type: man] debsums_gen.8 \
 	$lang:$lang/debsums_gen.$lang.8 \
 	add_fr:fr/addendum.fr \
 	add_pt_BR:pt_BR/addendum.pt_BR \
-	add_ru:ru/addendum.ru opt_ru:"-L KOI8-R" \
-	opt:"-o groff_code=verbatim -o untranslated=Id"
+	add_ru:ru/addendum.ru
