--- dh_make.orig	2012-12-26 23:37:13.419564524 +0900
+++ dh_make	2012-12-26 23:42:33.535996185 +0900
@@ -152,7 +152,7 @@
   -e, --email <address>     use <address> as the maintainer e-mail address
   -n, --native              the program is Debian native, don\'t generate .orig
   -f, --file <file>         specify file to use as the original source archive
-      --createorig          create orig.tar.gz file
+      --createorig          create orig.tar.xz file
   -s, --single              set package class to single
   -i, --indep               set package class to arch-independent
   -m, --multi               set package class to multiple binary
@@ -550,9 +554,9 @@
 			}
 		} else {
 			if ($create_orig_tar) {
-				system('tar', 'cfz', "../$package_name\_$version.orig.tar.gz", ".");
+				system('tar', 'cfJ', "../$package_name\_$version.orig.tar.xz", ".");
 			} else {
-				print "Could not find $package_name\_$version.orig.tar.gz\n";
+				print "Could not find $package_name\_$version.orig.tar.xz\n";
 				print "Either specify an alternate file to use with -f,\n";
 				print "or add --createorig to create one.\n";
 				exit 1;
