diff --git a/debian/docx2txt.mime b/debian/docx2txt.mime
index 1a486de..47a9ea2 100644
--- a/debian/docx2txt.mime
+++ b/debian/docx2txt.mime
@@ -1 +1 @@
-application/vnd.openxmlformats-officedocument.wordprocessingml.document; docx2txt '%s' - ; copiousoutput; description=Office Open XML Document
+application/vnd.openxmlformats-officedocument.wordprocessingml.document; docx2txt %s - ; copiousoutput; description=Office Open XML Document
