--- debian/control 2007-01-16 09:19:47.000000000 -0200
+++ debian/control 2007-01-17 05:57:53.000000000 -0200
@@ -3,7 +3,7 @@
 Section: otherosfs
 Maintainer: Joerg Jaspert <joerg@debian.org>
 Uploaders: Eduard Bloch <blade@debian.org>, Steve McIntyre <93sam@debian.org>
-Build-Depends: debhelper (>=5.0.37.3), zlib1g-dev, autotools-dev, cmake (>= 2.4.3-1), libcap-dev [!kfreebsd-i386 !kfreebsd-amd64 !hurd-i386], libcam-dev [kfreebsd-i386 kfreebsd-amd64], libmagic-dev
+Build-Depends: debhelper (>=5.0.37.3), zlib1g-dev, autotools-dev, cmake (>= 2.4.3-1) | cmake (<=2.4.2-1) [hurd-i386], libcap-dev [!kfreebsd-i386 !kfreebsd-amd64 !hurd-i386], libcam-dev [kfreebsd-i386 kfreebsd-amd64], libmagic-dev
 Standards-Version: 3.7.2

 Package: wodim
