diff -urN trunk.orig/debian/apt-file.update-notifier trunk/debian/apt-file.update-notifier
--- trunk.orig/debian/apt-file.update-notifier  2011-09-10 14:01:08.394256749 +0900
+++ trunk/debian/apt-file.update-notifier       2011-09-10 14:05:25.710251335 +0900
@@ -1,8 +1,10 @@
 Name: apt-file update needed
+Name-ja_JP: apt-file の更新が必要です
 Priority: Medium
 Command: "/usr/share/apt-file/do-apt-file-update"
 Terminal: True
 DisplayIf: /usr/share/apt-file/is-cache-empty
 Description: You may need to update or create the apt-file cache.
  Running this command likely needs an active internet connection.
-
+Description-ja_JP: apt-file のキャッシュを更新、または作成しなければいけません。
+ このコマンドの実行には、インターネット接続が有効になっている必要があります。
