diff -ur apt-0.6.46.3/apt-pkg/deb/debsrcrecords.cc apt-0.6.46.3.fix/apt-pkg/deb/debsrcrecords.cc
--- apt-0.6.46.3/apt-pkg/deb/debsrcrecords.cc	2006-03-02 14:44:28.000000000 +0100
+++ apt-0.6.46.3.fix/apt-pkg/deb/debsrcrecords.cc	2006-11-30 00:38:19.000000000 +0100
@@ -40,7 +40,7 @@
    char *Buf;
    if (Bins.length() > sizeof(Buffer))
    {
-      BigBuf = new char[Bins.length()];
+      BigBuf = new char[Bins.length()+1];
       Buf = BigBuf;
    }
    else
