diff -Nru binutils-2.29.51.20171208.orig/debian/tests/build binutils-2.29.51.20171208/debian/tests/build
--- binutils-2.29.51.20171208.orig/debian/tests/build	2017-12-11 13:24:16.729964293 -0200
+++ binutils-2.29.51.20171208/debian/tests/build	1969-12-31 21:00:00.000000000 -0300
@@ -1 +0,0 @@
-#!/bin/true
diff -Nru binutils-2.29.51.20171208.orig/debian/tests/control binutils-2.29.51.20171208/debian/tests/control
--- binutils-2.29.51.20171208.orig/debian/tests/control	2017-12-11 13:24:16.729964293 -0200
+++ binutils-2.29.51.20171208/debian/tests/control	2017-12-11 13:24:22.557958973 -0200
@@ -1,7 +1,3 @@
-Tests: build
-Depends: build-essential
-Restrictions: build-needed
-
 Tests: libc-link
 Depends: build-essential
 
