diff -u abntex-0.9~beta2/debian/changelog abntex-0.9~beta2/debian/changelog
--- abntex-0.9~beta2/debian/changelog
+++ abntex-0.9~beta2/debian/changelog
@@ -1,3 +1,9 @@
+abntex (0.9~beta2-2.1) unstable; urgency=low
+
+  * Fix Build-Depends-Indep on texlive packages (closes: #442313)
+
+ -- Don Armstrong <don@debian.org>  Sat, 08 Mar 2008 02:33:11 -0800
+
 abntex (0.9~beta2-2) unstable; urgency=low
 
   * Move back to main since it doesn't depends of latex2html for building.
diff -u abntex-0.9~beta2/debian/control abntex-0.9~beta2/debian/control
--- abntex-0.9~beta2/debian/control
+++ abntex-0.9~beta2/debian/control
@@ -2,7 +2,7 @@
 Section: tex
 Priority: optional
 Maintainer: Otavio Salvador <otavio@debian.org>
-Build-Depends-Indep: tetex-extra | texlive-latex-recommended
+Build-Depends-Indep: tetex-extra | texlive-latex-recommended, tetex-extra | texlive-latex-extra, tetex-extra | texlive-fonts-recommended, tetex-extra | texlive-fonts-extra, tetex-extra | texlive-lang-portuguese
 Build-Depends: debhelper (>= 5)
 Standards-Version: 3.7.2
 
