diff -urN arj-3.10.18.orig/debian/control arj-3.10.18/debian/control
--- arj-3.10.18.orig/debian/control	2003-12-10 10:43:02.000000000 +1100
+++ arj-3.10.18/debian/control	2003-12-10 10:43:13.000000000 +1100
@@ -3,7 +3,7 @@
 Priority: optional
 Maintainer: Ola Lundqvist <opal@debian.org>
 Uploaders: Guillem Jover <guillem@debian.org>
-Build-Depends: debhelper (>> 3.0.0), autoconf
+Build-Depends: debhelper (>> 4.0.0), autoconf
 Standards-Version: 3.6.1
 
 Package: arj
diff -urN arj-3.10.18.orig/environ.c arj-3.10.18/environ.c
--- arj-3.10.18.orig/environ.c	2003-12-10 10:43:01.000000000 +1100
+++ arj-3.10.18/environ.c	2003-12-10 10:43:28.000000000 +1100
@@ -54,11 +54,10 @@
  #include <utime.h>
  #include <sys/time.h>                  /* LIBC high-resolution timing */
  #include <sys/resource.h>              /* Priority control */
- /* For Linux-specific IOCTL. sys/statfs.h is no longer included,
-    since struct statfs is duplicated in linux/fs.h */
  #if defined(linux)
   #include <sys/ioctl.h>
   #include <linux/fs.h>
+  #include <sys/statfs.h>
  #elif defined(__FreeBSD__)
   #include <sys/param.h>
   #include <sys/mount.h>
