--- CONF.sh~	2013-01-15 01:56:42.000000000 +0000
+++ CONF.sh	2013-03-20 09:13:51.000000000 +0000
@@ -84,11 +84,11 @@
 
 # ... for arch
 if [ -z "$ARCHES" ]; then
-	CPU=`dpkg-architecture -qDEB_HOST_DPKG_CPU 2>/dev/null || true`
+	CPU=`dpkg-architecture -qDEB_HOST_ARCH_CPU 2>/dev/null || true`
 	if [ -z "$CPU" ]; then
 		CPU=`dpkg-architecture -qDEB_HOST_ARCH`
 	fi
-	KERNEL=`dpkg-architecture -qDEB_HOST_DPKG_OS 2>/dev/null || true`
+	KERNEL=`dpkg-architecture -qDEB_HOST_ARCH_OS 2>/dev/null || true`
 	if [ -z "$KERNEL" ]; then
 		KERNEL=linux
 	fi
