diff -ur dpkg-1.18.23.ilp32/data/abitable dpkg-1.18.23/data/abitable
--- dpkg-1.18.23.ilp32/data/abitable	2017-03-10 01:25:46.966846921 +0000
+++ dpkg-1.18.23/data/abitable	2017-02-22 21:50:39.000000000 +0000
@@ -10,5 +10,4 @@
 #
 # <Debian name>	<Bits>
 abin32		32
-ilp32		32
 x32		32
diff -ur dpkg-1.18.23.ilp32/data/ostable dpkg-1.18.23/data/ostable
--- dpkg-1.18.23.ilp32/data/ostable	2017-03-09 16:57:36.122840477 +0000
+++ dpkg-1.18.23/data/ostable	2017-02-22 21:50:39.000000000 +0000
@@ -24,7 +24,6 @@
 abi64-gnu-linux		linux-gnuabi64		linux[^-]*-gnuabi64
 spe-gnu-linux		linux-gnuspe		linux[^-]*-gnuspe
 x32-gnu-linux		linux-gnux32		linux[^-]*-gnux32
-ilp32-gnu-linux		linux-gnu_ilp32		linux[^-]*-gnu_ilp32
 base-gnu-linux		linux-gnu		linux[^-]*(-gnu.*)?
 eabihf-gnu-kfreebsd	kfreebsd-gnueabihf	kfreebsd[^-]*-gnueabihf
 base-gnu-kfreebsd	kfreebsd-gnu		kfreebsd[^-]*(-gnu.*)?
diff -ur dpkg-1.18.23.ilp32/data/tupletable dpkg-1.18.23/data/tupletable
--- dpkg-1.18.23.ilp32/data/tupletable	2017-03-10 01:27:54.292067239 +0000
+++ dpkg-1.18.23/data/tupletable	2017-02-22 21:50:39.000000000 +0000
@@ -9,7 +9,6 @@
 base-uclibc-linux-<cpu>		uclibc-linux-<cpu>
 eabihf-musl-linux-arm		musl-linux-armhf
 base-musl-linux-<cpu>		musl-linux-<cpu>
-ilp32-gnu-linux-arm64		arm64ilp32
 eabihf-gnu-linux-arm		armhf
 eabi-gnu-linux-arm		armel
 abin32-gnu-linux-mips64r6el	mipsn32r6el
