--- /usr/share/dokuwiki/feed.php	2009-02-14 13:13:24.000000000 +0100
+++ feed.php	2009-10-04 00:40:06.000000000 +0200
@@ -236,7 +236,7 @@
         $item->author = '';
         if($user && $conf['useacl'] && $auth){
             $userInfo = $auth->getUserData($user);
-            $item->author = $userInfo['name'];
+            $item->author = editorinfo($user);
             if($userInfo && !$opt['guardmail']){
                 $item->authorEmail = $userInfo['mail'];
             }else{
@@ -245,7 +245,7 @@
             }
         }elseif($user){
             // this happens when no ACL but some Apache auth is used
-            $item->author      = $user;
+            $item->author      = editorinfo($user);
             $item->authorEmail = $user.'@'.$recent['ip'];
         }else{
             $item->authorEmail = 'anonymous@'.$recent['ip'];
