--- main/main.c~	2003-10-26 05:03:21.000000000 +0900
+++ main/main.c	2004-03-14 17:35:11.000000000 +0900
@@ -45,10 +45,10 @@
   if (fputs(DPKG, stdout) < 0) werr("stdout");
   if (fputs(_("' package management program version "), stdout) < 0) werr("stdout");
   if (fputs( DPKG_VERSION_ARCH ".\n", stdout) < 0) werr("stdout");
-  if (fputs(_( "This is free software; see the GNU General Public Licence version 2 or\n"
+  if (fprintf(stdout, _( "This is free software; see the GNU General Public Licence version 2 or\n"
 		"later for copying conditions.  There is NO warranty.\n"
-		"See " DPKG " --licence for copyright and license details.\n"),
-		 stdout) < 0) werr("stdout");
+		"See %s --licence for copyright and license details.\n"),
+	      DPKG) < 0) werr("stdout");
 }
 /*
    options that need fixing:
