diff --git a/debian/control b/debian/control
index cb2623c..a04fa1e 100644
--- a/debian/control
+++ b/debian/control
@@ -26,7 +26,10 @@ Build-Depends:
  libxext-dev,
  libxfixes-dev,
  libx11-dev,
- ikiwiki
+ ikiwiki,
+ libtinfo-dev,
+ libedit-dev,
+ zlib1g-dev
 Standards-Version: 3.9.8
 Homepage: https://www.freedesktop.org/wiki/Software/Beignet/
 Vcs-Git: https://anonscm.debian.org/git/pkg-opencl/beignet.git
diff --git a/debian/patches/series b/debian/patches/series
index c340adc..05905d1 100644
--- a/debian/patches/series
+++ b/debian/patches/series
@@ -1,7 +1,7 @@
 Debian-compliant-compiler-flags-handling.patch
 support-kfreebsd.patch
 reduce-notfound-output.patch
-shared-llvm.patch
+#shared-llvm.patch
 update-docs.patch
 ship-test-tool.patch
 find-python35.patch
@@ -11,3 +11,4 @@ llvm39-support.patch
 add-appstream-metadata.patch
 spelling.patch
 opencl2-runtime-detection.patch
+static-llvm.patch
