--- debian/rules.org	2022-08-17 06:09:55.850853504 +0000
+++ debian/rules	2022-08-17 06:10:21.906723720 +0000
@@ -13,6 +13,8 @@
 CFLAGS := $(shell dpkg-buildflags --get CPPFLAGS; dpkg-buildflags --get CFLAGS)
 CXXFLAGS := $(shell dpkg-buildflags --get CPPFLAGS; dpkg-buildflags --get CXXFLAGS)
 LDFLAGS := $(shell dpkg-buildflags --get LDFLAGS)
+CFLAGS   += -D_LARGE_FILE_SOURCE=1  -D_FILE_OFFSET_BITS=64
+CXXFLAGS += -D_LARGE_FILE_SOURCE=1  -D_FILE_OFFSET_BITS=64
 export CFLAGS
 export CXXFLAGS
 export LDFLAGS
