Index: bsdmainutils-8.0.6/usr.bin/calendar/calendars/calendar.holiday
===================================================================
--- bsdmainutils-8.0.6.orig/usr.bin/calendar/calendars/calendar.holiday	2010-01-22 12:59:14.000000000 +0700
+++ bsdmainutils-8.0.6/usr.bin/calendar/calendars/calendar.holiday	2010-01-23 16:40:19.000000000 +0700
@@ -56,7 +56,9 @@
 02/10	Feast of St. Paul's Shipwreck, AD 60
 02/11	National Foundation Day in Japan
 02/12	Pyidaungsa Day in Burma
+/* Makha Bucha is not Gregorian
 02/16	Makha Bucha Day in Thailand
+*/
 02/18	Democracy Day in Nepal
 02/18	Independence Day in The Gambia
 02/23	Republic Day in Guyana
@@ -126,7 +128,7 @@
 04/09	Martyrs Day in Tunisia
 04/11	National Heroes Day in Costa Rica
 04/13	National Day in Chad
-04/13	Songkron Day in Thailand
+04/13	Songkran Day in Thailand
 04/14	Day of the Americas in Honduras
 04/15	Bengali New Year in Bangladesh
 04/16	De Diego's Birthday celebrated in Puerto Rico
@@ -182,7 +184,9 @@
 05/14	Unification Day in Liberia
 05/15	Kamuzu Day in Malawi
 05/15	Vesak Day, observed in Singapore and Malaysia
+/* Visakha Bucha is not Gregorian
 05/15	Visakha Bucha Day in Thailand
+*/
 05/16	Discovery Day in Cayman Islands
 05/17	Constitution Day in Nauru and Norway
 05/18	Flag Day in Haiti
@@ -275,7 +279,9 @@
 07/10	Independence Day in Bahamas
 07/11	National Holiday in the Mongolian People's Republic
 07/12	Battle of Boyne celebrated in Northern Ireland
+/* Buddhist Lent is not Gregorian
 07/13	Buddhist Lent in Thailand
+*/
 07/14	Anniversary of the Revolution in Iraq
 07/14	French National Festival
 07/14	National Holiday in Monaco
