--- /lnet/00filesystem.squashfs/etc/init.d/alsa-utils	2009-03-30 02:36:01.000000000 +0200
+++ /etc/init.d/alsa-utils	2009-04-09 08:48:48.545351854 +0200
@@ -221,7 +221,8 @@
 		sanify_levels_on_card "$1" || TTSDML_RETURNSTATUS=1
 		;;
 	esac
-	mute_and_zero_levels_on_card "pcsp"
+	cat /proc/asound/card[0-9]*/id | grep -qswe "pcsp" && \
+		mute_and_zero_levels_on_card "pcsp"
 	return $TTSDML_RETURNSTATUS
 }
 
