Index: autoproject-0.20/lib/cli/all/argp/program.texinfo
===================================================================
--- autoproject-0.20.orig/lib/cli/all/argp/program.texinfo	2001-02-28 02:49:38.000000000 +0100
+++ autoproject-0.20/lib/cli/all/argp/program.texinfo	2021-09-25 18:30:47.463527619 +0200
@@ -252,7 +252,7 @@
 If you find a bug in @command{#NAME#}, please send electronic mail to
 @email{#EEMAIL#}.  Include the version number, which you can find by
 running @w{@samp{#NAME# --version}}.  Also include in your message the
-output that the program produced and the output you expected.@refill
+output that the program produced and the output you expected.
 
 If you have other questions, comments or suggestions about
 @command{#NAME#}, contact the author via electronic mail to
Index: autoproject-0.20/lib/cli/all/none/program.texinfo
===================================================================
--- autoproject-0.20.orig/lib/cli/all/none/program.texinfo	2001-02-28 02:49:39.000000000 +0100
+++ autoproject-0.20/lib/cli/all/none/program.texinfo	2021-09-25 18:31:33.747498011 +0200
@@ -252,7 +252,7 @@
 If you find a bug in @command{#NAME#}, please send electronic mail to
 @email{#EEMAIL#}.  Include the version number, which you can find by
 running @w{@samp{#NAME# --version}}.  Also include in your message the
-output that the program produced and the output you expected.@refill
+output that the program produced and the output you expected.
 
 If you have other questions, comments or suggestions about
 @command{#NAME#}, contact the author via electronic mail to
