diff -ruN Alacarte.orig/MainWindow.py Alacarte/MainWindow.py
--- Alacarte.orig/MainWindow.py	2009-12-21 22:59:55.000000000 +0100
+++ Alacarte/MainWindow.py	2009-12-22 00:02:32.000000000 +0100
@@ -128,6 +128,8 @@
 						found = True
 				if item[3].get_type() == gmenu.TYPE_SEPARATOR:
 					if not isinstance(item_id, tuple):
+						#we may not skip the increment via "continue"
+						i += 1
 						continue
 					#separators have no id, have to find them manually
 					#probably won't work with two separators together
