diff --git a/build/boot/x86/xen/debian.cfg b/build/boot/x86/xen/debian.cfg
index 070dca4ce..2ca33e236 100644
--- a/build/boot/x86/xen/debian.cfg
+++ b/build/boot/x86/xen/debian.cfg
@@ -33,53 +33,53 @@
 #kernel = "vmlinuz"
 #ramdisk = "initrd.gz"
 
 #----------------------------------------------------------------------------
 # CDROM install:
 #----------------------------------------------------------------------------
 #
 # Download an ISO image which supports installation under Xen. Typically these
 # are the amd64+i386 multiarch images. e.g. those under:
 #   http://cdimage.debian.org/debian-cd/current/multi-arch/
 #
 # Uncomment "bootloader" and the correct "bootargs" for the
 # architecture you wish to install.
 #
 # Be sure to configure a CDROM device with the installation image in
 # addition to your regular disk HDD in your disk stanza.
 #
 # e.g. for an ISO image:
 #   file:/path/to/debian-6.0.0-amd64-i386-netinst.iso,xvdd:cdrom,r
 #    -or- if you have a phyiscal cdrom in your drive, phy:
 #   phy:/dev/cdrom,xvdd:cdrom,r
 
 #bootloader="pygrub"
 
 # For i386:
-#bootargs="--kernel=install.i386/xen/vmlinuz --ramdisk=install.i386/xen/initrd.gz"
+#bootloader_args="--kernel=install.i386/xen/vmlinuz --ramdisk=install.i386/xen/initrd.gz"
 # For amd64:
-#bootargs="--kernel=install.amd/xen/vmlinuz --ramdisk=install.amd/xen/initrd.gz"
+#bootloader_args="--kernel=install.amd/xen/vmlinuz --ramdisk=install.amd/xen/initrd.gz"
 
 #----------------------------------------------------------------------------
 # Installation options common to all methods
 #----------------------------------------------------------------------------
 #
 # If you configure a vfb below then use "console=tty0" instead of
 # "console=hvc0".
 # 
 # You can prepend your own command line options too
 
 #extra = "debian-installer/exit/always_halt=true -- quiet console=hvc0"
 
 #============================================================================
 # TO BOOT INSTALLED SYSTEM
 #
 # Comment all of the above installation options and uncomment the
 # below instead
 #============================================================================
 
 #bootloader="pygrub"
 
 #============================================================================
 # STANDARD OPTIONS
 #============================================================================
 #
