--- ./erts/emulator/drivers/common/inet_drv.c.orig	2007-06-11 14:53:49.000000000 +0200
+++ ./erts/emulator/drivers/common/inet_drv.c	2007-09-23 19:47:44.000000000 +0200
@@ -280,6 +280,18 @@
 #     define    SCTP_EOF        MSG_EOF
 #endif
 
+/* These changed in draft 12 (February 17, 2006) */
+#ifndef SCTP_ADAPTION_LAYER
+#define SCTP_ADAPTION_LAYER SCTP_ADAPTATION_LAYER
+#define SCTP_ADAPTION_INDICATION SCTP_ADAPTATION_INDICATION
+#define sctp_adaption_event sctp_adaptation_event
+#define sai_adaption_ind sai_adaptation_ind
+#define sctp_adaption_layer_event sctp_adaptation_layer_event
+#define sn_adaption_event sn_adaptation_event
+#define sctp_setadaption sctp_setadaptation
+#define ssb_adaption_ind ssb_adaptation_ind
+#endif	/* SCTP_ADAPTION_LAYER */
+
 #endif /* SCTP supported */
 
 #ifndef WANT_NONBLOCKING
