--- unpacked/usr/bin/chronicle	2011-05-15 20:15:12.000000000 +0200
+++ /usr/bin/chronicle	2012-12-23 18:24:13.868587817 +0100
@@ -1475,7 +1475,8 @@
     $template = loadTemplate( "tags.xml.template", die_on_bad_params => 0 );
     $template->param( blog_title => $CONFIG{ 'blog_title' } )
       if ( $CONFIG{ 'blog_title' } );
-    $template->param( entries => \@sortedEntries ) if (@sortedEntries);
+    my @rssEntries = @sortedEntries[0 .. $CONFIG{ 'rss-count' } - 1 ];
+    $template->param( entries => \@rssEntries ) if (@rssEntries);
     $template->param( tagname => $tagName )        if ($tagName);
     outputTemplate( $template, "$dir/$tagName.rss" );
 }
@@ -1748,7 +1749,8 @@
     $template = loadTemplate( "month.xml.template", die_on_bad_params => 0 );
     $template->param( blog_title => $CONFIG{ 'blog_title' } )
       if ( $CONFIG{ 'blog_title' } );
-    $template->param( entries => \@sortedEntries ) if (@sortedEntries);
+    my @rssEntries = @sortedEntries[0 .. $CONFIG{ 'rss-count' } - 1 ];
+    $template->param( entries => \@rssEntries ) if (@rssEntries);
     $template->param( month => $month, year => $year );
     $template->param( month_name => $names[$month - 1] );
     outputTemplate( $template, "$dir/$month.rss" );
