--- /usr/bin/adt-run.orig	2011-11-09 07:18:14.977723322 +0100
+++ /usr/bin/adt-run	2011-11-09 07:28:10.309752135 +0100
@@ -814,6 +814,12 @@
 	if rc:
 		pstderr("\n" "warning: failed to restore"
 			" testbed apt cache, exit code %d" % rc)
+	cmdl = ['rm', '-f', '/etc/apt/apt.conf.d/90autopkgtest',
+                '/etc/apt/preferences.d/90autopkgtest', '/etc/apt/sources.list.d/autopkgtest.list']
+	rc = tb.execute(what, cmdl, kind='install')
+	if rc:
+		pstderr("\n" "warning: failed to restore"
+			" testbed apt cache, exit code %d" % rc)
 	tb._need_reset_apt = False
  def close(tb):
 	tb._debug('close, scratch=%s' % tb.scratch)
