--- a/cowdancer-0.73/cowbuilder.c	2014-01-04 13:31:58.000000000 +0100
+++ b/cowdancer-0.73/cowbuilder.c	2014-01-04 13:33:49.000000000 +0100
@@ -198,13 +198,11 @@
       /* if there was no ilist file in the beginning, that's not a
 	 problem.
        */
-      fprintf(stderr, "I: unlink for ilistfile %s failed, it didn't exist?\n",
-	      ilistfile);
+      printf("I: unlink for ilistfile %s failed, it didn't exist?\n", ilistfile);
     }
   else
     {
-      fprintf(stderr, "I: removed stale ilistfile %s\n",
-	      ilistfile);
+      printf("I: removed stale ilistfile %s\n", ilistfile);
     }
   free(ilistfile);
 
