diff --git a/contrib/init/sysvinit-debian/docker b/contrib/init/sysvinit-debian/docker
index cf33c83..0a4ebba 100755
--- a/contrib/init/sysvinit-debian/docker
+++ b/contrib/init/sysvinit-debian/docker
@@ -38,6 +38,14 @@ if [ -f /etc/default/$BASE ]; then
 	. /etc/default/$BASE
 fi
 
+if [ -n "$http_proxy" ]; then
+	export http_proxy
+fi
+
+if [ -n "$TMPDIR" ]; then
+	export TMPDIR
+fi
+
 # see also init_is_upstart in /lib/lsb/init-functions (which isn't available in Ubuntu 12.04, or we'd use it)
 if [ -x /sbin/initctl ] && /sbin/initctl version 2>/dev/null | grep -q upstart; then
 	log_failure_msg "$DOCKER_DESC is managed via upstart, try using service $BASE $1"
diff --git a/contrib/init/sysvinit-debian/docker.default b/contrib/init/sysvinit-debian/docker.default
index 14e6601..fdb0438 100644
--- a/contrib/init/sysvinit-debian/docker.default
+++ b/contrib/init/sysvinit-debian/docker.default
@@ -7,7 +7,7 @@
 #DOCKER_OPTS="--dns 8.8.8.8 --dns 8.8.4.4"
 
 # If you need Docker to use an HTTP proxy, it can also be specified here.
-#export http_proxy="http://127.0.0.1:3128/"
+#http_proxy="http://127.0.0.1:3128/"
 
 # This is also a handy place to tweak where Docker's temporary files go.
-#export TMPDIR="/mnt/bigdrive/docker-tmp"
+#TMPDIR="/mnt/bigdrive/docker-tmp"
