*** automysqlbackup.orig	2013-02-26 21:19:36.265505513 +0100
--- automysqlbackup	2013-02-26 21:19:13.966198068 +0100
***************
*** 672,678 ****
  	if [ $MAXATTSIZE -ge $ATTSIZE ]
  	then
  		BACKUPFILES=`echo "$BACKUPFILES" | sed -e "s# # -a #g"`	#enable multiple attachments
! 		mutt -s "$ERRORNOTE MySQL Backup Log and SQL Files for $HOST - $DATE" $BACKUPFILES $MAILADDR < $LOGFILE		#send via mutt
  	else
  		cat "$LOGFILE" | mail -s "WARNING! - MySQL Backup exceeds set maximum attachment size on $HOST - $DATE" $MAILADDR
  	fi
--- 672,678 ----
  	if [ $MAXATTSIZE -ge $ATTSIZE ]
  	then
  		BACKUPFILES=`echo "$BACKUPFILES" | sed -e "s# # -a #g"`	#enable multiple attachments
! 		mutt -s "$ERRORNOTE MySQL Backup Log and SQL Files for $HOST - $DATE" $BACKUPFILES -- $MAILADDR < $LOGFILE		#send via mutt
  	else
  		cat "$LOGFILE" | mail -s "WARNING! - MySQL Backup exceeds set maximum attachment size on $HOST - $DATE" $MAILADDR
  	fi
