diff -Nru emacs-goodies-el-36.3+nmu1/elisp/dpkg-dev-el/debian-control-mode.el emacs-goodies-el-36.3+nmu1local1/elisp/dpkg-dev-el/debian-control-mode.el
--- emacs-goodies-el-36.3+nmu1/elisp/dpkg-dev-el/debian-control-mode.el	2017-01-28 03:14:24.000000000 +0100
+++ emacs-goodies-el-36.3+nmu1local1/elisp/dpkg-dev-el/debian-control-mode.el	2017-10-17 12:52:48.000000000 +0200
@@ -181,8 +181,9 @@
 (defvar debian-control-source-fields
   (append
   '("Section" "Priority" "Maintainer" "Build-Depends" "Build-Depends-Indep"
-     "Build-Conflicts" "Build-Conflicts-Indep" "Standards-Version" "Uploaders"
-     "DM-Upload-Allowed" "Homepage" "Vcs-Browser" "XS-Python-Version")
+     "Build-Depends-Arch" "Build-Conflicts" "Build-Conflicts-Indep"
+     "Build-Conflicts-Arch" "Standards-Version" "Uploaders" "Origin" "Bugs"
+     "Homepage" "Vcs-Browser" "Rules-Requires-Root" "XS-Python-Version")
    (mapcar (lambda (elt) (concat "Vcs-" elt))
            debian-control-vcs-names))
   "Valid source package field names, collected from several policy sections.")
