diff -ur cdbs-0.4.56.old/1/rules/buildcore.mk.in cdbs-0.4.56/1/rules/buildcore.mk.in
--- cdbs-0.4.56.old/1/rules/buildcore.mk.in	2009-04-08 21:14:15.000000000 +0200
+++ cdbs-0.4.56/1/rules/buildcore.mk.in	2009-05-18 22:00:15.000000000 +0200
@@ -51,7 +51,7 @@
 
 # This variable is used by tarball.mk, but we want it here in order to check
 # tarball contents before unpacking.  tarball.mk imports this file anyway.
-DEB_TARBALL ?= $(wildcard *.tar *.tgz *.tar.gz *.tar.bz *.tar.bz2 *.zip)
+DEB_TARBALL ?= $(wildcard *.tar *.tgz *.tar.gz *.tar.bz *.tar.bz2 *.zip *.tar.lzma)
 
 ifneq (, $(findstring .bz2, $(DEB_TARBALL)))
 CDBS_BUILD_DEPENDS := $(CDBS_BUILD_DEPENDS), bzip2
@@ -73,6 +73,7 @@
 			*.tar$(close_parenthesis) tar -tf $$i | grep "/config\.[^/]*$$" > $$i.cdbs-config_list ;; \
 			*.tgz|*.tar.gz$(close_parenthesis) tar -tzf $$i | grep "/config\.[^/]*$$" > $$i.cdbs-config_list ;; \
 			*.tar.bz|*.tar.bz2$(close_parenthesis) tar -tjf $$i | grep "/config\.[^/]*$$" > $$i.cdbs-config_list ;; \
+			*.tar.lzma$(close_parenthesis) lzma -d < $$i | tar -tjf - | grep "/config\.[^/]*$$" > $$i.cdbs-config_list ;; \
 			*.zip$(close_parenthesis) unzip -l $$i | grep "/config\.[^/]*$$" > $$i.cdbs-config_list ;; \
 			*$(close_parenthesis) echo Warning: tarball $$i with unknown format >&2 ;; \
 		esac ; \
diff -ur cdbs-0.4.56.old/1/rules/tarball.mk.in cdbs-0.4.56/1/rules/tarball.mk.in
--- cdbs-0.4.56.old/1/rules/tarball.mk.in	2009-04-08 21:14:15.000000000 +0200
+++ cdbs-0.4.56/1/rules/tarball.mk.in	2009-05-18 21:59:23.000000000 +0200
@@ -96,6 +96,10 @@
 	tar -C $(_cdbs_tarball_dir) $(_cdbs_tar_verbose) -x -j -f $(call _cdbs_stampname_to_tarname,$@)
 	touch $@
 
+$(addsuffix .lzma,$(_cdbs_tarball_stamp_base)):
+	lzma -d < $(call _cdbs_stampname_to_tarname,$@) | tar -C $(_cdbs_tarball_dir) $(_cdbs_tar_verbose) -x -f -
+	touch $@
+
 $(addsuffix .zip,$(_cdbs_tarball_stamp_base)):
 	unzip $(call _cdbs_stampname_to_tarname,$@) -d $(_cdbs_tarball_dir)
 	touch $@
