--- debian-security-support/security-support-limited.orig	2018-11-25 08:39:43 +0100
+++ debian-security-support/security-support-limited	2019-04-01 12:14:58 -0400
@@ -17,6 +17,7 @@
 mozjs17         Not covered by security support, only suitable for trusted content
 mozjs24         Not covered by security support, only suitable for trusted content
 ocsinventory-server Only supported behind an authenticated HTTP zone
+qtwebengine-opensource-src No security support upstream and backports not feasible, only for use on trusted content
 qtwebkit        No security support upstream and backports not feasible, only for use on trusted content
 qtwebkit-opensource-src No security support upstream and backports not feasible, only for use on trusted content
 sql-ledger      Only supported behind an authenticated HTTP zone
