diff -urN cryptsetup-1.1.3.orig/debian/cryptdisks-early.init cryptsetup-1.1.3/debian/cryptdisks-early.init
--- cryptsetup-1.1.3.orig/debian/cryptdisks-early.init	2010-06-27 01:03:36.000000000 +0200
+++ cryptsetup-1.1.3/debian/cryptdisks-early.init	2010-07-22 09:24:56.664716922 +0200
@@ -19,7 +19,7 @@
 . /lib/cryptsetup/cryptdisks.functions
 
 INITSTATE="early"
-LOUD=""
+DEFAULT_LOUD=""
 
 case "$CRYPTDISKS_ENABLE" in
 [Nn]*)
diff -urN cryptsetup-1.1.3.orig/debian/cryptdisks.functions cryptsetup-1.1.3/debian/cryptdisks.functions
--- cryptsetup-1.1.3.orig/debian/cryptdisks.functions	2010-07-21 10:23:24.000000000 +0200
+++ cryptsetup-1.1.3/debian/cryptdisks.functions	2010-07-22 10:32:53.336716604 +0200
@@ -549,7 +549,7 @@
 
 	# Make sure source device exists
 	if [ ! -r "$src" ]; then
-		if [ "$LOUD" = "yes" ] || [ "$INITSTATE" != "early" ]; then
+		if [ "$LOUD" = "yes" ]; then
 			device_msg "$dst" "skipped, device $src does not exist"
 			return 1
 		fi
diff -urN cryptsetup-1.1.3.orig/debian/cryptdisks.init cryptsetup-1.1.3/debian/cryptdisks.init
--- cryptsetup-1.1.3.orig/debian/cryptdisks.init	2010-06-26 14:53:56.000000000 +0200
+++ cryptsetup-1.1.3/debian/cryptdisks.init	2010-07-22 09:24:47.744716322 +0200
@@ -19,7 +19,7 @@
 . /lib/cryptsetup/cryptdisks.functions
 
 INITSTATE="remaining"
-LOUD="yes"
+DEFAULT_LOUD="yes"
 
 case "$CRYPTDISKS_ENABLE" in
 [Nn]*)
