diff -urN cryptsetup-1.1.3.orig/debian/cryptdisks.functions cryptsetup-1.1.3/debian/cryptdisks.functions
--- cryptsetup-1.1.3.orig/debian/cryptdisks.functions	2010-07-22 10:32:53.336716604 +0200
+++ cryptsetup-1.1.3/debian/cryptdisks.functions	2010-07-22 09:30:35.749216508 +0200
@@ -21,8 +21,6 @@
 
 MOUNT="$CRYPTDISKS_MOUNT"
 
-LOUD=""
-
 # Parses the option field from the crypttab file
 parse_opts () {
 	local opts opt IFS PARAM VALUE
@@ -41,7 +39,6 @@
 	IGNORE=""
 	CRYPTTAB_OPTIONS=""
 	LOUD="$DEFAULT_LOUD"
-	QUIET=""
 
 	# Parse the options field, convert to cryptsetup parameters
 	# and construct the command line
@@ -152,7 +149,7 @@
 			fi
 			;;
 		noauto)
-			if [ -n "$INITSTATE" ] && [ "$INITSTATE" != "manual" ]; then
+			if [ "$INITSTATE" != "manual" ]; then
 				IGNORE="yes"
 			fi
 			;;
