Description: add a fox 'cow'
Origin: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=888229#5
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=888229



--- /dev/null
+++ b/cows/fox.cow
@@ -0,0 +1,18 @@
+$the_cow = <<EOC;
+         $thoughts     ,-.      .-,     
+          $thoughts    |-.\\ __ /.-|     
+           $thoughts   \\  `    `  /     
+                /_     _ \\      
+              <  _`q  p _  >    
+              <.._=/  \\=_. >    
+                 {`\\()/`}`\\     
+                 {      }  \\    
+                 |{    }    \\   
+                 \\ '--'   .- \\  
+                 |-      /    \\ 
+                 | | | | |     ;
+                 | | |.;.,..__ |
+               .-"";`         `|
+              /    |           /
+              `-../____,..---'`
+EOC
--- a/debian/copyright
+++ b/debian/copyright
@@ -31,6 +31,10 @@
 Copyright: 1999 Geordan Rosario <geordan@csua.berkeley.edu>
 License: COWSAY
 
+Files: cows/fox.cow
+Copyright: 2019 Hansen Wu <hansen.wu.85@gmail.com>
+License: COWSAY
+
 Files: cows/gnu.cow cows/suse.cow
 Copyright: 2006 Gerfried Fuchs <rhonda@deb.at>
 License: WTFPL-2
