Description: Fix glib-2.0 include and link
Author: Angel Abad <angelabad@ubuntu.com>

--- a/Makefile.in
+++ b/Makefile.in
@@ -8,7 +8,8 @@
 
 
 AUDACIOUS_CFLAGS=@AUDACIOUS_CFLAGS@ -I/usr/include/dbus-1.0
-LDADD=-ldbus-glib-1 -lncursesw -laudclient
+GLIB_CFLAGS=`pkg-config --cflags glib-2.0`
+LDADD=-ldbus-glib-1 -lncursesw -laudclient `pkg-config --libs glib-2.0`
 
 SOURCES=main.c curses_printf.c playlist.c playlist_jump.c playlist_addurl.c settings.c connect.c browse.c playlist_create.c
 BINS=$(SOURCES:.c=.o)
@@ -21,7 +22,7 @@
 	cc -g -O2 -g2 -Wall -Werror ${LDFLAGS} -o audtty $(BINS) $(LDADD)
 
 .c.o:
-	cc -g -Wall ${AUDACIOUS_CFLAGS} ${CFLAGS} -o $@ -c $<
+	cc -g -Wall ${AUDACIOUS_CFLAGS} ${GLIB_CFLAGS} ${CFLAGS} -o $@ -c $<
 
 install: audtty
 	mkdir -p ${bindir}
