--- /etc/cron.daily/apt	2011-04-15 23:06:01.000000000 +0200
+++ apt	2012-08-02 10:37:00.184091894 +0200
@@ -307,6 +307,9 @@
 
 # ------------------------ main ----------------------------
 
+# See the matched ")"
+(
+
 # Backup the 7 last versions of APT's extended_states file
 # shameless copy from dpkg cron
 if cd /var/backups ; then
@@ -493,6 +496,9 @@
 # check cache size 
 check_size_constraints
 
+# Make work in background to not lock other crond tasks.
+) &
+
 #
 #     vim: set sts=4 ai :
 #
