diff -u autofs-4.1.4+debian/debian/rules autofs-4.1.4+debian/debian/rules
--- autofs-4.1.4+debian/debian/rules
+++ autofs-4.1.4+debian/debian/rules
@@ -2,7 +2,7 @@
 
 .PHONY: binary binary-arch binary-indep clean checkroot
 
-export CFLAGS=-Wall -g
+export CFLAGS=-Wall -g -DLDAP_DEPRECATED
 
 ifneq (,$(findstring noopt,$(DEB_BUILD_OPTIONS)))
   CFLAGS+=-O0
diff -u autofs-4.1.4+debian/debian/changelog autofs-4.1.4+debian/debian/changelog
--- autofs-4.1.4+debian/debian/changelog
+++ autofs-4.1.4+debian/debian/changelog
@@ -1,3 +1,12 @@
+autofs (4.1.4+debian-2.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * High-urgency upload for RC bugfix.
+  * Add -DLDAP_DEPRECATED to CFLAGS, to fix compatibility with OpenLDAP
+    2.4 on 64-bit architectures.  Closes: #463419.
+
+ -- Steve Langasek <vorlon@debian.org>  Sat, 08 Mar 2008 01:36:09 -0800
+
 autofs (4.1.4+debian-2) unstable; urgency=low
 
   * Add virtual package autofs to provide for smooth transitions
