Index: debian/changelog
===================================================================
RCS file: /cvsroot/pkg-goodies-el/emacs-goodies-el/debian/changelog,v
retrieving revision 1.435
diff -u -r1.435 changelog
--- debian/changelog	4 Sep 2009 02:46:35 -0000	1.435
+++ debian/changelog	5 Sep 2009 21:22:26 -0000
@@ -1,6 +1,7 @@
 emacs-goodies-el (30.6-1) UNRELEASED; urgency=low
 
   * emacs-goodies-el:
+   - cwebm.el: Stub file removed (was replaced with stub in 28.0-1)
    - dirvars.el: Updated to 1.3 (by Benjamin Rutt)
    - folding.el: Updated to 2009.0220.1404
    - highlight-completion.el: Updated to 0.08
@@ -98,6 +99,7 @@
   * emacs-goodies-el:
     - pod-mode.el: updated to V0.502, thanks to Kevin Ryde
       (Closes: #499473).
+    - emacs-goodies-loaddefs.el: rebuilt to remove old cruft
     New file:
     - quack.el: Added file (version 0.34), integration contributed by
       Daniel Moerner <dmoerner@gmail.com>, (Closes: #509486)
Index: debian/control
===================================================================
RCS file: /cvsroot/pkg-goodies-el/emacs-goodies-el/debian/control,v
retrieving revision 1.81
diff -u -r1.81 control
--- debian/control	2 Sep 2009 20:56:27 -0000	1.81
+++ debian/control	5 Sep 2009 21:22:26 -0000
@@ -32,7 +32,6 @@
   ctypes - enhanced Font lock support for custom defined types;
   cua - emulate CUA key bindings (C-z undo, C-x cut, C-c copy, C-v paste);
   cyclebuffer - select buffer by cycling through;
-  cwebm - a modified CWEB/WEB modified mode;
   dedicated - make a window dedicated to a single buffer;
   df - display in the mode line space left on devices;
   dict - wrapper around the 'dict' command. (Depends on bash and dict)
Index: elisp/emacs-goodies-el/cwebm.el
===================================================================
RCS file: /cvsroot/pkg-goodies-el/emacs-goodies-el/elisp/emacs-goodies-el/cwebm.el,v
retrieving revision 1.5
diff -u -r1.5 cwebm.el
--- elisp/emacs-goodies-el/cwebm.el	25 Sep 2007 00:02:52 -0000	1.5
+++ elisp/emacs-goodies-el/cwebm.el	5 Sep 2009 21:22:26 -0000
@@ -1,33 +0,0 @@
-;;; cwebm.el --- CWEB/WEB modified mode
-;;
-;; This file is replacement for the cweb.el from the Levi' CWEB.
-;;
-;; Copyright (C) 1987,1990,1993,2000 Silvio Levy and Donald E. Knuth
-;; Copyright (c) 2002,2003,2004,2005 Max Vasin
-;;
-;; Permission is granted to make and distribute verbatim copies of
-;; this document provided that the copyright notice and this
-;; permission notice are preserved on all copies.
-;;
-;; Permission is granted to copy and distribute modified versions of
-;; this document under the conditions for verbatim copying, provided
-;; that the entire resulting derived work is given a different name
-;; and distributed under the terms of a permission notice identical to
-;; this one.
-
-;;---------------------------------------------------------------------
-;; Important note:
-;;
-;;  I have removed the contents of this file at the request of the FSF
-;;  while the FSF try to get it licensed compatibly with Emacs, which the
-;;  FSF believes it modifies.  If it doesn't get sorted out, I will remove
-;;  the file completely.  If it does get sorted out, I'll replace this with
-;;  the new version.
-;;
-;;  Peter S Galbraith <psg@debian.org>
-;;  2007-09-24
-;;---------------------------------------------------------------------
-
-(provide 'cwebm)
-
-;;; cwebm.el ends here
