From: Emmanouil Maroudas <emmanouil.maroudas@gmail.com>
Date: Sun, 12 Jun 2016 20:41:24 +0300
Subject: show changelog of source packages

---
 src/cmdline/cmdline_changelog.cc | 10 +++++-----
 1 file changed, 5 insertions(+), 5 deletions(-)

diff --git a/src/cmdline/cmdline_changelog.cc b/src/cmdline/cmdline_changelog.cc
index 50e3f88..a68896d 100644
--- a/src/cmdline/cmdline_changelog.cc
+++ b/src/cmdline/cmdline_changelog.cc
@@ -352,11 +352,11 @@ void do_cmdline_changelog(const vector<string> &packages,
 	  // use an explicit version.
 	  if(p.valid())
 	    {
-	      get_changelog_from_source(p.get_package(),
-					p.get_version(),
-					p.get_section(),
-					p.get_package(),
-                                        term_metrics);
+	      filename = get_changelog_from_source(p.get_package(),
+                                                   p.get_version(),
+                                                   p.get_section(),
+                                                   p.get_package(),
+                                                   term_metrics);
 	    }
 	  else
 	    {
