Index: developers-reference-3.4.0/best-pkging-practices.dbk
===================================================================
--- developers-reference-3.4.0.orig/best-pkging-practices.dbk	2008-06-23 20:33:41.000000000 +0700
+++ developers-reference-3.4.0/best-pkging-practices.dbk	2008-06-23 20:37:14.000000000 +0700
@@ -1745,11 +1745,11 @@
 add an entry to <filename>debian/control</filename> for the debug package.
 </para>
 <para>
-Note that the Debian package should depend on the package that it provides
+Note that the debug package should depend on the package that it provides
 debugging symbols for, and this dependency should be versioned.  For example:
 </para>
 <screen>
-Depends: libfoo-dbg (= ${binary:Version})
+Depends: libfoo (= ${binary:Version})
 </screen>
 </section>
 
