diff -u courier-authlib-0.63.0/debian/rules courier-authlib-0.63.0/debian/rules
--- courier-authlib-0.63.0/debian/rules
+++ courier-authlib-0.63.0/debian/rules
@@ -101,7 +101,9 @@
 	dh_installdirs
 
 # Add here commands to install the package into debian/tmp.
-	$(MAKE) DESTDIR=$(BUILD) install 
+	$(MAKE) DESTDIR=$(BUILD) install
+	
+	sed -i "/dependency_libs/ s/'.*'/''/" `find $(BUILD) -name '*.la'`
 
 #	(cd $(BUILD)/usr/lib/courier/courier-authlib/changepwd; ln -fs ../../authsystem.passwd ./authsystem.passwd)
 
diff -u courier-authlib-0.63.0/debian/changelog courier-authlib-0.63.0/debian/changelog
--- courier-authlib-0.63.0/debian/changelog
+++ courier-authlib-0.63.0/debian/changelog
@@ -1,3 +1,10 @@
+courier-authlib (0.63.0-3.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Empty dependency_libs (Closes: #621191).
+
+ -- Luk Claes <luk@debian.org>  Tue, 28 Jun 2011 07:23:00 +0200
+
 courier-authlib (0.63.0-3) unstable; urgency=low
 
   [ Simon McVittie <smcv@debian.org> ]
