diff --git a/beyond-pkging.dbk b/beyond-pkging.dbk
index 121ddb9..08faea7 100644
--- a/beyond-pkging.dbk
+++ b/beyond-pkging.dbk
@@ -217,7 +217,7 @@ address.
 If you notice that a package is lacking maintenance, you should make sure that
 the package maintainer is active and will continue to work on their packages. 
 It is possible that the maintainer is not active anymore, but haven't formally
-orphaned his/her packages. It is also possible that he/she just needs a reminder.
+orphaned their packages. It is also possible that they just needs a reminder.
 </para>
 <para>
 The QA team maintains a simple system, the MIA database, with information about
@@ -242,6 +242,12 @@ important to take into account that real life is sometimes very hectic.  One
 way to handle this would be to send a reminder after two weeks.
 </para>
 <para>
+A non-functional e-mail address is a serious violation of Debian Policy
+(<ulink url="https://www.debian.org/doc/debian-policy/ch-binary.html#s-maintainer">Section 3.3</ulink>).
+If an e-mail "bounces", please file a bug against the package and submit this
+information to the MIA database.
+</para>
+<para>
 If the maintainer doesn't reply within four weeks (a month), one can assume
 that a response will probably not happen.  If that happens, you should
 investigate further and try to gather as much useful information about the
@@ -301,12 +307,13 @@ One last note: please remember to be polite.  We are all volunteers and cannot
 dedicate all of our time to Debian.  Also, you are not aware of the
 circumstances of the person who is involved.  Perhaps they might be seriously
 ill or might even have died â you do not know who may be on the receiving
-side.  Imagine how a relative will feel if they read the e-mail of the deceased
+side. Imagine how a relative will feel if they read the e-mail of the deceased
 and find a very impolite, angry and accusing message! On the other hand,
-although we are volunteers, we do have a responsibility.  So you can stress
-the importance of the greater good â if a package maintainer does not have
-the time or interest anymore, they should let go and give the package to
-someone with more time.
+although we are volunteers, a package maintainer has made a commitment and
+therefore has a responsibility to maintain the package. Therefore you can
+stress the importance of the greater good â if a package maintainer does not
+have the time or interest anymore, they should let go and give the package to
+someone with more time/interest.
 </para>
 <para>
 If you are interested in working in the MIA team (Debian Developers only),
