diff --git usr/bin/checkbashisms tmp/checkbashisms
index 40cffc9..b4a308e 100755
--- usr/bin/checkbashisms
+++ tmp/checkbashisms
@@ -538,7 +538,7 @@ sub init_hashes {
 
     %singlequote_bashisms = (
 	$LEADIN . qr'echo\s+(?:-[^e\s]+\s+)?\'[^\']*(\\[\\abcEfnrtv0])+.*?[\']' => q<unsafe echo with backslash>,
-	$LEADIN . qr'source\s+[\"\']?(?:\.\/|\/|\$|[\w.-])[^\s]+' =>
+	$LEADIN . qr'source\s+[\"\']?(?:\.\/|\/|\$|[\w.~-])[^\s]+' =>
 	                               q<should be '.', not 'source'>,
     );
 
