diff --git a/debian/doomsday-compat b/debian/doomsday-compat
index 05bda09..6efd3eb 100755
--- a/debian/doomsday-compat
+++ b/debian/doomsday-compat
@@ -28,7 +28,7 @@ DOOMSDAY="doomsday"
 next_is_iwad=0
 next_is_pwad=0
 
-iwaddir=""
+iwaddir="$(awk '/^iwaddir/{print $2}' /etc/doomsday/paths)"
 gamemode="doom2"
 
 for arg in "$@"
@@ -59,6 +59,10 @@ do
 		next_is_pwad=0
 		pwadpath=$arg
 		pwadfile=$(basename $arg)
+		if [ ! -f $pwadpath -a -f $iwaddir/$pwadpath ]
+		then
+			pwadpath=$iwaddir/$pwadpath
+		fi
 		continue
 	fi
 
