diff -r -u dvgrab-1.2.0deb.old/avi.cc dvgrab-1.2.0deb/avi.cc
--- dvgrab-1.2.0deb.old/avi.cc	2003-06-13 23:14:22.000000000 -0500
+++ dvgrab-1.2.0deb/avi.cc	2003-06-13 23:17:06.000000000 -0500
@@ -102,6 +102,7 @@
 #include <iostream>
 #include <strstream>
 #include <iomanip>
+#include <cassert>
 
 using std::cout;
 using std::hex;
diff -r -u dvgrab-1.2.0deb.old/filehandler.cc dvgrab-1.2.0deb/filehandler.cc
--- dvgrab-1.2.0deb.old/filehandler.cc	2003-06-13 23:14:22.000000000 -0500
+++ dvgrab-1.2.0deb/filehandler.cc	2003-06-13 23:17:36.000000000 -0500
@@ -25,6 +25,7 @@
 #include <iostream>
 #include <strstream>
 #include <iomanip>
+#include <cassert>
 
 using std::cerr;
 using std::cout;
diff -r -u dvgrab-1.2.0deb.old/riff.cc dvgrab-1.2.0deb/riff.cc
--- dvgrab-1.2.0deb.old/riff.cc	2003-06-13 23:14:22.000000000 -0500
+++ dvgrab-1.2.0deb/riff.cc	2003-06-13 23:18:15.000000000 -0500
@@ -65,6 +65,7 @@
 #include <strstream>
 #include <iomanip>
 #include <byteswap.h>
+#include <cassert>
 
 using std::cout;
 using std::hex;
