Index: resources.dbk
===================================================================
--- resources.dbk	(revision 6982)
+++ resources.dbk	(working copy)
@@ -152,10 +152,8 @@
 speak there of issues that are discussed in
 &email-debian-private;.  There's another channel for this
 purpose, it's called <literal>#debian-private</literal> and it's protected by
-a key.  This key is available in the archives of debian-private in
-<filename>master.debian.org:&file-debian-private-archive;</filename>,
-just <command>zgrep</command> for <literal>#debian-private</literal> in all
-the files.
+a key.  This key is available at
+<filename>master.debian.org:&file-debian-private-key;</filename>.
 </para>
 <para>
 There are other additional channels dedicated to specific subjects.
Index: common.ent
===================================================================
--- common.ent	(revision 6982)
+++ common.ent	(working copy)
@@ -152,6 +152,7 @@
 <!ENTITY file-ocaml-policy "<filename>/usr/share/doc/ocaml/ocaml_packaging_policy.gz</filename>">
 <!ENTITY file-lisp-controller "<filename>/usr/share/doc/common-lisp-controller/README.packaging</filename>">
 <!ENTITY file-debian-private-archive "~debian/archive/debian-private/">
+<!ENTITY file-debian-private-key "~debian/misc/irc-password">
 <!ENTITY file-bpp-autotools "<filename>/usr/share/doc/autotools-dev/README.Debian.gz</filename>">
 
 <!ENTITY cron-bug-report '0 17 * * fri   echo "index maint <replaceable>address</replaceable>" | mail request@&bugs-host;'>
