Index: binfmt-support-2.2.0/man/update-binfmts.man8
===================================================================
--- binfmt-support-2.2.0.orig/man/update-binfmts.man8
+++ binfmt-support-2.2.0/man/update-binfmts.man8
@@ -306,7 +306,7 @@ The
 .Ar credentials ,
 .Ar preserve ,
 and
-.Ar fix\-binary
+.Ar fix_binary
 options correspond to the command-line options of the same names.
 .Sh EXIT STATUS
 .Bl -tag -width 4n
