--- /etc/ssl/openssl.cnf.orig	2019-05-30 11:27:48.000000000 -0400
+++ /etc/ssl/openssl.cnf	2020-09-07 16:02:31.448309714 -0400
@@ -353,6 +353,7 @@
 				# identifier (optional, default: sha1)
 [default_conf]
 ssl_conf = ssl_sect
+engines = engine_section
 
 [ssl_sect]
 system_default = system_default_sect
@@ -360,3 +361,14 @@
 [system_default_sect]
 MinProtocol = TLSv1.2
 CipherString = DEFAULT@SECLEVEL=2
+
+[engine_section]
+pkcs11 = pkcs11_section
+
+[pkcs11_section]
+# https://github.com/openssl/openssl/blob/master/README.ENGINE
+engine_id = pkcs11
+# same as SO_PATH
+dynamic_path = /usr/lib/x86_64-linux-gnu/engines-1.1/pkcs11.so
+MODULE_PATH = opensc-pkcs11.so
+init = 0
