diff -Nru claws-mail-3.13.2/debian/claws-mail.install claws-mail-3.13.2/debian/claws-mail.install
--- claws-mail-3.13.2/debian/claws-mail.install	2016-01-26 12:01:39.000000000 +0100
+++ claws-mail-3.13.2/debian/claws-mail.install	2016-05-02 14:22:44.000000000 +0200
@@ -1,5 +1,5 @@
 usr/bin/claws-mail
-claws-mail.desktop usr/share/applications
+usr/share/applications
 usr/share/man
-claws-mail*.png usr/share/pixmaps/
+usr/share/icons/
 debian/claws-mail-32x32.xpm usr/share/pixmaps/
