diff -Nur dkms-2.2.0.3.orig/dkms_common.postinst dkms-2.2.0.3/dkms_common.postinst
--- dkms-2.2.0.3.orig/dkms_common.postinst	2012-10-18 18:53:35.000000000 +0200
+++ dkms-2.2.0.3/dkms_common.postinst	2012-10-18 18:58:28.000000000 +0200
@@ -282,7 +282,7 @@
             dkms_status=`dkms status -m $NAME -v $VERSION -k $KERNEL $ARCH`
         else
             echo "Module build for the currently running kernel was skipped since the"
-            echo "kernel source for this kernel does not seem to be installed."
+            echo "kernel headers for this kernel do not seem to be installed."
         fi
     fi
 
diff -Nur dkms-2.2.0.3.orig/sample.spec dkms-2.2.0.3/sample.spec
--- dkms-2.2.0.3.orig/sample.spec	2008-07-08 17:19:41.000000000 +0200
+++ dkms-2.2.0.3/sample.spec	2012-10-18 18:58:45.000000000 +0200
@@ -93,7 +93,7 @@
 	else
 		echo -e ""
 		echo -e "Module build for the currently running kernel was skipped since the"
-		echo -e "kernel source for this kernel does not seem to be installed."
+		echo -e "kernel headers for this kernel do not seem to be installed."
 	fi
 fi
 exit 0
