From 33e4556cd4b4372412d0689f6caf798b999b5fd6 Mon Sep 17 00:00:00 2001
From: Clint Adams <clint@debian.org>
Date: Sun, 22 Aug 2021 17:51:24 -0400
Subject: [PATCH] scripts/add-key: use mktemp instead of tempfile

---
 scripts/add-key | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/add-key b/scripts/add-key
index 881a3c13..313719fe 100755
--- a/scripts/add-key
+++ b/scripts/add-key
@@ -23,7 +23,7 @@ cleanup () {
 if echo -n "$1" | egrep -q '^[[:xdigit:]]{40}$'; then
     fpr=$1
     keyserver=${KEYSERVER:=pool.sks-keyservers.net}
-    keyfile=$(tempfile -p newky -d $GNUPGHOME )
+    keyfile=$(mktemp -p $GNUPGHOME newkyXXXXXX)
     echo "Retrieving key $fpr from keyserver $keyserver"
     gpg --keyserver $keyserver --recv-key "$fpr"
     gpg --export "$fpr" > $keyfile
-- 
2.32.0

