diff -up ./Makefile.org ./Makefile
--- ./Makefile.org	2014-04-09 20:05:54.396150182 +0000
+++ ./Makefile	2014-04-09 20:06:06.604147127 +0000
@@ -21,7 +21,7 @@ ifndef TASK
 TASK=Debian-generic
 endif
 ifndef MKISOFS
-ifneq (,$(filter i386 amd64,$(ARCHES)))
+ifneq (,$(filter i386 amd64 hppa,$(ARCHES)))
 export MKISOFS=xorriso
 export MKISOFS_OPTS=-as mkisofs -r -checksum_algorithm_iso md5,sha1
 else
diff -up ./tools/boot/jessie/boot-hppa.org ./tools/boot/jessie/boot-hppa
--- ./tools/boot/jessie/boot-hppa.org	2014-04-09 20:06:34.008140248 +0000
+++ ./tools/boot/jessie/boot-hppa	2014-04-09 20:07:19.500128763 +0000
@@ -51,9 +51,8 @@ fi
 K32=$(basename "$DI_DIR"/vmlinux-*-parisc)
 K64=$(basename "$DI_DIR"/vmlinux-*-parisc64*)
 
-# palo >= 1.91 can load gzip-compressed Linux kernels
-gzip -c "$DI_DIR"/$K32 > "$CDROOT/install"/$K32
-gzip -c "$DI_DIR"/$K64 > "$CDROOT/install"/$K64
+cp "$DI_DIR"/$K32  "$CDROOT/install"/$K32
+cp "$DI_DIR"/$K64  "$CDROOT/install"/$K64
 cp "$DI_DIR"/initrd.gz "$CDROOT/install"
 
 install_languages "$CDROOT"
