Index: faad2-2.5/debian/control
===================================================================
--- faad2-2.5.orig/debian/control	2007-08-20 00:27:55.000000000 +0100
+++ faad2-2.5/debian/control	2007-08-20 00:36:51.000000000 +0100
@@ -2,14 +2,13 @@
 Section: libs
 Priority: optional
 Maintainer: Matthew W. S. Bell <matthew@bells23.org.uk>
-Bugs: mailto:matthew@bells23.org.uk
 Standards-Version: 3.7.2
 Build-Depends: debhelper (>> 4), libsndfile1-dev, xmms-dev, libid3-dev, dpatch, libtool, autotools-dev
 
 Package: libfaad0
 Architecture: any
 Depends: ${shlibs:Depends}
-Conflicts: libfaad2-0 (<< ${Source-Version})
+Conflicts: libfaad2-0 (<< 2.5-1)
 Replaces: libfaad2-0
 Description: freeware Advanced Audio Decoder - runtime files
  FAAD2 is the fastest ISO AAC audio decoder available. FAAD2 correctly
@@ -18,7 +17,7 @@
 
 Package: libfaad2-0
 Architecture: all
-Depends: libfaad0 (>= ${Source-Version})
+Depends: libfaad0 (>= ${binary:Version})
 Description: freeware Advanced Audio Decoder - dummy package
  FAAD2 is the fastest ISO AAC audio decoder available. FAAD2 correctly
  decodes all MPEG-4 and MPEG-2 MAIN, LOW, LTP, LD and ER object type AAC
@@ -30,7 +29,7 @@
 Package: libfaad-dev
 Section: libdevel
 Architecture: any
-Depends: libfaad0 (= ${Source-Version}), libc6-dev | libc-dev
+Depends: libfaad0 (= ${binary:Version}), libc6-dev | libc-dev
 Conflicts: libfaad2-dev
 Replaces: libfaad2-dev
 Description: freeware Advanced Audio Decoder - development files