diff -ur cgoban-1.9.14/wmslib/src/but/but.c cgoban-1.9.14.new/wmslib/src/but/but.c
--- cgoban-1.9.14/wmslib/src/but/but.c	2000-01-24 20:40:25.000000000 -0800
+++ cgoban-1.9.14.new/wmslib/src/but/but.c	2004-02-21 12:43:58.000000000 -0800
@@ -478,12 +478,20 @@
       return(result);
     butWin_turnOffTimers(win);
     win->mapped = FALSE;
+    /*
+     * The following code causes the window to disappear
+     * when one changes the workplace in WindowMaker.
+     * Commenting it out fixes Debian bug #113342
+     * 
+     */
+    /*
     if (win->unmap != NULL)  {
       result |= win->unmap(win);
     }
     if (!win->isIcon && (win->iconWin == NULL))  {
       butWin_dList(win);
     }
+    */
     break;
   case ConfigureNotify:
     win = butEnv_findWin(env, event.xunmap.window);
