From e0e0e11f7ca0fd084c49d72c31ce213f0a1c9f57 Mon Sep 17 00:00:00 2001
From: Sebastian Harl <sh@tokkee.org>
Date: Fri, 7 Nov 2008 14:20:08 +0100
Subject: [PATCH] fai-mirror: Set Dir::Etc to $aptcache/etc/apt/.

---
 bin/fai-mirror |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/bin/fai-mirror b/bin/fai-mirror
index 187d504..29656b6 100755
--- a/bin/fai-mirror
+++ b/bin/fai-mirror
@@ -136,7 +136,7 @@ initialize() {
       -o Dir::Cache=$aptcache/var/cache/apt \
       -o Dir::State=$aptcache/var/cache/apt \
       -o Dir::Cache::Archives=$archivedir \
-      -o Dir::Etc::sourcelist=$aptcache/etc/apt/sources.list \
+      -o Dir::Etc=$aptcache/etc/apt/ \
       -o Dir::State::Lists=$aptcache/var/lib/apt/lists/"
 
     # not needed
-- 
1.6.0.3.523.g304d0

