From 70f3ceea4b3baf3191512b6f3734db7d7cbfa430 Mon Sep 17 00:00:00 2001
From: "FeRD (Frank Dana)" <ferdnyc@gmail.com>
Date: Sat, 13 Nov 2021 01:08:10 -0500
Subject: [PATCH] End stderr output with a newline

---
 chrpath.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/chrpath.c b/chrpath.c
index 207e369..c8580d8 100644
--- a/chrpath.c
+++ b/chrpath.c
@@ -186,7 +186,7 @@ chrpath(const char *filename, const char *newpath, int convert)
 
   if ((int)SHDR_O(sh_size) < rpathoff)
   {
-    fprintf(stderr, "%s string offset not contained in string table",
+    fprintf(stderr, "%s string offset not contained in string table\n",
             elf_tagname(DYNSS(rpath_dyns_index, d_tag)));
     free(strtab);
     free(dyns);
-- 
2.31.1

