diff --git i/data/ostable w/data/ostable
index 7fd2bff08..c1f9d7a7c 100644
--- i/data/ostable
+++ w/data/ostable
@@ -24,6 +24,7 @@ eabihf-gnu-linux	linux-gnueabihf		linux[^-]*-gnueabihf
 eabi-gnu-linux		linux-gnueabi		linux[^-]*-gnueabi
 abin32-gnu-linux	linux-gnuabin32		linux[^-]*-gnuabin32
 abi64-gnu-linux		linux-gnuabi64		linux[^-]*-gnuabi64
+f64-gnu-linux		linux-gnuf64		linux[^-]*-gnuf64
 spe-gnu-linux		linux-gnuspe		linux[^-]*-gnuspe
 x32-gnu-linux		linux-gnux32		linux[^-]*-gnux32
 ilp32-gnu-linux		linux-gnu_ilp32		linux[^-]*-gnu_ilp32
diff --git i/data/tupletable w/data/tupletable
index a7a878f5b..06d2c4e7f 100644
--- i/data/tupletable
+++ w/data/tupletable
@@ -27,6 +27,7 @@ base-musl-linux-<cpu>		musl-linux-<cpu>
 ilp32-gnu-linux-arm64		arm64ilp32
 eabihf-gnu-linux-arm		armhf
 eabi-gnu-linux-arm		armel
+f64-gnu-linux-loong64		loong64
 abin32-gnu-linux-mips64r6el	mipsn32r6el
 abin32-gnu-linux-mips64r6	mipsn32r6
 abin32-gnu-linux-mips64el	mipsn32el
diff --git i/scripts/t/Dpkg_Arch.t w/scripts/t/Dpkg_Arch.t
index 59855dfa4..f9e080f4e 100644
--- i/scripts/t/Dpkg_Arch.t
+++ w/scripts/t/Dpkg_Arch.t
@@ -16,7 +16,7 @@
 use strict;
 use warnings;
 
-use Test::More tests => 18900;
+use Test::More tests => 18907;
 
 use_ok('Dpkg::Arch', qw(debarch_to_debtuple debarch_to_multiarch
                         debarch_eq debarch_is debarch_is_wildcard
