Index: tools/sort_deps
===================================================================
--- tools/sort_deps	(revision 1983)
+++ tools/sort_deps	(working copy)
@@ -34,8 +34,8 @@
 
 $| = 1; # Autoflush for debugging
 
-open(LOG, ">$dir/log.sort_deps.$arch")
-       || die "Can't write in $dir/log.sort_deps.$arch !\n";
+open(LOG, ">$dir/sort_deps.$arch.log")
+       || die "Can't write in $dir/sort_deps.$arch.log !\n";
 
 sub msg {
 	my $level = shift;
Index: tools/start_new_disc
===================================================================
--- tools/start_new_disc	(revision 1983)
+++ tools/start_new_disc	(working copy)
@@ -271,13 +271,13 @@
         echo >> $CDDIR/README.excluded
     fi
 
-    grep "refused because of toobig" $BDIR/log.sort_deps* | sort -u | awk '{print $1}' > $BDIR/toobig
+    grep "refused because of toobig" $BDIR/sort_deps*.log | sort -u | awk '{print $1}' > $BDIR/toobig
     if [ -s $BDIR/toobig ] ; then
         echo "  Adding README.excluded (some package files were too big)"
         echo "For size reasons, the following packages were excluded from this disc set:" >> $CDDIR/README.excluded
         echo >> $CDDIR/README.excluded
         cat $BDIR/toobig >> $CDDIR/README.excluded
-        grep "Can't add.*toobig" $BDIR/log.sort_deps* | sort -u | awk '{print $3}' > $BDIR/toobig-deps
+        grep "Can't add.*toobig" $BDIR/sort_deps*.log | sort -u | awk '{print $3}' > $BDIR/toobig-deps
         if [ -s $BDIR/toobig-deps ] ; then
             echo >> $CDDIR/README.excluded
             echo "and that caused the following packages to be also excluded because" >> $CDDIR/README.excluded
