diff -Nru dh-autoreconf-1/debian/changelog dh-autoreconf-1ubuntu1/debian/changelog
--- dh-autoreconf-1/debian/changelog	2010-04-17 15:18:49.000000000 +0200
+++ dh-autoreconf-1ubuntu1/debian/changelog	2010-06-22 10:34:56.000000000 +0200
@@ -1,3 +1,10 @@
+dh-autoreconf (2) unstable; urgency=low
+
+  * dh_autoreconf:
+    - fix issue in find | xargs when files contain spaces (Closes: #586755)
+
+ -- Didier Roche <didrocks@ubuntu.com>  Tue, 22 Jun 2010 10:34:56 +0200
+
 dh-autoreconf (1) unstable; urgency=low
 
   * Initial release (Closes: #575938).
diff -Nru dh-autoreconf-1/dh_autoreconf dh-autoreconf-1ubuntu1/dh_autoreconf
--- dh-autoreconf-1/dh_autoreconf	2010-04-17 15:10:31.000000000 +0200
+++ dh-autoreconf-1ubuntu1/dh_autoreconf	2010-06-22 10:32:10.000000000 +0200
@@ -70,7 +70,7 @@
 }
 
 my %modes = (
-    'md5' => "find $find_options -type f | xargs md5sum",
+    'md5' => "find $find_options -type f -exec md5sum {} \\;",
     'timesize' => "find $find_options -type f -printf \"%s|%T@  %p\n\""
 );
 
