--- console-setup.orig/debian/rules	2006-10-19 21:23:54.000000000 +0100
+++ console-setup/debian/rules	2007-02-23 15:48:19.000000000 +0000
@@ -6,8 +6,8 @@
 export xkbdir
 
 main_build:
-	$(MAKE)
-	cd Keyboard && $(MAKE) build-mini
+	$(MAKE) xkbdir=$(xkbdir)
+	cd Keyboard && $(MAKE) build-mini xkbdir=$(xkbdir)
 	touch main_build
 
 debian/console-setup.config debian/console-setup-mini.config: debian/config.proto main_build
