--- blhc	2024-04-20 15:10:28.965108347 -0300
+++ blhc-new	2024-04-20 15:10:38.509055996 -0300
@@ -554,7 +554,7 @@
     }
 
     if (not (index($line, 'checking if you want to see long compiling messages... no') == 0
-                or $line =~ /^\s*\[?(?:CC|CCLD|C\+\+|CXX|CXXLD|LD|LINK)\]?\s+(.+?)$/
+                or $line =~ /^\s*\[?(?:CC|CCLD|C\+\+|CXX|CXXLD|LD|LINK)\]?\s+([^:]+?)$/
                 or $line =~ /^\s*[][\/0-9 ]*[Cc]ompiling\s+(.+?)(?:\.\.\.)?$/
                 or $line =~ /^\s*[Bb]uilding (?:program|shared library)\s+(.+?)$/
                 or $line =~ /^\s*\[[\d ]+%\] Building (?:C|CXX) object (.+?)$/)) {
