diff -rup mountmedia-0.23.orig/mountmedia mountmedia-0.23/mountmedia
--- mountmedia-0.23.orig/mountmedia	2012-11-23 20:49:31.000000000 +0100
+++ mountmedia-0.23/mountmedia	2013-11-24 09:52:57.020072623 +0100
@@ -27,13 +27,19 @@ media_mounted() {
 	mount | cut -d' ' -f3 | grep -q "^$MNT$"
 }
 
+get_fstype ()
+{
+	/sbin/blkid -s TYPE -o value $1 2>/dev/null
+}
+
 try_mount() {
 	# TO REMOVE, there is a bug somewhere in the kernel, the first
 	# mount command fail when changing floppy disk
 	# so we have to launch mount twice
-	mount $1 -tauto $MNT || true
+	fs_type=$(get_fstype $1)
+	mount -t $fs_type $1 $MNT || true
 	umount $MNT || true
-	mount $1 -tauto $MNT
+	mount -t $fs_type $1 $MNT
 	media_mounted && checkcontents $MNT
 }
 
