--- dpkg.old/scripts/dpkg-buildpackage.sh	2007-08-27 23:23:28.000000000 +0100
+++ dpkg-1.14.5/scripts/dpkg-buildpackage.sh	2007-08-28 20:06:39.000000000 +0100
@@ -84,6 +84,18 @@
 passopts=''
 admindir=''
 
+DPKGCROSSENABLE=0
+if [ -f /usr/share/dpkg-cross/buildcross ]; then
+ . /usr/share/dpkg-cross/buildcross
+ DPKGCROSSENABLE=1
+fi
+
+function enhance_cross {
+	if [ $DPKGCROSSENABLE -gt 0 ]; then
+		setup_cross
+	fi
+}
+
 while [ $# != 0 ]
 do
 	value="`echo x\"$1\" | sed -e 's/x--.*=//;s/^x-.//'`"
@@ -104,7 +116,7 @@
 	-us)	signsource=: ;;
 	-uc)	signchanges=: ;;
 	-ap)	usepause="true";;
-	-a*)    targetarch="$value"; checkbuilddep=false ;;
+	-a*)    targetarch="$value"; checkbuilddep=false; enhance_cross ;;
 	-si)	sourcestyle=-si ;;
 	-sa)	sourcestyle=-sa ;;
 	-sd)	sourcestyle=-sd ;;
