diff -urN btrfs-progs-xnox/debian/btrfs-progs.examples btrfs-progs-sten/debian/btrfs-progs.examples
--- btrfs-progs-xnox/debian/btrfs-progs.examples	1969-12-31 19:00:00.000000000 -0500
+++ btrfs-progs-sten/debian/btrfs-progs.examples	2016-05-28 19:50:15.750928562 -0400
@@ -0,0 +1 @@
+debian/local/btrfs-convert.hook
diff -urN btrfs-progs-xnox/debian/rules btrfs-progs-sten/debian/rules
--- btrfs-progs-xnox/debian/rules	2016-05-28 19:46:20.907183008 -0400
+++ btrfs-progs-sten/debian/rules	2016-05-28 19:51:39.934837184 -0400
@@ -28,5 +28,10 @@
 	install -D -m 0755 debian/local/btrfs.hook debian/btrfs-progs/usr/share/initramfs-tools/hooks/btrfs
 	install -D -m 0755 debian/local/btrfs.local-premount debian/btrfs-progs/usr/share/initramfs-tools/scripts/local-premount/btrfs
 
+# divert btrfs-convert to btrfs-convert.real
+# and use a wrapper script to notify it is dangerous
+	mv debian/btrfs-progs/bin/btrfs-convert debian/btrfs-progs/bin/btrfs-convert.real
+	install -D -m 0755 debian/local/btrfs-convert.wrapper debian/btrfs-progs/bin/btrfs-convert
+
 # Needs autopkgtest
 override_dh_auto_test:
