--- policy-orig.sgml	2010-06-16 18:41:50.000000000 -0400
+++ policy.sgml	2010-06-16 18:42:51.000000000 -0400
@@ -1753,7 +1753,7 @@
 		The <tt>build</tt> target should perform all the
 		configuration and compilation of the package.
 		If a package has an interactive pre-build
-		configuration routine, the Debianized source package
+		configuration routine, the Debianised source package
 		must either be built after this has taken place (so
 		that the binary package can be built without rerunning
 		the configuration) or the configuration routine
@@ -2954,7 +2954,7 @@
 	          <var>upstream_version</var> may not contain a hyphen.
 	          This format represents the case where a piece of
 	          software was written specifically to be turned into a
-	          Debian package, and so there is only one "debianisation"
+	          Debian package, and so there is only one "Debianisation"
 	          of it and therefore no revision indication is required.
 	        </p>
 
@@ -10286,7 +10286,7 @@
 	    The source packaging tools manage the changes between the
 	    original and Debianised source using <prgn>diff</prgn> and
 	    <prgn>patch</prgn>.  Turning the original source tree as
-	    included in the <file>.orig.tar.gz</file> into the debianised
+	    included in the <file>.orig.tar.gz</file> into the Debianised
 	    source must not involve any changes which cannot be
 	    handled by these tools.  Problematic changes which cause
 	    <prgn>dpkg-source</prgn> to halt with an error when
