diff -ru debarchiver-0.9.10.pristine/debian/control debarchiver-0.9.10/debian/control
--- debarchiver-0.9.10.pristine/debian/control	2011-11-02 21:29:40.000000000 +0000
+++ debarchiver-0.9.10/debian/control	2012-12-07 17:39:21.838723944 +0000
@@ -11,13 +11,13 @@
 Depends: opalmod (>= 0.1.10), apt-utils | dpkg-dev, adduser, ${misc:Depends}
 Recommends: exim4 | mail-transport-agent | mailutils | bsd-mailx | mailx
 Suggests: devscripts, gnupg
-Description: Tool to handle Debian package archives
- This tool can create a potato like file structure that dselect,
- apt-get and similar tools can use for easier installation.
+Description: tool to handle Debian package archives
+ This tool can sort packages into a directory structure that apt-get
+ and similar tools can use as an installation source (arranged in an
+ old-style simple repository layout without package pools). It is meant
+ to be used by local administrators who need special packages, or
+ tweaked versions to ease administration.
  .
- You just have to place the package files into a incoming
- directory and the tool does the sorting (if you place the
- generated '.changes' file there too).
- .
- OBSERVE! This package will create a cronjob that does the
- actual sorting.
+ Package files placed in debarchiver's incoming directory along with the
+ corresponding generated ".changes" file will be sorted by a cronjob
+ into the appropriate directory of the repository.
