--- /lib/udev/bridge-network-interface~	2021-02-16 13:01:37.000000000 +0100
+++ /lib/udev/bridge-network-interface	2021-02-21 00:24:11.936523677 +0100
@@ -31,10 +31,10 @@
 				if [ -d /sys/class/net/$port ]; then
 					ifup --allow auto $i
 					if [ -f /proc/sys/net/ipv6/conf/$port/disable_ipv6 ]; then echo 1 > /proc/sys/net/ipv6/conf/$port/disable_ipv6;fi
-					brctl addif $i $port && ip link set dev $port up &&
 					if [ "$(ifquery "$i"|sed -n -e's/^bridge[_-]hw: //p')" = "$port" ]; then
 						ip link set dev "$i" address "$(ip link show dev "$port" 2>/dev/null|sed -n "s|.*link/ether \([^ ]*\) brd.*|\1|p")"
 					fi
+					brctl addif $i $port && ip link set dev $port up
 				fi
 				break
 				;;
