diff --git i/scripts/Dpkg/Source/Patch.pm w/scripts/Dpkg/Source/Patch.pm
index 0da35b9e5..99020885b 100644
--- i/scripts/Dpkg/Source/Patch.pm
+++ w/scripts/Dpkg/Source/Patch.pm
@@ -513,6 +513,11 @@ sub analyze {
 	    error(g_("diff '%s' patches something which is not a plain file"),
 	          $diff);
 	}
+        my $nlink = (stat _)[3];
+        if ($nlink > 1) {
+            warning(g_("diff '%s' patches hard link %s which can have " .
+                       "unintended consequences"), $diff, $fn);
+        }
 
 	if ($filepatched{$fn}) {
             $filepatched{$fn}++;
