diff --git a/debdelta b/debdelta
index 0d23ff1..8be8c27 100755
--- a/debdelta
+++ b/debdelta
@@ -94,6 +94,7 @@ minibzip2='/usr/lib/debdelta/minibzip2'
 ####################################################################
 
 import sys , os , tempfile , string ,getopt , tarfile , shutil , time, hashlib, traceback, ConfigParser
+from subprocess import *
 
 from stat    import ST_SIZE, ST_MTIME, ST_MODE, S_IMODE, S_IRUSR, S_IWUSR, S_IXUSR 
 from os.path import abspath
@@ -380,11 +381,12 @@ def prepare_for_echo__(s):
 
 def apply_prepare_for_echo(shell,repres):
     a=ECHO_TEST  + " $echo '" + repres +  "' \n exit "
-    o,i=os.popen2(shell)
-    o.write(a)
-    o.close()
-    a=i.read()
+    p = Popen([shell], shell=True, stdin=PIPE, stdout=PIPE, close_fds=True)
+    (o, i) = (p.stdout, p.stdin)
+    i.write(a)
     i.close()
+    a=o.read()
+    o.close()
     return a
 
 #ack! I wanted to use 'dash' as preferred shell, but bug 379227 stopped me
