--- apt-file-2.0.3/apt-file.1.sgml.orig	2004-03-15 01:40:39.000000000 +0200
+++ apt-file-2.0.3/apt-file.1.sgml	2005-07-28 14:38:17.540812320 +0300
@@ -45,12 +45,12 @@
 	<listitem>
 	  <para>
 	    Resynchronize the package contents from their sources. The
-	    contents packages are fetched from the location(s)
+	    lists of the contents of packages are fetched from the location(s)
 	    specified in
 	    <filename>/etc/apt/sources.list</filename>. This command
-	    attempt to fetch the
-	    <filename>Contents-&lt;ARCH&gt;.gz</filename> file from
-	    remote source.
+	    attempts to fetch the
+	    <filename>Contents-&lt;ARCH&gt;.gz</filename> files from
+	    remote sources.
 	  </para>
 	</listitem>
       </varlistentry>
@@ -59,8 +59,8 @@
 	<term>search</term>
 	<listitem>
 	  <para>
-	    Search in which package file is included. A list of all
-	    package containing the <literal>pattern</literal> pattern
+	    Search in which package a file is included. A list of all
+	    packages containing the <literal>pattern</literal> pattern
 	    is returned.
 	  </para>
 	</listitem>
@@ -169,7 +169,7 @@
 	  <para>
 	  Sets architecture to <literal>architecture</literal>. This
 	  option is useful if you search a package for a different
-	  architecture the one installed on your system.
+	  architecture from the one installed on your system.
 	  </para>
 	</listitem>
       </varlistentry>
@@ -182,7 +182,7 @@
 	<listitem>
 	  <para>
 	    Sets the <filename>sources.list</filename> file to a
-	    different value its default
+	    different value from its default
 	    <filename>/etc/apt/sources.list</filename>.
 	  </para>
 	</listitem>
@@ -240,8 +240,8 @@
       proxies.
     </para>
     <para>
-      A string expension is done on severa value. See <literal>string
-	expension</literal> section.
+      A string expension is done on several values. See the <literal>string
+	expansion</literal> section.
     </para>
     <variablelist>
       <varlistentry>
@@ -256,19 +256,19 @@
 	<term>http | ftp | ssh | rsh | file | cdrom</term>
 	<listitem>
 	  <para>
-	    Here are the commands used to fetch files.
+	    Define are the commands used to fetch files.
 	  </para>
 	</listitem>
       </varlistentry>
     </variablelist>
     <refsect2>
-      <title>String expension</title>
+      <title>String expansion</title>
       <para>
 	A <filename>sources.list</filename> entry is defined as:
 	<screen>
-	  deb uri dist componant1 componant2 ...
+	  deb uri dist component1 component2 ...
 	</screen>
-	an uri, is defined as:
+	A <literal>uri</literal> is defined as:
 	<screen>
 	  proto:/[/][user[:password]@]host[:port][/path]
 	</screen>
@@ -328,7 +328,7 @@
 	  <term>&lt;dest&gt;</term>
 	  <listitem>
 	    <para>
-	      replace with <literal>destination</literal> expended
+	      replace with <literal>destination</literal> expanded
 	      value.
 	    </para>
 	  </listitem>
@@ -354,14 +354,14 @@
       <varlistentry>
 	<term><filename>/etc/apt/sources.list</filename></term>
 	<listitem>
-	  <para>Locations to fetch packages contents from.</para>
+	  <para>Locations to fetch package contents from.</para>
 	</listitem>
       </varlistentry>
       <varlistentry>
 	<term><filename>/etc/apt/apt-file.conf</filename></term>
 	<listitem>
 	  <para>
-	    Configuration file for <command></command>apt-file.
+	    Configuration file for <command>apt-file</command>.
 	  </para>
 	</listitem>
       </varlistentry>
@@ -384,7 +384,7 @@
   </refsect1>
   <refsect1>
     <title>BUGS</title>
-    <para>cdrom backend haddn't been tested.</para>
+    <para>cdrom backend hadn't been tested.</para>
     <para>
       Non-release line in sources.list is not handled by apt-file.
     </para>
