From 25363f5d31cf5d909801b00d1e733da156f77104 Mon Sep 17 00:00:00 2001
From: Francesca Ciceri <madamezou@yahoo.it>
Date: Sun, 8 Apr 2012 14:06:22 +0200
Subject: [PATCH] typo fixed in manpage, closes #657684

---
 doc/debtags-man-hooks |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/doc/debtags-man-hooks b/doc/debtags-man-hooks
index cdefeaf..0e22a26 100644
--- a/doc/debtags-man-hooks
+++ b/doc/debtags-man-hooks
@@ -12,7 +12,7 @@ DESCRIPTION end
 |rm\fP, and submitted to the Debtags database with \fBsubmit\fP.
 |.P
 |The \fBupdate\fP command can be used to download extra tags as listed in
-|\fI/etc/debtags/sources/list\fP: see the contents of that file for more
+|\fI/etc/debtags/sources.list\fP: see the contents of that file for more
 |information.
 
 OPTIONS end
-- 
1.7.9.5

