--- update-xmlcatalog.orig	2008-10-08 00:20:48.000000000 +0200
+++ update-xmlcatalog	2008-10-08 19:01:14.000000000 +0200
@@ -121,7 +121,7 @@
 use strict;
 
 ## ----------------------------------------------------------------------
-use File::Spec::Functions;
+use File::Spec;
 use Getopt::Long;
 
 ## ----------------------------------------------------------------------
@@ -196,7 +196,7 @@
     {
 	if ( defined( $package ) )
 	{
-	    my $catalog = catfile( $catalog_dir, "$package.xml" );
+	    my $catalog = File::Spec->catfile( $catalog_dir, "$package.xml" );
 	    if ( ! -f $catalog )
 	    {
 		print STDERR "$name: error: package catalog $catalog not found\n";
@@ -261,7 +261,7 @@
 {
     if ( defined( $root ) )
     {
-	my $catalog = catfile( $catalog_dir, 'catalog' );
+	my $catalog = File::Spec->catfile( $catalog_dir, 'catalog' );
 	if ( ! -f $catalog )
 	{
 	    print STDERR "$name: error: root catalog $catalog not found\n";
@@ -275,7 +275,7 @@
     }
     elsif ( defined( $package ) )
     {
-	my $catalog = catfile( $catalog_dir, "$package.xml" );
+	my $catalog = File::Spec->catfile( $catalog_dir, "$package.xml" );
 	if ( ! -f $catalog )
 	{
 	    print STDERR "$name: error: package catalog $catalog not found\n";
@@ -344,8 +344,8 @@
 if ( defined( $root ) )
 {
     $catalog = 'catalog';
-    $catalog_data = catfile( $catalog_data_dir, $catalog );
-    $catalog = catfile( $catalog_dir, $catalog );
+    $catalog_data = File::Spec->catfile( $catalog_data_dir, $catalog );
+    $catalog = File::Spec->catfile( $catalog_dir, $catalog );
     my $start = $type;
     $start .= 'Id' unless $type eq 'uri';
     $start .= 'StartString';
@@ -358,8 +358,8 @@
 }
 elsif ( defined( $package ) )
 {
-    $catalog_data = catfile( $catalog_data_dir, $package );
-    $catalog = catfile( $catalog_dir, "$package.xml" );
+    $catalog_data = File::Spec->catfile( $catalog_data_dir, $package );
+    $catalog = File::Spec->catfile( $catalog_dir, "$package.xml" );
     my $start = $type;
     $start .= 'Id' unless $type eq 'uri';
     $start .= 'StartString';
@@ -375,7 +375,7 @@
     $catalog = $local;
     $catalog_data = $local;
     $catalog_data =~ tr|/|_|;
-    $catalog_data = catfile( $catalog_data_dir, $catalog_data );
+    $catalog_data = File::Spec->catfile( $catalog_data_dir, $catalog_data );
     my $start = ( $type eq 'uri' ) ? 'name' : $type;
     $start .= 'Id' unless $type eq 'uri';
     $id = "$start=\"$id\"";
