--- consolation.Orig	2018-01-20 16:09:04.708448392 +0100
+++ consolation	2018-01-20 16:15:18.366830577 +0100
@@ -47,9 +47,8 @@
 	#   2 if daemon could not be started
 	start-stop-daemon --start --quiet --pidfile $PIDFILE --exec $DAEMON --test > /dev/null \
 		|| return 1
-	start-stop-daemon --start --quiet --pidfile $PIDFILE --exec $DAEMON -- \
-		$DAEMON_OPTS \
-		|| return 2
+	start-stop-daemon --start --quiet --pidfile $PIDFILE --make-pidfile --background \
+		--exec $DAEMON -- --no-daemon $DAEMON_OPTS || return 2
 }
 
 #
