diff -Nru bash-4.3/debian/rules bash-4.3/debian/rules
--- bash-4.3/debian/rules	2015-01-28 16:55:12.000000000 +0000
+++ bash-4.3/debian/rules	2015-05-20 03:24:37.000000000 +0000
@@ -43,7 +43,9 @@
 CFLAGS := $(shell $(dpkg_buildflags) --get CFLAGS)
 CPPFLAGS := $(shell $(dpkg_buildflags) --get CPPFLAGS)
 LDFLAGS := $(shell $(dpkg_buildflags) --get LDFLAGS)
-
+ifeq ($(shell $(CC) -v 2>&1 | grep -c "clang version"), 1)
+   LDFLAGS += -fuse-ld=gold
+endif

 SHELL	= /bin/bash
 YACC	= bison -y
 
