--- ccc-analyzer	2009-08-14 20:20:50.000000000 +0200
+++ /usr/bin/ccc-analyzer	2010-02-26 18:27:03.000000000 +0100
@@ -395,7 +395,7 @@
 
 # Determine what clang-cc executable to use.
 my $ClangCC = $ENV{'CLANG_CC'};
-if (!defined $ClangCC) { $ClangCC = 'clang-cc'; }
+if (!defined $ClangCC) { $ClangCC = '/usr/lib/clang/1.0/clang-cc'; }
 
 # Get the HTML output directory.
 my $HtmlDir = $ENV{'CCC_ANALYZER_HTML'};
--- scan-build	2009-08-04 19:05:18.000000000 +0200
+++ /usr/bin/scan-build	2010-02-26 18:19:46.000000000 +0100
@@ -82,7 +82,7 @@
 ##----------------------------------------------------------------------------##
 
 # First, look for 'clang-cc' in libexec.
-my $ClangCCSB = Cwd::realpath("$RealBin/libexec/clang-cc");
+my $ClangCCSB = Cwd::realpath("/usr/lib/clang/1.0/clang-cc");
 # Second, look for 'clang-cc' in the same directory as scan-build.
 if (!defined $ClangCCSB || ! -x $ClangCCSB) {
   $ClangCCSB = Cwd::realpath("$RealBin/clang-cc");
