--- a/examples/apache.conf
+++ b/examples/apache.conf
@@ -13,6 +13,8 @@
     #  RewriteLog /org/bugs.debian.org/apache-rewrite.log
     #  RewriteLogLevel 0
      
+    RewriteCond %{REQUEST_URI} ^/(Access\.html|Developer\.html|Reporting\.html|server-request\.html|server-control\.html|server-refcard\.html).* [NC]
+    RewriteRule .* - [L]
     RewriteRule ^/$ http://www.debian.org/Bugs/
     RewriteRule ^/(robots\.txt|release-critical|apt-listbugs\.html)$ - [L]
     # The following two redirect to up-to-date pages
