diff -ur apt-0.7.11/cmdline/apt-cache.cc apt-0.7.11.new/cmdline/apt-cache.cc
--- apt-0.7.11/cmdline/apt-cache.cc	2007-08-02 00:48:14.000000000 +0200
+++ apt-0.7.11.new/cmdline/apt-cache.cc	2008-02-19 16:09:17.000000000 +0100
@@ -244,7 +244,7 @@
 bool Stats(CommandLine &Cmd)
 {
    pkgCache &Cache = *GCache;
-   cout << _("Total package names : ") << Cache.Head().PackageCount << " (" <<
+   cout << _("Total package names: ") << Cache.Head().PackageCount << " (" <<
       SizeToStr(Cache.Head().PackageCount*Cache.Head().PackageSz) << ')' << endl;
 
    int Normal = 0;
@@ -292,7 +292,7 @@
    
    cout << _("Total distinct versions: ") << Cache.Head().VersionCount << " (" <<
       SizeToStr(Cache.Head().VersionCount*Cache.Head().VersionSz) << ')' << endl;
-   cout << _("Total Distinct Descriptions: ") << Cache.Head().DescriptionCount << " (" <<
+   cout << _("Total distinct descriptions: ") << Cache.Head().DescriptionCount << " (" <<
       SizeToStr(Cache.Head().DescriptionCount*Cache.Head().DescriptionSz) << ')' << endl;
    cout << _("Total dependencies: ") << Cache.Head().DependsCount << " (" << 
       SizeToStr(Cache.Head().DependsCount*Cache.Head().DependencySz) << ')' << endl;
diff -ur apt-0.7.11/cmdline/apt-get.cc apt-0.7.11.new/cmdline/apt-get.cc
--- apt-0.7.11/cmdline/apt-get.cc	2008-01-08 22:07:36.000000000 +0100
+++ apt-0.7.11.new/cmdline/apt-get.cc	2008-02-20 09:43:53.000000000 +0100
@@ -2618,7 +2618,7 @@
       "   upgrade - Perform an upgrade\n"
       "   install - Install new packages (pkg is libc6 not libc6.deb)\n"
       "   remove - Remove packages\n"
-      "   autoremove - Remove all automatic unused packages\n"
+      "   autoremove - Remove automatically all unused packages\n"
       "   purge - Remove and purge packages\n"
       "   source - Download source archives\n"
       "   build-dep - Configure build-dependencies for source packages\n"
