diff --git a/debian/control b/debian/control
index a7f07a1..46557ef 100644
--- a/debian/control
+++ b/debian/control
@@ -69,6 +69,7 @@ Recommends: at,
             unzip,
             wdiff,
             xz-utils,
+            file,
             ${vendor:Recommends}
 Suggests: bsd-mailx | mailx,
           build-essential,
diff --git a/scripts/bts.pl b/scripts/bts.pl
index 70f8987..c8fc757 100755
--- a/scripts/bts.pl
+++ b/scripts/bts.pl
@@ -38,7 +38,7 @@ package main;
 
 =head1 NAME
 
-bts - developers' command line interface to the BTS
+bts - command line interface to the BTS
 
 =cut
 
@@ -186,12 +186,11 @@ B<bts> [I<options>] I<command> [I<args>] [B<#>I<comment>] [B<.>|B<,> I<command>
 =head1 DESCRIPTION
 
 This is a command line interface to the Debian Bug Tracking System
-(BTS), intended mainly
-for use by developers. It lets the BTS be manipulated using simple commands
-that can be run at the prompt or in a script, does various sanity checks on
-the input, and constructs and sends a mail to the BTS control address for
-you. A local cache of web pages and e-mails from the BTS may also be
-created and updated.
+(BTS), originally written for developers, but usable by anyone.
+It lets the BTS be manipulated using simple commands that can be run at the
+prompt or in a script, does various sanity checks on the input, and constructs
+and sends a mail to the BTS control address for you. A local cache of web pages
+and e-mails from the BTS may also be created and updated.
 
 In general, the command line interface is the same as what you would write
 in a mail to control@bugs.debian.org, just prefixed with "bts". For
@@ -199,7 +198,7 @@ example:
 
  % bts severity 69042 normal
  % bts merge 69042 43233
- % bts retitle 69042 blah blah
+ % bts retitle 69042 "new title"
 
 A few additional commands have been added for your convenience, and this
 program is less strict about what constitutes a valid bug number. For example,
