--- lib/debian.rb	2010-08-26 08:25:15.000000000 -0700
+++ /usr/lib/ruby/1.8/debian.rb	2011-09-04 01:48:49.000000000 -0700
@@ -115,7 +115,10 @@
 
     def deb?(debfile)
       begin
-	Debian::Ar.new(debfile).open("debian-binary").read == DEBFORMAT_VERSION
+        f = Debian::Ar.new(debfile)
+        res = (f.open("debian-binary").read == DEBFORMAT_VERSION)
+        f.close
+        return res
       rescue NameError, Debian::ArError
 	false
       end
@@ -568,6 +571,7 @@
 	  @data.push(line.chomp)
 	}
       }
+      @artab.close
       freeze
     end
     def control= (c); @control = c; end
