--- /usr/share/acpi-support/policy-funcs.orig	2014-08-07 02:31:20.615414658 +0200
+++ /usr/share/acpi-support/policy-funcs	2014-08-07 02:19:29.994837968 +0200
@@ -64,5 +64,5 @@
 # for (We can't check the MediaKeys plugin, since that has been around for
 # longer).
 GnomeSettingsDaemonPowerRunning() {
-	DBusSend gnome-settings-daemon org.gnome.SettingsDaemon /org/gnome/SettingsDaemon/Power org.freedesktop.DBus.Introspectable.Introspect | grep -q 'interface name="org.gnome.SettingsDaemon.Power"'
+	DBusSend gnome-settings- org.gnome.SettingsDaemon /org/gnome/SettingsDaemon/Power org.freedesktop.DBus.Introspectable.Introspect | grep -q 'interface name="org.gnome.SettingsDaemon.Power"'
 }
