--- a/lib/dhelp/exporter/html.rb
+++ b/lib/dhelp/exporter/html.rb
@@ -83,7 +83,9 @@
       # Variables for the template
       @supported_formats = Dhelp::SUPPORTED_FORMATS
       @section_title     = capitalized_section
-      @item_list         = item_list
+      @item_list         = item_list.sort{ |a,b|
+        a.title.downcase <=> b.title.downcase
+      }
       @doc_path_prefix   = "../" *
         # also count "HTML/"
         (section.split('/').size + 1)
