diff -Naur a56.orig/a56-1.3+dfsg/debian/changelog a56/a56-1.3+dfsg/debian/changelog
--- a56.orig/a56-1.3+dfsg/debian/changelog	2014-02-23 18:38:36.763119344 -0600
+++ a56/a56-1.3+dfsg/debian/changelog	2014-02-23 18:41:17.771122148 -0600
@@ -1,3 +1,10 @@
+a56 (1.3+dfsg-8) unstable; urgency=low
+
+  * Fix the FTBFS with clang:
+    - Fixed non-void should return a value bug in main.c
+
+ -- Arthur Marble <arthur@info9.net>  Sun, 23 Feb 2014 18:41:17 -0600
+
 a56 (1.3+dfsg-7) unstable; urgency=low
 
   * new maintainer (Closes: #594375)
diff -Naur a56.orig/a56-1.3+dfsg/debian/patches/clang-ftbfs.diff a56/a56-1.3+dfsg/debian/patches/clang-ftbfs.diff
--- a56.orig/a56-1.3+dfsg/debian/patches/clang-ftbfs.diff	1969-12-31 18:00:00.000000000 -0600
+++ a56/a56-1.3+dfsg/debian/patches/clang-ftbfs.diff	2014-02-23 18:49:55.399131162 -0600
@@ -0,0 +1,20 @@
+--- a/main.c
++++ b/main.c
+@@ -185,7 +185,7 @@ double f;
+ 			pass = 2;				/* what a kludge */
+ 			yyerror("%s: multiply defined symbol", sym);
+ 			pass = 1;
+-			return;
++			return 1;
+ 		}
+ 		stop = &symtab[HASH(sym)];
+ 		sp = NEW(struct sym);
+@@ -395,7 +395,7 @@ struct psect *pp;
+ 	int used, avail, of;
+ 
+ 	if(pp == NULL)
+-		return;
++		return 1;
+ 
+ 	used = pp->pc - pp->bottom;
+ 	avail = pp->top - pp->pc;
diff -Naur a56.orig/a56-1.3+dfsg/debian/patches/series a56/a56-1.3+dfsg/debian/patches/series
--- a56.orig/a56-1.3+dfsg/debian/patches/series	2014-02-23 18:38:36.763119344 -0600
+++ a56/a56-1.3+dfsg/debian/patches/series	2014-02-23 18:38:55.483119670 -0600
@@ -3,3 +3,4 @@
 include.patch
 Makefile.patch
 strtol.patch
+clang-ftbfs.diff
