--- apt-show-versions.bash_completion.orig	2013-01-03 10:40:25.560502275 -0500
+++ apt-show-versions.bash_completion	2013-01-03 10:40:42.568639744 -0500
@@ -27,7 +27,7 @@
             return 0
             ;;
         -p|--package)
-            COMPREPLY=( $(_comp_dpkg_installed_packages $cur) )
+            COMPREPLY=( $(_xfunc dpkg _comp_dpkg_installed_packages $cur) )
             return 0
             ;;
         -stf|--status-file)
@@ -44,7 +44,7 @@
         COMPREPLY=( $(compgen -W "$opts" -- $cur) )
         return 0
     else
-        COMPREPLY=( $(_comp_dpkg_installed_packages $cur) )
+        COMPREPLY=( $(_xfunc dpkg _comp_dpkg_installed_packages $cur) )
         return 0
     fi
 }
