diff --git a/scripts/sadt b/scripts/sadt
--- a/scripts/sadt
+++ b/scripts/sadt
@@ -1,7 +1,7 @@
 #!/usr/bin/python3
 # encoding=UTF-8
 
-# Copyright © 2012, 2013 Jakub Wilk <jwilk@debian.org>
+# Copyright © 2012, 2013, 2014 Jakub Wilk <jwilk@debian.org>
 
 # Permission to use, copy, modify, and/or distribute this software for any
 # purpose with or without fee is hereby granted, provided that the above
@@ -185,6 +185,7 @@
         for or_clause in parsed_depends:
             if len(or_clause) == 1 and or_clause[0]['name'] == '@builddeps@':
                 or_clauses += build_depends
+                or_clauses += deb822.PkgRelation.parse_relations('make')
                 continue
             stripped_or_clause = [r for r in or_clause if r['name'] != '@']
             if len(stripped_or_clause) < len(or_clause):
diff --git a/scripts/sadt.pod b/scripts/sadt.pod
--- a/scripts/sadt.pod
+++ b/scripts/sadt.pod
@@ -64,7 +64,7 @@
 
 =head1 CONFORMING TO
 
-README.package-tests shipped by autopkgtest 2.6
+README.package-tests shipped by autopkgtest 2.7.2
 
 =head1 SEE ALSO
 
