diff -ur boinc-6.10.58+dfsg.orig/clientgui/BOINCBaseView.cpp boinc-6.10.58+dfsg/clientgui/BOINCBaseView.cpp
--- boinc-6.10.58+dfsg.orig/clientgui/BOINCBaseView.cpp	2010-03-19 16:56:58.000000000 +0100
+++ boinc-6.10.58+dfsg/clientgui/BOINCBaseView.cpp	2011-02-25 15:33:33.000000000 +0100
@@ -132,8 +132,8 @@
         wxNullFont
     );
     m_pGrayBackgroundAttr = new wxListItemAttr(
-        wxSystemSettings::GetColour(wxSYS_COLOUR_WINDOWTEXT),
-        wxColour(240, 240, 240),
+        wxSystemSettings::GetColour(wxSYS_COLOUR_BTNTEXT),
+        wxSystemSettings::GetColour(wxSYS_COLOUR_BTNFACE),
         wxNullFont
     );
 #endif
diff -ur boinc-6.10.58+dfsg.orig/clientgui/ViewMessages.cpp boinc-6.10.58+dfsg/clientgui/ViewMessages.cpp
--- boinc-6.10.58+dfsg.orig/clientgui/ViewMessages.cpp	2010-03-18 22:54:20.000000000 +0100
+++ boinc-6.10.58+dfsg/clientgui/ViewMessages.cpp	2011-02-25 15:38:22.000000000 +0100
@@ -125,8 +125,8 @@
         wxNullFont
     );
     m_pMessageErrorAttr = new wxListItemAttr(
+        *wxWHITE, 
         *wxRED, 
-        m_pWhiteBackgroundAttr->GetBackgroundColour(), 
         wxNullFont
     );
     m_pMessageInfoGrayAttr = new wxListItemAttr(
@@ -135,8 +135,8 @@
         wxNullFont
     );
     m_pMessageErrorGrayAttr = new wxListItemAttr(
+        *wxLIGHT_GREY, /*wxYELLOW, is not declared contrary to docs */
         *wxRED, 
-        m_pGrayBackgroundAttr->GetBackgroundColour(), 
         wxNullFont
     );
 #else
@@ -146,8 +146,8 @@
         wxNullFont
     );
     m_pMessageErrorAttr = new wxListItemAttr(
+        *wxWHITE,
         *wxRED, 
-        wxSystemSettings::GetColour(wxSYS_COLOUR_WINDOW),
         wxNullFont
     );
     m_pMessageInfoGrayAttr = new wxListItemAttr(*m_pMessageInfoAttr);
