diff -ru bind9-9.9.3.dfsg.P2.pristine/debian/control bind9-9.9.3.dfsg.P2/debian/control
--- bind9-9.9.3.dfsg.P2.pristine/debian/control	2013-11-12 11:17:30.000000000 +0000
+++ bind9-9.9.3.dfsg.P2/debian/control	2013-11-12 11:20:58.275509307 +0000
@@ -8,6 +8,7 @@
 Standards-Version: 3.7.3.0
 XS-Vcs-Browser: http://git.debian.org/?p=users/lamont/bind9.git
 XS-Vcs-Git: git://git.debian.org/~lamont/bind9.git
+Homepage: http://www.isc.org/software/bind
 
 Package: bind9
 Architecture: any
@@ -51,9 +52,10 @@
 Conflicts: host (<<1:9.7.0), dnsutils (<< 1:9.0.0)
 Replaces: dnsutils (<< 1:9.0.0), host (<< 1:9.7.0)
 Provides: host
-Description: Version of 'host' bundled with BIND 9.X
- This package provides the 'host' program in the form that is bundled with
- the BIND 9.X sources.
+Description: DNS lookup utility
+ This package provides /usr/bin/host, a simple utility (bundled with the
+ BIND 9.X sources) which can be used for converting domain names to IP
+ addresses and the reverse.
 
 Package: libbind-dev
 Section: libdevel
