diff -Nur connman-0.48+dfsg.orig/debian/watch connman-0.48+dfsg/debian/watch
--- connman-0.48+dfsg.orig/debian/watch	2010-05-31 19:14:23.000000000 +0200
+++ connman-0.48+dfsg/debian/watch	2010-05-31 19:20:51.000000000 +0200
@@ -1,5 +1,2 @@
-# This software is retrieved directly from the upstream git tree, using tags
-# on the tree.
-#
-# Please see debian/rules, especially GET_SOURCE, get-orig-source as well as
-# get-current-source targets.
+version=3
+http://www.kernel.org/pub/linux/network/connman/connman-(?:[\d\.]*)\.tar\.(?:gz|bz2)
