--- cur/encfs-1.7.4/encfs/encfsctl.cpp	2010-08-30 08:27:49.000000000 +0200
+++ new/encfs-1.7.4/encfs/encfsctl.cpp	2013-09-26 16:59:26.064837976 +0200
@@ -94,7 +94,7 @@
 	gettext_noop("  -- change password for volume, taking password"
 		" from standard input.\n\tNo prompts are issued.")},
     {"ls", 1, 2, cmd_ls, 0,0},
-    {"showcruft", 1, 1, cmd_showcruft, "(root dir)",
+    {"showcruft", 1, 2, cmd_showcruft, "[--extpass=prog] (root dir)",
 	// xgroup(usage)
         gettext_noop("  -- show undecodable filenames in the volume")},
     {"cat", 2, 2, cmd_cat, "(root dir) path",
@@ -677,9 +677,7 @@
 */
 static int cmd_showcruft( int argc, char **argv )
 {
-    (void)argc;
-
-    RootPtr rootInfo = initRootInfo(argv[1]);
+    RootPtr rootInfo = initRootInfo(argc, argv);
 
     if(!rootInfo)
 	return EXIT_FAILURE;
