Index: atom4-4.1/engine/ai.cc
===================================================================
--- atom4-4.1.orig/engine/ai.cc	2013-05-30 16:39:37.000000000 +0200
+++ atom4-4.1/engine/ai.cc	2013-05-30 16:42:24.964551041 +0200
@@ -340,4 +340,5 @@
 #endif //DEBUG_AI
 
   make_move();				// move if it's our turn after reset
+  return 0;
 }
Index: atom4-4.1/x/xscoreboard.cc
===================================================================
--- atom4-4.1.orig/x/xscoreboard.cc	2013-05-30 16:39:37.000000000 +0200
+++ atom4-4.1/x/xscoreboard.cc	2013-05-30 16:43:22.276552620 +0200
@@ -145,7 +145,7 @@
     XDrawString(disp, buffer, textgc, PL_X, starty+PL_MSG_Y+scorefont->ascent,
                 buf, strlen(buf));
   } else if (game->winner()==STALEMATE) {
-    snprintf(buf, TEXTBUF_MAX, "=== DRAW: no more moves left ===", player);
+    snprintf(buf, TEXTBUF_MAX, "=== DRAW: no more moves left ===");
     XDrawString(disp, buffer, textgc, PL_X, starty+PL_MSG_Y+scorefont->ascent,
                 buf, strlen(buf));
   }
