From a1e3825b765bcf7c53d094a1d5b73e360c279a50 Mon Sep 17 00:00:00 2001
From: Ben Finney <ben+debian@benfinney.id.au>
Date: Wed, 5 Apr 2017 19:37:23 +1000
Subject: [PATCH 1/5] Document the change made in this branch.

---
 debian/changelog | 7 ++++++-
 1 file changed, 6 insertions(+), 1 deletion(-)

diff --git a/debian/changelog b/debian/changelog
index 4015df8b..e09d002c 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -18,7 +18,12 @@ devscripts (2.17.6) UNRELEASED; urgency=medium
     bts chdist debcommit dscverify grep-excuses
     namecheck rc-alert svnpath wnpp-alert
 
- -- Osamu Aoki <osamu@debian.org>  Fri, 31 Mar 2017 02:03:30 +0900
+  [ Ben Finney ]
+  * suspicious-source: Correct references to GNU GPL definition of the
+    source form of a work.
+    Closes: bug#859617.
+
+ --
 
 devscripts (2.17.5) unstable; urgency=medium
 
-- 
2.11.0

