=== modified file 'test/integration/framework'
--- test/integration/framework	2013-03-18 07:08:37 +0000
+++ test/integration/framework	2013-03-18 07:22:26 +0000
@@ -1,6 +1,6 @@
 #!/bin/sh -- # no runable script, just for vi
 
-TESTFAILURES="no"
+TESTFAILURES=0
 
 # we all like colorful messages
 if expr match "$(readlink -f /proc/$$/fd/1)" '/dev/pts/[0-9]\+' > /dev/null && \
@@ -38,7 +38,7 @@
 }
 msgpass() { echo "${CPASS}PASS${CNORMAL}" >&2; }
 msgskip() { echo "${CWARNING}SKIP${CNORMAL}" >&2; }
-msgfail() { echo "${CFAIL}FAIL${CNORMAL}" >&2; TESTFAILURES="yes"; }
+msgfail() { echo "${CFAIL}FAIL${CNORMAL}" >&2; TESTFAILURES=$((TESTFAILURES+1)); }
 
 # enable / disable Debugging
 MSGLEVEL=${MSGLEVEL:-3}
@@ -116,7 +116,8 @@
 }
 
 exitwithstatus() {
-	[ "$TESTFAILURES" = "yes" ] && exit 1 || exit 0;
+        # the "% 255" ensures the exit it != 0 even on overflow
+        exit $((TESTFAILURES | TESTFAILURES % 255));
 }
 
 addtrap() {

