diff -u cmake-2.6.2/debian/control cmake-2.6.2/debian/control
--- cmake-2.6.2/debian/control
+++ cmake-2.6.2/debian/control
@@ -4,7 +4,7 @@
 Maintainer: A. Maitland Bottoms <bottoms@debian.org>
 Uploaders: Pierre Habouzit <madcoder@debian.org>, Modestas Vainius <modestas@vainius.eu>
 DM-Upload-Allowed: yes
-Build-Depends: debhelper (>= 6.0.7~), libncurses5-dev, tcl8.4, quilt, libcurl4-gnutls-dev
+Build-Depends: debhelper (>= 6.0.7~), libncurses5-dev, tcl8.4, quilt, libcurl4-gnutls-dev, libxmlrpc-c3-dev, libexpat1-dev, zlib1g-dev
 Standards-Version: 3.8.0
 
 Package: cmake
diff -u cmake-2.6.2/debian/rules cmake-2.6.2/debian/rules
--- cmake-2.6.2/debian/rules
+++ cmake-2.6.2/debian/rules
@@ -38,7 +38,7 @@
 	echo CMAKE_CXX_FLAGS:STRING=$(CFLAGS) >> Build/CMakeCache.txt
 	$(if $(USE_DARTP),echo DART_ROOT:PATH=/usr/share/Dart              >> Build/CMakeCache.txt)
 	$(if $(USE_DARTP),echo BUILDNAME:STRING=cmake_2.2.3-1_$(DARCH).deb >> Build/CMakeCache.txt)
-	cd Build && ../bootstrap --prefix=/usr --docdir=/share/doc/cmake --mandir=/share/man
+	cd Build && ../bootstrap --prefix=/usr --docdir=/share/doc/cmake --mandir=/share/man --system-libs
 	chmod -x ChangeLog.manual
 	chmod -x Modules/*.cmake  Modules/*.cmake.in Modules/*.cxx Modules/Platform/*
 	touch $@
