From 7d122a30c3e0e62fb6d163c345f720c588340a86 Mon Sep 17 00:00:00 2001
From: Xiyue Deng <manphiz@gmail.com>
Date: Fri, 22 Nov 2024 15:10:21 -0800
Subject: [PATCH 3/3] Update documentation accordingly.

---
 dh_elpa_test | 5 +++--
 1 file changed, 3 insertions(+), 2 deletions(-)

diff --git a/dh_elpa_test b/dh_elpa_test
index 5091bd7..2552054 100755
--- a/dh_elpa_test
+++ b/dh_elpa_test
@@ -125,8 +125,9 @@ B<dh_elpa_test> invokes Emacs to run ERT tests.
 =item B<ert_selectors>
 
 A Lisp expression specifying a set of tests.  This is the same as ERT test
-selectors and will be passed to ERT through command line, which will also work
-with custom test tools built upon ERT.
+selectors and will be passed to ERT through command line.  Note that this is
+disabled when B<ert_eval> is specified, and you'll have to manually specify the
+selectors int he helper script.
 
 =item B<autopkgtest_keep>
 
-- 
2.45.2

