--- camel/providers/imap/camel-imap-store-summary.c.orig	2008-02-04 14:42:58.000000000 +0100
+++ camel/providers/imap/camel-imap-store-summary.c	2008-02-04 14:39:54.000000000 +0100
@@ -335,6 +335,9 @@
 		pathu8 = camel_imap_store_summary_full_to_path(s, full_name, dir_sep);
 	}
 
+	if (!g_ascii_strcasecmp (pathu8, "INBOX"))
+		pathu8 = "INBOX";
+
 	info = (CamelImapStoreInfo *)camel_store_summary_add_from_path((CamelStoreSummary *)s, pathu8);
 	if (info) {
 		d(printf("  '%s' -> '%s'\n", pathu8, full_name));
