diff --git a/bin/keep-latest b/bin/keep-latest
index 43cfa82..ebe87ef 100755
--- a/bin/keep-latest
+++ b/bin/keep-latest
@@ -55,6 +55,9 @@ for my $file (@ARGV) {
 		if ($arch ne "source" && defined $architecture && $architecture ne "all" && $architecture ne $arch) {
 			next;
 		}
+		if ($arch eq "source" && /^Extra-Source-Only:\s*yes$/mi) {
+			next;
+		}
 		my $key = $name;
 
 		if ((!exists $version{$key}) or version_less($version{$key},$version)) {
