diff -u cron-3.0pl1/debian/cron.pam cron-3.0pl1/debian/cron.pam
--- cron-3.0pl1/debian/cron.pam
+++ cron-3.0pl1/debian/cron.pam
@@ -5,7 +5,7 @@
 @include common-auth
 session       required   pam_env.so
 @include common-account
-@include common-session
+@include common-session-noninteractive
 # Sets up user limits, please define limits for cron tasks
 # through /etc/security/limits.conf
 session    required   pam_limits.so
diff -u cron-3.0pl1/debian/control cron-3.0pl1/debian/control
--- cron-3.0pl1/debian/control
+++ cron-3.0pl1/debian/control
@@ -9,7 +9,7 @@
 
 Package: cron
 Architecture: any
-Depends: ${shlibs:Depends}, debianutils (>=1.7), adduser, lsb-base (>= 3.2-12ubuntu4)
+Depends: ${shlibs:Depends}, debianutils (>=1.7), adduser, lsb-base (>= 3.2-12ubuntu4), libpam-runtime (>= 1.0.1-11)
 Suggests: exim4 | postfix | mail-transport-agent, lockfile-progs, anacron (>=2.0-1), logrotate, checksecurity
 Conflicts: suidmanager (<< 0.50), lockfile-progs (<< 0.1.7)
 Provides:
