diff --git a/debian/apache2.preinst b/debian/apache2.preinst
index ed5a382..1adc647 100644
--- a/debian/apache2.preinst
+++ b/debian/apache2.preinst
@@ -49,8 +49,9 @@ obsolete_conffile_exists()
 		fi
 	done
 
-	for CONFFILE in $MOVED_CONFFILES_IN ; do
-		if [ -e "/etc/apache2/conf.d/$CONFFILE" ] ; then
+	for CONFFILE in $MOVED_CONFFILES ; do
+		CONFFILE=$( echo "$CONFFILE" | cut -d: -f1 )
+		if [ -e "$CONFFILE" ] ; then
 			return 0
 		fi
 	done
