diff -Nru biometric-authentication-0.9.71/debian/biometric-auth.install biometric-authentication-0.9.71/debian/biometric-auth.install
--- biometric-authentication-0.9.71/debian/biometric-auth.install	2022-02-25 09:57:20.000000000 +0100
+++ biometric-authentication-0.9.71/debian/biometric-auth.install	2024-07-08 00:38:08.000000000 +0200
@@ -1,5 +1,5 @@
 etc/
-lib/systemd/system/biometric-authentication.service
+usr/lib/systemd/system/biometric-authentication.service
 usr/lib/biometric-authentication/biometric-authenticationd
 usr/share/biometric-auth/
 usr/share/dbus-1/
diff -Nru biometric-authentication-0.9.71/debian/changelog biometric-authentication-0.9.71/debian/changelog
--- biometric-authentication-0.9.71/debian/changelog	2022-03-15 13:23:58.000000000 +0100
+++ biometric-authentication-0.9.71/debian/changelog	2024-07-08 00:38:08.000000000 +0200
@@ -1,3 +1,10 @@
+biometric-authentication (0.9.71-1.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Install systemd units into /usr. (Closes: #1073636)
+
+ -- Chris Hofstaedtler <zeha@debian.org>  Mon, 08 Jul 2024 00:38:08 +0200
+
 biometric-authentication (0.9.71-1) unstable; urgency=medium
 
   * New upstream release.
diff -Nru biometric-authentication-0.9.71/debian/rules biometric-authentication-0.9.71/debian/rules
--- biometric-authentication-0.9.71/debian/rules	2022-02-25 09:57:20.000000000 +0100
+++ biometric-authentication-0.9.71/debian/rules	2024-07-08 00:38:05.000000000 +0200
@@ -54,6 +54,8 @@
 	mkdir -p debian/tmp/usr/share/biometric-auth/
 	mv debian/tmp/etc/biometric-auth/biometric-drivers.conf \
 		debian/tmp/usr/share/biometric-auth/biometric-drivers.conf.template
+	mv debian/tmp/lib/* debian/tmp/usr/lib/
+	rmdir debian/tmp/lib
 
 override_dh_gencontrol:
 	echo "bio-app-api:Provides=$(APP_API_DEPEND)" >> debian/biometric-auth.substvars;
