--- ./debian/acpi-support/usr/share/acpi-support/screenblank.orig	2010-12-14 11:37:17.000000000 +0100
+++ ./debian/acpi-support/usr/share/acpi-support/screenblank	2010-12-14 13:58:45.000000000 +0100
@@ -1,9 +1,9 @@
-if [ `pidof xscreensaver` ]; then
+if pidof xscreensaver >/dev/null; then
 	su $user -s /bin/sh -c "(xscreensaver-command -throttle&)"
 		if [ x$LOCK_SCREEN = xtrue ]; then	
 		su $user -s /bin/sh -c "(xscreensaver-command -lock&)"
 	fi
-elif [ `pidof dcopserver` ]; then
+elif pidof dcopserver >/dev/null; then
 	if [ x$LOCK_SCREEN = xtrue ]; then
 		avail_sessions=`dcop --all-users --list-sessions | grep '.DCOP.*__0'`
     		# send the lock command to all sessions
