diff -ru ctwm-3.7.orig/menus.c ctwm-3.7/menus.c
--- ctwm-3.7.orig/menus.c	2006-07-20 00:37:33.000000000 -0400
+++ ctwm-3.7/menus.c	2006-07-20 00:38:44.000000000 -0400
@@ -1461,7 +1461,7 @@
   
 	ws = NULL;
 
-	if (! (all || allicons) && CurrentSelectedWorkspace) {
+	if (! (all || allicons) && CurrentSelectedWorkspace && Scr->workSpaceManagerActive) {
 	    for (ws = Scr->workSpaceMgr.workSpaceList; ws != NULL; ws = ws->next) {
         	if (strcmp (ws->name, CurrentSelectedWorkspace) == 0) break;
 	    }
