--- eboard-1.1.1/debian/rules~	2009-01-22 21:14:25.000000000 +0100
+++ eboard-1.1.1/debian/rules	2009-01-22 21:15:05.000000000 +0100
@@ -8,7 +8,8 @@
 configure: configure-stamp
 configure-stamp: patch-stamp
 	dh_testdir
-	./configure --prefix=/usr --data-prefix=/usr/share/games --man-prefix=/usr/share/man
+	./configure --prefix=/usr --data-prefix=/usr/share/games \
+	    --man-prefix=/usr/share/man --extra-flags=-fsigned-char
 
 	touch configure-stamp
 
