Index: pkgs.dbk
===================================================================
--- pkgs.dbk	(revision 10135)
+++ pkgs.dbk	(working copy)
@@ -290,9 +290,9 @@
 <literal>unstable</literal>.
 </para>
 <para>
-Actually, there are two other possible distributions: <literal>stable-security</literal>
-and <literal>testing-security</literal>, but read
-<xref linkend="bug-security"/> for more information on those.
+Actually, there are other possible distributions:
+<replaceable>codename</replaceable><literal>-security</literal>,
+but read <xref linkend="bug-security"/> for more information on those.
 </para>
 <para>
 It is not possible to upload a package into several distributions at the same
@@ -443,8 +443,8 @@
 <title>Security uploads</title>
 <para>
 Do <emphasis role="strong">NOT</emphasis> upload a package to the security
-upload queue (<literal>oldstable-security</literal>, <literal>stable-security</literal>,
-etc.) without prior authorization from the security team.  If the
+upload queue (on <literal>security-master.debian.org</literal>)
+without prior authorization from the security team.  If the
 package does not exactly meet the team's requirements, it will cause many
 problems and delays in dealing with the unwanted upload.  For details, please
 see <xref linkend="bug-security"/>.
@@ -1122,11 +1122,10 @@
 <listitem>
 <para>
 <emphasis role="strong">Target the right distribution</emphasis>
-in your <filename>debian/changelog</filename>.
-For <literal>stable</literal> this is <literal>stable-security</literal> and
-for <literal>testing</literal> this is <literal>testing-security</literal>, and for the previous
-stable release, this is <literal>oldstable-security</literal>.  Do not target
-<replaceable>distribution</replaceable><literal>-proposed-updates</literal> or
+in your <filename>debian/changelog</filename>:
+<replaceable>codename</replaceable><literal>-security</literal>
+(e.g. <literal>wheezy-security</literal>).
+Do not target <replaceable>distribution</replaceable><literal>-proposed-updates</literal> or
 <literal>stable</literal>!
 </para>
 </listitem>
@@ -1194,8 +1193,8 @@
 <title>Uploading the fixed package</title>
 <para>
 Do <emphasis role="strong">NOT</emphasis> upload a package to the security
-upload queue (<literal>oldstable-security</literal>, <literal>stable-security</literal>,
-etc.) without prior authorization from the security team.  If the
+upload queue (on <literal>security-master.debian.org</literal>)
+without prior authorization from the security team.  If the
 package does not exactly meet the team's requirements, it will cause many
 problems and delays in dealing with the unwanted upload.
 </para>