--- control	2006-07-27 01:43:50.000000000 -0300
+++ control.nmu	2006-07-27 01:49:15.461489750 -0300
@@ -3,7 +3,8 @@
 Priority: extra
 Maintainer: Mario Lang <mlang@debian.org>
 Build-Depends: debhelper (>= 4.2), bison, doxygen, linuxdoc-tools, groff,
- flite1-dev, libncurses5-dev, libxaw7-dev, libatspi-dev, libbluetooth1-dev
+ flite1-dev, libncurses5-dev, libxaw7-dev, libatspi-dev, 
+ libbluetooth2-dev | libbluetooth-dev
 Standards-Version: 3.6.1
 
 Package: brltty
