diff -uNrp debmirror-2.35.orig/debmirror debmirror-2.35/debmirror
--- debmirror-2.35.orig/debmirror	2021-06-03 12:23:33.000000000 +0200
+++ debmirror-2.35/debmirror	2021-06-29 10:38:46.230744497 +0200
@@ -820,6 +820,7 @@ $remoteroot =~ s%^[:/]%% unless download
 @arches=split(/,/,join(',',@arches));
 @arches=qw(i386) unless @arches;
 @arches=() if (join(',',@arches) eq "none");
+@keyrings=split(/,/,join(',',@keyrings));
 @di_dists=split(/,/,join(',',@di_dists));
 @di_arches=split(/,/,join(',',@di_arches));
 if (@di_dists) {
