From d4bc39ef9b3e33312eeeeb339a411391d40aa075 Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Sun, 26 Aug 2018 09:24:49 -0700
Subject: [PATCH 3/3] dgit-maint-debrebase(7): Formatting & wording tweaks

Signed-off-by: Sean Whitton <spwhitton@spwhitton.name>
---
 dgit-maint-debrebase.7.pod | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/dgit-maint-debrebase.7.pod b/dgit-maint-debrebase.7.pod
index cc393b5..e5abe17 100644
--- a/dgit-maint-debrebase.7.pod
+++ b/dgit-maint-debrebase.7.pod
@@ -627,7 +627,7 @@ package, but this should already be represented in your git history.)
 
 The git history made by git-debrebase can seem complicated.
 Here are some suggestions for helpful invocations of gitk and git.
-They can be adapted for other tools like tig, git log, etc.
+They can be adapted for other tools like tig(1), git-log(1), magit, etc.
 
 History of package in Debian, disregarding history from upstream:
 
@@ -665,7 +665,8 @@ Complete diff since the last upload:
 =over
 
     % git diff dgit/dgit/sid..HEAD -- :/ :!/debian/patches
-(Includes changes to upstream files.)
+
+This includes changes to upstream files.
 
 =back
 
@@ -674,7 +675,7 @@ Interdiff of delta queue since last upload, if you really want it:
     % git debrebase make-patches
     % git diff dgit/dgit/sid..HEAD -- debian/patches
 
-Also of course there is
+And of course there is:
 
     % git debrebase status
 
-- 
2.11.0

