diff --git a/adequate b/adequate
--- a/adequate
+++ b/adequate
@@ -434,6 +434,12 @@
             when (m/^\t(\S+) => not found$/) {
                 tag $pkg, 'library-not-found', $path, '=>', $1;
             }
+            when (m/^\t(\S+) .*(?<= )[(]0x[0-9a-f]+[)]$/) {
+                # skip
+            }
+            default {
+                print STDERR "ldd -r $pkg: $_";
+            }
         }
         wait or die "$ldd -r: $!";
         unless ($? == 0 or ($not_dynamic and $? == (1 << 8))) {
