--- emacsen-common-1.4.19/debian-startup.el~	2009-05-17 22:38:21.000000000 +0000
+++ emacsen-common-1.4.19/debian-startup.el	2010-09-14 09:48:58.766324218 +0000
@@ -80,7 +80,7 @@
 of the dirs, where XX must be a number.  The files will be run in
 alphabetical order.  If a file appears in more than one of the dirs,
 then the earlier dir takes precedence, and a .elc file always
-supercedes a .el file of the same name."
+supersedes a .el file of the same name."
 
   (let* ((paths dirs)
          ;; Get a list of all the files in all the specified
@@ -118,7 +118,7 @@
       ;; Now load the files.  "load" will make sure we get the byte
       ;; compiled one first, if any, and will respect load-path's
       ;; ordering.
-      (mapcar
+      (mapc
        (lambda (file)
          (condition-case err
              (load file nil)
