diff --git a/runner/adt-run b/runner/adt-run
index 758f055..0593f70 100755
--- a/runner/adt-run
+++ b/runner/adt-run
@@ -2070,7 +2070,7 @@ def build_source(act, control_override):
         if act.kind != 'dsc':
             testbed.prepare2([])
 
-        testbed.satisfy_dependencies_string('build-essential',
+        testbed.satisfy_dependencies_string(', '.join(build_essential),
                                             'install build-essential')
         testbed.satisfy_dependencies_dsc(dsc, 'build dependencies')
 
