diff -Nru gromacs-4.5.5/debian/changelog gromacs-4.5.5/debian/changelog
--- gromacs-4.5.5/debian/changelog	2012-06-09 20:43:07.000000000 +0200
+++ gromacs-4.5.5/debian/changelog	2012-07-10 11:41:52.000000000 +0200
@@ -1,3 +1,9 @@
+gromacs (4.5.5-2.1) UNRELEASED; urgency=low
+
+  * Non-maintainer upload.
+
+ -- Evgeni Golov <evgeni@debian.org>  Tue, 10 Jul 2012 11:41:49 +0200
+
 gromacs (4.5.5-2) unstable; urgency=low
 
   [ Andreas Tille ]
diff -Nru gromacs-4.5.5/debian/rules gromacs-4.5.5/debian/rules
--- gromacs-4.5.5/debian/rules	2012-06-08 04:21:18.000000000 +0200
+++ gromacs-4.5.5/debian/rules	2012-07-10 13:15:08.000000000 +0200
@@ -102,8 +102,8 @@
 
 build-mpich: configure-stamp
 	dh_testdir
-	$(MAKE) -C build/mpich mdrun 
-	$(MAKE) -C build/mpich-dp mdrun
+	$(MAKE) -C build/mpich 
+	$(MAKE) -C build/mpich-dp
 	sed -e 's/#MPIABBR#/mpich/g' -e 's/#MPIPROG#/MPICH2/g' \
 	    < debian/gromacs-parallel.README.Debian.in \
 	    > debian/gromacs-mpich.README.Debian
@@ -112,8 +112,8 @@
 build-openmpi: configure-stamp
 	dh_testdir
 ifneq (,$(findstring $(DEB_HOST_ARCH),$(OPENMPI_ARCH)))
-	$(MAKE) -C build/openmpi mdrun
-	$(MAKE) -C build/openmpi-dp mdrun
+	$(MAKE) -C build/openmpi
+	$(MAKE) -C build/openmpi-dp
 	sed -e 's/#MPIABBR#/openmpi/g' -e 's/#MPIPROG#/OpenMPI/g' \
 	    < debian/gromacs-parallel.README.Debian.in \
 	    > debian/gromacs-openmpi.README.Debian
@@ -204,8 +204,8 @@
 	dh_testroot
 	dh_prep -pgromacs-mpich
 	dh_installdirs -pgromacs-mpich
-	$(MAKE) -C build/mpich install-mdrun DESTDIR=$(CURDIR)/debian/gromacs-mpich
-	$(MAKE) -C build/mpich-dp install-mdrun DESTDIR=$(CURDIR)/debian/gromacs-mpich
+	$(MAKE) -C build/mpich install DESTDIR=$(CURDIR)/debian/gromacs-mpich
+	$(MAKE) -C build/mpich-dp install DESTDIR=$(CURDIR)/debian/gromacs-mpich
 	dh_installman -pgromacs-mpich debian/man/mdrun_mpi.mpich.1
 	dh_link -pgromacs-mpich usr/share/man/man1/mdrun_mpi.mpich.1.gz usr/share/man/man1/mdrun_mpi_d.mpich.1.gz
 	dh_lintian -pgromacs-mpich
@@ -216,8 +216,8 @@
 	dh_prep -pgromacs-openmpi
 ifneq (,$(findstring $(DEB_HOST_ARCH),$(OPENMPI_ARCH)))
 	dh_installdirs -pgromacs-openmpi
-	$(MAKE) -C build/openmpi install-mdrun DESTDIR=$(CURDIR)/debian/gromacs-openmpi
-	$(MAKE) -C build/openmpi-dp install-mdrun DESTDIR=$(CURDIR)/debian/gromacs-openmpi
+	$(MAKE) -C build/openmpi install DESTDIR=$(CURDIR)/debian/gromacs-openmpi
+	$(MAKE) -C build/openmpi-dp install DESTDIR=$(CURDIR)/debian/gromacs-openmpi
 	dh_installman -pgromacs-openmpi debian/man/mdrun_mpi.openmpi.1
 	dh_link -pgromacs-openmpi usr/share/man/man1/mdrun_mpi.openmpi.1.gz usr/share/man/man1/mdrun_mpi_d.openmpi.1.gz
 	dh_lintian -pgromacs-openmpi
