--- debian/control.orig	2008-11-04 13:26:44.000000000 -0600
+++ debian/control	2008-11-04 13:26:29.000000000 -0600
@@ -7,7 +7,7 @@
 Standards-Version: 3.8.0.1
 
 Package: crash
-Architecture: i386 ia64 alpha powerpc amd64
+Architecture: i386 ia64 alpha powerpc amd64 lpia
 Depends: ${shlibs:Depends}
 Suggests: kernel-patch-lkcd, dumputils
 Description: kernel debugging utility, allowing gdb like syntax
