From a5b9ca20c1799776b6632b9766bf1ce36d30824b Mon Sep 17 00:00:00 2001
From: Ben Harris <bjh21@bjh21.me.uk>
Date: Sat, 22 Feb 2025 18:52:54 +0000
Subject: [PATCH] dgit-user(7): correct "comments" to "commits"

---
 dgit-user.7.pod | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dgit-user.7.pod b/dgit-user.7.pod
index 43ac10c9..1db1df12 100644
--- a/dgit-user.7.pod
+++ b/dgit-user.7.pod
@@ -177,7 +177,7 @@ there will also be some things in C<debian/patches/>.
 It is best to ignore these.
 Insofar as they are relevant
 the changes there will have been applied to the actual files,
-probably by means of actual comments in the git history.
+probably by means of actual commits in the git history.
 The contents of debian/patches are ignored
 when building binaries
 from dgitish git branches.
-- 
2.39.5

