--- /usr/bin/archmbox	2021-08-16 13:27:45.000000000 +0200
+++ archmbox	2022-06-05 15:40:21.429860158 +0200
@@ -1861,7 +1861,7 @@
 my $mbs_f = $mbox_size / $SIZE_FACTOR;		# float size in MB
 my @fmt_l = (length($mbs),			# format values
 	    length($messages),
-	    length( ( $mbs_f =~ /(^.*?)\./ )[0] ) + 3);
+	    length( ( $mbs_f =~ /(^.*?)\./ ) ) + 3 );
 
 sub _p_total {
     my ( $msg_l, $n_msg_l, $size_l, $msg, $n_msg, $size, $on_msg ) = @_;
