From a14ed61bc758bad47d0f319a97d42dbf33c78f34 Mon Sep 17 00:00:00 2001
From: James Valleroy <james.valleroy@gmail.com>
Date: Sun, 9 Feb 2014 18:16:52 -0500
Subject: [PATCH] Enable http bind by default.

---
 debian/ejabberd.cfg | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/ejabberd.cfg b/debian/ejabberd.cfg
index 0dcb5de..9956f5d 100644
--- a/debian/ejabberd.cfg
+++ b/debian/ejabberd.cfg
@@ -571,6 +571,7 @@
   %%                       {docroot, "/var/www"}, 
   %%                       {accesslog, "/var/log/ejabberd/access.log"}
   %%                      ]},
+  {mod_http_bind, []},
   {mod_last,     []},
   {mod_muc,      [
 		  %%{host, "conference.@HOST@"},
-- 
1.8.5.3

