--- Modules/Compiler/GNU-Fortran.cmake~	2011-07-08 14:21:44.000000000 +0200
+++ Modules/Compiler/GNU-Fortran.cmake	2011-09-03 14:19:12.904135479 +0200
@@ -3,7 +3,7 @@
 
 # No -DNDEBUG for Fortran.
 SET(CMAKE_Fortran_FLAGS_MINSIZEREL_INIT "-Os")
-SET(CMAKE_Fortran_FLAGS_RELEASE_INIT "-O3")
+SET(CMAKE_Fortran_FLAGS_RELEASE_INIT "-O2")
 
 # We require updates to CMake C++ code to support preprocessing rules
 # for Fortran.
--- Modules/Compiler/GNU.cmake~	2011-07-08 14:21:44.000000000 +0200
+++ Modules/Compiler/GNU.cmake	2011-09-03 14:19:26.224156389 +0200
@@ -28,7 +28,7 @@
   set(CMAKE_${lang}_FLAGS_INIT "")
   set(CMAKE_${lang}_FLAGS_DEBUG_INIT "-g")
   set(CMAKE_${lang}_FLAGS_MINSIZEREL_INIT "-Os -DNDEBUG")
-  set(CMAKE_${lang}_FLAGS_RELEASE_INIT "-O3 -DNDEBUG")
+  set(CMAKE_${lang}_FLAGS_RELEASE_INIT "-O2 -DNDEBUG")
   set(CMAKE_${lang}_FLAGS_RELWITHDEBINFO_INIT "-O2 -g")
   set(CMAKE_${lang}_CREATE_PREPROCESSED_SOURCE "<CMAKE_${lang}_COMPILER> <DEFINES> <FLAGS> -E <SOURCE> > <PREPROCESSED_SOURCE>")
   set(CMAKE_${lang}_CREATE_ASSEMBLY_SOURCE "<CMAKE_${lang}_COMPILER> <DEFINES> <FLAGS> -S <SOURCE> -o <ASSEMBLY_SOURCE>")
--- Modules/Compiler/G95-Fortran.cmake~	2011-07-08 14:21:44.000000000 +0200
+++ Modules/Compiler/G95-Fortran.cmake	2011-09-03 14:19:05.194125125 +0200
@@ -1,7 +1,7 @@
 set(CMAKE_Fortran_FLAGS_INIT "")
 set(CMAKE_Fortran_FLAGS_DEBUG_INIT "-g")
 set(CMAKE_Fortran_FLAGS_MINSIZEREL_INIT "-Os")
-set(CMAKE_Fortran_FLAGS_RELEASE_INIT "-O3")
+set(CMAKE_Fortran_FLAGS_RELEASE_INIT "-O2")
 set(CMAKE_Fortran_FLAGS_RELWITHDEBINFO_INIT "-O2 -g")
 set(CMAKE_Fortran_MODDIR_FLAG "-fmod=")
 set(CMAKE_Fortran_VERBOSE_FLAG "-v")
