diff -ur dnssec-trigger-0.13~svn683.orig/01-dnssec-trigger.in dnssec-trigger-0.13~svn683/01-dnssec-trigger.in
--- dnssec-trigger-0.13~svn683.orig/01-dnssec-trigger.in	2014-07-14 22:24:40.000000000 -0700
+++ dnssec-trigger-0.13~svn683/01-dnssec-trigger.in	2014-07-14 22:26:51.437356743 -0700
@@ -50,11 +50,11 @@
 #########################
 # get global nameservers
 # try to get nmcli version
-NMCLI_VER="`nmcli -v 2> /dev/null | sed 's/.*version \([0-9]\)\.\([0-9]\)\.\([0-9]\)\.\([0-9]\).*/\1\2\3\4/'`"
+NMCLI_VER=$(printf '%03d%03d%03d%03d\n' $(nmcli -v 2>/dev/null | sed 's/.*version \([0-9]\+\)\.\([0-9]\+\)\.\([0-9]\+\)\.\([0-9]\+\).*/\1 \2 \3 \4/'))
 # if nmcli exists
-if [ -n $NMCLI_VER ]; then
+if [ -n "$NMCLI_VER" ]; then
     # if the version is greater or equal 0.9.9.0
-    if [ $NMCLI_VER -ge 0990 ]; then
+    if [ $NMCLI_VER -ge 000009009000 ]; then
         global_nameservers="`nmcli -f IP4,IP6 dev show | fgrep 'DNS' | awk '{print $2;}'`"
     else
         global_nameservers="`nmcli -f IP4,IP6 dev list | fgrep 'DNS' | awk '{print $2;}'`"
