diff -Naur devscripts-2.10.1.orig/scripts/bts.pl devscripts-2.10.1/scripts/bts.pl
--- devscripts-2.10.1.orig/scripts/bts.pl	2007-03-16 22:16:35.000000000 +0200
+++ devscripts-2.10.1/scripts/bts.pl	2007-03-22 04:59:17.000000000 +0200
@@ -1573,7 +1573,7 @@
 	$charset =~ s/^ANSI_X3\.4-19(68|86)$/US-ASCII/;
 	my $fromline = fold_from_header("From: " .
 			MIME_encode_mimewords($from, 'Charset' => $charset));
-	my $date = `822-date`;
+	my $date = `date -R`;
 	chomp $date;
 
 	my $pid = open(MAIL, "|-");
@@ -1642,7 +1642,7 @@
     my ($subject, $body, $to, $from) = @_;
 
     if (defined $from) {
-	my $date = `822-date`;
+	my $date = `date -R`;
 	chomp $date;
 
 	my $pid = open(MAIL, "|-");
diff -Naur devscripts-2.10.1.orig/scripts/debchange.pl devscripts-2.10.1/scripts/debchange.pl
--- devscripts-2.10.1.orig/scripts/debchange.pl	2007-02-26 01:16:03.000000000 +0200
+++ devscripts-2.10.1/scripts/debchange.pl	2007-03-22 04:59:00.000000000 +0200
@@ -684,7 +684,7 @@
 }
 
 # Get the date
-chomp(my $DATE=`822-date`);
+chomp(my $DATE=`date -R`);
 
 # Are we going to have to figure things out for ourselves?
 if (! $opt_i && ! $opt_v && ! $opt_d && ! $opt_a && ! $opt_e && ! $opt_r &&
diff -Naur devscripts-2.10.1.orig/scripts/mass-bug.pl devscripts-2.10.1/scripts/mass-bug.pl
--- devscripts-2.10.1.orig/scripts/mass-bug.pl	2007-02-26 01:16:03.000000000 +0200
+++ devscripts-2.10.1/scripts/mass-bug.pl	2007-03-22 04:57:10.000000000 +0200
@@ -262,7 +262,7 @@
     my ($subject, $body, $to, $from) = @_;
 
     if (defined $from) {
-	my $date = `822-date`;
+	my $date = `date -R`;
 	chomp $date;
 	
 	my $pid = open(MAIL, "|-");
diff -Naur devscripts-2.10.1.orig/scripts/nmudiff.sh devscripts-2.10.1/scripts/nmudiff.sh
--- devscripts-2.10.1.orig/scripts/nmudiff.sh	2007-02-26 01:16:03.000000000 +0200
+++ devscripts-2.10.1/scripts/nmudiff.sh	2007-03-22 04:57:01.000000000 +0200
@@ -300,7 +300,7 @@
 Cc: 
 Bcc: $BCC_ADDRESS_SENDMAIL
 Subject: $SOURCE: diff for NMU version $VERSION
-Date: `822-date`
+Date: `date -R`
 X-NMUDIFF-Version: ###VERSION###
 
 $TAGS
