--- apt-mirror.old	2011-04-09 16:57:09.237888427 +0400
+++ apt-mirror	2011-04-09 17:47:43.001889158 +0400
@@ -178,17 +178,16 @@
 }
 
 sub lock_aptmirror {
-	system ("touch " . get_variable("var_path") . "/apt-mirror.lock");
-}
-
-sub check_lock {
-	if(-e get_variable("var_path") . "/apt-mirror.lock")
+	open (LOCK_FILE, '>', get_variable("var_path") . "/apt-mirror.lock");
+	my $lock = flock(LOCK_FILE, 2 | 4); ## LOCK_EX { 2 }, LOCK_NB { 4 } ## i not know why not work
+	if (!$lock)
 	{
 		die("apt-mirror is already running, exiting");
 	}
 }
 
 sub unlock_aptmirror {
+	close(LOCK_FILE);
 	unlink(get_variable("var_path") . "/apt-mirror.lock");
 }
 
@@ -296,12 +295,6 @@
 #
 #######################################################################################
 
-check_lock();
-
-$SIG{INT} = "unlock_aptmirror";
-$SIG{HUP} = "unlock_aptmirror";
-$SIG{TERM} = "unlock_aptmirror";
-
 lock_aptmirror();
 
 
