Description: correct check on $0 in conditions for entering debugger
Author: Nicholas Bamber <nicholas@periapt.co.uk>
Bug: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=403304
Last-Update: 2015-10-12
Index: bash-4.3/shell.c
===================================================================
--- bash-4.3.orig/shell.c
+++ bash-4.3/shell.c
@@ -720,7 +720,7 @@ main (argc, argv, env)
   /* Bind remaining args to $1 ... $n */
   arg_index = bind_args (argv, arg_index, argc, 1);
 
-  if (debugging_mode && locally_skip_execution == 0 && running_setuid == 0 && dollar_vars[1])
+  if (debugging_mode && locally_skip_execution == 0 && running_setuid == 0 && dollar_vars[0])
     start_debugger ();
 
   /* Do the things that should be done only for interactive shells. */
