diff -r -C 3 /home/jamie/test/evolution-2.0.2/mail/em-junk-filter.c /tmp/jamie/evolution-2.0.2/mail/em-junk-filter.c
*** /home/jamie/test/evolution-2.0.2/mail/em-junk-filter.c	Tue Jun 22 03:14:37 2004
--- /tmp/jamie/evolution-2.0.2/mail/em-junk-filter.c	Thu Oct 28 00:24:16 2004
***************
*** 246,252 ****
  	if (em_junk_sa_local_only)
  		argv[i++] = "--local";
  		
! 	argv[i++] = "--daemonize";
  	argv[i] = NULL;
  		
  	for (i = 0; i < MAX_SPAMD_PORTS; i++, p++) {
--- 246,252 ----
  	if (em_junk_sa_local_only)
  		argv[i++] = "--local";
  		
! 	argv[i++] = "--max-children 1";
  	argv[i] = NULL;
  		
  	for (i = 0; i < MAX_SPAMD_PORTS; i++, p++) {
