diff --git a/apparmor.d/abstractions/lxc/container-base b/apparmor.d/abstractions/lxc/container-base
index c109baa..f3a215f 100644
--- a/apparmor.d/abstractions/lxc/container-base
+++ b/apparmor.d/abstractions/lxc/container-base
@@ -5,9 +5,9 @@
 
   # The following 3 entries are only supported by recent apparmor versions.
   # Comment them if the apparmor parser doesn't recognize them.
-  dbus,
-  signal,
-  ptrace,
+  #dbus,
+  #signal,
+  #ptrace,
 
   # ignore DENIED message on / remount
   deny mount options=(ro, remount) -> /,
