--- clamav-daemon	2007-03-12 20:51:30.000000000 -0300
+++ clamav-daemon.new	2007-03-19 16:54:21.000000000 -0300
@@ -24,7 +24,7 @@
 CLAMAVCONF=/etc/clamav/clamd.conf
 SUPERVISOR=/usr/bin/daemon
 SUPERVISORNAME=daemon
-SUPERVISORPIDFILE="/var/run/clamav/daemon-clamd.pid"
+SUPERVISORPIDFILE="/var/run/clamav/clamd.pid"
 SUPERVISORARGS="--name=$NAME --respawn $DAEMON -F $SUPERVISORPIDFILE"
 
 [ -x "$DAEMON" ] || exit 0
