--- amavisd-new-courier.patch	2004-08-28 05:41:08.000000000 +0200
+++ amavisd-new-courier.patch.frda	2004-08-28 05:51:17.000000000 +0200
@@ -4,8 +4,8 @@
  # Set the user and group to which the daemon will change if started as root
  # (otherwise just keeps the UID unchanged, and these settings have no effect):
 +# For courier, the group must be the same as the group which courier runs as
- $daemon_user  = 'vscan';	# (no default;  customary: vscan or amavis)
- $daemon_group = 'sweep';	# (no default;  customary: vscan or amavis)
+ $daemon_user  = 'amavis';	# (no default;  customary: vscan or amavis)
+ $daemon_group = 'amavis';	# (no default;  customary: vscan or amavis)
 @@ -122,4 +123,8 @@
  #$notify_method = $forward_method;
  
@@ -16,17 +16,18 @@
  # prefer to collect mail for forwarding as BSMTP files?
  #$forward_method = "bsmtp:$MYHOME/out-%i-%n.bsmtp";
 @@ -208,11 +213,14 @@
- 			          # (default is true)
+ 			          # (default is true (1) )
  
 -# AMAVIS-CLIENT PROTOCOL INPUT SETTINGS (e.g. with sendmail milter)
 +# AMAVIS-CLIENT AND COURIER PROTOCOL INPUT SETTINGS (e.g. with sendmail milter)
  #   (used with amavis helper clients like amavis-milter.c and amavis.c,
- #   NOT needed for Postfix or Exim or dual-sendmail - keep it undefined.
- $unix_socketname = "$MYHOME/amavisd.sock"; # amavis helper protocol socket
+ #   NOT needed for Postfix and Exim  or dual-sendmail - keep it undefined.)
+ #$unix_socketname = "/var/lib/amavis/amavisd.sock"; # amavis helper protocol socket
 +#$unix_socketname = "/var/lib/courier/filters/amavisd"; # courier socket
- #$unix_socketname = undef;        # disable listening on a unix socket
+-$unix_socketname = undef;         # disable listening on a unix socket
++#$unix_socketname = undef;        # disable listening on a unix socket
                                    # (default is undef, i.e. disabled)
-                                   # (usual setting is $MYHOME/amavisd.sock)
+
 +$unix_socket_protocol = 'amavis'; # either 'amavis' for milter or the amavis
 +                                  # client or 'courier' for courier
  
