diff -ruN /home/ernie/debian/dia-0.94-pre1/objects/standard/image.c dia-0.94-pre1/objects/standard/image.c
--- /home/ernie/debian/dia-0.94-pre1/objects/standard/image.c	2004-05-21 18:01:34.000000000 +0200
+++ dia-0.94-pre1/objects/standard/image.c	2004-07-14 23:26:20.000000000 +0200
@@ -692,9 +692,11 @@
       char *temp_string;
 
       temp_string = g_malloc(strlen(diafile_dir) +
-			     strlen(image->file) +1);
+	                     strlen(G_DIR_SEPARATOR_S) +
+			     strlen(image->file) + 1);
 
       strcpy(temp_string, diafile_dir);
+      strcat(temp_string, G_DIR_SEPARATOR_S);
       strcat(temp_string, image->file);
 
       image->image = dia_image_load(temp_string);
