From e32d12cc0df04d756eb9b5be19c5ce1f8f6f058b Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas@debian.org>
Date: Wed, 30 Dec 2020 22:38:47 +0100
Subject: [PATCH 11/21] Make flow of with_cross easyer to follow

---
 debian/rules | 6 ++----
 1 file changed, 2 insertions(+), 4 deletions(-)

diff --git a/debian/rules b/debian/rules
index d4346d1..3a9cc38 100755
--- a/debian/rules
+++ b/debian/rules
@@ -129,9 +129,9 @@ ifneq ($(distribution)-$(CROSS_ARCHS),Ubuntu-)
   NATIVE_ARCHS += mipsn32r6 mipsn32r6el mipsr6 mipsr6el mips64r6 mips64r6el
 endif
 
+with_cross = yes
 ifeq ($(DEB_SOURCE),binutils)
   with_native	= yes
-  with_cross	= yes
   same_source	= yes
   ifeq (,$(CROSS_ARCHS))
     ifneq (,$(filter $(DEB_HOST_ARCH), amd64 i386 x32))
@@ -164,7 +164,6 @@ ifeq ($(DEB_SOURCE),binutils)
   endif
 else ifeq ($(DEB_SOURCE),binutils-mipsen)
   with_native	=
-  with_cross	= yes
   same_source	=
   ifeq (,$(CROSS_ARCHS))
     ifneq (,$(filter $(DEB_HOST_ARCH), amd64 i386 x32 arm64 ppc64el))
@@ -181,7 +180,6 @@ else ifeq ($(DEB_SOURCE),binutils-mipsen)
   endif
 else
   # binutils-ports ...
-  with_cross	= yes
   ifeq (,$(CROSS_ARCHS))
     CROSS_ARCHS ?= m68k sh4 mips64 mips64el
   endif
@@ -257,7 +255,7 @@ endif
 
 ifneq (,$(TARGET))
   with_native	=
-  with_cross	=
+  with_cross	= disabled by TARGET
   same_source	=
   # Support TARGET both as Debian architecture specification (e.g. arm),
   # and as the target name (e.g. arm-linux-gnu).
-- 
2.30.1

