From 06b2a578f497bdfc7f7efa1122424a22f07bef89 Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas@debian.org>
Date: Tue, 5 Jan 2021 19:25:20 +0100
Subject: [PATCH 15/21] Simplify definition of CROSS_ARCH

---
 debian/rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules b/debian/rules
index 7e155ae..27fd81f 100755
--- a/debian/rules
+++ b/debian/rules
@@ -218,7 +218,7 @@ HOST_ARCHS_kfreebsd-amd64 = amd64 i386 x32
 HOST_ARCHS_kfreebsd-i386 = amd64 i386 x32
 HOST_ARCHS_hurd-i386 = amd64 i386 x32
 
-CROSS_ARCH	= $(subst .,,$(suffix $@))
+CROSS_ARCH	= $*
 _gnu_type	= $(shell dpkg-architecture -a$(1) -qDEB_HOST_GNU_TYPE -f 2>/dev/null)
 _multiarch	= $(shell dpkg-architecture -a$(1) -qDEB_HOST_MULTIARCH -f 2>/dev/null)
 CROSS_GNU_TYPE	= $(subst i586,i686,$(call _gnu_type,$(CROSS_ARCH)))
-- 
2.30.1

