From a0b622e64cc75a9e390a823354040fbc3da52f07 Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas@debian.org>
Date: Tue, 5 Jan 2021 19:27:29 +0100
Subject: [PATCH 16/21] Wrap two calls to dpkg-architecture into the _gnu_type
 macro

---
 debian/rules | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/rules b/debian/rules
index 27fd81f..6367477 100755
--- a/debian/rules
+++ b/debian/rules
@@ -602,8 +602,8 @@ else
 	$(foreach ca,$(NATIVE_ARCHS), \
 	  sed \
 	    -e 's/@host_archs@/$(ca)$(if $(and $(filter yes,$(with_cross)),$(filter $(ca),$(DEB_HOST_ARCH) $(CROSS_ARCHS))), $(HOST_ARCHS_$(ca)))/g' \
-	    -e 's/@target@/$(shell dpkg-architecture -a$(ca) -qDEB_HOST_GNU_TYPE 2>/dev/null | sed 's/_/-/g;s/i586/i686/')/g' \
-	    -e 's/@binutils_alt_triplet@/$(if $(findstring i386,$(ca)),binutils-$(shell dpkg-architecture -a$(ca) -qDEB_HOST_GNU_TYPE 2>/dev/null | sed 's/i686/i586/g'))/g' \
+	    -e 's/@target@/$(subst _,-,$(subst i586,i686,$(call _gnu_type,$(ca))))/g' \
+	    -e 's/@binutils_alt_triplet@/$(if $(findstring i386,$(ca)),binutils-$(subst i686,i586,$(call _gnu_type,$(ca))))/g' \
 	    -e '$(if $(filter yes,$(same_source)),/^Built-Using/d)' \
 	    debian/control.cross.in >> debian/control; \
 	)
-- 
2.30.1

