From c30ecc3edd96df8bb21567e8785e0deacb488cd8 Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas@debian.org>
Date: Tue, 5 Jan 2021 19:35:18 +0100
Subject: [PATCH 18/21] Drop two unused variable definitions

---
 debian/rules | 2 --
 1 file changed, 2 deletions(-)

diff --git a/debian/rules b/debian/rules
index 9d26c4a..032d08e 100755
--- a/debian/rules
+++ b/debian/rules
@@ -247,8 +247,6 @@ ifneq (,$(TARGET))
   DEB_TARGET_ARCH      = $(call vafilt,$(DPKG_TARGET_VARS),DEB_HOST_ARCH)
   for_target           = dpkg-architecture -f -t$(TARGET) -c
 else
-  DEB_TARGET_MULTIARCH = $(DEB_HOST_MULTIARCH)
-  DEB_TARGET_ARCH      = $(DEB_HOST_ARCH)
   for_target           =
 endif
 
-- 
2.30.1

