From c176041504ed84827dbf6d9bcb5196c3948a1439 Mon Sep 17 00:00:00 2001
From: Colin Watson <cjwatson@canonical.com>
Date: Tue, 3 Apr 2012 14:00:19 +0100
Subject: [PATCH] Fix parsing of '@' dependencies

---
 runner/adt-run |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/runner/adt-run b/runner/adt-run
index 310c39a..e6e81f1 100755
--- a/runner/adt-run
+++ b/runner/adt-run
@@ -1126,9 +1126,9 @@ class Test:
 			dn.append(d)
 		else:
 			for (pkg,bin) in t.act.binaries:
-				d = d.replace('@',pkg)
-				t._debug('  synthesised dependency '+d)
-				dn.append(d)
+				dp = d.replace('@',pkg)
+				t._debug('  synthesised dependency '+dp)
+				dn.append(dp)
 	testbed.prepare(dn)
  def run(t, tree):
 	t._debug('[----------------------------------------')
-- 
1.7.9.1

