diff --git a/project-history.en.dbk b/project-history.en.dbk
index 570591c..72efe8a 100644
--- a/project-history.en.dbk
+++ b/project-history.en.dbk
@@ -463,7 +463,7 @@ This was also the first release that supported the installation and booting in
 devices using UEFI firmware.
 </para>
 <para>
-Debian 7 had Long Term Support (LTS) for i386, amd64, armel and armh architectures
+Debian 7 had Long Term Support (LTS) for i386, amd64, armel and armhf architectures
 until the end of May 2018.
 </para>
 <para>
@@ -486,7 +486,7 @@ which had to be restarted in order to propagate security updates after an
 upgrade.
 </para>
 <para>
-Debian 8 had Long Term Support (LTS) for i386, amd64, armel and armh architectures
+Debian 8 had Long Term Support (LTS) for i386, amd64, armel and armhf architectures
 until the end of June 2020.
 </para>
 <para>
@@ -511,7 +511,7 @@ Thanks to the Reproducible Builds project, over 90% of the source packages inclu
 in Debian 9 were able to build bit-for-bit identical binary packages.
 </para>
 <para>
-Debian 9 had Long Term Support (LTS) for i386, amd64, armel and armh architectures
+Debian 9 had Long Term Support (LTS) for i386, amd64, armel and armhf architectures
 until the end of June 2022.
 </para>
 
@@ -532,7 +532,7 @@ amd64, i386 and arm64 architectures and working out of the box on most
 Secure Boot-enabled machines.
 </para>
 <para>
-Debian 10 has Long Term Support (LTS) for i386, amd64, armel and armh architectures
+Debian 10 has Long Term Support (LTS) for i386, amd64, armel and armhf architectures
 until the end of June 2024.
 </para>
 
@@ -628,7 +628,7 @@ however, systems using the older layout would be converted during the upgrade.
 
 Thanks to the combined work of the Debian Security team and the Debian Long
 Term Support team, <emphasis>bookworm</emphasis> will be supported on four
-architectures untill June 2028 (5 years after release).
+architectures until June 2028 (5 years after release).
 
 </para>
 
