--- simple-patchsys.mk.old	2005-03-03 23:38:17.000000000 +0000
+++ simple-patchsys.mk	2005-03-23 22:43:57.000000000 +0000
@@ -56,7 +56,7 @@
 	@echo "patches: $(DEB_PATCHES)"
 	@set -e ; reverse=""; patches="$(DEB_PATCHES)"; \
 	  if [ "$@" = "reverse-patches" ]; then \
-	    if [ ! -e debian/stamp-patched ]; then \
+	    if [ ! -e debian/stamp-patched -a ! -e debian/stamp-patch-failed ]; then \
 	      echo "Not reversing not applied patches."; \
 	      exit 0; \
 	    fi; \
@@ -96,6 +96,7 @@
 	    if test -z "$$success"; then \
 	      if test -z "$$reverse"; then \
 	        echo "failure."; \
+		touch debian/stamp-patch-failed; \
 	        exit 1; \
 	       else \
 	         echo "failure (ignored)."; \
