--- a/scripts/dpkg-source.pl	2007-11-23 08:12:51.000000000 +0530
+++ b/scripts/dpkg-source.pl	2007-12-20 12:13:58.000000000 +0530
@@ -324,7 +324,6 @@
 		error(_g("error occurred while parsing %s"), $_) unless defined $dep;
 		my $facts = Dpkg::Deps::KnownFacts->new();
 		$dep->simplify_deps($facts);
-		$dep->sort();
 		$f{$_}= $dep->dump();
 	    }
             elsif (s/^X[BC]*S[BC]*-//i) { $f{$_}= $v; }
