Index: debian/config.proto
===================================================================
--- debian/config.proto	(revision 230)
+++ debian/config.proto	(working copy)
@@ -1130,7 +1130,7 @@
 
     case "$XKBLAYOUT" in
 	# TODO: make s.t. to not forget to update this list
-	cs,cs|us,am|us,ara|us,ben|us,bd|us,bg|us,bt|us,by|us,deva|us,ge|us,gh|us,gr|us,guj|us,guru|us,il|us,in|us,ir|us,iku|jp,jp|us,jp|us,kan|us,kh|us,la|us,lao|us,lk|us,mk|us,mm|us,mn|us,mv|us,mal|us,ori|us,pk|us,ru|us,scc|us,sy|us,syr|us,tel|us,th|us,tj|us,tam|us,ua|us,uz)
+	cs,cs|us,am|us,ara|us,ben|us,bd|us,bg|us,bt|us,by|us,deva|us,ge|us,gh|us,gr|us,guj|us,guru|us,il|us,in|us,ir|us,iku|jp,jp|us,jp|us,kan|us,kh|us,la|us,lao|us,lk|us,lt|us,mk|us,mm|us,mn|us,mv|us,mal|us,ori|us,pk|us,ru|us,scc|us,sy|us,syr|us,tel|us,th|us,tj|us,tam|us,ua|us,uz)
 	    default_layout=${XKBLAYOUT#*,}
 	    default_variant="${XKBVARIANT#*,}"
 	    unsupported_layout=no
@@ -1315,7 +1315,7 @@
 			;;
 		    # TODO: make s.t. to not forget to update this list.  Don't
 		    # forget to update also the nonlatin list in kbdcompiler
-		    am|ara|ben|bd|bg|bt|by|deva|ge|gh|gr|guj|guru|il|in|ir|iku|kan|kh|la|lao|lk|mk|mm|mn|mv|mal|ori|pk|ru|scc|sy|syr|tel|th|tj|tam|ua|uz)
+		    am|ara|ben|bd|bg|bt|by|deva|ge|gh|gr|guj|guru|il|in|ir|iku|kan|kh|la|lao|lk|lt|mk|mm|mn|mv|mal|ori|pk|ru|scc|sy|syr|tel|th|tj|tam|ua|uz)
 			latin=no
 			real_layout=us,$layout
 			;;
Index: Keyboard/kbdcompiler
===================================================================
--- Keyboard/kbdcompiler	(revision 230)
+++ Keyboard/kbdcompiler	(working copy)
@@ -25,7 +25,7 @@
 
 # Don't forget to update also the list in config.proto
 my $nonlatin = ' am ara ben bd bg bt by deva ge gh gr guj guru \
-                 il in ir iku kan kh la lao lk mk mm mn mv mal \
+                 il in ir iku kan kh la lao lk lt mk mm mn mv mal \
                  ori pk ru scc sy syr tel th tj tam ua uz ';
 
 my %keymaps;
