diff -ru apng2gif-1.5.pristine/debian/control apng2gif-1.5/debian/control
--- apng2gif-1.5.pristine/debian/control	2012-03-11 07:16:42.000000000 +0000
+++ apng2gif-1.5/debian/control	2012-07-04 16:03:09.459392404 +0100
@@ -11,8 +11,9 @@
 Package: apng2gif
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}
-Description: convert APNG animated images to GIF files
- Progrmam apng2gif converts APNG into animated GIF format.
+Description: tool for converting APNG images to animated GIF format
+ This package provides a command line tool for converting images from
+ Animated PNG to animated GIF format.
  .
  The Animated Portable Network Graphics (APNG) file format is an
  extension to the Portable Network Graphics (PNG) specification. It
