diff -Nru connman-1.21/debian/changelog connman-1.21/debian/changelog
--- connman-1.21/debian/changelog	2014-02-26 20:16:52.000000000 +0100
+++ connman-1.21/debian/changelog	2014-08-15 14:38:20.000000000 +0200
@@ -1,3 +1,10 @@
+connman (1.21-1.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Build against libgnutls28-dev. Closes: #753022
+
+ -- Andreas Metzler <ametzler@debian.org>  Fri, 15 Aug 2014 14:38:16 +0200
+
 connman (1.21-1) unstable; urgency=low
 
   * New upstream release. (Closes: #688238)
diff -Nru connman-1.21/debian/control connman-1.21/debian/control
--- connman-1.21/debian/control	2014-02-26 20:15:40.000000000 +0100
+++ connman-1.21/debian/control	2014-08-15 14:39:21.000000000 +0200
@@ -9,7 +9,7 @@
                libglib2.0-dev,
                libdbus-1-dev,
                gtk-doc-tools,
-               libgnutls-dev,
+               libgnutls28-dev,
                libreadline-dev,
                iptables-dev,
                wpasupplicant,
