--- debian-ispell.el.orig	2021-06-26 21:35:17.993190868 +0200
+++ debian-ispell.el	2021-06-26 21:50:06.715333471 +0200
@@ -438,12 +438,8 @@
 
     ;; Set `ispell-dictionary' if still unbound. This will be done after
     ;; init files load, with real `ispell-program-name'
-    (or (boundp 'ispell-dictionary)
-	(defcustom ispell-dictionary default-dictionary
-	  "Default dictionary to use if `ispell-local-dictionary' is nil."
-	  :type '(choice string
-			 (const :tag "default" nil))
-	  :group 'ispell))
+    (or ispell-dictionary
+	(setq ispell-dictionary default-dictionary))
 
     ;; The debugging output if required
     (if debian-dict-common-debug
@@ -493,4 +489,11 @@
              (ispell-set-spellchecker-params)))
   :group 'ispell)
 
+
+(defcustom ispell-dictionary nil
+  "Default dictionary to use if `ispell-local-dictionary' is nil."
+  :type '(choice string
+		 (const :tag "default" nil))
+  :group 'ispell)
+
 ;;; -----------------------------------------------------------------------
