diff -ur docbook-xsl-ns.old/common/l10n.xsl docbook-xsl-ns-1.76.0~RC1+dfsg/common/l10n.xsl
--- docbook-xsl-ns.old/common/l10n.xsl	2011-01-09 23:52:13.000000000 +0000
+++ docbook-xsl-ns-1.76.0~RC1+dfsg/common/l10n.xsl	2011-01-09 23:52:58.000000000 +0000
@@ -423,7 +423,7 @@
 	  <xsl:for-each select="$context.node">
 	    <xsl:variable name="template.node"
 			  select="(key('l10n-template-style', concat($name, '#', $xrefstyle))
-				   |key('l10n-template', $name))[1]"/>
+				   |key('l10n-template', $name))[ancestor::*[1]=$context.node][1]"/>
 
 	    <xsl:choose>
 	      <xsl:when test="$template.node/@text">
