diff -r 73f19fe1017b init.d/cron
--- a/init.d/cron       Fri Mar 06 15:10:05 2009 +0100
+++ b/init.d/cron       Fri Mar 06 15:43:25 2009 +0100
@@ -67,6 +67,16 @@
        # cron reloads automatically
         log_end_msg 0
         ;;
+status)
+        pidofproc -p $PIDFILE $DAEMON >/dev/null
+        status=$?
+        if [ $status -eq 0 ]; then
+                log_success_msg "Periodic command scheduler crond is running"
+        else
+                log_failure_msg "Periodic command scheduler crond is not running"
+        fi
+        exit $status
+        ;;
 *)     log_action_msg "Usage: /etc/init.d/cron {start|stop|restart|reload|force-reload}"
         exit 2
         ;;
--- a/init.d/cron       Sat Mar 07 20:08:13 2009 +0100
+++ b/init.d/cron       Sat Mar 07 20:08:15 2009 +0100
@@ -77,7 +77,7 @@
         fi
         exit $status
         ;;
-*)     log_action_msg "Usage: /etc/init.d/cron {start|stop|restart|reload|force-reload}"
+*)     log_action_msg "Usage: /etc/init.d/cron {start|stop|restart|reload|force-reload|status}"
         exit 2
         ;;
 esac

