--- fbi-2.07/jpegtools.c	2006-06-13 14:47:24.000000000 +0200
+++ fbi-2.07_patched/jpegtools.c	2011-10-19 01:42:59.498275393 +0200
@@ -570,18 +570,16 @@ int jpeg_transform_inplace(char *file,
     }
     out = fdopen(fd,"w");
 
-    /* copy owner and permissions */
+    /* copy owner and permissions (if possible with the filesystem) */
     if (-1 == fstat(fileno(in),&st)) {
 	fprintf(stderr,"fstat(%s): %s\n",file,strerror(errno));
 	goto oops;
     }
     if (-1 == fchown(fileno(out),st.st_uid,st.st_gid)) {
 	fprintf(stderr,"fchown(%s): %s\n",tmpfile,strerror(errno));
-	goto oops;
     }
     if (-1 == fchmod(fileno(out),st.st_mode)) {
 	fprintf(stderr,"fchmod(%s): %s\n",tmpfile,strerror(errno));
-	goto oops;
     }
 
     /* transform */
