diff -ru catfish-0.3.2.orig/catfish.py catfish-0.3.2/catfish.py
--- catfish-0.3.2.orig/catfish.py	2010-07-20 23:14:37.263707525 +0100
+++ catfish-0.3.2/catfish.py	2010-07-20 23:14:48.759682295 +0100
@@ -12,7 +12,7 @@
 import sys
 
 try:
-    import os, stat, time, md5, optparse, subprocess, fnmatch, re
+    import os, stat, time, hashlib, optparse, subprocess, fnmatch, re
     import gobject, pygtk
     pygtk.require('2.0')
     import locale, gettext, gtk, gtk.glade, pango
@@ -713,7 +713,7 @@
     def get_thumbnail(self, path, icon_size=0, mime_type=None):
         """Try to fetch a small thumbnail."""
 
-        md5_hash = md5.new('file://' + path).hexdigest()
+        md5_hash = hashlib.md5('file://' + path).hexdigest()
         filename = '%s%s.png' % (self.folder_thumbnails, md5_hash)
         try:
             return gtk.gdk.pixbuf_new_from_file(filename)
