--- usr/bin/debchange 
+++ opt/debian/src/devscripts/devscripts-2.10.12/scripts/debchange.pl 
@@ -662,7 +662,7 @@
 	my $lastbug;
 
 	my $uripkg = uri_escape($PACKAGE);
-	my $bugs = `wget -q -O - 'http://bugs.debian.org/cgi-bin/pkgreport.cgi?src=$uripkg'`;
+	my $bugstr = `wget -q -O - 'http://bugs.debian.org/cgi-bin/pkgreport.cgi?src=$uripkg'`;
 	if ($? >> 8 != 0) {
 	    warn "$progname warning: wget failed, so cannot query the bug-tracking system\n";
 	    $opt_query=0;
@@ -670,8 +670,8 @@
 	    # This will now go and execute the "if (@closes and ! $opt_query)" code
 	}
 
-	foreach (split /\n/, $bugs) {
-	    if (m%<a(?: class=\".*?\")? href=\"bugreport.cgi\?bug=([0-9]*).*?>\#\1: (.*?)</a>%i) {
+	foreach (split /\n/, $bugstr) {
+		if (/bugreport.cgi\?bug=([0-9]*).*?>\#[0-9]*: (.*)<\/a>/) {
 		$bugs{$1} = [$2];
 		$lastbug=$1;
 	    }
@@ -686,6 +686,7 @@
 	foreach my $close (@closes) {
 	    if (exists $bugs{$close}) {
 		my ($title,$pkg) = @{$bugs{$close}};
+		$pkg = $PACKAGE if (not defined ($pkg));
 		$title =~ s/^($pkg|$PACKAGE): //;
 		$title =~ s/&quot;/\"/g;
 		$title =~ s/&lt;/</g;
