Only in libgenerics-1.2.1.new/debian/: copyright_newhints
diff -ur libgenerics-1.2.1/debian/rules libgenerics-1.2.1.new/debian/rules
--- libgenerics-1.2.1/debian/rules	2009-05-23 22:57:05.000000000 +0000
+++ libgenerics-1.2.1.new/debian/rules	2009-05-23 22:55:21.000000000 +0000
@@ -34,6 +34,14 @@
 # Suppress wrongly resolved build-essential libstdc++6-dev (see bug#374049)
 common-binary-post-install-arch:: debian/install-libs-stamp
 
+makebuilddir/libgenerics0::
+	#fixup config.sub and config.guess symlinks
+	rm -f config.sub
+	rm -f config.guess
+	ln -s /usr/share/misc/config.sub config.sub
+	ln -s /usr/share/misc/config.guess config.guess
+
+
 debian/install-libs-stamp:
 	d-shlibmove --commit \
 		--override s/libstdc++6-dev// \
@@ -44,6 +52,15 @@
 
 clean::
 	rm -f debian/install-libs-stamp
+	
+	#undo our changes to config.[sub|guess] symlinks
+	#and cdbs's change to ltmain.sh symlink
+	rm -f config.sub
+	rm -f config.guess
+	rm -f ltmain.sh
+	ln -s /usr/share/libtool/config.sub config.sub
+	ln -s /usr/share/libtool/config.guess config.guess
+	ln -s /usr/share/libtool/ltmain.sh ltmain.sh
 
 # Needed by upstream build process
 CDBS_BUILD_DEPENDS := $(CDBS_BUILD_DEPENDS), libxml2-dev
