diff -u dash-0.5.7/debian/changelog dash-0.5.7/debian/changelog
--- dash-0.5.7/debian/changelog
+++ dash-0.5.7/debian/changelog
@@ -1,3 +1,10 @@
+dash (0.5.7-1.1) UNRELEASED; urgency=low
+
+  * Non-maintainer upload.
+  * Revert http://thread.gmane.org/gmane.comp.shells.dash/556
+
+ -- Stéphane Glondu <glondu@debian.org>  Sun, 25 Sep 2011 19:05:10 +0200
+
 dash (0.5.7-1) unstable; urgency=low
 
   * new upstream release.
only in patch2:
unchanged:
--- dash-0.5.7.orig/src/main.c
+++ dash-0.5.7/src/main.c
@@ -171,7 +171,7 @@
 state3:
 	state = 4;
 	if (minusc)
-		evalstring(minusc, sflag ? 0 : EV_EXIT);
+		evalstring(minusc, 0);
 
 	if (sflag || minusc == NULL) {
 state4:	/* XXX ??? - why isn't this before the "if" statement */
