From 7a6d9d92f52c2a8728db87409e0d1dcd6e6dc910 Mon Sep 17 00:00:00 2001
From: Bill Allombert <Bill.Allombert@math.u-bordeaux1.fr>
Date: Sun, 11 Oct 2015 17:54:38 +0200
Subject: [PATCH] autopkgtest.desc: replace reference to packages-tests by
 autopkgtest

---
 autopkgtest.desc |    8 ++++----
 1 file changed, 4 insertions(+), 4 deletions(-)

diff --git a/autopkgtest.desc b/autopkgtest.desc
index dc7ea40..7d3fa3e 100644
--- a/autopkgtest.desc
+++ b/autopkgtest.desc
@@ -2,12 +2,12 @@ Document: autopkgtest
 Title: automatic as-installed package test specification
 Author: The Debian Project
 Abstract: Specification for automatic as-installed (AKA "autopkgtest",
-  or DEP-8) tests for Debian packages
+ or DEP-8) tests for Debian packages
 Section: Debian
 
 Format: text
-Files: /usr/share/doc/debian-policy/package-tests.txt.gz
+Files: /usr/share/doc/debian-policy/autopkgtest.txt.gz
 
 Format: HTML
-Index: /usr/share/doc/debian-policy/package-tests.html
-Files: /usr/share/doc/debian-policy/package-tests.html
+Index: /usr/share/doc/debian-policy/autopkgtest.html
+Files: /usr/share/doc/debian-policy/autopkgtest.html
-- 
1.7.10.4

