diff -pruN 2.~-1.1-1/debian/rules 2.~-1.1-1ubuntu1/debian/rules
--- 2.~-1.1-1/debian/rules	2008-02-18 19:47:43.000000000 +0000
+++ 2.~-1.1-1ubuntu1/debian/rules	2008-02-18 19:35:46.000000000 +0000
@@ -68,10 +68,13 @@ binary-indep: build-indep-stamp
 	dh_installdirs -i
 	$(MAKE) install -C docs DESTDIR=$(CURDIR)/debian/cuyo-data/
 	$(MAKE) install -C data DESTDIR=$(CURDIR)/debian/cuyo-data/
+	$(MAKE) install -C icons DESTDIR=$(CURDIR)/debian/cuyo-data/
 	dh_install -i
 	dh_installdocs -i
 	dh_installman -i
 	dh_installchangelogs -i
+	dh_icons -i
+	dh_desktop -i
 	dh_compress -i
 	dh_fixperms -i
 	dh_installdeb -i
diff -pruN 2.~-1.1-1/icons/cuyo.desktop 2.~-1.1-1ubuntu1/icons/cuyo.desktop
--- 2.~-1.1-1/icons/cuyo.desktop	2007-12-20 17:22:32.000000000 +0000
+++ 2.~-1.1-1ubuntu1/icons/cuyo.desktop	2008-02-18 19:35:46.000000000 +0000
@@ -3,9 +3,8 @@ Name=Cuyo
 Icon=cuyo
 Type=Application
 Terminal=false
-MimeType=application/x-executable
+MimeType=application/x-executable;
 Categories=Game;BlocksGame;
-Encoding=UTF-8
-Comment=A tetris like game with many levels
-Comment[de]=Ein Tetris-artiges Spiel mit vielen Leveln
+Comment=Play a tetris like game with many levels
+Comment[de]=Ein Tetris-artiges Spiel mit vielen Leveln spielen
 Exec=cuyo
diff -pruN 2.~-1.1-1/icons/Makefile.am 2.~-1.1-1ubuntu1/icons/Makefile.am
--- 2.~-1.1-1/icons/Makefile.am	2007-12-20 17:22:32.000000000 +0000
+++ 2.~-1.1-1ubuntu1/icons/Makefile.am	2008-02-18 19:35:46.000000000 +0000
@@ -1,4 +1,4 @@
-desktopdir = $(datadir)/applications
+desktopdir = $(datarootdir)/applications
 desktop_DATA = cuyo.desktop
 
 
@@ -25,7 +25,7 @@ desktop_DATA = cuyo.desktop
 # Das vorige war falsch: die png sollen alle cuyo.png heissen
 # und nicht cuyo-xxxx.png. Deshalb nachfolgend neue Version:
 
-iconsdir = $(datadir)/icons/hicolor
+iconsdir = $(datarootdir)/icons/hicolor
 icons_DATA = cuyo-32x32.png cuyo-64x64.png
 
 install-iconsDATA:
diff -pruN 2.~-1.1-1/icons/Makefile.in 2.~-1.1-1ubuntu1/icons/Makefile.in
--- 2.~-1.1-1/icons/Makefile.in	2008-02-13 19:58:29.000000000 +0000
+++ 2.~-1.1-1ubuntu1/icons/Makefile.in	2008-02-18 19:35:46.000000000 +0000
@@ -15,7 +15,7 @@
 @SET_MAKE@
 
 VPATH = @srcdir@
-pkgdatadir = $(datadir)/@PACKAGE@
+pkgdatadir = $(datarootdir)/@PACKAGE@
 pkglibdir = $(libdir)/@PACKAGE@
 pkgincludedir = $(includedir)/@PACKAGE@
 am__cd = CDPATH="$${ZSH_VERSION+.}$(PATH_SEPARATOR)" && cd
@@ -146,7 +146,7 @@ sysconfdir = @sysconfdir@
 target_alias = @target_alias@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
-desktopdir = $(datadir)/applications
+desktopdir = $(datarootdir)/applications
 desktop_DATA = cuyo.desktop
 
 # The following would automatically put cuyo with the full
@@ -168,7 +168,7 @@ desktop_DATA = cuyo.desktop
 
 # Das vorige war falsch: die png sollen alle cuyo.png heissen
 # und nicht cuyo-xxxx.png. Deshalb nachfolgend neue Version:
-iconsdir = $(datadir)/icons/hicolor
+iconsdir = $(datarootdir)/icons/hicolor
 icons_DATA = cuyo-32x32.png cuyo-64x64.png
 EXTRA_DIST = $(icons_DATA) cuyo.desktop
 all: all-am
