From d1229f61dffaeb516f3f652648fbaa1c3de570cb Mon Sep 17 00:00:00 2001
From: corubba <corubba@gmx.de>
Date: Mon, 28 Mar 2022 01:01:39 +0200
Subject: [PATCH] cryptroot hook: handle absolute keyfile path for decrypt_derived

---
 debian/initramfs/hooks/cryptroot | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/debian/initramfs/hooks/cryptroot b/debian/initramfs/hooks/cryptroot
index 81f65869..d9f01ea3 100644
--- a/debian/initramfs/hooks/cryptroot
+++ b/debian/initramfs/hooks/cryptroot
@@ -113,6 +113,9 @@ crypttab_print_entry() {
         # the boot scripts unlock it first); since _CRYPTTAB_* are local
         # to crypttab_find_and_print_entry() the new value won't
         # override the new ones
+        if [ "${CRYPTTAB_KEY#/}" != "$CRYPTTAB_KEY" ] && [ -b "$CRYPTTAB_KEY" ]; then
+            CRYPTTAB_KEY="$(dmsetup info -c --noheadings -o name -- "$CRYPTTAB_KEY")"
+        fi
         crypttab_find_and_print_entry "$CRYPTTAB_KEY"
     fi
     printf '%s %s %s %s\n' \
-- 
2.35.1

