diff -Nru dokuwiki-0.0.20120125/debian/add-ons/dokuwiki-addsite dokuwiki-0.0.20120125/debian/add-ons/dokuwiki-addsite
--- dokuwiki-0.0.20120125/debian/add-ons/dokuwiki-addsite	2012-01-26 22:03:02.000000000 +0100
+++ dokuwiki-0.0.20120125/debian/add-ons/dokuwiki-addsite	2012-04-01 00:33:16.000000000 +0200
@@ -30,6 +30,8 @@
 install -d -o www-data -g root -m 0700 "$site_dir/data/meta"
 install -d -o www-data -g root -m 0700 "$site_dir/data/pages"
 install -d -o www-data -g root -m 0700 "$site_dir/data/tmp"
+install -d -o www-data -g root -m 0700 "$site_dir/data/media_attic"
+install -d -o www-data -g root -m 0700 "$site_dir/data/media_meta"
 install -d -o root -g www-data -m 0775 "$conf_dir"
 
 
