Removed additional = sign from desktop file's comment line.
diff -u biniax2-1.30/debian/biniax2.desktop biniax2-1.30/debian/biniax2.desktop
--- biniax2-1.30/debian/biniax2.desktop
+++ biniax2-1.30/debian/biniax2.desktop
@@ -2,7 +2,7 @@
 Type=Application
 Version=1.0
 Name=Biniax-2
-Comment==Colorful Logic game with arcade and tactics modes
+Comment=Colorful Logic game with arcade and tactics modes
 Icon=biniax2
 Exec=biniax2
 Terminal=false
diff -u biniax2-1.30/debian/changelog biniax2-1.30/debian/changelog
