diff -ur cron-apt-0.6.6.pristine/debian/control cron-apt-0.6.6/debian/control
--- cron-apt-0.6.6.pristine/debian/control	2007-02-04 11:04:27.000000000 +0000
+++ cron-apt-0.6.6/debian/control	2008-06-26 15:11:35.000000000 +0100
@@ -8,7 +8,7 @@
 
 Package: cron-apt
 Architecture: all
-Depends: apt, debianutils (>= 1.7)
+Depends: apt
 Recommends: liblockfile1, mailx, cron
 Description: automatic update of packages using apt-get
  Contains a tool that is run by a cron job at regular intervals. By default it
