--- a/usr/share/doc/apt-utils/examples/apt-ftparchive.conf	2011-06-17 13:10:20.000000000 +0300
+++ b/usr/share/doc/apt-utils/examples/apt-ftparchive.conf	2012-06-03 17:59:35.330121443 +0300
@@ -21,21 +21,21 @@ Default {
 BinDirectory "pool/main" {
 	Packages "dists/sid/main/binary-i386/Packages";
 	Sources "dists/sid/main/source/Sources";
-	Contents "dists/sid/Contents-i386";
+	Contents "dists/sid/main/Contents-i386";
 }
 
 // This is the same for the contrib section
 BinDirectory "pool/contrib" {
 	Packages "dists/sid/contrib/binary-i386/Packages";
 	Sources "dists/sid/contrib/source/Sources";
-	Contents "dists/sid/Contents-i386";
+	Contents "dists/sid/contrib/Contents-i386";
 }
 
 // This is the same for the non-free section
 BinDirectory "pool/non-free" {
 	Packages "dists/sid/non-free/binary-i386/Packages";
 	Sources "dists/sid/non-free/source/Sources";
-	Contents "dists/sid/Contents-i386";
+	Contents "dists/sid/non-free/Contents-i386";
 };
 
 // By default all Packages should have the extension ".deb"
