--- debian-installer/amd64/boot-screens/prompt.cfg.orig	2008-10-29 13:09:04.000000000 +0100
+++ debian-installer/amd64/boot-screens/prompt.cfg	2008-11-13 13:07:06.000000000 +0100
@@ -1,7 +1,7 @@
+include debian-installer/amd64/boot-screens/menu.cfg
 prompt 1
 display debian-installer/amd64/boot-screens/f1.txt
 timeout 0
-include debian-installer/amd64/boot-screens/menu.cfg
 
 f1 debian-installer/amd64/boot-screens/f1.txt
 f2 debian-installer/amd64/boot-screens/f2.txt
