--- debian-installer/amd64/boot-screens/menu.cfg.orig	2008-10-29 13:09:04.000000000 +0100
+++ debian-installer/amd64/boot-screens/menu.cfg	2008-11-13 11:55:11.000000000 +0100
@@ -1,3 +1,5 @@
+serial 0 9600 0
+
 menu hshift 13
 menu width 49
 
@@ -18,6 +20,15 @@
 	include debian-installer/amd64/boot-screens/adgtk.cfg
 	include debian-installer/amd64/boot-screens/adamdgtk.cfg
 menu end
+menu begin serial
+	menu title Serial console 
+	label mainmenu
+		menu label ^Back..
+		menu exit
+	include debian-installer/amd64/boot-screens/stdmenu.cfg
+	include debian-installer/amd64/boot-screens/sadtext.cfg
+	include debian-installer/amd64/boot-screens/adamdtext.cfg
+menu end
 label help
 	menu label ^Help
 	config debian-installer/amd64/boot-screens/prompt.cfg
--- /dev/null	2008-10-17 17:20:56.736525434 +0200
+++ debian-installer/amd64/boot-screens/sadtext.cfg	2008-11-13 12:29:01.000000000 +0100
@@ -0,0 +1,18 @@
+default sinstall
+label sinstall
+        menu label ^Install
+        menu default
+        kernel debian-installer/amd64/linux
+        append vga=normal initrd=debian-installer/amd64/initrd.gz -- console=ttyS0,9600n8 quiet
+label expert
+	menu label ^Expert install
+	kernel debian-installer/amd64/linux
+	append priority=low vga=normal initrd=debian-installer/amd64/initrd.gz -- console=ttyS0,9600n8
+label rescue
+	menu label ^Rescue mode
+	kernel debian-installer/amd64/linux
+	append debian-installer/serial-console vga=normal initrd=debian-installer/amd64/initrd.gz rescue/enable=true -- console=ttyS0,9600n8 quiet 
+label auto
+	menu label ^Automated install
+	kernel debian-installer/amd64/linux
+	append auto=true priority=critical vga=normal initrd=debian-installer/amd64/initrd.gz -- console=ttyS0,9600n8 quiet 
