From 34d41b8480c3467304383df3633c03ce6c5f34e2 Mon Sep 17 00:00:00 2001
From: gregor herrmann <gregoa@debian.org>
Date: Sat, 25 Aug 2012 14:12:10 +0200
Subject: [PATCH 1/5] whitespace

---
 scripts/uscan.pl |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/uscan.pl b/scripts/uscan.pl
index e118142..1ea4489 100755
--- a/scripts/uscan.pl
+++ b/scripts/uscan.pl
@@ -2099,7 +2099,7 @@ sub get_main_source_dir($$$) {
     print "-- Dirty tarball found.\n" if $verbose;
     if ( $main_source_dir ) { # if tarball is dirty but does contain a $pkg-$newversion dir we will not undirty but leave it as is
         print "-- No idea how to create proper tarball structure - leaving as is.\n" if $verbose;
-    	return $tempdir;
+        return $tempdir;
     }
     print "-- Move files to subdirectory $pkg-$newversion.\n" if $verbose;
     $main_source_dir = $tempdir . '/' . $pkg . '-' . $newversion ;
@@ -2108,7 +2108,7 @@ sub get_main_source_dir($$$) {
 	unless ($file =~ /^\.\.?/) {
             # move("${tempdir}/$file", $main_source_dir) or die("Unable to move ${tempdir}/$file directory $main_source_dir\n");
             unless ( move("${tempdir}/$file", $main_source_dir) ) {
-        	# HELP: why can't perl move not move directories????
+            # HELP: why can't perl move not move directories????
                 print "Perl move seems to be not able to ` move(\"${tempdir}/$file\", $main_source_dir) ` ... trying system mv\n" if $debug;
                 system( "mv ${tempdir}/$file $main_source_dir" ) ;
             }
-- 
1.7.10.4

