From 67b13a72c801fb7419254784e443a90506819bc4 Mon Sep 17 00:00:00 2001
From: gregor herrmann <gregoa@debian.org>
Date: Sat, 25 Aug 2012 14:19:48 +0200
Subject: [PATCH 2/5] s/fuffix/suffix/g

---
 scripts/uscan.pl |    4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/uscan.pl b/scripts/uscan.pl
index 1ea4489..cd69217 100755
--- a/scripts/uscan.pl
+++ b/scripts/uscan.pl
@@ -1483,9 +1483,9 @@ EOF
             if ( $nfiles_before == $nfiles_after ) {
                 print "-- Source tree remains identical - no need for repacking.\n" if $verbose;
             } else {
-                my $excludefuffix = '+dfsg' ;
+                my $excludesuffix = '+dfsg' ;
                 my $suffix = 'gz' ;
-                my $newfile_base_dfsg = "${pkg}_${newversion}${excludefuffix}.orig.tar.$suffix" ;
+                my $newfile_base_dfsg = "${pkg}_${newversion}${excludesuffix}.orig.tar.$suffix" ;
                 system("cd $tempdir; GZIP='-n -9' tar --owner=root --group=root --mode=a+rX -czf \"$absdestdir/$newfile_base_dfsg\" $globpattern") == 0
                    or die("Excluding files failed (could not create tarball)\n");
                 $symlink = 'no' # prevent symlinking or renaming
-- 
1.7.10.4

