diff -Nru crash-7.1.8/debian/tests/control crash-7.1.8/debian/tests/control
--- crash-7.1.8/debian/tests/control	2017-04-17 22:25:08.000000000 +0200
+++ crash-7.1.8/debian/tests/control	2017-06-14 16:13:12.000000000 +0200
@@ -1,3 +1,3 @@
 Tests: live
-Restrictions: needs-root
-Depends: @, lsb-release, sudo
+Restrictions: needs-root isolation-machine
+Depends: @, lsb-release, software-properties-common, sudo
diff -Nru crash-7.1.8/debian/tests/live crash-7.1.8/debian/tests/live
--- crash-7.1.8/debian/tests/live	2017-04-17 22:25:08.000000000 +0200
+++ crash-7.1.8/debian/tests/live	2017-06-14 16:13:12.000000000 +0200
@@ -7,12 +7,13 @@
     apt-get update
     apt-get install linux-image-$(uname -r)-dbg
 elif [ "$(lsb_release -is)" = "Ubuntu" ]; then
-    sudo tee /etc/apt/sources.list.d/ddebs.list << EOF
-    deb http://ddebs.ubuntu.com/ $(lsb_release -cs)          main
-    deb http://ddebs.ubuntu.com/ $(lsb_release -cs)-updates  main
-EOF
+    sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys C8CAB6595FDFF622 2>&1
+    # add available pockets
+    for pocket in "" "-updates" "-proposed"; do
+        apt-add-repository "deb http://ddebs.ubuntu.com/ $(lsb_release -cs)${pocket} main"
+        apt-get update 2>&1 || apt-add-repository --remove "deb http://ddebs.ubuntu.com/ $(lsb_release -cs)${pocket} main"
+    done
     # avoid stderr output
-    sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys ECDCAD72428D7C01 2>&1
     sudo apt-get update
     sudo DEBIAN_FRONTEND=noninteractive apt-get install -y -q linux-image-$(uname -r)-dbgsym
 fi
