--- origs/debianutils-4.2.2/debian/control	2012-03-24 14:52:38.000000000 +0000
+++ patched/debianutils-4.2.2/debian/control	2012-03-27 11:45:44.000000000 +0000
@@ -13,6 +13,7 @@
 Depends: sensible-utils
 Section: utils
 Essential: yes
+Multi-Arch: foreign
 Description: Miscellaneous utilities specific to Debian
  This package provides a number of small utilities which are used
  primarily by the installation scripts of Debian packages, although
