--- /usr/share/xml/docbook/stylesheet/docbook-xsl-ns/template/titlepage.xsl	2009-07-21 17:59:17.000000000 +0000
+++ ./titlepage.xsl	2010-06-12 18:55:32.000000000 +0000
@@ -1309,7 +1309,7 @@
         <xsl:with-param name="orlist">
           <xsl:value-of select="$orlist"/>
           <xsl:if test="not($orlist='')">|</xsl:if>
-          <xsl:value-of select="name($elements[position()=$count])"/>
+          <xsl:value-of select="concat('d:', name($elements[position()=$count]))"/>
         </xsl:with-param>
       </xsl:call-template>
     </xsl:otherwise>
