diff -adNru developers-reference-3.3.8/developers-reference.sgml developers-reference-3.3.8.work/developers-reference.sgml
--- developers-reference-3.3.8/developers-reference.sgml	2006-11-12 11:05:22.000000000 +0000
+++ developers-reference-3.3.8.work/developers-reference.sgml	2007-02-27 20:02:05.000000000 +0000
@@ -2577,6 +2577,9 @@
 If you want to remove another package, you have to get the approval
 of its maintainer.
 	<p>
+Further information relating to these and other package removal related 
+topics may be found at <url id="http://wiki.debian.org/ftpmaster_Removals">.
+	<p>
 If in doubt concerning whether a package is disposable, email
 &email-debian-devel; asking for opinions.  Also of interest is the
 <prgn>apt-cache</prgn> program from the <package>apt</package>
