diff -ur exult-1.00.orig/conf/xmain.cc exult-1.00/conf/xmain.cc
--- exult-1.00.orig/conf/xmain.cc	2002-10-30 11:10:30.000000000 -0600
+++ exult-1.00/conf/xmain.cc	2003-11-18 09:21:50.000000000 -0600
@@ -27,6 +27,7 @@
 #include <iostream>
 #include <string>
 #include "common_types.h"
+#include <cassert>
 
 using std::string;
 
diff -ur exult-1.00.orig/files/databuf.h exult-1.00/files/databuf.h
--- exult-1.00.orig/files/databuf.h	2002-05-22 17:24:04.000000000 -0500
+++ exult-1.00/files/databuf.h	2003-11-18 09:22:14.000000000 -0600
@@ -28,6 +28,7 @@
 #include <iomanip>
 #include "U7file.h"
 #include "utils.h"
+#include <cassert>
 
 typedef char * charptr;
 
diff -ur exult-1.00.orig/files/rwregress.cc exult-1.00/files/rwregress.cc
--- exult-1.00.orig/files/rwregress.cc	2002-02-04 15:01:24.000000000 -0600
+++ exult-1.00/files/rwregress.cc	2003-11-18 09:21:30.000000000 -0600
@@ -6,6 +6,7 @@
 #include <iostream>
 #include "utils.h"
 #include <strstream>
+#include <cassert>
 
 using std::cout;
 using std::endl;
diff -ur exult-1.00.orig/tools/cmanip.cc exult-1.00/tools/cmanip.cc
--- exult-1.00.orig/tools/cmanip.cc	2002-10-30 11:11:36.000000000 -0600
+++ exult-1.00/tools/cmanip.cc	2003-11-18 09:22:35.000000000 -0600
@@ -29,6 +29,7 @@
 #include <string>
 #include "exult_constants.h"
 #include <vector>
+#include <cassert>
 
 const std::string c_empty_string;
 
diff -ur exult-1.00.orig/usecode/compiler/ucfun.cc exult-1.00/usecode/compiler/ucfun.cc
--- exult-1.00.orig/usecode/compiler/ucfun.cc	2002-05-23 04:09:00.000000000 -0500
+++ exult-1.00/usecode/compiler/ucfun.cc	2003-11-18 09:23:04.000000000 -0600
@@ -32,6 +32,7 @@
 #include "utils.h"
 #include "opcodes.h"
 #include "ucexpr.h"			/* Needed only for Write2(). */
+#include <cassert>
 
 using std::strlen;
 using std::memcpy;
diff -ur exult-1.00.orig/usecode/ucxt/include/ops.h exult-1.00/usecode/ucxt/include/ops.h
--- exult-1.00.orig/usecode/ucxt/include/ops.h	2002-06-09 23:17:20.000000000 -0500
+++ exult-1.00/usecode/ucxt/include/ops.h	2003-11-18 09:23:31.000000000 -0600
@@ -25,6 +25,7 @@
 #include <iostream>
 #include "Configuration.h"
 #include "ucc.h"
+#include <cassert>
 
 std::vector<std::string> qnd_ocsplit(const std::string &s);
 
