diff -u ecryptfs-utils-30/debian/patches/00list ecryptfs-utils-30/debian/patches/00list
--- ecryptfs-utils-30/debian/patches/00list
+++ ecryptfs-utils-30/debian/patches/00list
@@ -1,0 +2 @@
+02-dont_set_pamdir_to_lib64.dpatch
only in patch2:
unchanged:
--- ecryptfs-utils-30.orig/debian/patches/02-dont_set_pamdir_to_lib64.dpatch
+++ ecryptfs-utils-30/debian/patches/02-dont_set_pamdir_to_lib64.dpatch
@@ -0,0 +1,41 @@
+#! /bin/sh /usr/share/dpatch/dpatch-run
+## 02-dont_set_pamdir_to_lib64.dpatch by Michael Bienia <geser@ubuntu.com>
+##
+## All lines beginning with `## DP:' are a description of the patch.
+## DP: No description.
+
+@DPATCH@
+diff -urNad ecryptfs-utils-30~/configure ecryptfs-utils-30/configure
+--- ecryptfs-utils-30~/configure	2007-10-25 11:34:57.000000000 +0200
++++ ecryptfs-utils-30/configure	2008-01-03 16:07:04.000000000 +0100
+@@ -2226,13 +2226,6 @@
+ else
+ 
+ 		case $target in
+-			*64* | *s390x)
+-				if test "${prefix}" != "/usr" ; then
+-					pamdir="\$(prefix)/lib64/security"
+-				else
+-					pamdir="/lib64/security"
+-				fi
+-			;;
+ 			*)
+ 				if test "${prefix}" != "/usr" ; then
+ 					pamdir="\$(prefix)/lib/security"
+diff -urNad ecryptfs-utils-30~/configure.ac ecryptfs-utils-30/configure.ac
+--- ecryptfs-utils-30~/configure.ac	2007-10-25 10:45:37.000000000 +0200
++++ ecryptfs-utils-30/configure.ac	2008-01-03 16:06:57.000000000 +0100
+@@ -98,13 +98,6 @@
+ 	[pamdir="${withval}"],
+ 	[
+ 		case $target in
+-			*64* | *s390x)
+-				if test "${prefix}" != "/usr" ; then
+-					pamdir="\$(prefix)/lib64/security"
+-				else
+-					pamdir="/lib64/security"
+-				fi
+-			;;
+ 			*)
+ 				if test "${prefix}" != "/usr" ; then
+ 					pamdir="\$(prefix)/lib/security"
