From e31730ed93534d0a3bde14799cba9775165b2b21 Mon Sep 17 00:00:00 2001
From: Julian Andres Klode <jak@debian.org>
Date: Mon, 1 Mar 2010 14:58:02 +0100
Subject: [PATCH] Upgrade to the new python-apt API.

---
 debian/control |    2 +-
 src/debsecan   |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/control b/debian/control
index 3fa00af..6b3198e 100644
--- a/debian/control
+++ b/debian/control
@@ -8,7 +8,7 @@ Standards-Version: 3.7.2.0
 
 Package: debsecan
 Architecture: all
-Depends: debconf | debconf-2.0, python (>= 2.3), python-apt
+Depends: debconf | debconf-2.0, python (>= 2.3), python-apt (>= 0.7.93)
 Recommends: cron, exim4 | mail-transport-agent
 Description: Debian Security Analyzer
  debsecan is a tool to generate a list of vulnerabilities which affect
diff --git a/src/debsecan b/src/debsecan
index ae6df02..2893b76 100644
--- a/src/debsecan
+++ b/src/debsecan
@@ -85,7 +85,7 @@ class Version:
         return 'Version(%s)' % `self.__asString`
 
     def __cmp__(self, other):
-        return apt_pkg.VersionCompare(self.__asString, other.__asString)
+        return apt_pkg.version_compare(self.__asString, other.__asString)
 
 class PackageFile:
     """A Debian package file.
-- 
1.7.0

