diff --git a/init.d/bind9 b/init.d/bind9
index a4c55e8..4558b35 100755
--- a/init.d/bind9
+++ b/init.d/bind9
@@ -41,6 +41,7 @@ check_network() {
     return 0
 }
 
+ret=0
 case "$1" in
     start)
 	log_daemon_msg "Starting domain name service..." "bind9"
@@ -70,6 +71,7 @@ case "$1" in
 	    fi
 	    log_end_msg 0
 	else
+	    ret=$?
 	    log_end_msg 1
 	fi
     ;;
@@ -102,6 +104,7 @@ case "$1" in
 	fi
 
 	/usr/sbin/rndc reload >/dev/null
+	ret=$?
 	log_end_msg $?
     ;;
 
@@ -112,11 +115,12 @@ case "$1" in
 
 	$0 stop
 	$0 start
+	ret=$?
     ;;
     
     status)
-    	ret=0
-	status_of_proc -p ${PIDFILE} /usr/sbin/named bind9 2>/dev/null || ret=$?
+	status_of_proc -p ${PIDFILE} /usr/sbin/named bind9 2>/dev/null
+	ret=$?
 	;;
 
     *)
@@ -125,4 +129,4 @@ case "$1" in
     ;;
 esac
 
-exit 0
+exit $ret
