Index: src/generic/aptcache.cc
===================================================================
--- src/generic/aptcache.cc	(revision 3415)
+++ src/generic/aptcache.cc	(working copy)
@@ -354,19 +354,19 @@
 		if(ver.VerStr()==estate.candver)
 		  {
 		    SetCandidateVersion(ver);
-		    MarkInstall(ver.ParentPkg());
+		    MarkInstall(ver.ParentPkg(), false);
 		    break;
 		  }
 	    }
 	  else
 	    if(i.CurrentVer().end())
-	      MarkInstall(i);
+	      MarkInstall(i, false);
 	    else
 	      {
 		SetReInstall(i, estate.reinstall);
 
 		if(estate.upgrade && state.Upgradable())
-		  MarkInstall(i);
+		  MarkInstall(i, false);
 	      }
 	  break;
 	case pkgCache::State::Hold:
Index: ChangeLog
===================================================================
--- ChangeLog	(revision 3415)
+++ ChangeLog	(working copy)
@@ -1,3 +1,13 @@
+2005-06-22  Daniel Burrows  <dburrows@debian.org>
+
+	* src/generic/aptcache.cc:
+
+	  Disable auto-installation of dependencies when restoring package
+	  states, to make sure we reproduce the state the user expects and
+	  to avoid ordering bugs (my goodness, how did this big gaping
+	  monster manage to avoid detection for so long?).
+	  Closes: #315359
+
 2005-06-22 Luk Claes <luk@debian.org>
 
 	* Updated Dutch translation
