diff -u awstats-6.7.dfsg/debian/rules awstats-6.7.dfsg/debian/rules
--- awstats-6.7.dfsg/debian/rules
+++ awstats-6.7.dfsg/debian/rules
@@ -44,7 +44,7 @@
 		wwwroot/cgi-bin/awstats.model.conf > debian/awstats.conf
 
 clean::
-	rm -f debian/awstats.conf debian/firefox.png
+	rm -f debian/awstats.conf debian/icons/firefox.png
 
 # Remove badly coded PDF, and .cvsignore
 binary-post-install/awstats::
diff -u awstats-6.7.dfsg/debian/changelog awstats-6.7.dfsg/debian/changelog
--- awstats-6.7.dfsg/debian/changelog
+++ awstats-6.7.dfsg/debian/changelog
@@ -1,3 +1,11 @@
+awstats (6.7.dfsg-1.1) unstable; urgency=low
+
+  * Non-maintainer upload during
+    credativ BSP 2008
+  * Fix release goal: FTBFS if build twice in a row (Closes: #442500)
+
+ -- Martin Zobel-Helas <zobel@debian.org>  Sun, 06 Apr 2008 02:50:36 +0200
+
 awstats (6.7.dfsg-1) unstable; urgency=low
 
   * New upstream release. Closes: bug#436572, thanks to Daniel Baumann.
