--- /usr/bin/uscan	2013-12-23 17:31:35.000000000 -0300
+++ uscan.pl	2014-01-09 11:29:49.685459229 -0300
@@ -1809,6 +1809,15 @@
 		foreach my $dir (@dirs) { print STDERR "     $$dir[1]\n"; }
 	    }
 	    @dirs = Devscripts::Versort::upstream_versort(@dirs);
+            if ($download_version) {
+                print STDERR " $download_version\n";
+                foreach my $dir (@dirs) {
+                    if ($$dir[0] eq $download_version) {
+                        print STDERR "Found it: $$dir[2]\n";
+                        return $$dir[1];
+                    }
+                }
+            }
 	    my ($newversion, $newdir) = @{$dirs[0]};
 	    return $newdir;
 	} else {
