diff --git a/Makefile b/Makefile
index fef2588..f8b5808 100644
--- a/Makefile
+++ b/Makefile
@@ -73,6 +73,8 @@ CODE	:=	sid
 ARCH	:=	amd64
 UDEBA	:=	$(DEBM)/$(CODE)
 UDEBB	:=	$(DEBM)/experimental
+UDEBC	:=	$(DEBM)/buster
+UDEBD	:=	$(DEBM)/stretch
 DR_VERSION :=	$(shell dpkg-parsechangelog --show-field Version)
 
 # AsciiDoc source file names in $(DASC) directories for local update
@@ -152,6 +154,25 @@ packages.bkup.txt:
 	grep-dctrl -e -sPackage,Installed-Size -P "." packages.bkup.tmp > packages.bkup.txt
 	rm packages.bkup.tmp
 
+packages.buster.txt:
+	# FETCH PACKAGE (buster main)
+	@$(call check-command, wget, wget)
+	@$(call check-command, grep-dctrl, dctrl-tools)
+	wget -O - $(UDEBC)/main/binary-$(ARCH)/Packages.xz      | xzcat - > packages.buster.tmp
+	grep-dctrl -e -sPackage,Installed-Size -P "." packages.buster.tmp > packages.buster.txt
+	rm packages.buster.tmp
+
+packages.stretch.txt:
+	# FETCH PACKAGE (stretch main contrib)
+	@$(call check-command, wget, wget)
+	@$(call check-command, grep-dctrl, dctrl-tools)
+	wget -O - $(UDEBD)/main/binary-$(ARCH)/Packages.xz      | xzcat - > packages.stretch.tmp
+	grep-dctrl -e -sPackage,Installed-Size -P "." packages.stretch.tmp > packages.stretch.txt
+	rm packages.stretch.tmp
+	wget -O - $(UDEBD)/contrib/binary-$(ARCH)/Packages.xz      | xzcat - > packages.stretch.tmp
+	grep-dctrl -e -sPackage,Installed-Size -P "." packages.stretch.tmp >> packages.stretch.txt
+	rm packages.stretch.tmp
+
 all-popcon-results.txt:
 	# POPCON RESULTS
 	wget -O - $(UPOPC) | zcat - > all-popcon-results.txt
@@ -220,9 +241,9 @@ popcon.ent: all-popcon-results.txt all-popcon-pkgs.txt all-popcon-submissions.tx
 	echo "<!ENTITY pop-packages \"$$(grep -e '^Package:' all-popcon-pkgs.txt | wc -l)\">"				>> popcon.ent
 	grep -e '^Package:' all-popcon-pkgs.txt | grep -f pkg.lst | $(DBIN)/popconent `cat all-popcon-submissions.txt`	>> popcon.ent
 
-pkgsize.ent: pkg.lst packages.txt packages.bkup.txt
+pkgsize.ent: pkg.lst packages.txt packages.bkup.txt packages.buster.txt packages.stretch.txt
 	# GENERATE pkgsize.ent
-	sort pkg.lst | uniq | $(DBIN)/sizeent packages.txt packages.bkup.txt	> pkgsize.ent
+	sort pkg.lst | uniq | $(DBIN)/sizeent packages.txt packages.bkup.txt packages.buster.txt packages.stretch.txt > pkgsize.ent
 
 	# POPCON
 	wget -O - $(UPOPC) | zcat - > all-popcon-results.txt
diff --git a/bin/sizeent b/bin/sizeent
index 795f7aa..41f171c 100755
--- a/bin/sizeent
+++ b/bin/sizeent
@@ -12,6 +12,8 @@ set -e
 #<xsl:variable name="lletters">abcdefghijklmnopqrstuvwxyzabcdefghij</xsl:variable>
 packages=$1
 bkups=$2
+busterpackages=$3
+stretchpackages=$4
 while read X ; do
   #echo "process: $X" >&2
   echo -n "." >&2
@@ -23,6 +25,12 @@ while read X ; do
   elif SIZE=$(grep-dctrl -e -n --field=Package --show-field=Installed-Size  \
                     --pattern="^$(echo $X|sed -e 's/\+/\\+/g')\$" $bkups ) ; then
     echo "<!ENTITY size-${PKG} \"${SIZE}(*)\">"
+  elif SIZE=$(grep-dctrl -e -n --field=Package --show-field=Installed-Size  \
+                    --pattern="^$(echo $X|sed -e 's/\+/\\+/g')\$" $busterpackages ) ; then
+    echo "<!ENTITY size-${PKG} \"${SIZE}\">"
+  elif SIZE=$(grep-dctrl -e -n --field=Package --show-field=Installed-Size  \
+                    --pattern="^$(echo $X|sed -e 's/\+/\\+/g')\$" $stretchpackages ) ; then
+    echo "<!ENTITY size-${PKG} \"${SIZE}\">"
   else
     echo "<!ENTITY size-${PKG} \"NOT_FOUND\">"
     echo  "" >&2
