diff -u didiwiki-0.5/debian/rules didiwiki-0.5/debian/rules
--- didiwiki-0.5/debian/rules
+++ didiwiki-0.5/debian/rules
@@ -7 +7 @@
-	dh_install -m 644 debian/styles.css etc/didiwiki/
+	dh_install debian/styles.css debian/etc/didiwiki/
diff -u didiwiki-0.5/debian/changelog didiwiki-0.5/debian/changelog
--- didiwiki-0.5/debian/changelog
+++ didiwiki-0.5/debian/changelog
@@ -1,3 +1,12 @@
+didiwiki (0.5-5+nmu2) unstable; urgency=high
+
+  * Non-maintainer upload.
+  * Removed the non-existant '-m 644' option of dh_install from
+    debian/rules. (Closes: #518861)
+  * Urgency bumped to 'High' because it closes a RC Bug.
+
+ -- Mauro Lizaur <mauro@cacavoladora.org>  Mon, 18 May 2009 11:01:54 -0300
+
 didiwiki (0.5-5.1) unstable; urgency=low
 
   * Non-maintainer upload.
