From 760f373a87a6b78db732428a650cb5be79be21dc Mon Sep 17 00:00:00 2001
From: Osamu Aoki <osamu@debian.org>
Date: Sat, 5 Sep 2015 23:56:12 +0900
Subject: [PATCH 4/4] POD update for qx/script/

---
 scripts/uscan.pl | 23 +++++++++++++++--------
 1 file changed, 15 insertions(+), 8 deletions(-)

diff --git a/scripts/uscan.pl b/scripts/uscan.pl
index 5baf618..92d4d97 100644
--- a/scripts/uscan.pl
+++ b/scripts/uscan.pl
@@ -920,16 +920,23 @@ details.
 =item * multiple rules can be specified for a I<rule> by making a concatenated
 string of B<;> (semicolon) separated operations.
 
-=item * I<rule> may only use the B<s>, B<tr> and B<y> operations.
+=item * I<rule> may only use the B<qx> B<s>, B<tr> and B<y> operations.
 
-=item * When the B<s> operation is used, only the B<g>, B<i> and B<x> flags are
-available and rule may not contain any expressions which have the potential to
-execute code (i.e. the B<(?{})> and B<(??{})> constructs are not supported).
+=item * B<qx/>I<script>B</>
 
-=item * If the B<s> operation is used, the replacement can contain back references
-to expressions within parenthesis in the matching regex, like
-B<s/-alpha(\d*)/.a$1/>. These back references must use the B<$1> syntax, as the
-B<\1> syntax is not supported.
+Filter the target string with the F<debian/script> script.  F<debian/script>
+should be executable.
+
+=item * B<s/>I<regex>B</>I<replacement>B</>I<options>
+
+Regex pattern match and replace the target string.  Only the B<g>, B<i> and
+B<x> flags are available.  Use the B<$1> syntax for the back reference (No
+B<\1> syntax).  Code execution is not allowed (i.e. No B<(?{})> nor B<(??{})>
+constructs).
+
+=item * B<y/>I<source>B</>I<dest>B</> or B<tr/>I<source>B</>I<dest>B</>
+
+Transliterate the characters in the target string.
 
 =item * each operation can not contain B<;> (semicolon) nor B<,> (comma).
 
-- 
2.1.4

