diff -aur ejabberd-2.1.9.harden/debian/rules ejabberd-2.1.9/debian/rules
--- ejabberd-2.1.9.harden/debian/rules	2012-01-26 20:48:03.000000000 +0100
+++ ejabberd-2.1.9/debian/rules	2012-01-26 20:49:32.000000000 +0100
@@ -25,7 +25,7 @@
 configure-stamp: $(QUILT_STAMPFN)
 	dh_testdir
 
-	dh_auto_configure $(AUTO_OPTS) -- $(DEB_CONFIGURE_EXTRA_FLAGS)
+	dh_auto_configure $(AUTO_OPTS) -- $(DEB_CONFIGURE_EXTRA_FLAGS) $(shell dpkg-buildflags --export=configure)
 
 	touch configure-stamp
 
Nur in ejabberd-2.1.9/debian: rules~.
