diff --git a/debdelta/debdelta b/debdelta/debdelta
index 36e2d02..a0af15c 100755
--- a/debdelta/debdelta
+++ b/debdelta/debdelta
@@ -3809,7 +3809,8 @@ def delta_upgrade_(args):
     # synopsis lockf(   fd, operation, [length, [start, [whence]]])
     fcntl.lockf(a, fcntl.LOCK_EX | fcntl.LOCK_NB, 0,0,0)
   except IOError, s:
-    if s.errno == 11 :
+    from errno import EAGAIN
+    if s.errno == EAGAIN :
       a=' already locked!'
     else:
       a=str(s)
