--- debian/skel.bashrc
+++ debian/skel.bashrc
@@ -36,9 +36,10 @@
 fi
 
 # set a fancy prompt (non-color, unless we know we "want" color)
-case "$TERM" in
-    xterm-color|*-256color) color_prompt=yes;;
-esac
+if [ -x /usr/bin/tput ] && [ "$(/usr/bin/tput colors || echo 0)" -ge 256 ]
+then
+    color_prompt=yes
+fi
 
 # uncomment for a colored prompt, if the terminal has the capability; turned
 # off by default to not distract the user: the focus in a terminal window
