--- a/dh_ucf
+++ b/dh_ucf
@@ -14,7 +14,7 @@
 
 =head1 SYNOPSIS
 
-B<dh_ucf> [S<I<debhelper options>>] [B<-n>]
+B<dh_ucf> [S<I<debhelper options>>] [B<-A>] [B<-n>] [S<I<file> ...>]
 
 =head1 DESCRIPTION
 
@@ -41,10 +41,20 @@
 
 =over 4
 
+=item B<-A>, B<--all>
+
+Install all files specified by command line parameters in ALL packages
+acted on.
+
 =item B<-n>, B<--no-scripts>
 
 Do not modify F<postinst>/F<postrm> scripts. Turns this command into a no-op.
 
+=item I<file> ...
+
+Install these info files into the first package acted on. (Or in
+all packages if B<-A> is specified).
+
 =back
 
 =head1 NOTES
