--- /usr/sbin/extlinux-update.orig	2012-12-10 13:16:56.000000000 +0100
+++ /usr/sbin/extlinux-update	2013-05-12 18:36:21.559588617 +0200
@@ -155,7 +155,7 @@ _CONFIG="\
 "
 
 # Find linux versions
-_VERSIONS="$(cd /boot && ls vmlinuz-* | grep -v .dpkg-tmp | sed -e 's|vmlinuz-||g' | sort -nr)"
+_VERSIONS="$(cd /boot && ls vmlinuz-* | grep -v .dpkg-tmp | sed -e 's|vmlinuz-||g' | sort -Vr)"
 
 if [ "$(stat --printf %d /)" = "$(stat --printf %d /boot)" ]
 then
