commit fb673db9b8554375a388ba41eae8a3529b7911f8
Author: Jonathan Dowland <jmtd@debian.org>
Date:   Mon May 13 10:43:52 2019 +0100

    Update descriptions to reflect move of luksformat

diff --git a/debian/changelog b/debian/changelog
index d7e31ad3..61ffc795 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,11 @@
+cryptsetup (2:2.2.0~rc0-1.1) UNRELEASED; urgency=medium
+
+  * Non-maintainer upload.
+  * Update package descriptions to reflect the move of luksformat
+    from cryptsetup-bin to cryptsetup-run. Closes: #928751.
+
+ -- Jonathan Dowland <jmtd@debian.org>  Mon, 13 May 2019 10:43:09 +0100
+
 cryptsetup (2:2.2.0~rc0-1) experimental; urgency=low
 
   * New /testing/ upstream release 2.2.0 RC0.  Highlights include:
diff --git a/debian/control b/debian/control
index c09b080b..70cbc973 100644
--- a/debian/control
+++ b/debian/control
@@ -51,6 +51,9 @@ Description: disk encryption support - startup scripts
  automatically configuring encrypted devices at boot time via the config
  file /etc/crypttab. Additional features are cryptoroot support through
  initramfs-tools and several supported ways to read a passphrase or key.
+ .
+ This package provides the cryptdisk_start and stop wrappers and
+ luksformat.
 
 Package: cryptsetup-bin
 Architecture: linux-any
@@ -61,7 +64,8 @@ Description: disk encryption support - command line tools
  device mapper target dm-crypt. It features integrated Linux Unified Key
  Setup (LUKS) support.
  .
- This package provides cryptsetup, cryptsetup-reencrypt and luksformat.
+ This package provides cryptsetup, cryptsetup-reencrypt, integritysetup
+ and veritysetup.
 
 Package: cryptsetup-initramfs
 Architecture: all
