--- debmirror.orig	2011-02-22 18:26:05.996260712 +0100
+++ debmirror	2011-02-22 18:26:33.332207866 +0100
@@ -2505,7 +2505,7 @@
       $file =~ m:(^.*)/:;
       make_dir ("$tdir/$image_dir/$1") if $1;
       if (!remote_get("$image_dir/$file", $tdir) ||
-          !check_file(file => "$tdir/$image_dir/$file", size => -1, MD5Sum => $di_files{$image_dir}{$file}{md5sum})) {
+          !check_file(filename => "$tdir/$image_dir/$file", size => -1, MD5Sum => $di_files{$image_dir}{$file}{md5sum})) {
 	$lres = 0;
 	last if (! $do_dry_run);
       }
