diff -Nru efibootmgr-0.11.0/debian/changelog efibootmgr-0.11.0/debian/changelog
--- efibootmgr-0.11.0/debian/changelog	2014-10-29 05:41:15.000000000 +0100
+++ efibootmgr-0.11.0/debian/changelog	2014-11-22 15:56:31.000000000 +0100
@@ -1,3 +1,11 @@
+efibootmgr (0.11.0-1.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Skip dh_auto_install, it installs binary into /usr/sbin and is not
+    actually needed with the current packaging (Closes: 768889)
+
+ -- Andreas Bombe <aeb@debian.org>  Sat, 22 Nov 2014 15:38:43 +0100
+
 efibootmgr (0.11.0-1) unstable; urgency=medium
 
   * New upstream version
diff -Nru efibootmgr-0.11.0/debian/rules efibootmgr-0.11.0/debian/rules
--- efibootmgr-0.11.0/debian/rules	2014-10-29 05:29:33.000000000 +0100
+++ efibootmgr-0.11.0/debian/rules	2014-11-22 15:56:31.000000000 +0100
@@ -8,6 +8,10 @@
 %:
 	dh $@
 
+# upstream build installs into /usr/sbin ignoring target directory;
+# since the install step is not actually needed just skip it
+override_dh_auto_install:
+
 override_dh_installman:
 	dh_installman
 
