diff --git a/dl10n-check b/dl10n-check
index 4bad7a1..7b6b0cf 100755
--- a/dl10n-check
+++ b/dl10n-check
@@ -281,7 +281,7 @@ PKG: while ($dsc = shift @pkg_list) {
                 $data->maintainer($pkg, $maint);
                 my $newer = ($data->version($pkg) ne $ver);
                 if ($newer) {
-                        $newer = system ("dpkg","--compare-versions", $data->version($pkg), "\>", $ver);
+                        $newer = system ("dpkg","--compare-versions", $data->version($pkg), "gt", $ver);
                 }
                 if ((!$force) && $newer==0 && $data->version($pkg) ne ""
                         && !( $force_material &&
