diff -urNw dnsmasq-2.41rc2.orig/debian/control dnsmasq-2.41rc2/debian/control
--- dnsmasq-2.41rc2.orig/debian/control	2008-02-05 14:21:09.995049211 +0100
+++ dnsmasq-2.41rc2/debian/control	2008-02-05 14:21:31.483049072 +0100
@@ -6,8 +6,8 @@
 Maintainer: Simon Kelley <simon@thekelleys.org.uk>
 Standards-Version: 3.7.2
 
 Package: dnsmasq
-Architecture: any
+Architecture: all
 Depends: netbase, adduser, dnsmasq-base
 Suggests: resolvconf
 Conflicts: pdnsd, resolvconf (<<1.15)
--- dnsmasq-2.41rc2.orig/debian/rules	2008-02-05 14:21:09.995049211 +0100
+++ dnsmasq-2.41rc2/debian/rules	2008-02-05 14:21:27.035471412 +0100
@@ -9,7 +9,7 @@
 # paternity under the Copyright, Designs and Patents Act 1988.)
 # This file may have to be extensively modified
 
-package=dnsmasq
+package=dnsmasq-base
 
 # policy manual, section 10.1
 ifneq (,$(findstring noopt,$(DEB_BUILD_OPTIONS)))
@@ -52,6 +52,7 @@
 	install -d debian/daemon/DEBIAN\
 		-d debian/base/DEBIAN\
 	        -d debian/base/usr/share/doc/$(package)\
+	        -d debian/daemon/usr/share/doc\
 		-d debian/base/usr/share/doc/$(package)/examples\
 	        -d debian/daemon/etc/init.d\
 	        -d debian/daemon/etc/resolvconf/update.d\
@@ -66,6 +67,7 @@
 	install -m 644 debian/default debian/daemon/etc/default/dnsmasq
 	install -m 644 dnsmasq.conf.example debian/daemon/etc/dnsmasq.conf
 	install -m 644 dbus/dnsmasq.conf debian/daemon/etc/dbus-1/system.d/dnsmasq.conf
+	ln -s $(package) debian/daemon/usr/share/doc/dnsmasq
 	make $(TARGET) PREFIX=/usr DESTDIR=`pwd`/debian/base CFLAGS="$(CFLAGS)" COPTS="$(COPTS)" CC=gcc AWK=gawk
 ifeq (,$(findstring nodocs,$(DEB_BUILD_OPTIONS)))
 	cp doc.html debian/base/usr/share/doc/$(package)/.
