diff --git a/methods/https.cc b/methods/https.cc
index 0499af0..1b3d867 100644
--- a/methods/https.cc
+++ b/methods/https.cc
@@ -132,8 +132,14 @@ void HttpsMethod::SetupProxy()  					/*{{{*/
    {
       // Parse no_proxy, a comma (,) separated list of domains we don't want to use
       // a proxy for so we stop right here if it is in the list
-      if (getenv("no_proxy") != 0 && CheckDomainList(ServerName.Host,getenv("no_proxy")) == true)
+      if (getenv("no_proxy") != 0 &&
+          CheckDomainList(ServerName.Host,getenv("no_proxy")) == true)
+      {
+         Proxy = "";
+         curl_easy_setopt(curl, CURLOPT_PROXY, NULL);
+         curl_easy_setopt(curl, CURLOPT_PROXYPORT, 0);
 	 return;
+      }
    } else {
       const char* result = getenv("https_proxy");
       // FIXME: Fall back to http_proxy is to remain compatible with
