--- /usr/lib/cgi-bin/blosxom.orig-2.0.2	2007-06-11 15:47:48.000000000 +0100
+++ /usr/lib/cgi-bin/blosxom	2007-06-12 14:35:24.000000000 +0100
@@ -67,7 +67,10 @@
 
 ## On Debian GNU/Linux systems, read configuration files (if found)
 ## Dirk Eddelbuettel <edd@debian.org>
-for $rcfile ("/etc/blosxom/blosxom.conf", "/etc/blosxom.conf", param("-f")) {
+$conffile = undef;
+use Getopt::Long qw(:config pass_through debug);
+GetOptions("f=s" => \$conffile);
+for $rcfile ("/etc/blosxom/blosxom.conf", "/etc/blosxom.conf", (defined $conffile ? ($conffile) : ()) ) {
   if (-r $rcfile) {
     open (RC, "< $rcfile") or die "Cannot open $rcfile: $!";
     while (<RC>) {
