From 044f44ed267e084cc24103a662456b0c7199ee09 Mon Sep 17 00:00:00 2001
From: intrigeri <intrigeri@debian.org>
Date: Fri, 19 Jul 2019 14:55:38 +0000
Subject: [PATCH 3/3] cachefilesd (0.10.10-0.2)

Git-Dch: Ignore
---
 debian/changelog | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/debian/changelog b/debian/changelog
index 1ec7b2f..74c5188 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,12 @@
+cachefilesd (0.10.10-0.2) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Disable secctx in the default cachefilesd.conf (Closes: #909523).
+    Accordingly, document in README.Debian how to set the correct security
+    context under SELinux.
+
+ -- intrigeri <intrigeri@debian.org>  Fri, 19 Jul 2019 14:55:33 +0000
+
 cachefilesd (0.10.10-0.1) unstable; urgency=medium
 
   * Non-maintainer upload.
-- 
2.22.0

