diff -Npru binutils-2.22.orig/debian/rules binutils-2.22/debian/rules
--- binutils-2.22.orig/debian/rules	2013-03-19 22:45:42.000000000 +0100
+++ binutils-2.22/debian/rules	2013-03-19 22:48:53.626766646 +0100
@@ -212,7 +212,10 @@ ifeq ($(DEB_HOST_ARCH),s390x)
 	CONFARGS += --enable-targets=s390-linux-gnu
 endif
 ifeq ($(DEB_HOST_ARCH),i386)
-	CONFARGS += --enable-targets=x86_64-linux-gnu
+	CONFARGS += --enable-targets=x86_64-linux-gnu,x86_64-pep
+endif
+ifeq ($(DEB_HOST_ARCH),amd64)
+	CONFARGS += --enable-targets=x86_64-pep
 endif
 ifeq ($(DEB_HOST_ARCH),kfreebsd-i386)
 	CONFARGS += --enable-targets=x86_64-kfreebsd-gnu
