diff --recursive -u alsa-utils-1.0.18/debian/asoundconf alsa-utils-1.0.18.505089/debian/asoundconf
--- alsa-utils-1.0.18/debian/asoundconf	2009-02-12 08:29:51.000000000 +0100
+++ alsa-utils-1.0.18.505089/debian/asoundconf	2009-02-12 08:32:04.753031103 +0100
@@ -340,9 +340,9 @@
 	sep = re.compile(r'[ ]')
 	r = re.compile('^defaults.pcm.card')
 	s = re.compile('^defaults.ctl.card')
-	## !defaults.pcm.card and defaults.ctl.card should lead
+	## defaults.pcm.card and defaults.ctl.card should lead
 	## the user's custom asoundrc.
-	if set('!defaults.pcm.card', card) and \
+	if set('defaults.pcm.card', card) and \
 	   set('defaults.ctl.card', card):
 	    for i in clist:
 		(j, k) = sep.split(i)
diff --recursive -u alsa-utils-1.0.18/debian/changelog alsa-utils-1.0.18.505089/debian/changelog
--- alsa-utils-1.0.18/debian/changelog	2009-02-12 08:29:51.000000000 +0100
+++ alsa-utils-1.0.18.505089/debian/changelog	2009-02-12 08:34:00.873031419 +0100
@@ -1,3 +1,9 @@
+alsa-utils (1.0.18-1.0) experimental; urgency=low
+
+  * asoundconf set-default-card: no "!" (closes: #505089)
+
+ --
+
 alsa-utils (1.0.18-1) experimental; urgency=low
 
   * New upstream release.
