diff -ur aptitude-0.4.11.6/src/cmdline/cmdline_resolver.cc aptitude-0.4.11.6-new/src/cmdline/cmdline_resolver.cc
--- aptitude-0.4.11.6/src/cmdline/cmdline_resolver.cc	2008-06-11 02:26:49.000000000 +0000
+++ aptitude-0.4.11.6-new/src/cmdline/cmdline_resolver.cc	2008-06-27 17:09:36.000000000 +0000
@@ -236,7 +236,7 @@
 
   for(vector<pkgCache::VerIterator>::size_type i = 0;
       i < choices.size(); ++i)
-    cout << ssprintf(" (%d) %s", i+1, archives_text(choices[i]).c_str()) << endl;
+    cout << ssprintf(" (%d) %s", (int)(i+1), archives_text(choices[i]).c_str()) << endl;
 
   while(1)
     {
@@ -247,7 +247,7 @@
       in >> ws >> i >> ws;
 
       if(!in || !in.eof() || i < 1 || i > (signed)choices.size())
-	cerr << ssprintf(_("Invalid response.  Please enter an integer between 1 and %d."), choices.size()) << endl;
+	cerr << ssprintf(_("Invalid response.  Please enter an integer between 1 and %d."), (int)choices.size()) << endl;
       else
 	return choices[i];
     }
@@ -561,8 +561,8 @@
 	  resolver_manager::state state = resman->state_snapshot();
 
 	  spin.set_msg(ssprintf(_("open: %d; closed: %d; defer: %d; conflict: %d"),
-				state.open_size, state.closed_size,
-				state.deferred_size, state.conflicts_size));
+				(int)(state.open_size), (int)(state.closed_size),
+				(int)(state.deferred_size), (int)(state.conflicts_size)));
 	  spin.display();
 	  spin.tick();
 	}
diff -ur aptitude-0.4.11.6/src/cmdline/cmdline_util.cc aptitude-0.4.11.6-new/src/cmdline/cmdline_util.cc
--- aptitude-0.4.11.6/src/cmdline/cmdline_util.cc	2008-06-21 17:29:21.000000000 +0000
+++ aptitude-0.4.11.6-new/src/cmdline/cmdline_util.cc	2008-06-27 17:13:45.000000000 +0000
@@ -342,7 +342,7 @@
 	output_fragments.push_back(cw::text_fragment(ssprintf(ngettext("There is %d newly obsolete package.",
 								       "There are %d newly obsolete packages.",
 								       new_obsolete.size()),
-							      new_obsolete.size())));
+							      (int)new_obsolete.size())));
       }
     else if(new_obsolete.size() > 0)
       {
