diff --git a/dcut b/dcut
index 1063b10..4311477 100755
--- a/dcut
+++ b/dcut
@@ -275,8 +275,10 @@ def upload_stolen_from_dput_main(host, upload_methods, config,debug,simulate,fil
                 if debug and config.getboolean(host, 'scp_compress'):
                     print "D: Setting compression for scp"
                 scp_compress = config.getboolean(host, 'scp_compress')
+                ssh_config_options = filter(None, map(lambda x: x.strip(),
+                   config.get (host ,'ssh_config_options').split('\n')))
                 upload_methods[method](fqdn, login, incoming, \
-                    files_to_upload, debug, scp_compress)
+                    files_to_upload, debug, scp_compress, ssh_config_options)
             else:
                 upload_methods[method](fqdn, login, incoming, \
                     files_to_upload, debug, 0)
