diff --git a/debian/console-log.init.d b/debian/console-log.init.d
index 6a00d07..5105468 100755
--- a/debian/console-log.init.d
+++ b/debian/console-log.init.d
@@ -245,7 +245,7 @@ do_from_running()
     log_action_begin_msg "checking console-log"
   fi
   CHECKRET=0
-  for PIDPATH in $(find . -maxdepth 2 -type f); do
+  for PIDPATH in $(find . -maxdepth 2 -type f -not -name '*.clientpid'); do
     FILENAME=$(echo $PIDPATH | sed -n 's/.*\/\(.*\)/\1/p')
     PIDFILEDIR=$(echo $PIDPATH | sed -n 's/^.*\/\(.*\)\/.*/\1/p')
     if [ -z "$PIDFILEDIR" ]; then
