Index: bootcd-3.08/bootcdmk2diskconf.src
===================================================================
--- bootcd-3.08.orig/bootcdmk2diskconf.src	2007-11-26 21:00:36.000000000 -0500
+++ bootcd-3.08/bootcdmk2diskconf.src	2007-11-29 11:42:41.000000000 -0500
@@ -138,7 +138,7 @@
 
   CIFS="$(echo "$RIFSTAB"| eval "$MOUNTGREP" | awk '{printf("%s:%s\n",$1,$2)}' | labelsed )"
 
-  CIPART="$((echo "$CIFS" | awk -F: '{print $1}'; echo "$CISWAP"; 
+  CIPART="$( (echo "$CIFS" | awk -F: '{print $1}'; echo "$CISWAP"; 
     echo "$RIPV" | awk -F: '{print $1}') | rmvg | sort -u)"
 
   CIDISK="$(
