Index: bootcd-3.08/S12bootcdram.sh
===================================================================
--- bootcd-3.08.orig/S12bootcdram.sh	2007-11-28 01:37:24.000000000 -0500
+++ bootcd-3.08/S12bootcdram.sh	2007-11-28 01:37:24.000000000 -0500
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # bootcdram.sh	Necessary steps to boot diskless from cd
 # at Boottime /etc -> /ram1/etc -> /etc.ro (also dev, tmp, var, home, root)
 
Index: bootcd-3.08/S13bootcdflop.sh
===================================================================
--- bootcd-3.08.orig/S13bootcdflop.sh	2007-11-28 01:37:25.000000000 -0500
+++ bootcd-3.08/S13bootcdflop.sh	2007-11-28 01:37:25.000000000 -0500
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # bootcdflop.sh
 #
Index: bootcd-3.08/bootcd-run.lib.tst1
===================================================================
--- bootcd-3.08.orig/bootcd-run.lib.tst1	2007-11-28 01:37:24.000000000 -0500
+++ bootcd-3.08/bootcd-run.lib.tst1	2007-11-28 01:37:24.000000000 -0500
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 RUNLIB=../bootcd/bootcd-run.lib
 ERRLOG=/tmp/interactive.tst.log
Index: bootcd-3.08/bootcd-run.lib.tst2
===================================================================
--- bootcd-3.08.orig/bootcd-run.lib.tst2	2007-11-28 01:37:24.000000000 -0500
+++ bootcd-3.08/bootcd-run.lib.tst2	2007-11-28 01:37:24.000000000 -0500
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # bootcd-run.ib.tst2 - show all kind of examples of using interactive and run
 
 RUNLIB=../bootcd/bootcd-run.lib
Index: bootcd-3.08/bootcd2disk
===================================================================
--- bootcd-3.08.orig/bootcd2disk	2007-11-28 01:37:23.000000000 -0500
+++ bootcd-3.08/bootcd2disk	2007-11-28 01:37:23.000000000 -0500
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # bootcd2disk
 #
Index: bootcd-3.08/bootcdbackup
===================================================================
--- bootcd-3.08.orig/bootcdbackup	2007-11-28 01:37:23.000000000 -0500
+++ bootcd-3.08/bootcdbackup	2007-11-28 01:37:23.000000000 -0500
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # bootcdbackup
 #
Index: bootcd-3.08/bootcdbackupwizard.src
===================================================================
--- bootcd-3.08.orig/bootcdbackupwizard.src	2007-11-28 01:37:23.000000000 -0500
+++ bootcd-3.08/bootcdbackupwizard.src	2007-11-28 01:37:23.000000000 -0500
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # 	$Source: /cvsroot/bootcd/bootcd/bootcdbackupwizard.src,v $
 #       $Id: bootcdbackupwizard.src,v 1.53 2007-11-16 16:36:34 bs Exp $
Index: bootcd-3.08/bootcdbackwiz_extract.src
===================================================================
--- bootcd-3.08.orig/bootcdbackwiz_extract.src	2007-11-28 01:37:23.000000000 -0500
+++ bootcd-3.08/bootcdbackwiz_extract.src	2007-11-28 01:37:23.000000000 -0500
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # 	$Source: /cvsroot/bootcd/bootcd/bootcdbackwiz_extract.src,v $
 #       $Id: bootcdbackwiz_extract.src,v 1.4 2007-11-14 12:59:27 bs Exp $
Index: bootcd-3.08/bootcdcreate
===================================================================
--- bootcd-3.08.orig/bootcdcreate	2007-11-28 01:37:23.000000000 -0500
+++ bootcd-3.08/bootcdcreate	2007-11-28 01:37:23.000000000 -0500
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # 	$Source: /cvsroot/bootcd/bootcd/Attic/bootcdcreate,v $
 #       $Id: bootcdcreate,v 1.1 2007-09-13 13:34:58 debdin-guest Exp $
Index: bootcd-3.08/bootcddebootstrap.src
===================================================================
--- bootcd-3.08.orig/bootcddebootstrap.src	2007-11-28 01:37:23.000000000 -0500
+++ bootcd-3.08/bootcddebootstrap.src	2007-11-28 01:37:23.000000000 -0500
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # 	$Source: /cvsroot/bootcd/bootcd/bootcddebootstrap.src,v $
 #       $Id: bootcddebootstrap.src,v 1.18 2007-11-16 14:56:28 bs Exp $
Index: bootcd-3.08/bootcdflopcp
===================================================================
--- bootcd-3.08.orig/bootcdflopcp	2007-11-28 01:37:23.000000000 -0500
+++ bootcd-3.08/bootcdflopcp	2007-11-28 01:37:23.000000000 -0500
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # bootcdflopcp
 # History 
 #   28.07.2000 - Recognize empty files too.
Index: bootcd-3.08/bootcdinitramfshook
===================================================================
--- bootcd-3.08.orig/bootcdinitramfshook	2007-11-28 01:37:24.000000000 -0500
+++ bootcd-3.08/bootcdinitramfshook	2007-11-28 01:37:24.000000000 -0500
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 # See manpage initramfs-tools(8)
 . /usr/share/initramfs-tools/hook-functions
Index: bootcd-3.08/bootcdmk2diskconf.src
===================================================================
--- bootcd-3.08.orig/bootcdmk2diskconf.src	2007-11-28 01:37:24.000000000 -0500
+++ bootcd-3.08/bootcdmk2diskconf.src	2007-11-28 01:37:24.000000000 -0500
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 set -u # exit on using unset var
 
 #        name = bootcdmk2diskconf - create bootcd2disk.conf for backup
Index: bootcd-3.08/bootcdmk2diskconf.tst1
===================================================================
--- bootcd-3.08.orig/bootcdmk2diskconf.tst1	2007-11-28 01:37:24.000000000 -0500
+++ bootcd-3.08/bootcdmk2diskconf.tst1	2007-11-28 01:37:24.000000000 -0500
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 
 VERBOSE="" # VERBOSE="" means silent; VERBOSE="1" means verbose
 
Index: bootcd-3.08/bootcdmkinitrd
===================================================================
--- bootcd-3.08.orig/bootcdmkinitrd	2007-11-28 01:37:24.000000000 -0500
+++ bootcd-3.08/bootcdmkinitrd	2007-11-28 01:37:24.000000000 -0500
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # 	$Source: /cvsroot/bootcd/bootcd/bootcdmkinitrd,v $
 #       $Id: bootcdmkinitrd,v 1.26 2006-11-27 15:53:57 bs Exp $
@@ -185,7 +185,7 @@
 # 50bootcddiscover
 F1=/etc/mkinitrd/scripts/50bootcddiscover # see prerm
 cat <<end1 >$F1
-#!/bin/sh
+#!/bin/bash
 F2=\$INITRDDIR/scripts/bootcd
 cat <<end2 >\$F2
 /bin/echo "running /scripts/bootcd" >&2
@@ -200,7 +200,7 @@
 # 60bootcdproberoot
 F1=/etc/mkinitrd/scripts/60bootcdproberoot # see prerm
 cat <<END1 >$F1
-#!/bin/sh
+#!/bin/bash
 
 F2=\$INITRDDIR/scripts/getrootdev
 cat <<END2 >\$F2
Index: bootcd-3.08/bootcdmodprobe
===================================================================
--- bootcd-3.08.orig/bootcdmodprobe	2007-11-28 01:37:24.000000000 -0500
+++ bootcd-3.08/bootcdmodprobe	2007-11-28 01:37:24.000000000 -0500
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # We need CDROM and FLOPPY
 
 mydiscover1()
Index: bootcd-3.08/bootcdproberoot
===================================================================
--- bootcd-3.08.orig/bootcdproberoot	2007-11-28 01:37:24.000000000 -0500
+++ bootcd-3.08/bootcdproberoot	2007-11-28 01:37:24.000000000 -0500
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # 	$Source: /cvsroot/bootcd/bootcd/bootcdproberoot,v $
 #       $Id: bootcdproberoot,v 1.7 2006-11-27 16:36:34 bs Exp $
Index: bootcd-3.08/bootcdwrite
===================================================================
--- bootcd-3.08.orig/bootcdwrite	2007-11-28 01:37:24.000000000 -0500
+++ bootcd-3.08/bootcdwrite	2007-11-28 01:37:24.000000000 -0500
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 #
 # 	$Source: /cvsroot/bootcd/bootcd/bootcdwrite,v $
 #       $Id: bootcdwrite,v 1.69 2007-10-11 16:26:51 debdin-guest Exp $
Index: bootcd-3.08/runbootcdmodprobe
===================================================================
--- bootcd-3.08.orig/runbootcdmodprobe	2007-11-28 01:37:24.000000000 -0500
+++ bootcd-3.08/runbootcdmodprobe	2007-11-28 01:37:24.000000000 -0500
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # runbootcdmodprobe
 
 # See manpage initramfs-tools(8)
Index: bootcd-3.08/runbootcdproberoot
===================================================================
--- bootcd-3.08.orig/runbootcdproberoot	2007-11-28 01:37:24.000000000 -0500
+++ bootcd-3.08/runbootcdproberoot	2007-11-28 01:37:24.000000000 -0500
@@ -1,4 +1,4 @@
-#!/bin/sh
+#!/bin/bash
 # /etc/initramfs-tools/scripts/init-premount/runbootcdproberoot
 # local-top geht nicht weil root=auto nicht nach local kommt
 
