--- ejabberd-14.07.orig/debian/rules	2014-11-11 01:47:21.000000000 -0800
+++ ejabberd-14.07/debian/rules	2015-04-17 14:21:11.992254753 -0700
@@ -14,7 +14,8 @@
 .PHONY: override_dh_auto_configure
 override_dh_auto_configure:
 	        dh_auto_configure -- --enable-odbc --enable-mysql --enable-pgsql --enable-pam \
-			--enable-zlib --enable-json --enable-iconv --enable-lager --enable-stun
+			--enable-zlib --enable-json --enable-iconv --enable-lager --enable-stun \
+			--enable-transient_supervisors
 
 .PHONY: override_dh_auto_build
 override_dh_auto_build:
