--- /tmp/debmirror.mine	2007-11-27 11:17:22.000000000 +0100
+++ bin/debmirror.mine	2007-11-27 11:21:30.000000000 +0100
@@ -1105,7 +1105,7 @@
   # makes directories.. Sort so they are removable in bottom-up
   # order.
   chdir($mirrordir) or die "chdir $mirrordir: $!";
-  system("find . -depth -type d ! -name . ! -name .. -print0 | xargs -0 rmdir 2>/dev/null") if (! $dry_run);
+  system("find . -depth -type d -empty -not -name . -print0 | xargs -r0 rmdir") if (! $dry_run);
 }
 
 if ($res == 0) {
