--- ../../docs/manual/caching.html.en.orig	2011-11-30 11:11:31.000000000 -0500
+++ ../../docs/manual/caching.html.en	2011-11-30 11:28:08.000000000 -0500
@@ -640,7 +640,7 @@
       run takes, to avoid running more than one instance at a time.</p>
 
       <p class="figure">
-      <img src="images/caching_fig1.gif" alt="" width="600" height="406" /><br />
+      <img src="../images/caching_fig1.gif" alt="" width="600" height="406" /><br />
       <a id="figure1" name="figure1"><dfn>Figure 1</dfn></a>: Typical
       cache growth / clean sequence.</p>
 
--- ../../docs/manual/caching.html.fr.orig	2011-11-30 11:11:42.000000000 -0500
+++ ../../docs/manual/caching.html.fr	2011-11-30 11:28:20.000000000 -0500
@@ -723,7 +723,7 @@
       d'exécuter plusieurs instances à la fois.</p>
 
       <p class="figure">
-      <img src="images/caching_fig1.gif" alt="" width="600" height="406" /><br />
+      <img src="../images/caching_fig1.gif" alt="" width="600" height="406" /><br />
       <a id="figure1" name="figure1"><dfn>Figure 1</dfn></a>: Croissance
       typique du cache / séquence de nettoyage.</p>
 
