Index: debian/aide.conf.d/31_aide_acpid
===================================================================
--- debian/aide.conf.d/31_aide_acpid	(revision 785)
+++ debian/aide.conf.d/31_aide_acpid	(working copy)
@@ -3,4 +3,4 @@
 /var/log/acpid\.2\.gz$ LoSerMemberLog
 /var/log/acpid\.3\.gz$ SerMemberLog
 /var/log/acpid\.4\.gz$ HiSerMemberLog
-/var/run/acpid\.socket$ VarFile
+/var/run/acpid\.(socket|pid)$ VarFile
