--- fai-debconf.orig	2014-03-17 09:41:37.038020532 +0100
+++ fai-debconf	2014-03-17 09:44:14.581415100 +0100
@@ -60,7 +60,7 @@
 
      [ -s $LOGDIR/debconf.data ] || return # nothing to do if is size 0 or missing
      if [ -r $LOGDIR/debconf.old ] ; then
-       $ROOTCMD debconf-get-selections > $LOGDIR/debconf.new
+       $ROOTCMD debconf-get-selections | sort > $LOGDIR/debconf.new
        diff --changed-group-format="%>" --unchanged-line-format="" \
          $LOGDIR/debconf.old $LOGDIR/debconf.new > $LOGDIR/debconf.data
      fi
@@ -115,7 +115,7 @@
     exit 9
 fi
 
-[ $forcereconf -eq 0 -a -x $target/usr/bin/debconf-get-selections ] && $ROOTCMD debconf-get-selections > $LOGDIR/debconf.old
+[ $forcereconf -eq 0 -a -x $target/usr/bin/debconf-get-selections ] && $ROOTCMD debconf-get-selections | sort > $LOGDIR/debconf.old
 call_conf # add data to debconf database
 [ $reconf -eq 1 ] && reconfigure_packages
 exit 0
