--- developers-reference.sgml	2003-11-20 18:05:36.000000000 +0100
+++ developers-reference.patched.sgml	2003-11-20 18:07:23.000000000 +0100
@@ -332,7 +332,7 @@
 nothing as critical as that, but it's still appropriate to let the others
 know that you're unavailable.
 	<p>
-In order to inform the other developers, there's two things that you should do.
+In order to inform the other developers, there are two things that you should do.
 First send a mail to &email-debian-private; with "[VAC] " prepended to the
 subject of your message<footnote>This is so that the message can be easily
 filtered by people who don't want to read vacation notices.</footnote>
