--- a/debian/distcc.postinst	2012-11-09 12:49:31.005210874 +0800
+++ b/debian/distcc.postinst	2012-11-09 12:49:32.817538938 +0800
@@ -98,11 +98,12 @@
 		DISTCC_USER=distccd
 		DISTCC_LOGFILE=/var/log/distccd.log
 
-		# create distcc user if it does not already exist
-		if ! getent passwd $DISTCC_USER >/dev/null; then
-		  echo "Creating $DISTCC_USER user ..."
-		  adduser --quiet --system --ingroup nogroup \
-		    --home / --no-create-home $DISTCC_USER
+		# create distcc user
+		adduser --quiet --system --ingroup nogroup \
+		    --home /var/lib/distcc --no-create-home $DISTCC_USER
+		# 3.1-5 and earlier versions used "/" as home.
+		if [ x$(getent passwd grepza | cut -d: -f6) = "x/" ]; then
+			usermod --home /var/lib/distcc $DISTCC_USER || true
 		fi
 
 		if [ ! -s $DISTCC_LOGFILE ]; then
