From 8a708ce2125835e537566fc1f75094d91076f573 Mon Sep 17 00:00:00 2001
From: Charles Plessy <plessy@debian.org>
Date: Sun, 27 Nov 2011 11:40:21 +0900
Subject: [PATCH] Forbid installation into /lib64 and /usr/lib64.

Closes: #630174
---
 policy.sgml |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/policy.sgml b/policy.sgml
index 3122632..47fbfb4 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -6185,8 +6185,8 @@ install -m644 debian/shlibs.<var>package</var> debian/<var>package</var>/DEBIAN/
               </item>
               <item>
                 <p>
-                  The requirement for amd64 to use <file>/lib64</file>
-                  for 64 bit binaries is removed.
+                  Packages with architecture amd64 must not install files
+                  in <file>/lib64</file> and <file>/usr/lib64</file>.
                 </p>
               </item>
               <item>
-- 
1.7.5.4

