diff -u conky-1.7.1.1/debian/rules conky-1.7.1.1/debian/rules
--- conky-1.7.1.1/debian/rules
+++ conky-1.7.1.1/debian/rules
@@ -19,7 +19,7 @@
 endif
 
 ifeq ($(DEB_HOST_ARCH_OS),linux)
-       EXTRA_CONF_ARGS=--enable-wlan --enable-ibm --disable-lua
+       EXTRA_CONF_ARGS=--enable-wlan --enable-ibm
 endif
 
 DEB_QUILT_PATCHES = debian/patches
@@ -52,7 +52,7 @@
 	  --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE) \
 	  --prefix=/usr --sysconfdir=/etc \
 	  --mandir=\$${prefix}/share/man --infodir=\$${prefix}/share/info \
-	  --enable-debug --enable-rss $(EXTRA_CONF_ARGS)
+	  --enable-debug --enable-rss --disable-lua $(EXTRA_CONF_ARGS)
 	
 	touch debian/configure-stamp
 
diff -u conky-1.7.1.1/debian/changelog conky-1.7.1.1/debian/changelog
--- conky-1.7.1.1/debian/changelog
+++ conky-1.7.1.1/debian/changelog
@@ -1,3 +1,13 @@
+conky (1.7.1.1-1.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Fix FTBFS on non-Linux architectures:
+     - Move “--disable-lua” from the Linux-specific configure flags to the
+       common configure flags.
+     - Make libasound2-dev Linux-only in Build-Depends.
+
+ -- Cyril Brulebois <kibi@debian.org>  Thu, 09 Jul 2009 11:26:37 +0200
+
 conky (1.7.1.1-1) unstable; urgency=low
 
   * New upstream release (1.7.1.1).
diff -u conky-1.7.1.1/debian/control conky-1.7.1.1/debian/control
--- conky-1.7.1.1/debian/control
+++ conky-1.7.1.1/debian/control
@@ -4,7 +4,8 @@
 Maintainer: Kapil Hari Paranjape <kapil@debian.org>
 Build-Depends: debhelper (>= 5.0.0), quilt, libx11-dev, libxft-dev,
  libxdamage-dev, libxext-dev, pkg-config (>= 0.19), libglib2.0-dev,
- libxml2-dev, libcurl4-gnutls-dev, libasound2-dev,
+ libxml2-dev, libcurl4-gnutls-dev,
+ libasound2-dev [!kfreebsd-i386 !kfreebsd-amd64 !hurd-i386],
  libtool, automake1.10,
  libiw-dev [!kfreebsd-i386 !kfreebsd-amd64 !hurd-i386],
  libkvm-dev [kfreebsd-i386 kfreebsd-amd64],
