diff -Nru developers-reference-3.4.11/debian/changelog developers-reference-3.4.11+nmu1/debian/changelog
--- developers-reference-3.4.11/debian/changelog	2013-08-07 14:21:54.000000000 +0200
+++ developers-reference-3.4.11+nmu1/debian/changelog	2014-01-23 22:07:40.000000000 +0100
@@ -1,3 +1,10 @@
+developers-reference (3.4.11+nmu1) UNRELEASED; urgency=medium
+
+  * Non-maintainer upload.
+  * ressources: Mention that some of the goodies also apply to DMs.
+
+ -- Philipp Kaluza <debian@ghostroute.eu>  Thu, 23 Jan 2014 22:06:54 +0100
+
 developers-reference (3.4.11) unstable; urgency=low
 
   * Team upload.
diff -Nru developers-reference-3.4.11/resources.dbk developers-reference-3.4.11+nmu1/resources.dbk
--- developers-reference-3.4.11/resources.dbk	2013-08-07 13:57:05.000000000 +0200
+++ developers-reference-3.4.11+nmu1/resources.dbk	2014-01-23 22:10:15.000000000 +0100
@@ -4,7 +4,7 @@
   <!ENTITY % commondata SYSTEM "common.ent" > %commondata;
 ]>
 <chapter id="resources">
-<title>Resources for Debian Developers</title>
+<title>Resources for Debian Developers and Debian Maintainers</title>
 <para>
 In this chapter you will find a very brief road map of the Debian mailing
 lists, the Debian machines which may be available to you as a developer, and
@@ -1570,12 +1570,13 @@
 </section>
 
 <section id="developer-misc">
-<title>Goodies for Developers</title>
+<title>Goodies for Debian Developers and Debian Maintainers</title>
 <section id="lwn">
 <title>LWN Subscriptions</title>
 <para>
 Since October of 2002, HP has sponsored a subscription to LWN for all
-interested Debian developers.  Details on how to get access to this benefit are
+interested Debian Developers and Debian Maintainers.
+Details on how to get access to this benefit are
 in <ulink
 url="http://&lists-host;/debian-devel-announce/2002/10/msg00018.html"></ulink>.
 </para>
