--- a/cloud_sptheme/themes/cloud/static/cloud.js_t	2018-10-20 14:46:28.000000000 +0200
+++ b/cloud.js_t	2018-10-20 14:43:22.635643410 +0200
@@ -240,7 +240,7 @@
 
         var $doc = $('div.document'),
             $hide = $('button#sidebar-hide'),
-            $show = $('button#sidebar-show'),
+            $show = $('button#sidebar-show');
 
         // set sidebar state for current media size
         var lastSmall = false,
@@ -275,6 +275,7 @@
         if(smallScreen){
             setState(false);
         } else {
+            var value = "";
             {% if theme_defaultcollapsed | tobool %}
                 if (!value) value = "collapsed";
             {% endif %}

--- a/cloud_sptheme/themes/cloud/layout.html	2018-10-20 15:27:18.000000000 +0200
+++ b/layout.html	2018-10-20 16:02:39.185070146 +0200
@@ -22,9 +22,6 @@
 {% set script_files = script_files + ['_static/cloud.base.js',
                                       '_static/cloud.js'] %}
 
-{# add font stylesheets #}
-{% set css_files = css_files + [theme_fontcssurl] %}
-
 {# make root link redirectable #}
 {%- set theme_roottarget = (theme_roottarget == "<toc>" and master_doc or theme_roottarget) %}
 {% block rootrellink %}
