diff --git a/lib-base/policy-funcs b/lib-base/policy-funcs
index afd6375..3ebe448 100644
--- a/lib-base/policy-funcs
+++ b/lib-base/policy-funcs
@@ -8,7 +8,7 @@
 # daemon (e.g. through a fake key press).
 CheckPolicy() {
 	getXconsole
-	if pidof gnome-power-manager kpowersave > /dev/null ||
+	if pidof gnome-power-manager kpowersave xfce4-power-manager > /dev/null ||
 	   ( test "$XUSER" != "" && pidof dcopserver > /dev/null && test -x /usr/bin/dcop && /usr/bin/dcop --user $XUSER kded kded loadedModules | grep -q klaptopdaemon) ; then
 		echo 0;
 	else
