diff --git a/build/util/gen-sources.list.udeb b/build/util/gen-sources.list.udeb
index 26a0ac2..7f62d9d 100755
--- a/build/util/gen-sources.list.udeb
+++ b/build/util/gen-sources.list.udeb
@@ -86,6 +86,9 @@ for mirror in $MIRRORS; do
 		fi
 	else
 		echo "$mirror $SUITE $UDEB_COMPONENTS"
+		if [ "$USE_PROPOSED_UPDATES" = 1 ]; then
+			echo "$mirror $SUITE-proposed-updates $UDEB_COMPONENTS"
+		fi
 	fi
 
 done | perl -ne 'print unless $seen{$_}; $seen{$_}=1'
