diff -uNr autodia-2.03.orig/debian/rules autodia-2.03/debian/rules
--- autodia-2.03.orig/debian/rules	2008-12-08 11:14:10.000000000 +0100
+++ autodia-2.03/debian/rules	2008-12-07 20:11:55.000000000 +0100
@@ -27,15 +27,23 @@
 	dh_testdir
 
 	
+	mkdir -p tmp
+	cp -p autodia.pl tmp
+	cp -p autodia_java.pl tmp
+	sed -i -e 's/autodia.pl/autodia/' autodia.pl
+	sed -i -e 's/autodia.pl/autodia/' autodia_java.pl
 	# Add here commands to compile the package.
 	$(PERL) Makefile.PL INSTALLDIRS=vendor
 	$(MAKE) OPTIMIZE="-O2 -g -Wall"
-
+	cp -p tmp/autodia.pl autodia.pl
+	cp -p tmp/autodia_java.pl autodia_java.pl
 	touch build-stamp
 
+
 clean:
 	dh_testdir
 	dh_testroot
+	rm -rf tmp
 	rm -f build-stamp
 
 	# Add here commands to clean up after the build process.
