--- acpi-support-0.90-bckp/acpi_fakekey.c	2007-02-10 23:44:57.000000000 +0100
+++ acpi-support-0.90-work/acpi_fakekey.c	2007-02-10 23:52:03.000000000 +0100
@@ -16,7 +16,10 @@
         for (i=0; i<32; i++) {
                 snprintf(filename,sizeof(filename), "/dev/input/event%d", i);
 
-                fd = open(filename, O_RDWR);
+		if((fd = open(filename, O_RDWR))<0){
+			perror("open");
+			exit(EXIT_FAILURE);
+		}
                 ioctl(fd, EVIOCGBIT(EV_KEY, sizeof(key_bitmask)), key_bitmask);
 
 		for (j = 0; j < BTN_MISC; j++) {
@@ -38,17 +41,13 @@
 	struct input_event event;
 
 	if (argc == 2) {
-		key = atoi(argv[1]);
+		key = strtol(argv[1], NULL, 10);
 	} else {
-		return 1;
+		return -1;
 	}
 
 	fd = find_keyboard();
 
-	if (!fd) {
-		return 2;
-	}
-
 	event.type = EV_KEY;
 	event.code = key;
 	event.value = 1;
