--- acpi-support-0.90-bckp/asus-touchpad.sh	2007-02-10 23:44:57.000000000 +0100
+++ acpi-support-0.90-work/asus-touchpad.sh	2007-02-10 23:45:37.000000000 +0100
@@ -1,7 +1,7 @@
 #!/bin/sh
 
 # get the current state of the touchpad
-TPSTATUS=`synclient -l | grep TouchpadOff | awk '{print $3}'`
+TPSTATUS=`synclient -l | awk '{/TouchpadOff/ print $3}'`
 
 # if getting the status failed, exit
 test -z $TPSTATUS && exit 1
