diff -ur dpkg-1.10.22.orig/lib/fields.c dpkg-1.10.22/lib/fields.c
--- dpkg-1.10.22.orig/lib/fields.c	2004-06-02 06:18:16.000000000 +0900
+++ dpkg-1.10.22/lib/fields.c	2004-06-26 16:31:28.000000000 +0900
@@ -166,9 +166,9 @@
              _("value for `status' field not allowed in this context"));
   if (flags & pdb_recordavailable) return;
   
-  pigp->want= convert_string(filename,lno,"first (want) word in `status' field", -1,
+  pigp->want= convert_string(filename,lno,_("first (want) word in `status' field"), -1,
                              warnto,warncount,pigp, value,wantinfos,&ep);
-  pigp->eflag= convert_string(filename,lno,"second (error) word in `status' field", -1,
+  pigp->eflag= convert_string(filename,lno,_("second (error) word in `status' field"), -1,
                               warnto,warncount,pigp, ep,eflaginfos,&ep);
   if (pigp->eflag & eflagf_obsoletehold) {
     pigp->want= want_hold;
diff -ur dpkg-1.10.22.orig/lib/mlib.c dpkg-1.10.22/lib/mlib.c
--- dpkg-1.10.22.orig/lib/mlib.c	2004-06-02 06:18:16.000000000 +0900
+++ dpkg-1.10.22/lib/mlib.c	2004-06-27 02:33:56.000000000 +0900
@@ -119,10 +119,10 @@
     n= WTERMSIG(status); if (!n || ((flags & PROCPIPE) && n==SIGPIPE)) return 0;
     if (flags & PROCWARN)
       ohshit(_("dpkg: warning - %s killed by signal (%s)%s\n"),
-           description, strsignal(n), WCOREDUMP(status) ? ", core dumped" : "");
+           description, strsignal(n), WCOREDUMP(status) ? _(", core dumped") : "");
     else
       ohshit(_("subprocess %s killed by signal (%s)%s"),
-           description, strsignal(n), WCOREDUMP(status) ? ", core dumped" : "");
+           description, strsignal(n), WCOREDUMP(status) ? _(", core dumped") : "");
   } else {
     ohshit(_("subprocess %s failed with wait status code %d"),description,status);
   }
