Index: debian/apt-cacher-ng.postinst
===================================================================
--- debian/apt-cacher-ng.postinst	(Revision 4187)
+++ debian/apt-cacher-ng.postinst	(Arbeitskopie)
@@ -136,24 +136,24 @@
    db_get $NAME/cachedir
    case "$RET" in
       keep|"")
-         # not to be set by debconf mechanism, use whatever we have
-         CDIR=$(PRINTCFGVAR=CacheDir $BINPATH -c $CFG 2>/dev/null)
-         CDIRCONF="# not set by debconf"
+         # but making sure that it's really set to something sensible in the
+         # config, otherwise replace with the default
+         TESTPATH=$(PRINTCFGVAR=CacheDir $BINPATH -c $CFG 2>/dev/null ||:)
+         if test -d "$TESTPATH" ; then
+            CDIR="$TESTPATH"
+            CDIRCONF="# not set by debconf"
+         else
+            CDIR=/var/cache/$NAME
+            CDIRCONF="CacheDir: $CDIR # default or overridden since '$TESTPATH' directory didn't exist"
+         fi
          ;;
       *)
-         # ok, configured by debconf
+         # ok, configured by debconf, will be created as needed
          CDIR="$RET"
          CDIRCONF="CacheDir: $RET"
          ;;
    esac
 
-   # make sure it's set even if commented out by the user
-   TESTPATH=$(PRINTCFGVAR=CacheDir $BINPATH -c $CFG 2>/dev/null)
-   if test -e "$TESTPATH" ; then
-      CDIR=/var/cache/$NAME
-      CDIRCONF="CacheDir: $CDIR"
-   fi
-
    # user should exist. adduser sometimes fails (system range issue) but that's ok
    adduser --quiet --system --group --no-create-home --home "$CDIR" $NAME || id $NAME
    # also setup permissions ASAP
