--- control~	2009-01-21 11:08:35.000000000 +0000
+++ control	2009-01-21 11:08:51.000000000 +0000
@@ -8,7 +8,7 @@
 
 Package: eggdrop
 Architecture: any
-Depends: ${shlibs:Depends}, eggdrop-data (= ${source:Version})
+Depends: ${shlibs:Depends}, eggdrop-data (= ${source:Version}), ${misc:Depends}
 Description: Advanced IRC Robot
  Eggdrop is an IRC bot written in C, which sits on channels and takes
  protective measures, such as preventing it from being taken over (in
@@ -24,6 +24,7 @@
 Package: eggdrop-data
 Architecture: all
 Replaces: eggdrop (>= 1.6.16-2)
+Depends: ${misc:Depends}
 Description: Architecture independent files for eggdrop
  This package contains all the files that eggdrop places on /usr/share.
  These files are not specific to a single processor architecture once
