--- etc/exim4/exim4.conf.template	2011-01-31 21:18:20.000000000 +0300
+++ /etc/exim4/exim4.conf.template	2011-04-10 14:28:41.000000000 +0400
@@ -155,7 +155,7 @@
 # scanners. The second modification is in the acl_check_data access
 # control list.
 
-# av_scanner = clamd:/tmp/clamd
+# av_scanner = clamd:/var/run/clamav/clamd
 
 
 # For spam scanning, there is a similar option that defines the interface to
