--- main.c.originale	2006-06-02 13:13:09.000000000 +0200
+++ main.c	2006-06-02 13:13:30.000000000 +0200
@@ -1114,7 +1114,7 @@
 			break;
 		case E_DOC: {
 				char *browser;
-				browser = g_strconcat(prefs.browser," ",INSTALL_DIR,"/share/doc/euler/index.html &",NULL);
+				browser = g_strconcat(prefs.browser," ",INSTALL_DIR,"/share/doc/euler-doc/index.html &",NULL);
 				system(browser);
 				g_free(browser);
 			}
