--- /usr/share/perl5/App/ClusterSSH.pm	2015-02-02 00:08:39.000000000 +0100
+++ /usr/share/perl5/App/ClusterSSH.pm.new	2015-06-01 14:13:59.912495431 +0200
@@ -248,6 +248,23 @@
 
     $self->debug( 1, "Loading keymaps and keycodes" );
 
+    # BEGIN WORKAROUND
+    my %__workaround = (
+        ord("é") => ord("~"),
+        ord("\"") => ord("#"),
+        ord("'") => ord("{"),
+        ord("(") => ord("["),
+        ord("-") => ord("|"),
+        ord("è") => ord("`"),
+        ord("_") => ord("\\"),
+        ord("ç") => ord("^"),
+        ord("à") => ord("@"),
+        ord(")") => ord("]"),
+        ord("=") => ord("}"),
+    );
+    @keyboard = map { defined($__workaround{$_->[0]}) and $_->[2] = $__workaround{$_->[0]}; $_ } @keyboard;
+    # END WORKAROUND
+
     my %keyboard_modifier_priority = (
         'sa' => 3,    # lowest
         'a'  => 2,
