From 77715c263cd2e68fca5867e7591756fd48b93d56 Mon Sep 17 00:00:00 2001
From: Carlos Alberto Lopez Perez <clopez@igalia.com>
Date: Mon, 9 Jan 2012 12:07:00 +0100
Subject: [PATCH] Disable lame-servers, edns-disabled and resolver log
 categories

---
 named.conf.options |    7 +++++++
 1 files changed, 7 insertions(+), 0 deletions(-)

diff --git a/named.conf.options b/named.conf.options
index af79758..b0bdda9 100644
--- a/named.conf.options
+++ b/named.conf.options
@@ -18,3 +18,10 @@ options {
 	listen-on-v6 { any; };
 };
 
+// Disable EDNS, as it will spit out a big number of lines to syslog
+logging {
+	category lame-servers { null; };
+	category edns-disabled { null; };
+	category resolver { null; };
+};
+
-- 
1.7.7.3

