Index: fr/Makefile
===================================================================
--- fr/Makefile	(révision 11322)
+++ fr/Makefile	(copie de travail)
@@ -37,7 +37,7 @@
 html: $(manual).$(currentlang).html/index.$(currentlangcn).html
 $(manual).$(currentlang).html/index.$(currentlangcn).html: $(sources)
 	rm -rf $(manual).html $(manual).$(currentlang).html
-	debiandoc2html -c -l $(currentlang) $<
+	debiandoc2html -C -l $(currentlang).UTF-8 $<
 	mv $(manual).html $(manual).$(currentlang).html
 
 text: $(manual).$(currentlang).txt
@@ -51,7 +51,7 @@
 
 $(manual).$(currentlang).ps $(manual).$(currentlang).dvi $(manual).$(currentlang).pdf: \
   $(manual).$(currentlang).%: $(sources)
-	debiandoc2latex$* -l $(currentlang) $<
+	debiandoc2latex$* -l $(currentlang).UTF-8 $<
 	mv $(manual).$* $(manual).$(currentlang).$*
 
 ps dvi pdf info: %: $(manual).$(currentlang).%
Index: po4a/po4a.cfg
===================================================================
--- po4a/po4a.cfg	(révision 11322)
+++ po4a/po4a.cfg	(copie de travail)
@@ -2,5 +2,4 @@
 
 [type: sgml] debian-faq.sgml $lang:$lang/debian-faq.sgml \
 		add_$lang:?@po4a/add_$lang/$lang.add \
-		opt_fr:"-L ISO-8859-15" \
 		opt:"-o ignore-inclusion=faqdynamic.ent"
