--- /etc/init.d/exim4	2008-06-04 09:58:21.000000000 +0200
+++ /etc/init.d/exim4.new	2008-06-04 09:46:55.000000000 +0200
@@ -93,9 +93,9 @@ start_exim()
     combined)
       start_daemon -p "$PIDFILE" \
         "$DAEMON" -bd "-q${QFLAGS}${QUEUEINTERVAL}" \
-        ${COMMONOPTIONS:+"${COMMONOPTIONS}"} \
-	${QUEUERUNNEROPTIONS:+"${QUEUERUNNEROPTIONS}"} \
-	${SMTPLISTENEROPTIONS:+"${SMTPLISTENEROPTIONS}"}
+        ${COMMONOPTIONS} \
+	${QUEUERUNNEROPTIONS} \
+	${SMTPLISTENEROPTIONS}
       log_progress_msg "exim4"
       ;;
     separate)
@@ -107,23 +107,23 @@ start_exim()
       start_daemon -p "$QRPIDFILE" \
         "$DAEMON" -oP $QRPIDFILE \
         "-q${QFLAGS}${QUEUEINTERVAL}" \
-        ${COMMONOPTIONS:+"${COMMONOPTIONS}"} \
-	${QUEUERUNNEROPTIONS:+"${QUEUERUNNEROPTIONS}"}
+        ${COMMONOPTIONS} \
+	${QUEUERUNNEROPTIONS}
       log_progress_msg "exim4_queuerunner"
       ;;
     queueonly)
       start_daemon -p "$QRPIDFILE" \
         "$DAEMON" -oP $QRPIDFILE \
         "-q${QFLAGS}${QUEUEINTERVAL}" \
-        ${COMMONOPTIONS:+"${COMMONOPTIONS}"} \
-	${QUEUERUNNEROPTIONS:+"${QUEUERUNNEROPTIONS}"}
+        ${COMMONOPTIONS} \
+	${QUEUERUNNEROPTIONS}
       log_progress_msg "exim4_queuerunner"
       ;;
     no|ppp)
       start_daemon -p "$PIDFILE" \
         "$DAEMON" -bd \
-        ${COMMONOPTIONS:+"${COMMONOPTIONS}"} \
-	${SMTPLISTENEROPTIONS:+"${SMTPLISTENEROPTIONS}"}
+        ${COMMONOPTIONS} \
+	${SMTPLISTENEROPTIONS}
       log_progress_msg "exim4_listener"
       ;;
     nodaemon)
