Index: ./cpu/data_xfer8.cc
===================================================================
RCS file: /cvsroot/bochs/bochs/cpu/data_xfer8.cc,v
retrieving revision 1.14
diff -u -r1.14 data_xfer8.cc
--- ./cpu/data_xfer8.cc	25 Oct 2002 18:26:27 -0000	1.14
+++ ./cpu/data_xfer8.cc	13 Apr 2003 20:42:41 -0000
@@ -37,7 +37,7 @@
   void
 BX_CPU_C::MOV_RLIb(bxInstruction_c *i)
 {
-  BX_CPU_THIS_PTR gen_reg[i->b1() & 0x03].word.byte.rl = i->Ib();
+  BX_READ_8BIT_REGx(i->opcodeReg(),i->extend8bitL()) = i->Ib();
 }
 
   void
Index: ./cpu/fetchdecode64.cc
===================================================================
RCS file: /cvsroot/bochs/bochs/cpu/fetchdecode64.cc,v
retrieving revision 1.43
diff -u -r1.43 fetchdecode64.cc
--- ./cpu/fetchdecode64.cc	5 Apr 2003 12:16:53 -0000	1.43
+++ ./cpu/fetchdecode64.cc	13 Apr 2003 20:42:42 -0000
@@ -1597,10 +1597,10 @@
   /* B1 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RLIb },
   /* B2 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RLIb },
   /* B3 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RLIb },
-  /* B4 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RHIb },
-  /* B5 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RHIb },
-  /* B6 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RHIb },
-  /* B7 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RHIb },
+  /* B4 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RLIb },
+  /* B5 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RLIb },
+  /* B6 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RLIb },
+  /* B7 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RLIb },
   /* B8 */  { BxImmediate_Iv, &BX_CPU_C::MOV_RXIw },
   /* B9 */  { BxImmediate_Iv, &BX_CPU_C::MOV_RXIw },
   /* BA */  { BxImmediate_Iv, &BX_CPU_C::MOV_RXIw },
@@ -2115,10 +2115,10 @@
   /* B1 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RLIb },
   /* B2 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RLIb },
   /* B3 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RLIb },
-  /* B4 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RHIb },
-  /* B5 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RHIb },
-  /* B6 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RHIb },
-  /* B7 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RHIb },
+  /* B4 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RLIb },
+  /* B5 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RLIb },
+  /* B6 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RLIb },
+  /* B7 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RLIb },
   /* B8 */  { BxImmediate_Iv, &BX_CPU_C::MOV_ERXId },
   /* B9 */  { BxImmediate_Iv, &BX_CPU_C::MOV_ERXId },
   /* BA */  { BxImmediate_Iv, &BX_CPU_C::MOV_ERXId },
@@ -2630,10 +2630,10 @@
   /* B1 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RLIb },
   /* B2 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RLIb },
   /* B3 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RLIb },
-  /* B4 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RHIb },
-  /* B5 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RHIb },
-  /* B6 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RHIb },
-  /* B7 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RHIb },
+  /* B4 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RLIb },
+  /* B5 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RLIb },
+  /* B6 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RLIb },
+  /* B7 */  { BxImmediate_Ib, &BX_CPU_C::MOV_RLIb },
   /* B8 */  { BxImmediate_Iq, &BX_CPU_C::MOV_RRXIq },
   /* B9 */  { BxImmediate_Iq, &BX_CPU_C::MOV_RRXIq },
   /* BA */  { BxImmediate_Iq, &BX_CPU_C::MOV_RRXIq },
