>From 408228b09c7d1988a831249951976d8c72836b16 Mon Sep 17 00:00:00 2001
From: Adrian Bunk <bunk@stusta.de>
Date: Wed, 4 Dec 2013 13:33:05 +0200
Subject: debugfs(8): document lsdel

---
 debugfs/debugfs.8.in | 6 ++++++
 1 file changed, 6 insertions(+)

diff --git a/debugfs/debugfs.8.in b/debugfs/debugfs.8.in
index 65a30d5..debaa01 100644
--- a/debugfs/debugfs.8.in
+++ b/debugfs/debugfs.8.in
@@ -427,6 +427,12 @@ flag will list the files in a format which is more easily parsable by
 scripts, as well as making it more clear when there are spaces or other
 non-printing characters at the end of filenames.
 .TP
+.I lsdel [sec]
+List deleted inodes. The optional
+.I sec
+parameter limits the result to files deleted during the specified
+number of seconds before now.
+.TP
 .I modify_inode filespec
 Modify the contents of the inode structure in the inode
 .IR filespec .
-- 
1.8.5.rc3

