diff -ur djvulibre-3.5.20.orig/tools/csepdjvu.cpp djvulibre-3.5.20/tools/csepdjvu.cpp
--- djvulibre-3.5.20.orig/tools/csepdjvu.cpp	2007-12-29 15:38:21.000000000 +0100
+++ djvulibre-3.5.20/tools/csepdjvu.cpp	2007-12-29 15:40:20.000000000 +0100
@@ -1071,7 +1071,7 @@
     return 0;
   GP<GPixmap> pix = GPixmap::create(bs);
   // Check background reduction
-  for (bgred=1; bgred<12; bgred++) 
+  for (bgred=1; bgred<=12; bgred++) 
     {
       int subw = (w + bgred - 1) / bgred;
       int subh = (h + bgred - 1) / bgred;
