diff -Nrup evolution.orig/debian/rules evolution/debian/rules
--- evolution.orig/debian/rules	2010-04-28 22:16:55.000000000 +0200
+++ evolution/debian/rules	2010-04-28 22:17:23.000000000 +0200
@@ -31,7 +31,7 @@ DEB_CONFIGURE_EXTRA_FLAGS += \
 	--disable-pst-import \
 	--disable-contacts-map
 
-ifneq (DEB_HOST_ARCH_OS, linux)
+ifneq ($(DEB_HOST_ARCH_OS), linux)
 	DEB_CONFIGURE_EXTRA_FLAGS+=--disable-nm
 else
 	DEB_CONFIGURE_EXTRA_FLAGS+=--enable-nm
