diff -Nru /tmp/xl1G5jpjVX/dictionaries-common-0.80.1/policy/dsdt-policy.xml.in /tmp/4VA5hqonea/dictionaries-common-0.80.0.test.1/policy/dsdt-policy.xml.in
--- /tmp/xl1G5jpjVX/dictionaries-common-0.80.1/policy/dsdt-policy.xml.in	2007-04-10 16:52:47.000000000 +0200
+++ /tmp/4VA5hqonea/dictionaries-common-0.80.0.test.1/policy/dsdt-policy.xml.in	2007-05-07 16:56:13.000000000 +0200
@@ -1,5 +1,5 @@
 <?xml version="1.0"?>
-<!DOCTYPE article PUBLIC "-//Norman Walsh//DTD DocBk XML V3.1//EN"
+<!DOCTYPE article PUBLIC "-//Norman Walsh//DTD DocBk XML V4.0//EN"
      "dtd/docbook-xml/docbookx.dtd" [
 <!ENTITY doc-prefix "/usr/share/doc/dictionaries-common">
 <!ENTITY autoscript-prefix "/usr/share/debhelper/autoscripts">
@@ -51,7 +51,7 @@
 
 
 <article>
-  <artheader>
+  <articleinfo>
     <title>
       Debian Spelling Dictionaries and Tools Policy
     </title>
@@ -134,7 +134,7 @@
 	</ulink>.
       </para>
     </legalnotice>
-  </artheader>
+  </articleinfo>
 
   <warning>
     <para>
