Index: binutils/ar.c
===================================================================
RCS file: /cvs/src/src/binutils/ar.c,v
retrieving revision 1.53
diff -c -3 -p -r1.53 ar.c
*** binutils/ar.c	5 Jul 2007 16:54:45 -0000	1.53
--- binutils/ar.c	27 Feb 2008 10:51:08 -0000
*************** usage (int help)
*** 255,262 ****
        fprintf (s, _(" Generate an index to speed access to archives\n"));
        fprintf (s, _(" The options are:\n\
    @<file>                      Read options from <file>\n\
    -h --help                    Print this help message\n\
!   -V --version                 Print version information\n"));
      }
  
    list_supported_targets (program_name, s);
--- 255,263 ----
        fprintf (s, _(" Generate an index to speed access to archives\n"));
        fprintf (s, _(" The options are:\n\
    @<file>                      Read options from <file>\n\
+   -t                           Update the archive's symbol map timestamp\n\
    -h --help                    Print this help message\n\
!   -v --version                 Print version information\n"));
      }
  
    list_supported_targets (program_name, s);
