diff -uwr --unidirectional-new-file clamav-0.98.7+dfsg.orig/clamd/clamav-daemon.service.in clamav-0.98.7+dfsg/clamd/clamav-daemon.service.in
--- clamav-0.98.7+dfsg.orig/clamd/clamav-daemon.service.in	2015-05-06 22:33:56.000000000 +0200
+++ clamav-0.98.7+dfsg/clamd/clamav-daemon.service.in	2015-05-06 22:49:26.000000000 +0200
@@ -2,9 +2,6 @@
 Description=Clam AntiVirus userspace daemon
 Documentation=man:clamd(8) man:clamd.conf(5) http://www.clamav.net/lang/en/doc/
 Requires=clamav-daemon.socket
-# Check for database existence
-ConditionPathExistsGlob=@DBDIR@/main.{c[vl]d,inc}
-ConditionPathExistsGlob=@DBDIR@/daily.{c[vl]d,inc}
 
 [Service]
 ExecStart=@prefix@/sbin/clamd --foreground=true
Only in clamav-0.98.7+dfsg.orig/clamd: clamav-daemon.socket
diff -uwr --unidirectional-new-file clamav-0.98.7+dfsg.orig/clamd/clamav-daemon.socket.in clamav-0.98.7+dfsg/clamd/clamav-daemon.socket.in
--- clamav-0.98.7+dfsg.orig/clamd/clamav-daemon.socket.in	1970-01-01 01:00:00.000000000 +0100
+++ clamav-0.98.7+dfsg/clamd/clamav-daemon.socket.in	2015-05-06 22:49:23.000000000 +0200
@@ -0,0 +1,16 @@
+[Unit]
+Description=Socket for Clam AntiVirus userspace daemon
+Documentation=man:clamd(8) man:clamd.conf(5) http://www.clamav.net/lang/en/doc/
+# Check for database existence
+ConditionPathExistsGlob=@DBDIR@/main.{c[vl]d,inc}
+ConditionPathExistsGlob=@DBDIR@/daily.{c[vl]d,inc}
+
+[Socket]
+ListenStream=/run/clamav/clamd.ctl
+#ListenStream=127.0.0.1:1024
+SocketUser=clamav
+SocketGroup=clamav
+RemoveOnStop=True
+
+[Install]
+WantedBy=sockets.target
