diff --git a/debian/rules b/debian/rules
index 7e7ab4a..3f19cb8 100755
--- a/debian/rules
+++ b/debian/rules
@@ -267,7 +267,12 @@ install: build-arch build-indep test
 	# Alter the rpath of certain binaries and shared libraries.
 	chrpath -d $(TMPPKG_MIT)/usr/sbin/sasldblistusers2 \
 		$(TMPPKG_MIT)/usr/sbin/saslpasswd2
-	chrpath -d $(TMPPKG_MIT)/usr/lib/sasl2/libsql.so.2.0.23
+
+	for l in $(TMPPKG_MIT)/usr/lib/sasl2/lib*.so $(TMPPKG_HEIMDAL)/usr/lib/sasl2/lib*.so; do \
+	  rm $$l $$l.2; \
+	  mv $$l.2.0.23 $$l; \
+	  chrpath -d $$l; \
+	done
 
 	# Install the sasl-sample-client and -server man pages.
 	dh_installman -psasl2-bin sasl-sample-client.8 sasl-sample-server.8
@@ -311,7 +316,7 @@ binary-arch: build-arch install
 	dh_link -s
 	dh_strip -s -psasl2-bin -plibsasl2-2 -plibsasl2-modules -plibsasl2-modules-ldap -plibsasl2-modules-otp -plibsasl2-modules-sql -plibsasl2-modules-gssapi-mit -plibsasl2-dev -Nlibsasl2-modules-gssapi-heimdal --dbg-package=cyrus-sasl2-dbg
 	dh_strip -s -plibsasl2-modules-gssapi-heimdal -Nsasl2-bin -Nlibsasl2-2 -Nlibsasl2-modules -Nlibsasl2-modules-ldap -Nlibsasl2-modules-otp -Nlibsasl2-modules-sql -Nlibsasl2-modules-gssapi-mit -Nlibsasl2-dev --dbg-package=cyrus-sasl2-heimdal-dbg
-	mv debian/cyrus-sasl2-dbg/usr/lib/debug/usr/lib/sasl2/libgssapiv2.so.2.0.23 debian/cyrus-sasl2-mit-dbg/usr/lib/debug/usr/lib/sasl2/
+	mv debian/cyrus-sasl2-dbg/usr/lib/debug/usr/lib/sasl2/libgssapiv2.so debian/cyrus-sasl2-mit-dbg/usr/lib/debug/usr/lib/sasl2/
 	dh_compress -s
 	dh_fixperms -s
 	dh_makeshlibs -s -X/usr/lib/sasl2
