diff --git a/debian/libsasl2-dev.install b/debian/libsasl2-dev.install
index f4da345..77cc994 100644
--- a/debian/libsasl2-dev.install
+++ b/debian/libsasl2-dev.install
@@ -1,4 +1,4 @@
 usr/include/sasl
-usr/lib/libsasl2.??
+usr/lib/libsasl2.so
 usr/lib/libsasl2.a
 usr/share/man/man3
diff --git a/debian/libsasl2-modules-gssapi-heimdal.install b/debian/libsasl2-modules-gssapi-heimdal.install
index e3ed632..92b8b67 100644
--- a/debian/libsasl2-modules-gssapi-heimdal.install
+++ b/debian/libsasl2-modules-gssapi-heimdal.install
@@ -1 +1 @@
-usr/lib/sasl2/libgssapiv2.*
+usr/lib/sasl2/libgssapiv2.so*
diff --git a/debian/libsasl2-modules-gssapi-mit.install b/debian/libsasl2-modules-gssapi-mit.install
index e3ed632..92b8b67 100644
--- a/debian/libsasl2-modules-gssapi-mit.install
+++ b/debian/libsasl2-modules-gssapi-mit.install
@@ -1 +1 @@
-usr/lib/sasl2/libgssapiv2.*
+usr/lib/sasl2/libgssapiv2.so*
diff --git a/debian/libsasl2-modules-ldap.install b/debian/libsasl2-modules-ldap.install
index 429f8bf..47a9799 100644
--- a/debian/libsasl2-modules-ldap.install
+++ b/debian/libsasl2-modules-ldap.install
@@ -1 +1 @@
-usr/lib/sasl2/libldapdb.*
+usr/lib/sasl2/libldapdb.so*
diff --git a/debian/libsasl2-modules-otp.install b/debian/libsasl2-modules-otp.install
index a82b136..c3cc38d 100644
--- a/debian/libsasl2-modules-otp.install
+++ b/debian/libsasl2-modules-otp.install
@@ -1 +1 @@
-usr/lib/sasl2/libotp.*
+usr/lib/sasl2/libotp.so*
diff --git a/debian/libsasl2-modules-sql.install b/debian/libsasl2-modules-sql.install
index 4f2ae65..a202000 100644
--- a/debian/libsasl2-modules-sql.install
+++ b/debian/libsasl2-modules-sql.install
@@ -1 +1 @@
-usr/lib/sasl2/libsql.*
+usr/lib/sasl2/libsql.so*
diff --git a/debian/libsasl2-modules.install b/debian/libsasl2-modules.install
index 86e64a2..30490ec 100644
--- a/debian/libsasl2-modules.install
+++ b/debian/libsasl2-modules.install
@@ -1,6 +1,6 @@
-usr/lib/sasl2/libanonymous.*
-usr/lib/sasl2/libcrammd5.*
-usr/lib/sasl2/libdigestmd5.*
-usr/lib/sasl2/liblogin.*
-usr/lib/sasl2/libntlm.*
-usr/lib/sasl2/libplain.*
+usr/lib/sasl2/libanonymous.so*
+usr/lib/sasl2/libcrammd5.so*
+usr/lib/sasl2/libdigestmd5.so*
+usr/lib/sasl2/liblogin.so*
+usr/lib/sasl2/libntlm.so*
+usr/lib/sasl2/libplain.so*
