--- /etc/apparmor.d/usr.sbin.named~	2021-11-12 14:24:13.000000000 +0100
+++ /etc/apparmor.d/usr.sbin.named	2022-09-19 21:43:35.092730212 +0200
@@ -13,4 +13,7 @@
   capability sys_resource,
 
+  # named need to check if hugepages is avaiable.
+  /sys/kernel/mm/transparent_hugepage/enabled r,
+
   # /etc/bind should be read-only for bind
   # /var/lib/bind is for dynamically updated zone (and journal) files.
