Index: pkgs.dbk
===================================================================
--- pkgs.dbk	(revision 7057)
+++ pkgs.dbk	(working copy)
@@ -37,13 +37,13 @@
 header (don't use CC:, because that way the message's subject won't
 indicate the bug number). If you are packaging so many new packages (>10)
 that notifying the mailing list in seperate messages is too disruptive,
-do send a summary after filing the bugs to the debian-devel list instead.
+send a summary to the debian-devel list after filing the bugs instead.
 This will inform the other developers about upcoming packages and will
 allow a review of your description and package name.
 </para>
 <para>
 Please include a <literal>Closes:
-bug#<replaceable>nnnnn</replaceable></literal> entry in the changelog of the
+#<replaceable>nnnnn</replaceable></literal> entry in the changelog of the
 new package in order for the bug report to be automatically closed once the new
 package is installed in the archive (see <xref linkend="upload-bugfix"/> ).
 </para>
Index: resources.dbk
===================================================================
--- resources.dbk	(revision 7057)
+++ resources.dbk	(working copy)
@@ -141,7 +141,8 @@
 </para>
 <para>
 The main channel for Debian development is <literal>#debian-devel</literal>.
-It is a very active channel since usually over 150 people are always logged in.
+It is a very active channel; it will typically have a minimum of 150 people 
+at any time of day.
 It's a channel for people who work on Debian, it's not a support channel
 (there's <literal>#debian</literal> for that).  It is however open to anyone
 who wants to lurk (and learn).  Its topic is commonly full of interesting
@@ -488,7 +489,7 @@
 <para>
 Packages which do not conform to the DFSG are placed in the
 <literal>non-free</literal> section.  These packages are not considered as
-part of the Debian distribution, though we support their use, and we provide
+part of the Debian distribution, though we enable their use, and we provide
 infrastructure (such as our bug-tracking system and mailing lists) for non-free
 software packages.
 </para>
@@ -788,9 +788,9 @@
 <literal>testing</literal> as they approach stability, <literal>sid</literal>
 itself is never released.  As well as the usual contents of a Debian 
 distribution, <literal>sid</literal> contains packages for architectures which
-are not yet officially supported or released by Debian.  These architectures
-are planned to be integrated into the mainstream distribution at some future
-date.
+are not yet (or no longer) officially supported or released by Debian.  These
+architectures are planned to be integrated into the mainstream distribution at
+some future date.
 </para>
 <para>
 Since Debian has an open development model (i.e., everyone can participate and
@@ -1364,7 +1364,7 @@
 have to make sure it sends a copy of those mails to
 <literal><replaceable>sourcepackage</replaceable>_cvs@&pts-host;</literal>.
 Only the people who accept the <literal>cvs</literal> keyword will receive
-these notifications. Note that the mail need to be sent from a
+these notifications. Note that the mail needs to be sent from a
 <literal>debian.org</literal> machine, otherwise you'll have to add
 the <literal>X-PTS-Approved: 1</literal> header.
 </para>
