diff -ur dput-0.9.2.25.orig/COPYING dput-0.9.2.25/COPYING
--- dput-0.9.2.25.orig/COPYING	2001-04-07 23:28:10.000000000 +0200
+++ dput-0.9.2.25/COPYING	2006-07-08 14:19:52.000000000 +0200
@@ -2,7 +2,8 @@
 		       Version 2, June 1991
 
  Copyright (C) 1989, 1991 Free Software Foundation, Inc.
-                       59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
+ 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA
+
  Everyone is permitted to copy and distribute verbatim copies
  of this license document, but changing it is not allowed.
 
diff -ur dput-0.9.2.25.orig/FAQ dput-0.9.2.25/FAQ
--- dput-0.9.2.25.orig/FAQ	2004-01-20 21:56:21.000000000 +0100
+++ dput-0.9.2.25/FAQ	2006-07-08 14:15:04.000000000 +0200
@@ -33,7 +33,7 @@
    Because dput will make you rich and attract women.
 
    Now, seriously, it's a matter of preferences. If you use dupload
-   and like it's behaviour, then continue to use it. But in every
+   and like its behaviour, then continue to use it. But in every
    case, I would suggest that you try dput, since it will check the
    GnuPG signature and the checksums before uploading and offer to run
    dinstall in dry-run mode after the upload. It also offers the usage
@@ -95,7 +95,7 @@
    [unstable]
 
 
-8. How to set up dput to fill your remote apt repository
+8. How do I set up dput to fill my remote apt repository?
 
    Assuming you have remote ssh access, you can use the following
    You can use the same mini-dinstall configuration file (and setup
diff -ur dput-0.9.2.25.orig/Thanks dput-0.9.2.25/Thanks
--- dput-0.9.2.25.orig/Thanks	2004-06-10 11:16:20.000000000 +0200
+++ dput-0.9.2.25/Thanks	2006-07-08 14:13:19.000000000 +0200
@@ -1,5 +1,5 @@
 Here we thank some people who contributed either code, documentation or
-some good suggestion/ideas. There help improved this tool. 
+some good suggestion/ideas. Their help improved this tool.
 
 Here's now a list of people who contributed either some code, patches,
 documentation or other good stuff for dput.
diff -ur dput-0.9.2.25.orig/copyright dput-0.9.2.25/copyright
--- dput-0.9.2.25.orig/copyright	2005-03-25 18:20:58.000000000 +0100
+++ dput-0.9.2.25/copyright	2006-07-08 14:20:14.000000000 +0200
@@ -18,7 +18,7 @@
 
  You should have received a copy of the GNU General Public License along
  with this program; if not, write to the Free Software Foundation, Inc.,
- 59 Temple Place - Suite 330, Boston, MA 02111-1307, USA.
+ 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA
 
 On Debian GNU/Linux systems, the complete text of the GNU General
 Public License can be found in `/usr/share/common-licenses/GPL'.
