From 71ea4d17c6cc0e9b8664ba6c8ddf345ce86562e4 Mon Sep 17 00:00:00 2001
From: Jakub Wilk <jwilk@jwilk.net>
Date: Mon, 4 Feb 2019 19:43:54 +0100
Subject: [PATCH] rmadison: Use HTTPS for people.canonical.com

Signed-off-by: Jakub Wilk <jwilk@jwilk.net>
---
 scripts/rmadison.pl | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scripts/rmadison.pl b/scripts/rmadison.pl
index 580da87c..f0efb1cf 100755
--- a/scripts/rmadison.pl
+++ b/scripts/rmadison.pl
@@ -54,7 +54,7 @@ my %url_map = (
     'debian' => "https://api.ftp-master.debian.org/madison",
     'new'    => "https://api.ftp-master.debian.org/madison?s=new",
     'qa'     => "https://qa.debian.org/madison.php",
-    'ubuntu' => "http://people.canonical.com/~ubuntu-archive/madison.cgi",
+    'ubuntu' => "https://people.canonical.com/~ubuntu-archive/madison.cgi",
     'udd'    => 'https://qa.debian.org/cgi-bin/madison.cgi',
 );
 my $default_url = 'debian';
@@ -322,7 +322,7 @@ use I<URL> for the query. Supported shorthands are
  B<debian> https://api.ftp-master.debian.org/madison
  B<new> https://api.ftp-master.debian.org/madison?s=new
  B<qa> https://qa.debian.org/madison.php
- B<ubuntu> http://people.canonical.com/~ubuntu-archive/madison.cgi
+ B<ubuntu> https://people.canonical.com/~ubuntu-archive/madison.cgi
  B<udd> https://qa.debian.org/cgi-bin/madison.cgi
 
 See the B<RMADISON_URL_MAP_> variable below for a method to add
-- 
2.20.1

