diff --git a/src/main/help.c b/src/main/help.c
index 7762aca..f7d9d16 100644
--- a/src/main/help.c
+++ b/src/main/help.c
@@ -83,6 +83,10 @@ find_command(const char *prog)
   const char *path, *path_end;
   size_t path_len;
 
+  if (*prog == '/') {
+    return stat(prog, &stab) == 0 && (stab.st_mode & 0111);
+  }
+
   path_list = getenv("PATH");
   if (!path_list)
     ohshit(_("PATH is not set"));
