diff -u -r docbook-utils-0.6.14.orig/debian/changelog docbook-utils-0.6.14/debian/changelog
--- docbook-utils-0.6.14.orig/debian/changelog  2009-01-12 16:42:19.000000000 -0500
+++ docbook-utils-0.6.14/debian/changelog       2009-01-12 16:43:45.000000000 -0500
@@ -1,3 +1,10 @@
+docbook-utils (0.6.14-1.2) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Add missing Build-Depends-Indep on docbook
+
+ -- Stephen Depooter <stephend@xandros.com>  Mon, 27 Oct 2008 16:43:25 -0400
+
 docbook-utils (0.6.14-1.1) unstable; urgency=medium

   * Non-maintainer upload.
diff -u -r docbook-utils-0.6.14.orig/debian/control docbook-utils-0.6.14/debian/control
--- docbook-utils-0.6.14.orig/debian/control    2009-01-12 16:42:19.000000000 -0500
+++ docbook-utils-0.6.14/debian/control 2009-01-12 16:43:45.000000000 -0500
@@ -4,7 +4,7 @@
 Maintainer: Debian XML/SGML Group <debian-xml-sgml-pkgs@lists.alioth.debian.org>
 Uploaders: Ardo van Rangelrooij <ardo@debian.org>
 Standards-Version: 3.7.3
-Build-Depends-Indep: debhelper (>= 4.1), docbook-dsssl, jade, perl, sgmlspl, sp
+Build-Depends-Indep: debhelper (>= 4.1), docbook-dsssl, jade, perl, sgmlspl, sp, docbook

 Package: docbook-utils
 Section: text

