--- console-setup.orig/Keyboard/KeyboardNames.pl	2006-11-13 19:24:57.000000000 +0000
+++ console-setup/Keyboard/KeyboardNames.pl	2007-04-24 16:46:43.000000000 +0100
@@ -9,6 +9,8 @@
     'Acer AirKey V' => 'airkey',
     'Acer TravelMate 800' => 'acer_tm_800',
     'Advance Scorpius KI' => 'scorpius',
+    'Amiga' => 'amiga',
+    'Atari TT' => 'ataritt',
     'Azona RF2300 wireless Internet Keyboard' => 'azonaRF2300',
     'BTC 5090' => 'btc5090',
     'BTC 5113RF Multimedia' => 'btc5113rf',
@@ -111,6 +113,8 @@
     'SVEN Ergonomic 2500' => 'sven',
     'Samsung SDM 4500P' => 'samsung4500',
     'Samsung SDM 4510P' => 'samsung4510',
+    'Sun Type 4' => 'sun4',
+    'Sun Type 5' => 'sun5',
     'Super Power Multimedia Keyboard' => 'sp_inet',
     'Symplon PaceBook (tablet PC)' => 'symplon',
     'Toshiba Satellite S3000' => 'toshiba_s3000',
--- console-setup.orig/Keyboard/xmlreader	2006-09-28 08:00:03.000000000 +0100
+++ console-setup/Keyboard/xmlreader	2007-04-04 08:38:45.000000000 +0100
@@ -263,6 +263,21 @@
     }
 }
 
+# Fixups for model names we need
+my %modelvalues = map { $_ => 1 } values %models;
+if (not exists $modelvalues{amiga}) {
+    $models{'Amiga'} = 'amiga';
+}
+if (not exists $modelvalues{ataritt}) {
+    $models{'Atari TT'} = 'ataritt';
+}
+if (not exists $modelvalues{sun4}) {
+    $models{'Sun Type 4'} = 'sun4';
+}
+if (not exists $modelvalues{sun5}) {
+    $models{'Sun Type 5'} = 'sun5';
+}
+
 print <<'EOT';
 #!/usr/bin/perl -w
 
