diff -Nru ace-of-penguins-1.3/debian/ace-of-penguins.6.orig.man ace-of-penguins-1.3/debian/ace-of-penguins.6.orig.man
--- ace-of-penguins-1.3/debian/ace-of-penguins.6.orig.man	2010-06-26 13:08:57.000000000 +0200
+++ ace-of-penguins-1.3/debian/ace-of-penguins.6.orig.man	2010-10-31 13:24:59.000000000 +0100
@@ -1,4 +1,4 @@
-.TH ace_of_penguins 6 "9 September 1999" "1.2" "Ace of Penguins"
+.TH ace_of_penguins 6 "9 September 1999" "1.3" "Ace of Penguins"
 .SH NAME
 ace_of_penguins \- Solitaire-games with penguin-look.
 .SH SYNOPSIS
@@ -18,6 +18,8 @@
 .br
 .B ace_solitaire
 .br
+.B ace_spider
+.br
 .B ace_taipei
 .br
 .B ace_taipedit
@@ -37,11 +39,13 @@
 .TP
 .B ace_solitaire
 .TP
+.B ace_spider
+.TP
 .B ace_thornq
 solitary card-games
 .TP
 .B ace_golf
-annother solitray card-game
+another solitary card-game
 .TP
 .B ace_mastermind
 guess a secret combination of colors
diff -Nru ace-of-penguins-1.3/debian/ace-of-penguins.6.pod ace-of-penguins-1.3/debian/ace-of-penguins.6.pod
--- ace-of-penguins-1.3/debian/ace-of-penguins.6.pod	2010-06-26 13:08:57.000000000 +0200
+++ ace-of-penguins-1.3/debian/ace-of-penguins.6.pod	2010-10-31 13:26:58.000000000 +0100
@@ -27,6 +27,7 @@
   ace-minesweeper
   ace-pegged
   ace-solitaire
+  ace-spider
   ace-taipei
   ace-taipedit
   ace-thornq
@@ -51,6 +52,10 @@
 
 Solitary card-game
 
+=item ace-spider
+
+Solitary card-game
+
 =item ace-solitaire
 
 Solitary card-game
@@ -61,7 +66,7 @@
 
 =item ace-golf
 
-Another solitray card-game
+Another solitary card-game
 
 =item ace-mastermind
 
diff -Nru ace-of-penguins-1.3/debian/menu ace-of-penguins-1.3/debian/menu
--- ace-of-penguins-1.3/debian/menu	2010-06-26 13:08:57.000000000 +0200
+++ ace-of-penguins-1.3/debian/menu	2010-10-31 13:25:44.000000000 +0100
@@ -13,6 +13,8 @@
 ?package(ace-of-penguins):needs="X11" section="Games/Card"\
   title="Penguin Solitaire" command="/usr/games/ace-solitaire"
 ?package(ace-of-penguins):needs="X11" section="Games/Card"\
+  title="Penguin Spider" command="/usr/games/ace-spider"
+?package(ace-of-penguins):needs="X11" section="Games/Card"\
   title="Penguin Canfield" command="/usr/games/ace-canfield"
 ?package(ace-of-penguins):needs="X11" section="Games/Card"\
   title="Penguin Thornq" command="/usr/games/ace-thornq"
