Index: ftpfs.c
===================================================================
--- ftpfs.c	(revision 4411)
+++ ftpfs.c	(revision 4425)
@@ -1280,7 +1280,8 @@
   memset(&ftpfs, 0, sizeof(ftpfs));
 
   ftpfs.curl_version = curl_version_info(CURLVERSION_NOW);
-  ftpfs.safe_nobody = ftpfs.curl_version->version_num > CURLFTPFS_BAD_NOBODY;
+  //ftpfs.safe_nobody = ftpfs.curl_version->version_num > CURLFTPFS_BAD_NOBODY;
+  ftpfs.safe_nobody = 0;
   
   ftpfs.blksize = 4096;
   
