Description: Avoid crash on certain epub files

Author: Bernhard Übelacker <bernhardu@mailbox.org>
Bug-Debian: https://bugs.debian.org/972715
Forwarded: no
Last-Update: 2020-11-01

Index: atril-1.24.0/backend/epub/epub-document.c
===================================================================
--- atril-1.24.0.orig/backend/epub/epub-document.c
+++ atril-1.24.0/backend/epub/epub-document.c
@@ -1280,7 +1280,7 @@ setup_document_index(EpubDocument *epub_
     		xml_parse_children_of_node(navLabel,(xmlChar*)"text",NULL,NULL);
             linknode *newnode = g_new0(linknode,1);
     		newnode->linktext = NULL;
-    		while (newnode->linktext == NULL) {
+    		while (xmlretval && newnode->linktext == NULL) {
         		newnode->linktext = (gchar*)xml_get_data_from_node(xmlretval,XML_KEYWORD,NULL);
     			xmlretval = xmlretval->next;
     		}
@@ -1599,7 +1599,7 @@ page_set_function(linknode *Link, GList
 	contentListNode *pagedata;
 
 	guint flag=0;
-	while (!flag) {
+	while (listiter && !flag) {
 		pagedata = listiter->data;
 		if (link_present_on_page(Link->pagelink, pagedata->value)) {
 			flag=1;
