Index: bash-completion-2.0/completions/make
===================================================================
--- bash-completion-2.0.orig/completions/make	2012-06-15 11:21:55.000000000 +0200
+++ bash-completion-2.0/completions/make	2012-07-23 16:36:23.429434310 +0200
@@ -74,6 +74,6 @@
 
     fi
 } &&
-complete -F _make make gmake gnumake pmake
+complete -F _make make gmake gnumake pmake colormake
 
 # ex: ts=4 sw=4 et filetype=sh
