--- src/blackbox.cc.orig	2015-05-14 06:53:02.221198559 +0200
+++ src/blackbox.cc	2015-05-14 07:08:26.020864286 +0200
@@ -281,6 +281,7 @@
     bool lost_focus = true; // did the window really lose focus?
     bool no_focus = true;   // did another window get focus?
 
+    XSync(XDisplay(), False);
     XEvent event;
     if (XCheckIfEvent(XDisplay(), &event, scanForFocusIn, NULL)) {
       process_event(&event);
