From ff39c1ca5ce23a1b4ce100cf8d25db9878978a9a Mon Sep 17 00:00:00 2001
From: Hannes von Haugwitz <hannes@vonhaugwitz.com>
Date: Sat, 4 Oct 2014 09:42:31 +0200
Subject: [PATCH] Fix misspelled account name of Hannes von Haugwitz

Signed-off-by: Hannes von Haugwitz <hannes@vonhaugwitz.com>
---
 keyids | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/keyids b/keyids
index 296755a..2009149 100644
--- a/keyids
+++ b/keyids
@@ -2076,7 +2076,7 @@
 0xF675AAE2D2AB2220 Jeremy Malcolm <terminus>
 0xF67E2A5D1CF2D62A Micah Anderson <micah>
 0xF68DE0501BB18FFB Corrin Lakeland <lakeland>
-0xF6947DAB68E7B931 Hannes von Haugwitz <jvhaugwitz>
+0xF6947DAB68E7B931 Hannes von Haugwitz <hvhaugwitz>
 0xF6A57C9A24F2A001 Konstantinos Margaritis <markos>
 0xF6AB49D324618BF6 LaMont Jones <lamont>
 0xF6AEF2AFD17897FA Vincent Danjean <vdanjean>
-- 
2.1.0

