diff --git a/methods/mirror.cc b/methods/mirror.cc
index eb6d974..d6c5ba9 100644
--- a/methods/mirror.cc
+++ b/methods/mirror.cc
@@ -150,7 +150,7 @@ bool MirrorMethod::DownloadMirrorFile(string mirror_uri_str)
    for (std::vector<std::string>::const_iterator I = vec.begin();
         I != vec.end(); ++I)
       if (I == vec.begin())
-         fetch += "?arch" + (*I);
+         fetch += "?arch=" + (*I);
       else
          fetch += "&arch=" + (*I);
 
