diff -rau ../orig/docbook-xsl-1.73.2.dfsg.1/manpages/other.xsl ./docbook-xsl-1.73.2.dfsg.1/manpages/other.xsl
--- ../orig/docbook-xsl-1.73.2.dfsg.1/manpages/other.xsl	2007-06-23 11:07:45.000000000 +0200
+++ ./docbook-xsl-1.73.2.dfsg.1/manpages/other.xsl	2008-03-29 15:17:57.422038192 +0100
@@ -191,8 +191,6 @@
 
 <xsl:template name="escape.roff.specials">
   <xsl:param name="content"/>
-  <xsl:call-template name="convert.nobreak-space">
-    <xsl:with-param name="content">
       <xsl:call-template name="escape.apostrophe">
         <xsl:with-param name="content">
           <xsl:call-template name="escape.dash">
@@ -208,8 +206,6 @@
           </xsl:call-template>
         </xsl:with-param>
       </xsl:call-template>
-    </xsl:with-param>
-  </xsl:call-template>
 </xsl:template>
 
 <xsl:template name="escape.backslash">
@@ -252,30 +248,6 @@
   </xsl:call-template>
 </xsl:template>
 
-<xsl:template name="convert.nobreak-space">
-  <xsl:param name="content"/>
-  <xsl:call-template name="string.subst">
-    <xsl:with-param name="string" select="$content"/>
-    <xsl:with-param name="target">&#x00a0;</xsl:with-param>
-    <!-- * A no-break space can be written two ways in roff; the -->
-    <!-- * difference, according to the "Page Motions" node in the -->
-    <!-- * groff info page, is: -->
-    <!-- *  -->
-    <!-- *   "\ " = -->
-    <!-- *   An unbreakable and unpaddable (i.e. not expanded -->
-    <!-- *   during filling) space. -->
-    <!-- *  -->
-    <!-- *   "\~" = -->
-    <!-- *   An unbreakable space that stretches like a normal -->
-    <!-- *   inter-word space when a line is adjusted."  -->
-    <!-- *  -->
-    <!-- * Unfortunately, roff seems to do some weird things with -->
-    <!-- * long lines that only have words separated by "\~" -->
-    <!-- * spaces, so it's safer just to stick with the "\ " space -->
-    <xsl:with-param name="replacement">\ </xsl:with-param>
-  </xsl:call-template>
-</xsl:template>
-
 <!-- ==================================================================== -->
 
 <!-- * top.comment generates a comment containing metadata for the man -->
