diff -r 9b9ecdfff423 src/vscreen/vs_editline.cc
--- a/src/vscreen/vs_editline.cc	Wed Jun 20 06:14:18 2007 -0700
+++ b/src/vscreen/vs_editline.cc	Sun Jun 24 10:31:00 2007 -0700
@@ -369,7 +369,11 @@ bool vs_editline::handle_key(const key &
   else
     {
       if(clear_on_this_edit)
-	text.clear();
+	{
+	  text.clear();
+	  curloc = 0;
+	  startloc = 0;
+	}
 
       text.insert(curloc++, 1, k.ch);
       normalize_cursor();
