--- debian/control.orig	2004-06-05 19:11:49.000000000 +0200
+++ debian/control	2004-06-05 19:11:07.000000000 +0200
@@ -11,7 +11,8 @@
  libgtkmm-dev (>= 1.2.7), libgtk-canvas1-dev (>= 0.1.1-5),
  autoconf (>= 2.52), automake1.7 | automake, pkg-config, libtool, m4, flex,
  gettext, cvs, netbase (>= 4.13)
-Build-Conflicts: libardour0, libpbd0, libmidi++0, libgtkmmext0, libsoundtouch0
+Build-Conflicts: libardour0, libpbd0, libmidi++0, libgtkmmext0, libsoundtouch0,
+ automake1.4
 Standards-Version: 3.6.1
 
 Package: ardour-doc
