diff -r -u autolog-0.40.old/autolog.c autolog-0.40/autolog.c
--- autolog-0.40.old/autolog.c	2003-02-22 20:22:14.000000000 -0600
+++ autolog-0.40/autolog.c	2003-07-13 01:07:42.000000000 -0500
@@ -520,7 +520,7 @@
 	
 /*    if (debug)
         printf("pat_match:%s:%s:\n",patt,strg); */
-    len = strlen(patt+256);
+    len = strlen(patt) + 256;
     rpb.buffer = malloc(len);
     rpb.allocated = len;
     rpb.fastmap = 0;
