diff -u cmatrix-1.2a/debian/changelog cmatrix-1.2a/debian/changelog
--- cmatrix-1.2a/debian/changelog
+++ cmatrix-1.2a/debian/changelog
@@ -1,3 +1,15 @@
+cmatrix (1.2a-2.1) unstable; urgency=high
+
+  * Non-maintainer upload with high urgency for fixing RC-bug.
+  * Use debhelper v5
+  * Updated to Standards version 3.7.2
+  * Completed xfonts transitions (Closes: #362323)
+    + Build-dep on debhelper 5.0.29
+    + Install fonts in /usr/share/fonts/X11
+  * Adjust debian/copyright (Closes: #290089)
+
+ -- Mohammed Adnène Trojette <adn+deb@diwi.org>  Tue, 12 Dec 2006 05:28:21 +0100
+
 cmatrix (1.2a-2) unstable; urgency=low
 
   * cmatric.c: Fix segfault on terminal resize (Closes: Bug#268811)
diff -u cmatrix-1.2a/debian/copyright cmatrix-1.2a/debian/copyright
--- cmatrix-1.2a/debian/copyright
+++ cmatrix-1.2a/debian/copyright
@@ -6,7 +6,9 @@
 Upstream Author: Chris Allegretta <chrisa@asty.org>
 
 Copyright:
+Copyright (C) 1999 Chris Allegretta
 
+License:
 CMatrix is distributed under the terms of the GNU GPL version 2.
 
 On Debian GNU/Linux systems, the complete text of the GNU General
diff -u cmatrix-1.2a/debian/control cmatrix-1.2a/debian/control
--- cmatrix-1.2a/debian/control
+++ cmatrix-1.2a/debian/control
@@ -2,8 +2,8 @@
 Section: misc
 Priority: optional
 Maintainer: Edward Betts <edward@debian.org>
-Standards-Version: 3.6.1
-Build-Depends: cdbs, debhelper (>= 4.1.0), libncurses5-dev
+Standards-Version: 3.7.2
+Build-Depends: cdbs, debhelper (>= 5.0.29), libncurses5-dev
 
 Package: cmatrix
 Architecture: any
diff -u cmatrix-1.2a/debian/compat cmatrix-1.2a/debian/compat
--- cmatrix-1.2a/debian/compat
+++ cmatrix-1.2a/debian/compat
@@ -1 +1 @@
-4
+5
diff -u cmatrix-1.2a/debian/cmatrix-xfont.install cmatrix-1.2a/debian/cmatrix-xfont.install
--- cmatrix-1.2a/debian/cmatrix-xfont.install
+++ cmatrix-1.2a/debian/cmatrix-xfont.install
@@ -1 +1 @@
-mtx.pcf usr/X11R6/lib/X11/fonts/misc
+mtx.pcf usr/share/fonts/X11/misc
