diff --git a/debian/binfmt-support.conf b/debian/binfmt-support.conf
new file mode 100644
index 0000000..be7b3b7
--- /dev/null
+++ b/debian/binfmt-support.conf
@@ -0,0 +1,4 @@
+[Unit]
+# Do not run systemd-binfmt.service if binfmt-support is installed
+# as this can lead to race conditions during boot.
+ConditionFileIsExecutable=!/usr/sbin/update-binfmts
diff --git a/debian/binfmt-support.install b/debian/binfmt-support.install
new file mode 100644
index 0000000..48b7269
--- /dev/null
+++ b/debian/binfmt-support.install
@@ -0,0 +1 @@
+debian/binfmt-support.conf lib/systemd/system/systemd-binfmt.service.d/
