--- /home/peuh/crans/custom/packages/apt-file/apt-file-2.5.2.2/apt-file	2014-06-22 11:03:50.000000000 +0200
+++ /usr/bin/apt-file	2014-07-09 21:41:49.780877466 +0200
@@ -132,7 +132,7 @@
                            |rpm-
 			 )
                     /xo;
-            next if !m/^deb\s/;
+            next if !m/^\s*deb\s/;
             chomp;
             my $line = $_;
             debug "got \'$line\'";
