--- console-setup.orig/Makefile	2007-03-14 19:46:05.000000000 +0000
+++ console-setup/Makefile	2007-04-24 22:37:45.000000000 +0100
@@ -20,6 +20,9 @@
 	install -m 644 Fonts/*.psf.gz $(prefix)/share/consolefonts/
 	install -d $(prefix)/share/consoletrans
 	install -m 644 Keyboard/acm/*.acm.gz $(prefix)/share/consoletrans
+	install -d $(prefix)/share/console-setup
+	install -m 644 Keyboard/MyKeyboardNames.pl $(prefix)/share/console-setup/KeyboardNames.pl
+	install Keyboard/kbdnames-maker $(prefix)/share/console-setup
 	install -d  $(prefix)/bin/
 	install Keyboard/ckbcomp $(prefix)/bin/
 	install -d  $(bootprefix)/bin/

