From: Artur Rona <ari-tczew@ubuntu.com>
Description: Fix implicit pointer conversion.
Author: Julien Lavergne <gilir@ubuntu.com>

diff -urNad ace-1.3~/games/spider.c ace-1.3/games/spider.c
--- ace-1.3~/games/spider.c	2008-04-20 07:39:39.000000000 +0200
+++ ace-1.3/games/spider.c	2010-07-23 16:21:06.882845602 +0200
@@ -21,7 +21,7 @@
 #include <time.h>

+#include "imagelib.h" 
 #include "cards.h"
-//#include "imagelib.h"
 
 #define W CARD_WIDTH
 #define H CARD_HEIGHT
