Index: README
===================================================================
--- README	(revisione 4395)
+++ README	(copia locale)
@@ -144,7 +144,7 @@
 2.3 Skin Installation
 ---------------------
 
-Audacious will create a directory called ~/.share/audacious/Skins. You can unarchive
+Audacious will create a directory called ~/.local/share/audacious/Skins. You can unarchive
 the skins the same way as you do for Winamp.
 
 However, you don't need to as Audacious supports archived skins. Audacious
@@ -160,7 +160,7 @@
 Audacious looks for skins in these directories (in listing order):
 
 <prefix>/share/audacious/Skins
-~/.share/audacious/Skins
+~/.local/share/audacious/Skins
 ~/.audacious/Skins
 
 You can set the environment variable SKINSDIR to another location of
Index: man/audacious.1.in
===================================================================
--- man/audacious.1.in	(revisione 4395)
+++ man/audacious.1.in	(copia locale)
@@ -192,11 +192,11 @@
 User's set of Audacious-specific GTK config settings. Options such as widget
 color and fonts sizes can be set here.
 .TP
-~/.share/audacious/Skins, @datadir@/audacious/Skins
+~/.local/share/audacious/Skins, @datadir@/audacious/Skins
 Default location(s) where Audacious should look for skinfiles.
 .TP
-~/.config/audacious/Plugins
-Location of user-installed plugins. (subject to change)
+~/.local/share/audacious/Plugins
+Location of user-installed plugins.
 .SH ENVIRONMENT
 .TP 12
 .B SKINSDIR
