--- /etc/logrotate.d/clamav-freshclam.orig	2015-06-09 10:55:58.597662646 +1000
+++ /etc/logrotate.d/clamav-freshclam	2015-06-14 05:35:30.653954361 +1000
@@ -7,7 +7,7 @@
      create 640  clamav adm
      postrotate
      if [ -d /run/systemd/system ]; then
-         systemctl -q is-active clamav-freshclam && systemctl kill --signal=SIGHUP clamav-freshclam
+         systemctl -q is-active clamav-freshclam && systemctl kill --signal=SIGHUP clamav-freshclam || :
      else
          /etc/init.d/clamav-freshclam reload-log > /dev/null
      fi
