Description: Fix build failure when building with libarchive 3.0.2.
Author: Andres Mejia <amejia@debian.org>

--- a/src/main.c	2011-12-25 12:59:50.356201098 -0500
+++ b/src/main.c	2011-12-25 13:03:18.896199110 -0500
@@ -142,7 +142,8 @@
 	bind_textdomain_codeset (GETTEXT_PACKAGE, "UTF-8");
 	textdomain (GETTEXT_PACKAGE);
 #endif
-	g_assert (archive_api_version () == ARCHIVE_API_VERSION);
+	g_assert ((archive_version_number() / 1000000) ==
+	          (ARCHIVE_VERSION_NUMBER   / 1000000));
 	remaining_args = NULL;
 	window1 = NULL;
 	option_context = g_option_context_new ("");
