From c9b2cc6c6a715ad68a41c7a01c528bdf2aabf075 Mon Sep 17 00:00:00 2001
From: Daniel Hartwig <mandyke@gmail.com>
Date: Sat, 24 Dec 2011 14:15:26 +0800
Subject: [PATCH] Wrap long description of boolean_config_treeitems.

---
 src/apt_config_treeitems.cc |   13 +++++++------
 1 files changed, 7 insertions(+), 6 deletions(-)

diff --git a/src/apt_config_treeitems.cc b/src/apt_config_treeitems.cc
index 59f45f4..c168ec9 100644
--- a/src/apt_config_treeitems.cc
+++ b/src/apt_config_treeitems.cc
@@ -96,12 +96,13 @@ namespace aptitude
 	  bool value = aptcfg->FindB(item, dflt);
 
 	  std::vector<cw::fragment *> fragments;
-	  fragments.push_back(cw::fragf(_("%BOption:%b  %s\n"
-					  "%BDefault:%b %s\n"
-					  "%BValue:%b   %s\n"),
-					item.c_str(),
-					dflt  ? _("True") : _("False"),
-					value ? _("True") : _("False")));
+          fragments.push_back(drophardwrapbox(cw::fragf(_("%BOption:%b  ")),
+                                              item));
+          fragments.push_back(drophardwrapbox(cw::fragf(_("%BDefault:%b ")),
+                                              dflt  ? _("True") : _("False")));
+          fragments.push_back(drophardwrapbox(cw::fragf(_("%BValue:%b   ")),
+                                              value ? _("True") : _("False")));
+
 	  fragments.push_back(cw::newline_fragment());
 	  fragments.push_back(wrapbox(cw::fragf(long_description.c_str())));
 
-- 
1.7.5.4

