Index: fai-chboot
===================================================================
--- fai-chboot	(revision 3540)
+++ fai-chboot	(working copy)
@@ -251,6 +251,10 @@
   # rename network config file
   my ($host) = shift;
   my ($ipadr,$hex) = host2hex($host);
+  if (! -e "$pxedir/$hex") {
+    print "$host ($hex) is not enabled\n";
+    return;
+  }
   print "disable pxe config for $host in hex $hex\n" if $verbose;
   return if $opt_n;
   rename "$pxedir/$hex","$pxedir/$hex.disable" or $error .= "\nRename for $hex failed. $! $@";
