
---
Last-Update: 2019-03-18

--- bluez-5.43.orig/src/profile.c
+++ bluez-5.43/src/profile.c
@@ -198,7 +198,6 @@
 		<attribute id=\"0x0001\">				\
 			<sequence>					\
 				<uuid value=\"0x1101\" />		\
-				%s					\
 			</sequence>					\
 		</attribute>						\
 		<attribute id=\"0x0004\">				\
@@ -595,9 +594,7 @@
 			<sequence>					\
 				<sequence>				\
 					<uuid value=\"0x0100\" />	\
-					%s				\
 				</sequence>				\
-				%s					\
 			</sequence>					\
 		</attribute>						\
 		<attribute id=\"0x0005\">				\
@@ -605,7 +602,6 @@
 				<uuid value=\"0x1002\" />		\
 			</sequence>					\
 		</attribute>						\
-		%s							\
 		<attribute id=\"0x0100\">				\
 			<text value=\"%s\" />				\
 		</attribute>						\
