diff --git a/policy.sgml b/policy.sgml
index 404dc73..73b3391 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -2010,7 +2010,7 @@ zope.
 	      <p>
 		The <tt>build-arch</tt> target must
 		perform all the configuration and compilation required for
-		producing all architecture-dependant binary packages
+		producing all architecture-dependent binary packages
 		(those packages for which the body of the
 		<tt>Architecture</tt> field in <tt>debian/control</tt> is
 		not <tt>all</tt>).  Similarly, the <tt>build-indep</tt>
@@ -3069,7 +3069,7 @@ Package: libc6
 	    Specifying <tt>any all</tt> indicates that the source package
 	    isn't dependent on any particular architecture. The set of
 	    produced binary packages will include at least one
-	    architecture-dependant package and one architecture-independent
+	    architecture-dependent package and one architecture-independent
 	    package.
 	  </p>
 
@@ -6991,7 +6991,7 @@ Built-Using: grub2 (= 1.99-9), loadlin (= 1.6e-1)
                   be accessible through the search path
                   <file>/usr/include/<var>triplet</var></file> where
                   <tt><var>triplet</var></tt> is as above.  <footnote>
-                    This is necessary for architecture-dependant headers
+                    This is necessary for architecture-dependent headers
                     file to coexist in a <tt>multiarch</tt> setup.
                   </footnote>
                 </p>
