Description: Respect system locale instead of asking the user
Author: Fabian Greffrath <fabian@debian-unofficial.org>
Bug-Debian: http://bugs.debian.org/481424

diff -Nur -x '*.orig' -x '*~' audacity-1.3.9/src/AudacityApp.cpp audacity-1.3.9.new/src/AudacityApp.cpp
--- audacity-1.3.9/src/AudacityApp.cpp	2009-09-03 15:36:20.000000000 +0200
+++ audacity-1.3.9.new/src/AudacityApp.cpp	2009-09-03 15:36:30.000000000 +0200
@@ -71,6 +71,7 @@
 #include "FreqWindow.h"
 #include "Internat.h"
 #include "LangChoice.h"
+#include "Languages.h"
 #include "Prefs.h"
 #include "Project.h"
 #include "Screenshot.h"
@@ -943,7 +944,7 @@
 
    // Pop up a dialog the first time the program is run
    if (lang == wxT(""))
-      lang = ChooseLanguage(NULL);
+      lang = GetSystemLanguageCode();
 
 #ifdef NOT_RQD
 //TIDY-ME: (CleanSpeech) Language prompt??
