From 562a612feeb63a396d983afff047f58b8c5f63d6 Mon Sep 17 00:00:00 2001
From: Raphael Geissert <geissert@debian.org>
Date: Thu, 25 Oct 2012 17:18:49 -0500
Subject: [PATCH] checkbashisms: display the fancy filename in error messages

---
 scripts/checkbashisms.pl |    6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/scripts/checkbashisms.pl b/scripts/checkbashisms.pl
index 63cf510..7ea977d 100755
--- a/scripts/checkbashisms.pl
+++ b/scripts/checkbashisms.pl
@@ -468,11 +468,11 @@ foreach my $filename (@ARGV) {
 	}
     }
 
-    warn "error: $filename:  Unterminated heredoc found, EOF reached. Wanted: <$cat_string>, opened in line $start_lines{'cat_string'}\n"
+    warn "error: $display_filename:  Unterminated heredoc found, EOF reached. Wanted: <$cat_string>, opened in line $start_lines{'cat_string'}\n"
 	if ($cat_string ne '');
-    warn "error: $filename: Unterminated quoted string found, EOF reached. Wanted: <$quote_string>, opened in line $start_lines{'quote_string'}\n"
+    warn "error: $display_filename: Unterminated quoted string found, EOF reached. Wanted: <$quote_string>, opened in line $start_lines{'quote_string'}\n"
 	if ($quote_string ne '');
-    warn "error: $filename: EOF reached while on line continuation.\n"
+    warn "error: $display_filename: EOF reached while on line continuation.\n"
 	if ($buffered_line ne '');
 
     close C;
-- 
1.7.10

