diff --git a/debian/busybox-syslogd.busybox-klogd.init b/debian/busybox-syslogd.busybox-klogd.init
index 9acda52..4e9bed6 100644
--- a/debian/busybox-syslogd.busybox-klogd.init
+++ b/debian/busybox-syslogd.busybox-klogd.init
@@ -79,7 +79,7 @@ stop_server() {
 }
 
 running() {
-    cut -d ' ' -f 1-2 /proc/[0-9]*/stat 2> /dev/null | grep -F "($NAME)"
+    cut -d ' ' -f 1-2 /proc/[0-9]*/stat 2> /dev/null | grep -F "($DAEMON)"
 }
 
 case "$1" in
diff --git a/debian/busybox-syslogd.default b/debian/busybox-syslogd.default
index bc9a28f..4518307 100644
--- a/debian/busybox-syslogd.default
+++ b/debian/busybox-syslogd.default
@@ -3,5 +3,6 @@
 
 # Additional options that are passed to the daemons.  Default is to log
 # to ring buffer (to be read with logread(1)) and drop duplicates.
-SYSLOG_OPTS="-C128"
+#SYSLOG_OPTS="-C128"
+SYSLOG_OPTS=""
 KLOG_OPTS=""
diff --git a/debian/busybox-syslogd.init b/debian/busybox-syslogd.init
index f489dd2..5b93259 100644
--- a/debian/busybox-syslogd.init
+++ b/debian/busybox-syslogd.init
@@ -79,7 +79,7 @@ stop_server() {
 }
 
 running() {
-    cut -d ' ' -f 1-2 /proc/[0-9]*/stat 2> /dev/null | grep -F "($NAME)"
+    cut -d ' ' -f 1-2 /proc/[0-9]*/stat 2> /dev/null | grep -F "($DAEMON)"
 }
 
 case "$1" in
