--- /usr/lib/avr/include/avr/io.h.orig	2017-07-05 00:48:35.739212601 +0100
+++ /usr/lib/avr/include/avr/io.h	2017-07-05 00:47:21.191361651 +0100
@@ -270,6 +270,8 @@
 #  include <avr/iom3250pa.h>  
 #elif defined (__AVR_ATmega328P__)
 #  include <avr/iom328p.h>
+#elif (defined __AVR_ATmega328PB__)
+#  include <avr/iom328pb.h>
 #elif (defined __AVR_ATmega328__)
 #include <avr/iom328.h>
 #elif defined (__AVR_ATmega329__)
