diff -up ./src/mod_inputdev/inputdev_parallel/inputdev_parallel.cpp.org ./src/mod_inputdev/inputdev_parallel/inputdev_parallel.cpp
--- ./src/mod_inputdev/inputdev_parallel/inputdev_parallel.cpp.org	2016-09-29 17:51:32.786422248 +0200
+++ ./src/mod_inputdev/inputdev_parallel/inputdev_parallel.cpp	2016-09-29 18:04:22.030399857 +0200
@@ -28,7 +28,7 @@
 #if defined(__APPLE__) || defined(MACOSX)
 #else
 #  if defined(WIN32)
-#  elif defined(__powerpc__) || defined(__mips__) || defined(__mipsel__) ||\
+#  elif defined(__powerpc__) || defined(__mips__) || defined(__mipsel__) || defined(__hppa__) ||\
     defined(__s390__) || defined(__s390x__) || defined(__sparc__) || defined(__sh__) || defined(__aarch64__)
 #    define ioperm(a,b,c) -1
 #    define inb(a) 0
diff -up ./src/mod_inputdev/inputdev_rctran2/kernel_module/rctran2.c.org ./src/mod_inputdev/inputdev_rctran2/kernel_module/rctran2.c
