--- claws-mail-3.0.2-orig/src/compose.c 2007-11-01 03:27:03.000000000 -0300
+++ claws-mail-3.0.2/src/compose.c      2007-11-01 23:45:20.000000000 -0300
@@ -2366,18 +2366,21 @@
 {
        gchar *to = NULL;
        gchar *cc = NULL;
+       gchar *bcc = NULL;
        gchar *subject = NULL;
        gchar *body = NULL;
        gchar *temp = NULL;
        gsize  len = 0;
        gchar *attach = NULL;
 
-       scan_mailto_url(mailto, &to, &cc, NULL, &subject, &body, &attach);
+       scan_mailto_url(mailto, &to, &cc, &bcc, &subject, &body, &attach);
 
        if (to)
                compose_entry_append(compose, to, COMPOSE_TO);
        if (cc)
                compose_entry_append(compose, cc, COMPOSE_CC);
+       if (bcc)
+               compose_entry_append(compose, bcc, COMPOSE_BCC);
        if (subject) {
                if (!g_utf8_validate (subject, -1, NULL)) {
                        temp = g_locale_to_utf8 (subject, -1, NULL, &len, NULL);
