--- src/lib/agent-helper.c.orig	2014-08-03 18:39:59.000000000 -0700
+++ src/lib/agent-helper.c	2015-05-18 01:18:24.086901318 -0700
@@ -293,14 +293,25 @@
         g_object_unref(device_obj);
         
         /* Try to use found PIN */
 	if (pin != NULL)
         {
+            GVariant *pTemp = NULL, *pTemp2; // BBB - temporary fix
             if (_interactive)
                 g_print("Passkey found\n");
-            sscanf(pin, "%s", &ret);
-            g_dbus_method_invocation_return_value(invocation, g_variant_new_string(ret));
+// BBB - removed, since it page faults - no need to scan a string, right?
+//            sscanf(pin, "%s", &ret);
+
+            pTemp2 = g_variant_new_string(pin); // added since I need a tuple now
+            if(pTemp2)
+            {
+              pTemp = g_variant_new_tuple(&pTemp2, 1); // only one element
+            }
+            if(pTemp)
+            {
+              g_dbus_method_invocation_return_value(invocation, pTemp); // g_variant_new_string(ret));
+            }
             return;
 	}
         else if (_interactive)
         {
             g_print("Enter passkey: ");
