diff -u dns-browse-1.9/debian/control dns-browse-1.9/debian/control
--- dns-browse-1.9/debian/control
+++ dns-browse-1.9/debian/control
@@ -2,13 +2,13 @@
 Section: net
 Priority: extra
 Maintainer: Javier Fernandez-Sanguino Pen~a <jfs@debian.org>
-Build-Depends: debhelper (>= 3.0.18), autoconf, tk8.5 | wish, docbook-to-man
+Build-Depends: debhelper (>= 3.0.18), autoconf, tk, docbook-to-man
 Standards-Version: 3.7.2
 Homepage: http://www.isi.edu/~johnh/SOFTWARE/DNS/
 
 Package: dns-browse
 Architecture: all
-Depends: tk8.5 | wish, dnsutils, ${perl:Depends}
+Depends: tk, dnsutils, ${perl:Depends}
 Description: Front-ends to DNS search
  This package provides two programs to make user lookups on DNS
  servers: dns_tree and dns_browse.
diff -u dns-browse-1.9/debian/changelog dns-browse-1.9/debian/changelog
--- dns-browse-1.9/debian/changelog
+++ dns-browse-1.9/debian/changelog
@@ -1,3 +1,11 @@
+dns-browse (1.9-7.1) unstable; urgency=low
+
+  * Non-maintaner upload.
+  * Replaced build dependency on tk8.5 by tk to fix FTBFS after Tcl/Tk
+    packages droped alternatives for /usr/bin/wish.
+
+ -- Sergei Golovan <sgolovan@debian.org>  Mon, 30 Sep 2013 11:10:01 +0400
+
 dns-browse (1.9-7) unstable; urgency=low
 
   * Add docbook-to-man to the Build-Dependencies (Closes: #618248)
