--- apt-move.origin	2012-03-03 11:12:53.000000000 +0000
+++ apt-move	2012-03-03 11:13:11.000000000 +0000
@@ -1253,6 +1253,8 @@
 			$(md5sum $l) $size $j/$k/$l
 		printf ' %40s%.s %16d %s\n' \
 			$(sha1sum $l) $size $j/$k/$l >&3
+		printf ' %40s%.s %16d %s\n' \
+			$(sha256sum $l) $size $j/$k/$l >&4
 	done
 }
 
@@ -1312,7 +1314,7 @@
 		fi
 
 		cd $prev
-	done > $pf-md5sum 3> $pf-sha1sum
+	done > $pf-md5sum 3> $pf-sha1sum 4> $pf-sha256sum
 
 	[ -n "$compo" ] || return 0
 
@@ -1334,6 +1336,8 @@
 	cat $pf-md5sum
 	echo SHA1:
 	cat $pf-sha1sum
+	echo SHA256:
+	cat $pf-sha256sum
 
 	exec >&-
 
