--- ./debian/examples/acpi/resume.d/90-xscreensaver.sh.orig	2011-01-08 00:13:56.000000000 +0100
+++ ./debian/examples/acpi/resume.d/90-xscreensaver.sh	2011-01-08 01:21:31.000000000 +0100
@@ -2,8 +2,9 @@
 
 # now, we should poke xscreensaver so you get a dialog
 if pidof xscreensaver > /dev/null; then 
-    for x in /tmp/.X11-unix/*; do
-	displaynum=`echo $x | sed s#/tmp/.X11-unix/X##`
+    d=/tmp/.X11-unix
+    for x in $d/*; do
+	displaynum=${x#$d/X}
 	getXuser;
 	if [ x"$XAUTHORITY" != x"" ]; then
 	    export DISPLAY=":$displaynum"
--- ./debian/examples/acpi/suspendorhibernate.orig	2011-01-08 00:13:56.000000000 +0100
+++ ./debian/examples/acpi/suspendorhibernate	2011-01-08 01:19:14.000000000 +0100
@@ -170,8 +170,9 @@ for METHOD in $SUSPEND_METHODS; do
 
 			if [ x$LOCK_SCREEN = xtrue ]; then
 			    if pidof xscreensaver > /dev/null; then 
-				for x in /tmp/.X11-unix/*; do
-				    displaynum=`echo $x | sed s#/tmp/.X11-unix/X##`
+				d=/tmp/.X11-unix
+				for x in $d/*; do
+				    displaynum=${x#$d/X}
 				    getXuser;
 				    if [ x"$XAUTHORITY" != x"" ]; then	    
 					export DISPLAY=":$displaynum"
--- ./rotatescreen.sh.orig	2011-01-08 00:14:47.000000000 +0100
+++ ./rotatescreen.sh	2011-01-08 01:14:22.000000000 +0100
@@ -20,8 +20,9 @@ case "$ROTATION" in
 	;;
 esac
 
-for x in /tmp/.X11-unix/*; do
-	displaynum=`echo $x | sed s#/tmp/.X11-unix/X##`
+d=/tmp/.X11-unix
+for x in $d/*; do
+	displaynum=${x#$d/X}
 	getXconsole;
 	if [ x"$XAUTHORITY" != x"" ]; then
 	    export DISPLAY=":$displaynum"           
--- ./screenblank.sh.orig	2011-01-08 00:14:47.000000000 +0100
+++ ./screenblank.sh	2011-01-08 01:23:15.000000000 +0100
@@ -6,8 +6,9 @@ test -f /usr/share/acpi-support/key-cons
 . /usr/share/acpi-support/power-funcs
 . /etc/default/acpi-support
 
-for x in /tmp/.X11-unix/*; do
-    displaynum=`echo $x | sed s#/tmp/.X11-unix/X##`
+d=/tmp/.X11-unix
+for x in $d/*; do
+    displaynum=${x#$d/X}
     getXuser;
     if [ x"$XAUTHORITY" != x"" ]; then
         export DISPLAY=":$displaynum"
--- ./sleep.sh.orig	2011-01-08 00:14:47.000000000 +0100
+++ ./sleep.sh	2011-01-08 01:27:47.000000000 +0100
@@ -27,8 +27,9 @@ fi
 
 if [ x$LOCK_SCREEN = xtrue ]; then
     if pidof xscreensaver > /dev/null; then
-	for x in /tmp/.X11-unix/*; do
-	    displaynum=`echo $x | sed s#/tmp/.X11-unix/X##`
+	d=/tmp/.X11-unix
+	for x in $d/*; do
+	    displaynum=${x#$d/X}
 	    getXuser;
 	    if [ x"$XAUTHORITY" != x"" ]; then
 		export DISPLAY=":$displaynum"
