--- ostable	2015-11-26 23:53:38.000000000 +0000
+++ ostable	2016-02-15 11:40:09.000000000 +0000
@@ -37,3 +37,4 @@
 uclibceabi-uclinux	uclinux-uclibceabi	uclinux[^-]*-uclibceabi
 uclibc-uclinux		uclinux-uclibc		uclinux[^-]*(-uclibc.*)?
 tos-mint		mint			mint[^-]*
+cloudabi-none		cloudabi		cloudabi[^-]*
--- scripts/t/Dpkg_Arch.t	2015-12-25 03:53:50.000000000 +0000
+++ scripts/t/Dpkg_Arch.t	2016-02-09 15:46:22.000000000 +0000
@@ -98,7 +98,7 @@
 is(gnutriplet_to_debarch('unknown-unknown-unknown'), undef, 'unknown gnutriplet');
 is(gnutriplet_to_debarch('x86_64-linux-gnu'), 'amd64', 'known gnutriplet');
 
-is(scalar get_valid_arches(), 417, 'expected amount of known architectures');
+is(scalar get_valid_arches(), 446, 'expected amount of known architectures');
 
 {
     local $ENV{CC} = 'false';
--- triplettable	2015-11-26 23:53:41.000000000 +0000
+++ triplettable	2016-02-15 14:49:45.000000000 +0000
@@ -30,3 +30,4 @@
 uclibceabi-uclinux-arm	uclinux-armel
 uclibc-uclinux-<cpu>	uclinux-<cpu>
 tos-mint-m68k		mint-m68k
+cloudabi-none-<cpu>	cloudabi-none-<cpu>
