diff -Nru dolphin-emu-5.0-19870+dfsg/debian/changelog dolphin-emu-5.0-19870+dfsg/debian/changelog
--- dolphin-emu-5.0-19870+dfsg/debian/changelog	2023-09-06 19:42:46.000000000 +0200
+++ dolphin-emu-5.0-19870+dfsg/debian/changelog	2023-12-24 22:40:43.000000000 +0100
@@ -1,3 +1,10 @@
+dolphin-emu (5.0-19870+dfsg-1.1) UNRELEASED; urgency=medium
+
+  * Non-maintainer upload.
+  * Install udev rules file into /usr/lib. (Closes: #1057228)
+
+ -- Chris Hofstaedtler <zeha@debian.org>  Sun, 24 Dec 2023 22:40:43 +0100
+
 dolphin-emu (5.0-19870+dfsg-1) unstable; urgency=medium
 
   * New upstream beta release.
diff -Nru dolphin-emu-5.0-19870+dfsg/debian/dolphin-emu.install dolphin-emu-5.0-19870+dfsg/debian/dolphin-emu.install
--- dolphin-emu-5.0-19870+dfsg/debian/dolphin-emu.install	2023-09-06 14:13:20.000000000 +0200
+++ dolphin-emu-5.0-19870+dfsg/debian/dolphin-emu.install	2023-12-24 22:40:39.000000000 +0100
@@ -1,4 +1,4 @@
-Data/51-usb-device.rules /lib/udev/rules.d
+Data/51-usb-device.rules /usr/lib/udev/rules.d
 usr/games/dolphin-emu
 usr/games/dolphin-emu-nogui
 usr/games/dolphin-tool
