diff -pur calibre-0.9.31+dfsg/debian/local/calibre.desktop calibre-0.9.31+dfsg.modified/debian/local/calibre.desktop
--- calibre-0.9.31+dfsg/debian/local/calibre.desktop	2013-05-21 14:56:12.000000000 +0800
+++ calibre-0.9.31+dfsg.modified/debian/local/calibre.desktop	2013-07-07 16:45:51.404192906 +0800
@@ -4,6 +4,7 @@ Name=calibre
 Comment=E-book library management
 Comment[es]=aplicación para la gestión de libros electrónicos
 Comment[de]=E-Book Bibliotheksverwaltung
-Exec=calibre
+Exec=calibre %f
 Icon=calibre
 Categories=Office;Database;FileTools;Viewer;Qt;
+MimeType=x-content/ebook-reader;
