diff -ru dvdbackup.orig/src/dvdbackup.c dvdbackup/src/dvdbackup.c
--- dvdbackup.orig/src/dvdbackup.c	2009-02-20 00:00:52.000000000 +0100
+++ dvdbackup/src/dvdbackup.c	2009-05-25 12:07:17.000000000 +0200
@@ -284,7 +284,7 @@
 			to_read = left;
 		}
 		if ((rbuff = DVDReadBlocks(dvd_file,soffset, to_read, buffer)) < 0) {
-			fprintf(stderr, _("Error reading MENU VOB %d != %d\n"), rbuff, to_read);
+			fprintf(stderr, _("Error reading MENU VOB: %d != %d\n"), rbuff, to_read);
 			free(buffer);
 			DVDCloseFile(dvd_file);
 			close(streamout);
@@ -971,7 +971,7 @@
 
 
 	if (stat(targetname, &fileinfo) == 0) {
-		fprintf(stderr, _("The Title file %s exists will try to over write it.\n"), targetname);
+		fprintf(stderr, _("The Title file %s exists, will try to over write it.\n"), targetname);
 		if (! S_ISREG(fileinfo.st_mode)) {
 			fprintf(stderr,_("The Title %s file is not valid, it may be a directory\n"), targetname);
 			return(1);
