diff -ru dvdbackup.orig/src/main.c dvdbackup/src/main.c
--- dvdbackup.orig/src/main.c	2009-02-20 00:00:52.000000000 +0100
+++ dvdbackup/src/main.c	2009-05-25 12:14:09.000000000 +0200
@@ -403,7 +403,7 @@
 
 	if (stat(targetname, &fileinfo) == 0) {
 		if (! S_ISDIR(fileinfo.st_mode)) {
-			fprintf(stderr,_("The target directory is not valid; it may be a ordinary file\n"));
+			fprintf(stderr,_("The target directory is not valid; it may be an ordinary file\n"));
 		}
 	} else {
 		if (mkdir(targetname, 0777) != 0) {
@@ -419,7 +419,7 @@
 
 	if (stat(targetname, &fileinfo) == 0) {
 		if (! S_ISDIR(fileinfo.st_mode)) {
-			fprintf(stderr,_("The title directory is not valid; it may be a ordinary file\n"));
+			fprintf(stderr,_("The title directory is not valid; it may be an ordinary file\n"));
 		}
 	} else {
 		if (mkdir(targetname, 0777) != 0) {
@@ -434,7 +434,7 @@
 
 	if (stat(targetname, &fileinfo) == 0) {
 		if (! S_ISDIR(fileinfo.st_mode)) {
-			fprintf(stderr,_("The VIDEO_TS directory is not valid; it may be a ordinary file\n"));
+			fprintf(stderr,_("The VIDEO_TS directory is not valid; it may be an ordinary file\n"));
 		}
 	} else {
 		if (mkdir(targetname, 0777) != 0) {
