--- /usr/bin/dput~	2006-12-14 18:20:29.000000000 +0100
+++ /usr/bin/dput	2007-05-31 23:25:10.000000000 +0200
@@ -303,7 +303,7 @@
 
     # Run the check to verify that the package has been tested.
     if config.getboolean(host, 'check_version') == 1 or check_version:
-        version_check (changes, debug)
+        version_check (path, changes, debug)
 
     # Verify the signature of the maintainer
     verify_signature(host, change_file, dsc_file, check_only, debug,\
@@ -474,7 +474,7 @@
 
 # Check if the caller has installed the package also on his system
 # for testing purposes before uploading it. If not, we reject the upload.
-def version_check(changes, debug):
+def version_check(path, changes, debug):
     files_to_check = []
 
     # Get arch
@@ -491,8 +491,8 @@
     
     # Get filenames of deb files:
     for file in changes.dict['files'].split('\n'):
-        filename = string.split(file)[4] # filename only
-        if filename.find('.deb') != -1:
+        filename = os.path.join(path, string.split(file)[4])
+        if filename.endswith('.deb'):
             if debug:
                 print "D: Debian Package: %s" % filename
             dpkg_stdin,dpkg_stdout,dpkg_stderr = os.popen3('dpkg --field %s' % filename)
