Index: rapt-file.wrapper
===================================================================
--- rapt-file.wrapper	(revision 18324)
+++ rapt-file.wrapper	(working copy)
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-if [ -n "$(ls /usr/lib/python*/site-packages/apt/__init__.py 2> /dev/null)" ] ; then
+if [ -n "$(ls /usr/lib/python2.*/*-packages/apt/__init__.py 2> /dev/null)" ] ; then
 	exec /usr/share/apt-file/rapt-file "$@"
 else
 	echo You need to install python-apt to use rapt-file >&2
