diff --git a/debmirror b/debmirror
index 7e9e669..26bea48 100755
--- a/debmirror
+++ b/debmirror
@@ -629,6 +629,7 @@ our $doing_meta=1;
 our $ignore_missing_release=0;
 our $ignore_release_gpg=0;
 our $start_time = time;
+our $start_date =`LC_ALL=C date -R`;
 our $dry_run=0;
 our $do_dry_run=0;
 our $rsync_options="-aIL --partial";
@@ -1566,6 +1567,13 @@ if (! $do_dry_run) {
   make_dir("project/trace");
   open OUT, ">$tracefile" or die "$tracefile: $!";
   print OUT `LC_ALL=C date -u`;
+  print OUT "Date: ".`LC_ALL=C date -R`;
+  print OUT "Date-Started: $start_date";
+  print OUT "Used debmirror version: $version\n";
+  print OUT "Running on host: $hostname\n";
+  print OUT "Architectures: ".join(" ", @arches)."\n";
+  print OUT "Distributions: ".join(" ", @dists)."\n";
+  print OUT "Upstream-mirror: $host\n";
   close OUT;
 }
 


