diff -u audacity-1.3.2/debian/rules audacity-1.3.2/debian/rules
--- audacity-1.3.2/debian/rules
+++ audacity-1.3.2/debian/rules
@@ -5,6 +5,7 @@
 include /usr/share/cdbs/1/rules/simple-patchsys.mk
 include /usr/share/cdbs/1/rules/utils.mk
 
+ifeq (i386,$(DEB_HOST_ARCH))
 DEB_CONFIGURE_EXTRA_FLAGS := --with-libsndfile=system \
                              --with-vorbis=system \
 			     --with-id3tag=system \
@@ -13,6 +14,28 @@
 			     --with-expat=system \
 			     --with-portaudio=v19 \
 			     --enable-unicode
+else
+ifeq (powerpc,$(DEB_HOST_ARCH))
+DEB_CONFIGURE_EXTRA_FLAGS := --with-libsndfile=system \
+                             --with-vorbis=system \
+			     --with-id3tag=system \
+			     --with-libmad=system \
+			     --with-libflac=system \
+			     --with-expat=system \
+			     --with-portaudio=v19 \
+			     --enable-unicode
+else
+DEB_CONFIGURE_EXTRA_FLAGS := --with-libsndfile=system \
+                             --with-vorbis=system \
+			     --with-id3tag=system \
+			     --with-libmad=system \
+			     --with-libflac=system \
+			     --with-expat=system \
+			     --with-portaudio=v19 \
+			     --with-jack=no \
+			     --enable-unicode
+endif
+endif
 
 DEB_COMPRESS_EXCLUDE := *.htb
 
