From c5848db4f726cbf1b446fd19eaa835f6224a9326 Mon Sep 17 00:00:00 2001
From: Shirish Agarwal <shirishag75@gmail.com>
Date: Thu, 5 Jan 2017 23:37:05 +0530
Subject: [PATCH] debian-policy should ask for/have an Example Section in a manpage, if a manpage exists. 

---
 policy.sgml | 9 ++++++++-
 1 file changed, 8 insertions(+), 1 deletion(-)

diff --git a/policy.sgml b/policy.sgml
index 06d094c..a55c4ea 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -5777,7 +5777,7 @@ Built-Using: grub2 (= 1.99-9), loadlin (= 1.6e-1)
 
 	<p>
 	  Every time the shared library ABI changes in a way that may
-	  break binaries linked against older versions of the shared
+q	  break binaries linked against older versions of the shared
 	  library, the <tt>SONAME</tt> of the library and the
 	  corresponding name for the binary package containing the runtime
 	  shared library should change.  Normally, this means
@@ -10660,6 +10660,13 @@ name ["<var>syshostname</var>"]:
 	  auxiliary things are optional.
 	</p>
 
+        <p> 
+	  It is also suggested that all packages should also have an
+	  EXAMPLE section which describes with syntax how the package
+	  should be used. The Example section could be optional for 
+	  library packages.  
+	  
+	</p>  
 	<p>
           If no manual page is available, this is considered as a bug
           and should be reported to the Debian Bug Tracking System (the
-- 
2.11.0

