diff --git a/test/test_uscan b/test/test_uscan
index aabdadd7..299db0fb 100755
--- a/test/test_uscan
+++ b/test/test_uscan
@@ -50,6 +50,7 @@ cleanup(){
 }
 
 spawnHttpServer(){
+    unset http_proxy
     (
 	mkdir -p $TMPDIR/repo
 	cd $TMPDIR/repo
diff --git a/test/test_uscan_mangle b/test/test_uscan_mangle
index 496038cb..b80308a0 100755
--- a/test/test_uscan_mangle
+++ b/test/test_uscan_mangle
@@ -89,6 +89,7 @@ cleanup(){
 }
 
 spawnHttpServer(){
+    unset http_proxy
     (
 	USCAN_HTTP_SERVER=${USCAN_HTTP_SERVER:-server.py}
 	mkdir -p $TMPDIR/$REPOPATH
