diff --git a/policy.sgml b/policy.sgml
index deae2dd..f6d1aac 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -5674,7 +5674,6 @@ if [ ! -e /usr/local/share/emacs ]
 then
   if mkdir /usr/local/share/emacs 2>/dev/null
   then
-    chown root:staff /usr/local/share/emacs
     chmod 2775 /usr/local/share/emacs
   fi
 fi
@@ -5708,7 +5707,7 @@ rmdir /usr/local/share/emacs 2>/dev/null || true
 	    The <file>/usr/local</file> directory itself and all the
 	    subdirectories created by the package should (by default) have
 	    permissions 2775 (group-writable and set-group-id) and be
-	    owned by <tt>root:staff</tt>.
+	    owned by <tt>root:root</tt>.
 	  </p>
 	</sect1>
 
