diff -Nura crack-attack-1.1.14/debian/crack-attack.desktop crack-attack-1.1.14.mio/debian/crack-attack.desktop
--- crack-attack-1.1.14/debian/crack-attack.desktop	1969-12-31 21:00:00.000000000 -0300
+++ crack-attack-1.1.14.mio/debian/crack-attack.desktop	2006-08-03 11:44:05.000000000 -0300
@@ -0,0 +1,9 @@
+[Desktop Entry]
+Version=1.0
+Encoding=UTF-8
+Type=Application
+Name=Crack Attack
+Comment=puzzle game similar to the game "Tetris Attack"
+Exec=crack-attack
+Icon=crack-attack.xpm
+Categories=Application;Game;BlocksGame;
diff -Nura crack-attack-1.1.14/debian/rules crack-attack-1.1.14.mio/debian/rules
--- crack-attack-1.1.14/debian/rules	2006-08-03 11:45:23.000000000 -0300
+++ crack-attack-1.1.14.mio/debian/rules	2006-08-03 11:44:05.000000000 -0300
@@ -20,4 +20,6 @@
 
 binary-post-install/crack-attack::
 	mkdir -p debian/crack-attack/usr/share/pixmaps/
+	mkdir -p debian/crack-attack/usr/share/applications/
 	install -m0644 debian/crack-attack.xpm debian/crack-attack/usr/share/pixmaps/
+	install -m0644 debian/crack-attack.desktop debian/crack-attack/usr/share/applications/
