diff -Naur boinc-5.10.8.orig/client/cs_platforms.C boinc-5.10.8/client/cs_platforms.C
--- boinc-5.10.8.orig/client/cs_platforms.C	2007-05-07 21:38:27.000000000 +0200
+++ boinc-5.10.8/client/cs_platforms.C	2007-10-11 13:00:17.000000000 +0200
@@ -93,6 +93,8 @@
 
 #endif
 
+    if (strlen(config.alt_platform)>0) add_platform(config.alt_platform);
+
     if (config.no_alt_platform) {
         PLATFORM p = platforms[0];
         platforms.clear();
diff -Naur boinc-5.10.8.orig/client/log_flags.C boinc-5.10.8/client/log_flags.C
--- boinc-5.10.8.orig/client/log_flags.C	2007-06-11 23:30:26.000000000 +0200
+++ boinc-5.10.8/client/log_flags.C	2007-10-11 13:05:37.000000000 +0200
@@ -216,6 +216,7 @@
         if (xp.parse_bool(tag, "disallow_attach", disallow_attach)) continue;
         if (xp.parse_bool(tag, "os_random_only", os_random_only)) continue;
         if (xp.parse_bool(tag, "no_alt_platform", no_alt_platform)) continue;
+        if (xp.parse_str(tag, "alt_platform", alt_platform, sizeof(alt_platform))) continue;
         msg_printf(NULL, MSG_USER_ERROR, "Unparsed tag in %s: <%s>\n",
             CONFIG_FILE, tag
         );
diff -Naur boinc-5.10.8.orig/client/log_flags.h boinc-5.10.8/client/log_flags.h
--- boinc-5.10.8.orig/client/log_flags.h	2007-05-07 21:38:27.000000000 +0200
+++ boinc-5.10.8/client/log_flags.h	2007-10-11 13:02:23.000000000 +0200
@@ -91,6 +91,7 @@
     bool disallow_attach;
     bool os_random_only;
     bool no_alt_platform;
+    char alt_platform[256];
 
     CONFIG();
     void defaults();
