From 832804f583e596c06ea94c4226ec5ade899a7bf8 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Sat, 18 Feb 2017 01:51:11 +0100
Subject: [PATCH] wrap-and-sort: Add missing build dependency fields

---
 scripts/wrap-and-sort | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/scripts/wrap-and-sort b/scripts/wrap-and-sort
index 666c732b..4d1d24de 100755
--- a/scripts/wrap-and-sort
+++ b/scripts/wrap-and-sort
@@ -27,8 +27,12 @@ from devscripts.control import Control
 CONTROL_LIST_FIELDS = (
     "Breaks",
     "Build-Conflicts",
+    "Build-Conflicts-Arch",
+    "Build-Conflicts-Indep",
     "Build-Depends",
+    "Build-Depends-Arch",
     "Build-Depends-Indep",
+    "Built-Using",
     "Conflicts",
     "Depends",
     "Enhances",
-- 
2.11.0.483.g087da7b7c

