=== modified file 'templates/en_US/cgi/bugreport.tmpl'
--- templates/en_US/cgi/bugreport.tmpl	2008-09-19 00:21:05 +0000
+++ templates/en_US/cgi/bugreport.tmpl	2008-11-28 16:30:23 +0000
@@ -27,7 +27,7 @@
      $output .= sprintf qq(<p><a href="%s">Full log</a></p>),html_escape(bug_links(bug=>$ref,links_only=>1));
   }
   else {
-     $output .=  qq(<p><a href="mailto:$bug_num\@$config{email_domain}">Reply</a> ).
+     $output .=  qq(<p><a href="mailto:$bug_num\@$config{email_domain}?subject={html_escape($status{subject})}">Reply</a> ).
 	  qq(or <a href="mailto:$bug_num-subscribe\@$config{email_domain}">subscribe</a> ).
 	       qq(to this bug.</p>\n);
      $output .=  qq(<p><a href="javascript:toggle_infmessages();">Toggle useless messages</a></p>);

