commit eda17e4a5809fead37ebf9ed7badec546ca340c1
Author: Bernhard Schmidt <Bernhard.Schmidt@lrz.de>
Date:   Wed Jun 20 13:40:56 2012 +0200

    fix website for Sophie and SAVI

diff --git a/debian/etc/conf.d/15-av_scanners b/debian/etc/conf.d/15-av_scanners
index 59ac2f3..d9c39c6 100644
--- a/debian/etc/conf.d/15-av_scanners
+++ b/debian/etc/conf.d/15-av_scanners
@@ -6,13 +6,13 @@ use strict;
 
 @av_scanners = (
 
-# ### http://www.clanfield.info/sophie/ (http://www.vanja.com/tools/sophie/)
+# ### http://sophie.sourceforge.net/
 # ['Sophie',
 #   \&ask_daemon, ["{}/\n", '/var/run/sophie'],
 #   qr/(?x)^ 0+ ( : | [\000\r\n]* $)/m,  qr/(?x)^ 1 ( : | [\000\r\n]* $)/m,
 #   qr/(?x)^ [-+]? \d+ : (.*?) [\000\r\n]* $/m ],
 
-# ### http://www.csupomona.edu/~henson/www/projects/SAVI-Perl/
+# ### http://search.cpan.org/~phenson/SAVI-Perl/
 # ['Sophos SAVI', \&sophos_savi ],
 
 ### http://www.clamav.net/
