From ca8939041f868c9e094dccff70b4b59a0ebd7c0d Mon Sep 17 00:00:00 2001
From: Dominic Hargreaves <dom@earth.li>
Date: Thu, 17 Dec 2015 10:26:31 +0000
Subject: [PATCH] Fix outdated dependency on perl5

---
 debian/control | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/debian/control b/debian/control
index 80ebf15..dc8e0e0 100644
--- a/debian/control
+++ b/debian/control
@@ -34,7 +34,6 @@ Depends: adduser,
          bsd-mailx | mailx,
          debconf | debconf-2.0,
          openbsd-inetd | inet-superserver,
-         perl5,
          tar (>=1.12),
          update-inetd,
          ${misc:Depends},
@@ -57,7 +56,7 @@ Depends: amanda-common (= ${binary:Version}),
 Suggests: amanda-client (= ${binary:Version}),
           cpio | mt-st,
           gnuplot,
-          perl5,
+          perl,
 Conflicts: amanda,
            amanda-common (<< 1:2.5.2p1-3),
 Replaces: amanda-common (<< 1:2.5.2p1-3),
-- 
2.1.4

