--- logger.c	2013-04-11 13:30:48.000000000 +0200
+++ logger.c.fixed	2013-04-11 13:30:48.000000000 +0200
@@ -281,7 +281,7 @@
 			}
 		}
 		if (p != buf) {
-		    if (!usock)
+		    if (!usock && !udpserver)
 			syslog(pri, "%s", buf);
 		    else
 			mysyslog(LogSock, logflags, pri, tag, buf);
@@ -295,7 +295,7 @@
 		    if (len > 0 && buf[len - 1] == '\n')
 			    buf[len - 1] = '\0';
 
-		    if (!usock)
+		    if (!usock && !udpserver)
 			syslog(pri, "%s", buf);
 		    else
 			mysyslog(LogSock, logflags, pri, tag, buf);
