--- debuerreotype-0.15.orig/scripts/debuerreotype-minimizing-config
+++ debuerreotype-0.15/scripts/debuerreotype-minimizing-config
@@ -36,13 +36,6 @@ cat > "$targetDir/usr/sbin/policy-rc.d"
 EOF
 chmod 0755 "$targetDir/usr/sbin/policy-rc.d"
 
-# prevent upstart scripts from running during install/update (but only if the "upstart" package exists)
-if "$thisDir/debuerreotype-chroot" "$targetDir" apt-get install -qq -s upstart &> /dev/null; then
-	"$thisDir/debuerreotype-chroot" "$targetDir" dpkg-divert --local --rename --add /sbin/initctl > /dev/null
-	cp -a "$targetDir/usr/sbin/policy-rc.d" "$targetDir/sbin/initctl"
-	sed -i 's/^exit.*/exit 0/' "$targetDir/sbin/initctl"
-fi
-
 # force dpkg not to call sync() after package extraction (speeding up installs)
 if [ -d "$targetDir/etc/dpkg/dpkg.cfg.d" ] && dpkg --compare-versions "$dpkgVersion" '>=' '1.15.8.6~'; then
 	# --debian-eol lenny and older do not include /etc/dpkg/dpkg.cfg.d
