From f46b17b67d171bc6cbe4af24ac87b89b81ba4e94 Mon Sep 17 00:00:00 2001
From: Daniel Baumann <daniel@debian.org>
Date: Wed, 2 Feb 2011 14:14:46 +0100
Subject: [PATCH] Adding priority field for source package in control field (Closes: #611805).

---
 debian/control |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/debian/control b/debian/control
index 1e09434..7a8a4ef 100644
--- a/debian/control
+++ b/debian/control
@@ -1,4 +1,5 @@
 Source: debian-archive-keyring
+Priority: optional
 Section: misc
 Maintainer: Debian Release Team <packages@release.debian.org>
 Build-Depends: debhelper (>= 5), jetring, gnupg
-- 
1.7.2.3

