diff -Nru debhelper-9.20120410/Debian/Debhelper/Buildsystem/cmake.pm debhelper-9.20120410.1~hardening1/Debian/Debhelper/Buildsystem/cmake.pm
--- debhelper-9.20120410/Debian/Debhelper/Buildsystem/cmake.pm	2011-10-20 02:02:57.000000000 +0200
+++ debhelper-9.20120410.1~hardening1/Debian/Debhelper/Buildsystem/cmake.pm	2012-04-14 16:58:24.000000000 +0200
@@ -43,6 +43,12 @@
 	push @flags, "-DCMAKE_INSTALL_PREFIX=/usr";
 	push @flags, "-DCMAKE_VERBOSE_MAKEFILE=ON";
 
+	# CMake doesn't respect CPPFLAGS, see #653916.
+	if ($ENV{CPPFLAGS}) {
+		$ENV{CFLAGS}   .= ' ' . $ENV{CPPFLAGS};
+		$ENV{CXXFLAGS} .= ' ' . $ENV{CPPFLAGS};
+	}
+
 	$this->mkdir_builddir();
 	eval { 
 		$this->doit_in_builddir("cmake", $this->get_source_rel2builddir(), @flags, @_);
