diff -ur --exclude=debian evince-2.20.2-orig/data/evince.schemas.in evince-2.20.2/data/evince.schemas.in
--- evince-2.20.2-orig/data/evince.schemas.in	2007-11-26 15:55:24.000000000 -0500
+++ evince-2.20.2/data/evince.schemas.in	2008-05-25 03:18:32.000000000 -0400
@@ -5,7 +5,7 @@
       <applyto>/apps/evince/override_restrictions</applyto>
       <owner>evince</owner>
       <type>bool</type>
-      <default>false</default>
+      <default>true</default>
       <locale name="C">
         <short>Override document restrictions</short>
         <long>Override document restrictions, like restriction to copy or to print.</long>
