--- etc/conf.d/01-debian.orig	2015-09-23 16:09:48.210304984 -0400
+++ etc/conf.d/01-debian	2015-09-23 16:11:25.437966482 -0400
@@ -28,7 +28,7 @@
 $lzop   = 'lzop';
 $rpm2cpio   = ['rpm2cpio.pl','rpm2cpio'];
 $cabextract = 'cabextract';
-$uncompress = ['uncompress', 'gzip -d', 'zcat'];
+$uncompress = ['gzip -d', 'uncompress', 'zcat'];
 #$unfreeze   = ['unfreeze', 'freeze -d', 'melt', 'fcat']; #disabled (non-free, no security support)
 $unfreeze = undef; 
 $arc        = ['nomarch', 'arc'];
