diff -u d-feet-0.1.10/debian/control d-feet-0.1.10/debian/control
--- d-feet-0.1.10/debian/control
+++ d-feet-0.1.10/debian/control
@@ -21,6 +21,7 @@
          python-dbus (>= 0.82.3),
          python-gtk2,
          python-glade2,
+         hicolor-icon-theme
 Recommends: python-wnck
 XB-Python-Version: ${python:Versions}
 Description: A D-Bus object browser, viewer and debugger
diff -u d-feet-0.1.10/debian/changelog d-feet-0.1.10/debian/changelog
--- d-feet-0.1.10/debian/changelog
+++ d-feet-0.1.10/debian/changelog
@@ -1,3 +1,10 @@
+d-feet (0.1.10-1.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Add missing dependency on hicolor-icon-theme (Closes: #546764)
+
+ -- Stefano Zacchiroli <zack@debian.org>  Mon, 08 Feb 2010 13:30:09 +0100
+
 d-feet (0.1.10-1) unstable; urgency=low
 
   * New upstream release.
