diff -Naur cdebootstrap-0.4.3/helper/cdebootstrap-helper-diverts/sbin/start-stop-daemon cdebootstrap-0.4.3.improved/helper/cdebootstrap-helper-diverts/sbin/start-stop-daemon
--- cdebootstrap-0.4.3/helper/cdebootstrap-helper-diverts/sbin/start-stop-daemon	2004-09-19 07:24:35.000000000 -0300
+++ cdebootstrap-0.4.3.improved/helper/cdebootstrap-helper-diverts/sbin/start-stop-daemon	2007-06-15 15:58:14.000000000 -0300
@@ -1 +1,4 @@
 #!/bin/sh
+echo "*****"
+echo "/sbin/start-stop-daemon has no effect in cdebootstrap-helper-diverts"
+echo "*****"
