diff -ur ../tmp/cdbs-0.4.32/1/rules/debhelper.mk.in ./1/rules/debhelper.mk.in
--- ../tmp/cdbs-0.4.32/1/rules/debhelper.mk.in	2005-02-03 10:14:53.000000000 +0100
+++ ./1/rules/debhelper.mk.in	2005-12-08 11:28:29.000000000 +0100
@@ -213,7 +213,7 @@
 	dh_installdeb -p$(cdbs_curpkg) $(DEB_DH_INSTALLDEB_ARGS)
 	dh_perl -p$(cdbs_curpkg) $(DEB_DH_PERL_ARGS)
 	dh_shlibdeps -p$(cdbs_curpkg) $(DEB_DH_SHLIBDEPS_ARGS)
-	# udebs should only have a 'control' file
+# udebs should only have a 'control' file
 	rm -f debian/$(cdbs_curpkg)/DEBIAN/*
 	dh_gencontrol -p$(cdbs_curpkg) -- -fdebian/files~
 	dpkg-distaddfile $(cdbs_curpkg)_$(DEB_NOEPOCH_VERSION)_$(DEB_ARCH).udeb debian-installer optional
diff -ur ../tmp/cdbs-0.4.32/1/rules/dpatch.mk.in ./1/rules/dpatch.mk.in
--- ../tmp/cdbs-0.4.32/1/rules/dpatch.mk.in	2005-05-18 11:50:40.000000000 +0200
+++ ./1/rules/dpatch.mk.in	2005-12-08 11:26:14.000000000 +0100
@@ -74,10 +74,10 @@
 deapply-dpatches:
 	$(MAKE) -f debian/rules reverse-config
 	dpatch deapply-all
-	# after reverting patches, we don't need update-config because:
-	#  - the tree is already in a consistent state.
-	#  - the initialisation of this routine is already in charge of making
-	#    the tree not only consistent, but also prepared to accept patching.
+# after reverting patches, we don't need update-config because:
+#  - the tree is already in a consistent state.
+#  - the initialisation of this routine is already in charge of making
+#    the tree not only consistent, but also prepared to accept patching.
 
 # The patch subsystem 
 apply-dpatches: pre-build
diff -ur ../tmp/cdbs-0.4.32/1/rules/simple-patchsys.mk.in ./1/rules/simple-patchsys.mk.in
--- ../tmp/cdbs-0.4.32/1/rules/simple-patchsys.mk.in	2005-05-18 12:09:42.000000000 +0200
+++ ./1/rules/simple-patchsys.mk.in	2005-12-08 11:27:22.000000000 +0100
@@ -141,10 +141,10 @@
 	elif [ "$@" = "reverse-patches" ] ; then rm -f debian/stamp-patch* ; \
 	fi
 
-	# after reverting patches, we don't need update-config because:
-	#  - the tree is already in a consistent state.
-	#  - the initialisation of this routine is already in charge of making
-	#    the tree not only consistent, but also prepared to accept patching.
+# after reverting patches, we don't need update-config because:
+#  - the tree is already in a consistent state.
+#  - the initialisation of this routine is already in charge of making
+#    the tree not only consistent, but also prepared to accept patching.
 	if [ "$@" != "reverse-patches" ] ; then \
 		$(MAKE) -f debian/rules update-config ; \
 	fi
