--- /usr/lib/ruby/vendor_ruby/debian.rb.orig	2014-12-18 19:01:03.233496178 -0100
+++ /usr/lib/ruby/vendor_ruby/debian.rb.debug	2014-12-18 19:00:26.229041877 -0100
@@ -911,7 +911,14 @@
       @provides = {}
       @file = [file]
       @lists = Archives.parseArchiveFile(file) {|info|
-        info =~ /Package:\s(.*)$/;
+	begin
+          info =~ /Package:\s(.*)$/;
+        rescue => e
+          puts "Error parsing file #{file}"
+          puts "Contents of info:"
+          puts info
+          raise e
+        end
 	if pkgs.empty? || pkgs.include?($1)
 	  d = Deb.new(info,fields)
 	  add_provides(d)
