From 512bca9ac18d8bd4e07eba0b02463369d40420a0 Mon Sep 17 00:00:00 2001
From: Charles Plessy <plessy@debian.org>
Date: Wed, 27 Aug 2014 06:37:41 +0900
Subject: [PATCH] =?UTF-8?q?Document=20the=20=E2=80=98nodoc=E2=80=99=20buil?=
 =?UTF-8?q?d=20option.?=
MIME-Version: 1.0
Content-Type: text/plain; charset=UTF-8
Content-Transfer-Encoding: 8bit

Closes: #759186
---
 policy.sgml | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/policy.sgml b/policy.sgml
index 6eac491..62d2bdf 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -2256,6 +2256,11 @@ zope.
 		  This tag says to not run any build-time test suite
 		  provided by the package.
 	      </item>
+	      <tag>nodoc</tag>
+	      <item>
+		  Do not build the documentation and do not build the binary
+		  packages containing the generated documentation.
+	      </item>
 	      <tag>noopt</tag>
 	      <item>
 		  The presence of this tag means that the package should
-- 
2.0.1

