From 32a66dd450192ac959b1ec2e6a7030288e9e6f6e Mon Sep 17 00:00:00 2001
From: Sven Joachim <svenjoac@gmx.de>
Date: Sun, 4 Sep 2016 18:24:17 +0200
Subject: [PATCH] pkgdetails_perl: Strip the arch-qualifier (Closes: #836525)

---
 functions | 1 +
 1 file changed, 1 insertion(+)

diff --git a/functions b/functions
index 031721f..1254c41 100644
--- a/functions
+++ b/functions
@@ -1229,6 +1229,7 @@ while (<STDIN>) {
 		for $d (split /\s*,\s*/, $1) {
 			$d =~ s/\s*[|].*$//;
 			$d =~ s/\s*[(].*[)]\s*//;
+			$d =~ s/:.*//;
 			push @d, $d;
 		}
 	}
-- 
2.9.3

