diff --git a/Debian/Debhelper/Buildsystem/cmake.pm b/Debian/Debhelper/Buildsystem/cmake.pm
index 338f65e..e9a6bbb 100644
--- a/Debian/Debhelper/Buildsystem/cmake.pm
+++ b/Debian/Debhelper/Buildsystem/cmake.pm
@@ -21,7 +21,7 @@ my @STANDARD_CMAKE_FLAGS = qw(
 
 my %DEB_HOST2CMAKE_SYSTEM = (
 	'linux'    => 'Linux',
-	'kfreebsd' => 'FreeBSD',
+	'kfreebsd' => 'kFreeBSD',
 	'hurd'     => 'GNU',
 );
 
diff --git a/debian/changelog b/debian/changelog
index 2078897..d96cd5a 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -11,6 +11,10 @@ debhelper (10.2.6) UNRELEASED; urgency=medium
     of the order of the fields in a given paragraph of debian/control.
     (Closes: #847138)
 
+  [ Steven Chamberlain ]
+  * When cross-compiling for GNU/kFreeBSD host systems, set
+    CMAKE_SYSTEM_NAME to the correct value "kFreeBSD", not "FreeBSD".
+
  -- Helmut Grohne <helmut@subdivi.de>  Sat, 18 Feb 2017 17:49:08 +0000
 
 debhelper (10.2.5) unstable; urgency=medium
