diff -uBbwr emacsen-common-1.4.15.orig/debian-startup.el emacsen-common-1.4.15/debian-startup.el
--- emacsen-common-1.4.15.orig/debian-startup.el	2002-02-18 11:12:15.000000000 -0600
+++ emacsen-common-1.4.15/debian-startup.el	2005-01-22 00:35:34.000000000 -0600
@@ -41,7 +41,9 @@
     
     ;; Find the last /usr/local/ element.
     (while (not (null lp-rest))
-      (if (string-match "^/usr/local" (car lp-rest))
+      (if (and
+           (not (null (car lp-rest)))
+           (string-match "^/usr/local" (car lp-rest)))
           (setq last-local-pos pos))
       (setq pos (+ pos 1))
       (setq lp-rest (cdr lp-rest)))
