--- dkms.orig	2016-10-13 12:02:58.503846710 +0200
+++ dkms	2016-10-13 12:06:10.910932590 +0200
@@ -1244,7 +1244,7 @@
 
     # Error out if build_exclude is set
     [[ $build_exclude ]] && die 9 \
-        $" The dkms.conf for this module includes a BUILD_EXCLUSIVE directive which" \
+        $" The $base_dir/dkms.conf for module $module includes a BUILD_EXCLUSIVE directive which" \
         $"does not match this kernel/arch.  This indicates that it should not be built."
 
     # Error out if source_tree is basically empty (binary-only dkms tarball w/ --force check)
