diff -u deluge-1.0.7.dfsg/debian/control deluge-1.0.7.dfsg/debian/control
--- deluge-1.0.7.dfsg/debian/control
+++ deluge-1.0.7.dfsg/debian/control
@@ -17,8 +17,8 @@
          python-xdg,
          python-gobject,
          python-pkg-resources
-Conflicts: deluge-torrent-common
-Replaces: deluge-torrent-common
+Conflicts: deluge-torrent-common (<< 1.0.7.dfsg-1)
+Replaces: deluge-torrent-common (<< 1.0.7.dfsg-1)
 Description: bittorrent client written in Python/PyGTK (daemon)
  Deluge is a bittorrent client that utilizes a daemon/client model.
  .
@@ -99,0 +100,10 @@
+
+Package: deluge-torrent
+Architecture: all
+Depends: ${misc:Depends}, deluge
+Description: bittorrent client written in Python/PyGTK (transitional package)
+ Deluge is a bittorrent client that utilizes a daemon/client model.
+ .
+ This is a transitional dummy package to ensure clean upgrades from old
+ releases (the package deluge-torrent is replaced by deluge). If nothing
+ depends on it, this package can be safely removed after upgrade.
