From 81c836c859fc4e5dfece7f2015cc5bb94fcfd128 Mon Sep 17 00:00:00 2001
From: Daniel Hartwig <mandyke@gmail.com>
Date: Sun, 12 Feb 2012 00:36:37 +0800
Subject: [PATCH] Silence text progress when -q2

---
 src/cmdline/text_progress.cc |    6 +++++-
 1 files changed, 5 insertions(+), 1 deletions(-)

diff --git a/src/cmdline/text_progress.cc b/src/cmdline/text_progress.cc
index 05b8855..da9c2db 100644
--- a/src/cmdline/text_progress.cc
+++ b/src/cmdline/text_progress.cc
@@ -137,9 +137,13 @@ namespace aptitude
       bool hidden = false;
 
       if(!isatty(1) ||
-         aptcfg->FindI("Quiet", 0))
+         aptcfg->FindI("quiet", 0) >= 1 ||
+         aptcfg->FindB("quiet::NoUpdate", false) == true)
         hide_tty_decorations = true;
 
+      if(aptcfg->FindI("quiet", 0) >= 2)
+        hidden = true;
+
       if(require_tty_decorations && hide_tty_decorations)
         hidden = true;
 
-- 
1.7.9

