diff -Nru cifs-utils-7.0/debian/changelog cifs-utils-7.0/debian/changelog
--- cifs-utils-7.0/debian/changelog	2022-08-26 16:06:45.000000000 +0200
+++ cifs-utils-7.0/debian/changelog	2024-06-17 00:00:05.000000000 +0200
@@ -1,3 +1,15 @@
+cifs-utils (2:7.0-2.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+
+  [ Debian Janitor ]
+  * Remove constraints unnecessary since buster
+
+  [ Michael Biebl ]
+  * Install files into /usr (DEP17 M2). (Closes: #1061332)
+
+ -- Chris Hofstaedtler <zeha@debian.org>  Mon, 17 Jun 2024 00:00:05 +0200
+
 cifs-utils (2:7.0-2) unstable; urgency=medium
 
   * root_sbindir-hook.patch - fix upstream fix for non-parallel
diff -Nru cifs-utils-7.0/debian/cifs-utils.lintian-overrides cifs-utils-7.0/debian/cifs-utils.lintian-overrides
--- cifs-utils-7.0/debian/cifs-utils.lintian-overrides	2022-08-25 22:11:10.000000000 +0200
+++ cifs-utils-7.0/debian/cifs-utils.lintian-overrides	2024-06-17 00:00:04.000000000 +0200
@@ -1 +1 @@
-cifs-utils: elevated-privileges 4755 root/root [sbin/mount.cifs]
+cifs-utils: elevated-privileges 4755 root/root [usr/sbin/mount.cifs]
diff -Nru cifs-utils-7.0/debian/control cifs-utils-7.0/debian/control
--- cifs-utils-7.0/debian/control	2022-08-25 21:23:04.000000000 +0200
+++ cifs-utils-7.0/debian/control	2024-06-16 23:59:28.000000000 +0200
@@ -13,8 +13,8 @@
 	libcap-ng-dev,
 	libkrb5-dev,
 	libpam0g-dev,
-	libtalloc-dev (>= 1.2.0~git20080616),
-	libwbclient-dev (>= 2:3.6.5-3),
+	libtalloc-dev,
+	libwbclient-dev,
 	python3-docutils,
 	debhelper-compat (= 13),
 	bash-completion,
diff -Nru cifs-utils-7.0/debian/rules cifs-utils-7.0/debian/rules
--- cifs-utils-7.0/debian/rules	2022-05-10 21:59:48.000000000 +0200
+++ cifs-utils-7.0/debian/rules	2024-06-17 00:00:04.000000000 +0200
@@ -6,7 +6,7 @@
 	dh $@ --with bash-completion
 
 override_dh_auto_configure:
-	dh_auto_configure -- --enable-cifsidmap --enable-cifscreds --with-libcap-ng=auto --enable-pam --with-pamdir=/lib/$(DEB_HOST_MULTIARCH)/security
+	ROOTSBINDIR=/usr/sbin dh_auto_configure -- --enable-cifsidmap --enable-cifscreds --with-libcap-ng=auto --enable-pam --with-pamdir=/usr/lib/$(DEB_HOST_MULTIARCH)/security
 
 override_dh_auto_install-indep:
 override_dh_auto_install-arch:
@@ -16,7 +16,7 @@
 
 override_dh_fixperms:
 	dh_fixperms
-	chmod u+s debian/cifs-utils/sbin/mount.cifs
+	chmod u+s debian/cifs-utils/usr/sbin/mount.cifs
 
 override_dh_auto_clean-indep:
 override_dh_auto_clean-arch:
