diff -r 5ab897f781da -r 7cc7cbc6ac3a src/generic/apt/aptcache.cc
--- a/src/generic/apt/aptcache.cc	Wed Jul 25 22:05:02 2007 -0700
+++ b/src/generic/apt/aptcache.cc	Sat Jul 28 08:34:05 2007 -0700
@@ -1411,6 +1411,7 @@ void aptitudeDepCache::apply_solution(co
     {
       pkgCache::PkgIterator pkg=i->first.get_pkg();
       pkgCache::VerIterator curver=pkg.CurrentVer();
+      pkgCache::VerIterator instver = (*apt_cache_file)[pkg].InstVerIter(*apt_cache_file);
       pkgCache::VerIterator actionver=i->second.ver.get_ver();
 
       // Check what type of action it is.
@@ -1428,7 +1429,14 @@ void aptitudeDepCache::apply_solution(co
 	{
 	  set_candidate_version(actionver, NULL);
 	  internal_mark_install(pkg, false, false);
-	  MarkAuto(pkg, true);
+	  // Mark the package as automatic iff it isn't currently
+	  // going to be installed.  Thus packages that are currently
+	  // manually installed don't get marked as auto, packages
+	  // that are going to be manually installed don't get marked
+	  // as auto, but packages that are being removed *do* get
+	  // marked as auto.
+	  if(instver.end())
+	    MarkAuto(pkg, true);
 	}
     }
 }
