From 55f7dabe2ea5fde7dd5a14e350c67f9b4ca7d22a Mon Sep 17 00:00:00 2001
From: Balint Reczey <balint@balintreczey.hu>
Date: Sun, 7 Sep 2014 14:55:04 +0200
Subject: [PATCH 3/3] Fix signed overflow in hash()

---
 src/filesdb.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/src/filesdb.c b/src/filesdb.c
index af1e466..91004fc 100644
--- a/src/filesdb.c
+++ b/src/filesdb.c
@@ -577,7 +577,7 @@ void filesdbinit(void) {
 }
 
 static int hash(const char *name) {
-  int v= 0;
+  unsigned int v= 0;
   while (*name) { v *= 1787; v += *name; name++; }
   return v;
 }
-- 
2.1.0

