Description: Explicitly request traditional scroll events.
Author: John Lindgren <john.lindgren@aol.com>
Origin: upstream, https://github.com/audacious-media-player/audacious-plugins/commit/b5612773bb620dfbee9c54f8dfa8fa5ce0d87694
Bug: http://redmine.audacious-media-player.org/issues/119
Bug-Debian: http://bugs.debian.org/672498
Reviewed-By: Carlos Alberto Lopez Perez <clopez@igalia.com>
--
--- audacious-plugins-3.2.3.orig/src/skins/ui_skinned_equalizer_slider.c
+++ audacious-plugins-3.2.3/src/skins/ui_skinned_equalizer_slider.c
@@ -170,7 +170,7 @@ GtkWidget * eq_slider_new (const gchar *
     GtkWidget * slider = gtk_drawing_area_new ();
     gtk_widget_set_size_request (slider, 14, 63);
     gtk_widget_add_events (slider, GDK_BUTTON_PRESS_MASK |
-     GDK_BUTTON_RELEASE_MASK | GDK_POINTER_MOTION_MASK);
+     GDK_BUTTON_RELEASE_MASK | GDK_POINTER_MOTION_MASK | GDK_SCROLL_MASK);
 
     DRAW_CONNECT (slider, eq_slider_draw);
 
--- audacious-plugins-3.2.3.orig/src/skins/ui_skinned_window.c
+++ audacious-plugins-3.2.3/src/skins/ui_skinned_window.c
@@ -107,7 +107,7 @@ GtkWidget * window_new (gint * x, gint *
     gtk_window_resize ((GtkWindow *) window, w, h);
 
     gtk_widget_add_events (window, GDK_BUTTON_PRESS_MASK |
-     GDK_BUTTON_RELEASE_MASK | GDK_POINTER_MOTION_MASK);
+     GDK_BUTTON_RELEASE_MASK | GDK_POINTER_MOTION_MASK | GDK_SCROLL_MASK);
 
     DRAW_CONNECT (window, window_draw);
     g_signal_connect (window, "button-press-event", (GCallback) window_button_press, NULL);
