--- a/src/xlsparse.c	2012-11-02 11:18:36.000000000 +1300
+++ b/src/xlsparse.c	2012-11-01 15:11:12.000000000 +1300
@@ -730,11 +730,11 @@
 # ifdef WORDS_BIGENDIAN     
 		for(s=rec+4,d=dconv.cc,i=0; i<4;i++) 
 			*(d++)=*(--s);
-		dconv.cc[3]=dconv.cc[3] & 0xfc;
+		dconv.cc[0]=dconv.cc[0] & 0xfc;
 # else       
 		for(s=rec,d=dconv.cc+4,i=0;
 				i<4;i++) *(d++)=*(s++);
-		dconv.cc[4]=dconv.cc[4] & 0xfc;
+		dconv.cc[3]=dconv.cc[3] & 0xfc;
 # endif     
 		value=dconv.d;
 	}
