Index: project-history.sgml
===================================================================
--- project-history.sgml	(revisione 10351)
+++ project-history.sgml	(copia locale)
@@ -659,7 +659,7 @@
 <p>An interesting fact about Debian 2.2 is that it showed how 
 an free software effort could lead to a modern operating system despite
 all the issues around it. This was studied<footnote><p>The 
-<url id="http://libresoft.es/debian-counting/potato/index.php?menu=Statistics" name="raw statitics data"> for Potato are also available at <url id="http://libresoft.es/debian-counting/" name="Debian counting site">, as well
+<url id="http://debian-counting.libresoft.es/potato/" name="raw statistics data"> for Potato are also available at <url id="http://debian-counting.libresoft.es/" name="Debian counting site">, as well
 as papers analysing later releases.</p></footnote>
 thoroughly by a group of interested people in
 an article called <url
