--- ups-monitor.dpkg-new	2016-07-16 08:19:48.000000000 -0700
+++ ups-monitor	2017-10-25 12:05:43.462235341 -0700
@@ -20,7 +20,10 @@
 
 case "$1" in
 	poweroff | killpower)
-		if [ -f /etc/apcupsd/powerfail ]; then
+		faildir="$(grep -e^PWRFAILDIR /etc/apcupsd/apcupsd.conf)"
+		faildir="${faildir#PWRFAILDIR }"
+
+		if [ -f "${faildir:=/etc/apcupsd}/powerfail" ]; then
 			echo ""
 			echo -n "apcupsd: Ordering UPS to kill power... "
 			/etc/apcupsd/apccontrol killpower
