diff -ru d52-3.4.1-orig/common.c d52-3.4.1/common.c
--- d52-3.4.1-orig/common.c	2007-09-02 17:33:46.000000000 +0200
+++ d52-3.4.1/common.c	2013-05-03 20:12:03.000000000 +0200
@@ -1747,8 +1747,11 @@
 		--asc_cnt;
 	}
 
-	putc('\'', fp);						// terminate line
-	kcnt++;
+	if (cnt)
+	{
+		putc('\'', fp);					// terminate line
+		kcnt++;
+	}
 
 	if (hexflag && cnt)					// if comment field requested...
 	{
