diff --git a/apt_offline_core/AptOfflineCoreLib.py b/apt_offline_core/AptOfflineCoreLib.py
index e06f55b..6af1235 100644
--- a/apt_offline_core/AptOfflineCoreLib.py
+++ b/apt_offline_core/AptOfflineCoreLib.py
@@ -901,6 +901,7 @@ def installer( args ):
                                 
                         if keyring is None:
                                 self.opts="--keyring /etc/apt/trusted.gpg --ignore-time-conflict"
+                                self.opts += "--keyring /usr/share/keyrings/debian-archive-keyring.gpg"
                         else:
                                 self.opts = "--keyring %s --ignore-time-conflict" % (keyring)
                                 
