--- docbook.xsl	2010-11-01 21:31:09.000000000 +0100
+++ /usr/share/xml/docbook/stylesheet/docbook-xsl-ns/epub/docbook.xsl	2012-03-28 14:10:28.270238268 +0200
@@ -83,7 +83,7 @@
       <xsl:when test="/*[not(self::d:book)][not(d:sect1) or not(d:section)]">
         <xsl:text>1</xsl:text>
       </xsl:when>
-      <xsl:when test="/d:book[*[last()][self::d:bookinfo]]|d:book[d:bookinfo]">
+      <xsl:when test="/d:book[*[last()][self::d:info]]|d:book[d:info]">
         <xsl:text>1</xsl:text>
       </xsl:when>
       <xsl:when test="/d:book[*[last()][self::d:info]]|d:book[d:info]">
@@ -773,8 +773,8 @@
 
       <!-- TODO: be nice to have a id="titlepage" here -->
       <xsl:apply-templates select="//d:part|
-                                   //d:book[*[last()][self::d:bookinfo]]|
-                                   //d:book[d:bookinfo]|
+                                   //d:book[*[last()][self::d:info]]|
+                                   //d:book[d:info]|
                                    /d:set|
                                    /d:set/d:book|
                                    //d:reference|
@@ -1066,8 +1066,8 @@
   <xsl:template
       match="d:set|
             d:book[parent::d:set]|
-            d:book[*[last()][self::d:bookinfo]]|
-            d:book[d:bookinfo]|
+            d:book[*[last()][self::d:info]]|
+            d:book[d:info]|
             d:article|
             d:part|
             d:reference|
