diff --git a/dohtml b/dohtml
index bd83bf8..38255f5 100755
--- a/dohtml
+++ b/dohtml
@@ -113,13 +113,13 @@ tags</a>:
    <li><strong>R</strong>: unreproducible</li>
    <li><strong>S</strong>: security</li>
    <li><strong>U</strong>: upstream</li>
-   <li><strong>I</strong>: stretch-ignore or buster-ignore</li>
+   <li><strong>I</strong>: <codename>-ignore (like 'bullseye-ignore')</li>
 </ul>
 
 <p>
   The second set of tags indicate what releases a bug applies to:
-  O for oldstable (jessie), S for stable (stretch), T for testing (buster),
-  U for unstable (sid) or E for experimental.
+  O for oldstable, S for stable, T for testing,
+  U for unstable or E for experimental.
 </p>
 
 EOF
