diff -Nru devscripts-2.10.35lenny2/debian/changelog devscripts-2.10.35lenny3/debian/changelog
--- devscripts-2.10.35lenny2/debian/changelog	2009-01-10 18:08:11.000000000 +0000
+++ devscripts-2.10.35lenny3/debian/changelog	2009-02-11 07:52:05.000000000 +0000
@@ -1,3 +1,10 @@
+devscripts (2.10.35lenny3) testing; urgency=low
+
+  * debchange: Recognise lenny-backports and lenny-volatile as valid Debian
+    distributions. (Closes: #514818)
+
+ -- Adam D. Barratt <adam@adam-barratt.org.uk>  Wed, 11 Feb 2009 07:48:29 +0000
+
 devscripts (2.10.35lenny2) testing; urgency=low
 
   * debsign: Use mktemp rather than a directory based on the process ID.
diff -Nru devscripts-2.10.35lenny2/scripts/debchange.pl devscripts-2.10.35lenny3/scripts/debchange.pl
--- devscripts-2.10.35lenny2/scripts/debchange.pl	2008-07-03 19:51:04.000000000 +0100
+++ devscripts-2.10.35lenny3/scripts/debchange.pl	2009-02-11 07:48:12.000000000 +0000
@@ -383,8 +383,8 @@
     $distributor ||= 'Debian';
 
     if ($distributor eq 'Debian') {
-	unless ($opt_D =~ /^(unstable|((old)?stable|testing)(-security)?|experimental|UNRELEASED|(sarge|etch)(-volatile|-backports)?|((oldstable|testing)-)?proposed-updates)$/) {
-	    warn "$progname warning: Recognised distributions are: unstable, testing, stable,\noldstable, experimental, UNRELEASED, {sarge,etch}-{volatile,backports},\n{oldstable-,testing-,}proposed-updates and {testing,stable,oldstable}-security.\nUsing your request anyway.\n";
+	unless ($opt_D =~ /^(unstable|((old)?stable|testing)(-security)?|experimental|UNRELEASED|(sarge|etch|lenny)(-volatile|-backports)?|((oldstable|testing)-)?proposed-updates)$/) {
+	    warn "$progname warning: Recognised distributions are: unstable, testing, stable,\noldstable, experimental, UNRELEASED, {sarge,etch,lenny}-{volatile,backports},\n{oldstable-,testing-,}proposed-updates and {testing,stable,oldstable}-security.\nUsing your request anyway.\n";
 	    $warnings++ if not $opt_force_dist;
 	}
     } elsif ($distributor eq 'Ubuntu') {
