From 1bbe6dc1e9fa3d41e65af6a28268fcb8d51d4a5b Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Wed, 25 Jan 2017 09:11:11 -0700
Subject: [PATCH] dgit-maint-merge(7): Quote sample clone commands

Signed-off-by: Sean Whitton <spwhitton@spwhitton.name>
---
 dgit-maint-merge.7.pod | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/dgit-maint-merge.7.pod b/dgit-maint-merge.7.pod
index 3587dcd..54cf79d 100644
--- a/dgit-maint-merge.7.pod
+++ b/dgit-maint-merge.7.pod
@@ -258,8 +258,8 @@ first upload of upstream version 1.2.3, you could use:
 
 =back
 
-(If you have dgit, use dgit clone foo,
-rather than plain git clone.)
+(If you have dgit, use `dgit clone foo`,
+rather than plain `git clone`.)
 
 A single combined diff, containing all the changes, follows.
 
-- 
2.11.0

