Index: doc/Makefile
===================================================================
--- doc/Makefile	(revision 10970)
+++ doc/Makefile	(working copy)
@@ -40,7 +40,7 @@
 	wml -i bug-pkgreport-opts.inc -q $< >$@
 
 %.txt: %.html
-	lynx -dump -nolist $< >$@
+	LC_ALL=C.UTF-8 lynx -dump -nolist $< >$@
 
 # These rules are conditioned to the existence of $(WEBWML)
 # so that the package can be built regardless of its existence
