commit a3a64968b0ddebb50305d294b68d18c43eda4644
Author: Jelmer Vernooij <jelmer@samba.org>
Date:   Sat Feb 22 17:41:10 2014 +0000

    Fix old private key location in README.Debian. Closes: #702385

diff --git a/debian/changelog b/debian/changelog
index 143efe4..3683cac 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,10 @@
+dovecot (1:2.2.9-1.1) UNRELEASED; urgency=medium
+
+  * Non-maintainer upload.
+  * Fix old private key location in README.Debian. Closes: #702385
+
+ -- Jelmer Vernooij <jelmer@debian.org>  Sat, 22 Feb 2014 17:40:57 +0000
+
 dovecot (1:2.2.9-1) unstable; urgency=low
 
   [ Jaldhar H. Vyas ]
diff --git a/debian/dovecot-core.README.Debian b/debian/dovecot-core.README.Debian
index 6913bdf..90c9357 100644
--- a/debian/dovecot-core.README.Debian
+++ b/debian/dovecot-core.README.Debian
@@ -96,7 +96,7 @@ Upgrading from 1.2 (squeeze) to 2.0 (wheezy)
     1.  Move /etc/ssl/certs/dovecot.pem to /etc/dovecot
     2.  Create a directory called /etc/dovecot/private owned by root:root
         with 0700 permissions.
-    3.  Move /etc/ssl/certs/private/dovecot.pem to /etc/dovecot/private
+    3.  Move /etc/ssl/private/dovecot.pem to /etc/dovecot/private
     4.  In /etc/dovecot/conf.d/10-ssl.conf make sure you have these lines:
         ssl_cert = </etc/dovecot/dovecot.pem
         ssl_key = </etc/dovecot/private/dovecot.pem
