diff -ur old/discover-2.1.2/scripts/discover-pkginstall discover-2.1.2/scripts/discover-pkginstall
--- old/discover-2.1.2/scripts/discover-pkginstall	2010-12-04 16:03:36.000000000 -0400
+++ discover-2.1.2/scripts/discover-pkginstall	2010-12-04 16:08:34.050134457 -0400
@@ -171,7 +171,12 @@
 
         if [ "$packages" ] ; then
             if [ -x /usr/bin/debconf-apt-progress ] ; then
-                $nop debconf-apt-progress -- aptitude install -q -y $packages
+		if [ "$DI_PROGRESS_BAR_VISIBLE" ]; then
+		    debconf_apt_progress_opts=--no-progress
+		else
+		    debconf_apt_progress_opts=
+		fi
+                $nop debconf-apt-progress $debconf_apt_progress_opts -- aptitude install -q -y $packages
             else
                 $nop aptitude install -y $packages
             fi
