diff -u alsaplayer-0.99.80/debian/changelog alsaplayer-0.99.80/debian/changelog
--- alsaplayer-0.99.80/debian/changelog
+++ alsaplayer-0.99.80/debian/changelog
@@ -1,3 +1,14 @@
+alsaplayer (0.99.80-3.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Fix FTBFS due to missing <climits> include (Closes: #518734):
+     * Add patch: 06_climits.dpatch
+  * Set urgency to “medium”, this bug affects testing too.
+  * Switch from obsolete “xlibmesa-gl-dev | libgl-dev” to libgl1-mesa-dev
+    in Build-Depends.
+
+ -- Cyril Brulebois <kibi@debian.org>  Sun, 19 Jul 2009 15:09:12 +0200
+
 alsaplayer (0.99.80-3) unstable; urgency=low
 
   * debian/alsaplayer-xosd.menu: The xosd interface really shouldn't need to be
diff -u alsaplayer-0.99.80/debian/control alsaplayer-0.99.80/debian/control
--- alsaplayer-0.99.80/debian/control
+++ alsaplayer-0.99.80/debian/control
@@ -5,7 +5,7 @@
 Uploaders: Paul Brossier <piem@debian.org>
 Standards-Version: 3.7.3
 Homepage: http://www.alsaplayer.org/
-Build-Depends: debhelper (>= 5.0.0), dpatch, libasound2-dev, libgtk2.0-dev, libesd0-dev, libmikmod2-dev, libaudio-dev, libogg-dev, libvorbis-dev, libtool, xlibmesa-gl-dev | libgl-dev, libmad0-dev, libid3tag0-dev, zlib1g-dev, libjack-dev, libflac-dev (>= 1.1.4-1), libxosd-dev, libsndfile1-dev, doxygen, libxt-dev
+Build-Depends: debhelper (>= 5.0.0), dpatch, libasound2-dev, libgtk2.0-dev, libesd0-dev, libmikmod2-dev, libaudio-dev, libogg-dev, libvorbis-dev, libtool, libgl1-mesa-dev, libmad0-dev, libid3tag0-dev, zlib1g-dev, libjack-dev, libflac-dev (>= 1.1.4-1), libxosd-dev, libsndfile1-dev, doxygen, libxt-dev
 
 Package: alsaplayer-common
 Architecture: any
only in patch2:
unchanged:
--- alsaplayer-0.99.80.orig/debian/patches/00list
+++ alsaplayer-0.99.80/debian/patches/00list
@@ -0,0 +1 @@
+06_climits.dpatch
only in patch2:
unchanged:
--- alsaplayer-0.99.80.orig/debian/patches/06_climits.dpatch
+++ alsaplayer-0.99.80/debian/patches/06_climits.dpatch
@@ -0,0 +1,16 @@
+#! /bin/sh /usr/share/dpatch/dpatch-run
+## 06_climits.dpatch by Cyril Brulebois <kibi@debian.org>
+##
+## DP: Fix FTBFS due to missing declaration of PATH_MAX.
+
+@DPATCH@
+--- a/app/ControlSocket.cpp
++++ b/app/ControlSocket.cpp
+@@ -18,6 +18,7 @@
+  *
+  */ 
+ #include "AlsaPlayer.h"
++#include <climits>
+ #include <cstdio>
+ #include <cstdlib>
+ #include <unistd.h>
