From 4c208a8428bb9d897085c61cc5b077c0810281aa Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lo=C3=AFc=20Minier?= <loic.minier@ubuntu.com>
Date: Sun, 9 May 2010 13:43:16 +0200
Subject: [PATCH] Support debcommiting to bzr from debian/

---
 scripts/debcommit.pl |    5 ++++-
 1 files changed, 4 insertions(+), 1 deletions(-)

diff --git a/scripts/debcommit.pl b/scripts/debcommit.pl
index 8555b8a..a521786 100755
--- a/scripts/debcommit.pl
+++ b/scripts/debcommit.pl
@@ -446,10 +446,13 @@ sub getprog {
 	}
     }
 
-    # .git may be in a parent directory, rather than the current
+    # .bzr or .git may be in a parent directory, rather than the current
     # directory, if multiple packages are kept in one git repository.
     my $dir=getcwd();
     while ($dir=~s/[^\/]*\/?$// && length $dir) {
+	if (-d "$dir/.bzr") {
+	    return "bzr";
+	}
 	if (-d "$dir/.git") {
 	    return "git";
 	}
-- 
1.7.0.4

