--- getbuildlog.sh	2022-06-13 22:42:31.607100283 +0500
+++ getbuildlog.sh-681779	2022-06-13 22:36:46.879528040 +0500
@@ -93,7 +93,9 @@
 
     trap 'rm -f "$ALL_LOGS"' EXIT
 
-    wget -q -O $ALL_LOGS "$BASE/status/logs.php?pkg=$PACKAGE"
+    wget -q -O - "$BASE/status/logs.php?pkg=$PACKAGE" \
+    | tee $ALL_LOGS \
+    | { grep -q "No build logs found for $PACKAGE () in the database" && exit 1; }
 
     # Put each href in $ALL_LOGS on a separate line so that $PATTERN
     # matches only one href. This is required because grep is greedy.
