Index: debian/changelog
===================================================================
--- debian/changelog	(revision 46993)
+++ debian/changelog	(working copy)
@@ -1,3 +1,12 @@
+net-retriever (1.18) UNRELEASED; urgency=low
+
+  * Use /.installer/udebs_source as the default release to fetch udebs from.
+    This is the same release (codename) as was used to build the images and
+    thus ensures consistency. It is independent of the release selected for
+    the target system. For daily builds the source will default to 'unstable'.
+
+ -- Frans Pop <fjp@debian.org>  Sat, 26 May 2007 16:42:28 +0200
+
 net-retriever (1.17) unstable; urgency=low
 
   [ Joey Hess ]
Index: net-retriever
===================================================================
--- net-retriever	(revision 46993)
+++ net-retriever	(working copy)
@@ -98,7 +98,11 @@
 
 		# Setting codename to a suite is not very nice, but can do no harm
 		if ! db_get mirror/udeb/suite || [ -z "$RET" ]; then
-			db_get mirror/codename
+			if [ -f /.installer/udebs_source ]; then
+				RET=$(cat /.installer/udebs_source)
+			else
+				db_get mirror/codename
+			fi
 		fi
 		codename="$RET"
 
