diff -Nur cegui-mk2-0.5.0/build-tree/cegui_mk2/acinclude.m4 cegui-mk2-0.5.0.new/build-tree/cegui_mk2/acinclude.m4
--- cegui_mk2/acinclude.m4	2006-10-25 05:54:43.000000000 -0500
+++ cegui_mk2/acinclude.m4	2007-12-26 15:52:02.000000000 -0600
@@ -537,7 +537,13 @@
 ])
 
 AC_DEFUN([CEGUI_CHECK_LUA],[
-    PKG_CHECK_MODULES(Lua, lua >= 5.0 lua < 5.1, [cegui_found_lua=yes], [cegui_found_lua=no])
+    PKG_CHECK_MODULES(Lua, lua >= 5.0 lua < 5.1, [cegui_found_lua=yes],
+    [
+        PKG_CHECK_MODULES(Lua, lua50 >= 5.0 lua50 < 5.1 lualib50 >= 5.0 lualib50 < 5.1, [cegui_found_lua=yes],
+        [
+            [cegui_found_lua=no]
+        ])
+    ])
 
     AC_ARG_ENABLE([lua-module], AC_HELP_STRING([--disable-lua-module], [Disables building of the Lua scripting module.]),
                 [cegui_with_lua=$enableval], [cegui_with_lua=yes])
diff -Nur cegui-mk2-0.5.0/build-tree/cegui_mk2/ScriptingModules/CEGUILua/LuaScriptModule/src/Makefile.am cegui-mk2-0.5.0.new/build-tree/cegui_mk2/ScriptingModules/CEGUILua/LuaScriptModule/src/Makefile.am
--- cegui_mk2/ScriptingModules/CEGUILua/LuaScriptModule/src/Makefile.am	2006-07-25 03:34:51.000000000 -0500
+++ cegui_mk2/ScriptingModules/CEGUILua/LuaScriptModule/src/Makefile.am	2007-12-26 15:55:07.000000000 -0600
@@ -1,4 +1,5 @@
-INCLUDES = -I$(top_srcdir)/include \
+INCLUDES = $(Lua_CFLAGS) \
+           -I$(top_srcdir)/include \
            -I$(top_srcdir)/ScriptingModules/CEGUILua/tolua++ \
            -I$(top_srcdir)/ScriptingModules/CEGUILua/LuaScriptModule/include
 lib_LTLIBRARIES = libCEGUILuaScriptModule.la
diff -Nur cegui-mk2-0.5.0/build-tree/cegui_mk2/ScriptingModules/CEGUILua/tolua++bin/Makefile.am cegui-mk2-0.5.0.new/build-tree/cegui_mk2/ScriptingModules/CEGUILua/tolua++bin/Makefile.am
--- cegui_mk2/ScriptingModules/CEGUILua/tolua++bin/Makefile.am	2006-10-17 06:58:04.000000000 -0500
+++ cegui_mk2/ScriptingModules/CEGUILua/tolua++bin/Makefile.am	2007-12-26 15:55:47.000000000 -0600
@@ -1,4 +1,4 @@
-INCLUDES = -I$(top_srcdir)/ScriptingModules/CEGUILua/tolua++
+INCLUDES = $(Lua_CFLAGS) -I$(top_srcdir)/ScriptingModules/CEGUILua/tolua++
 
 bin_PROGRAMS = tolua++cegui
 tolua__cegui_SOURCES = tolua.c toluabind.c toluabind.h
