From fe2c1726319f0702a5c715f5204e3b64d9d07509 Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Mon, 23 Jul 2018 12:10:02 +0800
Subject: [PATCH 1/2] dgit: add dgit.default.build-products-dir git
 configuration key

Signed-off-by: Sean Whitton <spwhitton@spwhitton.name>
---
 dgit   |  9 ++++++++-
 dgit.1 | 11 +++++++++++
 2 files changed, 19 insertions(+), 1 deletion(-)

diff --git a/dgit b/dgit
index 357adc9..a3d3b6b 100755
--- a/dgit
+++ b/dgit
@@ -63,7 +63,7 @@ our @ropts;
 our $sign = 1;
 our $dryrun_level = 0;
 our $changesfile;
-our $buildproductsdir = '..';
+our $buildproductsdir;
 our $new_package = 0;
 our $ignoredirty = 0;
 our $rmonerror = 1;
@@ -4715,6 +4715,7 @@ sub prep_push () {
     parseopts();
     build_or_push_prep_early();
     pushing();
+    build_or_push_prep();
     check_not_dirty();
     my $specsuite;
     if (@ARGV==0) {
@@ -6088,6 +6089,11 @@ sub cmd_clean () {
     maybe_unapply_patches_again();
 }
 
+sub build_or_push_prep () {
+    $buildproductsdir //= access_cfg('build-products-dir', 'RETURN-UNDEF');
+    $buildproductsdir //= '..';
+}
+
 sub build_or_push_prep_early () {
     our $build_or_push_prep_early_done //= 0;
     return if $build_or_push_prep_early_done++;
@@ -6106,6 +6112,7 @@ sub build_prep_early () {
 
 sub build_prep () {
     build_prep_early();
+    build_or_push_prep();
     clean_tree();
     build_maybe_quilt_fixup();
     if ($rmchanges) {
diff --git a/dgit.1 b/dgit.1
index 1460938..ddb0c0a 100644
--- a/dgit.1
+++ b/dgit.1
@@ -842,6 +842,11 @@ regardless of this option.
 Specifies where to find the built files to be uploaded.
 By default, dgit looks in the parent directory
 .RB ( .. ).
+
+Also see the
+.BI dgit.default.build-products-dir
+configuration option
+(which this command line option overrides).
 .TP
 .BI --no-rm-on-error
 Do not delete the destination directory if clone fails.
@@ -1096,6 +1101,12 @@ on the dgit command line.
 .LP
 Settings likely to be useful for an end user include:
 .TP
+.BI dgit.default.build-products-dir
+Specifies where to find the built files to be uploaded,
+when --build-products-dir is not specified.  The default is
+the parent directory
+.RB ( .. ).
+.TP
 .BR dgit-suite. \fIsuite\fR .distro " \fIdistro\fR"
 Specifies the distro for a suite.  dgit keys off the suite name (which
 appears in changelogs etc.), and uses that to determine the distro
-- 
2.11.0

