From 2551bbde4f9fed0732e10defa7e2ed2191a12775 Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Tue, 24 Jul 2018 11:15:56 +0800
Subject: [PATCH 2/2] test suite lib: add t-buildproductsdir-config

Signed-off-by: Sean Whitton <spwhitton@spwhitton.name>
---
 tests/lib | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/tests/lib b/tests/lib
index 99345ce..3fda1bc 100644
--- a/tests/lib
+++ b/tests/lib
@@ -1164,3 +1164,8 @@ for import in ${autoimport-gnupg}; do
 		;;
 	esac
 done
+
+t-buildproductsdir-config () {
+	# use --local, not t-git-config, because the value is relative
+	git config --local dgit.default.build-products-dir ../bpd
+}
-- 
2.11.0

