--- /etc/logrotate.d/apt-cacher-ng	2014-03-22 15:08:53.000000000 +0100
+++ apt-cacher-ng	2014-11-26 21:50:43.446950621 +0100
@@ -11,6 +11,9 @@
                 if [ -s /var/run/apt-cacher-ng/pid ] ; then
                         kill -s USR1 "$(cat /var/run/apt-cacher-ng/pid)"
                 fi
+                if [ -e /run/systemd/system ] ; then
+                        systemctl is-active apt-cacher-ng && systemctl kill --signal=USR1 apt-cacher-ng
+                fi
         endscript
 }
 
@@ -27,6 +30,9 @@
                 if [ -s /var/run/apt-cacher-ng/pid ] ; then
                         kill -s USR1 "$(cat /var/run/apt-cacher-ng/pid)"
                 fi
+                if [ -e /run/systemd/system ] ; then
+                        systemctl is-active apt-cacher-ng && systemctl kill --signal=USR1 apt-cacher-ng
+                fi
         endscript
 }
 
