=== modified file 'CONF.sh'
--- CONF.sh	2010-07-09 11:54:19 +0000
+++ CONF.sh	2010-07-30 13:03:54 +0000
@@ -181,8 +181,8 @@
 # them too
 export MAX_PKG_SIZE=600000000
 
-# uncoment this to make build_all.sh try to build a simple CD image if
-# the proper official CD run does not work
+# Should build_all.sh try to build a simple CD image if the proper official
+# CD run does not work?
 ATTEMPT_FALLBACK=yes
 
 # Set your disk type here. Known types are:
@@ -357,14 +357,14 @@
 # If so we will link to them on the web site.
 export OMIT_RELEASE_NOTES=1
 
+# Set this to override the default location
+#export RELEASE_NOTES_LOCATION="http://www.debian.org/releases/$CODENAME"
+
 # Set to 1 to not include the doc/tools directories on CD1
 # Useful to save space and avoids failures if you have a mirror
 # without those directories.
 #export OMIT_DOC_TOOLS=1
 
-# Set this to override the default location
-#export RELEASE_NOTES_LOCATION="http://www.debian.org/releases/$CODENAME"
-
 case "$OFFICIAL" in
     "Official")
 	export OFFICIAL_VAL=2

