Index: menu.cfg
===================================================================
--- menu.cfg	(revision 56664)
+++ menu.cfg	(working copy)
@@ -12,6 +12,11 @@
 	label mainmenu
 		menu label ^Back..
 		menu exit
+		text help
+   Display help screens; type 'menu' at boot prompt to return to this menu.
+		endtext
+ 
+ 
 	include ${SYSDIR}stdmenu.cfg
 	include ${SYSDIR}adtext.cfg
 	include ${SYSDIR}adamdtext.cfg
@@ -21,3 +26,7 @@
 label help
 	menu label ^Help
 	config ${SYSDIR}prompt.cfg
+	text help
+   Display help screens; type 'menu' at boot prompt to return to this menu.
+	endtext
+
Index: stdmenu.cfg
===================================================================
--- stdmenu.cfg	(revision 56664)
+++ stdmenu.cfg	(working copy)
@@ -4,8 +4,14 @@
 menu color sel		* #ffffffff #76a1d0ff *
 menu color hotsel	1;7;37;40 #ffffffff #76a1d0ff *
 menu color tabmsg	* #ffffffff #00000000 *
+menu color help 	37;40 #ffdddd00 #00000000 none
 menu vshift 11
 menu rows 10
-menu tabmsgrow 16
-menu timeoutrow 17
+menu tabmsgrow 18
+menu timeoutrow 16
 menu tabmsg Press ENTER to boot or TAB to edit a menu entry
+
+menu helpmsgrow 15
+
+#The command line must be at least one line from the bottom.
+menu cmdlinerow 16
