diff --git a/installer/build/boot/x86/exithelp.cfg b/installer/build/boot/x86/exithelp.cfg
new file mode 100644
index 0000000..6d9579a
--- /dev/null
+++ b/installer/build/boot/x86/exithelp.cfg
@@ -0,0 +1,3 @@
+label menu
+	kernel ${SYSDIR}vesamenu.c32
+	config ${SYSDIR}${SYSLINUX_CFG}
diff --git a/installer/build/boot/x86/prompt.cfg b/installer/build/boot/x86/prompt.cfg
index cd2957d..5ee6b27 100644
--- a/installer/build/boot/x86/prompt.cfg
+++ b/installer/build/boot/x86/prompt.cfg
@@ -2,6 +2,7 @@ prompt 1
 display ${SYSDIR}f1.txt
 timeout 0
 include ${SYSDIR}menu.cfg
+include ${SYSDIR}exithelp.cfg
 
 f1 ${SYSDIR}f1.txt
 f2 ${SYSDIR}f2.txt
diff --git a/installer/build/config/x86.cfg b/installer/build/config/x86.cfg
index 39a52f6..75bdd5c 100644
--- a/installer/build/config/x86.cfg
+++ b/installer/build/config/x86.cfg
@@ -72,6 +72,7 @@ endif
 				VIDEO_MODE $(VIDEO_MODE) \
 				VIDEO_MODE_GTK $(VIDEO_MODE_GTK) \
 				BEEP "$$beep" \
+				SYSLINUX_CFG syslinux.cfg \
 			| todos | mcopy -i$(TEMP_BOOT) - ::$$outfile; \
 		fi; \
 	)
@@ -169,6 +170,7 @@ arch_cd_info_dir:
 			VIDEO_MODE $(VIDEO_MODE) \
 			VIDEO_MODE_GTK $(VIDEO_MODE_GTK) \
 			BEEP "$$beep" \
+			SYSLINUX_CFG isolinux.cfg \
 		> $(TEMP_CD_INFO_DIR)/$$outfile; \
 	)
 
@@ -228,6 +230,7 @@ arch_miniiso:
 			VIDEO_MODE $(VIDEO_MODE) \
 			VIDEO_MODE_GTK $(VIDEO_MODE_GTK) \
 			BEEP "$$beep" \
+			SYSLINUX_CFG isolinux.cfg \
 		| todos > $(TEMP_CD_TREE)/$$outfile; \
 	)
 	
@@ -282,6 +285,7 @@ arch_netboot_dir:
 			INITRD $(NETBOOT_PATH)/initrd.gz \
 			VIDEO_MODE $(VIDEO_MODE) \
 			BEEP "$$beep" \
+			SYSLINUX_CFG syslinux.cfg \
 		> $$outfile; \
 	)
 	
