--- /etc/bash_completion	2007-03-29 10:39:42.000000000 -0400
+++ /tmp/tmp.CQWoe19504/bash_completion	2007-06-30 16:07:50.000000000 -0400
@@ -916,7 +916,7 @@
 		COMPREPLY=( $( compgen -W '0 1 2 3 4 5 6 7 8 9' -- $cur ) )
 		return 0
 		;;
-	-?(a|c)newer|-fls|-fprint?(0|f)|-?(i)?(l)name)
+	-?(a|c)newer|-fls|-fprint?(0|f)|-?(i)?(l)name|-?(i)wholename)
 		_filedir
 		return 0
 		;;
@@ -980,6 +980,7 @@
 			-mindepth -mount -noleaf -version -xdev -amin -anewer \
 			-atime -cmin -cnewer -ctime -empty -false -fstype \
 			-gid -group -ilname -iname -inum -ipath -iregex \
+			-wholename \
 			-links -lname -mmin -mtime -name -newer -nouser \
 			-nogroup -perm -regex -size -true -type -uid -used \
 			-user -xtype -exec -fls -fprint -fprint0 -fprintf -ok \
