Index: debian/aide.conf.d/31_aide_apache2
===================================================================
--- debian/aide.conf.d/31_aide_apache2	(revision 785)
+++ debian/aide.conf.d/31_aide_apache2	(working copy)
@@ -6,7 +6,9 @@
 /var/log/apache2/@@{APACHE2_LOGS}\.log$ Log
 /var/log/apache2/@@{APACHE2_LOGS}\.log\.1$ LowLog
 /var/log/apache2/@@{APACHE2_LOGS}\.log\.2\.gz$ LoSerMemberLog
-/var/log/apache2/@@{APACHE2_LOGS}\.log\.[0-9]+\.gz$ SerMemberLog
+/var/log/apache2/@@{APACHE2_LOGS}\.log\.([3-9]|[1-4][0-9]|5[0-1])\.gz$ SerMemberLog
+/var/log/apache2/@@{APACHE2_LOGS}\.log\.52\.gz$ HiSerMemberLog
+
 /var/run/apache2\.pid$ VarFile
 /var/run/apache2/ssl_scache$ VarFile
 /var/(log|run)/apache2$ VarDir
