--- orig/elinks-0.11.1/debian/rules	2006-11-11 10:09:22.000000000 +0100
+++ elinks-0.11.1/debian/rules	2006-11-11 10:42:18.000000000 +0100
@@ -109,6 +109,8 @@
 	ln -sf /usr/share/misc/config.sub .
 	ln -sf /usr/share/misc/config.guess .
 
+	[ -f config.h.in_DEBIANORIG ] || cp -p config.h.in config.h.in_DEBIANORIG
+
 	mkdir build-main && cd build-main && ../configure $(CONFFLAGS) $(CONFIGURE_MODE) $(CONFIGURE_OPTIONS)
 	$(MAKE) -C build-main
 	touch debian/build-heavy
@@ -142,6 +144,8 @@
 	ln -sf /usr/share/misc/config.sub .
 	ln -sf /usr/share/misc/config.guess .
 
+	[ -f config.h.in_DEBIANORIG ] || cp -p config.h.in config.h.in_DEBIANORIG
+
 	mkdir build-lite && cd build-lite && ../configure CFLAGS="$(CFLAGS_LITE)" $(CONFFLAGS) $(CONFIGURE_MODE) $(CONFIGURE_OPTIONS_LITE)
 	$(MAKE) -C build-lite
 	touch debian/build-lite
@@ -181,6 +185,16 @@
 	> debian/control
 	
 #	-find . -name ".#*" -exec rm -rf "{}" \;
+#
+	rm -rf build-main
+	rm -rf build-lite
+	rm -f debian/elinks.conf.temp
+	rm -f debian/elinks-lite.conf.temp
+	rm -f config.guess
+	rm -f config.sub
+	rm -f stamp-h.in
+	[ ! -f config.h.in_DEBIANORIG ] || mv config.h.in_DEBIANORIG config.h.in
+
 	dh_clean
 
 
