diff --git a/scripts/debuild.pl b/scripts/debuild.pl
index f6e74dd..068b95a 100755
--- a/scripts/debuild.pl
+++ b/scripts/debuild.pl
@@ -837,7 +837,7 @@ if ($command_version eq 'dpkg') {
 	/^-e(.*)/ and $changedby=$1, push(@debsign_opts, $_),
 	    push(@dpkg_opts, $_), next;
 	/^-C(.*)/ and $desc=$1, push(@dpkg_opts, $_), next;
-	/^-j(\d*)$/ and $parallel=($1 || '-1'), push(@dpkg_opts, $_), next;
+	/^-j(auto|\d*)$/ and $parallel=($1 || '-1'), push(@dpkg_opts, $_), next;
 	$_ eq '-W' and $warnable_error=1, push(@passopts, $_),
 	    push(@dpkg_opts, $_), next;
 	$_ eq '-E' and $warnable_error=0, push(@passopts, $_),
@@ -889,7 +889,7 @@ if ($command_version eq 'dpkg') {
 	/^-e(.*)/ and $changedby=$1, push(@debsign_opts, $_),
 	    push(@dpkg_opts, $_), next;
 	/^-C(.*)/ and $desc=$1, push(@dpkg_opts, $_), next;
-	/^-j(\d*)$/ and $parallel=($1 || '-1'), push(@dpkg_opts, $_), next;
+	/^-j(auto|\d*)$/ and $parallel=($1 || '-1'), push(@dpkg_opts, $_), next;
 	$_ eq '-W' and $warnable_error=1, push(@passopts, $_),
 	    push(@dpkg_opts, $_), next;
 	$_ eq '-E' and $warnable_error=0, push(@passopts, $_),
