From e5a092a87a49e0d8495fddfd6e25b7a2788ea627 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?David=20Pr=C3=A9vot?= <david@tilapin.org>
Date: Fri, 13 May 2011 17:55:49 -0400
Subject: [PATCH 5/7] Use of Policy compliant relations

---
 grep-dctrl/grep-dctrl.c |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/grep-dctrl/grep-dctrl.c b/grep-dctrl/grep-dctrl.c
index 38f3b02..db23956 100644
--- a/grep-dctrl/grep-dctrl.c
+++ b/grep-dctrl/grep-dctrl.c
@@ -129,9 +129,9 @@ static struct argp_option options[] = {
 	{ "or",		    'o', 0,		    0, N_("Disjunct filters.") },
 	{ "not",	    '!', 0,		    0, N_("Negate the following filters.") },
 	{ "eq",		    OPT_EQ, 0,		    0, N_("Test for version number equality.") },
-	{ "lt",		    OPT_LT, 0,		    0, N_("Version number comparison: <.") },
+	{ "lt",		    OPT_LT, 0,		    0, N_("Version number comparison: <<.") },
 	{ "le",		    OPT_LE, 0,		    0, N_("Version number comparison: <=.") },
-	{ "gt",		    OPT_GT, 0,		    0, N_("Version number comparison: >.") },
+	{ "gt",		    OPT_GT, 0,		    0, N_("Version number comparison: >>.") },
 	{ "ge",		    OPT_GE, 0,		    0, N_("Version number comparison: >=.") },
 	{ "debug-optparse", OPT_OPTPARSE, 0,	    0, N_("Debug option parsing.") },
 	{ "quiet",	    'q', 0,		    0, N_("Do no output to stdout.") },
-- 
1.7.5.1

