diff --git a/debtagslayout/static/js/debtags-ui.js b/debtagslayout/static/js/debtags-ui.js
index bcba898..770dbd7 100644
--- a/debtagslayout/static/js/debtags-ui.js
+++ b/debtagslayout/static/js/debtags-ui.js
@@ -254,7 +254,7 @@ Settings.prototype = {
             if (count_changes > 0)
             {
                 if (count_changes == 1)
-                    self._submit_summary.text("There is one change change to submit:");
+                    self._submit_summary.text("There is one change to submit:");
                 else
                     self._submit_summary.text("There are " + count_changes + " changes to submit:");
                 self._submit_button.attr("disabled", false);
