=== modified file 'lib/licenses/apache'
--- lib/licenses/apache	2009-03-14 21:32:39 +0000
+++ lib/licenses/apache	2009-04-03 04:28:09 +0000
@@ -1,5 +1,6 @@
-This package was debianized by #USERNAME# <#EMAIL#> on
-#DATE#.
+This work was packaged for Debian by:
+
+    #USERNAME# <#EMAIL#> on #DATE#
 
 It was downloaded from <url://example.com>
 

=== modified file 'lib/licenses/artistic'
--- lib/licenses/artistic	2009-03-14 21:32:39 +0000
+++ lib/licenses/artistic	2009-04-03 04:28:09 +0000
@@ -1,5 +1,6 @@
-This package was debianized by #USERNAME# <#EMAIL#> on
-#DATE#.
+This work was packaged for Debian by:
+
+    #USERNAME# <#EMAIL#> on #DATE#
 
 It was downloaded from <url://example.com>
 

=== modified file 'lib/licenses/blank'
--- lib/licenses/blank	2009-03-14 21:32:39 +0000
+++ lib/licenses/blank	2009-04-03 04:28:09 +0000
@@ -1,5 +1,6 @@
-This package was debianized by #USERNAME# <#EMAIL#> on
-#DATE#.
+This work was packaged for Debian by:
+
+    #USERNAME# <#EMAIL#> on #DATE#
 
 It was downloaded from <url://example.com>
 

=== modified file 'lib/licenses/bsd'
--- lib/licenses/bsd	2009-03-14 21:32:39 +0000
+++ lib/licenses/bsd	2009-04-03 04:28:09 +0000
@@ -1,5 +1,6 @@
-This package was debianized by #USERNAME# <#EMAIL#> on
-#DATE#.
+This work was packaged for Debian by:
+
+    #USERNAME# <#EMAIL#> on #DATE#
 
 It was downloaded from <url://example.com>
 

=== modified file 'lib/licenses/gpl2'
--- lib/licenses/gpl2	2009-03-14 21:32:39 +0000
+++ lib/licenses/gpl2	2009-04-03 04:28:09 +0000
@@ -1,4 +1,4 @@
-This package was debianized by:
+This work was packaged for Debian by:
 
     #USERNAME# <#EMAIL#> on #DATE#
 

=== modified file 'lib/licenses/gpl3'
--- lib/licenses/gpl3	2009-03-14 21:32:39 +0000
+++ lib/licenses/gpl3	2009-04-03 04:28:09 +0000
@@ -1,4 +1,4 @@
-This package was debianized by:
+This work was packaged for Debian by:
 
     #USERNAME# <#EMAIL#> on #DATE#
 

=== modified file 'lib/licenses/lgpl2'
--- lib/licenses/lgpl2	2009-03-14 21:32:39 +0000
+++ lib/licenses/lgpl2	2009-04-03 04:28:09 +0000
@@ -1,5 +1,6 @@
-This package was debianized by #USERNAME# <#EMAIL#> on
-#DATE#.
+This work was packaged for Debian by:
+
+    #USERNAME# <#EMAIL#> on #DATE#
 
 It was downloaded from <url://example.com>
 

=== modified file 'lib/licenses/lgpl3'
--- lib/licenses/lgpl3	2009-03-14 21:32:39 +0000
+++ lib/licenses/lgpl3	2009-04-03 04:28:09 +0000
@@ -1,5 +1,6 @@
-This package was debianized by #USERNAME# <#EMAIL#> on
-#DATE#.
+This work was packaged for Debian by:
+
+    #USERNAME# <#EMAIL#> on #DATE#
 
 It was downloaded from <url://example.com>
 

