=== modified file 'debian/copyright'
--- debian/copyright	2009-03-14 21:32:39 +0000
+++ debian/copyright	2009-04-03 04:35:12 +0000
@@ -1,4 +1,4 @@
-dh-make: A program to help Debian developers Debianize source trees
+dh-make: A program to prepare Debian packaging for an original source archive
 
 Copyright (C) 1998-2009 Craig Small <csmall@debian.org>
 

=== modified file 'dh_make'
--- dh_make	2009-03-14 21:38:46 +0000
+++ dh_make	2009-04-03 04:35:12 +0000
@@ -1,6 +1,6 @@
 #!/usr/bin/perl 
 #
-# dh_make - Script to Debianize a source archive
+# dh_make - Script to prepare Debian packaging for an original source archive
 #
 use Getopt::Long;
 use Cwd;
@@ -94,7 +94,7 @@
 
 sub show_version
 {
-  print "dh_make - Script to Debianize a regular source archive, version $main::DH_MAKE_VERSION\n\n";
+  print "dh_make - prepare Debian packaging for an original source archive, version $main::DH_MAKE_VERSION\n\n";
   print "Copyright (C) 1998-2009 Craig Small <csmall\@debian.org>\n";
   print "This is free software; see the source for copying conditions.  There is NO\n";
   print "warranty; not even for MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.\n";

=== modified file 'dh_make.1'
--- dh_make.1	2009-03-14 21:32:39 +0000
+++ dh_make.1	2009-04-03 04:35:12 +0000
@@ -1,6 +1,6 @@
 .TH DH_MAKE 8 "12 March 2009" "Debian Project" \" -*- nroff -*-
 .SH NAME
-dh_make \- Debianize a regular source archive
+dh_make \- prepare Debian packaging for an original source archive
 .SH SYNOPSIS
 .B dh_make
 [\-bnlsmiadh] [\-c license] [\-e address] [\-f file] [\-t directory]

