diff -durN ax25-apps-0.0.6/ax25ipd/kiss.c fixed/ax25-apps-0.0.6/ax25ipd/kiss.c
--- ax25-apps-0.0.6/ax25ipd/kiss.c	2001-10-08 20:03:12.000000000 -0500
+++ fixed/ax25-apps-0.0.6/ax25ipd/kiss.c	2010-12-07 18:35:19.444623640 -0600
@@ -72,7 +72,7 @@
 		if (c == FEND) {
 			if (ifcount > 0) {
 				/* Make sure that the control byte is zero */
-				if (*iframe == '\0' || *iframe == 0x10) {
+				if (1) {
 					/* Room for CRC in buffer? */
 					if (ifcount < (MAX_FRAME - 2)) {
 						stats.kiss_in++;
