--- cryptroot-script.orig	2011-05-10 02:48:10.000000000 +0200
+++ cryptroot-script	2011-12-25 21:18:49.200781323 +0100
@@ -274,11 +274,12 @@
 			fi
 		fi
 
-
-		if ! crypttarget="$crypttarget" cryptsource="$cryptsource" \
-		     $cryptkeyscript "$cryptkey" | $cryptcreate --key-file=- ; then
-			message "cryptsetup: cryptsetup failed, bad password or options?"
-			continue
+		if [ ! -e "$NEWROOT" ]; then
+			if ! crypttarget="$crypttarget" cryptsource="$cryptsource" \
+			 $cryptkeyscript "$cryptkey" | $cryptcreate --key-file=- ; then
+				message "cryptsetup: cryptsetup failed, bad password or options?"
+				continue
+			fi
 		fi
 
 		if [ ! -e "$NEWROOT" ]; then
