From: Lisandro Damián Nicanor Pérez Meyer <lisandro@debian.org>
Subject: Make FindRuby work on multiarch systems
Bug-Debian: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=730095
Author: Lisandro Damián Nicanor Pérez Meyer <lisandro@debian.org>
Forwarded: no
Last-Update: 2014-03-23

diff --git a/Modules/FindRuby.cmake b/Modules/FindRuby.cmake
index c02158f..bfc0f2a 100644
--- a/Modules/FindRuby.cmake
+++ b/Modules/FindRuby.cmake
@@ -94,7 +94,10 @@ if(RUBY_EXECUTABLE  AND NOT  RUBY_VERSION_MAJOR)
    _RUBY_CONFIG_VAR("archdir" RUBY_ARCH_DIR)
    _RUBY_CONFIG_VAR("arch" RUBY_ARCH)
    _RUBY_CONFIG_VAR("rubyhdrdir" RUBY_HDR_DIR)
-   _RUBY_CONFIG_VAR("libdir" RUBY_POSSIBLE_LIB_DIR)
+   _RUBY_CONFIG_VAR("archlibdir" RUBY_POSSIBLE_LIB_DIR)
+   if("${RUBY_POSSIBLE_LIB_DIR}" STREQUAL "")
+      _RUBY_CONFIG_VAR("libdir" RUBY_POSSIBLE_LIB_DIR)
+   endif
    _RUBY_CONFIG_VAR("rubylibdir" RUBY_RUBY_LIB_DIR)
 
    # site_ruby
