Index: wheezy/yaboot/yaboot.conf
===================================================================
--- wheezy/yaboot/yaboot.conf	(revision 2268)
+++ wheezy/yaboot/yaboot.conf	(working copy)
@@ -22,7 +22,7 @@
 image=/install/powerpc/vmlinux
 	label=expert
 	initrd=/install/powerpc/initrd.gz
-	append="DEBCONF_PRIORITY=low --"
+	append="priority=low --"
 	initrd-size=10240
 	read-only
 
@@ -51,7 +51,7 @@
 image=/install/powerpc64/vmlinux
 	label=expert64
 	initrd=/install/powerpc64/initrd.gz
-	append="DEBCONF_PRIORITY=low --"
+	append="priority=low --"
 	initrd-size=10240
 	read-only
 
