--- patch/gnus-sum.el.orig	2020-10-26 02:19:24.938566201 +0100
+++ patch/gnus-sum.el	2020-10-26 02:20:18.062310644 +0100
@@ -12834,6 +12834,7 @@
 	 (uncached (and gnus-summary-use-undownloaded-faces
                         (memq article gnus-newsgroup-undownloaded)
                         (not (memq article gnus-newsgroup-cached)))))
+    (defvar article)
     (let ((face (funcall (gnus-summary-highlight-line-0))))
       (unless (eq face (gnus-get-text-property-excluding-characters-with-faces beg 'face))
 	(gnus-put-text-property-excluding-characters-with-faces
