diff --git a/debmake/copyright.py b/debmake/copyright.py
index bca2764..28b3c24 100755
--- a/debmake/copyright.py
+++ b/debmake/copyright.py
@@ -67,6 +67,7 @@ def copyright(package_name, license_file_masks, cdata, xml_html_files, binary_fi
     text = '''\
 Format: https://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Name: {}
+Upstream-Contact: <preferred name and address to reach the upstream project>
 Source: <url://example.com>
 '''.format(package_name)
     if tutorial:
