diff --git a/pkgs.dbk b/pkgs.dbk
index 5a34b95..ca5847f 100644
--- a/pkgs.dbk
+++ b/pkgs.dbk
@@ -2770,8 +2770,9 @@ Make sure that you included an appropriate explanation in the changelog;
 </listitem>
 <listitem>
 <para>
-Make sure that you've written <literal>testing</literal> or
-<literal>testing-proposed-updates</literal> into your target distribution;
+Make sure that you've written the testing
+<link linkend="codenames">code name</link> (e.g. <literal>jessie</literal>)
+into your target distribution;
 </para>
 </listitem>
 <listitem>
