Index: apt-cacher-1.6.1/apt-cacher2
===================================================================
--- apt-cacher-1.6.1.orig/apt-cacher2	2007-12-11 09:48:16.000000000 -0400
+++ apt-cacher-1.6.1/apt-cacher2	2008-01-28 08:42:11.000000000 -0400
@@ -1226,6 +1226,9 @@
 	    unshift(@{$pathmap{$vhost}}, $hostcand);
 	}
 	else {
+	    # truncate $pkfd to overwrite possible HTML text from HTTP error
+	    truncate($pkfd, 0);
+	    sysseek($pkfd, 0, 0);
 	    goto RETRY_ACTION;
 	}
     }
