--- abcde.svn.range	2006-09-28 12:43:05.000000000 -0500
+++ abcde.svn.mkcue	2006-09-29 10:14:19.407635831 -0500
@@ -1657,13 +1657,11 @@
     echo REM DISCID $DISCID
     echo FILE \"dummy.wav\" WAVE
 
-    if [ $1 -ne 150 ] ; then
-        if [ $MODE = PREGAP ] ; then
-            OFFSET=$1
-        else
-            OFFSET=150
-        fi
-    fi
+	if [ $1 -ne 150 -a $MODE = PREGAP ] ; then
+		OFFSET=$1
+	else
+		OFFSET=150
+	fi
 
     i=1
     while [ $i -le "$TRACKS" ] ; do
