diff --git a/policy.sgml b/policy.sgml
index ec1acee..07ae48f 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -5055,7 +5055,7 @@ Replaces: mail-transport-agent
 	    <p>
 	      There is no Build-Depends-Arch; this role is essentially
 	      met with Build-Depends.  Anyone building the
-	      <tt>build-indep</tt> and binary-indep<tt></tt> targets is
+	      <tt>build-indep</tt> and <tt>binary-indep</tt> targets is
 	      assumed to be building the whole package, and therefore
 	      installation of all build dependencies is required.
 	    </p>
