--- /tmp/cryptsetup_1.7.3-3/lib/cryptsetup/cryptdisks.functions	2016-12-09 01:18:17.000000000 +0100
+++ /lib/cryptsetup/cryptdisks.functions	2016-12-25 19:02:23.179147532 +0100
@@ -203,6 +203,20 @@
 			fi
 			LUKSPARAMS="$LUKSPARAMS --key-slot $VALUE"
 			;;
+		keyfile-size)
+			if [ -z "$VALUE" ]; then
+				log_warning_msg "$dst: no value for keyfile-size option, skipping"
+				return 1
+			fi
+			LUKSPARAMS="$LUKSPARAMS --keyfile-size $VALUE"
+			;;
+		keyfile-offset)
+			if [ -z "$VALUE" ]; then
+				log_warning_msg "$dst: no value for keyfile-offset option, skipping"
+				return 1
+			fi
+			LUKSPARAMS="$LUKSPARAMS --keyfile-offset $VALUE"
+			;;
 		tcrypthidden)
 			TCRYPTPARAMS="$TCRYPTPARAMS --tcrypt-hidden"
 			;;
@@ -213,7 +227,7 @@
 
 		CRYPTTAB_OPTIONS="$CRYPTTAB_OPTIONS $PARAM"
 		[ -z "$VALUE" ] && VALUE="yes"
-		eval export CRYPTTAB_OPTION_$PARAM="\"$VALUE\""
+		eval export CRYPTTAB_OPTION_$(echo $PARAM | sed 's/-/_/g')="\"$VALUE\""
 	done
 	export CRYPTTAB_OPTIONS
 
