--- /tmp/cryptsetup_1.7.3-3/usr/share/initramfs-tools/hooks/cryptroot	2016-12-09 01:18:17.000000000 +0100
+++ /usr/share/initramfs-tools/hooks/cryptroot	2016-12-25 19:03:12.954987653 +0100
@@ -444,8 +444,15 @@
 			resumedev)
 				OPTIONS="$OPTIONS,$opt"
 				;;
+			keyfile-size=*)
+				OPTIONS="$OPTIONS,$opt"
+				;;
+			keyfile-offset=*)
+				OPTIONS="$OPTIONS,$opt"
+				;;
 			*)
 				# Presumably a non-supported option
+				echo "option not supported: $opt" >&2
 				;;
 		esac
 	done
@@ -473,25 +480,33 @@
 				key="/cryptroot-keyfiles/${target}.key"
 				;;
 			*)
-				key=$(readlink -e "$key")
+				# only resolve sym-links for files, not for disks
+				if [ "$key" = "${key%/dev/disk/*}" ] ; then
+					key=$(readlink -e "$key")
+				fi
 				# test whether $target is a root device (or parent of the root device)
 				if printf '%s' "$OPTIONS" | grep -Eq '^(.*,)?rootdev(,.*)?$'; then
-					echo "cryptsetup: WARNING: root target $target uses a key file, skipped" >&2
-					return 1
+					if [ "$key" = "${key%/dev/disk/*}" ] ; then
+						echo "cryptsetup: WARNING: root target $target uses a key file, skipped" >&2
+						return 1
+					else
+						echo "cryptsetup: NOTE: root target $target uses a device, $key" >&2 
+					fi
 				# test whether a) key file is not on root fs
 				#           or b) root fs is not encrypted
 				elif [ "$(stat -c %m -- "$key" 2>/dev/null)" != / ] || ! node_or_pv_is_in_crypttab $rootdevs; then
 					echo "cryptsetup: WARNING: $target's key file $key is not on an encrypted root FS, skipped" >&2
 					return 1
+				else
+					if printf '%s' "$OPTIONS" | grep -Eq '^(.*,)?resumedev(,.*)?$'; then
+						# we'll be able to decrypt the device, but won't be able to use it for resuming
+						echo "cryptsetup: WARNING: resume device $source uses a key file" >&2
+					fi
+					# prepend "/root" (to be substituted by the real root FS
+					# mountpoint "$rootmnt" in the boot script) to the
+					# absolute filename
+					key="/root$key"
 				fi
-				if printf '%s' "$OPTIONS" | grep -Eq '^(.*,)?resumedev(,.*)?$'; then
-					# we'll be able to decrypt the device, but won't be able to use it for resuming
-					echo "cryptsetup: WARNING: resume device $source uses a key file" >&2
-				fi
-				# prepend "/root" (to be substituted by the real root FS
-				# mountpoint "$rootmnt" in the boot script) to the
-				# absolute filename
-				key="/root$key"
 				;;
 		esac
 		OPTIONS="$OPTIONS,keyscript=cat"
