--- /tmp/cryptsetup_1.7.3-3/usr/share/initramfs-tools/scripts/local-top/cryptroot	2016-12-09 01:18:17.000000000 +0100
+++ /usr/share/initramfs-tools/scripts/local-top/cryptroot	2016-12-25 19:07:16.661745962 +0100
@@ -70,6 +70,8 @@
 	cryptkeyscript=""
 	cryptkey="" # This is only used as an argument to an eventual keyscript
 	cryptkeyslot=""
+	cryptkeyfilesize=""
+	cryptkeyfileoffset=""
 	crypttries=3
 	crypttcrypt=""
 	cryptveracrypt=""
@@ -124,6 +126,12 @@
 		keyslot=*)
 			cryptkeyslot=${x#keyslot=}
 			;;
+		keyfile-size=*)
+			cryptkeyfilesize=${x#keyfile-size=}
+			;;
+		keyfile-offset=*)
+			cryptkeyfileoffset=${x#keyfile-offset=}
+			;;
 		tries=*)
 			crypttries="${x#tries=}"
 			case "$crypttries" in
@@ -152,7 +160,7 @@
 			VALUE="${x#*=}"
 		fi
 		CRYPTTAB_OPTIONS="$CRYPTTAB_OPTIONS $PARAM"
-		eval export CRYPTTAB_OPTION_$PARAM="\"$VALUE\""
+		eval export CRYPTTAB_OPTION_$(echo $PARAM | sed 's/-/_/g')="\"$VALUE\""
 	done
 	export CRYPTTAB_OPTIONS
 
@@ -288,6 +296,12 @@
 	if [ -n "$cryptkeyslot" ]; then
 		cryptopen="$cryptopen --key-slot=$cryptkeyslot"
 	fi
+	if [ -n "$cryptkeyfilesize" ]; then
+		cryptopen="$cryptopen --keyfile-size=$cryptkeyfilesize"
+	fi
+	if [ -n "$cryptkeyfileoffset" ]; then
+		cryptopen="$cryptopen --keyfile-offset=$cryptkeyfileoffset"
+	fi
 	if /sbin/cryptsetup isLuks ${cryptheader:-$cryptsource} >/dev/null 2>&1; then
 		cryptopen="$cryptopen open --type luks $cryptsource $crypttarget --key-file=-"
 	elif [ "$crypttcrypt" = "yes" ]; then
