From 949bd770d7fd0d80ba0457f61c03b18d352b08dd Mon Sep 17 00:00:00 2001
From: Sven Joachim <svenjoac@gmx.de>
Date: Sat, 18 Feb 2023 09:09:31 +0100
Subject: [PATCH] di-utils-terminfo: Include xterm terminfo entry

Closes: #1026027
---
 debian/rules | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/rules b/debian/rules
index 703d210..50ab30c 100755
--- a/debian/rules
+++ b/debian/rules
@@ -17,14 +17,14 @@ export DEB_CFLAGS_MAINT_APPEND := -Wall -W -Os -fomit-frame-pointer
 CFLAGS := $(shell dpkg-buildflags --get CPPFLAGS; dpkg-buildflags --get CFLAGS)
 LDFLAGS := $(shell dpkg-buildflags --get LDFLAGS)

-TERMNAMES = a/ansi d/dumb s/screen
+TERMNAMES = a/ansi d/dumb s/screen x/xterm

 ifeq ($(DEB_HOST_ARCH_OS),linux)
 TERMNAMES += l/linux
 endif

 ifeq ($(DEB_HOST_ARCH_OS),kfreebsd)
-TERMNAMES += c/cons25 x/xterm
+TERMNAMES += c/cons25
 endif

 ifeq ($(DEB_HOST_ARCH_OS),hurd)
--
2.39.2

