diff -pur clamav-0.95.3+dfsg/clamd/scanner.c clamav-0.95.3+dfsg-octo/clamd/scanner.c
--- clamav-0.95.3+dfsg/clamd/scanner.c  2009-10-28 16:34:13.000000000 +0100
+++ clamav-0.95.3+dfsg-octo/clamd/scanner.c     2010-01-28 19:39:37.000000000 +0100
@@ -83,7 +83,7 @@ extern pthread_mutex_t reload_mutex;
 int scan_callback(struct stat *sb, char *filename, const char *msg, enum cli_ftw_reason reason, struct cli_ftw_cbdata *data)
 {
     struct scan_cb_data *scandata = data->data;
-    const char *virname;
+    const char *virname = NULL;
     int ret;
     int type = scandata->type;
     const struct optstruct *opt;
