diff -u eperl-2.2.14/eperl_perl5.c eperl-2.2.14/eperl_perl5.c
--- eperl-2.2.14/eperl_perl5.c
+++ eperl-2.2.14/eperl_perl5.c
@@ -194,6 +194,7 @@
     }
     IO_redirect_stderr(er);
 
+    PERL_SYS_INIT3(&myargc,&myargv,&env);
     my_perl = perl_alloc();   
     perl_construct(my_perl); 
     perl_init_i18nl10n(1);
diff -u eperl-2.2.14/debian/changelog eperl-2.2.14/debian/changelog
--- eperl-2.2.14/debian/changelog
+++ eperl-2.2.14/debian/changelog
@@ -1,3 +1,11 @@
+eperl (2.2.14-15.2) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Call PERL_SYS_INIT3() before perl_alloc(), fixing uninitialized lock
+    issues on hppa. (Closes: #494191)
+
+ -- Marc 'HE' Brockschmidt <he@debian.org>  Sat, 06 Sep 2008 21:30:21 +0200
+
 eperl (2.2.14-15.1) unstable; urgency=low
 
   * Non-maintainer upload.
