diff -Naru audacious-plugins-1.5.1.orig/debian/audacious-plugins-extra.install audacious-plugins-1.5.1/debian/audacious-plugins-extra.install
--- audacious-plugins-1.5.1.orig/debian/audacious-plugins-extra.install	2009-03-07 21:42:22.154460138 +0100
+++ audacious-plugins-1.5.1/debian/audacious-plugins-extra.install	2009-03-07 21:43:23.951683245 +0100
@@ -15,7 +15,6 @@
 usr/lib/audacious/Input/cuesheet.so
 usr/lib/audacious/Input/metronom.so
 usr/lib/audacious/Input/modplug.so
-usr/lib/audacious/Input/musepack.so
 usr/lib/audacious/Input/sexypsf.so
 usr/lib/audacious/Input/sid.so
 usr/lib/audacious/Input/timidity.so
diff -Naru audacious-plugins-1.5.1.orig/debian/rules audacious-plugins-1.5.1/debian/rules
--- audacious-plugins-1.5.1.orig/debian/rules	2009-03-07 21:42:36.358462532 +0100
+++ audacious-plugins-1.5.1/debian/rules	2009-03-07 21:43:09.338459693 +0100
@@ -40,6 +40,7 @@
 		    --disable-altivec \
 		    --disable-sse2 \
 		    --enable-chardet \
+		    --disable-musepack \
 		    CFLAGS="$(CFLAGS)"
 
 build: build-stamp
