Index: audacious-plugins-1.5.1/configure.ac
===================================================================
--- audacious-plugins-1.5.1.orig/configure.ac	2009-03-07 21:29:42.162465322 +0100
+++ audacious-plugins-1.5.1/configure.ac	2009-03-07 21:30:26.614460255 +0100
@@ -767,7 +767,7 @@
 
 if test "x$enable_musepack" = "xyes"; then
     PKG_CHECK_MODULES(TAGLIB, [taglib >= 1.4], [ 
-       AC_CHECK_HEADERS(mpcdec/config_types.h,, [
+       AC_CHECK_HEADERS(mpc/mpc_types.h,, [
            enable_musepack=no
        ])
     ], [enable_musepack=no])
Index: audacious-plugins-1.5.1/src/musepack/libmpc.h
===================================================================
--- audacious-plugins-1.5.1.orig/src/musepack/libmpc.h	2009-03-07 21:30:49.174461815 +0100
+++ audacious-plugins-1.5.1/src/musepack/libmpc.h	2009-03-07 21:33:27.058461767 +0100
@@ -21,7 +21,7 @@
 //libmpcdec headers
 #undef TRUE
 #undef FALSE
-#include <mpcdec/mpcdec.h>
+#include <mpc/mpcdec.h>
 
 //GTK+ headers
 #include <glib.h>
