--- apt-pkg/acquire-item.cc.orig	2010-01-23 00:17:35.683393726 +0530
+++ apt-pkg/acquire-item.cc	2010-01-23 00:16:41.631390753 +0530
@@ -824,8 +824,10 @@
       // name than DestFile because otherwise the http method will
       // send If-Range requests and there are too many broken servers
       // out there that do not understand them
-      LastGoodSig = DestFile+".reverify";
-      Rename(Final,LastGoodSig);
+      if (_config->FindB("APT::Get::Simulate") == false) {
+	      LastGoodSig = DestFile+".reverify";
+	      Rename(Final,LastGoodSig);
+      }
    }
 
    QueueURI(Desc);
