--- orig/doxygen-1.7.4/src/htmlgen.cpp	2011-03-26 16:11:31.000000000 +0000
+++ doxygen-1.7.4/src/htmlgen.cpp	2011-04-10 15:40:07.149419575 +0100
@@ -1082,6 +1082,7 @@
     fprintf(stderr,"Warning: Cannot open file %s for writing\n",fileName.data());
   }
 
+  if (Config_getBool("GENERATE_TREEVIEW") || Config_getBool("SEARCHENGINE"))
   {
     QFile f(dname+"/jquery.js");
     if (f.open(IO_WriteOnly))
