--- a/bash_completion.cowbuilder	2014-04-16 23:59:12.182898325 +0200
+++ b/bash_completion.cowbuilder	2014-04-16 23:59:30.678990039 +0200
@@ -14,7 +14,7 @@
     _get_comp_words_by_ref cur prev words
     initialcommand=${words[1]}
     initialcommand_options='--create --update --build --login --execute'
-    other_options='--dumpconfig --distribution --mirror --basepath --architecture'
+    other_options='--dumpconfig --distribution --mirror --basepath --architecture --configfile'
     distribution='sid sarge etch woody lenny squeeze'
 
     case $prev in
@@ -26,6 +26,10 @@
 	    _filedir -d
 	    return 0
 	    ;;
+	--configfile)
+	    _filedir
+	    return 0
+	    ;;
 	*)
 	    ;;
     esac
