diff -Naur a/separableconvolution.hxx b/separableconvolution.hxx
--- a/separableconvolution.hxx	2021-12-14 12:47:58.561112628 +0100
+++ b/separableconvolution.hxx	2021-12-14 12:41:47.490153379 +0100
@@ -1409,11 +1409,7 @@
         {}
 
         ~InitProxy()
-#ifndef _MSC_VER
-            throw(PreconditionViolation)
-#elif _MSC_VER >= 1900
             noexcept(false)
-#endif
         {
             vigra_precondition(count_ == 1 || count_ == sum_,
                   "Kernel1D::initExplicitly(): "
diff -Naur a/stdconvolution.hxx b/stdconvolution.hxx
--- a/stdconvolution.hxx	2021-12-14 12:47:58.561112628 +0100
+++ b/stdconvolution.hxx	2021-12-14 12:43:12.728503532 +0100
@@ -792,11 +792,7 @@
         {}
 
         ~InitProxy()
-#ifndef _MSC_VER
-            throw(PreconditionViolation)
-#elif _MSC_VER >= 1900
             noexcept(false)
-#endif
         {
             vigra_precondition(count_ == 1 || count_ == sum_,
                                "Kernel2D::initExplicitly(): "
