diff -Naur a/conf.py.cmake2.in b/conf.py.cmake2.in
--- a/conf.py.cmake2.in	2021-12-14 12:38:10.000000000 +0100
+++ b/conf.py.cmake2.in	2021-12-14 14:13:42.030342506 +0100
@@ -23,7 +23,7 @@
 def _getargspec_workaround(*args, **kw):
     try:
         return _original_getargspec(*args, **kw)
-    except TypeError, e:
+    except (TypeError, e):
         if str(e).startswith('arg is not a Python function'):
             return inspect.ArgSpec([], None, None, None)
         else:
diff -Naur a/conf.py.in b/conf.py.in
--- a/conf.py.in	2021-12-14 12:38:10.000000000 +0100
+++ b/conf.py.in	2021-12-14 14:14:07.576442837 +0100
@@ -22,7 +22,7 @@
 def _getargspec_workaround(*args, **kw):
     try:
         return _original_getargspec(*args, **kw)
-    except TypeError, e:
+    except (TypeError, e):
         if str(e).startswith('arg is not a Python function'):
             return inspect.ArgSpec([], None, None, None)
         else:
