Index: src/deliver.c
===================================================================
RCS file: /home/cvs/exim/exim-src/src/deliver.c,v
retrieving revision 1.41
diff -u -r1.41 deliver.c
--- src/deliver.c	22 Jan 2007 16:29:54 -0000	1.41
+++ src/deliver.c	23 Jan 2007 01:44:01 -0000
@@ -6572,6 +6572,9 @@
       readconf_printtime(time(NULL) - received_time));
   else
     log_write(0, LOG_MAIN, "Completed");
+
+  /* Unset deliver_freeze so that we won't try to move the spool files further down */
+  deliver_freeze = FALSE;
   }
 
 /* If there are deferred addresses, we are keeping this message because it is
