--- /usr/bin/bdf2psf	2009-11-20 23:56:56.000000000 +1100
+++ bdf2psf-debian	2009-11-28 21:39:39.000000000 +1100
@@ -376,25 +376,23 @@
 	    next;
 	}
 	if (/^ENDCHAR/) {
-	    if ($rows > 0) {
-		$rows == $height - ($beforebox + $afterbox) / matrix_row_size ()
-		    or die ("$0: $bdf: invalid number of rows $rows " 
-			    ."at line $current_line\n");
-		if ($u == -123456) {
-		    die ("$0: $bdf: missing ENCODING before ENDCHAR " 
-			 ."at line $current_line\n");
+	    $rows == $height - ($beforebox + $afterbox) / matrix_row_size ()
+		or die ("$0: $bdf: invalid number of rows $rows " 
+			."at line $current_line\n");
+	    if ($u == -123456) {
+		die ("$0: $bdf: missing ENCODING before ENDCHAR " 
+		     ."at line $current_line\n");
+	    }
+	    if (! defined $gliphs{$u}) {
+		if ($beforebox < 0) {
+		    @gliph_bytes = @gliph_bytes[-$beforebox..$#gliph_bytes];
 		}
-		if (! defined $gliphs{$u}) {
-		    if ($beforebox < 0) {
-			@gliph_bytes = @gliph_bytes[-$beforebox..$#gliph_bytes];
-		    }
-		    if ($afterbox < 0) {
-			@gliph_bytes = @gliph_bytes[0 .. $#gliph_bytes+$afterbox];
-		    }
-		    $gliphs{$u} = [ (0) x $beforebox,
-				    @gliph_bytes,
-				    (0) x $afterbox];
+		if ($afterbox < 0) {
+		    @gliph_bytes = @gliph_bytes[0 .. $#gliph_bytes+$afterbox];
 		}
+		$gliphs{$u} = [ (0) x $beforebox,
+				@gliph_bytes,
+				(0) x $afterbox];
 	    }
 	}
 	if (/^ENDFONT$/) {
