diff -Nru ./kernel/tests/powerpc/ydl_powerstation.cpuinfo ../base-installer-1.113/kernel/tests/powerpc/ydl_powerstation.cpuinfo
--- ./kernel/tests/powerpc/ydl_powerstation.cpuinfo	1969-12-31 19:00:00.000000000 -0500
+++ ../base-installer-1.113/kernel/tests/powerpc/ydl_powerstation.cpuinfo	2010-11-07 15:04:24.000000000 -0500
@@ -0,0 +1,23 @@
+processor       : 0
+cpu             : PPC970MP, altivec supported
+clock           : 2500.000000MHz
+revision        : 1.1 (pvr 0044 0101)
+
+processor       : 1
+cpu             : PPC970MP, altivec supported
+clock           : 2500.000000MHz
+revision        : 1.1 (pvr 0044 0101)
+
+processor       : 2
+cpu             : PPC970MP, altivec supported
+clock           : 2500.000000MHz
+revision        : 1.1 (pvr 0044 0101)
+
+processor       : 3
+cpu             : PPC970MP, altivec supported
+clock           : 2500.000000MHz
+revision        : 1.1 (pvr 0044 0101)
+
+timebase        : 14318278
+platform        : Maple
+model           : TSS,7047-191
diff -Nru ./kernel/tests/powerpc/ydl_powerstation.test ../base-installer-1.113/kernel/tests/powerpc/ydl_powerstation.test
--- ./kernel/tests/powerpc/ydl_powerstation.test	1969-12-31 19:00:00.000000000 -0500
+++ ../base-installer-1.113/kernel/tests/powerpc/ydl_powerstation.test	2010-11-07 15:06:18.000000000 -0500
@@ -0,0 +1,17 @@
+subarch chrp_ibm
+cpuinfo ydl_powerstation.cpuinfo
+majors 2.6
+flavour powerpc64
+kernel-2.6 linux-image-2.6-powerpc64
+usable \
+  linux-image-2.6-powerpc64 \
+  linux-image-2.6.32-5-powerpc64
+unusable \
+  linux-image-2.6-powerpc \
+  linux-image-2.6-powerpc-miboot \
+  linux-image-2.6-powerpc-smp \
+  linux-image-2.6-prep \
+  linux-image-2.6.32-5-powerpc \
+  linux-image-2.6.32-5-powerpc-miboot \
+  linux-image-2.6.32-5-powerpc-smp \
+  linux-image-2.6.32-5-prep
