diff -uNrp amavisd-new-2.6.4~/debian/control amavisd-new-2.6.4/debian/control
--- amavisd-new-2.6.4~/debian/control	2009-08-17 14:43:47.657773678 -0700
+++ amavisd-new-2.6.4/debian/control	2009-08-20 17:20:38.000000000 -0700
@@ -4,6 +4,6 @@ Priority: extra
 Maintainer: Brian May <bam@snoopy.debian.net>
 Uploaders: Henrique de Moraes Holschuh <hmh@debian.org>, Alexander Wirt <formorer@debian.org>
-Build-Depends: debhelper (>= 5), libmilter-dev, po-debconf, dpatch (>= 2), patch (>= 2.5.9-3bpo1), dpkg-dev (>= 1.13.19), autoconf2.13
+Build-Depends: debhelper (>= 5), libmilter-dev, po-debconf, dpatch (>= 2), patch (>= 2.5.9-3bpo1), dpkg-dev (>= 1.13.19), autoconf2.13, hardening-wrapper
 Standards-Version: 3.8.2
 
 Package: amavisd-new
diff -uNrp amavisd-new-2.6.4~/debian/rules amavisd-new-2.6.4/debian/rules
--- amavisd-new-2.6.4~/debian/rules	2009-08-17 14:43:47.657773678 -0700
+++ amavisd-new-2.6.4/debian/rules	2009-08-20 17:20:48.000000000 -0700
@@ -6,6 +6,7 @@
 
 # Uncomment this to turn on verbose mode.
 #export DH_VERBOSE=1
+export DEB_BUILD_HARDENING=1
 
 export DH_ALWAYS_EXCLUDE=CVS
 
