diff -Naur dictionaries-common_orig/scripts/debhelper/installdeb.in dictionaries-common/scripts/debhelper/installdeb.in
--- dictionaries-common_orig/scripts/debhelper/installdeb.in	2014-02-07 17:01:23.676496366 +0000
+++ dictionaries-common/scripts/debhelper/installdeb.in	2014-02-07 18:36:36.043091943 +0000
@@ -445,11 +445,11 @@
 
 	# Install extra auto-scripts for auto-compat handling
 	autoscript ($package, "preinst", "preinst-compatfile-$class",
-		    "s/#COMPAT#/$auto_compats/");
+		    "s/#COMPAT#/$auto_compats/;s/#HASHES#/$auto_hashes/");
 	autoscript ($package, "postinst", "postinst-compatfile-$class",
 		    "s/#COMPAT#/$auto_compats/");
 	autoscript ($package, "postrm", "postrm-varlibrm-$class",
-		    "s/#VARLIBRM#/$varlibrm/");
+		    "s/#VARLIBRM#/$varlibrm/;s/#HASHES#/$auto_hashes/");
 
 	# Make sure /var/lib/{a,i}spell directory is available.
 	doit ("install", "-d", $var_lib_dir);
@@ -460,10 +460,6 @@
 	  # Not for extra hashes only for the clean list
 	  next if ( defined $auto_extra_hash_basenames{$hash} );
 	  $hash = $hash . '.' .$hash_extension{$class};
-	  unless ( -e "$usr_lib_dir/$hash" ){
-	    print STDERR "$program: Setting \"$usr_lib_dir/$hash\" symlink.\n" if $debug;
-	    doit ("ln", "-sf", "/var/lib/$class/$hash", "$usr_lib_dir/$hash");
-	  }
 	}
       }
     }
diff -Naur dictionaries-common_orig/scripts/maintainer/postrm-varlibrm.in dictionaries-common/scripts/maintainer/postrm-varlibrm.in
--- dictionaries-common_orig/scripts/maintainer/postrm-varlibrm.in	2014-02-07 17:01:23.680496445 +0000
+++ dictionaries-common/scripts/maintainer/postrm-varlibrm.in	2014-02-07 18:41:29.355945209 +0000
@@ -1,8 +1,15 @@
 dico_class="[I:ispell:I][A:aspell:A]"
 dico_varlib_remove="#VARLIBRM#"
+dico_hashes="#HASHES#"
 
 case "$1" in abort-install|remove)
 	if [ -d "/var/lib/$dico_class" ]; then
 	    ( cd "/var/lib/$dico_class" && rm -f $dico_varlib_remove )
 	fi
+
+	for i in $dico_hashes; do
+	    if [ -h "/usr/lib/$dico_class/$i" ]; then
+	        rm -f "/usr/lib/$dico_class/$i"
+	    fi
+	done
 esac
diff -Naur dictionaries-common_orig/scripts/maintainer/preinst-compatfile.in dictionaries-common/scripts/maintainer/preinst-compatfile.in
--- dictionaries-common_orig/scripts/maintainer/preinst-compatfile.in	2014-02-07 17:01:23.680496445 +0000
+++ dictionaries-common/scripts/maintainer/preinst-compatfile.in	2014-02-07 18:41:02.299499121 +0000
@@ -1,8 +1,14 @@
 dico_class="[I:ispell:I][A:aspell:A]"
 dico_compat="#COMPAT#"
+dico_hashes="#HASHES#"
 
 [ -d "/var/lib/$dico_class" ] || mkdir -p "/var/lib/$dico_class"
 
 for i in $dico_compat; do
     >"/var/lib/$dico_class/$i"
 done
+
+for i in $dico_hashes; do
+   [ -e /usr/lib/$dico_class/$i ] || ln -s /var/lib/$dico_class/$i /usr/lib/$dico_class/$i 
+done
+
