--- analog-6.0.orig/src/globals.c	2004-12-19 13:51:30.000000000 +0000
+++ analog-6.0/src/globals.c	2012-05-28 00:40:51.000000000 +0000
@@ -1266,7 +1266,6 @@
   else
     COPYSTR(commandname, comname);
 #ifndef NOFOLLOW
-  commandpath = (char *)xmalloc(PATH_MAX);
   done = FALSE;
   if (strpbrk(commandname, PATHSEPS) == NULL &&
       (path = getenv("PATH")) != NULL) {
@@ -1275,12 +1274,12 @@
     for (s = strtok(pathcp, ":"); !done && s != NULL; s = strtok(NULL, ":")) {
       ENSURE_LEN(t, l, strlen(s) + strlen(commandname) + 2);
       sprintf(t, "%s%c%s", s, DIRSEP, commandname);
-      if (realpath(t, commandpath) != NULL && access(commandpath, X_OK) == 0 &&
+      if ((commandpath = realpath(t, NULL)) != NULL && access(commandpath, X_OK) == 0 &&
 	  stat(commandpath, &buf) == 0 && S_ISREG(buf.st_mode))
 	done = TRUE;
     }
   }
-  else if (realpath(commandname, commandpath) != NULL)
+  else if ((commandpath = realpath(commandname, NULL)) != NULL)
     done = TRUE;
   if (!done)
     strcpy(commandpath, commandname);
