diff -ur digikam-2.6.0~beta2/core/libs/widgets/graphicsview/regionframeitem.cpp digikam-2.6.0~beta2.new/core/libs/widgets/graphicsview/regionframeitem.cpp
--- digikam-2.6.0~beta2/core/libs/widgets/graphicsview/regionframeitem.cpp	2012-03-05 12:48:20.000000000 +0000
+++ digikam-2.6.0~beta2.new/core/libs/widgets/graphicsview/regionframeitem.cpp	2012-04-05 20:28:00.000000000 +0000
@@ -639,8 +639,8 @@
 {
     const QSizeF maxSize = parentDImgItem()->boundingRect().size();
     const QPointF point  = mapToParent(event->pos());
-    qreal posX           = qBound(0., point.x(), maxSize.width());
-    qreal posY           = qBound(0., point.y(), maxSize.height());
+    qreal posX           = qBound(qreal(0.), point.x(), maxSize.width());
+    qreal posY           = qBound(qreal(0.), point.y(), maxSize.height());
     QRectF r             = rect();
 
     // Adjust edge
diff -ur digikam-2.6.0~beta2/core/utilities/gpssearch/gpssearchview.cpp digikam-2.6.0~beta2.new/core/utilities/gpssearch/gpssearchview.cpp
--- digikam-2.6.0~beta2/core/utilities/gpssearch/gpssearchview.cpp	2012-03-05 12:48:18.000000000 +0000
+++ digikam-2.6.0~beta2.new/core/utilities/gpssearch/gpssearchview.cpp	2012-04-05 21:34:00.000000000 +0000
@@ -422,7 +422,7 @@
 
     // NOTE: coordinates as lon1, lat1, lon2, lat2 (or West, North, East, South)
     // as left/top, right/bottom rectangle.
-    QList<qreal> coordinatesList = QList<qreal>() <<
+    QList<double> coordinatesList = QList<double>() <<
                                    coordinates.first.lon() << coordinates.first.lat() <<
                                    coordinates.second.lon() << coordinates.second.lat();
 
Only in digikam-2.6.0~beta2: download-repos~
diff -ur digikam-2.6.0~beta2/extra/kipi-plugins/photolayoutseditor/widgets/canvas/CropWidgetItem.cpp digikam-2.6.0~beta2.new/extra/kipi-plugins/photolayoutseditor/widgets/canvas/CropWidgetItem.cpp
--- digikam-2.6.0~beta2/extra/kipi-plugins/photolayoutseditor/widgets/canvas/CropWidgetItem.cpp	2012-03-05 12:48:08.000000000 +0000
+++ digikam-2.6.0~beta2.new/extra/kipi-plugins/photolayoutseditor/widgets/canvas/CropWidgetItem.cpp	2012-04-05 18:55:55.000000000 +0000
@@ -340,8 +340,8 @@
         QPointF dif = event->scenePos() - event->lastScenePos();
         temp.translate(dif);
 
-        temp.translate( qMin(maxRect.right()-temp.right(),0.0), qMin(maxRect.bottom()-temp.bottom(),0.0) );
-        temp.translate( qMax(maxRect.left()-temp.left(),0.0), qMax(maxRect.top()-temp.top(),0.0) );
+        temp.translate( qMin(maxRect.right()-temp.right(),qreal(0.0)), qMin(maxRect.bottom()-temp.bottom(),qreal(0.0)) );
+        temp.translate( qMax(maxRect.left()-temp.left(),qreal(0.0)), qMax(maxRect.top()-temp.top(),qreal(0.0)) );
     }
     // Size change
     else
Only in digikam-2.6.0~beta2: .gitslave~
