--- bash_completion.dpkg-dist	2006-03-26 00:27:24.000000000 +0100
+++ bash_completion	2006-03-26 00:39:17.000000000 +0100
@@ -2108,7 +2108,7 @@
 	
 	if [ "$cur" != show ]; then
 	    for (( i=0; i < ${#COMP_WORDS[@]}-1; i++ )); do
-		if [[ ${COMP_WORDS[i]} == @(add|depends|dotty|policy|rdepends|show?(pkg|src|)) ]]; then
+		if [[ ${COMP_WORDS[i]} == @(add|depends|dotty|policy|rdepends|madison|show?(pkg|src|)) ]]; then
 		    special=${COMP_WORDS[i]}
 		fi
 	    done
@@ -2156,7 +2156,7 @@
 		COMPREPLY=( $( compgen -W 'add gencaches show showpkg showsrc \
 				stats dump dumpavail unmet search search \
 				depends rdepends pkgnames dotty xvcg \
-				policy' -- $cur ) )
+				policy madison' -- $cur ) )
 
 	fi
 
@@ -2200,7 +2200,7 @@
 		     -Z -v --verbose'
 
 	for (( i=0; i < ${#COMP_WORDS[@]}-1; i++ )); do
-	    if [[ ${COMP_WORDS[i]} == @(install|reinstall|hold|unhold|markauto|unmarkauto|dist-upgrade|download|show|forbid-version|purge|remove) ]]; then
+	    if [[ ${COMP_WORDS[i]} == @(install|reinstall|hold|unhold|markauto|unmarkauto|dist-upgrade|download|show|forbid-version|purge|remove|changelog) ]]; then
 		special=${COMP_WORDS[i]}
 	    fi
 	    #exclude some mutually exclusive options
@@ -2210,7 +2210,7 @@
 
 	if [[ -n "$special" ]]; then
 	   case $special in
-	       @(install|hold|markauto|unmarkauto|dist-upgrade|download|show))
+	       @(install|hold|markauto|unmarkauto|dist-upgrade|download|show|changelog))
 		   COMPREPLY=( $( apt-cache pkgnames $cur 2> /dev/null ) )
 		   return 0
 		   ;;
@@ -2253,7 +2253,7 @@
 				       autoclean install reinstall remove \
 				       hold unhold purge markauto unmarkauto \
 				       dist-upgrade download search show \
-				       forbid-version' -- $cur ) )
+				       forbid-version changelog' -- $cur ) )
 	fi
 
 
