diff -urpN debhelper-7.0.15.orig/dh_auto_clean debhelper-7.0.15/dh_auto_clean
--- debhelper-7.0.15.orig/dh_auto_clean	2008-06-04 12:38:54.000000000 -0600
+++ debhelper-7.0.15/dh_auto_clean	2010-06-03 11:25:33.000000000 -0600
@@ -45,7 +45,7 @@ if (-e "Makefile" || -e "makefile" || -e
 	foreach my $target (qw{distclean realclean clean}) {
 		# Use make -n to check to see if the target would do
 		# anything. There's no good way to test if a target exists.
-		my $ret=`$ENV{MAKE} -s -n $target 2>/dev/null`;
+		my $ret=`$ENV{MAKE} -s -n --no-print-directory $target 2>/dev/null`;
 		chomp $ret;
 		if (length $ret) {
 			doit($ENV{MAKE}, $target, @{$dh{U_PARAMS}});
diff -urpN debhelper-7.0.15.orig/dh_auto_install debhelper-7.0.15/dh_auto_install
--- debhelper-7.0.15.orig/dh_auto_install	2008-06-25 10:24:21.000000000 -0600
+++ debhelper-7.0.15/dh_auto_install	2010-06-03 11:25:55.000000000 -0600
@@ -70,7 +70,7 @@ if (-e "Makefile" || -e "makefile" || -e
 	foreach my $target (qw{install}) {
 		# Use make -n to check to see if the target would do
 		# anything. There's no good way to test if a target exists.
-		my $ret=`$ENV{MAKE} -s -n $target 2>/dev/null`;
+		my $ret=`$ENV{MAKE} -s -n --no-print-directory $target 2>/dev/null`;
 		chomp $ret;
 		if (length $ret) {
 			doit($ENV{MAKE}, $target,
diff -urpN debhelper-7.0.15.orig/dh_auto_test debhelper-7.0.15/dh_auto_test
--- debhelper-7.0.15.orig/dh_auto_test	2008-05-19 10:40:23.000000000 -0600
+++ debhelper-7.0.15/dh_auto_test	2010-06-03 11:25:44.000000000 -0600
@@ -46,7 +46,7 @@ if (-e "Makefile" || -e "makefile" || -e
 	foreach my $target (qw{test check}) {
 		# Use make -n to check to see if the target would do
 		# anything. There's no good way to test if a target exists.
-		my $ret=`$ENV{MAKE} -s -n $target 2>/dev/null`;
+		my $ret=`$ENV{MAKE} -s -n --no-print-directory $target 2>/dev/null`;
 		chomp $ret;
 		if (length $ret) {
 			doit($ENV{MAKE}, $target, @{$dh{U_PARAMS}});
