--- /tmp/global.php	2012-01-03 08:27:12.142254227 +0100
+++ /usr/share/cacti/site/include/global.php	2012-01-03 09:22:33.000000000 +0100
@@ -30,16 +30,16 @@
 
 */
 
-/* Default database settings*/
-$database_type = "mysql";
-require('/etc/cacti/debian.php');
-
 /* Default session name - Session name must contain alpha characters */
 $cacti_session_name = "Cacti";
 
 /* Include configuration */
 include(dirname(__FILE__) . "/config.php");
 
+/* Default database settings*/
+$database_type = "mysql";
+require('/etc/cacti/debian.php');
+
 if (isset($config["cacti_version"])) {
 	die("Invalid include/config.php file detected.");
 	exit;
