Index: profiles/kde-profile/kdeglobals
===================================================================
--- profiles/kde-profile/kdeglobals     (revision 211)
+++ profiles/kde-profile/kdeglobals     (working copy)
@@ -1,3 +1,2 @@
 [Directories]
 dir_config=/usr/share/desktop-base/profiles/kde-profile/share/config/
-dir_wallpaper=/usr/share/images/desktop-base
Index: Makefile
===================================================================
--- Makefile    (revision 211)
+++ Makefile    (working copy)
@@ -24,6 +24,10 @@
        mkdir -p $(DESTDIR)/usr/share/images/desktop-base
        $(INSTALL) $(BACKGROUNDS) $(DESTDIR)/usr/share/images/desktop-base
        cd $(DESTDIR)/usr/share/images/desktop-base && ln -s $(DEFAULT_BACKGROUND) default
+       mkdir -p $(DESTDIR)/usr/share/wallpapers
+       for wp in $(filter-out $(wildcard backgrounds/*grub*),$(BACKGROUNDS)) ; do \
+           ln -s /usr/share/images/desktop-base/`basename $${wp}` /usr/share/wallpapers/ ; \
+       done
        # splash files
        $(INSTALL) $(SPLASH) $(DESTDIR)/usr/share/images/desktop-base
        # emblems
