Index: argyll-1.8.3+repack/spectro/dispwin.c
===================================================================
--- argyll-1.8.3+repack.orig/spectro/dispwin.c	2016-09-04 12:17:35.000000000 +1000
+++ argyll-1.8.3+repack/spectro/dispwin.c	2016-09-04 12:49:15.892851404 +1000
@@ -2341,7 +2341,10 @@
 
 	cd_found = NULL;
 
-	if ((cd_found = dlopen("libcolordcompat.so", RTLD_LAZY)) != NULL) {
+	/* Load from private library path. Load from /usr/lib/argyll under the assumption that
+	 * dispcal isn't going to be foreign-arch.
+	 */
+	if ((cd_found = dlopen("/usr/lib/argyll/libcolordcompat.so", RTLD_LAZY)) != NULL) {
 
 		cd_edid_install_profile = dlsym(cd_found, "cd_edid_install_profile");
 		cd_edid_remove_profile = dlsym(cd_found, "cd_edid_remove_profile");
