--- /etc/bash_completion
+++ /etc/bash_completion
@@ -622,7 +622,7 @@
 
 	# don't complete for things like killall, ssh and mysql if it's
 	# the standalone command, rather than the init script
-	[[ ${COMP_WORDS[0]} != @(*init.d/!(functions|~)|service) ]] && return 0
+	[[ ${COMP_WORDS[0]} != @(*init.d/!(functions|~)|service|invoke-rc.d) ]] && return 0
 
 	# don't complete past 2nd token
 	[ $COMP_CWORD -gt 2 ] && return 0
@@ -630,7 +630,7 @@
 	[ -d /etc/rc.d/init.d ] && sysvdir=/etc/rc.d/init.d \
 				|| sysvdir=/etc/init.d
 
-	if [[ $COMP_CWORD -eq 1 ]] && [[ $prev == "service" ]]; then
+	if [[ $COMP_CWORD -eq 1 ]] && [[ $prev == @(service|invoke-rc.d) ]]; then
 		_services
 	else
 		COMPREPLY=( $( compgen -W '`sed -ne "y/|/ /; \

