--- apt.old/configure.in	2007-10-31 14:47:45.000000000 +0000
+++ apt-0.7.9/configure.in	2007-11-07 13:58:05.000000000 +0000
@@ -196,8 +196,14 @@
 dnl AC_CHECK_PROG(YODL_MAN,yodl2man,"yes","")
 
 ah_NUM_PROCS
-rc_GLIBC_VER
 rc_LIBSTDCPP_VER
+if test "$GLIBC_VER" == ""; then
+  AC_MSG_CHECKING([for glibc version])
+  VER=`grep __GLIBC_MINOR__ /usr/include/features.h | grep define | cut -f 3`
+  GLIBC_VER="-libc6.$VER"
+  AC_MSG_RESULT([$GLIBC_VER])
+fi
+AC_SUBST(GLIBC_VER)
 ah_GCC3DEP
 
 dnl It used to be that the user could select translations and that could get
