diff --git a/debian/control b/debian/control
index 9ac5681..fc1dde8 100644
--- a/debian/control
+++ b/debian/control
@@ -9,7 +9,7 @@ Build-Depends-Indep: docbook-xsl (>= 1.71.0), dblatex (>= 0.2), xsltproc, libxml
  texlive-lang-cyrillic, texlive-lang-french, texlive-lang-german, texlive-xetex,
  xmlto, zip
 Build-Depends: debhelper (>= 8), dpkg-dev (>= 1.16.1~)
-Vcs-Git: git://anonscn.debian.org/collab-maint/developers-reference.git
+Vcs-Git: git://anonscm.debian.org/collab-maint/developers-reference.git
 Vcs-Browser: http://anonscm.debian.org/gitweb/?p=collab-maint/developers-reference.git
 
 Package: developers-reference
