diff -ur /binutils-2.28/debian/changelog /tmp/buildd/binutils/binutils-2.28.fixed/debian/changelog
--- /binutils-2.28/debian/changelog	2017-04-10 03:00:18.000000000 +0000
+++ /tmp/buildd/binutils/binutils-2.28.fixed/debian/changelog	2017-04-06 03:13:35.636904884 +0000
@@ -834,7 +834,7 @@
   * Snapshot, taken from the trunk 20140814.
     - Add ld options --push-state and --pop-state.
   * binutils-hppa64: Set architecture to hppa.
-  * Add aarch64_ilp32-linux-gnu multiarch directories to the search path.
+  * Add aarch64-linux-gnu_ilp32 multiarch directories to the search path.
 
  -- Matthias Klose <doko@debian.org>  Thu, 14 Aug 2014 13:28:40 +0200
 
--- /binutils-2.28/debian/patches/129_multiarch_libpath.patch	2017-04-10 03:00:18.000000000 +0000
+++ /tmp/buildd/binutils/binutils-2.28.fixed/debian/patches/129_multiarch_libpath.patch	2017-04-06 03:15:25.242512047 +0000
@@ -32,10 +32,10 @@
 +      multiarch_name=aarch64-linux-gnu
 +      ;;
 +    aarch64linux32)
-+      multiarch_name=aarch64_ilp32-linux-gnu
++      multiarch_name=aarch64-linux-gnu_ilp32
 +      ;;
 +    aarch64linux32b)
-+      multiarch_name=aarch64_be_ilp32-linux-gnu
++      multiarch_name=aarch64_be-linux-gnu_ilp32
 +      ;;
 +    aarch64linuxb)
 +      multiarch_name=aarch64_be-linux-gnu
diff -ur /binutils-2.28/debian/rules /tmp/buildd/binutils/binutils-2.28.fixed/debian/rules
--- /binutils-2.28/debian/rules	2017-04-10 03:00:18.000000000 +0000
+++ /tmp/buildd/binutils/binutils-2.28.fixed/debian/rules	2017-04-06 14:24:20.425121857 +0000
@@ -381,7 +381,7 @@
 DEB_TARGET_MULTIARCH32_mips64r6el	= mipsisa32r6el-linux-gnu
 DEB_TARGET_MULTIARCHN32_mips64r6el	= mipsisa64r6el-linux-gnuabin32
 
-DEB_TARGET_MULTIARCH32_arm64		= aarch64_ilp32-linux-gnu
+DEB_TARGET_MULTIARCH32_arm64		= aarch64-linux-gnu_ilp32
 
 SET_MULTIARCH_ENV = \
 	DEB_TARGET_MULTIARCH=$(call _multiarch,$1) \
@@ -628,6 +628,7 @@
 multiarch_targets = \
 	aarch64-linux-gnu \
 	aarch64_be-linux-gnu \
+	aarch64-linux-gnu_ilp32 \
 	alpha-linux-gnu \
 	arm-linux-gnueabi \
 	hppa-linux-gnu \
