diff -Nru enigma-1.10~~pre-alpha+r2210/debian/changelog enigma-1.10~~pre-alpha+r2210/debian/changelog
--- enigma-1.10~~pre-alpha+r2210/debian/changelog	2011-07-03 18:14:10.000000000 +0200
+++ enigma-1.10~~pre-alpha+r2210/debian/changelog	2012-04-04 19:56:28.000000000 +0200
@@ -1,3 +1,13 @@
+enigma (1.10~~pre-alpha+r2210-1.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Fix "removal of enigma makes files disappear from enigma-data":
+    remove the duplicate .desktop file in debian/rules; merge upstream and
+    Debian .desktop files; set Breaks/Replaces against to previous version of
+    enigma-data. (Closes: #661474)
+
+ -- gregor herrmann <gregoa@debian.org>  Wed, 04 Apr 2012 19:55:15 +0200
+
 enigma (1.10~~pre-alpha+r2210-1) unstable; urgency=low
 
   * New SVN checkout, r2210
diff -Nru enigma-1.10~~pre-alpha+r2210/debian/control enigma-1.10~~pre-alpha+r2210/debian/control
--- enigma-1.10~~pre-alpha+r2210/debian/control	2011-07-03 18:12:52.000000000 +0200
+++ enigma-1.10~~pre-alpha+r2210/debian/control	2012-04-04 19:50:35.000000000 +0200
@@ -7,7 +7,8 @@
 
 Package: enigma
 Architecture: any
-Replaces: enigma-data
+Breaks: enigma-data (<< 1.10~~pre-alpha+r2210-1.1)
+Replaces: enigma-data (<< 1.10~~pre-alpha+r2210-1.1)
 Depends: ${shlibs:Depends}, ${misc:Depends}, enigma-data (= ${source:Version})
 Recommends: enigma-level-previews, enigma-doc
 Description: A game where you control a marble with the mouse
diff -Nru enigma-1.10~~pre-alpha+r2210/debian/enigma.desktop enigma-1.10~~pre-alpha+r2210/debian/enigma.desktop
--- enigma-1.10~~pre-alpha+r2210/debian/enigma.desktop	2009-12-08 02:12:56.000000000 +0100
+++ enigma-1.10~~pre-alpha+r2210/debian/enigma.desktop	2012-04-04 19:49:38.000000000 +0200
@@ -2,8 +2,10 @@
 Version=1.0
 
 Name=Enigma
+Name[eo]=Enigmo
 
 Comment=Uncover pairs of stones while navigating obstacles using a marble
+Comment[eo]=Malkovru parojn de ŝtonojn dum vi navigas obstaklojn kun globeto
 
 TryExec=enigma
 Exec=enigma
@@ -13,4 +15,4 @@
 StartupNotify=false
 
 Type=Application
-Categories=Application;Game;LogicGame;
+Categories=Game;LogicGame;
diff -Nru enigma-1.10~~pre-alpha+r2210/debian/rules enigma-1.10~~pre-alpha+r2210/debian/rules
--- enigma-1.10~~pre-alpha+r2210/debian/rules	2009-12-08 03:02:32.000000000 +0100
+++ enigma-1.10~~pre-alpha+r2210/debian/rules	2012-04-04 19:59:57.000000000 +0200
@@ -9,6 +9,10 @@
 DEB_CONFIGURE_EXTRA_FLAGS := --datadir=\$${prefix}/share/games --bindir=\$${prefix}/games --docdir=\$${prefix}/share/doc/enigma --enable-optimize
 
 binary-post-install/enigma-data::
+	# remove duplicate .desktop file
+	-rm debian/enigma-data/usr/share/applications/enigma.desktop
+	-rmdir --ignore-fail-on-non-empty --parents debian/enigma-data/usr/share/applications
+
 	-rm debian/enigma-data/usr/share/games/enigma/fonts/DejaVuSansCondensed.ttf
 	-rm debian/enigma-data/usr/share/games/enigma/fonts/vera_sans.ttf
 	# yes, using DejaVuSans instead of Vera Sans, one dependency less
