diff -urNpd dvdisaster-0.72.1~/scsi-layer.c dvdisaster-0.72.1/scsi-layer.c
--- dvdisaster-0.72.1~/scsi-layer.c	2016-09-24 14:04:44.000000000 -0500
+++ dvdisaster-0.72.1/scsi-layer.c	2016-09-24 14:16:10.529774227 -0500
@@ -1008,7 +1008,7 @@ static int query_bd(DeviceHandle *dh, in
 
    if(!strncmp((char*)&buf[4+8], "BDO", 3))
    {  dh->typeDescr = g_strdup("BD-ROM");
-      dh->subType = UNSUPPORTED;
+      dh->subType = BD;
    }
 
    if(!strncmp((char*)&buf[4+8], "BDW", 3))
