From 8e9d1d5f1629877a7308bd5292feaceff307410f Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Mon, 4 Aug 2014 14:14:21 +0200
Subject: [PATCH 1/4] Switch from libparted0debian1 to libparted2

---
 makehurddir.sh | 4 ++--
 packages/gnu   | 2 +-
 2 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/makehurddir.sh b/makehurddir.sh
index 8caf06a..4f3670e 100755
--- a/makehurddir.sh
+++ b/makehurddir.sh
@@ -48,8 +48,8 @@ if [ -e $TARGET/var/cache/apt/archives/libbz2-1.0_* ] ; then
         extract libbz2-1.0
 fi
 
-if [ -e $TARGET/var/cache/apt/archives/libparted0debian1_* ] ; then
-        extract libparted0debian1
+if [ -e $TARGET/var/cache/apt/archives/libparted2_* ] ; then
+        extract libparted2
 fi
 
 x_feign_install dpkg
diff --git a/packages/gnu b/packages/gnu
index c7bc014..ae805a9 100644
--- a/packages/gnu
+++ b/packages/gnu
@@ -1,6 +1,6 @@
 gnumach-image-1-486
 hurd
-libparted0debian1
+libparted2
 libuuid1
 libc0.3
 inetutils-tools
-- 
2.0.1.570.gc6b0718

