diff -ur docbook-to-man-2.0.0.orig/debian/control docbook-to-man-2.0.0/debian/control
--- docbook-to-man-2.0.0.orig/debian/control	2012-03-08 16:58:18.000000000 +0000
+++ docbook-to-man-2.0.0/debian/control	2012-03-08 17:17:55.000000000 +0000
@@ -11,6 +11,7 @@
 Package: docbook-to-man
 Architecture: any
 Depends: ${shlibs:Depends}, ${misc:Depends}, docbook, sp | opensp
+Multi-Arch: foreign
 Description: converter from DocBook SGML into roff man macros
  docbook-to-man is a batch converter that transforms UNIX-style manpages from
  the DocBook SGML format into nroff/troff man macros.
