--- /usr/bin/dh_gconf	2009-05-18 18:50:52.000000000 +0200
+++ dh_gconf	2009-05-29 13:45:44.000000000 +0200
@@ -97,7 +97,6 @@
 		if ($schemas ne '') {
 			autoscript($package,"postinst","postinst-gconf","s%#SCHEMAS#%$schemas%");
 			autoscript($package,"prerm","prerm-gconf","s%#SCHEMAS#%$schemas%");
-			autoscript($package,"postrm","postrm-gconf","s%#SCHEMAS#%$schemas%");
 			addsubstvar($package, "misc:Depends", "gconf2 (>= 2.10.1-2)") unless $gconf_dep;
 		}
 	}
