Index: /current/host/trunk/debian-grip/trunk/dscextract
===================================================================
--- /current/host/trunk/debian-grip/trunk/dscextract	(revision 8083)
+++ /current/host/trunk/debian-grip/trunk/dscextract	(revision 8170)
@@ -72,5 +72,5 @@
 fi
 
-if DEBIANTARGZ=$(egrep '^ [0-9a-f]{32,64} [0-9]+ [^ ]+\.debian\.tar\.(gz|bz2)$' "$DSC") ; then
+if DEBIANTARGZ=$(egrep '^ [0-9a-f]{32,64} [0-9]+ [^ ]+\.debian\.tar\.(gz|bz2|xz)$' "$DSC") ; then
 	case $FILE in
 		debian/*)
@@ -80,4 +80,5 @@
 				*.gz) ZIP=--gzip ;;
 				*.bz2) ZIP=--bzip2 ;;
+				*.xz) ZIP=--xz ;;
 			esac
 			tar --extract --to-stdout $ZIP --file "$DSCDIR/$DEBIANTARGZ" "$FILE" > "$WORKDIR/output" 2>/dev/null || :
