From ebe01be7ccb62c2c76f4036d103c77c82eecd166 Mon Sep 17 00:00:00 2001
From: Adam Borowski <kilobyte@angband.pl>
Date: Thu, 16 May 2019 17:10:35 +0200
Subject: [PATCH] Stop cron spam if lvm2 is not installed.

Signed-off-by: Adam Borowski <kilobyte@angband.pl>
---
 scrub/e2scrub_all.cron.in | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/scrub/e2scrub_all.cron.in b/scrub/e2scrub_all.cron.in
index 7d42c3f2..e0e5addc 100644
--- a/scrub/e2scrub_all.cron.in
+++ b/scrub/e2scrub_all.cron.in
@@ -1,2 +1,2 @@
-30 3 * * 0 root test -e /run/systemd/system || @pkglibdir@/e2scrub_all_cron
-10 3 * * * root test -e /run/systemd/system || @root_sbindir@/e2scrub_all -A -r
+30 3 * * 0 root test -e /run/systemd/system || test ! -x /sbin/lvcreate || @pkglibdir@/e2scrub_all_cron
+10 3 * * * root test -e /run/systemd/system || test ! -x /sbin/lvcreate || @root_sbindir@/e2scrub_all -A -r
-- 
2.20.1

