diff -u -r a/m4/ax_check_glu.m4 b/m4/ax_check_glu.m4
--- a/m4/ax_check_glu.m4	2015-02-23 23:00:00.000000000 +0000
+++ b/m4/ax_check_glu.m4	2015-08-14 12:18:29.064421280 +0100
@@ -316,7 +316,7 @@
                          [ax_check_glu_have_headers="yes";break],
                          [ax_check_glu_have_headers_headers="no"],
 			 [_AX_CHECK_GLU_INCLUDES_DEFAULT()])
-        _AX_CHECK_GL_RESTORE_FLAGS()
+        _AX_CHECK_GLU_RESTORE_FLAGS()
 	AC_LANG_POP([C])
 	AC_CACHE_CHECK([for OpenGL Utility (GLU) headers],[ax_cv_check_glu_have_headers],
                	       [ax_cv_check_glu_have_headers="${ax_check_glu_have_headers}"])
diff -u -r a/m4/ax_check_glx.m4 b/m4/ax_check_glx.m4
--- a/m4/ax_check_glx.m4	2015-02-23 23:00:00.000000000 +0000
+++ b/m4/ax_check_glx.m4	2015-08-14 12:18:48.260513289 +0100
@@ -259,7 +259,7 @@
                          [ax_check_glx_have_headers="yes";break],
                          [ax_check_glx_have_headers_headers="no"],
 			 [_AX_CHECK_GLX_INCLUDES_DEFAULT()])
-        _AX_CHECK_GL_RESTORE_FLAGS()
+        _AX_CHECK_GLX_RESTORE_FLAGS()
 	AC_LANG_POP([C])
 	AC_CACHE_CHECK([for OpenGL Utility (GLX) headers],[ax_cv_check_glx_have_headers],
 		       [ax_cv_check_glx_have_headers="${ax_check_glx_have_headers}"])
