--- debian/external_deps/fetk/punc/src/vf2c/uninit-old.c	2022-06-03 17:08:45.698221199 +0800
+++ debian/external_deps/fetk/punc/src/vf2c/uninit.c	2022-06-03 17:15:06.365211794 +0800
@@ -148,6 +148,16 @@
 #endif
 #endif

+
+#if defined(__riscv) && defined(__riscv_xlen) && __riscv_xlen == 64
+/* Masks for interrupts.  */
+#define _FPU_MASK_V     0x0800  /* Invalid operation */
+#define _FPU_MASK_Z     0x0400  /* Division by zero  */
+#define _FPU_MASK_O     0x0200  /* Overflow          */
+#define _FPU_MASK_U     0x0100  /* Underflow         */
+#define _FPU_MASK_I     0x0080  /* Inexact operation */
+#endif
+
 #ifdef MSpc
 #define IEEE0_done
 #include "float.h"
@@ -178,7 +188,7 @@
 	}
 #endif /* MSpc */

-#if defined (__mips) && !defined (__linux__)	/* must link with -lfpe */
+#if defined (__mips) && !defined (__linux__)   /* must link with -lfpe */
 #define IEEE0_done
 /* code from Eric Grosse */
 #include <stdlib.h>
