From 3cbaa5973e34c34a59183626254cad07187f26cf Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Fri, 25 Nov 2016 11:08:54 +0800
Subject: [PATCH] Make APT::Periodic::Verbose 2 match the documentation

---
 debian/apt.systemd.daily | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/apt.systemd.daily b/debian/apt.systemd.daily
index d034d8c..03d41ba 100755
--- a/debian/apt.systemd.daily
+++ b/debian/apt.systemd.daily
@@ -320,8 +320,8 @@ fi
 VERBOSE=0
 eval $(apt-config shell VERBOSE APT::Periodic::Verbose)
 debug_echo "verbose level $VERBOSE"
-if [ "$VERBOSE" -le 2 ]; then
-    # quiet for 0,1,2
+if [ "$VERBOSE" -le 1 ]; then
+    # quiet for 0/1
     XSTDOUT=">/dev/null"
     XSTDERR="2>/dev/null"
     XAPTOPT="-qq"
-- 
2.10.2

