From d5bf504efdb5b433145ab0de76557d2c8b3e5ece Mon Sep 17 00:00:00 2001
From: Alessandro Ghedini <alessandro@ghedini.me>
Date: Mon, 28 Jul 2014 17:20:56 +0200
Subject: [PATCH] Enable debian/control comment stripping

Closes: #734785
---
 script/dh_golang | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/script/dh_golang b/script/dh_golang
index 6f248ad..57dbc59 100755
--- a/script/dh_golang
+++ b/script/dh_golang
@@ -43,7 +43,7 @@ my $built_using;
 
 # TODO: switch to Dpkg::Control::Info, maybe?
 my $control_parser = Parse::DebControl->new();
-my $control_data = $control_parser->parse_file('./debian/control');
+my $control_data = $control_parser->parse_file('./debian/control', { stripComments => 1 });
 my $source_data = $control_data->[0];
 if (!defined($source_data)) {
     error('Could not parse ./debian/control');
-- 
2.0.1

