diff -ur old/cron-3.0pl1/debian/changelog cron-3.0pl1/debian/changelog
--- old/cron-3.0pl1/debian/changelog	2016-03-08 14:33:16.000000000 +0100
+++ cron-3.0pl1/debian/changelog	2016-03-08 14:32:51.976319657 +0100
@@ -1,3 +1,10 @@
+cron (3.0pl1-128.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * make sure cron is started last and stopped first. Closes: #767016
+
+ -- Harald Dunkel <harri@afaics.de>  Tue, 08 Mar 2016 14:12:41 +0100
+
 cron (3.0pl1-128) unstable; urgency=medium
 
   * d/cron.service: Use KillMode=process to kill only the daemon.
diff -ur old/cron-3.0pl1/debian/cron.init cron-3.0pl1/debian/cron.init
--- old/cron-3.0pl1/debian/cron.init	2016-03-08 14:33:16.000000000 +0100
+++ cron-3.0pl1/debian/cron.init	2016-03-08 14:11:38.048543606 +0100
@@ -5,8 +5,8 @@
 # Provides:          cron
 # Required-Start:    $remote_fs $syslog $time
 # Required-Stop:     $remote_fs $syslog $time
-# Should-Start:      $network $named slapd autofs ypbind nscd nslcd winbind
-# Should-Stop:       $network $named slapd autofs ypbind nscd nslcd winbind
+# Should-Start:      $all
+# Should-Stop:       $all
 # Default-Start:     2 3 4 5
 # Default-Stop:
 # Short-Description: Regular background program processing daemon
diff -ur old/cron-3.0pl1/debian/cron.service cron-3.0pl1/debian/cron.service
--- old/cron-3.0pl1/debian/cron.service	2016-03-08 14:33:16.000000000 +0100
+++ cron-3.0pl1/debian/cron.service	2016-03-08 14:12:29.118294423 +0100
@@ -7,6 +7,7 @@
 ExecStart=/usr/sbin/cron -f $EXTRA_OPTS
 IgnoreSIGPIPE=false
 KillMode=process
+Type=idle
 
 [Install]
 WantedBy=multi-user.target
