diff -Nur emoslib-4.3.9/debian/changelog emoslib-4.3.9-repr/debian/changelog
--- emoslib-4.3.9/debian/changelog	2016-03-22 19:28:00.000000000 -0400
+++ emoslib-4.3.9-repr/debian/changelog	2016-04-17 10:42:19.868582724 -0400
@@ -1,3 +1,9 @@
+emoslib (2:4.3.9-1.0~reproducible1) UNRELEASED; urgency=medium
+
+  * Solve reproducibility issues with different shells
+
+ -- boyska <piuttosto@logorroici.org>  Sun, 17 Apr 2016 10:41:40 -0400
+
 emoslib (2:4.3.9-1) unstable; urgency=medium
 
   * New upstream release.
diff -Nur emoslib-4.3.9/debian/rules emoslib-4.3.9-repr/debian/rules
--- emoslib-4.3.9/debian/rules	2016-03-22 19:28:00.000000000 -0400
+++ emoslib-4.3.9-repr/debian/rules	2016-04-17 10:41:02.746387904 -0400
@@ -96,4 +96,5 @@
 	# Change default from emos to emos_shared to link against shared libs in metview, etc.
 	grep -v LIBEMOS_SELF_LIBRARIES ${CMAKE_DIR}/libemos-config.cmake > ${CMAKE_DIR}/tmp
 	mv ${CMAKE_DIR}/tmp ${CMAKE_DIR}/libemos-config.cmake  
-	echo "\nset( LIBEMOS_SELF_LIBRARIES    "emos_shared" )" >> ${CMAKE_DIR}/libemos-config.cmake 
+	echo >> ${CMAKE_DIR}/libemos-config.cmake 
+	echo "set( LIBEMOS_SELF_LIBRARIES    "emos_shared" )" >> ${CMAKE_DIR}/libemos-config.cmake 
