--- Packages-arch-specific.orig	2010-06-06 10:37:07.000000000 +0100
+++ Packages-arch-specific	2010-06-06 10:52:20.000000000 +0100
@@ -93,6 +93,11 @@
 %gnumach: hurd-i386 i386 kfreebsd-i386				      # hurd kernel
 %google-perftools: amd64 i386 ia64 powerpc      		      # not yet ported to other archs
 %gpart: i386 hurd-i386 ia64 alpha arm armel mipsel amd64	      # little endian specific
+%gpe-conf: !hurd-i386 !kfreebsd-amd64 !kfreebsd-i386		      # requires a linux kernel
+gpe-announce: !hurd-i386 !kfreebsd-amd64 !kfreebsd-i386	      # requires libsoundgen which needs a linux kernel
+gpe-bluetooth: !hurd-i386 !kfreebsd-amd64 !kfreebsd-i386	      # requires bluez
+gpe-calendar: !hurd-i386 !kfreebsd-amd64 !kfreebsd-i386	      # requires libsoundgen which needs a linux kernel
+gpe-mininet: !hurd-i386 !kfreebsd-amd64 !kfreebsd-i386		      # requires a linux kernel
 gprolog: i386 mips mipsel sparc alpha powerpc amd64		      # from source
 %grub: i386 hurd-i386 amd64 lpia kfreebsd-i386 kfreebsd-amd64         # i386 boot loader
 %grub2: i386 kfreebsd-i386 hurd-i386 amd64 kfreebsd-amd64 powerpc ppc64 lpia sparc mipsel # bootloader
@@ -134,6 +139,7 @@
 %libsem: hurd-i386						      # hurd specific
 %libsendmail-milter-perl: !ia64					      # ANAIS
 %libsmbios: i386 amd64 ia64 lpia				      # ANAIS
+%libsoundgen: !hurd-i386 !kfreebsd-amd64 !kfreebsd-i386	      # Needs a linux kernel
 %libparportled: i386						      # [?] ANAIS, <sys/io.h>
 %libpfm3-3.2: ia64 amd64 i386 mips mipsel powerpc		      # [ANAIS]
 %libspe2: powerpc pcp64						      # PowerPC specific
@@ -352,6 +358,7 @@
 %tpb: i386 amd64						      # x86 specific
 %tra: i386							      # [ANAIS]
 %tremulous: !m68k						      # No m68k support
+%tslib: !hurd-i386 !kfreebsd-amd64 !kfreebsd-i386		      # needs a linux kernel
 %tuxonice-userui: i386 ia64 powerpc ppc64 amd64                       # [ANAIS]
 %tvtime: i386 amd64 lpia					      # i386 (wine) specific
 %u++: i386 ia64 amd64						      # per rcbilson@plg.uwaterloo.ca
