diff -urN orig/bison-2.4.1.dfsg//debian/bison.install bison-2.4.1.dfsg//debian/bison.install
--- orig/bison-2.4.1.dfsg//debian/bison.install	1970-01-01 02:00:00.000000000 +0200
+++ bison-2.4.1.dfsg//debian/bison.install	2011-09-19 11:57:59.215952683 +0300
@@ -0,0 +1,6 @@
+/usr/share/aclocal/bison-i18n.m4
+/usr/share/locale/*/*
+/usr/share/bison/*
+/usr/share/bison/*/*
+/usr/share/man/man1/*
+/usr/bin/*
diff -urN orig/bison-2.4.1.dfsg//debian/changelog bison-2.4.1.dfsg//debian/changelog
--- orig/bison-2.4.1.dfsg//debian/changelog	2011-09-15 11:19:02.000000000 +0300
+++ bison-2.4.1.dfsg//debian/changelog	2011-09-19 12:11:27.925953057 +0300
@@ -1,3 +1,9 @@
+bison (1:2.4.1.dfsg-3.1) unstable; urgency=low
+
+  * add multiarch
+
+ -- Riku Voipio <riku.voipio@iki.fi>  Mon, 19 Sep 2011 12:11:12 +0300
+
 bison (1:2.4.1.dfsg-3) unstable; urgency=low
 
   * Move upstream yacc.1 man page to bison.yacc.1 and let the alternatives
diff -urN orig/bison-2.4.1.dfsg//debian/control bison-2.4.1.dfsg//debian/control
--- orig/bison-2.4.1.dfsg//debian/control	2011-09-15 11:19:02.000000000 +0300
+++ bison-2.4.1.dfsg//debian/control	2011-09-19 11:34:27.865952033 +0300
@@ -4,13 +4,14 @@
 Maintainer: Chuan-kai Lin <cklin@debian.org>
 Homepage: http://www.gnu.org/software/bison/
 Standards-Version: 3.8.3.0
-Build-Depends: debhelper (>= 7), gettext, m4 (>= 1.4-14), autotools-dev, cdbs
+Build-Depends: debhelper (>= 8.1.3), gettext, m4 (>= 1.4-14), autotools-dev, cdbs (>=0.4.93)
 
 Package: bison
 Section: devel
 Priority: optional
 Architecture: any
-Depends: m4, ${shlibs:Depends}, ${misc:Depends}
+Depends: m4, ${shlibs:Depends}, ${misc:Depends}, liby-dev (= ${binary:Version})
+Multi-Arch: foreign
 Suggests: bison-doc
 Description: A parser generator that is compatible with YACC
  Bison is a general-purpose parser generator that converts a
@@ -23,3 +24,19 @@
  grammars ought to work with Bison with no change.  Anyone familiar with
  Yacc should be able to use Bison with little trouble.  Documentation of
  the program is in the bison-doc package.
+
+Package: liby-dev
+Section: devel
+Priority: optional
+Architecture: any
+Depends: bison
+Multi-Arch: same
+Description: A parser generator that is compatible with YACC - development library
+ Bison is a general-purpose parser generator that converts a
+ grammar description for an LALR(1) context-free grammar into a C
+ program to parse that grammar.  Once you are proficient with Bison, you
+ may use it to develop a wide range of language parsers, from those used
+ in simple desk calculators to complex programming languages.
+ .
+ This package provides the liby.a file including functions needed by yacc parsers
+
diff -urN orig/bison-2.4.1.dfsg//debian/liby-dev.install bison-2.4.1.dfsg//debian/liby-dev.install
--- orig/bison-2.4.1.dfsg//debian/liby-dev.install	1970-01-01 02:00:00.000000000 +0200
+++ bison-2.4.1.dfsg//debian/liby-dev.install	2011-09-19 11:37:27.175952115 +0300
@@ -0,0 +1 @@
+/usr/lib/*/liby.a
diff -urN orig/bison-2.4.1.dfsg//debian/rules bison-2.4.1.dfsg//debian/rules
--- orig/bison-2.4.1.dfsg//debian/rules	2011-09-15 11:19:02.000000000 +0300
+++ bison-2.4.1.dfsg//debian/rules	2011-09-19 11:36:25.205952087 +0300
@@ -4,6 +4,7 @@
 include /usr/share/cdbs/1/class/autotools.mk
 include /usr/share/cdbs/1/rules/simple-patchsys.mk
 
+DEB_CONFIGURE_EXTRA_FLAGS += --libdir=\$${prefix}/lib/$(DEB_HOST_MULTIARCH)
 SRCDIR = $(CURDIR)
 DESTDIR = $(CURDIR)/debian/bison
 
