Index: pkgs.dbk
===================================================================
--- pkgs.dbk	(revision 10373)
+++ pkgs.dbk	(working copy)
@@ -843,10 +843,9 @@
 <para>
 When you become aware of a security-related bug in a Debian package, whether or
 not you are the maintainer, collect pertinent information about the problem,
-and promptly contact the security team, preferably by filing a ticket in
-their Request Tracker.
-See <ulink url="http://wiki.debian.org/rt.debian.org#Security_Team"></ulink>.
-Alternatively you may email &email-security-team;.
+and promptly contact the security team by emailing &email-security-team;. If
+desired, email can be encrypted with the Debian Security Contact key, see 
+<ulink url="https://www.debian.org/security/faq#contact"></ulink> for details.
 <emphasis role="strong">DO NOT UPLOAD</emphasis> any packages for
 <literal>stable</literal> without contacting the team.  Useful information
 includes, for example:
