From 14fbe59b41c15ddfd638bcbde8656f374669ecb1 Mon Sep 17 00:00:00 2001
From: Sebastian Andrzej Siewior <sebastian@breakpoint.cc>
Date: Fri, 16 Jan 2015 21:14:55 +0100
Subject: [PATCH] dupload.conf: drop volatile

Signed-off-by: Sebastian Andrzej Siewior <sebastian@breakpoint.cc>
---
 dupload.conf | 9 ---------
 1 file changed, 9 deletions(-)

diff --git a/dupload.conf b/dupload.conf
index 8d58ccf553a7..a4aaa5894f56 100644
--- a/dupload.conf
+++ b/dupload.conf
@@ -84,15 +84,6 @@ $cfg{'bpo'} = {
     dinstall_runs => 1,
 };
 
-# see http://www.debian.org/devel/debian-volatile/ for more information
-$cfg{'volatile'} = {
-        fqdn => "volatile-master.debian.org",
-        incoming => "/pub/UploadQueue/",
-        # files pass on to dinstall on ftp-master which sends emails itself
-        dinstall_runs => 1,
-        passive => 1,
-};
-
 # For Delayed uploads use this. You can use 0-day, which is uploaded
 # one hour before dinstall runs.
 $delay = (defined($ENV{DEBDELAY}) ? $ENV{DEBDELAY} : 7);
-- 
2.1.4

