--- Esetroot.c.orig	2011-03-16 16:16:59.000000000 +1100
+++ Esetroot.c	2013-11-06 06:05:10.000000000 +1100
@@ -180,6 +180,7 @@
     }
     imlib_context_set_display(Xdisplay);
     imlib_context_set_visual(DefaultVisual(Xdisplay, DefaultScreen(Xdisplay)));
+    imlib_context_set_colormap(DefaultColormap(Xdisplay, DefaultScreen(Xdisplay)));
     im = imlib_load_image_immediately(fname);
     if (!im) {
         fprintf(stderr, "%s:  Unable to load image file \"%s\".\n", *argv, fname);
