--- asciidoc/01_tutorial.txt	2013-11-09 22:39:22.465045726 +0100
+++ /dev/fd/63	2013-11-17 16:12:42.823365199 +0100
@@ -1121,7 +1121,7 @@
 Default values of environment variables are initially set by the PAM system and then some of them may be reset by some application programs.
 
 - The display manager such as `gdm3` resets environment variables.
-- The shell in its start up codes resets environment variables in "`\~/bash_profile`" and "`\~/.bashrc`".
+- The shell in its start up codes resets environment variables in "`\~/.bash_profile`" and "`\~/.bashrc`".
 
 ==== The "`$LANG`" variable
 
