diff -Nru cdebconf-0.143/debian/cdebconf.install cdebconf-0.143+nmu1/debian/cdebconf.install
--- cdebconf-0.143/debian/cdebconf.install	2006-01-27 01:59:05.000000000 +0000
+++ cdebconf-0.143+nmu1/debian/cdebconf.install	2009-07-06 13:40:51.000000000 +0100
@@ -1,3 +1,12 @@
 deb/etc/* etc
-deb/usr/lib/cdebconf usr/lib
+deb/usr/lib/cdebconf/debconf-copydb usr/lib/cdebconf
+deb/usr/lib/cdebconf/debconf-loadtemplate usr/lib/cdebconf
+deb/usr/lib/cdebconf/dpkg-reconfigure usr/lib/cdebconf
+deb/usr/lib/cdebconf/debconf-communicate usr/lib/cdebconf
+deb/usr/lib/cdebconf/db/* usr/lib/cdebconf/db
+deb/usr/lib/cdebconf/debconf usr/lib/cdebconf/
+deb/usr/lib/cdebconf/frontend/newt.so usr/lib/cdebconf/frontend
+deb/usr/lib/cdebconf/frontend/text.so usr/lib/cdebconf/frontend
+deb/usr/lib/cdebconf/libdebconf.so usr/lib/cdebconf
+deb/usr/lib/cdebconf/debconf-dumpdb usr/lib/cdebconf
 deb/usr/share usr
diff -Nru cdebconf-0.143/debian/changelog cdebconf-0.143+nmu1/debian/changelog
--- cdebconf-0.143/debian/changelog	2009-07-02 14:32:28.000000000 +0100
+++ cdebconf-0.143+nmu1/debian/changelog	2009-07-06 13:27:52.000000000 +0100
@@ -1,3 +1,10 @@
+cdebconf (0.143+nmu1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Drop gtk frontend from default cdebconf package (Closes: #480899)
+
+ -- Neil Williams <codehelp@debian.org>  Mon, 06 Jul 2009 13:27:27 +0100
+
 cdebconf (0.143) unstable; urgency=low
 
   [ Otavio Salvador ]
