--- bidentd-1.1.4.orig/debian/bidentd.postinst	2010-04-29 16:48:25.000000000 +0200
+++ bidentd-1.1.4/debian/bidentd.postinst	2020-03-11 13:31:55.260658323 +0100
@@ -26,7 +26,7 @@
  if [ "x$opt" != "xUSER" ]; then
   CONF_STR="ident\tstream\ttcp\tnowait\troot\t/usr/sbin/bidentd bidentd $opt"
   update-inetd --pattern "/usr/sbin/bidentd" --remove ident
-  update-inetd --pattern "/usr/sbin/bidentd" --group INFO --add "$CONF_STR"
+  update-inetd --group INFO --add "$CONF_STR"
  fi
  
  # Re-enable the manual settings too
