diff -u basket-0.5.0/debian/control basket-0.5.0/debian/control
--- basket-0.5.0/debian/control
+++ basket-0.5.0/debian/control
@@ -2,7 +2,7 @@
 Section: kde
 Priority: optional
 Maintainer: Jose Luis Tallon <jltallon@adv-solutions.net>
-Build-Depends: debhelper (>= 4.0.0), autotools-dev, libqt3-mt-dev, kdebase-dev(>= 3.2.2), kdelibs4-dev(>= 3.2.2)
+Build-Depends: debhelper (>= 4.0.0), autotools-dev, kdebase-dev(>= 3.2.2)
 Standards-Version: 3.6.2
 
 Package: basket
diff -u basket-0.5.0/debian/rules basket-0.5.0/debian/rules
--- basket-0.5.0/debian/rules
+++ basket-0.5.0/debian/rules
@@ -32,7 +32,7 @@
 		ln -sf /usr/share/misc/config.guess config.guess
 
 	# Add here commands to configure the package.
-	CFLAGS="$(CFLAGS)" ./configure --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE) \
+	CFLAGS="$(CFLAGS)" LDFLAGS="-Wl,--as-needed" ./configure --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE) \
 		--prefix=/usr --with-qt-dir=/usr/share/qt3 \
 		--with-qt-includes=/usr/include/qt3 \
 		--with-qt-libraries=/usr/share/qt3/lib \
diff -u basket-0.5.0/debian/changelog basket-0.5.0/debian/changelog
--- basket-0.5.0/debian/changelog
+++ basket-0.5.0/debian/changelog
@@ -1,3 +1,12 @@
+basket (0.5.0-5.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Added -Wl,--as-needed as ldflags (Closes: #349480)
+  * Trimmed build-depends to not include stuff from
+    kdebase dependency line
+
+ -- Sune Vuorela <debian@pusling.com>  Fri, 21 Apr 2006 21:25:43 +0000
+
 basket (0.5.0-5) unstable; urgency=low
 
   * Fix crash on cutting multiline item (Closes: #339874)
