diff -uprN debian/cron.daily debian/cron.daily
--- debian/cron.daily	2009-02-13 19:52:49.000000000 +0100
+++ debian/cron.daily	2009-02-16 13:57:13.000000000 +0100
@@ -3,6 +3,7 @@ set -e
 if [ -x /usr/sbin/etckeeper ] && [ -e /etc/etckeeper/etckeeper.conf ]; then
 	. /etc/etckeeper/etckeeper.conf
 	if [ "$AVOID_DAILY_AUTOCOMMITS" != "1" ]; then
+		USER=${USER:-root}
 		if etckeeper unclean; then
 			etckeeper commit "daily autocommit" >/dev/null
 		fi
