From 3872647babf807be7c6d51d4a866ebe9a28d6137 Mon Sep 17 00:00:00 2001
From: Sven Joachim <svenjoac@gmx.de>
Date: Fri, 13 Nov 2015 21:08:14 +0100
Subject: [PATCH] Remove call to update-python-modules from postinst script

---
 debian/changelog | 7 +++++++
 debian/postinst  | 2 --
 2 files changed, 7 insertions(+), 2 deletions(-)

diff --git a/debian/changelog b/debian/changelog
index 70ffa87..5a10735 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,10 @@
+apt-xapian-index (0.47+nmu1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Remove call to update-python-modules from postinst script. Closes: #793681
+
+ -- Sven Joachim <svenjoac@gmx.de>  Fri, 13 Nov 2015 21:06:50 +0100
+
 apt-xapian-index (0.47) unstable; urgency=low
 
   [ Enrico Zini ]
diff --git a/debian/postinst b/debian/postinst
index fad3683..616f42d 100644
--- a/debian/postinst
+++ b/debian/postinst
@@ -28,8 +28,6 @@ case "$1" in
                 if [ ! -x /usr/sbin/policy-rc.d ] || /usr/sbin/policy-rc.d apt-xapian-index start
                 then
                         echo "apt-xapian-index: Building new index in background..."
-                        # Ensure that we can use our module. See #537376
-                        update-python-modules -p
                         $IONICE nice /usr/sbin/update-apt-xapian-index --force --quiet &
                 fi
         fi
-- 
2.6.2

