--- /tmp/cvsweb/cvsweb	2008-06-02 20:10:01.000000000 +0200
+++ /usr/lib/cgi-bin/cvsweb	2008-06-02 21:02:32.000000000 +0200
@@ -4458,7 +4459,7 @@ sub TIEHANDLE
               crc    => 0,
               len    => 0,
             };
-  my ($header) = pack("c10",
+  my ($header) = pack("C10",
                       MAGIC1, MAGIC2, Compress::Zlib::Z_DEFLATED(),
                       0, 0, 0, 0, 0, 0, OSCODE);
   print {$o->{handle}} $header;
