--- console-setup.orig/debian/config.proto	2006-11-13 16:55:11.000000000 +0000
+++ console-setup/debian/config.proto	2007-04-13 08:42:16.000000000 +0100
@@ -1585,7 +1585,9 @@
 	    fi
 	    ;;
 	6)
-	    db_default console-setup/codeset "$default_codeset"
+	    if [ "$default_codeset" ]; then
+		db_default console-setup/codeset "$default_codeset"
+	    fi
 	    db_input $codeset_priority console-setup/codeset || true
 	    if db_go; then
 		db_get console-setup/codeset
--- console-setup.orig/debian/console-setup.templates	2006-11-13 16:54:38.000000000 +0000
+++ console-setup/debian/console-setup.templates	2007-04-01 16:43:46.000000000 +0100
@@ -1,7 +1,7 @@
 Template: console-setup/codeset
 Type: select
 __Choices: . Arabic, # Armenian, # Cyrillic - KOI8-R and KOI8-U, # Cyrillic - non-Slavic languages, # Cyrillic - Slavic languages (also Bosnian and Serbian Latin), . Ethiopic, # Georgian, # Greek, # Hebrew, # Lao, # Latin1 and Latin5 - western Europe and Turkic languages, # Latin2 - central Europe and Romanian, # Latin3 and Latin8 - Chichewa; Esperanto; Irish; Maltese and Welsh, # Latin7 - Lithuanian; Latvian; Maori and Marshallese, . Latin - Vietnamese, # Thai, . Combined - Latin; Slavic Cyrillic; Hebrew; basic Arabic, . Combined - Latin; Slavic Cyrillic; Greek, . Combined - Latin; Slavic and non-Slavic Cyrillic
-Default: . Combined: Latin; Slavic Cyrillic; Hebrew; basic Arabic
+Default: . Combined - Latin; Slavic Cyrillic; Hebrew; basic Arabic
 _Description: Set of characters that should be supported by the console font:
  If you don't use framebuffer the choices that start with dot will
  reduce the number of the available colors on the console.
