diff -u busybox-1.1.3/util-linux/mount.c busybox-1.1.3/util-linux/mount.c
--- busybox-1.1.3/util-linux/mount.c
+++ busybox-1.1.3/util-linux/mount.c
@@ -47,6 +47,9 @@
 #ifndef MS_SILENT
 #define MS_SILENT	32768
 #endif
+#ifndef MS_RELATIME
+#define MS_RELATIME	2097152
+#endif
 
 // Not real flags, but we want to be able to check for this.
 #define MOUNT_NOAUTO    (1<<29)
@@ -80,6 +83,8 @@
 	{"noatime", MS_NOATIME},
 	{"diratime", ~MS_NODIRATIME},
 	{"nodiratime", MS_NODIRATIME},
+	{"relatime", MS_RELATIME},
+	{"norelatime", ~MS_RELATIME},
 	{"loud", ~MS_SILENT},
 
 	// action flags
