Index: developer-duties.dbk
===================================================================
--- developer-duties.dbk	(revision 6671)
+++ developer-duties.dbk	(working copy)
@@ -152,6 +152,12 @@
 upstream version.  Whatever changes you need, always try not to fork from the
 upstream sources.
 </para>
+<para>
+If you find that the upstream developers are or become hostile towards Debian
+and or the free software community you may want to re-consider the need to
+include the upstream software in Debian. Sometimes the social cost to the
+Debian community is not worth the benefits the software may bring.
+</para>
 </section>
 
 <section id="rc-bugs">
