From d32a369499427c0f69a451adb1834a4450f001dc Mon Sep 17 00:00:00 2001
From: A Mennucc <mennucc1@debian.org>
Date: Tue, 7 Feb 2017 14:45:26 +0100
Subject: [PATCH] rename mismatched "url" variable, possible correction for
 Debian bug 708022

---
 debdelta/debdelta | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/debdelta/debdelta b/debdelta/debdelta
index 95acb24..7e307d1 100755
--- a/debdelta/debdelta
+++ b/debdelta/debdelta
@@ -4620,8 +4620,8 @@ def delta_upgrade_(args):
         if uri_p.scheme == 'http':
           a=[copy(z) for z in uri_p]
           a[2]=urllib2.quote(uri_p[2])
-          url=urlunparse(a)
-        req = urllib2.Request(url, headers=headers)
+          uri=urlunparse(a)
+        req = urllib2.Request(uri, headers=headers)
         r = urllib2.urlopen(req)
         #print r.info(),dir(r),r.code
         return r, getattr(r,'code',None), getattr(r,'msg','(no message)'), r.info()
-- 
2.1.4

