--- /usr/bin/debdelta.orig	2017-01-01 23:58:49.000000000 +0100
+++ /usr/bin/debdelta	2017-01-20 14:03:11.408818733 +0100
@@ -4604,10 +4604,9 @@
           return e, None, None, None
     else: #use urllib2
       try:
-        if uri_p.scheme == 'http':
-          a=[copy(z) for z in uri_p]
-          a[2]=urllib2.quote(uri_p[2])
-          url=urlunparse(a)
+        a=[copy(z) for z in uri_p]
+        a[2]=urllib2.quote(uri_p[2])
+        url=urlunparse(a)
         req = urllib2.Request(url, headers=headers)
         r = urllib2.urlopen(req)
         #print r.info(),dir(r),r.code
