diff -Nru dh-make-2.201604/debian/changelog dh-make-2.201604+nmu1/debian/changelog
--- dh-make-2.201604/debian/changelog	2016-01-25 19:11:43.000000000 +0900
+++ dh-make-2.201604+nmu1/debian/changelog	2016-02-07 05:42:44.000000000 +0900
@@ -1,3 +1,10 @@
+dh-make (2.201604+nmu1) UNRELEASED; urgency=medium
+
+  * Non-maintainer upload.
+  * Fix 'outfile'.
+
+ -- Katsuhiko Nishimra <ktns.87@gmail.com>  Sun, 07 Feb 2016 05:42:20 +0900
+
 dh-make (2.201604) unstable; urgency=medium
 
   * Depend on python-enum34 Closes: #811291
diff -Nru dh-make-2.201604/dh_make dh-make-2.201604+nmu1/dh_make
--- dh-make-2.201604/dh_make	2016-01-25 19:11:43.000000000 +0900
+++ dh-make-2.201604+nmu1/dh_make	2016-02-07 05:42:12.000000000 +0900
@@ -714,7 +714,7 @@
             exit(1)
 
     try:
-        with open('outfile', 'w') as f:
+        with open(outfile, 'w') as f:
             if args.native:
                 f.write('3.0 (native)\n')
             else:
