Index: sysdeps/linux/pci.c
===================================================================
--- sysdeps/linux/pci.c	(revision 4434)
+++ sysdeps/linux/pci.c	(working copy)
@@ -56,6 +56,14 @@
     node->model = _discover_xmalloc(5);
 }
 
+static inline u_int16_t hstole(u_int16_t n){
+#ifdef BIG_ENDIAN
+    return ((((n) << 8) & 0xff00) | (((n) >> 8) & 0xff));
+#else
+    return n;
+#endif
+}
+
 discover_sysdep_data_t *
 _discover_get_pci_raw_proc(void)
 {
@@ -130,6 +138,8 @@
 
             close(individual_device_file);
 
+            devtype = hstole(devtype);
+
             snprintf(node->busclass, 5, "%.4x", devtype);
 
             if (head == NULL) {
Index: configure.ac
===================================================================
--- configure.ac	(revision 4434)
+++ configure.ac	(working copy)
@@ -62,6 +62,8 @@
 LT_REVISION=0
 LT_AGE=0
 
+AC_C_BIGENDIAN
+
 ###############################################################################
 # Arguments
 
