--- /usr/share/perl5/Biber/LaTeX/Recode.pm.old	2014-10-09 13:19:09.157612660 +0200
+++ /usr/share/perl5/Biber/LaTeX/Recode.pm	2014-10-09 13:18:08.225310513 +0200
@@ -392,7 +392,7 @@
     my $re = $remap_e->{accents}{re};
 
     if ( $b =~ /$re/) {
-        return ($a eq 'i') or ($a eq 'j') ? "{\\$a}" : $a;
+        return (($a eq 'i') or ($a eq 'j') ? "{\\$a}" : $a);
     }
     else {
         return "{$a}"
