diff -Naur dl10n-3.00.orig/dl10n-check dl10n-3.00/dl10n-check
--- dl10n-3.00.orig/dl10n-check	2011-03-30 10:41:08.000000000 -0700
+++ dl10n-3.00/dl10n-check	2016-12-08 20:06:13.590737510 -0800
@@ -1133,6 +1133,7 @@
 httpd
 interpreters
 java
+javascript
 kde
 kernel
 libdevel
@@ -1151,6 +1152,7 @@
 php
 python
 ruby
+rust
 science
 shells
 sound
@@ -1187,6 +1189,7 @@
 contrib/httpd
 contrib/interpreters
 contrib/java
+contrib/javascript
 contrib/kde
 contrib/kernel
 contrib/libdevel
@@ -1205,6 +1208,7 @@
 contrib/php
 contrib/python
 contrib/ruby
+contrib/rust
 contrib/science
 contrib/shells
 contrib/sound
@@ -1241,6 +1245,7 @@
 non-free/httpd
 non-free/interpreters
 non-free/java
+non-free/javascript
 non-free/kde
 non-free/kernel
 non-free/libdevel
@@ -1259,6 +1264,7 @@
 non-free/php
 non-free/python
 non-free/ruby
+non-free/rust
 non-free/science
 non-free/shells
 non-free/sound
