Index: apt-show-versions
===================================================================
--- apt-show-versions	(revision 69)
+++ apt-show-versions	(working copy)
@@ -342,18 +342,18 @@
         printf("%s not available for architecture %s\n", $pkgname, $archname);
     } elsif (! defined $pkgs->{$pkgname}) {
         if ($mode != $MODE_SINGLE || $archname) {
-            printf("%s not installed\n", $package);
+	    print_package_internal($pkgname, $archname)
         } elsif (keys(%{$apackages->{$pkgname}})) {
             my $archlist = "";
             foreach my $a (sort keys(%{$apackages->{$pkgname}})) {
+		print_package_internal($pkgname, $a);
                 $archlist .= ($archlist ? ", $a" : "$a");
             }
-            printf("%s not installed (available for: %s)\n", $pkgname, $archlist);
         } else {
             printf("%s not installed (not available)\n", $pkgname);
         }
     } else {
-        foreach my $arch ($archname or sort keys $pkgs->{$pkgname}) {
+        foreach my $arch ($archname or sort keys(%{$apackages->{$pkgname}})) {
             print_package_internal($pkgname, $arch);
         }
     }
@@ -387,9 +387,7 @@
                 push @print_info, "$ipkg->{$VERS} ";
             }
             push @print_info, "$ipkg->{$STATUS}\n";
-        } else {
-            push @print_info, "Not installed\n";
-        }
+        } 
 
         # Index to @official_suites: Next official suite to mention if missing.
         my $official_idx = 0;
