diff -Naur dh-make-0.42.orig/dh_make dh-make-0.42/dh_make
--- dh-make-0.42.orig/dh_make	2006-10-30 05:16:02.000000000 +0200
+++ dh-make-0.42/dh_make	2007-03-22 05:04:12.000000000 +0200
@@ -318,15 +318,9 @@
 
 sub get_date
 {
-  my $tmpdate;
-  if (-x "/usr/bin/822-date")
-  {
-    $tmpdate = `/usr/bin/822-date`;
-    chomp($tmpdate);
-    return $tmpdate;
-  } else {
-    die "Unable to find 822-date program in /usr/bin!\n";
-  }
+  my $tmpdate = `date -uR`;
+  chomp($tmpdate);
+  return $tmpdate;
 }
 
 sub print_confirmation
