--- Makefile.old	2007-11-22 19:32:20.000000000 +0000
+++ Makefile	2007-11-22 19:17:30.000000000 +0000
@@ -46,6 +46,6 @@
 	mkdir -p $(DESTDIR)/usr/share/desktop-base/profiles/xdg-config/xfce4-session
 	$(INSTALL) $(wildcard profiles/xdg-config/xfce4-session/*) $(DESTDIR)/usr/share/desktop-base/profiles/xdg-config/xfce4-session
 	# GNOME background descriptor
-	$(INSTALL) -d $(DESTDIR)/usr/share/gnome-background-properties
+	install -d $(DESTDIR)/usr/share/gnome-background-properties
 	$(INSTALL) gnome-backgrounds.xml $(DESTDIR)/usr/share/gnome-background-properties/debian.xml
 
