diff -ur docbook-utils-0.6.14.orig/debian/control docbook-utils-0.6.14/debian/control
--- docbook-utils-0.6.14.orig/debian/control	2012-03-09 12:28:43.000000000 +0000
+++ docbook-utils-0.6.14/debian/control	2012-03-09 12:32:42.000000000 +0000
@@ -13,6 +13,7 @@
 Depends: docbook-dsssl, jadetex, lynx | links | w3m, sgmlspl, sp, ${perl:Depends}
 Recommends: docbook-xml
 Replaces: docbook2man, docbook2texi, docbk-xml2x, docbook2x-doc, cygnus-stylesheets
+Multi-Arch: foreign
 Description: Convert Docbook files to other formats (HTML, RTF, PS, man, PDF)
  The docbook-utils is a set of a few small programs intended to ease
  everyday use of technical documentation software and more generally use of
