=== modified file 'bash_completion'
--- bash_completion	2009-02-22 10:48:49 +0000
+++ bash_completion	2009-02-22 17:16:17 +0000
@@ -108,7 +108,7 @@
 complete -f -X '!*.@(pdf|PDF)' acroread gpdf xpdf
 complete -f -X '!*.@(?(e)ps|?(E)PS|pdf|PDF)' kpdf
 complete -f -X '!*.@(@(?(e)ps|?(E)PS|pdf|PDF)?(.gz|.GZ|.bz2|.BZ2)|cb[rz]|CB[RZ]|djv?(u)|DJV?(U)|dvi|DVI|gif|jp?(e)g|miff|tif?(f)|pn[gm]|p[bgp]m|bmp|xpm|ico|xwd|tga|pcx|GIF|JP?(E)G|MIFF|TIF?(F)|PN[GM]|P[BGP]M|BMP|XPM|ICO|XWD|TGA|PCX)' evince
-complete -f -X '!*.@(?(e)ps|?(E)PS)' ps2pdf
+complete -f -X '!*.@(?(e)ps|?(E)PS|pdf|PDF)' ps2pdf
 complete -f -X '!*.texi*' makeinfo texi2html
 complete -f -X '!*.@(?(la)tex|?(LA)TEX|texi|TEXI|dtx|DTX|ins|INS)' tex latex slitex jadetex pdfjadetex pdftex pdflatex texi2dvi
 complete -f -X '!*.@(mp3|MP3)' mpg123 mpg321 madplay

=== modified file 'debian/changelog'
--- debian/changelog	2009-02-22 10:48:49 +0000
+++ debian/changelog	2009-02-22 17:16:17 +0000
@@ -6,6 +6,10 @@
   [ Freddy Vulto ]
   * Make `_mii-tool()' and `_mii-diag()' POSIX-compliant
 
+  [ Jakob Unterwurzacher ]
+  * ps2pdf can run on .pdf files as well. (Closes: #516614, LP: #316943)
+
+
  -- Freddy Vulto <fvulto@gmail.com>  Sun, 22 Feb 2009 11:32:41 +0100
 
 bash-completion (1:1.0-1) UNRELEASED; urgency=low

