diff -Nru docker-compose-1.29.2/debian/changelog docker-compose-1.29.2/debian/changelog
--- docker-compose-1.29.2/debian/changelog	2024-07-08 12:49:42.000000000 -0300
+++ docker-compose-1.29.2/debian/changelog	2024-07-14 18:21:36.000000000 -0300
@@ -1,3 +1,11 @@
+docker-compose (1.29.2-6.2) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * d/patches/remove_distutils.patch: Update patch to fix a bug in the
+    use of packaging.version.Version (Closes: #1076201).
+
+ -- Emmanuel Arias <eamanu@debian.org>  Sun, 14 Jul 2024 18:21:36 -0300
+
 docker-compose (1.29.2-6.1) unstable; urgency=medium
 
   * Non-maintainer upload.
