diff -u docbook-xml-4.5/debian/control docbook-xml-4.5/debian/control
--- docbook-xml-4.5/debian/control
+++ docbook-xml-4.5/debian/control
@@ -14,6 +14,7 @@
 
 Package: docbook-xml
 Architecture: all
+Multi-Arch: foreign
 Depends: sgml-data (>= 2.0.2), ${misc:Depends}, xml-core(>= 0.12)
 Suggests: docbook, docbook-dsssl, docbook-xsl, docbook-defguide
 Provides: docbk-xml
diff -u docbook-xml-4.5/debian/changelog docbook-xml-4.5/debian/changelog
--- docbook-xml-4.5/debian/changelog
+++ docbook-xml-4.5/debian/changelog
@@ -1,3 +1,10 @@
+docbook-xml (4.5-7.3) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Add multiarch meta-data (Closes: #732096)
+
+ -- Wookey <wookey@debian.org>  Thu, 28 May 2015 17:29:38 +0100
+
 docbook-xml (4.5-7.2) unstable; urgency=high
 
   [ Helmut Grohne ]
