diff -Nru debhelper-8.9.8/Debian/Debhelper/Buildsystem/qmake.pm debhelper-8.9.8/Debian/Debhelper/Buildsystem/qmake.pm
--- debhelper-8.9.8/Debian/Debhelper/Buildsystem/qmake.pm
+++ debhelper-8.9.8/Debian/Debhelper/Buildsystem/qmake.pm
@@ -54,12 +54,12 @@
 	push @options, '-nocache';
 
 	if ($ENV{CFLAGS}) {
-		push @flags, "QMAKE_CFLAGS_RELEASE=$ENV{CFLAGS}";
-		push @flags, "QMAKE_CFLAGS_DEBUG=$ENV{CFLAGS}";
+		push @flags, "QMAKE_CFLAGS_RELEASE=$ENV{CFLAGS} $ENV{CPPFLAGS}";
+		push @flags, "QMAKE_CFLAGS_DEBUG=$ENV{CFLAGS} $ENV{CPPFLAGS}";
 	}
 	if ($ENV{CXXFLAGS}) {
-		push @flags, "QMAKE_CXXFLAGS_RELEASE=$ENV{CXXFLAGS}";
-		push @flags, "QMAKE_CXXFLAGS_DEBUG=$ENV{CXXFLAGS}";
+		push @flags, "QMAKE_CXXFLAGS_RELEASE=$ENV{CXXFLAGS} $ENV{CPPFLAGS}";
+		push @flags, "QMAKE_CXXFLAGS_DEBUG=$ENV{CXXFLAGS} $ENV{CPPFLAGS}";
 	}
 	if ($ENV{LDFLAGS}) {
 		push @flags, "QMAKE_LFLAGS_RELEASE=$ENV{LDFLAGS}";
