--- ftp.py~	2004-09-08 21:11:56.000000000 +0200
+++ ftp.py	2004-09-08 21:11:02.000000000 +0200
@@ -55,7 +55,7 @@
 				      f, 1024)
 	    f.close()
             print "done."
-        except ftplib.all_errors:
-            print "Error during ftp transfer of %s" % package_name
+        except ftplib.all_errors, e:
+            print "Error '%s' during ftp transfer of %s"%(str(e),package_name)
             sys.exit(1)
     ftp_connection.quit()
