--- dput~	2004-09-08 21:20:58.000000000 +0200
+++ dput	2004-09-08 21:20:51.000000000 +0200
@@ -671,13 +671,10 @@
         login = os.environ['USER']
         if debug: print "D: Login: %s" % login
     else:
-        print "$USER not set, will use login."
-    
-    # Else use the current username
-    if login == '':
-       print "D: Variable $USER not set."
-       login = pwd.getpwuid(os.getuid( ))[0]
-       if debug:
+	print "$USER not set, will use login."
+	# Else use the current username
+	login = pwd.getpwuid(os.getuid( ))[0]
+	if debug:
            print "D: User-ID: %s" % os.getuid()
            print "D: Login: %s" % login
 
