--- ftp.py~	2004-09-24 19:40:39.000000000 +0200
+++ ftp.py	2004-09-24 19:39:34.000000000 +0200
@@ -19,8 +19,7 @@
     except ftplib.error_perm:
         print "Wrong Password"
         sys.exit(1)
-    if ftp_mode == 1:
-        ftp_connection.set_pasv(1)
+    ftp_connection.set_pasv(ftp_mode==1)
     try:
         ftp_connection.cwd(incoming)
     except ftplib.error_perm,e:
@@ -55,7 +54,8 @@
 				      f, 1024)
 	    f.close()
             print "done."
-        except ftplib.all_errors:
-            print "Error during ftp transfer of %s" % package_name
+        except ftplib.all_errors, e:
+            print "Error '%s' during ftp transfer of %s"%(str(e),package_name)
+	    raise
             sys.exit(1)
     ftp_connection.quit()
