diff --git a/bash_completion b/bash_completion
index e7a4b6e..5c9483d 100644
--- a/bash_completion
+++ b/bash_completion
@@ -602,7 +602,7 @@ _filedir()
 {
     local i IFS=$'\t\n' xspec
 
-    __expand_tilde_by_ref cur
+    __expand_tilde_by_ref cur || return 0
 
     local -a toks
     local quoted tmp
@@ -838,7 +838,9 @@ __expand_tilde_by_ref() {
             eval $1="${!1/%\/*}"/'${!1#*/}'
         else 
             # No, $1 doesn't contain slash
-            eval $1="${!1}"
+            COMPREPLY=( $( compgen -u "${!1}" ) )
+            [ ${#COMPREPLY[@]} -eq 1 ] && eval COMPREPLY[0]=${COMPREPLY[0]}
+            return ${#COMPREPLY[@]}
         fi
     fi
 } # __expand_tilde_by_ref()
