--- main.cc.orig	2004-07-07 06:03:48.000000000 +0300
+++ main.cc	2004-09-25 18:22:53.000000000 +0300
@@ -182,6 +182,9 @@
   //      rocker.
   argv0=argv[0];
 
+  // This will be used to hold current locale.
+  char *locale;
+
   // Read the arguments:
   while((curopt=getopt_long(argc, argv, "DVZvhS:uiF:w:sO:fdyPt:Rro:", opts, NULL))!=-1)
     {
@@ -272,6 +275,13 @@
 	      exit(0);
 	    case OPTION_VISUAL_PREVIEW:
 	      visual_preview=true;
+	      locale = setlocale(LC_ALL, NULL);    
+	      
+              if (strstr(locale, "UTF-8")) {
+	      	      setlocale(LC_ALL, "C");
+		      locale = setlocale(LC_ALL, NULL);
+              }
+	      
 	      break;
 	    default:
 	      fprintf(stderr, "%s",
@@ -378,6 +388,15 @@
       exit(1);
     }
 
+  // Apart from visual preview we also have to take care of the normal
+  // ui mess with UTF-8, and fallback to "C" locale
+  locale = setlocale(LC_ALL, NULL);    
+	      	      
+  if (strstr(locale, "UTF-8")) {
+    setlocale(LC_ALL, "C");
+    locale = setlocale(LC_ALL, NULL);
+  }			    	
+  
   ui_init();
 
   vs_progress *p=gen_progress_bar();
