diff --git a/distro-info-util.c b/distro-info-util.c
index 48b5a5d..b262f71 100644
--- a/distro-info-util.c
+++ b/distro-info-util.c
@@ -848,6 +848,8 @@ int main(int argc, char *argv[]) {
 
 #ifdef DEBIAN
     if(alias_codename) {
+        const distro_t *oldstable = get_distro(distro_list, date, filter_oldstable,
+                                               select_oldstable);
         const distro_t *stable = get_distro(distro_list, date, filter_stable,
                                             select_latest_release);
         const distro_t *testing = get_distro(distro_list, date, filter_testing,
@@ -857,6 +859,8 @@ int main(int argc, char *argv[]) {
         if(unlikely(stable == NULL || testing == NULL || unstable == NULL)) {
             fprintf(stderr, NAME ": " OUTDATED_ERROR "\n");
             return_value = EXIT_FAILURE;
+        } else if(strcmp(oldstable->series, alias_codename) == 0) {
+            printf("oldstable\n");
         } else if(strcmp(stable->series, alias_codename) == 0) {
             printf("stable\n");
         } else if(strcmp(testing->series, alias_codename) == 0) {
