diff --git a/30_exim4-config_check_rcpt b/30_exim4-config_check_rcpt
index ac347aa..b7b2f70 100644
--- a/30_exim4-config_check_rcpt
+++ b/30_exim4-config_check_rcpt
@@ -267,8 +267,8 @@ acl_check_rcpt:
     condition = ${run{/usr/bin/spfquery.mail-spf-perl --ip \
                    \"$sender_host_address\" --identity \
                    ${if def:sender_address_domain \
-                       {--scope mfrom  --identity \"$sender_address\"}\
-                       {--scope helo --identity  \"$sender_helo_name\"}}}\
+                       {--scope mfrom  --identity ${quote:$sender_address}}\
+                       {--scope helo --identity ${quote:$sender_helo_name}}}}\
                    {no}{${if eq {$runrc}{1}{yes}{no}}}}
 
   defer
