diff -u encfs-1.4.1.1/debian/control encfs-1.4.1.1/debian/control
--- encfs-1.4.1.1/debian/control
+++ encfs-1.4.1.1/debian/control
@@ -2,7 +2,7 @@
 Section: utils
 Priority: optional
 Maintainer: Eduard Bloch <blade@debian.org>
-Build-Depends: debhelper (>= 4.0.0), librlog-dev, librlog1c2a, libfuse-dev (>> 2.6), libssl-dev (>> 0.9.8), quilt
+Build-Depends: debhelper (>= 4.0.0), librlog-dev, librlog1c2a, libfuse-dev (>> 2.6), libssl-dev (>> 0.9.8), quilt, libboost-dev, pkg-config
 Standards-Version: 3.7.2
 
 Package: encfs
diff -u encfs-1.4.1.1/debian/rules encfs-1.4.1.1/debian/rules
--- encfs-1.4.1.1/debian/rules
+++ encfs-1.4.1.1/debian/rules
@@ -27,7 +27,7 @@
 config.status: configure $(QUILT_STAMPFN)
 	dh_testdir
 	# Add here commands to configure the package.
-	CFLAGS="$(CFLAGS)" LDFLAGS="-Wl,--as-needed -lulockmgr" ./configure --build=$(DEB_BUILD_GNU_TYPE) --prefix=/usr --mandir=\$${prefix}/share/man --infodir=\$${prefix}/share/info
+	CFLAGS="$(CFLAGS)" LDFLAGS="-Wl,--as-needed -lulockmgr -lpthread" ./configure --build=$(DEB_BUILD_GNU_TYPE) --prefix=/usr --mandir=\$${prefix}/share/man --infodir=\$${prefix}/share/info
 	touch config.status
 
 build: build-stamp
diff -u encfs-1.4.1.1/debian/changelog encfs-1.4.1.1/debian/changelog
--- encfs-1.4.1.1/debian/changelog
+++ encfs-1.4.1.1/debian/changelog
@@ -1,3 +1,11 @@
+encfs (1.4.1.1-1.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Adding build dependencies to libboost-dev.
+  * Fixing pthread issue.
+
+ -- Klaus Ethgen <Klaus@Ethgen.de>  Mon, 11 Feb 2008 22:43:36 +0100
+
 encfs (1.4.1.1-1) unstable; urgency=low
 
   * New upstream release (closes: #460287)
