=== modified file 'debian/changelog'
--- debian/changelog	2011-03-26 00:07:33 +0000
+++ debian/changelog	2011-03-26 21:15:47 +0000
@@ -1,6 +1,12 @@
+davfs2 (1.4.6-1ubuntu2) unstable; urgency=low
+
+  * Add distccd to ignore_home to allow mount under / in Debian & Ubuntu.
+
+ -- Michael R. Crusoe <michael.crusoe@gmail.com>  Sat, 26 Mar 2011 11:08:55 -0700

=== modified file 'etc/davfs2.conf'
--- etc/davfs2.conf	2011-03-25 18:10:57 +0000
+++ etc/davfs2.conf	2011-03-26 21:15:47 +0000
@@ -21,7 +21,7 @@
 
 # dav_user        davfs2            # system wide config file only
 # dav_group       davfs2            # system wide config file only
-# ignore_home                       # system wide config file only
+ignore_home       distccd	     # system wide config file only
 # kernel_fs       fuse
 # buf_size        16                 # KiByte
 

