--- /usr/bin/uscan	2009-01-21 22:04:38.000000000 +0100
+++ /tmp/uscan	2009-02-08 08:12:30.000000000 +0100
@@ -1295,6 +1295,15 @@
 	$newfile_base = $newfile_base_gz;
     }
 
+    if ($repack and $newfile_base =~ /^(.*)\.(tar\.lzma|tlzma?)$/) {
+	print "-- Repacking from lzma to gzip\n" if $verbose;
+	my $newfile_base_gz = "$1.tar.gz";
+	system("lzma -cd $destdir/$newfile_base | gzip > $destdir/$newfile_base_gz") == 0
+	  or die "repacking from lzma to gzip failed\n";
+	unlink "$destdir/$newfile_base";
+	$newfile_base = $newfile_base_gz;
+    }
+
     if ($repack and $newfile_base =~ /^(.*)\.zip$/) {
 	print "-- Repacking from zip to .tar.gz\n" if $verbose;
 
