diff --git a/debian/deluged.postinst b/debian/deluged.postinst
index b064a99..4b2e0aa 100644
--- a/debian/deluged.postinst
+++ b/debian/deluged.postinst
@@ -8,7 +8,9 @@ case "$1" in
       adduser --system --group --quiet --no-create-home --home /var/lib/deluged debian-deluged
     fi
 
+    dpkg-statoverride --list /var/log/deluged >/dev/null || \
     dpkg-statoverride --update --add debian-deluged adm 2750 /var/log/deluged
+    dpkg-statoverride --list /var/lib/deluged/config >/dev/null || \
     dpkg-statoverride --update --add debian-deluged debian-deluged 0750 /var/lib/deluged/config
     ;;
 esac
