diff --git a/debian/installer/armhf/modules/armhf-armmp/i2c-modules b/debian/installer/armhf/modules/armhf-armmp/i2c-modules
new file mode 100644
index 0000000..9b4450e
--- /dev/null
+++ b/debian/installer/armhf/modules/armhf-armmp/i2c-modules
@@ -0,0 +1,4 @@
+#include <i2c-modules>
+i2c-mv64xxx
+i2c-rk3x
+
diff --git a/debian/installer/armhf/modules/armhf-armmp/kernel-image b/debian/installer/armhf/modules/armhf-armmp/kernel-image
index edd1bf7..be27bb8 100644
--- a/debian/installer/armhf/modules/armhf-armmp/kernel-image
+++ b/debian/installer/armhf/modules/armhf-armmp/kernel-image
@@ -15,3 +15,6 @@ phy-ti-pipe3 ?
 
 # Regulators
 drivers/regulator/**
+
+# Powersupply
+axp20x_usb_power ?
