Index: debian/control
===================================================================
--- debian/control	(Revision 5292)
+++ debian/control	(Arbeitskopie)
@@ -13,7 +13,7 @@
 Package: fai-client
 Architecture: all
 Section: admin
-Depends: perl, file, libapt-pkg-perl, libparse-recdescent-perl, liblinux-lvm-perl
+Depends: perl, file, libapt-pkg-perl, libparse-recdescent-perl, liblinux-lvm-perl, net-tools
 Recommends: debconf-utils, cfengine2
 Suggests: ntfsprogs, dmsetup, cryptsetup
 Conflicts: fai, fai-kernels
@@ -46,7 +46,7 @@
 Architecture: all
 Section: admin
 Depends: fai-client, debootstrap
-Recommends: nfs-kernel-server, dhcp3-server | bootp, tftpd-hpa | tftpd, openssh-server, openssh-client, syslinux-common, openbsd-inetd | inet-superserver
+Recommends: nfs-kernel-server, dhcp3-server | bootp, tftpd-hpa | tftpd, openssh-server, openssh-client, syslinux-common, openbsd-inetd | inet-superserver, net-tools
 Suggests: debmirror, apt-move, genisoimage, grub, aptitude, perl-tk, libproc-daemon-perl
 Conflicts: fai
 Replaces: fai
Index: debian/changelog
===================================================================
--- debian/changelog	(Revision 5292)
+++ debian/changelog	(Arbeitskopie)
@@ -54,6 +54,9 @@
     initialization of encrypted partitions configurable
   * setup-storage.8: Document new encryption possibilities
 
+  [ Holger Levsen ]
+  * Add depends to net-tools to fai-server and fai-client. 
+
  -- Thomas Lange <lange@debian.org>  Sun, 23 Feb 2009 15:55:55 +0100
 
  (3.2.17) unstable; urgency=medium
