diff -Naur cvs-1.12.9.orig/debian/control cvs-1.12.9/debian/control
--- cvs-1.12.9.orig/debian/control	2006-04-05 10:16:08.000000000 +0200
+++ cvs-1.12.9/debian/control	2006-04-05 10:16:56.000000000 +0200
@@ -2,8 +2,8 @@
 Section: devel
 Priority: optional
 Maintainer: Steve McIntyre <93sam@debian.org>
-Build-Depends: debhelper (>= 4.1.16), zlib1g-dev, texi2html, groff, texinfo, bison, dbs, automake1.7, libpam0g-dev, autotools-dev
-Build-Conflicts: heimdal-dev, kerberos4kth-dev, libkrb5-dev
+Build-Depends: debhelper (>= 4.1.16), zlib1g-dev, texi2html, groff, texinfo, bison, dbs, automake1.7, libpam0g-dev, autotools-dev, heimdal-dev
+Build-Conflicts: kerberos4kth-dev, libkrb5-dev
 Standards-Version: 3.6.1.0
 
 Package: cvs
diff -Naur cvs-1.12.9.orig/debian/rules cvs-1.12.9/debian/rules
--- cvs-1.12.9.orig/debian/rules	2006-04-05 10:16:08.000000000 +0200
+++ cvs-1.12.9/debian/rules	2006-04-05 10:16:39.000000000 +0200
@@ -46,7 +46,7 @@
 		--sysconfdir=/etc \
 		--mandir=/usr/share/man \
 		--infodir=/usr/share/info \
-		--without-krb4 --without-gssapi \
+		--without-krb4 --with-gssapi --enable-encryption \
 		--with-editor=/usr/bin/sensible-editor \
 		--enable-rootcommit --with-external-zlib \
 		--enable-pam --with-hardcoded-pam-service-name=cvs
