diff --git a/debmirror b/debmirror
index ff4bd08..b89751a 100755
--- a/debmirror
+++ b/debmirror
@@ -1584,7 +1584,7 @@ sub rsync_get {
   }
   $opt .= " --progress" if $progress;
   $opt .= " -v" if $debug;
-  $opt .= " -no-motd" unless $verbose;
+  $opt .= " --no-motd" unless $verbose;
   system ("rsync --timeout=$timeout $opt $rsyncremote --include=$file --exclude='*' .");
   if ($? == 0 && -f $file) {
     return 1;
