diff --git a/bash_completion.d/mutt b/bash_completion.d/mutt
index 62540bd..5109db8 100644
--- a/bash_completion.d/mutt
+++ b/bash_completion.d/mutt
@@ -75,7 +75,7 @@ _muttquery()
 _muttfiledir()
 {
     local cur folder spoolfile muttcmd=${COMP_WORDS[0]}
-    cur=`_get_cword`
+    cur=`_get_cword =`
 
     # This is currently not working so well. Perhaps this function should
     # just call _filedir() for the moment.
@@ -125,6 +125,10 @@ _mutt()
         -@(e|m|Q|s|h|p|R|v|y|z|Z))
             return 0
             ;;
+	=)
+	    _muttfiledir
+	    return 0
+	    ;;
         *)
             _muttaddr
             return 0
