Index: epiphany-browser-2.20.3/embed/mozilla/EphyBrowser.h
===================================================================
--- epiphany-browser-2.20.3.orig/embed/mozilla/EphyBrowser.h	2008-04-02 12:54:04.000000000 -0300
+++ epiphany-browser-2.20.3/embed/mozilla/EphyBrowser.h	2008-04-02 12:56:20.000000000 -0300
@@ -187,7 +187,7 @@
 
 	nsresult LoadURI(const char *aURI, 
 			 PRUint32 aLoadFlags = nsIWebNavigation::LOAD_FLAGS_NONE, 
-			 nsIURI *aURI = nsnull);
+			 nsIURI *nsaURI = nsnull);
 
         EphyEmbedDocumentType GetDocumentType ();
 
Index: epiphany-browser-2.20.3/embed/mozilla/mozilla-x509-cert.h
===================================================================
--- epiphany-browser-2.20.3.orig/embed/mozilla/mozilla-x509-cert.h	2008-04-02 13:00:57.000000000 -0300
+++ epiphany-browser-2.20.3/embed/mozilla/mozilla-x509-cert.h	2008-04-02 13:01:19.000000000 -0300
@@ -56,7 +56,7 @@
 
 MozillaX509Cert           *mozilla_x509_cert_new              (nsIX509Cert *aMozCert);
 
-nsresult                   mozilla_x509_cert_get_mozilla_cert (MozillaX509Cert *cert, nsIX509Cert **cert);
+nsresult                   mozilla_x509_cert_get_mozilla_cert (MozillaX509Cert *mozcert, nsIX509Cert **nscert);
 
 G_END_DECLS
 
