diff --git a/INSTALL b/INSTALL
index da32cbe..c1cc0e6 100644
--- a/INSTALL
+++ b/INSTALL
@@ -68,7 +68,7 @@ Configuration:
  - visit Command-and-Control web interface of apt-cacher-ng, link can be found
    among other instructions at http://yourHostName:portNumber/ , which might be
    http://localhost:3142/ with the default configuration
- - for apt clients, there is a config snipped in contrib/10-apt-cacher-ng.conf
+ - for apt clients, there is a config snippet in contrib/10-apt-cacher-ng.conf
    which might be installed into /etc/apt/apt.conf.d/.
 
 Developer notes:
diff --git a/doc/000apt-cacher-ng-proxy b/doc/000apt-cacher-ng-proxy
index e03fadd..9bb07c2 100644
--- a/doc/000apt-cacher-ng-proxy
+++ b/doc/000apt-cacher-ng-proxy
@@ -1,4 +1,4 @@
-# This configuration snipped is intended to be stored in /etc/apt/apt.conf.d/
+# This configuration snippet is intended to be stored in /etc/apt/apt.conf.d/
 # on the client system in order to change a regular setup to use apt-cacher-ng.
 #
 Acquire::http::Proxy "http://192.168.0.245:3142/";
