diff -urN /home/wookey/linaro/multiarch/packages/equivs-2.0.9/debian/control /home/wookey/linaro/multiarch/packages-modded/equivs-2.0.9/debian/control
--- /home/wookey/linaro/multiarch/packages/equivs-2.0.9/debian/control	2011-09-30 06:37:04.000000000 +0100
+++ /home/wookey/linaro/multiarch/packages-modded/equivs-2.0.9/debian/control	2013-01-09 14:39:18.000000000 +0000
@@ -4,6 +4,7 @@
 Maintainer: Peter Samuelson <peter@p12n.org>
 Standards-Version: 3.9.2
 Build-Depends: debhelper
+Multi-Arch: foreign
 
 Package: equivs
 Depends: perl, debhelper, dpkg-dev, make, fakeroot, ${misc:Depends}
diff -urN /home/wookey/linaro/multiarch/packages/equivs-2.0.9/usr/bin/equivs-build /home/wookey/linaro/multiarch/packages-modded/equivs-2.0.9/usr/bin/equivs-build
--- /home/wookey/linaro/multiarch/packages/equivs-2.0.9/usr/bin/equivs-build	2011-09-30 07:22:24.000000000 +0100
+++ /home/wookey/linaro/multiarch/packages-modded/equivs-2.0.9/usr/bin/equivs-build	2013-01-09 14:37:06.000000000 +0000
@@ -309,6 +309,7 @@
 			   "Breaks",
 			   "Provides",
 			   "Replaces",
+			   "Multi-Arch",
 			   "Description");
   close OUT;
 }
