diff -Naur audacious-1.5.1.orig/src/audacious/main.c audacious-1.5.1/src/audacious/main.c
--- audacious-1.5.1.orig/src/audacious/main.c	2008-05-24 00:00:34.000000000 +0200
+++ audacious-1.5.1/src/audacious/main.c	2008-08-13 03:02:52.362244501 +0200
@@ -715,6 +715,9 @@
 
     signal_handlers_init();
 
+    plugin_system_init();
+    playlist_system_init();
+
     handle_cmd_line_options();
 
     if (options.headless == FALSE)
@@ -732,9 +735,6 @@
         ui_manager_create_menus();
     }
 
-    plugin_system_init();
-    playlist_system_init();
-
 #ifdef USE_DBUS
     init_dbus();
 #endif
