--- lid.sh.orig	2010-06-23 04:27:29.000000000 +0200
+++ lid.sh	2010-08-17 18:36:00.000000000 +0200
@@ -1,46 +1,44 @@
-#!/bin/bash
-# TODO:  Change the above to /bin/sh
+#!/bin/sh
 
-test -f /usr/share/acpi-support/state-funcs || exit 0
+# FIXME: Is this really needed here?
+[ -f /usr/share/acpi-support/state-funcs ] || exit 0
 
 . /usr/share/acpi-support/power-funcs
 . /usr/share/acpi-support/policy-funcs
 . /etc/default/acpi-support
 
-[ -x /etc/acpi/local/lid.sh.pre ] && /etc/acpi/local/lid.sh.pre
+[ ! -x /etc/acpi/local/lid.sh.pre ] || /etc/acpi/local/lid.sh.pre
 
-if [ `CheckPolicy` = 0 ]; then exit; fi
+# FIXME: 'CheckPolicy' doesn't 'echo' stuff, it 'return's an exit status!
+! CheckPolicy || exit 0
 
-grep -q closed /proc/acpi/button/lid/*/state
-if [ $? = 0 ]
-then
+
+if grep -q closed /proc/acpi/button/lid/*/state; then
     for x in /tmp/.X11-unix/*; do
-	displaynum=`echo $x | sed s#/tmp/.X11-unix/X##`
-	getXuser;
-	if [ x"$XAUTHORITY" != x"" ]; then
-	    export DISPLAY=":$displaynum"	    
+	# FIXME: Skip fork and pipe
+	#displaynum=`echo $x | sed s#/tmp/.X11-unix/X##`
+	displaynum=${x#/tmp/.X11-unix/X}
+	getXuser
+	[ -z "$XAUTHORITY" ] || {
+	    export DISPLAY=:$displaynum
 	    . /usr/share/acpi-support/screenblank
-	fi
+	}
     done
 else
     for x in /tmp/.X11-unix/*; do
-	displaynum=`echo $x | sed s#/tmp/.X11-unix/X##`
-	getXuser;
-	if [ x"$XAUTHORITY" != x"" ]; then
-	    export DISPLAY=":$displaynum"
-	    grep -q off-line /proc/acpi/ac_adapter/*/state
-	    if [ $? = 1 ]
-		then
-		if pidof xscreensaver > /dev/null; then 
-		    su $user -c "xscreensaver-command -unthrottle"
-		fi
-	    fi
-	    if [ x$RADEON_LIGHT = xtrue ]; then
-		[ -x /usr/sbin/radeontool ] && radeontool light on
-	    fi
-	    if [ `pidof xscreensaver` ]; then
+	# FIXME: Skip fork and pipe
+	#displaynum=`echo $x | sed s#/tmp/.X11-unix/X##`
+	displaynum=${x#/tmp/.X11-unix/X}
+	getXuser
+	if [ "$XAUTHORITY" ]; then
+	    export DISPLAY=:$displaynum
+	    grep -q off-line /proc/acpi/ac_adapter/*/state ||
+		! pidof xscreensaver >/dev/null 2>&1 ||
+		su $user -c "xscreensaver-command -unthrottle"
+	    [ "$RADEON_LIGHT" != true ]
+		[ ! -x /usr/sbin/radeontool ] || radeontool light on
+	    ! pidof xscreensaver >/dev/null 2>&1 ||
 		su $user -c "xscreensaver-command -deactivate"
-	    fi
 	    su $user -c "xset dpms force on"
 	fi
     done
