--- rules.orig	2025-01-27 09:10:24.472485199 +0100
+++ rules	2025-01-27 09:10:29.192485181 +0100
@@ -19,7 +19,8 @@
 d=debian/coreutils
 
 override_dh_auto_configure:
-	dh_auto_configure -- --enable-install-program=arch --with-openssl=auto-gpl-compat
+	dh_auto_configure -- --enable-install-program=arch --with-openssl=auto-gpl-compat \
+		--with-packager=Debian --with-packager-version=$(DEB_VERSION)
 
 %:
 	dh $@ --with autoreconf
