--- etckeeper-0.46/etckeeper	2009-09-26 21:57:49.000000000 +0200
+++ etckeeper-0.46ubuntu1/etckeeper	2010-05-17 16:41:13.000000000 +0200
@@ -22,6 +22,13 @@
 fi
 export VCS
 
+if [ "$VCS" = "bzr" ]; then
+	# Avoid creating root-owned .bazaar and .bzr.log directories in users
+	# home directory when etckeeper is run under sudo (LP: #376388)
+	HOME=~root
+	export HOME
+fi
+
 if [ ! -z "$GIT_COMMIT_OPTIONS" ]; then
 	export GIT_COMMIT_OPTIONS
 fi
