diff -Nru cups-1.5.3/debian/changelog cups-1.5.3/debian/changelog
--- cups-1.5.3/debian/changelog	2012-05-23 14:49:38.000000000 +0200
+++ cups-1.5.3/debian/changelog	2012-08-03 17:46:52.000000000 +0200
@@ -1,3 +1,11 @@
+cups (1.5.3-1+UNRELEASED) UNRELEASED; urgency=low
+
+  * Non-maintainer upload.
+  * Make sure obsolete backends are removed on configure.
+    (Closes: #683754)
+
+ -- Didier Raboud <odyx@debian.org>  Fri, 03 Aug 2012 17:46:41 +0200
+
 cups (1.5.3-1) unstable; urgency=low
 
   [ Till Kamppeter ]
diff -Nru cups-1.5.3/debian/cups.postinst cups-1.5.3/debian/cups.postinst
--- cups-1.5.3/debian/cups.postinst	2012-05-23 14:49:38.000000000 +0200
+++ cups-1.5.3/debian/cups.postinst	2012-08-03 17:46:38.000000000 +0200
@@ -79,6 +79,10 @@
     db_get cupsys/backend && SELECTED=$RET
     # We remove the scsi backend from the output as it got removed in CUPS 1.5.0
     list=`echo $SELECTED | sed -e 's/, /,/g' | sed -e 's/scsi,*//g' | sed -e 's/parallel,*//g' | sed -e 's/serial,*//g'`
+    # Make sure the above dropped backends are effectively away
+    if dpkg --compare-versions "$2" lt-nl "1.5.3-1+UNRELEASED"; then
+        (cd /usr/lib/cups/backend && rm -f scsi parallel serial)
+    fi
     save_IFS=$IFS
     IFS=,
     (cd /usr/lib/cups/backend && rm -f http https ipp ipps lpd socket usb snmp dnssd mdns)
