diff -Nru buildbot-0.8.5/debian/changelog buildbot-0.8.5/debian/changelog
--- buildbot-0.8.5/debian/changelog	2011-10-23 00:10:36.000000000 +0200
+++ buildbot-0.8.5/debian/changelog	2012-01-17 17:53:32.000000000 +0100
@@ -1,3 +1,12 @@
+buildbot (0.8.5-1.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Fix "FTBFS due to missing build-dependency on python-twisted-words":
+    add python-twisted-words to Build-Depends (thanks to Ubuntu / Daniel T
+    Chen for the bug report and the fix). (Closes: #648384)
+
+ -- gregor herrmann <gregoa@debian.org>  Tue, 17 Jan 2012 17:53:00 +0100
+
 buildbot (0.8.5-1) unstable; urgency=low
 
   * New upstream version.
diff -Nru buildbot-0.8.5/debian/control buildbot-0.8.5/debian/control
--- buildbot-0.8.5/debian/control	2011-10-23 00:07:25.000000000 +0200
+++ buildbot-0.8.5/debian/control	2012-01-17 17:36:17.000000000 +0100
@@ -3,7 +3,7 @@
 Priority: optional
 Maintainer: Matthias Klose <doko@debian.org>
 Uploaders: Andriy Senkovych <jolly_roger@itblog.org.ua>
-Build-Depends: debhelper (>= 7.0.50~), python-all (>= 2.6.6-3), python-twisted-core (>= 2.0), python-twisted-web, python-twisted-mail, python-jinja2 (>= 2.1), python-mock, texinfo, python-sqlalchemy, python-migrate
+Build-Depends: debhelper (>= 7.0.50~), python-all (>= 2.6.6-3), python-twisted-core (>= 2.0), python-twisted-web, python-twisted-mail, python-jinja2 (>= 2.1), python-mock, texinfo, python-sqlalchemy, python-migrate, python-twisted-words
 Homepage: http://buildbot.net
 Standards-Version: 3.9.2
 X-Python-Version: >= 2.5
