diff --git a/src/archives.c b/src/archives.c
index a2cba6a..a94096f 100644
--- a/src/archives.c
+++ b/src/archives.c
@@ -683,6 +683,9 @@ tarobject(void *ctx, struct tar_entry *ti)
                _("backend dpkg-deb during `%.255s'"),
                path_quote_filename(fnamebuf, ti->name, 256));
     }
+
+    posix_fadvise(fd, 0, 0, POSIX_FADV_DONTNEED);
+
     r = ti->size % TARBLKSZ;
     if (r > 0)
       if (safe_read(tc->backendpipe, databuf, TARBLKSZ - r) == -1)
