From 9127b7868f3a28f35a7eb61984873983abea2dde Mon Sep 17 00:00:00 2001
From: Guido Guenther <agx@sigxcpu.org>
Date: Sun, 3 Jun 2007 18:10:45 +0200
Subject: [PATCH] mkboot: In case of yaboot/arcboot don't run anything to make the system bootable

---
 mkboot |   18 ++++++++++++++++++
 1 files changed, 18 insertions(+), 0 deletions(-)

diff --git a/mkboot b/mkboot
index 16d2b2c..711882d 100644
--- a/mkboot
+++ b/mkboot
@@ -52,6 +52,16 @@ grubcheck () {
     if ! which update-grub >/dev/null && ! which grub-install >/dev/null; then return 1; fi
 }
 
+# check whether Yaboot is installed
+yabootcheck () {
+    [ -f /etc/yaboot.conf ] && [ -x /usr/sbin/ybin ] && return 0 || return 1
+}
+
+# check whether Arcboot is installed
+arcbootcheck () {
+    [ -f /etc/arcboot.conf ] && [ -x /usr/sbin/arcboot ] && return 0 || return 1
+}
+
 # check whether LILO is installed
 lilocheck () {
     printf "\nChecking for LILO..."
@@ -360,6 +370,14 @@ if [ "$installkernel" ] ; then
         printf "\nGRUB is installed. To automatically switch to new kernels, point your\n"
         echo "default entry in menu.lst to $1"
         exit 0
+    elif yabootcheck; then
+        printf "\nYaboot is installed. To automatically switch to new kernels, point your\n"
+        echo "default entry in /etc/yaboot.conf to $1"
+        exit 0
+    elif arcbootcheck; then
+        printf "\nArcboot is installed. To automatically switch to new kernels, point your\n"
+        echo "default entry in /etc/arcboot.conf to $1"
+        exit 0
     fi
 
     if lilocheck; then
-- 
1.5.1.4

