--- djvulibre-3.5.19.orig/tools/c44.cpp	2007-12-09 18:22:27.000000000 +0100
+++ djvulibre-3.5.19/tools/c44.cpp	2007-12-09 19:06:35.000000000 +0100
@@ -704,7 +704,8 @@
       GP<ByteStream> gibs=ByteStream::create(g().pnmurl,"rb");
       ByteStream &ibs=*gibs;
       char prefix[16];
-      if (ibs.readall((void*)prefix, sizeof(prefix)) != sizeof(prefix))
+      memset(prefix, 0, sizeof(prefix));
+      if (ibs.readall((void*)prefix, sizeof(prefix)) < 10)
         G_THROW( ERR_MSG("c44.failed_pnm_header") );
 #ifdef DEFAULT_JPEG_TO_HALF_SIZE
       // Default specification for jpeg files
