diff -urN dwww-1.9.26.old/Makefile dwww-1.9.26/Makefile
--- dwww-1.9.26.old/Makefile	2005-04-07 23:40:08.000000000 +0000
+++ dwww-1.9.26/Makefile	2006-01-10 10:03:19.000000000 +0000
@@ -70,20 +70,20 @@
 
 
 %::%.in $(source_links)
-	perl -e '					\
-		use lib "."; 				\
-		use Debian::Dwww::Initialize; 		\
-		$$d=&DwwwInitialize;			\
-		$$v="";					\
-		foreach $$k (sort keys %{$$d}) {	\
-			$$v.="\t$$k=\"$$d->{$$k}\"\n" 	\
-				if $$k ne "DWWW_TITLE";	\
-		}					\
-		while (<>) {				\
-			s/#VERSION#/$(VERSION)/g;  	\
-			s/^.*#DWWWVARS#.*$$/$$v/g;	\
-			print;				\
-		}'					\
+	perl -e '					'\
+'		use lib "."; 				'\
+'		use Debian::Dwww::Initialize; 		'\
+'		$$d=&DwwwInitialize;			'\
+'		$$v="";					'\
+'		foreach $$k (sort keys %{$$d}) {	'\
+'			$$v.="\t$$k=\"$$d->{$$k}\"\n" 	'\
+'				if $$k ne "DWWW_TITLE";	'\
+'		}					'\
+'		while (<>) {				'\
+'			s/#VERSION#/$(VERSION)/g;  	'\
+'			s/^.*#DWWWVARS#.*$$/$$v/g;	'\
+'			print;				'\
+'		}'					\
 	  < $< > $@
 	touch -r $< $@
 
