--- toc2cue.cc	2006-09-28 11:52:00.134573980 -0500
+++ toc3cue.cc	2006-09-28 11:52:00.164569393 -0500
@@ -286,6 +286,9 @@
     return 1;
   }
 
+  if (toc->catalogValid())
+    out << "CATALOG " << toc->catalog() << "\n";
+
   out << "FILE \"" << binFileName << "\" BINARY" << "\n";
 
   long offset = 0;
@@ -326,13 +329,18 @@
       out << "    FLAGS DCP\n";
     }
 
+    if (trun->isrcValid() )
+      out << "    ISRC " << trun->isrc() << "\n";
+
     const SubTrack *strun;
     SubTrackIterator stitr(trun);
     int pregap = 0;
 
     for (strun = stitr.first(); strun != NULL; strun = stitr.next()) {
       if (strun->TrackData::type() == TrackData::ZERODATA) {
-	out << "    PREGAP " << trun->start().str() << "\n";
+//	out << "    PREGAP " << trun->start().str() << "\n";
+	out << "    INDEX 00 00:00:00\n";
+	offset += trun->start().lba();
 	pregap = 1;
       }
       else {

