diff -Nur desktop-base-4.0.5.old/debian/postrm desktop-base-4.0.5/debian/postrm
--- desktop-base-4.0.5.old/debian/postrm	1970-01-01 01:00:00.000000000 +0100
+++ desktop-base-4.0.5/debian/postrm	2008-01-14 13:32:25.000000000 +0100
@@ -0,0 +1,9 @@
+#!/bin/sh -e
+
+#DEBHELPER#
+
+if [ "$1" = "remove" ] || [ "$1" = "deconfigure" ]; then
+    if which update-grub2 > /dev/null ; then
+        update-grub2
+    fi
+fi
diff -Nur desktop-base-4.0.5.old/debian/prerm desktop-base-4.0.5/debian/prerm
--- desktop-base-4.0.5.old/debian/prerm	2008-01-09 15:44:36.000000000 +0100
+++ desktop-base-4.0.5/debian/prerm	2008-01-14 13:32:12.000000000 +0100
@@ -31,9 +31,5 @@
 
     update-alternatives --remove desktop-splash \
         /usr/share/images/desktop-base/Splash-Debian_red.png
-
-    if which update-grub2 > /dev/null ; then
-        update-grub2
-    fi
 fi
 
