diff -Nru chrony-2.1.1_ORIG/debian/control chrony-2.1.1_IPROUTE2/debian/control
--- chrony-2.1.1_ORIG/debian/control	2015-11-18 19:51:35.000000000 +0100
+++ chrony-2.1.1_IPROUTE2/debian/control	2016-03-14 20:53:30.943608440 +0100
@@ -21,7 +21,7 @@
  ${shlibs:Depends},
  ucf, timelimit,
  install-info,
- net-tools,
+ net-tools | iproute2
  lsb-base,
  adduser,
  util-linux (>= 2.20.1-5)
diff -Nru chrony-2.1.1_ORIG/debian/init chrony-2.1.1_IPROUTE2/debian/init
--- chrony-2.1.1_ORIG/debian/init	2015-11-18 19:51:35.000000000 +0100
+++ chrony-2.1.1_IPROUTE2/debian/init	2016-03-14 21:54:22.769482228 +0100
@@ -29,7 +29,8 @@
 
 putonline ()
 { # Do we have a default route?  If so put chronyd online.
-    if timelimit -q -s9 -t5 -- netstat -rn 2>/dev/null | grep -q '0\.0\.0\.0' 
+    if timelimit -q -s9 -t5 -- netstat -rn 2>/dev/null | grep -q '0\.0\.0\.0' \
+    || timelimit -q -s9 -t5 -- ip route list 0/0 >/dev/null
     then
 	sleep 2  # Chronyd can take a while to start.
 	KEY=$(awk '$1 ~ /^commandkey$/ { print $2; exit}' /etc/chrony/chrony.conf)
