--- ca-certificates-20090814/debian/postinst	2009-06-23 17:43:03.000000000 -0400
+++ ca-certificates-20090814+nmu1/debian/postinst	2010-07-30 14:46:24.000000000 -0400
@@ -74,6 +74,12 @@
 	     if memberp "$ca" "$CERTS_ENABLED"; then
 	       echo "$ca"
 	       # CERTS_ENABLED=$(delca "$ca" "$CERTS_ENABLED")
+         elif memberp "$ca" "$CERTS_AVAILABLE" ||
+              echo "$line" | grep -q '^!'; then
+           echo "!$ca"
+         elif [ -f /usr/share/ca-certificates/"$ca" ] || \
+              [ -f /usr/local/share/ca-certificates/"$ca" ]; then
+           echo "$ca"
 	     else
 	       echo "!$ca"
 	     fi
