Description: Fix uninitialised use on quit
Author: Olly Betts <olly@survex.com>
Bug-Debian: https://bugs.debian.org/786531
Forwarded: no
Last-Update: 2015-05-25

--- codeblocks-13.12.orig/src/plugins/debuggergdb/debuggergdb.cpp
+++ codeblocks-13.12/src/plugins/debuggergdb/debuggergdb.cpp
@@ -168,6 +168,7 @@ DebuggerGDB::DebuggerGDB() :
     m_NoDebugInfo(false),
     m_StoppedOnSignal(false),
     m_pProject(0),
+    m_bIsConsole(false),
     m_stopDebuggerConsoleClosed(false),
     m_TemporaryBreak(false),
     m_printElements(0)
