diff -u clamav-0.97+dfsg/debian/changelog clamav-0.97+dfsg/debian/changelog
--- clamav-0.97+dfsg/debian/changelog
+++ clamav-0.97+dfsg/debian/changelog
@@ -1,3 +1,11 @@
+clamav (0.97+dfsg-2.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Remove references to other libraries from dependency_libs field
+    (closes: #621206).
+
+ -- Luk Claes <luk@debian.org>  Sun, 29 May 2011 18:13:48 +0200
+
 clamav (0.97+dfsg-2) unstable; urgency=low
 
   [ Michael Tautschnig ]
diff -u clamav-0.97+dfsg/debian/rules clamav-0.97+dfsg/debian/rules
--- clamav-0.97+dfsg/debian/rules
+++ clamav-0.97+dfsg/debian/rules
@@ -192,7 +192,7 @@
 	dh_clean -k -a
 	# Add here commands to install the package into debian/clamav.
 	$(MAKE) install DESTDIR=$(CURDIR)/debian/tmp/
-	
+	sed -i "/dependency_libs/ s/'.*'/''/" `find . -name '*.la'`
 	touch install-arch-stamp
 
 clamav-dbg:
