diff -aur ctorrent-1.3.4.dnh3.3.2.orig/debian/rules ctorrent-1.3.4.dnh3.3.2/debian/rules
--- ctorrent-1.3.4.dnh3.3.2.orig/debian/rules	2011-12-29 06:46:48.000000000 +0100
+++ ctorrent-1.3.4.dnh3.3.2/debian/rules	2011-12-29 06:48:09.000000000 +0100
@@ -4,17 +4,9 @@
 DEB_BUILD_GNU_TYPE  ?= $(shell dpkg-architecture -qDEB_BUILD_GNU_TYPE)
 include /usr/share/dpatch/dpatch.make
 
-CFLAGS = -Wall -g
-
-ifneq (,$(findstring noopt,$(DEB_BUILD_OPTIONS)))
-	CFLAGS += -O0
-else
-	CFLAGS += -O2
-endif
-
 config.status: configure
 	dh_testdir
-	./configure --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE) --prefix=/usr --mandir=\$${prefix}/share/man --infodir=\$${prefix}/share/info CFLAGS="$(CFLAGS)" --disable-maintainer-mode --with-ssl=no
+	./configure --host=$(DEB_HOST_GNU_TYPE) --build=$(DEB_BUILD_GNU_TYPE) --prefix=/usr --mandir=\$${prefix}/share/man --infodir=\$${prefix}/share/info $(shell dpkg-buildflags --export=configure) --disable-maintainer-mode --with-ssl=no
 
 
 build: build-stamp
Nur in ctorrent-1.3.4.dnh3.3.2/debian: rules~.
