--- debian/control.old	2012-05-20 18:46:58.000000000 +0900
+++ debian/control	2012-05-20 18:47:38.000000000 +0900
@@ -135,7 +135,8 @@
 
 Package: dict-wn
 Architecture: all
-Depends: ${misc:Depends}, dictd | dict-server
+Depends: ${misc:Depends}
+Recommends: dictd | dict-server
 Provides: dictd-dictionary
 Suggests: dict-gcide
 Description: electronic lexical database of English language for dict
