=== modified file 'apt-pkg/acquire-item.cc'
--- apt-pkg/acquire-item.cc	2010-09-09 20:23:31 +0000
+++ apt-pkg/acquire-item.cc	2011-03-11 10:51:08 +0000
@@ -861,8 +861,8 @@
 
    struct stat Buf;
    if (stat(Final.c_str(),&Buf) != 0)
-      return "\nFail-Ignore: true";
-   return "\nFail-Ignore: true\nLast-Modified: " + TimeRFC1123(Buf.st_mtime);
+      return "\nFail-Ignore: true\nIndex-File: true";
+   return "\nFail-Ignore: true\nIndex-File: true\nLast-Modified: " + TimeRFC1123(Buf.st_mtime);
 }
 									/*}}}*/
 // AcqIndexTrans::Failed - Silence failure messages for missing files	/*{{{*/

