diff -u cacti-0.8.7b/debian/cacti.postinst cacti-0.8.7b/debian/cacti.postinst
--- cacti-0.8.7b/debian/cacti.postinst
+++ cacti-0.8.7b/debian/cacti.postinst
@@ -29,6 +29,9 @@
 #
 db_get cacti/webserver
 WWWTYPE="$RET"
+
+ucf /usr/share/doc/cacti/cacti.apache.conf /etc/cacti/apache.conf
+
 # done with debconf...
 db_stop
 
@@ -37,8 +40,6 @@
 chmod 0640 /var/log/cacti/rrd.log /var/log/cacti/cacti.log
 chown -R www-data.www-data /var/log/cacti/ 
 
-ucf /usr/share/doc/cacti/cacti.apache.conf /etc/cacti/apache.conf
-
 # update the webserver, if needed
 case "$WWWTYPE" in
 	"Apache")
diff -u cacti-0.8.7b/debian/changelog cacti-0.8.7b/debian/changelog
--- cacti-0.8.7b/debian/changelog
+++ cacti-0.8.7b/debian/changelog
@@ -1,3 +1,11 @@
+cacti (0.8.7b-1.1) unstable; urgency=high
+
+  * Non-maintainer upload.
+  * Move ucf call in cacti.postinst above db_stop to fix freeze during
+    installation. (Closes: #470066)
+
+ -- Andreas Henriksson <andreas@fatal.se>  Mon, 17 Mar 2008 12:52:17 +0100
+
 cacti (0.8.7b-1) unstable; urgency=high
 
   * New upstream release.  Fixes multiple security vulnerabilities (no
