Index: tools.dbk
===================================================================
--- tools.dbk	(revisione 8928)
+++ tools.dbk	(copia locale)
@@ -193,24 +193,6 @@
 </para>
 </section>
 
-<section id="yada">
-<title><systemitem role="package">yada</systemitem></title>
-<para>
-<systemitem role="package">yada</systemitem> is another packaging helper tool.
-It uses a <filename>debian/packages</filename> file to auto-generate
-<filename>debian/rules</filename> and other necessary files in the
-<filename>debian/</filename> subdirectory.  The
-<filename>debian/packages</filename> file contains instruction to build
-packages and there is no need to create any <filename>Makefile</filename>
-files.  There is possibility to use macro engine similar to the one used in
-SPECS files from RPM source packages.
-</para>
-<para>
-For more informations see <ulink
-url="http://yada.alioth.debian.org/"><literal>YADA</literal> site</ulink>.
-</para>
-</section>
-
 <section id="equivs">
 <title><systemitem role="package">equivs</systemitem></title>
 <para>
