From adee07a29e00c724e627434fe52d18ff009719f1 Mon Sep 17 00:00:00 2001
From: Sergey Alyoshin <alyoshin.s@gmail.com>
Date: Thu, 27 Mar 2014 23:29:38 +0400
Subject: [PATCH 2/3] Mark message as translatable in .glade file

---
 apt-listchanges/apt-listchanges.glade | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/apt-listchanges/apt-listchanges.glade b/apt-listchanges/apt-listchanges.glade
index 1389ec0..d5998a9 100644
--- a/apt-listchanges/apt-listchanges.glade
+++ b/apt-listchanges/apt-listchanges.glade
@@ -32,7 +32,7 @@
       <child>
 	<widget class="GtkLabel" id="label_header">
 	  <property name="visible">True</property>
-	  <property name="label" translatable="yes" context="yes">&lt;big&gt;&lt;b&gt;Changelogs&lt;/b&gt;&lt;/big&gt;
+	  <property name="label" translatable="yes">&lt;big&gt;&lt;b&gt;Changelogs&lt;/b&gt;&lt;/big&gt;
 
 The following changes are found in the packages you are about to install:</property>
 	  <property name="use_underline">True</property>
-- 
1.9.1

