diff -u cows/calvin.cow cows.new/calvin.cow
--- cows/calvin.cow	2012-05-28 00:09:11.000000000 -0300
+++ cows.new/calvin.cow	2014-03-21 15:11:28.151252247 -0300
@@ -5,7 +5,7 @@
  $thoughts                   .,
    $thoughts         .      .TR   d'
      $thoughts      k,l    .R.b  .t .Je
-       $thoughts   .P q.   a|.b .f .Z%		
+       $thoughts   .P q.   a|.b .f .Z%
            .b .h  .E` # J: 2`     .
       .,.a .E  ,L.M'  ?:b `| ..J9!`.,
        q,.h.M`   `..,   ..,""` ..2"`
diff -u cows/cock.cow cows.new/cock.cow
--- cows/cock.cow	2012-05-28 00:09:11.000000000 -0300
+++ cows.new/cock.cow	2014-03-21 15:10:42.132861863 -0300
@@ -5,8 +5,8 @@
        |  0 >>
        |___|
  __((_<|   |
-(          |	
-(__________)	
+(          |
+(__________)
    |      |
    |      |
    /\\     /\\
diff -u cows/moofasa.cow cows.new/moofasa.cow
--- cows/moofasa.cow	2012-05-28 00:09:05.000000000 -0300
+++ cows.new/moofasa.cow	2014-03-21 15:11:12.697136066 -0300
@@ -10,5 +10,5 @@
            \\____/|----w |
                 ||     ||
 
-	         Moofasa
+                 Moofasa
 EOC
