--- /usr/share/cowsay/cows/calvin.cow	2012-05-28 00:09:11.000000000 -0300
+++ ./calvin.cow	2014-03-21 14:41:50.371857310 -0300
@@ -5,7 +5,7 @@
  $thoughts                   .,
    $thoughts         .      .TR   d'
      $thoughts      k,l    .R.b  .t .Je
-       $thoughts   .P q.   a|.b .f .Z%		
+       $thoughts   .P q.   a|.b .f .Z%
            .b .h  .E` # J: 2`     .
       .,.a .E  ,L.M'  ?:b `| ..J9!`.,
        q,.h.M`   `..,   ..,""` ..2"`
