From dc2e46b620803ac12feeb6b95e5137643c863332 Mon Sep 17 00:00:00 2001
From: Antonio Terceiro <terceiro@debian.org>
Date: Wed, 12 Feb 2025 10:57:12 -0300
Subject: [PATCH] Fix check for avahi proxy

---
 auto-apt-proxy | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/auto-apt-proxy b/auto-apt-proxy
index 34cc1b6..670a853 100755
--- a/auto-apt-proxy
+++ b/auto-apt-proxy
@@ -204,7 +204,7 @@ detect_avahi_local() {
   local avahi_result proxy
   if avahi_result="$(_lookup_avahi_local)"; then
     proxy=$(_print_avahi_proxy "${avahi_result}")
-    if hit "${proxy}" >/dev/null 2>&1; then
+    if curl --silent --output /dev/null "${proxy}"; then
       echo "${proxy}"
     else
       return 1
-- 
2.47.2

