diff -Nru phonon-4.6.0.0/debian/changelog phonon-4.6.0.0/debian/changelog
--- phonon-4.6.0.0/debian/changelog	2012-05-02 20:55:41.000000000 +0900
+++ phonon-4.6.0.0/debian/changelog	2012-09-20 12:33:37.000000000 +0900
@@ -1,3 +1,13 @@
+phonon (4:4.6.0.0-2.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * debian/control
+    - libphonon-dev: "Depends: phonon-backend-null | phonon-backend" 
+      it fixes dependency problem (Closes: #680822) 
+      Thanks to Evgeni Golov <evgeni@debian.org> for investigation.
+
+ -- Hideki Yamane <henrich@debian.org>  Thu, 20 Sep 2012 12:33:34 +0900
+
 phonon (4:4.6.0.0-2) unstable; urgency=low
 
   [ Debconf translation updates ]
diff -Nru phonon-4.6.0.0/debian/control phonon-4.6.0.0/debian/control
--- phonon-4.6.0.0/debian/control	2012-05-02 20:55:41.000000000 +0900
+++ phonon-4.6.0.0/debian/control	2012-09-20 12:05:06.000000000 +0900
@@ -38,7 +38,7 @@
 Architecture: any
 Section: libdevel
 Depends: libphonon4 (= ${binary:Version}),
-         phonon-backend-null | phonon,
+         phonon-backend-null | phonon-backend,
          ${misc:Depends},
          ${sameVersionDep:libqt4-dev:libphonon4},
          ${shlibs:Depends}
