--- debian/control.orig	2021-02-25 16:27:19.056189876 -0300
+++ debian/control	2021-02-25 16:27:34.484574772 -0300
@@ -122,8 +122,9 @@
 
 Package: binutils-dev
 Architecture: any
+Multi-Arch: same
 Priority: optional
-Depends: binutils (= ${binary:Version}), libbinutils (= ${binary:Version}),
+Depends: libbinutils (= ${binary:Version}),
   libctf0 (= ${binary:Version}), libctf-nobfd0 (= ${binary:Version})
 Conflicts: libbfd-dev
 Provides: libbfd-dev
