Index: abiword-2.9.2+svn20120603/src/af/xap/xp/xap_EncodingManager.cpp
===================================================================
--- abiword-2.9.2+svn20120603.orig/src/af/xap/xp/xap_EncodingManager.cpp	2012-06-04 13:20:24.000000000 +0800
+++ abiword-2.9.2+svn20120603/src/af/xap/xp/xap_EncodingManager.cpp	2012-08-13 01:34:47.758718953 +0800
@@ -1260,10 +1260,10 @@
 		"UTF-16-BE",		// my guess
 		0 };
 	static const char * (szUCS2LENames[]) = {
+		"UTF-16LE",			// superset
 		"UCS-2LE",			// preferred
 		"UCS-2-LE",			// older libiconv
 		"UNICODELITTLE",	// older glibc
-		"UTF-16LE",			// superset
 		"UTF-16-LE",		// my guess
 		0 };
 
