diff -Nru antlr3-3.0.1+dfsg.orig//src/org/antlr/codegen/templates/Python/Python.stg antlr3-3.0.1+dfsg/src/org/antlr/codegen/templates/Python/Python.stg
--- antlr3-3.0.1+dfsg.orig//src/org/antlr/codegen/templates/Python/Python.stg	2007-08-14 00:40:47.000000000 +0200
+++ antlr3-3.0.1+dfsg/src/org/antlr/codegen/templates/Python/Python.stg	2010-04-28 08:02:14.000000000 +0200
@@ -77,7 +77,7 @@
         self.ruleMemo = {}
 <endif>
 
-        <cyclicDFAs:{dfa | <cyclicDFAInit(dfa)>}; seperator="\n">
+        <cyclicDFAs:{dfa | <cyclicDFAInit(dfa)>}; separator="\n">
 
         <actions.lexer.init>
 
@@ -179,7 +179,7 @@
         self.ruleMemo = {}
 <endif>
 
-        <cyclicDFAs:{dfa | <cyclicDFAInit(dfa)>}; seperator="\n">
+        <cyclicDFAs:{dfa | <cyclicDFAInit(dfa)>}; separator="\n">
 
         <scopes:{<if(it.isDynamicGlobalScope)><globalAttributeScopeStack(scope=it)><endif>}>
 	<rules:{<ruleAttributeScopeStack(scope=it.ruleDescriptor.ruleScope)>}>
