Description: fix bug #972585
Author: Julien Puydt
Forwarded: https://bugs.debian.org/cgi-bin/bugreport.cgi?bug=972585

--- devscripts-2.20.4.orig/lib/Devscripts/Uscan/Downloader.pm
+++ devscripts-2.20.4/lib/Devscripts/Uscan/Downloader.pm
@@ -188,6 +188,8 @@
             uscan_die "Unknown suffix file to repack: $suffix";
         }
         chdir "$curdir" or uscan_die("Unable to chdir($curdir): $!");
+
+	uscan_exec_no_fail('rm', '-fr', $gitrepo_dir);
     }
     return 1;
 }
