diff --git a/scripts/debcommit.pl b/scripts/debcommit.pl
index 815b786..a9886ab 100755
--- a/scripts/debcommit.pl
+++ b/scripts/debcommit.pl
@@ -452,7 +452,7 @@ sub getprog {
 	}
     }
 
-    # .bzr or .git may be in a parent directory, rather than the current
+    # .bzr, .git or .hg may be in a parent directory, rather than the current
     # directory, if multiple packages are kept in one repository.
     my $dir=getcwd();
     while ($dir=~s/[^\/]*\/?$// && length $dir) {
@@ -462,6 +462,9 @@ sub getprog {
 	if (-d "$dir/.git") {
 	    return "git";
 	}
+	if (-d "$dir/.hg") {
+	    return "hg";
+	}
     }
 
     die "debcommit: not in a cvs, subversion, baz, bzr, git, hg, svk or darcs working copy\n";
