Index: anarchism-14.0/debian/control
===================================================================
--- anarchism-14.0.orig/debian/control
+++ anarchism-14.0/debian/control
@@ -5,7 +5,7 @@ Maintainer: Holger Levsen <holger@debian
 Build-Depends: debhelper (>= 8.0.0)
 Build-Depends-Indep: lynx, sed (>= 4.2.1)
 Standards-Version: 3.9.6
-Homepage: http://www.infoshop.org/AnAnarchistFAQ
+Homepage: http://www.anarchistfaq.org
 Vcs-Git: git://anonscm.debian.org/collab-maint/anarchism.git
 Vcs-Browser: https://anonscm.debian.org/gitweb/?p=collab-maint/anarchism.git
 