--- /usr/sbin/checkrestart	2010-10-14 21:34:26.000000000 +0200
+++ /tmp/checkrestart	2011-10-01 17:54:26.893411102 +0200
@@ -180,15 +180,13 @@
             if path.startswith('/etc/init.d/'):
                 if path.endswith('.sh'):
                     continue
-                package.initscripts.append(path)
+                package.initscripts.add(path)
         # Alternatively, find init.d scripts that match the process name
         if len(package.initscripts) == 0:
             for process in package.processes:
                 path = '/etc/init.d/' + os.path.basename(process.program)
                 if os.path.exists(path):
-                    package.initscripts.append(path)
-            # Remove duplicate inits
-            package.initscripts = [ u for u in package.initscripts if u not in locals()['_[1]'] ]
+                    package.initscripts.add(path)
 
     restartable = []
     nonrestartable = []
@@ -452,7 +450,8 @@
 class Package:
     def __init__(self, name):
         self.name = name
-        self.initscripts = []
+        # use a set, we don't need duplicates
+        self.initscripts = set()
         self.processes = []
 
 if __name__ == '__main__':
