--- orig/apt-show-versions	2009-06-19 00:00:00.000000000 +0200
+++ /etc/bash_completion.d/apt-show-versions	2012-03-18 02:21:04.794074335 +0100
@@ -27,7 +27,7 @@
             return 0
             ;;
         -p|--package)
-            COMPREPLY=( $(_comp_dpkg_installed_packages $cur) )
+            COMPREPLY=( $( apt-cache --no-generate pkgnames "$cur" 2> /dev/null ) )
             return 0
             ;;
         -stf|--status-file)
@@ -44,7 +44,7 @@
         COMPREPLY=( $(compgen -W "$opts" -- $cur) )
         return 0
     else
-        COMPREPLY=( $(_comp_dpkg_installed_packages $cur) )
+        COMPREPLY=( $( apt-cache --no-generate pkgnames "$cur" 2> /dev/null ) )
         return 0
     fi
 }
