--- debianutils-2.8.4.orig/run-parts.c	2004-03-24 21:20:31.000000000 +0100
+++ debianutils-2.8.4/run-parts.c	2005-07-14 17:24:43.000000000 +0200
@@ -132,7 +132,7 @@
 	(&hierre, "^_?([a-z0-9_.]+-)+[a-z0-9]+$", REG_EXTENDED | REG_NOSUB)
 	|| regcomp(&excsre, "^[a-z0-9-].*dpkg-(old|dist)$",
 		   REG_EXTENDED | REG_NOSUB)
-	|| regcomp(&tradre, "^[a-z0-9][a-z0-9-]*$", REG_NOSUB)) {
+	|| regcomp(&tradre, "^[a-z0-9][a-z0-9-.]*$", REG_NOSUB)) {
       error("regex failure");
       exit(1);
     }
