From 8fcd7119c971a2181f4bf38332236b018ef5f9be Mon Sep 17 00:00:00 2001
From: Niels Thykier <niels@thykier.net>
Date: Wed, 6 Nov 2024 11:21:26 +0000
Subject: [PATCH 2/4] d/control: Add `Vcs-*` headers

---
 debian/control | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/debian/control b/debian/control
index 0a49d0e..0eafd6b 100644
--- a/debian/control
+++ b/debian/control
@@ -5,6 +5,8 @@ Maintainer: Ian Jackson <ijackson@chiark.greenend.org.uk>
 Standards-Version: 3.9.1
 Build-Depends: debhelper-compat (= 13),
 Rules-Requires-Root: no
+Vcs-Git: git://git.chiark.greenend.org.uk/~ian/authbind.git
+Vcs-Browser: https://www.chiark.greenend.org.uk/ucgi/~ian/git/authbind.git
 
 Package: authbind
 Architecture: any
-- 
2.45.2

