From 81197dddb614a87905dae925660851b64713d2e2 Mon Sep 17 00:00:00 2001
From: Niels Thykier <niels@thykier.net>
Date: Wed, 6 Nov 2024 11:23:05 +0000
Subject: [PATCH 3/4] Makefile: Fix missing `install -d` of `$(bin_dir)`

---
 Makefile     | 2 +-
 debian/rules | 2 --
 2 files changed, 1 insertion(+), 3 deletions(-)

diff --git a/Makefile b/Makefile
index 37a8673..486b98f 100644
--- a/Makefile
+++ b/Makefile
@@ -59,7 +59,7 @@ MANPAGES_8=		authbind-helper.8
 all:			$(TARGETS)
 
 install:		$(TARGETS)
-		$(INSTALL_DIR) $(lib_dir) $(man1_dir) $(man8_dir)
+		$(INSTALL_DIR) $(bin_dir) $(lib_dir) $(man1_dir) $(man8_dir)
 		$(INSTALL_PROGRAM) $(BINTARGETS) $(bin_dir)/.
 		$(INSTALL_FILE) $(LIBTARGET) $(lib_dir)/.
 		$(STRIP) --strip-unneeded $(lib_dir)/$(LIBTARGET)
diff --git a/debian/rules b/debian/rules
index b0407cc..579a34a 100755
--- a/debian/rules
+++ b/debian/rules
@@ -50,8 +50,6 @@ override_dh_auto_build:
 
 
 override_dh_auto_install:
-	# Upstream's make install forgets to create usr/bin (but only that one) as of 2.1.2
-	$(INSTALL_DIR) debian/$(package)/usr/bin
 	dh_auto_install -- prefix=debian/$(package)/usr etc_dir=debian/$(package)/etc/authbind \
 		install install_man
 
-- 
2.45.2

