diff -Naur alsa-tools.orig/alsa-tools-1.0.28/debian/changelog alsa-tools/alsa-tools-1.0.28/debian/changelog
--- alsa-tools.orig/alsa-tools-1.0.28/debian/changelog	2014-08-01 14:37:56.111476729 -0500
+++ alsa-tools/alsa-tools-1.0.28/debian/changelog	2014-08-01 14:41:21.059480298 -0500
@@ -1,3 +1,11 @@
+alsa-tools (1.0.28-2) unstable; urgency=low
+
+  * Fix FTBFS with clang
+    - Fixed undefined undefined reference error in
+      as10k1/as10k1.c
+
+ -- Arthur Marble <arthur@info9.net>  Fri, 01 Aug 2014 14:41:15 -0500
+
 alsa-tools (1.0.28-1) unstable; urgency=medium
 
   [ Luke Yelavich ]
diff -Naur alsa-tools.orig/alsa-tools-1.0.28/debian/patches/clang-ftbfs.diff alsa-tools/alsa-tools-1.0.28/debian/patches/clang-ftbfs.diff 
--- alsa-tools.orig/alsa-tools-1.0.28/debian/patches/clang-ftbfs.diff	1969-12-31 18:00:00.000000000 -0600
+++ alsa-tools/alsa-tools-1.0.28/debian/patches/clang-ftbfs.diff	2014-08-01 14:39:03.443477901 -0500
@@ -0,0 +1,11 @@
+--- a/as10k1/as10k1.c
++++ b/as10k1/as10k1.c
+@@ -366,7 +366,7 @@ void as_exit(const char *message)
+ 	exit(1);
+ }
+ 
+-inline void output_tram_line( struct list_head *line_head, int type){
++static inline void output_tram_line( struct list_head *line_head, int type){
+         
+         struct tram *tram_sym;
+         struct list_head *entry;
diff -Naur alsa-tools.orig/alsa-tools-1.0.28/debian/patches/series alsa-tools/alsa-tools-1.0.28/debian/patches/series 
--- alsa-tools.orig/alsa-tools-1.0.28/debian/patches/series	2014-08-01 14:37:56.111476729 -0500
+++ alsa-tools/alsa-tools-1.0.28/debian/patches/series	2014-08-01 14:38:41.307477516 -0500
@@ -1,3 +1,4 @@
 usx2yloader_udev.patch
 firmware_locations.patch
 envy24control_config_dir.patch
+clang-ftbfs.diff
