--- /tmp/DuaGWIs59w/dictionaries-common-0.80.1/scripts/Debian/DictionariesCommon.pm.in	2007-04-10 16:54:56.000000000 +0200
+++ /tmp/ODsPZYCuv9/dictionaries-common-0.80.2/scripts/Debian/DictionariesCommon.pm.in	2007-05-10 12:43:16.000000000 +0200
@@ -394,15 +394,17 @@
   my $class        = "ispell";
   my $dictionaries = loaddb ($class);
   my $php          = generate_comment ("### ");
+  my @dictlist;
   
   $php .= "<?php\n\$SQSPELL_APP = array (\n";
   foreach ( keys %$dictionaries ){
     next if m/.*[^a-z]tex[^a-z]/i;            # Discard tex variants
     my $spellchecker_params =
 	&dc_get_spellchecker_params($class,$dictionaries->{$_});
-    $php .= qq {  '$_' => 'ispell -a $spellchecker_params'\n};
+    push(@dictlist, qq {  '$_' => 'ispell -a $spellchecker_params'\n});
   }
-  $php .= ");\n?>\n";
+  $php .= join(",", @dictlist);
+  $php .= ");\n";
   
   open (PHP, "> $cachedir/$squirrelmailsupport")
       or die "Cannot open SquirrelMail cache file";
