diff -Naur squirrelmail-1.4.9a.thijs/plugins/squirrelspell/sqspell_config.php squirrelmail-1.4.9a/plugins/squirrelspell/sqspell_config.php
--- squirrelmail-1.4.9a.thijs/plugins/squirrelspell/sqspell_config.php	2007-01-31 17:46:14.000000000 +0100
+++ squirrelmail-1.4.9a/plugins/squirrelspell/sqspell_config.php	2007-01-31 18:03:20.000000000 +0100
@@ -27,21 +27,21 @@
  *                     'Spanish' => 'aspell -d spanish -a' );
  */
 
-# Debian: if dictionaries-common >= 2.50 is available, detect the
+# Debian: if dictionaries-common >= 0.22.50 is available, detect the
 # installed dictionaries automatically.
 if ( is_readable ( '/var/cache/dictionaries-common/ispell-dicts-list.txt' ) ) {
     $dicts = file( '/var/cache/dictionaries-common/ispell-dicts-list.txt' );
     $SQSPELL_APP = array();
     foreach ($dicts as $dict) {
         preg_match('/(\S+)\s+\((.+)\)/', $dict, $dparts);
-        $SQSPELL_APP[$dparts[2]] = 'ispell -d ' . $dparts[1] . ' -a';
+        $SQSPELL_APP[$dparts[2]] = 'ispell-wrapper --language="' . $dparts[2] . '" -a';
     }
 } else {
     $SQSPELL_APP = array('English' => 'ispell -a',
                  'Spanish' => 'ispell -d spanish -a');
 }
 
-# Debian: if dictionaries-common >= 2.50 is available, detect the
+# Debian: if dictionaries-common >= 0.22.50 is available, detect the
 # default dictionary automatically.
 if ( is_readable ( '/etc/dictionaries-common/ispell-default' ) ) {
     $dictdefault = file ( '/etc/dictionaries-common/ispell-default' );
