Index: developers-reference.sgml
===================================================================
RCS file: /cvs/debian-doc/ddp/manuals.sgml/developers-reference/developers-reference.sgml,v
retrieving revision 1.334
diff -u -r1.334 developers-reference.sgml
--- developers-reference.sgml	4 Aug 2007 17:33:46 -0000	1.334
+++ developers-reference.sgml	4 Oct 2007 01:30:44 -0000
@@ -4013,7 +4013,7 @@
           <p>
 There are additional fields for the location of the Version Control System
 in <file>debian/control</file>.
-          <sect2><heading>XS-Vcs-Browser</heading>
+          <sect2><heading>Vcs-Browser</heading>
           <p>
 Value of this field should be a <tt>http://</tt> URL pointing to a
 web-browsable copy of the Version Control System repository used to
@@ -4023,7 +4023,7 @@
 browse the latest work done on the package (e.g. when looking for the
 patch fixing a bug tagged as <tt>pending</tt> in the bug tracking
 system).
-          <sect2><heading>XS-Vcs-*</heading>
+          <sect2><heading>Vcs-*</heading>
           <p>
 Value of this field should be a string identifying unequivocally the
 location of the Version Control System repository used to maintain the
@@ -4050,14 +4050,14 @@
 repository of the <package>vim</package> package is shown. Note how the
 URL is in the <tt>svn://</tt> scheme (instead of <tt>svn+ssh://</tt>) and
 how it points to the <file>trunk/</file> branch. The use of the
-<tt>XS-Vcs-Browser</tt> field described above is also shown.
+<tt>Vcs-Browser</tt> field described above is also shown.
 <example>
   Source: vim
   Section: editors
   Priority: optional
   &lt;snip&gt;
-  XS-Vcs-Svn: svn://svn.debian.org/svn/pkg-vim/trunk/packages/vim
-  XS-Vcs-Browser: http://svn.debian.org/wsvn/pkg-vim/trunk/packages/vim
+  Vcs-Svn: svn://svn.debian.org/svn/pkg-vim/trunk/packages/vim
+  Vcs-Browser: http://svn.debian.org/wsvn/pkg-vim/trunk/packages/vim
 </example>
         </sect1>
       </sect>
