Index: resources.dbk
===================================================================
--- resources.dbk	(revision 9204)
+++ resources.dbk	(working copy)
@@ -881,7 +881,7 @@
 &email-debian-changes;.  If a package is released with
 <literal>Distribution</literal> set to <literal>unstable</literal> or
 <literal>experimental</literal>, the announcement will be posted to
-&email-debian-devel-changes; instead.
+&email-debian-devel-changes; or &email-debian-experimental-changes; instead.
 </para>
 <para>
 Though ftp-master is restricted, a copy of the installation is available to all
Index: common.ent
===================================================================
--- common.ent	(revision 9204)
+++ common.ent	(working copy)
@@ -139,6 +139,7 @@
 <!ENTITY email-debian-devel-announce "<email>debian-devel-announce@&lists-host;</email>">
 <!ENTITY email-debian-devel-changes "<email>debian-devel-changes@lists.debian.org</email>">
 <!ENTITY email-debian-devel-req "<email>debian-devel-REQUEST@&lists-host;</email>">
+<!ENTITY email-debian-experimental-changes "<email>debian-experimental-changes@&lists-host;</email>">
 <!ENTITY email-debian-i18n "<email>debian-i18n@&lists-host;</email>">
 <!ENTITY email-debian-mentors "<email>debian-mentors@&lists-host;</email>">
 <!ENTITY email-debian-private "<email>debian-private@&lists-host;</email>">
