diff --git a/doc/devel/internals/build.sh b/doc/devel/internals/build.sh
index 712172b..d7cec50 100755
--- a/doc/devel/internals/build.sh
+++ b/doc/devel/internals/build.sh
@@ -7,10 +7,10 @@ if [ -n "$xsltproc" ] ; then
     if [ -e "$stylesheet" ]; then
 	$xsltproc --xinclude style-html.xsl internals.xml
     else
-	echo stylesheet missing; please install the docbook-xsl Debian package
+	echo "stylesheet missing; please install the docbook-xsl Debian package"
 	exit 1
     fi
 else
-    echo xsltproc not found; please install the xsltproc Debian package
+    echo "xsltproc not found; please install the xsltproc Debian package"
     exit 1
 fi
