diff -urN elinks-0.11.1/debian/control elinks-0.11.1+with-JS/debian/control
--- elinks-0.11.1/debian/control	2007-06-12 16:31:24.000000000 +0200
+++ elinks-0.11.1+with-JS/debian/control	2007-06-12 16:32:28.000000000 +0200
@@ -3,7 +3,7 @@
 Priority: optional
 Maintainer: Peter Gervai <grin@tolna.net>
 Standards-Version: 3.6.1
-Build-Depends: debhelper (>= 4.0.0), m4, bison, libgnutls-dev, libgpmg1-dev [!hurd-i386 !kfreebsd-i386 !kfreebsd-amd64], liblua50-dev, liblualib50-dev, libbz2-dev, libexpat1-dev (>= 1.95.6), libperl-dev, smbclient, autotools-dev
+Build-Depends: debhelper (>= 4.0.0), m4, bison, libgnutls-dev, libgpmg1-dev [!hurd-i386 !kfreebsd-i386 !kfreebsd-amd64], liblua50-dev, liblualib50-dev, libbz2-dev, libexpat1-dev (>= 1.95.6), libperl-dev, smbclient, autotools-dev, libsmjs-dev
 
 Package: elinks
 Architecture: any
diff -urN elinks-0.11.1/debian/rules elinks-0.11.1+with-JS/debian/rules
--- elinks-0.11.1/debian/rules	2007-06-12 16:31:24.000000000 +0200
+++ elinks-0.11.1+with-JS/debian/rules	2007-06-12 16:32:19.000000000 +0200
@@ -65,7 +65,7 @@
  --without-x \
  --with-gnutls=/usr \
  --with-perl --enable-nntp --enable-256-colors --enable-leds  \
- --without-spidermonkey --enable-html-highlight --disable-smb
+ --enable-html-highlight --disable-smb
 # spidermonkey: libsmjs-dev
 # perl libperl-dev
 
