--- README.make.orig	2006-05-24 01:20:32.000000000 +0300
+++ README.make	2006-05-24 01:24:09.000000000 +0300
@@ -39 +39 @@
-one failure condition that occures when a header file is erased. In
+one failure condition that occurs when a header file is erased. In
@@ -43,2 +43,2 @@
-The final significant deviation from normal make practicies is 
-in how the build directory is managed. It is not mearly a mirror of
+The final significant deviation from normal make practices is 
+in how the build directory is managed. It is not nearly a mirror of
@@ -54 +54 @@
-     cmndline/
+     cmdline/
@@ -59 +59 @@
-all runnable programs are within the bin/ directory, a huge benifit for
+all runnable programs are within the bin/ directory, a huge benefit for
@@ -90 +90 @@
-  aclocal -I buidlib
+  aclocal -I buildlib
@@ -92 +92 @@
-[Altertatively you can run make startup in the top level build dir]
+[Alternatively you can run make startup in the top level build dir]
@@ -106 +106 @@
-Autoconf will also wite a makefile into the top level of the build dir, 
+Autoconf will also write a makefile into the top level of the build dir, 
