From a1f16b7822d1fc41e714e9517d2e7462929ea23c Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas@debian.org>
Date: Sat, 6 Mar 2021 17:13:13 +0100
Subject: [PATCH 3/3] Cosmetic improvement in readability of debian/rules

---
 debian/rules | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/debian/rules b/debian/rules
index 2f5a068..ae075b3 100755
--- a/debian/rules
+++ b/debian/rules
@@ -3,6 +3,5 @@
 %:
 	dh $@
 
-override_dh_install:
-	dh_install
-	chmod -x $(CURDIR)/debian/alsa-ucm-conf/usr/share/alsa/ucm2/kblrt5660/*
+execute_after_dh_fixperms:
+	chmod -x debian/alsa-ucm-conf/usr/share/alsa/ucm2/kblrt5660/*
-- 
2.30.1

