diff -u dash-0.5.8/debian/implicit dash-0.5.8/debian/implicit
--- dash-0.5.8/debian/implicit
+++ dash-0.5.8/debian/implicit
@@ -92,6 +92,7 @@
 	@cd debian/$* && find * -path 'DEBIAN' -prune -o \
 	  -type f -print0 | LC_ALL=C sort -z | \
 	    xargs -0r md5sum >>DEBIAN/md5sums
+	@chmod 0644 debian/*/DEBIAN/md5sums
 %.deb-DEBIAN: %.deb-checkdir %.deb-DEBIAN-base %.deb-DEBIAN-scripts \
 	  %.deb-DEBIAN-md5sums
 	: debian/$*/DEBIAN/ ok
