From 938bd71e0e8af3d3c854e54a12f5fea7d1a5e63e Mon Sep 17 00:00:00 2001
From: Miriam Espana Acebal <miriam.espana@canonical.com>
Date: Fri, 2 Feb 2024 14:19:36 +0100
Subject: [PATCH] * d/prerm, d/postinst: fix path according to /usr movement

---
 debian/postinst | 2 +-
 debian/prerm    | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/postinst b/debian/postinst
index e92a53d..456e73a 100644
--- a/debian/postinst
+++ b/debian/postinst
@@ -3,7 +3,7 @@
 set -e
 
 if [ "$1" = configure ]; then
-    update-alternatives --install /bin/mt mt /bin/mt-gnu 10 \
+    update-alternatives --install /usr/bin/mt mt /usr/bin/mt-gnu 10 \
       --slave \
 	/usr/share/man/man1/mt.1.gz mt.1.gz /usr/share/man/man1/mt-gnu.1.gz
 fi
diff --git a/debian/prerm b/debian/prerm
index 89490ea..08f3b39 100644
--- a/debian/prerm
+++ b/debian/prerm
@@ -3,5 +3,5 @@
 set -e
 
 if [ "$1" = remove ]; then
-    update-alternatives --remove mt /bin/mt-gnu
+    update-alternatives --remove mt /usr/bin/mt-gnu
 fi
-- 
2.40.1

