--- /usr/bin/dh_make	2007-08-31 08:36:25.000000000 +0200
+++ dh_make	2007-10-12 19:55:36.000000000 +0200
@@ -3,6 +3,7 @@
 # dh_make - Script to Debianize a source archive
 #
 use Getopt::Long;
+use File::Basename;
 use Cwd;
 use strict;
 
@@ -80,6 +81,7 @@
       $line =~ s/#PHONY_CONFIGURE#/$main::PHONY_CONFIGURE/g;
       $line =~ s/#CDBS_CLASS#/$main::CDBS_CLASS/g;
       $line =~ s/#BUILD_DEPS#/$main::BUILD_DEPS/g;
+      $line =~ s/#SOURCE_FILE_BASE#/$main::source_file_base/g;
 
       print OUT $line;
     }
@@ -370,6 +372,7 @@
 our $year = `date '+%Y'`;
 chomp $year;
 parse_args();
+our $source_file_base=basename($source_file);
 if ( ! $overlay eq "" )
 {
   #setup for overlay mode
