--- courier-0.65.0.orig/debian/rules	2010-09-23 04:48:42.000000000 +0000
+++ courier-0.65.0/debian/rules	2010-09-23 04:48:05.000000000 +0000
@@ -205,6 +205,7 @@
 # - start daemons by default
 	perl -i -pe 's/^POP3DSTART=NO/POP3DSTART=YES/' $(BUILD)/etc/courier/pop3d
 	perl -i -pe 's/^MAXPERIP=\d+/MAXPERIP=20/; s/^IMAPDSTART=NO/IMAPDSTART=YES/' $(BUILD)/etc/courier/imapd
+	perl -i -pe 's/^IMAP_ULIMITD=\d+/IMAP_ULIMITD=132072/;' $(BUILD)/etc/courier/imapd
 # - change default LDAP server
 	perl -i -pe 's/^(LDAP_SERVER\s+)ldap.example.com/$$1localhost/' $(BUILD)/etc/courier/ldapaliasrc
 	perl -i -pe 's/^POP3DSSLSTART=NO/POP3DSSLSTART=YES/' $(BUILD)/etc/courier/pop3d-ssl
