diff -ru fbi-2.07.pristine/debian/control fbi-2.07/debian/control
--- fbi-2.07.pristine/debian/control	2012-03-07 18:00:01.000000000 +0000
+++ fbi-2.07/debian/control	2012-07-02 10:51:53.499392776 +0100
@@ -20,10 +20,10 @@
 Package: exiftran
 Architecture: linux-any
 Depends: ${shlibs:Depends}, ${misc:Depends}
-Description: transform digital camera jpeg images
- exiftran is a command line utility to transform digital image jpeg
- images. It can do lossless rotations like jpegtran, but unlike
- jpegtran it cares about the EXIF data: It can rotate images
- automatically by checking the exif orientation tag, it updates the
- exif informaton if needed (image dimension, orientation), it also
- rotates the exif thumbnail. It can process multiple images at once.
+Description: digital camera JPEG image transformer
+ exiftran is a command line utility to transform digital camera JPEG
+ images. It can do lossless rotations like jpegtran, but unlike jpegtran
+ it can process multiple images at once, and it cares about the Exif data:
+ it can rotate images automatically by checking the Exif orientation tag,
+ it updates the Exif information (image dimensions/orientation) if needed,
+ and it also rotates the Exif thumbnail.
