From c58f5060eb38293eb5ce166de7c5f7546a77bb7b Mon Sep 17 00:00:00 2001
From: Philippe Rouquier <bonfire-app@wanadoo.fr>
Date: Fri, 21 May 2010 11:27:33 +0000
Subject: Do not use int value in a g_signal_emit when a long is expected

---
diff --git a/libbrasero-burn/brasero-burn.c b/libbrasero-burn/brasero-burn.c
index e9b3973..34861a8 100644
--- a/libbrasero-burn/brasero-burn.c
+++ b/libbrasero-burn/brasero-burn.c
@@ -1349,7 +1349,7 @@ start:
 				       0,
 				       1.0,
 				       1.0,
-				       -1);
+				       -1L);
 		}
 		return BRASERO_BURN_OK;
 	}
--
cgit v0.8.3.1
