diff -u digikam-debian.svn/debian/changelog digikam-5.2.0/debian/changelog
--- digikam-debian.svn/debian/changelog	2016-09-20 17:09:51.154784493 +0900
+++ digikam-5.2.0/debian/changelog	2016-09-26 11:35:21.286104244 +0900
@@ -1,10 +1,13 @@
-digikam (4:5.1.0-3) UNRELEASED; urgency=medium
+digikam (4:5.2.0-0.1) UNRELEASED; urgency=medium
 
   [ Pino Toscano ]
   * Remove the alternative mplayerthumbs recommend, since it is dead and
     scheduled for removal from Debian.
 
- -- Debian KDE Extras Team <pkg-kde-extras@lists.alioth.debian.org>  Wed, 24 Aug 2016 20:03:57 +0200
+  [ Norbert Preining ]
+  * new upstream release
+
+ -- Norbert Preining <preining@debian.org>  Mon, 26 Sep 2016 11:35:08 +0900
 
 digikam (4:5.1.0-2) unstable; urgency=medium
 
diff -u digikam-debian.svn/debian/digikam-data.install digikam-5.2.0/debian/digikam-data.install
--- digikam-debian.svn/debian/digikam-data.install	2016-08-10 09:55:13.839590944 +0900
+++ digikam-5.2.0/debian/digikam-data.install	2016-09-26 13:21:26.697360883 +0900
@@ -4,7 +4,6 @@
 usr/share/icons/hicolor/*/actions
 usr/share/digikam
 usr/share/solid
-usr/share/kservice*/digikam*
 usr/share/kxmlgui5/digikam
 usr/share/knotifications5/digikam*
 usr/share/locale/*/LC_MESSAGES/digikam.mo
diff -u digikam-debian.svn/debian/digikam.install digikam-5.2.0/debian/digikam.install
--- digikam-debian.svn/debian/digikam.install	2016-08-10 09:55:11.343639113 +0900
+++ digikam-5.2.0/debian/digikam.install	2016-09-26 13:22:57.941666914 +0900
@@ -1,10 +1,7 @@
 usr/bin/cleanup_digikamdb
 usr/bin/digikam
 usr/bin/digitaglinktree
-usr/lib/*/qt5/plugins/digikam*
-usr/share/appdata/digikam.appdata.xml
-usr/share/appdata/digiKam-*.metainfo.xml
-usr/share/applications/digikam.desktop
-usr/share/kconf_update/
+usr/share/metainfo/org.kde.digikam.appdata.xml
+usr/share/applications/org.kde.digikam.desktop
 usr/share/man/man1/cleanup_digikamdb.1
 usr/share/man/man1/digitaglinktree.1
diff -u digikam-debian.svn/debian/kipi-plugins-common.install digikam-5.2.0/debian/kipi-plugins-common.install
--- digikam-debian.svn/debian/kipi-plugins-common.install	2016-08-10 09:55:11.343639113 +0900
+++ digikam-5.2.0/debian/kipi-plugins-common.install	2016-09-26 13:22:03.313483729 +0900
@@ -1,4 +1,3 @@
-usr/share/doc/HTML/en/kipi-plugins
 usr/share/icons/hicolor/*/apps/kipi*
 usr/share/kxmlgui5/kipi
 usr/share/kipiplugin_*
Common subdirectories: digikam-debian.svn/debian/man and digikam-5.2.0/debian/man
Common subdirectories: digikam-debian.svn/debian/patches and digikam-5.2.0/debian/patches
diff -u digikam-debian.svn/debian/showfoto.install digikam-5.2.0/debian/showfoto.install
--- digikam-debian.svn/debian/showfoto.install	2016-08-12 09:24:57.971968504 +0900
+++ digikam-5.2.0/debian/showfoto.install	2016-09-26 13:23:10.457708870 +0900
@@ -1,6 +1,6 @@
 usr/bin/showfoto
-usr/share/appdata/showfoto.appdata.xml
-usr/share/applications/showfoto.desktop
+usr/share/metainfo/org.kde.showfoto.appdata.xml
+usr/share/applications/org.kde.showfoto.desktop
 usr/share/doc/HTML/*/showfoto
 usr/share/icons/hicolor/*/apps/showfoto.*
 usr/share/kxmlgui5/showfoto
Common subdirectories: digikam-debian.svn/debian/source and digikam-5.2.0/debian/source
