Index: debian/aide.conf.d/31_aide_mailman
===================================================================
--- debian/aide.conf.d/31_aide_mailman	(revision 785)
+++ debian/aide.conf.d/31_aide_mailman	(working copy)
@@ -32,6 +32,6 @@
 @@define LOGFILES7 (qrunner|fromusenet|locks|smtp(-failure)?)
 /var/log/mailman/@@{LOGFILES7}$ Log
 /var/log/mailman/@@{LOGFILES7}\.1$ LowLog
-/var/log/mailman/@@{LOGFILES7}\.2\.gz$ HiSerMemberLog
+/var/log/mailman/@@{LOGFILES7}\.2\.gz$ LoSerMemberLog
 /var/log/mailman/@@{LOGFILES7}\.[3-6]\.gz$ SerMemberLog
-/var/log/mailman/@@{LOGFILES7}\.7\.gz$ LoSerMemberLog
+/var/log/mailman/@@{LOGFILES7}\.7\.gz$ HiSerMemberLog
