
---
Last-Update: 2019-08-27

--- /dev/null
+++ bluez-5.43/obexd/src/org.bluez.obex.service
@@ -0,0 +1,4 @@
+[D-BUS Service]
+Name=org.bluez.obex
+Exec=/usr/lib/bluetooth/obexd
+SystemdService=obex.service
--- bluez-5.43.orig/src/adapter.c
+++ bluez-5.43/src/adapter.c
@@ -7688,7 +7688,6 @@ static void connected_callback(uint16_t
 	ba2str(&ev->addr.bdaddr, addr);
 
 	DBG("hci%u device %s connected eir_len %u", index, addr, eir_len);
-
 	device = btd_adapter_get_device(adapter, &ev->addr.bdaddr,
 								ev->addr.type);
 	if (!device) {
@@ -7696,7 +7695,7 @@ static void connected_callback(uint16_t
 				"Unable to get device object for %s", addr);
 		return;
 	}
-
+	btd_adapter_restore_powered(adapter);
 	memset(&eir_data, 0, sizeof(eir_data));
 	if (eir_len > 0)
 		eir_parse(&eir_data, ev->eir, eir_len);
