---
Last-Update: 2019-09-08

--- /dev/null
+++ bluez-5.43/obexd/src/org.bluez.obex.service
@@ -0,0 +1,4 @@
+[D-BUS Service]
+Name=org.bluez.obex
+Exec=/usr/lib/bluetooth/obexd
+SystemdService=obex.service
--- bluez-5.43.orig/src/adapter.c
+++ bluez-5.43/src/adapter.c
@@ -6118,10 +6118,6 @@ int btd_cancel_authorization(guint id)
 
 int btd_adapter_restore_powered(struct btd_adapter *adapter)
 {
-	if (adapter->current_settings & MGMT_SETTING_POWERED)
-		return 0;
-
-	set_mode(adapter, MGMT_OP_SET_POWERED, 0x01);
 
 	return 0;
 }
