--- compiz-0.5.0.dfsg.orig/plugins/compiz.schemas.in
+++ compiz-0.5.0.dfsg/plugins/compiz.schemas.in
@@ -15763,7 +15763,7 @@
             <owner>compiz</owner>
             <type>list</type>
             <list_type>string</list_type>
-            <default>[freedesktop]</default>
+            <default>[debian,freedesktop]</default>
             <locale name="C">
                 <short>Image files</short>
                 <long>List of PNG and SVG files that should be rendered on top face of cube</long>
