Index: prog/aspell.cpp
===================================================================
RCS file: /sources/aspell/aspell/prog/aspell.cpp,v
retrieving revision 1.103.2.2
diff -u -r1.103.2.2 aspell.cpp
--- prog/aspell.cpp	19 Jun 2005 12:00:46 -0000	1.103.2.2
+++ prog/aspell.cpp	23 Nov 2006 14:36:57 -0000
@@ -870,7 +870,8 @@
           if (ci->pre_strip_len > 0) 
             guess.append('-').append(ci->word.str(), ci->pre_strip_len);
           if (ci->suf_strip_len > 0) 
-            guess.append('-').append(ci->word.str() - ci->suf_strip_len, ci->suf_strip_len);
+            guess.append('-').append(ci->word.str() + ci->word.size() - ci->suf_strip_len, 
+                                     ci->suf_strip_len);
           if (ci->suf_add && ci->suf_add[0])
             guess.append('+').append(ci->suf_add, ci->suf_add_len);
           real_speller->lang().fix_case(casep, guess.data(), guess.data());

 	  	 
