--- /usr/lib/cgit/filters/syntax-highlighting.py.orig	2016-11-17 23:42:23.992555420 +0100
+++ /usr/lib/cgit/filters/syntax-highlighting.py	2016-11-17 23:42:20.000547312 +0100
@@ -41,7 +41,10 @@
 except ClassNotFound:
 	# check if there is any shebang
 	if data[0:2] == '#!':
-		lexer = guess_lexer(data)
+		try:
+			lexer = guess_lexer(data)
+		except ClassNotFound:
+			lexer = TextLexer()
 	else:
 		lexer = TextLexer()
 except TypeError:
