--- bts.pl	2008-04-09 15:59:08.000000000 +0200
+++ bts.pl.new	2008-04-13 19:43:20.000000000 +0200
@@ -1389,9 +1389,9 @@
     mailbts("severity of $bug is $matches[0]", "severity $bug $matches[0]");
 }
 
-=item forwarded <bug> <email>
+=item forwarded <bug> <address>
 
-Mark the bug as forwarded to the given email address.
+Mark the bug as forwarded to the given email address or upstream bug tracker.
 
 =cut
 
