diff --git a/internal/pgsql b/internal/pgsql
index 76f6352..b57a501 100644
--- a/internal/pgsql
+++ b/internal/pgsql
@@ -115,7 +115,7 @@ _dbc_createdb(){
 		extrasql=" WITH ENCODING = '$dbc_pgsql_createdb_encoding'"; 
 	fi
 	extra="$extra template1"
-	_dbc_pgsql_exec_command "CREATE DATABASE \"$dbc_dbname\"$extrasql"
+	_dbc_pgsql_exec_command "CREATE DATABASE \"$dbc_dbname\"$extrasql; GRANT ALL PRIVILEGES ON DATABASE \"$dbc_dbname\" TO \"$dbc_dbuser\";"
 	retval=$?
 	_dbc_psql_cmd_cleanup
 	return $retval
