--- a/debian/hooks/dropbear
+++ b/debian/hooks/dropbear
@@ -24,7 +24,8 @@ for so in $(ldconfig -p | sed -nr 's/^\s*libnss_files\.so\.[0-9]+\s.*=>\s*//p');
     copy_exec "$so"
 done

-home="$(mktemp --directory -- "$DESTDIR/root-XXXXXXXXXX")" # avoid collisions with $rootmnt
+home="$DESTDIR/root-dropbear-initramfs"
+mkdir "$home"
 chmod 0700 -- "$home"
 for x in passwd group; do echo "$x: files"; done >"$DESTDIR/etc/nsswitch.conf"
 echo "root:*:0:0::${home#"$DESTDIR"}:/bin/sh" >"$DESTDIR/etc/passwd"
