From 5f1d55d60a77afb4d66b025cb84509f45f646c78 Mon Sep 17 00:00:00 2001
From: Nicholas D Steeves <nsteeves@gmail.com>
Date: Mon, 31 Jul 2017 15:56:26 -0400
Subject: [PATCH] fixup! Exclude non-free-RFC-BCP78 files (affects test suite)

---
 debian/copyright | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/copyright b/debian/copyright
index 91b88570..55b172d3 100644
--- a/debian/copyright
+++ b/debian/copyright
@@ -1,7 +1,7 @@
 Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Contact: linux-btrfs@vger.kernel.org
 Source: https://git.kernel.org/pub/scm/linux/kernel/git/kdave/btrfs-progs.git
-Files-Excluded: tests/sha224-256.c tests/sha.h tests/sha-private.h
+Files-Excluded: tests/sha224-256.c
 
 Files: *
 Copyright: 2007-2012 Oracle <http://www.oracle.com/>
-- 
2.11.0

