From 034dc48c7291ed0c19719b111b05fdd741191205 Mon Sep 17 00:00:00 2001
From: Nicholas D Steeves <nsteeves@gmail.com>
Date: Sat, 11 Mar 2017 21:14:15 -0500
Subject: [PATCH 2/5] Move all binaries back to /sbin (Closes: #786893)

---
 debian/rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules b/debian/rules
index d797ac8f..b5511049 100755
--- a/debian/rules
+++ b/debian/rules
@@ -19,7 +19,7 @@ override_dh_autoreconf:
 	dh_autoreconf ./autogen.sh
 
 override_dh_auto_configure:
-	dh_auto_configure -- --bindir=/bin --disable-convert
+	dh_auto_configure -- --bindir=/sbin --disable-convert
 
 override_dh_auto_build:
 	dh_auto_build -- V=1
-- 
2.11.0

