From dbf42a8606559212902f628cd71b69aba0cd4376 Mon Sep 17 00:00:00 2001
From: Nicholas D Steeves <nsteeves@gmail.com>
Date: Sat, 11 Mar 2017 20:27:13 -0500
Subject: [PATCH 02/14] Ignore .pc, the quilt state tracking dir

---
 .gitignore | 1 +
 1 file changed, 1 insertion(+)

diff --git a/.gitignore b/.gitignore
index 98b3657b..ba8c8590 100644
--- a/.gitignore
+++ b/.gitignore
@@ -76,3 +76,4 @@ stamp-h.in
 stamp-h1
 config/*
 
+.pc
-- 
2.11.0

