From e87b785f7b137221212148f2083a0755803f6cb7 Mon Sep 17 00:00:00 2001
From: Nicholas D Steeves <nsteeves@gmail.com>
Date: Sun, 19 Mar 2017 15:03:32 -0400
Subject: [PATCH 05/14] Update Source, because Chris Mason's branch hasn't been
 updated since v4.4

---
 debian/copyright | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/copyright b/debian/copyright
index 3412f64b..021c89ab 100644
--- a/debian/copyright
+++ b/debian/copyright
@@ -1,6 +1,6 @@
 Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Contact: linux-btrfs@vger.kernel.org
-Source: http://git.kernel.org/?p=linux/kernel/git/mason/btrfs-progs.git
+Source: https://git.kernel.org/pub/scm/linux/kernel/git/kdave/btrfs-progs.git
 
 Files: *
 Copyright: 2007-2012 Oracle <http://www.oracle.com/>
-- 
2.11.0

