From 9e41daf5e64728c48c33a1836f140f5af991ab1e Mon Sep 17 00:00:00 2001
From: Nicholas D Steeves <nsteeves@gmail.com>
Date: Sun, 19 Mar 2017 15:05:56 -0400
Subject: [PATCH 06/14] Exclude non-free-RFC-BCP78 files (affects test suite)

---
 debian/copyright | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/copyright b/debian/copyright
index 021c89ab..914dd2e6 100644
--- a/debian/copyright
+++ b/debian/copyright
@@ -1,6 +1,7 @@
 Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
 Upstream-Contact: linux-btrfs@vger.kernel.org
 Source: https://git.kernel.org/pub/scm/linux/kernel/git/kdave/btrfs-progs.git
+Files-Excluded: tests/sha224-256.c tests/sha.h tests/sha-private.h
 
 Files: *
 Copyright: 2007-2012 Oracle <http://www.oracle.com/>
-- 
2.11.0

