From 4f8bd0f44e6a85ca59928bb95c702abbc08d382c Mon Sep 17 00:00:00 2001
From: Nicholas D Steeves <nsteeves@gmail.com>
Date: Sun, 19 Mar 2017 16:02:53 -0400
Subject: [PATCH 08/14] Add dversionmangle to handle dfsg version suffix

---
 debian/watch | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/watch b/debian/watch
index ab6a7caa..6126779d 100644
--- a/debian/watch
+++ b/debian/watch
@@ -1,3 +1,3 @@
 version=3
-opts="uversionmangle=s/(\d)[_\.\-\+]?(rc\d*)$/$1~$2/,pgpsigurlmangle=s/\.tar.*$/\.tar.sign/,decompress" \
+opts="dversionmangle=s/\+(debian|dfsg|ds|deb)(\.?\d+)?$//,uversionmangle=s/(\d)[_\.\-\+]?(rc\d*)$/$1~$2/,pgpsigurlmangle=s/\.tar.*$/\.tar.sign/,decompress" \
 https://www.kernel.org/pub/linux/kernel/people/kdave/btrfs-progs/btrfs-progs-v(.+)\.tar\.xz
-- 
2.11.0

