From e0ea798b52c3595d097f26a6af55b67b0a2dd958 Mon Sep 17 00:00:00 2001
From: Nicholas D Steeves <nsteeves@gmail.com>
Date: Thu, 2 Feb 2017 16:01:49 -0500
Subject: [PATCH 09/14] Update copyright for new locations of: radix-tree.c and
 radix-tree.h

---
 debian/copyright | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/copyright b/debian/copyright
index 3ae2af71..91b88570 100644
--- a/debian/copyright
+++ b/debian/copyright
@@ -7,14 +7,14 @@ Files: *
 Copyright: 2007-2012 Oracle <http://www.oracle.com/>
 License: GPL-2
 
-Files: radix-tree.c
+Files: kernel-lib/radix-tree.c
 Copyright: 2007-2009 Oracle <http://www.oracle.com/>
            2001 Momchil Velikov
            2001 Christoph Hellwig
            2005 SGI, Christoph Lameter <clameter@sgi.com>
 License: GPL-2
 
-Files: radix-tree.h
+Files: kernel-lib/radix-tree.h
 Copyright: 2007-2009 Oracle <http://www.oracle.com/>
            2001 Momchil Velikov
            2001 Christoph Hellwig
-- 
2.11.0

