From ac469a6b75d559a38fd4052fdc5c7f42e7559c25 Mon Sep 17 00:00:00 2001
From: Nicholas D Steeves <nsteeves@gmail.com>
Date: Thu, 2 Feb 2017 15:08:13 -0500
Subject: [PATCH 10/14] d/control: add whitespace (cosmetic fix)

---
 debian/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index 21979a50..ca0b1a47 100644
--- a/debian/control
+++ b/debian/control
@@ -10,7 +10,7 @@ Build-Depends: debhelper (>= 9),
                libblkid-dev,
                liblzo2-dev,
                uuid-dev,
-	       udev,
+               udev,
                zlib1g-dev,
                asciidoc,
                xmlto,
-- 
2.11.0

