From 522ff303e900bc90207f11597d4c4da965814d09 Mon Sep 17 00:00:00 2001
From: Nicholas D Steeves <nsteeves@gmail.com>
Date: Fri, 30 Jun 2017 15:33:32 -0400
Subject: [PATCH 12/14] Drop btrfs-tools transitional dummy package

---
 debian/control | 15 ---------------
 1 file changed, 15 deletions(-)

diff --git a/debian/control b/debian/control
index ca0b1a47..951d1475 100644
--- a/debian/control
+++ b/debian/control
@@ -34,13 +34,6 @@ Description: Checksumming Copy on Write Filesystem utilities
  This package contains utilities (mkfs, fsck) used to work with btrfs
  and an utility (btrfs-convert) to make a btrfs filesystem from an ext3.
 
-Package: btrfs-tools
-Section: oldlibs
-Architecture: linux-any
-Depends: btrfs-progs, ${misc:Depends}, ${shlibs:Depends}
-Description: transitional dummy package
- This is a transitional dummy package. It can safely be removed.
-
 Package: btrfs-progs-udeb
 Package-Type: udeb
 Section: debian-installer
@@ -62,11 +55,3 @@ Description: Checksumming Copy on Write Filesystem utilities (debug)
  administration.
  .
  This package contains the debugging symbols.
-
-Package: btrfs-tools-dbg
-Section: oldlibs
-Priority: extra
-Architecture: linux-any
-Depends: btrfs-progs-dbg, ${misc:Depends}
-Description: transitional dummy package
- This is a transitional dummy package. It can safely be removed.
-- 
2.11.0

