From b40bb2e59bc7e3b84f52c530db14beddbcdbc292 Mon Sep 17 00:00:00 2001
From: Nicholas D Steeves <nsteeves@gmail.com>
Date: Fri, 30 Jun 2017 16:18:36 -0400
Subject: [PATCH 13/15] Switch to automatically generated -dbgsym package

---
 debian/control | 16 +---------------
 debian/rules   |  2 +-
 2 files changed, 2 insertions(+), 16 deletions(-)

diff --git a/debian/control b/debian/control
index 951d1475..eb95073f 100644
--- a/debian/control
+++ b/debian/control
@@ -2,7 +2,7 @@ Source: btrfs-progs
 Section: admin
 Priority: optional
 Maintainer: Dimitri John Ledkov <xnox@debian.org>
-Build-Depends: debhelper (>= 9),
+Build-Depends: debhelper (>= 10),
                dh-autoreconf,
                e2fslibs-dev,
                pkg-config,
@@ -41,17 +41,3 @@ Architecture: linux-any
 Depends: ${misc:Depends}, ${shlibs:Depends}
 Provides: btrfs-tools-udeb
 Description: Checksumming Copy on Write Filesystem utilities (udeb)
-
-Package: btrfs-progs-dbg
-Section: debug
-Priority: extra
-Architecture: linux-any
-Depends: btrfs-progs (= ${binary:Version}), ${misc:Depends}
-Replaces: btrfs-tools-dbg (<< 4.4.1-1)
-Breaks: btrfs-tools-dbg (<< 4.4.1-1)
-Description: Checksumming Copy on Write Filesystem utilities (debug)
- Btrfs is a new copy on write filesystem for Linux aimed at implementing
- advanced features while focusing on fault tolerance, repair and easy
- administration.
- .
- This package contains the debugging symbols.
diff --git a/debian/rules b/debian/rules
index 0ac43c76..e72cd4f4 100755
--- a/debian/rules
+++ b/debian/rules
@@ -35,7 +35,7 @@ override_dh_auto_install:
 override_dh_auto_test:
 
 override_dh_strip:
-	dh_strip --dbg-package=btrfs-progs-dbg
+	dh_strip --dbgsym-migration='btrfs-progs (<< 4.11.1+dfsg-1~)'
 
 override_dh_auto_clean:
 	if test -f Makefile.inc; then \
-- 
2.11.0

