From 0b6f953e4122211740462ce443bb8510c5219afe Mon Sep 17 00:00:00 2001
From: Nicholas D Steeves <nsteeves@gmail.com>
Date: Thu, 13 Jul 2017 15:17:27 -0400
Subject: [PATCH 14/15] Bump compat to 10

---
 debian/compat | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/compat b/debian/compat
index ec635144..f599e28b 100644
--- a/debian/compat
+++ b/debian/compat
@@ -1 +1 @@
-9
+10
-- 
2.11.0

