From 18848210e6a0cc928ad4bb196f4e09e5e6ef5b8d Mon Sep 17 00:00:00 2001
From: Nicholas D Steeves <nsteeves@gmail.com>
Date: Thu, 13 Jul 2017 19:45:06 -0400
Subject: [PATCH 14/14] Update changelog

---
 debian/changelog | 31 +++++++++++++++++++++++++++++++
 1 file changed, 31 insertions(+)

diff --git a/debian/changelog b/debian/changelog
index cb0b3db1..3e1cd2bb 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,34 @@
+btrfs-progs (4.11.1+dfsg-1) unstable; urgency=medium
+
+  [ Nicholas D Steeves ]
+  * Drop btrfs-convert. (Closes: #824895, #854489)
+  * Move all binaries back to /sbin. (Closes: #786893)
+  * debian/copyright:
+    - Update Source, because Chris Mason's branch hasn't been updated
+      since v4.4.
+    - Exclude non-free-RFC-BCP78 files (affects test suite).
+    - Fix license, because Btrfs-progs is explicitly GPL2 and not GPL2+
+      and 4.6.1-1 did not fix this.
+    - Update copyright for new locations of: radix-tree.c and radix-tree.h
+  * debian/watch:
+    - Add dversionmangle to handle dfsg version suffix.
+    - Switch to version 4 and add repacksuffix=+dfsg.
+  * d/control:
+    - Override dh_auto_clean to work around upstream bug where make clean
+      cannot be run before ./configure.
+    - Add whitespace (cosmetic fix).
+    - Drop btrfs-tools transitional dummy packages.
+  * Switch to debhelper 10
+    - debian/control: Bump Build-Depends to debhelper >= 10.
+    - debian/compat: Bump version.
+    - debian/rules: Simplify rules, because --parallel and --with autoreconf
+      are now default.
+  * Switch to automatically generated -dbgsym package.
+    - debian/rules: Switch to dh_strip --dbgsym-migration=this version~.
+    - debian/control: Drop btrfs-progs-dbg.
+
+ -- Nicholas D Steeves <nsteeves@gmail.com>  Thu, 13 Jul 2017 19:39:24 -0400
+
 btrfs-progs (4.9.1-1) unstable; urgency=medium
 
   [ Dimitri John Ledkov ]
-- 
2.11.0

