diff -ru debtags-1.7.2.old/debian/debtags.cron.daily debtags-1.7.2/debian/debtags.cron.daily
--- debtags-1.7.2.old/debian/debtags.cron.daily	2007-06-22 14:35:01.000000000 +0100
+++ debtags-1.7.2/debian/debtags.cron.daily	2007-07-06 15:30:48.000000000 +0100
@@ -1,4 +1,4 @@
 #!/bin/sh
 
 # Update the data and index files of Debtags
-debtags update
+debtags update 2>/dev/null || echo debtags update error
