=== modified file 'apt-pkg/acquire-item.cc'
--- old/apt-pkg/acquire-item.cc	2008-10-29 17:58:48 +0000
+++ new/apt-pkg/acquire-item.cc	2009-04-14 05:47:41 +0000
@@ -1055,8 +1055,8 @@
          if (!Record)
          {
             Status = StatAuthError;
-            ErrorText = "Unable to find expected entry  "
-               + (*Target)->MetaKey + " in Meta-index file (malformed Release file?)";
+	    ErrorText = "Fetched successfully, but expected entry " 
+               + (*Target)->MetaKey + " not found in Meta-index (misspelled, or Release file malformed?)";
             return;
          }
          ExpectedIndexHash = Record->Hash;

