--- fai.orig	2013-09-23 09:54:51.000000000 +0200
+++ fai	2014-01-15 17:33:36.854516640 +0100
@@ -308,9 +308,6 @@
     mkdir -p $LOGDIR
     ln -snf $action-$fai_rundate $LOGDIR/../last-$action
     ln -snf $action-$fai_rundate $LOGDIR/../last
-    if [ -x /usr/sbin/logtail ] ; then
-      logtail -f /var/log/kern.log -o /var/run/fai/kern.log.offset > /dev/null
-    fi
 fi
 chown root:adm $LOGDIR
 chmod 0750 $LOGDIR
