diff -Nru --exclude pbuilder-doc.html /tmp/lJIc1Y81Rq/pbuilder-0.152/debian/changelog /tmp/H2m1HBzCBJ/pbuilder-0.152/debian/changelog
--- /tmp/lJIc1Y81Rq/pbuilder-0.152/debian/changelog	2006-04-26 02:33:36.000000000 +0200
+++ /tmp/H2m1HBzCBJ/pbuilder-0.152/debian/changelog	2006-05-13 14:30:39.000000000 +0200
@@ -1,3 +1,9 @@
+pbuilder (0.152-0.1) TEST; urgency=low
+
+  * Support for new style architecture specifications (closes: #363193)
+
+ -- Peter Eisentraut <petere@debian.org>  Sat, 13 May 2006 14:29:52 +0200
+
 pbuilder (0.152) unstable; urgency=low
 
   * fix pdebuild-internal, pass --uid/--gid options.
diff -Nru --exclude pbuilder-doc.html /tmp/lJIc1Y81Rq/pbuilder-0.152/pbuilder-satisfydepends /tmp/H2m1HBzCBJ/pbuilder-0.152/pbuilder-satisfydepends
--- /tmp/lJIc1Y81Rq/pbuilder-0.152/pbuilder-satisfydepends	2006-03-31 02:14:31.000000000 +0200
+++ /tmp/H2m1HBzCBJ/pbuilder-0.152/pbuilder-satisfydepends	2006-05-13 14:46:52.000000000 +0200
@@ -40,16 +40,29 @@
     return 1;
 }
 
+function expand_arch () {
+    local ARCH="$1"
+    local EXPANDED_ARCH
+    if echo "$ARCH" | grep "-" > /dev/null; then
+        EXPANDED_ARCH=$ARCH
+    else
+        EXPANDED_ARCH="linux-$ARCH"
+    fi
+    local WC1=$(echo $EXPANDED_ARCH | sed 's/^[^-]*/any/')
+    local WC2=$(echo $EXPANDED_ARCH | sed 's/[^-]*$/any/')
+    echo "$ARCH\\|$EXPANDED_ARCH\\|$WC1\\|$WC2"
+}
+
 function checkbuilddep_archdeps () {
     # returns FALSE on INSTALL
     local INSTALLPKG="$1"
     local ARCH="$2"
-    if echo "$INSTALLPKG" | sed 's/.*\(\[.*\]\)/\1/' | grep "[[/][!]$ARCH[]/]" > /dev/null; then
+    if echo "$INSTALLPKG" | sed 's/.*\(\[.*\]\)/\1/' | grep "[[/][!]\($(expand_arch $ARCH)\)[]/]" > /dev/null; then
 	# if !$ARCH exists in there, ERROR.
 	return 0;
     fi
     if ! echo "$INSTALLPKG" | sed 's/.*\(\[.*\]\)/\1/' | grep "[!]" > /dev/null; then
-	if ! echo "$INSTALLPKG" | sed 's/.*\(\[.*\]\)/\1/' | grep "[[/]$ARCH[]/]" > /dev/null; then
+	if ! echo "$INSTALLPKG" | sed 's/.*\(\[.*\]\)/\1/' | grep "[[/]\($(expand_arch $ARCH)\)[]/]" > /dev/null; then
 	# if $ARCH does not exist, ERROR.
 	    return 0;
 	fi
