From ac48b48288b33d5277449f28c74077401068dded Mon Sep 17 00:00:00 2001
From: Nicholas D Steeves <nsteeves@gmail.com>
Date: Sat, 1 Jul 2017 18:28:09 -0400
Subject: [PATCH] Add extra step documentation to be extra safe; align comments

---
 dh-make-elpa | 4 +++-
 1 file changed, 3 insertions(+), 1 deletion(-)

diff --git a/dh-make-elpa b/dh-make-elpa
index c0a9af4..a8d0647 100755
--- a/dh-make-elpa
+++ b/dh-make-elpa
@@ -40,7 +40,9 @@ directory before uploading
 
 % cd foo
 
-% git reset --hard 1.0.0    # package latest stable release
+% git reset --hard 1.0.0      # package latest stable release
+
+% git branch --unset-upstream # detaches master branch from upstream remote
 
 % dh-make-elpa --pkg-emacsen
 
-- 
2.11.0

