*** extlinux-update.orig	2012-06-30 14:00:10.000000000 +0200
--- extlinux-update	2012-10-27 23:55:52.159496864 +0200
***************
*** 192,197 ****
--- 192,205 ----
  	linux ${_BOOT_DIRECTORY}/vmlinuz-${_VERSION}
  	append ${_INITRD} ${EXTLINUX_ROOT} ${EXTLINUX_PARAMETERS}"
  
+ 	if [ "o${EXTLINUX_DEFAULT}" = "ol${_NUMBER}" ];
+ 	then
+ 
+ 		# Selecting default entry
+ 		_CONFIG="${_CONFIG}
+ 	menu default"
+ 	fi
+ 
  	fi
  
  	if echo ${EXTLINUX_ALTERNATIVES} | grep -q live
***************
*** 208,213 ****
--- 216,229 ----
     This option boots the system into live mode (non-persistent)
  	endtext"
  
+ 	if [ "o${EXTLINUX_DEFAULT}" = "ol${_NUMBER}l" ];
+ 	then
+ 
+ 		# Selecting default entry
+ 		_CONFIG="${_CONFIG}
+ 	menu default"
+ 	fi
+ 
  	fi
  
  	if echo ${EXTLINUX_ALTERNATIVES} | grep -q recovery
***************
*** 224,229 ****
--- 240,253 ----
     This option boots the system into recovery mode (single-user)
  	endtext"
  
+ 	if [ "o${EXTLINUX_DEFAULT}" = "ol${_NUMBER}r" ];
+ 	then
+ 
+ 		# Selecting default entry
+ 		_CONFIG="${_CONFIG}
+ 	menu default"
+ 	fi
+ 
  	fi
  
  	_NUMBER="$((${_NUMBER} + 1))"
***************
*** 302,307 ****
--- 326,339 ----
  	initrd ${_IMAGE}
  	append ${_IMAGE_APPEND}"
  
+ 	if [ "o${EXTLINUX_DEFAULT}" = "om${_NUMBER}" ];
+ 	then
+ 
+ 		# Selecting default entry
+ 		_CONFIG="${_CONFIG}
+ 	menu default"
+ 	fi
+ 
  		_NUMBER="$((${_NUMBER} + 1))"
  	done
  
***************
*** 363,368 ****
--- 395,408 ----
  	kernel chain.c32
  	append hd${_DEVICENAME}"
  
+ 	if [ "o${EXTLINUX_DEFAULT}" = "oo${_NUMBER}" ];
+ 	then
+ 
+ 		# Selecting default entry
+ 		_CONFIG="${_CONFIG}
+ 	menu default"
+ 	fi
+ 
  		_NUMBER="$((${_NUMBER} + 1))"
  	done
  
