diff --git i/scripts/dpkg-buildpackage.pl w/scripts/dpkg-buildpackage.pl
index bf43fb5cf..f119f11f5 100755
--- i/scripts/dpkg-buildpackage.pl
+++ w/scripts/dpkg-buildpackage.pl
@@ -846,7 +846,7 @@ sub parse_rules_requires_root {
             if ($keyword =~ m{^dpkg/target/(.*)$}p and $target_official{$1}) {
                 error(g_('disallowed target in %s field keyword "%s"'),
                       'Rules-Requires-Root', $keyword);
-            } elsif ($keyword ne 'dpkg/target-subcommand') {
+            } elsif ($keyword =~ m{^dpkg/(.*)$} and $1 ne 'target-subcommand') {
                 error(g_('%s field keyword "%s" is unknown in dpkg namespace'),
                       'Rules-Requires-Root', $keyword);
             }
