diff --git a/server-src/amstatus.pl b/server-src/amstatus.pl
index be2b8fe..775d188 100644
--- a/server-src/amstatus.pl
+++ b/server-src/amstatus.pl
@@ -6,13 +6,13 @@ eval '(exit $?0)' && eval 'exec @PERL@ -S $0 ${1+"$@"}'
 	 & eval 'exec @PERL@ -S $0 $argv:q'
 		if 0;
 
-require "newgetopt.pl";
 use warnings;
 use lib '@amperldir@';
 use Time::Local;
 use Text::ParseWords;
 use Amanda::Util;
 use Amanda::Process;
+use Getopt::Long;
 
 delete @ENV{'IFS', 'CDPATH', 'ENV', 'BASH_ENV', 'PATH'};
 $ENV{'PATH'} = "/bin:/usr/bin:/usr/sbin:/sbin";       # force known path
@@ -33,7 +33,7 @@ $STATUS_MISSING =  8;
 $STATUS_TAPE    = 16;
 $exit_status    =  0;
 
-$result = &NGetOpt (	"summary",
+$result = GetOptions (	"summary",
 			"stats|statistics",
 			"dumping|d",
 			"waitdumping|wdumping",
@@ -49,7 +49,7 @@ $result = &NGetOpt (	"summary",
 			"file:s",
 			"locale-independent-date-format",
 			);
-if($result !=1 ) {
+if( !$result ) {
 	&usage();
 }
 
