diff -u -r courier-0.65.0/courier/cmlm.C courier-0.65.0-hk/courier/cmlm.C
--- courier-0.65.0/courier/cmlm.C	2009-11-08 19:14:47.000000000 +0100
+++ courier-0.65.0-hk/courier/cmlm.C	2012-02-11 18:07:40.000000000 +0100
@@ -1024,6 +1024,9 @@
 
 	} while ((rc=tryboundary(io, boundary.c_str())) > 0);
 	if (rc)	return ("");
+        if ( boundary.size() >= 70 ) // ein C string hat auch noch eine abschließende \0
+                boundary.erase(70);  // the function deletes everything from position pos forwards
+
 	return (boundary);
 }
 
diff -u -r courier-0.65.0/debian/changelog courier-0.65.0-hk/debian/changelog
--- courier-0.65.0/debian/changelog	2012-02-15 05:57:07.000000000 +0100
+++ courier-0.65.0-hk/debian/changelog	2012-02-11 18:09:13.000000000 +0100
@@ -1,3 +1,9 @@
+courier (0.65.0-3.1) unstable; urgency=low
+
+  * Limit MIME boundary to maximum size, Closes: 582765
+
+ -- Helge Kreutzmann <debian@helgefjell.de>  Sat, 11 Feb 2012 18:08:11 +0100
+
 courier (0.65.0-3) unstable; urgency=low
 
   * add Danish translation of Debconf templates (Closes: #605073, thanks

