From: vincent.mcintyre@csiro.au
Date: Tue, 28 Jan 2014 15:50:23 +1100
Subject: [PATCH] spelling and grammar improvements

---
 apt-mirror |   20 ++++++++++----------
 1 files changed, 10 insertions(+), 10 deletions(-)

diff --git a/apt-mirror b/apt-mirror
index 455fa85..81b13e3 100755
--- a/apt-mirror
+++ b/apt-mirror
@@ -16,12 +16,12 @@ A small and efficient tool that lets you mirror a part of or
 the whole Debian GNU/Linux distribution or any other apt sources.
 
 Main features:
- * It uses a config similar to apts F<sources.list>
- * It's fully pool comply
+ * It uses a config similar to apt's F<sources.list>
+ * It's fully pool compliant
  * It supports multithreaded downloading
  * It supports multiple architectures at the same time
  * It can automatically remove unneeded files
- * It works well on overloaded channel to internet
+ * It works well on an overloaded internet connection
  * It never produces an inconsistent mirror including while mirroring
  * It works on all POSIX compliant systems with perl and wget
 
@@ -35,7 +35,7 @@ After you setup the configuration file you may run as root:
 
     # su - apt-mirror -c apt-mirror
 
-Or uncomment line in F</etc/cron.d/apt-mirror> to enable daily mirror updates.
+Or uncomment the line in F</etc/cron.d/apt-mirror> to enable daily mirror updates.
 
 =head1 FILES
 
@@ -52,17 +52,17 @@ F</var/spool/apt-mirror/skel>
         Place for temporarily downloaded indexes
 
 F</var/spool/apt-mirror/var>
-        Log files placed here. URLs and MD5 summs also here.
+        Log files placed here. URLs and MD5 checksums also here.
 
 =head1 CONFIGURATION EXAMPLES
 
-The mirror.list configuration supports many options, the file is well commented explinging each option.
-here are some sample mirror configuration lines showing the various supported ways :
+The mirror.list configuration supports many options, the file is well commented explaining each option.
+Here are some sample mirror configuration lines showing the various supported ways :
 
 Normal:
 deb http://example.com/debian stable main contrib non-free
 
-Arch Specific: ( many other arch's are supported )
+Arch Specific: ( many other architectures are supported )
 deb-powerpc http://example.com/debian stable main contrib non-free
 
 HTTP and FTP Auth or non-standard port:
@@ -399,7 +399,7 @@ foreach (keys %urls_to_download) {
 }
 
 ######################################################################################
-## Main download prepair
+## Main download preparations
 
 %urls_to_download = ();
 
@@ -501,7 +501,7 @@ sub proceed_index_gz {
     close STREAM;
 }
 
-print "Proceed indexes: [";
+print "Processing indexes: [";
 
 foreach (@config_sources) {
     my ($uri, $distribution, @components) = @{$_};
-- 
1.7.2.5

