Index: modules-4.1.2/modulecmd.tcl.in
===================================================================
--- modules-4.1.2.orig/modulecmd.tcl.in
+++ modules-4.1.2/modulecmd.tcl.in
@@ -3659,7 +3659,7 @@ $g_shell \$*`;
    return $_mlstatus;}
          append fdef "\n};"
          if {[isStderrTty]} {
-            append fdef "\nmodule() { _moduleraw \"\$*\" 2>&1; };"
+            append fdef "\nmodule() { _moduleraw \$* 2>&1; };"
          }
       }
       {fish} {
