diff -Nru curl-7.63.0/debian/changelog curl-7.63.0/debian/changelog
--- curl-7.63.0/debian/changelog	2019-01-15 21:47:40.000000000 +0100
+++ curl-7.63.0/debian/changelog	2019-01-31 15:37:27.000000000 +0100
@@ -1,3 +1,11 @@
+curl (7.63.0-2) unstable; urgency=medium
+
+  * debian/control, debian/rules:
+    - build with libssh instead of libssh2, that's a better maintained
+      library and other distributions already switched to it
+
+ -- Sebastien Bacher <seb128@ubuntu.com>  Thu, 31 Jan 2019 15:29:39 +0100
+
 curl (7.63.0-1) unstable; urgency=medium
 
   * New upstream release
diff -Nru curl-7.63.0/debian/control curl-7.63.0/debian/control
--- curl-7.63.0/debian/control	2019-01-15 21:47:40.000000000 +0100
+++ curl-7.63.0/debian/control	2019-01-31 15:31:23.000000000 +0100
@@ -18,7 +19,7 @@
  libnss3-dev,
  libpsl-dev,
  librtmp-dev (>= 2.4+20131018.git79459a2-3~),
- libssh2-1-dev,
+ libssh-dev,
  libssl-dev (>= 1.1),
  libtool,
  openssh-server <!nocheck>,
diff -Nru curl-7.63.0/debian/rules curl-7.63.0/debian/rules
--- curl-7.63.0/debian/rules	2019-01-15 21:47:40.000000000 +0100
+++ curl-7.63.0/debian/rules	2019-01-31 15:29:29.000000000 +0100
@@ -15,15 +15,10 @@
 CONFIGURE_ARGS = -- --disable-dependency-tracking		\
 	--disable-symbol-hiding --enable-versioned-symbols	\
 	--enable-threaded-resolver --with-lber-lib=lber		\
-	--with-gssapi=/usr --with-libssh2 --with-nghttp2	\
+	--with-gssapi=/usr --with-libssh --with-nghttp2	\
 	--includedir=/usr/include/$(DEB_HOST_MULTIARCH)		\
 	--with-zsh-functions-dir=/usr/share/zsh/vendor-completions
 
-# disable libssh2 on Ubuntu (see #888449)
-ifeq ($(shell dpkg-vendor --derives-from Ubuntu && echo yes),yes)
-	CONFIGURE_ARGS += --without-libssh2
-endif
-
 %:
 	dh $@
 
