From 2e254fb802635742bb81f2e0a776e87c71360e1f Mon Sep 17 00:00:00 2001
From: Bo YU <tsu.yubo@gmail.com>
Date: Sun, 23 Apr 2023 22:22:47 +0800
Subject: [PATCH] add support for riscv32

Signed-off-by: Bo YU <tsu.yubo@gmail.com>
---
 data/cputable | 1 +
 1 file changed, 1 insertion(+)

diff --git a/data/cputable b/data/cputable
index 7b1ee2c58..7c2c1c1c8 100644
--- a/data/cputable
+++ b/data/cputable
@@ -45,6 +45,7 @@ powerpc		powerpc		(powerpc|ppc)		32	big
 powerpcel	powerpcle	powerpcle		32	little
 ppc64		powerpc64	(powerpc|ppc)64		64	big
 ppc64el		powerpc64le	powerpc64le		64	little
+riscv32		riscv32		riscv32			32	little
 riscv64		riscv64		riscv64			64	little
 s390		s390		s390			32	big
 s390x		s390x		s390x			64	big
-- 
2.36.1

