diff --git a/build/config/hppa.cfg b/build/config/hppa.cfg
index c3c8e57..d2b84f3 100644
--- a/build/config/hppa.cfg
+++ b/build/config/hppa.cfg
@@ -2,9 +2,9 @@ MEDIUM_SUPPORTED = cdrom netboot miniiso
 
 KERNEL_FLAVOUR = di
 
-BASEVERSION = 2.6.37-2
+BASEVERSION = 3.12-1
 KERNELMAJOR = 2.6
-KERNELIMAGEVERSION = $(BASEVERSION)-parisc $(BASEVERSION)-parisc64
+KERNELIMAGEVERSION = $(BASEVERSION)-parisc $(BASEVERSION)-parisc64-smp
 KERNELVERSION = $(foreach ver,${KERNELIMAGEVERSION},$(ver))
 KERNELNAME = $(foreach ver,${KERNELVERSION},vmlinux-$(ver))
  
@@ -16,7 +16,7 @@ arch_tree:
  
 arch_netboot:
 	palo -f /dev/null -k $(TEMP)/vmlinux*parisc \
-		-k $(TEMP)/vmlinux*parisc64 \
+		-k $(TEMP)/vmlinux*parisc64-smp \
 		-r $(TEMP_INITRD) -b /usr/share/palo/iplboot \
 		-c "0/linux initrd=0/ramdisk" \
 		-s $(TEMP_BOOT)
diff --git a/build/config/hppa/miniiso.cfg b/build/config/hppa/miniiso.cfg
index 80bda31..c99717b 100644
--- a/build/config/hppa/miniiso.cfg
+++ b/build/config/hppa/miniiso.cfg
@@ -12,7 +12,7 @@ arch_miniiso:
 	-rm -rf $(TEMP_CD_TREE)/*
 	install -m 644 -D $(BASE_TMP)miniiso/initrd.gz $(TEMP_CD_TREE)/boot/initrd.gz
 	install -m 644 -D $(BASE_TMP)miniiso/vmlinux*parisc $(TEMP_CD_TREE)/boot/vmlinux-parisc
-	install -m 644 -D $(BASE_TMP)miniiso/vmlinux*parisc64 $(TEMP_CD_TREE)/boot/vmlinux-parisc64
+	install -m 644 -D $(BASE_TMP)miniiso/vmlinux*parisc64-smp $(TEMP_CD_TREE)/boot/vmlinux-parisc64-smp
 	install -m 644 -D /usr/share/palo/iplboot $(TEMP_CD_TREE)/boot/iplboot
 
 	genisoimage -r -J -o $(TEMP_MINIISO) $(TEMP_CD_TREE)
diff --git a/build/pkg-lists/cdrom/hppa.cfg b/build/pkg-lists/cdrom/hppa.cfg
index e0508b1..fe833f0 100644
--- a/build/pkg-lists/cdrom/hppa.cfg
+++ b/build/pkg-lists/cdrom/hppa.cfg
@@ -1,5 +1,5 @@
 cdrom-core-modules-${kernel:Version}
-ide-modules-${kernel:Version}
+pata-modules-${kernel:Version}
 input-modules-${kernel:Version}
 nic-modules-${kernel:Version}
 scsi-modules-${kernel:Version}
