diff -Nur eog-2.6.1.old/debian/changelog eog-2.6.1/debian/changelog
--- eog-2.6.1.old/debian/changelog	2004-06-26 15:49:38.000000000 +0200
+++ eog-2.6.1/debian/changelog	2004-06-26 15:49:48.000000000 +0200
@@ -1,3 +1,12 @@
+eog (2.6.1-0.4) unstable; urgency=high
+
+  * NMU
+  * Fixed the FTBFS with the new libexif (Closes: #255186).
+  * debian/control:
+    - updated the Build-Depends on libexif-dev.
+
+ -- Sebastien Bacher <seb128@debian.org>  Sat, 26 Jun 2004 14:24:01 +0200
+
 eog (2.6.1-0.3) unstable; urgency=low
 
   * NMU
diff -Nur eog-2.6.1.old/debian/control eog-2.6.1/debian/control
--- eog-2.6.1.old/debian/control	2004-06-26 15:49:38.000000000 +0200
+++ eog-2.6.1/debian/control	2004-06-26 15:49:48.000000000 +0200
@@ -3,7 +3,7 @@
 Priority: optional
 Maintainer: Davide Puricelli (evo) <evo@debian.org>
 Standards-Version: 3.5.8.0
-Build-Depends: debhelper (>= 3), gettext, intltool, libtool, librsvg2-dev (>= 2.6.4-5), libart-2.0-dev, libbonobo2-dev (>= 2.6.0-2), libbonoboui2-dev (>= 2.6.1-1), libfreetype6-dev, libgconf2-dev (>= 2.6.1-2), libgnome2-dev (>= 2.6.1-2), libgnomecanvas2-dev (>= 2.6.1.1-2), libgnomeprint2.2-dev (>= 2.6.1-6), libgnomeui-dev (>= 2.6.1.1-2), libgnomevfs2-dev (>= 2.6.1.1-3), libgtk2.0-dev (>= 2.4.1-3), liborbit2-dev (>= 2.10.2-1.1), libpopt-dev, libxml2-dev, scrollkeeper, libjpeg62-dev, libpng12-dev, zlib1g-dev, libeel2-dev (>= 2.6.1-2), libgnomeprintui2.2-dev (>= 2.6.1-4), libexif-dev, libxt-dev
+Build-Depends: debhelper (>= 3), gettext, intltool, libtool, librsvg2-dev (>= 2.6.4-5), libart-2.0-dev, libbonobo2-dev (>= 2.6.0-2), libbonoboui2-dev (>= 2.6.1-1), libfreetype6-dev, libgconf2-dev (>= 2.6.1-2), libgnome2-dev (>= 2.6.1-2), libgnomecanvas2-dev (>= 2.6.1.1-2), libgnomeprint2.2-dev (>= 2.6.1-6), libgnomeui-dev (>= 2.6.1.1-2), libgnomevfs2-dev (>= 2.6.1.1-3), libgtk2.0-dev (>= 2.4.1-3), liborbit2-dev (>= 2.10.2-1.1), libpopt-dev, libxml2-dev, scrollkeeper, libjpeg62-dev, libpng12-dev, zlib1g-dev, libeel2-dev (>= 2.6.1-2), libgnomeprintui2.2-dev (>= 2.6.1-4), libexif-dev (>= 0.6.9), libxt-dev
 
 Package: eog
 Architecture: any
diff -Nur eog-2.6.1.old/libeog/eog-info-view.c eog-2.6.1/libeog/eog-info-view.c
--- eog-2.6.1.old/libeog/eog-info-view.c	2004-02-15 16:33:11.000000000 +0100
+++ eog-2.6.1/libeog/eog-info-view.c	2004-06-26 15:49:48.000000000 +0200
@@ -255,11 +255,11 @@
 	path = g_hash_table_lookup (priv->id_path_hash, GINT_TO_POINTER (entry->tag));
 
 	if (path != NULL) {
-		set_row_data (store, path, exif_tag_get_name (entry->tag), exif_entry_get_value (entry));	
+		set_row_data (store, path, exif_tag_get_name (entry->tag), exif_entry_get_value (entry, NULL, 4096));	
 	}
 	else {
 		path = set_row_data (store, NULL, 
-				     exif_tag_get_name (entry->tag), exif_entry_get_value (entry));	
+				     exif_tag_get_name (entry->tag), exif_entry_get_value (entry, NULL, 4096));	
 
 		g_hash_table_insert (priv->id_path_hash,
 				     GINT_TO_POINTER (entry->tag),
