From 47930e954324f62ce08d91aa2211a8b59c5913de Mon Sep 17 00:00:00 2001
From: Dmitry Shachnev <mitya57@gmail.com>
Date: Sat, 30 Sep 2017 23:55:31 +0300
Subject: [PATCH 2/2] qmake.pm: Use a custom qt.conf file for cross builds

---
 lib/Debian/Debhelper/Buildsystem/qmake.pm | 9 +++++++++
 1 file changed, 9 insertions(+)

diff --git a/lib/Debian/Debhelper/Buildsystem/qmake.pm b/lib/Debian/Debhelper/Buildsystem/qmake.pm
index e92c905f..e22d07de 100644
--- a/lib/Debian/Debhelper/Buildsystem/qmake.pm
+++ b/lib/Debian/Debhelper/Buildsystem/qmake.pm
@@ -8,6 +8,7 @@ package Debian::Debhelper::Buildsystem::qmake;
 
 use strict;
 use warnings;
+use File::Temp qw(tempfile);
 use Debian::Debhelper::Dh_Lib qw(dpkg_architecture_value error is_cross_compiling);
 use parent qw(Debian::Debhelper::Buildsystem::makefile);
 
@@ -60,6 +61,14 @@ sub configure {
 			'hurd'     => 'hurd-g++',
 		);
 		push @options, ("-spec", $os_mkspec_mapping{$host_os});
+
+		my ($fh, $filename) = tempfile("qt.XXXX", SUFFIX => ".conf", TMPDIR => 1, UNLINK => 1);
+		$fh->print("[Paths]\n");
+		$fh->print("Prefix=/usr\n");
+		$fh->print("HostData=lib/" . dpkg_architecture_value("DEB_HOST_MULTIARCH") . "/qt5\n");
+		$fh->print("Headers=include/" . dpkg_architecture_value("DEB_HOST_MULTIARCH") . "/qt5\n");
+		close $fh;
+		push @options, ("-qtconf", $filename);
 	}
 
 	if ($ENV{CFLAGS}) {
-- 
2.14.2

