--- busybox-1.10.2/debian/config/config.udeb.orig	2010-12-10 16:13:59.000000000 +0100
+++ busybox-1.10.2/debian/config/config.udeb	2010-12-10 16:14:28.000000000 +0100
@@ -589,7 +589,7 @@
 # CONFIG_FEATURE_PREFER_IPV4_ADDRESS is not set
 # CONFIG_VERBOSE_RESOLUTION_ERRORS is not set
 # CONFIG_ARP is not set
-# CONFIG_ARPING is not set
+CONFIG_ARPING=y
 # CONFIG_BRCTL is not set
 # CONFIG_FEATURE_BRCTL_FANCY is not set
 # CONFIG_DNSD is not set
