=== modified file 'init.d/ejabberd'
--- init.d/ejabberd	2010-03-23 21:16:58 +0000
+++ init.d/ejabberd	2014-04-09 22:41:49 +0000
@@ -133,8 +133,15 @@
 	    live
 	fi
     ;;
+    status)
+	if ctl status ; then
+	    echo -n "ejabberd is running"
+	else
+	    echo -n "ejabberd is stopped"
+	fi
+    ;;
     *)
-	echo "Usage: $0 {start|stop|restart|force-reload|live}" >&2
+	echo "Usage: $0 {start|stop|restart|force-reload|live|status}" >&2
 	exit 1
     ;;
 esac

