diff --git a/hooks/keymap b/hooks/keymap
index fa97209..cb770ac 100755
--- a/hooks/keymap
+++ b/hooks/keymap
@@ -21,20 +21,14 @@ if [ "$KEYMAP" != "y" ] && [ "$KEYMAP" != "Y" ]; then
 fi
 
 # Step 1 - Basic tools
-if [ ! -x /bin/loadkeys ] ||
-! ( [ -r /etc/console/boottime.kmap.gz ] \
-	|| [ -r /etc/console-setup/cached.kmap.gz ] ); then
+if [ ! -x /bin/loadkeys ] || [ ! -x /bin/setupcon ]; then
 	exit 0
 fi
 
 . /usr/share/initramfs-tools/hook-functions
 copy_exec /bin/loadkeys /bin
 
-if [ -e /etc/console-setup/cached.kmap.gz ]; then
-	cp /etc/console-setup/cached.kmap.gz ${DESTDIR}/etc/boottime.kmap.gz
-else
-	cp /etc/console/boottime.kmap.gz ${DESTDIR}/etc/
-fi
+/bin/setupcon --save-keyboard ${DESTDIR}/etc/boottime.kmap.gz
 
 # Step 2 - Check for UTF8 console
 if [ ! -x /usr/bin/kbd_mode ]; then
