diff -Nru audacious-docklet-0.1.1.orig/src/docklet.c audacious-docklet-0.1.1/src/docklet.c
--- audacious-docklet-0.1.1.orig/src/docklet.c	2006-01-21 02:01:39.000000000 +0100
+++ audacious-docklet-0.1.1/src/docklet.c	2006-11-29 04:43:40.000000000 +0100
@@ -109,6 +109,9 @@
     case 1:
       toggle_all_windows ();
       break;
+    case 2:
+      xmms_remote_playlist_next (docklet_plugin.xmms_session);
+      break;
     case 3:
       docklet_menu_show (event);
       break;
