From 9ef54cf63a24e489802dcf8a814618b633c8308a Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas@debian.org>
Date: Fri, 19 Jul 2024 14:08:10 +0200
Subject: [PATCH 3/3] scripts/mk: fix expansions passed to dpkg-buildflags.
 Closes: #1076568

---
 scripts/mk/buildflags.mk | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/scripts/mk/buildflags.mk b/scripts/mk/buildflags.mk
index a8eac9e54..bc0448881 100644
--- a/scripts/mk/buildflags.mk
+++ b/scripts/mk/buildflags.mk
@@ -57,7 +57,7 @@ dpkg_buildflags_run = $(eval $(shell \
       $(foreach operation,SET STRIP APPEND PREPEND,\
         DEB_$(flag)_MAINT_$(operation))),\
     $(if $(value $(exported)),\
-      $(exported)="$(value $(exported))"))\
+      $(exported)="$($(exported))"))\
   dpkg-buildflags | sed 's/\([^=]*\)\(.*\)/$$(eval \1:\2)/'))
 
 ifdef DPKG_EXPORT_BUILDFLAGS
-- 
2.39.2

