--- a/policy.sgml	2013-10-18 01:02:39.000000000 +0200
+++ b/policy.sgml	2014-06-30 16:30:42.573238978 +0200
@@ -3674,7 +3674,7 @@
 	  <p>
 	    The special value <tt>byhand</tt> for the section in a
 	    <tt>.changes</tt> file indicates that the file in question
-	    is not an ordinary package file and must by installed by
+	    is not an ordinary package file and must be installed by
 	    hand by the distribution maintainers.  If the section is
 	    <tt>byhand</tt> the priority should be <tt>-</tt>.
 	  </p>
