--- /usr/share/cdbs/1/class/makefile.mk.orig	2007-09-06 09:22:50.000000000 +0200
+++ /usr/share/cdbs/1/class/makefile.mk	2007-09-21 21:19:53.000000000 +0200
@@ -36,7 +36,7 @@
 
 clean:: makefile-clean
 makefile-clean::
-	$(if $(DEB_MAKE_CLEAN_TARGET),+-$(DEB_MAKE_INVOKE) -k $(DEB_MAKE_CLEAN_TARGET),@echo "DEB_MAKE_CLEAN_TARGET unset, not running clean")
+	$(if $(DEB_MAKE_CLEAN_TARGET),+[ ! -f $(if $(DEB_MAKE_MAKEFILE),$(DEB_MAKE_MAKEFILE),Makefile -a ! -f makefile -a ! -f GNUMakefile) ] || $(DEB_MAKE_INVOKE) -k $(DEB_MAKE_CLEAN_TARGET),@echo "DEB_MAKE_CLEAN_TARGET unset, not running clean")
 	rm -f debian/stamp-makefile-build
 
 common-install-arch common-install-indep:: common-install-impl
