diff -ur a//cups/http-support.c b//cups/http-support.c
--- a//cups/http-support.c	2011-10-15 22:13:35.000000000 +0100
+++ b//cups/http-support.c	2011-10-15 22:10:43.000000000 +0100
@@ -1335,9 +1335,12 @@
     case HTTP_WEBIF_DISABLED :
         s = _("Web Interface is Disabled");
 	break;
+    case HTTP_ERROR :
+        s = _("http error");
+	break;
 
     default :
-        s = _("Unknown");
+        s = _("Unknown http status");
 	break;
   }
 
