Index: debian/changelog
===================================================================
--- debian/changelog	(revision 2248)
+++ debian/changelog	(revision 2249)
@@ -1,3 +1,13 @@
+debconf (1.5.18) UNRELEASED; urgency=low
+
+  [ Søren Hansen ]
+  * debconf-apt-progress sometimes gets captured by buggy daemons, causing
+    tasksel to hang because $debconf_command_eof never becomes true.
+    STATUS_READ should be the last fd to close, so checking $status_eof is
+    sufficient (LP: #141601).
+
+ -- Colin Watson <cjwatson@debian.org>  Tue, 04 Dec 2007 17:09:43 +0000
+
 debconf (1.5.17) unstable; urgency=low
 
   * Partial support for cancelable progress bars. While the dialog frontend
Index: debconf-apt-progress
===================================================================
--- debconf-apt-progress	(revision 2248)
+++ debconf-apt-progress	(revision 2249)
@@ -409,11 +409,13 @@
 	my $status_buf = '';
 	my $debconf_command_buf = '';
 
-	while (not $status_eof or not $debconf_command_eof) {
+	# STATUS_READ should be the last fd to close. DEBCONF_COMMAND_WRITE
+	# may end up captured by buggy daemons, so terminate the loop even
+	# if we haven't hit $debconf_command_eof.
+	while (not $status_eof) {
 		my $rin = '';
 		my $rout;
-		vec($rin, fileno(STATUS_READ), 1) = 1
-			unless $status_eof;
+		vec($rin, fileno(STATUS_READ), 1) = 1;
 		vec($rin, fileno(DEBCONF_COMMAND_READ), 1) = 1
 			unless $debconf_command_eof;
 		my $sel = select($rout = $rin, undef, undef, undef);
