--- debconf/usr/sbin/dpkg-preconfigure	2007-07-25 14:59:59.000000000 +0100
+++ /tmp/dpkg-preconfigure	2007-08-22 08:51:34.000000000 +0100
@@ -35,6 +35,16 @@
 @ARGV=();
 
 my $have_tty=1;
+	# apt-get is closing the FD without unset the ENV
+	if ($ENV{DEBCONF_REDIR}) {
+	  local (*TMP);
+	  if (! open(TMP,"<&3")) {
+            delete $ENV{DEBCONF_REDIR};
+            delete $ENV{DEBIAN_HAS_FRONTEND};
+          } else {
+            close TMP;
+          }
+	}
 
 if ($apt) {
 	while (<>) {
@@ -100,6 +109,7 @@
 	exit;
 }
 my @buffer=<INFO>;
+close INFO;
 if ($apt && @buffer) {
 	print gettext("Preconfiguring packages ...\n");
 }
