From e0d128e11506ca142999bc79104cd1442d0105df Mon Sep 17 00:00:00 2001
From: Charles Plessy <plessy@debian.org>
Date: Sun, 24 Feb 2013 12:11:00 +0900
Subject: [PATCH] Installed file names must be in UTF-8 and should use only
 ASCII characters.

Closes: #701081
---
 policy.sgml | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/policy.sgml b/policy.sgml
index d81891c..c619d44 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -9463,6 +9463,16 @@ done
 	  </p>
 	</sect1>
       </sect>
+
+      <sec id="filenames">
+	<heading>File names</heading>
+
+	<p>
+	  The name of the files and directories installed by binary packages
+	  must be encoded in UTF-8 and should be restricted to ASCII when they
+	  can be represented in that character set.
+	</p>
+      </sec>
     </chapt>
 
 
-- 
1.8.2.rc0

