--- dcut~	2004-09-08 21:37:18.000000000 +0200
+++ dcut	2004-09-08 21:38:01.000000000 +0200
@@ -61,10 +61,10 @@
   # parse command line arguments
   try:
     (opts, arguments) = getopt.getopt(sys.argv[1:],
-                            'c:dDhsvmPU:O:',
+                            'c:dDhsvm:PU:O:',
                             ['config=', 'debug',
                              'help', 'simulate', 'version','host=',
-			     'maintainteraddress','passive','upload=',
+			     'maintainteraddress=','passive','upload=',
 			     'output='
 			     ])
   except getopt.error, msg:
@@ -104,6 +104,13 @@
     options['host'] = arguments[0]
     if options['debug']: print 'D: first argument "%s" treated as host'%(options['host'])
     del arguments[0]
+    
+  # we don't create command files without uploader
+  if not options['uploader'] and not options['filetoupload']:
+    print "%s error: command file cannot be create without maintainer email"%progname
+    print '%s        please either set $DEBEMAIL or use the "-m" option'%(len(progname)*' ')
+    sys.exit(1)
+    
   return options, arguments
 
 def parse_queuecommands(arguments,options,config):
