diff -Nru debmirror-2.4/debian/NEWS debmirror-2.4/debian/NEWS
--- debmirror-2.4/debian/NEWS	2009-10-03 21:33:44.000000000 +1000
+++ debmirror-2.4/debian/NEWS	2010-01-23 20:57:10.000000000 +1100
@@ -44,7 +44,7 @@
 
   * Option --root=directory no longer requires "/" or ":" prefix
 
-    It is not possible to leave the default at "debian" and all
+    It is now possible to leave the default at "debian" and all
     transfer methods should just work. For backwards compatibility
     debmirror will remove a leading "/" or ":" if one is passed.
 
@@ -128,14 +128,14 @@
     the index files are updated. That can result in files missing from
     the mirror when the mirroring fails and the index files aren't
     updated at the end (and also while debmirror runs). The
-    --postcleanup does not have that effect but can lead to temporaily
+    --postcleanup does not have that effect but can lead to temporarily
     more space usage on the mirror. If you are short on space you
     might want to make sure you use --cleanup.
 
   * Autodetecting non-existing archs and sections
 
     In the past it was impossible to mirror i386,amd64 and sarge,etch
-    because sarge has no amd64 architecture. Similary there is no
+    because sarge has no amd64 architecture. Similarly there is no
     debian-installer section in contrib. Debmirror now ignores any
     combination of arch, suite and section that does not exist locally
     and is not listed in the Release file for the suite. This
@@ -143,7 +143,7 @@
     mirror unknown archives like Ubuntu without problems.
 
     Note that debmirror will fail when a combination of arch, suite
-    and section that exists locally gets droped from the Release
+    and section that exists locally gets dropped from the Release
     file. There is no danger of loosing a branch when the Release file
     is corrupted or the upstream changes.
 
