From 8ae4a3a0f403b7ae3da5b8149cd606e6f8db88d0 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lo=C3=AFc=20Minier?= <lool@dooz.org>
Date: Mon, 11 Jan 2010 23:47:59 +0100
Subject: [PATCH 3/3] Honor nocheck in DEB_BUILD_OPTIONS

---
 debian/rules |    4 +++-
 1 files changed, 3 insertions(+), 1 deletions(-)

diff --git a/debian/rules b/debian/rules
index 7eb4a7d..35ffbaa 100755
--- a/debian/rules
+++ b/debian/rules
@@ -224,8 +224,10 @@ binary-indep: build install checkroot
 	cd debian/bogofilter-common && find * -type f ! -path "etc/bogofilter.cf" ! -regex '^DEBIAN/.*' -print0 | xargs -r0 md5sum > DEBIAN/md5sums
 	dpkg --build debian/bogofilter-common ..
 
+maybe_check = $(if $(findstring nocheck,$(DEB_BUILD_OPTIONS)),,check)
+
 # Build architecture-dependent files here.
-binary-arch: build install check checkroot
+binary-arch: build install $(maybe_check) checkroot
 	$(checkdir)
 	$(INSTALL_DIR) debian/bogofilter-bdb/usr/share/doc \
 		       debian/bogofilter-sqlite/usr/share/doc \
-- 
1.6.5

