--- /etc/bash_completion.d/aptitude	2010-11-21 20:45:04.000000000 +0100
+++ completion/bash-completion-1.2/contrib/aptitude	2010-06-13 12:21:19.000000000 +0200
@@ -29,7 +29,7 @@
         -Z -v --verbose --purge-unused --schedule-only'
 
     for (( i=0; i < ${#COMP_WORDS[@]}-1; i++ )); do
-        if [[ ${COMP_WORDS[i]} == @(@(|re)install|@(|un)hold|@(|un)markauto|@(dist|full)-upgrade|versions|download|show|forbid-version|purge|remove|changelog|why@(|-not)|keep@(|-all)|build-dep|@(add|remove)-user-tag) ]]; then
+        if [[ ${COMP_WORDS[i]} == @(@(|re)install|@(|un)hold|@(|un)markauto|@(dist|full)-upgrade|download|show|forbid-version|purge|remove|changelog|why@(|-not)|keep@(|-all)|build-dep|@(add|remove)-user-tag) ]]; then
         special=${COMP_WORDS[i]}
         fi
         #exclude some mutually exclusive options
@@ -39,7 +39,7 @@
 
     if [[ -n "$special" ]]; then
        case $special in
-           versions|install|hold|markauto|unmarkauto|dist-upgrade|full-upgrade| \
+           install|hold|markauto|unmarkauto|dist-upgrade|full-upgrade| \
            download|show|changelog|why|why-not|build-dep|add-user-tag| \
            remove-user-tag)
                COMPREPLY=( $( apt-cache pkgnames $cur 2> /dev/null ) )
@@ -81,7 +81,7 @@
             hold unhold purge markauto unmarkauto why why-not \
             dist-upgrade full-upgrade download search show \
             forbid-version changelog keep-all build-dep \
-            versions add-user-tag remove-user-tag' -- "$cur" ) )
+            add-user-tag remove-user-tag' -- "$cur" ) )
     fi
 
     return 0

