diff --git a/debian/changelog b/debian/changelog
index ea8a91d..29b752d 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,10 @@
+console-setup (1.190) UNRELEASED; urgency=medium
+
+  * console-setup(5) manpage: correct section of setfont manpage and remove
+    reference to no-longer existing consolechars manpage. Closes: #920312
+
+ -- Holger Wansing <hwansing@mailbox.org>  Thu, 14 Feb 2019 20:18:47 +0100
+
 console-setup (1.189) unstable; urgency=medium
   * Team upload
 
diff --git a/man/console-setup.5 b/man/console-setup.5
index 9153b2f..8bc515d 100644
--- a/man/console-setup.5
+++ b/man/console-setup.5
@@ -434,7 +434,6 @@ can be used in all video modes.
 .SH SEE ALSO
 .BR setupcon (1),
 .BR keyboard (5),
-.BR setfont (1),
-.BR consolechars (1),
+.BR setfont (8),
 .BR vidcontrol (1)
 
