diff -Nru email-reminder-0.7.7/debian/changelog email-reminder-0.7.7/debian/changelog
--- email-reminder-0.7.7/debian/changelog	2014-01-27 21:43:10.000000000 +0000
+++ email-reminder-0.7.7/debian/changelog	2014-05-01 22:56:11.000000000 +0000
@@ -1,3 +1,10 @@
+email-reminder (0.7.7-2.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Adding keywords to .desktop file
+
+ -- Chris Cormack <chrisc@catalyst.net.nz>  Thu, 01 May 2014 22:55:41 +0000
+
 email-reminder (0.7.7-2) unstable; urgency=medium
 
   * Force a shell for the daily cronjob's send-reminders su call
diff -Nru email-reminder-0.7.7/debian/patches/add_keywords_to_desktop.patch email-reminder-0.7.7/debian/patches/add_keywords_to_desktop.patch
--- email-reminder-0.7.7/debian/patches/add_keywords_to_desktop.patch	1970-01-01 00:00:00.000000000 +0000
+++ email-reminder-0.7.7/debian/patches/add_keywords_to_desktop.patch	2014-05-01 22:55:03.000000000 +0000
@@ -0,0 +1,10 @@
+Adding keywords to the .desktop file
+Index: email-reminder-0.7.7/email-reminder.desktop
+===================================================================
+--- email-reminder-0.7.7.orig/email-reminder.desktop	2008-01-26 23:54:21.000000000 +0000
++++ email-reminder-0.7.7/email-reminder.desktop	2014-05-01 22:54:24.965489710 +0000
+@@ -7,3 +7,4 @@
+ Type=Application
+ Categories=Utility;Calendar
+ StartupNotify=true
++Keywords=reminders todolist
diff -Nru email-reminder-0.7.7/debian/patches/series email-reminder-0.7.7/debian/patches/series
--- email-reminder-0.7.7/debian/patches/series	1970-01-01 00:00:00.000000000 +0000
+++ email-reminder-0.7.7/debian/patches/series	2014-05-01 22:52:04.000000000 +0000
@@ -0,0 +1 @@
+add_keywords_to_desktop.patch
