--- /usr/share/emacs/site-lisp/erlang/erlang.el	2014-11-30 20:50:50.000000000 +0100
+++ erlang.el	2014-12-11 13:40:04.881052009 +0100
@@ -2055,7 +2055,7 @@
   (if (or (null module) (string= module ""))
       (error "No Erlang module name given"))
   (let ((dir-list erlang-man-dirs)
-	(pat (concat "/" (regexp-quote module) "\\.\\([124-9]\\|3\\(erl\\)\\)\\(\\.gz\\)?$"))
+	(pat (concat "/" (regexp-quote module) "\\.\\(3erl\\)\\(\\.gz\\)?$"))
 	(file nil)
 	file-list)
     (while (and dir-list (null file))
