diff -ru a/debian/carmetal.mime b/debian/carmetal.mime
--- a/debian/carmetal.mime	2011-03-07 12:47:14.000000000 +0100
+++ b/debian/carmetal.mime	2021-04-23 11:01:48.791570425 +0200
@@ -1,2 +1,2 @@
-application/x-carmetal; carmetal '%s' ; edit=carmetal '%s' ; test=test "$DISPLAY" != "" ; description="CaRMetal figure" ; priority=2
+application/x-carmetal; carmetal %s ; edit=carmetal %s ; test=test "$DISPLAY" != "" ; description="CaRMetal figure" ; priority=2

