diff -Nru crash-7.0.7.orig/debian/tests/control crash-7.0.7/debian/tests/control
--- crash-7.0.7.orig/debian/tests/control	2014-08-02 15:42:32.967522345 -0300
+++ crash-7.0.7/debian/tests/control	2014-08-03 23:52:54.206595695 -0300
@@ -1,3 +1,3 @@
 Tests: live
 Restrictions: needs-root
-Depends: @, lsb-release
+Depends: @, lsb-release, sudo
diff -Nru crash-7.0.7.orig/debian/tests/live crash-7.0.7/debian/tests/live
--- crash-7.0.7.orig/debian/tests/live	2014-08-02 15:42:32.967522345 -0300
+++ crash-7.0.7/debian/tests/live	2014-08-03 23:57:23.513177432 -0300
@@ -3,14 +3,15 @@
 
 echo "Adding linux-image debug symbols."
 if [ "$(lsb_release -is)" = "Debian" ]; then
+    sudo apt-get update
     sudo apt-get install linux-image-$(uname -r)-dbg
 elif [ "$(lsb_release -is)" = "Ubuntu" ]; then
     sudo tee /etc/apt/sources.list.d/ddebs.list << EOF
-    deb http://ddebs.ubuntu.com/ $(lsb_release -cs)          main restricted universe multiverse
-    deb http://ddebs.ubuntu.com/ $(lsb_release -cs)-security main restricted universe multiverse
-    deb http://ddebs.ubuntu.com/ $(lsb_release -cs)-updates  main restricted universe multiverse
-    deb http://ddebs.ubuntu.com/ $(lsb_release -cs)-proposed main restricted universe multiverse
-    EOF
+deb http://ddebs.ubuntu.com/ $(lsb_release -cs)          main restricted universe multiverse
+deb http://ddebs.ubuntu.com/ $(lsb_release -cs)-security main restricted universe multiverse
+deb http://ddebs.ubuntu.com/ $(lsb_release -cs)-updates  main restricted universe multiverse
+deb http://ddebs.ubuntu.com/ $(lsb_release -cs)-proposed main restricted universe multiverse
+EOF
     sudo apt-key adv --keyserver keyserver.ubuntu.com --recv-keys ECDCAD72428D7C01
     sudo apt-get update
     sudo apt-get install linux-image-$(uname -r)-dbgsym
