Description: Add support for loongarch
Last-Update: 2023-09-13

--- a2jmidid-9.orig/sigsegv.c
+++ a2jmidid-9/sigsegv.c
@@ -93,7 +93,7 @@ static void signal_segv(int signum, sigi
     a2j_error("info.si_addr  = %p", info->si_addr);
 #if !defined(__alpha__) && !defined(__ia64__) && \
     !defined(__FreeBSD_kernel__) && !defined(__arm__) && !defined(__hppa__) && \
-    !defined(__sh__) && !defined(__aarch64__) && !defined(__riscv)
+    !defined(__sh__) && !defined(__aarch64__) && !defined(__riscv) && !defined(__loongarch__)
     for(i = 0; i < NGREG; i++)
         a2j_error("reg[%02d]       = 0x" REGFORMAT, i,
 #if defined(__powerpc__) && !defined(__powerpc64__)
@@ -106,7 +106,7 @@ static void signal_segv(int signum, sigi
                 ucontext->uc_mcontext.gregs[i]
 #endif
                 );
-#endif /* alpha, ia64, kFreeBSD, arm, hppa, aarch64, riscv */
+#endif /* alpha, ia64, kFreeBSD, arm, hppa, aarch64, riscv, loongarch */
 
 #if defined(SIGSEGV_STACK_X86) || defined(SIGSEGV_STACK_IA64)
 # if defined(SIGSEGV_STACK_IA64)
