=== modified file 'apt-pkg/contrib/strutl.cc'
--- apt-pkg/contrib/strutl.cc	2008-10-03 18:57:25 +0000
+++ apt-pkg/contrib/strutl.cc	2008-11-18 21:58:49 +0000
@@ -304,13 +304,13 @@
    {
       if (ASize < 100 && I != 0)
       {
-         sprintf(S,"%.1f%c",ASize,Ext[I]);
+         sprintf(S,"%'.1f %c",ASize,Ext[I]);
 	 break;
       }
       
       if (ASize < 10000)
       {
-         sprintf(S,"%.0f%c",ASize,Ext[I]);
+         sprintf(S,"%'.0f %c",ASize,Ext[I]);
 	 break;
       }
       ASize /= 1000.0;

=== modified file 'cmdline/acqprogress.cc'
--- cmdline/acqprogress.cc	2008-11-16 20:55:04 +0000
+++ cmdline/acqprogress.cc	2008-11-18 21:57:18 +0000
@@ -198,7 +198,7 @@
             
       // Add the current progress
       if (Mode == Long)
-	 snprintf(S,End-S," %lu",I->CurrentSize);
+	 snprintf(S,End-S," %'lu",I->CurrentSize);
       else
       {
 	 if (Mode == Medium || I->TotalSize == 0)

