diff -wru apt-file-2.0.3/apt-file apt-file-2.0.3-he/apt-file
--- apt-file-2.0.3/apt-file	2003-05-16 15:14:31.000000000 +0200
+++ apt-file-2.0.3-he/apt-file	2003-10-21 17:45:51.000000000 +0200
@@ -99,9 +99,11 @@
 
 	# Handle $(ARCH) in sources.list
 	$line =~ s/\$\(ARCH\)/$Conf->{arch}/g;
-	debug "kept $line";
 
 	my( $pkg, $uri, $dist, @extra) = split /\s+/, $line;
+	next if $dist =~ /experimental/;
+	
+	debug "kept $line";	
 	my($scheme, $user, $passwd, $host, $port, $path, $query,
 	   $fragment) =
 	       $uri =~
Only in apt-file-2.0.3-he: apt-file.orig
