=== modified file 'apt-pkg/cacheiterators.h'
--- apt-pkg/cacheiterators.h	2006-12-14 11:39:29 +0000
+++ apt-pkg/cacheiterators.h	2008-12-17 21:44:22 +0000
@@ -360,7 +360,7 @@
    // Iteration
    void operator ++(int) {if (FileP != Owner->VerFileP) FileP = Owner->VerFileP + FileP->NextFile;};
    inline void operator ++() {operator ++(0);};
-   inline bool end() const {return FileP == Owner->VerFileP?true:false;};
+   inline bool end() const {return Owner == NULL || FileP == Owner->VerFileP?true:false;};
 
    // Comparison
    inline bool operator ==(const VerFileIterator &B) const {return FileP == B.FileP;};

