=== modified file 'apt-pkg/policy.cc'
--- apt-pkg/policy.cc	2007-06-08 23:44:03 +0000
+++ apt-pkg/policy.cc	2008-11-14 20:17:51 +0000
@@ -117,7 +117,7 @@
 pkgCache::VerIterator pkgPolicy::GetCandidateVer(pkgCache::PkgIterator Pkg)
 {
    // Look for a package pin and evaluate it.
-   signed Max = GetPriority(Pkg);
+   signed short Max = GetPriority(Pkg);
    pkgCache::VerIterator Pref = GetMatch(Pkg);
 
    /* Falling through to the default version.. Setting Max to zero
@@ -145,7 +145,7 @@
 	     Pkg.CurrentVer() != Ver)
 	    continue;
 	 	 	 
-	 signed Prio = PFPriority[VF.File()->ID];
+	 signed short Prio = PFPriority[VF.File()->ID];
 	 if (Prio > Max)
 	 {
 	    Pref = Ver;

