=== modified file 'apt-pkg/depcache.cc'
--- apt-pkg/depcache.cc	2008-10-28 17:14:29 +0000
+++ apt-pkg/depcache.cc	2008-12-17 21:07:07 +0000
@@ -1146,7 +1146,7 @@
 {
    /* Not source/not automatic versions cannot be a candidate version 
       unless they are already installed */
-   VerIterator Last(*(pkgCache *)this,0);
+   VerIterator Last(*(pkgCache *)this,NULL);
    
    for (VerIterator I = Pkg.VersionList(); I.end() == false; I++)
    {

