--- debian/acpi-support/etc/acpi/asus-wireless.sh.orig	2010-12-19 23:32:58.000000000 +0100
+++ debian/acpi-support/etc/acpi/asus-wireless.sh	2010-12-20 17:27:08.000000000 +0100
@@ -5,12 +5,18 @@ test -f /usr/share/acpi-support/state-fu
 
 . /usr/share/acpi-support/state-funcs
 
-VENDOR=`cat /sys/class/dmi/id/sys_vendor | tr [:upper:] [:lower:] | cut -c1-4`
-test $VENDOR = "asus" || exit 0
+read vendor </sys/class/dmi/id/sys_vendor 2>/dev/null || exit 0
+case $vendor in
+	[As][Ss][Uu][Ss]*)
+		;;
+	*)
+		exit 0
+		;;
+esac
 
 if [ "$1" = "" ] ; then
 	toggleAllWirelessStates;
-elif ( isAnyWirelessPoweredOn ) ; then
+elif isAnyWirelessPoweredOn; then
 	if [ "$1" = "off" ] ; then
 		toggleAllWirelessStates;
 	fi
