--- policy.sgml	2006-10-03 01:36:50.000000000 +0300
+++ policy.sgml	2007-04-09 21:40:45.000000000 +0300
@@ -4711,7 +4711,9 @@ Replaces: mail-transport-agent
 	  Typically the development version should have an exact
 	  version dependency on the runtime library, to make sure that
 	  compilation and linking happens correctly.  The
-	  <tt>${Source-Version}</tt> substitution variable can be
+	  <tt>${binary:Version}</tt><footnote>Note that the
+	  <tt>${Source-Version}</tt> subsitution variable is deprecated
+	  since dpkg 1.13.19</footnote> substitution variable can be
 	  useful for this purpose.
 	</p>
       </sect>
