diff -ur dnssec-trigger-0.13~svn685.orig/panel/dnssec-trigger-panel.desktop.in dnssec-trigger-0.13~svn685/panel/dnssec-trigger-panel.desktop.in
--- dnssec-trigger-0.13~svn685.orig/panel/dnssec-trigger-panel.desktop.in	2014-07-15 01:14:57.000000000 -0700
+++ dnssec-trigger-0.13~svn685/panel/dnssec-trigger-panel.desktop.in	2014-07-26 11:43:13.494258925 -0700
@@ -5,8 +5,8 @@
 Name=DNSSEC Trigger
 GenericName=Network Applet
 Comment=Shows DNS state and warning dialog
-Exec=0bindir0/dnssec-trigger-panel
-Icon=0uidir0/status-icon.png
+Exec=@bindir@/dnssec-trigger-panel
+Icon=@uidir@/status-icon.png
 Terminal=false
 Categories=Utility;
 X-KDE-StartupNotify=false
