Index: clusterssh-4.00.01/bin/cssh
===================================================================
--- clusterssh-4.00.01.orig/bin/cssh	2010-06-18 09:51:46.000000000 +0200
+++ clusterssh-4.00.01/bin/cssh	2010-06-18 09:52:07.000000000 +0200
@@ -110,6 +110,7 @@
     'term-args|t=s',
     'title|T=s',
     'output-config|u',
+    'font|f=s',
 );
 my %options;
 my %config;
@@ -399,6 +400,9 @@
     }
 
     $config{internal_previous_state} = "";    # set to default
+
+    # option font overrides config file font setting
+    $config{terminal_font} = $options{font} if ( $options{font} );
     get_font_size();
 
     $config{extra_cluster_file} =~ s/\s+//g;
@@ -2450,6 +2454,10 @@
 Output the current configuration in the same format used by the 
 F<$HOME/.csshrc> file.
 
+=item --font,-u "5x8"
+
+Specify the font to use in the terminal windows. Use standard X font notation.
+
 =item --version,-v
 
 Show version information and exit
