Index: debian/patches/bristuff/app-dial-priority-202
===================================================================
--- debian/patches/bristuff/app-dial-priority-202	(revision 7415)
+++ debian/patches/bristuff/app-dial-priority-202	(working copy)
@@ -21,7 +21,7 @@
  "    k    - Allow the called party to enable parking of the call by sending\n"
  "           the DTMF sequence defined for call parking in features.conf.\n"
  "    K    - Allow the calling party to enable parking of the call by sending\n"
-@@ -1292,14 +1297,16 @@ static int dial_exec_full(struct ast_cha
+@@ -1292,14 +1297,18 @@ static int dial_exec_full(struct ast_cha
  	}
  
  	if (!outgoing) {
@@ -32,7 +32,9 @@
  			goto out;
  		}
 +		/* See if there is a special message */
-+		ast_goto_if_exists(chan, chan->context, chan->exten, chan->priority + 201);
++		if (ast_opt_priority_jumping || ast_test_flag(&opts, OPT_PRIORITY_JUMP)) {
++			ast_goto_if_exists(chan, chan->context, chan->exten, chan->priority + 201);
++		}
  	} else {
  		/* Our status will at least be NOANSWER */
 -		strcpy(status, "NOANSWER");
