From 91e4168bb3326c304d945fb794641ae5f1da504c Mon Sep 17 00:00:00 2001
From: Ondrej Certik <ondrej@certik.cz>
Date: Thu, 25 Dec 2008 18:35:35 +0100
Subject: [PATCH] Fixing all paths to /usr/sbin/etckeeper.

---
 apt.conf       |    8 ++++----
 pacman-g2.hook |    4 ++--
 2 files changed, 6 insertions(+), 6 deletions(-)

diff --git a/apt.conf b/apt.conf
index b2e0f9b..bd6afc5 100644
--- a/apt.conf
+++ b/apt.conf
@@ -1,5 +1,5 @@
-DPkg::Pre-Install-Pkgs { "if [ -x /usr/bin/etckeeper ]; then etckeeper pre-install; fi"; };
-DPkg::Post-Invoke      { "if [ -x /usr/bin/etckeeper ]; then etckeeper post-install; fi"; };
+DPkg::Pre-Install-Pkgs { "if [ -x /usr/sbin/etckeeper ]; then etckeeper pre-install; fi"; };
+DPkg::Post-Invoke      { "if [ -x /usr/sbin/etckeeper ]; then etckeeper post-install; fi"; };
 
-RPM::Pre-Install-Pkgs { "if [ -x /usr/bin/etckeeper ]; then etckeeper pre-install; fi"; };
-RPM::Post-Invoke      { "if [ -x /usr/bin/etckeeper ]; then etckeeper post-install; fi"; };
+RPM::Pre-Install-Pkgs { "if [ -x /usr/sbin/etckeeper ]; then etckeeper pre-install; fi"; };
+RPM::Post-Invoke      { "if [ -x /usr/sbin/etckeeper ]; then etckeeper post-install; fi"; };
diff --git a/pacman-g2.hook b/pacman-g2.hook
index ac2e099..bd93d73 100644
--- a/pacman-g2.hook
+++ b/pacman-g2.hook
@@ -1,13 +1,13 @@
 #!/bin/sh
 
 pre_sysupgrade() {
-	if [ -x /usr/bin/etckeeper ]; then
+	if [ -x /usr/sbin/etckeeper ]; then
 		etckeeper pre-install
 	fi
 }
 
 post_sysupgrade() {
-	if [ -x /usr/bin/etckeeper ]; then
+	if [ -x /usr/sbin/etckeeper ]; then
 		etckeeper post-install
 	fi
 }
-- 
1.5.6.5

