Index: scripts/debcommit.pl
===================================================================
--- scripts/debcommit.pl	(revision 1410)
+++ scripts/debcommit.pl	(working copy)
@@ -576,7 +576,7 @@
 	} elsif ($prog eq 'baz') {
 	    @diffcmd = ($prog, 'file-diff');
 	} elsif ($prog eq 'bzr') {
-	    @diffcmd = ($prog, 'diff', '--using', '/usr/bin/diff', '--diff-options', '-wu');
+	    @diffcmd = ($prog, 'diff', '--diff-options', '-wu');
 	} elsif ($prog eq 'git') {
 	    if ($all) {
 		@diffcmd = ('git', 'diff', '-w', '--no-color');
