diff -r 5ec42fce9483 src/generic/apt/aptcache.cc
--- a/src/generic/apt/aptcache.cc	Thu Jul 19 07:29:45 2007 -0700
+++ b/src/generic/apt/aptcache.cc	Thu Jul 26 19:19:59 2007 -0700
@@ -1428,7 +1428,10 @@ void aptitudeDepCache::apply_solution(co
 	{
 	  set_candidate_version(actionver, NULL);
 	  internal_mark_install(pkg, false, false);
-	  MarkAuto(pkg, true);
+	  // Mark the package as automatic iff it isn't currently
+	  // installed.
+	  if(curver.end())
+	    MarkAuto(pkg, true);
 	}
     }
 }

