diff --git a/Makefile.in b/Makefile.in
index 49cdf12..6a93f8f 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -180,7 +180,7 @@ install: all
 	  b=`basename $$i` ; \
 	  ( cd $(bin_dir) ; \
             pod2man --release=$(release) --section=1 --date=$(date) \
-              --center=" "  $$b ) | sed s/ê/\\\\[^e]/ > $(man1_dir)/$$b.1 ; \
+              --center=" "  $$b ) | sed s/E\<ecirc\>/\\\\[^e]/ > $(man1_dir)/$$b.1 ; \
 	done
 	( cd scripts/Debian ; \
           pod2man --release=$(release) --section=3pm --date=$(date) \
diff --git a/scripts/system/ispell-wrapper b/scripts/system/ispell-wrapper
index 4592e18..9a7edb4 100755
--- a/scripts/system/ispell-wrapper
+++ b/scripts/system/ispell-wrapper
@@ -200,8 +200,8 @@ Let us say that the following dictionaries are installed in the system
 
     castellano (Spanish TeX mode)
     castellano8 (Spanish 8 bit)
-    português (European Portuguese)
-    português brasileiro (Brazilian Portuguese)
+    portuguE<ecirc>s (European Portuguese)
+    portuguE<ecirc>s brasileiro (Brazilian Portuguese)
 
 Choosing the regexp (either in the --language option or in the
 environment variable ISPELLDEFAULT) to be "span" will yield an error,
