--- debian/control.bak	2011-06-17 11:24:05.017333500 +0200
+++ debian/control	2011-06-17 11:28:39.087845149 +0200
@@ -7,7 +7,7 @@
 
 Package: backuppc
 Architecture: any
-Depends: ${shlibs:Depends}, ${perl:Depends}, libdigest-md5-perl, libcompress-zlib-perl, libarchive-zip-perl, tar (>> 1.13), adduser (>=3.9), dpkg (>=1.8.3), apache2 | httpd, ${misc:Depends}, smbclient, samba-common-bin, bzip2, default-mta | exim4 | mail-transport-agent, iputils-ping, ucf, libtime-modules-perl, libwww-perl
+Depends: ${shlibs:Depends}, ${perl:Depends}, libdigest-md5-perl, libcompress-zlib-perl, libarchive-zip-perl, tar (>> 1.13), adduser (>=3.9), dpkg (>=1.8.3), apache2 | httpd, ${misc:Depends}, smbclient, samba-common-bin, bzip2, default-mta | exim4 | mail-transport-agent, iputils-ping | ping, ucf, libtime-modules-perl, libwww-perl
 Recommends: rsync, libfile-rsyncp-perl (>= 0.68), openssh-client | ssh-client, rrdtool, libio-dirent-perl
 Suggests: w3m | www-browser, par2
 Conflicts: libfile-rsyncp-perl (<< 0.68)
