--- debtags.py.old	2009-05-01 23:24:51.000000000 +0100
+++ debtags.py.new	2009-05-01 23:00:19.000000000 +0100
@@ -89,7 +89,7 @@
         document  is the document to update
         pkg       is the python-apt Package object for this package
         """
-        for tag in self.db.tagsOfPackage(pkg.name):
+        for tag in self.db.tags_of_package(pkg.name):
             document.add_term("XT"+tag)
 
     def indexDeb822(self, document, pkg):
@@ -102,7 +102,7 @@
         document  is the document to update
         pkg       is the Deb822 object for this package
         """
-        for tag in self.db.tagsOfPackage(pkg["Package"]):
+        for tag in self.db.tags_of_package(pkg["Package"]):
             document.add_term("XT"+tag)
 
 def init():
