Index: debian/installer/package-list
===================================================================
--- debian/installer/package-list	(revision 19377)
+++ debian/installer/package-list	(working copy)
@@ -332,6 +332,12 @@
  This package contains core modules for the kernel, that will almost 
  always be needed.
 
+Package: fuse-modules
+Depends: kernel-image
+Priority: standard
+Description: FUSE support modules
+ This package contains filesystem in userspace support modules.
+
 Package: acpi-modules
 Depends: kernel-image
 Priority: extra
Index: debian/installer/modules/fuse-modules
===================================================================
--- debian/installer/modules/fuse-modules	(revision 0)
+++ debian/installer/modules/fuse-modules	(revision 0)
@@ -0,0 +1 @@
+fuse
