diff --git a/debian/doomsday-compat b/debian/doomsday-compat
index 6efd3eb..998c81d 100755
--- a/debian/doomsday-compat
+++ b/debian/doomsday-compat
@@ -138,6 +138,14 @@ case "$(echo $pwadfile | tr '[:upper:]' '[:lower:]')" in
 		;;
 esac
 
+if [ $gamemode = "doom2" \
+     -a ! -f $iwaddir/doom2.wad \
+     -a ! -f $iwaddir/doom2f.wad \
+     -a -f $iwaddir/freedoom2.wad ]
+then
+	gamemode=freedoom2
+fi
+
 if [ -n "$pwadpath" ]
 then
 	cmdline="-file $pwadpath $cmdline"
