diff -u ../t/deja-dup-14.2/debian//changelog debian//changelog
--- ../t/deja-dup-14.2/debian//changelog	2011-04-17 13:42:26.000000000 +0200
+++ debian//changelog	2011-04-17 09:35:23.000000000 +0200
@@ -1,3 +1,14 @@
+deja-dup (18.1.1-1) unstable; urgency=low
+
+  [ Sylvain Beucler ]
+  * New upstream version (Closes: #613336)
+  * Precise dependency version for libglib2.0-dev (requires new
+    'glib-compile-schemas' tool)
+  * Bump dependency to libgtk2.0-dev to 2.18
+  * Bump dependency to libnotify-dev to 0.7
+
+ -- Jose Carlos Garcia Sogo <jsogo@debian.org>  Sun, 17 Apr 2011 09:34:58 +0200
+
 deja-dup (14.2-1) unstable; urgency=low
 
   * New upstream release
diff -u ../t/deja-dup-14.2/debian//control debian//control
--- ../t/deja-dup-14.2/debian//control	2011-04-17 13:42:26.000000000 +0200
+++ debian//control	2011-04-17 13:28:31.000000000 +0200
@@ -7,12 +7,12 @@
                intltool,
                debhelper (>= 5),
                pkg-config,
-               libglib2.0-dev,
-               libgtk2.0-dev (>= 2.12),
+               libglib2.0-dev (>= 2.25.3),
+               libgtk2.0-dev (>= 2.18),
                libgconf2-dev,
                libgnomeui-dev,
                libgnome-keyring-dev,
-               libnotify-dev,
+               libnotify-dev (>= 0.7),
                libunique-dev,
                libnautilus-extension-dev,
                gnome-doc-utils (>= 0.3.2),
@@ -27,7 +27,8 @@
 Depends: ${shlibs:Depends},
          ${misc:Depends},
          duplicity (>= 0.6.07),
-	 gvfs-fuse
+	 gvfs-fuse,
+	 dconf-gsettings-backend
 Recommends: python-boto (>= 0.9d), ssh-client | openssh-client
 Suggests: util-linux (>= 2.13)
 Description: Backup utility
Seulement dans debian/: source
