diff -u bitlbee-3.2.1+otr4/debian/changelog bitlbee-3.2.1+otr4/debian/changelog
--- bitlbee-3.2.1+otr4/debian/changelog
+++ bitlbee-3.2.1+otr4/debian/changelog
@@ -1,3 +1,13 @@
+bitlbee (3.2.1+otr4-1.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Rebuild against GnuTLS v3. Closes: #753020
+  * Comment unbuilt packages bitlbee-plugin-skype and skyped in
+    debian/control. As a side-effect of allowing source only uploads listing
+    unbuilt packages in debian/control triggers NEW processing.
+
+ -- Andreas Metzler <ametzler@debian.org>  Sat, 23 Aug 2014 19:39:58 +0200
+
 bitlbee (3.2.1+otr4-1) unstable; urgency=low
 
   * New upstream version, fixes Twitter API compatibility (now using HTTP/1.1).
diff -u bitlbee-3.2.1+otr4/debian/control bitlbee-3.2.1+otr4/debian/control
--- bitlbee-3.2.1+otr4/debian/control
+++ bitlbee-3.2.1+otr4/debian/control
@@ -4,7 +4,7 @@
 Maintainer: Wilmer van der Gaast <wilmer@gaast.net>
 Uploaders: Jelmer Vernooij <jelmer@samba.org>
 Standards-Version: 3.9.1
-Build-Depends: libglib2.0-dev (>= 2.4), libevent-dev, libgnutls-dev | gnutls-dev, po-debconf, libpurple-dev, libotr5-dev | libotr2-dev, debhelper (>= 6.0.7~), asciidoc
+Build-Depends: libglib2.0-dev (>= 2.4), libevent-dev, libgnutls28-dev | gnutls-dev, po-debconf, libpurple-dev, libotr5-dev | libotr2-dev, debhelper (>= 6.0.7~), asciidoc
 Homepage: http://www.bitlbee.org/
 Vcs-Bzr: http://code.bitlbee.org/bitlbee/
 DM-Upload-Allowed: yes
@@ -72,22 +72,22 @@
-Package: bitlbee-plugin-skype
-Architecture: any
-Depends: ${misc:Depends}, ${shlibs:Depends}, bitlbee (= ${binary:Version}) | bitlbee-libpurple (= ${binary:Version}), bitlbee-common (= ${source:Version})
-Recommends: skyped
-Description: An IRC to other chat networks gateway (Skype plugin)
- This program can be used as an IRC server which forwards everything you
- say to people on other chat networks: Jabber (which includes Google Talk
- and Facebook Chat), ICQ, AIM, MSN, Yahoo! and Twitter/Identica/Status.net.
- .
- This package contains a plugin that adds support for the Skype IM network.
- You need to download and install the Skype client for this to work.
-
-Package: skyped
-Architecture: all
-Depends: ${misc:Depends}, ${shlibs:Depends}, python (>= 2.5), python-gnutls, python-skype (>=0.9.28.7)
-Recommends: skype
-Description: Daemon to control Skype remotely
- Daemon to control the GUI Skype client. Currently required to control Skype
- from the BitlBee IRC2IM gateway. Skyped and Skype can run on a different
- host than the BitlBee server, the communication is encrypted.
- .
- You need to download and install the Skype client for this to work.
+# Package: bitlbee-plugin-skype
+# Architecture: any
+# Depends: ${misc:Depends}, ${shlibs:Depends}, bitlbee (= ${binary:Version}) | bitlbee-libpurple (= ${binary:Version}), bitlbee-common (= ${source:Version})
+# Recommends: skyped
+# Description: An IRC to other chat networks gateway (Skype plugin)
+#  This program can be used as an IRC server which forwards everything you
+#  say to people on other chat networks: Jabber (which includes Google Talk
+#  and Facebook Chat), ICQ, AIM, MSN, Yahoo! and Twitter/Identica/Status.net.
+#  .
+#  This package contains a plugin that adds support for the Skype IM network.
+#  You need to download and install the Skype client for this to work.
+# 
+# Package: skyped
+# Architecture: all
+# Depends: ${misc:Depends}, ${shlibs:Depends}, python (>= 2.5), python-gnutls, python-skype (>=0.9.28.7)
+# Recommends: skype
+# Description: Daemon to control Skype remotely
+#  Daemon to control the GUI Skype client. Currently required to control Skype
+#  from the BitlBee IRC2IM gateway. Skyped and Skype can run on a different
+#  host than the BitlBee server, the communication is encrypted.
+#  .
+#  You need to download and install the Skype client for this to work.
