--- checkrestart	2013-07-05 14:37:50.000000000 +0200
+++ checkrestart_	2014-04-22 16:47:50.503469698 +0200
@@ -552,13 +552,14 @@
                             m = re.match("^-", data[0])
                             while (m):
                                 data.pop(0)
+                                if not data: break
                                 m = re.match("^-", data[0])
-                            if data and data[0]: data = data[0]
-                            data = self.which(data)
-                            m = re.search(r'^(/usr/\S+)$', data)
-                            if m:
-                                # store the real full path of script as the program
-                                self.program = m.group(1)
+                            if data and data[0]:
+                                data = self.which(data[0])
+                                m = re.search(r'^(/usr/\S+)$', data)
+                                if m:
+                                    # store the real full path of script as the program
+                                    self.program = m.group(1)
         except OSError, e:
             if e.errno != errno.ENOENT:
                 if self.pid == 1:
