diff -rN -u old-aptitude/src/vscreen/fragment.cc new-aptitude/src/vscreen/fragment.cc
--- old-aptitude/src/vscreen/fragment.cc	2006-02-05 18:50:00.000000000 +0100
+++ new-aptitude/src/vscreen/fragment.cc	2006-02-05 18:50:03.000000000 +0100
@@ -1367,8 +1367,6 @@
 	  arguments[i].F=va_arg(arglst, fragment *);
 	  break;
 	case 's':
-	  arguments[i].s=va_arg(arglst, const char *);
-	  break;
 	case 'S':
 	  arguments[i].s=va_arg(arglst, const char *);
 	  break;

