diff -ruN easytag-1.99.10.orig/src/setting.c easytag-1.99.10/src/setting.c
--- easytag-1.99.10.orig/src/setting.c	2005-11-25 23:23:25.000000000 +0100
+++ easytag-1.99.10/src/setting.c	2005-12-06 23:27:57.579884512 +0100
@@ -132,7 +132,7 @@
     {"ogg_tag_write_xmms_comment",           CV_TYPE_BOOL,    &OGG_TAG_WRITE_XMMS_COMMENT               },
     {"set_focus_to_same_tag_field",          CV_TYPE_BOOL,    &SET_FOCUS_TO_SAME_TAG_FIELD              },
     {"set_focus_to_first_tag_field",         CV_TYPE_BOOL,    &SET_FOCUS_TO_FIRST_TAG_FIELD             },
-    {"sorting_file_mode",                    CV_TYPE_BOOL,    &SORTING_FILE_MODE                        },
+    {"sorting_file_mode",                    CV_TYPE_INT,     &SORTING_FILE_MODE                        },
     {"sorting_file_case_sensitive",          CV_TYPE_BOOL,    &SORTING_FILE_CASE_SENSITIVE              },
 
     {"replace_illegal_character_in_filename",CV_TYPE_BOOL,    &REPLACE_ILLEGAL_CHARACTERS_IN_FILENAME   },
@@ -168,7 +168,7 @@
 
     {"audio_file_player",                       CV_TYPE_STRING,&AUDIO_FILE_PLAYER                       },
 
-    {"scanner_type",                             CV_TYPE_BOOL,&SCANNER_TYPE                             },
+    {"scanner_type",                             CV_TYPE_INT, &SCANNER_TYPE                             },
     {"fts_convert_underscore_and_p20_into_space",CV_TYPE_BOOL,&FTS_CONVERT_UNDERSCORE_AND_P20_INTO_SPACE},
     {"fts_convert_space_into_underscore",        CV_TYPE_BOOL,&FTS_CONVERT_SPACE_INTO_UNDERSCORE        },
     {"rfs_convert_underscore_and_p20_into_space",CV_TYPE_BOOL,&RFS_CONVERT_UNDERSCORE_AND_P20_INTO_SPACE},
