--- dnssec-trigger-0.12.orig/dnssec-trigger-script.in	2014-05-06 23:45:37.000000000 -0700
+++ dnssec-trigger-0.12/dnssec-trigger-script.in	2014-07-01 16:56:33.071052995 -0700
@@ -1,4 +1,4 @@
-#!/usr/bin/python
+#!/usr/bin/python3
 # -*- coding: utf-8 -*-
 """
 @author: Tomas Hozza <thozza@redhat.com>
@@ -190,9 +190,9 @@
                 if fields.pop(0) in ('forward', 'forward:'):
                     fields.pop(0)
                 secure = False
-                if fields[0] == '+i':
-                    secure = True
-                    fields.pop(0)
+                #if fields[0] == '+i':
+                #    secure = True
+                #    fields.pop(0)
                 self.cache[name] = set(fields[3:]), secure
         log.debug(self)
 
