--- dibbler-0.7.1.orig/debian/dibbler-client.postinst	2008-08-11 02:35:02.000000000 +0100
+++ dibbler-0.7.1/debian/dibbler-client.postinst	2008-08-11 02:38:18.000000000 +0100
@@ -94,10 +94,10 @@
     echo "" >> ${CONFFILE_NEW}
 done
 
-db_stop
-
 # register this config
-test -f /usr/bin/ucf && ucf ${CONFFILE_NEW} ${CONFFILE} </dev/tty
+test -x /usr/bin/ucf && ucf ${CONFFILE_NEW} ${CONFFILE}
+
+db_stop
 
 # Start service if necessary
 if [ $START == "true" ]; then
