diff -Naur epiphany-browser-1.0_old/embed/mozilla/mozilla-embed-single.cpp epiphany-browser-1.0/embed/mozilla/mozilla-embed-single.cpp
--- epiphany-browser-1.0_old/embed/mozilla/mozilla-embed-single.cpp	2003-09-21 04:34:28.000000000 +0200
+++ epiphany-browser-1.0/embed/mozilla/mozilla-embed-single.cpp	2003-09-21 04:39:40.000000000 +0200
@@ -328,7 +328,7 @@
 	pref->SetBoolPref ("mozilla.widget.raise-on-setfocus", PR_FALSE);
 
 	/* set default search engine */
-	pref->SetCharPref ("keyword.URL", "http://www.google.com/search?q=");
+	pref->SetCharPref ("keyword.URL", "http://www.google.com/search?ie=UTF-8&oe=UTF-8&q=");
 	pref->SetBoolPref ("keyword.enabled", PR_TRUE);
 	pref->SetBoolPref ("security.checkloaduri", PR_FALSE);
 
