=== modified file 'cmdline/apt-get.cc'
--- cmdline/apt-get.cc	2009-04-14 12:17:40 +0000
+++ cmdline/apt-get.cc	2009-05-19 18:47:39 +0000
@@ -1870,7 +1870,8 @@
    // cache.commit()
    if (AutoMarkChanged > 0 &&
        Cache->DelCount() == 0 && Cache->InstCount() == 0 &&
-       Cache->BadCount() == 0)
+       Cache->BadCount() == 0 &&
+       _config->FindB("APT::Get::Simulate",false) == false)
       Cache->writeStateFile(NULL);
 
    // See if we need to prompt

