diff -Nru apt-setup-0.49/generators/92volatile apt-setup-0.50/generators/92volatile
--- apt-setup-0.49/generators/92volatile	2009-07-23 17:53:14.000000000 +0100
+++ apt-setup-0.50/generators/92volatile	2010-12-02 00:50:03.000000000 +0000
@@ -18,6 +18,15 @@
 	codename="$RET"
 	db_get mirror/suite
 	suite="$RET"
+
+	if [ "$codename" != "lenny" ]; then
+		db_get mirror/protocol
+		protocol="$RET"
+		db_get mirror/$protocol/hostname
+		host="$RET"
+		db_get mirror/$protocol/directory
+		directory="/${RET#/}"
+	fi
 else
 	db_get cdrom/codename
 	codename="$RET"
@@ -35,7 +44,11 @@
 done
 
 # Don't test mirror if no network selected in netcfg
-echo "deb http://$host/debian-volatile $codename/volatile $dists" >> $file
+if [ "$codename" = "lenny" ]; then
+	echo "deb http://$host/debian-volatile $codename/volatile $dists" >> $file
+else
+	echo "deb $protocol://${host}${directory} ${codename}-updates $dists" >> $file
+fi
 if db_get netcfg/dhcp_options && \
    [ "$RET" = "Do not configure the network at this time" ]; then
 	CODE=9
@@ -52,6 +65,10 @@
 	fi
 fi
 
-echo "deb-src http://$host/debian-volatile $codename/volatile $dists" >> $file
+if [ "$codename" = "lenny" ]; then
+	echo "deb-src http://$host/debian-volatile $codename/volatile $dists" >> $file
+else
+	echo "deb-src $protocol://${host}${directory} ${codename}-updates $dists" >> $file
+fi
 
 exit $CODE

