--- /usr/sbin/extlinux-update	2011-11-02 07:57:53.000000000 +0100
+++ /usr/local/sbin/extlinux-update	2011-11-23 22:00:49.103876891 +0100
@@ -182,7 +182,7 @@
 
 label l${_NUMBER}
 	menu label ${EXTLINUX_MENU_LABEL} ${_VERSION}
-	kernel ${_BOOT_DIRECTORY}/vmlinuz-${_VERSION}
+	linux ${_BOOT_DIRECTORY}/vmlinuz-${_VERSION}
 	append ${_INITRD} ${EXTLINUX_ROOT} ${EXTLINUX_PARAMETERS}"
 
 	fi
@@ -195,7 +195,7 @@
 
 label l${_NUMBER}l
 	menu label ${EXTLINUX_MENU_LABEL} ${_VERSION} (live mode)
-	kernel ${_BOOT_DIRECTORY}/vmlinuz-${_VERSION}
+	linux ${_BOOT_DIRECTORY}/vmlinuz-${_VERSION}
 	append ${_INITRD} ${EXTLINUX_ROOT} ${EXTLINUX_PARAMETERS} boot=live plainroot
 	text help
    This option boots the system into live mode (non-persistent)
@@ -211,7 +211,7 @@
 
 label l${_NUMBER}r
 	menu label ${EXTLINUX_MENU_LABEL} ${_VERSION} (recovery mode)
-	kernel ${_BOOT_DIRECTORY}/vmlinuz-${_VERSION}
+	linux ${_BOOT_DIRECTORY}/vmlinuz-${_VERSION}
 	append ${_INITRD} ${EXTLINUX_ROOT} $(echo ${EXTLINUX_PARAMETERS} | sed -e 's| quiet||') single
 	text help
    This option boots the system into recovery mode (single-user)
