=== modified file 'cmdline/apt-cache.cc'
--- cmdline/apt-cache.cc	2010-08-28 15:54:29 +0000
+++ cmdline/apt-cache.cc	2010-09-24 12:18:52 +0000
@@ -1321,7 +1321,7 @@
 	 continue;
       pkgCache::VerIterator V = Plcy->GetCandidateVer(P);
       if (V.end() == false)
-	 DFList[G->ID].Df = V.DescriptionList().FileList();
+	 DFList[G->ID].Df = V.TranslatedDescription().FileList();
 
       if (DFList[G->ID].NameMatch == false)
 	 continue;
@@ -1334,7 +1334,7 @@
 	    continue;
 
 	 unsigned long id = Prv.OwnerPkg().Group()->ID;
-	 DFList[id].Df = V.DescriptionList().FileList();
+	 DFList[id].Df = V.TranslatedDescription().FileList();
 	 DFList[id].NameMatch = true;
       }
    }

=== modified file 'debian/changelog'
--- debian/changelog	2010-09-17 20:22:50 +0000
+++ debian/changelog	2010-09-24 12:18:52 +0000
@@ -1,3 +1,12 @@
+apt (0.8.6) UNRELEASED; urgency=low
+
+  [ David Kalnischkies ]
+  * cmdline/apt-cache.cc:
+    - use the TranslatedDescription for searching and not the first
+      available one as it is maybe not an expected language (Closes: #597925)
+
+ -- David Kalnischkies <kalnischkies@gmail.com>  Fri, 24 Sep 2010 14:11:52 +0200
+
 apt (0.8.5) unstable; urgency=low
 
   [ Manpages translations ]

