--- control.orig	2013-07-29 12:56:57.947908330 +0200
+++ control	2013-07-29 12:57:04.708120642 +0200
@@ -7,7 +7,7 @@
 Maintainer: Jean-Philippe MENGUAL <texou@accelibreinfo.eu>
 Build-Depends: debhelper (>= 9), libreadline-dev, libpcre3-dev, libssl-dev,
  libcurl4-openssl-dev, libmozjs185-dev
-Vcs-Svn: svn://anoncsm.debian.org/collab-maint/deb-maint/edbrowse
+Vcs-Svn: svn://anonscm.debian.org/collab-maint/deb-maint/edbrowse
 Homepage: http://the-brannons.com/edbrowse
 Standards-Version: 3.9.4
 
