diff -uar usr/share/debian-cd.orig/Makefile usr/share/debian-cd/Makefile
--- usr/share/debian-cd.orig/Makefile	2012-02-26 07:18:25.000000000 +0000
+++ usr/share/debian-cd/Makefile	2012-02-26 15:35:07.000000000 +0000
@@ -160,12 +160,11 @@
 # Make sure unstable/sid points to testing/squeeze, as there is no build
 # rule for unstable/sid.
 unstable-map:
-	$(Q)if [ ! -d $(BASEDIR)/data/sid ] ; then \
-		ln -s squeeze $(BASEDIR)/data/sid ; \
-	fi
-	$(Q)if [ ! -d $(BASEDIR)/tools/boot/sid ] ; then \
-		ln -s squeeze $(BASEDIR)/tools/boot/sid ; \
-	fi
+	$(Q)for xd in testing sid; do \
+		test -d $(BASEDIR)/data/$${xd} \
+			|| ln -s squeeze $(BASEDIR)/data/$${xd}; \
+		test -d $(BASEDIR)/tools/boot/$${xd} \
+			|| ln -s squeeze $(BASEDIR)/tools/boot/$${xd}; done
 
 #################
 ## CLEAN RULES ##
