From d9e268bd9e3da473491ee6f3940c514d6c762c81 Mon Sep 17 00:00:00 2001
From: Sean Whitton <spwhitton@spwhitton.name>
Date: Sat, 10 Nov 2018 19:33:16 -0700
Subject: [PATCH 1/2] dgit: pass --no-source-only-changes to pbuilder

Closes: #904862

Signed-off-by: Sean Whitton <spwhitton@spwhitton.name>
---
 debian/control | 2 +-
 dgit           | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/control b/debian/control
index eef4dca..a6e3d72 100644
--- a/debian/control
+++ b/debian/control
@@ -17,7 +17,7 @@ Depends: perl, libwww-perl, libdpkg-perl, git-core, devscripts, dpkg-dev,
          libjson-perl, ca-certificates,
          libtext-iconv-perl, libtext-glob-perl
 Recommends: ssh-client
-Suggests: sbuild | pbuilder | cowbuilder
+Suggests: sbuild | pbuilder (>= 0.230) | cowbuilder
 Architecture: all
 Description: git interoperability with the Debian archive
  dgit (with the associated infrastructure) makes it possible to
diff --git a/dgit b/dgit
index e104838..7faf1c4 100755
--- a/dgit
+++ b/dgit
@@ -6762,7 +6762,7 @@ sub pbuilder ($) {
     # verb.  opts_opt_multi_cmd does all of that.
     binary_builder([opts_opt_multi_cmd ["build"], @$pbuilder], undef,
                    qw(--debbuildopts), "@ARGV", qw(--distribution), $isuite,
-                   $dscfn);
+                   qw(--no-source-only-changes), $dscfn);
 }
 
 sub cmd_pbuilder {
-- 
2.11.0

