From 2d16340cfecb690dc25ad186e18e074d108d8ad4 Mon Sep 17 00:00:00 2001
From: Yaroslav Halchenko <debian@onerussian.com>
Date: Thu, 20 Aug 2015 11:46:31 +0200
Subject: [PATCH 2/2] BF: set PYTHONPATH while running diffoscope while
 building pkg

---
 debian/rules | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/rules b/debian/rules
index 263eeba..35898ca 100755
--- a/debian/rules
+++ b/debian/rules
@@ -13,7 +13,7 @@ override_dh_auto_build:
 	dh_auto_build -O--buildsystem=pybuild
 
 override_dh_gencontrol:
-	echo "diffoscope:Recommends=$$(bin/diffoscope --list-tools | tail -n 1 | \
+	echo "diffoscope:Recommends=$$(PYTHONPATH=$(CURDIR) bin/diffoscope --list-tools | tail -n 1 | \
 		sed -e 's/\(^\| \)\(coreutils\|diffutils\|e2fsprogs\|findutils\|gzip\)\(,\|$$\)//g')" >> debian/diffoscope.substvars
 	dh_gencontrol -O--buildsystem=pybuild
 
-- 
2.4.3.573.g4eafbef

