--- /tmp/CayC8R7eZa/debsums-2.0.34/man/po4a.cfg	2007-11-29 02:15:49.000000000 +0100
+++ /tmp/1MQuDIJ8Gf/debsums-2.0.34.1/man/po4a.cfg	2007-12-03 13:25:28.000000000 +0100
@@ -2,7 +2,7 @@
 [po4a_paths] po/debsums.pot $lang:po/$lang.po
 
 [po4a_alias:man] man opt:"-o groff_code=verbatim -o untranslated=Id" \
-                     opt_fr:"-L ISO-8859-1" \
+                     opt_fr:"-L UTF-8" \
                      opt_pt_BR:"-L ISO-8859-1" \
                      opt_ru:"-L KOI8-R" \
                      opt_sv:"-L ISO-8859-1"
