diff -u dillo-0.8.5/debian/patches/00list dillo-0.8.6/debian/patches/00list
--- dillo-0.8.5/debian/patches/00list
+++ dillo-0.8.6/debian/patches/00list
@@ -1 +1 @@
-01_i18n
+
diff -u dillo-0.8.5/debian/changelog dillo-0.8.6/debian/changelog
--- dillo-0.8.5/debian/changelog
+++ dillo-0.8.6/debian/changelog
@@ -1,3 +1,9 @@
+dillo (0.8.6-1) unstable; urgency=low
+
+  * New upstream release
+
+ -- Don Armstrong <don@debian.org>  Wed, 21 Nov 2007 15:23:20 -0800
+
 dillo (0.8.5-4.1) unstable; urgency=low
 
   * Non-maintainer upload.
diff -u dillo-0.8.5/debian/rules dillo-0.8.6/debian/rules
--- dillo-0.8.5/debian/rules
+++ dillo-0.8.6/debian/rules
@@ -21,7 +21,7 @@
 configure: configure-stamp
 configure-stamp: patch-stamp
 	dh_testdir
-	./configure --prefix=/usr --sysconfdir=/etc --enable-ipv6 --enable-ssl --disable-tabs --enable-meta-refresh
+	./configure --prefix=/usr --sysconfdir=/etc --enable-ipv6 --enable-ssl --disable-dlgui --disable-tabs --enable-meta-refresh
 	touch configure-stamp
 
 build: build-stamp
@@ -36,10 +36,10 @@
 	dh_testroot
 	rm -f build-stamp configure-stamp
 	-$(MAKE) distclean
-	-rm -f `find . -name "*~"`
-	-rm -rf debian/dillo debian/files* core debian/substvars debian/prerm.debhelper debian/*postinst.debhelper debian/*postrm.debhelper
-	-rm -f `find . -name "*.o"` `find . -name "*.dpi"` `find . -name "*.a"`
-	-rm -f `find . -name "*.gmo"` src/dillo dpid/dpid config/dillocfg
+	rm -f `find . -name "*~"`
+	rm -rf debian/dillo debian/files* core debian/substvars debian/prerm.debhelper debian/*postinst.debhelper debian/*postrm.debhelper
+	rm -f `find . -name "*.o"` `find . -name "*.dpi"` `find . -name "*.a"`
+	rm -f `find . -name "*.gmo"` src/dillo dpid/dpid config/dillocfg
 	-$(MAKE) clean
 	dh_clean 
 
