--- passdev.c	2008-12-18 20:16:50.000000000 +0100
+++ passdev.c	2008-12-18 20:18:06.000000000 +0100
@@ -121,7 +121,7 @@
 
 	/* We only take one argument */
 	if (argc != 2) {
-		fprintf(stderr, "Incorrect number of arguments");
+		fprintf(stderr, "Incorrect number of arguments\n");
 		goto error;
 	}
 
@@ -134,7 +134,7 @@
 	devpath = argv[1];
 	filepath = strchr(devpath, ':');
 	if (!filepath || !(*filepath) || !(*(filepath + 1))) {
-		fprintf(stderr, "Invalid key path");
+		fprintf(stderr, "Invalid key path\n");
 		goto error;
 	}
 	*filepath = '\0';
