diff --git a/scripts/perl5/Debian/DictionariesCommon.pm.in b/scripts/perl5/Debian/DictionariesCommon.pm.in
index 48edc11..e3ac624 100644
--- a/scripts/perl5/Debian/DictionariesCommon.pm.in
+++ b/scripts/perl5/Debian/DictionariesCommon.pm.in
@@ -425,8 +425,8 @@ sub dc_merge_installed_hunspell_dicts {
   my $hunspelldir = shift;
   my $main_dicts  = shift;
 
-  die "dmihd: No hunspell dir passed or available\n" unless ( -d $hunspelldir);
   $main_dicts = {} unless $main_dicts;
+  return $main_dicts unless ( -d $hunspelldir);
 
   my @hunspell_aff = <$hunspelldir/*.aff>;
   my $parsed_dicts = {};
