From c6ed90d72e39e1a44f594480efc933c1da25142e Mon Sep 17 00:00:00 2001
From: Matt Kraai <kraai@ftbfs.org>
Date: Mon, 22 Feb 2010 21:09:49 -0800
Subject: [PATCH] Change apt.progress.OpTextProgress to apt.progress.text.OpProgress. Closes #570219

---
 update-apt-xapian-index |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/update-apt-xapian-index b/update-apt-xapian-index
index f39acad..86b3b9a 100755
--- a/update-apt-xapian-index
+++ b/update-apt-xapian-index
@@ -582,7 +582,7 @@ import os.path, re, imp, glob, xapian, textwrap, shutil, fcntl, errno, itertools
 import socket, errno
 import cPickle as pickle
 
-class AptSilentProgress(apt.progress.OpTextProgress) :
+class AptSilentProgress(apt.progress.text.OpProgress) :
     def __init__(self): 
         pass
     def done(self):
-- 
1.7.0

