--- a/dh_autoreconf
+++ b/dh_autoreconf
@@ -75,7 +75,7 @@
 =item I<program> B<--> I<params>
 
 Run the program given by I<program> with the arguments given by I<params>
-instead of autoreconf -f -i. If you need to run multiple commands, put them in
+instead of autoreconf -f -i -v. If you need to run multiple commands, put them in
 a script and pass the script instead (or add a target to debian/rules).
 
 =back
@@ -175,8 +175,10 @@
         if (grep_configure("^XDT_")) {
             $ENV{NOCONFIGURE}='1';
             doit('xdt-autogen', @directories);
-        } else {
+        } elsif (get_buildoption("terse")) {
             doit('autoreconf', '-f', '-i', @directories);
+        } else {
+            doit('autoreconf', '-f', '-i', '-v', @directories);
         }
     }
     1;
