--- a/powerbtn-acpi-support.sh	2013-01-02 19:25:40.739291024 -0500
+++ b/powerbtn-acpi-support.sh	2013-01-02 19:35:50.942314991 -0500
@@ -15,7 +15,7 @@
 
 . /usr/share/acpi-support/policy-funcs
 
-if CheckPolicy; then
+if { CheckPolicy || CheckUPowerPolicy; }; then
 	exit 0
 fi
 
@@ -26,6 +26,9 @@
         # Compatibility with old config script from acpid package
 	# which is still around because it was changed by the admin
         /etc/acpi/powerbtn.sh.dpkg-bak
+elif [ -x /etc/acpi/power.sh ] ; then
+        # Compatibility with another(?) old config script
+        /etc/acpi/power.sh
 else
 	# Normal handling.
 	/sbin/shutdown -h -P now "Power button pressed"
