From ef929769bd67ea3de4ab2850612e9dc1f1ef1631 Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas.boulenguez@free.fr>
Date: Sat, 30 Jul 2016 22:20:49 +0200
Subject: [PATCH 01/12] Switch copyright file to 1.0 machine-readable format.

---
 debian/copyright | 39 ++++++++++++++++++++-------------------
 1 file changed, 20 insertions(+), 19 deletions(-)

diff --git a/debian/copyright b/debian/copyright
index b4c3ced..a5d4639 100644
--- a/debian/copyright
+++ b/debian/copyright
@@ -1,35 +1,36 @@
-This package was debianized by Maitland Bottoms <bottoms@debian.org> on
-Fri,  2 Aug 2002 16:43:20 -0400.
-
-It is now being maintained by Jochen Friedrich <jochen@scram.de> on
-Wed, 20 Aug 2003 22:55:52 +0200
-
-It was downloaded from http://darkice.sourceforge.net/
- DarkIce is being written by Tyrell Hungary, Ákos Maróy, Rafael Diniz
+Format: http://www.debian.org/doc/packaging-manuals/copyright-format/1.0/
+Upstream-Name: DarkIce
+Upstream-Contact: http://www.freelists.org/list/darkice
+Source: http://www.darkice.org/
 
+Files: *
 Copyright:
-
    Copyright (c) 2000-2007, Tyrell Hungary, http://tyrell.hu/
    Copyright (c) 2008, Ákos Maróy, akos@maroy.hu
    Copyright (c) 2009, Rafael Diniz, rafael@riseup.net
- 
-   Tyrell DarkIce
- 
-   Copyright notice:
- 
+License: GPL-2+
+
+Files: debian/*
+Copyright: 2001-2002 Maitland Bottoms <bottoms@debian.org>
+           2003-2011 Jochen Friedrich <jochen@scram.de>
+License: GPL-2+
+
+License: GPL-2+
     This program is free software; you can redistribute it and/or
     modify it under the terms of the GNU General Public License
     as published by the Free Software Foundation; either version 2
     of the License, or (at your option) any later version.
-    
+    .
     This program is distributed in the hope that it will be useful,
     but WITHOUT ANY WARRANTY; without even the implied warranty of
     MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
     GNU General Public License for more details.
-                                                                                
+    .
     You should have received a copy of the GNU General Public License
     along with this program; if not, write to the Free Software
-    Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, 
+    Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston,
     MA 02110-1301, USA.
-
-the GNU GPL may be viewed on Debian systems in /usr/share/common-licenses/GPL
+    .
+    On Debian systems, the full text of the GNU General Public
+    License version 2 can be found in the file
+    `/usr/share/common-licenses/GPL-2'.
-- 
2.8.1

