From 9921c08893e35f3a9df4b3d467107f893444429e Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas.boulenguez@free.fr>
Date: Sat, 30 Jul 2016 22:31:29 +0200
Subject: [PATCH 02/12] Simplify watch file with sourceforge redirector and
 version 4 format.

---
 debian/watch | 10 ++++++----
 1 file changed, 6 insertions(+), 4 deletions(-)

diff --git a/debian/watch b/debian/watch
index dc71667..9e50381 100644
--- a/debian/watch
+++ b/debian/watch
@@ -1,4 +1,6 @@
-version=3
-opts=uversionmangle=s/(\d)[_\.\-\+]?((RC|rc|pre|dev|beta|alpha|b|a)[\-\.]?\d*)$/$1~$2/,\
-downloadurlmangle=s%.*/([^/]*)/downloads/detail\?name=([^=&]*).*%http://$1.googlecode.com/files/$2%,filenamemangle=s/.*name=([^&]+).*/$1/ \
-http://code.google.com/p/darkice/downloads/list?can=1 .*=darkice-(\d.*)\.(?:tgz|tbz2|txz|tar\.(?:gz|bz2|xz)|zip).*
+version=4
+
+# qa.debian.org runs a redirector which allows a simpler form of URL
+# for SourceForge based projects. The format below will automatically
+# be rewritten to use the redirector.
+http://sf.net/darkice/darkice@ANY_VERSION@@ARCHIVE_EXT@
-- 
2.8.1

