From 4991c4f954ded86ed7bf6c1de5261825b09f90ce Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas.boulenguez@free.fr>
Date: Sat, 30 Jul 2016 22:34:00 +0200
Subject: [PATCH 03/12] Update README.Debian: mp3lame has been enabled for a
 while.

---
 debian/README.debian | 5 +----
 1 file changed, 1 insertion(+), 4 deletions(-)

diff --git a/debian/README.debian b/debian/README.debian
index 03d496f..576b26b 100644
--- a/debian/README.debian
+++ b/debian/README.debian
@@ -1,12 +1,9 @@
 DarkIce for Debian
 ----------------------
 
-This package was built with Ogg Vorbis support but without mp3 (lame)
-support for patent reasons.
-
 A sample configuration in installed into /usr/share/doc/darkice/examples.
 In order to use DarkIce, please copy this example to your preferred
 configuration directory, make any required changes for your site and reference
 it using darkice -c <config file>.
 
-Jochen Friedrich <jochen@scram.de>, Wed, 20 Aug 2003 21:02:26 +0200
+ -- Nicolas Boulenguez <nicolas@debian.org>, Fri,  1 Jul 2016 23:59:37 +0200
-- 
2.8.1

