From 78c9f787a148f3afe13038ca3bf28cab0c951661 Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas.boulenguez@free.fr>
Date: Sat, 30 Jul 2016 22:43:47 +0200
Subject: [PATCH 04/12] Remove debian/dirs, automake handles creation of /bin.

---
 debian/dirs | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 debian/dirs

diff --git a/debian/dirs b/debian/dirs
deleted file mode 100644
index e772481..0000000
--- a/debian/dirs
+++ /dev/null
@@ -1 +0,0 @@
-usr/bin
-- 
2.8.1

