From 96cbbdb110e3eb8f8a68e26846325d12e75b09ae Mon Sep 17 00:00:00 2001
From: Nicolas Boulenguez <nicolas.boulenguez@free.fr>
Date: Sun, 31 Jul 2016 00:29:52 +0200
Subject: [PATCH 09/12] Remove --prefix configure option, already set by
 dh_auto_configure.

---
 debian/rules | 1 -
 1 file changed, 1 deletion(-)

diff --git a/debian/rules b/debian/rules
index 9c57cbf..5de5d8b 100755
--- a/debian/rules
+++ b/debian/rules
@@ -31,7 +31,6 @@ override_dh_auto_configure:
 		--with-pulseaudio-prefix=/usr/lib/$(DEB_HOST_MULTIARCH) \
 		--with-samplerate-prefix=/usr/lib/$(DEB_HOST_MULTIARCH) \
 		--with-jack-prefix=/usr/lib/$(DEB_HOST_MULTIARCH) \
-		--prefix=/usr \
 		--sysconfdir=/usr/share/doc/darkice/examples \
 		--without-aacplus \
 		--without-faac
-- 
2.8.1

