diff -u adonthell-0.3.5/debian/patches/series adonthell-0.3.5/debian/patches/series
--- adonthell-0.3.5/debian/patches/series
+++ adonthell-0.3.5/debian/patches/series
@@ -1,0 +2 @@
+10_ftbfs_with_gcc-4.4.diff
only in patch2:
unchanged:
--- adonthell-0.3.5.orig/debian/patches/10_ftbfs_with_gcc-4.4.diff
+++ adonthell-0.3.5/debian/patches/10_ftbfs_with_gcc-4.4.diff
@@ -0,0 +1,25 @@
+Patch backported from http://github.com/ksterker/adonthell/tree/master,
+commit c620028d36510103d15efe15fed11532a4cd4a71
+Index: adonthell-0.3.5/src/dialog.cc
+===================================================================
+--- adonthell-0.3.5.orig/src/dialog.cc	2009-09-12 09:35:25.000000000 +0400
++++ adonthell-0.3.5/src/dialog.cc	2009-09-12 09:49:58.000000000 +0400
+@@ -265,7 +265,8 @@
+ {
+     u_int32 begin, end, len;
+     PyObject *result;
+-    char *tmp, *start, *mid, *str = NULL;
++    char *tmp, *mid, *str = NULL;
++    const char *start;
+     character *the_player = data::the_player;
+     string newstr (s); 
+ 
+@@ -318,7 +319,7 @@
+ 
+         // Error!
+         cout << "\n*** Error, unknown macro " << start << flush;
+-        start[0] = ' ';
++        newstr[newstr.length () - strlen (start)] = ' ';
+     }
+     
+     // execute python functions
