--- debian/arpon.default~	2010-08-18 12:39:17.873711619 +0200
+++ debian/arpon.default	2010-08-18 12:40:32.013462171 +0200
@@ -7,10 +7,10 @@
 # dynamic ARP inspection (DARPI)
 #
 # For SARPI uncomment the following line
-# DAEMON_OPTS="-d -f /var/log/arpon/arpon.log -g -s"
+# DAEMON_OPTS="-q -f /var/log/arpon/arpon.log -g -s"
 
 # For DARPI uncomment the following line
-# DAEMON_OPTS="-d -f /var/log/arpon/arpon.log -g -y"
+# DAEMON_OPTS="-q -f /var/log/arpon/arpon.log -g -d"
 
 # Modify to RUN="yes" when you are ready
 RUN="no"
