--- changelog.Debian.orig	2008-10-15 17:27:08.000000000 +0200
+++ changelog.Debian	2008-10-15 17:28:19.000000000 +0200
@@ -6,7 +6,7 @@
   * E2fsck will offer to clear the test_fs flag if the ext4 filesystem
     is available.
   * Fix a file descriptor leak in libblkid
-  * Avoid a poteintial infinite loop in e2fsck when there are disk I/O
+  * Avoid a potential infinite loop in e2fsck when there are disk I/O
     errors while trying to close a filesystem.
   * Fix a potential infinite loop in resize2fs when a bogus new size of
     0 blocks is specified on the command line.
