From: Santiago Vila <sanvila@debian.org>
Subject: Modify two Makefile.in files to avoid /usr/share/info/dir.gz

--- a/doc/Makefile.in
+++ b/doc/Makefile.in
@@ -590,8 +590,7 @@
 	  done; \
 	done
 	@$(POST_INSTALL)
-	@if (install-info --version && \
-	     install-info --version 2>&1 | sed 1q | grep -i -v debian) >/dev/null 2>&1; then \
+	@if false; then \
 	  list='$(INFO_DEPS)'; \
 	  for file in $$list; do \
 	    relfile=`echo "$$file" | sed 's|^.*/||'`; \
--- a/xslt/documentation/Makefile.in
+++ b/xslt/documentation/Makefile.in
@@ -480,8 +480,7 @@
 	  done; \
 	done
 	@$(POST_INSTALL)
-	@if (install-info --version && \
-	     install-info --version 2>&1 | sed 1q | grep -i -v debian) >/dev/null 2>&1; then \
+	@if false; then \
 	  list='$(INFO_DEPS)'; \
 	  for file in $$list; do \
 	    relfile=`echo "$$file" | sed 's|^.*/||'`; \
