From f6687447d2289cca37c4c1e8592bad6bbefaa381 Mon Sep 17 00:00:00 2001
From: =?utf-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sun, 3 May 2009 18:13:36 +0200
Subject: [PATCH 1/3] update build-deps according to configure.ac

---
 debian/control    |    8 ++++----
 debian/control.in |    8 ++++----
 2 files changed, 8 insertions(+), 8 deletions(-)

diff --git a/debian/control b/debian/control
index 7b6805c..ee00a4b 100644
--- a/debian/control
+++ b/debian/control
@@ -8,13 +8,13 @@ Build-Depends: cdbs,
                intltool (>= 0.35.0),
                gnome-pkg-tools (>= 0.10),
                gnome-doc-utils (>= 0.3.2),
-               libglib2.0-dev (>= 2.15.6),
-               libgtk2.0-dev (>= 2.10.0),
+               libglib2.0-dev (>= 2.18.0),
+               libgtk2.0-dev (>= 2.12.0),
                libglade2-dev,
                libgconf2-dev,
                libnautilus-extension-dev,
-               libgnome-keyring-dev (>= 0.4.0),
-               libdbus-glib-1-dev (>= 0.33),
+               libgnome-keyring-dev (>= 2.22.0),
+               libdbus-glib-1-dev (>= 0.70),
                librsvg2-dev (>> 2.14.4-2),
                gnome-icon-theme (>= 2.17.1),
                libxml-parser-perl,
diff --git a/debian/control.in b/debian/control.in
index 6f23125..ecd3810 100644
--- a/debian/control.in
+++ b/debian/control.in
@@ -8,13 +8,13 @@ Build-Depends: cdbs,
                intltool (>= 0.35.0),
                gnome-pkg-tools (>= 0.10),
                gnome-doc-utils (>= 0.3.2),
-               libglib2.0-dev (>= 2.15.6),
-               libgtk2.0-dev (>= 2.10.0),
+               libglib2.0-dev (>= 2.18.0),
+               libgtk2.0-dev (>= 2.12.0),
                libglade2-dev,
                libgconf2-dev,
                libnautilus-extension-dev,
-               libgnome-keyring-dev (>= 0.4.0),
-               libdbus-glib-1-dev (>= 0.33),
+               libgnome-keyring-dev (>= 2.22.0),
+               libdbus-glib-1-dev (>= 0.70),
                librsvg2-dev (>> 2.14.4-2),
                gnome-icon-theme (>= 2.17.1),
                libxml-parser-perl,
-- 
1.6.2.4

