From a4d17d5cc9d5c7b8daf8fed208b91481900a6717 Mon Sep 17 00:00:00 2001
From: =?utf-8?q?Guido=20G=C3=BCnther?= <agx@sigxcpu.org>
Date: Sun, 3 May 2009 19:10:45 +0200
Subject: [PATCH 2/3] move -dbg package to debug section

---
 debian/control |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/debian/control b/debian/control
index ee00a4b..fca2348 100644
--- a/debian/control
+++ b/debian/control
@@ -55,6 +55,7 @@ Description: Document (postscript, pdf) viewer
 
 Package: evince-dbg
 Priority: extra
+Section: debug
 Architecture: any
 Depends: ${misc:Depends},
          evince (= ${binary:Version})
-- 
1.6.2.4

