From: Jordi Pujol Palomer <jordipujolp@gmail.com>
Subject: apt-get download, strip initial version number and colon
Date: 2016-05-02 08:03:13.887017907 +0200

--- apt-1.2.11-a/apt-pkg/acquire-item.cc
+++ apt-1.2.11-b/apt-pkg/acquire-item.cc	2016-05-02 08:03:13.887017907 +0200
@@ -2896,8 +2896,13 @@ pkgAcqArchive::pkgAcqArchive(pkgAcquire
 	 return;
             
       // Generate the final file name as: package_version_arch.foo
+      const char *V;
+      if (V = strchr((const char *)Version.VerStr(), ':'))
+            V++;
+      else
+            V = Version.VerStr();
       StoreFilename = QuoteString(Version.ParentPkg().Name(),"_:") + '_' +
-	              QuoteString(Version.VerStr(),"_:") + '_' +
+	              QuoteString(V,"_:") + '_' +
      	              QuoteString(Version.Arch(),"_:.") + 
 	              "." + flExtension(Parse.FileName());
    }
