From: Jordi Pujol Palomer <jordipujolp@gmail.com>
Subject: apt-get autoclean, strip initial version number and colon
Date: 2016-04-28 16:03:29.613722454 +0200

--- apt-1.2.11-a/apt-pkg/clean.cc
+++ apt-1.2.11-b/apt-pkg/clean.cc	2016-04-28 16:03:29.613722454 +0200
@@ -119,8 +119,14 @@ bool pkgArchiveCleaner::Go(std::string D
 	    }
 	    
 	    // See if this version matches the file
-	    if (IsFetchable == true && Ver == V.VerStr())
-	       break;
+	    if (IsFetchable == true) {
+			if (Start = strchr((const char *)V.VerStr(), ':'))
+				Start++;
+			else
+				Start = V.VerStr();
+			if (strcmp(Ver.c_str(), Start) == 0)
+				break;
+		}
 	 }
 	 
 	 // We found a match, keep the file
