diff -ru bash-4.2+dfsg.pristine/debian/control bash-4.2+dfsg/debian/control
--- bash-4.2+dfsg.pristine/debian/control	2013-05-11 12:53:42.000000000 +0100
+++ bash-4.2+dfsg/debian/control	2013-05-11 12:55:37.153645766 +0100
@@ -63,7 +63,7 @@
 Section: doc
 Priority: optional
 Replaces: bash (<< 3.2-1)
-Description: Documentation and examples for the The GNU Bourne Again SHell
+Description: GNU Bourne Again SHell - documentation and examples
  Bash is an sh-compatible command language interpreter that executes
  commands read from the standard input or from a file.  Bash also
  incorporates useful features from the Korn and C shells (ksh and csh).
