diff -Nru autoconf-archive-20150224/debian/changelog autoconf-archive-20150224/debian/changelog
--- autoconf-archive-20150224/debian/changelog	2015-07-06 22:54:30.000000000 +0100
+++ autoconf-archive-20150224/debian/changelog	2015-07-10 13:04:49.000000000 +0100
@@ -1,3 +1,12 @@
+autoconf-archive (20150224-2.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Add Breaks and Replaces on gnome-common (<< 3.14.0-2~) due to
+    autoconf-archive and gnome-common both shipping the same .m4 files.
+    (Closes: #790937)
+
+ -- Iain Lane <iain@orangesquash.org.uk>  Fri, 10 Jul 2015 13:04:49 +0100
+
 autoconf-archive (20150224-2) unstable; urgency=medium
 
   * Acknowledge NMU.
diff -Nru autoconf-archive-20150224/debian/control autoconf-archive-20150224/debian/control
--- autoconf-archive-20150224/debian/control	2015-07-03 17:50:40.000000000 +0100
+++ autoconf-archive-20150224/debian/control	2015-07-10 13:04:45.000000000 +0100
@@ -14,9 +14,9 @@
 Pre-Depends: dpkg (>= 1.17.5)
 Depends: ${misc:Depends}
 Recommends: autoconf
-Replaces: autoconf-gl-macros (<< 20130609-1~)
+Replaces: autoconf-gl-macros (<< 20130609-1~), gnome-common (<< 3.14.0-2~)
 Enhances: autoconf
-Breaks: autoconf-gl-macros (<< 20130609-1~)
+Breaks: autoconf-gl-macros (<< 20130609-1~), gnome-common (<< 3.14.0-2~)
 Description: Autoconf Macro Archive
  Autoconf Macro Archive aims to provide a central repository of
  useful and tested Autoconf macros for software developers around the
@@ -29,4 +29,4 @@
 Depends: ${misc:Depends}, autoconf-archive
 Section: oldlibs
 Description: Autoconf OpenGL Macro Archive -- transitional dummy package
- This is a transitional dummy package. It can safely be removed.
\ No newline at end of file
+ This is a transitional dummy package. It can safely be removed.
