--- a/debian/apt-cacher-ng.logrotate
+++ b/debian/apt-cacher-ng.logrotate
@@ -9,7 +9,7 @@
         create
         postrotate
                 if [ -e /run/systemd/system ] ; then
-                        systemctl is-active apt-cacher-ng && systemctl kill --signal=USR1 apt-cacher-ng
+                        systemctl -q is-active apt-cacher-ng && systemctl kill --signal=USR1 apt-cacher-ng
                 elif [ -s /var/run/apt-cacher-ng/pid ] ; then
                         kill -s USR1 "$(cat /var/run/apt-cacher-ng/pid)"
                 fi
@@ -27,7 +27,7 @@
         create
         postrotate
                 if [ -e /run/systemd/system ] ; then
-                        systemctl is-active apt-cacher-ng && systemctl kill --signal=USR1 apt-cacher-ng
+                        systemctl -q is-active apt-cacher-ng && systemctl kill --signal=USR1 apt-cacher-ng
                 elif [ -s /var/run/apt-cacher-ng/pid ] ; then
                         kill -s USR1 "$(cat /var/run/apt-cacher-ng/pid)"
                 fi
