diff -Nur kdeutils-3.5.3/ark/sevenzip.cpp kdeutils-3.5.3.modif/ark/sevenzip.cpp
--- kdeutils-3.5.3/ark/sevenzip.cpp	2006-05-22 20:08:38.000000000 +0200
+++ kdeutils-3.5.3.modif/ark/sevenzip.cpp	2006-08-14 12:10:10.000000000 +0200
@@ -44,10 +44,10 @@
   : Arch( gui, filename ), m_nameColumnPos( -1 )
 {
   // Check if 7z is available
-  bool have_7z = !KGlobal::dirs()->findExe( "7z" ).isNull();
+  bool have_7z = !KGlobal::dirs()->findExe( "7zr" ).isNull();
 
   if ( have_7z )
-    m_archiver_program = m_unarchiver_program = "7z";  // Use 7z
+    m_archiver_program = m_unarchiver_program = "7zr";  // Use 7z
   else
     m_archiver_program = m_unarchiver_program = "7za"; // Try 7za
 
