--- /usr/bin/licensecheck	2011-11-12 17:24:28.000000000 +0100
+++ /home/lrineau/CGAL/working_copy/Scripts/developer_scripts/licensecheck	2012-02-09 12:23:00.036607368 +0100
@@ -410,7 +410,7 @@
 	$license = "GENERATED FILE";
     }
 
-    if ($licensetext =~ /is free software.? you can redistribute it and\/or modify it under the terms of the (GNU (Library|Lesser) General Public License|LGPL)/i) {
+    if ($licensetext =~ /(is free software.? )?[Yy]ou can redistribute (it|them) and\/or +modify (it|them) under the terms of the (GNU (Library|Lesser) General Public License|LGPL)/i) {
 	$license = "LGPL$gplver$extrainfo $license";
     }
 
@@ -418,7 +418,7 @@
 	$license = "AGPL$gplver$extrainfo $license";
     }
 
-    if ($licensetext =~ /is free software.? you (can|may) redistribute it and\/or modify it under the terms of (?:version [^ ]+ (?:\(?only\)? )?of )?the GNU General Public License/i) {
+    if ($licensetext =~ /(is free software.? )?you (can|may) redistribute it and\/or modify it under the terms of (?:version [^ ]+ (?:\(?only\)? )?of )?the GNU General Public License/i) {
 	$license = "GPL$gplver$extrainfo $license";
     }
 
@@ -433,7 +433,7 @@
 
     if ($licensetext =~ /This file is part of the .*Qt GUI Toolkit. This file may be distributed under the terms of the Q Public License as defined/) {
 	$license = "QPL (part of Qt) $license";
-    } elsif ($licensetext =~ /may be distributed under the terms of the Q Public License as defined/) {
+    } elsif ($licensetext =~ /may (be distributed|redistribute it) under the terms of the Q Public License/) {
 	$license = "QPL $license";
     }
 
