diff -ru dosfstools-3.0.4/src/check.c dosfstools-3.0.4.new/src/check.c
--- dosfstools-3.0.4/src/check.c	2009-07-21 07:58:00.000000000 +0200
+++ dosfstools-3.0.4.new/src/check.c	2009-07-26 21:19:08.459704042 +0200
@@ -254,7 +254,7 @@
 {
     int i, spc, suspicious = 0;
     char *bad_chars = atari_format ? "*?\\/:" : "*?<>|\"\\/:";
-    char *name = file->dir_ent.name;
+    unsigned char *name = file->dir_ent.name;
 
     /* Do not complain about (and auto-correct) the extended attribute files
      * of OS/2. */
