--- a/gtk/gnome/Makefile.am
+++ b/gtk/gnome/Makefile.am
@@ -2,7 +2,8 @@
 
 if USE_GNOME
 libcompiz_la_LDFLAGS = -export-dynamic -avoid-version -rpath $(moduledir)
-libcompiz_la_LIBADD  = @GNOME_WINDOW_SETTINGS_LIBS@
+libcompiz_la_LIBADD  = @GNOME_WINDOW_SETTINGS_LIBS@ @GCONF_LIBS@
+libcompiz_la_CFLAGS  = @GCONF_CFLAGS@
 libcompiz_la_SOURCES = 		\
 	compiz-window-manager.c	\
 	compiz-window-manager.h
