diff --git a/1/class/python-distutils.mk.in b/1/class/python-distutils.mk.in
index 267afaf..8ba232d 100644
--- a/1/class/python-distutils.mk.in
+++ b/1/class/python-distutils.mk.in
@@ -26,7 +26,7 @@ include $(_cdbs_class_path)/python-module.mk$(_cdbs_makefile_suffix)
 
 DEB_PYTHON_SETUP_CMD ?= setup.py
 DEB_PYTHON_CLEAN_ARGS ?= -a
-DEB_PYTHON_BUILD_ARGS ?= --build-base="$(CURDIR)/$(DEB_BUILDDIR)/build"
+DEB_PYTHON_BUILD_ARGS ?= --build-base="$(CURDIR)/$(DEB_BUILDDIR)/build" $(if $(cdbs_python_nondefault_version),,--executable=/usr/bin/python)
 DEB_PYTHON_INSTALL_ARGS_ALL ?= --prefix=/usr --no-compile -O0
 
 # DEB_PYTHON_MODULE_PACKAGE is deprecated.
