--- /usr/sbin/luksformat.orig	2011-01-14 00:01:00.000000000 +0100
+++ /usr/sbin/luksformat	2011-02-13 14:23:19.000000000 +0100
@@ -10,7 +10,7 @@
 
 use Getopt::Long qw(:config pass_through);
 
-sub help() {
+sub usage() {
     print "luksformat - Create and format an encrypted LUKS device
 Usage: luksformat [-t <file system>] <device> [ mkfs options ]\n";
     exit 1;
@@ -20,7 +20,10 @@
 $fs = 'vfat';
 exit 1 unless GetOptions ('t|type=s' => \$fs);
 
-help() if $#ARGV < 0;
+GetOptions ('help', \$help);
+if (($#ARGV < 0) || ($help)) {
+    usage();
+}
 
 if ($> != 0) {
     print STDERR "This program needs to be started as root\n";
