From b1561363f086cfdab837b74fecad4bb2c045917d Mon Sep 17 00:00:00 2001
From: Simon Josefsson <simon@josefsson.org>
Date: Tue, 17 Nov 2020 19:21:19 +0100
Subject: [PATCH] B-D on libidn2-dev instead of obsolete libidn2-0-dev.

---
 debian/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index cd69a7be..1cb19fc9 100644
--- a/debian/control
+++ b/debian/control
@@ -11,7 +11,7 @@ Build-Depends: debhelper (>= 12),
  groff-base,
  libbrotli-dev,
  libgnutls28-dev,
- libidn2-0-dev,
+ libidn2-dev,
  libkrb5-dev,
  libldap2-dev,
  libnghttp2-dev,
-- 
2.20.1

