--- Packages-arch-specific.orig	2009-06-26 12:25:51.000000000 -0700
+++ Packages-arch-specific	2009-06-26 12:26:21.000000000 -0700
@@ -297,7 +297,7 @@
 qprof: ia64 amd64 i386 mips mipsel powerpc			      # Build-Depends on libpfm3-3.2
 quik: powerpc							      # powerpc boot loader
 quik-installer: powerpc						      # powerpc boot loader
-%read-edid: i386 powerpc lpia 					      # ioperm()/iopl(), vm86() etc. i386 specific
+%read-edid: i386 amd64 powerpc lpia 					      # ioperm()/iopl(), libx86 etc. x86 specific
 %ree: i386 ia64 amd64 hurd-i386 kfreebsd-i386 kfreebsd-amd64          # i386 specific
 %refit: i386							      # EFI bootloader
 replicator: i386						      # i386 specific
