From 1448b390495c930da6753b6a4e8cbff59bca5922 Mon Sep 17 00:00:00 2001
From: Guillem Jover <guillem@debian.org>
Date: Fri, 3 Jul 2015 22:40:57 +0200
Subject: [PATCH] dh_strip: Fix typo

---
 dh_strip | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/dh_strip b/dh_strip
index 91197ab..ea86300 100755
--- a/dh_strip
+++ b/dh_strip
@@ -209,7 +209,7 @@ sub testfile {
 	
 	# Is it a static library, and not a debug library?
 	if (m/lib.*\.a$/ && ! m/.*_g\.a$/) {
-		# Is it a binary file, or something else (maybe a liner
+		# Is it a binary file, or something else (maybe a linker
 		# script on Hurd, for example? I don't use file, because
 		# file returns a variety of things on static libraries.
 		if (-B $_) {
-- 
2.4.3.573.g4eafbef

