From 61091f445090f2fb5d5bc761f34e74b3b734a7ed Mon Sep 17 00:00:00 2001
From: Nobuhiro Iwamatsu <iwamatsu@nigauri.org>
Date: Thu, 9 Jun 2011 22:51:45 +0900
Subject: [PATCH] Fix typo from deamon to daemon

---
 src/log.c     |    2 +-
 tracer/main.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/log.c b/src/log.c
index 6fbbba9..2c492e9 100644
--- a/src/log.c
+++ b/src/log.c
@@ -111,7 +111,7 @@ void __btd_log_init(const char *debug, int detach)
 
 	openlog("bluetoothd", option, LOG_DAEMON);
 
-	syslog(LOG_INFO, "Bluetooth deamon %s", VERSION);
+	syslog(LOG_INFO, "Bluetooth daemon %s", VERSION);
 }
 
 void __btd_log_cleanup(void)
diff --git a/tracer/main.c b/tracer/main.c
index 0806ffe..3b37f59 100644
--- a/tracer/main.c
+++ b/tracer/main.c
@@ -118,7 +118,7 @@ int main(int argc, char *argv[])
 
 	openlog("hcitrace", LOG_PID | LOG_NDELAY | LOG_PERROR, LOG_DAEMON);
 
-	syslog(LOG_INFO, "HCI trace deamon %s", VERSION);
+	syslog(LOG_INFO, "HCI trace daemon %s", VERSION);
 
 	memset(&sa, 0, sizeof(sa));
 	sa.sa_flags = SA_NOCLDSTOP;
-- 
1.7.5.3

