diff -u cups-1.4.1/debian/changelog cups-1.4.1/debian/changelog
--- cups-1.4.1/debian/changelog
+++ cups-1.4.1/debian/changelog
@@ -1,3 +1,10 @@
+cups (1.4.1-4.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * 
+
+ -- Sune Vuorela <debian@pusling.com>  Tue, 06 Oct 2009 09:02:59 +0000
+
 cups (1.4.1-4) unstable; urgency=low
 
   [ Till Kamppeter ]
only in patch2:
unchanged:
--- cups-1.4.1.orig/ppdc/Makefile
+++ cups-1.4.1/ppdc/Makefile
@@ -237,14 +237,14 @@
 # genstrings - generate GNU gettext strings.
 #
 
-genstrings:		genstrings.o libcupsppdc.a ../cups/libcups.a \
+genstrings:		genstrings.o libcupsppdc.so ../cups/libcups.a \
 			sample.drv ../data/media.defs
 	echo Linking $@...
 	$(CXX) $(ARCHFLAGS) $(LDFLAGS) -o genstrings genstrings.o \
-		libcupsppdc.a ../cups/libcups.a $(LIBGSSAPI) $(SSLLIBS) \
+		-lcupsppdc ../cups/libcups.a $(LIBGSSAPI) $(SSLLIBS) \
 		$(DNSSDLIBS) $(COMMONLIBS) $(LIBZ)
 	echo Generating localization strings...
-	./genstrings >sample.c
+	LD_LIBRARY_PATH=.:../cups/ ./genstrings >sample.c
 
 
 #
@@ -258,12 +258,12 @@
 
 ppdc-static:		ppdc.o libcupsppdc.a  ../cups/libcups.a foo.drv foo-fr.po
 	echo Linking $@...
-	$(CXX) $(ARCHFLAGS) $(LDFLAGS) -o ppdc-static ppdc.o libcupsppdc.a \
+	$(CXX) $(ARCHFLAGS) $(LDFLAGS) -o ppdc-static ppdc.o -lcupsppdc \
 		../cups/libcups.a $(LIBGSSAPI) $(SSLLIBS) $(DNSSDLIBS) \
 		$(COMMONLIBS) $(LIBZ)
 	echo Testing PPD compiler...
-	./ppdc-static -l en,fr -I ../data foo.drv
-	./ppdc-static -l en,fr -z -I ../data foo.drv
+	LD_LIBRARY_PATH=.:../cups/ ./ppdc-static -l en,fr -I ../data foo.drv
+	LD_LIBRARY_PATH=.:../cups/ ./ppdc-static -l en,fr -z -I ../data foo.drv
 
 
 #
@@ -286,14 +286,14 @@
 
 ppdi-static:		ppdc-static ppdi.o libcupsppdc.a  ../cups/libcups.a
 	echo Linking $@...
-	$(CXX) $(ARCHFLAGS) $(LDFLAGS) -o ppdi-static ppdi.o libcupsppdc.a \
+	$(CXX) $(ARCHFLAGS) $(LDFLAGS) -o ppdi-static ppdi.o -lcupsppdc \
 		../cups/libcups.a $(LIBGSSAPI) $(SSLLIBS) $(DNSSDLIBS) \
 		$(COMMONLIBS) $(LIBZ)
 	echo Testing PPD importer...
 	$(RM) -r ppd ppd2 sample-import.drv
-	./ppdc-static -I ../data sample.drv
-	./ppdi-static -I ../data -o sample-import.drv ppd/*
-	./ppdc-static -I ../data -d ppd2 sample-import.drv
+	LD_LIBRARY_PATH=.:../cups/ ./ppdc-static -I ../data sample.drv
+	LD_LIBRARY_PATH=.:../cups/ ./ppdi-static -I ../data -o sample-import.drv ppd/*
+	LD_LIBRARY_PATH=.:../cups/ ./ppdc-static -I ../data -d ppd2 sample-import.drv
 	if diff -qr ppd ppd2; then \
 		echo PPD import OK; \
 	else \
@@ -326,7 +326,7 @@
 
 testcatalog:		testcatalog.o libcupsppdc.a ../cups/libcups.a
 	echo Linking $@...
-	$(CXX) $(LDFLAGS) -o $@ testcatalog.o libcupsppdc.a \
+	$(CXX) $(LDFLAGS) -o $@ testcatalog.o -lcupsppdc \
 		../cups/libcups.a $(LIBGSSAPI) $(SSLLIBS) $(DNSSDLIBS) \
 		$(COMMONLIBS) $(LIBZ)
 
