--- html2xml.py	2004-04-20 19:55:14.000000000 +0100
+++ html2xml.py	2004-06-29 14:33:56.000000000 +0100
@@ -131,17 +131,17 @@
 print '<chapters>'
 for chap in dict['order']:
     print '  <sub name="%s" link="%s">' % (chap, dict[chap]['link'])
-    if not dict[chap].has_key ('order'):
-	continue
-    for sub in dict[chap]['order']:
-	if not does_dict_have_keys (dict[chap][sub], ['link']):
-	    print '    <sub name="%s" link="%s">' % (sub, dict[chap][sub]['link'])
+    if dict[chap].has_key ('order'):
+        for sub in dict[chap]['order']:
+            if not does_dict_have_keys (dict[chap][sub], ['link']):
+                print '    <sub name="%s" link="%s">' % (sub, dict[chap][sub]['link'])
 	    
-	    for sub2 in dict[chap][sub]['order']:
-		print '      <sub name="%s" link="%s"/>' % (sub2, dict[chap][sub][sub2]['link'])
-            print '    </sub>'
-	else:
-	    print '    <sub name="%s" link="%s"/>' % (sub, dict[chap][sub]['link'])	    
+                for sub2 in dict[chap][sub]['order']:
+                    print '      <sub name="%s" link="%s"/>' % (sub2, dict[chap][sub][sub2]['link'])
+                print '    </sub>'
+            else:
+                print '    <sub name="%s" link="%s"/>' % (sub, dict[chap][sub]['link'])
+                    
     print '  </sub>'
     print
     
