From 574d5557c1aa1143cc1fa1fb0f639984804dd3a2 Mon Sep 17 00:00:00 2001
From: Stefan Kisdaroczi <skisda@gmail.com>
Date: Thu, 4 Aug 2011 18:28:12 +0200
Subject: [PATCH 1/2] minor documentation fixes

---
 debmirror |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/debmirror b/debmirror
index aaf0c41..73c52ae 100755
--- a/debmirror
+++ b/debmirror
@@ -125,7 +125,7 @@ Specifies the http proxy (like Squid) to use for http or ftp methods.
 
 =item B<-d>, B<--dist>=I<foo[,bar,..]>
 
-Specify the distribution (etch, lenny, squeeze, sid) of Debian to
+Specify the distribution (lenny, squeeze, wheezy, sid) of Debian to
 mirror. This switch may be used multiple times, and multiple
 distributions may be specified at once, separated by commas. 
 
@@ -476,9 +476,9 @@ settings defined in F<debmirror.conf>):
   debmirror /srv/mirror/debian
 
 Make a mirror of i386 and sparc binaries, main only, and include both unstable
-and testing versions of Debian; download from 'ftp.kernel.org':
+and testing versions of Debian; download from 'ftp.nl.debian.org':
 
-  debmirror -a i386,sparc -d sid -d etch -s main --nosource \
+  debmirror -a i386,sparc -d sid -d wheezy -s main --nosource \
             -h ftp.nl.debian.org --progress $HOME/mirror/debian
 
 Make a mirror using rsync (rsync server is 'ftp.debian.org::debian'),
-- 
1.7.2.5

