From fc6b7fe25cc0014f8c6f14dd3498b2ee5271c023 Mon Sep 17 00:00:00 2001
From: Stefan Kisdaroczi <skisda@gmail.com>
Date: Thu, 4 Aug 2011 19:26:18 +0200
Subject: [PATCH 2/2] skip d-i images for suites without d-i

---
 debmirror |    4 ++++
 1 files changed, 4 insertions(+), 0 deletions(-)

diff --git a/debmirror b/debmirror
index 73c52ae..1f77eb6 100755
--- a/debmirror
+++ b/debmirror
@@ -2428,6 +2428,10 @@ sub cleanup_unknown_files {
 sub di_check_dists {
   DI_DIST:
   for my $di_dist (@di_dists) {
+    # some dists don't have d-i
+    next if $di_dist eq "woody";
+    next if $di_dist eq "experimental";
+    next if $di_dist =~ /.*-updates/;
     if (exists $distset{$di_dist}) {
       # Valid dist and also mirroring the archive itself
       $distset{$di_dist}{"d-i"} = 1;
-- 
1.7.2.5

