diff -aur bsdmainutils-8.2.3.harden/debian/rules bsdmainutils-8.2.3/debian/rules
--- bsdmainutils-8.2.3.harden/debian/rules	2011-01-26 12:43:29.000000000 +0100
+++ bsdmainutils-8.2.3/debian/rules	2012-01-15 22:03:27.000000000 +0100
@@ -5,16 +5,15 @@
 
 include /usr/share/quilt/quilt.make
 
-# support noopt; nostrip is handled dh_strip
-CFLAGS = -g
-ifeq (,$(findstring noopt,$(DEB_BUILD_OPTIONS)))
-	CFLAGS += -O2
-endif
-export CFLAGS
-
 DEB_HOST_GNU_TYPE=$(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)
 DEB_BUILD_GNU_TYPE=$(shell dpkg-architecture -qDEB_BUILD_GNU_TYPE)
 
+CFLAGS = `dpkg-buildflags --get CFLAGS`
+CFLAGS += `dpkg-buildflags --get CPPFLAGS`
+LDFLAGS = `dpkg-buildflags --get LDFLAGS`
+export CFLAGS
+export LDFLAGS
+
 ifneq ($(DEB_HOST_GNU_TYPE),$(DEB_BUILD_GNU_TYPE))
 	CROSS += "CC=$(DEB_HOST_GNU_TYPE)-gcc"
 else
Nur in bsdmainutils-8.2.3/debian: rules~.
