diff --git a/debian/desktop-file-validate b/debian/desktop-file-validate
index 8034dce..9ec20eb 100644
--- a/debian/desktop-file-validate
+++ b/debian/desktop-file-validate
@@ -9,4 +9,4 @@ _desktop_file_validate()
 	cur=${COMP_WORDS[COMP_CWORD]}
 	_filedir '@(desktop)'
 }
-[ "${have:-}" ] && complete -F _desktop_file_validate $filenames desktop-file-validate
+[ "${have:-}" ] && complete -F _desktop_file_validate ${filenames:-} desktop-file-validate
