--- dump-0.4b43/restore/tape.c	2010-06-11 13:19:17.000000000 +0200
+++ dump-0.4b43-ema/restore/tape.c	2010-11-01 19:32:11.952668986 +0100
@@ -937,6 +937,21 @@
 			warn("%s: lchown", name);
 #endif
 		extractattr(name);
+
+#if defined(__linux__)
+        {
+    	struct timespec times[2];
+    
+	    times[0].tv_sec  =  timep[0].tv_sec;
+	    times[0].tv_nsec =  timep[0].tv_usec/1000;
+	    times[1].tv_sec  =  timep[1].tv_sec;
+	    times[1].tv_nsec =  timep[1].tv_usec/1000;
+
+        if (utimensat(AT_FDCWD, name, times, AT_SYMLINK_NOFOLLOW) <0)
+			warn("%s: utimensat", name);
+        }
+#endif
+
 		return (GOOD);
 	}
 
