--- abiword-2.6.4.orig/abiword/src/wp/ap/unix/ap_UnixDialog_Styles.cpp	2009-02-10 22:04:53.000000000 +0100
+++ abiword-2.6.4/abiword/src/wp/ap/unix/ap_UnixDialog_Styles.cpp	2009-02-10 22:04:56.000000000 +0100
@@ -661,7 +661,7 @@
 		// select first
 		GtkTreePath *path = gtk_tree_path_new_from_string("0");
 		gtk_tree_selection_select_path(selection, path);
-		g_free(path);
+		gtk_tree_path_free(path);
 	}
 	
 	// selection "changed" doesn't fire here, so hack manually
