diff -u backuppc-3.2.0/debian/control backuppc-3.2.0/debian/control
--- backuppc-3.2.0/debian/control
+++ backuppc-3.2.0/debian/control
@@ -8,7 +8,7 @@
 
 Package: backuppc
 Architecture: all
-Depends: ${perl:Depends}, libdigest-md5-perl, libcompress-zlib-perl, libarchive-zip-perl, tar (>> 1.13), adduser (>=3.9), dpkg (>=1.8.3), apache2 | httpd, perl-suid, ${misc:Depends}, smbclient, samba-common-bin, bzip2, default-mta | exim4 | mail-transport-agent, iputils-ping, ucf, libtime-modules-perl
+Depends: ${perl:Depends}, libdigest-md5-perl, libcompress-zlib-perl, libarchive-zip-perl, tar (>> 1.13), adduser (>=3.9), dpkg (>=1.8.3), apache2 | httpd, perl-suid, ${misc:Depends}, smbclient, samba-common-bin, bzip2, default-mta | exim4 | mail-transport-agent, iputils-ping, ucf, libtime-modules-perl, libwww-perl
 Recommends: rsync, libfile-rsyncp-perl (>= 0.68), openssh-client | ssh-client, rrdtool, libio-dirent-perl
 Suggests: w3m | www-browser, par2
 Conflicts: libfile-rsyncp-perl (<< 0.68)
diff -u backuppc-3.2.0/debian/changelog backuppc-3.2.0/debian/changelog
--- backuppc-3.2.0/debian/changelog
+++ backuppc-3.2.0/debian/changelog
@@ -1,3 +1,10 @@
+backuppc (3.2.0-1.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Depend on libwww-perl. (Closes: #602787)
+
+ -- Adrian Friedli <adi@koalatux.ch>  Sat, 27 Nov 2010 20:41:04 +0100
+
 backuppc (3.2.0-1) unstable; urgency=low
 
   * New upstream release
