--- debian-bug.el.ORG	2005-11-12 19:09:43.000000000 +0100
+++ debian-bug.el	2005-11-12 19:10:24.000000000 +0100
@@ -642,8 +642,11 @@
           (set-buffer "*mail*"))   ; Bug in emacs21.1?  Moves to " *nntpd*"
       (goto-char (point-min))
       (when (re-search-forward "cc:" nil t)
-        (delete-region (match-beginning 0)(match-end 0))
-        (insert "X-Debbugs-CC:"))
+	(goto-char (match-beginning 0))
+	(if (not (equal "Gcc" (word-at-point)))
+	    (progn
+	      (delete-region (match-beginning 0)(match-end 0))
+	      (insert "X-Debbugs-CC:"))))
       (goto-char (point-min))
       (cond
        ((re-search-forward "To: " nil t)
