diff -dur scowl-2020.12.07/debian/rules scowl-2020.12.07-f/debian/rules
--- scowl-2020.12.07/debian/rules	2022-01-20 16:14:35.000000000 +1100
+++ scowl-2020.12.07-f/debian/rules	2023-07-03 11:14:50.909568820 +1000
@@ -55,7 +55,7 @@
 				echo "  $${word_list}" >> w$$SPELLING$$SIZE.scowl-word-lists-used;\
 			fi;\
 		done; \
-	  cat $$SPELLING-english$$SIZE.unsorted | LC_ALL=C sort -d | uniq | iconv -f 'iso8859-1' -t 'utf-8' > $$SPELLING-english$$SIZE; rm $$SPELLING-english$$SIZE.unsorted;\
+	  cat $$SPELLING-english$$SIZE.unsorted | LC_ALL=C sort -d -f | uniq | iconv -f 'iso8859-1' -t 'utf-8' > $$SPELLING-english$$SIZE; rm $$SPELLING-english$$SIZE.unsorted;\
 	  done;\
 	done
 	cd speller && $(MAKE) hunspell
