diff -r -U3 byobu-2.80/debian/changelog byobu-2.80-mine/debian//changelog
--- byobu-2.80/debian/changelog	2011-10-02 14:51:18.000000000 -0400
+++ byobu-2.80-mine/debian//changelog	2011-10-02 14:46:06.000000000 -0400
@@ -1,3 +1,9 @@
+byobu (2.80-2) unstable; urgency=low
+
+  * Correct postinst chmod semantics.
+
+ -- Alexander Chernyakhovsky <achernya@mit.edu>  Sun, 02 Oct 2011 14:45:34 -0400
+
 byobu (2.80-1) unstable; urgency=low
 
   * Upload to Debian unstable.
diff -r -U3 byobu-2.80/debian/postinst byobu-2.80-mine/debian//postinst
--- byobu-2.80/debian/postinst	2011-10-02 14:51:18.000000000 -0400
+++ byobu-2.80-mine/debian//postinst	2011-10-02 14:34:04.000000000 -0400
@@ -17,14 +17,16 @@
 [ -e "/usr/bin/screen.real" ] && [ ! -e "/usr/bin/screen" ] && cp -a /usr/bin/screen.real /usr/bin/screen
 
 # Notify users that they should reload their profile
+touch_flag() {
+	touch "$1"
+	chown --reference $(dirname "$1") "$1" || true
+	chmod 700 "$1"
+}
 DIR="/var/run/screen"
 if [ -d "$DIR" ]; then
-	for d in "$DIR"/*; do
+	for d in "$DIR"/S-*; do
 		[ -d "$d" ] || continue
-		touch "$d/$PKG.reload-required"
-		u=$(echo "$d" | sed "s:^.*/S-::")
-		chown $u "$d/$PKG.reload-required"
-		chmod 700 "$d/$PKG.reload-required"
+		touch_flag "$d/$PKG.reload-required"
 	done
 fi
 
