diff --git a/debian/changelog b/debian/changelog
index 27e512c..4e26ac8 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,9 @@
+debootstrap (1.0.84) unstable; urgency=medium
+
+  * Enable merged-/usr by default.
+
+ -- Marco d'Itri <md@linux.it>  Thu, 29 Sep 2016 17:49:19 +0200
+
 debootstrap (1.0.83) unstable; urgency=medium
 
   [ Ansgar Burchardt ]
diff --git a/debootstrap b/debootstrap
index 084a541..c781a6c 100755
--- a/debootstrap
+++ b/debootstrap
@@ -27,7 +27,7 @@ KEYRING=""
 DISABLE_KEYRING=""
 FORCE_KEYRING=""
 VARIANT=""
-MERGED_USR="no"
+MERGED_USR="yes"
 ARCH=""
 HOST_ARCH=""
 HOST_OS=""
diff --git a/debootstrap.8 b/debootstrap.8
index 3d19683..bb590d7 100644
--- a/debootstrap.8
+++ b/debootstrap.8
@@ -86,6 +86,10 @@ Debian installation in
 .IP
 .IP "\fB\-\-merged-usr\fP"
 Create /{bin,sbin,lib}/ symlinks pointing to their counterparts in /usr/.
+(Default.)
+.IP
+.IP "\fB\-\-no-merged-usr\fP"
+Do not create /{bin,sbin,lib}/ symlinks pointing to their counterparts in /usr/.
 .IP
 .IP "\fB\-\-keyring=KEYRING\fP"
 Override the default keyring for the distribution being bootstrapped,
