Index: blender-2.78.a+dfsg0/intern/cycles/device/device_cuda.cpp
===================================================================
--- blender-2.78.a+dfsg0.orig/intern/cycles/device/device_cuda.cpp
+++ blender-2.78.a+dfsg0/intern/cycles/device/device_cuda.cpp
@@ -384,6 +384,7 @@ public:
 		string command = string_printf("\"%s\" "
 		                               "-arch=sm_%d%d "
 		                               "--cubin \"%s\" "
+                                               "-ccbin clang-3.8 " 
 		                               "-o \"%s\" "
 		                               "%s ",
 		                               nvcc,
