diff -urN ../tmp-orig/cyrus-sasl2-2.1.22/debian/patches/0001_versioned_symbols.dpatch ./debian/patches/0001_versioned_symbols.dpatch
--- ../tmp-orig/cyrus-sasl2-2.1.22/debian/patches/0001_versioned_symbols.dpatch	2006-11-08 13:51:38.000000000 +0000
+++ ./debian/patches/0001_versioned_symbols.dpatch	2006-11-08 13:51:28.000000000 +0000
@@ -14,7 +14,7 @@
  libsasl2_la_SOURCES = $(common_sources) $(common_headers)
 -libsasl2_la_LDFLAGS = -version-info $(sasl_version)
 -libsasl2_la_DEPENDENCIES = $(LTLIBOBJS)
-+libsasl2_la_LDFLAGS = -version-info $(sasl_version) -Wl,--version-script=Versions
++libsasl2_la_LDFLAGS = -version-info $(sasl_version)
 +libsasl2_la_DEPENDENCIES = $(LTLIBOBJS) Versions
  libsasl2_la_LIBADD = $(LTLIBOBJS) $(SASL_DL_LIB) $(LIB_SOCKET) $(LIB_DOOR)
  
