--- /usr/share/dh-lua/make/dh-lua.Makefile.single.orig	2018-02-08 09:38:33.240629090 +0000
+++ /usr/share/dh-lua/make/dh-lua.Makefile.single	2018-02-08 09:38:42.776798990 +0000
@@ -44,7 +44,7 @@
 LUA=lua$(LUA_VERSION)
 C_TEST_FILE=$(DH_LUA_SUPPORT_FILES)test/$(LUA_VERSION)/app.c
 PREFIX=$(DESTDIR)/usr/
-LBTL=libtool --silent --tag=CC
+LBTL=libtool --tag=CC
 ifeq "$(LUA_SOURCES_MANGLER)" ""
 	LUA_SOURCES_MANGLER:=cat
 endif
