From 0c550d9674f00be30a6db559c749d3ec5443c92b Mon Sep 17 00:00:00 2001
From: Rock Storm <rockstorm@gmx.com>
Date: Fri, 15 Jan 2021 16:17:42 +0000
Subject: [PATCH] d/*.links: Add a link for programmers.txt

Closes: #980118
---
 debian/arduino-core-avr.links | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/arduino-core-avr.links b/debian/arduino-core-avr.links
index 9c6d216..c1ebc84 100644
--- a/debian/arduino-core-avr.links
+++ b/debian/arduino-core-avr.links
@@ -3,3 +3,4 @@ usr/share/common-licenses/GPL-2    usr/share/arduino/hardware/arduino/avr/bootlo
 
 usr/share/arduino/hardware/arduino/cores/platform.txt  usr/share/arduino/hardware/arduino/avr/platform.txt
 usr/share/arduino/hardware/arduino/cores/boards.txt    usr/share/arduino/hardware/arduino/avr/boards.txt
+usr/share/arduino/hardware/arduino/cores/programmers.txt    usr/share/arduino/hardware/arduino/avr/programmers.txt
-- 
2.30.0

