Description: Replace xmalloc with xzalloc
Author: Maximiliano Curia <maxy@debian.org>

--- findutils-4.6.0+git+20160517.orig/find/tree.c
+++ findutils-4.6.0+git+20160517/find/tree.c
@@ -865,7 +865,7 @@ set_new_parent (struct predicate *curr,
 {
   struct predicate *new_parent;
 
-  new_parent = xmalloc (sizeof (struct predicate));
+  new_parent = xzalloc (sizeof (struct predicate));
   new_parent->p_type = BI_OP;
   new_parent->p_prec = high_prec;
   new_parent->need_stat = false;
@@ -1491,12 +1491,12 @@ get_new_pred (const struct parser_table
   if (predicates == NULL)
     {
       predicates = (struct predicate *)
-	xmalloc (sizeof (struct predicate));
+	xzalloc (sizeof (struct predicate));
       last_pred = predicates;
     }
   else
     {
-      new_pred = xmalloc (sizeof (struct predicate));
+      new_pred = xzalloc (sizeof (struct predicate));
       last_pred->pred_next = new_pred;
       last_pred = new_pred;
     }
