Index: README
===================================================================
--- README	(revision 66643)
+++ README	(working copy)
@@ -19,7 +19,7 @@
 
   Or for an anonymous checkout:
 
-     svn co svn+ssh://<username>@svn.d-i.alioth.debian.org/svn/d-i/trunk d-i
+     svn co svn://svn.d-i.alioth.debian.org/svn/d-i/trunk d-i
 
   Now run these commands to check out the rest of the installer, from git:
 
