Index: dselect/main.cc
===================================================================
--- dselect/main.cc	(révision 650)
+++ dselect/main.cc	(copie de travail)
@@ -36,7 +36,7 @@
 #include <ctype.h>
 #include <assert.h>
 
-#include <term.h>
+#include <ncursesw/term.h>
 
 extern "C" {
 #include <dpkg.h>
Index: dselect/dselect.h
===================================================================
--- dselect/dselect.h	(révision 650)
+++ dselect/dselect.h	(copie de travail)
@@ -28,7 +28,7 @@
 
 #include <signal.h>
 #undef ERR
-#include <curses.h>
+#include <ncursesw/curses.h>
 
 struct helpmenuentry {
   char key;
Index: dselect/Makefile.am
===================================================================
--- dselect/Makefile.am	(révision 650)
+++ dselect/Makefile.am	(copie de travail)
@@ -41,7 +41,7 @@
 
 curkeys.$(OBJEXT): curkeys.h
 curkeys.h: $(srcdir)/keyoverride $(srcdir)/mkcurkeys.pl
-	cursesfile=`echo '#include <curses.h>' | \
+	cursesfile=`echo '#include <ncursesw/curses.h>' | \
 		$(CC) -E - | grep 'curses.h' | head -n 1 | \
 		sed -e 's/^[^"]*"//; s/".*$$//'`; \
 	if [ "$$cursesfile" = "" ]; then \
Index: m4/libs.m4
===================================================================
--- m4/libs.m4	(révision 650)
+++ m4/libs.m4	(copie de travail)
@@ -93,7 +93,7 @@
 # Check for curses library.
 AC_DEFUN([DPKG_LIB_CURSES],
 [AC_ARG_VAR([CURSES_LIBS], [linker flags for curses library])dnl
-AC_CHECK_LIB([ncurses], [initscr], [CURSES_LIBS="${CURSES_LIBS:+$CURSES_LIBS }-lncurses"],
+AC_CHECK_LIB([ncursesw], [initscr], [CURSES_LIBS="${CURSES_LIBS:+$CURSES_LIBS }-lncursesw"],
 	[AC_CHECK_LIB([curses], [initscr], [CURSES_LIBS="${CURSES_LIBS:+$CURSES_LIBS }-lcurses"],
 		[AC_MSG_WARN([no curses library found])])])
 ])# DPKG_LIB_CURSES
