Index: amule-2.1.3/src/ExternalConnector.h
===================================================================
--- amule-2.1.3.orig/src/ExternalConnector.h	2008-03-17 13:09:42.000000000 -0300
+++ amule-2.1.3/src/ExternalConnector.h	2008-03-17 13:09:54.000000000 -0300
@@ -143,7 +143,7 @@
 	bool Parse_Command(const wxString& buffer);
 	void GetCommand(const wxString &prompt, char* buffer, size_t buffer_size);
 	const CECPacket *SendRecvMsg_v2(const CECPacket *request) { return m_ECClient->SendRecvPacket(request); }
-	void ConnectAndRun(const wxString &ProgName, const wxString& ProgName);
+	void ConnectAndRun(const wxString &ProgName1, const wxString& ProgName2);
 	void ShowGreet();
 
 	//
Index: amule-2.1.3/src/ED2KLinkParser.cpp
===================================================================
--- amule-2.1.3.orig/src/ED2KLinkParser.cpp	2008-03-17 13:29:45.000000000 -0300
+++ amule-2.1.3/src/ED2KLinkParser.cpp	2008-03-17 13:30:11.000000000 -0300
@@ -42,6 +42,7 @@
 #endif
 
 #include "FileLock.h"
+#include <cstdlib>
 
 using std::string;
 
