=== modified file 'apt-pkg/deb/dpkgpm.cc'
--- apt-pkg/deb/dpkgpm.cc	2009-12-10 15:06:40 +0000
+++ apt-pkg/deb/dpkgpm.cc	2009-12-14 15:32:45 +0000
@@ -879,7 +879,8 @@
 	    const char *s = _("Can not write log, openpty() "
 	                      "failed (/dev/pts not mounted?)\n");
 	    fprintf(stderr, "%s",s);
-	    fprintf(term_out, "%s",s);
+            if(term_out)
+              fprintf(term_out, "%s",s);
 	    master = slave = -1;
 	 }  else {
 	    struct termios rtt;

