--- crash-6.0.1.orig/debian/control	2012-01-16 18:25:11.000000000 +0000
+++ crash-6.0.1/debian/control	2012-01-16 16:30:04.000000000 +0000
@@ -7,7 +7,7 @@
 Standards-Version: 3.9.2.0
 
 Package: crash
-Architecture: i386 ia64 alpha powerpc amd64
+Architecture: i386 ia64 alpha powerpc amd64 armel
 Depends: ${shlibs:Depends}, ${misc:Depends}
 Suggests: kernel-patch-lkcd, dumputils
 Description: kernel debugging utility, allowing gdb like syntax
