Index: cvsps-2.1/cvs_direct.c
===================================================================
--- cvsps-2.1.orig/cvs_direct.c
+++ cvsps-2.1/cvs_direct.c
@@ -180,7 +180,7 @@ static CvsServerCtx * open_ctx_pserver(C
     char * p = root, *tok, *tok2;
     char user[BUFSIZ];
     char server[BUFSIZ];
-    char pass[BUFSIZ];
+    char pass[BUFSIZ] = "\0";
     char port[8];
 
     strcpy(root, p_root);
