diff -urp old/debtags-1.7.3/debian/changelog new/debtags-1.7.3+nmu1/debian/changelog
--- old/debtags-1.7.3/debian/changelog	2007-07-06 15:21:25.000000000 +0200
+++ new/debtags-1.7.3+nmu1/debian/changelog	2008-03-13 11:16:35.000000000 +0100
@@ -1,3 +1,13 @@
+debtags (1.7.3+nmu1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Don't try to run debtags in cronjob when binary is not present.  (Closes:
+    #435067, #435235, #435243, #457058)
+  * Don't make cron mail output unless an error is encountered.  (Closes:
+    #431994)
+
+ -- Bas Wijnen <wijnen@debian.org>  Thu, 13 Mar 2008 11:16:22 +0100
+
 debtags (1.7.3) unstable; urgency=low
 
   * Allow to install extra tag sources in /etc/debtags/sources.list.d/
diff -urp old/debtags-1.7.3/debian/debtags.cron.daily new/debtags-1.7.3+nmu1/debian/debtags.cron.daily
--- old/debtags-1.7.3/debian/debtags.cron.daily	2007-07-06 15:21:25.000000000 +0200
+++ new/debtags-1.7.3+nmu1/debian/debtags.cron.daily	2008-03-13 11:15:25.000000000 +0100
@@ -1,4 +1,4 @@
 #!/bin/sh
 
 # Update the data and index files of Debtags
-debtags update
+test ! -x /usr/bin/debtags || /usr/bin/debtags update 2>/dev/null || echo debtags update error
