--- openssl-easyrsa.cnf	2017-12-14 23:37:00.718627177 +0100
+++ openssl-easyrsa.cnf.reordered-1.0.0	2017-12-14 23:43:41.827280226 +0100
@@ -64,12 +64,6 @@
 ####################################################################
 # Easy-RSA DN (Subject) handling
 
-# Easy-RSA DN for cn_only support:
-[ cn_only ]
-commonName		= Common Name (eg: your user, host, or server name)
-commonName_max		= 64
-commonName_default	= $ENV::EASYRSA_REQ_CN
-
 # Easy-RSA DN for org support:
 [ org ]
 countryName			= Country Name (2 letter code)
@@ -97,6 +91,12 @@
 emailAddress_default		= $ENV::EASYRSA_REQ_EMAIL
 emailAddress_max		= 64
 
+# Easy-RSA DN for cn_only support:
+[ cn_only ]
+commonName		= Common Name (eg: your user, host, or server name)
+commonName_max		= 64
+commonName_default	= $ENV::EASYRSA_REQ_CN
+
 ####################################################################
 # Easy-RSA cert extension handling
 
