From 5ce7bb42ddbb3ad9bd59500ce1f601d1619da107 Mon Sep 17 00:00:00 2001
From: Roger Leigh <rleigh@debian.org>
Date: Tue, 14 Jun 2011 20:43:03 +0100
Subject: [PATCH 2/2] Don't add package name to libexecdir in v9

---
 Debian/Debhelper/Buildsystem/autoconf.pm |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/Debian/Debhelper/Buildsystem/autoconf.pm b/Debian/Debhelper/Buildsystem/autoconf.pm
index c211d8d..1fbc80d 100644
--- a/Debian/Debhelper/Buildsystem/autoconf.pm
+++ b/Debian/Debhelper/Buildsystem/autoconf.pm
@@ -40,7 +40,7 @@ sub configure {
 	my $multiarch=dpkg_architecture_value("DEB_HOST_MULTIARCH");
 	if (! compat(8) && defined $multiarch) {
 		push @opts, "--libdir=\${prefix}/lib/$multiarch";
-		push @opts, "--libexecdir=\${prefix}/lib/$multiarch/" . sourcepackage();
+		push @opts, "--libexecdir=\${prefix}/lib/$multiarch";
 	}
 	else {
 		push @opts, "--libexecdir=\${prefix}/lib/" . sourcepackage();
-- 
1.7.5.4

