diff -u apwal-0.4.5/debian/changelog apwal-0.4.5/debian/changelog
--- apwal-0.4.5/debian/changelog
+++ apwal-0.4.5/debian/changelog
@@ -1,3 +1,13 @@
+apwal (0.4.5-1.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Change d/compat to level 9, update B-D on debhelper (Closes: #817352)
+  * Add Homepage to d/control, remove it from package descripiton
+    (Closes: #615184)
+  * Replace dh_clean -k with dh_prep
+
+ -- Tobias Frost <tobi@debian.org>  Sun, 23 Oct 2016 16:20:32 +0200
+
 apwal (0.4.5-1) unstable; urgency=low
 
   * New upstream release. Includes all Debian-specific patches.
diff -u apwal-0.4.5/debian/control apwal-0.4.5/debian/control
--- apwal-0.4.5/debian/control
+++ apwal-0.4.5/debian/control
@@ -2,8 +2,9 @@
 Section: gnome
 Priority: optional
 Maintainer: Sam Hocevar (Debian packages) <sam+deb@zoy.org>
-Build-Depends: debhelper (>= 4.0), libgtk2.0-dev, libxml2-dev
+Build-Depends: debhelper (>= 9), libgtk2.0-dev, libxml2-dev
 Standards-Version: 3.6.1.0
+Homepage: http://apwal.free.fr/
 
 Package: apwal
 Section: gnome
@@ -23,2 +23,0 @@
- .
- Homepage: http://apwal.free.fr/
diff -u apwal-0.4.5/debian/compat apwal-0.4.5/debian/compat
--- apwal-0.4.5/debian/compat
+++ apwal-0.4.5/debian/compat
@@ -1,2 +1 @@
-4
-4
+9
only in patch2:
unchanged:
--- apwal-0.4.5.orig/debian/rules
+++ apwal-0.4.5/debian/rules
@@ -27,7 +27,7 @@
 install: build
 	dh_testdir
 	dh_testroot
-	dh_clean -k
+	dh_prep
 	dh_installdirs
 
 	# Add here commands to install the package into debian/<packagename>
