diff -BurN cryptsetup-1.0.6+20090405.svn49.orig/debian/rules cryptsetup-1.0.6+20090405.svn49/debian/rules
--- cryptsetup-1.0.6+20090405.svn49.orig/debian/rules	2009-04-06 16:27:48.000000000 +0200
+++ cryptsetup-1.0.6+20090405.svn49/debian/rules	2009-04-06 13:55:30.000000000 +0200
@@ -46,11 +46,7 @@
 	rm -f ltmain.sh
 	cp /usr/share/libtool/config/ltmain.sh ltmain.sh
 	./setup-gettext
-	libtoolize --force --copy
-	aclocal-1.9
-	autoheader
-	automake-1.9 --add-missing --gnu
-	autoconf
+	autoreconf --force --install
 	CFLAGS="$(CFLAGS)" ./configure $(confflags) \
 		--prefix=/usr \
 		--sbindir=/sbin \
