From 0fb3d81c77168aada7fe9d1f0337f0114a361fb2 Mon Sep 17 00:00:00 2001
From: Wolfgang Schweer <wschweer@arcor.de>
Date: Sun, 3 Mar 2019 17:42:08 +0100
Subject: [PATCH 2/2] Also pull in second level recommends

---
 tools/sort_deps | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/tools/sort_deps b/tools/sort_deps
index 8582b90..a7c5049 100755
--- a/tools/sort_deps
+++ b/tools/sort_deps
@@ -999,6 +999,10 @@ sub get_missing {
 		return ();
 	}
 
+	if (not add_missing (\@deps_list, $packages{$p}{"Recommends"}, \%t, 0, \@parents, $add_rec, $check_backports)) {
+		return ();
+	}
+
 	# Explicitly move the package itself to the end of the list,
 	# i.e. *after* all its dependencies
 	remove_entry(\%t, \@deps_list);
-- 
2.34.0

