From 522e63bfe145ebc87970f4e20567a3776b975c13 Mon Sep 17 00:00:00 2001
From: Holger Levsen <holger@layer-acht.org>
Date: Mon, 4 Mar 2019 19:00:15 +0100
Subject: [PATCH 3/3] add changelog entry for Wolfgang

Signed-off-by: Holger Levsen <holger@layer-acht.org>
---
 debian/changelog | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/debian/changelog b/debian/changelog
index a1d2ce4..6dbb670 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -2,6 +2,8 @@ debian-cd (3.1.24) UNRELEASED; urgency=medium
 
   [ Wolfgang Schweer ]
   * tasks/buster/Debian-edu-full: Force missing packages onto the Edu BD.
+  * tools/sort_deps: add support for recursively including Recommends. This is
+    disabled by default, set NORECOMMENDS=0 for enabling this. Closes: #601203
 
  -- Holger Levsen <holger@debian.org>  Wed, 20 Feb 2019 14:12:16 +0100
 
-- 
2.11.0

