From 8205b0efe0072a6a6447e930621c5976f30ff1a2 Mon Sep 17 00:00:00 2001
From: James Clarke <jrtc27@jrtc27.com>
Date: Tue, 14 Feb 2017 23:28:19 +0000
Subject: [PATCH] grab_md5: Don't require catz to be in PATH

Closes: #855174
---
 tools/grab_md5 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tools/grab_md5 b/tools/grab_md5
index e56aad7..bb706d4 100755
--- a/tools/grab_md5
+++ b/tools/grab_md5
@@ -71,7 +71,7 @@ do
                 echo "No files found for arch $ARCH. Abort!"
                 exit 1
             fi
-            catz $FILES | MIRROR=$MIRROR perl -ne '
+            $BASEDIR/tools/catz $FILES | MIRROR=$MIRROR perl -ne '
                 chomp;
                 my $mirror = $ENV{"MIRROR"};
                 my $filename;
-- 
2.11.0

