From 0534f8384784e7a4660dcae5cf51941325f2c623 Mon Sep 17 00:00:00 2001
From: Alexander Sbitnev <alexander.sbitnev@gmail.com>
Date: Fri, 9 Jun 2017 19:50:20 +0300
Subject: [PATCH] Run dh_systemd_enable after dh_install

---
 1/rules/debhelper.mk.in | 12 ++++++------
 1 file changed, 6 insertions(+), 6 deletions(-)

diff --git a/1/rules/debhelper.mk.in b/1/rules/debhelper.mk.in
index f4fde04..b4bb135 100644
--- a/1/rules/debhelper.mk.in
+++ b/1/rules/debhelper.mk.in
@@ -234,12 +234,6 @@ $(patsubst %,binary-install/%,$(DEB_ALL_PACKAGES)):: binary-install/%:
 	dh_installinfo -p$(cdbs_curpkg) $(call cdbs_expand_curvar,DEB_INSTALL_INFO)
 	dh_installmenu -p$(cdbs_curpkg) $(call cdbs_expand_curvar,DEB_DH_INSTALL_MENU_ARGS)
 	dh_installcron -p$(cdbs_curpkg) $(call cdbs_expand_curvar,DEB_DH_INSTALL_CRON_ARGS)
-	$(if $(wildcard /usr/bin/dh_systemd_enable),\
-		dh_systemd_enable -p$(cdbs_curpkg) $(call cdbs_expand_curvar,DEB_DH_SYSTEMD_ENABLE_ARGS))
-	dh_installinit -p$(cdbs_curpkg) $(strip \
-		$(if $(call cdbs_expand_curvar,DEB_UPDATE_RCD_PARAMS),\
-			--update-rcd-params="$(call cdbs_strip_quotes,$(call cdbs_expand_curvar,DEB_UPDATE_RCD_PARAMS))") \
-			$(call cdbs_expand_curvar,DEB_DH_INSTALLINIT_ARGS))
 	dh_installdebconf -p$(cdbs_curpkg) $(call cdbs_expand_curvar,DEB_DH_INSTALLDEBCONF_ARGS)
 	dh_installemacsen -p$(cdbs_curpkg) $(strip \
 		$(if $(DEB_EMACS_PRIORITY),\
@@ -264,6 +258,12 @@ $(patsubst %,binary-install/%,$(DEB_ALL_PACKAGES)):: binary-install/%:
 		$(if $(DEB_DH_INSTALL_SOURCEDIR),\
 			--sourcedir=$(DEB_DH_INSTALL_SOURCEDIR)) \
 		$(call cdbs_expand_curvar,DEB_DH_INSTALL_ARGS))
+	$(if $(wildcard /usr/bin/dh_systemd_enable),\
+		dh_systemd_enable -p$(cdbs_curpkg) $(call cdbs_expand_curvar,DEB_DH_SYSTEMD_ENABLE_ARGS))
+	dh_installinit -p$(cdbs_curpkg) $(strip \
+		$(if $(call cdbs_expand_curvar,DEB_UPDATE_RCD_PARAMS),\
+			--update-rcd-params="$(call cdbs_strip_quotes,$(call cdbs_expand_curvar,DEB_UPDATE_RCD_PARAMS))") \
+			$(call cdbs_expand_curvar,DEB_DH_INSTALLINIT_ARGS))
 	$(if $(wildcard /usr/bin/dh_systemd_start),\
 		dh_systemd_start -p$(cdbs_curpkg) $(call cdbs_expand_curvar,DEB_DH_SYSTEMD_START_ARGS))
 # TODO: Use DEB_DH_LINK_ARGS as package-default var (not global addon)
-- 
2.7.4

