--- debian/dnscrypt-proxy.service.orig	2015-03-08 12:00:00.000000000 +0000
+++ debian/dnscrypt-proxy.service	2015-03-08 12:00:00.000000000 +0000
@@ -5,10 +5,10 @@
 Requires=dnscrypt-proxy.socket
 
 [Service]
-Type=notify
+Type=simple
 Environment=DNSCRYPT_PROXY_RESOLVER_NAME=opendns DNSCRYPT_PROXY_USER=_dnscrypt-proxy "DNSCRYPT_PROXY_OPTIONS="
 EnvironmentFile=-/etc/default/dnscrypt-proxy
-ExecStart=/usr/sbin/dnscrypt-proxy
+ExecStart=/usr/sbin/dnscrypt-proxy \
     --user=${DNSCRYPT_PROXY_USER} \
     --resolver-name=${DNSCRYPT_PROXY_RESOLVER_NAME} \
     $DNSCRYPT_PROXY_OPTIONS
