diff -urN dahdi-tools-2.2.1_old/xpp/perl_modules/Dahdi/Chans.pm dahdi-tools-2.2.1/xpp/perl_modules/Dahdi/Chans.pm
--- dahdi-tools-2.2.1_old/xpp/perl_modules/Dahdi/Chans.pm	2010-05-11 15:31:43.000000000 +0200
+++ dahdi-tools-2.2.1/xpp/perl_modules/Dahdi/Chans.pm	2010-05-11 15:47:31.000000000 +0200
@@ -142,7 +142,7 @@
 		$type = "FXO"; # likely Rhino
 	} elsif ($fqn =~ m{---/.*}) {
 		$type = "EMPTY"; # likely Rhino, empty slot.
-	} elsif ($fqn =~ m{\b(TE[24]|WCT1|Tor2|TorISA|WP[TE]1|cwain[12]|R[124]T1)/.*}) {
+	} elsif ($fqn =~ m{\b(TE[24]|WCT1|Tor2|TorISA|WP[TE]1|cwain[12]|R[124]T1|AP40[124]|APE40[124])/.*}) {
 		# TE[24]: Digium wct4xxp
 		# WCT1: Digium single span card drivers?
 		# Tor2: Tor PCI cards
@@ -150,6 +150,8 @@
 		# WP[TE]1: Sangoma. TODO: this one tells us if it is TE or NT.
 		# cwain: Junghanns E1 card.
 		# R[124]: Rhino r1t1/rxt1 cards
+		# AP40[124]: Aligera AP40X cards
+		# APE40[124]: Aligera APE40X cards
 		$type = "PRI";
 	} elsif ($fqn =~ m{\b(B4|ZTHFC\d*|ztqoz\d*)/.*}) {
 		# B4: The Digium wcb4xxp DAHDI driver
diff -urN dahdi-tools-2.2.1_old/xpp/perl_modules/Dahdi/Hardware/PCI.pm dahdi-tools-2.2.1/xpp/perl_modules/Dahdi/Hardware/PCI.pm
--- dahdi-tools-2.2.1_old/xpp/perl_modules/Dahdi/Hardware/PCI.pm	2010-05-11 15:31:43.000000000 +0200
+++ dahdi-tools-2.2.1/xpp/perl_modules/Dahdi/Hardware/PCI.pm	2010-05-11 15:40:31.000000000 +0200
@@ -108,27 +108,30 @@
 	'1397:f001'		=> { DRIVER => 'ztgsm', DESCRIPTION => 'HFC-GSM Cologne Chips GSM' },
 
 	# Rhino cards (based on pci.ids)
-	'0b0b:0105'	=> { DRIVER => 'r1t1', DESCRIPTION => 'Rhino R1T1' },
-	'0b0b:0205'	=> { DRIVER => 'r4fxo', DESCRIPTION => 'Rhino R14FXO' },
-	'0b0b:0206'	=> { DRIVER => 'rcbfx', DESCRIPTION => 'Rhino RCB4FXO 4-channel FXO analog telphony card' },
-	'0b0b:0305'	=> { DRIVER => 'r1t1', DESCRIPTION => 'Rhino R1T1' },
-	'0b0b:0405'	=> { DRIVER => 'rcbfx', DESCRIPTION => 'Rhino R8FXX' },
-	'0b0b:0406'	=> { DRIVER => 'rcbfx', DESCRIPTION => 'Rhino RCB8FXX 8-channel modular analog telphony card' },
-	'0b0b:0505'	=> { DRIVER => 'rcbfx', DESCRIPTION => 'Rhino R24FXX' },
-	'0b0b:0506'	=> { DRIVER => 'rcbfx', DESCRIPTION => 'Rhino RCB24FXS 24-Channel FXS analog telphony card' },
-	'0b0b:0605'	=> { DRIVER => 'rxt1', DESCRIPTION => 'Rhino R2T1' },
-	'0b0b:0705'	=> { DRIVER => 'rcbfx', DESCRIPTION => 'Rhino R24FXS' },
-	'0b0b:0706'	=> { DRIVER => 'rcbfx', DESCRIPTION => 'Rhino RCB24FXO 24-Channel FXO analog telphony card' },
-	'0b0b:0906'	=> { DRIVER => 'rcbfx', DESCRIPTION => 'Rhino RCB24FXX 24-channel modular analog telphony card' },
+	'0b0b:0105'		=> { DRIVER => 'r1t1', DESCRIPTION => 'Rhino R1T1' },
+	'0b0b:0205'		=> { DRIVER => 'r4fxo', DESCRIPTION => 'Rhino R14FXO' },
+	'0b0b:0206'		=> { DRIVER => 'rcbfx', DESCRIPTION => 'Rhino RCB4FXO 4-channel FXO analog telephony card' },
+	'0b0b:0305'		=> { DRIVER => 'r1t1', DESCRIPTION => 'Rhino R1T1' },
+	'0b0b:0405'		=> { DRIVER => 'rcbfx', DESCRIPTION => 'Rhino R8FXX' },
+	'0b0b:0406'		=> { DRIVER => 'rcbfx', DESCRIPTION => 'Rhino RCB8FXX 8-channel modular analog telephony card' },
+	'0b0b:0505'		=> { DRIVER => 'rcbfx', DESCRIPTION => 'Rhino R24FXX' },
+	'0b0b:0506'		=> { DRIVER => 'rcbfx', DESCRIPTION => 'Rhino RCB24FXS 24-Channel FXS analog telephony card' },
+	'0b0b:0605'		=> { DRIVER => 'rxt1', DESCRIPTION => 'Rhino R2T1' },
+	'0b0b:0705'		=> { DRIVER => 'rcbfx', DESCRIPTION => 'Rhino R24FXS' },
+	'0b0b:0706'		=> { DRIVER => 'rcbfx', DESCRIPTION => 'Rhino RCB24FXO 24-Channel FXO analog telephony card' },
+	'0b0b:0906'		=> { DRIVER => 'rcbfx', DESCRIPTION => 'Rhino RCB24FXX 24-channel modular analog telephony card' },
 
 	# Sangoma cards (based on pci.ids)
-	'1923:0040'	=> { DRIVER => 'wanpipe', DESCRIPTION => 'Sangoma Technologies Corp. A200/Remora FXO/FXS Analog AFT card' },
-	'1923:0100'	=> { DRIVER => 'wanpipe', DESCRIPTION => 'Sangoma Technologies Corp. A104d QUAD T1/E1 AFT card' },
-	'1923:0300'	=> { DRIVER => 'wanpipe', DESCRIPTION => 'Sangoma Technologies Corp. A101 single-port T1/E1' },
-	'1923:0400'	=> { DRIVER => 'wanpipe', DESCRIPTION => 'Sangoma Technologies Corp. A104u Quad T1/E1 AFT' },
+	'1923:0040'		=> { DRIVER => 'wanpipe', DESCRIPTION => 'Sangoma Technologies Corp. A200/Remora FXO/FXS Analog AFT card' },
+	'1923:0100'		=> { DRIVER => 'wanpipe', DESCRIPTION => 'Sangoma Technologies Corp. A104d QUAD T1/E1 AFT card' },
+	'1923:0300'		=> { DRIVER => 'wanpipe', DESCRIPTION => 'Sangoma Technologies Corp. A101 single-port T1/E1' },
+	'1923:0400'		=> { DRIVER => 'wanpipe', DESCRIPTION => 'Sangoma Technologies Corp. A104u Quad T1/E1 AFT' },
 
 	# Yeastar (from output of modinfo):
-	'e159:0001/2151' => { DRIVER => 'ystdm8xx', DESCRIPTION => 'Yeastar YSTDM8xx'},
+	'e159:0001/2151'	=> { DRIVER => 'ystdm8xx', DESCRIPTION => 'Yeastar YSTDM8xx' },
+
+	# Aligera
+	'10ee:1004'		=> { DRIVER => 'ap400', DESCRIPTION => 'Aligera AP40X/APE40X 1E1/2E1/4E1 card' },
 	);
 
 $ENV{PATH} .= ":/usr/sbin:/sbin:/usr/bin:/bin";
