Author: Wolfgang Schweer <wschweer@arcor.de>
Description: Adjust custom dblatex conf file after Inkscape changes
--- a/docs/custom/dblatex.xconf
+++ b/docs/custom/dblatex.xconf
@@ -18,7 +18,7 @@
   <imagedata>
     <converter src="svg" dst="*" docformat="pdf">
       <command>
-        inkscape -z -D --export-dpi=600 --export-%(dst)s=%(output)s %(input)s
+        inkscape -D --export-dpi=600 --export-filename=%(output)s %(input)s
       </command>
     </converter>
   </imagedata>
