--- a/debian/rules
+++ b/debian/rules
@@ -9,9 +9,10 @@ override_dh_clean:
 	dh_clean
 
 override_dh_auto_build:
-	# - Nothing to be compiled
 	# - Don't call build_doc, as it's superfluous and will run into
 	#   bootstrap problems.
+	# - But rebuild the PDF manuals, just in case source files contain changes.
+	rm --force docs/*.pdf
 
 override_dh_installchangelogs:
 	dh_installchangelogs docs/changes/changes.txt
