From f69e12808f2dc2915cb4222e5b3c3cf9107d04a4 Mon Sep 17 00:00:00 2001
From: Don Armstrong <don@debian.org>
Date: Tue, 13 Nov 2012 13:03:05 -0800
Subject: [PATCH 1/2] fix git-ignore to only ignore the build directory in
 debian

---
 .gitignore |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/.gitignore b/.gitignore
index 2c8599c..704688d 100644
--- a/.gitignore
+++ b/.gitignore
@@ -1,6 +1,6 @@
 build-stamp
 configure-stamp
-debian/ca-certificates*
+debian/ca-certificates
 debian/config
 debian/files
 cacert.org/cacert.org.crt
-- 
1.7.10.4

