diff -u apcupsd-3.14.12/debian/changelog apcupsd-3.14.12/debian/changelog
--- apcupsd-3.14.12/debian/changelog
+++ apcupsd-3.14.12/debian/changelog
@@ -1,3 +1,11 @@
+apcupsd (3.14.12-1.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * src/apcaccess.c: declare "ch" as int to match getopt return type. Fixes
+    apcaccess on ARM. (Closes: #770984)
+
+ -- Emanuele Rocca <ema@debian.org>  Tue, 14 Apr 2015 09:14:33 +0200
+
 apcupsd (3.14.12-1) unstable; urgency=low
 
   * [84a0ea2] Imported Upstream version 3.14.12
diff -u apcupsd-3.14.12/debian/patches/series apcupsd-3.14.12/debian/patches/series
--- apcupsd-3.14.12/debian/patches/series
+++ apcupsd-3.14.12/debian/patches/series
@@ -4,0 +5 @@
+05-apcaccess-arm.patch
only in patch2:
unchanged:
--- apcupsd-3.14.12.orig/debian/patches/05-apcaccess-arm.patch
+++ apcupsd-3.14.12/debian/patches/05-apcaccess-arm.patch
@@ -0,0 +1,13 @@
+Declare "ch" as int to match getopt return type. Fix apcaccess behavior on
+arm.
+--- a/src/apcaccess.c
++++ b/src/apcaccess.c
+@@ -146,7 +146,7 @@
+ #endif
+ 
+    // Process standard options
+-   char ch;
++   int ch;
+    while ((ch = getopt(argc, argv, "f:h:p:u")) != -1)
+    {
+       switch (ch)
