$ git diff master..wheezy-backports debian/asterisk.init 
diff --git a/debian/asterisk.init b/debian/asterisk.init
index 0ab18a7..1462e21 100644
--- a/debian/asterisk.init
+++ b/debian/asterisk.init
@@ -178,7 +178,10 @@ case "$1" in
                                  --pidfile $ASTSAFE_PIDFILE
                rm -f $ASTSAFE_PIDFILE
        fi
-       start-stop-daemon --stop --quiet --oknodo --exec $DAEMON --pidfile=$PIDFILE
+       # just making sure it's really, really dead. 
+       # KILL is necessary just in case there's an asterisk -r in the background
+       start-stop-daemon --stop --quiet --oknodo --retry=0/2/TERM/2/KILL/5 --exec $DAEMON
+       start-stop-daemon --stop --quiet --oknodo --retry=0/2/TERM/2/KILL/5 --exec $CANARY
        echo "."
        ;;
   reload)
