Index: zaphfc/base.c
===================================================================
--- a/drivers/dahdi/zaphfc/base.c	(revision 4)
+++ b/drivers/dahdi/zaphfc/base.c	(working copy)
@@ -626,10 +626,7 @@ static int hfc_zap_chanconfig(struct dahdi_chan *d
 	struct hfc_card *card = chan->card;
 	struct dahdi_hfc *hfccard = card->ztdev;
 
-	if ((sigtype == DAHDI_SIG_HARDHDLC) || (hfccard->sigchan == d_chan)) {
-		hfccard->sigchan = (sigtype == DAHDI_SIG_HARDHDLC)
-			? d_chan : NULL;
-
+	if ((sigtype == DAHDI_SIG_HARDHDLC) && (hfccard->sigchan == d_chan)) {
 		hfccard->sigactive = 0;
 		atomic_set(&hfccard->hdlc_pending, 0);
 	}
