diff -Naur debhelper-9.20120909/Debian/Debhelper/Buildsystem/qmake.pm debhelper-9.20120909.mod/Debian/Debhelper/Buildsystem/qmake.pm
--- debhelper-9.20120909/Debian/Debhelper/Buildsystem/qmake.pm	2011-12-08 19:26:46.000000000 -0300
+++ debhelper-9.20120909.mod/Debian/Debhelper/Buildsystem/qmake.pm	2013-04-14 16:11:56.762544395 -0300
@@ -50,6 +50,11 @@
 	my @options;
 	my @flags;
 
+	# Select Qt4 by default except if overriden otherwise.
+	if (!$ENV{QT_SELECT}) {
+		push @options, "-qt=qt4";
+	}
+
 	push @options, '-makefile';
 	push @options, '-nocache';
 
