diff -Nru diod-1.0.24/debian/changelog diod-1.0.24/debian/changelog
--- diod-1.0.24/debian/changelog	2020-03-16 23:20:44.000000000 +0100
+++ diod-1.0.24/debian/changelog	2023-11-27 12:39:55.000000000 +0100
@@ -1,3 +1,11 @@
+diod (1.0.24-5.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Move mount.diod symlink into /usr. (Closes: #1056972)
+  * Update Build-Depends libncurses5-dev to libncurses-dev.
+
+ -- Chris Hofstaedtler <zeha@debian.org>  Mon, 27 Nov 2023 12:39:55 +0100
+
 diod (1.0.24-5) unstable; urgency=medium
 
   * Don't depend on libattr1-dev for <attr/xattr.h> use <sys/xattr.h> from
diff -Nru diod-1.0.24/debian/control diod-1.0.24/debian/control
--- diod-1.0.24/debian/control	2020-03-16 23:20:44.000000000 +0100
+++ diod-1.0.24/debian/control	2023-11-27 12:39:55.000000000 +0100
@@ -3,7 +3,7 @@
 Priority: optional
 Maintainer: Євгеній Мещеряков <eugen@debian.org>
 Build-Depends: debhelper (>= 10~), libpopt-dev, liblua5.1-0-dev,
-               libmunge-dev, libcap-dev, libncurses5-dev, libwrap0-dev,
+               libmunge-dev, libcap-dev, libncurses-dev, libwrap0-dev,
                valgrind [amd64 i386 powerpc], pkg-config
 Build-Conflicts: valgrind [armel]
 Standards-Version: 4.5.0
diff -Nru diod-1.0.24/debian/diod.dirs diod-1.0.24/debian/diod.dirs
--- diod-1.0.24/debian/diod.dirs	2020-03-16 23:20:44.000000000 +0100
+++ diod-1.0.24/debian/diod.dirs	2023-11-27 12:39:55.000000000 +0100
@@ -1,2 +1,2 @@
-sbin
+usr/sbin
 usr/share/doc/diod/examples
diff -Nru diod-1.0.24/debian/diod.links diod-1.0.24/debian/diod.links
--- diod-1.0.24/debian/diod.links	2020-03-16 23:20:44.000000000 +0100
+++ diod-1.0.24/debian/diod.links	2023-11-27 12:39:55.000000000 +0100
@@ -1,2 +1,2 @@
-usr/sbin/diodmount sbin/mount.diod
+usr/sbin/diodmount usr/sbin/mount.diod
 usr/share/man/man8/diodmount.8.gz usr/share/man/man8/mount.diod.8.gz
