diff --git a/debmirror b/debmirror
index 0c50a2f..7ab6ad6 100755
--- a/debmirror
+++ b/debmirror
@@ -2466,6 +2466,7 @@ sub di_add_files {
   foreach my $dist (keys %distset) {
     next unless exists $distset{$dist}{"d-i"};
     foreach my $arch (@di_arches) {
+      next if $arch eq "all";
       next if $arch =~ /kfreebsd-/;
 
       my $image_dir = "dists/$dist/main/installer-$arch/current/images";
