--- 30_exim4-config_check_rcpt_sender	2017-11-05 15:06:25.839001108 +0100
+++ exim4-4.86.2-patched/debian/debconf/conf.d/acl/30_exim4-config_check_rcpt	2017-11-05 15:07:54.158520031 +0100
@@ -189,12 +189,6 @@
     !verify = sender
   .endif
 
-  # We also require all accepted addresses to be verifiable. This check will
-  # do local part verification for local domains, but only check the domain
-  # for remote domains.
-  require
-    verify = recipient
-
 
   # Verify recipients listed in local_rcpt_callout with a callout.
   # This is especially handy for forwarding MX hosts (secondary MX or
@@ -340,6 +334,16 @@
   .include CHECK_RCPT_LOCAL_ACL_FILE
   .endif
 
+  # Recipient check leaks information about which local accounts exist.
+  # This could be exploited in a brute-force scan to find valid addresses (usually
+  # for spam purposes). Performing this check last gives the other checks a chance
+  # to deny the request before they succeed at extracting this information.
+
+  # We also require all accepted addresses to be verifiable. This check will
+  # do local part verification for local domains, but only check the domain
+  # for remote domains.
+  require
+    verify = recipient
 
   #############################################################################
   # This check is commented out because it is recognized that not every
