--- 30_exim4-config_check_rcpt_orig	2017-11-05 15:13:31.253064618 +0100
+++ 30_exim4-config_check_rcpt_sender	2017-11-05 15:15:15.076702584 +0100
@@ -107,24 +107,6 @@
     domains = +local_domains : +relay_to_domains
 
 
-  # Deny unless the sender address can be verified.
-  #
-  # This is disabled by default so that DNSless systems don't break. If
-  # your system can do DNS lookups without delay or cost, you might want
-  # to enable this feature.
-  #
-  # This feature does not work in smarthost and satellite setups as
-  # with these setups all domains pass verification. See spec.txt section
-  # "Access control lists" subsection "Address verification" with the added
-  # information that a smarthost/satellite setup routes all non-local e-mail
-  # to the smarthost.
-  .ifdef CHECK_RCPT_VERIFY_SENDER
-  deny
-    message = Sender verification failed
-    !acl = acl_local_deny_exceptions
-    !verify = sender
-  .endif
-
   # Verify senders listed in local_sender_callout with a callout.
   #
   # In smarthost and satellite setups, this causes the callout to be
@@ -183,6 +165,34 @@
     message = relay not permitted
     domains = +local_domains : +relay_to_domains
 
+  # If sender check is enabled, it should be after relay check. Otherwise, it
+  # would be possible to tell if a local address exists by attempting to send
+  # a mail with:
+  # - the targetted local address as sender
+  # - an arbitrary remote address as recipient
+  # Thus if the mail were rejected with "relay not permitted", this means it
+  # passed sender verification.
+  #
+  # This could be exploited in a brute-force scan to find valid addresses (usually
+  # for spam purposes).
+
+  # Deny unless the sender address can be verified.
+  #
+  # This is disabled by default so that DNSless systems don't break. If
+  # your system can do DNS lookups without delay or cost, you might want
+  # to enable this feature.
+  #
+  # This feature does not work in smarthost and satellite setups as
+  # with these setups all domains pass verification. See spec.txt section
+  # "Access control lists" subsection "Address verification" with the added
+  # information that a smarthost/satellite setup routes all non-local e-mail
+  # to the smarthost.
+  .ifdef CHECK_RCPT_VERIFY_SENDER
+  deny
+    message = Sender verification failed
+    !acl = acl_local_deny_exceptions
+    !verify = sender
+  .endif
 
   # We also require all accepted addresses to be verifiable. This check will
   # do local part verification for local domains, but only check the domain
