--- ispell.el.orig	2003-09-03 11:33:12.000000000 +0100
+++ ispell.el	2003-09-03 11:37:18.000000000 +0100
@@ -723,10 +723,7 @@
 			       (const "~nroff") (const "~list")
 			       (const "~latin1") (const "~latin3")
  			       (const :tag "default" nil))
-		       (choice :tag "Coding System"
-			       (const iso-8859-1)
-			       (const iso-8859-2)
-			       (const koi8-r))))
+		       (coding-system :tag "Coding System")))
   :group 'ispell)
 
 ;;; update the dictionaries at load time
