--- cron-apt	2015-12-15 23:01:13.549108700 +0100
+++ /usr/sbin/cron-apt	2015-12-17 14:26:50.568026395 +0100
@@ -246,10 +246,9 @@
 MINTMPDIRSIZE=10
 # The command to use (can be aptitude instead)
 APTCOMMAND="/usr/bin/apt-get"
-# If HOSTNAME is non-empty, the contents will be used to generate the
-# e-mail subject for notifications sent out. If HOSTNAME is empty, the
-# output of $(uname -n) will be used.
-HOSTNAME=""
+# HOSTNAME will be used to generate the  e-mail subject for notifications
+# sent out.
+HOSTNAME="$(uname -n)"
 # Ignore lines matching this regexp to determine whether changes occurred
 # for MAILON="changes".
 DIFFIGNORE=""
