--- functions	2015-12-15 23:01:33.221130707 +0100
+++ /usr/share/cron-apt/functions	2015-12-17 14:25:12.104333094 +0100
@@ -94,12 +94,9 @@
     fi
     if [ -f "$MAIL" ] && [ "$MAILON" != "never" ] && [ -n "$MAILON" ] ; then
 	if command -v mail >/dev/null; then
-	    if [ -z "$HOSTNAME" ]; then
-		HOSTNAME="$(uname -n)"
-	    fi
-	    SUBJECT="CRON-APT completed on $HOSTNAME [$CONFIG]"
+	    SUBJECT="${OKSUBJECT:-CRON-APT ${SUBJECTSUPPL:+$SUBJECTSUPPL }completed on $HOSTNAME [$CONFIG]}"
 	    if [ -f "$ERROR" ] ; then
-		SUBJECT="CRON-APT error on $HOSTNAME [$CONFIG]"
+		SUBJECT="${ERRSUBJECT:-CRON-APT  ${SUBJECTSUPPL:+$SUBJECTSUPPL }error on $HOSTNAME [$CONFIG]}"
 	    fi
 	    # Support for XHEADERS
 	    IFSOLD=$IFS
