diff --git a/Makefile.in b/Makefile.in
index d64efbe..5a5184d 100644
--- a/Makefile.in
+++ b/Makefile.in
@@ -18,7 +18,6 @@ perl_optimize   = @perl_optimize@
 perl_ccflags    = @perl_ccflags@
 perl_ldflags    = @perl_ldflags@
 perl_libs       = @perl_libs@
-perl_dla        = @perl_dla@
 perl_ccdlflags  = @perl_ccdlflags@
 perl_cccdlflags = @perl_cccdlflags@
 perl_archlib    = @perl_archlib@
@@ -26,9 +25,9 @@ perl_archlib    = @perl_archlib@
 #   compiler tools
 CC       = $(perl_cc)
 CFLAGS   = @CFLAGS@ $(perl_optimize) $(perl_ccflags) -I$(perl_archlib)/CORE -I.
-DLCFLAGS = @CFLAGS@ $(perl_optimize) $(perl_cccdlflags) $(perl_ccflags) -I$(perl_archlib)/CORE -I.
-LDFLAGS  = @LDFLAGS@ $(perl_ccdlflags) $(perl_ldflags) -L$(perl_archlib)/CORE 
-LDLIBS   = $(perl_dla) -lperl $(perl_libs)
+DLCFLAGS = @CFLAGS@ $(perl_optimize) $(perl_cccdlflags) $(perl_ccflags) -I.
+LDFLAGS  = @LDFLAGS@ $(perl_ccdlflags) $(perl_ldflags)
+LDLIBS   = $(perl_libs)
 AR       = @AR@
 RANLIB   = @RANLIB@
 SIZE     = @SIZE@
@@ -213,7 +212,7 @@ OBJS = \
  eperl_powered.o
 
 eperl: $(OBJS)
-	$(CC) $(LDFLAGS) -o eperl $(OBJS) $(LDLIBS) $(dmalloc)
+	$(CC) -o eperl $(OBJS) $(LDFLAGS) $(LDLIBS) $(dmalloc)
 	@[ "x$(debug)" = xon ] || strip eperl
 
 eperl_main.o: eperl_main.c
diff --git a/configure.in b/configure.in
index 690e58b..fc8dc89 100644
--- a/configure.in
+++ b/configure.in
@@ -159,18 +159,15 @@ case $usedl in
     define )
         rc=yes
         AC_DEFINE(HAVE_PERL_DYNALOADER)
-        perl_dla=$perl_archlib/auto/DynaLoader/DynaLoader.a
         ;;
     * )
         rc=no
-        perl_dla=
         ;;
 esac
-AC_SUBST(perl_dla)
 AC_MSG_RESULT([$rc])
 
 AC_MSG_CHECKING([for Perl dynamic loading compilation flags])
-perl_cccdlflags="`$perlprog -e 'use Config; print $Config{cccdlflags}'`";
+perl_cccdlflags="`$perlprog -e 'use Config; print $Config{cccdlflags} . " "; use ExtUtils::Embed; ccopts'`";
 case $perl_cccdlflags in
     " " ) x="none" ;;
     * )   x="$perl_cccdlflags" ;;
@@ -179,7 +176,7 @@ AC_MSG_RESULT([$x])
 AC_SUBST(perl_cccdlflags)
 
 AC_MSG_CHECKING([for Perl dynamic loading link flags])
-perl_ccdlflags="`$perlprog -e 'use Config; print $Config{ccdlflags}'`";
+perl_ccdlflags="`$perlprog -e 'use Config; print $Config{ccdlflags} . " "; use ExtUtils::Embed; ldopts'`";
 case $perl_os in
     *aix* ) perl_ccdlflags="`echo $perl_ccdlflags | sed -e 's;-bE:perl.exp;-bE:${perl_archlib}/CORE/perl.exp;'`" ;;
 esac
