diff -u dash-0.5.7/debian/control dash-0.5.7/debian/control
--- dash-0.5.7/debian/control
+++ dash-0.5.7/debian/control
@@ -2,7 +2,7 @@
 Section: shells
 Priority: optional
 Maintainer: Gerrit Pape <pape@smarden.org>
-Build-Depends: po-debconf
+Build-Depends: po-debconf, dpkg-dev (>= 1.16.1)
 Standards-Version: 3.9.3.0
 Homepage: http://gondor.apana.org.au/~herbert/dash/
 Vcs-Git: http://smarden.org/git/dash.git/
diff -u dash-0.5.7/debian/rules dash-0.5.7/debian/rules
--- dash-0.5.7/debian/rules
+++ dash-0.5.7/debian/rules
@@ -1,7 +1,10 @@
 #!/usr/bin/make -f
 
+DPKG_EXPORT_BUILDFLAGS = 1
+include /usr/share/dpkg/buildflags.mk
+
 CC =gcc
-CFLAGS =-g -O2 -Wall
+CFLAGS +=-Wall
 STRIP =strip
 
 DEB_HOST_GNU_TYPE =$(shell dpkg-architecture -qDEB_HOST_GNU_TYPE)
only in patch2:
unchanged:
--- dash-0.5.7.orig/src/jobs.c
+++ dash-0.5.7/src/jobs.c
@@ -427,7 +427,7 @@
 				goto out;
 #endif
 		}
-		col = fmtstr(s, 32, strsignal(st));
+		col = fmtstr(s, 32, "%s", strsignal(st));
 #ifdef WCOREDUMP
 		if (WCOREDUMP(status)) {
 			col += fmtstr(s + col, 16, " (core dumped)");
