--- debhelper.mk	2007-11-02 14:22:46.000000000 +0200
+++ debhelper.mk	2008-01-22 20:00:44.000000000 +0200
@@ -122,7 +122,7 @@
 endif
 endif
 cdbs_dbg_package = $(if $(DEB_DBG_PACKAGE_$(cdbs_curpkg)),$(DEB_DBG_PACKAGE_$(cdbs_curpkg)),$(DEB_DBG_PACKAGE_ALL))
-cdbs_dbg_package_option = $(if $(cdbs_dbg_package),$(shell if [ "$(DH_COMPAT)" -eq 5 ]; then echo "--dbg-package=$(strip $(cdbs_dbg_package))"; fi))
+cdbs_dbg_package_option = $(if $(cdbs_dbg_package),$(shell if [ "$(DH_COMPAT)" -ge 5 ]; then echo "--dbg-package=$(strip $(cdbs_dbg_package))"; fi))
 DEB_DH_STRIP_ARGS = $(cdbs_dbg_package_option)
 endif
 
