only in patch2:
unchanged:
--- alsa-tools-1.0.13.orig/debian/echomixer.desktop
+++ alsa-tools-1.0.13/debian/echomixer.desktop
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Encoding=UTF-8
+Name=Echomixer
+GenericName=
+Comment=control tool for Echoaudio soundcards
+Icon=
+Exec=echomixer
+Terminal=false
+Categories=AudioVideo;Audio;Utility;
only in patch2:
unchanged:
--- alsa-tools-1.0.13.orig/debian/envy24_control.desktop
+++ alsa-tools-1.0.13/debian/envy24_control.desktop
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Encoding=UTF-8
+Name=Envy24 control
+GenericName=
+Comment=control tool for Envy24 (ice1712) based soundcards
+Icon=
+Exec=envy24control
+Terminal=false
+Categories=AudioVideo;Audio;Utility;
only in patch2:
unchanged:
--- alsa-tools-1.0.13.orig/debian/hdspconf.desktop
+++ alsa-tools-1.0.13/debian/hdspconf.desktop
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Encoding=UTF-8
+Name=HDSPConf
+GenericName=
+Comment=GUI program to control the Hammerfall HDSP Alsa Settings
+Icon=
+Exec=hdspconf
+Terminal=false
+Categories=AudioVideo;Audio;Utility;
only in patch2:
unchanged:
--- alsa-tools-1.0.13.orig/debian/hdspmixer.desktop
+++ alsa-tools-1.0.13/debian/hdspmixer.desktop
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Encoding=UTF-8
+Name=HDSPMixer
+GenericName=
+Comment=tool to control the advanced routing features of the RME Hammerfall DSP
+Icon=
+Exec=hdspmixer
+Terminal=false
+Categories=AudioVideo;Audio;Utility;
only in patch2:
unchanged:
--- alsa-tools-1.0.13.orig/debian/rmedigicontrol.desktop
+++ alsa-tools-1.0.13/debian/rmedigicontrol.desktop
@@ -0,0 +1,10 @@
+[Desktop Entry]
+Type=Application
+Encoding=UTF-8
+Name=Rmedigicontrol
+GenericName=
+Comment=control tool for RME Digi32 and RME Digi96 soundcards
+Icon=
+Exec=rmedigicontrol
+Terminal=false
+Categories=AudioVideo;Audio;Utility;
only in patch2:
unchanged:
--- alsa-tools-1.0.13.orig/debian/alsa-tools-gui.install
+++ alsa-tools-1.0.13/debian/alsa-tools-gui.install
@@ -0,0 +1,5 @@
+debian/echomixer.desktop usr/share/applications
+debian/envy24_control.desktop usr/share/applications
+debian/hdspconf.desktop usr/share/applications
+debian/hdspmixer.desktop usr/share/applications
+debian/rmedigicontrol.desktop usr/share/applications
only in patch2:
unchanged:
--- alsa-tools-1.0.13.orig/debian/alsa-tools-gui.dirs
+++ alsa-tools-1.0.13/debian/alsa-tools-gui.dirs
@@ -0,0 +1 @@
+usr/share/applications
