diff -Naur cryptsetup-1.0.3.orig/debian/initramfs-cryptroot-hook cryptsetup-1.0.3/debian/initramfs-cryptroot-hook
--- cryptsetup-1.0.3.orig/debian/initramfs-cryptroot-hook	2006-05-24 02:25:35.000000000 +0200
+++ cryptsetup-1.0.3/debian/initramfs-cryptroot-hook	2006-05-24 02:27:21.000000000 +0200
@@ -79,6 +79,9 @@
 			size=*)
 				echo -n ",$opt"
 				;;
+			lvm=*)
+				echo -n ",$opt"
+				;;
 			*)
 				# Presumably a non-supported option
 				;;
