--- cryptsetup-1.0.3/debian/initramfs-cryptroot-script	2006-05-24 02:28:59.000000000 +0200
+++ cryptsetup-1.0.3.orig/debian/initramfs-cryptroot-script	2006-05-24 02:25:35.000000000 +0200
@@ -136,7 +136,7 @@
 
 	if [ $? -eq 0 ] && fstype < "$NEWROOT" > /conf/param.conf; then
 		. /conf/param.conf
-		([ "$FSTYPE" = "lvm" ] || [ "$FSTYPE" = "lvm2" ]) && setup_lvm "$cryptlvm"
+		[ "$FSTYPE" = "lvm" ] && setup_lvm "$cryptlvm"
 		[ "$FSTYPE" != "unknown" ] && break
 	fi
 
