diff --git a/dh_makeshlibs b/dh_makeshlibs
index a5ba10e..367aa64 100755
--- a/dh_makeshlibs
+++ b/dh_makeshlibs
@@ -235,7 +235,7 @@ foreach my $package (@{$dh{DOPACKAGES}}) {
 
 	# Historically, --noscripts would disable the creation of
 	# maintscripts for calling ldconfig.
-	if (! $dh{NOSCRIPTS} && $need_ldconfig) {
+	if (! $dh{NOSCRIPTS} && ! $dh{NO_ACT} && $need_ldconfig) {
 		autotrigger($package, 'activate-noawait', 'ldconfig');
 	}
 
