From e7ae7aa485c484826da7017941035203f098633c Mon Sep 17 00:00:00 2001
From: Jan Dittberner <jan@dittberner.info>
Date: Wed, 4 Feb 2009 07:50:57 +0100
Subject: [PATCH] fix spelling mistake in dh_prep

---
 dh_prep |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/dh_prep b/dh_prep
index 690ab54..d9e4bf8 100755
--- a/dh_prep
+++ b/dh_prep
@@ -2,7 +2,7 @@
 
 =head1 NAME
 
-dh_prep - preform cleanups in preparation for building a binary package
+dh_prep - perform cleanups in preparation for building a binary package
 
 =cut
 
-- 
1.5.6.5

