diff -Nru distcc-3.1/debian/changelog distcc-3.1/debian/changelog
--- distcc-3.1/debian/changelog	2013-05-08 14:33:03.000000000 +0200
+++ distcc-3.1/debian/changelog	2014-10-27 19:19:18.000000000 +0100
@@ -1,3 +1,13 @@
+distcc (3.1-6.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Fix "Trigger cycle causes dpkg to fail processing":
+    change triggers from "interest" to "interest-noawait", as proposed by
+    Guillem Jover.
+    (Closes: #764667)
+
+ -- gregor herrmann <gregoa@debian.org>  Mon, 27 Oct 2014 19:18:30 +0100
+
 distcc (3.1-6) unstable; urgency=low
 
   [ Daniel Hartwig ]
diff -Nru distcc-3.1/debian/distcc.triggers.in distcc-3.1/debian/distcc.triggers.in
--- distcc-3.1/debian/distcc.triggers.in	2013-01-10 11:55:32.000000000 +0100
+++ distcc-3.1/debian/distcc.triggers.in	2014-10-27 19:18:27.000000000 +0100
@@ -1,2 +1,2 @@
-interest /usr/lib/gcc
-interest /usr/lib/%DEB_HOST_MULTIARCH%/gcc
+interest-noawait /usr/lib/gcc
+interest-noawait /usr/lib/%DEB_HOST_MULTIARCH%/gcc
