--- cyrus-imapd-2.4-2.4.8/debian/control	2011-05-12 02:00:11.000000000 -0700
+++ ../cyrus-imapd-2.4-2.4.8/debian/control	2011-05-12 14:41:18.000000000 -0700
@@ -32,7 +32,8 @@
 	       po-debconf,
 	       quilt (>> 0.46-7~),
 	       transfig,
-	       xutils-dev
+	       xutils-dev,
+	       libldap2-dev
 Vcs-Git: git://git.debian.org/git/pkg-cyrus-imapd/pkg-cyrus-imapd-2.4.git
 Vcs-Browser: http://git.debian.org/?p=git/pkg-cyrus-imapd/pkg-cyrus-imapd-2.4.git
 Homepage: http://www.cyrusimap.org/
--- cyrus-imapd-2.4-2.4.8/debian/rules	2011-05-12 02:00:11.000000000 -0700
+++ ../cyrus-imapd-2.4-2.4.8/debian/rules	2011-05-12 14:15:37.000000000 -0700
@@ -94,7 +94,8 @@
 	 --with-com_err="" \
 	 --with-pidfile=/var/run/cyrmaster.pid \
 	 --with-syslogfacility=MAIL \
-	 --with-ucdsnmp=/usr
+	 --with-ucdsnmp=/usr \
+	 --with-ldap=/usr
 	echo 'To build this package, configure was called as follows:' \
 		> debian/README.configure-options
 	grep '^ac_cs_config=' config.status \
--- cyrus-imapd-2.4-2.4.8/debian/cyrus-common-2.4.install	2011-05-12 02:00:11.000000000 -0700
+++ ../cyrus-imapd-2.4-2.4.8/debian/cyrus-common-2.4.install	2011-05-12 14:26:00.000000000 -0700
@@ -25,6 +25,9 @@
 usr/lib/cyrus/bin/smmapd
 usr/lib/cyrus/bin/notifyd
 usr/lib/cyrus/bin/fud
+usr/lib/cyrus/bin/ptloader
+usr/lib/cyrus/bin/ptdump
+usr/lib/cyrus/bin/ptexpire
 usr/lib/cyrus/upgrade/
 usr/sbin/cyr_dbtool
 usr/sbin/cyr_df
