diff -ur dupload-2.6.3.2.orig/debian/control dupload-2.6.3.2/debian/control
--- dupload-2.6.3.2.orig/debian/control	2003-09-21 13:32:29.000000000 +0200
+++ dupload-2.6.3.2/debian/control	2006-07-08 14:30:27.000000000 +0200
@@ -10,7 +10,7 @@
 Architecture: all
 Depends: perl (>= 5.003), libnet-perl
 Recommends: ssh
-Suggests: lintian, mail-transport-agent
+Suggests: lintian, yaclc, mail-transport-agent
 Description: utility to upload Debian packages
  dupload will automagically upload Debian packages to a remote host
  with a Debian upload queue. The default host is configurable,
