--- /usr/bin/chronicle	2010-08-24 08:58:36.000000000 +0200
+++ tmp/chronicle	2011-02-28 19:24:52.000000000 +0100
@@ -2339,7 +2339,7 @@
     #
     if ( !defined( $meta{ 'date' } ) || ( !length( $meta{ 'date' } ) ) )
     {
-        my @lt = localtime($mtime);
+        my @lt = gmtime($mtime);
         $meta{ 'date' } = strftime( "%d %B %Y", @lt );
     }
 
@@ -2372,7 +2372,7 @@
     #
     if ( $hms eq '00:00:00' )
     {
-        $hms = time2str( "%H:%M:%S", $mtime );
+        $hms = time2str( "%H:%M:%S", $mtime, "GMT" );
         $tz = "GMT";
     }
 
