diff -wru apt-file-2.0.3/apt-file apt-file-2.0.3-he/apt-file
--- apt-file-2.0.3/apt-file	2003-05-16 15:14:31.000000000 +0200
+++ apt-file-2.0.3-he/apt-file	2003-10-21 14:56:42.000000000 +0200
@@ -325,7 +325,7 @@
 	"/cdrom";
 
     $Conf->{action} = shift @ARGV || "none";
-    $Conf->{pattern} = shift @ARGV;
+    $Conf->{pattern} = shift @ARGV || "";
     $Conf->{pattern} =~ s/^\///;
     $Conf->{pattern} = quotemeta($Conf->{pattern});
     undef $!;
