diff -urN ../chromium-browser-36.0.1985.125/debian/changelog chromium-browser-36.0.1985.125/debian/changelog
--- ../chromium-browser-36.0.1985.125/debian/changelog	2014-07-16 09:50:42.000000000 +0900
+++ chromium-browser-36.0.1985.125/debian/changelog	2014-07-23 19:42:37.000000000 +0900
@@ -1,3 +1,10 @@
+chromium-browser (36.0.1985.125-0.1) UNRELEASED; urgency=low
+
+  * Non-maintainer upload.
+  * Enable gnome-keyring.
+
+ -- Arnaud Fontaine <arnau@debian.org>  Wed, 23 Jul 2014 10:39:33 +0000
+
 chromium-browser (36.0.1985.125-0) experimental; urgency=medium
 
   * New upstream beta release.
diff -urN ../chromium-browser-36.0.1985.125/debian/control chromium-browser-36.0.1985.125/debian/control
--- ../chromium-browser-36.0.1985.125/debian/control	2014-07-16 09:34:39.000000000 +0900
+++ chromium-browser-36.0.1985.125/debian/control	2014-07-23 19:28:30.000000000 +0900
@@ -10,6 +10,7 @@
 Homepage: http://www.chromium.org/Home
 Build-Depends:
  debhelper (>= 9),
+ libgnome-keyring-dev,
  gyp,
  python3,
  pkg-config,
diff -urN ../chromium-browser-36.0.1985.125/debian/rules chromium-browser-36.0.1985.125/debian/rules
--- ../chromium-browser-36.0.1985.125/debian/rules	2014-07-16 09:39:36.000000000 +0900
+++ chromium-browser-36.0.1985.125/debian/rules	2014-07-23 19:27:39.000000000 +0900
@@ -28,7 +28,6 @@
          use_ozone=0 \
 	 use_gconf=0 \
 	 use_allocator=none \
-         use_gnome_keyring=0 \
 	 linux_breakpad=0 \
          linux_use_libgps=0 \
          linux_use_gold_flags=0 \
@@ -41,6 +40,7 @@
 
 # enabled features
 defines+=use_pulseaudio=1 \
+         use_gnome_keyring=1 \
 	 linux_link_libpci=1 \
          linux_link_gsettings=1 \
          linux_link_libspeechd=1 \
