--- eggdrop-1.6.20.orig/src/eggdrop.h	2014-04-01 20:19:57.906060000 +0200
+++ eggdrop-1.6.20/src/eggdrop.h	2014-04-01 20:20:50.260920850 +0200
@@ -64,7 +64,7 @@
 
 
 /* Language stuff */
-#define LANGDIR  "./language" /* language file directory                   */
+#define LANGDIR  "/usr/share/eggdrop/language" /* language file directory                   */
 #define BASELANG "english"    /* language which always gets loaded before
                                  all other languages. You do not want to
                                  change this.                              */

