commit 79013c353d63565d294b33d5475a31747af2a40c
Author: Mark Hindley <mark@hindley.org.uk>
Date:   Mon Jul 7 10:58:24 2008 +0100

    Fix regexp in header parsing

diff --git a/apt-cacher2 b/apt-cacher2
index 053d81e..87d2725 100755
--- a/apt-cacher2
+++ b/apt-cacher2
@@ -1327,7 +1327,7 @@ sub libcurl {
 			  die "Parse of $_ failed";
 			last SWITCH;
 		    };
-		    /^(\S+): (\S+)$/ && do {
+		    /^(\S+): (.+)$/ && do {
 			debug_message("Got header $_\n");
 			$response->headers->push_header($1, $2);
 			last SWITCH;
@@ -1338,7 +1338,7 @@ sub libcurl {
 			&write_header(\$response) if defined $pkfdref;
 			last SWITCH;
 		    };
-		    debug_message("Warning, unrecognised line in head_callback: $_");
+		    info_message("Warning, unrecognised line in head_callback: $_");
 		}
 	    }
 	    else {
