diff -Naur old/usr/share/texmf/scripts/context/perl/mptopdf.pl new/usr/share/texmf/scripts/context/perl/mptopdf.pl
--- old/usr/share/texmf/scripts/context/perl/mptopdf.pl	2008-05-11 20:27:25.000000000 +0200
+++ new/usr/share/texmf/scripts/context/perl/mptopdf.pl	2008-05-11 20:27:34.000000000 +0200
@@ -120,7 +120,9 @@
 
 foreach my $file (@files) {
     $_ = $file ;
-    if (s/\.(\d+|mps)$// && -e $file) {
+    if (s/\.(\d+|mps|ps)$// && -e $file) {
+        my $suffix = $1 ;
+        my $pdf = basename($_).".pdf" ;
         if ($miktex) {
             $command = "pdftex -undump=mptopdf" ;
         } else {
@@ -132,15 +134,17 @@
             $command = "$command \\\\relax $file" ;
         }
         system($command) ;
-        my $pdfsrc = basename($_).".pdf";
-        rename ($pdfsrc, "$_-$1.pdf") ;
-        if (-e $pdfsrc) {
-            CopyFile ($pdfsrc, "$_-$1.pdf") ;
+        if ($suffix =~ m/\.\d+$/) {
+            rename ($pdf, "$_-$suffix.pdf") ;
+            if (-e $pdf) {
+                CopyFile ($pdf, "$_-$suffix.pdf") ;
+            }
+            $pdf = "$_-$suffix.pdf" ;
         }
         if ($done) {
             $report .= " +" ;
         }
-        $report .= " $_-$1.pdf" ;
+        $report .= " $pdf" ;
         ++$done  ;
     }
 }
