diff -u bind9-9.9.5.dfsg/debian/changelog bind9-9.9.5.dfsg/debian/changelog
--- bind9-9.9.5.dfsg/debian/changelog
+++ bind9-9.9.5.dfsg/debian/changelog
@@ -1,3 +1,12 @@
+bind9 (1:9.9.5.dfsg-4.2) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Disable parallel build.  Closes: #762766
+  * Fix dependencies for libbind-export-udeb.  Closes: #762762
+  * Don't install configuration files to /usr.  Closes: #762948
+
+ -- Michael Gilbert <mgilbert@debian.org>  Sun, 28 Sep 2014 02:56:44 +0000
+
 bind9 (1:9.9.5.dfsg-4.1) unstable; urgency=low
 
   * Non-maintainer upload.
diff -u bind9-9.9.5.dfsg/debian/control bind9-9.9.5.dfsg/debian/control
--- bind9-9.9.5.dfsg/debian/control
+++ bind9-9.9.5.dfsg/debian/control
@@ -186,8 +186,8 @@
 Section: debian-installer
 Architecture: any
 Priority: extra
-Depends: ${shlibs:Depends}
-XC-Package-Type: udeb
+Depends: libc6-udeb
+Package-Type: udeb
 Description: Exported BIND libraries for debian-installer
  libbind-export-udeb is a minimal bind package used by the debian-installer.
 
diff -u bind9-9.9.5.dfsg/debian/rules bind9-9.9.5.dfsg/debian/rules
--- bind9-9.9.5.dfsg/debian/rules
+++ bind9-9.9.5.dfsg/debian/rules
@@ -25,7 +25,7 @@
 
 ifneq (,$(filter parallel=%,$(DEB_BUILD_OPTIONS)))
 NUMJOBS = $(patsubst parallel=%,%,$(filter parallel=%,$(DEB_BUILD_OPTIONS)))
-export MAKEFLAGS += -j$(NUMJOBS)
+# export MAKEFLAGS += -j$(NUMJOBS)
 endif
 
 export CFLAGS=-fno-strict-aliasing -DDIG_SIGCHASE $(DEBUG) $(OPT)
@@ -126,6 +126,7 @@
 	dh_installdirs
 	$(MAKE) -C export install DESTDIR=`pwd`/debian/bind9
 	$(MAKE) install DESTDIR=`pwd`/debian/bind9
+	rm -rf debian/bind9/usr/etc
 	rm -f debian/bind9/usr/lib/*.la
 	install -c -o bin -g bin -m 444 debian/db.0 ${ETCBIND}/db.0
 	install -c -o bin -g bin -m 444 debian/db.0 ${ETCBIND}/db.255
