--- catfish.py.0	2010-06-11 01:51:11.034060921 +0700
+++ catfish.py	2010-06-11 07:29:00.958033927 +0700
@@ -247,8 +247,8 @@
             self.checkbox_find_limit.toggled()
             self.spin_find_limit.set_value(self.options.limit_results)
         self.folder_thumbnails = os.path.expanduser('~/.thumbnails/normal/')
-	self.options.path = os.path.abspath(self.options.path)
-        self.button_find_folder.set_filename(os.path.expanduser(self.options.path))
+        self.options.path = os.path.abspath(os.path.expanduser(self.options.path))
+        self.button_find_folder.set_current_folder(self.options.path)
         try:
             self.link_color = self.treeview_files.style_get_property('link-color')
         except:
