diff -u binstats-1.08/binstats binstats-1.08/binstats
--- binstats-1.08/binstats
+++ binstats-1.08/binstats
@@ -282,7 +282,7 @@
 if [ $FILEOPS -gt 0 ]; then
 
 echo "Look in $EXEDIR for executables"
-find $EXEDIR -perm +111 -maxdepth 1 -type f -follow -print 2>/dev/null| \
+find $EXEDIR -perm /111 -maxdepth 1 -type f -follow -print 2>/dev/null| \
  derefsymlink -s | sort | uniq | \
  awk -F '/' '{ printf "%s:%s\n", $NF, $0 }' > $ETEMPA
 
diff -u binstats-1.08/debian/changelog binstats-1.08/debian/changelog
--- binstats-1.08/debian/changelog
+++ binstats-1.08/debian/changelog
@@ -1,3 +1,10 @@
+binstats (1.08-8.1) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Use "find -perm /x" instead of "find -perm +x". Closes: #808929
+
+ -- Andreas Metzler <ametzler@debian.org>  Fri, 25 Dec 2015 14:32:38 +0100
+
 binstats (1.08-8) unstable; urgency=high
 
   * Fixed call to mktemp (Closes: #303261)
