--- cdbs.orig/simple-patchsys.mk
+++ cdbs.patched/simple-patchsys.mk
@@ -35,7 +35,7 @@
 _cdbs_patch_system_apply_rule := apply-patches
 _cdbs_patch_system_unapply_rule := reverse-patches
 
-DEB_PATCH_SUFFIX ?= .diff .diff.gz .diff.bz2 .patch .patch.gz .patch.bz2 
+DEB_PATCH_SUFFIX ?= .diff .diff.gz .diff.bz2 .diff.uu .patch .patch.gz .patch.bz2 .patch.uu
 DEB_PATCHDIRS ?= debian/patches
 close_parenthesis ?= )
 DEB_PATCHES := $(shell\
@@ -86,6 +86,7 @@
           case $$patch in \
             *.gz) cat=zcat ;; \
             *.bz2) cat=bzcat ;; \
+            *.uu) cat="uudecode -o -" ;; \
             *) cat=cat ;; \
           esac; \
 	  level=$$(head $$patch | egrep '^#DPATCHLEVEL=' | cut -f 2 -d '='); \
