--- /usr/lib/dnssec-trigger/dnssec-trigger-script.orig	2016-02-25 10:17:23.296285495 +0100
+++ /usr/lib/dnssec-trigger/dnssec-trigger-script	2016-02-25 10:34:25.856239132 +0100
@@ -5,6 +5,8 @@
 @author: Pavel Šimerda <psimerda@redhat.com>
 """
 
+import gi
+gi.require_version('NMClient', '1.0')
 from gi.repository import NMClient
 import os, sys, shutil, glob, subprocess
 import logging, logging.handlers
@@ -333,7 +335,7 @@
         except AttributeError:
             self.usage()
         self.config = Config()
-        self.client = NMClient.Client()
+        self.client = NMClient.Client().new()
 
         self.resolvconf = "/etc/resolv.conf"
         self.resolvconf_backup = "/run/dnssec-trigger/resolv.conf.bak"
