From 9d9fc8e0a1b46af0c9bf13e7ff982af1b9e15c07 Mon Sep 17 00:00:00 2001
From: Harlan Lieberman-Berg <hlieberman@setec.io>
Date: Sun, 28 Jan 2024 20:56:58 -0500
Subject: [PATCH] Ensure stock configs are installed (Closes: #1061692)

---
 debian/install | 1 +
 1 file changed, 1 insertion(+)

diff --git a/debian/install b/debian/install
index d2033f8..af982ac 100644
--- a/debian/install
+++ b/debian/install
@@ -2,3 +2,4 @@ chirp/share/chirp.png /usr/share/pixmaps
 chirp/share/chirp.desktop /usr/share/applications
 chirp/share/*.png /usr/share/chirp
 chirp/locale/* /usr/lib/python3/dist-packages/chirp/locale
+chirp/stock_configs /usr/lib/python3/dist-packages/chirp
-- 
2.43.0

