diff -ru bsd-mailx-8.1.2-0.20081101cvs.pristine/debian/control bsd-mailx-8.1.2-0.20081101cvs/debian/control
--- bsd-mailx-8.1.2-0.20081101cvs.pristine/debian/control	2009-02-17 16:03:17.000000000 +0000
+++ bsd-mailx-8.1.2-0.20081101cvs/debian/control	2009-02-17 16:04:24.000000000 +0000
@@ -11,10 +11,9 @@
 Provides: mail-reader, mailx
 Conflicts: mailutils (<< 1:1.1+dfsg1-4), mailx (<< 1:20071201)
 Replaces: mailx (<< 1:20071201)
-Description: A simple mail user agent
+Description: simple mail user agent
  mailx is the traditional command-line-mode mail user agent.
- Even if you don't use it may be required by other programs.
-
+ Even if you don't use it, it may be required by other programs.
  
 Package: mailx
 Priority: extra
