--- debian-changelog-mode.el.~1.42.~	2010-05-12 11:47:48.000000000 +1000
+++ debian-changelog-mode.el	2010-07-01 18:05:31.000000000 +1000
@@ -1774,10 +1774,12 @@
 
 ;;; Setup auto-mode-alist
 ;; (in case /etc/emacs/site-start.d/50dpkg-dev.el not used)
+;;
+;; Crib note: no need for "NEWS.Debian.gz" or "changelog.Debian.gz" entries
+;; since jka-compr.el dispatches using the basename after uncompressing.
 
 (add-to-list 'auto-mode-alist '("/debian/*NEWS" . debian-changelog-mode))
 (add-to-list 'auto-mode-alist '("NEWS.Debian" . debian-changelog-mode))
-(add-to-list 'auto-mode-alist '("NEWS.Debian.gz" . debian-changelog-mode))
 
 ;;(add-to-list 'auto-mode-alist '("/debian/changelog\\'" . debian-changelog-mode))
 ;;; Instead use this.  See http://bugs.debian.org/457047 by Trent W. Buck
@@ -1788,16 +1790,13 @@
    . debian-changelog-mode))
 
 (add-to-list 'auto-mode-alist '("changelog.Debian" . debian-changelog-mode))
-(add-to-list 'auto-mode-alist '("changelog.Debian.gz" . debian-changelog-mode))
   ;; For debchange
 (add-to-list 'auto-mode-alist '("changelog.dch" . debian-changelog-mode))
 
 ;;;###autoload(add-to-list 'auto-mode-alist '("/debian/*NEWS" . debian-changelog-mode))
 ;;;###autoload(add-to-list 'auto-mode-alist '("NEWS.Debian" . debian-changelog-mode))
-;;;###autoload(add-to-list 'auto-mode-alist '("NEWS.Debian.gz" . debian-changelog-mode))
 ;;;###autoload(add-to-list 'auto-mode-alist '("/debian/\\([[:lower:][:digit:]][[:lower:][:digit:].+-]+\\.\\)?changelog\\'" . debian-changelog-mode))
 ;;;###autoload(add-to-list 'auto-mode-alist '("changelog.Debian" . debian-changelog-mode))
-;;;###autoload(add-to-list 'auto-mode-alist '("changelog.Debian.gz" . debian-changelog-mode))
 ;;;###autoload(add-to-list 'auto-mode-alist '("changelog.dch" . debian-changelog-mode))
 
 (provide 'debian-changelog-mode)
