--- /home/mojo/src/aptitude-0.7.5/src/cmdline/cmdline_action.cc	2015-12-08 01:32:51.000000000 +0100
+++ /tmp/diffu_tmp	2016-01-23 10:57:45.408131025 +0100
@@ -607,7 +607,7 @@
 	    {
 	      // Don't overwhelm the user.
 	      if (possible.size() > package_limit)
-	        printf(_("Couldn't find package \"%s\", and more than %zu\npackages contain \"%s\" in their name.\n"),
+	        printf(_("Couldn't find package \"%s\", and more than %zu\npackages containing \"%s\" in their name.\n"),
 		       package.c_str(), package_limit, package.c_str());
 	      else
 		{
@@ -636,7 +636,7 @@
 		printf(_("Couldn't find any package whose name or description matched \"%s\"\n"), package.c_str());
 	      else if (possible.size() > package_limit)
 		// Don't overwhelm the user.
-		printf(_("Couldn't find any package matching \"%s\", and more than %zu\npackages contain \"%s\" in their description.\n"),
+		printf(_("Couldn't find any package matching \"%s\", and more than %zu\npackages containing \"%s\" in their description.\n"),
 		       package.c_str(), package_limit, package.c_str());
 	      else
 		{
