From 5715f4648fb1d3b8d9ac462f62253f9782f50cf4 Mon Sep 17 00:00:00 2001
From: A Mennucc <mennucc1@debian.org>
Date: Mon, 24 Sep 2012 11:53:22 +0200
Subject: [PATCH] debpatch, debdelta-upgrade : do not get confused by broken
 symlinks

---
 debdelta/debdelta |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/debdelta/debdelta b/debdelta/debdelta
index b5eebad..9d921d6 100755
--- a/debdelta/debdelta
+++ b/debdelta/debdelta
@@ -1627,6 +1627,9 @@ def do_patch_(delta, olddeb, newdeb, TD, runtime,
           file_triples.append((orig,divert,z))
         else:
           file_triples.append((orig,divert,None))
+      elif not os.path.exists(divert) and os.path.islink(divert):
+        file_triples.append((orig,divert,None))
+        if VERBOSE > 1 : print '  Broken symlink? ',divert
       elif not os.path.exists(divert):
         file_triples.append((orig,divert,None))
         if VERBOSE : print ' Disappeared file? ',divert
-- 
1.7.10.4

