diff -Nru asterisk-13.10.0~dfsg/debian/asterisk.install asterisk-13.10.0~dfsg/debian/asterisk.install
--- asterisk-13.10.0~dfsg/debian/asterisk.install	2016-08-29 17:07:40.000000000 +0200
+++ asterisk-13.10.0~dfsg/debian/asterisk.install	2016-07-31 10:55:45.000000000 +0200
@@ -4,5 +4,4 @@
 usr/share/asterisk/keys
 usr/share/asterisk/conf
 usr/share/asterisk/documentation
-usr/share/asterisk/rest-api
 lib/systemd/system/asterisk.service
