Index: debian/apparmor-profile.abstraction
===================================================================
--- debian/apparmor-profile.abstraction	(revision 51782)
+++ debian/apparmor-profile.abstraction	(working copy)
@@ -46,8 +46,8 @@
   /usr/bin/dvipdfmx Cx -> sanitized_helper,
 
   # supported archivers
-  /bin/gzip ixr,
-  /bin/bzip2 ixr,
+  /{usr/,}bin/gzip ixr,
+  /{usr/,}bin/bzip2 ixr,
   /usr/bin/unrar* ixr,
   /usr/bin/unzip ixr,
   /usr/bin/7zr ixr,
@@ -55,7 +55,7 @@
   /usr/bin/7za ixr,
   /usr/lib/p7zip/7za ixr,
   /usr/bin/zipnote ixr,
-  /bin/tar ixr,
+  /{usr/,}bin/tar ixr,
   /usr/bin/xz ixr,
 
   # allow read access to anything in /usr/share, for plugins and input methods
