From 3d353f41e05f7ce2a925960a9dcbcaaab055752b Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?David=20Pr=C3=A9vot?= <taffit@debian.org>
Date: Sat, 15 Feb 2014 20:59:17 -0400
Subject: [PATCH] uscan: Honor Referer

---
 scripts/uscan.pl | 1 +
 1 file changed, 1 insertion(+)

diff --git a/scripts/uscan.pl b/scripts/uscan.pl
index 4438e25..198cdb1 100755
--- a/scripts/uscan.pl
+++ b/scripts/uscan.pl
@@ -1377,6 +1377,7 @@ EOF
 	    print STDERR "$progname debug: requesting URL $url\n" if $debug;
 	    my $headers = HTTP::Headers->new;
 	    $headers->header('Accept' => '*/*');
+	    $headers->header('Referer' => $base);
 	    $request = HTTP::Request->new('GET', $url, $headers);
 	    $response = $user_agent->request($request, $fname);
 	    if (! $response->is_success) {
-- 
1.9.0.rc3

