diff -ru blobwars-1.05/src/CEngine.cpp /home/daniel/src/blobwars-1.05/src/CEngine.cpp
--- blobwars-1.05/src/CEngine.cpp	2006-12-10 12:21:28.000000000 -0800
+++ /home/daniel/src/blobwars-1.05/src/CEngine.cpp	2006-12-10 12:12:18.000000000 -0800
@@ -1096,6 +1096,20 @@
 		// joystick navigation is impossible.
 		lockJoy(300);
 	}
+	else if(keyState[SDLK_DELETE] || keyState[SDLK_BACKSPACE])
+	{
+		if(highlightedWidget->value == NULL)
+		{
+			debug(("%s has not been implemented!\n", highlightedWidget->name));
+		}
+		else
+		{
+			if (highlightedWidget->type == WG_JOYPAD)
+			{
+				*highlightedWidget->value = -1;
+			}
+		}
+	}
 
 	return update;
 }
diff -ru blobwars-1.05/src/widgets.cpp /home/daniel/src/blobwars-1.05/src/widgets.cpp
--- blobwars-1.05/src/widgets.cpp	2005-05-03 14:05:40.000000000 -0700
+++ /home/daniel/src/blobwars-1.05/src/widgets.cpp	2006-12-10 12:20:28.000000000 -0800
@@ -154,6 +154,10 @@
 	{
 		sprintf(text, "...");
 	}
+	else if(*widget->value == -1)
+	{
+		sprintf(text, "No Binding");
+	}
 	else
 	{
 		sprintf(text, "Button #%d", *widget->value);
