Index: debian/control
===================================================================
--- debian/control	(revision 1192)
+++ debian/control	(working copy)
@@ -47,6 +47,7 @@
     dupload | dput, ssh]
   - cvs-debuild: run cvs-buildpackage using debuild (see below) as the
     package building program [cvs-buildpackage, fakeroot, lintian, gnupg]
+  - dcmd: expand file lists of .dsc/.changes files in the command line
   - dd-list: given a list of packages, pretty-print it ordered by maintainer
   - debc: display the contents of just-built .debs
   - debchange/dch: automagically add entries to debian/changelog files
Index: po4a/devscripts-po4a.conf
===================================================================
--- po4a/devscripts-po4a.conf	(revision 1192)
+++ po4a/devscripts-po4a.conf	(working copy)
@@ -29,6 +29,8 @@
 	fr:fr/cvs-debrelease.fr.1 add_fr:add_fr/translator_man.add
 [type:man] ../scripts/cvs-debuild.1 \
 	fr:fr/cvs-debuild.fr.1 add_fr:add_fr/translator_man.add
+[type:man] ../scripts/dcmd.1 \
+	fr:fr/dcmd.fr.1 add_fr:add_fr/translator_man.add
 [type:man] ../scripts/dd-list.1 \
 	fr:fr/dd-list.fr.1 add_fr:add_fr/translator_man.add
 [type:man] ../scripts/debc.1 \
Index: README
===================================================================
--- README	(revision 1192)
+++ README	(working copy)
@@ -48,6 +48,8 @@
   wrapper for cvs-buildpackage to use debuild as its package building
   program.
 
+- dcmd: Expand file lists of .dsc/.changes files in the command line.
+
 - dd-list: Given a list of packages, pretty-print it ordered by maintainer.
 
 - debc: List contents of current package.  Do this after a successful
Index: conf.default.in
===================================================================
--- conf.default.in	(revision 1192)
+++ conf.default.in	(working copy)
@@ -125,6 +125,10 @@
 # 
 # No variables currently
 
+##### dcmd
+# 
+# No variables currently
+
 ##### debc
 # 
 # debc recognises the DEBRELEASE_DEBS_DIR variable; see debrelease
