diff -Nur desktop-file-utils-0.10/src/eggdesktopentries.c desktop-file-utils-0.10.new/src/eggdesktopentries.c
--- desktop-file-utils-0.10/src/eggdesktopentries.c	2004-10-18 18:02:46.000000000 +0200
+++ desktop-file-utils-0.10.new/src/eggdesktopentries.c	2006-05-11 12:00:11.000000000 +0200
@@ -1441,7 +1441,7 @@
 
   last_char_index = strlen (value) - 1;
 
-  if (value[last_char_index] == ';')
+  if (last_char_index >= 0 && value[last_char_index] == ';')
     value[last_char_index] = '\0';
 
   value_vector = g_strsplit (value, ";", 0);
@@ -2784,7 +2784,7 @@
       p++;
     }
 
-  if (p[-1] == '\\' && error == NULL)
+  if (p > value && p[-1] == '\\' && error == NULL)
     {
       g_set_error (error, EGG_DESKTOP_ENTRIES_ERROR,
                    EGG_DESKTOP_ENTRIES_ERROR_INVALID_VALUE,
diff -Nur desktop-file-utils-0.10/src/update-desktop-database.c desktop-file-utils-0.10.new/src/update-desktop-database.c
--- desktop-file-utils-0.10/src/update-desktop-database.c	2004-09-08 17:43:14.000000000 +0200
+++ desktop-file-utils-0.10.new/src/update-desktop-database.c	2006-05-11 11:59:59.000000000 +0200
@@ -449,6 +449,8 @@
   for (i = 0; data_dirs[i] != NULL; i++)
     args[i] = g_build_filename (data_dirs[i], "applications", NULL);
 
+  args[i] = NULL;
+
   g_strfreev (data_dirs);
 
   return (const char **) args;
