From f9be1eeaf1e9650fe2d4929917533f43fd260552 Mon Sep 17 00:00:00 2001
From: Jan Dittberner <jan@dittberner.info>
Date: Fri, 19 Feb 2010 17:52:19 +0100
Subject: [PATCH 1/2] spring contains x86 only assembly code

---
 Packages-arch-specific |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/Packages-arch-specific b/Packages-arch-specific
index d8d0f53..0395baa 100644
--- a/Packages-arch-specific
+++ b/Packages-arch-specific
@@ -329,6 +329,7 @@ silo: sparc							      # sparc lilo
 %sparc-utils: sparc                                                   # sparc specific
 %spicctrl: i386							      # For Sony VAIO laptops
 %splay: i386 alpha arm armel m68k powerpc			      # [?] broken on sparc (see BTS]
+%spring: i386 amd64 kfreebsd-i386 kfreebsd-amd64 hurd-i386	      # x86 specific
 %srm-reader: alpha						      # Alpha specific udeb
 %stalin: i386 amd64 ia64 sparc kfreebsd-i386 kfreebsd-amd64	      # [ANAIS]
 subterfugue: i386						      # i386 specific process tracing etc.
-- 
1.6.6.1

