diff --git a/index.dbk b/index.dbk
index ea3d33f..2068638 100644
--- a/index.dbk
+++ b/index.dbk
@@ -86,14 +86,13 @@ it by writing to the &fsf-addr;.
 </para>
 <para>
 If you want to print this reference, you should use the <ulink
-url="developers-reference.pdf">pdf version</ulink>.  This page is also
-available in <ulink url="index.fr.html">French</ulink>, <ulink
+url="developers-reference.pdf">pdf version</ulink>.  This manual is
+available in <ulink url="index.en.html">English</ulink>,
+<ulink url="index.fr.html">French</ulink>, <ulink
 url="index.de.html">German</ulink>, <ulink
 url="index.it.html">Italian</ulink>, <ulink
 url="index.ru.html">Russian</ulink>, and <ulink
 url="index.ja.html">Japanese</ulink>.
-<!-- TODO: Maybe better: "This document has originally been written
-in English.  Translations into different languages are available." -->
 </para>
 </legalnotice>
 <!-- toc -->
