diff -Nru ebtables-2.0.10.4/debian/changelog ebtables-2.0.10.4/debian/changelog
--- ebtables-2.0.10.4/debian/changelog	2013-03-20 16:30:15.000000000 +0000
+++ ebtables-2.0.10.4/debian/changelog	2013-05-05 20:11:30.000000000 +0100
@@ -1,3 +1,9 @@
+ebtables (2.0.10.4-3) UNRELEASED; urgency=low
+
+  * Fix update-rc.d call to include trailing ".". 
+
+ -- Iain Lane <laney@debian.org>  Sun, 05 May 2013 20:11:07 +0100
+
 ebtables (2.0.10.4-2) unstable; urgency=low
 
   * packaging update:
diff -Nru ebtables-2.0.10.4/debian/rules ebtables-2.0.10.4/debian/rules
--- ebtables-2.0.10.4/debian/rules	2013-03-15 14:21:39.000000000 +0000
+++ ebtables-2.0.10.4/debian/rules	2013-05-05 20:11:07.000000000 +0100
@@ -10,7 +10,7 @@
 CFLAGS+=$(HARDENING_CFLAGS) -fstack-protector-all
 LDFLAGS+=$(HARDENING_LDFLAGS)
 
-DEB_DH_INSTALLINIT_ARGS:=-- start 20 S . stop 80 0 1 6
+DEB_DH_INSTALLINIT_ARGS:=-- start 20 S . stop 80 0 1 6 .
 
 build/ebtables::
 	make CFLAGS="$(CFLAGS)" $(MAKE_PATH_REDIRECTIONS)
