--- dpkg-1.15.7.2+nmu0anbe1/scripts/dpkg-divert.pl	2010-05-19 07:55:01.000000000 +0200
+++ dpkg-1.15.7.2+nmu0anbe2/scripts/dpkg-divert.pl	2010-07-07 14:51:29.000000000 +0200
@@ -269,6 +269,13 @@
     ($rsrc,$rdest) = @_;
     (@ssrc = lstat($rsrc)) || $! == ENOENT ||
         quit(sprintf(_g("cannot stat old name \`%s': %s"), $rsrc, $!));
+    unless(@ssrc) {
+	# If the source file is not present and we are not going to do the
+	# rename anyway there's no point in checking the target or the
+	# write permissions of any directory.
+	$dorename = !$dorename;
+	return;
+    }
     (@sdest = lstat($rdest)) || $! == ENOENT ||
         quit(sprintf(_g("cannot stat new name \`%s': %s"), $rdest, $!));
     # Unfortunately we have to check for write access in both
@@ -281,11 +288,6 @@
     if (open (TMP, ">>", "${rsrc}.dpkg-devert.tmp")) {
 	close TMP;
 	unlink ("${rsrc}.dpkg-devert.tmp");
-    } elsif ($! == ENOENT) {
-	$dorename = !$dorename;
-	# If the source file is not present and we are not going to do the
-	# rename anyway there's no point in checking the target.
-	return;
     } else {
 	quit(sprintf(_g("error checking \`%s': %s"), $rsrc, $!));
     }
