--- cream.orig	2004-08-15 16:36:10.000000000 +0200
+++ cream	2004-08-15 16:36:49.000000000 +0200
@@ -10,10 +10,10 @@
 export CREAM
 
 # If this script is called as 'gcream' then try to run gvim
-if [ $(basename $0) == "gcream" -a -x /usr/bin/gvim -a "$DISPLAY" != "" ] ; then
+if [ $(basename $0) = "gcream" ] && [ -x /usr/bin/gvim ] && [ "$DISPLAY" != "" ] ; then
 	exec /usr/bin/gvim -U NONE -u "\$VIM/cream/creamrc" "$@"
 # If this script is called as 'kcream' then try to run kvim
-elif [ $(basename $0) == "kcream" -a -x /usr/bin/kvim -a "$DISPLAY" != "" ] ; then
+elif [ $(basename $0) = "kcream" ] && [ -x /usr/bin/kvim ] && [ "$DISPLAY" != "" ] ; then
 	exec /usr/bin/kvim -U NONE -u "\$VIM/cream/creamrc" "$@"
 # ...otherwise try 'vim' (e.g. on a console)
 else
