diff -Nru djvulibre-3.5.21/debian/rules djvulibre-3.5.21/debian/rules
--- djvulibre-3.5.21/debian/rules	2008-11-14 05:13:56.000000000 +0000
+++ djvulibre-3.5.21/debian/rules	2008-11-14 05:13:56.000000000 +0000
@@ -127,7 +127,8 @@
 	@echo remove from djvulibre-bin, just links
 	find debian/djvulibre-bin -name djview\* -print -delete
 	@echo remove from djvulibre-bin, already in djvulibre-plugin
-	rm debian/djvulibre-bin/usr/share/man/{.,*}/man*/nsdejavu.1
+	rm debian/djvulibre-bin/usr/share/man/*/man*/nsdejavu.1
+	rm debian/djvulibre-bin/usr/share/man/man*/nsdejavu.1
 	@echo remove from djvulibre-bin, already in djvuserve
 	find debian/djvulibre-bin -name djvuserve\* -print -delete
 # 	@echo remove from djvulibre-bin, not in Debian: needs gsdjvu, direct complaints to ATT
