*** easytag-2.1.4-orig/src/cddb.c	2007-11-30 02:09:49.000000000 +0300
--- easytag-2.1.4/src/cddb.c	2009-06-04 23:39:34.000000000 +0400
*************** gboolean Cddb_Search_Album_List_From_Str
*** 2527,2533 ****
      // Do a loop to load all the pages of results
      do
      {
!         cddb_server_name     = g_strdup(CDDB_SERVER_NAME_MANUAL_SEARCH);    //"www.gnudb.org");
          cddb_server_port     = CDDB_SERVER_PORT_MANUAL_SEARCH;              //80;
          cddb_server_cgi_path = g_strdup(CDDB_SERVER_CGI_PATH_MANUAL_SEARCH);//"/~cddb/cddb.cgi");
  
--- 2527,2533 ----
      // Do a loop to load all the pages of results
      do
      {
!         cddb_server_name     = g_strdup_printf("www.%s", CDDB_SERVER_NAME_MANUAL_SEARCH);    //"www.gnudb.org");
          cddb_server_port     = CDDB_SERVER_PORT_MANUAL_SEARCH;              //80;
          cddb_server_cgi_path = g_strdup(CDDB_SERVER_CGI_PATH_MANUAL_SEARCH);//"/~cddb/cddb.cgi");
  
*************** gboolean Cddb_Search_Album_List_From_Str
*** 2676,2682 ****
                  cddbalbum = g_malloc0(sizeof(CddbAlbum));
  
                  // Parameters of the server used
!                 cddbalbum->server_name     = g_strdup(cddb_server_name);
                  cddbalbum->server_port     = cddb_server_port;
                  cddbalbum->server_cgi_path = g_strdup(cddb_server_cgi_path);
                  cddbalbum->bitmap          = Cddb_Get_Pixbuf_From_Server_Name(cddbalbum->server_name);
--- 2676,2682 ----
                  cddbalbum = g_malloc0(sizeof(CddbAlbum));
  
                  // Parameters of the server used
!                 cddbalbum->server_name     = g_strdup_printf("gnudb.%s", CDDB_SERVER_NAME_MANUAL_SEARCH);
                  cddbalbum->server_port     = cddb_server_port;
                  cddbalbum->server_cgi_path = g_strdup(cddb_server_cgi_path);
                  cddbalbum->bitmap          = Cddb_Get_Pixbuf_From_Server_Name(cddbalbum->server_name);
