--- trunk/src/media/media.gyp	2012/02/04 16:53:31	120484
+++ trunk/src/media/media.gyp	2012/02/04 16:53:39	120485
@@ -467,14 +467,6 @@
         [ 'os_posix == 1 and OS != "mac" and OS != "android"', {
           'cflags': [
             '-msse2',
-            '-msse3',
-            '-mssse3',
-          ],
-        }],
-        [ 'OS == "openbsd"', {
-          # OpenBSD's gcc (4.2.1) does not support -mssse3
-          'cflags!': [
-            '-mssse3',
           ],
         }],
         [ 'OS == "mac"', {
