Index: bluez-tools-0.1.38+git662e/src/Makefile.am
===================================================================
--- bluez-tools-0.1.38+git662e.orig/src/Makefile.am	2011-05-07 17:06:38.000000000 +0200
+++ bluez-tools-0.1.38+git662e/src/Makefile.am	2011-05-07 17:07:10.000000000 +0200
@@ -1,5 +1,5 @@
 AM_CPPFLAGS = $(DBUS_CFLAGS) $(GLIB_CFLAGS) $(DBUS_GLIB_CFLAGS)
-AM_LDFLAGS = $(DBUS_LIBS) $(GLIB_LIBS) $(DBUS_GLIB_LIBS)
+LDADD = $(DBUS_LIBS) $(GLIB_LIBS) $(DBUS_GLIB_LIBS)
 
 # Marshallers generation
 #BUILT_SOURCES = lib/marshallers.c lib/marshallers.h
@@ -57,7 +57,7 @@
 if OBEX
 bin_PROGRAMS += bt-obex
 bt_obex_SOURCES = $(lib_sources) $(bluez_sources) $(obexd_sources) bt-obex.c
-bt_obex_LDADD = $(LIBREADLINE)
+bt_obex_LDADD = $(LDADD) $(LIBREADLINE)
 endif
 
 dist_man_MANS = bt-monitor.1 bt-adapter.1 bt-agent.1 bt-device.1 bt-input.1 bt-audio.1 bt-network.1 bt-serial.1
Index: bluez-tools-0.1.38+git662e/src/Makefile.in
===================================================================
--- bluez-tools-0.1.38+git662e.orig/src/Makefile.in	2011-05-07 17:07:16.000000000 +0200
+++ bluez-tools-0.1.38+git662e/src/Makefile.in	2011-05-07 17:07:58.000000000 +0200
@@ -419,7 +419,7 @@
 top_builddir = @top_builddir@
 top_srcdir = @top_srcdir@
 AM_CPPFLAGS = $(DBUS_CFLAGS) $(GLIB_CFLAGS) $(DBUS_GLIB_CFLAGS)
-AM_LDFLAGS = $(DBUS_LIBS) $(GLIB_LIBS) $(DBUS_GLIB_LIBS)
+LDADD = $(DBUS_LIBS) $(GLIB_LIBS) $(DBUS_GLIB_LIBS)
 
 # Marshallers generation
 #BUILT_SOURCES = lib/marshallers.c lib/marshallers.h
@@ -462,7 +462,7 @@
 bt_network_SOURCES = $(lib_sources) $(bluez_sources) $(obexd_sources) bt-network.c
 bt_serial_SOURCES = $(lib_sources) $(bluez_sources) $(obexd_sources) bt-serial.c
 @OBEX_TRUE@bt_obex_SOURCES = $(lib_sources) $(bluez_sources) $(obexd_sources) bt-obex.c
-@OBEX_TRUE@bt_obex_LDADD = $(LIBREADLINE)
+@OBEX_TRUE@bt_obex_LDADD = $(LDADD) $(LIBREADLINE)
 dist_man_MANS = bt-monitor.1 bt-adapter.1 bt-agent.1 bt-device.1 \
 	bt-input.1 bt-audio.1 bt-network.1 bt-serial.1 $(am__append_3)
 all: all-am
