Index: acpid-1.0.6/debian/acpid.init.d
===================================================================
--- acpid-1.0.6.orig/debian/acpid.init.d	2007-11-02 21:02:53.000000000 +0000
+++ acpid-1.0.6/debian/acpid.init.d	2007-11-02 21:07:31.000000000 +0000
@@ -43,12 +43,17 @@
 
 	# Get list of available modules
         LOC="/lib/modules/`uname -r`/kernel/drivers/acpi"
+        LOC2="/lib/modules/`uname -r`/ubuntu/drivers/acpi"
         if [ -d $LOC ]; then
 	  MODAVAIL=`( find $LOC -type f -name "*.o" -printf "basename %f .o\n"; \
 		find $LOC -type f -name "*.ko" -printf "basename %f .ko\n" ) | /bin/sh`
 	else
 	  MODAVAIL=""
 	fi
+        if [ -d $LOC2 ]; then
+            MODAVAIL="$MODAVAIL `( find $LOC2 -type f -name "*.o" -printf "basename %f .o\n"; \
+                find $LOC2 -type f -name "*.ko" -printf "basename %f .ko\n" ) | /bin/sh`"
+        fi
 
         if [ "$MODULES" = "all" ]; then
 		MODULES="$MODAVAIL"
