From c5b766bc47a4e07dced13903409a32172fd547b1 Mon Sep 17 00:00:00 2001
From: Sebastian Andrzej Siewior <sebastian@breakpoint.cc>
Date: Fri, 16 Jan 2015 21:07:23 +0100
Subject: [PATCH] dupload.conf: make ftp-master use passive FTP

Signed-off-by: Sebastian Andrzej Siewior <sebastian@breakpoint.cc>
---
 dupload.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/dupload.conf b/dupload.conf
index adb14340ec13..8d58ccf553a7 100644
--- a/dupload.conf
+++ b/dupload.conf
@@ -74,6 +74,7 @@ $cfg{'anonymous-ftp-master'} = {
 	incoming => "/pub/UploadQueue/",
 	# The dinstall on ftp-master sends emails itself
 	dinstall_runs => 1,
+	passive => 1,
 };
 
 # see http://backports.debian.org/Contribute/ for more information
-- 
2.1.4

