--- dupload.conf~	2011-02-24 00:41:58.000000000 +0100
+++ dupload.conf	2016-01-14 07:08:23.975176394 +0100
@@ -74,6 +74,7 @@
 	incoming => "/pub/UploadQueue/",
 	# The dinstall on ftp-master sends emails itself
 	dinstall_runs => 1,
+	passive => 1,
 };
 
 # see http://backports.debian.org/Contribute/ for more information
@@ -81,6 +82,7 @@
     fqdn => "backports-master.debian.org",
     incoming => "/pub/UploadQueue/",
     dinstall_runs => 1,
+    passive => 1,
 };
 
 # see http://www.debian.org/devel/debian-volatile/ for more information
@@ -100,6 +102,7 @@
 	incoming => "/pub/UploadQueue/DELAYED/$delay-day/",
 	# The dinstall on ftp-master sends emails itself
 	dinstall_runs => 1,
+	passive => 1,
 };
 
 # Mentors upload queue, see
@@ -132,6 +135,7 @@
 #	# files pass on to dinstall on security which sends emails itself
 #	dinstall_runs => 1,
 #	nonus => 1,
+#	passive => 1,
 #};
 
 
