diff -urp debian-policy-3.7.2.2.old/policy.sgml debian-policy-3.7.2.2/policy.sgml
--- debian-policy-3.7.2.2.old/policy.sgml	2006-10-03 00:36:50.000000000 +0200
+++ debian-policy-3.7.2.2/policy.sgml	2007-11-23 08:21:25.000000000 +0100
@@ -695,9 +695,8 @@
 		    trying to produce, amongst other things, a free
 		    Unix.
 		</footnote>
-		Other packages without which the system will not run
-		well or be usable must also have priority
-		<tt>important</tt>.  This does
+		Other packages needed to make the system usable
+		must also have priority	<tt>important</tt>.  This does
 		<em>not</em> include Emacs, the X Window System, TeX
 		or any other large applications.  The
 		<tt>important</tt> packages are just a bare minimum of
