diff -u distcc-2.18.3/debian/distcc.init.d distcc-2.18.3/debian/distcc.init.d
--- distcc-2.18.3/debian/distcc.init.d
+++ distcc-2.18.3/debian/distcc.init.d
@@ -128,9 +128,12 @@
 	}
 	log_end_msg 0
 	;;
+  status)
+	status_of_proc $DAEMON $NAME
+	;;
   *)
 	N=/etc/init.d/$NAME
-	echo "Usage: $N {start|stop|restart|force-reload}" >&2
+	echo "Usage: $N {start|stop|restart|force-reload|status}" >&2
 	exit 1
 	;;
 esac
diff -u distcc-2.18.3/debian/control distcc-2.18.3/debian/control
--- distcc-2.18.3/debian/control
+++ distcc-2.18.3/debian/control
@@ -7,7 +7,7 @@
 
 Package: distcc
 Architecture: any
-Depends: ${shlibs:Depends}, adduser (>= 3.52), debconf (>= 1.2.0) | debconf-2.0, netbase (>= 4.09), dbus, lsb-base
+Depends: ${shlibs:Depends}, adduser (>= 3.52), debconf (>= 1.2.0) | debconf-2.0, netbase (>= 4.09), dbus, lsb-base (>= 3.2-13)
 Suggests: distccmon-gnome, ccache
 Description: Simple distributed compiler client and server
  distcc is a program to distribute compilation of C or C++ code across
