--- fai-cd.orig	2011-04-01 09:33:29.000000000 +0200
+++ fai-cd	2011-04-01 09:34:15.000000000 +0200
@@ -275,10 +275,10 @@
 
     elif [ "$grub_version" -eq 2 ]; then
     # detect device of mounted usb stick, grub2
-    device=$(grub-probe -tdrive $usbdir | perl -ane 'm#(/dev/\w+),# && print "$1\n"')
+    device=$(grub-probe -tdrive $usbdir | perl -ane 'm#(\w+),# && print "($1)\n"')
     echo "Installing grub2 to $device."
     # this call seems to remove file and copies them again onto the stick
-    grub-install --no-floppy --root-directory=$usbdir $device
+    grub-install --no-floppy --root-directory=$usbdir "$device"
     fi
 }
 # - - - - - - - - - - - - - - - - - - - - - - - - - -
