--- a/configure.ac
+++ b/configure.ac
@@ -887,12 +887,12 @@ case $host_os in
       WINE_TRY_CFLAGS([-fPIC -Wl,--export-dynamic],
                       [LDEXECFLAGS="-Wl,--export-dynamic"])
 
-      WINE_TRY_CFLAGS([-fPIC -Wl,--rpath,\$ORIGIN/../lib],
-                      [LDRPATH_INSTALL="-Wl,--rpath,\\\$\$ORIGIN/\`\$(MAKEDEP) -R \${bindir} \${libdir}\`"
-                       LDRPATH_LOCAL="-Wl,--rpath,\\\$\$ORIGIN/\$(top_builddir)/libs/wine"],
-          [WINE_TRY_CFLAGS([-fPIC -Wl,-R,\$ORIGIN/../lib],
-                           [LDRPATH_INSTALL="-Wl,-R,\\\$\$ORIGIN/\`\$(MAKEDEP) -R \${bindir} \${libdir}\`"
-                            LDRPATH_LOCAL="-Wl,-R,\\\$\$ORIGIN/\$(top_builddir)/libs/wine"])])
+      WINE_TRY_CFLAGS([-fPIC -Wl,--rpath,./lib],
+                      [LDRPATH_INSTALL="-Wl,--rpath,\`\$(MAKEDEP) -R \${bindir} \${libdir}\`"
+                       LDRPATH_LOCAL="-Wl,--rpath,\$(top_builddir)/libs/wine"],
+          [WINE_TRY_CFLAGS([-fPIC -Wl,-R,./lib],
+                           [LDRPATH_INSTALL="-Wl,-R,\`\$(MAKEDEP) -R \${bindir} \${libdir}\`"
+                            LDRPATH_LOCAL="-Wl,-R,\$(top_builddir)/libs/wine"])])
 
       WINE_TRY_CFLAGS([-Wl,--enable-new-dtags],
                       [LDRPATH_INSTALL="$LDRPATH_INSTALL -Wl,--enable-new-dtags"])
