diff -Nru emacs24-24.4+1/debian/changelog emacs24-24.4+1/debian/changelog
--- emacs24-24.4+1/debian/changelog	2014-12-09 16:25:45.000000000 +0100
+++ emacs24-24.4+1/debian/changelog	2015-03-07 10:36:39.000000000 +0100
@@ -1,3 +1,12 @@
+emacs24 (24.4+1-4.2) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * debian/control: add Breaks against wheezy version of apel. This
+    fixes partial upgrades when emacs24 is installed before apel is
+    upgraded. (Closes: #775564)
+
+ -- Sébastien Villemot <sebastien@debian.org>  Sat, 07 Mar 2015 10:34:14 +0100
+
 emacs24 (24.4+1-4.1) unstable; urgency=medium
 
   * Non-maintainer upload.
diff -Nru emacs24-24.4+1/debian/control emacs24-24.4+1/debian/control
--- emacs24-24.4+1/debian/control	2014-12-09 16:27:41.000000000 +0100
+++ emacs24-24.4+1/debian/control	2015-03-07 10:59:24.000000000 +0100
@@ -22,6 +22,7 @@
 Suggests: emacs24-common-non-dfsg
 Conflicts: emacs24, emacs24-nox
 Replaces: emacs24, emacs24-nox
+Breaks: apel (<< 10.8+0.20120427-4)
 Description: GNU Emacs editor (with Lucid GUI support)
  GNU Emacs is the extensible self-documenting text editor.  This
  package contains a version of Emacs with support for a graphical user
@@ -50,6 +51,7 @@
 Suggests: emacs24-common-non-dfsg
 Conflicts: emacs24, emacs24-lucid
 Replaces: emacs24, emacs24-lucid
+Breaks: apel (<< 10.8+0.20120427-4)
 Description: GNU Emacs editor (without GUI support)
  GNU Emacs is the extensible self-documenting text editor.  This
  package contains a version of Emacs compiled without support for X,
@@ -72,6 +74,7 @@
 Suggests: emacs24-common-non-dfsg
 Conflicts: emacs24-lucid, emacs24-nox
 Replaces: emacs24-lucid, emacs24-nox
+Breaks: apel (<< 10.8+0.20120427-4)
 Description: GNU Emacs editor (with GTK+ GUI support)
  GNU Emacs is the extensible self-documenting text editor.  This
  package contains a version of Emacs with a graphical user interface
diff -Nru emacs24-24.4+1/debian/control.in emacs24-24.4+1/debian/control.in
--- emacs24-24.4+1/debian/control.in	2014-10-25 21:41:42.000000000 +0200
+++ emacs24-24.4+1/debian/control.in	2015-03-07 10:59:07.000000000 +0100
@@ -22,6 +22,7 @@
 Suggests: @DEB_FLAVOR@-common-non-dfsg
 Conflicts: @DEB_FLAVOR@, @DEB_FLAVOR@-nox
 Replaces: @DEB_FLAVOR@, @DEB_FLAVOR@-nox
+Breaks: apel (<< 10.8+0.20120427-4)
 Description: GNU Emacs editor (with Lucid GUI support)
  GNU Emacs is the extensible self-documenting text editor.  This
  package contains a version of Emacs with support for a graphical user
@@ -50,6 +51,7 @@
 Suggests: @DEB_FLAVOR@-common-non-dfsg
 Conflicts: @DEB_FLAVOR@, @DEB_FLAVOR@-lucid
 Replaces: @DEB_FLAVOR@, @DEB_FLAVOR@-lucid
+Breaks: apel (<< 10.8+0.20120427-4)
 Description: GNU Emacs editor (without GUI support)
  GNU Emacs is the extensible self-documenting text editor.  This
  package contains a version of Emacs compiled without support for X,
@@ -72,6 +74,7 @@
 Suggests: @DEB_FLAVOR@-common-non-dfsg
 Conflicts: @DEB_FLAVOR@-lucid, @DEB_FLAVOR@-nox
 Replaces: @DEB_FLAVOR@-lucid, @DEB_FLAVOR@-nox
+Breaks: apel (<< 10.8+0.20120427-4)
 Description: GNU Emacs editor (with GTK+ GUI support)
  GNU Emacs is the extensible self-documenting text editor.  This
  package contains a version of Emacs with a graphical user interface
