From f7b98ecfa36e26c22ed2d1bccd74cf0dc1761f13 Mon Sep 17 00:00:00 2001
From: Charles Plessy <plessy@debian.org>
Date: Sun, 4 Dec 2011 16:04:17 +0900
Subject: [PATCH] Disallow non-public libraries in /usr/lib/triplet, but not
 its subdirectories.

Closes: #636383
---
 policy.sgml |    5 +++--
 1 files changed, 3 insertions(+), 2 deletions(-)

diff --git a/policy.sgml b/policy.sgml
index b8db0ab..d0baa1b 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -7697,8 +7697,9 @@ strip --strip-unneeded <var>your-lib</var>
 	  Shared object files (often <file>.so</file> files) that are not
 	  public libraries, that is, they are not meant to be linked
 	  to by third party executables (binaries of other packages),
-	  should be installed in subdirectories of the
-	  <file>/usr/lib</file> directory.  Such files are exempt from the
+	  should be installed in subdirectories of the <file>/usr/lib</file>
+	  or <file>/usr/lib/<var>triplet</var></file> directories (see
+	  <ref id="fhs"> for a definition).  Such files are exempt from the
 	  rules that govern ordinary shared libraries, except that
 	  they must not be installed executable and should be
 	  stripped.<footnote>
-- 
1.7.7.3

