--- debarchiver/src/debarchiver.pl	2005-09-09 21:43:52.000000000 +0200
+++ /usr/bin/debarchiver	2005-09-15 13:53:37.000000000 +0200
@@ -49,7 +49,9 @@ $verifysignaturesdistinput = 0;
 $userconfigfile = "$ENV{HOME}/.debarchiver.conf";
 $bzip = 0;
 $gpgpassfile = "$ENV{HOME}/.gnupg/passphrase";
-$gpgpassfile = "" if (! -f $gpgpassfile);
+if ($gpgpassfile) {
+	$gpgpassfile = 0 if (! -f $gpgpassfile);
+}
 
 ###############################################################################
 ########################## DECLARATION ########################################
@@ -967,10 +969,10 @@ sub generateRelease($$\@\@) {
     if ($gpgkey) {
 	unlink("$path/Release.gpg");
 	if ($gpgpassfile) {
-	    cmdaction("gpg -a -b -s --passphrase-fd 2 -u '$gpgkey' -o $path/Release.gpg"
-		      . " $path/Release 2</path/to/passphrase",
-		      "Signing Release file for $path",
-		      3);	    
+	    cmdaction("gpg -a -b -s -u '$gpgkey' --passphrase-fd 2 2<'$gpgpassfile'"
+		     . " -o $path/Release.gpg $path/Release",
+		     "Signing Release file for $path with key '$gpgkey'",
+		      3);
 	} else {
 	    cmdaction("gpg -a -b -s -u '$gpgkey' -o $path/Release.gpg"
 		      . " $path/Release",
