diff -ur bzr-2.0.3.orig/debian/rules bzr-2.0.3/debian/rules
--- bzr-2.0.3.orig/debian/rules	2010-01-17 12:52:17.000000000 +0100
+++ bzr-2.0.3/debian/rules	2010-01-16 23:54:06.000000000 +0100
@@ -36,3 +36,6 @@
 	( cd doc && find -name "*.png" -print0 ) | \
 	    xargs -r0 -i'{}' -n1 install -D -m 644 'doc/{}' \
 	    "debian/bzr-doc/usr/share/doc/bzr/html/{}"
+
+install/bzr::
+	rm -Rf debian/tmp/usr/lib/python*/*-packages/bzrlib/util/elementtree
