Index: debian/source/new/autopkgtest-3.20.1/tools/adt-buildvm-ubuntu-cloud
===================================================================
--- debian.orig/source/new/autopkgtest-3.20.1/tools/adt-buildvm-ubuntu-cloud
+++ debian/source/new/autopkgtest-3.20.1/tools/adt-buildvm-ubuntu-cloud
@@ -83,7 +83,8 @@ def parse_args():
     default_arch = subprocess.check_output(['dpkg', '--print-architecture'],
                                            universal_newlines=True).strip()
     qemu_default_cmds = {'i386': 'qemu-system-i386',
-                         'amd64': 'qemu-system-x86_64'}
+                         'amd64': 'qemu-system-x86_64',
+                         'ppc64el': 'qemu-system-ppc64'}
 
     parser.add_argument('-a', '--arch', default=default_arch,
                         help='Ubuntu architecture (default: %(default)s)')
