From 6fc3bc1a150239a11b19217d1bbb39127492770d Mon Sep 17 00:00:00 2001
From: Ansgar Burchardt <ansgar@debian.org>
Date: Sat, 4 Aug 2012 12:02:12 +0200
Subject: [PATCH] gnome.mk: Use xz compression by default.

---
 1/class/gnome.mk.in |    3 +++
 1 file changed, 3 insertions(+)

diff --git a/1/class/gnome.mk.in b/1/class/gnome.mk.in
index 5d2e188..3febdf7 100644
--- a/1/class/gnome.mk.in
+++ b/1/class/gnome.mk.in
@@ -42,4 +42,7 @@ $(patsubst %,binary-install/%,$(DEB_PACKAGES)) :: binary-install/%:
 	$(if $(wildcard /usr/bin/dh_gconf),dh_gconf -p$(cdbs_curpkg) $(DEB_DH_GCONF_ARGS))
 	$(if $(wildcard /usr/bin/dh_icons),dh_icons -p$(cdbs_curpkg) $(DEB_DH_ICONS_ARGS))
 
+# Use xz compression.
+DEB_DH_BUILDDEB_ARGS ?= -u-Zxz
+
 endif
-- 
1.7.10.4

