Index: dh-make-2.201602/dh_make
===================================================================
--- dh-make-2.201602.orig/dh_make
+++ dh-make-2.201602/dh_make
@@ -759,7 +759,7 @@ def do_debianize(args, subs):
             setup_make()
 
     # Create debian directory
-    if args.addmissing is None:
+    if args.addmissing is False:
         if os.path.isdir('debian'):
             print('You already have a debian/ subdirectory in the source'
                   'tree.\ndh_make will not try to overwrite anything.')
@@ -809,7 +809,7 @@ def do_debianize(args, subs):
         process_dir(args, subs_func, args.templates)
 
     process_docs(args.python, args.docs, subs['PACKAGE'])
-    process_infos(args.docs, subs['PACKAGE'])
+    process_infos(args.python, args.docs, subs['PACKAGE'])
 
     rename_package_files(subs['PACKAGE'])
 
