--- a/debsign.sh	2014-10-13 21:35:45.000000000 -0500
+++ b/debsign.sh	2014-10-30 17:17:51.058874668 -0500
@@ -256,7 +256,7 @@
 
     # set defaults
     for var in $VARS; do
-	eval "$var=\$DEFAULT_$var"
+	eval $var=\${$var:-\$DEFAULT_$var}
     done
 else
     # Run in a subshell for protection against accidental errors
@@ -264,7 +264,7 @@
     eval $(
 	set +e
 	for var in $VARS; do
-	    eval "$var=\$DEFAULT_$var"
+	    eval $var=\${$var:-\$DEFAULT_$var}
 	done
 
 	for file in /etc/devscripts.conf ~/.devscripts
