diff -Nru a/debian/control b/debian/control
--- a/debian/control	2015-10-20 12:44:27.000000000 +0100
+++ b/debian/control	2016-01-12 00:29:26.813611864 +0000
@@ -26,12 +26,12 @@
  libsigc++-2.0-dev (>= 2.2.10),
  uuid-dev,
  libxml2-dev (>= 2.5.7),
- libcwiid-dev,
+ libcwiid-dev [linux-any],
  libcairomm-1.0-dev (>= 1.10.0),
  libgtkmm-2.4-dev (>= 2.24.2),
  libpangomm-1.4-dev (>= 2.28.4),
  ladspa-sdk (>= 1.1-2),
- libasound2-dev (>= 0.9.4),
+ libasound2-dev (>= 0.9.4) [linux-any],
  libaubio-dev (>= 0.3.2),
  libjack-dev,
  liblo-dev (>= 0.26~),
diff -Nru a/debian/rules b/debian/rules
--- a/debian/rules	2015-10-20 12:44:27.000000000 +0100
+++ b/debian/rules	2016-01-12 00:29:13.954613461 +0000
@@ -44,6 +44,12 @@
 	DIST_TARGET = x86_64
 endif
 
+BACKENDS = jack
+ifneq (,$(findstring linux,$(DEB_HOST_ARCH_OS)))
+	# ALSA Backend is only available on Linux
+	BACKENDS = jack,alsa
+endif
+
 LD_LIBRARY_PATH += :$(DEB_DESTDIR)/usr/lib/ardour4/
 
 waf-configure-options = --lv2 \
@@ -52,7 +58,7 @@
 	--configdir=/etc/ \
 	--noconfirm \
 	--prefix=/usr/ \
-	--with-backends=jack,alsa \
+	--with-backends=$(BACKENDS) \
 	--no-phone-home \
 	--use-external-libs \
 	--dist-target=$(DIST_TARGET) \
