Index: pkgs.dbk
===================================================================
--- pkgs.dbk	(révision 9331)
+++ pkgs.dbk	(copie de travail)
@@ -370,6 +370,12 @@
 long as it hasn't been archived. The same rules as for <literal>stable</literal>
 apply.
 </para>
+<para>
+Version numbers should follow advice from <xref linkend="nmu-changelog"/>, and
+the distribution name should be preferred over <literal>stable</literal> or
+<literal>oldstable</literal> in the changelog entry.
+</para>
+
 </section>
 
 <section id="upload-t-p-u">
@@ -2094,21 +2100,20 @@
 If you upload a package to testing or stable, you sometimes need to "fork" the
 version number tree. This is the case for security uploads, for example.  For
 this, a version of the form
-<literal>+deb<replaceable>XY</replaceable>u<replaceable>Z</replaceable></literal>
-should be used, where <replaceable>X</replaceable> and
-<replaceable>Y</replaceable> are the major and minor release numbers, and
+<literal>+deb<replaceable>X</replaceable>u<replaceable>Z</replaceable></literal>
+should be used, where <replaceable>X</replaceable>
+is the major and minor release numbers, and
 <replaceable>Z</replaceable> is a counter starting at <literal>1</literal>.
 When the release number is not yet known (often the case for
 <literal>testing</literal>, at the beginning of release cycles), the lowest
 release number higher than the last stable release number must be used.  For
-example, while Lenny (Debian 5.0) is stable, a security NMU to stable for a
+example, while Squeeze (Debian 6.0) is stable, a security NMU to stable for a
 package at version <literal>1.5-3</literal> would have version
-<literal>1.5-3+deb50u1</literal>, whereas a security NMU to Squeeze would get
-version <literal>1.5-3+deb60u1</literal>. After the release of Squeeze, security
+<literal>1.5-3+deb6u1</literal>, whereas a security NMU to Wheezy would get
+version <literal>1.5-3+deb7u1</literal>. After the release of Wheezy, security
 uploads to the <literal>testing</literal> distribution will be versioned
-<literal>+deb61uZ</literal>, until it is known whether that release will be
-Debian 6.1 or Debian 7.0 (if that becomes the case, uploads will be versioned
-as <literal>+deb70uZ</literal>).
+<literal>+deb8u<replaceable>Z</replaceable></literal> since Jessie will be
+Debian 8.0.
 </para>
 </section>
 
@@ -2689,11 +2694,10 @@
 <literal>unstable</literal> does not pull in any new dependencies.
 </para>
 <para>
-Version numbers are usually selected by adding the codename of the
-<literal>testing</literal> distribution and a running number, like
-<literal>1.2squeeze1</literal> for the first upload through
-<literal>testing-proposed-updates</literal> of package version
-<literal>1.2</literal>.
+<para>
+Version numbers should follow advice from <xref linkend="nmu-changelog"/>, and
+the distribution name should be preferred over <literal>testing</literal> in
+the changelog entry.
 </para>
 <para>
 Please make sure you didn't miss any of these items in your upload:
