--- a/aide.conf.d/70_aide_dev	2012-03-04 11:10:49.258794862 +0100
+++ b/aide.conf.d/70_aide_dev	2012-03-04 07:57:26.282852021 +0100
@@ -6,20 +6,11 @@ echo "!/dev/pts/[0-9]{1,2}$"
 if [ -e /dev/.udev ]; then
   # we have udev
 
-  # We generate a RamdiskData entry for each file found in /dev (as
-  # these files change inode and mtim/ctime at each boot anyway).
-  # /dev/.static is excluded as this is bind-mounted from the persistent
-  # root /dev, which should be a lot more static.
-
   # Making this any more paranoid would probably mean to implementing most
   # of udev. Please feel free to submit patches ;)
-
-  # -path is -wholename on sid, but -path still works on sarge and sid
-  find /dev -path /dev/.static -prune -o -print | \
-     sed -e 's/^\(.*\)/\1$ RamdiskData/'   \
-         -e 's,\\,\\\\,g' -e 's,\.,\\\.,g' \
-         -e 's,#,\\#,g'
   echo "/dev/\.static$ RamdiskData"
+  echo "/dev/\.static Full"
+  echo "/dev RamdiskData"
 else
   # we have static /dev
   cat <<EOF
