diff --git a/debian/changelog b/debian/changelog
index 45337ce..fe4a858 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -28,6 +28,11 @@ debian-policy (3.9.5.0) UNRELEASED; urgency=low
     Seconded: Charles Plessy <plessy@debian.org>
     Seconded: Russ Allbery <rra@debian.org>
     Closes: #669915
+  * Policy: Stop recommending to serve HTML documents from /usr/share/doc.
+    Wording: Thomas Goirand <zigo@debian.org>
+    Seconded: Charles Plessy <plessy@debian.org>
+    Seconded: Jonathan Nieder <jrnieder@gmail.com>
+    Closes: #715804
   * debconf_spec: Document the 'escape' capability.
     Wording: Jonathan Nieder <jrnieder@gmail.com>
     Seconded: Charles Plessy <plessy@debian.org>
diff --git a/policy.sgml b/policy.sgml
index 1508231..953d5d2 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -9669,24 +9669,7 @@ http://localhost/cgi-bin/<var>cgi-bin-name</var>
 	    </item>
 
 	    <item>
-	      <p>Access to HTML documents</p>
-
-	      <p>
-		HTML documents for a package are stored in
-                <file>/usr/share/doc/<var>package</var></file>
-		and can be referred to as
-		<example compact="compact">
-http://localhost/doc/<var>package</var>/<var>filename</var>
-		</example>
-	      </p>
-
-	      <p>
-                The web server should restrict access to the document
-                tree so that only clients on the same host can read
-                the documents. If the web server does not support such
-                access controls, then it should not provide access at
-                all, or ask about providing access during installation.
-	      </p>
+	      <p>(Deleted)</p>
 	    </item>
 
             <item>
diff --git a/upgrading-checklist.sgml b/upgrading-checklist.sgml
index bcc4c1d..5714e84 100644
--- a/upgrading-checklist.sgml
+++ b/upgrading-checklist.sgml
@@ -55,6 +55,10 @@ Unreleased.
   <item>New section documenting the <tt>Package-Type</tt> field in source
   package control files.
   </item>
+<tag>11.5.2</tag>
+  <item>Stop recommending to serve HTML documents from
+  <file>/usr/share/doc/<var>package</var></file>.
+  </item>
 <tag>12.2</tag>
   <item>Packages distributing Info documents should use
   <package>install-info</package>'s trigger, and do not need anymore
