From b23927006592b80b83541c274c00c092d56ffaf2 Mon Sep 17 00:00:00 2001
From: Julian Wollrath <jwollrath@web.de>
Date: Sun, 19 Mar 2017 10:51:51 +0100
Subject: [PATCH 1/6] Run wrap-and-sort

---
 debian/awesome.install |  2 +-
 debian/control         | 72 +++++++++++++++++++++++++-------------------------
 2 files changed, 37 insertions(+), 37 deletions(-)

diff --git a/debian/awesome.install b/debian/awesome.install
index e50e2c74..7ff29bf2 100644
--- a/debian/awesome.install
+++ b/debian/awesome.install
@@ -1,2 +1,2 @@
-debian/awesome.xpm usr/share/pixmaps
 debian/55awesome-javaworkaround /etc/X11/Xsession.d
+debian/awesome.xpm usr/share/pixmaps
diff --git a/debian/control b/debian/control
index 43fb79f1..7a6c1ed3 100644
--- a/debian/control
+++ b/debian/control
@@ -3,34 +3,34 @@ Section: x11
 Priority: optional
 Maintainer: Julien Danjou <acid@debian.org>
 Uploaders: Arnaud Fontaine <arnau@debian.org>
-Build-Depends: debhelper (>= 9),
-  libcairo2-dev,
-  xmlto,
-  asciidoc,
-  cmake (>= 2.6.0),
-  lua5.1,
-  lua-ldoc,
-  libxcb-xtest0-dev,
-  libxcb-icccm4-dev (>= 0.3.8),
-  libxcb-randr0-dev (>= 0.3.0),
-  libxcb-keysyms1-dev (>= 0.3.4),
-  libxcb-xinerama0-dev,
-  liblua5.1-dev,
-  libdbus-1-dev,
-  libxdg-basedir-dev (>= 1.0.0),
-  libstartup-notification0-dev (>= 0.10),
-  imagemagick,
-  libxcb1-dev (>= 1.6),
-  libxcb-shape0-dev,
-  libxcb-util0-dev,
-  libxcb-cursor-dev (>= 0.1.0-2),
-  libgdk-pixbuf2.0-dev,
-  libglib2.0-dev,
-  libxcb-xrm-dev,
-  libxcb-xkb-dev,
-  libxkbcommon-x11-dev,
-  libxkbcommon-dev,
-  lua-lgi (>= 0.7.0),
+Build-Depends: asciidoc,
+               cmake (>= 2.6.0),
+               debhelper (>= 9),
+               imagemagick,
+               libcairo2-dev,
+               libdbus-1-dev,
+               libgdk-pixbuf2.0-dev,
+               libglib2.0-dev,
+               liblua5.1-dev,
+               libstartup-notification0-dev (>= 0.10),
+               libxcb-cursor-dev (>= 0.1.0-2),
+               libxcb-icccm4-dev (>= 0.3.8),
+               libxcb-keysyms1-dev (>= 0.3.4),
+               libxcb-randr0-dev (>= 0.3.0),
+               libxcb-shape0-dev,
+               libxcb-util0-dev,
+               libxcb-xinerama0-dev,
+               libxcb-xkb-dev,
+               libxcb-xrm-dev,
+               libxcb-xtest0-dev,
+               libxcb1-dev (>= 1.6),
+               libxdg-basedir-dev (>= 1.0.0),
+               libxkbcommon-dev,
+               libxkbcommon-x11-dev,
+               lua-ldoc,
+               lua-lgi (>= 0.7.0),
+               lua5.1,
+               xmlto
 Standards-Version: 3.9.7
 Homepage: http://awesome.naquadah.org
 Vcs-Git: git://anonscm.debian.org/collab-maint/awesome.git
@@ -38,14 +38,14 @@ Vcs-Browser: http://anonscm.debian.org/git/collab-maint/awesome.git
 
 Package: awesome
 Architecture: any
-Recommends: x11-xserver-utils, rlwrap, feh
-Depends: ${shlibs:Depends},
-  ${misc:Depends},
-  menu,
-  dbus-x11,
-  lua-lgi (>= 0.7.0),
-  gir1.2-freedesktop,
-  gir1.2-pango-1.0
+Recommends: feh, rlwrap, x11-xserver-utils
+Depends: dbus-x11,
+         gir1.2-freedesktop,
+         gir1.2-pango-1.0,
+         lua-lgi (>= 0.7.0),
+         menu,
+         ${misc:Depends},
+         ${shlibs:Depends}
 Provides: x-window-manager
 Description: highly configurable X window manager
  awesome manages windows dynamically in floating or tiled layouts. It is
-- 
2.11.0

