From d4011e244070cbb884b180391cb877cbdbd0afd8 Mon Sep 17 00:00:00 2001
From: Julian Wollrath <jwollrath@web.de>
Date: Thu, 18 May 2017 14:36:51 +0200
Subject: [PATCH 2/4] Do not install README.md link (Closes: #856442)

---
 debian/docs | 1 -
 1 file changed, 1 deletion(-)
 delete mode 100644 debian/docs

diff --git a/debian/docs b/debian/docs
deleted file mode 100644
index b43bf86b..00000000
--- a/debian/docs
+++ /dev/null
@@ -1 +0,0 @@
-README.md
-- 
2.13.2

