From 4540e01a36973244413e14a282afd5f756349a31 Mon Sep 17 00:00:00 2001
From: Julian Wollrath <jwollrath@web.de>
Date: Sun, 19 Mar 2017 11:14:28 +0100
Subject: [PATCH 2/6] Update build dependencies

---
 debian/control | 13 ++++++++-----
 1 file changed, 8 insertions(+), 5 deletions(-)

diff --git a/debian/control b/debian/control
index 7a6c1ed3..3287d0a2 100644
--- a/debian/control
+++ b/debian/control
@@ -12,24 +12,27 @@ Build-Depends: asciidoc,
                libgdk-pixbuf2.0-dev,
                libglib2.0-dev,
                liblua5.1-dev,
+               libpango1.0-dev,
                libstartup-notification0-dev (>= 0.10),
-               libxcb-cursor-dev (>= 0.1.0-2),
+               libx11-xcb-dev,
+               libxcb-cursor-dev,
                libxcb-icccm4-dev (>= 0.3.8),
                libxcb-keysyms1-dev (>= 0.3.4),
-               libxcb-randr0-dev (>= 0.3.0),
+               libxcb-randr0-dev,
                libxcb-shape0-dev,
-               libxcb-util0-dev,
+               libxcb-util0-dev (>= 0.3.8),
                libxcb-xinerama0-dev,
                libxcb-xkb-dev,
                libxcb-xrm-dev,
                libxcb-xtest0-dev,
-               libxcb1-dev (>= 1.6),
                libxdg-basedir-dev (>= 1.0.0),
                libxkbcommon-dev,
                libxkbcommon-x11-dev,
+               lua-discount,
                lua-ldoc,
-               lua-lgi (>= 0.7.0),
+               lua-lgi (>= 0.8.0),
                lua5.1,
+               x11proto-core-dev (>= 7.0.15),
                xmlto
 Standards-Version: 3.9.7
 Homepage: http://awesome.naquadah.org
-- 
2.11.0

