From bd5c408ad03baaa6278e60486011bd418c2eeb9d Mon Sep 17 00:00:00 2001
From: Julian Wollrath <jwollrath@web.de>
Date: Sun, 19 Mar 2017 11:55:00 +0100
Subject: [PATCH 3/6] Bump required lua-lgi version to at least 0.8.0

---
 debian/control | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/debian/control b/debian/control
index 3287d0a2..a08dd57b 100644
--- a/debian/control
+++ b/debian/control
@@ -45,7 +45,7 @@ Recommends: feh, rlwrap, x11-xserver-utils
 Depends: dbus-x11,
          gir1.2-freedesktop,
          gir1.2-pango-1.0,
-         lua-lgi (>= 0.7.0),
+         lua-lgi (>= 0.8.0),
          menu,
          ${misc:Depends},
          ${shlibs:Depends}
-- 
2.11.0

