From 0473d1e118ee687132ddc6776b1da76850ff4b71 Mon Sep 17 00:00:00 2001
From: Julian Wollrath <jwollrath@web.de>
Date: Mon, 17 Jul 2017 11:38:18 +0200
Subject: [PATCH 4/4] Update debian/changelog

---
 debian/changelog | 14 ++++----------
 1 file changed, 4 insertions(+), 10 deletions(-)

diff --git a/debian/changelog b/debian/changelog
index ce6a40c0..f75000c9 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,20 +1,14 @@
-awesome (4.2-0~exp0) unstable; urgency=medium
-
-  * New upstream snapshot.
-  * Provide notification-daemon (Closes: #734645).
-  * Do not install README.md link (Closes: #856442).
-
- -- Julian Wollrath <jwollrath@web.de>  Sat, 15 Jul 2017 17:08:22 +0200
-
-awesome (4.1-0~exp0) unstable; urgency=medium
+awesome (4.2-0~exp0) UNRELEASED; urgency=medium
 
   * New upstream version: (Closes: #800885, #858170)
     - Refresh install-and-debian-menu.
     - Drop remove_useless_build_lgi_check.patch.
     - Update build dependencies.
     - Bump required lua-lgi version to at least 0.8.0.
+  * Provide notification-daemon (Closes: #734645).
+  * Do not install README.md link (Closes: #856442).
 
- -- Julian Wollrath <jwollrath@web.de>  Sun, 19 Mar 2017 11:57:14 +0100
+ -- Julian Wollrath <jwollrath@web.de>  Mon, 17 Jul 2017 11:37:46 +0200
 
 awesome (4.0-1) unstable; urgency=medium
 
-- 
2.13.2

