From 3332b2cc443397530bb38c85fbc3ef86f73d70e8 Mon Sep 17 00:00:00 2001
From: Julian Wollrath <jwollrath@web.de>
Date: Sun, 19 Mar 2017 11:16:05 +0100
Subject: [PATCH 6/6] Update changelog for 4.1-0~exp0

---
 debian/changelog | 10 ++++++++++
 1 file changed, 10 insertions(+)

diff --git a/debian/changelog b/debian/changelog
index b8b5f7dd..43884596 100644
--- a/debian/changelog
+++ b/debian/changelog
@@ -1,3 +1,13 @@
+awesome (4.1-0~exp0) unstable; urgency=medium
+
+  * New upstream version:
+    - Refresh install-and-debian-menu.
+    - Drop remove_useless_build_lgi_check.patch.
+    - Update build dependencies.
+    - Bump required lua-lgi version to at least 0.8.0.
+
+ -- Julian Wollrath <jwollrath@web.de>  Sun, 19 Mar 2017 11:57:14 +0100
+
 awesome (4.0-1) unstable; urgency=medium
 
   * New upstream release. Closes: #850701, #851791.
-- 
2.11.0

