diff --git a/adequate b/adequate
--- a/adequate
+++ b/adequate
@@ -450,7 +450,7 @@
             next file if $path =~ m{^/lib\d*/.*(?<=/)ld(?:-.+)[.]so(?:$|[.])}; # dynamic linker
             next file unless defined $interesting_dirs{$dir};
             if (-l $path) {
-                my $realpath = Cwd::realpath($path);
+                my $realpath = Cwd::realpath($path) // die "resolving $path failed: $!";
                 my ($realdir) = $realpath =~ m{(.*)/[^/]+$};
                 # If the symlink target is still in an “interesting” directory,
                 # then any issue hopefully will be reported against another
