diff -ur ecl-11.1.1/debian/rules ecl-11.1.1.modified/debian/rules
--- ecl-11.1.1/debian/rules	2011-03-11 19:00:29.000000000 +0000
+++ ecl-11.1.1.modified/debian/rules	2011-11-16 17:12:19.000000000 +0000
@@ -33,6 +33,9 @@
 ifneq ($(DEB_HOST_ARCH),ia64)
   confflags += --enable-gengc
 endif
+ifneq ($(DEB_HOST_ARCH),hurd-i386)
+  confflags += --enable-threads=yes
+endif
 
 # force to use ginstall-info:
 export INSTALL_INFO=/usr/bin/ginstall-info
@@ -56,7 +59,6 @@
 	   --with-system-gmp=yes \
 	   --with-tcp \
 	   --with-clx  \
-	   --enable-threads=yes \
 	   --enable-boehm=system  \
 	   --with-x
 
