--- orig/functions	2023-06-12 19:44:03.169739289 +0200
+++ /usr/lib/cryptsetup/functions	2023-06-12 19:45:25.780386107 +0200
@@ -83,6 +83,7 @@
              CRYPTTAB_OPTION_keyslot \
              CRYPTTAB_OPTION_header \
              CRYPTTAB_OPTION_tcrypthidden \
+             CRYPTTAB_OPTION_timeout \
              CRYPTTAB_OPTION_same_cpu_crypt \
              CRYPTTAB_OPTION_submit_from_crypt_cpus \
              CRYPTTAB_OPTION_no_read_workqueue \
@@ -186,7 +187,7 @@
             fi
         ;;
         # numeric options >=0
-        offset|skip|tries|keyslot|keyfile-offset)
+        offset|skip|tries|keyslot|keyfile-offset|timeout)
             if ! printf '%s' "${VALUE-}" | grep -Exq "[0-9]+"; then
                 return 1
             fi
