--- binfmt-support.orig	2009-09-30 14:12:44.982007593 -0400
+++ binfmt-support	2009-09-30 14:13:00.814256909 -0400
@@ -31,7 +31,7 @@
     log_daemon_msg "Enabling $DESC" "$NAME"
     PROCDIR=/proc/sys/fs/binfmt_misc
     if [ ! -e "$PROCDIR/register" ]; then
-	modprobe -q binfmt_misc
+	modprobe -q binfmt_misc || true
 	mount -t binfmt_misc -o nodev,noexec,nosuid binfmt_misc "$PROCDIR"
     fi
     cachefail=0
