diff --git a/debian/initramfs/scripts/local-top/cryptroot b/debian/initramfs/scripts/local-top/cryptroot
index 1da5ad1f..c348cf7f 100644
--- a/debian/initramfs/scripts/local-top/cryptroot
+++ b/debian/initramfs/scripts/local-top/cryptroot
@@ -189,7 +189,7 @@ setup_mapping() {
                 return 1
             elif vg="$(lvm pvs --noheadings -o vg_name --config 'log{prefix=""}' -- "$dev")"; then
                 # activate the VG held by the PV we just unlocked
-                lvm lvchange -a y --sysinit -- "$vg"
+                lvm lvchange -a ay --sysinit -- "$vg"
             fi
         fi
 
