Index: debian/changelog
===================================================================
--- debian/changelog	(Revision 819)
+++ debian/changelog	(Arbeitskopie)
@@ -21,8 +21,11 @@
   * debcommit: Name git tags "debian/$version" or just with the package
     version number for native version numbers. Closes: #446699
 
- -- Adam D. Barratt <debian-bts@adam-barratt.org.uk>  Mon, 22 Oct 2007 18:49:49 +0100
+  [ Patrick Schoenfeld ]
+  * licensecheck: Implemented checking for ISC license (Closes: #448000)
 
+ -- Patrick Schoenfeld <schoenfeld@in-medias-res.com>  Thu, 25 Oct 2007 11:07:26 +0200
+
 devscripts (2.10.9) unstable; urgency=low
 
   [ Mohammed Adnène Trojette ]
Index: scripts/licensecheck.pl
===================================================================
--- scripts/licensecheck.pl	(Revision 819)
+++ scripts/licensecheck.pl	(Arbeitskopie)
@@ -398,6 +398,11 @@
 	}
     }
 
+	if ($licensetext =~ /Permission to use, copy, modify, and\/or distribute this software for any purpose with or without fee is hereby granted, provided.*copyright notice.*permission notice.*all copies/)
+	{
+	$license = "ISC";
+	}
+
     if ($licensetext =~ /Mozilla Public License Version ([^ ]+)/) {
 	$license = "MPL (v$1) $license";
     }
