--- Mailer.class.php.orig	2024-10-18 19:37:25.667341916 +0200
+++ Mailer.class.php	2024-10-18 19:37:58.632008894 +0200
@@ -13,7 +13,7 @@
 
 // end of line for mail lines - RFC822 says CRLF but postfix (and other MTAs?)
 // think different
-if(!defined('MAILHEADER_EOL')) define('MAILHEADER_EOL', "\n");
+if(!defined('MAILHEADER_EOL')) define('MAILHEADER_EOL', "\r\n");
 #define('MAILHEADER_ASCIIONLY',1);
 
 /**
