From e8f3f081993f23233b9be68935bc1abde84ca283 Mon Sep 17 00:00:00 2001
From: intrigeri <intrigeri@debian.org>
Date: Sat, 3 Dec 2016 19:10:19 +0000
Subject: [PATCH] AppArmor profile: support merged-/usr.

---
 debian/local/apparmor-profile | 16 ++++++++--------
 1 file changed, 8 insertions(+), 8 deletions(-)

diff --git a/debian/local/apparmor-profile b/debian/local/apparmor-profile
index e0c7e40..7ef397d 100644
--- a/debian/local/apparmor-profile
+++ b/debian/local/apparmor-profile
@@ -42,9 +42,9 @@
   network econet dgram,
   network ash dgram,
 
-  /bin/bash ixr,
-  /bin/dash ixr,
-  /bin/hostname ixr,
+  /{usr/,}bin/bash ixr,
+  /{usr/,}bin/dash ixr,
+  /{usr/,}bin/hostname ixr,
   /dev/lp* rw,
   deny /dev/tty rw,  # silence noise
   /dev/ttyS* rw,
@@ -72,8 +72,8 @@
   /sys/** r,
   /usr/bin/* ixr,
   /usr/sbin/* ixr,
-  /bin/* ixr,
-  /sbin/* ixr,
+  /{usr/,}bin/* ixr,
+  /{usr/,}sbin/* ixr,
   /usr/lib/** rm,
 
   # backends which come with CUPS can be confined
@@ -180,9 +180,9 @@
 
   @{PROC}/*/auxv r,
 
-  /bin/dash ixr,
-  /bin/bash ixr,
-  /bin/cp ixr,
+  /{usr/,}bin/dash ixr,
+  /{usr/,}bin/bash ixr,
+  /{usr/,}bin/cp ixr,
   /etc/papersize r,
   /etc/cups/cups-pdf.conf r,
   @{HOME}/PDF/ rw,
-- 
2.10.2

