Index: debian/cdrom-detect.templates
===================================================================
--- debian/cdrom-detect.templates	(revision 66241)
+++ debian/cdrom-detect.templates	(working copy)
@@ -71,6 +71,12 @@
 Description: for internal use only
  Set if the CD appears to be on a USB stick.
 
+Template: cdrom-detect/usb-hdd
+Type: boolean
+Default: false
+Description: for internal use only
+ Set if the CD appears to be a live USB-HDD image.
+
 Template: cdrom-detect/scanning_progress_title
 Type: text
 # :sl1:
Index: debian/changelog
===================================================================
--- debian/changelog	(revision 66241)
+++ debian/changelog	(working copy)
@@ -1,3 +1,10 @@
+cdrom-detect (1.38) UNRELEASED; urgency=low
+
+  * Set cdrom-detect/usb-hdd when the "CD" is really a live USB-HDD
+    filesystem.
+
+ -- Joey Hess <joeyh@debian.org>  Sat, 15 Jan 2011 14:32:29 -0400
+
 cdrom-detect (1.37) unstable; urgency=low
 
   [ Colin Watson ]
Index: debian/cdrom-detect.postinst
===================================================================
--- debian/cdrom-detect.postinst	(revision 66241)
+++ debian/cdrom-detect.postinst	(working copy)
@@ -122,6 +122,7 @@
 			break 2
 		fi
 		if try_mount $device $FATFS; then
+			db_set cdrom-detect/usb-hdd true
 			break 2
 		fi
 	done
