diff --git a/scripts/build/lb_binary_debian-installer b/scripts/build/lb_binary_debian-installer
index e2d9d7c..7fb7773 100755
--- a/scripts/build/lb_binary_debian-installer
+++ b/scripts/build/lb_binary_debian-installer
@@ -323,8 +323,7 @@ then
 
 	case "${LB_ARCHITECTURE}" in
 		amd64)
-			#DI_REQ_PACKAGES="lilo grub grub2"
-			DI_REQ_PACKAGES="lilo grub"
+			DI_REQ_PACKAGES="lilo grub grub-pc"
 
 			case "${LB_MODE}" in
 				ubuntu)
@@ -338,8 +337,7 @@ then
 			;;
 
 		i386)
-			#DI_REQ_PACKAGES="elilo lilo grub grub2"
-			DI_REQ_PACKAGES="elilo lilo grub"
+			DI_REQ_PACKAGES="elilo lilo grub grub-pc"
 
 			case "${LB_MODE}" in
 				ubuntu)
