From 3a00ca0e08f19926a6e64e0e47a8d823eda9846f Mon Sep 17 00:00:00 2001
From: Ben Finney <ben@benfinney.id.au>
Date: Mon, 18 Apr 2016 13:46:44 +1000
Subject: [PATCH 2/2] Remove restriction of which control files may contain
 comments.

---
 policy.sgml | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/policy.sgml b/policy.sgml
index 236bf743..9aca360c 100644
--- a/policy.sgml
+++ b/policy.sgml
@@ -2636,9 +2636,7 @@ Package: libc6
 
 	<p>
 	  Lines starting with U+0023 “<tt>#</tt>”, without any
-	  preceding whitespace, are comment lines that are only
-	  permitted in source package control files
-	  (<file>debian/control</file>). These comment lines are
+	  preceding whitespace, are comment lines. These comment lines are
 	  ignored, even between two continuation lines. They do not
 	  end logical lines.
 	</p>
-- 
2.8.0.rc3

