diff -urN broadcom-sta-5.100.82.112/src/wl/sys/wl_linux.c broadcom-sta-5.100.82.112-k3.4.0/src/wl/sys/wl_linux.c
--- broadcom-sta-5.100.82.112/src/wl/sys/wl_linux.c	2012-04-17 16:25:03.000000000 +0300
+++ broadcom-sta-5.100.82.112-k3.4.0/src/wl/sys/wl_linux.c	2012-08-21 23:41:28.578729210 +0300
@@ -40,7 +40,12 @@
 #include <linux/pci_ids.h>
 #define WLC_MAXBSSCFG		1	
 
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(3, 4, 0)
+#include <asm/switch_to.h>
+#endif
+#if LINUX_VERSION_CODE < KERNEL_VERSION(3, 4, 0)
 #include <asm/system.h>
+#endif
 #include <asm/io.h>
 #include <asm/irq.h>
 #include <asm/pgtable.h>
@@ -390,7 +395,7 @@
 #if LINUX_VERSION_CODE >= KERNEL_VERSION(3, 2, 0)
 	.ndo_set_rx_mode = wl_set_multicast_list,
 #else
-	.ndo_set_multicast_list = wl_set_multicast_list,
+    .ndo_set_rx_mode = wl_set_multicast_list,
 #endif
 	.ndo_do_ioctl = wl_ioctl
 };
