Index: disktype-9-3/debian/control
===================================================================
--- disktype-9-3/debian/control
+++ disktype-9-3/debian/control
@@ -45,9 +45,8 @@
     QNX4 FS
     3DO CD-ROM FS
     Veritas VxFS
     Xbox DVD file system
-    .
  Partitioning:
     DOS/PC style
     EFI GPT
     Apple
@@ -58,19 +57,16 @@
     Linux LVM1 physical volumes
     Linux LVM2 physical volumes
     Solaris SPARC disklabel
     Solaris x86 disklabel (vtoc)
-    .
  Other structures:
     Debian split floppy header
     Linux swap
-    .
  Disk images:
     Raw CD image (.bin)
     Virtual PC hard disk image
     Apple UDIF disk image (limited)
     Linux cloop (limited)
-    .
  Boot loaders:
     LILO
     GRUB
     SYSLINUX
@@ -79,14 +75,12 @@
     FreeBSD loader
     Windows/MS-DOS loader
     BeOS loader, Haiku loader
     Sega Dreamcast
-    .
  Compression formats:
     gzip
     compress
     bzip2
-    .
  Archive formats:
     tar
     cpio
     bar
