Index: debian/aide.conf.d/31_aide_clamav-freshclam
===================================================================
--- debian/aide.conf.d/31_aide_clamav-freshclam	(revision 785)
+++ debian/aide.conf.d/31_aide_clamav-freshclam	(working copy)
@@ -6,3 +6,4 @@
 /var/lib/clamav/(daily|main)\.inc$ VarDir
 /var/lib/clamav/daily\.inc/daily\.(info|[nmhp]db)$ VarFile
 /var/lib/clamav/mirrors.dat$ VarFile
+/var/run/clamav/freshclam\.pid$ VarFile
Index: debian/aide.conf.d/31_aide_clamav-data
===================================================================
--- debian/aide.conf.d/31_aide_clamav-data	(revision 785)
+++ debian/aide.conf.d/31_aide_clamav-data	(working copy)
@@ -2,7 +2,7 @@
 !/usr/share/doc/clamav-data/(README\.Debian|copyright|changelog\.gz)$
 /usr/share/doc/clamav-data$ VarDir
 !/var/lib/dpkg/info/clamav-data\.(config|list|post(inst|rm)|templates|conffiles|md5sums)$
-/var/lib/clamav/(daily|main)\.cvd$ VarFile
+/var/lib/clamav/(daily|main)\.c[vl]d$ VarFile
 /var/lib/clamav$ VarDir
 /var/lib/clamav-data/warn-on-old-databases$ VarFile
 /var/lib/clamav-data$ VarDir
