From b17efeb4ea527e3e0f58d4eb57a38977a0ca6483 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lo=C3=AFc=20Minier?= <lool@debian.org>
Date: Mon, 17 Oct 2011 16:23:47 +0200
Subject: [PATCH] Remove valgrind; Auto-not-for-us is enough

---
 Packages-arch-specific |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/Packages-arch-specific b/Packages-arch-specific
index 9358f9c..fcdbb73 100644
--- a/Packages-arch-specific
+++ b/Packages-arch-specific
@@ -268,7 +268,6 @@ toshutils: i386 amd64                                                 # x86 spec
 %user-mode-linux: i386 amd64					      # [?] ANAIS
 %uswsusp: i386 amd64 powerpc
 v86d: amd64 i386						      # x86 specific
-%valgrind: i386 amd64 powerpc lpia armel			      # Needs ported (asm, arch knwldge)
 %vbetool: !hppa !ia64 !m68k !mips !mipsel !powerpc !sh4 !sparc	      # sys/io.h
 %vegastrike: !m68k						      # requested by Mike Furr <mfurr@debian.org>, see 207578
 vmelilo: m68k							      # m68k (VME) lilo
-- 
1.7.5.4

