diff -u bb-1.3rc1/debian/changelog bb-1.3rc1/debian/changelog
--- bb-1.3rc1/debian/changelog
+++ bb-1.3rc1/debian/changelog
@@ -1,3 +1,13 @@
+bb (1.3rc1-8.4) unstable; urgency=medium
+
+  * Non-maintainer upload.
+  * Drop the dependency on oss-compat, it's no longer needed. libmikmod
+    now supports ALSA. Closes: #617426.
+  * Build-depend on libmikmod-dev instead of libmikmod2-dev.
+    Closes: #744906.
+
+ -- Stephen Kitt <skitt@debian.org>  Sun, 21 Jun 2015 20:30:55 +0200
+
 bb (1.3rc1-8.3) unstable; urgency=medium
 
   * Non-maintainer upload.
diff -u bb-1.3rc1/debian/control bb-1.3rc1/debian/control
--- bb-1.3rc1/debian/control
+++ bb-1.3rc1/debian/control
@@ -2,13 +2,13 @@
 Section: games
 Priority: optional
 Maintainer: Uwe Hermann <uwe@debian.org>
-Build-Depends: debhelper (>> 4.0.0), libmikmod2-dev, libaa1-dev (>= 1.4p5-28)
+Build-Depends: debhelper (>> 4.0.0), libmikmod-dev, libaa1-dev (>= 1.4p5-28)
 Standards-Version: 3.7.2.2
 Homepage: http://aa-project.sourceforge.net/bb/
 
 Package: bb
 Architecture: any
-Depends: ${shlibs:Depends}, oss-compat
+Depends: ${shlibs:Depends}
 Description: ASCII-art demo based on AAlib
  This package contains a 'high quality audio-visual demonstration'
  of ASCII-art using AAlib, a portable ASCII-art graphics library.
