diff -Nru apitrace-7.1+git20160531.2d78bef0+repack/debian/changelog apitrace-7.1+git20160531.2d78bef0+repack/debian/changelog
--- apitrace-7.1+git20160531.2d78bef0+repack/debian/changelog	2016-06-03 08:58:07.000000000 +0200
+++ apitrace-7.1+git20160531.2d78bef0+repack/debian/changelog	2016-06-09 16:24:21.000000000 +0200
@@ -1,3 +1,11 @@
+apitrace (7.1+git20160531.2d78bef0+repack-1.1) UNRELEASED; urgency=medium
+
+  * Non-maintainer upload.
+  * d/{control,rules}: enable waffle support.
+    (Closes: #826623)
+
+ -- Héctor Orón Martínez <zumbi@debian.org>  Thu, 09 Jun 2016 16:23:43 +0200
+
 apitrace (7.1+git20160531.2d78bef0+repack-1) unstable; urgency=medium
 
   [ Christopher James Halse Rogers ]
diff -Nru apitrace-7.1+git20160531.2d78bef0+repack/debian/control apitrace-7.1+git20160531.2d78bef0+repack/debian/control
--- apitrace-7.1+git20160531.2d78bef0+repack/debian/control	2016-06-03 08:36:39.000000000 +0200
+++ apitrace-7.1+git20160531.2d78bef0+repack/debian/control	2016-06-09 16:23:38.000000000 +0200
@@ -9,6 +9,7 @@
  libegl1-mesa-dev,
  libgles1-mesa-dev,
  libgles2-mesa-dev,
+ libwaffle-dev,
  qtbase5-dev,
  libqt5webkit5-dev,
  pkg-config,
diff -Nru apitrace-7.1+git20160531.2d78bef0+repack/debian/rules apitrace-7.1+git20160531.2d78bef0+repack/debian/rules
--- apitrace-7.1+git20160531.2d78bef0+repack/debian/rules	2016-06-03 08:36:39.000000000 +0200
+++ apitrace-7.1+git20160531.2d78bef0+repack/debian/rules	2016-06-09 16:23:23.000000000 +0200
@@ -16,7 +16,7 @@
 DEB_HOST_MULTIARCH ?= $(shell dpkg-architecture -qDEB_HOST_MULTIARCH)
 
 override_dh_auto_configure:
-	dh_auto_configure -- -DENABLE_EGL=YES -DENABLE_GUI=YES
+	dh_auto_configure -- -DENABLE_EGL=YES -DENABLE_GUI=YES -DENABLE_WAFFLE=YES
 
 override_dh_install:
 	# Remove some files we don't want, then fail if there's anything left
