Index: expect-5.45/expect.c
===================================================================
--- expect-5.45.orig/expect.c
+++ expect-5.45/expect.c
@@ -1863,7 +1863,11 @@ expRead(
 	/* little, so that from now on it means "no new data arrived */
 	/* but it should be looked at again anyway". */
 	if (cc == 0) {
-	    cc = EXP_EOF;
+	    /* the meaning of 0 from i_read just may mean eof. So, whether */
+	    /* the eof condition really exists must be checked explicitly. */
+	    if (Tcl_Eof(esPtr->channel)) {
+		cc = EXP_EOF;
+	    }
 	} else if (cc > 0) {
 	    /* successfully read data */
 	} else {
