--- alsaplayer.desktop.orig	2008-06-16 20:19:29.000000000 +0100
+++ alsaplayer.desktop	2009-08-30 14:54:58.000000000 +0100
@@ -7,6 +7,6 @@
 Exec=alsaplayer
 TryExec=alsaplayer
 Icon=alsaplayer
-MimeType=audio/x-scpls;audio/x-mpegurl;audio/mpegurl;audio/mp3;audio/x-mp3;audio/mpeg;audio/x-mpeg;audio/x-wav;audio/ogg+vorbis;audio/ogg;application/flac;
+MimeType=audio/x-scpls;audio/x-mpegurl;audio/mpegurl;audio/mp3;audio/x-mp3;audio/mpeg;audio/x-mpeg;audio/x-wav;audio/ogg+vorbis;audio/ogg;audio/flac;
 Categories=AudioVideo;Audio;Player;
 StartupNotify=true
