Index: debian/aide.conf.d/31_aide_bind9
===================================================================
--- debian/aide.conf.d/31_aide_bind9	(revision 793)
+++ debian/aide.conf.d/31_aide_bind9	(working copy)
@@ -2,7 +2,7 @@
 @@{BINDCHROOT}/dev/log$ LowLogs
 @@{BINDCHROOT}/dev VarDir
 @@endif
-@@{BINDCHROOT}/var/log/bind/queries\.log$ Logs
+@@{BINDCHROOT}/var/log/bind/queries\.log$ Log
 @@{BINDCHROOT}/var/log/bind/queries\.log\.0$ LoSerMemberLog
 @@{BINDCHROOT}/var/log/bind/queries\.log\.[1-8]$ SerMemberLog
 @@{BINDCHROOT}/var/log/bind/queries\.log\.9$ HiSerMemberLog
