diff -Naur o/alsa-utils n/alsa-utils
--- o/alsa-utils	2019-11-05 16:39:55.000000000 +0100
+++ n/alsa-utils	2022-02-02 22:40:16.075852685 +0100
@@ -26,8 +26,9 @@
 PATH=/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin
 MYNAME=/etc/init.d/alsa-utils
 ALSACTLHOME=/run/alsa
+ALSACTLRUNTIME="${ALSACTLHOME}/runtime"
 
-[ -d "$ALSACTLHOME" ] || mkdir -p "$ALSACTLHOME"
+[ -d "$ALSACTLRUNTIME" ] || mkdir -p "$ALSACTLRUNTIME"
 
 . /lib/lsb/init-functions
 . /usr/share/alsa/utils.sh
@@ -82,7 +83,7 @@
 		sleep 1
 		return 0
 	else
-		log_action_cont_msg "warning: 'alsactl store${CARD:+ $CARD}' -E HOME="$ALSACTLHOME" -E XDG_RUNTIME_DIR=@alsactlruntime@ failed with error message '$MSG'"
+		log_action_cont_msg "warning: 'alsactl store${CARD:+ $CARD}' -E HOME="$ALSACTLHOME" -E XDG_RUNTIME_DIR="${ALSACTLRUNTIME}" failed with error message '$MSG'"
 		return 1
 	fi
 }
