diff -u ddclient-3.8.0/debian/postinst ddclient-3.8.0/debian/postinst
--- ddclient-3.8.0/debian/postinst
+++ ddclient-3.8.0/debian/postinst
@@ -39,7 +39,7 @@
 
   # if the hostslist was blank, let the user know some possible reasons
   if [ -z $hostslist ]; then
-    db_input high ddclient/blankhostslist
+    db_input high ddclient/blankhostslist || true
     db_go
   fi
 }
diff -u ddclient-3.8.0/debian/changelog ddclient-3.8.0/debian/changelog
--- ddclient-3.8.0/debian/changelog
+++ ddclient-3.8.0/debian/changelog
@@ -1,3 +1,12 @@
+ddclient (3.8.0-6.1) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * postinst: ignore debconf errors about blankhostslist, thus enabling
+    installation with non-interactive mode. Patch by Jonathan Niehof.
+    (Closes: #533888)
+
+ -- Stefano Zacchiroli <zack@debian.org>  Thu, 19 Nov 2009 10:23:24 +0100
+
 ddclient (3.8.0-6) unstable; urgency=low
 
   * Fix pending l10n issues. Debconf translations:
