diff -u clisp-2.48/debian/control clisp-2.48/debian/control
--- clisp-2.48/debian/control
+++ clisp-2.48/debian/control
@@ -17,6 +17,7 @@
 Conflicts: common-lisp-controller (<< 6.19)
 Provides: lisp-compiler, ${clisp:fasl-version}
 Suggests: gdb, clisp-doc, clisp-dev, slime
+Breaks: cl-asdf (<< 2:1.704-1)
 Description: GNU CLISP, a Common Lisp implementation
  ANSI Common Lisp is a high-level, general-purpose programming language.
  GNU CLISP is a Common Lisp implementation by Bruno Haible of Karlsruhe
diff -u clisp-2.48/debian/changelog clisp-2.48/debian/changelog
--- clisp-2.48/debian/changelog
+++ clisp-2.48/debian/changelog
@@ -1,3 +1,10 @@
+clisp (1:2.48-1.2) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * Add "Breaks: cl-asdf (<< 2:1.704-1)" (Closes: #579460)
+
+ -- Alexander Reichle-Schmehl <tolimar@debian.org>  Wed, 05 May 2010 10:39:00 +0200
+
 clisp (1:2.48-1.1) unstable; urgency=low
 
   * Non-maintainer upload.
