diff -ru dctrl-tools-2.20.orig//man/grep-dctrl.1.cp dctrl-tools-2.20/man/grep-dctrl.1.cp
--- dctrl-tools-2.20.orig//man/grep-dctrl.1.cp	2011-12-06 07:54:54.000000000 +0100
+++ dctrl-tools-2.20/man/grep-dctrl.1.cp	2011-12-06 07:57:46.000000000 +0100
@@ -39,7 +39,7 @@
 .IR "What is the Debian package foo?" , 
 .IR "Which version of the Debian package bar is now current?" ,
 .IR "Which Debian packages does John Doe maintain?" ,
-.I  "Which Debian packages are somehow related to the Scheme"
+.I  "Which Debian packages are somehow related to the Scheme?"
 .IR "programming language?" ,
 and with some help,
 .IR "Who maintain the essential packages of a Debian system?" ,
