From f3e84bdfe9a6d5d881a9bfe0227ce5e1b7b4d561 Mon Sep 17 00:00:00 2001
From: Kumar Appaiah <akumar@debian.org>
Date: Wed, 15 Oct 2008 23:15:31 -0500
Subject: [PATCH] Add support for gfortran.

---
 Makefile |    4 ++--
 1 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/Makefile b/Makefile
index 41f3cf6..8c86023 100644
--- a/Makefile
+++ b/Makefile
@@ -2,7 +2,7 @@
 
 APT_BUILD = apt-build
 APT_BUILD_WRAPPER = apt-build-wrapper
-WRAP_PROGRAMMS = gcc cc g++ c++ make
+WRAP_PROGRAMMS = gcc cc g++ c++ make gfortran
 CFLAGS = -W -Wall -Wextra -std=gnu99 -pipe -Wundef -Wshadow -Wcast-align \
 	 -Wwrite-strings -Wsign-compare -Wunused -Winit-self -Wpointer-arith -Wredundant-decls \
 	 -Wmissing-prototypes -Wmissing-format-attribute -Wmissing-noreturn -D_GNU_SOURCE -g
@@ -27,7 +27,7 @@ install:	$(APT_BUILD_WRAPPER) $(APT_BUILD)
 		$(LN_S) $(APT_BUILD_WRAPPER) $(APT_BUILD_DIR)/$$prog; \
 	done
 ifneq ($(HOST_TYPE),)
-	for prog in gcc g++; do \
+	for prog in gcc g++ gfortran; do \
 		$(LN_S) $(APT_BUILD_WRAPPER) $(APT_BUILD_DIR)/$$prog-$(HOST_TYPE); \
 	done
 endif
-- 
1.5.6.5

