diff -u apsfilter-7.2.6/debian/rules apsfilter-7.2.6/debian/rules
--- apsfilter-7.2.6/debian/rules
+++ apsfilter-7.2.6/debian/rules
@@ -43,7 +43,9 @@
 			template/apsfilterrc \
 			debian/apsfilter/usr/share/apsfilter
 	rm debian/apsfilter/usr/share/apsfilter/bin/*.in
-	mv debian/apsfilter/usr/share/apsfilter/bin/{aps2file,apspreview,apsfilter-bug} \
+	mv debian/apsfilter/usr/share/apsfilter/bin/aps2file \
+		debian/apsfilter/usr/share/apsfilter/bin/apspreview \
+		debian/apsfilter/usr/share/apsfilter/bin/apsfilter-bug \
 			debian/apsfilter/usr/bin
 	cp -p uniprint/*.upp debian/apsfilter/usr/share/ghostscript/common
 	dh_installdebconf
diff -u apsfilter-7.2.6/debian/changelog apsfilter-7.2.6/debian/changelog
--- apsfilter-7.2.6/debian/changelog
+++ apsfilter-7.2.6/debian/changelog
@@ -1,3 +1,10 @@
+apsfilter (7.2.6-1.2) unstable; urgency=low
+
+  * Non-maintainer upload.
+  * debian/rules: Fix bashim. (Closes: #459048)
+
+ -- Marc 'HE' Brockschmidt <he@debian.org>  Mon, 28 Jan 2008 00:42:50 +0100
+
 apsfilter (7.2.6-1.1) unstable; urgency=low
 
   * Non-maintainer upload to fix longstanding l10n and debconf issues
