diff --git a/scheduler/main.c b/scheduler/main.c
index e9ced71..a526628 100644
--- a/scheduler/main.c
+++ b/scheduler/main.c
@@ -431,14 +431,6 @@ main(int  argc,				/* I - Number of command-line args */
 
 #ifndef DEBUG
    /*
-    * Disable core dumps...
-    */
-
-    getrlimit(RLIMIT_CORE, &limit);
-    limit.rlim_cur = 0;
-    setrlimit(RLIMIT_CORE, &limit);
-
-   /*
     * Disconnect from the controlling terminal...
     */
 
