Description: Fix Little Endian for mipsel
Author: Mathieu Malaterre <malat@debian.org

--- cmake-3.0.2.orig/Source/kwsys/CPU.h.in
+++ cmake-3.0.2/Source/kwsys/CPU.h.in
@@ -78,7 +78,12 @@
 
 /* MIPS */
 #elif defined(__mips) || defined(__mips__) || defined(__MIPS__)
+/* MIPSEL */
+#if defined(__MIPSEL__)
+# define @KWSYS_NAMESPACE@_CPU_ENDIAN_ID @KWSYS_NAMESPACE@_CPU_ENDIAN_ID_LITTLE
+#else
 # define @KWSYS_NAMESPACE@_CPU_ENDIAN_ID @KWSYS_NAMESPACE@_CPU_ENDIAN_ID_BIG
+#endif
 
 /* OpenRISC 1000 */
 #elif defined(__or1k__)
