diff -ru dpkg-1.14.12ubuntu3.orig/src/archives.c dpkg-1.14.12ubuntu3/src/archives.c
--- dpkg-1.14.12ubuntu3.orig/src/archives.c	2007-12-05 10:42:16.000000000 +0100
+++ dpkg-1.14.12ubuntu3/src/archives.c	2007-12-20 03:16:29.169061806 +0100
@@ -648,7 +648,7 @@
      * it until we apply the proper mode, which might be a statoverride.
      */
     fd= open(fnamenewvb.buf, (O_CREAT|O_EXCL|O_WRONLY), 0);
-    if (fd < 0) ohshite(_("unable to create `%.255s'"),ti->Name);
+    if (fd < 0) ohshite(_("unable to create `%.255s' (while processing `%.255s')"),fnamenewvb.buf,ti->Name);
     push_cleanup(cu_closefd,ehflag_bombout, 0,0, 1,(void*)&fd);
     debug(dbg_eachfiledetail,"tarobject NormalFile[01] open size=%lu",
           (unsigned long)ti->Size);
