From 37a4874908069cc11872c1fb71aebc760ead27eb Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Fri, 1 Mar 2013 15:01:53 +0800
Subject: [PATCH 2/3] Fix the URLs for a couple of DPL elections.

---
 data/events/voting | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/data/events/voting b/data/events/voting
index 3835efe..bc6a5c0 100644
--- a/data/events/voting
+++ b/data/events/voting
@@ -163,12 +163,12 @@ Source: http://www.debian.org/vote/2009/vote_001
 
 Title: Steve McIntyre re-elected as DPL
 Date: Apr 11 2009
-Source: http://www.debian.org/vote/2008/vote_001
+Source: http://www.debian.org/vote/2009/vote_001
 
 Title: DPL 2010 elections
 Start-Date: Mar 05 2010
 End-Date: Apr 15 2010
-Source: http://www.debian.org/vote/2009/vote_001
+Source: http://www.debian.org/vote/2010/vote_001
 
 Title: Stefano Zacchiroli elected as DPL
 Date: Apr 15 2010
-- 
1.8.2.rc0

