From 3c482d1a0fe3baf192866867d5118ca06ba9cde4 Mon Sep 17 00:00:00 2001
From: Paul Wise <pabs@debian.org>
Date: Fri, 1 Mar 2013 15:02:18 +0800
Subject: [PATCH 3/3] Add some missing DPL elections.

---
 data/events/voting | 15 +++++++++++++++
 1 file changed, 15 insertions(+)

diff --git a/data/events/voting b/data/events/voting
index bc6a5c0..4c90504 100644
--- a/data/events/voting
+++ b/data/events/voting
@@ -178,10 +178,20 @@ Title: Debian welcomes non-packaging contributors
 Date: Oct 18 2010
 Source: http://www.debian.org/vote/2010/vote_002
 
+Title: DPL 2011 elections
+Start-Date: Mar 05 2011
+End-Date: Apr 15 2011
+Source: http://www.debian.org/vote/2011/vote_001
+
 Title: Stefano Zacchiroli re-elected as DPL
 Date: Apr 17 2011
 Source: http://www.debian.org/vote/2011/vote_001
 
+Title: DPL 2012 elections
+Start-Date: Mar 04 2010
+End-Date: Apr 14 2010
+Source: http://www.debian.org/vote/2012/vote_001
+
 Title: Stefano Zacchiroli re-elected as DPL
 Date: April 14 2012
 Source: http://www.debian.org/vote/2012/vote_001
@@ -189,3 +199,8 @@ Source: http://www.debian.org/vote/2012/vote_001
 Title: Diversity statement General Resolution
 Date: Jun 2 2012
 Source: http://www.debian.org/vote/2012/vote_002
+
+Title: DPL 2013 elections
+Start-Date: Mar 03 2013
+End-Date: Apr 13 2013
+Source: http://www.debian.org/vote/2013/vote_001
-- 
1.8.2.rc0

