--- bind9.dist	2011-09-09 17:16:47.000000000 +0200
+++ bind9	2011-09-09 17:18:14.000000000 +0200
@@ -89,10 +89,13 @@
 		    --pidfile ${PIDFILE} -- $OPTIONS
 	fi
 	if [ -n $pid ]; then
-	  while kill -0 $pid 2>/dev/null; do
-	    log_progress_msg "waiting for pid $pid to die"
-	    sleep 1
-	  done
+	    count=1
+	    while kill -0 $pid 2>/dev/null && [ $count -lt 60 ]; do
+		log_progress_msg "waiting for pid $pid to die"
+		sleep 1
+		count=$(($count+1))
+	    done
+	    [ $count -lt 60 ] || log_end_msg 1
 	fi
 	log_end_msg 0
     ;;
