--- audtool_main.0.c	2009-03-15 20:43:04.000000000 -0400
+++ audtool_main.c	2009-03-15 20:43:12.000000000 -0400
@@ -192,7 +192,7 @@
 			     !g_ascii_strcasecmp(g_strconcat("--", handlers[i].name, NULL), argv[j]))
 			    && g_ascii_strcasecmp("<sep>", handlers[i].name))
   			{
-				int numargs = handlers[i].args + 1 < argc - 1 ? handlers[i].args + 1 : argc - 1;
+				int numargs = handlers[i].args + 1 < argc - j ? handlers[i].args + 1 : argc - j;
 				handlers[i].handler(numargs, &argv[j]);
 				j += handlers[i].args;
 				k++;
