From 51d3341d9d6a0b7ac7dc9dedbbfb0dc8aa0a0664 Mon Sep 17 00:00:00 2001
From: Jeremy Bicha <jbicha@ubuntu.com>
Date: Sat, 12 Aug 2017 22:59:33 -0400
Subject: [PATCH 3/3] Adapt to python3-doit 0.30.3 to fix FTBFS

---
 debian/control | 2 +-
 debian/rules   | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/debian/control b/debian/control
index fe4e0476..257935bb 100644
--- a/debian/control
+++ b/debian/control
@@ -4,7 +4,7 @@ Priority: extra
 Maintainer: Debian Fonts Task Force <pkg-fonts-devel@lists.alioth.debian.org>
 Uploaders: Hideki Yamane <henrich@debian.org>
 Build-Depends: debhelper (>= 10), valac (>= 0.17.3),
- python3-doit (>= 0.20), libglib2.0-dev, libgtk2.0-dev,
+ python3-doit (>= 0.30.3), libglib2.0-dev, libgtk2.0-dev,
  libgee-0.8-dev, libnotify-dev, libwebkit2gtk-4.0-dev, libgit2-dev, libsqlite3-dev,
  libxmlbird-dev, python3-pkg-resources
 Standards-Version: 4.0.0
diff --git a/debian/rules b/debian/rules
index d203a750..21a9fbf2 100755
--- a/debian/rules
+++ b/debian/rules
@@ -13,7 +13,7 @@ package=birdfont
 
 override_dh_auto_configure:
 	$(CURDIR)/configure
-	doit3
+	doit
 
 override_dh_auto_install:
 	$(CURDIR)/install.py --dest=debian/$(package) -n /share/man/man1/ -l /lib/$(DEB_HOST_MULTIARCH)
-- 
2.11.0

