--- completions/useradd.orig	2012-11-08 00:40:00.866440036 +0000
+++ completions/useradd	2012-11-08 00:40:24.426529064 +0000
@@ -9,7 +9,7 @@
     #       with -u/--uid
 
     case $prev in
-        -c|--comment|-h|--help|-e|--expiredate|-f|--inactive|-k|--key|\
+        -c|--comment|-h|--help|-e|--expiredate|-f|--inactive|-K|--key|\
         -p|--password|-u|--uid|-Z|--selinux-user)
             return 0
             ;;
