--- scripts/uupdate.sh.orig	2010-10-21 20:37:03.000000000 +0900
+++ scripts/uupdate.sh	2010-10-31 19:17:29.612000002 +0900
@@ -685,6 +685,11 @@
 	# Otherwise, we put them into a new directory
 	mkdir $PACKAGE-$SNEW_VERSION
 	mv $TEMP_DIR/* $PACKAGE-$SNEW_VERSION
+
+	# move .* files if they exist
+	if [ `ls $TEMP_DIR/.[0-9a-zA-Z]* | wc -l` -ne 0 ]; then
+	  mv $TEMP_DIR/.[0-9a-zA-Z]* $PACKAGE-$SNEW_VERSION
+	fi
     fi
     rm -rf $TEMP_DIR
     cp -a $PACKAGE-$SNEW_VERSION $PACKAGE-$SNEW_VERSION.orig
