--- debian/control.orig	2009-12-06 13:37:07.000000000 +0100
+++ debian/control	2009-12-06 13:37:36.000000000 +0100
@@ -96,6 +96,7 @@
   - licensecheck: attempt to determine the license of source files
   - list-unreleased: searches for unreleased packages
   - manpage-alert: locate binaries without corresponding manpages [man-db]
+  - man-reportbug: manpages bugs and improvements reporting made easy
   - mass-bug: mass-file bug reports [bsd-mailx | mailx | mailutils]
   - mergechanges: merge .changes files from a package built on different
     architectures
