--- debian/control~	2009-12-06 13:37:36.000000000 +0100
+++ debian/control	2009-12-06 13:39:58.000000000 +0100
@@ -24,7 +24,7 @@
  sensible-utils
 Suggests: build-essential, cvs-buildpackage, devscripts-el, gnuplot,
  libfile-desktopentry-perl, libnet-smtp-ssl-perl (>= 1.01-2), mutt,
- svn-buildpackage, w3m
+ svn-buildpackage, w3m, reportbug
 Description: scripts to make the life of a Debian Package maintainer easier
  Contains the following scripts, dependencies/recommendations shown in
  brackets afterwards:
