--- README.orig	2009-12-06 13:35:08.000000000 +0100
+++ README	2009-12-06 13:34:47.000000000 +0100
@@ -170,6 +170,9 @@
 
 - manpage-alert: locate binaries without corresponding manpages. [man-db]
 
+- man-reportbug: manpages bugs and improvements reporting made easy (edit
+  the rendered manpage in a text-editor) [reportbug]
+
 - mass-bug: mass-file bug reports [bsd-mailx | mailx | mailutils]
 
 - mergechanges: merge .changes files from the same release but built
