From 8b194e2ce9850c40f75d8a79aa6fc952971710b7 Mon Sep 17 00:00:00 2001
From: Francois Marier <francois@debian.org>
Date: Sun, 30 Aug 2015 11:31:21 -0700
Subject: Use HTTPS when talking to elpa.gnu.org

diff --git a/lisp/emacs-lisp/package.el b/lisp/emacs-lisp/package.el
index 7c4f21f..445af4e 100644
--- a/lisp/emacs-lisp/package.el
+++ b/lisp/emacs-lisp/package.el
@@ -204,7 +204,7 @@ If VERSION is nil, the package is not loaded (it is \"disabled\")."
   :risky t
   :version "24.1")
 
-(defcustom package-archives '(("gnu" . "http://elpa.gnu.org/packages/"))
+(defcustom package-archives '(("gnu" . "https://elpa.gnu.org/packages/"))
   "An alist of archives from which to fetch.
 The default value points to the GNU Emacs package repository.
 
-- 
2.5.1

