From f4944b26d2023546858c5e4240f8d7570fa3c73a Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Vilius=20Panev=C4=97=C5=BEys?= <floss@pane.lt>
Date: Thu, 23 Jan 2025 19:30:59 +0200
Subject: [PATCH] Don't enable systemd timer by default

---
 debian/rules | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/debian/rules b/debian/rules
index 4dac9f0..b8e1d3d 100755
--- a/debian/rules
+++ b/debian/rules
@@ -11,3 +11,6 @@ override_dh_installdocs:
 override_dh_auto_install:
 	dh_auto_install
 	find debian/etckeeper -name '__pycache__' -print0 | xargs -0 -r rm -rv
+
+override_dh_installsystemd:
+	dh_installsystemd --no-enable --no-start etckeeper.timer
-- 
2.39.5

