diff -urN binutils-2.19.51.20090805/debian/rules binutils-2.19.51.20090805.cross/debian/rules
--- binutils-2.19.51.20090805/debian/rules	2009-08-13 18:02:54.000000000 +0200
+++ binutils-2.19.51.20090805.cross/debian/rules	2009-08-13 18:25:05.000000000 +0200
@@ -877,6 +877,22 @@
 ADDITIONAL_TARGETS = --enable-targets=s390-linux-gnu
 endif
 
+#-----------------------------------------------------------------
+# sysroot options
+ifdef WITH_SYSROOT
+  with_sysroot = $(WITH_SYSROOT)
+endif
+ifdef WITH_BUILD_SYSROOT
+  with_build_sysroot = $(WITH_BUILD_SYSROOT)
+endif
+
+ifneq ($(with_sysroot),)
+  CONFARGS += --with-sysroot=$(with_sysroot)
+endif
+ifneq ($(with_build_sysroot),)
+  CONFARGS += --with-build-sysroot=$(with_build_sysroot)
+endif
+
 configure-$(TARGET)-stamp: patch-stamp
 	$(checkdir)
 	test "" != "$(TARGET)"
@@ -885,7 +901,7 @@
 	cd builddir-$(TARGET) \
 	    && env CC="$(CC)" CXX="$(CXX)" ../configure --host=$(DEB_HOST_GNU_TYPE) \
 	        --build=$(DEB_BUILD_GNU_TYPE) --target=$(TARGET) --prefix=/usr \
-		$(ADDITIONAL_TARGETS)
+		$(ADDITIONAL_TARGETS) $(CONFARGS)
 	touch $@
 
 build-$(TARGET)-stamp: configure-$(TARGET)-stamp
