Index: debian/aide.conf.d/70_aide_dev
===================================================================
--- debian/aide.conf.d/70_aide_dev	(revision 762)
+++ debian/aide.conf.d/70_aide_dev	(working copy)
@@ -16,9 +16,10 @@
 
   # -path is -wholename on sid, but -path still works on sarge and sid
   find /dev -path /dev/.static -prune -o -print | \
-     sed -e 's/^\(.*\)/\1$ RamdiskData/' \
+     sed -e 's/^\(.*\)/\1$ RamdiskData/'   \
+         -e 's,\\,\\\\,g' -e 's,\.,\\\.,g' \
          -e 's,#,\\#,g'
-  echo "/dev/.static$ RamdiskData"
+  echo "/dev/\.static$ RamdiskData"
 else
   # we have static /dev
   cat <<EOF
