diff -Nru antlr3-3.5.2/debian/changelog antlr3-3.5.2/debian/changelog
--- antlr3-3.5.2/debian/changelog	2017-08-02 11:53:56.000000000 +0200
+++ antlr3-3.5.2/debian/changelog	2018-03-14 15:21:31.000000000 +0100
@@ -1,3 +1,11 @@
+antlr3 (3.5.2-9) UNRELEASED; urgency=medium
+
+  * Add generate-sources to docs goals to activate antlr before javadoc's
+    goals, otherwise javadoc:jar fails because the sourcepath is incomplete.
+    (Closes: #892760)
+
+ -- Tiago Stürmer Daitx <tiago.daitx@ubuntu.com>  Wed, 14 Mar 2018 14:21:31 +0000
+
 antlr3 (3.5.2-8) unstable; urgency=medium
 
   * Team upload.
diff -Nru antlr3-3.5.2/debian/rules antlr3-3.5.2/debian/rules
--- antlr3-3.5.2/debian/rules	2017-06-30 11:10:45.000000000 +0200
+++ antlr3-3.5.2/debian/rules	2018-03-14 15:20:48.000000000 +0100
@@ -8,7 +8,7 @@
 
 DEB_MAVEN_INSTALL_TO_USJ := false
 DEB_MAVEN_BUILD_TARGET := package install
-DEB_MAVEN_DOC_TARGET := javadoc:jar javadoc:aggregate
+DEB_MAVEN_DOC_TARGET := generate-sources javadoc:jar javadoc:aggregate
 PACKAGE              := $(DEB_SOURCE_PACKAGE)
 VERSION              := $(shell echo $(DEB_UPSTREAM_VERSION) | cut -d'+' -f1 -)
 JAVA_HOME            := /usr/lib/jvm/default-java
