--- a/easyrsa           2021-11-05 10:21:02.783260266 +0100
+++ b/easyrsa           2021-11-05 10:22:12.591259497 +0100
@@ -1664,9 +1664,8 @@
        vars=
 
        # set up program path
-       prog_file="$0"
+       prog_file="$(dirname $0)/$(basename $0)"
        prog_file2="$(which -- "$prog_file" 2>/dev/null)" && prog_file="$prog_file2"
-       prog_file2="$(readlink -f "$prog_file" 2>/dev/null)" && prog_file="$prog_file2"
        prog_dir="${prog_file%/*}"
        prog_vars="${prog_dir}/vars"
        # set up PKI path
