--- ax_python_devel.m4-s32.original	2022-10-01 17:42:46.000000000 +0200
+++ ax_python_devel.m4	2022-10-16 18:40:38.873380357 +0200
@@ -125,7 +125,7 @@
         return tuple(map(int, s.strip().replace("rc", ".").split(".")))
     def __init__(self):
         import sys
-        self.vpy = tuple(sys.version_info)
+        self.vpy = tuple(sys.version_info)[[:3]]
     def __eq__(self, s):
         return self.vpy == self.vtup(s)
     def __ne__(self, s):
