diff -aur exiv2-0.20.harden/debian/rules exiv2-0.20/debian/rules
--- exiv2-0.20.harden/debian/rules	2011-05-29 19:49:29.000000000 +0200
+++ exiv2-0.20/debian/rules	2012-01-18 15:57:33.020481301 +0100
@@ -4,7 +4,7 @@
 	dh $@ --parallel
 
 override_dh_auto_configure:
-	dh_auto_configure -- --disable-rpath
+	dh_auto_configure -- --disable-rpath $(shell dpkg-buildflags --export=configure)
 
 override_dh_clean:
 	dh_clean
Nur in exiv2-0.20/debian: rules~.
