diff --git a/bin/fai-make-nfsroot b/bin/fai-make-nfsroot
index 8aebd6d..a6874bc 100755
--- a/bin/fai-make-nfsroot
+++ b/bin/fai-make-nfsroot
@@ -443,6 +443,7 @@ upgrade_nfsroot() {
 
     mount -t proc   /proc  $NFSROOT/proc
     mount -t sysfs  /sys   $NFSROOT/sys
+    [ -d $NFSROOT/dev/pts ] || mkdir -p $NFSROOT/dev/pts
     mount -t devpts devpts $NFSROOT/dev/pts
     /usr/lib/fai/mkramdisk $NFSROOT/var/lib/dpkg $NFSROOT/var/cache
     mkdir $NFSROOT/etc/mdadm; touch $NFSROOT/etc/mdadm/mdadm.conf # stop mdadm from calling mkconf
