# DP: do not transliterate raw base64 bodies

--- a/pith/mailview.c
+++ b/pith/mailview.c
@@ -255,7 +255,7 @@ format_body(long int msgno, BODY *body,
 	    else
 	      charset = ps_global->display_charmap;
 
-	    if(strucmp(charset, "us-ascii") && strucmp(charset, "utf-8")){
+	    if(body->encoding != ENCBASE64 && strucmp(charset, "us-ascii") && strucmp(charset, "utf-8")){
 		/* transliterate message text to UTF-8 */
 		gf_link_filter(gf_utf8, gf_utf8_opt(charset));
 	    }
