Index: cdebootstrap-0.5.5/src/frontend/standalone/cdebootstrap.man
===================================================================
--- cdebootstrap-0.5.5.orig/src/frontend/standalone/cdebootstrap.man	2009-12-22 10:02:22.000000000 -0200
+++ cdebootstrap-0.5.5/src/frontend/standalone/cdebootstrap.man	2009-12-22 10:02:55.000000000 -0200
@@ -86,4 +86,4 @@
 All rc.d operations are disabled by a policy-rc.d script (cdebootstrap-helper-rc.d package).
 .TP
 \fBstandard\fR
-Installs required and important priority packages.
+Installs required and important priority packages.  This is the default flavour.
