diff --git a/src/debarchiver.pl b/src/debarchiver.pl
index efa3797..ce3cf0a 100755
--- a/src/debarchiver.pl
+++ b/src/debarchiver.pl
@@ -76,7 +76,7 @@ use OpaL::read qw(readfile readcommand);
 #  2009-03-16 Franck Joncourt <franck.mail@dthconnex.com>
 #       Added usermailcmd, mailsearch, usermailcmd and cmds. Removed mailcmd.
 
-$mailformat    = "sendmail";
+$mailformat    = "";
 @mailsearch    = ();
 $usermailcmd   = '';
 
@@ -2554,7 +2554,7 @@ sub  check_mailconfig()
     if ( ($mailformat ne "") && ($usermailcmd eq "") ) {
         pdebug(2, "The mailformat option must be used in conjunction to the mailcmd option.");
 
-    } elsif ($mailformat eq "sendmail") {
+    } elsif ( ($mailformat eq "sendmail") || ($mailformat eq "") ) {
         @mailsearch = ("sendmail", "mail");
 
     } elsif ($mailformat eq "mail") {
