From 895f3993022306da52c54994b9aa3add03ccb164 Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Lo=C3=AFc=20Minier?= <lool@debian.org>
Date: Mon, 17 Oct 2011 15:32:33 +0200
Subject: [PATCH] Drop debian-edu-artwork-usplash; usplash was RMed

---
 Packages-arch-specific |    1 -
 1 files changed, 0 insertions(+), 1 deletions(-)

diff --git a/Packages-arch-specific b/Packages-arch-specific
index 9358f9c..240d954 100644
--- a/Packages-arch-specific
+++ b/Packages-arch-specific
@@ -49,7 +49,6 @@ crash: amd64 i386 ia64 alpha powerpc				      # not yet ported to other platform
 %digitools: i386						      # [ANAIS]
 %drawterm: !hppa						      # No source support
 %drdsl: i386 amd64						      # [ANAIS]
-debian-edu-artwork-usplash: i386 amd64 powerpc sparc armel            # needs usplash
 dosemu: i386 amd64                                                    # Hardcoded i386 assembler
 e3: i386 kfreebsd-i386 amd64 kfreebsd-amd64			      # i386 assembly
 %eep24c: amd64 i386 kfreebsd-i386 kfreebsd-amd64 hurd-i386	      # [?] ANAIS, <sys/io.h>
-- 
1.7.5.4

