--- ../cdebootstrap-0.7.3/src/install.c	2016-05-14 17:21:23.678022968 +0100
+++ src/install.c	2016-05-14 19:51:30.675105075 +0100
@@ -285,8 +285,6 @@
   *c++ = "dpkg";
   *c++ = "--configure";
   *c++ = "-a";
-  if (force)
-    *c++ = "--force-all";
   if (!simulate_handler)
     *c++ = "--status-fd=3";
   *c = NULL;
@@ -312,7 +310,7 @@
   *c++ = "dpkg";
   *c++ = "--install";
   if (force)
-    *c++ = "--force-all";
+    *c++ = "--force-depends";
   if (!simulate_handler)
     *c++ = "--status-fd=3";
   *c = NULL;
@@ -326,7 +324,7 @@
 
   *c++ = "dpkg";
   *c++ = "--unpack";
-  *c++ = "--force-all";
+  *c++ = "--force-depends";
   if (!simulate_handler)
     *c++ = "--status-fd=3";
   *c = NULL;
