diff -Npru busybox-1.17.1.orig/debian/config/pkg/udeb busybox-1.17.1/debian/config/pkg/udeb
--- busybox-1.17.1.orig/debian/config/pkg/udeb	2010-11-09 05:37:10.000000000 -0500
+++ busybox-1.17.1/debian/config/pkg/udeb	2011-02-06 21:13:04.455899401 -0500
@@ -697,7 +697,7 @@ CONFIG_NC_EXTRA=y
 # CONFIG_FEATURE_PREFER_IPV4_ADDRESS is not set
 # CONFIG_VERBOSE_RESOLUTION_ERRORS is not set
 # CONFIG_ARP is not set
-# CONFIG_ARPING is not set
+CONFIG_ARPING=y
 # CONFIG_BRCTL is not set
 # CONFIG_FEATURE_BRCTL_FANCY is not set
 # CONFIG_FEATURE_BRCTL_SHOW is not set
