From: "Georgios M. Zarkadas" <gz@member.fsf.org>
Date: Sat, 9 Mar 2019 19:51:13 +0200
Subject: [PATCH 2/2] Added documentation of the new feature.

Signed-off-by: Georgios M. Zarkadas <gz@member.fsf.org>
---

diff --git a/apt-mirror b/apt-mirror
index fafd482..9d6f609 100755
--- a/apt-mirror
+++ b/apt-mirror
@@ -20,6 +20,7 @@ Main features:
  * It's fully pool compliant
  * It supports multithreaded downloading
  * It supports multiple architectures at the same time
+ * It supports partial mirroring (with optional package include filter)
  * It can automatically remove unneeded files
  * It works well on an overloaded Internet connection
  * It never produces an inconsistent mirror including while mirroring
@@ -80,6 +81,9 @@ deb https://example.com:443/debian stable main contrib non-free
 Source Mirroring:
 deb-src http://example.com/debian stable main contrib non-free
 
+Partial Mirroring: (optional regex to only include some packages)
+deb http://example.com/debian stable main contrib non-free [linux-source|linux-doc|linux-(image|config|kbuild|source|headers|doc)-.+|fetchmail]
+
 =head1 AUTHORS
 
 Dmitry N. Hramtsov E<lt>hdn@nsu.ruE<gt>
