--- duplicity	2014-06-10 13:24:51.000000000 +0200
+++ duplicity.now	2015-10-18 17:57:14.912781530 +0200
@@ -938,6 +938,8 @@
                    "manually purge the repository."))
 
     chainlist = col_stats.get_chains_older_than(globals.remove_time)
+    if globals.remove_all_inc_of_but_n_full_mode:
+        chainlist = [e for e in chainlist if len(e.incset_list) > 0]
     if not chainlist:
         log.Notice(_("No old backup sets found, nothing deleted."))
         return
