From 3d42c9bba7eb65ac5ff70a0a73919368caced533 Mon Sep 17 00:00:00 2001
From: Balint Reczey <balint@balintreczey.hu>
Date: Thu, 4 Sep 2014 16:31:29 +0200
Subject: [PATCH 1/2] Fix off-by-one error in command argv size calculation

---
 lib/dpkg/command.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/lib/dpkg/command.c b/lib/dpkg/command.c
index 0044d83..f5b08ea 100644
--- a/lib/dpkg/command.c
+++ b/lib/dpkg/command.c
@@ -78,7 +78,7 @@ static void
 command_grow_argv(struct command *cmd, int need)
 {
 	/* Check if we already have enough room. */
-	if ((cmd->argv_size - cmd->argc) >= need)
+	if ((cmd->argv_size - (cmd->argc + 1)) >= need)
 		return;
 
 	cmd->argv_size = (cmd->argv_size + need) * 2;
-- 
2.1.0

