diff -ur bluez-4.93_without_hid2hci/debian/bluez_agent.udev bluez-4.93/debian/bluez_agent.udev
--- bluez-4.93_without_hid2hci/debian/bluez_agent.udev	2011-05-30 20:11:37.727321967 +0200
+++ bluez-4.93/debian/bluez_agent.udev	2011-05-30 20:12:36.682466070 +0200
@@ -3,6 +3,6 @@
 
 (
 	. /lib/udev/hotplug.functions
-	wait_for_file /usr/sbin/hid2hci
-	exec /usr/sbin/hid2hci "$@"
+	wait_for_file /lib/udev/hid2hci
+	exec /lib/udev/hid2hci "$@"
 ) &
diff -ur bluez-4.93_without_hid2hci/debian/bluez.bluetooth.init bluez-4.93/debian/bluez.bluetooth.init
--- bluez-4.93_without_hid2hci/debian/bluez.bluetooth.init	2011-05-30 20:11:37.735321846 +0200
+++ bluez-4.93/debian/bluez.bluetooth.init	2011-05-30 20:12:36.682466070 +0200
@@ -26,7 +26,7 @@
 DAEMON=/usr/sbin/bluetoothd
 HCIATTACH=/usr/sbin/hciattach
 
-HID2HCI=/usr/sbin/hid2hci
+HID2HCI=/lib/udev/hid2hci
 HID2HCI_ENABLED=1
 HID2HCI_UNDO=1
 
diff -ur bluez-4.93_without_hid2hci/debian/bluez.install bluez-4.93/debian/bluez.install
--- bluez-4.93_without_hid2hci/debian/bluez.install	2011-05-30 20:11:37.743321725 +0200
+++ bluez-4.93/debian/bluez.install	2011-05-30 20:12:36.690465949 +0200
@@ -4,3 +4,4 @@
 debian/tmp/usr/sbin/*
 debian/tmp/usr/bin/*
 debian/tmp/usr/share/man/man*
+debian/tmp/usr/lib/bluez/udev/hid2hci /lib/udev
