Index: scripts/bts.pl
===================================================================
--- scripts/bts.pl	(revision 542)
+++ scripts/bts.pl	(working copy)
@@ -1100,9 +1100,9 @@
 Change the "owner" address of a bug, with `!' meaning
 `use the address on the current email as the new owner address'.
 
-The owner of a bug accepts the responsibility of dealing with it, and
-will receive all of the email corresponding to the bug instead of the
-usual maintainer.
+The owner of a bug accepts the responsibility of dealing with it. Note that the
+"owner" of a bug does not receive automatically all of the email corresponding
+to the bug; use "subscribe" to achieve that.
 
 =cut
 
