Index: build/boot/kfreebsd/grub-kfreebsd-i386.cfg
===================================================================
--- build/boot/kfreebsd/grub-kfreebsd-i386.cfg	(revision 60995)
+++ build/boot/kfreebsd/grub-kfreebsd-i386.cfg	(working copy)
@@ -17,12 +17,12 @@
 menuentry "Install Debian GNU/kFreeBSD 32 bits" {
 	echo "Loading ..."
 	set root=$cd
-	freebsd /boot/kernel/kfreebsd.gz
-	freebsd_module_elf /boot/kernel/acpi.ko
-	freebsd_module /boot/mfsroot.gz type=mfs_root
-	set FreeBSD.vfs.root.mountfrom=ufs:/dev/md0
-	set FreeBSD.hw.ata.ata_dma=0	# needed for qemu hard disk # TODO: delete
-	set FreeBSD.hw.ata.atapi_dma=0	# needed for qemu cd # TODO: 1
+	kfreebsd /boot/kernel/kfreebsd.gz
+	kfreebsd_module_elf /boot/kernel/acpi.ko
+	kfreebsd_module /boot/mfsroot.gz type=mfs_root
+	set kFreeBSD.vfs.root.mountfrom=ufs:/dev/md0
+	set kFreeBSD.hw.ata.ata_dma=0	# needed for qemu hard disk # TODO: delete
+	set kFreeBSD.hw.ata.atapi_dma=0	# needed for qemu cd # TODO: 1
 }
 
 menuentry "Boot from first hard disk" {
Index: build/boot/kfreebsd/grub-kfreebsd-amd64.cfg
===================================================================
--- build/boot/kfreebsd/grub-kfreebsd-amd64.cfg	(revision 60995)
+++ build/boot/kfreebsd/grub-kfreebsd-amd64.cfg	(working copy)
@@ -17,11 +17,11 @@
 menuentry "Install Debian GNU/kFreeBSD 64 bits" {
 	echo "Loading ..."
 	set root=$cd
-	freebsd /boot/kernel/kfreebsd.gz
-	freebsd_module /boot/mfsroot.gz type=mfs_root
-	set FreeBSD.vfs.root.mountfrom=ufs:/dev/md0
-	set FreeBSD.hw.ata.ata_dma=0	# needed for qemu hard disk # TODO: delete
-	set FreeBSD.hw.ata.atapi_dma=0	# needed for qemu cd # TODO: 1
+	kfreebsd /boot/kernel/kfreebsd.gz
+	kfreebsd_module /boot/mfsroot.gz type=mfs_root
+	set kFreeBSD.vfs.root.mountfrom=ufs:/dev/md0
+	set kFreeBSD.hw.ata.ata_dma=0	# needed for qemu hard disk # TODO: delete
+	set kFreeBSD.hw.ata.atapi_dma=0	# needed for qemu cd # TODO: 1
 }
 
 menuentry "Boot from first hard disk" {
